<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rsc="urn:rsc.org" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Review: a comprehensive summary of a decade development of the recombinase polymerase amplification  - Analyst (RSC Publishing) DOI:10.1039/C8AN01621F</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2019/an/c8an01621f"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":62,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};;/*! For license information please see nr-loader-spa-1.236.0.min.js.LICENSE.txt */
(()=>{"use strict";var e,t,r={5763:(e,t,r)=>{r.d(t,{P_:()=>l,Mt:()=>g,C5:()=>s,DL:()=>v,OP:()=>T,lF:()=>D,Yu:()=>y,Dg:()=>h,CX:()=>c,GE:()=>b,sU:()=>_});var n=r(8632),i=r(9567);const o={beacon:n.ce.beacon,errorBeacon:n.ce.errorBeacon,licenseKey:void 0,applicationID:void 0,sa:void 0,queueTime:void 0,applicationTime:void 0,ttGuid:void 0,user:void 0,account:void 0,product:void 0,extra:void 0,jsAttributes:{},userAttributes:void 0,atts:void 0,transactionName:void 0,tNamePlain:void 0},a={};function s(e){if(!e)throw new Error("All info objects require an agent identifier!");if(!a[e])throw new Error("Info for ".concat(e," was never set"));return a[e]}function c(e,t){if(!e)throw new Error("All info objects require an agent identifier!");a[e]=(0,i.D)(t,o),(0,n.Qy)(e,a[e],"info")}var u=r(7056);const d=()=>{const e={blockSelector:"[data-nr-block]",maskInputOptions:{password:!0}};return{allow_bfcache:!0,privacy:{cookies_enabled:!0},ajax:{deny_list:void 0,enabled:!0,harvestTimeSeconds:10},distributed_tracing:{enabled:void 0,exclude_newrelic_header:void 0,cors_use_newrelic_header:void 0,cors_use_tracecontext_headers:void 0,allowed_origins:void 0},session:{domain:void 0,expiresMs:u.oD,inactiveMs:u.Hb},ssl:void 0,obfuscate:void 0,jserrors:{enabled:!0,harvestTimeSeconds:10},metrics:{enabled:!0},page_action:{enabled:!0,harvestTimeSeconds:30},page_view_event:{enabled:!0},page_view_timing:{enabled:!0,harvestTimeSeconds:30,long_task:!1},session_trace:{enabled:!0,harvestTimeSeconds:10},harvest:{tooManyRequestsDelay:60},session_replay:{enabled:!1,harvestTimeSeconds:60,sampleRate:.1,errorSampleRate:.1,maskTextSelector:"*",maskAllInputs:!0,get blockClass(){return"nr-block"},get ignoreClass(){return"nr-ignore"},get maskTextClass(){return"nr-mask"},get blockSelector(){return e.blockSelector},set blockSelector(t){e.blockSelector+=",".concat(t)},get maskInputOptions(){return e.maskInputOptions},set maskInputOptions(t){e.maskInputOptions={...t,password:!0}}},spa:{enabled:!0,harvestTimeSeconds:10}}},f={};function l(e){if(!e)throw new Error("All configuration objects require an agent identifier!");if(!f[e])throw new Error("Configuration for ".concat(e," was never set"));return f[e]}function h(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");f[e]=(0,i.D)(t,d()),(0,n.Qy)(e,f[e],"config")}function g(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");var r=l(e);if(r){for(var n=t.split("."),i=0;i<n.length-1;i++)if("object"!=typeof(r=r[n[i]]))return;r=r[n[n.length-1]]}return r}const p={accountID:void 0,trustKey:void 0,agentID:void 0,licenseKey:void 0,applicationID:void 0,xpid:void 0},m={};function v(e){if(!e)throw new Error("All loader-config objects require an agent identifier!");if(!m[e])throw new Error("LoaderConfig for ".concat(e," was never set"));return m[e]}function b(e,t){if(!e)throw new Error("All loader-config objects require an agent identifier!");m[e]=(0,i.D)(t,p),(0,n.Qy)(e,m[e],"loader_config")}const y=(0,n.mF)().o;var w=r(385),x=r(6818);const A={buildEnv:x.Re,bytesSent:{},queryBytesSent:{},customTransaction:void 0,disabled:!1,distMethod:x.gF,isolatedBacklog:!1,loaderType:void 0,maxBytes:3e4,offset:Math.floor(w._A?.performance?.timeOrigin||w._A?.performance?.timing?.navigationStart||Date.now()),onerror:void 0,origin:""+w._A.location,ptid:void 0,releaseIds:{},session:void 0,xhrWrappable:"function"==typeof w._A.XMLHttpRequest?.prototype?.addEventListener,version:x.q4},E={};function T(e){if(!e)throw new Error("All runtime objects require an agent identifier!");if(!E[e])throw new Error("Runtime for ".concat(e," was never set"));return E[e]}function _(e,t){if(!e)throw new Error("All runtime objects require an agent identifier!");E[e]=(0,i.D)(t,A),(0,n.Qy)(e,E[e],"runtime")}function D(e){return function(e){try{const t=s(e);return!!t.licenseKey&&!!t.errorBeacon&&!!t.applicationID}catch(e){return!1}}(e)}},9567:(e,t,r)=>{r.d(t,{D:()=>i});var n=r(50);function i(e,t){try{if(!e||"object"!=typeof e)return(0,n.Z)("Setting a Configurable requires an object as input");if(!t||"object"!=typeof t)return(0,n.Z)("Setting a Configurable requires a model to set its initial properties");const r=Object.create(Object.getPrototypeOf(t),Object.getOwnPropertyDescriptors(t)),o=0===Object.keys(r).length?e:r;for(let a in o)if(void 0!==e[a])try{"object"==typeof e[a]&&"object"==typeof t[a]?r[a]=i(e[a],t[a]):r[a]=e[a]}catch(e){(0,n.Z)("An error occurred while setting a property of a Configurable",e)}return r}catch(e){(0,n.Z)("An error occured while setting a Configurable",e)}}},6818:(e,t,r)=>{r.d(t,{Re:()=>i,gF:()=>o,q4:()=>n});const n="1.236.0",i="PROD",o="CDN"},385:(e,t,r)=>{r.d(t,{FN:()=>a,IF:()=>u,Nk:()=>f,Tt:()=>s,_A:()=>o,il:()=>n,pL:()=>c,v6:()=>i,w1:()=>d});const n="undefined"!=typeof window&&!!window.document,i="undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self.navigator instanceof WorkerNavigator||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis.navigator instanceof WorkerNavigator),o=n?window:"undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis),a=""+o?.location,s=/iPad|iPhone|iPod/.test(navigator.userAgent),c=s&&"undefined"==typeof SharedWorker,u=(()=>{const e=navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);return Array.isArray(e)&&e.length>=2?+e[1]:0})(),d=Boolean(n&&window.document.documentMode),f=!!navigator.sendBeacon},1117:(e,t,r)=>{r.d(t,{w:()=>o});var n=r(50);const i={agentIdentifier:"",ee:void 0};class o{constructor(e){try{if("object"!=typeof e)return(0,n.Z)("shared context requires an object as input");this.sharedContext={},Object.assign(this.sharedContext,i),Object.entries(e).forEach((e=>{let[t,r]=e;Object.keys(i).includes(t)&&(this.sharedContext[t]=r)}))}catch(e){(0,n.Z)("An error occured while setting SharedContext",e)}}}},8e3:(e,t,r)=>{r.d(t,{L:()=>d,R:()=>c});var n=r(2177),i=r(1284),o=r(4322),a=r(3325);const s={};function c(e,t){const r={staged:!1,priority:a.p[t]||0};u(e),s[e].get(t)||s[e].set(t,r)}function u(e){e&&(s[e]||(s[e]=new Map))}function d(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"feature";if(u(e),!e||!s[e].get(t))return a(t);s[e].get(t).staged=!0;const r=[...s[e]];function a(t){const r=e?n.ee.get(e):n.ee,a=o.X.handlers;if(r.backlog&&a){var s=r.backlog[t],c=a[t];if(c){for(var u=0;s&&u<s.length;++u)f(s[u],c);(0,i.D)(c,(function(e,t){(0,i.D)(t,(function(t,r){r[0].on(e,r[1])}))}))}delete a[t],r.backlog[t]=null,r.emit("drain-"+t,[])}}r.every((e=>{let[t,r]=e;return r.staged}))&&(r.sort(((e,t)=>e[1].priority-t[1].priority)),r.forEach((e=>{let[t]=e;a(t)})))}function f(e,t){var r=e[1];(0,i.D)(t[r],(function(t,r){var n=e[0];if(r[0]===n){var i=r[1],o=e[3],a=e[2];i.apply(o,a)}}))}},2177:(e,t,r)=>{r.d(t,{c:()=>f,ee:()=>u});var n=r(8632),i=r(2210),o=r(1284),a=r(5763),s="nr@context";let c=(0,n.fP)();var u;function d(){}function f(e){return(0,i.X)(e,s,l)}function l(){return new d}function h(){u.aborted=!0,u.backlog={}}c.ee?u=c.ee:(u=function e(t,r){var n={},c={},f={},g=!1;try{g=16===r.length&&(0,a.OP)(r).isolatedBacklog}catch(e){}var p={on:b,addEventListener:b,removeEventListener:y,emit:v,get:x,listeners:w,context:m,buffer:A,abort:h,aborted:!1,isBuffering:E,debugId:r,backlog:g?{}:t&&"object"==typeof t.backlog?t.backlog:{}};return p;function m(e){return e&&e instanceof d?e:e?(0,i.X)(e,s,l):l()}function v(e,r,n,i,o){if(!1!==o&&(o=!0),!u.aborted||i){t&&o&&t.emit(e,r,n);for(var a=m(n),s=w(e),d=s.length,f=0;f<d;f++)s[f].apply(a,r);var l=T()[c[e]];return l&&l.push([p,e,r,a]),a}}function b(e,t){n[e]=w(e).concat(t)}function y(e,t){var r=n[e];if(r)for(var i=0;i<r.length;i++)r[i]===t&&r.splice(i,1)}function w(e){return n[e]||[]}function x(t){return f[t]=f[t]||e(p,t)}function A(e,t){var r=T();p.aborted||(0,o.D)(e,(function(e,n){t=t||"feature",c[n]=t,t in r||(r[t]=[])}))}function E(e){return!!T()[c[e]]}function T(){return p.backlog}}(void 0,"globalEE"),c.ee=u)},5546:(e,t,r)=>{r.d(t,{E:()=>n,p:()=>i});var n=r(2177).ee.get("handle");function i(e,t,r,i,o){o?(o.buffer([e],i),o.emit(e,t,r)):(n.buffer([e],i),n.emit(e,t,r))}},4322:(e,t,r)=>{r.d(t,{X:()=>o});var n=r(5546);o.on=a;var i=o.handlers={};function o(e,t,r,o){a(o||n.E,i,e,t,r)}function a(e,t,r,i,o){o||(o="feature"),e||(e=n.E);var a=t[o]=t[o]||{};(a[r]=a[r]||[]).push([e,i])}},3239:(e,t,r)=>{r.d(t,{bP:()=>s,iz:()=>c,m$:()=>a});var n=r(385);let i=!1,o=!1;try{const e={get passive(){return i=!0,!1},get signal(){return o=!0,!1}};n._A.addEventListener("test",null,e),n._A.removeEventListener("test",null,e)}catch(e){}function a(e,t){return i||o?{capture:!!e,passive:i,signal:t}:!!e}function s(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;window.addEventListener(e,t,a(r,n))}function c(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;document.addEventListener(e,t,a(r,n))}},4402:(e,t,r)=>{r.d(t,{Ht:()=>u,M:()=>c,Rl:()=>a,ky:()=>s});var n=r(385);const i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";function o(e,t){return e?15&e[t]:16*Math.random()|0}function a(){const e=n._A?.crypto||n._A?.msCrypto;let t,r=0;return e&&e.getRandomValues&&(t=e.getRandomValues(new Uint8Array(31))),i.split("").map((e=>"x"===e?o(t,++r).toString(16):"y"===e?(3&o()|8).toString(16):e)).join("")}function s(e){const t=n._A?.crypto||n._A?.msCrypto;let r,i=0;t&&t.getRandomValues&&(r=t.getRandomValues(new Uint8Array(31)));const a=[];for(var s=0;s<e;s++)a.push(o(r,++i).toString(16));return a.join("")}function c(){return s(16)}function u(){return s(32)}},7056:(e,t,r)=>{r.d(t,{Bq:()=>n,Hb:()=>o,oD:()=>i});const n="NRBA",i=144e5,o=18e5},7894:(e,t,r)=>{function n(){return Math.round(performance.now())}r.d(t,{z:()=>n})},7243:(e,t,r)=>{r.d(t,{e:()=>o});var n=r(385),i={};function o(e){if(e in i)return i[e];if(0===(e||"").indexOf("data:"))return{protocol:"data"};let t;var r=n._A?.location,o={};if(n.il)t=document.createElement("a"),t.href=e;else try{t=new URL(e,r.href)}catch(e){return o}o.port=t.port;var a=t.href.split("://");!o.port&&a[1]&&(o.port=a[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===a[0]?"443":"80"),o.hostname=t.hostname||r.hostname,o.pathname=t.pathname,o.protocol=a[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var s=!t.protocol||":"===t.protocol||t.protocol===r.protocol,c=t.hostname===r.hostname&&t.port===r.port;return o.sameOrigin=s&&(!t.hostname||c),"/"===o.pathname&&(i[e]=o),o}},50:(e,t,r)=>{function n(e,t){"function"==typeof console.warn&&(console.warn("New Relic: ".concat(e)),t&&console.warn(t))}r.d(t,{Z:()=>n})},2587:(e,t,r)=>{r.d(t,{N:()=>c,T:()=>u});var n=r(2177),i=r(5546),o=r(8e3),a=r(3325);const s={stn:[a.D.sessionTrace],err:[a.D.jserrors,a.D.metrics],ins:[a.D.pageAction],spa:[a.D.spa],sr:[a.D.sessionReplay,a.D.sessionTrace]};function c(e,t){const r=n.ee.get(t);e&&"object"==typeof e&&(Object.entries(e).forEach((e=>{let[t,n]=e;void 0===u[t]&&(s[t]?s[t].forEach((e=>{n?(0,i.p)("feat-"+t,[],void 0,e,r):(0,i.p)("block-"+t,[],void 0,e,r),(0,i.p)("rumresp-"+t,[Boolean(n)],void 0,e,r)})):n&&(0,i.p)("feat-"+t,[],void 0,void 0,r),u[t]=Boolean(n))})),Object.keys(s).forEach((e=>{void 0===u[e]&&(s[e]?.forEach((t=>(0,i.p)("rumresp-"+e,[!1],void 0,t,r))),u[e]=!1)})),(0,o.L)(t,a.D.pageViewEvent))}const u={}},2210:(e,t,r)=>{r.d(t,{X:()=>i});var n=Object.prototype.hasOwnProperty;function i(e,t,r){if(n.call(e,t))return e[t];var i=r();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,t,{value:i,writable:!0,enumerable:!1}),i}catch(e){}return e[t]=i,i}},1284:(e,t,r)=>{r.d(t,{D:()=>n});const n=(e,t)=>Object.entries(e||{}).map((e=>{let[r,n]=e;return t(r,n)}))},4351:(e,t,r)=>{r.d(t,{P:()=>o});var n=r(2177);const i=()=>{const e=new WeakSet;return(t,r)=>{if("object"==typeof r&&null!==r){if(e.has(r))return;e.add(r)}return r}};function o(e){try{return JSON.stringify(e,i())}catch(e){try{n.ee.emit("internal-error",[e])}catch(e){}}}},3960:(e,t,r)=>{r.d(t,{K:()=>a,b:()=>o});var n=r(3239);function i(){return"undefined"==typeof document||"complete"===document.readyState}function o(e,t){if(i())return e();(0,n.bP)("load",e,t)}function a(e){if(i())return e();(0,n.iz)("DOMContentLoaded",e)}},8632:(e,t,r)=>{r.d(t,{EZ:()=>u,Qy:()=>c,ce:()=>o,fP:()=>a,gG:()=>d,mF:()=>s});var n=r(7894),i=r(385);const o={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net"};function a(){return i._A.NREUM||(i._A.NREUM={}),void 0===i._A.newrelic&&(i._A.newrelic=i._A.NREUM),i._A.NREUM}function s(){let e=a();return e.o||(e.o={ST:i._A.setTimeout,SI:i._A.setImmediate,CT:i._A.clearTimeout,XHR:i._A.XMLHttpRequest,REQ:i._A.Request,EV:i._A.Event,PR:i._A.Promise,MO:i._A.MutationObserver,FETCH:i._A.fetch}),e}function c(e,t,r){let i=a();const o=i.initializedAgents||{},s=o[e]||{};return Object.keys(s).length||(s.initializedAt={ms:(0,n.z)(),date:new Date}),i.initializedAgents={...o,[e]:{...s,[r]:t}},i}function u(e,t){a()[e]=t}function d(){return function(){let e=a();const t=e.info||{};e.info={beacon:o.beacon,errorBeacon:o.errorBeacon,...t}}(),function(){let e=a();const t=e.init||{};e.init={...t}}(),s(),function(){let e=a();const t=e.loader_config||{};e.loader_config={...t}}(),a()}},7956:(e,t,r)=>{r.d(t,{N:()=>i});var n=r(3239);function i(e){let t=arguments.length>1&&void 0!==arguments[1]&&arguments[1],r=arguments.length>2?arguments[2]:void 0,i=arguments.length>3?arguments[3]:void 0;return void(0,n.iz)("visibilitychange",(function(){if(t)return void("hidden"==document.visibilityState&&e());e(document.visibilityState)}),r,i)}},1214:(e,t,r)=>{r.d(t,{em:()=>v,u5:()=>N,QU:()=>S,_L:()=>I,Gm:()=>L,Lg:()=>M,gy:()=>U,BV:()=>Q,Kf:()=>ee});var n=r(2177);const i="nr@original";var o=Object.prototype.hasOwnProperty,a=!1;function s(e,t){return e||(e=n.ee),r.inPlace=function(e,t,n,i,o){n||(n="");var a,s,c,u="-"===n.charAt(0);for(c=0;c<t.length;c++)d(a=e[s=t[c]])||(e[s]=r(a,u?s+n:n,i,s,o))},r.flag=i,r;function r(t,r,n,a,s){return d(t)?t:(r||(r=""),nrWrapper[i]=t,u(t,nrWrapper,e),nrWrapper);function nrWrapper(){var i,u,d,f;try{u=this,i=[...arguments],d="function"==typeof n?n(i,u):n||{}}catch(t){c([t,"",[i,u,a],d],e)}o(r+"start",[i,u,a],d,s);try{return f=t.apply(u,i)}catch(e){throw o(r+"err",[i,u,e],d,s),e}finally{o(r+"end",[i,u,f],d,s)}}}function o(r,n,i,o){if(!a||t){var s=a;a=!0;try{e.emit(r,n,i,t,o)}catch(t){c([t,r,n,i],e)}a=s}}}function c(e,t){t||(t=n.ee);try{t.emit("internal-error",e)}catch(e){}}function u(e,t,r){if(Object.defineProperty&&Object.keys)try{return Object.keys(e).forEach((function(r){Object.defineProperty(t,r,{get:function(){return e[r]},set:function(t){return e[r]=t,t}})})),t}catch(e){c([e],r)}for(var n in e)o.call(e,n)&&(t[n]=e[n]);return t}function d(e){return!(e&&e instanceof Function&&e.apply&&!e[i])}var f=r(2210),l=r(385);const h={},g=XMLHttpRequest,p="addEventListener",m="removeEventListener";function v(e){var t=function(e){return(e||n.ee).get("events")}(e);if(h[t.debugId]++)return t;h[t.debugId]=1;var r=s(t,!0);function i(e){r.inPlace(e,[p,m],"-",o)}function o(e,t){return e[1]}return"getPrototypeOf"in Object&&(l.il&&b(document,i),b(l._A,i),b(g.prototype,i)),t.on(p+"-start",(function(e,t){var n=e[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var i=(0,f.X)(n,"nr@wrapped",(function(){var e={object:function(){if("function"!=typeof n.handleEvent)return;return n.handleEvent.apply(n,arguments)},function:n}[typeof n];return e?r(e,"fn-",null,e.name||"anonymous"):n}));this.wrapped=e[1]=i}})),t.on(m+"-start",(function(e){e[1]=this.wrapped||e[1]})),t}function b(e,t){let r=e;for(;"object"==typeof r&&!Object.prototype.hasOwnProperty.call(r,p);)r=Object.getPrototypeOf(r);for(var n=arguments.length,i=new Array(n>2?n-2:0),o=2;o<n;o++)i[o-2]=arguments[o];r&&t(r,...i)}var y="fetch-",w=y+"body-",x=["arrayBuffer","blob","json","text","formData"],A=l._A.Request,E=l._A.Response,T="prototype",_="nr@context";const D={};function N(e){const t=function(e){return(e||n.ee).get("fetch")}(e);if(!(A&&E&&l._A.fetch))return t;if(D[t.debugId]++)return t;function r(e,r,n){var i=e[r];"function"==typeof i&&(e[r]=function(){var e,r=[...arguments],o={};t.emit(n+"before-start",[r],o),o[_]&&o[_].dt&&(e=o[_].dt);var a=i.apply(this,r);return t.emit(n+"start",[r,e],a),a.then((function(e){return t.emit(n+"end",[null,e],a),e}),(function(e){throw t.emit(n+"end",[e],a),e}))})}return D[t.debugId]=1,x.forEach((e=>{r(A[T],e,w),r(E[T],e,w)})),r(l._A,"fetch",y),t.on(y+"end",(function(e,r){var n=this;if(r){var i=r.headers.get("content-length");null!==i&&(n.rxSize=i),t.emit(y+"done",[null,r],n)}else t.emit(y+"done",[e],n)})),t}const O={},j=["pushState","replaceState"];function S(e){const t=function(e){return(e||n.ee).get("history")}(e);return!l.il||O[t.debugId]++||(O[t.debugId]=1,s(t).inPlace(window.history,j,"-")),t}var P=r(3239);const C={},R=["appendChild","insertBefore","replaceChild"];function I(e){const t=function(e){return(e||n.ee).get("jsonp")}(e);if(!l.il||C[t.debugId])return t;C[t.debugId]=!0;var r=s(t),i=/[?&](?:callback|cb)=([^&#]+)/,o=/(.*)\.([^.]+)/,a=/^(\w+)(\.|$)(.*)$/;function c(e,t){var r=e.match(a),n=r[1],i=r[3];return i?c(i,t[n]):t[n]}return r.inPlace(Node.prototype,R,"dom-"),t.on("dom-start",(function(e){!function(e){if(!e||"string"!=typeof e.nodeName||"script"!==e.nodeName.toLowerCase())return;if("function"!=typeof e.addEventListener)return;var n=(a=e.src,s=a.match(i),s?s[1]:null);var a,s;if(!n)return;var u=function(e){var t=e.match(o);if(t&&t.length>=3)return{key:t[2],parent:c(t[1],window)};return{key:e,parent:window}}(n);if("function"!=typeof u.parent[u.key])return;var d={};function f(){t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}function l(){t.emit("jsonp-error",[],d),t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}r.inPlace(u.parent,[u.key],"cb-",d),e.addEventListener("load",f,(0,P.m$)(!1)),e.addEventListener("error",l,(0,P.m$)(!1)),t.emit("new-jsonp",[e.src],d)}(e[0])})),t}var k=r(5763);const H={};function L(e){const t=function(e){return(e||n.ee).get("mutation")}(e);if(!l.il||H[t.debugId])return t;H[t.debugId]=!0;var r=s(t),i=k.Yu.MO;return i&&(window.MutationObserver=function(e){return this instanceof i?new i(r(e,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype),t}const z={};function M(e){const t=function(e){return(e||n.ee).get("promise")}(e);if(z[t.debugId])return t;z[t.debugId]=!0;var r=n.c,o=s(t),a=k.Yu.PR;return a&&function(){function e(r){var n=t.context(),i=o(r,"executor-",n,null,!1);const s=Reflect.construct(a,[i],e);return t.context(s).getCtx=function(){return n},s}l._A.Promise=e,Object.defineProperty(e,"name",{value:"Promise"}),e.toString=function(){return a.toString()},Object.setPrototypeOf(e,a),["all","race"].forEach((function(r){const n=a[r];e[r]=function(e){let i=!1;[...e||[]].forEach((e=>{this.resolve(e).then(a("all"===r),a(!1))}));const o=n.apply(this,arguments);return o;function a(e){return function(){t.emit("propagate",[null,!i],o,!1,!1),i=i||!e}}}})),["resolve","reject"].forEach((function(r){const n=a[r];e[r]=function(e){const r=n.apply(this,arguments);return e!==r&&t.emit("propagate",[e,!0],r,!1,!1),r}})),e.prototype=a.prototype;const n=a.prototype.then;a.prototype.then=function(){var e=this,i=r(e);i.promise=e;for(var a=arguments.length,s=new Array(a),c=0;c<a;c++)s[c]=arguments[c];s[0]=o(s[0],"cb-",i,null,!1),s[1]=o(s[1],"cb-",i,null,!1);const u=n.apply(this,s);return i.nextPromise=u,t.emit("propagate",[e,!0],u,!1,!1),u},a.prototype.then[i]=n,t.on("executor-start",(function(e){e[0]=o(e[0],"resolve-",this,null,!1),e[1]=o(e[1],"resolve-",this,null,!1)})),t.on("executor-err",(function(e,t,r){e[1](r)})),t.on("cb-end",(function(e,r,n){t.emit("propagate",[n,!0],this.nextPromise,!1,!1)})),t.on("propagate",(function(e,r,n){this.getCtx&&!r||(this.getCtx=function(){if(e instanceof Promise)var r=t.context(e);return r&&r.getCtx?r.getCtx():this})}))}(),t}const B={},F="requestAnimationFrame";function U(e){const t=function(e){return(e||n.ee).get("raf")}(e);if(!l.il||B[t.debugId]++)return t;B[t.debugId]=1;var r=s(t);return r.inPlace(window,[F],"raf-"),t.on("raf-start",(function(e){e[0]=r(e[0],"fn-")})),t}const q={},G="setTimeout",V="setInterval",X="clearTimeout",W="-start",Z="-",$=[G,"setImmediate",V,X,"clearImmediate"];function Q(e){const t=function(e){return(e||n.ee).get("timer")}(e);if(q[t.debugId]++)return t;q[t.debugId]=1;var r=s(t);return r.inPlace(l._A,$.slice(0,2),G+Z),r.inPlace(l._A,$.slice(2,3),V+Z),r.inPlace(l._A,$.slice(3),X+Z),t.on(V+W,(function(e,t,n){e[0]=r(e[0],"fn-",null,n)})),t.on(G+W,(function(e,t,n){this.method=n,this.timerDuration=isNaN(e[1])?0:+e[1],e[0]=r(e[0],"fn-",this,n)})),t}var Y=r(50);const K={},J=["open","send"];function ee(e){var t=e||n.ee;const r=function(e){return(e||n.ee).get("xhr")}(t);if(K[r.debugId]++)return r;K[r.debugId]=1,v(t);var i=s(r),o=k.Yu.XHR,a=k.Yu.MO,c=k.Yu.PR,u=k.Yu.SI,d="readystatechange",f=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],h=[],g=l._A.XMLHttpRequest.listeners,p=l._A.XMLHttpRequest=function(e){var t=new o(e);function n(){try{r.emit("new-xhr",[t],t),t.addEventListener(d,b,(0,P.m$)(!1))}catch(e){(0,Y.Z)("An error occured while intercepting XHR",e);try{r.emit("internal-error",[e])}catch(e){}}}return this.listeners=g?[...g,n]:[n],this.listeners.forEach((e=>e())),t};function m(e,t){i.inPlace(t,["onreadystatechange"],"fn-",E)}function b(){var e=this,t=r.context(e);e.readyState>3&&!t.resolved&&(t.resolved=!0,r.emit("xhr-resolved",[],e)),i.inPlace(e,f,"fn-",E)}if(function(e,t){for(var r in e)t[r]=e[r]}(o,p),p.prototype=o.prototype,i.inPlace(p.prototype,J,"-xhr-",E),r.on("send-xhr-start",(function(e,t){m(e,t),function(e){h.push(e),a&&(y?y.then(A):u?u(A):(w=-w,x.data=w))}(t)})),r.on("open-xhr-start",m),a){var y=c&&c.resolve();if(!u&&!c){var w=1,x=document.createTextNode(w);new a(A).observe(x,{characterData:!0})}}else t.on("fn-end",(function(e){e[0]&&e[0].type===d||A()}));function A(){for(var e=0;e<h.length;e++)m(0,h[e]);h.length&&(h=[])}function E(e,t){return t}return r}},7825:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.ajax},6660:(e,t,r)=>{r.d(t,{A:()=>i,t:()=>n});const n=r(3325).D.jserrors,i="nr@seenError"},3081:(e,t,r)=>{r.d(t,{gF:()=>o,mY:()=>i,t9:()=>n,vz:()=>s,xS:()=>a});const n=r(3325).D.metrics,i="sm",o="cm",a="storeSupportabilityMetrics",s="storeEventMetrics"},4649:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageAction},7633:(e,t,r)=>{r.d(t,{Dz:()=>i,OJ:()=>a,qw:()=>o,t9:()=>n});const n=r(3325).D.pageViewEvent,i="firstbyte",o="domcontent",a="windowload"},9251:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageViewTiming},3614:(e,t,r)=>{r.d(t,{BST_RESOURCE:()=>i,END:()=>s,FEATURE_NAME:()=>n,FN_END:()=>u,FN_START:()=>c,PUSH_STATE:()=>d,RESOURCE:()=>o,START:()=>a});const n=r(3325).D.sessionTrace,i="bstResource",o="resource",a="-start",s="-end",c="fn"+a,u="fn"+s,d="pushState"},7836:(e,t,r)=>{r.d(t,{BODY:()=>A,CB_END:()=>E,CB_START:()=>u,END:()=>x,FEATURE_NAME:()=>i,FETCH:()=>_,FETCH_BODY:()=>v,FETCH_DONE:()=>m,FETCH_START:()=>p,FN_END:()=>c,FN_START:()=>s,INTERACTION:()=>l,INTERACTION_API:()=>d,INTERACTION_EVENTS:()=>o,JSONP_END:()=>b,JSONP_NODE:()=>g,JS_TIME:()=>T,MAX_TIMER_BUDGET:()=>a,REMAINING:()=>f,SPA_NODE:()=>h,START:()=>w,originalSetTimeout:()=>y});var n=r(5763);const i=r(3325).D.spa,o=["click","submit","keypress","keydown","keyup","change"],a=999,s="fn-start",c="fn-end",u="cb-start",d="api-ixn-",f="remaining",l="interaction",h="spaNode",g="jsonpNode",p="fetch-start",m="fetch-done",v="fetch-body-",b="jsonp-end",y=n.Yu.ST,w="-start",x="-end",A="-body",E="cb"+x,T="jsTime",_="fetch"},5938:(e,t,r)=>{r.d(t,{W:()=>o});var n=r(5763),i=r(2177);class o{constructor(e,t,r){this.agentIdentifier=e,this.aggregator=t,this.ee=i.ee.get(e,(0,n.OP)(this.agentIdentifier).isolatedBacklog),this.featureName=r,this.blocked=!1}}},9144:(e,t,r)=>{r.d(t,{j:()=>m});var n=r(3325),i=r(5763),o=r(5546),a=r(2177),s=r(7894),c=r(8e3),u=r(3960),d=r(385),f=r(50),l=r(3081),h=r(8632);function g(){const e=(0,h.gG)();["setErrorHandler","finished","addToTrace","inlineHit","addRelease","addPageAction","setCurrentRouteName","setPageViewName","setCustomAttribute","interaction","noticeError","setUserId"].forEach((t=>{e[t]=function(){for(var r=arguments.length,n=new Array(r),i=0;i<r;i++)n[i]=arguments[i];return function(t){for(var r=arguments.length,n=new Array(r>1?r-1:0),i=1;i<r;i++)n[i-1]=arguments[i];let o=[];return Object.values(e.initializedAgents).forEach((e=>{e.exposed&&e.api[t]&&o.push(e.api[t](...n))})),o.length>1?o:o[0]}(t,...n)}}))}var p=r(2587);function m(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},m=arguments.length>2?arguments[2]:void 0,v=arguments.length>3?arguments[3]:void 0,{init:b,info:y,loader_config:w,runtime:x={loaderType:m},exposed:A=!0}=t;const E=(0,h.gG)();y||(b=E.init,y=E.info,w=E.loader_config),(0,i.Dg)(e,b||{}),(0,i.GE)(e,w||{}),(0,i.sU)(e,x),y.jsAttributes??={},d.v6&&(y.jsAttributes.isWorker=!0),(0,i.CX)(e,y),g();const T=function(e,t){t||(0,c.R)(e,"api");const h={};var g=a.ee.get(e),p=g.get("tracer"),m="api-",v=m+"ixn-";function b(t,r,n,o){const a=(0,i.C5)(e);return null===r?delete a.jsAttributes[t]:(0,i.CX)(e,{...a,jsAttributes:{...a.jsAttributes,[t]:r}}),x(m,n,!0,o||null===r?"session":void 0)(t,r)}function y(){}["setErrorHandler","finished","addToTrace","inlineHit","addRelease"].forEach((e=>h[e]=x(m,e,!0,"api"))),h.addPageAction=x(m,"addPageAction",!0,n.D.pageAction),h.setCurrentRouteName=x(m,"routeName",!0,n.D.spa),h.setPageViewName=function(t,r){if("string"==typeof t)return"/"!==t.charAt(0)&&(t="/"+t),(0,i.OP)(e).customTransaction=(r||"http://custom.transaction")+t,x(m,"setPageViewName",!0)()},h.setCustomAttribute=function(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2];if("string"==typeof e){if(["string","number"].includes(typeof t)||null===t)return b(e,t,"setCustomAttribute",r);(0,f.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t,"> was provided."))}else(0,f.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.setUserId=function(e){if("string"==typeof e||null===e)return b("enduser.id",e,"setUserId",!0);(0,f.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.interaction=function(){return(new y).get()};var w=y.prototype={createTracer:function(e,t){var r={},i=this,a="function"==typeof t;return(0,o.p)(v+"tracer",[(0,s.z)(),e,r],i,n.D.spa,g),function(){if(p.emit((a?"":"no-")+"fn-start",[(0,s.z)(),i,a],r),a)try{return t.apply(this,arguments)}catch(e){throw p.emit("fn-err",[arguments,this,"string"==typeof e?new Error(e):e],r),e}finally{p.emit("fn-end",[(0,s.z)()],r)}}}};function x(e,t,r,i){return function(){return(0,o.p)(l.xS,["API/"+t+"/called"],void 0,n.D.metrics,g),i&&(0,o.p)(e+t,[(0,s.z)(),...arguments],r?null:this,i,g),r?void 0:this}}function A(){r.e(439).then(r.bind(r,7438)).then((t=>{let{setAPI:r}=t;r(e),(0,c.L)(e,"api")})).catch((()=>(0,f.Z)("Downloading runtime APIs failed...")))}return["actionText","setName","setAttribute","save","ignore","onEnd","getContext","end","get"].forEach((e=>{w[e]=x(v,e,void 0,n.D.spa)})),h.noticeError=function(e,t){"string"==typeof e&&(e=new Error(e)),(0,o.p)(l.xS,["API/noticeError/called"],void 0,n.D.metrics,g),(0,o.p)("err",[e,(0,s.z)(),!1,t],void 0,n.D.jserrors,g)},d.il?(0,u.b)((()=>A()),!0):A(),h}(e,v);return(0,h.Qy)(e,T,"api"),(0,h.Qy)(e,A,"exposed"),(0,h.EZ)("activatedFeatures",p.T),T}},3325:(e,t,r)=>{r.d(t,{D:()=>n,p:()=>i});const n={ajax:"ajax",jserrors:"jserrors",metrics:"metrics",pageAction:"page_action",pageViewEvent:"page_view_event",pageViewTiming:"page_view_timing",sessionReplay:"session_replay",sessionTrace:"session_trace",spa:"spa"},i={[n.pageViewEvent]:1,[n.pageViewTiming]:2,[n.metrics]:3,[n.jserrors]:4,[n.ajax]:5,[n.sessionTrace]:6,[n.pageAction]:7,[n.spa]:8,[n.sessionReplay]:9}}},n={};function i(e){var t=n[e];if(void 0!==t)return t.exports;var o=n[e]={exports:{}};return r[e](o,o.exports,i),o.exports}i.m=r,i.d=(e,t)=>{for(var r in t)i.o(t,r)&&!i.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},i.f={},i.e=e=>Promise.all(Object.keys(i.f).reduce(((t,r)=>(i.f[r](e,t),t)),[])),i.u=e=>(({78:"page_action-aggregate",147:"metrics-aggregate",242:"session-manager",317:"jserrors-aggregate",348:"page_view_timing-aggregate",412:"lazy-feature-loader",439:"async-api",538:"recorder",590:"session_replay-aggregate",675:"compressor",733:"session_trace-aggregate",786:"page_view_event-aggregate",873:"spa-aggregate",898:"ajax-aggregate"}[e]||e)+"."+{78:"ac76d497",147:"3dc53903",148:"1a20d5fe",242:"2a64278a",317:"49e41428",348:"bd6de33a",412:"2f55ce66",439:"30bd804e",538:"1b18459f",590:"cf0efb30",675:"ae9f91a8",733:"83105561",786:"06482edd",860:"03a8b7a5",873:"e6b09d52",898:"998ef92b"}[e]+"-1.236.0.min.js"),i.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),e={},t="NRBA:",i.l=(r,n,o,a)=>{if(e[r])e[r].push(n);else{var s,c;if(void 0!==o)for(var u=document.getElementsByTagName("script"),d=0;d<u.length;d++){var f=u[d];if(f.getAttribute("src")==r||f.getAttribute("data-webpack")==t+o){s=f;break}}s||(c=!0,(s=document.createElement("script")).charset="utf-8",s.timeout=120,i.nc&&s.setAttribute("nonce",i.nc),s.setAttribute("data-webpack",t+o),s.src=r),e[r]=[n];var l=(t,n)=>{s.onerror=s.onload=null,clearTimeout(h);var i=e[r];if(delete e[r],s.parentNode&&s.parentNode.removeChild(s),i&&i.forEach((e=>e(n))),t)return t(n)},h=setTimeout(l.bind(null,void 0,{type:"timeout",target:s}),12e4);s.onerror=l.bind(null,s.onerror),s.onload=l.bind(null,s.onload),c&&document.head.appendChild(s)}},i.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},i.j=364,i.p="https://js-agent.newrelic.com/",(()=>{var e={364:0,953:0};i.f.j=(t,r)=>{var n=i.o(e,t)?e[t]:void 0;if(0!==n)if(n)r.push(n[2]);else{var o=new Promise(((r,i)=>n=e[t]=[r,i]));r.push(n[2]=o);var a=i.p+i.u(t),s=new Error;i.l(a,(r=>{if(i.o(e,t)&&(0!==(n=e[t])&&(e[t]=void 0),n)){var o=r&&("load"===r.type?"missing":r.type),a=r&&r.target&&r.target.src;s.message="Loading chunk "+t+" failed.\n("+o+": "+a+")",s.name="ChunkLoadError",s.type=o,s.request=a,n[1](s)}}),"chunk-"+t,t)}};var t=(t,r)=>{var n,o,[a,s,c]=r,u=0;if(a.some((t=>0!==e[t]))){for(n in s)i.o(s,n)&&(i.m[n]=s[n]);if(c)c(i)}for(t&&t(r);u<a.length;u++)o=a[u],i.o(e,o)&&e[o]&&e[o][0](),e[o]=0},r=window.webpackChunkNRBA=window.webpackChunkNRBA||[];r.forEach(t.bind(null,0)),r.push=t.bind(null,r.push.bind(r))})();var o={};(()=>{i.r(o);var e=i(3325),t=i(5763);const r=Object.values(e.D);function n(e){const n={};return r.forEach((r=>{n[r]=function(e,r){return!1!==(0,t.Mt)(r,"".concat(e,".enabled"))}(r,e)})),n}var a=i(9144);var s=i(5546),c=i(385),u=i(8e3),d=i(5938),f=i(3960),l=i(50);class h extends d.W{constructor(e,t,r){let n=!(arguments.length>3&&void 0!==arguments[3])||arguments[3];super(e,t,r),this.auto=n,this.abortHandler,this.featAggregate,this.onAggregateImported,n&&(0,u.R)(e,r)}importAggregator(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};if(this.featAggregate||!this.auto)return;const r=c.il&&!0===(0,t.Mt)(this.agentIdentifier,"privacy.cookies_enabled");let n;this.onAggregateImported=new Promise((e=>{n=e}));const o=async()=>{let t;try{if(r){const{setupAgentSession:e}=await Promise.all([i.e(860),i.e(242)]).then(i.bind(i,3228));t=e(this.agentIdentifier)}}catch(e){(0,l.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.",e)}try{if(!this.shouldImportAgg(this.featureName,t))return void(0,u.L)(this.agentIdentifier,this.featureName);const{lazyFeatureLoader:r}=await i.e(412).then(i.bind(i,8582)),{Aggregate:o}=await r(this.featureName,"aggregate");this.featAggregate=new o(this.agentIdentifier,this.aggregator,e),n(!0)}catch(e){(0,l.Z)("Downloading and initializing ".concat(this.featureName," failed..."),e),this.abortHandler?.(),n(!1)}};c.il?(0,f.b)((()=>o()),!0):o()}shouldImportAgg(r,n){return r!==e.D.sessionReplay||!1!==(0,t.Mt)(this.agentIdentifier,"session_trace.enabled")&&(!!n?.isNew||!!n?.state.sessionReplay)}}var g=i(7633),p=i(7894);class m extends h{static featureName=g.t9;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];if(super(r,n,g.t9,i),("undefined"==typeof PerformanceNavigationTiming||c.Tt)&&"undefined"!=typeof PerformanceTiming){const n=(0,t.OP)(r);n[g.Dz]=Math.max(Date.now()-n.offset,0),(0,f.K)((()=>n[g.qw]=Math.max((0,p.z)()-n[g.Dz],0))),(0,f.b)((()=>{const t=(0,p.z)();n[g.OJ]=Math.max(t-n[g.Dz],0),(0,s.p)("timing",["load",t],void 0,e.D.pageViewTiming,this.ee)}))}this.importAggregator()}}var v=i(1117),b=i(1284);class y extends v.w{constructor(e){super(e),this.aggregatedData={}}store(e,t,r,n,i){var o=this.getBucket(e,t,r,i);return o.metrics=function(e,t){t||(t={count:0});return t.count+=1,(0,b.D)(e,(function(e,r){t[e]=w(r,t[e])})),t}(n,o.metrics),o}merge(e,t,r,n,i){var o=this.getBucket(e,t,n,i);if(o.metrics){var a=o.metrics;a.count+=r.count,(0,b.D)(r,(function(e,t){if("count"!==e){var n=a[e],i=r[e];i&&!i.c?a[e]=w(i.t,n):a[e]=function(e,t){if(!t)return e;t.c||(t=x(t.t));return t.min=Math.min(e.min,t.min),t.max=Math.max(e.max,t.max),t.t+=e.t,t.sos+=e.sos,t.c+=e.c,t}(i,a[e])}}))}else o.metrics=r}storeMetric(e,t,r,n){var i=this.getBucket(e,t,r);return i.stats=w(n,i.stats),i}getBucket(e,t,r,n){this.aggregatedData[e]||(this.aggregatedData[e]={});var i=this.aggregatedData[e][t];return i||(i=this.aggregatedData[e][t]={params:r||{}},n&&(i.custom=n)),i}get(e,t){return t?this.aggregatedData[e]&&this.aggregatedData[e][t]:this.aggregatedData[e]}take(e){for(var t={},r="",n=!1,i=0;i<e.length;i++)t[r=e[i]]=A(this.aggregatedData[r]),t[r].length&&(n=!0),delete this.aggregatedData[r];return n?t:null}}function w(e,t){return null==e?function(e){e?e.c++:e={c:1};return e}(t):t?(t.c||(t=x(t.t)),t.c+=1,t.t+=e,t.sos+=e*e,e>t.max&&(t.max=e),e<t.min&&(t.min=e),t):{t:e}}function x(e){return{t:e,min:e,max:e,sos:e*e,c:1}}function A(e){return"object"!=typeof e?[]:(0,b.D)(e,E)}function E(e,t){return t}var T=i(8632),_=i(4402),D=i(4351);var N=i(7956),O=i(3239),j=i(9251);class S extends h{static featureName=j.t;constructor(e,r){let n=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,r,j.t,n),c.il&&((0,t.OP)(e).initHidden=Boolean("hidden"===document.visibilityState),(0,N.N)((()=>(0,s.p)("docHidden",[(0,p.z)()],void 0,j.t,this.ee)),!0),(0,O.bP)("pagehide",(()=>(0,s.p)("winPagehide",[(0,p.z)()],void 0,j.t,this.ee))),this.importAggregator())}}var P=i(3081);class C extends h{static featureName=P.t9;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,P.t9,r),this.importAggregator()}}var R,I=i(2210),k=i(1214),H=i(2177),L={};try{R=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(L.console=!0,-1!==R.indexOf("dev")&&(L.dev=!0),-1!==R.indexOf("nr_dev")&&(L.nrDev=!0))}catch(e){}function z(e){try{L.console&&z(e)}catch(e){}}L.nrDev&&H.ee.on("internal-error",(function(e){z(e.stack)})),L.dev&&H.ee.on("fn-err",(function(e,t,r){z(r.stack)})),L.dev&&(z("NR AGENT IN DEVELOPMENT MODE"),z("flags: "+(0,b.D)(L,(function(e,t){return e})).join(", ")));var M=i(6660);class B extends h{static featureName=M.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,M.t,i),this.skipNext=0;try{this.removeOnAbort=new AbortController}catch(e){}const o=this;o.ee.on("fn-start",(function(e,t,r){o.abortHandler&&(o.skipNext+=1)})),o.ee.on("fn-err",(function(t,r,n){o.abortHandler&&!n[M.A]&&((0,I.X)(n,M.A,(function(){return!0})),this.thrown=!0,(0,s.p)("err",[n,(0,p.z)()],void 0,e.D.jserrors,o.ee))})),o.ee.on("fn-end",(function(){o.abortHandler&&!this.thrown&&o.skipNext>0&&(o.skipNext-=1)})),o.ee.on("internal-error",(function(t){(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,o.ee)})),this.origOnerror=c._A.onerror,c._A.onerror=this.onerrorHandler.bind(this),c._A.addEventListener("unhandledrejection",(t=>{const r=function(e){let t="Unhandled Promise Rejection: ";if(e instanceof Error)try{return e.message=t+e.message,e}catch(t){return e}if(void 0===e)return new Error(t);try{return new Error(t+(0,D.P)(e))}catch(e){return new Error(t)}}(t.reason);(0,s.p)("err",[r,(0,p.z)(),!1,{unhandledPromiseRejection:1}],void 0,e.D.jserrors,this.ee)}),(0,O.m$)(!1,this.removeOnAbort?.signal)),(0,k.gy)(this.ee),(0,k.BV)(this.ee),(0,k.em)(this.ee),(0,t.OP)(r).xhrWrappable&&(0,k.Kf)(this.ee),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}onerrorHandler(t,r,n,i,o){"function"==typeof this.origOnerror&&this.origOnerror(...arguments);try{this.skipNext?this.skipNext-=1:(0,s.p)("err",[o||new F(t,r,n),(0,p.z)()],void 0,e.D.jserrors,this.ee)}catch(t){try{(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,this.ee)}catch(e){}}return!1}}function F(e,t,r){this.message=e||"Uncaught error with no additional information",this.sourceURL=t,this.line=r}let U=1;const q="nr@id";function G(e){const t=typeof e;return!e||"object"!==t&&"function"!==t?-1:e===c._A?0:(0,I.X)(e,q,(function(){return U++}))}function V(e){if("string"==typeof e&&e.length)return e.length;if("object"==typeof e){if("undefined"!=typeof ArrayBuffer&&e instanceof ArrayBuffer&&e.byteLength)return e.byteLength;if("undefined"!=typeof Blob&&e instanceof Blob&&e.size)return e.size;if(!("undefined"!=typeof FormData&&e instanceof FormData))try{return(0,D.P)(e).length}catch(e){return}}}var X=i(7243);class W{constructor(e){this.agentIdentifier=e,this.generateTracePayload=this.generateTracePayload.bind(this),this.shouldGenerateTrace=this.shouldGenerateTrace.bind(this)}generateTracePayload(e){if(!this.shouldGenerateTrace(e))return null;var r=(0,t.DL)(this.agentIdentifier);if(!r)return null;var n=(r.accountID||"").toString()||null,i=(r.agentID||"").toString()||null,o=(r.trustKey||"").toString()||null;if(!n||!i)return null;var a=(0,_.M)(),s=(0,_.Ht)(),c=Date.now(),u={spanId:a,traceId:s,timestamp:c};return(e.sameOrigin||this.isAllowedOrigin(e)&&this.useTraceContextHeadersForCors())&&(u.traceContextParentHeader=this.generateTraceContextParentHeader(a,s),u.traceContextStateHeader=this.generateTraceContextStateHeader(a,c,n,i,o)),(e.sameOrigin&&!this.excludeNewrelicHeader()||!e.sameOrigin&&this.isAllowedOrigin(e)&&this.useNewrelicHeaderForCors())&&(u.newrelicHeader=this.generateTraceHeader(a,s,c,n,i,o)),u}generateTraceContextParentHeader(e,t){return"00-"+t+"-"+e+"-01"}generateTraceContextStateHeader(e,t,r,n,i){return i+"@nr=0-1-"+r+"-"+n+"-"+e+"----"+t}generateTraceHeader(e,t,r,n,i,o){if(!("function"==typeof c._A?.btoa))return null;var a={v:[0,1],d:{ty:"Browser",ac:n,ap:i,id:e,tr:t,ti:r}};return o&&n!==o&&(a.d.tk=o),btoa((0,D.P)(a))}shouldGenerateTrace(e){return this.isDtEnabled()&&this.isAllowedOrigin(e)}isAllowedOrigin(e){var r=!1,n={};if((0,t.Mt)(this.agentIdentifier,"distributed_tracing")&&(n=(0,t.P_)(this.agentIdentifier).distributed_tracing),e.sameOrigin)r=!0;else if(n.allowed_origins instanceof Array)for(var i=0;i<n.allowed_origins.length;i++){var o=(0,X.e)(n.allowed_origins[i]);if(e.hostname===o.hostname&&e.protocol===o.protocol&&e.port===o.port){r=!0;break}}return r}isDtEnabled(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.enabled}excludeNewrelicHeader(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.exclude_newrelic_header}useNewrelicHeaderForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!1!==e.cors_use_newrelic_header}useTraceContextHeadersForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.cors_use_tracecontext_headers}}var Z=i(7825),$=["load","error","abort","timeout"],Q=$.length,Y=t.Yu.REQ,K=c._A.XMLHttpRequest;class J extends h{static featureName=Z.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,Z.t,i),(0,t.OP)(r).xhrWrappable&&(this.dt=new W(r),this.handler=(e,t,r,n)=>(0,s.p)(e,t,r,n,this.ee),(0,k.u5)(this.ee),(0,k.Kf)(this.ee),function(r,n,i,o){function a(e){var t=this;t.totalCbs=0,t.called=0,t.cbTime=0,t.end=E,t.ended=!1,t.xhrGuids={},t.lastSize=null,t.loadCaptureCalled=!1,t.params=this.params||{},t.metrics=this.metrics||{},e.addEventListener("load",(function(r){_(t,e)}),(0,O.m$)(!1)),c.IF||e.addEventListener("progress",(function(e){t.lastSize=e.loaded}),(0,O.m$)(!1))}function s(e){this.params={method:e[0]},T(this,e[1]),this.metrics={}}function u(e,n){var i=(0,t.DL)(r);i.xpid&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",i.xpid);var a=o.generateTracePayload(this.parsedOrigin);if(a){var s=!1;a.newrelicHeader&&(n.setRequestHeader("newrelic",a.newrelicHeader),s=!0),a.traceContextParentHeader&&(n.setRequestHeader("traceparent",a.traceContextParentHeader),a.traceContextStateHeader&&n.setRequestHeader("tracestate",a.traceContextStateHeader),s=!0),s&&(this.dt=a)}}function d(e,t){var r=this.metrics,i=e[0],o=this;if(r&&i){var a=V(i);a&&(r.txSize=a)}this.startTime=(0,p.z)(),this.listener=function(e){try{"abort"!==e.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==e.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof t.onload)&&"function"==typeof o.end)&&o.end(t)}catch(e){try{n.emit("internal-error",[e])}catch(e){}}};for(var s=0;s<Q;s++)t.addEventListener($[s],this.listener,(0,O.m$)(!1))}function f(e,t,r){this.cbTime+=e,t?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof r.onload||"function"!=typeof this.end||this.end(r)}function l(e,t){var r=""+G(e)+!!t;this.xhrGuids&&!this.xhrGuids[r]&&(this.xhrGuids[r]=!0,this.totalCbs+=1)}function h(e,t){var r=""+G(e)+!!t;this.xhrGuids&&this.xhrGuids[r]&&(delete this.xhrGuids[r],this.totalCbs-=1)}function g(){this.endTime=(0,p.z)()}function m(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-added",[e[1],e[2]],t)}function v(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-removed",[e[1],e[2]],t)}function b(e,t,r){t instanceof K&&("onload"===r&&(this.onload=!0),("load"===(e[0]&&e[0].type)||this.onload)&&(this.xhrCbStart=(0,p.z)()))}function y(e,t){this.xhrCbStart&&n.emit("xhr-cb-time",[(0,p.z)()-this.xhrCbStart,this.onload,t],t)}function w(e){var t,r=e[1]||{};"string"==typeof e[0]?t=e[0]:e[0]&&e[0].url?t=e[0].url:c._A?.URL&&e[0]&&e[0]instanceof URL&&(t=e[0].href),t&&(this.parsedOrigin=(0,X.e)(t),this.sameOrigin=this.parsedOrigin.sameOrigin);var n=o.generateTracePayload(this.parsedOrigin);if(n&&(n.newrelicHeader||n.traceContextParentHeader))if("string"==typeof e[0]||c._A?.URL&&e[0]&&e[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),s(i.headers,n)&&(this.dt=n),e.length>1?e[1]=i:e.push(i)}else e[0]&&e[0].headers&&s(e[0].headers,n)&&(this.dt=n);function s(e,t){var r=!1;return t.newrelicHeader&&(e.set("newrelic",t.newrelicHeader),r=!0),t.traceContextParentHeader&&(e.set("traceparent",t.traceContextParentHeader),t.traceContextStateHeader&&e.set("tracestate",t.traceContextStateHeader),r=!0),r}}function x(e,t){this.params={},this.metrics={},this.startTime=(0,p.z)(),this.dt=t,e.length>=1&&(this.target=e[0]),e.length>=2&&(this.opts=e[1]);var r,n=this.opts||{},i=this.target;"string"==typeof i?r=i:"object"==typeof i&&i instanceof Y?r=i.url:c._A?.URL&&"object"==typeof i&&i instanceof URL&&(r=i.href),T(this,r);var o=(""+(i&&i instanceof Y&&i.method||n.method||"GET")).toUpperCase();this.params.method=o,this.txSize=V(n.body)||0}function A(t,r){var n;this.endTime=(0,p.z)(),this.params||(this.params={}),this.params.status=r?r.status:0,"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var o={txSize:this.txSize,rxSize:n,duration:(0,p.z)()-this.startTime};i("xhr",[this.params,o,this.startTime,this.endTime,"fetch"],this,e.D.ajax)}function E(t){var r=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var o=0;o<Q;o++)t.removeEventListener($[o],this.listener,!1);r.aborted||(n.duration=(0,p.z)()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==r.status&&(r.status=0):_(this,t),n.cbTime=this.cbTime,i("xhr",[r,n,this.startTime,this.endTime,"xhr"],this,e.D.ajax))}}function T(e,t){var r=(0,X.e)(t),n=e.params;n.hostname=r.hostname,n.port=r.port,n.protocol=r.protocol,n.host=r.hostname+":"+r.port,n.pathname=r.pathname,e.parsedOrigin=r,e.sameOrigin=r.sameOrigin}function _(e,t){e.params.status=t.status;var r=function(e,t){var r=e.responseType;return"json"===r&&null!==t?t:"arraybuffer"===r||"blob"===r||"json"===r?V(e.response):"text"===r||""===r||void 0===r?V(e.responseText):void 0}(t,e.lastSize);if(r&&(e.metrics.rxSize=r),e.sameOrigin){var n=t.getResponseHeader("X-NewRelic-App-Data");n&&(e.params.cat=n.split(", ").pop())}e.loadCaptureCalled=!0}n.on("new-xhr",a),n.on("open-xhr-start",s),n.on("open-xhr-end",u),n.on("send-xhr-start",d),n.on("xhr-cb-time",f),n.on("xhr-load-added",l),n.on("xhr-load-removed",h),n.on("xhr-resolved",g),n.on("addEventListener-end",m),n.on("removeEventListener-end",v),n.on("fn-end",y),n.on("fetch-before-start",w),n.on("fetch-start",x),n.on("fn-start",b),n.on("fetch-done",A)}(r,this.ee,this.handler,this.dt),this.importAggregator())}}var ee=i(3614);const{BST_RESOURCE:te,RESOURCE:re,START:ne,END:ie,FEATURE_NAME:oe,FN_END:ae,FN_START:se,PUSH_STATE:ce}=ee;var ue=i(7836);const{FEATURE_NAME:de,START:fe,END:le,BODY:he,CB_END:ge,JS_TIME:pe,FETCH:me,FN_START:ve,CB_START:be,FN_END:ye}=ue;var we=i(4649);class xe extends h{static featureName=we.t;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,we.t,r),this.importAggregator()}}new class{constructor(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:(0,_.ky)(16);c._A?(this.agentIdentifier=t,this.sharedAggregator=new y({agentIdentifier:this.agentIdentifier}),this.features={},this.desiredFeatures=new Set(e.features||[]),this.desiredFeatures.add(m),Object.assign(this,(0,a.j)(this.agentIdentifier,e,e.loaderType||"agent")),this.start()):(0,l.Z)("Failed to initial the agent. Could not determine the runtime environment.")}get config(){return{info:(0,t.C5)(this.agentIdentifier),init:(0,t.P_)(this.agentIdentifier),loader_config:(0,t.DL)(this.agentIdentifier),runtime:(0,t.OP)(this.agentIdentifier)}}start(){const t="features";try{const r=n(this.agentIdentifier),i=[...this.desiredFeatures];i.sort(((t,r)=>e.p[t.featureName]-e.p[r.featureName])),i.forEach((t=>{if(r[t.featureName]||t.featureName===e.D.pageViewEvent){const n=function(t){switch(t){case e.D.ajax:return[e.D.jserrors];case e.D.sessionTrace:return[e.D.ajax,e.D.pageViewEvent];case e.D.sessionReplay:return[e.D.sessionTrace];case e.D.pageViewTiming:return[e.D.pageViewEvent];default:return[]}}(t.featureName);n.every((e=>r[e]))||(0,l.Z)("".concat(t.featureName," is enabled but one or more dependent features has been disabled (").concat((0,D.P)(n),"). This may cause unintended consequences or missing data...")),this.features[t.featureName]=new t(this.agentIdentifier,this.sharedAggregator)}})),(0,T.Qy)(this.agentIdentifier,this.features,t)}catch(e){(0,l.Z)("Failed to initialize all enabled instrument classes (agent aborted) -",e);for(const e in this.features)this.features[e].abortHandler?.();const r=(0,T.fP)();return delete r.initializedAgents[this.agentIdentifier]?.api,delete r.initializedAgents[this.agentIdentifier]?.[t],delete this.sharedAggregator,r.ee?.abort(),delete r.ee?.get(this.agentIdentifier),!1}}}({features:[J,m,S,class extends h{static featureName=oe;constructor(t,r){if(super(t,r,oe,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;const n=this.ee;let i;(0,k.QU)(n),this.eventsEE=(0,k.em)(n),this.eventsEE.on(se,(function(e,t){this.bstStart=(0,p.z)()})),this.eventsEE.on(ae,(function(t,r){(0,s.p)("bst",[t[0],r,this.bstStart,(0,p.z)()],void 0,e.D.sessionTrace,n)})),n.on(ce+ne,(function(e){this.time=(0,p.z)(),this.startPath=location.pathname+location.hash})),n.on(ce+ie,(function(t){(0,s.p)("bstHist",[location.pathname+location.hash,this.startPath,this.time],void 0,e.D.sessionTrace,n)}));try{i=new PerformanceObserver((t=>{const r=t.getEntries();(0,s.p)(te,[r],void 0,e.D.sessionTrace,n)})),i.observe({type:re,buffered:!0})}catch(e){}this.importAggregator({resourceObserver:i})}},C,xe,B,class extends h{static featureName=de;constructor(e,r){if(super(e,r,de,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;if(!(0,t.OP)(e).xhrWrappable)return;try{this.removeOnAbort=new AbortController}catch(e){}let n,i=0;const o=this.ee.get("tracer"),a=(0,k._L)(this.ee),s=(0,k.Lg)(this.ee),u=(0,k.BV)(this.ee),d=(0,k.Kf)(this.ee),f=this.ee.get("events"),l=(0,k.u5)(this.ee),h=(0,k.QU)(this.ee),g=(0,k.Gm)(this.ee);function m(e,t){h.emit("newURL",[""+window.location,t])}function v(){i++,n=window.location.hash,this[ve]=(0,p.z)()}function b(){i--,window.location.hash!==n&&m(0,!0);var e=(0,p.z)();this[pe]=~~this[pe]+e-this[ve],this[ye]=e}function y(e,t){e.on(t,(function(){this[t]=(0,p.z)()}))}this.ee.on(ve,v),s.on(be,v),a.on(be,v),this.ee.on(ye,b),s.on(ge,b),a.on(ge,b),this.ee.buffer([ve,ye,"xhr-resolved"],this.featureName),f.buffer([ve],this.featureName),u.buffer(["setTimeout"+le,"clearTimeout"+fe,ve],this.featureName),d.buffer([ve,"new-xhr","send-xhr"+fe],this.featureName),l.buffer([me+fe,me+"-done",me+he+fe,me+he+le],this.featureName),h.buffer(["newURL"],this.featureName),g.buffer([ve],this.featureName),s.buffer(["propagate",be,ge,"executor-err","resolve"+fe],this.featureName),o.buffer([ve,"no-"+ve],this.featureName),a.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"],this.featureName),y(l,me+fe),y(l,me+"-done"),y(a,"new-jsonp"),y(a,"jsonp-end"),y(a,"cb-start"),h.on("pushState-end",m),h.on("replaceState-end",m),window.addEventListener("hashchange",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("load",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("popstate",(function(){m(0,i>1)}),(0,O.m$)(!0,this.removeOnAbort?.signal)),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}}],loaderType:"spa"})})(),window.NRBA=o})();</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Jia Li"/><meta name="DC.Creator" content="Joanne Macdonald"/><meta name="DC.Creator" content="Felix von Stetten"/><meta name="DC.title" content="Review: a comprehensive summary of a decade development of the recombinase polymerase amplification "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2018/12/17"/><meta name="DC.Identifier" scheme="doi" content="10.1039/C8AN01621F"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Review: a comprehensive summary of a decade development of the recombinase polymerase amplification "/><meta name="citation_author" content="Jia Li"/><meta name="citation_author" content="Joanne Macdonald"/><meta name="citation_author" content="Felix von Stetten"/><meta name="citation_online_date" content="2018/11/14"/><meta name="citation_date" content="2019"/><meta name="citation_journal_title" content="Analyst"/><meta name="citation_volume" content="144"/><meta name="citation_issue" content="1"/><meta name="citation_firstpage" content="31"/><meta name="citation_lastpage" content="67"/><meta name="citation_doi" content="10.1039/C8AN01621F"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2019/an/c8an01621f"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2019/an/c8an01621f"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2019/an/c8an01621f"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2019/an/c8an01621f">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:c8an01667d)" href="/en/content/articlehtml/2019/an/c8an01667d" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:c8an01526k)" href="/en/content/articlehtml/2019/an/c8an01526k" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div><br/><span class="oa"><img src="/content/newimages/open_access_blue.png" alt=""/> Open Access Article<br/><img src="/content/newimages/CCBY.svg" alt=""/>
This Open Access Article is licensed under a <br/><a rel="license" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 Unported Licence</a></span></div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/C8AN01621F">10.1039/C8AN01621F</a>
(Critical Review)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1364-5528/1876">Analyst</a></span>, 2019, <strong>144</strong>, 31-67</div><h1 id="sect213"><span class="title_heading">Review: a comprehensive summary of a decade development of the recombinase polymerase amplification</span></h1><p class="header_text">
      <span class="bold">
        
          
            Jia 
            Li
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Jia Li (orcid.org/0000-0002-4480-7899) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-4480-7899"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Joanne 
            Macdonald
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Joanne Macdonald (orcid.org/0000-0003-4624-3545) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-4624-3545"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">bc</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Felix 
            von Stetten
          
          
        
      </span>*<sup><span class="sup_ref italic">ad</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Laboratory for MEMS Applications, IMTEK – Department of Microsystems Engineering, University of Freiburg, Georges-Köhler-Allee 103, 79110 Freiburg, Germany. E-mail: <a href="mailto:Felix.von.Stetten@Hahn-Schickard.de">Felix.von.Stetten@Hahn-Schickard.de</a>;   Tel: +49 761 203-73243</span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Inflammation and Healing Research Cluster, Genecology Research Centre, School of Science and Engineering, University of the Sunshine Coast, Qld, Australia. E-mail: <a href="mailto:jmacdon1@usc.edu.au">jmacdon1@usc.edu.au</a>;   Tel: +61 7 5456 5944</span>
      <br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">Division of Experimental Therapeutics, Columbia University, New York, NY, USA. E-mail: <a href="mailto:jm2236@columbia.edu">jm2236@columbia.edu</a></span>
      <br/><a id="affd"><sup><span class="sup_ref italic">d</span></sup></a><span class="italic">Hahn-Schickard, Georges-Köhler-Allee 103, 79110 Freiburg, Germany</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      21st August 2018
    </span><span class="bold italic">, Accepted 8th October 2018</span></div><p class="bold italic">First published on 14th November 2018</p><hr/><div class="abstract"><h2>Abstract</h2><p>Nucleic acid amplification has permeated every field in the life sciences since the introduction of the classic polymerase chain reaction (PCR) method in 1983. Yet, despite its fundamental reach, PCR has been constrained within the walls of a laboratory, due to its requirement for a sophisticated thermocycling machine, limiting external application in low-resource settings. New isothermal amplification strategies are seeking to break through traditional laboratory boundaries by providing nucleic acid replication at constant temperatures. Of these methods, recombinase polymerase amplification (RPA) is one of the fastest developing, experiencing rapid uptake and market, even though it was introduced comparatively late. Critically, RPA's technology potentiates highly accessible and sensitive nucleic acid amplification outside of laboratory, and even self-testing. Here we provide a comprehensive review of the equipment-free simplicity of RPA over its first decade of development. Our review includes key knowledge of RPA technology, such as its reaction components, mechanism, sensitivities and specificities, and distinctive detection methods. The review also provides know-how for developing RPA assays, and information about commercially available RPA reaction kits and accessories. We summarise critical RPA experimental tips and issues available through data mining the published literature, to assist researchers in mastering the RPA reaction. We also outline influential hotspots of RPA development, and conclude with outlooks for future development and implications for eclipsing PCR and further revolutionising the life sciences.</p></div><hr/><table><tr><td class="biogPlate"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-p1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-p1.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-p1.gif"/></a><p><span class="graphic_title">Jia Li</span></p></td><td><i><p>Dr Jia Li did her undergraduate degree in the University of Sydney, working on the structure and activity relationship of carborane phosphonium salts for the Boron Neutron Capture Therapy (BNCT). She then went to England for a research project about onco-protein–protein interactions at the University of Leeds. She did her PhD under Dr Joanne Macdonald's supervision at the University of the Sunshine Coast, developing rapid and novel virus detection biosensors towards point-of-care. Afterwards, she has been working as a post-doc in the University of Freiburg under Alexander von Humboldt Fellowship, researching in assay development for micro-total diagnostic systems.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-p2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-p2.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-p2.gif"/></a><p><span class="graphic_title">Joanne Macdonald</span></p></td><td><i><p>Dr Joanne Macdonald's research focuses on the molecular engineering of advanced devices, including a DNA automaton able to play tic-tac-toe against a human opponent, and rapid field-based biosensors for the detection of pathogens. She is an Associate Professor in Molecular Engineering at the University of the Sunshine Coast (QLD, Australia), holds a joint appointment in Clinical Medical Sciences at Columbia University (New York, USA), and is the Chief Technical Officer of the biosensing company BioCifer Pty Ltd. Dr Macdonald received her PhD in Microbiology in 2003 from the University of Queensland, Australia.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-p3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-p3.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-p3.gif"/></a><p><span class="graphic_title">Felix von Stetten</span></p></td><td><i><p>Professor Felix von Stetten studied agricultural sciences and biotechnology. He completed his PhD in Microbiology in 1999 from the Technical University of Munich, Germany. Thereafter, he joined in the diagnostic industry, where he was involved in the development of methods for sample preparation, real-time PCR and DNA-arrays. Subsequently, he joined the Laboratory for MEMS Applications at IMTEK, University of Freiburg, where he was involved in lab-on-a-chip research. In 2008, he became head of the Hahn-Schickard Lab-on-a-Chip division. Now he is associate director of the Hahn-Schickard-Institut für Mikroanalysesysteme, and apl. professor at IMTEK, University of Freiburg.</p></i></td></tr></table><hr/>
    
      
      <h2 id="sect293"><span class="a_heading">1. Introduction and overview</span></h2>
      <span>Nucleic acid amplification (NAA) <span class="italic">in vitro</span>, the artificial replication of genetic material, has infiltrated all areas of life sciences and biotechnology, such as pathogen detection, cancer research, cloning, sequencing, genetic engineering, synthetic biology, genotyping, mutagenesis, forensic identification of crimes, drug discovery, molecular archaeology, food testing, wellness and lifestyle testing <span class="italic">etc.</span> This explosive revolution began with the invention of the polymerase chain reaction (PCR) by Kary Mullis in 1983.<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> Fundamentally, PCR is a cyclic process that performs exponential amplification from a single nucleic acid molecule to billions of copies <span class="italic">in vitro</span>, by providing successive temperatures favourable to nucleic acid replication processes (strand denaturation, primer annealing, and enzymatic extension). Increasing molecular quantities makes the handling and subsequent applications of nucleic acids easier, reducing the requirement for use of toxic radioactive probes to track molecular presence, and spawning immense creativity around applications for use. Yet, as valuable as PCR is, the requirement for a sophisticated thermocycler to provide the cyclic heating and cooling process, has largely bound PCR to implementation within the walls of a laboratory, impeding its application in low-resource settings.</span>
      <p class="otherpara">Recent advances in isothermal nucleic acid amplification have provided simplified incubation conditions for artificial nucleic acid replication, requiring only a constant temperature rather than thermocycling. The single temperature incubation reduces equipment requirements, opening new avenues to break through the boundaries of the laboratory and perform amplification in low-resource settings. The elimination of repeated heating and cooling steps also provides a second advantage for low-resource implementation, through reduced amplification times. Faster reactions occur not only because of a reduction in heating and cooling times, but also because multiple molecular reactions can proceed asynchronously rather than being forced to operate sequentially within an artificial heating and cooling cycle. Since the early 1990s, a plethora of isothermal nucleic acid amplification methods have adopted various reaction mechanisms. The most well-established methods are exemplified by nucleic acid sequence-based amplification (NASBA, also known as transcription mediated amplification, TMA), signal-mediated amplification of ribonucleic acid (RNA) technology (SMART), helicase-dependent amplification (HDA), recombinase polymerase amplification (RPA), rolling circle amplification (RCA), multiple displacement amplification (MDA), loop-mediated isothermal amplification (LAMP) and strand displacement amplification (SDA); readers can refer to details of these methods in a few reviews.<a title="Select to navigate to reference" href="#cit2"><sup><span class="sup_ref">2–5</span></sup></a> One technology in particular, recombinase polymerase amplification (RPA), is experiencing rapid development and increasing market share (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>), despite its comparatively late introduction, due to its simplified equipment requirements and fast reaction times.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f1.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Summary of RPA market share and publication numbers from its first introduction. (A): RPA market share percentage in the isothermal nucleic acid amplification technology. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit7">ref. 7</a>. Copyright 2018 Grand View Research, Inc. (B): RPA publication numbers from year 2006 to 2017 based on collected data from web of science.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">RPA was first introduced in 2006 by Niall Armes from ASM Scientific Ltd (Cambridge, United Kingdom, founded by the Wellcome Trust Sanger Institute).<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> Although RPA has not yet occupied a large market share percentage in the isothermal nucleic acid amplification technology (according to the data from the Grand View Research report,<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a><a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>), it is experiencing the most rapid uptake. More than 250 publications about RPA have been published so far, with a consistent increase in RPA publication numbers in the last six years; noticeably, the RPA publication number started growing exponentially from year 2014 (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1B</a>). Among these publications, five RPA review papers were published in consecutive years from 2014 to 2018, respectively. The review of Zaghloul and El-shahat focuses on the application of RPA for hepatitis C virus diagnosis;<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> the review of Moore and Jaykus emphasises RPA assays developed for the detection of enteric viruses;<a title="Select to navigate to references" href="#cit9"><sup><span class="sup_ref">9</span></sup></a> the reviews of James and Macdonald,<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> Daher <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> and Lobato and O'Sullivan<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> describe and summarise the characteristics and strengths for applications of RPA in point-of-care (POC) diagnostics. In comparison, here we provide a comprehensive review that focuses on the radical properties and development potential of RPA. Beginning with an introduction of the key aspects of RPA technology, namely reaction components and mechanisms. We subsequently provide know-how about developing RPA assays, including design and selection of oligonucleotides (primer, probe and template); the information about commercially available RPA reaction kits and accessories are also provided. For those interested in the technical implementation of RPA, we summarise critical RPA experimental tips and issues available through data mining the published literature to assist researchers better master RPA reaction. This is followed by elucidating the clinical/field performance of RPA <span class="italic">via</span> collated data such as sensitivity and specificity from RPA literature. We also describe some distinctive RPA detection methods for those who want to detect RPA assay signal using the methods other than the commonly used PCR detection methods. To understand the critical significance of this technology for eclipsing PCR and breaking out of the boundaries of the laboratory walls, we discuss the development hot-spots of RPA, including quantitative RPA, multiplex RPA reaction, mobile RPA diagnostic, integrated RPA assays on microfluidics and one-step RPA assays. Our review then concludes with outlooks of future development of RPA.</p>
    
    
      
      <h2 id="sect319"><span class="a_heading">2. Recombinase polymerase amplification (RPA) reaction</span></h2>
      <span>The prominence of RPA as revolutionary method to eclipse PCR stems from its specific reaction components (<a title="Select to navigate to table" href="#tab1">Table 1</a>) and mechanism. For a successful RPA assay, the nuances are hinged on the intrinsic factors, the design of the primers, probe and nucleic acid template; and are related to the extrinsic factors, such as reaction temperature and agitation, tolerability to mismatches, inhibitors and background DNA. In addition, the nucleic acid labelling during RPA, and RPA amplicon clean-up and post-amplification treatment are also important details for successful RPA detection. This section provides these practical information summarised from the RPA literature to serve as a guideline for RPA assay design. In addition, readers can also get information about commercially available RPA reaction kits and accessories (<a title="Select to navigate to table" href="#tab2">Tables 2</a> and <a title="Select to navigate to table" href="#tab3">3</a>). At the end, this section elucidates the clinical/field performance of RPA <span class="italic">via</span> data mining of RPA literature, which are also succinctly collated (<a title="Select to navigate to table" href="#tab4">Tables 4</a> and <a title="Select to navigate to table" href="#tab5">5</a>).</span>
      
<div class="table_caption"><b>Table 1</b> <span id="tab1">Summary of RPA reaction components, their typical concentrations and functions</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="135.67pt"/><col align="left" width="138.48pt"/><col align="left" width="209.18pt"/><col align="left" width="38.90pt"/></colgroup>
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Reaction components</th>
                <th align="left" valign="bottom" class="border_black">Typical concentration</th>
                <th align="left" valign="bottom" class="border_black">Functions</th>
                <th align="left" valign="bottom" class="border_black">Ref.</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td valign="top">T4 UvsX protein</td>
                <td valign="top">120 ng μL<small><sup>−1</sup></small></td>
                <td valign="top">Recombinase that possesses pairing and strand-transfer activity that is important in genetic recombination, DNA repair and replication (or <span class="italic">E. coli</span> RecA; recombinase is a central component in the related processes of recombinational DNA repair and homologous genetic recombination that is the ortholog of the UvsX protein).</td>
                <td valign="top">
                  <a title="Select to navigate to reference" href="#cit13">13 and 14</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">T4 UvsY protein</td>
                <td valign="top">60 ng μL<small><sup>−1</sup></small></td>
                <td valign="top">Recombinase loading factor that is classified as a recombination-mediator protein that stimulates the single-stranded DNA-dependent ATPase activity of T4 UvsX and lowers the critical concentration of T4 UvsX required for activity.</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit15">15</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">T4 gp32</td>
                <td valign="top">600 ng μL<small><sup>−1</sup></small></td>
                <td valign="top">Single-stranded binding (SSB) protein is involved in DNA replication, repair and recombination, and binds preferentially to single-stranded DNA. The T4 UvsX, T4 UvsY and T4 gp32 proteins work co-operatively to initiate the RPA reaction <span class="italic">via</span> unwinding, D-loop formation and stabilisation of the DNA template.</td>
                <td valign="top">
                  <a title="Select to navigate to reference" href="#cit16">16 and 17</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">Bacillus subtilis</span> DNA polymerase I (<span class="italic">Bsu</span>) or <span class="italic">Staphylococcus aureus</span> polymerase (<span class="italic">Sau</span>)</td>
                <td valign="top">
                  <span class="italic">Bsu</span>: 30 ng μL<small><sup>−1</sup></small>; <span class="italic">Sau</span>: 8.6 or 12.8 μg</td>
                <td valign="top">DNA polymerase synthesises new DNA templates homologous to the target nucleic acid, by extending nucleotide building blocks from the bound primers, complementary to the original target nucleic acid sequence or “template”.</td>
                <td valign="top">
                  <a title="Select to navigate to reference" href="#cit18">18 and 19</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Deoxynucleotide triphosphate (dNTP, N = A, T, C, G)</td>
                <td valign="top">200 μM each</td>
                <td valign="top">An equimolar solution of dATP, dCTP, dGTP and dTTP are building blocks used by the DNA polymerase to synthesise new templates.</td>
                <td align="left" valign="top">—</td>
              </tr>
              <tr align="left">
                <td valign="top">Forward and reverse primers</td>
                <td valign="top">Usually at 420 nM each, but can be varied in the concentration range of 150 nM to 600 nM</td>
                <td valign="top">Primers are critical to directing the amplification event to the nucleic acid target of interest through homologous binding. After binding, the primers provide the essential 3′-OH for polymerase to perform strand extension.</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit20">20</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">DNA template</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The oligonucleotide that the primers bind to for the synthesis of exact new oligonucleotides</td>
                <td align="left" valign="top">—</td>
              </tr>
              <tr align="left">
                <td valign="top">Carbowax20M (a high molecular weight polyethylene glycol (PEG))</td>
                <td valign="top">PEG 35K (5%)</td>
                <td valign="top">The crowding reagent is a good mimic of the real biomacromolecules condition <span class="italic">in vivo</span> and facilitates amplification, as the crowding agents can enhance the catalytic activity of the enzymes.</td>
                <td valign="top">
                  <a title="Select to navigate to reference" href="#cit6">6 and 21–24</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Dithiothreitol</td>
                <td valign="top">2 mM</td>
                <td valign="top">Stablisation of the enzymes by baring free sulfhydryl groups.</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit25">25</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Phosphocreatine</td>
                <td valign="top">50 mM</td>
                <td rowspan="3" valign="top">The three components form the energy-supply system for the activities of the recombinase and the DNA polymerase.</td>
                <td rowspan="3" valign="top">
                  <a title="Select to navigate to references" href="#cit26">26</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Creatine kinase</td>
                <td valign="top">100 ng μL<small><sup>−1</sup></small></td>
              </tr>
              <tr align="left">
                <td valign="top">Adenosine triphosphate (ATP)</td>
                <td valign="top">3 mM</td>
              </tr>
              <tr align="left">
                <td valign="top">Tris(hydroxymethyl)aminomethane (Tris)</td>
                <td valign="top">50 mM (pH 7.9)</td>
                <td rowspan="2" valign="top">The two components serve to stabilise and solubilise the DNA in solution.</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to reference" href="#cit27">27 and 28</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Potassium acetate</td>
                <td valign="top">100 mM</td>
              </tr>
              <tr align="left">
                <td valign="top">Magnesium acetate</td>
                <td valign="top">14 mM</td>
                <td valign="top">Acting as a cofactor for the performance of the enzymes. The RPA reaction initiates once the magnesium acetate is added.</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit20">20</a>
                </td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
<div class="table_caption"><b>Table 2</b> <span id="tab2">Summary of commercialised RPA reaction kits by TwistDx<small><sup>TM</sup></small></span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="80.41pt"/><col align="left" width="64.42pt"/><col align="left" width="100.53pt"/><col align="left" width="93.12pt"/><col align="left" width="183.75pt"/></colgroup>
            
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Product name</th>
                <th align="left" valign="bottom" class="border_black">Category</th>
                <th align="left" valign="bottom" class="border_black">Nucleic acid detection</th>
                <th align="left" valign="bottom" class="border_black">Compatible general detection method</th>
                <th align="left" valign="bottom" class="border_black">Product information</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td valign="top">TwistAmp® Basic</td>
                <td valign="top">Lyophilised kit</td>
                <td valign="top">DNA</td>
                <td rowspan="4" valign="top">Gel electrophoresis</td>
                <td rowspan="4" valign="top">The <span class="bold">lyophilised kits</span> contain pre-mixed enzymes and reagents necessary for the amplification, the user needs only supply primers and template (and dNTPs for the liquid kits). The <span class="bold">RT kits</span> afford one-step RNA amplification, which contain pre-mixed enzymes and reagents necessary for the amplification. The user need only supply primers, template and RNase inhibitor.</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® Basic RT</td>
                <td valign="top">Lyophilised kit</td>
                <td valign="top">RNA</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® Liquid Basic</td>
                <td valign="top">Liquid kit</td>
                <td valign="top">DNA</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® Liquid Basic RT</td>
                <td valign="top">Liquid kit</td>
                <td valign="top">RNA</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® exo</td>
                <td valign="top">Lyophilised kit</td>
                <td valign="top">DNA</td>
                <td rowspan="4" valign="top">Real-time fluorogenic probe-based</td>
                <td rowspan="4" valign="top">Recommended for users who want to combine TwistDx's RPA amplification technology with the use of TwistDx's proprietary fluorescent <span class="bold">TwistAmp® exo probe</span> in a homogenous format. The <span class="bold">lyophilised kits</span> contain pre-mixed enzymes and reagents necessary for the amplification, the user needs only supply primers, probe and template (and dNTPs for the liquid kits). The <span class="bold">RT kits</span> afford one-step RNA amplification, which contain pre-mixed enzymes and reagents necessary for the amplification. The user need only supply primers, probe, template and RNase inhibitor.</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® exo RT</td>
                <td valign="top">Lyophilised kit</td>
                <td valign="top">RNA</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® Liquid exo</td>
                <td valign="top">Liquid kit</td>
                <td valign="top">DNA</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® Liquid exo RT</td>
                <td valign="top">Liquid kit</td>
                <td valign="top">RNA</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® fpg</td>
                <td valign="top">Lyophilised kit</td>
                <td valign="top">DNA</td>
                <td valign="top">Real-time and end-point fluorogenic probe-based</td>
                <td valign="top">Tailored for users who want to combine TwistDx's amplification technology with <span class="bold">an alternative TwistDx reporter probe system</span> – fluorescent <span class="bold">TwistAmp® fpg probe</span> other than the TwistAmp® exo probe system in a homogenous format. The kit contains pre-mixed enzymes and reagents necessary for the amplification, the user needs only supply primers, probe, template and RNase inhibitor.</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® nfo</td>
                <td valign="top">Lyophilised kit</td>
                <td valign="top">DNA</td>
                <td valign="top">Lateral flow strip</td>
                <td valign="top">Designed for users who want to detect the amplicons based on <span class="bold">sandwich assays</span>. The kit contains pre-mixed enzymes and reagents necessary for the amplification, the user need only supply primers, probe and template.</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® exo +ListeriaM</td>
                <td rowspan="2" valign="top">Food safety lyophilised kit</td>
                <td valign="top">DNA (<span class="italic">Listeria monocytogenes</span> hly gene)</td>
                <td rowspan="2" valign="top">Real-time fluorogenic probe-based</td>
                <td rowspan="2" valign="top">The kits contain pre-mixed enzymes, oligos and reagents necessary for detection of specific genes of <span class="italic">Listeria monocytogenes</span> and Campylobacter species respectively in less than 10 minutes. User must perform sample preparation.</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistAmp® exo +Campylobacter</td>
                <td valign="top">DNA (Campylobacter species including <span class="italic">jejuni</span> and <span class="italic">coli</span>)</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistGlow® Salmonella</td>
                <td rowspan="2" valign="top">Food safety lyophilised kit</td>
                <td rowspan="2" valign="top">DNA (<span class="italic">Salmonella enterica</span> INVA gene)</td>
                <td valign="top">Real-time and end-point fluorogenic probe-based</td>
                <td rowspan="2" valign="top">The kits contain pre-mixed enzymes, oligos and reagents necessary, users need only add DNA, with provided buffer and magnesium acetate to the reactions. The kits also feature internal control DNA and probes in the lyophilised pellets, and lysis buffer for a two-step lysis of up to 5 μL of sample. The turn-around time of the <span class="bold">Glow kit</span> and the <span class="bold">Flow kit</span> are less than 10 minutes and approximately 20 minutes, respectively.</td>
              </tr>
              <tr align="left">
                <td valign="top">TwistFlow® Salmonella</td>
                <td valign="top">Lateral flow strip</td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
<div class="table_caption"><b>Table 3</b> <span id="tab3">Summary of commercialised RPA devices and accessories by TwistDx<small><sup>TM</sup></small></span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="110.96pt"/><col align="left" width="80.75pt"/><col align="left" width="96.16pt"/><col align="left" width="234.36pt"/></colgroup>
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Product name</th>
                <th align="left" valign="bottom" class="border_black">Category</th>
                <th align="left" valign="bottom" class="border_black">Compatible general detection method</th>
                <th align="left" valign="bottom" class="border_black">Product information</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td valign="top">Twirla™ Portable Mixing Incubator</td>
                <td valign="top">Device for incubation</td>
                <td valign="top">Gel electrophoresis and lateral flow strip</td>
                <td valign="top">Small and portable, and incubates up to 6 RPA reactions at optimal temperature. It can be powered by battery or mains-power <span class="italic">via</span> micro USB. Magnetic mixing of RPA reactions is also possible when fitted with <span class="bold">Micro Balls</span> (0.2 mL; dispensed with <span class="bold">Micro Ball Dispenser</span>) in the reaction tube.</td>
              </tr>
              <tr align="left">
                <td valign="top">T8-ISO Instrument</td>
                <td valign="top">Device for incubation and detection</td>
                <td valign="top">Real-time</td>
                <td valign="top">Incubates up to 8 RPA reactions with 2 channel fluorescence detection per tube; the testing temperature range is from 37 °C to 65 °C. It can be powered by mains-power <span class="italic">via</span> micro USB or <span class="bold">PowerGorilla external battery</span>. Magnetic mixing of RPA reactions is also possible when fitted with <span class="bold">Micro Balls</span> (0.2 mL; dispensed with <span class="bold">Micro Ball Dispenser</span>) in the reaction tube. It can also be adapted into a <span class="bold">T8-ISO Carry Case</span> when travelling to demanding environments.</td>
              </tr>
              <tr align="left">
                <td valign="top">T16-ISO Instrument</td>
                <td valign="top">Device for incubation and detection</td>
                <td valign="top">Real-time</td>
                <td valign="top">An advanced version of the T8-ISO Instrument, which incubates up to 16 RPA reactions with 3 channel fluorescence detection per tube.</td>
              </tr>
              <tr align="left">
                <td valign="top">Milenia HybriDetect 1</td>
                <td valign="top">Device for lateral flow detection</td>
                <td valign="top">Lateral flow strip</td>
                <td valign="top">Single-plex detection designed to detect a biotin and FITC/FAM labelled amplicon. Detection is based on sandwich assay using <span class="bold">gold nanoparticles</span> as tracer.</td>
              </tr>
              <tr align="left">
                <td valign="top">Milenia HybriDetect 2</td>
                <td valign="top">Device for lateral flow detection</td>
                <td valign="top">Lateral flow strip</td>
                <td valign="top">Duplex detection designed to simultaneously detect two amplicons labelled with FITC (or FAM)/biotin and/or FITC (or FAM)/DIG. Detection is based on sandwich assay using <span class="bold">gold nanoparticles</span> as tracer.</td>
              </tr>
              <tr align="left">
                <td valign="top">PCRD Nucleic Acid Detection</td>
                <td valign="top">Device for lateral flow detection</td>
                <td valign="top">Lateral flow strip</td>
                <td valign="top">Duplex detection designed to simultaneously detect two amplicons labelled with DIG/biotin and/or FITC (or FAM)/biotin. Detection is based on sandwich assay using <span class="bold">carbon nanoparticles</span> as tracer and is <span class="bold">performed in an open cartridge</span>.</td>
              </tr>
              <tr align="left">
                <td valign="top">U-Star disposable nucleic acid lateral flow detection units</td>
                <td valign="top">Device for lateral flow detection</td>
                <td valign="top">Lateral flow strip</td>
                <td valign="top">Single-plex detection designed to detect a biotin and FITC (or FAM) labelled amplicon. Detection is based on sandwich assay using <span class="bold">carbon nanoparticles</span> as tracer and is <span class="bold">performed in a sealed cartridge</span>.</td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
<div class="table_caption"><b>Table 4</b> <span id="tab4">RPA literature reporting analytical sensitivity nearing the single analyte detection limit</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="169.54pt"/><col align="left" width="193.54pt"/><col align="left" width="133.54pt"/><col align="left" width="25.62pt"/></colgroup>
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Analyte(s)</th>
                <th align="left" valign="bottom" class="border_black">Detection method</th>
                <th align="left" valign="bottom" class="border_black">Limit of detection</th>
                <th align="left" valign="bottom" class="border_black">Ref.</th>
              </tr>
            </thead>
            <tfoot align="left">
              <tr align="left">
                <th colspan="4">
                  <a id="tab4fna"><span class="tfootnote">a</span></a> <span class="sup_inf">CFU: colony forming unit.</span>
                </th>
              </tr>
            </tfoot>
            <tbody align="left">
              <tr align="left">
                <td valign="top">
                  <span class="italic">Bla</span>CTX-M-15 antimicrobial resistance gene</td>
                <td valign="top">Electrowetting-on-dielectric (EWOD)-based digital droplets end-point fluorescent detection</td>
                <td valign="top">5.6 fg (∼a single DNA copy)</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit80">80</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">RNA polymerase beta subunit (RPOB) gene of <span class="italic">Mycobacterium tuberculosis</span></td>
                <td valign="top">Electrochemical detection using gold nanoparticles on a solid phase</td>
                <td valign="top">1 CFU<i><small><sup style="color: red;"><a title="" href="#tab4fna">a</a></sup></small></i></td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit103">103</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Early secretory antigenic target-6 (ESAT-6) gene of <span class="italic">Mycobacterium tuberculosis</span></td>
                <td valign="top">Electrochemical detection on a screen-printed carbon electrode (SPCE)</td>
                <td valign="top">1 CFU<i><small><sup style="color: red;"><a title="" href="#tab4fna">a</a></sup></small></i></td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit102">102</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Serotype-specific Enteritidis sequence fragment <span class="italic">sdfl</span> of <span class="italic">Salmonella Enterica</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">1 CFU<i><small><sup style="color: red;"><a title="" href="#tab4fna">a</a></sup></small></i></td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit119">119</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Genomic DNA of <span class="italic">Plasmodium falciparum</span> 3D7</td>
                <td valign="top">Real-time waveguide-based detection</td>
                <td valign="top">&lt;1 parasite per μL</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit120">120</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">Leishmania donovani</span> kinetoplast minicircle DNA</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">1 cell</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit121">121</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">β-Conglutin for Lup an 1 anaphylactic allergen</td>
                <td valign="top">(Competitive) lateral flow strip detection</td>
                <td valign="top">0.17 attomol</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit39">39</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">Ceu</span>E gene of <span class="italic">Campylobacter jejun</span>i; <span class="italic">hip</span>O gene of <span class="italic">Campylobacter coli</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">1 CFU<i><small><sup style="color: red;"><a title="" href="#tab4fna">a</a></sup></small></i> ml<small><sup>−1</sup></small> in pure culture and chicken broth without enrichment</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit122">122</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lentivviruses harboring genome fragment of Zika virus</td>
                <td valign="top">Clustered regularly interspaced short palindromic repeats (CRISPR)-based end-point fluorescent detection</td>
                <td valign="top">2 attomol L<small><sup>−1</sup></small></td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit123">123</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">B1 gene of <span class="italic">Toxoplasma gondii</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">0.1 oocyst</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit71">71</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Small subunit ribosomal RNA (18S RNA) gene of <span class="italic">Plasmodium knowlesi</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">1 plasmid</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit124">124</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">IS6110</span> gene of <span class="italic">Mycobacterium tuberculosis</span> H37Rv genomic DNA</td>
                <td valign="top">Real-time silicon photonic microring-based detection</td>
                <td valign="top">3.2 genomic DNA copies (= single cell of H37Rv)</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit113">113</a>
                </td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
<div class="table_caption"><b>Table 5</b> <span id="tab5">RPA literature describing clinical/field trials</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="91.93pt"/><col align="left" width="63.49pt"/><col align="left" width="91.33pt"/><col align="left" width="86.17pt"/><col align="left" width="39.46pt"/><col align="left" width="39.46pt"/><col align="left" width="56.32pt"/><col align="left" width="82.94pt"/><col align="left" width="50.93pt"/><col align="left" width="50.93pt"/><col align="left" width="25.34pt"/></colgroup>
            
            
            
            
            
            
            
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Analyte(s)</th>
                <th align="left" valign="bottom" class="border_black">Detection method</th>
                <th align="left" valign="bottom" class="border_black">Limit of detection</th>
                <th align="left" valign="bottom" class="border_black">Clinical/field sample(s)</th>
                <th align="left" valign="bottom" class="border_black">Clinical sensitivity</th>
                <th align="left" valign="bottom" class="border_black">Clinical specificity</th>
                <th align="left" valign="bottom" class="border_black">Benchmark method</th>
                <th align="left" valign="bottom" class="border_black">Limit of detection of benchmark method</th>
                <th align="left" valign="bottom" class="border_black">Clinical sensitivity of benchmark method compared to RPA</th>
                <th align="left" valign="bottom" class="border_black">Clinical specificity of benchmark method compared to RPA</th>
                <th align="left" valign="bottom" class="border_black">Ref.</th>
              </tr>
            </thead>
            <tfoot align="left">
              <tr align="left">
                <th colspan="11">
                  <a id="tab5fna"><span class="tfootnote">a</span></a> <span class="sup_inf">RT: reverse transcription.</span>
                </th>
              </tr>
            </tfoot>
            <tbody align="left">
              <tr align="left">
                <td valign="top">Nucleocapsid (N) gene of bovine coronavirus</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">10 to 100 RNA copies (19 RNA copies by probit analysis)</td>
                <td valign="top">16 fecal and 14 nasal swab specimens collected from cattle showing intestinal and/or respiratory manifestations</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td valign="top">1000 RNA copies</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit125">125</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Chlamydia trachomatis CDS2 gene</td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">5–12 pathogens per reaction</td>
                <td valign="top">70 self-collected first void morning urine samples from young adults (19 males and 51 females)</td>
                <td valign="top">83%</td>
                <td valign="top">100%</td>
                <td valign="top">Roche Cobas Amplicor CT assay</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit92">92</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">cAMP factor (<span class="italic">cfb</span>) gene of Group B Streptococci</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">98 genome copies</td>
                <td valign="top">50 vaginal/anal samples collected from women</td>
                <td valign="top">96%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit126">126</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">DNA target sequence specific to <span class="italic">Cryptosporidium spp.</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">100 oocysts per mL stool</td>
                <td valign="top">A total of 10 human stool samples clinically verified to contain Cryptosporidium by a reference laboratory and 11 stool samples from healthy volunteers presumed to be uninfected</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Lower</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit127">127</a>
                </td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">5′-Untranslated region of Yellow fever virus (YFV)</td>
                <td valign="top">Real-time fluorescent detection on the tube scanner</td>
                <td rowspan="2" valign="top">44 genomic copies per reaction in YFV RNA extracts; 21 genomic copies per reaction of YFV-spiked human plasma samples</td>
                <td valign="top">34 samples of monospecific pools of wild-caught mosquitoes collected from Kedougou, southern Senegal</td>
                <td valign="top">80%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td rowspan="2" valign="top">8 genomic copies per reaction in YFV RNA extracts</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit117">117</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Real-time fluorescent detection on the microfluidic platform</td>
                <td valign="top">27 RNA samples of mosquito pools</td>
                <td valign="top">71.4%</td>
                <td valign="top">100%</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">IS6110</span> gene of <span class="italic">Mycobacterium tuberculoss</span> (MTB)</td>
                <td rowspan="2" valign="top">Real-time fluorescent detection</td>
                <td valign="top">6.25 fg</td>
                <td rowspan="2" valign="top">121 specimens including induced and expectorated sputum (<span class="italic">n</span> = 119) and respiratory washes (bronchial and tracheal, <span class="italic">n</span> = 2) collected from a total of 101 tuberculosis suspect cases (no more than 3 specimens/individual were tested)</td>
                <td valign="top">87.5%</td>
                <td valign="top">95.4%</td>
                <td rowspan="2" valign="top">Culture</td>
                <td rowspan="2" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">Higher</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit128">128</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">IS1081</span> gene of <span class="italic">Mycobacterium tuberculoss</span></td>
                <td valign="top">20 fg</td>
                <td valign="top">91.4%</td>
                <td valign="top">100%</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">Giardia</span> beta giardin gene</td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">10<small><sup>3</sup></small>–10<small><sup>3.5</sup></small> cysts per mL of stool</td>
                <td valign="top">104 clinical stool samples</td>
                <td valign="top">73%</td>
                <td valign="top">96%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">10<small><sup>2.5</sup></small> cysts per mL of stool</td>
                <td valign="top">Higher</td>
                <td valign="top">Higher</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit129">129</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">IS6110</span> gene of <span class="italic">Mycobacterium tuberculoss</span></td>
                <td valign="top">Real-time photonic detection</td>
                <td valign="top">10<small><sup>−6</sup></small>-fold diluted MTB sample</td>
                <td valign="top">33 clinical samples including 13 smear and culture positive samples and 20 smear and culture negative samples</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Conventional microscopy of smear and solid culture</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit130">130</a>
                </td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">A highly conserved 3′-untranslated region that cover DENV 1–4</td>
                <td rowspan="2" valign="top">Real-time fluorescent detection</td>
                <td rowspan="2" valign="top">DENV serotype 1: 237 RNA copies; DENV serotype 2: 618 RNA copies; DENV serotype 3: 363 RNA copies; DENV serotype 4: 383 RNA copies</td>
                <td valign="top">Inactivated DENV 1–4 spiked plasma and 31 DENV positive samples in Kedougou region in Senegal</td>
                <td valign="top">98%</td>
                <td align="left" valign="top">—</td>
                <td rowspan="2" valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td rowspan="2" valign="top">—</td>
                <td valign="top">Higher</td>
                <td align="left" valign="top">—</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit131">131</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">RNA of 90 plasma samples extracted and tested between 2012–2013 by RT-PCR in Bangkok (Thailand)</td>
                <td valign="top">72%</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td align="left" valign="top">—</td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">47 kDa gene sequence from the Karp strain of <span class="italic">Orientia tsutsugamushi</span> (47-RPA) and the 17 kDa gene sequence from the Wilmington strain of <span class="italic">Rickettsia typhi</span></td>
                <td rowspan="2" valign="top">Lateral flow strip detection</td>
                <td valign="top">47 kDa gene: 53 DNA copies per reaction</td>
                <td valign="top">10 positive and 10 negative human samples</td>
                <td valign="top">80%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td valign="top">47 kDa gene: 10 DNA copies per reaction</td>
                <td valign="top">Higher</td>
                <td valign="top">Higher</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit95">95</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">17 kDa gene: 20 copies per reaction</td>
                <td align="left" valign="top">—</td>
                <td align="left" valign="top">—</td>
                <td align="left" valign="top">—</td>
                <td valign="top">17 kDa gene: 6 DNA copies per reaction</td>
                <td align="left" valign="top">—</td>
                <td align="left" valign="top">—</td>
              </tr>
              <tr align="left">
                <td valign="top">Ribosomal 18S DNA of <span class="italic">Entamoeba histolytica</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">2.5 fg from serial dilutions of pure DNA extracted from parasites; 40 parasites from spiked stool sample</td>
                <td valign="top">32 samples of DNA extracted from clinical samples</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">2.5 fg from serial dilutions of pure DNA extracted from parasites</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit132">132</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">A sequence designed based on ITS sequences of the <span class="italic">Madurella mycetomatis</span> type strain CBS 109801</td>
                <td valign="top">Gel electrophoresis detection</td>
                <td valign="top">0.23 ng of DNA</td>
                <td valign="top">12 patient biopsy specimens</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Conventional PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit133">133</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Ebola virus (EBOV) nucleocapsid sequence</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">5 genomic copies per reaction of a molecular RNA standard; 15 genomic copies per reaction in EBOV-spiked human plasma samples</td>
                <td valign="top">928 post-mortem swab samples</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit134">134</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Orf virus (ORFV) DNA polymerase gene segments</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 DNA copies</td>
                <td valign="top">22 samples collected from suspected cases of Orf, 8 nasal swabs collected from experimentally infected sheep and 5 samples obtained from healthy goats</td>
                <td valign="top">86%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit135">135</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Leader peptidase A (<span class="italic">LepA</span>) gene of <span class="italic">Streptococcus pneumoniae</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">4.1 genome equivalents per reaction</td>
                <td valign="top">15 blood samples including 11 confirmed culture positive and 4 confirmed culture negative for <span class="italic">Streptococcus pneumoniae</span></td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">5.1 genome equivalents per reaction</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit97">97</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Orf virus (ORFV) DNA polymerase gene segments</td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">80 copies per reaction of DNA plasmid</td>
                <td valign="top">24 ORFV-spiked tissues samples, 53 samples collected from goats with suspected ORFV infection, 8 nasal swabs samples and 5 tissues samples from healthy goats</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit64">64</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">Leishmania donovani</span> (LD) kinetoplast minicircle DNA</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 DNA copies applying the LD DNA linearised plasmid; 1 genomic DNA copy</td>
                <td valign="top">96 buffy coats and skin biopsies collected from visceral leishmaniasis, asymptomatic and post-kala-azar dermal leishmaniasis</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit121">121</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Highly pathogenic porcine reproductive and respiratory syndrome virus (HP-PRRSV) NSP2 gene</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">70 RNA copies per reaction</td>
                <td valign="top">68 tissue samples and 10 serum samples collected from suspected pigs of HP-PRRSV, 35 serum samples and 12 tissue samples collected from healthy pigs</td>
                <td valign="top">97.6%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit136">136</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">100% conserved sequence of a major capsid protein gene of all cyprinid herpesvirus 3 strains</td>
                <td valign="top">Gel electrophoresis detection</td>
                <td valign="top">10 copies of genomic DNA</td>
                <td valign="top">12 confirmed latently infected fish and 1 confirmed uninfected fish</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Lower</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit66">66</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">cAMP factor (cfb) gene of Group B Streptococci</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">6.25–12.5 genome equivalents</td>
                <td valign="top">124 clinical samples</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">3.1–6.25 genome equivalents</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit137">137</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Non-structure protein 1 (nsP1) of Chikungunya virus (CHIKV)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">80 genome copies of extracted RNA from CHIKV isolate LR strain</td>
                <td valign="top">58 suspect Chikungunya fever cases</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td valign="top">80 genome copies of extracted RNA from CHIKV isolate LR strain</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit87">87</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">A sequence designed in NS2A region conserved among all Zika virus lineages</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">21 RNA copies</td>
                <td valign="top">25 positive and 9 negative urine samples collected during the Zika virus epidemic in Tuparetama, Brazil</td>
                <td valign="top">92%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit138">138</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">G-protein-coupled chemokine receptor (GPCR) gene of lumpy skin disease virus (LSDV)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 DNA copies (179 DNA copies by probit analysis)</td>
                <td valign="top">12 negative skin samples and 22 skin nodules of suspected LSDV-infected cattle collected during the summer of 2012 in Dakahlia Governorate, Egypt</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">37 DNA copies</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit139">139</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">IS900</span> gene of <span class="italic">Mycobacterium avium</span> subsp. p<span class="italic">aratuberculosis</span> (MAP)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">16 plasmid copies per μL; 500 fg genomic DNA/reaction</td>
                <td valign="top">Archived DNA of MAP positive blood (<span class="italic">n</span> = 14), sperm (<span class="italic">n</span> = 18), faecal (<span class="italic">n</span> = 12) and tissue (<span class="italic">n</span> = 4) samples and 20 MAP-negative faecal samples</td>
                <td valign="top">89.5%</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">1 plasmid copies per μL; 50 fg genomic DNA/reaction</td>
                <td valign="top">Higher</td>
                <td align="left" valign="top">—</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit140">140</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">T1E4 gene of prostate cancer</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">1000 RNA copies</td>
                <td valign="top">9 urine samples obtained from prostate cancer and 2 urine samples from healthy individuals</td>
                <td valign="top">90%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit141">141</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">NS1 gene of porcine parvovirus (PPV)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">300 DNA copies</td>
                <td valign="top">101 clinical tissue samples (serum, liver, kidney, lymph node, spleen and duodenum) collected from pig farms with suspected cases of PPV in Gansu province, China, and 27 clinical samples (serum, kidney and duodenum) collected from healthy pigs</td>
                <td valign="top">94.4%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit54">54</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Nucleocapsid gene of type 2 porcine reproductive and respiratory syndrome virua (PRRSV)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 RNA copies (690 RNA copies by probit analysis)</td>
                <td valign="top">60 clinical samples (lymph node, lung, spleen and liver) collected from diseased pigs suspected of having PRRS from 5 pigs farms in Hebei province, China from 2015–2016</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td valign="top">100 RNA copies</td>
                <td valign="top">The same</td>
                <td valign="top">Lower</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit142">142</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Cytochrome b gene of <span class="italic">Theileria annulata</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">2 pg genomic DNA</td>
                <td valign="top">17 anticoagulated blood samples collected from tropical theileriosis endemic areas in Gansu province, China</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit67">67</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">pirA-like gene of <span class="italic">Vibrio owensii</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">2 plasmid copies (2.84 plasmid copies by probit analysis)</td>
                <td valign="top">138 clinical shrimp obtained from immersion bioassay, including 70 shrimp acute hepatopancreatic necrosis disease (AHPND) infected shrimp and 68 non-AHPND infected shrimp</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Lower</td>
                <td valign="top">Lower</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit143">143</a>
                </td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">rRNA gene of <span class="italic">Fasciola hepatica</span></td>
                <td valign="top">Gel electrophoresis detection</td>
                <td valign="top">1.6 pg μL<small><sup>−1</sup></small> DNA copies</td>
                <td rowspan="2" valign="top">102 human stool samples selected from banked specimens</td>
                <td valign="top">87.8%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td rowspan="2" valign="top">1.6 pg μL<small><sup>−1</sup></small> DNA copies</td>
                <td valign="top">Lower</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit144">144</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">1.0 pg μL<small><sup>−1</sup></small> DNA copies</td>
                <td valign="top">95.2%</td>
                <td valign="top">90.4%</td>
                <td valign="top">Lower</td>
                <td valign="top">Higher</td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">N gene of pest des petits ruminants virus (PPRV)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 plasmid copies</td>
                <td rowspan="2" valign="top">32 clinical samples collected from suspected cases of PPRV in Gansu province, China and 5 samples obtained from healthy sheep</td>
                <td valign="top">90%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td rowspan="2" valign="top">150 plasmid copies</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit145">145</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">10 plasmid copies</td>
                <td valign="top">90%</td>
                <td valign="top">100%</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">ITS2 gene of <span class="italic">Phytophthora infestans</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">50 fg μL<small><sup>−1</sup></small> of genomic DNA</td>
                <td valign="top">24 potato leaf samples collected from fields with and without visible symptoms of late blight infections in New Brunswick and Quebec provinces, Canada, respectively</td>
                <td valign="top">60%</td>
                <td valign="top">100%</td>
                <td valign="top">LAMP</td>
                <td valign="top">50 fg μL<small><sup>−1</sup></small> of genomic DNA</td>
                <td valign="top">Higher</td>
                <td valign="top">Lower</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit146">146</a>
                </td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">ORF2 gene of porcine circovirus type 2 (PCV2)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 plasmid copies</td>
                <td rowspan="2" valign="top">65 clinical samples (spleen, inguinal lymph node, tonsil, lung and serum) collected from suspected PCV2 infection pigs from 8 pig farms in Shandong province, China; 37 clinical samples (inguinal lymph node, tonsil, lung and serum) collected from Gansu Province, China, and 10 PCV1 positive samples conserved in the laboratory</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td rowspan="2" valign="top">80 plasmid copies</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit69">69</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">100 plasmid copies</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">gD gene of pseudorabies virus</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 DNA copies</td>
                <td rowspan="2" valign="top">76 clinical samples (tonsil, heart, spleen, lymph nodes, lung and serum) collected from pig farms in Shandong province, China, and 26 clinical samples (lymph nodes, tonsil and serum) collected from healthy pigs</td>
                <td valign="top">93.3%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td rowspan="2" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit70">70</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">160 DNA copies</td>
                <td valign="top">93.3%</td>
                <td valign="top">100%</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">B1 gene of <span class="italic">Toxoplasma gondii</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">0.1 oocysts per reaction</td>
                <td valign="top">35 soil samples and 15 water samples collected from parks, residential areas, schools and gutterways in Lanzhou city, Gansu rovince, China, during August 2016</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Nested PCR</td>
                <td valign="top">1 oocyst/reaction</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit71">71</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">RNA transcript of TMPRSS2:ERG (a fusion gene for prostate cancer)</td>
                <td valign="top">RPA fluocculation assay</td>
                <td valign="top">10<small><sup>5</sup></small> RNA copies</td>
                <td valign="top">Clinical urine specimens from 10 metastatic castration-resistant promising prostate cancer patients and 5 healthy control patients</td>
                <td valign="top">70%</td>
                <td valign="top">100%</td>
                <td valign="top">Conventional RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit101">101</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">VP2 gene of porcine parvovirus</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">100 DNA copies (103 DNA copies by probit analysis)</td>
                <td valign="top">115 clinical samples (lymph node, lung, spleen, kidney and duodenum collected from pigs with reproductive disorders, diarrhea or respiratory disease in Hebei province, China from 2014 to 2016</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">100 DNA copies</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit147">147</a>
                </td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">G-protein-coupled chemokine receptor (GPCR) gene of <span class="italic">Capripoxvirus</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">300 plasmid copies</td>
                <td rowspan="2" valign="top">107 clinical samples (liver, lung, kidney, spleen, skin and blood) collected from 14 suspected sheep and 6 suspected goats in Gansu province which were characterised by pyrexia, excessive salivation and generalised pock lesions in the skin during the period of October 2014 to August 2015</td>
                <td valign="top">97%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td rowspan="2" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit148">148</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">300 plasmid copies</td>
                <td valign="top">97%</td>
                <td valign="top">100%</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">Nucleocapsid protein gene of canine distemper virus</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">9.4 RNA copies (31.8 RNA copies by probit analysis)</td>
                <td valign="top">32 nasal/oropharyngeal swabs collected from 20 dogs of both sexes (various breeds and ages) from the animal hospital of Agricultural University of Hebei and 12 raccoon dogs from the farms in Hebei Province, China from 2014 to 2016</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time RT<i><small><sup style="color: red;"><a title="" href="#tab5fna">a</a></sup></small></i>-PCR</td>
                <td valign="top">94 RNA copies</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit149">149</a>
                </td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">
                  <span class="italic">imp</span> gene of <span class="italic">Candidatus</span> Phytoplasma oryzae</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">1–10 plasmid copies</td>
                <td rowspan="2" valign="top">66 Napier grass samples from various geographical locations in western Kenya</td>
                <td valign="top">100%</td>
                <td valign="top">57.1%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td rowspan="2" valign="top">—</td>
                <td rowspan="2" valign="top">Lower</td>
                <td rowspan="2" valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit79">79</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">10–100 plasmid copies</td>
                <td align="left" valign="top">—</td>
                <td align="left" valign="top">—</td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">
                  <span class="italic">imp</span> gene of <span class="italic">Candidatus</span> Phytoplasma mali</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">10 copies of cloned plasmid</td>
                <td rowspan="2" valign="top">38 roots of field samples from apple (<span class="italic">Malus domestica</span>) trees collected in autumn 2014, in spring 2015 and in June 2016 in private orchards or in the experimental field of the Institute for Fruit Growing in Samochvalovichi, Belarus</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td rowspan="2" valign="top">—</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit72">72</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">10 copies of cloned plasmid</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">N gene of rabies</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">1000 RNA copies per μL of strains SAD B19, Bobcat USA and Kelev</td>
                <td valign="top">A panel of RNA from 33 field samples</td>
                <td valign="top">97%</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">1 RNA copies per μL of strains SAD B19, Bobcat USA and Kelev</td>
                <td valign="top">Higher</td>
                <td align="left" valign="top">—</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit150">150</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">KRAS</span> oncogenic mutation gene G12D on Exon 12</td>
                <td rowspan="2" valign="top">Real-time silicon photonic microring-based detection</td>
                <td rowspan="2" valign="top">1% to 100% of the mutant cells</td>
                <td rowspan="2" valign="top">70 frozen tissues samples from colorectal cancer patients in Bio-Resource Center of Asian Medical Center, including 24 samples with the G12D mutation (34.3%), 26 samples with G13D mutation (37.1%) and 20 samples with no mutation (28.6%)</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Conventional PCR</td>
                <td rowspan="2" valign="top">30% to 100% of the mutant cells</td>
                <td valign="top">Lower</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit151">151</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">KRAS</span> oncogenic mutation genes G13D on Exon 13</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Lower</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">A consensus region that covers all 7 S-segment clades of Crimean-Congo Hemorrhagic fever virus (CCHFV)</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">500 RNA copies (251 RNA copies by probit analysis)</td>
                <td valign="top">21 extracted patient sera samples obtained in relation to outbreaks of CCHFV in 2013–2015 in Tajikistan</td>
                <td valign="top">88%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit152">152</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Canine parvovirus 2 (CPV-2) nucleocapsid protein gene</td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">10 copies of recombinant plasmid</td>
                <td valign="top">91 fecal swab samples collected from the dogs from 2012 to 2016</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">10 copies of recombinant plasmid</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit153">153</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">G gene of bovine ephemeral fever virus (BEFV)</td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">8 plasmid copies per reaction (corresponding to 24 RNA copies)</td>
                <td valign="top">104 clinical blood specimens and 24 tissue samples including 16 lung tissue specimens, 8 lymph gland specimens collected from suspected dairy cattle cases of BEFV infections in eastern China</td>
                <td valign="top">97.89%</td>
                <td valign="top">90.91%</td>
                <td valign="top">Real-time PCR</td>
                <td align="left" valign="top">—</td>
                <td valign="top">Higher</td>
                <td valign="top">Higher</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit74">74</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">IS900</span> gene of <span class="italic">Mycobacterium avium</span> subsp. <span class="italic">paratuberculosis</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">8 plasmid copies per reaction</td>
                <td valign="top">320 individual fecal samples collected between September 2016 and September 2017 from 10 different dairy farms located in 10 distinct geographic regions of Shandong province, China</td>
                <td valign="top">100%</td>
                <td valign="top">97.63%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">8 plasmid copies per reaction</td>
                <td valign="top">The same</td>
                <td valign="top">Higher</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit76">76</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <span class="italic">Fno</span> FSC771 hypothetical protein gene of <span class="italic">Francisella noatunensis</span> subsp. <span class="italic">Orientalis</span></td>
                <td valign="top">Real-time fluorescent detection</td>
                <td valign="top">10 plasmid copies (15 plasmid copies by probit analysis)</td>
                <td valign="top">Samples of spleen (<span class="italic">n</span> = 78), head kidney (<span class="italic">n</span> = 78) and water (<span class="italic">n</span> = 5)</td>
                <td valign="top">100%</td>
                <td valign="top">84,89%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">10 plasmid copies (11 plasmid copies by probit analysis)</td>
                <td valign="top">The same</td>
                <td valign="top">Higher</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit154">154</a>
                </td>
              </tr>
              <tr align="left">
                <td rowspan="2" valign="top">VP1 gene of Enterovirus 71 subgenotype C4 (EV71-C4)</td>
                <td rowspan="2" valign="top">Real-time fluorescent detection</td>
                <td rowspan="2" valign="top">3.767<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>10 genomic copies (LGC)</td>
                <td valign="top">Stool samples (<span class="italic">n</span> = 44) collected in 2017 by Shenzhen Center for Disease Control and Prevention</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td rowspan="2" valign="top">Real-time PCR</td>
                <td rowspan="2" valign="top">2.026<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>10 genomic copies (LGC)</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td rowspan="2" valign="top">
                  <a title="Select to navigate to references" href="#cit155">155</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Stool samples (<span class="italic">n</span> = 134) collected from patients with suspected hand–foot–mouth disease at the pediatrics department of Zhujiang Hospital (Southern Medical University, Guangzhou, China) in 2009</td>
                <td valign="top">89.5%</td>
                <td valign="top">100%</td>
                <td valign="top">Lower</td>
                <td valign="top">The same</td>
              </tr>
              <tr align="left">
                <td valign="top">56 Kda gene of a Karp-like strain of <span class="italic">Orientia tsutsugamushi</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">10 copies (recombinant plasmid); 12 copies of genomic DNA</td>
                <td valign="top">62 animal (including <span class="italic">Apodemus agrarius</span>, <span class="italic">Rattus norvegicus</span>, <span class="italic">Microtus fortis</span> and <span class="italic">Neomys fodiens</span>) organ samples including 5 infected animals trapped in the wild, 2 infected in the laboratory and 55 uninfected animals trapped in the wild</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">12 copies of genomic DNA</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit156">156</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">23S rRNA gene of <span class="italic">Coxiella burnetii</span></td>
                <td valign="top">Lateral flow strip detection</td>
                <td valign="top">10 copies (recombinant plasmid); 7 copies of genomic DNA</td>
                <td valign="top">DNA of spleens from 9 5-week old C57BL/6 female <span class="italic">Coxiella burnetii</span>-infected mice and 9 control PBS-infected mice</td>
                <td valign="top">100%</td>
                <td valign="top">100%</td>
                <td valign="top">Real-time PCR</td>
                <td valign="top">7 copies of genomic DNA</td>
                <td valign="top">The same</td>
                <td valign="top">The same</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit157">157</a>
                </td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
        
        <h3 id="sect1575"><span class="b_heading">2.1 Reaction components</span></h3>
        <span>The fundamental reaction mechanism of RPA relies on a synthetically engineered adaptation of a natural cellular process called homologous recombination, a key process in DNA metabolism. The standard RPA reaction reagents comprise three key proteins (recombinase, recombinase loading factor and single-stranded binding protein), which subsequently co-ordinate with ancillary components such as deoxyribonucleic acid (DNA) polymerase, crowding agent, energy/fuel components (<span class="italic">e.g.</span> adenosine triphosphate, ATP) and salt molecules to perform the RPA reaction mechanism (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>).<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> The detailed reaction components, and their typical concentration and function are provided in <a title="Select to navigate to table" href="#tab1">Table 1</a>:</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f2.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">RPA reaction mechanism. The reaction starts from the binding of the recombinase (T4 UvsX) to the primers with the help of the loading factor (T4 UvsY). This forms a nucleoprotein filament that searches for the homologous sequence in the double-stranded DNA. Once the homology is located, the complex invades the duplex DNA, forming a D-loop structure to initiate a strand exchange reaction while the unwound strand is stabilised by the single-stranded binding proteins (T4 gp32). The recombinase (<span class="italic">Bsu</span> or <span class="italic">Sau</span>) disassembles from the nucleoprotein filament once the strand exchange is performed, and will be available for the next pair of primers. Next, the DNA polymerase extends from the 3′ end of primers. As the polymerisation continues, the two parental strands begin to separate and eventually form two duplexes, and then the whole process repeats.<a title="Select to navigate to references" href="#cit31"><sup><span class="sup_ref">31</span></sup></a></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1588"><span class="b_heading">2.2 Mechanism</span></h3>
        <span>RPA starts with the binding of the T4 UvsX protein (recombinase), assisted by the T4 UvsY (loading factor), to the primers to form a nucleoprotein filament. The resulting complex searches for homologous sequences in duplex DNA (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>).<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> Once the homology is located, the complex invades the double-stranded DNA, forming a D-loop structure. One side of the D-loop is double-stranded where the primer hybridises with the template strand, initiating a strand exchange reaction, whereas the other side of the D-loop remains single-stranded – the unwound complementary strand that is stabilised by the SSB proteins (T4 gp32).<a title="Select to navigate to reference" href="#cit29"><sup><span class="sup_ref">29,30</span></sup></a> Subsequently, the recombinase disassembles from the nucleoprotein filament and becomes immediately available to initiate another strand displacement reaction with a new primer. Primer incorporation allows the DNA polymerase (<span class="italic">Bsu</span> or <span class="italic">Sau</span>) to initiate the synthesis from the free 3′-OH at the end of the primer. As the polymerisation continues, the two parental strands continue to separate. Incorporation of both forward and reverse primers enables strand synthesis to occur in both directions simultaneously, and ultimately results in the exponential accumulation of amplified duplex DNA, consisting of the sequence between the forward and reverse primers.</span>
        <p class="otherpara">During RPA, the formation of the recombinase-primer complex is the rate limiting for the D-loop formation.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> It was reported that the D-loop formation was most efficient at the stoichiometries at which the T4 UvsX protein fully coated the primers but did not bind substantially to the double-stranded DNA.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> The SSB proteins and the T4 UvsY (together with the ATP) have been shown to be essential for cooperating in strand exchange reaction along with the T4 UvsX protein.<a title="Select to navigate to reference" href="#cit13"><sup><span class="sup_ref">13,30,32</span></sup></a> However, the presence of both of these proteins requires a higher concentration of T4 UvsX protein than what is required in the presence of only one of these proteins.<a title="Select to navigate to reference" href="#cit29"><sup><span class="sup_ref">29,33,34</span></sup></a> The SSB proteins can stimulate the strand exchange reaction if T4 UvsX degrades.<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a> Importantly, the T4 UvsY protein neutralises the competition between the SSB proteins and the T4 UvsX for binding sites on the primers preventing the SSB from binding the primer from initiating the recombination event.<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> When the primer concentration is low, the SSB proteins inhibits the strand exchange activity of the T4 UvsX protein.<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> However, once the T4 UvsY protein is supplied, the T4 UvsY protein is able to invade the SSB proteins-covered primers to promote the binding of T4 UvsX protein to primers (from a site that is adjacent to the bound T4 UvsY protein), thereby displacing the SSB proteins from the primers.<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a></p>
      
      
        
        <h3 id="sect1606"><span class="b_heading">2.3 Template, primers, probe and their designs</span></h3>
        <span>RPA was initially demonstrated to be a nucleic acid amplification method for DNA,<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> later it was shown that RNA also could be the template by addition of reverse transcriptase (<span class="italic">e.g.</span> Murine Leukemia virus (MuLV) reverse transcriptase) in the same reaction tube.<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> Regardless of nucleic acid template type, the recommended RPA amplicon length should be below 500 nucleotides for efficient amplification. Most published RPA papers have applied amplicon lengths between 100 and 250 nucleotides, which usually incur fast and efficient amplification. However, shorter amplicons (79 nucleotides;<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> 94 nucleotides<a title="Select to navigate to reference" href="#cit38"><sup><span class="sup_ref">38–40</span></sup></a> and longer amplicon up to 1500 nucleotides<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> and 1941 nucleotides<a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a> have also been reported.</span>
        <p class="otherpara">Unlike PCR, the length of RPA primers is relatively long (a recommended minimum of 30 nucleotides, but typically between 32 and 35 nucleotides). Shorter PCR primers (typically between 18 and 25 nucleotides) can also be used in the RPA reaction but may decrease the reaction speed and sensitivity.<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> Application of short PCR primers in RPA has been demonstrated by Mayboroda <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit43"><sup><span class="sup_ref">43</span></sup></a> Martorell <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> and Fuller <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> The latter two authors have shown that the PCR primers used in RPA resulted higher analytical sensitivity of detection compared to their usage in PCR: RPA detected 100 DNA copies of genetically modified GTS 40-3-2 soybean and 3.5 pg of genomic DNA of <span class="italic">Agrobacterium</span> spp. respectively, while the benchmark method PCR detected 1000 DNA copies and 350 pg of genomic DNA respectively.<a title="Select to navigate to reference" href="#cit44"><sup><span class="sup_ref">44,45</span></sup></a></p>
        <p class="otherpara">The company that sells the commercialised RPA reagents, TwistDx™ Ltd (see section 2.4 and <a title="Select to navigate to table" href="#tab2">Tables 2</a> and <a title="Select to navigate to table" href="#tab3">3</a> for more details) provides additional probes that can be incorporated during the RPA reaction. The TwistAmp™ exo probe (typically between 46 and 52 nucleotides) and the TwistAmp® fpg probe (typically between 32 and 35 nucleotides) are used for fluorogenic real-time detection (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3A and B</a>). These two probes are usually labelled with a fluorophore, a quencher (<span class="italic">e.g.</span> Black Hole Quencher) that is in close proximity to the fluorophore, to temporarily deter the fluorescent signal, and a blocker (<span class="italic">e.g.</span> C3-spacer, a phosphate, a biotin-TEG or an amine) at the 3′-end serving to prevent polymerase extension from the 3′-end (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3A and B</a>). The real-time detection is based on cleavage of fluorogenic probes at an abasic site (also known as an apurinic/apyrimidinic site that is a location in DNA (less often in RNA), which has neither a purine nor a pyrimidine base) between the fluorophor and the quencher. The abasic site can either be tetrahydrofuran (THF) or a dSpacer (a derivative of the THF) or a dR group (the deoxyribose of the abasic site <span class="italic">via</span> a C–O–C linker). The <span class="italic">E. coli</span> exonuclease III cleaves the TwistAmp™ exo probe at a THF or a dSpacer site, while the glycosylase/lyase <span class="italic">E. coli</span> fpg cleaves the TwistAmp™ fpg probe at the dR position (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3A and B</a>). After the enzymatic cleavage, the TwistAmp® exo probe can serve as a forward primer. However, the TwistAmp™ fpg probe cannot serve as a primer due to different catalytic mode (beta-elimination) of the glycosylase/lyase <span class="italic">E. coli</span> fpg protein, which does not generate an extendable 3′-OH group but a 3′-phosphate group.<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a></p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f3.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">RPA probes. A: TwistAmp™ <span class="italic">exo</span> probe. This probe is cleaved by the <span class="italic">E. coli</span> exonuclease III at the abasic site (<span class="italic">e.g.</span> tetrahydrofuran, THF) to depart the fluorophore from the quencher and generate an extensible 3′-OH group for polymerisation. B: TwistAmp™ fpg probe. This probe is cleaved by the glycosylase/lyase <span class="italic">E. coli</span> fpg at the dR position (the deoxyribose of the abasic site <span class="italic">via</span> a C–O–C linker) to depart the fluorophore from the quencher and generate a 3′-phosphate group which is non-extensible for polymerisation. C: TwistAmp™ LF probe. This probe is cleaved by the Nfo endonucleases IV at the abasic site (<span class="italic">e.g.</span> tetrahydrofuran, THF) to generate an extensible 3′-OH group for polymerisation. The DNA polymerases extend and displace from 3′-ends of the primers and cleaved probe to produce the minor amplicons (from the forward and reverse primers) and a displaced strand. The displaced strand combines with the labelled reverse primer, and leads to the production of a dual-labelled amplicon (the major amplicon) for the down-stream sandwich assay detection.<a title="Select to navigate to references" href="#cit31"><sup><span class="sup_ref">31</span></sup></a></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">A third probe, the TwistAmp™ LF (typically between 46 and 52 nucleotides), is used for lateral flow strip detection (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3C</a>). This probe is labelled at the 5′-end (<span class="italic">e.g.</span> with fluorescein), has a blocker at the 3′-end, and an internal abasic site (THF or dSpacer). The Nfo endonucleases IV cleaves at this abasic site of the TwistAmp™ LF probe, and generates an extendable 3′-OH group for polymerisation. However, unlike the <span class="italic">E. coli</span> exonuclease III which degrades most of the amplicons during RPA reaction, the Nfo endonuclease IV generates a slower signal and incomplete cleavage to avoid amplicon degradation (also see section 2.8).<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> Therefore, the TwistAmp™ LF probe can also be used for cases when gel electrophoresis (GE) is chosen as a detection method.</p>
        <p class="otherpara">To select suitable RPA templates and to design primers and probes, users can refer to the criteria suggested in the TwistAmp™ reaction kit manual.<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a> In brief, (1) GC content of the DNA template should be between 40% and 60%, and should avoid long homo-polymer tracks, few direct/inverted repeats and palindromes; (2) GC content of the primers should be between 30% and 70%, and should avoid long tracks of guanines at the 5′ end but recommend cytidines; and (3) guanines and cytidines are recommended at 3′-end of the primer for improved performance. From the RPA literature, it is further recommended that users evaluate the melting temperature, hybridisation stability, secondary structures and dimer formations among these oligonucleotides.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> Specific softwares such as BioEdit version 7.0.5.3,<a title="Select to navigate to references" href="#cit48"><sup><span class="sup_ref">48</span></sup></a> Primer3,<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> UNAFold,<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> mFOLD,<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> Oligoanalyzer 3.1 (IDT, Leuven, Belgium), PrimerQuest software (Integrated DNA Technologies, Coralville, IA) and Visual OMP (DNA software, MI, USA) have been used in the literature for analysing RPA oligonucleotide properties. An effective way to avoid primer dimer formation is to employ self-avoiding molecular recognition systems (SAMRS), by including SAMRS nucleotides 2-aminopurine-2′deoxyriboside (A*), 2′-deoxy-2-thiothymidine (T*), 2′-deoxyinosine (G*) and <span class="italic">N</span>4-thyl-2′-deoxycytidine (C*) in the primers.<a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a> The inclusion of these SAMRS nucleotides strategically replaces the hydrogen-bonding units from natural A pairs with T (and G pairs with C) to SAMRS A* pairs with natural T, SAMRS T* pairs with natural A, SAMRS G* pairs with natural C and SAMRS C* pairs with natural G. However, the SAMRS A* and SAMRS G* nucleotides do not interact with the SAMRS T* and SAMRS C* nucleotides respectively no matter what their concentration, and in this way, the undesired products due to primer dimer can be avoided during nucleic acid amplification.<a title="Select to navigate to reference" href="#cit52"><sup><span class="sup_ref">52,53</span></sup></a> Such SAMRS system has been demonstrated in RPA reaction by Sharma and co-workers, and was shown to successfully eliminate RPA artifacts.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> In addition, caution is needed to avoid overlap between the primer and probe which has shown to impede the desired amplification efficiency.<a title="Select to navigate to reference" href="#cit54"><sup><span class="sup_ref">54,55</span></sup></a> Collectively speaking, it is sufficient to follow the described guidelines as a starting point for <span class="italic">in silico</span> optimal design of the RPA oligonucleotide candidates, however, the resulting candidates should be screened through RPA reactions to select the preferred final oligonucleotide set applicable for a specific RPA assay (<span class="italic">e.g.</span> TwistDx™ Ltd recommends designing five forward and reverse primers, and 3 probes).</p>
      
      
        
        <h3 id="sect1670"><span class="b_heading">2.4 Commercial kits and instrumentation by TwistDx™</span></h3>
        <span>All the RPA reagents are available for commercial purchase through TwistDx™, a subsidiary of Abbott.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> The company provides various kits for RPA reactions that can be customised towards specific applications by the end user. The company also sells RPA kits for the detection of specific food-borne pathogens (<span class="italic">e.g. Listeria monocytogenes</span>, Campylobacter and <span class="italic">Salmonella enterica</span>) (<a title="Select to navigate to table" href="#tab2">Table 2</a>). The company not only provides RPA reagents in liquid format, but also in lyophilised pellet format which allows in-field application. These lyophilised pellets have shelf-lives up to 12 weeks at 25 °C or up to 3 weeks at 45 °C.<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> In addition, TwistDx™ offers a custom freeze-drying service to create RPA reaction pellets containing primers, probes, and concentrations of protein components or other components (<span class="italic">e.g.</span> internal control DNA or RNA species), which can be encased in various holding vessels with different volumes.<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a></span>
        <p class="otherpara">Apart from various RPA reaction kits, TwistDx™ also develops tailor-made devices and accessories for RPA reactions; these devices and accessories enable incubation, dispensing, mixing, detection, power supply and portability (<a title="Select to navigate to table" href="#tab3">Table 3</a> and <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>). The Twirla™ device is a hand-held sized battery-powered incubator, which allows up to six parallel RPA reactions and subsequent end-point detection (<span class="italic">e.g.</span> gel electrophoresis and lateral flow strip detection; <a title="Select to navigate to figure" href="#imgfig4">Fig. 4A</a>) (note that the Twirla™ is an upgraded version of an earlier Twista® device that does not support constant mixing during incubation, which has now been discontinued). Alternatively, the T8-ISO allows up to eight parallel incubations and two-channel real-time fluorescent detection per tube (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4B</a>). The T16-ISO is an advanced version of the T8-ISO, which supports up to sixteen parallel reactions with three-channel fluorescent detection per tube (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4C</a>). The T8-ISO and the T16-ISO can be powered by mains-power supply, micro USB, or PowerGorilla external battery (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4D</a>). Moreover, all three incubators mentioned so far support magnetic mixing (programmed or constant) when fitted with Micro Ball(s) (0.2 mL; dispensed with Micro Ball Dispenser) in the reaction tube (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4E</a>). For lateral flow strip detection, TwistDx™ provides four different lateral flow devices: Milenia HybriDetect 1, Milenia HybriDetect 2 (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4F</a>), PCRD Nucleic Acid Detection (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4G</a>) and U-Star Disposable Nucleic Acid Lateral Flow Detection Units (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4H</a>). The Milenia HybriDetect 1 and the U-Star Disposable Nucleic Acid Lateral Flow Detection Units allow single-plex detection while the other two devices allow duplex detection. All the lateral flow devices except for the PCRD Nucleic Acid Detection device are based on sandwich assay using gold nanoparticles as tracer; the PCRD Nucleic Acid Detection device employs carbon nanoparticles, which can be more sensitive than the gold nanoparticles.<a title="Select to navigate to reference" href="#cit59"><sup><span class="sup_ref">59,60</span></sup></a> The Milenia HybriDetect are provided as strips, whereas the PCRD Nucleic Acid Detection strips are encased in a semi-sealed cartridge, and the U-Star Disposable Nucleic Acid Lateral Flow Detection strips are embedded in a sealed cartridge designed to enable the RPA reaction to flow to the strips in a completely closed environment that prevents cross-contamination of amplified products.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f4.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">RPA devices and accessories. (A): Twirla™ Portable Mixing Incubator. (B): T8-ISO Instrument. (C): T16-ISO Instrument. (D): PowerGorilla external battery. (E): Micro Balls (0.2 mL, left) and Micro Ball Dispenser (right). (F): Milenia HybriDetect lateral flow strips. (G): PCRD Nucleic Acid Detection device. (H): U-Star Disposable Nucleic Acid Lateral Flow Detection Units device. Reprinted and reproduced with permission from TwistDx™ Limited. Copyright 2009–2018 TwistDx™ Limited.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1696"><span class="b_heading">2.5 Influence of temperature and agitation</span></h3>
        <span>For RPA reactions to achieve optimal efficiency and analytical sensitivity, the choice of target sequence and the designs of corresponding primers and probe are the intrinsic determinants, however, the reaction temperature and agitation during RPA reaction are two of the most important contributing extrinsic factors.</span>
        <p class="otherpara">The recommended RPA reaction temperature is between 37 °C and 42 °C,<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> and Crannell <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> and Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> have also demonstrated that RPA reaction can be performed using body temperature, which can be used advantageously for in-field application. However, several research groups have studied RPA reaction temperatures that lie outside of the recommended range.<a title="Select to navigate to reference" href="#cit38"><sup><span class="sup_ref">38,44,45,60,62–78</span></sup></a> The largest temperature range was tested between 15 °C and 50 °C;<a title="Select to navigate to reference" href="#cit62"><sup><span class="sup_ref">62,64,69,70,76</span></sup></a> and results indicated the marginal reaction temperature to produce a positive result should be greater than 30 °C.<a title="Select to navigate to reference" href="#cit62"><sup><span class="sup_ref">62–64,66,67,69,71,74,76,77</span></sup></a> However, Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> and Poulton and Webster<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> showed that temperature as low as 25 °C could still generate a positive signal after RPA amplification and subsequent lateral flow strip detection. Moreover, Lillis <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> showed that the ambient temperature also had an effect on RPA reaction: the RPA reaction was unstable if the ambient temperature was below 10 °C, however, extension of the reaction time could improve positive results attainability. Such reaction temperature range studies indicate that RPA reaction does not require precise temperature control.</p>
        <p class="otherpara">While reaction temperature provides a suitable working environment for the RPA enzymes, agitation increases the interactions among the RPA components in a homogenous reaction solution. TwistDx™ recommends the user performing two times mixing steps for the RPA reaction, one is at the beginning of the process and the other is after 4 minutes of the reaction. The former is to mix all the RPA reagents to initiate the reaction, the latter is to prevent from local depletion of the reaction reagents, thereby increasing the reaction rate. Wambua <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> reported that threshold fluorescence values were reached in 5–8 minutes when agitation was performed after 4 minutes, whereas the time to reach detectable levels ranged between 8 and 14 minutes without this agitation. In addition, constant shaking throughout the RPA reaction has been shown to further accelerate the RPA reaction rate, achieve more stable positive results and improve sensitivity, especially when the template concentration is close to the limit of detection.<a title="Select to navigate to reference" href="#cit57"><sup><span class="sup_ref">57,62,80</span></sup></a> Kersting <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> reported that constant shaking resulted in faster and more stable signals on the lateral flow strips than with the recommended two-shaking event. Kalsi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a> also reported that continuous mixing of microdroplets from a RPA exo assay led to faster time to result, increased fluorescence and improved sensitivity. In addition, Moody <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit81"><sup><span class="sup_ref">81</span></sup></a> built up a mathematical model and showed that mechanical stirring is better than manually shaking to eliminate inter-operator variations and obtain consistent quantitative experimental result; yet the ideal mixing frequency is assay dependent, and should be determined prior to the reaction.<a title="Select to navigate to reference" href="#cit57"><sup><span class="sup_ref">57,81</span></sup></a> Nevertheless, if a shaking condition is not available, Lillis <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> demonstrated that a decrease of the reaction volume (<span class="italic">e.g.</span> from 50 μL to 5 μL) could compensate for the shaking effect, as smaller volume increased interactions between the reagents and oligonucleotides required for the amplification.</p>
      
      
        
        <h3 id="sect1728"><span class="b_heading">2.6 Tolerability to mismatches, inhibitors and background DNA</span></h3>
        <span>Apart from temperature and agitation, tolerability to mismatches, inhibitors and background DNA are other vital factors for efficient and sensitive RPA reaction. RPA has the ability to tolerate mismatches, and the highest mismatch tolerability reported so far is nine nucleotide base pairs across the primer and probe binding sites.<a title="Select to navigate to reference" href="#cit82"><sup><span class="sup_ref">82–89</span></sup></a> Studies also showed that the mismatches at the 5′-end or centre of primers only mildly affect the RPA reaction, but mismatches located at the 3′-end of primers significantly affect the reaction.<a title="Select to navigate to reference" href="#cit84"><sup><span class="sup_ref">84,86</span></sup></a> This is consistent with the RPA reaction mechanism (see section 2.2), since the polymerase extends the primers and probe (once cleaved) from the 3′-terminus. A useful application for such mismatch sensitivity at the 3′-end is to distinguish single nucleotide polymorphism (SNP). Yamanaka <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a> applied this property to differentiate polymorphisms for the tobacco use disorder genes; the DNA polymerase extension was efficient when the 3′-terminal base of a primer matched its target, whereas the DNA polymerase extension was inefficient or non-existent when the 3′-terminal base was mismatched.<a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a> However, of the general RPA mismatch tolerability (outside of the 3′-end of the primer) can be advantageous, as it enables some flexibility in primer design for highly polymorphic targets, where long conserved target regions are hard to locate. Conversely, the drawback of such mismatch tolerability is a tendency towards non-specific detection of closely-related species. Indeed, non-specific detections have been observed by Patel <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> Moore <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> and Yang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> when detecting chikungunya virus, epidemic human noroviruses and porcine circovirus Type 2, respectively.</span>
        <p class="otherpara">When testing clinical or field samples, numerous substances (<span class="italic">e.g.</span> inhibitors) are either present or could be introduced during sample preparation and processing steps, which can potentially interfere with nucleic acid amplification. RPA has been demonstrated to tolerate certain (PCR) inhibitors, including: (1) haemoglobin (20 g L<small><sup>−1</sup></small>), heparin (0.5 U) and urine (1.25%) showed no effect on RPA reaction;<a title="Select to navigate to reference" href="#cit62"><sup><span class="sup_ref">62,91</span></sup></a> and (2) haemoglobin (50 g L<small><sup>−1</sup></small>), ethanol (4% v/v) and urine (up to 5%), which only slightly affected reactions.<a title="Select to navigate to reference" href="#cit62"><sup><span class="sup_ref">62,91,92</span></sup></a> However, RPA reaction was totally inhibited in the presence of SDS (0.05% v/v) and urine (10%).<a title="Select to navigate to reference" href="#cit62"><sup><span class="sup_ref">62,91</span></sup></a> It was also observed that RPA reactions were more susceptible to inhibitors when the DNA template concentration was close to the limit of detection.<a title="Select to navigate to reference" href="#cit62"><sup><span class="sup_ref">62,91</span></sup></a> However, it is also important to carefully consider the choice of extraction buffer or incubation medium for the sample preparation and processing steps, as these working solutions may also contain potential inhibitors. For example, Valasevich and Schneider<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> found that Cetyltrimethyl ammonium bromide (CTAB) DNA extraction buffer strongly inhibited RPA reaction. Similarly, Liu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit93"><sup><span class="sup_ref">93</span></sup></a> found that selenite cystine broth (bacterial enrichment medium) significantly affected RPA reactions, resulting in a large number of primer dimers that led to false positive results on the lateral flow strip detection.</p>
        <p class="otherpara">In addition to tolerating inhibitors, RPA is capable of amplifying target nucleic acids in the presence of background DNA.<a title="Select to navigate to reference" href="#cit94"><sup><span class="sup_ref">94–97</span></sup></a> However, similar to the tolerability for inhibitors, the tolerability for background DNA is also concentration dependent. Clancy <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> observed that the RPA reaction was significantly inhibited when 400 ng of background human DNA was present, but was much less inhibited when 200 ng of background human DNA was present. Rohrman and Richards-Kortum<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> showed that RPA was completely inhibited by 0.5 μg of sheared salmon sperm DNA when 50 copies of human immunodeficiency virus-1 (HIV-1) target DNA were present, while only inhibited by 2 or 5 μg of sheared salmon sperm DNA when 10<small><sup>3</sup></small> or 10<small><sup>6</sup></small> copies of the target DNA were present respectively. In addition, Rohrman and Richards-Kortum<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> also pointed out that the primer, probe and target sequences used in the assay could influence the maximum background DNA concentration tolerability. Both HIV-1 and <span class="italic">Plasmodium falciparum</span> RPA assays were completely inhibited by 2 μg of sheared salmon sperm DNA respectively when 10<small><sup>3</sup></small> copies of HIV-1 and <span class="italic">Plasmodium falciparum</span> target DNAs were present.<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> However, when the same amount of target DNA were present (10<small><sup>3</sup></small> copies), the <span class="italic">Entamoeba histolytica</span> and <span class="italic">Giardia duodenalis</span> assays were completely inhibited only by 1 and 0.5 μg of sheared salmon sperm DNA, respectively.<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a></p>
      
      
        
        <h3 id="sect1770"><span class="b_heading">2.7 Nucleic acid labelling during RPA</span></h3>
        <span>One vital process for diverse down-stream RPA applications (<span class="italic">e.g.</span> lateral flow strip detection and enzyme-linked immunosorbent assay) is to incorporate labels into nucleic acid template during RPA reaction, so that the incorporated labels allow capture, detection and/or assist the signal generation of RPA assays. Such nucleic acid labelling can be achieved terminally using 5′-labelled primers or internally <span class="italic">via</span> labelled nucleotides.<a title="Select to navigate to reference" href="#cit39"><sup><span class="sup_ref">39,40,98–104</span></sup></a> The labels used for nucleic acid labelling can be fluorescent entity (<span class="italic">e.g.</span> fluorescein), ligand (<span class="italic">e.g.</span> biotin) or even a short segment of nucleotides (overhang). Most terminal nucleic acid labelling using RPA employs both 5′-labelled forward and backward primers, such that the amplicons possess dual-labels that can be captured and detected by corresponding recognition molecules in down-stream assays. However, RPA only tolerates to certain labels <span class="italic">via</span> 5′-labelling process. Crannell <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a> reported a failure of RPA incorporation of five different 5′-labels (Cy5, Cy3, bromodeoxyuridine, tetrachlorofluorescein and hexachlorofluorescein) compared to successful incorporation with three 5′-labels, Alexa Fluor488, fluorescein and digoxigenin.</span>
        <p class="otherpara">For internal nucleic acid labelling during RPA, the reaction mixture can be supplemented with labelled nucleotides, mostly using digoxigenin-dUTPs, which randomly substitute dTTPs during polymerase extension to create labelled amplicons.<a title="Select to navigate to reference" href="#cit101"><sup><span class="sup_ref">101–104</span></sup></a> In comparison to the terminal labelling, the internal labelling allows more labels to be incorporated into a single nucleic acid template, thus having more binding opportunities in down-stream assays. However, terminal labelling can be a better choice when the down-stream application is for a sandwich assay, as the two labels incorporated <span class="italic">via</span> terminal labelling are further apart (separated by the length of amplicon), which could prevent steric hindrance of binding if the labels were too close together.</p>
      
      
        
        <h3 id="sect1785"><span class="b_heading">2.8 Amplicon clean-up and post-amplification treatment</span></h3>
        <span>The above-mentioned issues have considered the conditions both before and during RPA reactions. In addition, post-reaction procedures are critical for successful RPA signal detections, and should be determined according to the intended use of RPA amplicons. The generation of RPA amplicons are RPA reaction kit dependent. Usage of the TwistAmp® Basic kit (also the Basic RT kits) produces a single amplicon from the forward and reverse primers. Conversely, usage of the TwistAmp® exo kit (also the exo RT kits) and the TwistAmp® fpg kits do not produce a single amplicon, the former is due to the exonuclease present in the reaction mixture digesting most of the amplicons during RPA reaction,<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> and the latter is due to the glycosylase/lyase <span class="italic">E. coli</span> fpg cleavage generating a non-extensible 3′-phosphate group (also see section 2.3).<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> For the TwistAmp® nfo kit, however, two types of amplicons are generated, due to the DNA polymerase displacement activity to the probe-primed template: a dual-labelled amplicon emerges as a short product from the probe and one of the primers, whereas a singlely-labelled amplicon emerges as a longer product from the forward and reverse primers (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3C</a>) (note that only the dual-labelled product will generate a positive signal in the test zone of a lateral flow strip detection based on a sandwich assay).<a title="Select to navigate to reference" href="#cit105"><sup><span class="sup_ref">105–108</span></sup></a></span>
        <p class="otherpara">Nevertheless, the RPA amplicons are initially associated with proteins and crowding agents, and the resulting DNA–protein-crowding agent complexes prevent direct use of DNA molecules for gel electrophoresis detection.<a title="Select to navigate to reference" href="#cit109"><sup><span class="sup_ref">109,110</span></sup></a> This is because these complexes affect the proper migration of the amplicons in gel electrophoresis, leading to a lump of smears on the gel pattern. Several methods have been reported in the literature to process RPA amplicons before gel electrophoresis detection, these include protein denaturation by heating (at 65 °C or 95 °C for 10 minutes) and detergent treatment (<span class="italic">e.g.</span> sodium dodecyl sulfate, SDS), enzymatic digestion (<span class="italic">e.g.</span> proteinase K), protein sedimentation <span class="italic">via</span> high-speed centrifugation and purification using commercial DNA clean-up kit.<a title="Select to navigate to reference" href="#cit37"><sup><span class="sup_ref">37,109–113</span></sup></a> Among these methods, the heating method worked equivalently to the methods by proteinase K digestion or SDS treatment.<a title="Select to navigate to reference" href="#cit109"><sup><span class="sup_ref">109,111</span></sup></a> However, heating at 65 °C for 10 minutes showed better result than that of heating at higher temperature (95 °C).<a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> SDS treatment (20% in loading buffer) generated brighter and thicker gel bands than the proteinase K digestion method (0.2 mg mL<small><sup>−1</sup></small> or 20 mg mL<small><sup>−1</sup></small>);<a title="Select to navigate to references" href="#cit109"><sup><span class="sup_ref">109</span></sup></a> heating at 65 °C for 10 minutes generated equivalent brightness gel bands to the ones generated by SDS treatment method (5% or 10% in the loading buffer) in Londono and co-workers’ results, however, Kapoor and co-workers showed that the SDS treatment method (5% in the loading buffer) resulted in brighter gel bands than the heating method (65 °C for 10 minutes) but also resulted in a smear-like pattern above the target band.<a title="Select to navigate to reference" href="#cit111"><sup><span class="sup_ref">111,112</span></sup></a> In comparison to the heating, proteinase K digestion and SDS treatment methods, usage of the commercial DNA clean-up kit produced only the target band but in a much lower band intensity.<a title="Select to navigate to reference" href="#cit37"><sup><span class="sup_ref">37,109,111</span></sup></a> In addition, as an alternative method, centrifugation (3 minutes) to pellet RPA proteins showed equivalent performance to the heating method (65 °C for 10 minutes).<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a></p>
        <p class="otherpara">As with lateral flow strip detection, direct usage of RPA amplicons is possible, but it is recommended to dilute the amplicons with the running buffer (<span class="italic">e.g.</span> 1/100 dilution) before running on the strip to (1) improve its wicking performance<a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a> and (2) avoid “ghost band” effects.<a title="Select to navigate to reference" href="#cit45"><sup><span class="sup_ref">45,54,115–117</span></sup></a> Notably, Powell and co-workers pointed out that the viscous wicking problem on the lateral flow strip can be mitigated by replacing the high molecular weight PEG (5.5% 35 kDa; see also section 2.1) with the low molecular weight PEG (6.5% 3 kDa) in the RPA reagent formulae.<a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a> Moreover, Powell and co-workers also developed methodology to alleviate the dilution step for lateral flow strip detection. They found that sometimes the RPA amplicons are being rendered unavailable in the “RPA globule” (the core of nucleic acid amplification which contains localised RPA reagents; <a title="Select to navigate to figure" href="#imgfig5">Fig. 5A</a>), the formation of which is highly associated with the PEG, for binding to the test line of lateral flow strip. However, applying a dual-labelled probe (two labels are connected <span class="italic">via</span> short length linkers) enabled escape from the “RPA globule” after the enzymatic cleavage, permitting amplicon ready for down-stream sandwich assay detection (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5B</a>).<a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a></p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f5.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">Influence of TwistAmp® nfo formulae and reaction mechanism on lateral flow strip detection. (A) The TwistAmp® nfo amplicons tend to be trapped in a RPA globule (a core of nucleic acid amplification that contains localised RPA reagents), and are consequently impended for binding to the test line of lateral flow strip. (B) The introduction of a dual-labelled probe (two labels are connected <span class="italic">via</span> short length linkers) allows escape from the RPA globule trap after enzymatic cleavage. The escaped cleaved dual-labelled probes are readily available for the down-stream sandwich assay detection. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit114">ref. 114</a>. Copyright 2017 Powell <span class="italic">et al.</span> Published by Elsevier Inc.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1822"><span class="b_heading">2.9 Sensitivities and specificities</span></h3>
        <span>The sensitivities and specificities of RPA can be evaluated in two categories, namely analytical and clinical (or field). The analytical sensitivity indicates the lowest amount of analyte that an assay can detect (also known as limit of detection); the analytical specificity is the ability of an assay to measure one particular analyte rather than others in a sample.<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a> In comparison, the clinical sensitivity is the percentage of correct detection of positive clinical samples, while the clinical specificity is the percentage of correct detection of negative clinical samples.</span>
        <p class="otherpara">From the analytical sensitivity and specificity perspective, RPA is very sensitive and can detect as little as a few molecules (copies) of the analyte, which approaches the analytical sensitivity of PCR (<a title="Select to navigate to table" href="#tab5">Table 5</a>). Furthermore, ultra-sensitive detection down to even a single copy of the analyte can also be achieved in RPA (<a title="Select to navigate to table" href="#tab4">Table 4</a>). In most cases, RPA is very specific for distinguishing one species from other non-closely related species, however, the natural function of these enzymes for performing homology directed repair becomes a disadvantage of RPA to discriminate towards closely-related species, especially when these species share high sequence similarity.<a title="Select to navigate to reference" href="#cit69"><sup><span class="sup_ref">69,87,88</span></sup></a> Conversely, this indicates that RPA can tolerate to a certain degree of primers or probe mismatches to the target sequence (more details see section 2.6).</p>
        <p class="otherpara">Apart from measuring the analytical sensitivity and analytical specificity, many researchers have applied RPA for testing clinical or field samples, and compared the results with a standard method (mostly PCR). From the summary <a title="Select to navigate to table" href="#tab5">Table 5</a>, the clinical sensitivity of RPA is only half as sensitive as the benchmark method, whereas the clinical specificity of RPA is most of time as specific as the benchmark method. These results indicate that RPA may (in some cases only), mis-detect a positive sample (false negative), but is unlikely to show a false positive. In short, RPA is still at the beginning of undergoing clinical/field test evaluation, but not yet matured to be a routine test in the clinical/field settings.</p>
      
    
    
      
      <h2 id="sect1833"><span class="a_heading">3. Distinctive RPA detection methods</span></h2>
      <span>In the proceeding sections, we have discussed how RPA adapts to commonly used PCR detection methods, such as real-time fluorescent detection, gel electrophoresis and lateral flow strip detection. However, a myriad of different detection methods have been coupled with RPA, including flocculation assay, electrochemical, chemiluminescent, silicon microring resonator (SMR)-based photonic and surface-enhanced Raman scattering (SERS) detections. This section describes these distinctive RPA detection methods and discusses their advantages, which have enabled RPA assays to be more efficient and sensitive, yet sometimes simpler and faster.</span>
      
        
        <h3 id="sect1837"><span class="b_heading">3.1 Flocculation assay detection</span></h3>
        <span>Flocculation assay detection is based on a bridging flocculation phenomenon in colloid chemistry described by Ruehrwein and Ward (in 1952),<a title="Select to navigate to references" href="#cit158"><sup><span class="sup_ref">158</span></sup></a> and later explained by La Mer and Healy (in 1960s).<a title="Select to navigate to reference" href="#cit159"><sup><span class="sup_ref">159–161</span></sup></a> The basic principle of bridging flocculation involves the use of long polymers to cross-link multiple particles and thus flocculate out of solution at a specific buffer condition (<span class="italic">e.g.</span> pH and salt concentration; <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>). A RPA reaction in combination with a flocculation assay detection was first demonstrated by Wee and co-workers, where RPA amplicons greater than 100 bp (from a plant pathogen) that resembled long polymers were precipitated onto a magnetic bead surface (at low buffer pH).<a title="Select to navigate to references" href="#cit162"><sup><span class="sup_ref">162</span></sup></a> The resulted conjugated particles underwent flocculation due to cross-linking of magnetic beads <span class="italic">via</span> RPA amplicons. The flocculation could only be triggered with amplified nucleic acids of lengths above 100 nucleotides, which is much longer than standard RPA primers; RPA reaction with no or non-target template did not produce such long “DNA polymer segments” and thus incurred no flocculation.<a title="Select to navigate to references" href="#cit162"><sup><span class="sup_ref">162</span></sup></a> Following from this first application, such RPA-flocculation assays were further applied to detect gene-specific DNA methylation,<a title="Select to navigate to references" href="#cit163"><sup><span class="sup_ref">163</span></sup></a><span class="italic">Mycobacterium tuberculosis</span><a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> and prostate cancer biomarkers (TMPRSS2:ERG).<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a> Analytical sensitivity of these detections were 10% methylation (from 5 ng of starting material),<a title="Select to navigate to references" href="#cit163"><sup><span class="sup_ref">163</span></sup></a> 10 bacteria colony forming unit (CFU)<a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> and 10<small><sup>5</sup></small> copies of TMPRSS2:ERG RNA (equivalent to a single cell),<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a> respectively. In addition, the detection of TMPRSS2:ERG biomarker underwent clinical sample test, and achieved 70% clinical sensitivity and 100% clinical specificity in comparison to the standard method reverse transcription-PCR.<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a></span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f6.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">RPA in combination with flocculation assay detection. The RPA amplicons are incubated with magnetic beads at low pH buffer condition. Consequently, the precipitated RPA amplicons on the magnetic bead surfaces cross-link multiple other RPA-magnetic bead conjugates, and thus flocculate out of solution, causing a sharp transition between solution phase and flocculate. RPA reactions with no or non-target template do not produce long “DNA polymer segments”, hence no flocculation takes place. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit163">ref. 163</a>. Copyright 2015 Springer Nature.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">The flocculation assay detection is a good alternative to the commonly used end-point detection methods (<span class="italic">e.g.</span> lateral flow strip detection) for quick qualitative detection of RPA amplicons. The total detection time is within ten minutes and only requires a minimum amount of RPA amplicons (10% of the reaction volume). Such detection method does not involve any DNA labelling or modifications during RPA reaction, and it also does not require equipment to confirm the RPA amplicons. The detection is a sharp transition between solution phase and flocculate, and is therefore better as naked-eye visualisation in comparison to other colourimetric visualisation detections.</p>
      
      
        
        <h3 id="sect1862"><span class="b_heading">3.2 Electrochemical detection</span></h3>
        <span>Another alternative detection strategy for RPA is electrochemical detection, which employs electrochemically active compounds to produce a signal in relation to the amplified nucleic acids. Most electrochemical approaches for RPA detection measure amperometric signals from RPA-enzyme-linked immunosorbent (ELISA) assay or RPA-enzymatic assay (more details please refer to literature<a title="Select to navigate to reference" href="#cit37"><sup><span class="sup_ref">37,38,43,104</span></sup></a>) due to the electrochemically active property of 3,3′,5,5′-tetramethylbenzidine (TMB) (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a>).<a title="Select to navigate to reference" href="#cit68"><sup><span class="sup_ref">68,101,102,165–167</span></sup></a> Results suggest that electrochemical detection could be up to 10<small><sup>5</sup></small>-fold more sensitive than optical detection (by ELISA).<a title="Select to navigate to references" href="#cit166"><sup><span class="sup_ref">166</span></sup></a> Alternatively, Ng <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> and Lau <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a> have applied gold nanoparticles as signal transducers for RPA amplicons detection. The gold nanoparticles can selectively bind to the RPA amplicons <span class="italic">via</span> specific conjugation, and transfers concentration of RPA amplicons into a measurable electrochemical signal. Again, such electrochemical detection exhibited high analytical sensitivity; Ng <span class="italic">et al.</span> was able to detect as low as 1 CFU of <span class="italic">Mycobacterium tuberculosis</span> bacilli DNA, while Lau <span class="italic">et al.</span> could detect down to 214 pM of <span class="italic">Pseudomonas syringae</span> (pathogenic bacteria for crops) DNA, which is 100 times more sensitive than RPA-GE detection. Recently, Tsaloglou <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit168"><sup><span class="sup_ref">168</span></sup></a> applied [Ru(NH<small><sub>3</sub></small>)<small><sub>6</sub></small>]<small><sup>3+</sup></small> as an electroactive mediator for RPA-electrochemical detection. The [Ru(NH<small><sub>3</sub></small>)<small><sub>6</sub></small>]<small><sup>3+</sup></small> can bind to the double-stranded DNA analogous to the intercalating fluorescent dye SYBR Green I, consequently causing a drop in the diffusion-controlled current as more double-stranded DNAs are synthesised during RPA. This ruthenium compound-based electrochemical detection achieved 11 CFU mL<small><sup>−1</sup></small> of <span class="italic">Mycobacterium tuberculosis</span> analytical sensitivity, which is even more sensitive than the GeneXpert MTB/RIF (Cepheid Inc.) detection (a World Health Organisation recommended tuberculosis diagnostic system that employs PCR real-time fluorescent detection; 131 CFU mL<small><sup>−1</sup></small>).</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f7.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">RPA in combination with electrochemical detection. The example demonstrates electrochemical detection from a RPA-enzymatic assay. The RPA-enzymatic assay is performed in an asymmetric manner, where one of the RPA primers is pre-immobilised on the solid surface, and another RPA primer, together with the DNA template and RPA reagents are free in the solution. After the RPA reaction, streptavidin-conjugated horseradish peroxidase is added, and followed by addition of electrochemically active 3,3′,5,5′-tetramethylbenzidine (TMB). The generated electrochemical signal can be detected on a screen printed carbon electrode.<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">It is obvious that the RPA-electrochemical detection is advantageous for high analytical sensitivity. In addition, RPA-electrochemical detection is fast, low cost and field amenable, as it can be implemented easily with small components manufactured by inexpensive materials (<span class="italic">e.g.</span> screen printed carbon electrodes) with signal measured using a portable device (<span class="italic">e.g.</span> μSTAT 400 bipotentiostat/galvanostat).<a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a></p>
      
      
        
        <h3 id="sect1898"><span class="b_heading">3.3 Chemiluminescent detection</span></h3>
        <span>As an alternative to the fluorescent probe-based detection that requires a light source, RPA reactions can also be detected <span class="italic">via</span> chemiluminescence. The chemiluminescent detection converts chemical energy into the emission of visible light (luminescence) as the result of an oxidation or hydrolysis reaction.<a title="Select to navigate to references" href="#cit169"><sup><span class="sup_ref">169</span></sup></a> Seidel's research group has been applying RPA coupled with chemiluminescent detection of water-borne microbes on flow-based microarrarys.<a title="Select to navigate to reference" href="#cit170"><sup><span class="sup_ref">170,171</span></sup></a> Their chemiluminescent detection employs conversion of energy from oxidation between luminol and peroxide catalysed by horseradish peroxidase to give off luminescent signals detected by a charge-coupled device (CCD) camera (<a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>).<a title="Select to navigate to reference" href="#cit170"><sup><span class="sup_ref">170,171</span></sup></a> Seidel and co-workers first demonstrated a triplex detection of Human adenovirus 41(HAdV 41), Phi X 174 (a virus) and the bacterium <span class="italic">Enterococcus faecalis</span> on the chemiluminescent flow-based microarray, and achieved the limit of detections of 35 genomic units (GU) μL<small><sup>−1</sup></small>, 1 GU μL<small><sup>−1</sup></small> and 5 × 10<small><sup>3</sup></small> GU μL<small><sup>−1</sup></small>, respectively.<a title="Select to navigate to references" href="#cit170"><sup><span class="sup_ref">170</span></sup></a> The limit of detection of HAdV 41 was in the same range as the TaqMan-qPCR result reported by Heim <span class="italic">et al.</span> (15 GU μL<small><sup>−1</sup></small>);<a title="Select to navigate to references" href="#cit172"><sup><span class="sup_ref">172</span></sup></a> the limit of detection of <span class="italic">Enterococcus faecalis</span> was one order of magnitude lower than the reported RPA-electrochemical detection (of a <span class="italic">Francisella tularensis</span>, 2 × 10<small><sup>4</sup></small> GU μL<small><sup>−1</sup></small>) by Del Rio <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit166"><sup><span class="sup_ref">166</span></sup></a> Seidel and co-workers’ second demonstration was a duplex detection of <span class="italic">Legionella</span> spp. and <span class="italic">Legionella pneumophila</span>, and they achieved the limit of detection of 87 GU μL<small><sup>−1</sup></small> and 26 GU μL<small><sup>−1</sup></small>, respectively.<a title="Select to navigate to references" href="#cit171"><sup><span class="sup_ref">171</span></sup></a> The <span class="italic">Legionella</span> spp. could be quantified over four log-steps while the <span class="italic">Legionella pneumophila</span> could be quantified over five log-steps.<a title="Select to navigate to references" href="#cit171"><sup><span class="sup_ref">171</span></sup></a> In short, the chemiluminescent detection is highly sensitive, having long and stable dynamic range, and is a good candidate for coupling with portable detection device (<span class="italic">e.g.</span> a smart phone) in comparison to the fluorescent probe-based detection.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f8.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">RPA in combination with chemiluminescent detection. The example demonstrates chemiluminescent detection using luminol and peroxide catalysed by horseradish peroxidase on flow-based microarray. The luminescent signal is detected by the CCD camera.<a title="Select to navigate to references" href="#cit170"><sup><span class="sup_ref">170</span></sup></a></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1934"><span class="b_heading">3.4 Silicon microring resonator (SMR)-based photonic detection</span></h3>
        <span>Detection of RPA reactions can also be performed using silicon microring resonator (SMR)-based photonic detection, which involves performing nucleic acid amplification in an asymmetric manner (pre-immobilised on one of the primers on the SMR, and all the other oligonucleotides and reagents are free in the solution, see also section 4.2) in the evanescent field of a resonator waveguide.<a title="Select to navigate to reference" href="#cit173"><sup><span class="sup_ref">173–175</span></sup></a> The binding of nucleic acids to pre-immobilised primers induces changes in the refractive index proximal to the waveguide surface. As the nucleic acid amplification progresses, the wavelength shift due to binding can be monitored in real-time on the SMR (<a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>). The SMR-based photonic detection is an alternative detection method for fluorophore-based real-time detection, yet is label-free and much more sensitive. Shin <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> achieved the limit of detection for a 10<small><sup>−4</sup></small>-fold diluted sample for <span class="italic">Mycobacterium tuberculosis</span> detection using real-time PCR, while they could detect down to a 10<small><sup>−6</sup></small>-fold diluted sample when using RPA-SMR detection. Sabaté del Río <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit176"><sup><span class="sup_ref">176</span></sup></a> demonstrated that RPA-SMR (2 fg μL<small><sup>−1</sup></small>) was 1000 times more sensitive than the real-time PCR (5 pg μL<small><sup>−1</sup></small>) for the detection of <span class="italic">Francisella tularensis</span>. Jin <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit151"><sup><span class="sup_ref">151</span></sup></a> showed that real-time PCR could only detect greater than 30% of the mutant allele in wild-type <span class="italic">KRAS</span> (a mutant gene in colorectal cancer) populations, while the RPA-SMR detection could detect 1% to 100% of the mutant allele. Since several silicon microrings can be accommodated on the resonator surface, a multiplexed RPA assay is achievable simultaneously on multiple microrings in parallel (also see section 4.2). Liu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit113"><sup><span class="sup_ref">113</span></sup></a> demonstrated a duplex detection of <span class="italic">IS6110</span> and <span class="italic">IS1081</span> insertion sequences of <span class="italic">Mycobacterium bovis</span> Bulgarian BCG using RPA-SMR assay, and achieved 3.2 and 12 genomic DNA copies per reaction analytical sensitivity respectively. Dao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit177"><sup><span class="sup_ref">177</span></sup></a> also demonstrated a duplex RPA-SMR assay for the detection of <span class="italic">Salmonella Typhimurium</span> and <span class="italic">Brucella ovis</span>, and they could identify 50 CFU (in 10 mL urine) and 100 CFU (in 10 mL urine) respectively. In comparison to the multiplex fluorophore-based real-time detection, the multiplex SMR-based photonic detection mitigates potential signal interferences due to fluorophore spectrum overlap, as the wavelength shifts are measured individually on each silicon microring.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f9.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">RPA in combination with silicon microring resonator (SMR)-based photonic detection. An asymmetric RPA reaction is performed on the SMR, where one of the RPA primers is pre-immobilised on the SMR, and another RPA primer, DNA template together with RPA reagents are free in the solution. The binding of DNAs to the pre-immobilised primers induces changes in the refractive index proximal to the waveguide surface, and can be monitored in real-time. The signal is detected as a wavelength shift. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit130">ref. 130</a>. Copyright 2015 Elsevier B.V. All rights reserved.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1965"><span class="b_heading">3.5 Surface-enhanced Raman scattering (SERS) detection</span></h3>
        <span>The final method to be highlighted for RPA detection is surface-enhanced Raman scattering (SERS). This is the phenomenon of when a laser excites nanoscale roughened metal surfaces (<span class="italic">e.g.</span> gold or silver), which resonantly drives surface charges, creating a highly localised (plasmonic) light field. When a molecule is absorbed, or lies close to, the enhanced field at the surface, a large enhancement in the Raman signal can be observed.<a title="Select to navigate to references" href="#cit178"><sup><span class="sup_ref">178</span></sup></a> SERS is a highly sensitive spectroscopic detection technique, which shows narrow and distinct spectral peaks of the detection molecules, and is particularly prominent for multiple target molecules detection (<a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>). Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit179"><sup><span class="sup_ref">179</span></sup></a> demonstrated a duplex RPA-SERS assay for the detection of two prostate cancer biomarkers T1E4 and RN7SL1. The peaks assigned to the phosphate backbone (913 cm<small><sup>−1</sup></small>), purines (560, 742 and 1334 cm<small><sup>−1</sup></small>) and pyrimidines (1632 cm<small><sup>−1</sup></small>) could readily be observed in the SERS spectral profiles of both T1E4 and RN7SL1 target amplicons. Lau <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> demonstrated a triplex RPA-SERS detection of plant pathogens (<span class="italic">Botrytis cinerea</span>, <span class="italic">Pseudomonas syringae</span> and <span class="italic">Fusarium oxysporum</span>); distinct peaks corresponding to the three pathogens (due to the labelled Raman reporters) could be clearly displayed on the spectrum. Their results also showed that the RPA-SERS detection (2.3 DNA copies) was 100 and 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 times more sensitive than RPA-Gel electrophoresis (GE) (2.32 × 10<small><sup>2</sup></small> DNA copies) and PCR-GE (2.3 × 10<small><sup>4</sup></small> DNA copies) detections, respectively. Koo <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit99"><sup><span class="sup_ref">99</span></sup></a> however, has demonstrated the up-to-date highest multiplexity of RPA-SERS assay, a penta-plex detection of prostate cancer biomarkers (T1E4, T1E5, PCA3, ARV7 and RN7SL1). Distinct peaks corresponding to each prostate cancer biomarker was well-separated and clearly observed at 1075 cm<small><sup>−1</sup></small> (T1E4), 1175 cm<small><sup>−1</sup></small> (T1E5), 1285 cm<small><sup>−1</sup></small> (PCA3), 1338 cm<small><sup>−1</sup></small> (ARV7) and 1380 cm<small><sup>−1</sup></small> (RN7SL1). In comparison to the SMR-based photonic detection, the SERS detection is also a better suited alternative to the fluorescent-based detection, especially for multiplexed detection. The multiplex ability of SMR-based photonic detection is limited by the number of silicon microrings that can be accommodated on the resonator surface, while the multiplex ability of SERS-based detection is limited by the number of Raman reporters (typically labelled to the target molecules) that produce sharp and unique peaks in the SERS detection.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f10.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">RPA in combination with surface-enhanced Raman scattering (SERS) detection. The example demonstrates a penta-plex RPA-SERS assay. The RPA amplicons labelled with Raman reporters are absorbed onto metal nanoparticles. When the laser excites these nanoparticles, a large enhancement in the Raman signals can be observed due to surface charge induced plasmonic light field surrounding the nanoparticles. The signal is displayed as narrow and distinct spectral peaks of the Raman reporters, correlating to different target templates. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit99">ref. 99</a>. Copyright 2016 WILEY-VCH Verlag GmbH &amp; Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
    
    
      
      <h2 id="sect1994"><span class="a_heading">4. Hot spots for RPA development</span></h2>
      <span>As demonstrated in the proceeding sections, RPA is a fast developing nucleic acid amplification technique with a growing number of applications and detection systems. In this section, we consider some of the upcoming “hot spots” for RPA development, and discuss their progress and limitations. In particular, we discuss critical advancements in moving RPA into an accurate quantitative technique through the use of digital RPA, as well as increasing the detection capacity through multiplexing. We also outline progress in realising the ability of RPA to break through the bounds of the laboratory for true field implementation, either through mobile laboratory setups (such as the “RPA in a suitcase”), microfluidic integration, or simple single-step RPA procedures.</span>
      
        
        <h3 id="sect1998"><span class="b_heading">4.1 Quantitative RPA – digital RPA</span></h3>
        <span>One of the most important developments of RPA is to quantify nucleic acid amplification, as quantitative nucleic acid amplification can be informative of gene expression levels, enabling better understanding of biological mechanisms, such as gene regulation during replication or pharmacological treatments, or distinguishing symptomatic from asymptomatic infection. However, unlike the classical real-time PCR (or quantitative PCR), real-time RPA is not considered to be robust for nucleic acid quantification. This is because the RPA reaction employs a chemical start (by adding the magnesium acetate) rather than a thermal start (by increasing reaction temperature to 95 °C), thus the reaction starting point cannot be precisely controlled. In addition, PCR is synchronised during each thermal cycle, as the annealing occurs only at low temperature, while there is no such synchronisation in RPA, as the annealing takes places all the time at the optimal reaction temperature range. Because of these reasons, RPA generates non-linear calibration curves for quantification.</span>
        <p class="otherpara">Nevertheless, the quantification of nucleic acids can be achieved using digital RPA that alleviates calibration curves by the ability of absolute quantification. Digital RPA involves fractionating a nucleic acid sample into thousands to hundred thousands of micro- to pico-litre volume compartments. Nucleic acid amplification subsequently occurs in each individual compartment. The amplification is usually detected <span class="italic">via</span> fluorescent probe, and compartments that encase nucleic acid from the sample will give rise to fluorescent signals. At the end of amplification, counting the positive and negative droplets gives precise, absolute quantification of the initial amount of nucleic acid template based on Poisson statistics (the targets end up in partitions independently and with a fixed rate). The published digital RPA reactions can be based on pre-fabricated partition and droplet partition of nucleic acid sample (<a title="Select to navigate to figure" href="#imgfig11">Fig. 11</a>).</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig11"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f11.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f11.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 11 </b> <span id="fig11"><span class="graphic_title">Digital RPA. (A): Digital RPA reaction based on pre-fabricated compartments. (i): Digital RPA on a slipchip. The slipchip contains top and bottom plates that contain wells and ducts which are pre-loaded with RPA reaction reagents respectively. The RPA reaction is initiated only when the top and bottom plate are brought in contact to each other and slipped to a specific configuration. The wells that encase the target DNA will give rise to an end-point fluorescent signal. Reprinted with permission from <a title="Select to navigate to references" href="#cit180">ref. 180</a>. Copyright 2011 American Chemical Society. (ii): Pre-fabricated partition-based digital RPA on a “microcliff” structured microchip. RPA reagents and extracted DNA are delivered into the microchip <span class="italic">via</span> passive polydimethylsiloxane (PDMS) degas pumping. The wells that trap target DNA will give rise to an end-point fluorescent signal. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit183">ref. 183</a>. Copyright 2017 Yeh <span class="italic">et al.</span> (iii) Pre-fabricated partition-based digital RPA on a pico-litre well array (PWA) chip. The mixture of RPA reagents and <span class="italic">Listeria monocytogenes</span> genomic DNA is evenly distributed into the wells (314 pL per well) using a scraping blade. The pico-litre wells that trap target DNA will give rise to an end-point fluorescent signal. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit184">ref. 184</a>. Copyright 2016 Li <span class="italic">et al.</span> (B): Digital RPA reaction based on droplet partition. (i) Water-in-oil emulsion-based digital droplet RPA reaction produced by multiple nozzles. Mixture of RPA reagents and <span class="italic">Listeria monocytogenes</span> DNA is dispersed into continuous oil phase. The droplets that encapsulate the <span class="italic">Listeria monocytogenes</span> DNA will give rise to an end-point fluorescent signal. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit185">ref. 185</a>. Copyright 2015 The Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">The pre-fabricated partition employs a large amount of “mini-cavities” – wells that can hold a defined volume, which allows massively parallel nucleic acid amplifications. One demonstration of such pre-fabricated partition-based digital RPA is the slipchip.<a title="Select to navigate to reference" href="#cit180"><sup><span class="sup_ref">180,181</span></sup></a> The slipchip contains two plates (top and bottom) containing wells and ducts which are pre-loaded with RPA reaction reagents respectively. The RPA reaction is initiated only when the top and bottom plates are aligned in a specific configuration by slipping. Tsaloglou <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit181"><sup><span class="sup_ref">181</span></sup></a> demonstrated a slipchip which allowed 8 parallel RPA reactions (each of the eight reactions comprises 500 nL of volume) for the detection of <span class="italic">Clostridium difficile</span>, and achieved a limit of detection of 1000 DNA copies. Shen <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit180"><sup><span class="sup_ref">180</span></sup></a> also demonstrated a slipchip that allowed 1550 parallel RPA reactions (each with 9 nL of volume) for methicillin-resistant <span class="italic">Staphylococcus aureus</span> (MRSA) detection, and achieved a limit of detection of 300 copies per mL of genomic DNA (<a title="Select to navigate to figure" href="#imgfig11">Fig. 11A(i)</a>). The other demonstration is on a “microcliff” structured microchip (<a title="Select to navigate to figure" href="#imgfig11">Fig. 11A(ii)</a>).<a title="Select to navigate to reference" href="#cit182"><sup><span class="sup_ref">182,183</span></sup></a> Unlike the slipchip, the sample (nucleic acid and RPA reagents) delivery in this microchip is <span class="italic">via</span> passive polydimethylsiloxane (PDMS) degas pumping that alleviates external pumps and power source. RPA incubation starts once the sample is delivered into the wells. The “microcliff” structured microchip demonstrated by Yeh <span class="italic">et al.</span><a title="Select to navigate to reference" href="#cit182"><sup><span class="sup_ref">182,183</span></sup></a> encased 200 to 1500 wells (30–100 nL per well), which allowed detection of 10–10<small><sup>5</sup></small> copies per μL of MRSA DNA. Another demonstration of pre-fabricated partition-based digital RPA is on a pico-liter well array (PWA) chip (27<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 wells, with each well of 314 pico-litres, in a 6 cm<small><sup>2</sup></small> area; <a title="Select to navigate to figure" href="#imgfig11">Fig. 11A(iii)</a>).<a title="Select to navigate to references" href="#cit184"><sup><span class="sup_ref">184</span></sup></a> In this chip, the sample was evenly distributed into the wells using a scraping blade. The chip was sealed with oil to prevent evaporation before RPA incubation. The PWA chip could quantify serial dilutions of <span class="italic">Listeria monocytogenes</span> genomic DNA from 9 × 10<small><sup>−1</sup></small> to 4 × 10<small><sup>−3</sup></small> copies per well with an average error of less than 11% (<span class="italic">n</span> = 15).<a title="Select to navigate to references" href="#cit184"><sup><span class="sup_ref">184</span></sup></a></p>
        <p class="otherpara">The digital RPA by droplet partitions is based on water-in-oil emulsion and is generated by dispersing liquid within an oil phase.<a title="Select to navigate to reference" href="#cit185"><sup><span class="sup_ref">185,186</span></sup></a> In this case, each generated droplet acts as an individual micro-reactor, which contains all the reagents for a miniaturised RPA reaction (in the volume range from nano-litre to pico-litre; <a title="Select to navigate to figure" href="#imgfig11">Fig. 11B(i)</a>). Schuler <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit185"><sup><span class="sup_ref">185</span></sup></a> is the first to demonstrate the RPA reaction in water emulsion-based droplets (also called digital droplet RPA, ddRPA), and they successfully quantified <span class="italic">Listeria monocytogenes</span> DNA (100, 215, 464 and 1000 copies) that was concordant to the number of copies measured with digital droplet PCR.</p>
        <p class="otherpara">In summary, the digital RPA partitionates a RPA reaction into nano- or pico-litre sized cavities to allow absolute quantification of the starting nucleic acid. This provides an opportunity to reduce the quantitative variability associated with real-time fluorescent probe-based quantification, as it does not rely on calibration curves. In addition, the digital RPA quantification is of significant value when testing samples that only contain negligible quantity (prospectively circulating tumour cells) in the original sample; the nucleic acid amplification from such samples can be overwhelmed in the tube (bulk) reaction, whereas, they become conspicuous in the digital RPA reaction.</p>
      
      
        
        <h3 id="sect2047"><span class="b_heading">4.2 Multiplex RPA</span></h3>
        <span>While quantitative RPA is critical to measure the amount of a single genetic marker in a system, multiplex testing is critical for assessing a series of different genetic markers, which can be equally important in gaining an improved understanding of a biological system, or for disease monitoring or detection. Multiplexing is highly preferred from a time and precision of understanding, as it greatly increases the result output per sample input in comparison to the single-plex detection within defined analytical turn-around time. Multiplex RPA reaction has been demonstrated either in a single tube (homogenous) or in a parallel fashion (sometimes refers to heterogeneous). The single-tube multiplex RPA can be performed using single or multiple nucleic acid templates. For the former, Kim <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit122"><sup><span class="sup_ref">122</span></sup></a> performed a duplex detection of Campylobacter species: <span class="italic">Campylobacter coli</span> and <span class="italic">Campylobacter jejuni</span> (detection limits of each: 1 CFU mL<small><sup>−1</sup></small> in pure culture and in chicken broth samples without enrichment; 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>log CFU g<small><sup>−1</sup></small> in non-enriched egg and chicken meat samples; 1 CFU g<small><sup>−1</sup></small> in enriched egg and chicken meat samples), and Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit187"><sup><span class="sup_ref">187</span></sup></a> conducted a duplex assay to differentiate wild-type and gE-deleted vaccine pseudorabies (limit of detection of 100 viral DNA copies each). However, the highest level of multiplexing is a triplex assay demonstrated by Piepenburg and co-workers for the detection of MRSA I, II and III genotypes (detection limits are 10 genomic copies each).<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> For single-tube multiplex RPA using multiple different nucleic acid templates, a few duplex and triplex assays have been demonstrated for the detection of bacteria,<a title="Select to navigate to reference" href="#cit126"><sup><span class="sup_ref">126,188,189</span></sup></a> parasites,<a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a> fungus,<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> genetic modified soybeans<a title="Select to navigate to references" href="#cit21"><sup><span class="sup_ref">21</span></sup></a> and prostate cancer biomarkers.<a title="Select to navigate to references" href="#cit179"><sup><span class="sup_ref">179</span></sup></a> In particular, Song and co-workers demonstrated the highest multiplexity <span class="italic">via</span> a 16-plex assay for pathogen detection by combing RPA reactions with LAMP on a microchip, which was termed a two-stage isothermal amplification (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12A</a>).<a title="Select to navigate to references" href="#cit190"><sup><span class="sup_ref">190</span></sup></a> In Song's demonstration, a first-stage RPA reaction was applied to simultaneously amplify sixteen different DNA targets in the RPA reactor, while a second-stage LAMP reaction was performed in sixteen branching chambers that stemmed from the RPA reactor to amplify and detect the resultant RPA amplicons in parallel. Each of the branching chambers contained a specific set of primers and probe for amplifying a specific pathogen among a pool of sixteen RPA amplicons from the first stage reaction.<a title="Select to navigate to references" href="#cit190"><sup><span class="sup_ref">190</span></sup></a> The RPA-LAMP assay could detect down to 1 plaque-forming units (PFU) using Zika virus-American strain (mex 2-81, Mexico) as a demonstration (amplifying the DNA template with the present of 16 primer pairs), with a linear correlation between 5 PFU and 500 PFU.<a title="Select to navigate to references" href="#cit190"><sup><span class="sup_ref">190</span></sup></a></span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig12"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f12_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f12.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f12.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 12 </b> <span id="fig12"><span class="graphic_title">Multiplex RPA. (A): Example of single-tube multiplex RPA <span class="italic">via</span> a two-stage isothermal amplification (RPA in combination with LAMP). Sixteen different DNA targets are amplified using RPA in the RPA reactor at the first stage, followed by metering the resulting mixture of RPA amplicons into sixteen branching LAMP reactors that have been pre-stored with set of LAMP primers and probe for a specific pathogen detection respectively. If the DNA targets are present in the sample and are successfully amplified by RPA, the branching LAMP reactors will produce fluorescent signals. Reprinted with permission from <a title="Select to navigate to references" href="#cit190">ref. 190</a>. Copyright AMERICAN ASSOCIATION FOR CLINICAL CHEMISTRY, INC. (B): Example of in-parallel multiplex RPA that adopts homogeneous assay format on a DVD. RPA reaction mixtures are pipetted onto a DVD as a microarray. Reprinted with permission from <a title="Select to navigate to references" href="#cit191">ref. 191</a>. Copyright 2014 Elsevier Ltd. All rights reserved. (C) Example of in-parallel multiplex RPA that adopts homogeneous assay format on electrowetting-on-dielectric (EWOD) surface. The example demonstrates a triplex RPA reaction (each sample with 3 replicates and duplicates negative control) for the detection of anti-microbial resistance genes in Gram-negative microorganisms. The mixing of different components of RPA reagents can be performed at desired time-points and in a sequential manner, as the RPA reaction droplets can be individually manipulated (dispensing, splitting or mixing) <span class="italic">via</span> the phenomenon of EWOD. The RPA reaction droplets that contain the DNA target will give rise to an end-point fluorescent signal. Reprinted and reproduced with permission from <a title="Select to navigate to references" href="#cit192">ref. 192</a>. Copyright 2017 Kalsi <span class="italic">et al.</span> (D) Example of in parallel multiplex RPA that adopts homogeneous assay format on a 3D origami paper-based device. Schematic view of the 3D origami paper-based device showing the location of freeze-dried biological components (real-time RPA reagents with three different primers and associated RNA templates) on wax patterned areas (left). Application of three different coloured dyes to display the multiplex RPA detection (right). After supplying the sample, dipping into the water and being folded, the input sample will flow along the defined patterns <span class="italic">via</span> capillary force between adjacent paper layers, and finally reach to the desired areas for RPA amplification. Reprinted with permission from <a title="Select to navigate to references" href="#cit193">ref. 193</a>. Copyright 2017 Springer Nature.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Collectively speaking, the single-tube multiplex RPA reaction saves both reaction space and volume. However, further increasing multiplexing levels is very challenging. In the case of using a single nucleic acid template, the multiplex capacity is mainly restricted by the capacity to accommodate multiple genes within a certain length of consensus sequence. However, the main restriction becomes the non-specific interactions among multiple oligonucleotides (primers, probes and nucleic acid templates) when using multiple nucleic acid templates. Although Song and co-workers’ methodology alleviates such restriction (as long as there were few correct RPA amplicons generated in the first-stage reaction, despite concurrently produced non-specific amplicons, these correct RPA amplicons would be specifically further amplified <span class="italic">via</span> a second-stage LAMP reaction), their methodology is still limited by the capacity of primer concentrations in the first-stage RPA reaction.</p>
        <p class="otherpara">In contrast to single-tube multiplex RPA, performing multiplex RPA in a parallel fashion omits the non-specific interactions among multiple oligonucleotides. This is because (1) each single-plex RPA reaction occurs next to each other but is independent of each other (performed as homogeneous assays) or (2) the RPA reaction is performed in an asymmetrical manner (heterogenous assay), where one of the RPA primer is pre-immobilised on a matrix while the other RPA primer remains in the solution together with the DNA template (and probe), in this way, only the RPA amplicons will be captured on the matrix for detection.</p>
        <p class="otherpara">The in-parallel multiplex RPA that adopts homogeneous assay format can be performed either in solution or on a solid phase. For the former, multiple amplicons from collateral single-plex RPA reactions (in tubes) are combined together and submitted to a subsequent multiplex detection method, exemplified by quantum dot beads-coated microwell detection (duplex),<a title="Select to navigate to references" href="#cit194"><sup><span class="sup_ref">194</span></sup></a> quantum dot-based flow cytometry detection (penta-plex)<a title="Select to navigate to references" href="#cit195"><sup><span class="sup_ref">195</span></sup></a> and SERS detection (penta-plex).<a title="Select to navigate to references" href="#cit99"><sup><span class="sup_ref">99</span></sup></a> For the latter, one demonstration is on the digital video disk (DVD) by Maquieira research group, and the resulting signals can be detected by a DVD player (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12B</a>).<a title="Select to navigate to reference" href="#cit188"><sup><span class="sup_ref">188,191,194</span></sup></a> The other demonstration is on an electrowetting-on-dielectric (EWOD) surface, where the EWOD-based RPA reaction droplets are controlled using electric potential by varying electric energy across the thin dielectric film between the liquid and conducting substrate.<a title="Select to navigate to reference" href="#cit80"><sup><span class="sup_ref">80,192</span></sup></a> Kalsi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit192"><sup><span class="sup_ref">192</span></sup></a> demonstrated a triplex EWOD-based RPA reaction in a parallel format for the detection of three antibiotic resistant genes (CTX-M-15, KPC and NDM-1). Each single-plex droplet RPA reaction was independent of other single-plex droplet RPA reactions, and could perform a set of programmed operations including move, merge, split, dispense and mix in an automatic manner (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12C</a>). The EWOD-based RPA reaction achieved the limit of detections of 1000 DNA copies of each antibiotic resistant gene, which is 100 times less sensitive than the bench-top RPA reactions.<a title="Select to navigate to references" href="#cit192"><sup><span class="sup_ref">192</span></sup></a> Another demonstration is on paper by Magro and co-workers, which they constructed a wax patterned five-layer paper device in three-dimensional (3D) origami configuration (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12D</a>).<a title="Select to navigate to references" href="#cit193"><sup><span class="sup_ref">193</span></sup></a> Freeze-dried RPA reaction reagents were pre-stored on multiple individual wax patterned areas. Once the liquid sample mixture was pipetted onto the sample inlet, the input liquid sample mixture could be distributed <span class="italic">via</span> capillary force between adjacent paper layers by folding, and ultimately reached specific areas that contained pre-stored reagents, to initiate RPA reactions.</p>
        <p class="otherpara">Unlike the in-parallel multiplex RPA that adopts homogeneous assay format, the demonstrations of the in-parallel multiplex RPA that adopts heterogeneous assay takes place only on solid surfaces. The majority of such assay performs multiplex RPA in an asymmetric manner on a microarray. Kersting <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit196"><sup><span class="sup_ref">196</span></sup></a> performed triplex RPA in a 4 × 6 microarray for the detection of <span class="italic">Salmonella enterica</span>, <span class="italic">Neisseria gonorrhoeae</span> and MRSA on the epoxy-silanised glass slides, and achieved detection limits of 10 CFU, 100 CFU and 10 CFU respectively. Kunze <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit170"><sup><span class="sup_ref">170</span></sup></a> conducted triplex RPA in a 5 × 5 microarray for the detection of <span class="italic">Enterococcus faecalis</span>, Human adenovirus 41 (HADV 41) and Phi X 174 virus on the poly(propylene glycol) diamine (DAPPG) coated glass slides (also see section 3.3), and achieved detection limits of 35 GU μL<small><sup>−1</sup></small>, 1 GU μL<small><sup>−1</sup></small> and 5 × 10<small><sup>3</sup></small> GU μL<small><sup>−1</sup></small> respectively. Another demonstration of such an assay is the performance of multiple single-plex RPA reactions on multiple silicon microrings fitted on a resonator surface, demonstrated by Liu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit113"><sup><span class="sup_ref">113</span></sup></a> and Dao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit177"><sup><span class="sup_ref">177</span></sup></a> (more details refer to section 3.4).</p>
        <p class="otherpara">Taken together, the multiplex RPA in a parallel fashion has a much higher assay throughput, circumvents cross-reaction (among oligonucleotides) issues, and possesses higher multiplex capacity compared to the single-tube multiplex RPA. However, its degree of multiplexing is limited by (1) the number of available unique labels conjugated to multiple nucleic acid templates which permit distinctive detection; (2) the size of the reaction surface; and (3) EWOD-based RPA reaction is also limited by the minimum volume of the droplet that permits an effective reaction.</p>
      
      
        
        <h3 id="sect2115"><span class="b_heading">4.3 RPA in a suitcase for mobile laboratory</span></h3>
        <span>The ultimate purpose of isothermal amplification such as RPA is to take the nucleic acid testing away from the centralised lab to the field or resource-limited settings. Abd El Wahed and co-workers turned this thought into reality by miniaturising the RPA diagnostic into a standard suitcase (56 cm × 45.5 cm × 26.5 cm by size), dubbed diagnostics-in-a-suitcase (Dias; costs around 5000 Euro; <a title="Select to navigate to figure" href="#imgfig13">Fig. 13A</a>).<a title="Select to navigate to references" href="#cit197"><sup><span class="sup_ref">197</span></sup></a> The Dias contained all the reagents and equipment necessary for the real-time RPA assay to detect the emerging avian influenza A (H7N9) virus at the site of an outbreak. The reagents and the equipment were fixed in the foam at the base of the suitcase which acted as a shock absorber during transport; a solar panel and a power pack provided the power support; and the storage box was refillable whenever needed.<a title="Select to navigate to references" href="#cit197"><sup><span class="sup_ref">197</span></sup></a> The RPA-Dias detection of influenza A (H7N9) was successful, and achieved detection limits of 10 and 100 RNA copies for the detection of H gene and N gene, respectively.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig13"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f13_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f13.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f13.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 13 </b> <span id="fig13"><span class="graphic_title">Mobile RPA assay. (A): RPA in-a-suitcase. The standard suitcase (56 cm × 45.5 cm × 26.5 cm by size) contains all equipment and reagents for performing the real-time RPA assay. The equipment and reagents are imbedded into the foam or fixed for stable portability. Reprinted with permission from <a title="Select to navigate to references" href="#cit197">ref. 197</a>. Copyright 2015 Abd El Wahed <span class="italic">et al.</span> Published by Elsevier B.V. (B): A mobile laboratory set up from the “RPA in-a-suitcase”. The mobile lab is arranged into 4 sites in close proximity, including the RNA extraction, master mix, sample mix and detection. The RNA extraction area encompasses magnetic separator stand, vortex, rotator, 1.5–2 mL Eppendorf tube rack, automatic 100–1000 μL micopipette, micropipette tips, digital timer, 1.5 mL disposable plastic Eppendorf tubes and a waste container with autoclavable plastic bags. Both the master mix and sample mix areas contain vortex, minicentrifuge, automatic 1–10 and 10–100 μL micopipettes, micropipette tips, scissor and 0.2 mL tubes rack. The detection was done using the tubescanner (Twista device, TwistDx, Cambridge, UK). Reprinted with permission from <a title="Select to navigate to references" href="#cit131">ref. 131</a>. Copyright 2015 Abd El Wahed <span class="italic">et al.</span></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">This highly promising demonstration of the first RPA-Dias inspired Abd El Wahed and co-workers to perform further testing. They carried the Dias (a total weight of 23 kilograms including the aluminium case) to the field of both of Kedougou (Senegal) and Bangkok (Thailand), and set up a mobile laboratory for the real-time RPA detection of dengue virus (1–4 serotypes).<a title="Select to navigate to references" href="#cit131"><sup><span class="sup_ref">131</span></sup></a> The mobile lab was organised into 4 sites, including the RNA extraction, master mix, sample mix and detection (using a tubescanner, TwistDx, Cambridge, UK) in close proximity (<a title="Select to navigate to figure" href="#imgfig13">Fig. 13B</a>).<a title="Select to navigate to references" href="#cit131"><sup><span class="sup_ref">131</span></sup></a> All the reagents were cold-chain independent; the power was supplied either from a motor vehicle battery (<span class="italic">via</span> inverter) or from the solar panel. The real-time RPA assay performed successfully in such an open-air environment and observed no influence of dust on the assay quality. The clinical sensitivities and specificities tested in Kedougou (Senegal) and Bangkok (Thailand) were 98% and 100% and 72% and 100%, respectively.<a title="Select to navigate to references" href="#cit131"><sup><span class="sup_ref">131</span></sup></a> Later, Abd El Wahed and co-workers also carried the Dias to the local hospitals in Matoto (Guinea)<a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> and Mymensingh (Bangladesh)<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a> for the on-site detection of Ebola virus and <span class="italic">Leishmania donovani</span>, respectively. The real-time RPA Ebola assay was able to correctly include positives (91% of clinical sensitivity) and exclude negatives (100% of clinical specificity);<a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> while the real-time RPA <span class="italic">Leishmania donovani</span> assay achieved 100% clinical sensitivity and 100% clinical specificity.<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a> The demonstration of RPA-Dias brings the rapid nucleic acid testing to the site of need, in particular benefiting the places with poor laboratory infrastructure. With its continuous development, there is no doubt that the RPA-Dias will become pervasive within five years.</p>
      
      
        
        <h3 id="sect2140"><span class="b_heading">4.4 Microfluidic integration of RPA assays</span></h3>
        <span>Apart from multiplex RPA reactions and mobile RPA reactions that provide increasing diagnostic efficiency and portability respectively, a growing attention has been focused on developing RPA assays that encompasses both of the two merits. The solution is to integrate RPA amplification with other chemical processes on a miniaturised device using microfluidics. The major chemical processes to combine with RPA amplification for an intact diagnosis are sample preparation, amplification and signal detection. There have been a few examples of integrating RPA amplification with signal detection, however, only few demonstrations have been performed integrating all of the three processes. Branavan <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit198"><sup><span class="sup_ref">198</span></sup></a> developed a microfluidics that integrated RPA amplification and fluorescent signal detection for the detection of sexually transmitted infections; Renner <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> developed a degas-actuated microfluidics that integrated multiplex RPA amplification and fluorescent signal detection for the detection of ESKAPE (<span class="italic">Enterococcus faecium</span>, <span class="italic">Staphylococcus aureus</span>, <span class="italic">Klebsiella pneumoniae</span>, <span class="italic">Acinetobacter baumannii</span>, <span class="italic">Pseudomonas aeruginosa</span> and <span class="italic">Enterobacter</span> spp.) bacterial pathogens; and Hu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit199"><sup><span class="sup_ref">199</span></sup></a> developed thin film transistor-based microfluidics that integrated RPA amplification and electronic signal detection for the detection of anti-microbial resistance genes. We note that all these three examples require the input of manually pre-mixed RPA reaction reagents. For fully integrated assays, one demonstration is the self-powered integrated microfluidic by Yeh <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit183"><sup><span class="sup_ref">183</span></sup></a> The self-powered integrated microfluidic integrated sample preparation (plasma separation from blood cells), digital RPA amplification (pre-partition-based) and detection (fluorescent-probe based) in an automatic manner <span class="italic">via</span> a built-in vacuum battery system (that frees the chip from external pumps sources for pumping).<a title="Select to navigate to references" href="#cit183"><sup><span class="sup_ref">183</span></sup></a> The self-powered integrated microfluidic was able to detect 2 × 10<small><sup>5</sup></small> copies per μL of HIV-1 RNA spiked in human blood within 18 minutes, and was able to quantitatively detect 10 to 10<small><sup>5</sup></small> copies per μL of MRSA DNA in water or directly from spiked human whole blood.<a title="Select to navigate to references" href="#cit183"><sup><span class="sup_ref">183</span></sup></a> Another demonstration of fully integrated assays is the application of centrifugal microfluidics that perform all the required procedure for RPA reaction in an automatic manner governing by centrifugal forces. Zengerle's research group was the first one to demonstrate such performance. They developed a foil disc (<a title="Select to navigate to figure" href="#imgfig14">Fig. 14A</a>) that performs metering of the RPA rehydration buffer to re-suspend RPA reaction pellet, and subsequently aliquot the resulting mixture into five parallel reaction chambers that were pre-stored with oligonucleotides (primers, probe and DNA template).<a title="Select to navigate to references" href="#cit200"><sup><span class="sup_ref">200</span></sup></a> The foil disc was fitted onto a modified Rotor-Gene 2000, which executed centrifugation to manipulate the liquid movement between different chambers on the foil disc and real-time fluorescent signal detection. Followed up this demonstration, they developed another foil disc that integrated sample preparation (DNA extraction from blood plasma), nucleic acid amplification and fluorescent detection of <span class="italic">Bacillus anthracis</span> and <span class="italic">Francisella tularensis</span>.<a title="Select to navigate to references" href="#cit201"><sup><span class="sup_ref">201</span></sup></a> And this time, they tailor-made a portable LabDisk Player (approximately 18 × 28×15 cm<small><sup>3</sup></small> and 2 kilograms by weight) which could process the integrated chemical processes on the foil disc in an automatic manner. The fully integrated microfluidic RPA assay detected 6 × 10<small><sup>4</sup></small> genome equivalents of <span class="italic">Bacillus anthracis</span> and 6 × 10<small><sup>6</sup></small> genome equivalents of <span class="italic">Francisella tularensis</span> in less than 45 minutes.<a title="Select to navigate to references" href="#cit201"><sup><span class="sup_ref">201</span></sup></a> Their centrifugal microfluidics is the first true example of fully integrated RPA assay device, with the entire procedure from DNA extraction through to detection, in a fully automated fashion. Later, Kim <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit202"><sup><span class="sup_ref">202</span></sup></a> also developed a centrifugal microfluidics that integrated sample lysis, RPA amplification, metering, dilution and lateral flow strip detection of food-borne pathogen (<span class="italic">Salmonella enteritidis</span>; <a title="Select to navigate to figure" href="#imgfig14">Fig. 14B</a>). The centrifugal microfluidics was fitted onto a computer-controlled unit that included a spinning motor, a laser for sample lysis and a local heating apparatus to support RPA assay procedure. Their demonstration achieved the limit of detections of 10 CFU mL<small><sup>−1</sup></small> and 100 CFU mL<small><sup>−1</sup></small> in phosphate-buffered saline (PBS) and milk respectively with a total analysis time of 30 minutes.<a title="Select to navigate to references" href="#cit202"><sup><span class="sup_ref">202</span></sup></a></span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig14"><a href="/image/article/2019/AN/c8an01621f/c8an01621f-f14_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8an01621f-f14.tif" src="/image/article/2019/AN/c8an01621f/c8an01621f-f14.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 14 </b> <span id="fig14"><span class="graphic_title">Integrated RPA assays on microfluidics. (A): Centrifugal microfluidics that integrates RPA reagents metering, mixing and aliquoting, amplification and fluorescent signal detection. Photograph of a foil disc assembled with RPA reaction reagents (liquid and lyophilisate reagents) featuring 6 fluidic structures (top), each capable of processing 5 geometrically multiplexed assays. Detailed fluidic structure for one unit is denoted; the buffer is replaced by red ink for demonstration purposes (bottom left). Elucidation of automated RPA assay on a centrifugal microfluidic (bottom right); fluids movement between chamber to chamber are manipulated <span class="italic">via</span> centrifugal force. (i) Inside the glass capillary contains the RPA liquid reagents, including DNA template, primers, probe, rehydration buffer, magnesium acetate, while the lyophilisate chamber contains the lyophilised pellet; (ii) the RPA liquid reagents is spun into the lyophilisate chamber after the glass capillary is crushed; (iii) the siphon allows valving between the lyophilisate chamber and the aliquoting structure; (iv) the 50 μL RPA liquid reagents volume is split into 5 × 10 μL aliquots; and (v) the fluid fills the reaction chambers <span class="italic">via</span> a centrifugal pneumatic valve. Reprinted with permission from <a title="Select to navigate to references" href="#cit200">ref. 200</a>. Copyright 2010 Royal Society of Chemistry. (B): Centrifugal microfluidics that integrated sample lysis, RPA amplification, metering, dilution and lateral flow strip detection. Reprinted with permission from <a title="Select to navigate to references" href="#cit202">ref. 202</a>. Copyright 2014 American Chemical Society. (C): Paper-based microfluidics that integrates RPA amplification, dilution and lateral flow strip detection. (i) Layout and components of the device; (ii) operations of the device: (a) sample slider at position 1, mixing RPA reagents and target to initiate the amplification (30 minutes of incubation); (b) after amplification, the sample slider pulled down to position 2, in contact with dry dilution pads to absorb RPA buffer (10 minutes of duration); (c) the sample slider pulled down to position 3, in contact with wet dilution pads, for dilution with TBST buffer (10 minutes of duration); and (d) the sample slider pulled down to position 4 into contact with the lateral flow strip, and the running buffer slider is pulled down to activate the lateral flow detection (5 minutes of duration). Reprinted with permission from <a title="Select to navigate to references" href="#cit106">ref. 106</a>. Copyright 2015 Cordray and Richards-Kortum.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">The foil-based microfluidics described above are a compact and affordable platform for performing integrated RPA assays, but they require complicated and expensive design and fabrication. Nonetheless, paper-based microfluidics can be an alternative for performing integrated RPA assays, as they can also conduct many of the functions of the foil-based microfluidics. Cordray and Richards-Kortum<a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> developed a paper-based microfluidics that integrated RPA amplification, dilution and lateral flow strip detection. All the reagents for this three processes were pre-stored in different paper pads, and were mixed with each other sequentially by pulling the sliders (<a title="Select to navigate to figure" href="#imgfig14">Fig. 14C</a>).<a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> Their paper-based integrated RPA assay could detect as few as 50 synthetic <span class="italic">Plasmodium</span> DNA copies, which was equivalent to the analytical sensitivity of the bench-top RPA-GE assay. In comparison, Magro <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit193"><sup><span class="sup_ref">193</span></sup></a> developed a 3D origami paper-based device that integrated multiplex RPA amplification and fluorescent signal detection (refer to section 4.2). The pre-stored reagents and oligonucleotides in different wax-patterned areas were rehydrated once the device is folded and dipped into water. Although the integration of RPA amplification and signal detection showed successful paper-based microfluidics, it required human intervention to complete the integrated RPA assay, as opposed to a fully integrated RPA assay on foil-based microfluidics. Moreover, it is very challenging to integrate sample preparation with RPA amplification and signal detection on paper-based microfluidics, not to mention in an automated execution. The reason is that it is very difficult to manipulate liquid movement on paper merely <span class="italic">via</span> capillary force, which is a weak force, and thus results in a slow moving process (despite of the liquid has a high wettability and the matrix is very hydrophilic and porous).</p>
        <p class="otherpara">Overall, developing a microfluidically integrated RPA assay is a transition towards the realisation of fully automated sample-in-answer-out RPA assay to increase the diagnostic efficiency and portability. Nevertheless, the sample preparation step is the bottleneck to achieve this ultimate goal. In particular, some samples (<span class="italic">e.g.</span> blood) require multiple processing steps before the extracted nucleic acids are ready for the subsequent amplification and detection. Zengerle's research group and Kim and colleagues’ work by employing multiple sample preparation chambers together with the amplification chamber (and detection chamber) on a single foil-based microfluidics, demonstrates one feasible way, albeit with challenging and costly microfluidics design and fabrication.</p>
      
      
        
        <h3 id="sect2198"><span class="b_heading">4.5 One-step RPA assays</span></h3>
        <span>A simple and feasible way to facilitate full automation of the RPA assay without introducing multiple sample preparation chambers is to perform a one-step RPA assay, or in other words, to perform RPA amplification directly from the crude sample (in the same tube). RPA has been able to amplify nucleic acids directly from crude samples, such as plant tissue extract,<a title="Select to navigate to reference" href="#cit111"><sup><span class="sup_ref">111,203</span></sup></a> crude sap,<a title="Select to navigate to reference" href="#cit79"><sup><span class="sup_ref">79,112</span></sup></a> soil and water samples,<a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> food samples (eggs and chicken meat)<a title="Select to navigate to references" href="#cit119"><sup><span class="sup_ref">119</span></sup></a> and vaginal swab lysate.<a title="Select to navigate to references" href="#cit137"><sup><span class="sup_ref">137</span></sup></a></span>
        <p class="otherpara">Nevertheless, Choi and co-workers were the first to demonstrate a fully integrated RPA assay on foil-based microfluidics that was facilitated by the one-step assay approach. Choi <span class="italic">et al.</span> employed direct PCR buffer in RPA reaction for the detection of food poisoning bacteria (<span class="italic">Salmonella enterica</span>, <span class="italic">Escherichia coli</span> O157:H7 or <span class="italic">Vibrio parahaemolyticus</span>).<a title="Select to navigate to reference" href="#cit204"><sup><span class="sup_ref">204,205</span></sup></a> The direct PCR buffer was able to lyse the bacterial cells, inactivate amplification inhibitors from the lysed sample and was compatible to the RPA reaction system.<a title="Select to navigate to references" href="#cit204"><sup><span class="sup_ref">204</span></sup></a> The entire assay completed within 30 minutes, which was much shorter than the time required for a non-sample preparation integrated microfluidic or a sample preparation integrated microfluidic system (<span class="italic">e.g.</span> at least 60 minutes). Although the direct RPA assay is a shortcut to assist full automation of foil-based microfluidics, there is no rule of thumb for the direct RPA assay buffer. This is because each sample has different lysis difficulty levels, and the effective lysis buffer may not be compatible with the RPA system and may also have potential impact on fluid movement within the microfluidics. Therefore, it is a trail-and-error process to identify such effective direct RPA buffers.</p>
      
    
    
      
      <h2 id="sect2215"><span class="a_heading">5. Conclusions and future perspectives</span></h2>
      <span>The PCR is a revolutionary watershed in nucleic acid test, however, isothermal nucleic acid amplification, as an alternative to PCR, has a lower requirement in heating conditions and performs nucleic acid replication more rapidly. RPA was a comparatively late introduced isothermal amplification method, however, it is one of the fastest developing methods. This is due to its true isothermal properties, simple reaction scheme, fast reaction time and excellent reports of sensitivity and specificity. In this review, we provide comprehensive knowledge of RPA technology: from its reaction components and mechanism, to the design of RPA assay and detection methods. In particular, we summarise many experimental tips from practical implementation, dispose clinical/field performance data and point out focused RPA development, to help researchers make better use of RPA and make contributions to boost its development.</span>
      <p class="otherpara">RPA has just passed its first decade of development, and is now stepping into a next stage of evolution. We have considered some critical issues for future development. Firstly, more attention is needed for developing field-amenable sample preparation methods including concentration, extraction and purification, as this would largely facilitate a complete RPA assay for on-site or field application. Secondly, developing portable and fully automated RPA diagnostic devices is of paramount importance not only to further increase the diagnostic efficiency, but more importantly to bring RPA diagnostic closer to the layperson and people who live in places with poor healthcare infrastructure. Thirdly, RPA can be applied for developing wearable sensors by virtue of its close-to-body reaction temperature. Imagine if people can simply take some samples from their own body (<span class="italic">e.g.</span> body fluid) and perform a fast RPA assay using their body temperature to detect potential pathogens, this could revolutionise RPA diagnostics to be a self-testing. Fourthly, unlike real-time PCR which is controlled by heat cycle, RPA employs continuous amplification after reaction initiation; it would be beneficial to develop a method or mechanism to precisely control the starting point of each RPA amplification cycle, so as to assist obtaining reliable quantitative RPA results (other than the absolute quantification by digital RPA). Fifthly, the current market share of RPA (7.08%) is far less than that of PCR, and is less in comparison to other isothermal nucleic acid amplification methods (<span class="italic">e.g.</span> SDA and LAMP). To boost its market share, more studies in comparative clinical validations between RPA and existing PCR assays can be performed. In addition, practical websites or software should be developed to ease and streamline the RPA primers and probe design, and screen for optimal oligonucleotide pairs for the reaction. Rationally speaking, RPA may not replace the status of PCR in years to come, but it can be a versatile complement of PCR. At this time point, RPA technology is maturing for application in the clinic, however, it is still at a transition period towards on-site or field application. With its continuous fast development, we foresee that RPA technology will ultimately become robust mobile and point-of-need applications in the future.</p>
    
    
      <h2 id="sect2222"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <h2 id="sect2226"><span class="a_heading">Acknowledgements</span></h2>
      <span>This work was funded by Alexander von Humboldt Fellowship. We also acknowledge the financial support from the Baden-Württemberg Ministry for Economy, Labour and Housing (project “IDAK” AZ 7-4332.62-HSG/69) and the Queensland Government, Department of Science, Information Technology, Innovation and the Arts (DSITIA, Australia).</span>
    
    <span id="sect2228"><h2 id="sect2225"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">K. Mullis, F. Faloona, S. Scharf, R. Saiki, G. Horn and H. Erlich, <span class="italic">Cold Spring Harbor Symp. Quant. Biol.</span>, 1986, <span class="bold">51</span>(Part 1), 263–273 <a target="_blank" class="DOILink" href="https://doi.org/10.1101/SQB.1986.051.01.032" title="DOI Link to resource 10.1101/SQB.1986.051.01.032">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2sXkslajtb0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=3472723%5Buid%5D" title="PubMed Link to resource 3472723">PubMed</a>.</span></li>
      <li><span id="cit2">J. Li and J. Macdonald, <span class="italic">Biosens. Bioelectron.</span>, 2014, <span class="bold">64C</span>, 196–211 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Biosens.%20Bioelectron.%5BJour%5D%20AND%2064C%5Bvolume%5D%20AND%20196%5Bpage%5D%20and%202014%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit3">P. Craw and W. Balachandran, <span class="italic">Lab Chip</span>, 2012, <span class="bold">12</span>, 2469–2486 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2LC40100B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2LC40100B">RSC</a>.</span></li>
      <li><span id="cit4">P. Gill and A. Ghaemi, <span class="italic">Nucleosides, Nucleotides Nucleic Acids</span>, 2008, <span class="bold">27</span>, 224–243 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/15257770701845204" title="DOI Link to resource 10.1080/15257770701845204">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhs1WmsLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18260008%5Buid%5D" title="PubMed Link to resource 18260008">PubMed</a>.</span></li>
      <li><span id="cit5">Y. Zhao, F. Chen, Q. Li, L. Wang and C. Fan, <span class="italic">Chem. Rev.</span>, 2015, <span class="bold">115</span>, 12491–12545 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.5b00428" title="DOI Link to resource 10.1021/acs.chemrev.5b00428">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvVKqsbfO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26551336%5Buid%5D" title="PubMed Link to resource 26551336">PubMed</a>.</span></li>
      <li><span id="cit6">O. Piepenburg, C. H. Williams, D. L. Stemple and N. A. Armes, <span class="italic">PLoS Biol.</span>, 2006, <span class="bold">4</span>, e204 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pbio.0040204" title="DOI Link to resource 10.1371/journal.pbio.0040204">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16756388%5Buid%5D" title="PubMed Link to resource 16756388">PubMed</a>.</span></li>
      <li><span id="cit7">Isothermal nucleic acid amplification technology (INAAT) market analysis by product, by technology (NASBA, HDA, LAMP, SDA, SPIA, NEAR, TMA, RCA, RPA, SMAP2), and segment forecasts, 2018–2025, Report GVR-1-68038-588-5, Grand View Research, 2017.</span></li>
      <li><span id="cit8">H. Zaghloul and M. El-Shahat, <span class="italic">World J. Hepatol.</span>, 2014, <span class="bold">6</span>, 916–922 <a target="_blank" class="DOILink" href="https://doi.org/10.4254/wjh.v6.i12.916" title="DOI Link to resource 10.4254/wjh.v6.i12.916">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25544878%5Buid%5D" title="PubMed Link to resource 25544878">PubMed</a>.</span></li>
      <li><span id="cit9">M. D. Moore and L.-A. Jaykus, <span class="italic">Future Virol.</span>, 2017, <span class="bold">12</span>, 421–429 <a target="_blank" class="DOILink" href="https://doi.org/10.2217/fvl-2017-0034" title="DOI Link to resource 10.2217/fvl-2017-0034">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtl2gsLzK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit10">A. James and J. Macdonald, <span class="italic">Expert Rev. Mol. Diagn.</span>, 2015, <span class="bold">15</span>, 1475–1489 <a target="_blank" class="DOILink" href="https://doi.org/10.1586/14737159.2015.1090877" title="DOI Link to resource 10.1586/14737159.2015.1090877">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksrbK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26517245%5Buid%5D" title="PubMed Link to resource 26517245">PubMed</a>.</span></li>
      <li><span id="cit11">R. K. Daher, G. Stewart, M. Boissinot and M. G. Bergeron, <span class="italic">Clin. Chem.</span>, 2016, <span class="bold">62</span>, 947–958 <a target="_blank" class="DOILink" href="https://doi.org/10.1373/clinchem.2015.245829" title="DOI Link to resource 10.1373/clinchem.2015.245829">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFOhsLbE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27160000%5Buid%5D" title="PubMed Link to resource 27160000">PubMed</a>.</span></li>
      <li><span id="cit12">I. M. Lobato and C. K. O'Sullivan, <span class="italic">TrAC, Trends Anal. Chem.</span>, 2018, <span class="bold">98</span>, 19–35 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.trac.2017.10.015" title="DOI Link to resource 10.1016/j.trac.2017.10.015">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsleksbzI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit13">J. Griffith and T. Formosa, <span class="italic">J. Biol. Chem.</span>, 1985, <span class="bold">260</span>, 4484–4491 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2MXhvVWit70%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit14">M. Bianchi, B. Riboli and G. Magni, <span class="italic">EMBO J.</span>, 1985, <span class="bold">4</span>, 3025–3030 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/j.1460-2075.1985.tb04039.x" title="DOI Link to resource 10.1002/j.1460-2075.1985.tb04039.x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL28Xht1Ggt7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=3905387%5Buid%5D" title="PubMed Link to resource 3905387">PubMed</a>.</span></li>
      <li><span id="cit15">J. M. Boyle and N. Symonds, <span class="italic">Mutat. Res.</span>, 1969, <span class="bold">8</span>, 431–439 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0027-5107(69)90060-8" title="DOI Link to resource 10.1016/0027-5107(69)90060-8">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaE3cXktVSltw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=5370738%5Buid%5D" title="PubMed Link to resource 5370738">PubMed</a>.</span></li>
      <li><span id="cit16">Y. Shamoo, A. M. Friedman, M. R. Parsons, W. H. Konigsberg and T. A. Steitz, <span class="italic">Nature</span>, 1995, <span class="bold">376</span>, 362–366 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/376362a0" title="DOI Link to resource 10.1038/376362a0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXnt1SjsL4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=7630406%5Buid%5D" title="PubMed Link to resource 7630406">PubMed</a>.</span></li>
      <li><span id="cit17">B. M. Alberts and L. Frey, <span class="italic">Nature</span>, 1970, <span class="bold">227</span>, 1313–1318 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/2271313a0" title="DOI Link to resource 10.1038/2271313a0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaE3cXlt1OqtbY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=5455134%5Buid%5D" title="PubMed Link to resource 5455134">PubMed</a>.</span></li>
      <li><span id="cit18">T. Okazaki and A. Kornberg, <span class="italic">J. Biol. Chem.</span>, 1964, <span class="bold">239</span>, 259–268 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaF2cXjt1egsw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit19">M. Euler, Y. Wang, P. Otto, H. Tomaso, R. Escudero, P. Anda, F. T. Hufert and M. Weidmann, <span class="italic">J. Clin. Microbiol.</span>, 2012, <span class="bold">50</span>, 2234–2238 <a target="_blank" class="DOILink" href="https://doi.org/10.1128/JCM.06504-11" title="DOI Link to resource 10.1128/JCM.06504-11">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhtFCmsL7K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22518861%5Buid%5D" title="PubMed Link to resource 22518861">PubMed</a>.</span></li>
      <li><span id="cit20">RPA Assay Design, https://www.twistdx.co.uk/en/support/rpa-assay-design-2, (accessed Oct 4th, 2018).</span></li>
      <li><span id="cit21">D. Chandu, S. Paul, M. Parker, Y. Dudin, J. King-Sitzes, T. Perez, D. W. Mittanck, M. Shah, K. C. Glenn and O. Piepenburg, <span class="italic">BioMed. Res. Int.</span>, 2016, <span class="bold">2016</span>, 3145921 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=BioMed.%20Res.%20Int.%5BJour%5D%20AND%202016%5Bvolume%5D%20AND%203145921%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit22">S. B. Zimmerman and B. Harrison, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 1987, <span class="bold">84</span>, 1871–1875 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.84.7.1871" title="DOI Link to resource 10.1073/pnas.84.7.1871">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2sXktlGmtL8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit23">R. J. Ellis, <span class="italic">Trends Biochem. Sci.</span>, 2001, <span class="bold">26</span>, 597–604 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0968-0004(01)01938-7" title="DOI Link to resource 10.1016/S0968-0004(01)01938-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXnt1Wjsbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=11590012%5Buid%5D" title="PubMed Link to resource 11590012">PubMed</a>.</span></li>
      <li><span id="cit24">Y. Sasaki, D. Miyoshi and N. Sugimoto, <span class="italic">Biotechnol. J.</span>, 2006, <span class="bold">1</span>, 440–446 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/biot.200500032" title="DOI Link to resource 10.1002/biot.200500032">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xmt1emsLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16892271%5Buid%5D" title="PubMed Link to resource 16892271">PubMed</a>.</span></li>
      <li><span id="cit25">S. Fjelstrup, M. B. Andersen, J. Thomsen, J. Wang, M. Stougaard, F. S. Pedersen, Y.-P. Ho, M. S. Hede and B. R. Knudsen, <span class="italic">Sensors</span>, 2017, <span class="bold">17</span>, 1201 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/s17061201" title="DOI Link to resource 10.3390/s17061201">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28538659%5Buid%5D" title="PubMed Link to resource 28538659">PubMed</a>.</span></li>
      <li><span id="cit26">Creatine kinase, https://en.wikipedia.org/wiki/Creatine_kinase, (accessed Oct 4, 2018).</span></li>
      <li><span id="cit27">Tris, https://en.wikipedia.org/wiki/Tris, (accessed Oct 4th, 2018).</span></li>
      <li><span id="cit28">cambio, Lab Reagents: Smart Buffers and Reagents, https://www.cambio.co.uk/24/1283/77/products/potassium-acetate/#tab-2, (accessed July 27th, 2018).</span></li>
      <li><span id="cit29">T. Yonesaki and T. Minagawa, <span class="italic">EMBO J.</span>, 1985, <span class="bold">4</span>, 3321–3327 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/j.1460-2075.1985.tb04083.x" title="DOI Link to resource 10.1002/j.1460-2075.1985.tb04083.x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL28XhsVChs7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=2936601%5Buid%5D" title="PubMed Link to resource 2936601">PubMed</a>.</span></li>
      <li><span id="cit30">L. D. Harris and J. Griffith, <span class="italic">J. Biol. Chem.</span>, 1987, <span class="bold">262</span>, 9285–9292 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2sXksFChu78%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit31">TwistDxLtd., TwistAmp® combined instruction manual, https://www.twistdx.co.uk/docs/default-source/RPA-assay-design/ta01cmanual-combined-manual_revo_v1–3a.pdf?sfvrsn=6, (accessed Oct 4th, 2018).</span></li>
      <li><span id="cit32">M. M. Cox and I. R. Lehman, <span class="italic">J. Biol. Chem.</span>, 1982, <span class="bold">257</span>, 8523–8532 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL38Xlt1Whsb8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit33">L. D. Harris and J. D. Griffith, <span class="italic">J. Mol. Biol.</span>, 1989, <span class="bold">206</span>, 19–27 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0022-2836(89)90520-2" title="DOI Link to resource 10.1016/0022-2836(89)90520-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL1MXhvVKqtr0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=2522995%5Buid%5D" title="PubMed Link to resource 2522995">PubMed</a>.</span></li>
      <li><span id="cit34">T. Yonesaki and T. Minagawa, <span class="italic">J. Biol. Chem.</span>, 1989, <span class="bold">264</span>, 7814–7820 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL1MXksFamt7k%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit35">K. Hashimoto and T. Yonesaki, <span class="italic">J. Biol. Chem.</span>, 1991, <span class="bold">266</span>, 4883–4888 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3MXhvVSku7c%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit36">M. Euler, Y. Wang, O. Nentwich, O. Piepenburg, F. T. Hufert and M. Weidmann, <span class="italic">J. Clin. Virol.</span>, 2012, <span class="bold">54</span>, 308–312 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jcv.2012.05.006" title="DOI Link to resource 10.1016/j.jcv.2012.05.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XosFWitbY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22683006%5Buid%5D" title="PubMed Link to resource 22683006">PubMed</a>.</span></li>
      <li><span id="cit37">S. Martorell, S. Palanca, A. Maquieira and L. A. Tortajada-Genaro, <span class="italic">Anal. Biochem.</span>, 2017, <span class="bold">544</span>, 49–56 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2017.12.013" title="DOI Link to resource 10.1016/j.ab.2017.12.013">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29248501%5Buid%5D" title="PubMed Link to resource 29248501">PubMed</a>.</span></li>
      <li><span id="cit38">M. Jauset-Rubio, J. Sabate Del Rio, T. Mairal, M. Svobodova, M. S. El-Shahawi, A. S. Bashammakh, A. O. Alyoubi and C. K. O'Sullivan, <span class="italic">Anal. Bioanal. Chem.</span>, 2017, <span class="bold">409</span>, 143–149 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-016-9973-2" title="DOI Link to resource 10.1007/s00216-016-9973-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhslSms7jN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27766362%5Buid%5D" title="PubMed Link to resource 27766362">PubMed</a>.</span></li>
      <li><span id="cit39">M. Jauset-Rubio, M. Svobodova, T. Mairal, C. McNeil, N. Keegan, M. S. El-Shahawi, A. S. Bashammakh, A. O. Alyoubi and C. K. O'Sullivan, <span class="italic">Anal. Chem.</span>, 2016, <span class="bold">88</span>, 10701–10709 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.6b03256" title="DOI Link to resource 10.1021/acs.analchem.6b03256">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yiu7bP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27753473%5Buid%5D" title="PubMed Link to resource 27753473">PubMed</a>.</span></li>
      <li><span id="cit40">M. Jauset-Rubio, M. Svobodova, T. Mairal, C. McNeil, N. Keegan, A. Saeed, M. N. Abbas, M. S. El-Shahawi, A. S. Bashammakh, A. O. Alyoubi and O. S. CK, <span class="italic">Sci. Rep.</span>, 2016, <span class="bold">6</span>, 37732 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep37732" title="DOI Link to resource 10.1038/srep37732">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XitFSmurfO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27886248%5Buid%5D" title="PubMed Link to resource 27886248">PubMed</a>.</span></li>
      <li><span id="cit41">J. Wang, J. Wang, Y. Geng and W. Yuan, <span class="italic">Can. J. Vet. Res.</span>, 2017, <span class="bold">81</span>, 308–312 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Can.%20J.%20Vet.%20Res.%5BJour%5D%20AND%2081%5Bvolume%5D%20AND%20308%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit42">TwistAmp® manuals, https://www.twistdx.co.uk/en/support/manuals/twistamp-manuals, (accessed Oct 4th, 2018).</span></li>
      <li><span id="cit43">O. Mayboroda, A. Gonzalez Benito, J. Sabate del Rio, M. Svobodova, S. Julich, H. Tomaso, C. K. O'Sullivan and I. Katakis, <span class="italic">Anal. Bioanal. Chem.</span>, 2016, <span class="bold">408</span>, 671–676 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-015-9177-1" title="DOI Link to resource 10.1007/s00216-015-9177-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvVChsrvI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26563112%5Buid%5D" title="PubMed Link to resource 26563112">PubMed</a>.</span></li>
      <li><span id="cit44">R. Wang, F. Zhang, L. Wang, W. Qian, C. Qian, J. Wu and Y. Ying, <span class="italic">Anal. Chem.</span>, 2017, <span class="bold">89</span>, 4413–4418 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.7b00964" title="DOI Link to resource 10.1021/acs.analchem.7b00964">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXkvVyhsrg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28345860%5Buid%5D" title="PubMed Link to resource 28345860">PubMed</a>.</span></li>
      <li><span id="cit45">S. L. Fuller, E. A. Savory, A. J. Weisberg, J. Z. Buser, M. I. Gordon, M. L. Putnam and J. H. Chang, <span class="italic">Phytopathology</span>, 2017, <span class="bold">107</span>, 1062–1068 <a target="_blank" class="DOILink" href="https://doi.org/10.1094/PHYTO-04-17-0144-R" title="DOI Link to resource 10.1094/PHYTO-04-17-0144-R">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28569126%5Buid%5D" title="PubMed Link to resource 28569126">PubMed</a>.</span></li>
      <li><span id="cit46">
          O. Piepenburg and N. A. Armes, DNA glycosylase/lyase and AP endonuclease substrates, Alere San Diego, Inc., US20110053153 A1,  2011.</span></li>
      <li><span id="cit47">L. D. Renner, J. Zan, L. I. Hu, M. Martinez, P. J. Resto, A. C. Siegel, C. Torres, S. B. Hall, T. R. Slezak, T. H. Nguyen and D. B. Weibel, <span class="italic">Appl. Environ. Microbiol.</span>, 2017, <span class="bold">83</span>, e02449–e02416 <a target="_blank" class="DOILink" href="https://doi.org/10.1128/AEM.02449-16" title="DOI Link to resource 10.1128/AEM.02449-16">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27986722%5Buid%5D" title="PubMed Link to resource 27986722">PubMed</a>.</span></li>
      <li><span id="cit48">
          T. A. Hall, BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT, <span class="italic">Nucleic acids symposium series</span>,  1999, vol. 41, pp. 95–98 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Nucleic%20acids%20symposium%20series%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201999%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit49">T. Koressaar and M. Remm, <span class="italic">Bioinformatics</span>, 2007, <span class="bold">23</span>, 1289–1291 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/bioinformatics/btm091" title="DOI Link to resource 10.1093/bioinformatics/btm091">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXntVOjur0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17379693%5Buid%5D" title="PubMed Link to resource 17379693">PubMed</a>.</span></li>
      <li><span id="cit50">N. R. Markham and M. Zuker, <span class="italic">Methods Mol. Biol.</span>, 2008, <span class="bold">453</span>, 3–31 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/978-1-60327-429-6_1" title="DOI Link to resource 10.1007/978-1-60327-429-6_1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXosFGgt78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18712296%5Buid%5D" title="PubMed Link to resource 18712296">PubMed</a>.</span></li>
      <li><span id="cit51">M. Zuker, <span class="italic">Nucleic Acids Res.</span>, 2003, <span class="bold">31</span>, 3406–3415 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nar/gkg595" title="DOI Link to resource 10.1093/nar/gkg595">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXltVWisr8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12824337%5Buid%5D" title="PubMed Link to resource 12824337">PubMed</a>.</span></li>
      <li><span id="cit52">S. Hoshika, F. Chen, N. A. Leal and S. A. Benner, <span class="italic">Nucleic Acids Symp. Ser.</span>, 2008, 129–130 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nass/nrn066" title="DOI Link to resource 10.1093/nass/nrn066">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhtFGgtb%252FP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18776287%5Buid%5D" title="PubMed Link to resource 18776287">PubMed</a>.</span></li>
      <li><span id="cit53">N. Sharma, S. Hoshika, D. Hutter, K. M. Bradley and S. A. Benner, <span class="italic">ChemBioChem</span>, 2014, <span class="bold">15</span>, 2268–2274 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cbic.201402250" title="DOI Link to resource 10.1002/cbic.201402250">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhsFagtLvE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25209570%5Buid%5D" title="PubMed Link to resource 25209570">PubMed</a>.</span></li>
      <li><span id="cit54">Y. Yang, X. Qin, W. Zhang, Y. Li and Z. Zhang, <span class="italic">Mol. Cell. Probes</span>, 2016, <span class="bold">30</span>, 300–305 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mcp.2016.08.011" title="DOI Link to resource 10.1016/j.mcp.2016.08.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFCrtrrN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27593155%5Buid%5D" title="PubMed Link to resource 27593155">PubMed</a>.</span></li>
      <li><span id="cit55">X. Xia, Y. Yu, L. Hu, M. Weidmann, Y. Pan, S. Yan and Y. Wang, <span class="italic">Arch. Virol.</span>, 2015, <span class="bold">160</span>, 987–994 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00705-015-2357-7" title="DOI Link to resource 10.1007/s00705-015-2357-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXitlejs7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25655264%5Buid%5D" title="PubMed Link to resource 25655264">PubMed</a>.</span></li>
      <li><span id="cit56">TwistDxLtd., TwistDx, https://www.twistdx.co.uk, (accessed Oct 4th, 2018).</span></li>
      <li><span id="cit57">L. Lillis, J. Siverson, A. Lee, J. Cantera, M. Parker, O. Piepenburg, D. A. Lehman and D. S. Boyle, <span class="italic">Mol. Cell. Probes</span>, 2016, <span class="bold">30</span>, 74–78 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mcp.2016.01.009" title="DOI Link to resource 10.1016/j.mcp.2016.01.009">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XitlSntbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26854117%5Buid%5D" title="PubMed Link to resource 26854117">PubMed</a>.</span></li>
      <li><span id="cit58">TwistDxLtd., Custom freeze-drying, https://www.twistdx.co.uk/en/products/custom-freeze-drying, (accessed May 3rd, 2018).</span></li>
      <li><span id="cit59">G. Posthuma-Trumpie, J. Wichers, M. Koets, L. J. M. Berendsen and A. Amerongen, <span class="italic">Anal. Bioanal. Chem.</span>, 2012, <span class="bold">402</span>, 593–600 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-011-5340-5" title="DOI Link to resource 10.1007/s00216-011-5340-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhtFeqtLrO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21904801%5Buid%5D" title="PubMed Link to resource 21904801">PubMed</a>.</span></li>
      <li><span id="cit60">K. Poulton and B. Webster, <span class="italic">Anal. Biochem.</span>, 2018, <span class="bold">546</span>, 65–71 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2018.01.031" title="DOI Link to resource 10.1016/j.ab.2018.01.031">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXis1Snu70%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29425749%5Buid%5D" title="PubMed Link to resource 29425749">PubMed</a>.</span></li>
      <li><span id="cit61">Z. A. Crannell, B. Rohrman and R. Richards-Kortum, <span class="italic">PLoS One</span>, 2014, <span class="bold">9</span>, e112146 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0112146" title="DOI Link to resource 10.1371/journal.pone.0112146">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25372030%5Buid%5D" title="PubMed Link to resource 25372030">PubMed</a>.</span></li>
      <li><span id="cit62">S. Kersting, V. Rausch, F. F. Bier and M. von Nickisch-Rosenegk, <span class="italic">Malar. J.</span>, 2014, <span class="bold">13</span>, 99 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/1475-2875-13-99" title="DOI Link to resource 10.1186/1475-2875-13-99">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24629133%5Buid%5D" title="PubMed Link to resource 24629133">PubMed</a>.</span></li>
      <li><span id="cit63">L. Lillis, D. Lehman, M. C. Singhal, J. Cantera, J. Singleton, P. Labarre, A. Toyama, O. Piepenburg, M. Parker, R. Wood, J. Overbaugh and D. S. Boyle, <span class="italic">PLoS One</span>, 2014, <span class="bold">9</span>, e108189 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0108189" title="DOI Link to resource 10.1371/journal.pone.0108189">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25264766%5Buid%5D" title="PubMed Link to resource 25264766">PubMed</a>.</span></li>
      <li><span id="cit64">Y. Yang, X. Qin, G. Wang, J. Jin, Y. Shang and Z. Zhang, <span class="italic">Virol. J.</span>, 2016, <span class="bold">13</span>, 46 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12985-016-0502-x" title="DOI Link to resource 10.1186/s12985-016-0502-x">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26993468%5Buid%5D" title="PubMed Link to resource 26993468">PubMed</a>.</span></li>
      <li><span id="cit65">K. Sun, W. Xing, X. Yu, W. Fu, Y. Wang, M. Zou, Z. Luo and D. Xu, <span class="italic">Parasites Vectors</span>, 2016, <span class="bold">9</span>, 476 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13071-016-1745-5" title="DOI Link to resource 10.1186/s13071-016-1745-5">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27577576%5Buid%5D" title="PubMed Link to resource 27577576">PubMed</a>.</span></li>
      <li><span id="cit66">M. A. Prescott, A. N. Reed, L. Jin and M. K. Pastey, <span class="italic">J. Aquat. Anim. Health</span>, 2016, <span class="bold">28</span>, 173–180 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/08997659.2016.1185048" title="DOI Link to resource 10.1080/08997659.2016.1185048">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xhtlaku7jO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27485254%5Buid%5D" title="PubMed Link to resource 27485254">PubMed</a>.</span></li>
      <li><span id="cit67">F. Yin, J. Liu, A. Liu, Y. Li, J. Luo, G. Guan and H. Yin, <span class="italic">Vet. Parasitol.</span>, 2017, <span class="bold">237</span>, 125–129 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.vetpar.2017.02.019" title="DOI Link to resource 10.1016/j.vetpar.2017.02.019">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGhurs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28249769%5Buid%5D" title="PubMed Link to resource 28249769">PubMed</a>.</span></li>
      <li><span id="cit68">J. S. Del Rio, I. M. Lobato, O. Mayboroda, I. Katakis and C. K. O'Sullivan, <span class="italic">Anal. Bioanal. Chem.</span>, 2017, <span class="bold">409</span>, 3261–3269 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-017-0269-y" title="DOI Link to resource 10.1007/s00216-017-0269-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXjslWgtrk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28255921%5Buid%5D" title="PubMed Link to resource 28255921">PubMed</a>.</span></li>
      <li><span id="cit69">Y. Yang, X. Qin, Y. Sun, G. Cong, Y. Li and Z. Zhang, <span class="italic">BioMed. Res. Int.</span>, 2017, <span class="bold">2017</span>, 8403642 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=BioMed.%20Res.%20Int.%5BJour%5D%20AND%202017%5Bvolume%5D%20AND%208403642%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit70">Y. Yang, X. Qin, W. Zhang, Z. Li, S. Zhang, Y. Li and Z. Zhang, <span class="italic">Mol. Cell. Probes</span>, 2017, <span class="bold">33</span>, 32–35 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mcp.2017.03.005" title="DOI Link to resource 10.1016/j.mcp.2017.03.005">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXltVajt7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28342800%5Buid%5D" title="PubMed Link to resource 28342800">PubMed</a>.</span></li>
      <li><span id="cit71">Y. D. Wu, M. J. Xu, Q. Q. Wang, C. X. Zhou, M. Wang, X. Q. Zhu and D. H. Zhou, <span class="italic">Vet. Parasitol.</span>, 2017, <span class="bold">243</span>, 199–203 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.vetpar.2017.06.026" title="DOI Link to resource 10.1016/j.vetpar.2017.06.026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXht1GitrzP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28807294%5Buid%5D" title="PubMed Link to resource 28807294">PubMed</a>.</span></li>
      <li><span id="cit72">N. Valasevich and B. Schneider, <span class="italic">J. Phytopathol.</span>, 2017, <span class="bold">165</span>, 762–770 <a target="_blank" class="DOILink" href="https://doi.org/10.1111/jph.12616" title="DOI Link to resource 10.1111/jph.12616">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWltLjP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit73">E. Tian, Q. Liu, H. Ye, F. Li and Z. Chao, <span class="italic">Molecules</span>, 2017, <span class="bold">22</span>, 2261–2270 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/molecules22122261" title="DOI Link to resource 10.3390/molecules22122261">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29258287%5Buid%5D" title="PubMed Link to resource 29258287">PubMed</a>.</span></li>
      <li><span id="cit74">P. Hou, G. Zhao, H. Wang, C. He, Y. Huan and H. He, <span class="italic">Mol. Cell. Probes</span>, 2018, <span class="bold">38</span>, 31–37 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mcp.2017.12.003" title="DOI Link to resource 10.1016/j.mcp.2017.12.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjvVKjuw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29288049%5Buid%5D" title="PubMed Link to resource 29288049">PubMed</a>.</span></li>
      <li><span id="cit75">M. Y. Lai, C. H. Ooi and Y. L. Lau, <span class="italic">Am. J. Trop. Med. Hyg.</span>, 2018, <span class="bold">98</span>, 700–703 <a target="_blank" class="DOILink" href="https://doi.org/10.4269/ajtmh.17-0738" title="DOI Link to resource 10.4269/ajtmh.17-0738">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitFymu77J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29260656%5Buid%5D" title="PubMed Link to resource 29260656">PubMed</a>.</span></li>
      <li><span id="cit76">Z. Guimin, W. Hongmei, H. Peili, H. Chengqiang and H. Hongbin, <span class="italic">J. Vet. Sci.</span>, 2017, <span class="bold">19</span>, 242–250 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Vet.%20Sci.%5BJour%5D%20AND%2019%5Bvolume%5D%20AND%20242%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit77">G. Zhao, H. Wang, P. Hou, C. He and H. He, <span class="italic">J. Vet. Sci.</span>, 2018, <span class="bold">19</span>, 242–250 <a target="_blank" class="DOILink" href="https://doi.org/10.4142/jvs.2018.19.2.242" title="DOI Link to resource 10.4142/jvs.2018.19.2.242">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29284204%5Buid%5D" title="PubMed Link to resource 29284204">PubMed</a>.</span></li>
      <li><span id="cit78">H. Soliman and M. El-Matbouli, <span class="italic">J. Fish Dis.</span>, 2018, <span class="bold">41</span>, 761–772 <a target="_blank" class="DOILink" href="https://doi.org/10.1111/jfd.12774" title="DOI Link to resource 10.1111/jfd.12774">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXns1OjtLk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29315637%5Buid%5D" title="PubMed Link to resource 29315637">PubMed</a>.</span></li>
      <li><span id="cit79">L. Wambua, B. Schneider, A. Okwaro, J. O. Wanga, O. Imali, P. N. Wambua, L. Agutu, C. Olds, C. S. Jones, D. Masiga, C. Midega, Z. Khan, J. Jores and A. Fischer, <span class="italic">Mol. Cell. Probes</span>, 2017, <span class="bold">35</span>, 44–56 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mcp.2017.06.004" title="DOI Link to resource 10.1016/j.mcp.2017.06.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSmtLrJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28647581%5Buid%5D" title="PubMed Link to resource 28647581">PubMed</a>.</span></li>
      <li><span id="cit80">S. Kalsi, M. Valiadi, M. N. Tsaloglou, L. Parry-Jones, A. Jacobs, R. Watson, C. Turner, R. Amos, B. Hadwen, J. Buse, C. Brown, M. Sutton and H. Morgan, <span class="italic">Lab Chip</span>, 2015, <span class="bold">15</span>, 3065–3075 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5LC00462D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5LC00462D">RSC</a>.</span></li>
      <li><span id="cit81">C. Moody, H. Newell and H. Viljoen, <span class="italic">Biochem. Eng. J.</span>, 2016, <span class="bold">112</span>, 193–201 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bej.2016.04.017" title="DOI Link to resource 10.1016/j.bej.2016.04.017">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XnsFWgtLg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit82">D. S. Boyle, D. A. Lehman, L. Lillis, D. Peterson, M. Singhal, N. Armes, M. Parker, O. Piepenburg and J. Overbaugh, <span class="italic">mBio</span>, 2013, <span class="bold">4</span>, e00135-13 <a target="_blank" class="DOILink" href="https://doi.org/10.1128/mBio.00135-13" title="DOI Link to resource 10.1128/mBio.00135-13">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23549916%5Buid%5D" title="PubMed Link to resource 23549916">PubMed</a>.</span></li>
      <li><span id="cit83">A. Abd El Wahed, A. El-Deeb, M. El-Tholoth, H. Abd El Kader, A. Ahmed, S. Hassan, B. Hoffmann, B. Haas, M. A. Shalaby, F. T. Hufert and M. Weidmann, <span class="italic">PLoS One</span>, 2013, <span class="bold">8</span>, e71642 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0071642" title="DOI Link to resource 10.1371/journal.pone.0071642">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhtlKmurfN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23977101%5Buid%5D" title="PubMed Link to resource 23977101">PubMed</a>.</span></li>
      <li><span id="cit84">R. K. Daher, G. Stewart, M. Boissinot, D. K. Boudreau and M. G. Bergeron, <span class="italic">Mol. Cell. Probes</span>, 2015, <span class="bold">29</span>, 116–121 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mcp.2014.11.005" title="DOI Link to resource 10.1016/j.mcp.2014.11.005">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitV2mtrjP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25481659%5Buid%5D" title="PubMed Link to resource 25481659">PubMed</a>.</span></li>
      <li><span id="cit85">N. Yehia, A. S. Arafa, A. Abd El Wahed, A. A. El-Sanousi, M. Weidmann and M. A. Shalaby, <span class="italic">J. Virol. Methods</span>, 2015, <span class="bold">223</span>, 45–49 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jviromet.2015.07.011" title="DOI Link to resource 10.1016/j.jviromet.2015.07.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXht1GrsLnP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26225482%5Buid%5D" title="PubMed Link to resource 26225482">PubMed</a>.</span></li>
      <li><span id="cit86">L. Lillis, D. A. Lehman, J. B. Siverson, J. Weis, J. Cantera, M. Parker, O. Piepenburg, J. Overbaugh and D. S. Boyle, <span class="italic">J. Virol. Methods</span>, 2016, <span class="bold">230</span>, 28–35 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jviromet.2016.01.010" title="DOI Link to resource 10.1016/j.jviromet.2016.01.010">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xhs12hu7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26821087%5Buid%5D" title="PubMed Link to resource 26821087">PubMed</a>.</span></li>
      <li><span id="cit87">P. Patel, A. Abd El Wahed, O. Faye, P. Pruger, M. Kaiser, S. Thaloengsok, S. Ubol, A. Sakuntabhai, I. Leparc-Goffart, F. T. Hufert, A. A. Sall, M. Weidmann and M. Niedrig, <span class="italic">PLoS Neglected Trop. Dis.</span>, 2016, <span class="bold">10</span>, e0004953 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pntd.0004953" title="DOI Link to resource 10.1371/journal.pntd.0004953">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27685649%5Buid%5D" title="PubMed Link to resource 27685649">PubMed</a>.</span></li>
      <li><span id="cit88">M. D. Moore and L. A. Jaykus, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 40244 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep40244" title="DOI Link to resource 10.1038/srep40244">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXotlensg%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28067278%5Buid%5D" title="PubMed Link to resource 28067278">PubMed</a>.</span></li>
      <li><span id="cit89">J. Kissenkötter, S. Hansen, S. Böhlken-Fascher, O. G. Ademowo, O. E. Oyinloye, A. S. Bakarey, G. Dobler, D. Tappe, P. Patel, C.-P. Czerny and A. Abd El Wahed, <span class="italic">Anal. Biochem.</span>, 2017, <span class="bold">544</span>, 29–33 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2017.12.018" title="DOI Link to resource 10.1016/j.ab.2017.12.018">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29258829%5Buid%5D" title="PubMed Link to resource 29258829">PubMed</a>.</span></li>
      <li><span id="cit90">E. S. Yamanaka, L. A. Tortajada-Genaro and Á. Maquieira, <span class="italic">Microchim. Acta</span>, 2017, <span class="bold">184</span>, 1453–1462 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-017-2144-0" title="DOI Link to resource 10.1007/s00604-017-2144-0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXjslWgt70%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit91">A. Rosser, D. Rollinson, M. Forrest and B. L. Webster, <span class="italic">Parasites Vectors</span>, 2015, <span class="bold">8</span>, 446 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13071-015-1055-3" title="DOI Link to resource 10.1186/s13071-015-1055-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BC287psVWqsw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26338510%5Buid%5D" title="PubMed Link to resource 26338510">PubMed</a>.</span></li>
      <li><span id="cit92">K. Krolov, J. Frolova, O. Tudoran, J. Suhorutsenko, T. Lehto, H. Sibul, I. Mager, M. Laanpere, I. Tulp and U. Langel, <span class="italic">J. Mol. Diagn.</span>, 2014, <span class="bold">16</span>, 127–135 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmoldx.2013.08.003" title="DOI Link to resource 10.1016/j.jmoldx.2013.08.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhvFyqs73P" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24331366%5Buid%5D" title="PubMed Link to resource 24331366">PubMed</a>.</span></li>
      <li><span id="cit93">H. B. Liu, Y. X. Zang, X. J. Du, P. Li and S. Wang, <span class="italic">J. Dairy Sci.</span>, 2017, <span class="bold">100</span>, 7016–7025 <a target="_blank" class="DOILink" href="https://doi.org/10.3168/jds.2017-12566" title="DOI Link to resource 10.3168/jds.2017-12566">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFKht7jE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28711269%5Buid%5D" title="PubMed Link to resource 28711269">PubMed</a>.</span></li>
      <li><span id="cit94">B. Rohrman and R. Richards-Kortum, <span class="italic">Anal. Chem.</span>, 2015, <span class="bold">87</span>, 1963–1967 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac504365v" title="DOI Link to resource 10.1021/ac504365v">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXks1KmtA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25560368%5Buid%5D" title="PubMed Link to resource 25560368">PubMed</a>.</span></li>
      <li><span id="cit95">C. C. Chao, T. Belinskaya, Z. Zhang and W. M. Ching, <span class="italic">PLoS Neglected Trop. Dis.</span>, 2015, <span class="bold">9</span>, e0003884 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pntd.0003884" title="DOI Link to resource 10.1371/journal.pntd.0003884">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26161793%5Buid%5D" title="PubMed Link to resource 26161793">PubMed</a>.</span></li>
      <li><span id="cit96">B. Y. C. Ng, E. J. H. Wee, N. P. West and M. Trau, <span class="italic">Sci. Rep.</span>, 2015, <span class="bold">5</span>, 15027 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep15027" title="DOI Link to resource 10.1038/srep15027">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KksrrP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit97">E. Clancy, O. Higgins, M. S. Forrest, T. W. Boo, M. Cormican, T. Barry, O. Piepenburg and T. J. Smith, <span class="italic">BMC Infect. Dis.</span>, 2015, <span class="bold">15</span>, 481 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12879-015-1212-5" title="DOI Link to resource 10.1186/s12879-015-1212-5">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26515409%5Buid%5D" title="PubMed Link to resource 26515409">PubMed</a>.</span></li>
      <li><span id="cit98">H. Y. Lau, Y. Wang, E. J. Wee, J. R. Botella and M. Trau, <span class="italic">Anal. Chem.</span>, 2016, <span class="bold">88</span>, 8074–8081 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.6b01551" title="DOI Link to resource 10.1021/acs.analchem.6b01551">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtFGmtL%252FM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27403651%5Buid%5D" title="PubMed Link to resource 27403651">PubMed</a>.</span></li>
      <li><span id="cit99">K. M. Koo, E. J. Wee, P. N. Mainwaring, Y. Wang and M. Trau, <span class="italic">Small</span>, 2016, <span class="bold">12</span>, 6233–6242 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/smll.201602161" title="DOI Link to resource 10.1002/smll.201602161">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFOms7bJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27717152%5Buid%5D" title="PubMed Link to resource 27717152">PubMed</a>.</span></li>
      <li><span id="cit100">H. Y. Lau, H. Wu, E. J. Wee, M. Trau, Y. Wang and J. R. Botella, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 38896 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep38896" title="DOI Link to resource 10.1038/srep38896">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ogs7Y%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28094255%5Buid%5D" title="PubMed Link to resource 28094255">PubMed</a>.</span></li>
      <li><span id="cit101">K. M. Koo, E. J. Wee and M. Trau, <span class="italic">Theranostics</span>, 2016, <span class="bold">6</span>, 1415–1424 <a target="_blank" class="DOILink" href="https://doi.org/10.7150/thno.15250" title="DOI Link to resource 10.7150/thno.15250">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXns1Cisrw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27375789%5Buid%5D" title="PubMed Link to resource 27375789">PubMed</a>.</span></li>
      <li><span id="cit102">B. Y. C. Ng, E. J. H. Wee, N. P. West and M. Trau, <span class="italic">ACS Sens.</span>, 2016, <span class="bold">1</span>, 173–178 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssensors.5b00171" title="DOI Link to resource 10.1021/acssensors.5b00171">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWhsLbM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit103">B. Y. Ng, W. Xiao, N. P. West, E. J. Wee, Y. Wang and M. Trau, <span class="italic">Anal. Chem.</span>, 2015, <span class="bold">87</span>, 10613–10618 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.5b03121" title="DOI Link to resource 10.1021/acs.analchem.5b03121">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWqtLbE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26382883%5Buid%5D" title="PubMed Link to resource 26382883">PubMed</a>.</span></li>
      <li><span id="cit104">S. Santiago-Felipe, L. A. Tortajada-Genaro, R. Puchades and A. Maquieira, <span class="italic">Anal. Chim. Acta</span>, 2014, <span class="bold">811</span>, 81–87 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2013.12.017" title="DOI Link to resource 10.1016/j.aca.2013.12.017">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXkvFyqug%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24456598%5Buid%5D" title="PubMed Link to resource 24456598">PubMed</a>.</span></li>
      <li><span id="cit105">Z. Crannell, A. Castellanos-Gonzalez, G. Nair, R. Mejia, A. C. White and R. Richards-Kortum, <span class="italic">Anal. Chem.</span>, 2016, <span class="bold">88</span>, 1610–1616 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.5b03267" title="DOI Link to resource 10.1021/acs.analchem.5b03267">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXitVejur7N" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26669715%5Buid%5D" title="PubMed Link to resource 26669715">PubMed</a>.</span></li>
      <li><span id="cit106">M. S. Cordray and R. R. Richards-Kortum, <span class="italic">Malar. J.</span>, 2015, <span class="bold">14</span>, 472 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12936-015-0995-6" title="DOI Link to resource 10.1186/s12936-015-0995-6">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26611141%5Buid%5D" title="PubMed Link to resource 26611141">PubMed</a>.</span></li>
      <li><span id="cit107">J. Wang, L. Liu, R. Li, J. Wang, Q. Fu and W. Yuan, <span class="italic">Arch. Virol.</span>, 2016, <span class="bold">161</span>, 1015–1018 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00705-015-2738-y" title="DOI Link to resource 10.1007/s00705-015-2738-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XislalsA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26729477%5Buid%5D" title="PubMed Link to resource 26729477">PubMed</a>.</span></li>
      <li><span id="cit108">P. A. Tu, J. S. Shiu, S. H. Lee, V. F. Pang, D. C. Wang and P. H. Wang, <span class="italic">J. Virol. Methods</span>, 2017, <span class="bold">243</span>, 98–104 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jviromet.2017.01.023" title="DOI Link to resource 10.1016/j.jviromet.2017.01.023">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXislyiur0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28159666%5Buid%5D" title="PubMed Link to resource 28159666">PubMed</a>.</span></li>
      <li><span id="cit109">L. Glais and E. Jacquot, <span class="italic">Methods Mol. Biol.</span>, 2015, <span class="bold">1302</span>, 207–225 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/978-1-4939-2620-6_16" title="DOI Link to resource 10.1007/978-1-4939-2620-6_16">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25981257%5Buid%5D" title="PubMed Link to resource 25981257">PubMed</a>.</span></li>
      <li><span id="cit110">B. Babu, B. K. Washburn, S. H. Miller, K. Poduch, T. Sarigul, G. W. Knox, F. M. Ochoa-Corona and M. L. Paret, <span class="italic">J. Virol. Methods</span>, 2017, <span class="bold">240</span>, 78–84 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jviromet.2016.11.014" title="DOI Link to resource 10.1016/j.jviromet.2016.11.014">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XitVyjurbP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27915036%5Buid%5D" title="PubMed Link to resource 27915036">PubMed</a>.</span></li>
      <li><span id="cit111">M. A. Londono, C. L. Harmon and J. E. Polston, <span class="italic">Virol. J.</span>, 2016, <span class="bold">13</span>, 48 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12985-016-0504-8" title="DOI Link to resource 10.1186/s12985-016-0504-8">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27000806%5Buid%5D" title="PubMed Link to resource 27000806">PubMed</a>.</span></li>
      <li><span id="cit112">R. Kapoor, N. Srivastava, S. Kumar, R. K. Saritha, S. K. Sharma, R. K. Jain and V. K. Baranwal, <span class="italic">Arch. Virol.</span>, 2017, <span class="bold">162</span>, 2791–2796 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00705-017-3399-9" title="DOI Link to resource 10.1007/s00705-017-3399-9">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXotVKmtrw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28500444%5Buid%5D" title="PubMed Link to resource 28500444">PubMed</a>.</span></li>
      <li><span id="cit113">Q. Liu, B. K. L. Lim, S. Y. Lim, W. Y. Tang, Z. Gu, J. Chung, M. K. Park and T. Barkham, <span class="italic">Sens. Actuators, B</span>, 2018, <span class="bold">255</span>, 1595–1603 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.08.181" title="DOI Link to resource 10.1016/j.snb.2017.08.181">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVGhs7jP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit114">M. L. Powell, F. R. Bowler, A. J. Martinez, C. J. Greenwood, N. Armes and O. Piepenburg, <span class="italic">Anal. Biochem.</span>, 2017, <span class="bold">543</span>, 108–115 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2017.12.003" title="DOI Link to resource 10.1016/j.ab.2017.12.003">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29233680%5Buid%5D" title="PubMed Link to resource 29233680">PubMed</a>.</span></li>
      <li><span id="cit115">O. A. Saldarriaga, A. Castellanos-Gonzalez, R. Porrozzi, G. C. Baldeviano, A. G. Lescano, M. B. de Los Santos, O. L. Fernandez, N. G. Saravia, E. Costa, P. C. Melby and B. L. Travi, <span class="italic">PLoS Neglected Trop. Dis.</span>, 2016, <span class="bold">10</span>, e0004638 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pntd.0004638" title="DOI Link to resource 10.1371/journal.pntd.0004638">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27115155%5Buid%5D" title="PubMed Link to resource 27115155">PubMed</a>.</span></li>
      <li><span id="cit116">Y. D. Wu, D. H. Zhou, L. X. Zhang, W. B. Zheng, J. G. Ma, M. Wang, X. Q. Zhu and M. J. Xu, <span class="italic">Parasitol. Res.</span>, 2016, <span class="bold">115</span>, 3551–3555 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00436-016-5120-4" title="DOI Link to resource 10.1007/s00436-016-5120-4">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27174027%5Buid%5D" title="PubMed Link to resource 27174027">PubMed</a>.</span></li>
      <li><span id="cit117">C. Escadafal, O. Faye, A. A. Sall, O. Faye, M. Weidmann, O. Strohmeier, F. von Stetten, J. Drexler, M. Eberhard, M. Niedrig and P. Patel, <span class="italic">PLoS Neglected Trop. Dis.</span>, 2014, <span class="bold">8</span>, e2730 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pntd.0002730" title="DOI Link to resource 10.1371/journal.pntd.0002730">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24603874%5Buid%5D" title="PubMed Link to resource 24603874">PubMed</a>.</span></li>
      <li><span id="cit118">A. J. Saah and D. R. Hoover, <span class="italic">Ann. Intern. Med.</span>, 1997, <span class="bold">126</span>, 91–94 <a target="_blank" class="DOILink" href="https://doi.org/10.7326/0003-4819-126-1-199701010-00026" title="DOI Link to resource 10.7326/0003-4819-126-1-199701010-00026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADyaK2s7isl2ltA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=8992938%5Buid%5D" title="PubMed Link to resource 8992938">PubMed</a>.</span></li>
      <li><span id="cit119">J. Y. Kim and J.-L. Lee, <span class="italic">J. Food Saf.</span>, 2016, <span class="bold">36</span>, 402–411 <a target="_blank" class="DOILink" href="https://doi.org/10.1111/jfs.12261" title="DOI Link to resource 10.1111/jfs.12261">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1eht7zK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit120">Q. Liu, J. Nam, S. Kim, C. T. Lim, M. K. Park and Y. Shin, <span class="italic">Biosens. Bioelectron.</span>, 2016, <span class="bold">82</span>, 1–8 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2016.03.050" title="DOI Link to resource 10.1016/j.bios.2016.03.050">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27031184%5Buid%5D" title="PubMed Link to resource 27031184">PubMed</a>.</span></li>
      <li><span id="cit121">D. Mondal, P. Ghosh, M. A. Khan, F. Hossain, S. Bohlken-Fascher, G. Matlashewski, A. Kroeger, P. Olliaro and A. Abd El Wahed, <span class="italic">Parasites Vectors</span>, 2016, <span class="bold">9</span>, 281 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13071-016-1572-8" title="DOI Link to resource 10.1186/s13071-016-1572-8">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27177926%5Buid%5D" title="PubMed Link to resource 27177926">PubMed</a>.</span></li>
      <li><span id="cit122">J. Y. Kim and J.-L. Lee, <span class="italic">Food Control</span>, 2017, <span class="bold">73</span>(Part B), 1247–1255 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.foodcont.2016.10.041" title="DOI Link to resource 10.1016/j.foodcont.2016.10.041">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xhslyjt7fP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit123">J. S. Gootenberg, O. O. Abudayyeh, J. W. Lee, P. Essletzbichler, A. J. Dy, J. Joung, V. Verdine, N. Donghia, N. M. Daringer, C. A. Freije, C. Myhrvold, R. P. Bhattacharyya, J. Livny, A. Regev, E. V. Koonin, D. T. Hung, P. C. Sabeti, J. J. Collins and F. Zhang, <span class="italic">Science</span>, 2017, <span class="bold">356</span>, 438–442 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aam9321" title="DOI Link to resource 10.1126/science.aam9321">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXmvFejtb8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28408723%5Buid%5D" title="PubMed Link to resource 28408723">PubMed</a>.</span></li>
      <li><span id="cit124">M. Y. Lai, C. H. Ooi and Y. L. Lau, <span class="italic">Am. J. Trop. Med. Hyg.</span>, 2017, <span class="bold">97</span>, 1597–1599 <a target="_blank" class="DOILink" href="https://doi.org/10.4269/ajtmh.17-0427" title="DOI Link to resource 10.4269/ajtmh.17-0427">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitV2lsL7N" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28820700%5Buid%5D" title="PubMed Link to resource 28820700">PubMed</a>.</span></li>
      <li><span id="cit125">H. M. Amer, A. Abd El Wahed, M. A. Shalaby, F. N. Almajhdi, F. T. Hufert and M. Weidmann, <span class="italic">J. Virol. Methods</span>, 2013, <span class="bold">193</span>, 337–340 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jviromet.2013.06.027" title="DOI Link to resource 10.1016/j.jviromet.2013.06.027">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhsVegsr%252FM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23811231%5Buid%5D" title="PubMed Link to resource 23811231">PubMed</a>.</span></li>
      <li><span id="cit126">R. K. Daher, G. Stewart, M. Boissinot and M. G. Bergeron, <span class="italic">Clin. Chem.</span>, 2014, <span class="bold">60</span>, 660–666 <a target="_blank" class="DOILink" href="https://doi.org/10.1373/clinchem.2013.213504" title="DOI Link to resource 10.1373/clinchem.2013.213504">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXmt1ClurY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24463560%5Buid%5D" title="PubMed Link to resource 24463560">PubMed</a>.</span></li>
      <li><span id="cit127">Z. A. Crannell, A. Castellanos-Gonzalez, A. Irani, B. Rohrman, A. C. White and R. Richards-Kortum, <span class="italic">Anal. Chem.</span>, 2014, <span class="bold">86</span>, 2565–2571 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac403750z" title="DOI Link to resource 10.1021/ac403750z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Wgsbs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24479858%5Buid%5D" title="PubMed Link to resource 24479858">PubMed</a>.</span></li>
      <li><span id="cit128">D. S. Boyle, R. McNerney, H. Teng Low, B. T. Leader, A. C. Perez-Osorio, J. C. Meyer, D. M. O'Sullivan, D. G. Brooks, O. Piepenburg and M. S. Forrest, <span class="italic">PLoS One</span>, 2014, <span class="bold">9</span>, e103091 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0103091" title="DOI Link to resource 10.1371/journal.pone.0103091">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25118698%5Buid%5D" title="PubMed Link to resource 25118698">PubMed</a>.</span></li>
      <li><span id="cit129">Z. A. Crannell, M. M. Cabada, A. Castellanos-Gonzalez, A. Irani, A. C. White and R. Richards-Kortum, <span class="italic">Am. J. Trop. Med. Hyg.</span>, 2015, <span class="bold">92</span>, 583–587 <a target="_blank" class="DOILink" href="https://doi.org/10.4269/ajtmh.14-0593" title="DOI Link to resource 10.4269/ajtmh.14-0593">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXktl2msLk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25510713%5Buid%5D" title="PubMed Link to resource 25510713">PubMed</a>.</span></li>
      <li><span id="cit130">Y. Shin, A. P. Perera, W. Y. Tang, D. L. Fu, Q. Liu, J. K. Sheng, Z. Gu, T. Y. Lee, T. Barkham and M. Kyoung Park, <span class="italic">Biosens. Bioelectron.</span>, 2015, <span class="bold">68C</span>, 390–396 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2015.01.030" title="DOI Link to resource 10.1016/j.bios.2015.01.030">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25615836%5Buid%5D" title="PubMed Link to resource 25615836">PubMed</a>.</span></li>
      <li><span id="cit131">A. Abd El Wahed, P. Patel, O. Faye, S. Thaloengsok, D. Heidenreich, P. Matangkasombut, K. Manopwisedjaroen, A. Sakuntabhai, A. A. Sall, F. T. Hufert and M. Weidmann, <span class="italic">PLoS One</span>, 2015, <span class="bold">10</span>, e0129682 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0129682" title="DOI Link to resource 10.1371/journal.pone.0129682">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26075598%5Buid%5D" title="PubMed Link to resource 26075598">PubMed</a>.</span></li>
      <li><span id="cit132">G. Nair, M. Rebolledo, A. C. White Jr., Z. Crannell, R. R. Richards-Kortum, A. E. Pinilla, J. D. Ramirez, M. C. Lopez and A. Castellanos-Gonzalez, <span class="italic">Am. J. Trop. Med. Hyg.</span>, 2015, <span class="bold">93</span>, 591–595 <a target="_blank" class="DOILink" href="https://doi.org/10.4269/ajtmh.15-0276" title="DOI Link to resource 10.4269/ajtmh.15-0276">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XmtV2ltbg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26123960%5Buid%5D" title="PubMed Link to resource 26123960">PubMed</a>.</span></li>
      <li><span id="cit133">S. A. Ahmed, W. W. van de Sande, M. Desnos-Ollivier, A. H. Fahal, N. A. Mhmoud and G. S. de Hoog, <span class="italic">J. Clin. Microbiol.</span>, 2015, <span class="bold">53</span>, 3280–3285 <a target="_blank" class="DOILink" href="https://doi.org/10.1128/JCM.01544-15" title="DOI Link to resource 10.1128/JCM.01544-15">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtlGqu7zI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26246484%5Buid%5D" title="PubMed Link to resource 26246484">PubMed</a>.</span></li>
      <li><span id="cit134">O. Faye, O. Faye, B. Soropogui, P. Patel, A. A. El Wahed, C. Loucoubar, G. Fall, D. Kiory, N. Magassouba, S. Keita, M. K. Konde, A. A. Diallo, L. Koivogui, H. Karlberg, A. Mirazimi, O. Nentwich, O. Piepenburg, M. Niedrig, M. Weidmann and A. A. Sall, <span class="italic">Euro. Surveill.</span>, 2015, <span class="bold">20</span>, 10–18 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Euro.%20Surveill.%5BJour%5D%20AND%2020%5Bvolume%5D%20AND%2010%5Bpage%5D%20and%202015%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit135">Y. Yang, X. Qin, G. Wang, Y. Zhang, Y. Shang and Z. Zhang, <span class="italic">Virol. J.</span>, 2015, <span class="bold">12</span>, 206 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12985-015-0440-z" title="DOI Link to resource 10.1186/s12985-015-0440-z">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26631157%5Buid%5D" title="PubMed Link to resource 26631157">PubMed</a>.</span></li>
      <li><span id="cit136">Y. Yang, X. Qin, Y. Sun, T. Chen and Z. Zhang, <span class="italic">Virus Genes</span>, 2016, <span class="bold">52</span>, 883–886 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11262-016-1378-y" title="DOI Link to resource 10.1007/s11262-016-1378-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtlGmtb3O" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27534870%5Buid%5D" title="PubMed Link to resource 27534870">PubMed</a>.</span></li>
      <li><span id="cit137">C. Clarke, L. O'Connor, H. Carre-Skinner, O. Piepenburg and T. J. Smith, <span class="italic">BMC Microbiol.</span>, 2016, <span class="bold">16</span>, 221–226 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12866-016-0836-y" title="DOI Link to resource 10.1186/s12866-016-0836-y">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27658382%5Buid%5D" title="PubMed Link to resource 27658382">PubMed</a>.</span></li>
      <li><span id="cit138">A. Abd El Wahed, S. S. Sanabani, O. Faye, R. Pessoa, J. V. Patriota, R. R. Giorgi, P. Patel, S. Bohlken-Fascher, O. Landt, M. Niedrig, P. M. Zanotto, C. P. Czerny, A. A. Sall and M. Weidmann, <span class="italic">PLoS Curr.</span>, 2017, <span class="bold">9</span>, 1–10 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=PLoS%20Curr.%5BJour%5D%20AND%209%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit139">M. A. Shalaby, A. El-Deeb, M. El-Tholoth, D. Hoffmann, C. P. Czerny, F. T. Hufert, M. Weidmann and A. Abd El Wahed, <span class="italic">BMC Vet. Res.</span>, 2016, <span class="bold">12</span>, 244 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12917-016-0875-5" title="DOI Link to resource 10.1186/s12917-016-0875-5">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27806722%5Buid%5D" title="PubMed Link to resource 27806722">PubMed</a>.</span></li>
      <li><span id="cit140">S. Hansen, J. Schafer, K. Fechner, C. P. Czerny and A. Abd El Wahed, <span class="italic">PLoS One</span>, 2016, <span class="bold">11</span>, e0168733 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0168733" title="DOI Link to resource 10.1371/journal.pone.0168733">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27992571%5Buid%5D" title="PubMed Link to resource 27992571">PubMed</a>.</span></li>
      <li><span id="cit141">K. M. Koo, E. J. H. Wee and M. Trau, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">89</span>, 715–720 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2016.11.024" title="DOI Link to resource 10.1016/j.bios.2016.11.024">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhvVymt7%252FJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27865106%5Buid%5D" title="PubMed Link to resource 27865106">PubMed</a>.</span></li>
      <li><span id="cit142">J. C. Wang, W. Z. Yuan, Q. A. Han, J. F. Wang and L. B. Liu, <span class="italic">J. Virol. Methods</span>, 2017, <span class="bold">243</span>, 55–60 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jviromet.2017.01.017" title="DOI Link to resource 10.1016/j.jviromet.2017.01.017">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXit1Wisb8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28122203%5Buid%5D" title="PubMed Link to resource 28122203">PubMed</a>.</span></li>
      <li><span id="cit143">L. Liu, L. Jiang, Y. Yu, X. Xia, Y. Pan, S. Yan and Y. Wang, <span class="italic">Mol. Cell. Probes</span>, 2017, <span class="bold">33</span>, 4–7 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mcp.2017.02.001" title="DOI Link to resource 10.1016/j.mcp.2017.02.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXisF2jtbw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28179103%5Buid%5D" title="PubMed Link to resource 28179103">PubMed</a>.</span></li>
      <li><span id="cit144">M. M. Cabada, J. L. Malaga, A. Castellanos-Gonzalez, K. A. Bagwell, P. A. Naeger, H. K. Rogers, S. Maharsi, M. Mbaka and A. C. White Jr., <span class="italic">Am. J. Trop. Med. Hyg.</span>, 2016, <span class="bold">96</span>, 341–346 <a target="_blank" class="DOILink" href="https://doi.org/10.4269/ajtmh.16-0601" title="DOI Link to resource 10.4269/ajtmh.16-0601">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27821691%5Buid%5D" title="PubMed Link to resource 27821691">PubMed</a>.</span></li>
      <li><span id="cit145">Y. Yang, X. Qin, Y. Song, W. Zhang, G. Hu, Y. Dou, Y. Li and Z. Zhang, <span class="italic">Virol. J.</span>, 2017, <span class="bold">14</span>, 24 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12985-017-0688-6" title="DOI Link to resource 10.1186/s12985-017-0688-6">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28173845%5Buid%5D" title="PubMed Link to resource 28173845">PubMed</a>.</span></li>
      <li><span id="cit146">M. Si Ammour, G. J. Bilodeau, D. M. Tremblay, H. Van der Heyden, T. Yaseen, L. Varvaro and O. Carisse, <span class="italic">Plant Dis.</span>, 2017, <span class="bold">101</span>, 1269–1277 <a target="_blank" class="DOILink" href="https://doi.org/10.1094/PDIS-12-16-1780-RE" title="DOI Link to resource 10.1094/PDIS-12-16-1780-RE">CrossRef</a>.</span></li>
      <li><span id="cit147">J. C. Wang, L. B. Liu, Q. A. Han, J. F. Wang and W. Z. Yuan, <span class="italic">J. Virol. Methods</span>, 2017, <span class="bold">248</span>, 145–147 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jviromet.2017.06.011" title="DOI Link to resource 10.1016/j.jviromet.2017.06.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFCru7jN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28690087%5Buid%5D" title="PubMed Link to resource 28690087">PubMed</a>.</span></li>
      <li><span id="cit148">Y. Yang, X. Qin, X. Zhang, Z. Zhao, W. Zhang, X. Zhu, G. Cong, Y. Li and Z. Zhang, <span class="italic">Virol. J.</span>, 2017, <span class="bold">14</span>, 131 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12985-017-0792-7" title="DOI Link to resource 10.1186/s12985-017-0792-7">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28716095%5Buid%5D" title="PubMed Link to resource 28716095">PubMed</a>.</span></li>
      <li><span id="cit149">J. Wang, J. Wang, R. Li, L. Liu and W. Yuan, <span class="italic">BMC Vet. Res.</span>, 2017, <span class="bold">13</span>, 241 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12917-017-1180-7" title="DOI Link to resource 10.1186/s12917-017-1180-7">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28810858%5Buid%5D" title="PubMed Link to resource 28810858">PubMed</a>.</span></li>
      <li><span id="cit150">K. Schlottau, C. M. Freuling, T. Muller, M. Beer and B. Hoffmann, <span class="italic">Virol. J.</span>, 2017, <span class="bold">14</span>, 184 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12985-017-0853-y" title="DOI Link to resource 10.1186/s12985-017-0853-y">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28938887%5Buid%5D" title="PubMed Link to resource 28938887">PubMed</a>.</span></li>
      <li><span id="cit151">C. E. Jin, S. S. Yeom, B. Koo, T. Y. Lee, J. H. Lee, Y. Shin and S. B. Lim, <span class="italic">Oncotarget</span>, 2017, <span class="bold">8</span>, 83860–83871 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Oncotarget%5BJour%5D%20AND%208%5Bvolume%5D%20AND%2083860%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit152">L. C. Bonney, R. J. Watson, B. Afrough, M. Mullojonova, V. Dzhuraeva, F. Tishkova and R. Hewson, <span class="italic">PLoS Neglected Trop. Dis.</span>, 2017, <span class="bold">11</span>, e0006013 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pntd.0006013" title="DOI Link to resource 10.1371/journal.pntd.0006013">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29028804%5Buid%5D" title="PubMed Link to resource 29028804">PubMed</a>.</span></li>
      <li><span id="cit153">Y. Geng, J. Wang, L. Liu, Y. Lu, K. Tan and Y.-Z. Chang, <span class="italic">BMC Vet. Res.</span>, 2017, <span class="bold">13</span>, 311 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s12917-017-1232-z" title="DOI Link to resource 10.1186/s12917-017-1232-z">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29110666%5Buid%5D" title="PubMed Link to resource 29110666">PubMed</a>.</span></li>
      <li><span id="cit154">K. Shahin, J. Gustavo Ramirez-Paredes, G. Harold, B. Lopez-Jimena, A. Adams and M. Weidmann, <span class="italic">PLoS One</span>, 2018, <span class="bold">13</span>, e0192979 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0192979" title="DOI Link to resource 10.1371/journal.pone.0192979">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29444148%5Buid%5D" title="PubMed Link to resource 29444148">PubMed</a>.</span></li>
      <li><span id="cit155">D. Yin, Y. Zhu, K. Wang, J. Wang, X. Zhang, M. Han, Y. He, Q. Chen and G. Hu, <span class="italic">Arch. Virol.</span>, 2018, <span class="bold">163</span>, 2459–2463 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00705-018-3859-x" title="DOI Link to resource 10.1007/s00705-018-3859-x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpsFyiu7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29767300%5Buid%5D" title="PubMed Link to resource 29767300">PubMed</a>.</span></li>
      <li><span id="cit156">Y. Qi, Q. Yin, Y. Shao, M. Cao, S. Li, H. Chen, W. Shen, J. Rao, J. Li, X. Li, Y. Sun, Y. Lin, Y. Deng, W. Zeng, S. Zheng, S. Liu and Y. Li, <span class="italic">Int. J. Infect. Dis.</span>, 2018, <span class="bold">70</span>, 42–50 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ijid.2018.03.003" title="DOI Link to resource 10.1016/j.ijid.2018.03.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXntlentrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29548879%5Buid%5D" title="PubMed Link to resource 29548879">PubMed</a>.</span></li>
      <li><span id="cit157">Y. Qi, Q. Yin, Y. Shao, S. Li, H. Chen, W. Shen, J. Rao, J. Li, X. Li, Y. Sun, Y. Lin, Y. Deng, W. Zeng, S. Zheng, S. Liu and Y. Li, <span class="italic">BioMed Res. Int.</span>, 2018, <span class="bold">2018</span>, 1–10 <a target="_blank" class="DOILink" href="https://doi.org/10.1155/2018/6417354" title="DOI Link to resource 10.1155/2018/6417354">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29850545%5Buid%5D" title="PubMed Link to resource 29850545">PubMed</a>.</span></li>
      <li><span id="cit158">R. A. Ruehrwein and D. W. Ward, <span class="italic">Soil Sci.</span>, 1952, <span class="bold">73</span>, 485–492 <a target="_blank" class="DOILink" href="https://doi.org/10.1097/00010694-195206000-00007" title="DOI Link to resource 10.1097/00010694-195206000-00007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaG3sXksFOmtw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit159">V. K. La Mer, <span class="italic">Discuss. Faraday Soc.</span>, 1966, <span class="bold">42</span>, 248–254 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=DF9664200248&amp;newsite=1" title="Link to RSC resource DOI:10.1039/DF9664200248">RSC</a>.</span></li>
      <li><span id="cit160">T. W. Healy and V. K. La Mer, <span class="italic">J. Colloid Sci.</span>, 1964, <span class="bold">19</span>, 323–332 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0095-8522(64)90034-0" title="DOI Link to resource 10.1016/0095-8522(64)90034-0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaF2cXktFGjtr4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit161">R. H. Smellie and V. K. La Mer, <span class="italic">J. Colloid Sci.</span>, 1958, <span class="bold">13</span>, 589–599 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0095-8522(58)90071-0" title="DOI Link to resource 10.1016/0095-8522(58)90071-0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaG1MXhvFehtQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit162">E. J. Wee, H. Y. Lau, J. R. Botella and M. Trau, <span class="italic">Chem. Commun.</span>, 2015, <span class="bold">51</span>, 5828–5831 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4CC10068A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4CC10068A">RSC</a>.</span></li>
      <li><span id="cit163">E. J. Wee, T. Ha Ngo and M. Trau, <span class="italic">Sci. Rep.</span>, 2015, <span class="bold">5</span>, 15028 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep15028" title="DOI Link to resource 10.1038/srep15028">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhs1KksrrL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26458746%5Buid%5D" title="PubMed Link to resource 26458746">PubMed</a>.</span></li>
      <li><span id="cit164">K. M. Koo, E. J. Wee, P. N. Mainwaring and M. Trau, <span class="italic">Sci. Rep.</span>, 2016, <span class="bold">6</span>, 30722 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep30722" title="DOI Link to resource 10.1038/srep30722">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1ygs73J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27470540%5Buid%5D" title="PubMed Link to resource 27470540">PubMed</a>.</span></li>
      <li><span id="cit165">E. J. H. Wee, T. H. Ngo and M. Trau, <span class="italic">Clin. Epigenet.</span>, 2015, <span class="bold">7</span>, 1–9 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13148-014-0036-2" title="DOI Link to resource 10.1186/s13148-014-0036-2">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25628764%5Buid%5D" title="PubMed Link to resource 25628764">PubMed</a>.</span></li>
      <li><span id="cit166">J. S. Del Rio, N. Yehia Adly, J. L. Acero-Sanchez, O. Y. Henry and C. K. O'Sullivan, <span class="italic">Biosens. Bioelectron.</span>, 2014, <span class="bold">54</span>, 674–678 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2013.11.035" title="DOI Link to resource 10.1016/j.bios.2013.11.035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOltLg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24334283%5Buid%5D" title="PubMed Link to resource 24334283">PubMed</a>.</span></li>
      <li><span id="cit167">J. S. Del Rio, M. Svobodova, P. Bustos, P. Conejeros and C. K. O'Sullivan, <span class="italic">Anal. Bioanal. Chem.</span>, 2016, <span class="bold">408</span>, 8611–8620 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-016-9639-0" title="DOI Link to resource 10.1007/s00216-016-9639-0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xos1Gmu7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27220649%5Buid%5D" title="PubMed Link to resource 27220649">PubMed</a>.</span></li>
      <li><span id="cit168">M. N. Tsaloglou, A. Nemiroski, G. Camci-Unal, D. C. Christodouleas, L. P. Murray, J. T. Connelly and G. M. Whitesides, <span class="italic">Anal. Biochem.</span>, 2017, <span class="bold">543</span>, 116–121 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2017.11.025" title="DOI Link to resource 10.1016/j.ab.2017.11.025">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29224732%5Buid%5D" title="PubMed Link to resource 29224732">PubMed</a>.</span></li>
      <li><span id="cit169">
          R. E. Farrell, in <span class="italic">RNA Methodologies</span>, ed. R. E. Farrell, Academic Press, San Diego, 4th edn,  2010, ch. 14, pp. 301–320 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=RNA%20Methodologies%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202010%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit170">A. Kunze, M. Dilcher, A. Abd El Wahed, F. Hufert, R. Niessner and M. Seidel, <span class="italic">Anal. Chem.</span>, 2016, <span class="bold">88</span>, 898–905 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.5b03540" title="DOI Link to resource 10.1021/acs.analchem.5b03540">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWrtrbM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26624222%5Buid%5D" title="PubMed Link to resource 26624222">PubMed</a>.</span></li>
      <li><span id="cit171">C. Kober, R. Niessner and M. Seidel, <span class="italic">Biosens. Bioelectron.</span>, 2018, <span class="bold">100</span>, 49–55 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2017.08.053" title="DOI Link to resource 10.1016/j.bios.2017.08.053">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVSlt7nK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28863324%5Buid%5D" title="PubMed Link to resource 28863324">PubMed</a>.</span></li>
      <li><span id="cit172">A. Heim, C. Ebnet, G. Harste and P. Pring-Akerblom, <span class="italic">J. Med. Virol.</span>, 2003, <span class="bold">70</span>, 228–239 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jmv.10382" title="DOI Link to resource 10.1002/jmv.10382">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXktF2kurg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12696109%5Buid%5D" title="PubMed Link to resource 12696109">PubMed</a>.</span></li>
      <li><span id="cit173">W. Bogaerts, P. De Heyn, T. Van Vaerenbergh, K. De Vos, S. K. Selvaraja, T. Claes, P. Dumon, P. Bienstman, D. Van Thourhout and R. Baets, <span class="italic">Laser Photonics Rev.</span>, 2012, <span class="bold">6</span>, 47–73 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/lpor.201100017" title="DOI Link to resource 10.1002/lpor.201100017">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhs1GlsrfP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit174">B. Koo, C. E. Jin, S. Y. Park, T. Y. Lee, J. Nam, Y. R. Jang, S. M. Kim, J. Y. Kim, S. H. Kim and Y. Shin, <span class="italic">J. Biophotonics</span>, 2017, <span class="bold">11</span>, e201700167 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jbio.201700167" title="DOI Link to resource 10.1002/jbio.201700167">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29024445%5Buid%5D" title="PubMed Link to resource 29024445">PubMed</a>.</span></li>
      <li><span id="cit175">Y. Shin, A. P. Perera, K. W. Kim and M. K. Park, <span class="italic">Lab Chip</span>, 2013, <span class="bold">13</span>, 2106–2114 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3LC50129A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3LC50129A">RSC</a>.</span></li>
      <li><span id="cit176">J. Sabaté del Río, T. Steylaerts, O. Y. Henry, P. Bienstman, T. Stakenborg, W. Van Roy and C. K. O'Sullivan, <span class="italic">Biosens. Bioelectron.</span>, 2015, <span class="bold">73</span>, 130–137 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2015.05.063" title="DOI Link to resource 10.1016/j.bios.2015.05.063">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26056956%5Buid%5D" title="PubMed Link to resource 26056956">PubMed</a>.</span></li>
      <li><span id="cit177">T. N. T. Dao, E. Y. Lee, B. Koo, C. E. Jin, T. Y. Lee and Y. Shin, <span class="italic">Anal. Biochem.</span>, 2017, <span class="bold">544</span>, 87–92 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2017.12.030" title="DOI Link to resource 10.1016/j.ab.2017.12.030">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29289485%5Buid%5D" title="PubMed Link to resource 29289485">PubMed</a>.</span></li>
      <li><span id="cit178">S. Schlucker, <span class="italic">Angew. Chem., Int. Ed.</span>, 2014, <span class="bold">53</span>, 4756–4795 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201205748" title="DOI Link to resource 10.1002/anie.201205748">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24711218%5Buid%5D" title="PubMed Link to resource 24711218">PubMed</a>.</span></li>
      <li><span id="cit179">J. Wang, K. M. Koo, E. J. Wee, Y. Wang and M. Trau, <span class="italic">Nanoscale</span>, 2017, <span class="bold">9</span>, 3496–3503 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6NR09928A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6NR09928A">RSC</a>.</span></li>
      <li><span id="cit180">F. Shen, E. K. Davydova, W. Du, J. E. Kreutz, O. Piepenburg and R. F. Ismagilov, <span class="italic">Anal. Chem.</span>, 2011, <span class="bold">83</span>, 3533–3540 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac200247e" title="DOI Link to resource 10.1021/ac200247e">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXksFansLo%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21476587%5Buid%5D" title="PubMed Link to resource 21476587">PubMed</a>.</span></li>
      <li><span id="cit181">M. N. Tsaloglou, R. J. Watson, C. M. Rushworth, Y. Zhao, X. Niu, J. M. Sutton and H. Morgan, <span class="italic">Analyst</span>, 2015, <span class="bold">140</span>, 258–264 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4AN01683A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4AN01683A">RSC</a>.</span></li>
      <li><span id="cit182">
          E.-C. Yeh and L. P. Lee, Presented in part at the 17th International Conference on Miniaturised Systems for Chemistry and Life Sciences, Freiburg, Germany, 27–31 October, 2013.</span></li>
      <li><span id="cit183">E.-C. Yeh, C.-C. Fu, L. Hu, R. Thakur, J. Feng and L. P. Lee, <span class="italic">Sci. Adv.</span>, 2017, <span class="bold">3</span>, 1–11 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Sci.%20Adv.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit184">Z. Li, Y. Liu, Q. Wei, Y. Liu, W. Liu, X. Zhang and Y. Yu, <span class="italic">PLoS One</span>, 2016, <span class="bold">11</span>, e0153359 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0153359" title="DOI Link to resource 10.1371/journal.pone.0153359">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27074005%5Buid%5D" title="PubMed Link to resource 27074005">PubMed</a>.</span></li>
      <li><span id="cit185">F. Schuler, F. Schwemmer, M. Trotter, S. Wadle, R. Zengerle, F. von Stetten and N. Paust, <span class="italic">Lab Chip</span>, 2015, <span class="bold">15</span>, 2759–2766 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5LC00291E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5LC00291E">RSC</a>.</span></li>
      <li><span id="cit186">C. J. DeJournette, J. Kim, H. Medlen, X. Li, L. J. Vincent and C. J. Easley, <span class="italic">Anal. Chem.</span>, 2013, <span class="bold">85</span>, 10556–10564 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac4026048" title="DOI Link to resource 10.1021/ac4026048">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhsFWks7jI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24070333%5Buid%5D" title="PubMed Link to resource 24070333">PubMed</a>.</span></li>
      <li><span id="cit187">J. Wang, L. Liu, J. Wang, X. Pang and W. Yuan, <span class="italic">Anal. Biochem.</span>, 2017, <span class="bold">543</span>, 122–127 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2017.12.012" title="DOI Link to resource 10.1016/j.ab.2017.12.012">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29246749%5Buid%5D" title="PubMed Link to resource 29246749">PubMed</a>.</span></li>
      <li><span id="cit188">S. Santiago-Felipe, L. A. Tortajada-Genaro, S. Morais, R. Puchades and A. Maquieira, <span class="italic">Food Chem.</span>, 2015, <span class="bold">174</span>, 509–515 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.foodchem.2014.11.080" title="DOI Link to resource 10.1016/j.foodchem.2014.11.080">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOit7rP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25529713%5Buid%5D" title="PubMed Link to resource 25529713">PubMed</a>.</span></li>
      <li><span id="cit189">H. B. Liu, X. J. Du, Y. X. Zang, P. Li and S. Wang, <span class="italic">J. Agric. Food Chem.</span>, 2017, <span class="bold">65</span>, 10290–10299 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jafc.7b03957" title="DOI Link to resource 10.1021/acs.jafc.7b03957">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhslGisr%252FI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29095602%5Buid%5D" title="PubMed Link to resource 29095602">PubMed</a>.</span></li>
      <li><span id="cit190">J. Song, C. Liu, M. G. Mauk, S. C. Rankin, J. B. Lok, R. M. Greenberg and H. H. Bau, <span class="italic">Clin. Chem.</span>, 2017, <span class="bold">63</span>, 714–722 <a target="_blank" class="DOILink" href="https://doi.org/10.1373/clinchem.2016.263665" title="DOI Link to resource 10.1373/clinchem.2016.263665">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtVSltLzM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28073898%5Buid%5D" title="PubMed Link to resource 28073898">PubMed</a>.</span></li>
      <li><span id="cit191">S. Santiago-Felipe, L. A. Tortajada-Genaro, S. Morais, R. Puchades and Á. Maquieira, <span class="italic">Sens. Actuators, B</span>, 2014, <span class="bold">204</span>, 273–281 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2014.07.073" title="DOI Link to resource 10.1016/j.snb.2014.07.073">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXht1KlurzM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit192">S. Kalsi, S. L. Sellars, C. Turner, J. M. Sutton and H. Morgan, <span class="italic">Micromachines</span>, 2017, <span class="bold">8</span>, 1–12 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/mi8040111" title="DOI Link to resource 10.3390/mi8040111">CrossRef</a>.</span></li>
      <li><span id="cit193">L. Magro, B. Jacquelin, C. Escadafal, P. Garneret, A. Kwasiborski, J. C. Manuguerra, F. Monti, A. Sakuntabhai, J. Vanhomwegen, P. Lafaye and P. Tabeling, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 1347 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-017-00758-9" title="DOI Link to resource 10.1038/s41598-017-00758-9">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28465576%5Buid%5D" title="PubMed Link to resource 28465576">PubMed</a>.</span></li>
      <li><span id="cit194">K. Ming, J. Kim, M. J. Biondi, A. Syed, K. Chen, A. Lam, M. Ostrowski, A. Rebbapragada, J. J. Feld and W. C. Chan, <span class="italic">ACS Nano</span>, 2015, <span class="bold">9</span>, 3060–3074 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/nn5072792" title="DOI Link to resource 10.1021/nn5072792">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXit12lt74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25661584%5Buid%5D" title="PubMed Link to resource 25661584">PubMed</a>.</span></li>
      <li><span id="cit195">J. Kim, M. J. Biondi, J. J. Feld and W. C. W. Chan, <span class="italic">ACS Nano</span>, 2016, <span class="bold">10</span>, 4742–4753 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.6b01254" title="DOI Link to resource 10.1021/acsnano.6b01254">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XltlOnu7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27035744%5Buid%5D" title="PubMed Link to resource 27035744">PubMed</a>.</span></li>
      <li><span id="cit196">S. Kersting, V. Rausch, F. F. Bier and M. von Nickisch-Rosenegk, <span class="italic">Mikrochim. Acta</span>, 2014, <span class="bold">181</span>, 1715–1723 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-014-1198-5" title="DOI Link to resource 10.1007/s00604-014-1198-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXislKntLs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25253912%5Buid%5D" title="PubMed Link to resource 25253912">PubMed</a>.</span></li>
      <li><span id="cit197">A. Abd El Wahed, M. Weidmann and F. T. Hufert, <span class="italic">J. Clin. Virol.</span>, 2015, <span class="bold">69</span>, 16–21 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jcv.2015.05.004" title="DOI Link to resource 10.1016/j.jcv.2015.05.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXpt1Ojsr0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26209370%5Buid%5D" title="PubMed Link to resource 26209370">PubMed</a>.</span></li>
      <li><span id="cit198">M. Branavan, R. E. Mackay, P. Craw, A. Naveenathayalan, J. C. Ahern, T. Sivanesan, C. Hudson, T. Stead, J. Kremer, N. Garg, M. Baker, S. T. Sadiq and W. Balachandran, <span class="italic">Med. Eng. Phys.</span>, 2016, <span class="bold">38</span>, 741–748 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.medengphy.2016.04.022" title="DOI Link to resource 10.1016/j.medengphy.2016.04.022">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27238759%5Buid%5D" title="PubMed Link to resource 27238759">PubMed</a>.</span></li>
      <li><span id="cit199">C. Hu, S. Kalsi, I. Zeimpekis, K. Sun, P. Ashburn, C. Turner, J. M. Sutton and H. Morgan, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">96</span>, 281–287 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2017.05.016" title="DOI Link to resource 10.1016/j.bios.2017.05.016">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXotF2ltrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28505562%5Buid%5D" title="PubMed Link to resource 28505562">PubMed</a>.</span></li>
      <li><span id="cit200">S. Lutz, P. Weber, M. Focke, B. Faltin, J. Hoffmann, C. Muller, D. Mark, G. Roth, P. Munday, N. Armes, O. Piepenburg, R. Zengerle and F. von Stetten, <span class="italic">Lab Chip</span>, 2010, <span class="bold">10</span>, 887–893 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B921140C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B921140C">RSC</a>.</span></li>
      <li><span id="cit201">
          O. Strohmeier, B. Kanat, D. Bär, P. Patel, J. Drexler, M. Weidmann, T. v. Oordt, G. Roth, D. Mark, R. Zengerle and F. v. Stetten, Presented in part at the MicroTAS 2012, Okinawa, Japan, Oct. 28th to Nov. 1st 2012, 2012.</span></li>
      <li><span id="cit202">T. H. Kim, J. Park, C. J. Kim and Y. K. Cho, <span class="italic">Anal. Chem.</span>, 2014, <span class="bold">86</span>, 3841–3848 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac403971h" title="DOI Link to resource 10.1021/ac403971h">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXksVWjt78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24635032%5Buid%5D" title="PubMed Link to resource 24635032">PubMed</a>.</span></li>
      <li><span id="cit203">G. Silva, J. Oyekanmi, C. K. Nkere, M. Bomer, P. L. Kumar and S. E. Seal, <span class="italic">Anal. Biochem.</span>, 2018, <span class="bold">546</span>, 17–22 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2018.01.019" title="DOI Link to resource 10.1016/j.ab.2018.01.019">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitV2ks7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29378167%5Buid%5D" title="PubMed Link to resource 29378167">PubMed</a>.</span></li>
      <li><span id="cit204">G. Choi, J. H. Jung, B. H. Park, S. J. Oh, J. H. Seo, J. S. Choi, H. Kim do and T. S. Seo, <span class="italic">Lab Chip</span>, 2016, <span class="bold">16</span>, 2309–2316 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6LC00329J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6LC00329J">RSC</a>.</span></li>
      <li><span id="cit205">
          G. Choi, J. H. Jung, B. H. Park, S. J. Oh and T. S. Seo, Presented in part at the 19th International Conference on Miniaturized Systems for Chemistry and Life Sciences, Gyeongju, Korea, October 25–29, 2015.</span></li>
    </ol>
  <table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2019</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>