<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Machine learning for renewable energy materials  - Journal of Materials Chemistry A (RSC Publishing) DOI:10.1039/C9TA02356A</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2019/ta/c9ta02356a"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":56,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Geun Ho Gu"/><meta name="DC.Creator" content="Juhwan Noh"/><meta name="DC.Creator" content="Inkyung Kim"/><meta name="DC.Creator" content="Yousung Jung"/><meta name="DC.title" content="Machine learning for renewable energy materials "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2019/07/23"/><meta name="DC.Identifier" scheme="doi" content="10.1039/C9TA02356A"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Machine learning for renewable energy materials "/><meta name="citation_author" content="Geun Ho Gu"/><meta name="citation_author" content="Juhwan Noh"/><meta name="citation_author" content="Inkyung Kim"/><meta name="citation_author" content="Yousung Jung"/><meta name="citation_online_date" content="2019/04/30"/><meta name="citation_date" content="2019"/><meta name="citation_journal_title" content="Journal of Materials Chemistry A"/><meta name="citation_volume" content="7"/><meta name="citation_issue" content="29"/><meta name="citation_firstpage" content="17096"/><meta name="citation_lastpage" content="17117"/><meta name="citation_doi" content="10.1039/C9TA02356A"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2019/ta/c9ta02356a"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2019/ta/c9ta02356a"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2019/ta/c9ta02356a"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2019/ta/c9ta02356a">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:c9ta03595h)" href="/en/content/articlehtml/2019/ta/c9ta03595h" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:c9ta00867e)" href="/en/content/articlehtml/2019/ta/c9ta00867e" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div> </div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/C9TA02356A">10.1039/C9TA02356A</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2050-7496/2013">J. Mater. Chem. A</a></span>, 2019, <strong>7</strong>, 17096-17117</div><h1 id="sect214"><span class="title_heading">Machine learning for renewable energy materials</span></h1><p class="header_text">
      <span class="bold">
        
          
            Geun Ho 
            Gu
          
        
      </span>, 
      <span class="bold">
        
          
            Juhwan 
            Noh
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Juhwan Noh (orcid.org/0000-0003-1183-9955) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-1183-9955"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>, 
      <span class="bold">
        
          
            Inkyung 
            Kim
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Inkyung Kim (orcid.org/0000-0001-5715-8253) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-5715-8253"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><span class="bold"> and </span>
      <span class="bold">
        
          
            Yousung 
            Jung
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Yousung Jung (orcid.org/0000-0003-2615-8394) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-2615-8394"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*
      <br/><span class="italic">Graduate School of EEWS, Korea Advanced Institute of Science and Technology (KAIST), 291 Daehak-ro 34141, Daejeon, 305-335, South Korea. E-mail: <a href="mailto:ysjn@kaist.ac.kr">ysjn@kaist.ac.kr</a></span>
    </p><div id="art-admin"><span class="italic bold">Received 
      4th March 2019
    </span><span class="bold italic">, Accepted 29th April 2019</span></div><p class="bold italic">First published on 30th April 2019</p><hr/><div class="abstract"><h2>Abstract</h2><p>Achieving the 2016 Paris agreement goal of limiting global warming below 2 °C and securing a sustainable energy future require materials innovations in renewable energy technologies. While the window of opportunity is closing, meeting these goals necessitates deploying new research concepts and strategies to accelerate materials discovery by an order of magnitude. Recent advancements in machine learning have provided the science and engineering community with a flexible and rapid prediction framework, showing a tremendous potential impact. Here we summarize the recent progress in machine learning approaches for developing renewable energy materials. We demonstrate applications of machine learning methods for theoretical approaches in key renewable energy technologies including catalysis, batteries, solar cells, and crystal discovery. We also analyze notable applications resulting in significant real discoveries and discuss critical gaps to further accelerate materials discovery.</p></div><hr/>
    
      
      <h2 id="sect253"><span class="a_heading">1. Introduction</span></h2>
      <span>Despite the international effort emerging from the 2016 Paris agreement,<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> the international commitment aiming to hold global warming below 2 °C above the pre-industrial level by 2100 is projected to be inadequate to meet the goal<a title="Select to navigate to references" href="#cit2"><sup><span class="sup_ref">2</span></sup></a> (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>). Beyond enforcing policies and supporting non-state actions, the research community can help meet this goal by innovating and deploying new economic materials that produce fuels, chemicals, and energy from renewable resources.<a title="Select to navigate to reference" href="#cit2"><sup><span class="sup_ref">2,3</span></sup></a> While the carbon budget – the upper limit of CO<small><sub>2</sub></small> emission to remain under the target temperature – is expected to run out by 2030 with 66 percent probability,<a title="Select to navigate to references" href="#cit2"><sup><span class="sup_ref">2</span></sup></a> discovering new materials and their deployment in the market takes on average 10–20 years with a large investment.<a title="Select to navigate to reference" href="#cit4"><sup><span class="sup_ref">4,5</span></sup></a> Thus, the research community will have to develop new concepts and research approaches that accelerate materials discovery. In this regard, Mission Innovation, a global initiative led by 24 countries to accelerate global clean energy innovation, has advocated combining theoretical physical chemistry and materials science with high-throughput computational data generation, machine learning (ML), and robotics.<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> This data-driven approach has the potential to reduce the cost and computation–experiment cycle time in the conventional trial-and-error approach, and is expected to accelerate materials discovery by a factor of ten.<a title="Select to navigate to reference" href="#cit5"><sup><span class="sup_ref">5,6</span></sup></a></span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f1.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Global greenhouse gas (GHG) emissions as implied by the Intended Nationally Determined Contributions (INDCs) compared to the no-policy baseline, current-policy and 2 °C scenarios reproduced with permission from <a title="Select to navigate to references" href="#cit2">ref. 2</a> Copyright 2016 Springer, <span class="italic">Nature</span>. White lines show the median, and 20th–80th-percentile ranges are shown for no-policy and least-energy-economy-cost 2 °C scenarios, and the 10th–90th percentile across all studies is shown.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Machine learning-based data-driven approaches have demonstrated tremendous impact in a number of aspects. The advancement in computational power and the emergence of big data have led to the success of machine learning methods in retail,<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a> medical diagnoses,<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> image recognition,<a title="Select to navigate to references" href="#cit9"><sup><span class="sup_ref">9</span></sup></a> and speech understanding.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> These successes accompanying the advancement in machine learning have attracted interest in its application to science and engineering led initially by the U.S. Materials Genome Initiative.<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a></p>
      <p class="otherpara">Conventional trial-and-error theoretical and experimental research studies involve an in-depth understanding of interesting, useful phenomena followed by the exploitation of new knowledge to test better materials. Such intuition-based approaches provide insightful knowledge but are not the most efficient approach for discovering new materials. Experimental approaches involve synthesis, characterization and property analysis by manual labor, resulting in a slow turnover. On the other hand, computational simulations can allow easier integration of high-throughput screening through automation. However, finding suitable descriptors or a model for the material's activity of interest is difficult and the ability to bridge the time-and-length scale to obtain macroscale properties is limited due to the large computational cost. In addition, computational simulations alone are not enough to discover materials as the simulations may not be realistic. For this, machine learning and robotics offer a systematic solution to speed up discovery.</p>
      <p class="otherpara">Machine learning offers a flexible and accessible framework that correlates the materials' <span class="italic">a priori</span> knowledge with the properties of interest with respectable accuracy and speed given a large data set. For the experimental approach, interpreting and designing of the experiment can be delegated to the machine learning algorithms while robotics is used to automate the aforementioned manual labor.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> Notably, the Autonomous Research System, ARES, has used this approach successfully to optimize carbon nanotube synthesis.<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> For the computational approach, machine-learning models offer a practical solution for high-throughput screening as theoretically developing a highly flexible model that correlates <span class="italic">a priori</span> knowledge with the properties of interest is often difficult. We do note that developing theoretical models and understanding is important for developing machine learning models as it provides insights into effective descriptors and designs.</p>
      <p class="otherpara">Machine learning is also useful in coupling the small and large time-and-length scales. For example, molecular dynamics simulations are often performed with a surrogate model such as a force-field, but the accuracy of the energy and force is typically lower than that of the <span class="italic">ab initio</span> methods. Machine learning models can provide high accuracy while maintaining the large speed up from <span class="italic">ab initio</span> calculations.<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> Another exciting and promising area of machine learning is inverse design where the desired properties of the material are given to the model, and materials with those properties are outputted.<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a> Its application to functional materials discovery has enormous potential in academia and industry, but the inverse design model development for materials is still in its infancy.</p>
      <p class="otherpara">The U.S. Department of Energy has identified several major technical advancement goals for realizing a secure and sustainable energy future: (1) production of fuels from sunlight, (2) carbon-neutral electricity generation, and (3) revolutionizing energy efficiency and use.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> Among renewable sources of energy including wind, geothermal heat, and hydropower, solar energy is the most available and practical resource to harness.<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> However, with current solar cell technology and its growth rate, solar cells are predicted to supply only 10 percent of the carbon-free energy demand in 2050.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> While the solar cell has grown to be the least expensive technology to produce energy in many countries,<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> continued improvements in the cost, solar conversion efficiency, and energy storage are needed to meet the global energy demand sustainably.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a></p>
      <p class="otherpara">Currently, the transportation sector is almost completely dependent on fossil fuels. Replacing them with renewable and clean-burning fuel is thus essential, and discovering catalysts that convert water, biomass, and CO<small><sub>2</sub></small> to hydrogen fuels and small hydrocarbons such as methanol and ethanol is critical.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> A large emphasis has been placed on discovering water-splitting catalysts as hydrogen is the cleanest fuel.<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> Hydrogen fuel production, delivery and dispensing have become nearly cost-competitive with gasoline, and continued innovation and deployment is key.<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a> In addition to fuels, developing catalysts for renewable production of chemicals and plastics from biomass and CO<small><sub>2</sub></small> is widely investigated as well. While renewable fuels can power transportation either by direct burning or generating electricity through fuel-cells, battery-operated cars are another solution to renewable transportation. In this regard, developing better batteries is necessary to increase range, reduce weight and improve efficiency.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> Also, renewable energy sources such as solar and wind energy suffer from inconsistent power output due to the day and night cycles and weather. Thus, developing battery technologies that provide large amounts of electricity over a long period of time is important.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> To address this, improvement of redox flow fuel-cells, which can be thought of as a hybrid of a battery and a fuel cell, to exhibit the potential to store electricity in large quantities, was carried out. In addition, improving electric grid reliability can reduce economic overhead expenses. The Lawrence Berkeley National Laboratory estimated that power outages cost the US about $80 billion annually.<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a> The materials discovery of high power energy storages to address micro-outages is a critical issue to improve electric grid reliance.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> As briefly discussed above, the trend of replacing the energy supply infrastructure with renewable energy sources is currently not encouraging, and CO<small><sub>2</sub></small> emission from combustion is likely to continue.<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> In this regard, developing CO<small><sub>2</sub></small> capture materials can negate the CO<small><sub>2</sub></small> emissions from using fossil fuels.<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a><a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a> summarizes and categorizes these key renewable energy technologies.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f2.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Key renewable technologies addressing the energy needs of society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">These challenges in realizing a renewable society can be addressed to some extent at the manufacturing level and by (non-)state actions, but, in essence, discovering game-changing materials with superior efficiency and low cost offers a fundamental solution. In this review, we aim to provide a broad overview of machine learning research in innovating renewable energy technology with a focus on the computational approaches. We refer to other in-depth reviews of machine learning methods for the physical system,<a title="Select to navigate to reference" href="#cit6"><sup><span class="sup_ref">6,21–26</span></sup></a> and here we focus on applications of various machine learning techniques towards renewable energy materials development. Section 2 introduces key studies on critical renewable energy technologies discussed above such as solar cells, catalysts, CO<small><sub>2</sub></small> capture, and batteries. We also summarize the emerging inverse design machine learning and others with large implications in accelerating materials design.<a title="Select to navigate to references" href="#cit27"><sup><span class="sup_ref">27</span></sup></a> To move forward, we report the success stories that have led to the real discovery of practical materials in Section 3. Finally, we close this review with the remaining gaps and prospects.</p>
    
    
      
      <h2 id="sect315"><span class="a_heading">2. Machine learning in renewable energy materials</span></h2>
      <span>We begin this section by briefly summarizing the workflow of implementing machine learning. The subsequent sections discuss the literature overview of the applications and are organized by the characteristics of the machine learning applications. Section 2.2 summarizes the most widely used form where materials' properties are predicted from descriptors, and Section 2.3 is dedicated to machine learning potential, where energy and forces of atoms are calculated from atomic coordinates. The last section discusses the machine-learning based inverse modeling applied to materials.</span>
      
        
        <h3 id="sect319"><span class="b_heading">2.1. Developing machine learning modelling</span></h3>
        <span>Here, we briefly discuss the development of the machine learning model (see <a title="Select to navigate to references" href="#cit21">ref. 21</a> for in-depth discussion). The premise of the machine learning model is its ability to automatically learn and predict physical properties given a large enough dataset. As such, the critical steps in developing a successful model are (1) gathering a quality data set, and developing (2) a machine-understandable data representation and (3) a model that correlates the representation with the physical properties.<a title="Select to navigate to references" href="#cit25"><sup><span class="sup_ref">25</span></sup></a> The data collection method needs to be designed to minimize noise and to unbiasedly sample the data space, and the dataset needs to be sufficiently large. Data are typically collected experimentally and through <span class="italic">ab initio</span> calculations. However, the large resources needed to collect the data often become the bottleneck for machine learning development. In this regard, a number of databases are mostly freely available for molecules (ZINC,<a title="Select to navigate to references" href="#cit28"><sup><span class="sup_ref">28</span></sup></a> ChEMBL,<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> GDB-13,<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> GDB-17,<a title="Select to navigate to references" href="#cit31"><sup><span class="sup_ref">31</span></sup></a> and others), as well as for inorganic crystals (International Crystal Structure Database (ICSD),<a title="Select to navigate to reference" href="#cit32"><sup><span class="sup_ref">32,33</span></sup></a> Materials Project (MP),<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> Open Quantum Materials Database (OQMD),<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> Atomic-FLOW for materials discovery (AFLOW),<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> NOMAD, Pearson's Crystal Data, and others). After collecting data, the data need to be transformed into a mathematical representation that machines can understand, typically involving multi-dimensional matrices of real numbers. This so-called featurization is critical as the better the representation describing the physical properties of interest, the higher the accuracy of the model. In this regard, the field experts are critical as they can advise on suitable descriptors. Most of the research studies introduced here implement descriptors that have been known for some time from previous theoretical and experimental investigations. Lastly, the choice of model determines the model performance. Methods such as ridge regression,<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> LASSO,<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> elastic net,<a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a> and partial least squares regression<a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a> enhance linear regression by controlling over- and under-fitting and by enabling descriptor selection and so on. Classification methods such as support vector machines,<a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a> decision trees,<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> random forests,<a title="Select to navigate to references" href="#cit43"><sup><span class="sup_ref">43</span></sup></a> and logistic regression<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> categorize data, and researchers have often used them to classify high and low activity of the materials. Kernel ridge regression<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> learns a linear function from the similarity between the data point descriptors. In particular, the generalized theory of Kernel ridge regression called the Gaussian process<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> offers uncertainty quantification, opening the door to the interesting applications discussed below. Recently, neural network modeling is rapidly entering the field.<a title="Select to navigate to reference" href="#cit6"><sup><span class="sup_ref">6,21–26</span></sup></a> The type of neural network varies widely, as the framework enables flexible additions and connections between various GPU-optimized operations. The set of operations and the input and output connections between them are called the neural network architecture and are customized to solve the problem of interest. A neural network with a highly complex architecture is often referred to as deep learning.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> Deep learning has become a popular topic in the application of the physical system. Choosing the right model involves careful examination of the underlying physics between the representation and the properties of interest.</span>
      
      
        
        <h3 id="sect346"><span class="b_heading">2.2. Property predictions</span></h3>
        <span>The most prevalent form of the application of machine learning is property prediction in order to rapidly screen materials. Using density functional theory (DFT) to calculate the desired properties remains a bottleneck for computational research due to its high cost; thus machine learning models are used to make rapid predictions. Machine learning model research typically involves the development of effective representation/descriptors. Effective descriptors ideally directly describe the activtiy and stability of materials with minimal calculations. In such a case, large-scale materials screening can be performed as shown in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f3.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">Widely used workflow for machine learning based materials screening <span class="italic">via</span> property prediction.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <div>
          
          <span id="sect354"/><span class="c_heading_indent">2.2.1 Adsorbate binding energy prediction for catalyst design. </span>
          <span>Binding energy has been the staple descriptor for correlating the activity of a catalyst since the early 20<small><sup>th</sup></small> century, by the empirical Sabatier principle.<a title="Select to navigate to references" href="#cit48"><sup><span class="sup_ref">48</span></sup></a> In general, an optimal adsorbate binding energy exists for catalyst activity as strong binding leads to catalyst poisoning and weak binding fails to activate the adsorbates (see <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>).<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> While it is rather empirical, the simplicity of computationally calculating the binding energy has led the field as the descriptor for catalyst discovery. As discussed above, the CO<small><sub>2</sub></small> reduction reaction is one of the most studied reactions in this field. For this reaction, the (100) terminated surface has been identified as the actives site,<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> and the CO binding energy on it has been shown to describe the electrocatalytic activity well.<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> Based on this, Xin and co-workers pioneered the employment of machine learning to correlate surface properties with the binding energy.<a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a> Here, 13 theoretically determined electronic properties are used together with a neural network to predict CO binding energies on a (100) terminated bi-metallic surface. The model shows a 0.13 eV root mean squared error (RMSE) and was used to suggest various (100) Cu-based bimetallic surfaces that may be active. The sensitivity analysis showed that electronegativity and d-band characteristics are critical to describing the CO binding energy. The model introduced in this study requires DFT calculations of the clean surface, thus using this model requires significant computational overhead. To bypass DFT calculations, Xin and co-workers extended their previous work and introduced local electronegativity and effective coordination numbers which do not require DFT calculations. Using these descriptors with the neural network demonstrated a RMSE of 0.12 eV.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> In the follow up study, the developed descriptors are tested using various regression techniques (linear regression, ridge regression, LASSO, elastic net, kernel ridge regression, support vector machines, Gaussian processes, K-nearest neighbors, and random forests), the best of which showed a RMSE near 0.2 eV for CO binding energy (the higher RMSE is due to the more versatile dataset). CO and OH binding energies are screened over diverse model bimetallic surfaces using the neural network, which is used to compute the theoretical limiting potential of methanol electro-oxidation in the interest of methanol fuel cells. This study has suggested many effective bimetallic materials including known active alloy catalysts.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f4.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">(a) Demonstration of the Sabatier principle for predicting optimal catalyst activity, and (b) workflow chart of catalyst screening <span class="italic">via</span> machine learning.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">While electronegativity and coordination number descriptors have shown good performance, the reactivity difference among alloys is often derived from d-band characteristics.<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> However, the quantifying d-band characteristics for an alloy surface require DFT calculations, and are thus not ideal for high-throughput screening. In order to exploit d-band information without DFT, Noh <span class="italic">et al.</span> have introduced the d-band width of the mean-field muffin-tin orbital theory and used it together with electronegativity to compute the CO binding energy. Using the active-learning algorithm to sample the most informative data, the kernel ridge regression model demonstrated the current state-of-the-art performance of 0.05 eV RMSE for CO binding energy on the (100) surface.<a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> This study also has suggested a number of promising catalyst candidates for the CO<small><sub>2</sub></small> reduction reaction.</p>
          <p class="otherpara">More recently, machine learning is used to guide DFT calculations to find optimal electrocatalysts for CO<small><sub>2</sub></small> reduction and H<small><sub>2</sub></small> evolution through binding energy prediction. Here, the TPOT machine learning package is employed to automatically select and run DFT calculations where 80% and 20% of the DFT calculations were dedicated to optimizing the model and finding optimal materials, respectively.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> The automated calculation framework performed for little over a year and resulted in the identification of 131 promising candidate surfaces from various alloys made from a pool of 31 elements. As opposed to the studies above, the binding energy is computed with DFT accuracy as opposed to ML accuracy, thus adding credibility to the suggested candidates.</p>
          <p class="otherpara">In a similar vein, Takigawa and co-workers have investigated models predicting the d-band center.<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> Ordinary least squares regression, partial least squares regression, Gaussian process regression, and gradient boosting regression are used with nine readily accessible physical descriptors of metal atoms that do not require <span class="italic">ab initio</span> calculations. Bimetallic alloys with impure and overlay surfaces are considered. Gradient boosting regression showed the highest accuracy at 0.17 eV and 0.19 eV RMSE for impurity and overlay surfaces, respectively. In the interest of designing catalysts for hydrocarbon selectivity from the conversion of renewably obtained methane, Takigawa and co-workers have employed accessible descriptors to predict the binding energy of CH<small><sub>3</sub></small>, CH<small><sub>2</sub></small>, CH, C and H on Cu-based alloys.<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> Out of nine regression methods, extra tree regression demonstrated a RMSE near 0.3 eV. Besides predicting the active components, controlling the structure of the catalyst can improve activity and binding energy, differing based on the facets of the surface. In this regard, Yıldırım and co-workers implemented a neural network to predict the CO binding energy on Au clusters for CO oxidation.<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> CO oxidation is an important process for reducing CO emission from transportation and industries. Au catalysts have demonstrated good activity for CO oxidation, but the active site is not well known. Yildirim and co-workers have used the cluster size, overall charge, unpaired-electrons and coordination number to predict binding energies <span class="italic">via</span> a neural network. Asahi and co-workers have predicted N, NO and O binding energies and the formation energy of RhAu octahedral nanoparticles with various surface compositions using SOAP kernel regression.<a title="Select to navigate to reference" href="#cit60"><sup><span class="sup_ref">60,61</span></sup></a> The binding energies and formation energies are predicted at ∼0.1 and 0.02 eV RMSE, respectively. Then, the binding energy and formation energy are used to estimate catalyst activity and predict nanoparticle stability, respectively. This work demonstrates the one-shot approach to directly predict catalyst activity given the composition of the nanoparticle.</p>
        </div>
        <div>
          
          <span id="sect388"/><span class="c_heading_indent">2.2.2. Materials design for CO<small><sub>2</sub></small> capture. </span>
          <span>CO<small><sub>2</sub></small> capture technologies are most widely implemented in industry using amine sorbents <span class="italic">via</span> absorption.<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> However, the process requires a large number of heats for the regeneration step, and amine sorbents are corrosive and unstable.<a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> Due to this, metal–organic frameworks (MOFs) are considered promising due to their versatility, high porous volume and the ease of active site design. However MOFs suffer from relatively low chemical and thermal stability which complicates the temperature or pressure swing recovery of CO<small><sub>2</sub></small>; thus, discovering materials that are stable and have high CO<small><sub>2</sub></small> capacity is key. Woo and co-workers introduced quantitative structure–property relationship (QSPR) models to classify MOFs with high and low CO<small><sub>2</sub></small> uptake capacity.<a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> Here, the atomic property-weighted radial distribution function descriptor developed previously by Woo and co-workers that captures chemical and geometric features<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> is employed with a support vector machine. A database of 324<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>500 MOF structures is considered, where grand canonical Monte Carlo combined with the Lennard-Jones potential and DFT-derived electrostatic potential is used to model electronic interactions. The model was able to capture most of the known active MOFs. Froudakis and co-workers used the presence or absence of substructure patterns in MOFs as descriptors and “Just Add Data”, an automated machine learning analysis tool, to predict the CO<small><sub>2</sub></small> and H<small><sub>2</sub></small> uptake capacity.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> Compared to the earlier study by Woo and co-workers, this model is able to predict the continuous uptake capacity value and is trained using experimental data of 100 MOFs. Similar to Woo and co-workers, Gómez-Gualdrón and co-workers investigated CO<small><sub>2</sub></small> capture prediction of MOFs using DFT, grand canonical Monte Carlo, and machine learning.<a title="Select to navigate to references" href="#cit67"><sup><span class="sup_ref">67</span></sup></a> Here, various storage properties of 400 MOFs are determined which are then used to train six different machine learning models <span class="italic">via</span> 13 different electronic and geometric descriptors. Instead of CO<small><sub>2</sub></small>, Woo and co-workers investigated storage capacity prediction for methane using the geometric features, such as pore size and void fraction, of 130<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 MOFs with multilinear regression, decision trees, and nonlinear support vector machines.<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> Analysis of the model revealed the desired geometric properties of the MOF that could lead to high methane storage.</span>
        </div>
        <div>
          
          <span id="sect410"/><span class="c_heading_indent">2.2.3. Battery electrolytes and electrode discovery. </span>
          <span>Developing new battery technology involves innovations in the electrolyte and electrodes. <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a> shows the battery device and the desired characteristics for the device, electrodes, and electrolytes. The Li-ion-based battery has been in focus as Li is superior to other elements in terms of energy density and power density.<a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a> An electrolyte is a medium in which ions migrate between electrodes (the activity of which is called ionic conductivity) while electron transfer is limited. In order to maximize Li ion conductivity in a liquid electrolyte, the solvent solubility of Li salts and viscosity of Li-ions need to be maximized and minimized, respectively. Moreover, the solvent needs to remain in the liquid phase below zero °C to ensure its operability in cold weather. In this regard, the solvation energy and melting temperature of the solvent for liquid electrolytes have been demonstrated to be more accurately predicted by an exhaustive search with linear regression compared to multiple linear regression and LASSO.<a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> The voltage range in which the electrolyte is electrochemically stable is called the electrochemical window and is critical for electrolyte performance. In this regard, electrolyte additives can aid in widening the voltage range. Additives dissociate at the initial stage of (dis)charge and react in electrolyte to form a stable solid electrolyte interphase layer between the liquid electrolyte and electrodes.<a title="Select to navigate to reference" href="#cit70"><sup><span class="sup_ref">70,71</span></sup></a> Okamoto <span class="italic">et al.</span> predicted the oxidation potential of molecular additives using the regression model.<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a></span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f5.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">(a) Schematic of a Li-ion battery. The direction of electrons and Li-ions is in respect to electron discharge; (b) key properties of batteries for their performance; (c) desired properties of electrodes and electrolytes.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Liquid electrolytes are often based on organic solvents and suffer from flammability. In this regard, solid electrolytes are promising as they are generally less flammable and have a larger electrochemical window than liquid electrolytes. However, solid electrolytes suffer from low Li ion conductivity and thus research on them focuses on finding highly conductive materials. In computational approaches, the Li-ion migration barrier is widely computed as a surrogate measurement of ionic conductivity. Jalem <span class="italic">et al.</span> applied the partial least squares algorithm to predict the migration barrier of olivine-type LiMXO<small><sub>4</sub></small>.<a title="Select to navigate to references" href="#cit73"><sup><span class="sup_ref">73</span></sup></a> Analyzing the model coefficient plot and variable importance in projection plot revealed that descriptors such as the ionic size of M and local lattice distortion are important for migration barriers and new promising olivine-type solid electrolytes were suggested. Furthermore, Jalem <span class="italic">et al.</span> employed the neural network framework for olivine-type LiMXO<small><sub>4</sub></small> as solid electrolyte candidates to predict the migration barrier and cohesive energy.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> The neural network migration barrier prediction model reported by Jalem <span class="italic">et al.</span> was applied to tavorite type LiMTO<small><sub>4</sub></small>F materials as a solid electrolyte.<a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> The Li migration trajectory has also been widely studied to understand the Li migration mechanism and elucidate the rate determining step which can be improved for ionic conductivity. Chen <span class="italic">et al.</span> developed a density-based clustering method to elucidate Li migration trajectories of garnet-type solid electrolytes.<a title="Select to navigate to references" href="#cit76"><sup><span class="sup_ref">76</span></sup></a> In addition, Li ion conductivity was directly evaluated using experimental measurements where the support vector regression algorithm with descriptors of diffusivity at 1600 K, average volume of the disordered structure, ordered/disordered phase transition temperature and temperature at which ionic conductivity was measured, were employed to classify the Li ion conductivity of LiSiCON type solid electrolytes<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a> and garnet-type solid electrolytes.<a title="Select to navigate to references" href="#cit78"><sup><span class="sup_ref">78</span></sup></a> Jalem <span class="italic">et al.</span> applied the Gaussian process with Bayesian sampling to predict the migration barrier of tavorite type solid electrolytes.<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a></p>
          <p class="otherpara">For electrodes, Li ions intercalate and deintercalate into and from the electrodes to store and release energy. Intercalation characteristics of the electrode are critical for energy and power densities. To achieve high energy density, researchers have focused on understanding structure–performance relationships by controlling the elements and structure of electrode materials. In this regard, Shandiz <span class="italic">et al.</span> classified three crystal structures (monoclinic, orthorhombic, and triclinic) from Li–Si–(Mn, Fe, Co)–O compositions using five algorithms.<a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a> Among them, random forests and extremely randomized trees showed the highest accuracy. Wang <span class="italic">et al.</span> used partial least squares regression to distinguish the descriptors which described the volume change during delithiation of cathode candidates well.<a title="Select to navigate to references" href="#cit81"><sup><span class="sup_ref">81</span></sup></a> Eremin <span class="italic">et al.</span> applied ridge regression to predict the energy of LiNiO<small><sub>2</sub></small> and LiNi<small><sub>0.8</sub></small>Co<small><sub>0.15</sub></small>Al<small><sub>0.05</sub></small>O.<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> From the results, common features that (de)stabilize the structure and stabilization effect of doping were discovered. The investigation revealed important descriptors for evaluating the migration barrier of LiMTO<small><sub>4</sub></small>F structures and the model was extended to predict the values of a database containing two structure types, both having the 1D Li path in common. Okamoto employed Kernel ridge regression with Bayesian optimization sampling to evaluate the change in Gibbs free energy before and after Li intercalation into a graphite anode.<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a></p>
          <p class="otherpara">One of the high power energy storage techonology is called, Superconducting Magenetic Energy Storage (SMES). Majority of the SMES cost is in the superconductor materials. Discovering economic superconductors may aid in the SMES distribution. However, machine learning-based research on superconductive materials is preliminary, as the mechanisms behind high-temperature superconductivity are not clear. Stanev <span class="italic">et al.</span> applied the random forest method to classify critical temperatures to identify superconductive materials. 35 promising materials and their common characteristics were determined.<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> For the redox flow battery, Kim <span class="italic">et al.</span> have developed a multiple descriptor multiple kernel method that predicts the solubility of active molecule derivatives of the molecular candidates in an aqueous electrolyte depending on the pH.<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect456"/><span class="c_heading_indent">2.2.4. Materials screening for solar cells. </span>
          <span>For solar cell materials, one of the pivotal parameters is the Shockley–Queisser limit,<a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a> which refers to the maximum efficiency of a single p–n junction solar cell given the bandgap. The efficiency of the realistic device depends on many different properties, but, due to the convenience in DFT calculation, the bandgap is widely used as the screening criterion. In addition, thermodynamic stability of the material is critical as well, and is estimated using the energy of the material. A number of proposed machine learning models have focused on perovskite materials because of the ease of fabrication, high solar absorption, and low non-radiative carrier recombination rates that affect the efficiency.<a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> Allam <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> utilized DFT calculations and the neural network model to analyze the bandgap of elemental halide perovskites and identified the importance of the basic atomic features used to train the machine learning model. A similar approach for double perovskites was proposed by Pilania <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a> by using the kernel ridge regression model in which the author identified which combination of the input features can make an accurate bandgap prediction model. Pilania <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a> proposed the 2-level co-kriging fidelity machine learning model for accurate bandgap prediction of the double perovskite halides where the model predicts the bandgap computed by high-level DFT calculations (specifically, hybrid functionals) by learning from the low-level calculations (<span class="italic">e.g.</span>, generalized gradient approximation functionals) as high-level calculations are expensive. Interestingly, using the low-level DFT bandgap as a descriptor can reduce the uncertainty of the predicted high-level DFT bandgap suggesting that properties calculated using the high-level methods can be predicted using the low-level methods. Also, Takahashi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit91"><sup><span class="sup_ref">91</span></sup></a> used a random forest-based classification machine learning model to classify the bandgap of the perovskite materials in which a total of 18 physical descriptors are used. The model was used to classify whether the given perovskite material is in the ideal bandgap range (1.5–3.0 eV) or not and to screen data of 15<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 perovskites with further analysis of the stability using the DFT calculations. Dey <span class="italic">et al.</span> investigated a machine learning approach for predicting the bandgap of chalcopyrite type materials using OLS, SPLS, elastic net, and LASSO coupled with rough set and principal component analysis methods.<a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a> A total of 15 accessible elemental properties are used as descriptors to predict the bandgap, where 28 data points are used for training. The bandgaps of 227 chalcopyrite materials are predicted. Though this study greatly expanded the knowledge of the solar cell performance of chalcopyrite materials, the lack of training data limited the testing of model accuracy. Beyond specific types of materials, Ward <span class="italic">et al.</span> developed a flexible framework that can be used for crystalline or amorphous materials.<a title="Select to navigate to references" href="#cit93"><sup><span class="sup_ref">93</span></sup></a> Here, the OQMD database is used where properties such as the volume, formation energy, and bandgap were the target properties and 145 attributes were used as descriptors. The algorithms such as partial least squares regression, LASSO, decision tree, kernel ridge regression, Gaussian process regression, and neural networks are used together with an effective data partitioning strategy for high accuracy. On the other hand, the organic solar cell has been gaining popularity due to its accessible electronic property engineering as well as its cheap cost. Aspuru-Guzik and co-workers have used the Harvard Organic Photovoltaic Dataset where molecular fingerprint methods were used together with the Gaussian process to predict properties such as frontier orbital energies, optical gaps, current density, open circuit voltage, and power conversion efficiency. The same strategy is applied to discover non-fullerene acceptors where the molecular fingerprint based Gaussian process is used to leverage the difference between the DFT calculated and experimental frontier molecular orbital energies.<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> Schmidt and co-workers have developed the variational autoencoder method where context-free grammar representations of molecules are implemented to predict the LUMO and optical transition energy. The variational autoencoder enables inverse modeling, which is used to identify a number of molecules with desired properties.</span>
          <p class="otherpara">In addition to the bandgap, the stability of the materials is critical for their longevity. The stability of the materials can be analyzed by calculating the energy above the convex hull (in eV per atom) within the phase diagram constructed from the given elements in a theoretical or experimental manner. We note that the energy above the convex hull is defined as the difference between the enthalpy of the formation of a target material and the most stable enthalpy of formation evaluated from the phase diagram. In this regard, Li <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a> proposed classification and regression models for predicting the energy of the convex hull of a perovskite oxide material using the elemental property data (the best F1 score for classification is 0.881 and the best RMSE for regression is 28.5 meV per atom). Furthermore, the model was used to discover 15 new promising perovskite materials. In addition, Schmidt <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> tested various types of machine learning models including ridge regression, random forests, extremely randomized trees (including adaptive boosting), and neural networks to predict the energy above the hull for 250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 cubic perovskite materials (ABX<small><sub>3</sub></small>), where extremely randomized trees outperformed all the other models with a mean absolute error of 121 meV per atom.</p>
        </div>
        <div>
          
          <span id="sect482"/><span class="c_heading_indent">2.2.5. Other renewable energy topics. </span>
          <span>In the field of catalysis, understanding the surface adsorbate concentrations is difficult in a system involving a large number of adsorbates due to the combinatorial increase in the computational cost. In this regard, Ulissi <span class="italic">et al.</span> have developed Gaussian process-based learning to construct a surface-adsorbate phase diagram.<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> This methodology resulted in the elimination of 80% of the DFT calculations in the case of H and OH adsorption on an IrO<small><sub>2</sub></small>(110) surface. Similarly, investigating the reaction mechanism of large molecules involves DFT calculations for a large number of adsorbates and reactions, often rendering DFT based investigation impossible. In this regard, the Gaussian process is used together with group additivity descriptors to identify the most likely reaction pathways of syngas on Rh(111), and only the most likely reactions are investigated using DFT.<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a></span>
          <p class="otherpara">In order to search for sunlight absorbing molecules, Xin and co-workers tested LASSO, kernel ridge regression, support vector machines, and neural networks to predict energy gaps of porphyrin molecules using molecular fingerprints, Coulomb matrices, chemoinformatics, and electro topological-state indices as descriptors.<a title="Select to navigate to references" href="#cit99"><sup><span class="sup_ref">99</span></sup></a> The model captures the energy gaps within 0.06 eV RMSE, and its analysis suggests structural motifs that influence energy gaps <span class="italic">via</span> sensitivity analysis. Besides utilizing CO<small><sub>2</sub></small> and splitting water, biomass utilization is another attractive pathway to produce chemicals and fuels. Lignocellulosic monomers are already functionalized as opposed to petrochemicals which need functionalization, and aromatic structures, widely synthesized in the petrochemical industry, are already present. However, theoretical studies of biomass conversion are difficult due to their large reaction network.<a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a> In this regard, Vlachos and co-workers have applied LASSO to predict the thermochemistry of biomass monomer adsorbates on metal surfaces using subgraph descriptors.<a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> The model demonstrated a RMSE of 0.09 eV for the heat of formation of lignin monomers on the Pt(111) surface. Such a model reinforces the drawbacks of a popular semi-empirical method for predicting energy called group additivity, where manually identifying graph descriptors is difficult.<a title="Select to navigate to reference" href="#cit102"><sup><span class="sup_ref">102–104</span></sup></a> Metal heterogeneous catalysts are often synthesized by supporting metal particles on oxides, but theoretical metal catalyst studies often use ideal surfaces as a model catalyst. In this regard, Hammer and co-workers introduced a genetic algorithm to globally optimize the metal nanoparticle structure on supports, where the pairing of candidates involved cutting of two candidate particles in half and splicing.<a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a></p>
          <p class="otherpara">While a large number of machine learning studies on catalysts focused on metal alloys, several studies addressed homogeneous catalysts. Rothenberg and co-workers investigated cross-coupling reactions <span class="italic">via</span> a neural network where steric and electronic descriptors of ligands, substrates, catalyst precursors and 412 Heck reactions are correlated with experimentally computed catalyst activity.<a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> Also, Kulik and co-workers have been leading the efforts in developing machine learning frameworks for transition metal complexes, which are used often in homogeneous catalysts.<a title="Select to navigate to references" href="#cit107"><sup><span class="sup_ref">107</span></sup></a> Kulik and co-workers introduced molSimplify, an automated toolkit for screening and discovery of inorganic and intermolecular complexes.<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a> The neural network has been implemented to predict important properties such as energetics, metal–ligand bond lengths, spin-states, and oxidation states and the model accounts for DFT functional sensitivities. An example is the redox potential design of octahedral Fe(<span class="small_caps">II</span>/<span class="small_caps">III</span>) redox couples with nitrogen ligands. Furthermore, they have introduced revised autocorrelation functions that encode atomic properties to molecular graphs demonstrating great accuracy (0.26 eV mean absolute error for atomization energies) for various properties of metal complexes.<a title="Select to navigate to references" href="#cit109"><sup><span class="sup_ref">109</span></sup></a> Finally, the genetic algorithm and neural network are employed to discover spin-crossover complexes, which have applications as spin-based switches.<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a> This work demonstrates the accelerated discovery of machine learning augmented screening.</p>
          <p class="otherpara">Beyond the specific applications discussed above, machine learning methods have been extensively applied for diverse materials applications due to their flexibility.<a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> Faber <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit112"><sup><span class="sup_ref">112</span></sup></a> proposed the formation energy prediction machine learning model for Elpasolite materials by utilizing kernel ridge regression. The developed model was used to screen all possible candidates suggesting that 90 out of 212 new structures are predicted to be on the convex hull. Legrain <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit113"><sup><span class="sup_ref">113</span></sup></a> utilized a random forest machine learning model to predict the force constant of the 121 metastable phase of KZnF<small><sub>3</sub></small> with a mean absolute error of 0.17 eV Å<small><sup>−2</sup></small>, and the predicted force constant was used to estimate phonon spectral features, heat capacities, vibrational entropies, and vibrational free energies, which were in good agreement with the <span class="italic">ab initio</span> calculations. Furthermore, Pilania <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a> proposed a machine learning model to estimate diverse physical properties (formation energy, bandgap, elastic constants and so on) for more than 1200 binary wurtzite superlattices. Dixon and co-workers have developed a linear model predicting thermochemistry using fragments of ZnO nanoparticles, predicting various phase transitions and providing insights on particle growth for ZnO.<a title="Select to navigate to references" href="#cit115"><sup><span class="sup_ref">115</span></sup></a> All of the studies discussed here lead to a concrete conclusion that combining the <span class="italic">ab initio</span> calculations with the novel machine learning model can help to accelerate the understanding of novel materials.</p>
        </div>
      
      
        
        <h3 id="sect520"><span class="b_heading">2.3. Machine learning potentials</span></h3>
        <span>Though DFT has realized simulation-based research, DFT calculation is expensive and, therefore, limited to simulations of small time and length scales. For example, computations of bandgaps and formation energies require a single snapshot calculation of the unit-cell; the macro- and meso/micro-scale simulations such as catalyst surface dynamics, surface–solvent interface dynamics, and lithium diffusion coefficients are difficult. Effectively bridging time and length scales is critical to materials discovery as many properties of interest are not directly accessible from DFT calculations but need multi-scale strategies. For example, catalysis is one of the most complex multi-scale phenomena, where the surface and adsorbates dynamically interact and change. In the simulation of gaseous phase catalysts, micro-kinetic modeling and kinetic Monte Carlo have been used successfully to bridge these gaps<a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a> (similarly for the battery as well<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a>). However, these multi-scale approaches rely on the ordered structure of the materials, and such multi-scaling becomes difficult for disordered structures.<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a> In such a case, developing a force-field-like model for the complex system could bridge these scales by its fast-prediction ability (see <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>). Force-field modeling has been used typically to address larger-scale simulations, but flexible models that can predict energy and force at <span class="italic">ab initio</span>-level accuracy is desired. In this regard, machine learning has been used to correlate atomic coordinates with the potential surface. Machine learning potential directly learns the potential surface from <span class="italic">ab initio</span> calculations, and aims to predict the <span class="italic">ab initio</span> potential surface at a significantly lower cost.<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">14,119,120</span></sup></a> While the non-physical form of machine learning limits model interpretation and extrapolation is difficult, the fast calculation speed and the <span class="italic">ab initio</span> comparable accuracy are the combination of the advantages of force-field and DFT.<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a> We refer to other reviews for an overview of the various machine learning potential methods.<a title="Select to navigate to reference" href="#cit119"><sup><span class="sup_ref">119,120</span></sup></a></span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f6.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">Machine learning potential enables microscale simulations to understand other key properties of interest.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <div>
          
          <span id="sect537"/><span class="c_heading_indent">2.3.1. Proofs of concept. </span>
          <span>Some of the earliest examples of using machine learning potentials (MLPs) to study material bulk have been pioneered by Behler and co-workers. Recently, application of MLPs has been extended from a single component to multicomponent systems.<a title="Select to navigate to references" href="#cit122"><sup><span class="sup_ref">122</span></sup></a> Here, a zinc oxide database consisting of 7366 clusters, 26<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>037 bulk structures, and 5347 slab geometries is used to achieve a 2 meV per atom RMSE. Such modeling has been applied to copper bulk and surfaces as well.<a title="Select to navigate to references" href="#cit123"><sup><span class="sup_ref">123</span></sup></a> The MLP framework of Behler and co-workers has been developed into the Aenet software package that can systematically generate data sets to develop a model.<a title="Select to navigate to references" href="#cit124"><sup><span class="sup_ref">124</span></sup></a> Software evaluation demonstrates independence of the CPU time from the number of atoms, attractive for the multi-scale approach. Similarly, Kitchin and co-workers have employed a MLP framework for zirconia to test its ability to predict diverse bulk properties.<a title="Select to navigate to references" href="#cit125"><sup><span class="sup_ref">125</span></sup></a> A total of 2178 DFT calculations are used to train the model which demonstrated high accuracy for formation energy, the equation of states, oxygen vacancy formation energies, and diffusion barrier prediction. MLP has also shown great predictive ability for predicting surface energy, palladium vacancy formation, diffusion barriers, and adatom diffusion barriers for palladium particles.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a></span>
        </div>
        <div>
          
          <span id="sect546"/><span class="c_heading_indent">2.3.2. Particle, adsorbate, and solvent dynamics for catalysis. </span>
          <span>In the interest of catalysis, metal particle structure prediction is critical, as the surface structure determines the catalytic activity. Previously, particle structures have been determined using Wulff construction, which involves minimizing the surface energy. However, Wulff construction is difficult to use for systems involving alloys, adsorbates, and solvents. Several studies have leveraged the MLP's scalability to address such challenges. Conventionally, the <span class="italic">ab initio</span> calculations for the catalytic surface are performed using a single facet surface, assuming that the catalyst particle is large enough. In the case of small particle catalysts, deciding the shape of catalysts for performing <span class="italic">ab initio</span> calculations is not clear. In this regard, Jiang and co-workers have combined MLP with the basin-hopping algorithm to find a global minimum structure for Au particles.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a> Similarly, Alexandrova and co-workers have introduced a particle structure prediction framework called NN-PGOPT that performs global structure minimization using neural net potential.<a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> The training involved efficient sampling of the configuration space <span class="italic">via</span> bond length distribution analysis. The approach has been applied to Pt<small><sub>9</sub></small> and Pt<small><sub>13</sub></small> nanoclusters and the minimum stability structure changed at 200 K and 100 K, respectively.</span>
          <p class="otherpara">Artrith and co-workers have investigated solvents and Au–Cu alloys using MLP. Au–Cu alloys have shown promising overpotential and stability for CO<small><sub>2</sub></small> reduction electrochemistry.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> However, identifying the active site for electrochemical reactions is difficult due to the solvent and adsorbate. The MLP model is trained using 24<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>995 DFT calculations consisting of the Au–Cu alloy bulk, slab, and clusters in a vacuum and in water. The developed model was combined with molecular dynamics and Monte Carlo simulations and predicted a Cu–Au core–shell structure in agreement with experimental results. The temperature dependence of the core–shell structure is observed and a potential strategy for nanoparticle structure control during experimental synthesis is suggested. A computation-based synthesis suggestion is atypical due to the complexity of the synthesis simulation, demonstrating the MLP's ability to couple the time–length-scale. In addition, Artrith and co-workers have shown that the Cu–Au alloy structure changes from the core–shell structure in a vacuum to a mixed surface in an aqueous solvent.<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> This demonstrates that MLP can be used to understand the surface structure under reaction conditions in order to perform DFT investigation more in-line with experimental conditions. The MLP predicted nanoparticle structure agreed well with the Wulff-construction predicted structure, validating the neural network potential-based particle prediction. While Artrith and co-workers used a simple frozen water shell model to account for the solvent effect, Behler and co-workers performed water–copper interface dynamics simulations for various surfaces.<a title="Select to navigate to references" href="#cit131"><sup><span class="sup_ref">131</span></sup></a> Here, the water–copper interaction strength has been shown to depend on the facets, and structures of the interface hydration layer have been analyzed.</p>
          <p class="otherpara">MLP has also been applied for the gas phase adsorbate surface system as well. Kroes and co-workers investigated the N<small><sub>2</sub></small>/Ru(0001) system where the phonons, wave-like vibrations of surface atoms, are used to describe dissociative chemisorption of N<small><sub>2</sub></small> more in-line with the experimental conditions.<a title="Select to navigate to references" href="#cit132"><sup><span class="sup_ref">132</span></sup></a> Combining MLP with molecular dynamics allows the computation of a sticking coefficient lower than previously possible using <span class="italic">ab initio</span> molecular dynamics, which also shows good agreement with the experiment. Kitchin and co-workers have implemented MLP to predict dynamic interactions between oxygen atoms on the Pd(111) surface which enables molecular dynamics simulation of adsorbates on catalytic surfaces.<a title="Select to navigate to references" href="#cit133"><sup><span class="sup_ref">133</span></sup></a></p>
          <p class="otherpara">On the other hand, Nørskov and coworkers implemented MLP for classic binding energy calculation problems.<a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> In the interest of predicting CO<small><sub>2</sub></small> reduction activity, MLP is used to learn the CO binding energy for various catalytic site environments of the Ni–Ga alloy. The geometries of CO at 583 binding sites were relaxed using neural network potential, and if the neural network potential error rose above 0.2 eV, DFT calculation is performed which is subsequently used as a training set. This modeling revealed the active sites for the Ni–Ga alloy, providing a rationalization for its high activity for CO<small><sub>2</sub></small> reduction. Compared to full explicit DFT calculations, only 10% of DFT calculations are used, demonstrating computational time efficiency of the machine learning approach.</p>
        </div>
        <div>
          
          <span id="sect572"/><span class="c_heading_indent">2.3.3. Machine learning potential in other topics. </span>
          <span>In the field of batteries, the applications of MLP are typically limited, as battery materials are solid and the lattice-based multi-scale approach is usually sufficient for coupling the time–length scale, and <span class="italic">ab initio</span> calculations are often sufficient for battery materials discovery. However, amorphous phase Li–Si has received attention due to its high capacity for Li.<a title="Select to navigate to references" href="#cit135"><sup><span class="sup_ref">135</span></sup></a> In order to address the amorphous structure, Artrith <span class="italic">et al.</span> used neural network potential with genetic algorithm sampling effectively to discover low energy amorphous Li–Si structures and their energy.<a title="Select to navigate to references" href="#cit136"><sup><span class="sup_ref">136</span></sup></a> MLP is trained using only the crystalline Li–Si structure, and, in combination with molecular dynamics, demonstrates near-ground-state amorphous Li–Si structures. Onat <span class="italic">et al.</span>, also applied MLP to evaluate the voltage and diffusion coefficient of amorphous Li–Si structures.<a title="Select to navigate to references" href="#cit137"><sup><span class="sup_ref">137</span></sup></a> Using three sub-networks including crystalline Si, amorphous Si, and an amorphous Li–Si alloy and freezing weights reduced the validation error. Fujikake <span class="italic">et al.</span> applied MLP to predict the Li intercalation energy of graphitic and disordered carbon materials.<a title="Select to navigate to references" href="#cit138"><sup><span class="sup_ref">138</span></sup></a> The MLP model's predictive ability for the Li adsorption energy and Li migration energy of graphitic carbon and Li diffusion trajectories of disordered carbon is demonstrated.</span>
        </div>
      
      
        
        <h3 id="sect584"><span class="b_heading">2.4. Crystal discovery</span></h3>
        <span>Besides predicting the properties of interest of a defined chemical space, discovering new stable crystal structures is another critical aspect of materials discovery. Leveraging the former approach to discover materials typically involves generation of idealized materials followed by using machine learning to predict target properties. However, generated idealized materials may not be stable materials, may be unsynthesizable or the synthesis routine may not be known. In this regard, the latter approach can provide the community with a database of stable materials which researchers can use as a starting point for searching for new materials. For example, Jeffrey and co-workers have discovered 15 new photoanode materials for splitting water by step-by-step screening.<a title="Select to navigate to references" href="#cit139"><sup><span class="sup_ref">139</span></sup></a> This approach has been realized considerably through the Materials Genome Initiative, a result of which is the Materials Project crystal structure database. In addition, designing materials with target properties, the so-called inverse design of materials, has received tremendous attention recently due to its implications of guided materials discovery. Although the theoretical and experimental high-throughput screening has demonstrated success in finding new promising candidates, these approaches omit materials that are not in the screening scope. In this section, we present a literature review on various data-driven models for crystal structure prediction and high-throughput screening as well as inverse designing. <a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a> summarizes the methods that are discussed below.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f7.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">Four widely used inverse designing methods to discover crystal.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <div>
          
          <span id="sect592"/><span class="c_heading_indent">2.4.1. Crystal structure type prediction. </span>
          <span>Identifying the symmetry group of a crystal is one of the initial characterization and analysis steps in materials science. In computational investigations, knowing the symmetry group of the crystal structure is critical for efficient sampling in reciprocal space and reducing the computation cost for matrix operations. Although many open-source packages have been developed to identify symmetry groups, their performance is not reliable for a crystal with random atom displacement and defects, and they are sensitive to user-defined thresholds. In this regard, Ziletti <span class="italic">et al.</span> have introduced the data-driven machine learning model to classify the crystal structures into 8 space groups.<a title="Select to navigate to references" href="#cit140"><sup><span class="sup_ref">140</span></sup></a> Here, a total of 100<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 theoretically calculated diffraction patterns from the 3D atomic arrangement are used to train the deep learning model. Although this model is limited to only 8 crystal systems, the proposed model outperforms other packages significantly regardless of defects in the crystal structures. In the case of an experimental investigation, the extraction of symmetry group information from the spectroscopy data can enhance experimental characterization capability. In this regard, Park <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit141"><sup><span class="sup_ref">141</span></sup></a> trained a deep learning model to classify X-ray diffraction patterns into 230 space groups, 101 extinction groups, and 7 crystal systems simultaneously. The model demonstrates a reliable accuracy of 81.14, 83.83 and 94.99% for the space group, extinction group, and crystal system, respectively. On the other hand, a couple of models have been developed to predict the crystal structure type given the fixed chemical formula type such as equiatomic binary (AB)<a title="Select to navigate to references" href="#cit142"><sup><span class="sup_ref">142</span></sup></a> and ternary (ABC)<a title="Select to navigate to references" href="#cit143"><sup><span class="sup_ref">143</span></sup></a> compounds. Here, the support vector machine was used for the high-throughput classification model. Notably, the model identified a new experimentally validated material, an RhCd compound with the CsCl-type structure.<a title="Select to navigate to references" href="#cit142"><sup><span class="sup_ref">142</span></sup></a> Furthermore, Oliynyk <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit143"><sup><span class="sup_ref">143</span></sup></a> experimentally confirmed 19-polymorphs between TiNiSi- and ZrNiAl-type structures in agreement with experimental results.</span>
        </div>
        <div>
          
          <span id="sect605"/><span class="c_heading_indent">2.4.2. Crystal structure prediction given composition. </span>
          <span>Previously, a combinatorial chemical space search strategy is used where stable structures from DFT calculations are chosen based on thermodynamic stability. However, DFT calculations are expensive, and synthesizability is not guaranteed; thus recent investigations have focused on developing a crystal structure prediction model using statistical theory and machine learning.</span>
          <p class="otherpara">Fischer <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit144"><sup><span class="sup_ref">144</span></sup></a> approached materials discovery using statistics and proposed data mining structure prediction (DMSP), a probability-theory-based model, for binary alloys, and the model was used to predict novel nitrogen-rich nitride materials.<a title="Select to navigate to references" href="#cit145"><sup><span class="sup_ref">145</span></sup></a> Furthermore, this concept was extended by Hautier <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit146"><sup><span class="sup_ref">146</span></sup></a> to ternary materials where 209 new compounds were discovered with a minimal computational budget. The model predicted two new compounds in the Mg–Mn–O system, MgMnO<small><sub>3</sub></small> and Mg<small><sub>2</sub></small>Mn<small><sub>3</sub></small>O<small><sub>8</sub></small>, and for MgMnO<small><sub>3</sub></small>, the diffraction pattern matched the experimental diffraction pattern. On the other hand, Ryan <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit147"><sup><span class="sup_ref">147</span></sup></a> proposed a neural network based model using the normalized atomic fingerprints to predict crystal structures given alloy compositions.</p>
          <p class="otherpara">Besides the supervised-learning models introduced above, DFT-based evolutionary algorithms have been widely employed to predict crystal structures and generate materials with target properties. The software packages Crystal structure analysis by particle swarm optimization<a title="Select to navigate to references" href="#cit148"><sup><span class="sup_ref">148</span></sup></a> and XTALOPT<a title="Select to navigate to reference" href="#cit149"><sup><span class="sup_ref">149,150</span></sup></a> are well known. Zhou <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit151"><sup><span class="sup_ref">151</span></sup></a> used XTALOPT to predict host–guest Na–Fe intermetallics at high pressures and Na<small><sub>3</sub></small>Fe and Na<small><sub>4</sub></small>Fe were predicted to be stable at pressures above 120 and 155 GPa, respectively. All the predicted materials have formed a host–guest-like Na sublattice structure. These structures are similar to the host framework of the self-hosting incommensurate phases observed in group I and II elements. In addition, the model is further used to find 2D B<small><sub>2</sub></small>S materials and discover new anisotropic 2D-Dirac cone materials.<a title="Select to navigate to references" href="#cit152"><sup><span class="sup_ref">152</span></sup></a> Furthermore, Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit153"><sup><span class="sup_ref">153</span></sup></a> used the evolutionary algorithm to predict new metastable allotropes of Li<small><sub>2</sub></small>MnO<small><sub>3</sub></small> as cathode materials under a high pressure of 20 GPa. Similarly, Shamp <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit154"><sup><span class="sup_ref">154</span></sup></a> predicted the most stable hydrides of phosphorus (PH<small><sub><span class="italic">n</span></sub></small>, <span class="italic">n</span> = 1–6) at 100, 150, and 200 GPa, pressure of which the phosphorus hydrides decomposes to elemental phases such as PH<small><sub>2</sub></small> and H<small><sub>2</sub></small>. Interestingly, three metallic PH<small><sub>2</sub></small> phases have been found that are dynamically stable and superconducting between 100 and 200 GPa providing new insights on high-pressure-driven materials with properties that cannot be observed at 1 atm.</p>
        </div>
        <div>
          
          <span id="sect643"/><span class="c_heading_indent">2.4.3. Stability classification models. </span>
          <span>Another form of crystal prediction involves a classification model, where the model predicts the stability of the given compositions. These models are typically trained with a well-defined chemical space which could result in better prediction accuracy than aforementioned models that are designed to be universal. For example, Heusler compounds (AB<small><sub>2</sub></small>C-type compounds), intermetallic compounds that are also known as Cu<small><sub>2</sub></small>MnAl-type crystal structures, have attracted attention for thermoelectric and spintronic applications. However, Oliynyk <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit155"><sup><span class="sup_ref">155</span></sup></a> showed that identifying the structure type of a synthesized material is difficult as several structure types (Heusler, CsCl-type, and inverse-Heusler type) can form given the same chemical composition. Therefore the authors constructed the Heusler compound classification model by utilizing a random forest model and experimentally confirmed 12 novel gallide materials (MRu<small><sub>2</sub></small>Ga and RuM<small><sub>2</sub></small>Ga) that were identified using high-throughput machine learning screening. A similar approach for the half-Heusler compound was proposed by Legrain <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit156"><sup><span class="sup_ref">156</span></sup></a> and 481 candidates were identified from the high-throughput of 71<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>178 candidates. Similarly, Balachandran <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit157"><sup><span class="sup_ref">157</span></sup></a> investigated into perovskite structure classification using the two-step machine learning models: one for classifying perovskite and the other for classifying the cubic perovskite structure. The proposed models were trained with the experimentally known ABO<small><sub>3</sub></small> compounds. High-throughput screening was performed and revealed 625 ABO<small><sub>3</sub></small> compounds which were further analyzed using DFT calculations, suggesting 87 highly promising cubic perovskite materials. All the listed results suggest that the data-driven approach can be effectively used to determine the class of the crystal structure.</span>
          <p class="otherpara">Machine learning can also be applied to perform optimization for various materials and device designs. For example, Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit158"><sup><span class="sup_ref">158</span></sup></a> proposed a deep learning framework for design parameter prediction for on-demand design of the chiral metamaterials where the developed model enables not only the prediction of light–matter interaction properties of devices but also the proposal of design parameters for nano-photonic devices suggesting that the deep-learning-based model effectively used real world device design. A similar approach was also proposed by Peurifoy <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit159"><sup><span class="sup_ref">159</span></sup></a> where the author proposed a neural network model for the inverse design of nanophotonic particle simulation with the analytical gradient method. Furthermore, Liu <span class="italic">et al.</span>, by combining a forward neural network (property prediction network) and an inverse network (input feature prediction network), overcame non-uniqueness in all inverse scattering problems. Interestingly, the structure of the proposed model is quite similar to that of the novel autoencoder widely used for generative models, but the authors used each part of the autoencoder (<span class="italic">i.e.</span> encoder and decoder) as independent regression models to handle the fundamental non-uniqueness of the inverse scattering problem effectively. One interesting study on machine learning for real materials synthesis was conducted by Yuan <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit160"><sup><span class="sup_ref">160</span></sup></a> to predict electrostrain of Pb-free BaTiO<small><sub>3</sub></small> (BTO)-based materials. Here, the author used both exploration (using uncertainty) and exploitation (using only model prediction) to find out the optimal criterion for new novel BTO-based materials, and (Ba<small><sub>0.84</sub></small>Ca<small><sub>0.16</sub></small>) (Ti<small><sub>0.90</sub></small>Zr<small><sub>0.07</sub></small>Sn<small><sub>0.03</sub></small>)O<small><sub>3</sub></small> was confirmed to be a novel piezoelectric material with large electrostrain both experimentally and theoretically.</p>
        </div>
      
    
    
      
      <h2 id="sect675"><span class="a_heading">3. Successful materials discovery</span></h2>
      <span>The summaries above demonstrate that the machine learning strategy is becoming widespread in the materials science and engineering field. While many studies demonstrate significant potential to accelerate research in high-throughput discovery, only some studies discuss the discovery of new economical materials using the machine learning approach. In this section, we summarize these studies to understand their key ingredients to discuss future prospects in Section 4.</span>
      <p class="otherpara">Aspuru-Guzik and co-workers have been leading a one-shot approach where high-throughput computational screening is performed followed by an experimental demonstration of the discovered materials (<a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>). While organic light-emitting diodes (OLEDs) have many industrial applications due to their high efficiency and color properties, blue OLED development has been particularly difficult due to the higher energy needed for excitation. In this regard, Aspuru-Guzik and co-workers have presented a highly integrated design process involving theoretical insight, quantum mechanics, machine learning, industrial expertise, and experiments to discover new highly efficient blue OLEDs.<a title="Select to navigate to references" href="#cit161"><sup><span class="sup_ref">161</span></sup></a> Here, the chemical space is defined using a combinatorial enumeration of defined fragments. These fragments are selected using theoretical intuition. Molecules with unstable substructures known from experiments are filtered as well. Then, a neural network is employed to find the best OLED candidate, which is analyzed by time-dependent DFT calculations, resulting in a total of 400<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 calculations. From the 400<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 candidates, four candidates were experimentally validated after 2500 human experts voted for property novelty of candidates and synthetic accessibility. The study demonstrated one validated candidate showing 22% external quantum efficiency as well as about one thousand potential candidates with equal or better performance. This study demonstrates that an end-to-end highly integrated approach directly leads to the discovery of new materials.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f8.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">Collaborative discovery approach adopted by Aspuru-Guzik and co-workers to discover blue OLED materials (adapted with permission from <a title="Select to navigate to references" href="#cit161">ref. 161</a> Copyright 2016 Springer, <span class="italic">Nat. Mater.</span>). The screening stages integrating theoretical and computational approaches and experimental input and testing were the key to successful discovery.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">One of the methods to produce white light using LEDs involves phosphor coating on a light emitting diode (LED), where part of the LED emission is absorbed and re-emitted as photons at different wavelengths. The combination of all the photons results in white light. Such engineering simplifies the design and improves the efficiency of the white light LED, but only a handful of phosphor materials have been reported. In this regard, Brgoch and co-workers employed DFT, the support vector machine regression model, and experimental validation to discover NaBaB<small><sub>9</sub></small>O<small><sub>15</sub></small> which is highly efficient and stable.<a title="Select to navigate to references" href="#cit162"><sup><span class="sup_ref">162</span></sup></a> Here, support vector machine regression is trained with 2610 DFT-based Debye temperature from the Materials Project database that correlates with the quantum efficiency of the materials. Then, the model is used to screen 2071 materials (1) that are available in Pearson's crystal database, (2) for which the bandgap is available in the Materials Project database, (3) that are ternary, and (4) that are non-metals. Out of these, NaBaB<small><sub>9</sub></small>O<small><sub>15</sub></small> shows the most ideal Debye temperature and bandgaps, which are further validated using experiments (see <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>). The key in this study was the screening of materials from the database which contains experimentally observed materials.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f9.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">Machine learning predicted Debye temperature against the calculated bandgap. Machine learning predicted Debye temperatures (<span class="italic">Θ</span><small><sub>D,SVR</sub></small>) against the density functional theory calculated bandgaps (<span class="italic">E</span><small><sub>g,DFT</sub></small>) for 2071 compounds predicted (adapted with permission from <a title="Select to navigate to references" href="#cit162">ref. 162</a> Copyright 2018 Springer, <span class="italic">Nat. Commun.</span>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Degradation of battery performance caused by electrolyte decomposition can be improved by adding electrolyte additives as discussed above. Anode additives are reduced prior to electrolyte solvents and cathode additives are oxidized prior to electrolyte solvents to form a stable solid electrolyte interphase layer to reduce the irreversible capacity.<a title="Select to navigate to references" href="#cit163"><sup><span class="sup_ref">163</span></sup></a> In this regard, calculating the reduction and oxidation potential can help find promising electrolyte additives. Park <span class="italic">et al.</span> used a neural network model to predict the oxidation and reduction potentials for organic additives and solvents using 86 descriptors, such as bonding types, functional groups and so on.<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a> The relationship between the redox potential and the functional group was proposed as in <a title="Select to navigate to figure" href="#imgfig10">Fig. 10(a)</a>. From the results, it can be seen that organic compounds containing double bonds are prone to reduction and unsusceptible to oxidation, <span class="italic">i.e.</span>, the compounds can be used as anode additives. Among various candidates that meet these conditions, quinoxaline was tested for full cell applications and validated to improve cycle life as shown in <a title="Select to navigate to figure" href="#imgfig10">Fig. 10(b)</a>.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f10.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">(a) Schematic of the distribution of functional groups on the potential plane, (b) cyclic performance of a Li(Ni<small><sub>0.88</sub></small>Co<small><sub>0.11</sub></small>Al<small><sub>0.01</sub></small>)O<small><sub>2</sub></small>/graphite full cell with and without a quinoxaline additive (adapted with permission from <a title="Select to navigate to references" href="#cit164">ref. 164</a> Copyright 2016 Royal Society of Chemistry, <span class="italic">Phys. Chem. Chem. Phys.</span>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Saeki and co-workers applied a similar strategy, where molecular fingerprinting techniques are combined with neural networks and random forests to predict the bandgap, molecular weight and power conversion efficiency for fullerene polymer using approximately a thousand experimentally calculated properties of polymer-fullerene. To demonstrate materials design (see <a title="Select to navigate to figure" href="#imgfig11">Fig. 11</a>), 2.3 million molecules from the Harvard Clean Energy Project database were screened. A total of 1000 molecules were selected from the database based on the first-principles calculated properties, 149 molecules of which were selected after the screening using a random forest model. One molecule was manually chosen based on its possibility of synthesis. The study identified a new polymer with a power conversion efficiency of ∼5.4%.<a title="Select to navigate to references" href="#cit165"><sup><span class="sup_ref">165</span></sup></a> The study shows stage by stage screening starting from the existing large first-principles database, followed by a machine learning model trained using experimental values to narrow the gap between the theory and experiments. Finally, manual consideration was used to decide the synthesis accessibility of the screened material.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig11"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f11.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f11.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 11 </b> <span id="fig11"><span class="graphic_title">Polymer design scheme combining first-principles, machine learning, and manual consideration to discover a new polymer for organic photovoltaics (adapted with permission from <a title="Select to navigate to references" href="#cit165">ref. 165</a> Copyright 2018 ACS publications, <span class="italic">J. Phys. Chem. Lett.</span>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit166"><sup><span class="sup_ref">166</span></sup></a> used the DMSP scheme, already discussed in the previous section, to expand the chemical space of the various nitride systems since it can be used for various applications such as solid-state lighting, ammonia-synthesis catalysts, superconductors, superinsulators, electrodes and so on. In spite of the aforementioned high potential, the nitride systems (&lt;400) are relatively under-explored in the ICSD compared to the ternary metal oxides (&gt;4000) suggesting that it is important to find new (meta-) stable metal nitrides. Because the DMSP scheme can easily be applied to predict the crystal structure from the given composition of the target metal nitride system, they first constructed a map of the metal nitrides after doing DFT calculations to identify the stability of the predicted materials as shown in <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a>. One interesting point is that although there are many previously known metal nitrides in the nitride map, there are still plausible new ternary metal nitrides indicating that machine learning can be effectively used to discover a large materials space compared to the conventional combinatorial explorations.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig12"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f12_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f12.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f12.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 12 </b> <span id="fig12"><span class="graphic_title">Map of the constructed metal nitrides using the DMSP scheme and DFT calculations to identify stability (adopted with permission from the corresponding author of <a title="Select to navigate to references" href="#cit166">ref. 166</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">The other interesting point is that from the theoretical predictions the authors experimentally identified 7 new phases of Zn- and Mg-based ternary metal nitrides (Zn–Mo–N, Zn–W–N, Zn–Sb–N, Mg–Ti–N, Mg–Zr–N, Mg–Hf–N, and Mg–Nb–N) of which the latter new materials can be classified into the two unique crystal structures (the wurtzite and rocksalt structure; see <a title="Select to navigate to figure" href="#imgfig13">Fig. 13</a>). Although there is still a need for human intuition in experimental synthesis from the newly discovered materials, one can reduce unnecessary trial and error for exploring un-plausible chemical space by utilizing machine learning models.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig13"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f13_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f13.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f13.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 13 </b> <span id="fig13"><span class="graphic_title">(a) 7 new phases of the ternary metal nitrides with the corresponding space group and formation energies, (b) detailed structures for the newly discovered nitrides, (c) synchrotron measured XRD patterns of new Zn- and Mg-based ternary nitrides and (d) discovery histogram for new ternary nitride spaces, based on entries as cataloged in the ICSD (adopted with permission from the <a title="Select to navigate to references" href="#cit166">ref. 166</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Reed and co-workers have leveraged machine-learning with experiments to predict Li ion conductivity in order to discover a solid electrolyte for Li-ion batteries.<a title="Select to navigate to references" href="#cit167"><sup><span class="sup_ref">167</span></sup></a> Here, 12<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>831 Li-containing crystal structures from the Materials Project were extracted, and performance-related properties such as electronic conductivity and electrochemical stability were computed using DFT and theory (see <a title="Select to navigate to figure" href="#imgfig14">Fig. 14</a>). Another critical performance measure is the Li-ion conductivity, but DFT is difficult to use to compute this metric as it is a larger scale phenomenon. However, ionic conductivity for 40 crystal structures was available; thus the authors have implemented logistic regression to classify high and low conductivity <span class="italic">via</span> 20 features extracted from the crystal's elemental and structural properties. The screening narrowed the search space down to 21 structures, the performances of which were confirmed by experiments. This work highlights the difficulty of theoretical approaches to simulate larger scale phenomena as well as highlighting the importance of integrating experiments to screen materials.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig14"><a href="/image/article/2019/TA/c9ta02356a/c9ta02356a-f14_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9ta02356a-f14.tif" src="/image/article/2019/TA/c9ta02356a/c9ta02356a-f14.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 14 </b> <span id="fig14"><span class="graphic_title">Flowchart of the discovery of a new Li solid electrolyte by integrating DFT, machine learning, and experiments. Machine learning is used to predict Li ion conductivity which is difficult to compute using DFT due to its multi-scale nature (adopted with permission from <a title="Select to navigate to references" href="#cit167">ref. 167</a> Copyright 2017 Royal Society of Chemistry, <span class="italic">Energy Environ. Sci.</span>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
    
    
      
      <h2 id="sect743"><span class="a_heading">4. Future prospects</span></h2>
      
        
        <h3 id="sect746"><span class="b_heading">4.1. Summary</span></h3>
        <span>Within 5 years of the launch of the Materials Genome Initiative in 2011, some of the machine learning models have shown performances approaching chemical accuracy (1 kcal mol<small><sup>−1</sup></small> or ∼43 meV) for given training and test cases for electronic energies (within the error of data acquisition methods) from atomic coordinates. Many models are flexible and can achieve respectable accuracy given a large enough data set. A majority of the studies have demonstrated significant potential in terms of accelerating the DFT calculations. This signifies that the machine learning, science, and engineering interdisciplinary community has effectively established a strategy to perform accurate theoretical high-throughput screening. In particular, renewable technologies with accessible activity-descriptors have been widely investigated with ML-based high-throughput screening and promising candidates have been suggested.</span>
        <p class="otherpara">Machine learning potentials (MLPs) have demonstrated their potential to couple the DFT time–length scale to a larger scale. In particular, much attention has been devoted to understanding the surface and nanocluster dynamics in the interest of catalysis. Notably, MLP has been effectively applied to the multi-scaling phenomenon of the nanocluster structure change under reaction conditions, demonstrating its ability to reveal new catalytic phenomena. Furthermore, MLP shows promise for identifying active sites of an alloy by learning the binding energy activity descriptor.</p>
        <p class="otherpara">Discovering new stable crystals is critical to expanding our knowledge of viable materials. In this respect, several studies focused on predicting crystal structures given the materials composition constraint. In addition, a couple of machine learning augmented DFT based materials discovery methods are introduced and suggested as a standard strategy for discovering materials. Another approach has involved stability screening within a defined chemical space. Many of these introduced approaches have successfully identified previously unknown materials, signifying that the community has a good idea in leveraging machine learning to discover new materials.</p>
      
      
        
        <h3 id="sect753"><span class="b_heading">4.2. Moving forward</span></h3>
        <div>
          
          <span id="sect756"/><span class="c_heading_indent">4.2.1. Experimental validation. </span>
          <span>Reviewing some of the studies discussed above has clearly proven the great acceleration effect of machine learning in theoretical research, especially advancing the high-throughput screening strategy. However, there is only a little end-to-end screening research where the final candidates are validated using experiments. This may in part be related to the lack of an accurate measurement of synthesizability of computationally generated or suggested materials that can be easily computed, although a metric has been suggested for molecules in the interest of drug discovery.<a title="Select to navigate to references" href="#cit168"><sup><span class="sup_ref">168</span></sup></a> Typically, materials in the screening scope are generated using heuristic rules; thus they are not guaranteed to be synthesizable and their synthesis procedures are not necessarily known.</span>
          <p class="otherpara">Several possible strategies can be suggested. For example, a number of studies leverage the experimental experts to measure the synthesizability. The blue OLED discovery study developed a web interface for experimentalists to vote on the synthesizability of molecules screened using a machine.<a title="Select to navigate to references" href="#cit161"><sup><span class="sup_ref">161</span></sup></a> Similarly, manual screening of the synthetic aspect is considered for the discovery of polymers for organic photovoltaics.<a title="Select to navigate to references" href="#cit165"><sup><span class="sup_ref">165</span></sup></a> Another popular strategy is to avoid hypothetical materials entirely by defining the screening scope as the experimentally known materials. The discovery of white LED materials introduced above is an example.<a title="Select to navigate to references" href="#cit162"><sup><span class="sup_ref">162</span></sup></a> This approach has been one of the most successful strategies for screening studies not involving machine learning. In addition, theoretical screening criteria are often limited to properties that are easily computable due to practical consideration (tractable time–length scale), instead of properties that are more directly relevant to experiments, and for these cases surrogate models are helpful to predict experimentally determined properties. All this shows that close collaboration between the computational and experimental investigators is key. Also, it would be helpful if the theoretical and experimental researchers closely communicate coherently at the beginning of collaboration to improve the success rate of machine prediction followed by experimental validation, instead of performing separate roles of “design” and “validation” by theory and experiments, respectively.</p>
        </div>
        <div>
          
          <span id="sect765"/><span class="c_heading_indent">4.2.2. Data infrastructure. </span>
          <span>Despite the introduction and the great utility of several existing databases, many important issues remain to be addressed for data infrastructure. While the community generates a large amount of <span class="italic">ab initio</span> and experimental data, a significant portion remains unshared often because it did not come to fruition. However, these “failed data” are valuable to machine learning to learn,<a title="Select to navigate to references" href="#cit169"><sup><span class="sup_ref">169</span></sup></a> just as a human learns from failed experiments. An accessible platform to share failed data could help increase data quality and size dramatically but the effort to collect and share such data is lacking. In addition, details are critical to reproduce the theoretical and experimental results. Reproducibility can reduce the duplication in the effort, enable eye-to-eye comparison between different set-ups, and provide all the ingredients needed for machine learning to learn the results. For example, the DFT simulation parameters vary for different applications. An effort to build and follow a standard reporting procedure has shown an impact in the medical community, and such effort can also aid data-based materials science well. In terms of theoretical simulations, developing strategies to bridge results of different settings can enable exploiting community-generated data that are currently too diverse in simulation settings to be used. As discussed before, Pilania <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a> have shown that the band-gap calculated using the high accuracy DFT method can be predicted by low accuracy DFT methods, suggesting existing correlations. Thus, developing machine-learning models that can transform results in one parameter space into the others shows high potential. Also, compared to the crystal databases, a theoretical and experimental database for the surface is lacking. Notably, the CatApp database developed by Nørskov and co-workers<a title="Select to navigate to references" href="#cit170"><sup><span class="sup_ref">170</span></sup></a> has developed the binding energy database for catalysis, but the database is limited to a few adsorbates' binding energy and idealized surface. Continuously expanding the surface database as well as developing an experimental database can encourage machine learning applications and end-to-end catalyst screening.</span>
        </div>
        <div>
          
          <span id="sect774"/><span class="c_heading_indent">4.2.3. New directions and applications. </span>
          <span>In terms of future prospects for machine learning applications, Aspuru-Guzik and co-workers have pioneered inverse design models that generate molecules with target properties and have been an active area of research for drug discovery.<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a> Previous machine learning based screening was limited by the assumption that the screening scope contains promising materials. Inverse modeling differs from previous screening methods as the machine outputs the materials with target properties, thus having critical implications in materials discovery. However, such models for crystal and surface structures are limited, despite their transformative implications in materials discovery. Developing reversible representation, where the encoded input feature from the crystal structure can be recovered to its original crystal structure, is the critical next step that can enable the inverse design of renewable energy materials.</span>
          <p class="otherpara">Although, here, we have mainly focused on the application of machine learning in terms of computational prediction of novel functional materials mainly using computational data, utilizing actual experimental data to predict materials properties of unknown compounds or even suggest new materials can be highly impactful. The critical aspect here is to collect a large set of data that have been obtained consistently using the same experimental setup under controlled conditions. Most of the existing experimental data in the literature are sparse and inhomogeneous for use in machine learning. For this reason, the number of quality experimental machine learning studies is limited. Recently, Gregoire and co-workers have demonstrated the potential of high-throughput consistent experiments where 178<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>994 data samples are used to map the visual image of samples and their adsorption spectra.<a title="Select to navigate to references" href="#cit171"><sup><span class="sup_ref">171</span></sup></a> Furthermore, compositions and Raman signal data of 1379 BiVO<small><sub>4</sub></small> alloys have been correlated to their photoelectrochemical activity.<a title="Select to navigate to references" href="#cit172"><sup><span class="sup_ref">172</span></sup></a> These promising results demonstrate that the consistent experiments enable end-to-end data science for materials science.</p>
          <p class="otherpara">Augmenting machine learning with robotics, or the so-called self-driving laboratory, has been emerging as a significant new direction.<a title="Select to navigate to reference" href="#cit4"><sup><span class="sup_ref">4,12,124</span></sup></a> Developing the self-driving laboratory requires non-human-interrupted closed-loop flow work, where a machine learning model designs the experiments, followed by using robotics to perform the experiments and characterize the sample. Then, the new knowledge is learned by machine learning which can design the next experiment to repeat the cycle. Maruyama and co-workers are the pioneers in this regard <span class="italic">via</span> the Autonomous Research System (ARES) where the carbon nanotube growth rate is learned by machine learning model to grow the carbon nanotube at target rates, showing its potential,<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> but such an effort is still in its infancy.<a title="Select to navigate to reference" href="#cit173"><sup><span class="sup_ref">173,174</span></sup></a> The self-driving lab enables robust end-to-end materials search and is expected to revolutionize materials discovery in the future, which can be adopted in industry.</p>
        </div>
      
    
    
      <h2 id="sect788"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <h2 id="sect792"><span class="a_heading">Acknowledgements</span></h2>
      <span>We acknowledge the generous support of the Saudi Aramco–KAIST CO<small><sub>2</sub></small> Management Center and National Research Foundation of Korea (award number 2018M3D1A1089310) to perform this research.</span>
    
    <span id="sect795"><h2 id="sect791"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">
          <span class="italic">Synthesis report on the aggregate effect of INDCs</span>, United Nations Framework Convention on Climate Change, United Nations,  2016, http://https://unfccc.int/sites/default/files/resource/docs/2016/cop22/eng/02.pdf <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Synthesis%20report%20on%20the%20aggregate%20effect%20of%20INDCs%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit2">J. Rogelj, M. den Elzen, N. Höhne, T. Fransen, H. Fekete, H. Winkler, R. Schaeffer, F. Sha, K. Riahi and M. Meinshausen, <span class="italic">Nature</span>, 2016, <span class="bold">534</span>, 631 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature18307" title="DOI Link to resource 10.1038/nature18307">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gju7vI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27357792%5Buid%5D" title="PubMed Link to resource 27357792">PubMed</a>.</span></li>
      <li><span id="cit3">Mission Innovation, http://mission-innovation.net/, accessed January, 2019.</span></li>
      <li><span id="cit4">
          A. Aspuru-Guzik, K. Persson, A. Alexander-Katz, C. Amador, D. Solis-Ibarra, M. Antes, A. Mosby, M. Aykol, E. Chan, S. Dwaraknath, J. Montoya, E. Rotenberg, J. Gregoire, A. HattrickSimpers, D. M. Huang, J. Hein, G. Hutchison, O. Isayev, Y. Jung, J. Kiviaho, C. Kreisbeck, L. Roch, S. Saikin, D. Tabor, J. Lambert, S. Odom, J. Pijpers, M. Ross, J. Schrier, R. Segalman, M. Sfeir, H. Tribukait and T. Vegge, <span class="italic">Materials Acceleration Platform: Accelerating Advanced Energy Materials Discovery by Integrating High-Throughput Methods with Artificial Intelligence: Report of the Clean Energy Materials Innovation Challenge Expert Workshop</span>, Mission Innovation,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Materials%20Acceleration%20Platform:%20Accelerating%20Advanced%20Energy%20Materials%20Discovery%20by%20Integrating%20High-Throughput%20Methods%20with%20Artificial%20Intelligence:%20Report%20of%20the%20Clean%20Energy%20Materials%20Innovation%20Challenge%20Expert%20Workshop%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit5">E. Maine and E. Garnsey, <span class="italic">Resour. Policy</span>, 2006, <span class="bold">35</span>, 375–393 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.respol.2005.12.006" title="DOI Link to resource 10.1016/j.respol.2005.12.006">CrossRef</a>.</span></li>
      <li><span id="cit6">
          T. Mueller, A. G. Kusne and R. Ramprasad, in <span class="italic">Reviews in Computational Chemistry</span>, ed. A. L. Parrill and K. B. Lipkowitz, John Wiley &amp; Sons, Inc., Hoboken, New Jersey,  2016, vol. 29, ch. 4, pp. 186–273 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Reviews%20in%20Computational%20Chemistry%5BJour%5D%20AND%20vol. 29%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit7">
          X. Su and T. M. Khoshgoftaar, <span class="italic">Lect. Notes Artif. Int.</span>,  2009, vol. 2009, p. 19 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Lect.%20Notes%20Artif.%20Int.%5BJour%5D%20AND%20vol. 2009%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit8">I. Kononenko, <span class="italic">Artif. Intell. Med.</span>, 2001, <span class="bold">23</span>, 89–109 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0933-3657(01)00077-X" title="DOI Link to resource 10.1016/S0933-3657(01)00077-X">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BD3MvitVyisw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=11470218%5Buid%5D" title="PubMed Link to resource 11470218">PubMed</a>.</span></li>
      <li><span id="cit9">N. M. Nasrabadi, <span class="italic">J. Electron. Imaging</span>, 2007, <span class="bold">16</span>, 049901 <a target="_blank" class="DOILink" href="https://doi.org/10.1117/1.2819119" title="DOI Link to resource 10.1117/1.2819119">CrossRef</a>.</span></li>
      <li><span id="cit10">
          A. Graves, A. Mohamed and G. Hinton, <span class="italic">Speech recognition with deep recurrent neural networks</span>,  2013 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Speech%20recognition%20with%20deep%20recurrent%20neural%20networks%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202013%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit11">M. L. Green, C. L. Choi, J. R. Hattrick-Simpers, A. M. Joshi, I. Takeuchi, S. C. Barron, E. Campo, T. Chiang, S. Empedocles, J. M. Gregoire, A. G. Kusne, J. Martin, A. Mehta, K. Persson, Z. Trautt, J. Van Duren and A. Zakutayev, <span class="italic">Appl. Phys. Rev.</span>, 2017, <span class="bold">4</span>, 011105 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Appl.%20Phys.%20Rev.%5BJour%5D%20AND%204%5Bvolume%5D%20AND%20011105%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit12">D. P. Tabor, L. M. Roch, S. K. Saikin, C. Kreisbeck, D. Sheberla, J. H. Montoya, S. Dwaraknath, M. Aykol, C. Ortiz, H. Tribukait, C. Amador-Bedolla, C. J. Brabec, B. Maruyama, K. A. Persson and A. Aspuru-Guzik, <span class="italic">Nat. Rev. Mater.</span>, 2018, <span class="bold">3</span>, 5–20 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41578-018-0005-z" title="DOI Link to resource 10.1038/s41578-018-0005-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cjs73F" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit13">P. Nikolaev, D. Hooper, F. Webber, R. Rao, K. Decker, M. Krein, J. Poleski, R. Barto and B. Maruyama, <span class="italic">npj Comput. Mater.</span>, 2016, <span class="bold">2</span>, 16031 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/npjcompumats.2016.31" title="DOI Link to resource 10.1038/npjcompumats.2016.31">CrossRef</a>.</span></li>
      <li><span id="cit14">J. Behler, <span class="italic">J. Chem. Phys.</span>, 2016, <span class="bold">145</span>, 170901 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.4966192" title="DOI Link to resource 10.1063/1.4966192">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27825224%5Buid%5D" title="PubMed Link to resource 27825224">PubMed</a>.</span></li>
      <li><span id="cit15">R. Gómez-Bombarelli, J. N. Wei, D. Duvenaud, J. M. Hernández-Lobato, B. Sánchez-Lengeling, D. Sheberla, J. Aguilera-Iparraguirre, T. D. Hirzel, R. P. Adams and A. Aspuru-Guzik, <span class="italic">ACS Cent. Sci.</span>, 2018, <span class="bold">4</span>, 268–276 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.7b00572" title="DOI Link to resource 10.1021/acscentsci.7b00572">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29532027%5Buid%5D" title="PubMed Link to resource 29532027">PubMed</a>.</span></li>
      <li><span id="cit16">
          J. Hemminger, G. Crabtree, M. Kastner, S. Bare, B. Nora, C. Sylvia, S. Clark, P. Cummings, F. Disalvo, M. El-sayed, G. Flynn, B. Gates, L. Greene, S. Hammes-Schiffer, M. Hochella, B. Kay, K. Kirby, J. McCurdy, W. D. Morse, M. Moskovits, K. Nagy, J. Richards, J. Spence, K. Taylor, D. Tobias and J. Tranquada, <span class="italic">New Science for a Secure and Sustainable Energy Future: A report of a subcommittee to the Basic Energy Science Advisory Committee</span>, U.S. Department of Energy, U.S.A,  2008 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=New%20Science%20for%20a%20Secure%20and%20Sustainable%20Energy%20Future:%20A%20report%20of%20a%20subcommittee%20to%20the%20Basic%20Energy%20Science%20Advisory%20Committee%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit17">
          J. Stringer, L. Horton, M. Singer, J. Ahearne, G. Crabtree, C. Baker, L. DeJonghe, J. Herbst, M. Dresselhaus, R. Smalley and R. Stoller, <span class="italic">Basic Research Needs To Assure A Secure Energy Future: A Report from the Basic Energy Science Advisory Committee</span>, U.S. Department of Energy, Oak Ridge National Laboratory, U.S.A.,  2003 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Basic%20Research%20Needs%20To%20Assure%20A%20Secure%20Energy%20Future:%20A%20Report%20from%20the%20Basic%20Energy%20Science%20Advisory%20Committee%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit18">
          <span class="italic">SunShot Initiative 2030 Goals</span>, U.S. Department of Energy: Office of Energy Efficiency &amp; Renewable Energy: Solar Energy Technologies Office, U.S.A.,  2017, http://https://www.energy.gov/sites/prod/files/2018/05/f51/SunShot%202030%20Fact%20Sheet.pdf <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=SunShot%20Initiative%202030%20Goals%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit19">
          <span class="italic">Progress in Hydrogen and Fuel Cells</span>, Office of Energy Efficiency &amp; Renewable Energy, U.S. Department of Energy, Fuel Cell Technologies Office, U.S.A.,  2017, http://https://www.energy.gov/sites/prod/files/2017/10/f37/fcto-progress-fact-sheet-august-2017.pdf <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Progress%20in%20Hydrogen%20and%20Fuel%20Cells%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit20">
          A. Chen, <span class="italic">Berkeley Lab Study Estimates $80 Billion Annual Cost of Power Interruptions</span>, https://newscenter.lbl.gov/2005/02/02/berkeley-lab-study-estimates-80-billion-annual-cost-of-power-interruptions/, accessed January,  2019 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Berkeley%20Lab%20Study%20Estimates%20$80%20Billion%20Annual%20Cost%20of%20Power%20Interruptions%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit21">K. T. Butler, D. W. Davies, H. Cartwright, O. Isayev and A. Walsh, <span class="italic">Nature</span>, 2018, <span class="bold">559</span>, 547–555 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41586-018-0337-2" title="DOI Link to resource 10.1038/s41586-018-0337-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtl2jt7vL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit22">S. V. Kalinin, B. G. Sumpter and R. K. Archibald, <span class="italic">Nat. Mater.</span>, 2015, <span class="bold">14</span>, 973 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat4395" title="DOI Link to resource 10.1038/nmat4395">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGlu7nO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26395941%5Buid%5D" title="PubMed Link to resource 26395941">PubMed</a>.</span></li>
      <li><span id="cit23">L. Ward and C. Wolverton, <span class="italic">Curr. Opin. Solid State Mater. Sci.</span>, 2017, <span class="bold">21</span>, 167–176 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cossms.2016.07.002" title="DOI Link to resource 10.1016/j.cossms.2016.07.002">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht12jtb3I" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit24">R. Ramprasad, R. Batra, G. Pilania, A. Mannodi-Kanakkithodi and C. Kim, <span class="italic">npj Comput. Mater.</span>, 2017, <span class="bold">3</span>, 54 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-017-0056-5" title="DOI Link to resource 10.1038/s41524-017-0056-5">CrossRef</a>.</span></li>
      <li><span id="cit25">Y. Liu, T. Zhao, W. Ju and S. Shi, <span class="italic">J. Materiomics</span>, 2017, <span class="bold">3</span>, 159–177 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmat.2017.08.002" title="DOI Link to resource 10.1016/j.jmat.2017.08.002">CrossRef</a>.</span></li>
      <li><span id="cit26">B. R. Goldsmith, J. Esterhuizen, J.-X. Liu, C. J. Bartel and C. Sutton, <span class="italic">AIChE J.</span>, 2018, <span class="bold">64</span>, 2311–2323 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aic.16198" title="DOI Link to resource 10.1002/aic.16198">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtVSltrzE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit27">B. Sanchez-Lengeling and A. Aspuru-Guzik, <span class="italic">Science</span>, 2018, <span class="bold">361</span>, 360–365 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aat2663" title="DOI Link to resource 10.1126/science.aat2663">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyitr3L" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30049875%5Buid%5D" title="PubMed Link to resource 30049875">PubMed</a>.</span></li>
      <li><span id="cit28">J. J. Irwin and B. K. Shoichet, <span class="italic">J. Chem. Inf. Model.</span>, 2005, <span class="bold">45</span>, 177–182 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci049714+" title="DOI Link to resource 10.1021/ci049714+">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15667143%5Buid%5D" title="PubMed Link to resource 15667143">PubMed</a>.</span></li>
      <li><span id="cit29">A. Gaulton, A. Hersey, A. Karlsson, D. Mendez, E. Cibrián-Uhalte, F. Atkinson, G. Papadatos, I. Smit, J. P. Overington, J. Chambers, L. J. Bellis, M. Davies, M. Nowotka, N. Dedman, P. Mutowo, A. R. Leach, A. P. Bento and M. P. Magariños, <span class="italic">Nucleic Acids Res.</span>, 2016, <span class="bold">45</span>, D945–D954 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nar/gkw1074" title="DOI Link to resource 10.1093/nar/gkw1074">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27899562%5Buid%5D" title="PubMed Link to resource 27899562">PubMed</a>.</span></li>
      <li><span id="cit30">L. C. Blum and J.-L. Reymond, <span class="italic">J. Am. Chem. Soc.</span>, 2009, <span class="bold">131</span>, 8732–8733 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja902302h" title="DOI Link to resource 10.1021/ja902302h">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXmvFWru7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19505099%5Buid%5D" title="PubMed Link to resource 19505099">PubMed</a>.</span></li>
      <li><span id="cit31">L. Ruddigkeit, R. van Deursen, L. C. Blum and J.-L. Reymond, <span class="italic">J. Chem. Inf. Model.</span>, 2012, <span class="bold">52</span>, 2864–2875 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci300415d" title="DOI Link to resource 10.1021/ci300415d">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhsFClsL3J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23088335%5Buid%5D" title="PubMed Link to resource 23088335">PubMed</a>.</span></li>
      <li><span id="cit32">R. Allmann and R. Hinek, <span class="italic">Acta Crystallogr., Sect. A: Found. Crystallogr.</span>, 2007, <span class="bold">63</span>, 412–417 <a target="_blank" class="DOILink" href="https://doi.org/10.1107/S0108767307038081" title="DOI Link to resource 10.1107/S0108767307038081">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXpt1Sntro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17703075%5Buid%5D" title="PubMed Link to resource 17703075">PubMed</a>.</span></li>
      <li><span id="cit33">A. Belsky, M. Hellenbrandt, V. L. Karen and P. Luksch, <span class="italic">Acta Crystallogr., Sect. B: Struct. Sci.</span>, 2002, <span class="bold">58</span>, 364–369 <a target="_blank" class="DOILink" href="https://doi.org/10.1107/S0108768102006948" title="DOI Link to resource 10.1107/S0108768102006948">CrossRef</a>.</span></li>
      <li><span id="cit34">A. Jain, S. P. Ong, G. Hautier, W. Chen, W. D. Richards, S. Dacek, S. Cholia, D. Gunter, D. Skinner, G. Ceder and K. A. Persson, <span class="italic">APL Mater.</span>, 2013, <span class="bold">1</span>, 011002 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.4812323" title="DOI Link to resource 10.1063/1.4812323">CrossRef</a>.</span></li>
      <li><span id="cit35">S. Kirklin, J. E. Saal, B. Meredig, A. Thompson, J. W. Doak, M. Aykol, S. Rühl and C. Wolverton, <span class="italic">npj Comput. Mater.</span>, 2015, <span class="bold">1</span>, 15010 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/npjcompumats.2015.10" title="DOI Link to resource 10.1038/npjcompumats.2015.10">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXlslant7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit36">S. Curtarolo, W. Setyawan, G. L. W. Hart, M. Jahnatek, R. V. Chepulskii, R. H. Taylor, S. Wang, J. Xue, K. Yang, O. Levy, M. J. Mehl, H. T. Stokes, D. O. Demchenko and D. Morgan, <span class="italic">Comput. Mater. Sci.</span>, 2012, <span class="bold">58</span>, 218–226 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2012.02.005" title="DOI Link to resource 10.1016/j.commatsci.2012.02.005">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XksVyktL8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit37">A. E. Hoerl and R. W. Kennard, <span class="italic">Technometrics</span>, 1970, <span class="bold">12</span>, 55–67 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/00401706.1970.10488634" title="DOI Link to resource 10.1080/00401706.1970.10488634">CrossRef</a>.</span></li>
      <li><span id="cit38">R. Tibshirani, <span class="italic">J. R. Stat. Ser. Soc. B Stat. Methodol.</span>, 1996, <span class="bold">58</span>, 267–288 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20R.%20Stat.%20Ser.%20Soc.%20B%20Stat.%20Methodol.%5BJour%5D%20AND%2058%5Bvolume%5D%20AND%20267%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit39">H. Zou and T. Hastie, <span class="italic">J. R. Stat. Soc. Ser. B</span>, 2005, <span class="bold">67</span>, 301–320 <a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1467-9868.2005.00503.x" title="DOI Link to resource 10.1111/j.1467-9868.2005.00503.x">CrossRef</a>.</span></li>
      <li><span id="cit40">S. Wold, M. Sjöström and L. Eriksson, <span class="italic">Chemom. Intell. Lab. Syst.</span>, 2001, <span class="bold">58</span>, 109–130 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0169-7439(01)00155-1" title="DOI Link to resource 10.1016/S0169-7439(01)00155-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXotF2mtLw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit41">J. A. K. Suykens and J. Vandewalle, <span class="italic">Neural Process. Lett.</span>, 1999, <span class="bold">9</span>, 293–300 <a target="_blank" class="DOILink" href="https://doi.org/10.1023/A:1018628609742" title="DOI Link to resource 10.1023/A:1018628609742">CrossRef</a>.</span></li>
      <li><span id="cit42">S. R. Safavian and D. Landgrebe, <span class="italic">IEEE Trans. Syst. Man Cybern.</span>, 1991, <span class="bold">21</span>, 660–674 <a target="_blank" class="DOILink" href="https://doi.org/10.1109/21.97458" title="DOI Link to resource 10.1109/21.97458">CrossRef</a>.</span></li>
      <li><span id="cit43">
          H. Tin Kam, Random decision forests, <span class="italic">Proceedings of 3<small><sup>rd</sup></small> International Conference on Document Analysis and Recognition</span>,  1995, pp. 278–282, <small> DOI:<a class="DOILink" href="https://doi.org/10.1109/ICDAR.1995.598994" TARGET="_BLANK" title="DOI Link to 10.1109/ICDAR.1995.598994">10.1109/ICDAR.1995.598994</a></small>.</span></li>
      <li><span id="cit44">J. S. Cramer, <span class="italic">Stud. Hist. Philos. Sci.</span>, 2004, <span class="bold">35</span>, 613–626 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.shpsc.2004.09.003" title="DOI Link to resource 10.1016/j.shpsc.2004.09.003">CrossRef</a>.</span></li>
      <li><span id="cit45">
          C. Saunders, A. Gammerman and V. Vovk, Ridge regression learning algorithm in dual variables, <span class="italic">International Conference on Machine Learning</span>,  1998, pp. 515–521 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Machine%20Learning%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201998%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit46">
          C. E. Rasmussen, in <span class="italic">Advanced Lectures on Machine Learning: ML Summer Schools 2003</span>, Canberra, Australia, February 2–14, 2003, Tübingen, Germany, August 4–16, 2003, Revised Lectures, ed. O. Bousquet, U. von Luxburg and G. Rätsch, Springer Berlin Heidelberg, Berlin, Heidelberg,  2004, pp. 63–71, <small> DOI:<a class="DOILink" href="https://doi.org/10.1007/978-3-540-28650-9_4" TARGET="_BLANK" title="DOI Link to 10.1007/978-3-540-28650-9_4">10.1007/978-3-540-28650-9_4</a></small>.</span></li>
      <li><span id="cit47">Y. LeCun, Y. Bengio and G. Hinton, <span class="italic">Nature</span>, 2015, <span class="bold">521</span>, 436 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature14539" title="DOI Link to resource 10.1038/nature14539">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXht1WlurzP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26017442%5Buid%5D" title="PubMed Link to resource 26017442">PubMed</a>.</span></li>
      <li><span id="cit48">
          P. Sabatier, <span class="italic">La catalyse en chimie organique</span>, Librairie polytechnique, Paris et Liege,  1920 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=La%20catalyse%20en%20chimie%20organique%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201920%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit49">A. J. Medford, A. Vojvodic, J. S. Hummelshøj, J. Voss, F. Abild-Pedersen, F. Studt, T. Bligaard, A. Nilsson and J. K. Nørskov, <span class="italic">J. Catal.</span>, 2015, <span class="bold">328</span>, 36–42 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jcat.2014.12.033" title="DOI Link to resource 10.1016/j.jcat.2014.12.033">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtVeju7bM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit50">F. S. Roberts, K. P. Kuhl and A. Nilsson, <span class="italic">Angew. Chem., Int. Ed.</span>, 2015, <span class="bold">54</span>, 5179–5182 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201412214" title="DOI Link to resource 10.1002/anie.201412214">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2msb8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25728325%5Buid%5D" title="PubMed Link to resource 25728325">PubMed</a>.</span></li>
      <li><span id="cit51">X. Liu, J. Xiao, H. Peng, X. Hong, K. Chan and J. K. Nørskov, <span class="italic">Nat. Commun.</span>, 2017, <span class="bold">8</span>, 15438 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/ncomms15438" title="DOI Link to resource 10.1038/ncomms15438">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXot1egsL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28530224%5Buid%5D" title="PubMed Link to resource 28530224">PubMed</a>.</span></li>
      <li><span id="cit52">X. Ma, Z. Li, L. E. K. Achenie and H. Xin, <span class="italic">J. Phys. Chem. Lett.</span>, 2015, <span class="bold">6</span>, 3528–3533 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.5b01660" title="DOI Link to resource 10.1021/acs.jpclett.5b01660">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtlymsLbO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26722718%5Buid%5D" title="PubMed Link to resource 26722718">PubMed</a>.</span></li>
      <li><span id="cit53">Z. Li, X. Ma and H. Xin, <span class="italic">Catal. Today</span>, 2017, <span class="bold">280</span>, 232–238 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cattod.2016.04.013" title="DOI Link to resource 10.1016/j.cattod.2016.04.013">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XnsVylsbs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit54">
          B. Hammer and J. K. Nørskov, in <span class="italic">Advances in Catalysis</span>, Academic Press,  2000, vol. 45, pp. 71–129 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Advances%20in%20Catalysis%5BJour%5D%20AND%20vol. 45%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202000%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit55">J. Noh, S. Back, J. Kim and Y. Jung, <span class="italic">Chem. Sci.</span>, 2018, <span class="bold">9</span>, 5152–5159 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7SC03422A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7SC03422A">RSC</a>.</span></li>
      <li><span id="cit56">K. Tran and Z. W. Ulissi, <span class="italic">Nat. Catal.</span>, 2018, <span class="bold">1</span>, 696–703 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41929-018-0142-1" title="DOI Link to resource 10.1038/s41929-018-0142-1">CrossRef</a>.</span></li>
      <li><span id="cit57">I. Takigawa, K.-i. Shimizu, K. Tsuda and S. Takakusagi, <span class="italic">RSC Adv.</span>, 2016, <span class="bold">6</span>, 52587–52595 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6RA04345C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6RA04345C">RSC</a>.</span></li>
      <li><span id="cit58">T. Toyao, K. Suzuki, S. Kikuchi, S. Takakusagi, K.-i. Shimizu and I. Takigawa, <span class="italic">J. Phys. Chem. C</span>, 2018, <span class="bold">122</span>, 8315–8326 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b12670" title="DOI Link to resource 10.1021/acs.jpcc.7b12670">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXlvVKkt78%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit59">T. Davran-Candan, M. E. Günay and R. Yıldırım, <span class="italic">J. Chem. Phys.</span>, 2010, <span class="bold">132</span>, 174113 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.3369007" title="DOI Link to resource 10.1063/1.3369007">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20459162%5Buid%5D" title="PubMed Link to resource 20459162">PubMed</a>.</span></li>
      <li><span id="cit60">R. Jinnouchi and R. Asahi, <span class="italic">J. Phys. Chem. Lett.</span>, 2017, <span class="bold">8</span>, 4279–4283 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.7b02010" title="DOI Link to resource 10.1021/acs.jpclett.7b02010">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtlyhsL3K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28837771%5Buid%5D" title="PubMed Link to resource 28837771">PubMed</a>.</span></li>
      <li><span id="cit61">R. Jinnouchi, H. Hirata and R. Asahi, <span class="italic">J. Phys. Chem. C</span>, 2017, <span class="bold">121</span>, 26397–26405 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b08686" title="DOI Link to resource 10.1021/acs.jpcc.7b08686">CrossRef</a>.</span></li>
      <li><span id="cit62">H. Yarveicy, M. M. Ghiasi and A. H. Mohammadi, <span class="italic">J. Mol. Liq.</span>, 2018, <span class="bold">255</span>, 375–383 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.molliq.2017.11.156" title="DOI Link to resource 10.1016/j.molliq.2017.11.156">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitlahu7g%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit63">Y. Liu, Z. U. Wang and H.-C. Zhou, <span class="italic">Greenhouse Gases: Sci. Technol.</span>, 2012, <span class="bold">2</span>, 239–259 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ghg.1296" title="DOI Link to resource 10.1002/ghg.1296">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhvVahsbvN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit64">M. Fernandez, N. R. Trefiak and T. K. Woo, <span class="italic">J. Phys. Chem. C</span>, 2013, <span class="bold">117</span>, 14095–14105 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp404287t" title="DOI Link to resource 10.1021/jp404287t">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXpsFGltLY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit65">M. Fernandez, T. K. Woo, C. E. Wilmer and R. Q. Snurr, <span class="italic">J. Phys. Chem. C</span>, 2013, <span class="bold">117</span>, 7681–7689 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp4006422" title="DOI Link to resource 10.1021/jp4006422">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXktlyltLo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit66">G. Borboudakis, T. Stergiannakos, M. Frysali, E. Klontzas, I. Tsamardinos and G. E. Froudakis, <span class="italic">npj Comput. Mater.</span>, 2017, <span class="bold">3</span>, 40 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-017-0045-8" title="DOI Link to resource 10.1038/s41524-017-0045-8">CrossRef</a>.</span></li>
      <li><span id="cit67">R. Anderson, J. Rodgers, E. Argueta, A. Biong and D. A. Gómez-Gualdrón, <span class="italic">Chem. Mater.</span>, 2018, <span class="bold">30</span>, 6325–6337 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.8b02257" title="DOI Link to resource 10.1021/acs.chemmater.8b02257">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOltrzI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit68">V. Etacheri, R. Marom, R. Elazari, G. Salitra and D. Aurbach, <span class="italic">Energy Environ. Sci.</span>, 2011, <span class="bold">4</span>, 3243–3262 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1EE01598B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1EE01598B">RSC</a>.</span></li>
      <li><span id="cit69">K. Sodeyama, Y. Igarashi, T. Nakayama, Y. Tateyama and M. Okada, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2018, <span class="bold">20</span>, 22585–22591 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7CP08280K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7CP08280K">RSC</a>.</span></li>
      <li><span id="cit70">D. Aurbach, E. Pollak, R. Elazari, G. Salitra, C. S. Kelley and J. Affinito, <span class="italic">J. Electrochem. Soc.</span>, 2009, <span class="bold">156</span>, A694–A702 <a target="_blank" class="DOILink" href="https://doi.org/10.1149/1.3148721" title="DOI Link to resource 10.1149/1.3148721">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXosV2lsb8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit71">Z. Lin, Z. Liu, W. Fu, N. J. Dudney and C. Liang, <span class="italic">Adv. Funct. Mater.</span>, 2013, <span class="bold">23</span>, 1064–1069 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201200696" title="DOI Link to resource 10.1002/adfm.201200696">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xos1enu7g%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit72">Y. Okamoto and Y. Kubo, <span class="italic">ACS Omega</span>, 2018, <span class="bold">3</span>, 7868–7874 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsomega.8b00576" title="DOI Link to resource 10.1021/acsomega.8b00576">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlSqtb3L" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit73">R. Jalem, T. Aoyama, M. Nakayama and M. Nogami, <span class="italic">Chem. Mater.</span>, 2012, <span class="bold">24</span>, 1357–1364 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/cm3000427" title="DOI Link to resource 10.1021/cm3000427">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XjslKmt74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit74">R. Jalem, M. Nakayama and T. Kasuga, <span class="italic">J. Mater. Chem. A</span>, 2014, <span class="bold">2</span>, 720–734 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3TA13235H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3TA13235H">RSC</a>.</span></li>
      <li><span id="cit75">R. Jalem, M. Kimura, M. Nakayama and T. Kasuga, <span class="italic">J. Chem. Inf. Model.</span>, 2015, <span class="bold">55</span>, 1158–1168 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci500752n" title="DOI Link to resource 10.1021/ci500752n">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXovVSgt70%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26000780%5Buid%5D" title="PubMed Link to resource 26000780">PubMed</a>.</span></li>
      <li><span id="cit76">C. Chen, Z. Lu and F. Ciucci, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 40769 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep40769" title="DOI Link to resource 10.1038/srep40769">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohurc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28094317%5Buid%5D" title="PubMed Link to resource 28094317">PubMed</a>.</span></li>
      <li><span id="cit77">K. Fujimura, A. Seko, Y. Koyama, A. Kuwabara, I. Kishida, K. Shitara, C. A. J. Fisher, H. Moriwake and I. Tanaka, <span class="italic">Adv. Energy
Mater.</span>, 2013, <span class="bold">3</span>, 980–985 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201300060" title="DOI Link to resource 10.1002/aenm.201300060">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWhsbzM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit78">N. Kireeva and V. S. Pervov, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2017, <span class="bold">19</span>, 20904–20918 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7CP00518K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7CP00518K">RSC</a>.</span></li>
      <li><span id="cit79">R. Jalem, K. Kanamori, I. Takeuchi, M. Nakayama, H. Yamasaki and T. Saito, <span class="italic">Sci. Rep.</span>, 2018, <span class="bold">8</span>, 5845 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-018-23852-y" title="DOI Link to resource 10.1038/s41598-018-23852-y">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29643423%5Buid%5D" title="PubMed Link to resource 29643423">PubMed</a>.</span></li>
      <li><span id="cit80">M. Attarian Shandiz and R. Gauvin, <span class="italic">Comput. Mater. Sci.</span>, 2016, <span class="bold">117</span>, 270–278 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2016.02.021" title="DOI Link to resource 10.1016/j.commatsci.2016.02.021">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XivVKhsLY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit81">X. Wang, R. Xiao, H. Li and L. Chen, <span class="italic">J. Materiomics</span>, 2017, <span class="bold">3</span>, 178–183 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmat.2017.02.002" title="DOI Link to resource 10.1016/j.jmat.2017.02.002">CrossRef</a>.</span></li>
      <li><span id="cit82">R. A. Eremin, P. N. Zolotarev, O. Y. Ivanshina and I. A. Bobrikov, <span class="italic">J. Phys. Chem. C</span>, 2017, <span class="bold">121</span>, 28293–28305 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b09760" title="DOI Link to resource 10.1021/acs.jpcc.7b09760">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2mtrzL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit83">Y. Okamoto, <span class="italic">J. Phys. Chem. A</span>, 2017, <span class="bold">121</span>, 3299–3304 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpca.7b01629" title="DOI Link to resource 10.1021/acs.jpca.7b01629">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXmtFClurc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28418244%5Buid%5D" title="PubMed Link to resource 28418244">PubMed</a>.</span></li>
      <li><span id="cit84">V. Stanev, C. Oses, A. G. Kusne, E. Rodriguez, J. Paglione, S. Curtarolo and I. Takeuchi, <span class="italic">npj Comput. Mater.</span>, 2018, <span class="bold">4</span>, 29 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-018-0085-8" title="DOI Link to resource 10.1038/s41524-018-0085-8">CrossRef</a>.</span></li>
      <li><span id="cit85">S. Kim, A. Jinich and A. Aspuru-Guzik, <span class="italic">J. Chem. Inf. Model.</span>, 2017, <span class="bold">57</span>, 657–668 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.6b00332" title="DOI Link to resource 10.1021/acs.jcim.6b00332">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXks12hsL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28328209%5Buid%5D" title="PubMed Link to resource 28328209">PubMed</a>.</span></li>
      <li><span id="cit86">W. Shockley and H. J. Queisser, <span class="italic">J. Appl. Phys.</span>, 1961, <span class="bold">32</span>, 510–519 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.1736034" title="DOI Link to resource 10.1063/1.1736034">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaF3MXpslGqsQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit87">M. A. Green, A. Ho-Baillie and H. J. Snaith, <span class="italic">Nat. Photonics</span>, 2014, <span class="bold">8</span>, 506 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nphoton.2014.134" title="DOI Link to resource 10.1038/nphoton.2014.134">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhtVGqu7zN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit88">O. Allam, C. Holmes, Z. Greenberg, K. C. Kim and S. S. Jang, <span class="italic">ChemPhysChem</span>, 2018, <span class="bold">19</span>, 2559–2565 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cphc.201800382" title="DOI Link to resource 10.1002/cphc.201800382">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1Kqu73F" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29928788%5Buid%5D" title="PubMed Link to resource 29928788">PubMed</a>.</span></li>
      <li><span id="cit89">G. Pilania, A. Mannodi-Kanakkithodi, B. P. Uberuaga, R. Ramprasad, J. E. Gubernatis and T. Lookman, <span class="italic">Sci. Rep.</span>, 2016, <span class="bold">6</span>, 19375 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep19375" title="DOI Link to resource 10.1038/srep19375">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1elt74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26783247%5Buid%5D" title="PubMed Link to resource 26783247">PubMed</a>.</span></li>
      <li><span id="cit90">G. Pilania, J. E. Gubernatis and T. Lookman, <span class="italic">Comput. Mater. Sci.</span>, 2017, <span class="bold">129</span>, 156–163 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2016.12.004" title="DOI Link to resource 10.1016/j.commatsci.2016.12.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XitVGmsbvN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit91">K. Takahashi, L. Takahashi, I. Miyazato and Y. Tanaka, <span class="italic">ACS Photonics</span>, 2018, <span class="bold">5</span>, 771–775 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsphotonics.7b01479" title="DOI Link to resource 10.1021/acsphotonics.7b01479">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjs1ajtg%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit92">P. Dey, J. Bible, S. Datta, S. Broderick, J. Jasinski, M. Sunkara, M. Menon and K. Rajan, <span class="italic">Comput. Mater. Sci.</span>, 2014, <span class="bold">83</span>, 185–195 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2013.10.016" title="DOI Link to resource 10.1016/j.commatsci.2013.10.016">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXlvVahtg%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit93">L. Ward, A. Agrawal, A. Choudhary and C. Wolverton, <span class="italic">npj Comput. Mater.</span>, 2016, <span class="bold">2</span>, 16028 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/npjcompumats.2016.28" title="DOI Link to resource 10.1038/npjcompumats.2016.28">CrossRef</a>.</span></li>
      <li><span id="cit94">S. A. Lopez, B. Sanchez-Lengeling, J. de Goes Soares and A. Aspuru-Guzik, <span class="italic">Joule</span>, 2017, <span class="bold">1</span>, 857–870 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2017.10.006" title="DOI Link to resource 10.1016/j.joule.2017.10.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpvFGhu7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit95">W. Li, R. Jacobs and D. Morgan, <span class="italic">Comput. Mater. Sci.</span>, 2018, <span class="bold">150</span>, 454–463 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2018.04.033" title="DOI Link to resource 10.1016/j.commatsci.2018.04.033">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXos1Sns7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit96">J. Schmidt, J. Shi, P. Borlido, L. Chen, S. Botti and M. A. L. Marques, <span class="italic">Chem. Mater.</span>, 2017, <span class="bold">29</span>, 5090–5103 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.7b00156" title="DOI Link to resource 10.1021/acs.chemmater.7b00156">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXns1Ogu7k%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit97">Z. W. Ulissi, A. R. Singh, C. Tsai and J. K. Nørskov, <span class="italic">J. Phys. Chem. Lett.</span>, 2016, <span class="bold">7</span>, 3931–3935 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.6b01254" title="DOI Link to resource 10.1021/acs.jpclett.6b01254">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsVSgtbrP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27558978%5Buid%5D" title="PubMed Link to resource 27558978">PubMed</a>.</span></li>
      <li><span id="cit98">Z. W. Ulissi, A. J. Medford, T. Bligaard and J. K. Nørskov, <span class="italic">Nat. Commun.</span>, 2017, <span class="bold">8</span>, 14621 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/ncomms14621" title="DOI Link to resource 10.1038/ncomms14621">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28262694%5Buid%5D" title="PubMed Link to resource 28262694">PubMed</a>.</span></li>
      <li><span id="cit99">Z. Li, N. Omidvar, W. S. Chin, E. Robb, A. Morris, L. Achenie and H. Xin, <span class="italic">J. Phys. Chem. A</span>, 2018, <span class="bold">122</span>, 4571–4578 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpca.8b02842" title="DOI Link to resource 10.1021/acs.jpca.8b02842">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXot1Citbg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29688014%5Buid%5D" title="PubMed Link to resource 29688014">PubMed</a>.</span></li>
      <li><span id="cit100">J. E. Sutton and D. G. Vlachos, <span class="italic">Chem. Eng. Sci.</span>, 2015, <span class="bold">121</span>, 190–199 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ces.2014.09.011" title="DOI Link to resource 10.1016/j.ces.2014.09.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhsFyksrzL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit101">G. H. Gu, P. Plechac and D. G. Vlachos, <span class="italic">React. Chem. Eng.</span>, 2018, <span class="bold">3</span>, 454–466 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7RE00210F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7RE00210F">RSC</a>.</span></li>
      <li><span id="cit102">G. H. Gu and D. G. Vlachos, <span class="italic">J. Phys. Chem. C</span>, 2016, <span class="bold">120</span>, 19234–19241 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.6b06430" title="DOI Link to resource 10.1021/acs.jpcc.6b06430">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht12jsL3P" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit103">G. H. Gu, B. Schweitzer, C. Michel, S. N. Steinmann, P. Sautet and D. G. Vlachos, <span class="italic">J. Phys. Chem. C</span>, 2017, <span class="bold">121</span>, 21510–21519 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b07340" title="DOI Link to resource 10.1021/acs.jpcc.7b07340">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKhtbzI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit104">K. Han, A. Jamal, C. A. Grambow, Z. J. Buras and W. H. Green, <span class="italic">Int. J. Chem. Kinet.</span>, 2018, <span class="bold">50</span>, 294–303 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/kin.21158" title="DOI Link to resource 10.1002/kin.21158">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXit12rsrg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit105">L. B. Vilhelmsen and B. Hammer, <span class="italic">J. Chem. Phys.</span>, 2014, <span class="bold">141</span>, 044711 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.4886337" title="DOI Link to resource 10.1063/1.4886337">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25084941%5Buid%5D" title="PubMed Link to resource 25084941">PubMed</a>.</span></li>
      <li><span id="cit106">E. Burello, D. Farrusseng and G. Rothenberg, <span class="italic">Adv. Synth. Catal.</span>, 2004, <span class="bold">346</span>, 1844–1853 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adsc.200404170" title="DOI Link to resource 10.1002/adsc.200404170">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXhslOrug%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit107">J. P. Janet, T. Z. H. Gani, A. H. Steeves, E. I. Ioannidis and H. J. Kulik, <span class="italic">Ind. Eng. Chem. Res.</span>, 2017, <span class="bold">56</span>, 4898–4910 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.iecr.7b00808" title="DOI Link to resource 10.1021/acs.iecr.7b00808">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXls1Snsrg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit108">E. I. Ioannidis, T. Z. H. Gani and H. J. Kulik, <span class="italic">J. Comput. Chem.</span>, 2016, <span class="bold">37</span>, 2106–2117 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.24437" title="DOI Link to resource 10.1002/jcc.24437">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtV2gu7bN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27364957%5Buid%5D" title="PubMed Link to resource 27364957">PubMed</a>.</span></li>
      <li><span id="cit109">J. P. Janet and H. J. Kulik, <span class="italic">J. Phys. Chem. A</span>, 2017, <span class="bold">121</span>, 8939–8954 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpca.7b08750" title="DOI Link to resource 10.1021/acs.jpca.7b08750">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhslGhsLnL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29095620%5Buid%5D" title="PubMed Link to resource 29095620">PubMed</a>.</span></li>
      <li><span id="cit110">J. P. Janet, L. Chan and H. J. Kulik, <span class="italic">J. Phys. Chem. Lett.</span>, 2018, <span class="bold">9</span>, 1064–1071 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.8b00170" title="DOI Link to resource 10.1021/acs.jpclett.8b00170">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisFarsr4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29425453%5Buid%5D" title="PubMed Link to resource 29425453">PubMed</a>.</span></li>
      <li><span id="cit111">A. Seko, H. Hayashi, K. Nakayama, A. Takahashi and I. Tanaka, <span class="italic">Phys. Rev. B</span>, 2017, <span class="bold">95</span>, 144110 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.95.144110" title="DOI Link to resource 10.1103/PhysRevB.95.144110">CrossRef</a>.</span></li>
      <li><span id="cit112">F. A. Faber, A. Lindmaa, O. A. von Lilienfeld and R. Armiento, <span class="italic">Phys. Rev. Lett.</span>, 2016, <span class="bold">117</span>, 135502 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevLett.117.135502" title="DOI Link to resource 10.1103/PhysRevLett.117.135502">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27715098%5Buid%5D" title="PubMed Link to resource 27715098">PubMed</a>.</span></li>
      <li><span id="cit113">
          F. Legrain, A. v. Roekeghem, S. Curtarolo, J. Carrete, G. K. H. Madsen and N. Mingo, arXiv:1803.09827,  2018.</span></li>
      <li><span id="cit114">G. Pilania and X. Y. Liu, <span class="italic">J. Mater. Sci.</span>, 2018, <span class="bold">53</span>, 6652–6664 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10853-018-1987-z" title="DOI Link to resource 10.1007/s10853-018-1987-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXotFahtQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit115">M. Chen and D. A. Dixon, <span class="italic">J. Phys. Chem. C</span>, 2018, <span class="bold">122</span>, 18621–18639 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.8b01667" title="DOI Link to resource 10.1021/acs.jpcc.8b01667">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGit7vN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit116">M. Salciccioli, M. Stamatakis, S. Caratzoulas and D. G. Vlachos, <span class="italic">Chem. Eng. Sci.</span>, 2011, <span class="bold">66</span>, 4319–4355 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ces.2011.05.050" title="DOI Link to resource 10.1016/j.ces.2011.05.050">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhtVagsLfO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit117">A. Van der Ven, G. Ceder, M. Asta and P. D. Tepesch, <span class="italic">Phys. Rev. B: Condens. Matter Mater. Phys.</span>, 2001, <span class="bold">64</span>, 184307 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.64.184307" title="DOI Link to resource 10.1103/PhysRevB.64.184307">CrossRef</a>.</span></li>
      <li><span id="cit118">S. Curtarolo, G. L. W. Hart, M. B. Nardelli, N. Mingo, S. Sanvito and O. Levy, <span class="italic">Nat. Mater.</span>, 2013, <span class="bold">12</span>, 191 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat3568" title="DOI Link to resource 10.1038/nmat3568">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXislWju7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23422720%5Buid%5D" title="PubMed Link to resource 23422720">PubMed</a>.</span></li>
      <li><span id="cit119">J. Behler, <span class="italic">Angew. Chem., Int. Ed.</span>, 2017, <span class="bold">56</span>, 12828–12840 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201703114" title="DOI Link to resource 10.1002/anie.201703114">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKhurnK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28520235%5Buid%5D" title="PubMed Link to resource 28520235">PubMed</a>.</span></li>
      <li><span id="cit120">M. Gastegger, C. Kauffmann, J. Behler and P. Marquetand, <span class="italic">J. Chem. Phys.</span>, 2016, <span class="bold">144</span>, 194110 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.4950815" title="DOI Link to resource 10.1063/1.4950815">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27208939%5Buid%5D" title="PubMed Link to resource 27208939">PubMed</a>.</span></li>
      <li><span id="cit121">J. R. Boes, M. C. Groenenboom, J. A. Keith and J. R. Kitchin, <span class="italic">Int. J. Quantum Chem.</span>, 2016, <span class="bold">116</span>, 979–987 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/qua.25115" title="DOI Link to resource 10.1002/qua.25115">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XjtlOnsr8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit122">N. Artrith, T. Morawietz and J. Behler, <span class="italic">Phys. Rev. B: Condens. Matter Mater. Phys.</span>, 2011, <span class="bold">83</span>, 153101 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.83.153101" title="DOI Link to resource 10.1103/PhysRevB.83.153101">CrossRef</a>.</span></li>
      <li><span id="cit123">N. Artrith and J. Behler, <span class="italic">Phys. Rev. B: Condens. Matter Mater. Phys.</span>, 2012, <span class="bold">85</span>, 045439 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.85.045439" title="DOI Link to resource 10.1103/PhysRevB.85.045439">CrossRef</a>.</span></li>
      <li><span id="cit124">N. Artrith and A. Urban, <span class="italic">Comput. Mater. Sci.</span>, 2016, <span class="bold">114</span>, 135–150 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2015.11.047" title="DOI Link to resource 10.1016/j.commatsci.2015.11.047">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtFGisA%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit125">C. Wang, A. Tharval and J. R. Kitchin, <span class="italic">Mol. Simul.</span>, 2018, <span class="bold">44</span>, 623–630 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/08927022.2017.1420185" title="DOI Link to resource 10.1080/08927022.2017.1420185">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXkvFKitA%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit126">T. Gao and J. R. Kitchin, <span class="italic">Catal. Today</span>, 2018, <span class="bold">312</span>, 132–140 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cattod.2018.03.045" title="DOI Link to resource 10.1016/j.cattod.2018.03.045">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXntV2ntb4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit127">R. Ouyang, Y. Xie and D.-e. Jiang, <span class="italic">Nanoscale</span>, 2015, <span class="bold">7</span>, 14817–14821 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5NR03903G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5NR03903G">RSC</a>.</span></li>
      <li><span id="cit128">H. Zhai and A. N. Alexandrova, <span class="italic">J. Chem. Theory Comput.</span>, 2016, <span class="bold">12</span>, 6213–6226 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jctc.6b00994" title="DOI Link to resource 10.1021/acs.jctc.6b00994">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhvVWnurrL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27951667%5Buid%5D" title="PubMed Link to resource 27951667">PubMed</a>.</span></li>
      <li><span id="cit129">N. Artrith and A. M. Kolpak, <span class="italic">Comput. Mater. Sci.</span>, 2015, <span class="bold">110</span>, 20–28 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2015.07.046" title="DOI Link to resource 10.1016/j.commatsci.2015.07.046">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXht12mtrzE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit130">N. Artrith and A. M. Kolpak, <span class="italic">Nano Lett.</span>, 2014, <span class="bold">14</span>, 2670–2676 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/nl5005674" title="DOI Link to resource 10.1021/nl5005674">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXmsVGmu78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24742028%5Buid%5D" title="PubMed Link to resource 24742028">PubMed</a>.</span></li>
      <li><span id="cit131">S. K. Natarajan and J. Behler, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2016, <span class="bold">18</span>, 28704–28725 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6CP05711J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6CP05711J">RSC</a>.</span></li>
      <li><span id="cit132">K. Shakouri, J. Behler, J. Meyer and G.-J. Kroes, <span class="italic">J. Phys. Chem. Lett.</span>, 2017, <span class="bold">8</span>, 2131–2136 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.7b00784" title="DOI Link to resource 10.1021/acs.jpclett.7b00784">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXms1Wnsbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28441867%5Buid%5D" title="PubMed Link to resource 28441867">PubMed</a>.</span></li>
      <li><span id="cit133">J. R. Boes and J. R. Kitchin, <span class="italic">Mol. Simul.</span>, 2017, <span class="bold">43</span>, 346–354 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/08927022.2016.1274984" title="DOI Link to resource 10.1080/08927022.2016.1274984">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXitVKjur4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit134">Z. W. Ulissi, M. T. Tang, J. Xiao, X. Liu, D. A. Torelli, M. Karamad, K. Cummins, C. Hahn, N. S. Lewis, T. F. Jaramillo, K. Chan and J. K. Nørskov, <span class="italic">ACS Catal.</span>, 2017, <span class="bold">7</span>, 6600–6608 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscatal.7b01648" title="DOI Link to resource 10.1021/acscatal.7b01648">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXht1elu7vF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit135">A. Ostadhossein, E. D. Cubuk, G. A. Tritsaris, E. Kaxiras, S. Zhang and A. C. T. van Duin, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2015, <span class="bold">17</span>, 3832–3840 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4CP05198J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4CP05198J">RSC</a>.</span></li>
      <li><span id="cit136">N. Artrith, A. Urban and G. Ceder, <span class="italic">J. Chem. Phys.</span>, 2018, <span class="bold">148</span>, 241711 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5017661" title="DOI Link to resource 10.1063/1.5017661">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29960321%5Buid%5D" title="PubMed Link to resource 29960321">PubMed</a>.</span></li>
      <li><span id="cit137">B. Onat, E. D. Cubuk, B. D. Malone and E. Kaxiras, <span class="italic">Phys. Rev. B</span>, 2018, <span class="bold">97</span>, 094106 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.97.094106" title="DOI Link to resource 10.1103/PhysRevB.97.094106">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXltVGhsLo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit138">S. Fujikake, V. L. Deringer, T. H. Lee, M. Krynski, S. R. Elliott and G. Csányi, <span class="italic">J. Chem. Phys.</span>, 2018, <span class="bold">148</span>, 241714 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5016317" title="DOI Link to resource 10.1063/1.5016317">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29960342%5Buid%5D" title="PubMed Link to resource 29960342">PubMed</a>.</span></li>
      <li><span id="cit139">Q. Yan, J. Yu, S. K. Suram, L. Zhou, A. Shinde, P. F. Newhouse, W. Chen, G. Li, K. A. Persson, J. M. Gregoire and J. B. Neaton, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2017, <span class="bold">114</span>, 3040–3043 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.1619940114" title="DOI Link to resource 10.1073/pnas.1619940114">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXjvVyjsbw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28265095%5Buid%5D" title="PubMed Link to resource 28265095">PubMed</a>.</span></li>
      <li><span id="cit140">A. Ziletti, D. Kumar, M. Scheffler and L. M. Ghiringhelli, <span class="italic">Nat. Commun.</span>, 2018, <span class="bold">9</span>, 2775 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-018-05169-6" title="DOI Link to resource 10.1038/s41467-018-05169-6">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30018362%5Buid%5D" title="PubMed Link to resource 30018362">PubMed</a>.</span></li>
      <li><span id="cit141">W. B. Park, J. Chung, J. Jung, K. Sohn, S. P. Singh, M. Pyo, N. Shin and K.-S. Sohn, <span class="italic">IUCrJ</span>, 2017, <span class="bold">4</span>, 486–494 <a target="_blank" class="DOILink" href="https://doi.org/10.1107/S205225251700714X" title="DOI Link to resource 10.1107/S205225251700714X">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFegurbK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28875035%5Buid%5D" title="PubMed Link to resource 28875035">PubMed</a>.</span></li>
      <li><span id="cit142">A. O. Oliynyk, L. A. Adutwum, J. J. Harynuk and A. Mar, <span class="italic">Chem. Mater.</span>, 2016, <span class="bold">28</span>, 6672–6681 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.6b02905" title="DOI Link to resource 10.1021/acs.chemmater.6b02905">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtlOqurbE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit143">A. O. Oliynyk, L. A. Adutwum, B. W. Rudyk, H. Pisavadia, S. Lotfi, V. Hlukhyy, J. J. Harynuk, A. Mar and J. Brgoch, <span class="italic">J. Am. Chem. Soc.</span>, 2017, <span class="bold">139</span>, 17870–17881 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.7b08460" title="DOI Link to resource 10.1021/jacs.7b08460">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7bN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29129069%5Buid%5D" title="PubMed Link to resource 29129069">PubMed</a>.</span></li>
      <li><span id="cit144">C. C. Fischer, K. J. Tibbetts, D. Morgan and G. Ceder, <span class="italic">Nat. Mater.</span>, 2006, <span class="bold">5</span>, 641 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat1691" title="DOI Link to resource 10.1038/nmat1691">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XnsFeitbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16845417%5Buid%5D" title="PubMed Link to resource 16845417">PubMed</a>.</span></li>
      <li><span id="cit145">W. Sun, A. Holder, B. Orvañanos, E. Arca, A. Zakutayev, S. Lany and G. Ceder, <span class="italic">Chem. Mater.</span>, 2017, <span class="bold">29</span>, 6936–6946 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.7b02399" title="DOI Link to resource 10.1021/acs.chemmater.7b02399">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFyqsb7P" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit146">G. Hautier, C. C. Fischer, A. Jain, T. Mueller and G. Ceder, <span class="italic">Chem. Mater.</span>, 2010, <span class="bold">22</span>, 3762–3767 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/cm100795d" title="DOI Link to resource 10.1021/cm100795d">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXms1Siur4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit147">K. Ryan, J. Lengyel and M. Shatruk, <span class="italic">J. Am. Chem. Soc.</span>, 2018, <span class="bold">140</span>, 10158–10168 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.8b03913" title="DOI Link to resource 10.1021/jacs.8b03913">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29874459%5Buid%5D" title="PubMed Link to resource 29874459">PubMed</a>.</span></li>
      <li><span id="cit148">Y. Wang, J. Lv, L. Zhu and Y. Ma, <span class="italic">Comput. Phys. Commun.</span>, 2012, <span class="bold">183</span>, 2063–2070 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cpc.2012.05.008" title="DOI Link to resource 10.1016/j.cpc.2012.05.008">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XnvFGis7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit149">D. C. Lonie and E. Zurek, <span class="italic">Comput. Phys. Commun.</span>, 2011, <span class="bold">182</span>, 372–387 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cpc.2010.07.048" title="DOI Link to resource 10.1016/j.cpc.2010.07.048">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWqsL%252FN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit150">D. C. Lonie and E. Zurek, <span class="italic">Comput. Phys. Commun.</span>, 2011, <span class="bold">182</span>, 2305–2306 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cpc.2011.06.003" title="DOI Link to resource 10.1016/j.cpc.2011.06.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXos1eisbc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit151">Y. Zhou, H. Wang, C. Zhu, H. Liu, J. S. Tse and Y. Ma, <span class="italic">Inorg. Chem.</span>, 2016, <span class="bold">55</span>, 7026–7032 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.6b00881" title="DOI Link to resource 10.1021/acs.inorgchem.6b00881">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtVertr7P" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27341197%5Buid%5D" title="PubMed Link to resource 27341197">PubMed</a>.</span></li>
      <li><span id="cit152">Y. Zhao, X. Li, J. Liu, C. Zhang and Q. Wang, <span class="italic">J. Phys. Chem. Lett.</span>, 2018, <span class="bold">9</span>, 1815–1820 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.8b00616" title="DOI Link to resource 10.1021/acs.jpclett.8b00616">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXlvVKku7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29575891%5Buid%5D" title="PubMed Link to resource 29575891">PubMed</a>.</span></li>
      <li><span id="cit153">S. Wang, J. Liu and Q. Sun, <span class="italic">J. Mater. Chem. A</span>, 2017, <span class="bold">5</span>, 16936–16943 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7TA04941B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7TA04941B">RSC</a>.</span></li>
      <li><span id="cit154">A. Shamp, T. Terpstra, T. Bi, Z. Falls, P. Avery and E. Zurek, <span class="italic">J. Am. Chem. Soc.</span>, 2016, <span class="bold">138</span>, 1884–1892 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.5b10180" title="DOI Link to resource 10.1021/jacs.5b10180">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XptlGisw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26777416%5Buid%5D" title="PubMed Link to resource 26777416">PubMed</a>.</span></li>
      <li><span id="cit155">A. O. Oliynyk, E. Antono, T. D. Sparks, L. Ghadbeigi, M. W. Gaultois, B. Meredig and A. Mar, <span class="italic">Chem. Mater.</span>, 2016, <span class="bold">28</span>, 7324–7331 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.6b02724" title="DOI Link to resource 10.1021/acs.chemmater.6b02724">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFamsrrN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit156">F. Legrain, J. Carrete, A. van Roekeghem, G. K. H. Madsen and N. Mingo, <span class="italic">J. Phys. Chem. B</span>, 2018, <span class="bold">122</span>, 625–632 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcb.7b05296" title="DOI Link to resource 10.1021/acs.jpcb.7b05296">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28742351%5Buid%5D" title="PubMed Link to resource 28742351">PubMed</a>.</span></li>
      <li><span id="cit157">P. V. Balachandran, A. A. Emery, J. E. Gubernatis, T. Lookman, C. Wolverton and A. Zunger, <span class="italic">Phys. Rev. Mater.</span>, 2018, <span class="bold">2</span>, 043802 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevMaterials.2.043802" title="DOI Link to resource 10.1103/PhysRevMaterials.2.043802">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXltV2qsr4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit158">W. Ma, F. Cheng and Y. Liu, <span class="italic">ACS Nano</span>, 2018, <span class="bold">12</span>, 6326–6334 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.8b03569" title="DOI Link to resource 10.1021/acsnano.8b03569">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtVKgtbfF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29856595%5Buid%5D" title="PubMed Link to resource 29856595">PubMed</a>.</span></li>
      <li><span id="cit159">J. Peurifoy, Y. Shen, L. Jing, Y. Yang, F. Cano-Renteria, B. G. DeLacy, J. D. Joannopoulos, M. Tegmark and M. Soljačić, <span class="italic">Sci. Adv.</span>, 2018, <span class="bold">4</span>, eaar4206 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/sciadv.aar4206" title="DOI Link to resource 10.1126/sciadv.aar4206">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29868640%5Buid%5D" title="PubMed Link to resource 29868640">PubMed</a>.</span></li>
      <li><span id="cit160">R. Yuan, Z. Liu, P. V. Balachandran, D. Xue, Y. Zhou, X. Ding, J. Sun, D. Xue and T. Lookman, <span class="italic">Adv. Mater.</span>, 2018, <span class="bold">30</span>, 1702884 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201702884" title="DOI Link to resource 10.1002/adma.201702884">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29315814%5Buid%5D" title="PubMed Link to resource 29315814">PubMed</a>.</span></li>
      <li><span id="cit161">R. Gómez-Bombarelli, J. Aguilera-Iparraguirre, T. D. Hirzel, D. Duvenaud, D. Maclaurin, M. A. Blood-Forsythe, H. S. Chae, M. Einzinger, D.-G. Ha, T. Wu, G. Markopoulos, S. Jeon, H. Kang, H. Miyazaki, M. Numata, S. Kim, W. Huang, S. I. Hong, M. Baldo, R. P. Adams and A. Aspuru-Guzik, <span class="italic">Nat. Mater.</span>, 2016, <span class="bold">15</span>, 1120 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat4717" title="DOI Link to resource 10.1038/nmat4717">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27500805%5Buid%5D" title="PubMed Link to resource 27500805">PubMed</a>.</span></li>
      <li><span id="cit162">Y. Zhuo, A. Mansouri Tehrani, A. O. Oliynyk, A. C. Duke and J. Brgoch, <span class="italic">Nat. Commun.</span>, 2018, <span class="bold">9</span>, 4377 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-018-06625-z" title="DOI Link to resource 10.1038/s41467-018-06625-z">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30348949%5Buid%5D" title="PubMed Link to resource 30348949">PubMed</a>.</span></li>
      <li><span id="cit163">A. M. Haregewoin, A. S. Wotango and B.-J. Hwang, <span class="italic">Energy Environ. Sci.</span>, 2016, <span class="bold">9</span>, 1955–1988 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6EE00123H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6EE00123H">RSC</a>.</span></li>
      <li><span id="cit164">M. S. Park, I. Park, Y.-S. Kang, D. Im and S.-G. Doo, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2016, <span class="bold">18</span>, 26807–26815 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6CP05800K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6CP05800K">RSC</a>.</span></li>
      <li><span id="cit165">S. Nagasawa, E. Al-Naamani and A. Saeki, <span class="italic">J. Phys. Chem. Lett.</span>, 2018, <span class="bold">9</span>, 2639–2646 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.8b00635" title="DOI Link to resource 10.1021/acs.jpclett.8b00635">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXovVyms74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29733216%5Buid%5D" title="PubMed Link to resource 29733216">PubMed</a>.</span></li>
      <li><span id="cit166">
          W. Sun, C. Bartel, E. Arca, S. Bauers, B. Matthews, B. Orvañanos, B.-R. Chen, M. F. Toney, L. T. Schelhas, W. Tumas, J. Tate, A. Zakutayev, S. Lany, A. Holder and G. Ceder, arXiv:1809.09202,  2018.</span></li>
      <li><span id="cit167">A. D. Sendek, Q. Yang, E. D. Cubuk, K. N. Duerloo, Y. Cui and E. J. Reed, <span class="italic">Energy. Environ. Sci.</span>, 2017, <span class="bold">10</span>, 306–320 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6EE02697D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6EE02697D">RSC</a>.</span></li>
      <li><span id="cit168">P. Ertl and A. Schuffenhauer, <span class="italic">J. Cheminf.</span>, 2009, <span class="bold">1</span>, 8 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%201%5Bvolume%5D%20AND%208%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit169">P. Raccuglia, K. C. Elbert, P. D. F. Adler, C. Falk, M. B. Wenny, A. Mollo, M. Zeller, S. A. Friedler, J. Schrier and A. J. Norquist, <span class="italic">Nature</span>, 2016, <span class="bold">533</span>, 73 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature17439" title="DOI Link to resource 10.1038/nature17439">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XntlKhtL0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27147027%5Buid%5D" title="PubMed Link to resource 27147027">PubMed</a>.</span></li>
      <li><span id="cit170">J. S. Hummelshøj, F. Abild-Pedersen, F. Studt, T. Bligaard and J. K. Nørskov, <span class="italic">Angew. Chem., Int. Ed.</span>, 2012, <span class="bold">51</span>, 272–274 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201107947" title="DOI Link to resource 10.1002/anie.201107947">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22162297%5Buid%5D" title="PubMed Link to resource 22162297">PubMed</a>.</span></li>
      <li><span id="cit171">H. S. Stein, D. Guevarra, P. F. Newhouse, E. Soedarmadji and J. M. Gregoire, <span class="italic">Chem. Sci.</span>, 2019, <span class="bold">10</span>, 47–55 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8SC03077D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8SC03077D">RSC</a>.</span></li>
      <li><span id="cit172">M. Umehara, H. S. Stein, D. Guevarra, P. F. Newhouse, D. A. Boyd and J. M. Gregoire, <span class="italic">npj Comput. Mater.</span>, 2019, <span class="bold">5</span>, 34 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-019-0172-5" title="DOI Link to resource 10.1038/s41524-019-0172-5">CrossRef</a>.</span></li>
      <li><span id="cit173">L. M. Roch, F. Häse, C. Kreisbeck, T. Tamayo-Mendoza, L. P. E. Yunker, J. E. Hein and A. Aspuru-Guzik, <span class="italic">Science Robotics</span>, 2018, <span class="bold">3</span>, eaat5559 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/scirobotics.aat5559" title="DOI Link to resource 10.1126/scirobotics.aat5559">CrossRef</a>.</span></li>
      <li><span id="cit174">F. Häse, L. M. Roch, C. Kreisbeck and A. Aspuru-Guzik, <span class="italic">ACS Cent. Sci.</span>, 2018, <span class="bold">4</span>, 1134–1145 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.8b00307" title="DOI Link to resource 10.1021/acscentsci.8b00307">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30276246%5Buid%5D" title="PubMed Link to resource 30276246">PubMed</a>.</span></li>
    </ol>
    
    
    
    
    
  <table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2019</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>