<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Computational protein design, from single domain soluble proteins to membrane  proteins  - Chemical Society Reviews (RSC Publishing) DOI:10.1039/B810924A</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2010/cs/b810924a"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":61,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Pu Tian"/><meta name="DC.title" content="Computational protein design, from single domain soluble proteins to membrane  proteins "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2010/05/25"/><meta name="DC.Identifier" scheme="doi" content="10.1039/B810924A"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Computational protein design, from single domain soluble proteins to membrane  proteins "/><meta name="citation_author" content="Pu Tian"/><meta name="citation_online_date" content="2010/04/20"/><meta name="citation_date" content="2010"/><meta name="citation_journal_title" content="Chemical Society Reviews"/><meta name="citation_volume" content="39"/><meta name="citation_issue" content="6"/><meta name="citation_firstpage" content="2071"/><meta name="citation_lastpage" content="2082"/><meta name="citation_doi" content="10.1039/B810924A"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2010/cs/b810924a"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2010/cs/b810924a"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2010/cs/b810924a"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/prospect.js?ver=6_2_1"> </script><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2010/cs/b810924a">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:b911431a)" href="/en/content/articlehtml/2010/cs/b911431a" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:b811349c)" href="/en/content/articlehtml/2010/cs/b811349c" target="_BLANK">Next Article</a></span></div><div class="right_head"> </div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/B810924A">10.1039/B810924A</a>
(Tutorial Review)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1460-4744/1972">Chem. Soc. Rev.</a></span>, 2010, <strong>39</strong>, 2071-2082<div id="chemnav"><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of compound names" onclick="toggleColour('TC');"><span alt="Show Compounds button" id="btnTC">Show Compounds</span></a><input id="inpTC" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of chem-ontology terms" onclick="toggleColour('CH');"><span alt="Show Chemical Terms button" id="btnCH">Show Chemical Terms</span></a><input id="inpCH" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of bio-ontology terms" onclick="toggleColour('AN');"><span alt="Show Biomedical Terms button" id="btnAN">Show Biomedical Terms</span></a><input id="inpAN" type="hidden" value="off"/></div></div><h1 id="sect157"><span class="title_heading">Computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design, from single domain soluble <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> to <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span></span></h1><p class="header_text">
      <span class="bold">
        
          
            Pu 
            Tian
          
          
        
      </span>
      <br/><span class="italic">College of Life Sciences &amp; Key Laboratory for Molecular Enzymology and Engineering of Ministry of Education, Jilin University, 2699 Qian-Jin, Street, Changchun, China 130012. E-mail: <a href="mailto:tianpu@jlu.edu.cn">tianpu@jlu.edu.cn</a></span>
    </p><div id="art-admin"><span class="italic bold">Received 
      15th October 2009
    </span></div><p class="bold italic">First published on 
      
      
      
      
      
      20th April 2010
    </p><hr/><div class="abstract"><h2>Abstract</h2><p>Computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design is becoming more and more important in both understanding the fundamental <span class="CH"><a class="simple" title="ChEBI Ontology link for: biological roles" href="javascript:popupOBO('CHEBI:24432','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24432')">biological roles</a></span> of <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and facilitating their utilization in biotechnology. Based upon the significant progress in our understanding of <span class="AN"><a class="simple" title="Gene Ontology link for: protein folding" href="javascript:popupOBO('GO:0006457','b810924a')">protein folding</a></span>, development of efficient sequence and conformational search algorithms and accurate scoring functions, and ever increasing computational resources, great progress has been made in design of single domain soluble <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. There are also reports of <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design, albeit very limited. This <span class="italic">tutorial review</span> will give a brief introduction, the state of the art, and future prospects and implications of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design in improving healthcare and environment. The interplay between <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design, molecular simulations and experiments will be presented to emphasize the interdisciplinary nature of the field.</p></div><hr/><table><tr><td class="biogPlate"><img alt="&#xA;                  Pu Tian&#xA;                " src="/image/article/2010/CS/b810924a/b810924a-p1.gif"/><p><span class="graphic_title">
                  <span class="bold">Pu Tian</span>
                </span></p></td><td><i><p>Pu Tian received his BS in Chemical Engineering from Sichuan University (Chengdu, China) in 1995 and PhD in Materials Science and Engineering from the University of Utah, in 2003. He joined the University of Michigan as a research fellow in Department of Chemistry in 2003 and moved to National Institute of Diabetes &amp; Digestive &amp; Kidney Diseases as a research fellow in 2005. He has been a member of faculty in College of Life Science at Jilin University (Changchun, China) since late 2009. His current research interests include molecular simulations and design of macrobiomolecules with emphasis on membrane proteins.</p></i></td></tr></table><hr/>
    
      
      <h2 id="sect201"><span class="a_heading">1. Introduction</span></h2>
      <span>Molecular biology research has been focusing on understanding the structure, dynamics, function and interactions of macrobiomolecules (<span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, <span class="AN"><a class="simple" title="Sequence Ontology link for: DNAs" href="javascript:popupOBO('SO:0000352','b810924a')">DNAs</a></span> and RNAs) of given sequences, with the theoretical aspects hinging on how <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> fold into their native structures. The inverse problem (see <a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>), to design biological molecules with desired structural and dynamic properties, is gaining more and more attention. The ability to design functional biomolecules will provide us diverse ways to realize controlled intervention of biological systems. Potential benefits include, but are not limited to, improved health care through innovative <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span> and more efficient disease diagnostics<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1,2</span></sup></a> and designed enzymes that can catalyze necessary chemical reactions under environmentally friendly conditions.<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> This review only touches a small subset of these issues, the design of <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>.</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><img alt="A schematic illustration of the inverse relationship between structure prediction (protein folding) and protein design. (A) A known sequence (indicated by different gray scale of the tube protein representation, color online) in a random coil state. (B) The backbone of a given target fold without any information on the sequence. (C) A folded protein with both structure and sequence (indicated by different gray scale of the carton representation, color online) determined." src="/image/article/2010/CS/b810924a/b810924a-f1.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">A schematic illustration of the inverse relationship between structure prediction (<span class="AN"><a class="simple" title="Gene Ontology link for: protein folding" href="javascript:popupOBO('GO:0006457','b810924a')">protein folding</a></span>) and <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. (A) A known sequence (indicated by different gray scale of the tube <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> representation, color online) in a random coil state. (B) The backbone of a given target fold without any information on the sequence. (C) A folded <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> with both structure and sequence (indicated by different gray scale of the carton representation, color online) determined.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Naturally occurring <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> have two common properties, (i) a stable structure (or a set of stable structural states) and proper dynamical behavior under physiological conditions (this property implies that a sufficiently large free energy gap of approximately 10–100 kcal mol<small><sup>−1</sup></small> exists between the native state ensemble and other molten globule states); and (ii) the ability to rapidly fold into their native structure either alone or with the help of some chaperons or interaction partners. As a matter of fact, most functional <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> molecules, with the exception of inherently disordered <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, do have these properties. Based upon the belief that dominant folding pathways of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> are those with transition states that maximize the formation of favorable native interactions and simultaneously minimize the entropy loss of the given <span class="CH"><a class="simple" title="ChEBI Ontology link for: polypeptide" href="javascript:popupOBO('CHEBI:15841','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=15841')">polypeptide</a></span> chain, computational design<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a> was successfully utilized to switch the preferred folding pathway of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> G through stabilization of a β hairpin that is not well structured in the original transition state. Folding kinetics of a few designed small globular <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> was found to be somewhat different from their natural counterparts.<a title="Select to navigate to references" href="#cit5"><sup><span class="sup_ref">5</span></sup></a> However, we have very limited knowledge on the relationship between sequence and dynamics, and between sequence and folding mechanisms of <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> with realistic size and structural complexity. Therefore, the majority of current rational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> designs mainly focus on finding thermodynamically stable sequences with desired structures and interaction specificity, thus leaving the analysis of dynamics and folding kinetics to posterior experimental characterizations and functional assays.</p>
      <p class="otherpara">Since the term “inverse folding problem” appeared in the early 1980s, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design community has made steady progress in design of many different <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. Such exercises have evolved from redesign of natural <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> by mutating a few key residues based upon simple motifs<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> to rational design of a novel fold<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a> (see <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>) which do not exist in the current <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> data bank (PDB), and to <span class="italic">de novo</span> design of a new <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b810924a')">enzyme</a></span><a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> which catalyzes chemical reactions that are not catalyzed by any known natural enzymes. Additionally, although most of the early studies were focusing on small soluble <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, efforts have been made toward the design of integral <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>.<a title="Select to navigate to references" href="#cit9"><sup><span class="sup_ref">9</span></sup></a> While still in its infancy when compared to mature engineering design of cars, planes and electronic circuits, <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design has started to show promises in helping solving two of the most pressing issues facing our society, namely the health care and the energy crisis. On the disease treatment side, <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> (<span class="AN"><a class="simple" title="Gene Ontology link for: antibody" href="javascript:popupOBO('GO:0042571','b810924a')">antibody</a></span>) based <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span> are now important new <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span> that are on the market and in the pipeline, and designing <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptides" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptides</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> which can interact correctly with their targeted biomolecules (usually <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> or <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span>) is starting to make important contributions in this regard.<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> On the disease diagnostics side, sensitive <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> based biosensors are designed to improve the <span class="AN"><a class="simple" title="Gene Ontology link for: detection of" href="javascript:popupOBO('GO:0014865','b810924a')">detection of</a></span> specific <span class="CH"><a class="simple" title="ChEBI Ontology link for: biomarkers" href="javascript:popupOBO('CHEBI:59163','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=59163')">biomarkers</a></span>.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> The conversion of biomass to easily transportable <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span>, together with other clean energy sources (solar, wind and nuclear), will greatly reduce our <span class="CH"><a class="simple" title="ChEBI Ontology link for: fossil fuel" href="javascript:popupOBO('CHEBI:35230','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35230')">fossil fuel</a></span> consumption that is the culprit of the green house effects. The critical step of the biomass conversion process, breaking cellulose fibers into smaller <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> molecules, could be potentially facilitated by rational design of more industrially friendly cellulase.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><img alt="Comparison of the computationally designed model (black, blue online) to the solved X-ray crystal structure (gray, red online) of Top7. (A) Cα overlay of the model and structure in stereo (backbone RMSD = 1.17 Å). (B) The C-terminal halves of the X-ray crystal structure and model are extraordinarily similar. The representative region shown (Asp78 to Gly85) has an all-atom RMSD of 0.79 Å and a backbone RMSD of 0.54 Å. (C) Stereo-representation of the effectively superposable side chains in the cores of the designed model and the solved structure (Science, 2003, 302, 1364–1368. Copyright 2003, AAAS)." src="/image/article/2010/CS/b810924a/b810924a-f2.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Comparison of the computationally designed model (black, blue online) to the solved X-ray crystal structure (gray, red online) of Top7. (A) Cα overlay of the model and structure in stereo (backbone RMSD = 1.17 Å). (B) The C-terminal halves of the X-ray crystal structure and model are extraordinarily similar. The representative region shown (Asp78 to Gly85) has an all-atom RMSD of 0.79 Å and a backbone RMSD of 0.54 Å. (C) Stereo-representation of the effectively superposable side chains in the cores of the designed model and the solved structure (<span class="italic">Science</span>, 2003, <span class="bold">302</span>, 1364–1368. Copyright 2003, AAAS).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">According to different extent of utilizing existing sequences from structures in the PDB database, <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design is usually classified into redesign and <span class="italic">de novo</span> design. The former involves improving and enhancing original functions or introducing new functions into naturally occurring <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> with limited amount of mutations, while the latter refers to assignment of sequences that are not related to any natural <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> sequences adopting the target fold. However, the border is not clear cut. For example, in the full sequence redesign of <span class="italic">Drosophila melanogaster</span> engrailed homeodomain by Shah <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> two highly stable sequences (with midpoint thermal denaturation temperature higher than 99 °C) of only 22% and 24% native sequence identity were obtained. Different strategies are utilized in these two types of designs. For limited sequence redesigns, deterministic sequence and conformational search are the method of choice, while for extensive design with many mutable residues, only stochastic methods can realistically provide approximate solutions (see Section 2.1). However, the general principles are the same and no special attention will be given to differentiate them in this review. It is important to point out that current “<span class="italic">de novo</span>” design still extensively utilizes empirical knowledge obtained from high-resolution structures deposited in the PDB, in contrast to quantum mechanics based “<span class="italic">ab initio</span>” small molecule simulations, which uses pure electronic structure theory as the start point.</p>
      <p class="otherpara">The first step of a <span class="italic">de novo</span> design is to choose a target fold based upon the desired function. As there is no rule existing for the designability (a designable fold should has at least one solution sequence) of a given fold, the safest bet would be to pick from the PDB a natural occurring one or a combination of them. Since the number of distinctive folds in the PDB is approximately 1100 as classified by CATH<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> and is continuing to increase, this resource should satisfy most of our needs for a target fold. Many methods (<span class="italic">e.g.</span> ScaffoldSelection<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> and references therein) have been developed to search for appropriate scaffolds in the PDB for <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b810924a')">enzyme</a></span> design. Once a target fold is selected, we face major challenges of sequence/conformational search, energy evaluations, and final experimental validation.</p>
      <p class="otherpara">There are many excellent detailed reviews either focusing on the performance of various methodologies<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">14–17</span></sup></a> or regarding the latest progresses on design of different type of <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>.<a title="Select to navigate to reference" href="#cit9"><sup><span class="sup_ref">9,18</span></sup></a> As a tutorial review, this article will first give a brief introduction to the fundamental problems that need to be solved by the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design community, then summarize where we are in terms of addressing these challenges, with emphasis on the computational design of both soluble and <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. The important interplay between computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design, experiments and simulations is presented. Finally, the prospects, challenges and opportunities in this rapidly expanding field are discussed.</p>
    
    
      
      <h2 id="sect299"><span class="a_heading">2. Fundamentals of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design</span></h2>
      <span>The purpose of current <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design is to select one or more sequences which will fold into a desired structure with reasonable stability. A 100-residue <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> has 20<small><sup>100</sup></small> possible different sequences, which dictate that the complexity of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design is literally much more challenging than picking a needle in a haystack. This problem is exacerbated by the fact that each residue has backbone degrees of freedom represented by dihedral angles <span class="italic">ϕ</span> and <span class="italic">ψ</span>. Additionally, depending upon its identity, each residue has various number of side-chain degrees of freedom named rotameric states (rotations around various C–C bonds [<span class="italic">e.g.</span> C<small><sub>α</sub></small>–C<small><sub>β</sub></small>, C<small><sub>β</sub></small>–C<small><sub>γ</sub></small>] in side chains). The rule of the selection is that sequences folding into a more stable desired structure should have lower free energy than those folding into less stable ones. The process of such selection, usually referred to as the sequence and conformational search problem, is only one of the two challenges facing <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> designers. The other challenge is that we do not know how to accurately and rapidly compute the free energy of a given <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> sequence adopting a desired structure. Scientists have come up with various approximate functions, usually called potential functions or scoring functions, to characterize the free energy of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> sequence in a given structure.</span>
      
        
        <h3 id="sect320"><span class="b_heading">2.1 Methodologies for sequence and conformational search</span></h3>
        <span>By utilizing a fixed backbone and discrete rotameric states for each <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span>, the backbone degrees of freedom is reduced to one and the side-chain conformational space is also greatly decreased. Therefore the search for the global minimum energy conformation (GMEC) becomes a combinatorial problem of the sequence and discrete rotameric states for a given fold. However, even after such simplification, a brute force exhaustive calculation of free energy for all possible sequences and conformations and then picking the lowest energy candidates is apparently out of the question even for a small <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>. With a conservative sampling of only five rotameric states for each <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span>, a 100-residue design involves evaluating and comparing energies for (5 × 20)<small><sup>100</sup></small> = 10<small><sup>200</sup></small> states, far exceeding capability of any current supercomputer. This challenge will remain true in the foreseeable future when petaflop (10<small><sup>15</sup></small> float point operations per second) machines are on everybody's desktop. However, it is widely known that in the PDB database, the number of folds is only a small fraction of that of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> structures and typical structures are shared by many different sequences. In addition, there are cases that <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> with low sequence identity share an almost identical structure,<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a> indicating that sequences far away from each other in the sequence space can be equally good candidates for some target fold. This is a comforting fact since we need only one or a few sequences that can stably fold into our desired structure. It is also important from a theoretical point of view that due to this fact, the sequence search problem in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design can be treated as a local minimization problem rather than a global minimization problem. Consequently, unlike importance sampling of conformational space in molecular simulations, the detailed balance requirement (which dictates that the probability of transition from a microstate A to a microstate B has to be equal to the probability of the inverse transition from B to A) is not necessary in the sequence search of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. Therefore, in principle, there could be a lot of different ways to bias the random walk in the sequence space to facilitate our search. Despite the fact that side chains (especially ones on the surface of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>) can easily flip between different rotameric states belonging to the native ensemble, it is desirable in most cases that the native structural ensemble itself settles in a unique lowest free energy valley. This is because a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> with multiple comparable low free energy valleys would potentially have promiscuous functions and low specificity. Therefore, the conformational search process is usually a global minimization problem. Currently, sequence and conformational search are conducted either simultaneously (all possible amino acids and their rotameric states for each design position are considered in each search move, <span class="italic">e.g.</span> DESIGNER), or alternately (first a search in sequence space is conducted and then a conformational search is carried out with the sequence fixed, this two procedures are performed alternately to find lowest energy sequence and conformation combination. <span class="italic">e.g.</span> RosettaDesign). There is not much discussion on the benefit (or lack of it) of separating sequence and conformational search in the literature. The conformational search has been the center of search algorithm research. Development of the novel sequence sampling technique has potential to become a fertile frontier of computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design.</span>
        <p class="otherpara">There are mainly two categories of sequence and conformational sampling methods, namely deterministic and stochastic ones. Deterministic search methods give a unique solution (or a unique set of solutions) once converged while solution(s) from stochastic search methods depend upon specific sets of random numbers used and the length of search operations. A widely used deterministic method is the Dead End Elimination (DEE) algorithm,<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a> which rigorously finds the global free energy minimum in the sequence and conformational space through eliminations of sequences and conformations that are provably not compatible with GMEC. However, because of the exponential increase of the sequence and conformational space with the number of residues, the convergence of DEE can only be achieved for <span class="italic">de novo</span> design of small <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> or redesign of <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> with limited mutable positions. Since being first introduced by Desmet <span class="italic">et al.</span> in 1992,<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a> this method has been improved significantly through introduction of more efficient elimination rules<a title="Select to navigate to reference" href="#cit21"><sup><span class="sup_ref">21,22</span></sup></a> and the divide and conquer strategy.<a title="Select to navigate to reference" href="#cit23"><sup><span class="sup_ref">23,24</span></sup></a> The DEE was later on combined with other innovative procedures to further improve its search efficiency. Desmet <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit25"><sup><span class="sup_ref">25</span></sup></a> developed an iterative algorithm named FASTER (Fast and Accurate Side-Chain Topology and Energy Refinement), which uses DEE for initial rotamer eliminations. The method was found to be two to three orders of magnitude faster than DEE in reaching the GMEC for most of a set of 33 small <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and it scales favorably with the size of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>. Additionally, the FASTER method always converges (not true for DEE in many cases). FASTER was further improved<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a> by use of Monte Carlo (MC) simulation to create a number of high quality initial structures. By focusing relaxation steps on residues that interacting strongly with perturbed positions, this strategy effectively utilizes structural correlation between interacting residues. DEE was also combined with a search tree based algorithms (A*)<a title="Select to navigate to references" href="#cit27"><sup><span class="sup_ref">27</span></sup></a> to identify all of the low energy conformations within a threshold energy value of the GMEC in strict order of increasing energy, thus providing a detailed picture of the energy profile in the vicinity of the GMEC.</p>
        <p class="otherpara">Another deterministic method is based upon self consistent mean field (SCMF) approximation,<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> this method is guaranteed to converge and its computational cost scales linearly with the number of residues in a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>. But the minimum obtained from SCMF are not guaranteed to be the real GMEC and as a matter of fact, it does not converge to the correct GMEC in many cases.<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a></p>
        <p class="otherpara">Stochastic methods, such as MC and genetic algorithm methods,<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> do not aim at rigorous search of GMEC. Instead, a random walk through the sequence and conformational space is carried out and the lowest energy conformation found is used as an approximate solution to the given design problem. In a typical MC search procedure, after a random move is made in sequence or conformational space, the Metropolis MC<a title="Select to navigate to references" href="#cit28"><sup><span class="sup_ref">28</span></sup></a> criteria is used to accept the move with a probability<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn1"/><span id="eqn1"><span class="italic">P</span> = 1: <span class="italic">E</span><small><sub>b</sub></small> &lt; <span class="italic">E</span><small><sub>a</sub></small>;<span><span class="italic">P</span> = exp[−(<span class="italic">E</span><small><sub>b</sub></small> − <span class="italic">E</span><small><sub>a</sub></small>)]/<span class="italic">k</span><small><sub>b</sub></small><span class="italic">T</span>: <span class="italic">E</span><small><sub>b</sub></small> &gt; <span class="italic">E</span><small><sub>a</sub></small></span></span></td><td class="rightEqn">(1)</td></tr></table>with <span class="italic">k</span><small><sub>b</sub></small> being the Boltzmann constant, <span class="italic">E</span><small><sub>a</sub></small> and <span class="italic">E</span><small><sub>b</sub></small> are the free energy before and after the mutation or conformational change, and <span class="italic">T</span> is the temperature. For the conformational search, it is critical to create moves that have both good accept ratio and the ability to rapidly explore the free energy landscape. One way to accelerate the MC sampling is by starting from a high temperature so that even moves with relatively large energy change will have a significant probability to be accepted, then gradually reduce the temperature so that the search settles down in low free energy valleys. This process is termed simulated <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: annealing" href="javascript:popupOBO('CMO:0001465','b810924a')">annealing</a></span> (<span class="CH"><a class="simple" title="Chemical Methods Ontology link for: SA" href="javascript:popupOBO('CMO:0001305','b810924a')">SA</a></span>). Each <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: SA" href="javascript:popupOBO('CMO:0001305','b810924a')">SA</a></span> search process will settle in a local free energy minimum in the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> conformational space and many cycles of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: SA" href="javascript:popupOBO('CMO:0001305','b810924a')">SA</a></span> are usually performed by cyclically raising and lowering the temperature, the lowest free energy conformation found is used as the GMEC. However, increasing temperature does not solve all the potential problems in MC sampling. For example, any moves that cause significant steric clashes will result in a huge free energy pulse and consequently will not have much chance to be accepted even at a relatively high temperature. Therefore creative moves are essential to facilitate conformational sampling using MC. A set of moves were specifically developed for <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> in Rosetta package.</p>
        <p class="otherpara">Genetic algorithm (GA) is another widely used stochastic search method for <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design.<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> This method explores the sequence space through random mutation and recombination to find low energy sequences for the target fold. It usually starts with many <span class="AN"><a class="simple" title="Sequence Ontology link for: random sequences" href="javascript:popupOBO('SO:0000449','b810924a')">random sequences</a></span>, mutations are then applied randomly at a certain rate to generate a <span class="CH"><a class="simple" title="ChEBI Ontology link for: group" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">group</a></span> of sequences and free energy of generated sequences/conformations are evaluated and ranked. Solutions with lowest free energy will be paired for recombination and new mutations are started for the second cycle. The population dynamics of GA give it some advantage over MC for crossing higher energy barriers. However, because the correlation between the inter-residue spatial distance and inter-residue sequence distance is quite weak, finding clean crossover points during the recombination step is not straight forward.</p>
        <p class="otherpara">The extreme optimization (EO) method, originally applied to spin glass optimization problems in condensed matter physics, was adapted to <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design problems.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> Using <span class="italic">i</span><small><sub><span class="italic">r</span></sub></small> to denote the <span class="italic">r</span>-th rotamer at position <span class="italic">i</span> and <span class="italic">j</span><small><sub><span class="italic">s</span></sub></small> to denote the <span class="italic">s</span>-th rotamer at position <span class="italic">j</span>, the free energy based on single-residue <span class="italic">E</span>(<span class="italic">i</span><small><sub><span class="italic">r</span></sub></small>) and pair-residue <span class="italic">E</span>(<span class="italic">i</span><small><sub><span class="italic">r</span></sub></small>, <span class="italic">j</span><small><sub><span class="italic">s</span></sub></small>)<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn2"/><img class="centered" alt="ugraphic, filename = b810924a-t1.gif" src="/image/article/2010/CS/b810924a/b810924a-t1.gif"/></td><td class="rightEqn">(2)</td></tr></table>can be written as<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn3"/><img class="centered" alt="ugraphic, filename = b810924a-t2.gif" src="/image/article/2010/CS/b810924a/b810924a-t2.gif"/></td><td class="rightEqn">(3)</td></tr></table>with <span class="italic">N</span> being the number of residues in the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>. This stochastic search method divides the total energy into a sum of local pair energies in the following way:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn4"/><img class="centered" alt="ugraphic, filename = b810924a-t3.gif" src="/image/article/2010/CS/b810924a/b810924a-t3.gif"/></td><td class="rightEqn">(4)</td></tr></table>and rank these local energies in decreasing order:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn5"/><span id="eqn5"><span class="italic">ε</span><small><sub>Π (1)</sub></small> ≥ <span class="italic">ε</span><small><sub>Π (2)</sub></small> ≥ ⋯ ≥ <span class="italic">ε</span><small><sub>Π(<span class="italic">k</span>)</sub></small> ≥ ⋯ ≥ <span class="italic">ε</span><small><sub>Π(<span class="italic">N</span>)</sub></small></span></td><td class="rightEqn">(5)</td></tr></table>Starting from a random state, with <span class="italic">k</span> being the rank of site index Π(<span class="italic">k</span>), the EO pick a rank from a power-law distribution with an exponent −<span class="italic">τ</span>: <span class="italic">P</span>(<span class="italic">k</span>) ∼ <span class="italic">k</span><small><sup>−<span class="italic">τ</span></sup></small>. At site <span class="italic">k</span>, the rotamer energy is ranked in increasing order:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn6"/><span id="eqn6"><span class="italic">ε</span><small><sub><span class="italic">i</span>Π (1)</sub></small> ≤ <span class="italic">ε</span><small><sub><span class="italic">i</span>Π (2)</sub></small> ≤ ⋯ ≤ ⋯ ≤ <span class="italic">ε</span><small><sub>Π(<span class="italic">k</span>′)</sub></small> ≤ <span class="italic">ε</span><small><sub><span class="italic">i</span>Π(Mrot(i))</sub></small></span></td><td class="rightEqn">(6)</td></tr></table>with <span class="italic">k</span>′ being the rank of rotamer index Π′(<span class="italic">k</span>′) and <span class="italic">M</span><small><sub>rot(i)</sub></small> being the number of candidate rotamers at this site. Then a rotamer index <span class="italic">k</span>′ is picked from a power distribution with exponents −<span class="italic">τ</span>′: <span class="italic">P</span>(<span class="italic">k</span>′) ∼ (<span class="italic">k</span>′)<small><sup>−<span class="italic">τ</span>′</sup></small>. A change of rotamer at site Π(<span class="italic">k</span>) to rotamer Π′(<span class="italic">k</span>′) is performed and the move is accepted unconditionally. Local energies are updated after the rotamer change and such ranking and changing is repeated until energy converges to a preset target or a certain number of iterations have been reached. After optimizing the two exponents <span class="italic">τ</span> and <span class="italic">τ</span>′, EO was found to be comparable with <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: SA" href="javascript:popupOBO('CMO:0001305','b810924a')">SA</a></span> for conformational space sampling. Based upon characteristics of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> local energy landscape, the authors further improved the method by using the difference between the local energy and a reference local energy for ranking, and the improved method is termed Reference Energy Extremal Optimization (REEO).</p>
        <p class="otherpara">Each of the above mentioned methods has its own pros and cons, it is essential to choose a suitable strategy for a given design problem. As demonstrated by FASTER, combination of different methods has greatly advanced our ability to carry out rapid conformational searches. In a typical divide and conquer strategy, sub-problems at each level may have different characteristics, therefore, further development in combining various methods in such a sequence and conformational search cascade has great potential.</p>
      
      
        
        <h3 id="sect521"><span class="b_heading">2.2 Potential functions for <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design</span></h3>
        <span>To make free energy evaluation tractable for many conformational states of given sequences and target fold, all current <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design potential functions approximate molecular interactions through pairwise additive terms. In principle, the energy of a sequence on a fixed backbone can be expanded into the following form:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn7"/><img class="centered" alt="ugraphic, filename = b810924a-t4.gif" src="/image/article/2010/CS/b810924a/b810924a-t4.gif"/></td><td class="rightEqn">(7)</td></tr></table>with <span class="italic">E</span><small><sub>0</sub></small> accounting for the energy of the backbone itself, <span class="italic">E</span><small><sub><span class="italic">i</span></sub></small> accounting for the interaction between each residue and the backbone and the residue self energy, and <span class="italic">E</span><small><sub><span class="italic">ij</span></sub></small>, <span class="italic">E</span><small><sub><span class="italic">ijk</span></sub></small>,… representing pair, triple and higher order interactions among different residues. In reality, this expansion is usually cut off at the second order. The reasons for ignoring the triple and higher order interactions are threefold: (i) computation of triple and higher order interactions are much more expensive (scale as <span class="italic">n</span><small><sup>3</sup></small>, <span class="italic">n</span><small><sup>4</sup></small>…, <span class="italic">n</span> being the number of residues), while computational cost for pairwise interactions scales with <span class="italic">n</span><small><sup>2</sup></small> and can be reduced to <span class="italic">n</span> if only short range interactions are considered; (ii) due to the limited number of high resolution experimental structures, no accurate parameters can be derived for triple and higher order interactions based upon current PDB, and (iii) pairwise interaction approximation has been proven to be effective by many successful design studies.</span>
        <p class="otherpara">The pairwise approximation is also widely used for describing non-bonded interactions in molecular mechanics (MM) force fields (FF). Many mature MMFF have very similar terms and functional forms for each term.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> The basis for these MMFF include both small molecule <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: experimental data" href="javascript:popupOBO('CMO:0000793','b810924a')">experimental data</a></span> and quantum mechanical calculation, consequently they are referred to as physics based FF. There are two main reasons that MM force fields were not widely used in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. Firstly, MMFF are very expensive and are not suitable for initial energetic screening for large numbers of sequences and configurations; secondly, MMFF are parameterized to represent molecular configurations very close to the equilibrium structure and are highly sensitive to small coordinate changes, thus resulting in very rugged energy landscapes. In <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design, the placement of side chain positions are usually based on discrete rotamer libraries and small steric clashes between atoms are very common in the initial model building stage. The repulsive term accounting for van der Waals interaction is usually softened in some ways to alleviate this problem. A study conducted by Summa and Levitt<a title="Select to navigate to references" href="#cit31"><sup><span class="sup_ref">31</span></sup></a> demonstrated that knowledge based potentials perform better than MMFF in terms of bringing denatured or close-to-native structures to the native state.</p>
        <p class="otherpara">Potential functions used for <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design by different <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> have widely different forms at various resolutions. They usually contain a mixture of knowledge based pair potentials derived from high resolution structures in PDB and physical based energy terms accounting for van der Waals, solvation and electrostatics interactions. Entropy is usually implicitly considered through the knowledge-based pair potentials and reference state energies for each <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> type. Explicit consideration of side chain conformational entropy in design of 110 native backbones by Hu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit32"><sup><span class="sup_ref">32</span></sup></a> indicated that conformational entropy does not play a significant role in determining sequence preference in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. However, a recent study by Sciretti <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a> suggested that side chain entropy does make a non-trivial difference in that the rank of the best scoring sequences of 1<span class="italic">cka</span> is significantly changed with explicit entropy consideration. However, in the former case, the entropy term was added to the Rosetta potentials and in the latter case, it was added to the DESIGNER potentials. The differences between these two set of potentials (as stated below) makes the direct comparison between these two studies difficult. In a given set of potential functions, <span class="italic">ad hoc</span> weighting factors are used for each term and are optimized empirically to fit either <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: experimental data" href="javascript:popupOBO('CMO:0000793','b810924a')">experimental data</a></span> (<span class="italic">e.g.</span> melting temperature) or native sequences (or sequence profiles). Rosetta is one of the most successful <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> structure prediction and design packages up to date. It has two sets of potential functions with different resolutions. The low-resolution set represents each residue by its centroid. The detailed explanation for each term can be found in Rohl <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> One important contribution separating the pairwise potentials in Rosetta from other earlier derivations is that it rigorously decouples the residue–environment interactions from residue–pair interactions. The high-resolution set consists of a Ramachandran torsion preference term, a Lennard-Jones term, a statistical <span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span> bonding term, a solvation term, a term for rotamer energy and a reference term representing unfolded states. In contrast, DESIGNER has only three adjustable parameters with atomistic resolution. They are the dielectric constant (<span class="italic">ε</span>) and two weighting factors accounting for the solvation contribution of the reference (unfolded) state (<span class="italic">ω</span>) and the pairwise solvation term in the folded state (<span class="italic">γ</span>). Recently, a set of orientation dependent statistical all-atom potentials, dubbed <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:PSP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4602.html">PSP</a></span> (potential derived from side-chain packing) were developed by Lu <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> the PSP is based on a basis set of 19 rigid-body blocks extracted from chemical structures of the 20 amino acids (see <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>) and exhibited great performance in the limited tests that have been carried out. In addition to the accuracy of individual terms, the performance of a set of potential functions is also determined by the proper balance among various interactions. Increasing accuracy of any single term in isolation is not an effective way for improving the performance for a specific set of potential functions.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><img alt="Definition of 19 rigid-body blocks. (a) Illustration of block types. Blocks are categorized into nine symmetry classes from I to IX. Block classes I, II, III and VI are line shapes, and the others are plane shapes. Heavy atoms are numbered for determining contacts, steric clashes and reference frames (see Supplementary Data in the reference). R and R′ are not considered parts of the blocks but are shown to indicate connectivity only. The reference frames for line shapes and plane shapes are shown schematically alongside their corresponding block types at the bottom of the Figure. (b) Block composition of residues. All blocks are circled for all amino acids. The block types (numbers in parentheses) are defined in (a). Each residue contains at least one block type, and within that residue, no block type occurs more than once (J. Mol. Biol., 2008, 376. 288–301, Copyright 2008, Elsevier)." src="/image/article/2010/CS/b810924a/b810924a-f3.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">Definition of 19 rigid-body blocks. (a) Illustration of block types. Blocks are categorized into nine symmetry classes from I to IX. Block classes I, II, III and VI are line shapes, and the others are plane shapes. Heavy atoms are numbered for determining contacts, steric clashes and reference frames (see Supplementary Data in the reference). R and R′ are not considered parts of the blocks but are shown to indicate connectivity only. The reference frames for line shapes and plane shapes are shown schematically alongside their corresponding block types at the bottom of the Figure. (b) Block composition of residues. All blocks are circled for all amino acids. The block types (numbers in parentheses) are defined in (a). Each residue contains at least one block type, and within that residue, no block type occurs more than once (<span class="italic">J. Mol. </span><span class="italic">Biol</span><span class="italic">.</span>, 2008, <span class="bold">376</span>. 288–301, Copyright 2008, Elsevier).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect603"><span class="b_heading">2.3 The importance of unfolded states reference energies</span></h3>
        <span>Understanding the energetic properties of unfolded states is critical for improving accuracy of potential functions. In principle, the goal of the sequence search is to find the sequence with lowest folding free energy Δ<span class="italic">G</span><small><sub>folding</sub></small> = <span class="italic">G</span><small><sub>folded</sub></small> − <span class="italic">G</span><small><sub>reference</sub></small>. The reference is the collection of all unfolded states. Due to the difficulty in characterizing these unfolded states, many potential functions choose to ignore them<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> (that is equivalent to assume all <span class="CH"><a class="simple" title="ChEBI Ontology link for: polypeptides" href="javascript:popupOBO('CHEBI:15841','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=15841')">polypeptides</a></span> with the same length have the same free energy in unfolded states). Another approximation is to use the sum of contributions from individual isolated amino acids,<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> this treatment assumes that the free energy for the unfolded states of <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> are only determined by their <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> composition. In a recent study by Alvizo and Mayo,<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> based upon the random energy model, they derived potential function parameters through fixed-composition optimization, the resulted potentials were shown to have significantly greater predicting power.</span>
      
      
        
        <h3 id="sect619"><span class="b_heading">2.4 Treatment of explicit <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> molecules in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design</span></h3>
        <span>As reflected in many high-resolution crystal structures, <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> molecules can be very tightly bound to <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. Apparently the molecular geometry of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> molecules becomes important under such situations and various widely used implicit solvation models break down. To model <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span>-mediated <span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span> bonds at <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>–<span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> interfaces, Jiang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a> came up with a “solvated rotamer” approach which includes <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> molecules in rotamers. In the design of retro-Aldol enzymes by Jiang <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a> inclusion of explicit <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> molecules to mediate <span class="CH"><a class="simple" title="ChEBI Ontology link for: proton" href="javascript:popupOBO('CHEBI:24636','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">proton</a></span> shuffling was found to be significantly more successful than the results from utilizing charged side-chain networks. The failure to predict a water molecule at a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>–<span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> interface in the crystal structure<a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a> indicate that, within the certainty of the involved crystal structure, there are apparent deficiencies in the current modeling of explicitly bound <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> molecules. Improvement of explicit <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> modeling, especially at the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>–<span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> interface or <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>–ligand interface, is necessary to achieve more accurate design of interactions in inter-facial region of <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>.</span>
      
      
        
        <h3 id="sect650"><span class="b_heading">2.5 Backbone flexibility</span></h3>
        <span>In reality, <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> backbones are not rigid. As a matter of fact, the backbone flexibility is of great importance to <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> function. A direct consequence of incorporating backbone flexibility is the increased conformational space that needs to be explored. However, many studies demonstrated that the additional sampling effort engendered is worthwhile. With an alternating design scheme (optimizing the sequence for a fixed <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> backbone using rotamer based sequence search and optimizing the backbone for a fixed sequence with atomic resolution structure prediction), a globular <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> Top7 with a novel fold was designed.<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> A set of MC moves were developed to systematically introduce backbone flexibility in this study. Using the same optimization strategy, Saunders and Baker<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> found that flexible backbone design better recapitulate sequence variation within <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> families than sequence optimization with fixed or randomly perturbed backbones for ten diverse <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> structures. Fu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit43"><sup><span class="sup_ref">43</span></sup></a> utilized normal mode analysis to introduce the backbone flexibility. In this study, their designed BH3 <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptides" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptides</a></span> effectively broadened the set of sequences when compared with fixed backbone design, and provided insight into positions important for binding Bcl-xL.</span>
        <p class="otherpara">The most common local backbone structural variation seen in high-resolution crystal structures was given a term “Backrub”<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> (see <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>). It involves a primary rotation around an axis (Cα<small><sub><span class="italic">i</span>−1</sub></small>–Cα<small><sub><span class="italic">i</span></sub></small><small><sub>+1</sub></small>), which shifts the Cα–Cβ orientation of a given side-chain by rigid-body <span class="CH"><a class="simple" title="ChEBI Ontology link for: dipeptide" href="javascript:popupOBO('CHEBI:46761','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46761')">dipeptide</a></span> rotation. Two accompanying secondary rotations around axes Cα<small><sub><span class="italic">i</span>−1</sub></small>–Cα<small><sub><span class="italic">i</span></sub></small> and Cα<span class="italic"><small><sub>i</sub></small></span>–Cα<small><sub><span class="italic">i</span></sub></small><small><sub>+1</sub></small> rotate the two neighboring planar <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> bonds of the central Cα. Such concerted local motion costs little energy as bonding parameters (bond lengths, angles and <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> planarity) are maintained close to ideal during the process. The corresponding algorithm was utilized<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> to generate an ensemble of backbone conformations to account for backbone flexibility. The sequence profile obtained from such ensembles of backbone conformations was shown to distinguish restrictive positions (positions where only a small number of amino acids are allowed) from highly plastic positions (positions where a large number of amino acids are allowed) at <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>–<span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> interfaces, and to qualitatively predict the tolerance profile of restricted positions. Another important consequence of the flexible backbone design is that the search for GMEC becomes more complex. For stochastic search methods such as MC, incorporating flexibility simply requires additional moves modeling various backbone conformations. For DEE, the traditional pruning criteria may eliminate the real GMEC as the flexibility may change the relative free energy ranking obtained for a fixed backbone. To address this issue, Georgiev and Donald introduced a flexible-<span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_0062_0332.gif" alt="[b with combining low line]"/></span>ackbone <span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_0044_0332.gif" alt="[D with combining low line]"/></span>EE (BD) algorithm<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> with the global backbone flexibility represented by a bounded continuous family of solutions. The same <span class="CH"><a class="simple" title="ChEBI Ontology link for: group" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">group</a></span> also developed another algorithm, BRDEE,<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> which represent local backbone flexibility by a finite set of conformations generated <span class="italic">via</span> the “backrub” algorithm. When applied in redesigns of core residues of the β<small><sub>1</sub></small> domain of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> G (Gβ<small><sub>1</sub></small>) and the active site of the non-ribosomal <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> synthetase (NRPS) <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b810924a')">enzyme</a></span> Gramicidin Synthetase A (GrsA-PheA), both algorithms are found to be superior to original rigid backbone DEE in identifying more low-energy sequences and conformations. These two methods are complementary since BD represents flexibility with continuous global motions while BRDEE represents flexibility with discrete local conformations.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><img alt="Schematic representations of the Backrub motion (A) A schematic diagram of the Backrub motion. The primary rotation (θ1,3) moves the central residue and its adjacent peptides around the axis (connecting Cαi–1 to Cαi+1) as a rigid body, causing the central Cα to trace out the black (red online) dotted circle. Secondary rotations (θ1,2 and θ2,3, gray/blue online dotted circle) move the individual peptides as rigid bodies around the axes Cαi−1–Cα and Cα–Cαi+1. A small amount of distortion is introduced into the two τ angles (N–Cα–C), but they generally remain well within the range of values seen in typical crystal structures. (B) A series of backbones generated with Backrub by making 5° steps around the primary rotation axis (hydrogen atoms not shown). (C) Another series of backbones generated with Backrub by making 5° steps around the primary rotation axis, while also rotating each peptide to roughly maintain the H bonding position of the NH and CO groups. (Structure, 2006, 14, 265–274, Copyright 2006, Cell Press)." src="/image/article/2010/CS/b810924a/b810924a-f4.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Schematic representations of the Backrub motion (A) A schematic diagram of the Backrub motion. The primary rotation (<span class="italic">θ</span><small><sub>1,3</sub></small>) moves the central residue and its adjacent <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptides" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptides</a></span> around the axis (connecting Cα<small><sub><span class="italic">i</span>–1</sub></small> to Cα<small><sub><span class="italic">i</span></sub></small><small><sub>+1</sub></small>) as a rigid body, causing the central Cα to trace out the black (red online) dotted circle. Secondary rotations (<span class="italic">θ</span><small><sub>1,2</sub></small> and <span class="italic">θ</span><small><sub>2,3</sub></small>, gray/blue online dotted circle) move the individual <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptides" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptides</a></span> as rigid bodies around the axes Cα<small><sub><span class="italic">i</span>−1</sub></small>–Cα and Cα–Cα<small><sub><span class="italic">i</span></sub></small><small><sub>+1</sub></small>. A small amount of distortion is introduced into the two <span class="italic">τ</span> angles (N–Cα–C), but they generally remain well within the range of values seen in typical crystal structures. (B) A series of backbones generated with Backrub by making 5° steps around the primary rotation axis (<span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span> atoms not shown). (C) Another series of backbones generated with Backrub by making 5° steps around the primary rotation axis, while also rotating each <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> to roughly maintain the <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> bonding position of the NH and CO <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span>. (<span class="italic">Structure</span>, 2006, <span class="bold">14</span>, 265–274, Copyright 2006, <span class="AN"><a class="simple" title="Cell Ontology link for: Cell" href="javascript:popupOBO('CL:0000000','b810924a')">Cell</a></span> Press).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect751"><span class="b_heading">2.6 The concerted progresses of search methodology and energy functions</span></h3>
        <span>Because all accuracy tests of potential functions are limited by the extent of conformational sampling performed, the development of conformational search methods and potential functions are highly correlated. As an example, Shmygelska and Levitt<a title="Select to navigate to references" href="#cit48"><sup><span class="sup_ref">48</span></sup></a> found that the Rosetta force fields actually create false native states in addition to real ones. The deficiency was only observed when sufficient conformational sampling was achieved by using Hamiltonian Replica Exchange<a title="Select to navigate to footnote" href="#fn1">†</a> Monte Carlo sampling method, which was demonstrated to perform better than both Temperature Replica Exchange Monte Carlo and the original Monte Carlo <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: SA" href="javascript:popupOBO('CMO:0001305','b810924a')">SA</a></span> used in the Rosetta package. Additionally, they also discovered that the long-range and short-range backbone <span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span>-bonding energy terms of the Rosetta energy discriminate between the non-native and native-like structures significantly better than the low-resolution scores used in Rosetta.</span>
        <p class="otherpara">The specifics of a search algorithm always depend upon the form of potential functions. Grigoryan <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> developed a <span class="CH"><a class="simple" title="ChEBI Ontology link for: cluster" href="javascript:popupOBO('CHEBI:33731','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33731')">cluster</a></span> expansion method in which energies for a subset of conformational space were computed with structure based potential functions and then used to train a set of expansion coefficients. Specifically, denote the energy of the sequence <span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_e1c9.gif" alt="[small sigma, Greek, vector]"/></span> on the <span class="italic">N</span>-residue target fold backbone <span class="italic">E</span><small><sub>min</sub></small>(<span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_e1c9.gif" alt="[small sigma, Greek, vector]"/></span>), where <span class="italic"><span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_e1c9.gif" alt="[small sigma, Greek, vector]"/></span></span> = {<span class="italic">σ</span><small><sub>1</sub></small>, ... ,<span class="italic">σ</span><small><sub>N</sub></small>}, and <span class="italic">σ</span><small><sub>i</sub></small> is a variable representing the <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> at the <span class="italic">i</span>-th position. Let <span class="italic">M</span> be the number of allowed amino acids at each position and <span class="italic">σ</span><small><sub>i</sub></small> may assume any value from 0 to <span class="italic">M</span> − 1, then <span class="italic">E</span><small><sub>min</sub></small>(<span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_e1c9.gif" alt="[small sigma, Greek, vector]"/></span>) can be expanded into the following form:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn8"/><img class="centered" alt="ugraphic, filename = b810924a-t5.gif" src="/image/article/2010/CS/b810924a/b810924a-t5.gif"/></td><td class="rightEqn">(8)</td></tr></table>where <span class="italic">φ(si,i)</span> is a binary function and equals 1 if site <span class="italic">si</span> is occupied by an <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> <span class="italic">i</span> or 0 otherwise. The summations are over all sites and <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> identities, Terms <span class="italic">J</span> are the effective contributions of each function to the total energy. This expansion will be exact if used in its entirety. In reality, truncation at the second or third order is necessary. Terms <span class="italic">J</span> can be derived by using a training set of sequence-energy pairs obtained from traditional structured-based energy evaluation procedures. Once these terms are established, the energy is expressed in terms of sequence instead of structure. Great accuracy was achieved for bZIPs (basic-region <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:leucine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.834.html">leucine</a></span> zipper) and the search is extremely fast when combined with optimization by Integer Linear Programming (ILP)<a title="Select to navigate to footnote" href="#fn2">‡</a>. The downside is that such expansion has to be performed for each specific fold, which constitutes a significant challenge for designing multiple structural states involving different folds. However, for single fold design, the integrated frame termed CLASSY<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> (<span class="CH"><a class="simple" title="ChEBI Ontology link for: cluster" href="javascript:popupOBO('CHEBI:33731','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33731')">cluster</a></span> expansion and linear programming based analysis of specificity and stability) is in principle extremely flexible and both physical/structure-based models and empirical/statistics based models can be accommodated. The only requirement is that potential functions can be expressed as a linear function of sequence variables.</p>
      
    
    
      
      <h2 id="sect813"><span class="a_heading">3. Significant achievements of soluble <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design</span></h2>
      
        
        <h3 id="sect817"><span class="b_heading">3.1 Improving thermal stability through computational design</span></h3>
        <span>Thermal stability is one of basic goals in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design as <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> that are intended to carry out certain functions should maintain their function under ambient conditions or even withstand reasonably high temperature for robustness. Since it has been understood that hydrophobic packing is the major driving force for the folding of globular <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, most design studies successfully increased <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> stability by increasing the hydrophobic burial.<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> Fundamentally, high thermal stability of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> requires two conditions to be satisfied. First, in the absolute sense, the <span class="AN"><a class="simple" title="Sequence Ontology link for: designed sequences" href="javascript:popupOBO('SO:0000546','b810924a')">designed sequences</a></span> should withstand ambient or higher temperature in its intended structure and this property is represented by the computed value of the lowest folding free energy; second, there should be a large enough gap between the free energies of the <span class="AN"><a class="simple" title="Sequence Ontology link for: designed sequence" href="javascript:popupOBO('SO:0000546','b810924a')">designed sequence</a></span> in the target structure and in alternative structures, should there be any. The fact that the design process is a search for the lowest folding energy sequence automatically considers the first condition. Therefore, even without explicit consideration of heuristic rules such as large hydrophobic burial, high stability can be achieved in automated design procedures as demonstrated by the redesign of <span class="italic">Drosophila melanogaster</span> engrailed homeodomain.<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a> However, in reality, there are always functional constraints that have to be satisfied while increasing the stability. By fixing residues directly involved in catalysis, Korkegian <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> successfully redesigned the yeast cytosine deaminase to be 10 °C more stable while maintaining its <span class="AN"><a class="simple" title="Gene Ontology link for: catalytic" href="javascript:popupOBO('GO:0003824','b810924a')">catalytic</a></span> efficiency. Preserving function of natural <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> while increasing their stability will continue to be an important area of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design since a lot of functions performed by natural <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> (<span class="italic">e.g.</span> fixation of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> under ambient conditions, breaking of the glycosidic bonds in cellulose by cellulase) could potentially be utilized in industries.</span>
      
      
        
        <h3 id="sect839"><span class="b_heading">3.2 Interaction specificity in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design</span></h3>
        <span>A designed <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> or <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> is usually meant to carry out certain functions through interactions with its intended target/partner. For optimal effects, the designed <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>/<span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> has to have sufficient stability and specificity. The stability can be achieved by simply searching for the lowest free energy sequence for a given fold (which matches well with the intended target) and such a process is usually termed positive design. Specificity design is more complex because it has to explicitly consider the possible alternative structures the design sequence may adopt, thus facing a structure prediction problem accompanying the design process for the intended target. Further complications arise when selection against <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> that are homologous to the target has to be considered. Designing processes which explicitly address these issues are termed negative design. While there are examples of specificity being convenient by-products of a pure positive design<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> (Yosef <span class="italic">et al.</span> 2009, and references therein), stability and specificity are conflicting goals in many cases and appropriate compromises have to be made to achieve a suitable balance, especially when the intended target has many structurally similar molecules that coexist in its environment.</span>
        <p class="otherpara">Despite the above mentioned challenges, great progress has been made in achieving specificity in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. Havranek and Harbury<a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a> developed an algorithm that selects sequences with an energetic preference for the desired state over competing states. They implemented the algorithm for automated design of specificity in molecular recognition, and designed two coiled-coil sequences that prefer homodimers over heterodimers. Joachimiak <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a> introduced a new <span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span>-bond network into the colicin E7 Dnase-Im7 immunity <span class="AN"><a class="simple" title="Gene Ontology link for: protein complex" href="javascript:popupOBO('GO:0043234','b810924a')">protein complex</a></span> and obtained at least 300-fold specificity over one of the cognate interactions. Grigoryan <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> implemented a “specificity sweep” procedure within their CLASSY framework to design bZIP binding <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptides" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptides</a></span>. Many of their designs were selective for the intended targets over all 19 other families (20 bZIP families were investigated in the study).</p>
      
      
        
        <h3 id="sect865"><span class="b_heading">3.3 Computational design of enzymes</span></h3>
        <span>With the ability to accelerate chemical reactions up to 10<small><sup>20</sup></small> (<span class="italic">e.g.</span> <span class="AN"><a class="simple" title="Gene Ontology link for: Arginine Decarboxylase" href="javascript:popupOBO('GO:0008792','b810924a')">Arginine Decarboxylase</a></span>) and to realize their performance under ambient or physiological conditions, it is not surprising that enzymes have attracted significant effort from the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design community. A typical catalyzed chemical reaction involves substrate binding → conformational transition(s) to the transition state → reaction → conformational transition(s) to the product release state → product release. It is demonstrated for <span class="AN"><a class="simple" title="Gene Ontology link for: dihydrofolate reductase" href="javascript:popupOBO('GO:0004146','b810924a')">dihydrofolate reductase</a></span><a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> that the rate limiting step coincides with a step exhibiting slowest dynamical transition. Current design of enzymes mainly targets the transition-state interactions involving high-energy states of substrates. In this regard, Lassila <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> redesigned <span class="italic">E. coli</span> <span class="AN"><a class="simple" title="Gene Ontology link for: chorismate mutase" href="javascript:popupOBO('GO:0004106','b810924a')">chorismate mutase</a></span> by stabilizing transition-state binding and only mutating second-shell residues. One of the five designed single mutations increased efficiency by 60%. Recently, Rothlisberger <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> successfully designed an <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b810924a')">enzyme</a></span> that catalyzes the Kemp <span class="CH"><a class="simple" title="RXNO Ontology link for: elimination reaction" href="javascript:popupOBO('RXNO:0000123','b810924a')">elimination reaction</a></span>, for which no known natural enzymes is available. They achieved reaction acceleration of up to 10<small><sup>5</sup></small>. Using hashing based new algorithms, Jiang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> designed retro-aldol enzymes with rate acceleration up to four orders of magnitude. Compared to earlier <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b810924a')">enzyme</a></span> redesigns with limited mutations, these two <span class="italic">de novo</span> designs created enzymes for non-biological reactions. While the rate enhancement in both studies is not impressive at all when compared with natural enzymes, these achievements are significant milestones toward our goal of designing novel <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b810924a')">enzyme</a></span> with natural-<span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b810924a')">enzyme</a></span> like capacities. Bolon <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> stated in their 2002 review that “<span class="italic">Computational design is not currently an effective tool for increasing fitness</span>, <span class="italic">which may occur through subtle changes in backbone structure and </span><span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')"><span class="italic">protein</span></a></span><span class="italic"> dynamics</span>”. Unfortunately, this statement remains true today although we are certainly much closer to solving the problem. The inability to capture subtle structural changes has its origin in both sequence and conformational search capability and insufficient accuracy of potential functions. This is because a distal mutation can potentially cause subtle or even significant structural changes in the active site. From a pure structural view point, a design with backbone deviation less than 2 Å would be considered high quality. However, chemical reactions require key atoms within a few tenths of an angstrom, presenting much more stringent tests for the accuracy of potential functions. With increasing number of <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> joining the community, further progress in both sequence/conformational search and accuracy of potential functions are expected in the coming years, and design of higher quality structures will appear. However, new methodologies on designing multiple relevant structural states and dynamical transitions between them have to be developed before we can routinely design useful enzymes. As large conformational changes are often necessary in enzymatic reactions, ensemble design considering only structures or backbone flexibility in the vicinity of a single native state may be insufficient for such purposes. In this regard, Allen and Mayo<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> developed a multistate design (MSD) method in which the sequence selection is based on the energetic contributions from multiple structural states. Depending upon the intended importance of various states, either Boltzmann-weighted or arithmetically weighted average energy from the multiple target states is used in the sequence selection criteria. This method is combined with both MC (MSD-MC) and FASTER (MSD-FASTER) search algorithm and MSD-FASTER is found to have significantly better performance than MSD-MC.</span>
      
    
    
      
      <h2 id="sect902"><span class="a_heading">4. Progress of <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design</span></h2>
      <span>Our knowledge of <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> certainly does not match their biological and potential technological importance. From a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design point of view, limited availability of high-resolution structures and biochemical data makes it difficult to simply replicate the successes of soluble <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design in their <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> counterparts. Compared with design of soluble <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, there are two major differences need to be considered for <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. First, the partition of residues among the aqueous, hydrophobic core and inter-facial regions of biomembranes; second, molecular interactions for residues partitioned into <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> bilayers are different from both surface and buried residues in soluble <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. Great progress has been made in the last decade on these issues. In the molecular simulation community, a wide range of implicit <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> models have been developed<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> (see <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>) to account for the <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> environment in simulations. One of such models, namely IMM1,<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> was proven to be sufficient to recapitulate side chain conformational preference and residue partitions in a series of tests by Barth <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> to recover sequences and conformations for a set of polytopic <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. For transmembrane helical (THM) <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, inter-helical interactions are the key to their tertiary structure. In this regard, using the double-mutant cycle analysis, Joh <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> found that the average contribution of eight inter-helical side-chain <span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span> bonds in bacteriorhodopsin is only 0.6 kcal mol<small><sup>−1</sup></small>. Utilizing well-defined sequence–structure correlations at TMH interfaces, Yin <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> designed <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptides" href="javascript:popupOBO('CHEBI:16670','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptides</a></span> that specifically recognize the TM helices of two closely related integrins (α<small><sub>IIb</sub></small>β<small><sub>3</sub></small> and α<small><sub>v</sub></small>β<small><sub>3</sub></small>) in micelles, bacterial <span class="AN"><a class="simple" title="Gene Ontology link for: membranes" href="javascript:popupOBO('GO:0016020','b810924a')">membranes</a></span>, and mammalian <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b810924a')">cells</a></span>. Their results demonstrated that geometric complementarity alone is sufficient to drive specific recognition and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: activation" href="javascript:popupOBO('CMO:0002196','b810924a')">activation</a></span> of target integrin variants. The importance of complementary packing was also demonstrated in another study by North <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a></span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><img alt="Schematic representation of a typical implicit membrane model; ε represents the dielectric constant. Acyl chains of a lipid bilayer are represented by a slab of low dielectric media (“hydrophobic core”), head groups (“interfacial region”) with a slab with intermediate dielectric constant." src="/image/article/2010/CS/b810924a/b810924a-f5.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">Schematic representation of a typical implicit <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> model; <span class="italic">ε</span> represents the dielectric constant. Acyl chains of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> bilayer are represented by a slab of low dielectric media (“hydrophobic core”), head <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> (“interfacial region”) with a slab with intermediate dielectric constant.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">In addition to helical <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, the other class of integral <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> features beta barrels. Due to their excellent thermal stability and relative ease for expression and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: purification" href="javascript:popupOBO('CMO:0002231','b810924a')">purification</a></span>, a number of them (<span class="italic">e.g.</span> Alpha-hemolysin, OmpG) were studied<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> as potential biosensors. However, the mutation design in these studies are limited to empirical trials based on visual inspection or molecular dynamics (MD) simulations of high resolution crystal structures. No systematic design study has been carried out for <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b810924a')">membrane</a></span> embedded beta barrels.</p>
    
    
      
      <h2 id="sect964"><span class="a_heading">5. Interplay of computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design, experiments and molecular simulations</span></h2>
      <span>Experimental validations certainly play critical role in testing and improving our computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design protocols. As demonstrated by many successful studies,<a title="Select to navigate to reference" href="#cit16"><sup><span class="sup_ref">16,4</span></sup></a> the most direct test of our <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design capability is the comparison between computationally built models and high-resolution X-ray or <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: NMR" href="javascript:popupOBO('CMO:0000613','b810924a')">NMR</a></span> structures. On the other hand, <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design can be very useful in facilitating experimental studies. While computational design can sample much larger sequence space than <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design using directed evolution,<a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> the latter performs much better in terms of fine tuning the free energy landscapes. Fundamentally, in computational design approximate potential functions are applied to a larger set of sequences while in directed evolution, the exact “potential functions” were applied to a smaller set of sequences through appropriate selection assays. If an effective selection assay is available for the directed evolution, combining these two classes of methodologies by utilizing the minimally functional design from computational studies as the starting point of directed evolution is a great option. Computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design has been proved to generate libraries with greater diversity and better preservation of function than random library generation.<a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> With rapid and continuing progress in modeling <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>–<span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> interfaces, computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design will find more utilities in facilitating experimental molecular biology research.</span>
      <p class="otherpara">MD simulation has been extensively used to study the dynamics of biomolecules at atomic resolution. Its strength in this regard will reserve the method a well-deserved niche in the future molecular biological sciences. With development of mature MD packages and increasingly affordable computation resources, MD simulation is now carried out in both computational and experimental research <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span>. Currently, the interaction between the computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design field and MD simulation field is limited to sharing of some potential functions or MD simulations of some structural models derived from <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. The complementary strength of these two types of methodology may lead to deeper interaction between them. Possible scenarios include, but are not limited to, inclusion of MD simulation steps in structural refinement stage of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design or derivation of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> dynamics–sequence relationship from both MD trajectories and experiments and utilization of such relationship in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design. While bioinformatics methods, such as BLAST or similar sequence analysis methodologies, quickly find their way into experimental labs, MD simulations are slowly gaining popularity among some experimental molecular biologists. With improved availability of mature packages, computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design will certainly become an important tool for many molecular biologists in the future.</p>
    
    
      
      <h2 id="sect989"><span class="a_heading">6. Challenges and prospects of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design</span></h2>
      <span>The major challenge at the moment is still insufficient sampling capabilities of research <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> with only small computing <span class="CH"><a class="simple" title="ChEBI Ontology link for: clusters" href="javascript:popupOBO('CHEBI:33731','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33731')">clusters</a></span> available. So faster search algorithms is still the top priority for design of targets with more than 150 amino acids. On the bright side, in addition to the ever faster clock of processing units, with rapid development of GPU (Graphic Processing Units) based, massively parallel supercomputing facilities and internet based cloud computing, the computational power available to small research <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> will get a big boost in the coming years. As a matter of fact, some projects, such as Folding@home (http://folding.stanford.edu) and Rossetta@home (http://boinc.bakerlab.org/rosetta), have been using internet resources for many years.</span>
      <p class="otherpara">The capability of fine tuning the free energy landscape is critical in realizing effective design of functional <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> such as enzymes. Sub-angstrom structural accuracy and significant improvement of current potential functions is necessary to achieve this goal. One direction in this regard is to gradually improve general potential functions (which can be utilized for arbitrary folds) based upon our increasing understanding of physical interactions underlining the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> structure and function (<span class="italic">e.g.</span> Rosetta). The other direction would be to develop specialized potential functions for a given type of fold (<span class="italic">e.g.</span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: Cluster" href="javascript:popupOBO('CHEBI:33731','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33731')">Cluster</a></span> Expansion<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a>).</p>
      <p class="otherpara"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Protein</a></span> dynamics is critically important for their functions. However, most current <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> designs do not explicitly include dynamical properties as design objectives. The Elastic Network Model (ENM)<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> has been very successful in rapidly predicting functionally important domain motions on relatively long time scale (micron second to millisecond and beyond). Since no <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvent" href="javascript:popupOBO('CHEBI:46787','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvent</a></span> is considered in this model, it can be easily added as a filter in future design of larger and/or multi-domain <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. Rational and explicit consideration of the effect of <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvent" href="javascript:popupOBO('CHEBI:46787','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvent</a></span>–molecule dynamics on <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> dynamics on the ns to μs time scale await for development of novel algorithms to account for it.</p>
      <p class="otherpara">Given the impressive progress in the last two decades, with continuing improvement of computational power, sampling algorithms and potential functions, we may well see in the coming decade that the activity of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> design gradually propagate from computational <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> to experimental laboratories, and from academic laboratories to <span class="CH"><a class="simple" title="ChEBI Ontology link for: pharmaceutical" href="javascript:popupOBO('CHEBI:52217','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=52217')">pharmaceutical</a></span>, medical and chemical engineering industries. There are plenty of opportunities in both developing better searching algorithms and more accurate potential functions. For example, without the constraint of the detailed balance requirement, there is great potential in development of smarter MC or other stochastic search methodologies in sequence space. Most of the current algorithms are implemented for running on regular CPUs. To utilize rapid hardware development of GPUs, efficient implementations of many classical search algorithms on GPU based or mixed CPU/GPU computing facilities will greatly boost our computational capability in the next few years. While one major traditional test for potential functions is to maximally recapitulate sequence identity of native <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, matching the sequence profiles from <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> designs (which incorporating backbone flexibility or backbone structural ensemble) with that of naturally occurring <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> may be a better and more general test. Additionally, rigorous experimental structural verification of designed models is essential in improving potential functions.</p>
    
  
    <span id="sect1027"><h2 id="sect1026"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">M. Rubinstein and M. Y. Niv, <span class="italic">Biopolymers</span>, 2009, <span class="bold">91</span>, 505 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/bip.21164" title="DOI Link to resource 10.1002/bip.21164">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXltlCiu7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/1_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit2">A. J. Moss, S. Sharma and N. P. J. Brindle, <span class="italic">Biochem. Soc. Trans.</span>, 2009, <span class="bold">37</span>, 717 <a target="_blank" class="DOILink" href="https://doi.org/10.1042/BST0370717" title="DOI Link to resource 10.1042/BST0370717">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXovVOjtLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/2_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit3">S. Martí, J. Andres, V. Moliner, E. Silla, I. Tunon and J. Bertran, <span class="italic">Chem. Soc. Rev.</span>, 2008, <span class="bold">37</span>, 2634 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=b710705f&amp;newsite=1" title="Link to RSC resource DOI:10.1039/b710705f">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/3_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit4">S. Nauli, B. Kuhlman and D. Baker, <span class="italic">Nat. Struct. Biol.</span>, 2001, <span class="bold">8</span>, 602 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/89638" title="DOI Link to resource 10.1038/89638">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BD3Mzms1yjtQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/4_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit5">B. Kuhlman and D. Baker, <span class="italic">Curr. Opin. Struct. Biol.</span>, 2004, <span class="bold">14</span>, 89 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbi.2004.01.002" title="DOI Link to resource 10.1016/j.sbi.2004.01.002">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXhsFWqsrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/5_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit6">W. F. Degrado, C. M. Summa, V. Pavone, F. Nastri and A. Lombardi, <span class="italic">Annu. Rev. Biochem.</span>, 1999, <span class="bold">68</span>, 779 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev.biochem.68.1.779" title="DOI Link to resource 10.1146/annurev.biochem.68.1.779">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1MXlvFajt7o%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/6_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit7">B. Kuhlman, G. Dantas, G. Ireton, G. Varani, B. Stoddard and D. Baker, <span class="italic">Science</span>, 2003, <span class="bold">302</span>, 1364 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1089427" title="DOI Link to resource 10.1126/science.1089427">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXptVCqu7o%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/7_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit8">D. Röthlisberger, O. Khersonsky, A. M. Wollacott, L. Jiang, J. Dechancie, J. Betker, J. L. Gallaher, E. A. Althoff, A. Zanghellini, O. Dym, S. Albeck, K. N. Kouk, D. S. Tawfik and D. Baker, <span class="italic">Nature</span>, 2008, <span class="bold">453</span>, 190 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature06879" title="DOI Link to resource 10.1038/nature06879">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/8_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit9">P. Barth, <span class="italic">Curr. Opin. Struct. Biol.</span>, 2007, <span class="bold">17</span>, 460 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbi.2007.08.006" title="DOI Link to resource 10.1016/j.sbi.2007.08.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSltbrF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/9_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit10">G. Maglia, M. R. Restrepo, E. Mikhailova and H. Bayley, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2008, <span class="bold">105</span>, 19720 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0808296105" title="DOI Link to resource 10.1073/pnas.0808296105">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhsFCltr3F" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/10_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>; M. Chen, S. Khalid, M. S. P. Sansom and H. Bayley, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2008, <span class="bold">105</span>, 6272 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0711561105" title="DOI Link to resource 10.1073/pnas.0711561105">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXlsFyht7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/10_2" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit11">P. S. Shah, G. K. Hom, S. A. Ross, J. K. Lassila, K. A. Crowhurst and S. L. Mayo, <span class="italic">J. Mol. Biol.</span>, 2007, <span class="bold">372</span>, 1 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2007.06.032" title="DOI Link to resource 10.1016/j.jmb.2007.06.032">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXovFeru74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/11_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit12">O. C. Redfern, A. Harrison, T. Dallman, F. M. G. Perl and C. A. Orengo, <span class="italic">PLoS Comput. Biol.</span>, 2007, <span class="bold">3</span>, e232 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=PLoS%20Comput.%20Biol.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%20e232%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/12_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit13">C. Malisi, O. Kohlbacher and B. Hocker, <span class="italic">Proteins: Struct., Funct., Bioinf.</span>, 2009, <span class="bold">77</span>, 74 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proteins:%20Struct.,%20Funct.,%20Bioinf.%5BJour%5D%20AND%2077%5Bvolume%5D%20AND%2074%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/13_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit14">C. A. Voigt, D. B. Gordon and S. L. Mayo, <span class="italic">J. Mol. Biol.</span>, 2000, <span class="bold">299</span>, 789 <a target="_blank" class="DOILink" href="https://doi.org/10.1006/jmbi.2000.3758" title="DOI Link to resource 10.1006/jmbi.2000.3758">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXjs12nsLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/14_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit15">C. L. Vizcarra and S. L. Mayo, <span class="italic">Curr. Opin. Chem. Biol.</span>, 2005, <span class="bold">9</span>, 622 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXht1aqu7bE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/15_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit16">F. E. Boas and P. B. Harbury, <span class="italic">Curr. Opin. Struct. Biol.</span>, 2007, <span class="bold">17</span>, 199 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbi.2007.03.006" title="DOI Link to resource 10.1016/j.sbi.2007.03.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXksFehtb4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/16_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit17">R. Das and D. Baker, <span class="italic">Annu. Rev. Biochem.</span>, 2008, <span class="bold">77</span>, 363 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev.biochem.77.062906.171838" title="DOI Link to resource 10.1146/annurev.biochem.77.062906.171838">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXos1ekurw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/17_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit18">Y. Lu, N. Yeung, N. Sieracki and N. M. Marshall, <span class="italic">Nature</span>, 2009, <span class="bold">460</span>, 855 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature08304" title="DOI Link to resource 10.1038/nature08304">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXpvFCkt7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/18_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit19">W. D. Tian and J. Sholnick, <span class="italic">J. Mol. Biol.</span>, 2003, <span class="bold">333</span>, 863 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2003.08.057" title="DOI Link to resource 10.1016/j.jmb.2003.08.057">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXot1GktLs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/19_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit20">J. Desmet, M. De Maeyer, B. Hazes and I. Lasters, <span class="italic">Nature</span>, 1992, <span class="bold">356</span>, 539 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/356539a0" title="DOI Link to resource 10.1038/356539a0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK38XisFWls7Y%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/20_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit21">R. F. Goldstein, <span class="italic">Biophys. J.</span>, 1994, <span class="bold">66</span>, 1335 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2cXkslKnu7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/21_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit22">D. B. Gordon and S. L. Mayo, <span class="italic">J. Comput. Chem.</span>, 1998, <span class="bold">19</span>, 1505 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1096-987X(199810)19:13&lt;1505::AID-JCC7&gt;3.0.CO;2-U" title="DOI Link to resource 10.1002/(SICI)1096-987X(199810)19:13&lt;1505::AID-JCC7&gt;3.0.CO;2-U">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXmsFait7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/22_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit23">N. A. Pierce, J. A. Spriet, J. Desmet and S. L. Mayo, <span class="italic">J. Comput. Chem.</span>, 2000, <span class="bold">21</span>, 999 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/1096-987X(200008)21:11&lt;999::AID-JCC9&gt;3.3.CO;2-1" title="DOI Link to resource 10.1002/1096-987X(200008)21:11&lt;999::AID-JCC9&gt;3.3.CO;2-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXkvVOkt7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/23_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit24">I. Georgiev, R. H. Lilien and B. R. Donald, <span class="italic">Bioinformatics</span>, 2006, <span class="bold">22</span>, e174 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/bioinformatics/btl220" title="DOI Link to resource 10.1093/bioinformatics/btl220">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xotl2rtLo%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/24_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit25">J. Desmet, J. Spriet and I. Lasters, <span class="italic">Proteins: Struct., Funct., Genet.</span>, 2002, <span class="bold">48</span>, 31 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/prot.10131" title="DOI Link to resource 10.1002/prot.10131">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XksFGkt7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/25_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit26">B. D. Allen and S. L. Mayo, <span class="italic">J. Comput. Chem.</span>, 2006, <span class="bold">27</span>, 1071 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.20420" title="DOI Link to resource 10.1002/jcc.20420">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XlvVegsr4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/26_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit27">A. R. Leach and A. P. Lemon, <span class="italic">Proteins: Struct., Funct., Genet.</span>, 1998, <span class="bold">33</span>, 227 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1097-0134(19981101)33:2&lt;227::AID-PROT7&gt;3.0.CO;2-F" title="DOI Link to resource 10.1002/(SICI)1097-0134(19981101)33:2&lt;227::AID-PROT7&gt;3.0.CO;2-F">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXmsFOnsrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/27_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit28">N. Metropolis, A. W. Rosenbluth, M. N. Rosenbluth, A. H. Teller and E. Teller, <span class="italic">J. Chem. Phys.</span>, 1953, <span class="bold">21</span>, 1087 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.1699114" title="DOI Link to resource 10.1063/1.1699114">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaG3sXltlKhsw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/28_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit29">N. Zhang and C. Zeng, <span class="italic">J. Comput. Chem.</span>, 2008, <span class="bold">29</span>, 1762 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.20937" title="DOI Link to resource 10.1002/jcc.20937">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXosVKksLc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/29_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit30">A. D. Mackerell Jr, <span class="italic">J. Comput. Chem.</span>, 2004, <span class="bold">25</span>, 1584 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.20082" title="DOI Link to resource 10.1002/jcc.20082">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/30_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit31">C. M. Summa and M. Levitt, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2007, <span class="bold">104</span>, 3177 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0611593104" title="DOI Link to resource 10.1073/pnas.0611593104">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXjtVCnsbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/31_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit32">X. Hu and B. Kuhlman, <span class="italic">Proteins: Struct., Funct., Bioinf.</span>, 2006, <span class="bold">62</span>, 739 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proteins:%20Struct.,%20Funct.,%20Bioinf.%5BJour%5D%20AND%2062%5Bvolume%5D%20AND%20739%5Bpage%5D%20and%202006%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/32_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit33">D. Sciretti, P. Bruscolini, P. Alessandro, M. Pretti and A. Jaramillo, <span class="italic">Proteins: Struct., Funct., Bioinf.</span>, 2009, <span class="bold">74</span>, 176 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proteins:%20Struct.,%20Funct.,%20Bioinf.%5BJour%5D%20AND%2074%5Bvolume%5D%20AND%20176%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/33_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit34">C. A. Rohl, C. E. M. Strauss, K. M. S. Misura and D. Baker, <span class="italic">Methods Enzymol.</span>, 2004, <span class="bold">383</span>, 66 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXkt1Siurs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/34_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit35">M. Lu, A. Dousis and J. Ma, <span class="italic">J. Mol. Biol.</span>, 2008, <span class="bold">376</span>, 288 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2007.11.033" title="DOI Link to resource 10.1016/j.jmb.2007.11.033">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXpt1ymug%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/35_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit36">D. B. Gordon, S. A. Marshall and S. L. Mayo, <span class="italic">Curr. Opin. Struct. Biol.</span>, 1999, <span class="bold">9</span>, 509 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0959-440X(99)80072-4" title="DOI Link to resource 10.1016/S0959-440X(99)80072-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1MXlt1Cgtrg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/36_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit37">A. Jaramillo, L. Wernisch, S. Hery and S. J. Wodak, <span class="italic">Comb. Chem. High Throughput Screening</span>, 2001, <span class="bold">4</span>, 643 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38Xks1OktA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/37_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit38">O. Alvizo and S. L. Mayo, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2008, <span class="bold">105</span>, 12242 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0805858105" title="DOI Link to resource 10.1073/pnas.0805858105">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhtVOhtrrN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/38_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit39">L. Jiang, B. Kuhlman, T. Kortemme and D. Baker, <span class="italic">Proteins: Struct., Funct., Bioinf.</span>, 2005, <span class="bold">58</span>, 893 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proteins:%20Struct.,%20Funct.,%20Bioinf.%5BJour%5D%20AND%2058%5Bvolume%5D%20AND%20893%5Bpage%5D%20and%202005%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/39_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit40">L. Jiang, E. A. Althoff, F. R. Clemente, L. Doyle, D. Rothlisberger, A. Zanghellini, J. L. Gallaher, J. L. Betker, F. Tanaka, C. G. Barbas III, D. Hilvert, K. N. Houk, B. L. Stoddard and D. Baker, <span class="italic">Science</span>, 2008, <span class="bold">319</span>, 1387 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1152692" title="DOI Link to resource 10.1126/science.1152692">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXislSkt74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/40_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit41">L. A. Joachimiak, T. Kortemme, B. L. Stoddard and D. Baker, <span class="italic">J. Mol. Biol.</span>, 2006, <span class="bold">361</span>, 195 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2006.05.022" title="DOI Link to resource 10.1016/j.jmb.2006.05.022">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XntFSgtbo%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/41_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit42">C. T. Saunders and D. Baker, <span class="italic">J. Mol. Biol.</span>, 2005, <span class="bold">346</span>, 631 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2004.11.062" title="DOI Link to resource 10.1016/j.jmb.2004.11.062">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXntVSgtw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/42_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit43">X. Fu, J. R. Apgar and A. E. Keating, <span class="italic">J. Mol. Biol.</span>, 2007, <span class="bold">371</span>, 1099 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2007.04.069" title="DOI Link to resource 10.1016/j.jmb.2007.04.069">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXot12rt78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/43_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit44">I. W. Davis, W. B. Arendall, D. C. Richardson and J. S. Richardson, <span class="italic">Structure</span>, 2006, <span class="bold">14</span>, 265 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.str.2005.10.007" title="DOI Link to resource 10.1016/j.str.2005.10.007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XhtlKltr8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/44_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit45">E. L. Humphris and T. Kortemme, <span class="italic">Structure</span>, 2008, <span class="bold">16</span>, 1777 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.str.2008.09.012" title="DOI Link to resource 10.1016/j.str.2008.09.012">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhsVOis7rO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/45_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit46">I. Georgiev and B. R. Donald, <span class="italic">Bioinformatics</span>, 2007, <span class="bold">23</span>, i185 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/bioinformatics/btm197" title="DOI Link to resource 10.1093/bioinformatics/btm197">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXos1yrtrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/46_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit47">I. Georgiev, D. Keedy, J. S. Richardson, D. C. Richardson and B. R. Donald, <span class="italic">Bioinformatics</span>, 2008, <span class="bold">24</span>, i196 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/bioinformatics/btn169" title="DOI Link to resource 10.1093/bioinformatics/btn169">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXnvV2rsb8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/47_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit48">A. Shmygelska and M. Levitt, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2009, <span class="bold">106</span>, 1415 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0812510106" title="DOI Link to resource 10.1073/pnas.0812510106">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXhvV2qtbo%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/48_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit49">G. Grigoryan, A. W. Reinke and W. E. Keating, <span class="italic">Nature</span>, 2009, <span class="bold">458</span>, 859 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature07885" title="DOI Link to resource 10.1038/nature07885">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXksFGgsbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/49_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit50">A. Korkegian, M. E. Black, D. Baker and B. L. Stoddard, <span class="italic">Science</span>, 2005, <span class="bold">308</span>, 857 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1107387" title="DOI Link to resource 10.1126/science.1107387">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXjvVant7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/50_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit51">E. Yosef, R. Politi, M. H. Chio and J. M. Shifman, <span class="italic">J. Mol. Biol.</span>, 2009, <span class="bold">385</span>, 1470 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2008.09.053" title="DOI Link to resource 10.1016/j.jmb.2008.09.053">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSmsLs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/51_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit52">J. J. Havranek and B. P. Harbury, <span class="italic">Nat. Struct. Biol.</span>, 2003, <span class="bold">10</span>, 45 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nsb877" title="DOI Link to resource 10.1038/nsb877">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38Xps1Sitrc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/52_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit53">D. D. Boehr, D. McElheny, H. J. Dyson and P. E. Wright, <span class="italic">Science</span>, 2006, <span class="bold">313</span>, 1638 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1130258" title="DOI Link to resource 10.1126/science.1130258">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XpsVOktL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/53_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit54">J. K. Lassila, J. R. Keeffe, P. Oelschlaeger and S. L. Mayo, <span class="italic">Protein Eng., Des. Sel.</span>, 2005, <span class="bold">18</span>, 161 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/protein/gzi015" title="DOI Link to resource 10.1093/protein/gzi015">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXjvFymsb0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/54_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit55">D. N. Bolon, C. A. Voigt and S. L. Mayo, <span class="italic">Curr. Opin. Chem. Biol.</span>, 2002, <span class="bold">6</span>, 125 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S1367-5931(02)00303-4" title="DOI Link to resource 10.1016/S1367-5931(02)00303-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XitlGgu70%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/55_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit56">B. D. Allen and S. L. Mayo, <span class="italic">J. Comput. Chem.</span>, 2010, <span class="bold">31</span>, 904 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXitFegsLo%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/56_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit57">M. Feig and C. L. Brooks III, <span class="italic">Curr. Opin. Struct. Biol.</span>, 2004, <span class="bold">14</span>, 217 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbi.2004.03.009" title="DOI Link to resource 10.1016/j.sbi.2004.03.009">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXjt1Cmt7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/57_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit58">T. Lazaridis, <span class="italic">Proteins: Struct., Funct., Genet.</span>, 2003, <span class="bold">52</span>, 176 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/prot.10410" title="DOI Link to resource 10.1002/prot.10410">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXlsFaht7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/58_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit59">P. Barth, J. Schonbrun and D. Baker, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2007, <span class="bold">104</span>, 15682 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0702515104" title="DOI Link to resource 10.1073/pnas.0702515104">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGns77K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/59_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit60">N. H. Joh, A. Min, S. Faham, J. P. Whitelegge, D. Yang, V. L. Woods and J. U. Bowie, <span class="italic">Nature</span>, 2008, <span class="bold">453</span>, 1266 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature06977" title="DOI Link to resource 10.1038/nature06977">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXnslGksLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/60_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit61">H. Yin, J. S. Slusky, B. W. Berger, R. S. Walters, G. Vilaire, R. I. Litvinov, J. D. Lear, G. A. Caputo, J. S. Bennett and W. F. Degrado, <span class="italic">Science</span>, 2007, <span class="bold">315</span>, 1817–1822 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1136782" title="DOI Link to resource 10.1126/science.1136782">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXjsFSitrc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/61_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit62">B. North, L. Cristian, X. Fu, J. D. Lear, J. G. Saven and W. F. Degrado, <span class="italic">J. Mol. Biol.</span>, 2006, <span class="bold">359</span>, 930 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmb.2006.04.001" title="DOI Link to resource 10.1016/j.jmb.2006.04.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xls1ehs7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/62_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit63">C. Jäckel, P. Kast and D. Hilvert, <span class="italic">Annu. Rev. Biophys.</span>, 2008, <span class="bold">37</span>, 153 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Annu.%20Rev.%20Biophys.%5BJour%5D%20AND%2037%5Bvolume%5D%20AND%20153%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/63_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit64">T. Treynor, C. L. Vizcarra, D. Nedelcu and S. L. Mayo, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2007, <span class="bold">104</span>, 48 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0609647103" title="DOI Link to resource 10.1073/pnas.0609647103">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXjt1Krsw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/64_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
      <li><span id="cit65">M. M. Tirion, <span class="italic">Phys. Rev. Lett.</span>, 1996, <span class="bold">77</span>, 1905 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevLett.77.1905" title="DOI Link to resource 10.1103/PhysRevLett.77.1905">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28XlsVGjt7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003405/b810924a/65_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003405"/></a>.</span></li>
    </ol>
    
    
    
    
    
  <hr/><table><tr><td><h3>Footnotes</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">Replica Exchange (RE) is a sampling method where multiple simulations are carried out in parallel, and exchange between neighboring replica pairs is performed under the principle of detailed balance to facilitate exploration of the conformational space. For temperature RE, each replica has a different temperature. Replicas with higher temperatures will be able to cross energy barrier more rapidly and therefore facilitate sampling of configuration space of the lowest temperature through replica exchange. Hamiltonian RE uses different Hamiltonian (interactions) instead of temperature for each replica to facilitate sampling. In both cases, one replica (usually the first one) is simulated with the interested target parameter (temperature or Hamiltonian). More details can be found in <span class="italic">J. Chem. Phys.</span>. 116<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>9058–9067, and references wherein.</span></span></td></tr><tr><td><span class="sup_ref">‡ <span id="fn2">Linear Programming is a mathematical method for optimization of a given model that is represented by linear equations. In the special case when variables in a model are integers, it is called Integer Linear Programming (ILP). <span class="CH"><a class="simple" title="ChEBI Ontology link for: Protein" href="javascript:popupOBO('CHEBI:36080','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Protein</a></span> design, as it is formulated in <a title="Select to navigate to references" href="#cit48">ref. 48</a>, is a special case of ILP called binary integer programming as occupation of each site by a specific <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b810924a','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> is either 1 or 0. See <a title="Select to navigate to references" href="#cit48">ref. 48</a> and reference wherein for more details.</span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2010</b></td></tr></table></div></div></div></body><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>