<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Machine learning for high performance organic solar cells: current scenario and future prospects  - Energy &amp; Environmental Science (RSC Publishing) DOI:10.1039/D0EE02838J</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2021/ee/d0ee02838j"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":49,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Asif Mahmood"/><meta name="DC.Creator" content="Jin-Liang Wang"/><meta name="DC.title" content="Machine learning for high performance organic solar cells: current scenario and future prospects "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2021/01/26"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D0EE02838J"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Machine learning for high performance organic solar cells: current scenario and future prospects "/><meta name="citation_author" content="Asif Mahmood"/><meta name="citation_author" content="Jin-Liang Wang"/><meta name="citation_online_date" content="2020/11/26"/><meta name="citation_date" content="2021"/><meta name="citation_journal_title" content="Energy &amp; Environmental Science"/><meta name="citation_volume" content="14"/><meta name="citation_issue" content="1"/><meta name="citation_firstpage" content="90"/><meta name="citation_lastpage" content="105"/><meta name="citation_doi" content="10.1039/D0EE02838J"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2021/ee/d0ee02838j"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2021/ee/d0ee02838j"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2021/ee/d0ee02838j"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2021/ee/d0ee02838j">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d0ee01860k)" href="/en/content/articlehtml/2021/ee/d0ee01860k" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d0ee02491k)" href="/en/content/articlehtml/2021/ee/d0ee02491k" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div> </div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D0EE02838J">10.1039/D0EE02838J</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1754-5706/2008">Energy Environ. Sci.</a></span>, 2021, <strong>14</strong>, 90-105</div><h1 id="sect201"><span class="title_heading">Machine learning for high performance organic solar cells: current scenario and future prospects</span></h1><p class="header_text">
      <span class="bold">
        
          
            Asif 
            Mahmood
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Asif Mahmood (orcid.org/0000-0001-9412-1011) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-9412-1011"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><span class="bold"> and </span>
      <span class="bold">
        
          
            Jin-Liang 
            Wang
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Jin-Liang Wang (orcid.org/0000-0001-5726-3336) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-5726-3336"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*
      <br/><span class="italic">Key Laboratory of Cluster Science of Ministry of Education, Beijing Key Laboratory of Photoelectronic/Electrophotonic Conversion Materials, School of Chemistry and Chemical Engineering, Beijing Institute of Technology, Beijing, 100081, China. E-mail: <a href="mailto:jinlwang@bit.edu.cn">jinlwang@bit.edu.cn</a></span>
    </p><div id="art-admin"><span class="italic bold">Received 
      4th September 2020
    </span><span class="bold italic">, Accepted 25th November 2020</span></div><p class="bold italic">First published on 26th November 2020</p><hr/><div class="abstract"><h2>Abstract</h2><p>Machine learning (ML) is a field of computer science that uses algorithms and techniques for automating solutions to complex problems that are hard to program using conventional programming methods. Owing to the chemical versatility of organic building blocks, a large number of organic semi-conductors have been used for organic solar cells. Selecting a suitable organic semi-conductor is like searching for a needle in a haystack. Data-driven science, the fourth paradigm of science, has the potential to guide experimentalists to discover and develop new high-performance materials. The last decade has seen impressive progress in materials informatics and data science; however, data-driven molecular design of organic solar cell materials is still challenging. The data-analysis capability of machine learning methods is well known. This review is written about the use of machine learning methods for organic solar cell research. In this review, we have outlined the basics of machine learning and common procedures for applying machine learning. A brief introduction on different classes of machine learning algorithms as well as related software and tools is provided. Then, the current research status of machine learning in organic solar cells is reviewed. We have discussed the challenges in anticipating the data driven material design, such as the complexity metric of organic solar cells, diversity of chemical structures and necessary programming ability. We have also proposed some suggestions that can enhance the usefulness of machine learning for organic solar cell research enterprises.</p></div><hr/><table><tr><td class="biogPlate"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-p1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-p1.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-p1.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Asif Mahmood</span>
                </span></p></td><td><i><p>Asif Mahmood, received his PhD degree from the National Center for Nanoscience and Technology (NCNST), China. Currently, he is a Post-Doctoral at School of Chemistry and Chemical Engineering, Beijing Institute of Technology under the supervision of Prof. Jin-Liang Wang. His research interests include the design and synthesis of organic semi-conductor materials for organic solar cells and computational analysis of organic solar cells.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-p2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-p2.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-p2.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Jin-Liang Wang</span>
                </span></p></td><td><i><p>Jin-Liang Wang, received his PhD degree in College of Chemistry and Molecular Engineering, Peking University under the supervision of Prof. Jian Pei, from 2003-2008. From 2008 to 2012, he was a Postdoctoral research fellow in The University of Akron and The University of North Carolina at Chapel Hill. In 2013, he was awarded the junior thousand talent award and joined Beijing Institute of Technology as a full professor. His major is organic optoelectronic materials chemistry and his interests focus on synthesis of functional organic molecular materials for optoelectronic molecular devices and investigation of the relationship of chemical <span class="bold">s</span>tructure, computational analysis, optimization of film morphology and device performances.</p></i></td></tr></table><hr/>
    
      <br/><table class="box_border"><tr><td>
        
          <h2 id="sect246"><span class="a_heading">Broader context</span></h2>
          <span>To provide an environment friendly solution to the energy crisis, extensive research in various fields of solar cells is going on. Organic solar cells are the most prestigious candidate. After tremendous success of machine learning in language translation and image recognition, it has gained significant attention from material scientists. Finding efficient materials for organic solar cells is a prime task for further improvement. An innovative approach is required to replace the tedious and expensive trial and error method because a large number of organic semi-conductors are possible due to the chemical versatility of organic building blocks. In this review, we have discussed the current state of machine learning assisted material design for organic solar cells. There are serious challenges for the scientific community to accelerate the data-driven research paradigm. However, this field has the potential to overcome obstacles. We have proposed suggestions to overcome these obstacles. Machine learning based design can be a cost-effective way to foster innovation in organic solar cell research.</span>
        
      </td></tr></table><br/>
    
    
      
      <h2 id="sect249"><span class="a_heading">1. Introduction</span></h2>
      <span>Organic solar cells (OSCs) have gained huge attention due to their unique qualities such as lightweight, transparency, flexibility and low-cost.<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1–5</span></sup></a> In recent years, the power conversion efficiency (PCE) has significantly increased.<a title="Select to navigate to reference" href="#cit6"><sup><span class="sup_ref">6–8</span></sup></a> The great success is the result of device architecture design, design of active layer materials and processing.<a title="Select to navigate to reference" href="#cit9"><sup><span class="sup_ref">9–13</span></sup></a> New design ideas are based on knowledge obtained from expensive and time-consuming synthesis, characterization and device optimization. The results are further used for redesigning.<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> This approach leads to an extended iterative cycle that not always leads to high performance in the end. A large number of donors and acceptors have been tried for organic solar cell devices. Still, an infinite number of candidate materials are possible due to intrinsic degrees of freedom in organic molecules. Selection of materials is carried out through a trial and error method that is time consuming and laborious. From chemical structure, it is hard to guess whether a material will work well or not because the working of organic solar cells is complex. It is based on light absorption, formation, migration, and dissociation of excitons and transportation of generated charges to respective electrodes. Complicated multiple mechanisms and donor–acceptor morphology make it more difficult to design a rule.<a title="Select to navigate to reference" href="#cit15"><sup><span class="sup_ref">15–17</span></sup></a> Human experts are required to select suitable candidates, synthesize and fabricate devices, and then determine the performance. In this scenario, computer assisted prediction of performance of organic solar cells is an ultimate need.</span>
      <p class="otherpara">Scharber's model was developed to predict photovoltaic efficiency of organic solar cells.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> It uses few electronic parameters of active layer materials (donor and acceptor) for prediction. Its scope is limited due to assumptions based on it. There are so many obstacles in its extension <span class="italic">i.e.</span> it is hard to include other descriptors such structural, topological and thermodynamic. The performance of this model is quite poor.<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19–21</span></sup></a></p>
      <p class="otherpara">Data driven research has started a new paradigm and shown promising results within materials science. It can provide an understanding of fundamental factors that govern the performance of materials for specific applications. Material discovery through a data-driven paradigm is efficient and effective to utilize relevant information.<a title="Select to navigate to reference" href="#cit22"><sup><span class="sup_ref">22,23</span></sup></a> The systematic way of doing this is known as machine learning, which learns from past data and helps to screen the candidates for laboratory work. The discovery of excellent candidate materials for organic solar cells can be made fast and cheaper through multidimensional design by combining machine learning (ML), DFT calculations and available experimental data survey (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>).</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f1.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Computer assisted design and screening of materials for organic solar cells.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">The use of machine learning (ML) should be encouraged because it can provide a chance to get hidden information and trends. It uses a digitized form of chemical structures, and theoretically calculated and experimentally measured properties.<a title="Select to navigate to reference" href="#cit24"><sup><span class="sup_ref">24–26</span></sup></a> Further development in the ML model's explainability and interpretability will increase their usefulness. As compared to other fields of materials science, the use of ML for organic photovoltaic is limited. However, in the last year, the number of published papers has increased significantly as compared to the past. To the best of our knowledge, this is the first review article related to ML use in organic solar cell research.</p>
      <p class="otherpara">This review article is written to provide a comprehensive view of the machine learning methods used for organic photovoltaic materials. In Section 2, a brief description of machine learning and its classes is provided. In Section 3, basic steps to train ML models are given. In Section 4, the research status with respect to the applications of machine learning in organic solar cells is discussed. Section 5 discusses the limitations and drawbacks of machine learning and possible solutions.</p>
    
    
      
      <h2 id="sect270"><span class="a_heading">2. Machine-learning</span></h2>
      <span>Machine learning is a branch of artificial intelligence. Machine learning is based on the collection of sufficient amounts of reliable materials data (from experiment or computation) that best explain behavior or properties of materials or their applications to develop a model that is used to discover new materials without performing the same experiment or computation. Machine learning allows us to find hidden insights from data. Machine learning methods learn the rules from a given dataset and build a model to make predictions. A brief description of types of machine learning is given in <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>. Generally speaking, machine learning techniques can be divided into three groups, namely, supervised learning, unsupervised learning, and reinforcement learning.</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f2.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Different types of machine learning.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">The selection of an appropriate ML algorithm plays a key role since it significantly affects the prediction performance. There is no single best method for all cases. There are so many machine learning algorithms (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>). To achieve a highly effective model, it is crucial to select an appropriate algorithm that is mostly done on the basis of a trial and error process. Detailed literature search can help in this aspect. The learner should have enough knowledge of algorithms. It can help to select the best algorithms. A range of learning algorithms can be applied (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>), depending on the type of data and the question posed. The scope of this review article does not allow discussion of all the algorithms. Readers interested in a more comprehensive understanding of different algorithms can look at the following review articles.<a title="Select to navigate to reference" href="#cit27"><sup><span class="sup_ref">27–30</span></sup></a> Information about popular machine learning frameworks and libraries is summarized in <a title="Select to navigate to table" href="#tab1">Table 1</a>.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f3.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">Different types of machine learning algorithms.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
<div class="table_caption"><b>Table 1</b> <span id="tab1">Popular machine learning frameworks and libraries</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/></colgroup>
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Name</th>
                <th align="left" valign="bottom" class="border_black">Description</th>
                <th align="left" valign="bottom" class="border_black">Website</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td valign="top">Scikit-learn</td>
                <td valign="top">Free python-based library of machine learning algorithms</td>
                <td valign="top">
                  https://scikit-learn.org/stable/index.html
                </td>
              </tr>
              <tr align="left">
                <td valign="top">TensorFlow</td>
                <td valign="top">Open-source software library for dataflow and programming</td>
                <td valign="top">
                  https://www.tensorflow.org
                </td>
              </tr>
              <tr align="left">
                <td valign="top">CRAN-machine learning</td>
                <td valign="top">The Comprehensive R Archive Network (CRAN) is based on R programming language and contains verity of machine learning algorithms</td>
                <td valign="top">
                  https://cran.rstudio.com/web/views/MachineLearning.html
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Keras</td>
                <td valign="top">Open-source python library, supports data mining using deep learning algorithms</td>
                <td valign="top">
                  https://keras.io
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Weka</td>
                <td valign="top">Waikato Environment for Knowledge Analysis (Weka) offers verity of ML algorithms with graphical user interface</td>
                <td valign="top">
                  https://www.cs.waikato.ac.nz/ml/weka
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Amazon Web Services machine learning</td>
                <td valign="top">Offers fully managed solution for machine learning through various libraries such keras, TensorFlow, and Apache Spark ML <span class="italic">etc.</span></td>
                <td valign="top">
                  https://aws.amazon.com/machine-learning
                </td>
              </tr>
              <tr align="left">
                <td valign="top">IBM Watson machine learning</td>
                <td valign="top">Efficient and fast machine learning facilities based on Apache Spark libraries.</td>
                <td valign="top">
                  https://www.ibm.com/cloud/machine-learning
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Google artificial intelligence</td>
                <td valign="top">Cloud-based artificial intelligence machine learning framework.</td>
                <td valign="top">
                  https://ai.google/
                </td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
    
    
      
      <h2 id="sect338"><span class="a_heading">3. Basic steps of machine learning</span></h2>
      <span>Machine learning analysis consists of four steps.</span>
      
        
        <h3 id="sect342"><span class="b_heading">3.1. Sample collection</span></h3>
        <span>The first step is to collect data. Data can be from computational simulations and experimental measurements. Sometimes data cleaning or data transformation should be performed to avoid noise and inconsistency. Data can be split in various ratios; split sizes can also differ. Depending on the situation, it could be 60<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>40, 70<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>30 and 80<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>20, or can even be 90<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>10. The data can be split into training and testing sets in a number of ways. The simplest way is to select non-overlapping subsets of data while preserving the order of the data records, <span class="italic">i.e.</span> first 70% as the training set and the remaining 30% as the test set. However, this method will be problematic when the response is not evenly distributed. Random sampling is another way, selecting data randomly for the training set and test set. The range of the response values of both subsets should be covered from the lowest to the highest value. The data split strongly influences the performance of a model.<a title="Select to navigate to references" href="#cit31"><sup><span class="sup_ref">31</span></sup></a> High performance of a model can be due to chance variations in the training and testing sets rather than a robust association between the features and the target variable.<a title="Select to navigate to references" href="#cit32"><sup><span class="sup_ref">32</span></sup></a></span>
        <p class="otherpara">The required quantity of data varies model to model but a general rule of thumb is that at least 50 data points are necessary for a reasonable ML model. However, some models such as neural networks require much larger quantities. The data quantity and quality are perhaps the main challenges in the application of ML in materials science. A large fraction of data is available in journal publications.</p>
      
      
        
        <h3 id="sect350"><span class="b_heading">3.2. Data preparation and processing</span></h3>
        <span>As the machine learning model learns the hidden patterns of given data and predicts the pattern in new data. To improve the quality of modelling, data are cleaned to find missing data and outliers. If the data is better, the ML model will perform well.</span>
        <p class="otherpara">Values of different descriptors fall in different scales. It is better to normalize the values, as this will make it easy to find the difference between observations and use within a single algorithm. If the number of features (descriptors) is higher than the number of observations or features highly correlate with each other, then dimensionality reduction tools are used. Principal component analysis (PCA), discriminant analysis (LDA) and independent component analysis (ICA) are the most famous ones. They help to reduce the dimension of the feature space and identify the most relevant features. These tools also visualize the data.</p>
      
      
        
        <h3 id="sect355"><span class="b_heading">3.3. Model building</span></h3>
        <span>In organic solar cells, a complex relationship exists between parameters and performance. The accuracy and generalization ability of a machine learning model strongly depends on the machine learning algorithm. Every algorithm has its own advantages and disadvantages. Classification, clustering, regression and probability estimation are the most widely used types of ML algorithms for materials science. Classification, clustering and regression are used for material property prediction. Probability estimation is used for new materials discovery.</span>
      
      
        
        <h3 id="sect359"><span class="b_heading">3.4. Model evaluation</span></h3>
        <span>A model will be considered good, if it performs well on both the training and test data. Statistical analysis is used to check the performance of machine-learning models. For this purpose, root mean squared error (RMSE) and coefficient of determination (<span class="italic">R</span><small><sup>2</sup></small>) are used.<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn1"/><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-t1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-t1.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-t1.gif"/></a></td><td class="rightEqn">(1)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn2"/><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-t2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-t2.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-t2.gif"/></a></td><td class="rightEqn">(2)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn3"/><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-t3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-t3.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-t3.gif"/></a></td><td class="rightEqn">(3)</td></tr></table><span class="italic">x</span><small><sub><span class="italic">i</span></sub></small> is the predicted value of the model, <span class="italic">y</span><small><sub><span class="italic">i</span></sub></small> is the target variable and Var(<span class="italic">y</span>) is the variance of the sample data. The coefficient of determination (<span class="italic">R</span><small><sup>2</sup></small>) indicates the prediction ability of the model. Its value ranges between 0 and 1, and a value close to one indicates more prediction accuracy.</span>
        <p class="otherpara">An alternative approach for estimating the model's performance is splitting the available data into training/test sets several times, each time with a different group of observations in each set. This is a way to check performance several times, then sum up for each train/test split. This process also provides us with a measure of variability and stability of the model performance. This method can be executed using the cross validation (CV) and bootstrapping procedures.</p>
      
    
    
      
      <h2 id="sect384"><span class="a_heading">4. Machine learning analysis of organic solar cells</span></h2>
      <span>For the successful screening of potential candidates for organic solar cells, it is a prerequisite to understand the relationships between molecular properties and PCE. However, it is also important to study the relationship between molecular properties and specific device performance parameter to fulfil the requirements of a device for a particular application such as solar-to-fuel energy conversion (high open-circuit voltage (<span class="italic">V</span><small><sub>OC</sub></small>)) or solar window applications (high short circuit current (<span class="italic">J</span><small><sub>SC</sub></small>)). In this regard, ML has wide scope in organic solar cells research. The prediction ability of a ML model strongly depends on the type of used descriptors. The descriptors are the languages that researchers use to speak with the database. They play a significant role in making trustful predictions. If the target property is not clear then selection of candidate descriptors will become difficult.<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a></span>
      <p class="otherpara">Generally, a material property strongly depends on specific factors. Selection of suitable descriptors for specific properties is a crucial step before applying the ML process, especially microscopic descriptors that are experimentally and computationally expensive to determine. A good material descriptor should at least meet the following three criteria: It should be (i) a unique characterization of the material, (ii) sensitive to the target property, and (iii) easy to calculate.</p>
      <p class="otherpara">We have grouped the reported studies on the basis of input data. However, this grouping is not highly accurate due to the use of multiple types of inputs in some studies.</p>
      
        
        <h3 id="sect395"><span class="b_heading">4.1. Molecular descriptors</span></h3>
        <span>Molecular descriptors are driven from a compound's molecular structure, and represent physical and chemical properties of a molecule. They range from simple features such as count of specific atoms to more complex features such as charge distribution. There are thousands of molecular descriptors of various categories, zero-dimensional (0D) to three-dimensional (3D).<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> 0D descriptors describe molecular information without topological or atom connectivity inferences such as atom number, atom types, and molecular weight. 1D descriptors describe chemical fragment types and counts. In contrast, 2D descriptors define topological and topo-chemical molecular features. Finally, 3D descriptors capture geometrical data and include conformation data such as molecular volume and partial surface charges. An ideal expression must provide most of the features of the molecule and be free from redundant information. Various expressions for the same molecule comprise vastly different chemical information, or this information is presented in different abstract levels. There are many expressions, some are shown in <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>. Molecular descriptors are easy and fast to calculate, which allows the rapid screening of a huge number of materials. There are many free and commercial tools to calculate molecular descriptors. A list of software and website resources is presented in <a title="Select to navigate to table" href="#tab2">Table 2</a>.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f4.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Different types of molecular representations applied to one molecule. Adopted with permission from <a title="Select to navigate to references" href="#cit34">ref. 34</a> Copyright 2018, AAAS.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        
<div class="table_caption"><b>Table 2</b> <span id="tab2">Tools to calculate molecular descriptors and fingerprints</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Name</th>
                  <th align="left" valign="bottom" class="border_black">Description</th>
                  <th align="left" valign="bottom" class="border_black">Website</th>
                  <th align="left" valign="bottom" class="border_black">Ref.</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">DRAGON</td>
                  <td valign="top">Software, it can calculate 5270 descriptors</td>
                  <td valign="top">
                    http://www.talete.mi.it
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit36">36</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">E-DRAGON</td>
                  <td valign="top">Web version of DRAGON can calculate over 3000 descriptors for molecules with atoms 150 atoms</td>
                  <td valign="top">
                    http://www.vcclab.org/lab/edragon/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit37">37</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Mold2</td>
                  <td valign="top">Software to calculate 779 descriptors</td>
                  <td valign="top">
                    https://www.fda.gov/science-research/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit38">38</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Mordred</td>
                  <td valign="top">Software, it can calculate over 1800 2D and 3D descriptors</td>
                  <td valign="top">
                    https://github.com/mordred-descriptor/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit39">39</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PaDEL-Descriptor</td>
                  <td valign="top">Tool, can calculate 1444 1D and 2D descriptors, 431 3D descriptors, and 12 different types of fingerprint</td>
                  <td valign="top">
                    http://www.yapcwsoft.com/dd/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit40">40</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">MOE</td>
                  <td valign="top">tool to calculate over 300 topological, physical and structural descriptors</td>
                  <td valign="top">
                    http://www.chemcomp.com/
                  </td>
                  <td valign="top">
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">MOLGEN QSPR</td>
                  <td valign="top">software to calculate 708 arithmetical, topological, and geometrical descriptors</td>
                  <td valign="top">
                    http://molgen.de/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit41">41</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">ChemoPY</td>
                  <td valign="top">Python package to calculate 1135 2D and 3D descriptors</td>
                  <td valign="top">
                    http://code.google.com/p/pychem/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit42">42</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BlueDesc</td>
                  <td valign="top">software to calculate 174 descriptors</td>
                  <td valign="top">
                    http://www.ra.cs.uni-tuebingen.de/software/bluedesc/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit43">43</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PowerMV</td>
                  <td valign="top">PC software for calculation of 1000 descriptors</td>
                  <td valign="top">
                    https://www.niss.org/research/software/
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit44">44</a>
                  </td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">In 2011, Aspuru Guzik's group used ML for the discovery of promising OPV donor materials.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> Current–voltage properties of 2.6 million molecular motifs were modelled using linear regression. On the basis of feature predictions, they identified benzothiadiazole, pyridinethiadazole and thienopyrrole as the top candidates.</p>
        <p class="otherpara">Zhang <span class="italic">et al.</span> established a data set of 111<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 molecules and trained an ML model using random forest (RF).<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> Through this model, they predicted the LUMO and HOMO with below 0.16 eV error without any DFT calculations. This can speed up high-throughput screening of organic semi-conductors for solar cells.</p>
        <p class="otherpara">Su <span class="italic">et al</span>. designed a series of novel acceptors based on multi-conformational bistricyclic aromatic ene (BAE) derivative.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> They have predicted their PCE using an ML model build using experimental data through cascaded support vector machine (CasSVM). The CasSVM model is a novel two-level network (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>), which consists of three subset SVM models taking <span class="italic">J</span><small><sub>SC</sub></small>, <span class="italic">V</span><small><sub>OC</sub></small>, and FF, respectively, as outputs in the first level. Then, the second level was used to establish the relationship between the first level outputs and the ultimate endpoint PCE. The best established CasSVM model has predicted the PCE value of OPVs with mean absolute error (MAE) of 0.35 (%), which is approximately 10% (3.89%) of the mean PCE. The value of <span class="italic">R</span><small><sup>2</sup></small> was 0.96. This approach can be very useful for experimental chemists to screen the potential candidates before synthesis.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f5.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">The structure of the cascaded SVM QSAR model. Sub-1–3 are the input descriptors, respectively, for <span class="italic">J</span><small><sub>SC</sub></small>, <span class="italic">V</span><small><sub>OC</sub></small>, and FF. SVM1∼4 are the subset SVM models used for the prediction of <span class="italic">J</span><small><sub>SC</sub></small>, <span class="italic">V</span><small><sub>OC</sub></small>, FF and PCE, respectively. Reprinted with permission from <a title="Select to navigate to references" href="#cit47">ref. 47</a> Copyright 2018, Wiley-VCH Verlag GmbH &amp; Co. KGaA.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect515"><span class="b_heading">4.2. Molecular fingerprints</span></h3>
        <span>Molecular fingerprints are a digitized form of chemical structures. They do not include structural data such as coordinates. They are used to search the database and find the similarity between two molecules. There are several methods to transfer a molecular structure into digitized form. Keys-based fingerprints, circular fingerprints and topological or path-based fingerprints are famous types of fingerprints. These types have further subtypes. We are suggesting a few articles for further reading.<a title="Select to navigate to reference" href="#cit48"><sup><span class="sup_ref">48–50</span></sup></a></span>
        <p class="otherpara">In recent years, non-fullerene acceptors have been extensively used in organic photovoltaics (OPVs).<a title="Select to navigate to reference" href="#cit3"><sup><span class="sup_ref">3–5,51–55</span></sup></a> In 2017, Aspuru-Guzik <span class="italic">et al.</span> collected a data set of over 51<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 non-fullerene acceptors based on benzothiadiazole (BT), diketopyrrolopyrroles (DPPs), perylene diimides (PDIs), tetraazabenzodifluoranthenes (BFIs) and fluoranthene-fused imides from Harvard Clean Energy Project (HCEP).<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a> A data set of 94 experimentally reported molecules was used to calibrate the DFT methods for the calculation of HOMO and LUMO values of new non-fullerene acceptors. Gaussian process regression was used instead of the widely used linear regression because of the absence of a linear trend. They used the Scharber model to calculate the PCE for organic solar cells based on designed non-fullerene acceptors and poly[<span class="italic">N</span>-90-heptadecanyl-2,7-carbazole-<span class="italic">alt</span>-5,5-(40,70-di-2-thienyl-20,10,30-benzothiadiazole)] (PCDTBT), a standard electron-donor material. DFT calculated HOMO and LUMO values of acceptors and experimentally reported HOMO and LUMO values of PCDTBT were used as input for the Scharber model. To check the PCE prediction ability of the Scharber model, they utilized a set of 49 reported experimental values and predicted their Scharber model PCE values. A weak correlation was found (<span class="italic">r</span> = 0.43 and <span class="italic">R</span><small><sup>2</sup></small> = 0.11).</p>
        <p class="otherpara">Prediction of PCE and specific device properties are equally important. To improve a specific property, it is important to find the relationship between the specific property and molecular descriptors.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> For example, most high performing OSC devices showed lower open-circuit voltage (<span class="italic">V</span><small><sub>OC</sub></small>). In BHJ OSCs, charge separation is typically associated with large voltage losses because of the extra energy required to split excitons into free carriers. This voltage loss in OSCs with high performance is usually around 0.6 V, which is 0.2–0.3 V higher than that for c-Si and GaAs-based solar cells.<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> Non-fullerene acceptors with extended thin-film absorption and suitable energy levels can help to achieve a balanced trade-off between <span class="italic">V</span><small><sub>OC</sub></small> and <span class="italic">J</span><small><sub>SC</sub></small>.<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> Their structural versatility allows highly tunable absorption and molecular energy levels. Machine learning can speed up the screening of suitable materials. Predication of specific parameters will be helpful to further enhance the PCE. Aspuru Guzik <span class="italic">et al.</span> calibrated the <span class="italic">V</span><small><sub>OC</sub></small> and <span class="italic">J</span><small><sub>SC</sub></small> values, which were calculated from the Scharber model with available experimental data using structural similarity.<a title="Select to navigate to references" href="#cit21"><sup><span class="sup_ref">21</span></sup></a> Information on the molecular graph is extracted with extended connectivity fingerprints, and exploited using a Gaussian process. This calibration method reduced the functional dependence of the calculated properties, and it will help to ease the high-throughput virtual screening.</p>
        <p class="otherpara">In 2019, Sun <span class="italic">et al.</span> collected the dataset of 1719 donor materials.<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> They tested different inputs including seven types of molecular fingerprints, two types of descriptors, ASCII strings and images. Classification of donor materials was carried out in two categories “low” and “high” PCE. Models developed using fingerprints showed the best performance in predicting the PCE class (86.76% accuracy). They have verified the ML results by synthesizing 10 donor materials. The model classified eight molecules into the correct category. Experimental results were in good agreement with predicted results. Classification in just two categories (0–2.9%, 3–14.6%) is very easy as compared to predicating the PCE of individual semi-conductors. This study does not have much practical value <span class="italic">i.e.</span> the second category is too wide.</p>
        <p class="otherpara">In the same year, Saeki <span class="italic">et al.</span> extracted 2.3 million molecules from the Harvard Clean Energy Project database.<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> 1000 molecules were selected on the basis of calculated PCE. They used molecular access system (MACCS) fingerprints and the extended connectivity fingerprint (ECFP6) key to train the ML model. 149 molecules were selected using random forest (RF) screening (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>). The RF method for PCE prediction showed an accuracy of 48%. They selected one polymer on the basis of synthetic feasibility. A solar cell device based on the new polymer showed a PCE of 0.53% that is much lower than the RF prediction (5.0–5.8%). There are two reasons behind this failure. Firstly, the PCE calculated from the Scharber model is used to train the RF model. The performance of the Scharber model is very poor. Secondly, the structures of polymer donors reported in the literature are more complex than that of semi-conductors in the HCEP database. Even if we ignore these factors, the PCE prediction accuracy of the RF model is very low. Therefore, the ML model should be more accurate and multiple materials should be synthesized for experimental validation.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f6.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">Scheme of polymer design by combining RF screening and manual screening/modification. Adopted with permission from <a title="Select to navigate to references" href="#cit60">ref. 60</a> Copyright 2018, American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Schmidt and co-workers collected a dataset of 3,989 monomers and trained a model using a grammar variational autoencoder (GVA).<a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> The trained model can predict the lowest optical transition energy and lowest unoccupied molecular orbital (LUMO) without the knowledge of the atomic positions. Moreover, this model can generate new molecular structures with the desired LUMO and optical gap energies. The prediction accuracy of the deep neural network (DNN) was higher than that of GVA. However, it is necessary to perform DFT calculations to find the atomic position that is necessary to predict the LUMO. So, in the case of the DNN model, it impossible to skip the DFT calculations.</p>
        <p class="otherpara">Paul <span class="italic">et al.</span> used extremely randomized tree to predict the HOMO values for donor compounds.<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> Their proposed models showed better results than neural networks trained on molecular fingerprints, SMILES, Chemception and Molecular Graph.</p>
        <p class="otherpara">Peng and Zhao used convolutional neural network (CNN) to construct generative and prediction models for the design and analysis of non-fullerene acceptors.<a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> Different molecular descriptors have been used such as extended-connectivity fingerprints, coulomb matrix, molecular graph, bagof-bonds, and SMILES string. The depth of convolutional layers influences the diversity of the generated NFAs. Quantum chemistry calculations were performed to verify the predicted molecules. In the prediction model, the dilated convolution layers are adopted for feature extraction, and the attention mechanism is used as an interpretable module. The authors concluded that graph representation is better than a string representation.</p>
        <p class="otherpara">In most of the experimental studies, donor and acceptor materials are optimized separately. Just optimization of one of the two components of the cell results in a limited exploration of the space of combinations. Troisi used ML to find the answer to the question: can components be optimized separately or should the optimization occur simultaneously.<a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> They used molecular fingerprints as input. Combinations of 262 donors (D) and 76 acceptors (A) were collected from the literature. They have predicted the PCE of BHJ solar cells based on combination of donor and acceptor materials. A high accuracy (<span class="italic">r</span> = 0.78) was obtained even though the data set was small. The best combination is proposed for experimental investigation. Min <span class="italic">et al.</span> reported an excellent work. They trained five ML models using linear regression (LR), multiple linear regression (MLR), boosted regression trees (BRT), RF, and ANN algorithms. 565 D/A combinations collected from the literature were used as training data sets.<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> For polymer-NFA OSC devices, correlation between D/A pairs and the PCE predication is validated. BRT and RF models showed higher prediction ability, with an <span class="italic">r</span> value of 0.71 and 0.70, respectively. These two models were used to predict PCE of &gt;32 million D/A combinations. Six D/A pairs were selected and incorporated into OSC devices. Experimental PCEs were close to the predicted ones. All the synthesized non-fullerene acceptors belong to high performing Y6 series. The workflow of the whole study is summarized in <a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a>.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f7.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">Workflow of building, application, and evaluations of machine learning methods. (a) Scheme of collecting experimental data and converting chemical structure to digitized data. (b) Scheme of machine training, predicting, and method evaluation. Adopted from <a title="Select to navigate to references" href="#cit65">ref. 65</a> Creative Commons Attribution 4.0 International License.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect573"><span class="b_heading">4.3. Images</span></h3>
        <span>Machine learning has achieved tremendous success in image recognition. It identifies features out of a complicated background and associates them with the same output. Sun <span class="italic">et al.</span> used a deep neural network that can recognize the chemical structures and automatically classify them to predict the PCE.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> Therefore, pictures of chemical structures were used without any transformation. This model was fast with low computational cost, and possible to run on a personal computer. It showed 91.02% accuracy in the prediction of PCE of donor materials. The working pattern of this study is given in <a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>. There are several drawbacks in this study; firstly, data from the Harvard Clean Energy Project (HCEP) is used to train the ML model for the screening of OSC materials. However, molecules reported in the literature are usually much more complicated than those in the HCEP. Secondly, DFT calculated energy levels are used as input for the Scharber model to estimate PCE. Both are not accurate. The working of organic solar cells is very complex, and the active layer materials, solubility, solvent additives, crystallinity and molecular orientation affect the performance of solar cells. In this study just, pictures of chemical structures of donor materials without any transformation were used as input. Such kind of approach cannot provide realistic results. Molecular descriptors are a better option as compared with just a picture of the structure. So far use of images as input for ML analysis of OSCs is rare (<a title="Select to navigate to table" href="#tab3">Table 3</a>).</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f8.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">Structure of the convolutional neural network (CNN). Reprinted with permission from <a title="Select to navigate to references" href="#cit66">ref. 66</a> Copyright 2018, Wiley-VCH Verlag GmbH &amp; Co. KGaA.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        
<div class="table_caption"><b>Table 3</b> <span id="tab3">A comparative overview of ML analysis of organic solar cells</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/><col/></colgroup>
              
              
              
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Data set</th>
                  <th align="left" valign="bottom" class="border_black">Source</th>
                  <th align="left" valign="bottom" class="border_black">Input</th>
                  <th align="left" valign="bottom" class="border_black">Method</th>
                  <th align="left" valign="bottom" class="border_black">ML models</th>
                  <th align="left" valign="bottom" class="border_black">Performance*</th>
                  <th align="left" valign="bottom" class="border_black">Experimental validation</th>
                  <th align="left" valign="bottom" class="border_black">Ref.</th>
                </tr>
              </thead>
              <tfoot align="left">
                <tr align="left">
                  <th colspan="8">*PCE unless mention. ** mean absolute errors. *** Between ML predicted and Scharber's model calculated PCE. AOC = Accuracy of classification.</th>
                </tr>
              </tfoot>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">2.3 million</td>
                  <td valign="top">HCEP</td>
                  <td valign="top">Descriptors</td>
                  <td valign="top">regression</td>
                  <td valign="top">Linear regression</td>
                  <td valign="top">0.84 <span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit45">45</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">111<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                  <td valign="top">Literature + database</td>
                  <td valign="top">Descriptors</td>
                  <td valign="top">Regression</td>
                  <td valign="top">RF</td>
                  <td valign="top">HOMO(0.85), LUMO (0.94) <span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit46">46</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">161</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Descriptors</td>
                  <td valign="top">Regression</td>
                  <td valign="top">CasSVM</td>
                  <td valign="top">0.96<span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit47">47</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">51<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                  <td valign="top">HCEP</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Regression</td>
                  <td valign="top">GPR</td>
                  <td valign="top">0.43(<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit20">20</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">2.3 million</td>
                  <td valign="top">HCEP</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Regression</td>
                  <td valign="top">GPR</td>
                  <td valign="top">0.65(<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit21">21</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">1719</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Classification (two groups)</td>
                  <td valign="top">RF</td>
                  <td valign="top">86.67% AOC</td>
                  <td valign="top">Yes (successful)</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit59">59</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">1000</td>
                  <td valign="top">HCEP</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Classification (four groups)</td>
                  <td valign="top">RF</td>
                  <td valign="top">48% AOC</td>
                  <td valign="top">Yes (failed)</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit60">60</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">3989</td>
                  <td valign="top">Quantum-Machine.org</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Regression</td>
                  <td valign="top">deep tensor network</td>
                  <td valign="top">HOMO(45) LUMO(31)**</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit61">61</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">350</td>
                  <td valign="top">HOPV15</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Regression</td>
                  <td valign="top">decision trees</td>
                  <td valign="top">HOMO (0.74) <span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">62</td>
                </tr>
                <tr align="left">
                  <td valign="top">51 000</td>
                  <td valign="top">HCEP</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Regression</td>
                  <td valign="top">CNN</td>
                  <td valign="top">0.91 (<span class="italic">r</span>)***</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit63">63</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">320</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Regression</td>
                  <td valign="top">KRR</td>
                  <td valign="top">0.78 (<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit64">64</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">565</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Fingerprints</td>
                  <td valign="top">Regression</td>
                  <td valign="top">BRT</td>
                  <td valign="top">0.71</td>
                  <td valign="top">Yes (successful)</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit65">65</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">5000</td>
                  <td valign="top">HCEP</td>
                  <td valign="top">Image</td>
                  <td valign="top">Classification (two groups)</td>
                  <td valign="top">CNN</td>
                  <td valign="top">91.02% AOC ***</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit66">66</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">270</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Microscopic properties</td>
                  <td valign="top">Regression</td>
                  <td valign="top">GB</td>
                  <td valign="top">0.79(<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit67">67</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">300</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Microscopic properties</td>
                  <td valign="top">Regression</td>
                  <td valign="top">GBRT</td>
                  <td valign="top">0.78(<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit68">68</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">566</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Microscopic properties</td>
                  <td valign="top">Regression</td>
                  <td valign="top">k-NN</td>
                  <td valign="top">0.72(<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit69">69</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">290</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Microscopic properties</td>
                  <td valign="top">Regression</td>
                  <td valign="top">GBRT</td>
                  <td valign="top">0.80 (<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit70">70</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">249</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Microscopic properties</td>
                  <td valign="top">Regression</td>
                  <td valign="top">kRR</td>
                  <td valign="top">0.68 (<span class="italic">r</span>)</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit19">19</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">2.3 million</td>
                  <td valign="top">HCEP</td>
                  <td valign="top">Energy level</td>
                  <td valign="top">Scharber's model</td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit71">71</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">380</td>
                  <td valign="top">Auto-generation</td>
                  <td valign="top">Energy level</td>
                  <td valign="top">Scharber's model</td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit72">72</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">135</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Energy level</td>
                  <td valign="top">regression</td>
                  <td valign="top">RF</td>
                  <td valign="top">0.80<span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit73">73</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">124</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Energy level</td>
                  <td valign="top">regression</td>
                  <td valign="top">RF</td>
                  <td valign="top">0.77<span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit74">74</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">121</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Energy level</td>
                  <td valign="top">regression</td>
                  <td valign="top">RF</td>
                  <td valign="top">0.77(<span class="italic">V</span><small><sub>OC</sub></small>)<span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit75">75</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">70</td>
                  <td valign="top">Literature</td>
                  <td valign="top">Energy level</td>
                  <td valign="top">regression</td>
                  <td valign="top">RF</td>
                  <td valign="top">0.69<span class="italic">R</span><small><sup>2</sup></small></td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit76">76</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">1800</td>
                  <td valign="top">Simulation</td>
                  <td valign="top">Simulated properties</td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit77">77</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">65<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                  <td valign="top">Simulation</td>
                  <td valign="top">Simulated properties</td>
                  <td valign="top">Classification</td>
                  <td valign="top">CNN</td>
                  <td valign="top">95.80%(<span class="italic">J</span><small><sub>SC</sub></small>) AOC</td>
                  <td valign="top">No</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit78">78</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                  <td valign="top">Simulation</td>
                  <td valign="top">Simulated properties</td>
                  <td valign="top">
                  </td>
                  <td valign="top">ANN</td>
                  <td valign="top">
                  </td>
                  <td valign="top">Yes</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit79">79</a>
                  </td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
      
      
        
        <h3 id="sect908"><span class="b_heading">4.4. Microscopic properties</span></h3>
        <span>The efficiency of OSCs depends on many microscopic properties of the organic materials such as optical gap, charge-carrier mobility, ionization potential of the donor, electron affinity of the acceptor, hole–electron binding energy <span class="italic">etc.</span> These microscopic descriptors are more realistic for solar cell applications as compared with simple topological descriptors. However, they are expensive to compute or experimentally determined. Ma <span class="italic">et al.</span> used 13 microscopic properties as descriptors to train a model for the prediction of PCE. They used the dataset of 270 small molecules.<a title="Select to navigate to references" href="#cit67"><sup><span class="sup_ref">67</span></sup></a> Different techniques such as random forest, gradient boosting and artificial neural network were used. The gradient boosting model showed an impressive performance (Pearson's coefficient (<span class="italic">r</span>) = 0.79). However, the used descriptors are computationally expensive such as polarizability and excited state, which is an obstacle for large-scale high-throughput virtual screening of potential molecules.</span>
        <p class="otherpara">Ma <span class="italic">et al.</span> used RF and Gradient boosting regression tree (GBRT) algorithms to predict device characteristics (<span class="italic">V</span><small><sub>OC</sub></small>, <span class="italic">J</span><small><sub>SC</sub></small>, and FF) through microscopic properties.<a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a><span class="italic">J</span><small><sub>SC</sub></small> (<span class="italic">r</span> = 0.78) and FF (<span class="italic">r</span> = 0.73) showed strong correlation with PCE, however, <span class="italic">V</span><small><sub>OC</sub></small> (<span class="italic">r</span> = 0.15) showed very weak correlation with PCE, these results are consistent with recently reported results.<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> The <span class="italic">J</span><small><sub>SC</sub></small> and FF are found to be poorly correlated (<span class="italic">r</span> = 0.33), with almost no correlation between <span class="italic">V</span><small><sub>OC</sub></small> and <span class="italic">J</span><small><sub>SC</sub></small> (<span class="italic">r</span> = −0.18) as well as <span class="italic">V</span><small><sub>OC</sub></small> and FF (<span class="italic">r</span> = −0.09).</p>
        <p class="otherpara">To analyze the effect of descriptor types on the prediction ability of ML models, Trois <span class="italic">et al.</span> trained <span class="italic">k</span>-Nearest Neighbors (<span class="italic">k</span>-NN), Kernel Ridge Regression (KRR) and Support Vector Regression (SVR) models on data of 566 donor/acceptor pairs collected from the literature.<a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> Both structural (topology properties) and physical descriptors (energy-levels, molecular size, light absorption and mixing properties) are used. The structural descriptors are the ones that contributed most to the ML models. Some physical properties showed high correlation with PCE but didn’t improve the model prediction ability because information they carry was already encoded in the structural descriptors.</p>
        <p class="otherpara">To achieve push–pull conjugated systems, different types of building blocks such as electron-deficient, electron-rich and π-spacer are used to design organic semi-conductors. Ma <span class="italic">et al.</span> performed ML modelling to screen 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 molecules designed from 32 building blocks.<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> The purpose of this study was to understand the effect of the nature and arrangement of building blocks. Descriptors were calculated from ground and excited states of candidate molecules. 126 potential candidates with predicted efficiencies ≥8% were selected on the basis of GBRT and ANN models. This approach is efficient to screen the potential candidates for organic solar cells.</p>
        <p class="otherpara">Troisi <span class="italic">et al.</span> used electronic and geometrical properties to train ML models to predict device parameters, and it performed much better than the Scharber model (Pearson's coefficient (<span class="italic">r</span>) = 0.68).<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a></p>
        <p class="otherpara">In organic solar cells, the thermodynamics of mixing of active layer materials controls the evolution of film morphology, and consequently, changes the charge-transport and light-harvesting, so controls the overall performance and stability of the final device.<a title="Select to navigate to reference" href="#cit80"><sup><span class="sup_ref">80,81</span></sup></a> It is important to study the relationship between molecular interaction parameters and phase behaviour of thin films. For this purpose, Perea <span class="italic">et al.</span> used the ANN model and the Flory–Huggins solution theory to study the phase evolution of polymers and fullerenes.<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> The ANN model was used to predict solubility parameters by surface charge distribution. Combined with solubility parameters, a figure of merit was established to describe the stability of polymer–fullerene blends (<a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>).</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f9.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">Computational flowchart describing the routine for determining the relative stability capable of describing the microstructure of polymer:fullerene blends. (i) Creation of the σ-profile from the conductor-like screening model (COSMO); (ii) σ-moments as extracted from COSMO are fed into an artificial neural network (ANN) to determine Hansen solubility parameters (HSPs); (iii) HSPs are used to calculate the qualitative Flory–Huggings interaction parameters (χ<small><sub>1,2</sub></small>); (iv) implementation of moiety-monomer-structure properties (reduced molar volumes/weights); (v) spinodal demixing diagrams resulting from polymer blend theory; and (vi) figure of merit (FoM) defined as the ratio of the Flory–Huggins intermolecular parameter and the spinodal diagram forms the basis of a relative stability metric. Adopted with permission from <a title="Select to navigate to references" href="#cit82">ref. 82</a> Copyright 2017, American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect963"><span class="b_heading">4.5. Energy levels</span></h3>
        <span>Energy levels of donor and acceptor materials are among the factors that affect the performance of organic solar cells. Energy level mismatch between the donor and acceptor reduces the PCE of OSCs due to large energy loss resulting from radiative recombination loss.<a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a></span>
        <p class="otherpara">In 2017, Aspuru Guzik's group investigated millions of molecular motifs using 150 million DFT calculations.<a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> PCE was predicted using Scharber's model,<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> and the calculated energy level was used as input. Candidates with a PCE of more than 10% were identified. In 2017, Imamura <span class="italic">et al.</span> reported the automatic generation of thiophene-based polymers from donor and acceptor units, estimation of the orbital levels by Hückel-based models and an evaluation of photovoltaic characteristics.<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> PCE was calculated using Scharber's model, but its performance is very poor.<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19–21</span></sup></a> Molecular descriptors and microscopic properties of semi-conductors were totally ignored.</p>
        <p class="otherpara">Min-Hsuan Lee performed Random Forest (RF) modelling on a database of &gt;100 bulk heterojunction solar cells and achieved high prediction accuracy (<span class="italic">R</span><small><sup>2</sup></small> of 0.85 and 0.80 for the training set and testing set, respectively).<a title="Select to navigate to references" href="#cit73"><sup><span class="sup_ref">73</span></sup></a> In this study, the number of descriptors was small (HOMO, LUMO and band gap). Inclusion of parameters such as solubility parameters, interaction parameters and surface energies as input for the ML model can further enhance the usefulness of similar studies.</p>
        <p class="otherpara">Various examples of ML of binary solar cells are discussed above. Generally, ternary OSCs show higher performance than binary OSCs. Binary OSCs are facing the problem of insufficient light harvesting due to narrow absorption range of organic semi-conductors. In ternary OSCs, the third component, can be a donor or an acceptor, and not only works as an additional absorber to enhance photon harvesting but also helps to achieve favourable morphology.<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> As the working of ternary solar cells is complex as compared with binary solar cells,<a title="Select to navigate to reference" href="#cit85"><sup><span class="sup_ref">85,86</span></sup></a> it is a challenging task to find ideal third components for ternary solar cells. Min-Hsuan Lee has trained an ML model for ternary solar cells using Random Forest, Gradient Boosting, <span class="italic">k</span>-Nearest Neighbors (<span class="italic">k</span>-NN), Linear-Regression and Support Vector Regression. The value of LUMO for the donor (D1) showed a noticeable linear correlation with PCE (<span class="italic">r</span> = −0.55), and the correlations of other indicators with PCE were weak.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> The value of <span class="italic">V</span><small><sub>OC</sub></small> is highly correlated with the HOMO of the donor (<span class="italic">r</span> = −0.54) and the LUMO of the donor (<span class="italic">r</span> = −0.54), respectively, suggesting that the energy levels of the donor need further consideration to find the origin of <span class="italic">V</span><small><sub>OC</sub></small> in ternary OSCs. The Random Forest model showed higher <span class="italic">R</span><small><sup>2</sup></small> (0.77 on test set) value among all ML methods. In another study, he trained the ML model to predict the <span class="italic">V</span><small><sub>OC</sub></small> of fullerene derivatives based ternary organic solar cells. The descriptors were the same as in a previous study.<a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> The Random Forest model showed an <span class="italic">R</span><small><sup>2</sup></small> value of 0.77. In both studies, just energy levels of organic semi-conductors were used as descriptors and other molecular descriptors and the thin film morphology effect were ignored. Therefore, it is required to develop a hybrid modelling framework that should include thin-film characteristics (<span class="italic">e.g.</span>, the appropriate ratio of three components) and fabrication conditions (<span class="italic">e.g.</span>, annealing temperature and solvent additive). Optimization of all the factors can enhance charge generation and reduce voltage loss, and resultantly can improve device efficiency.<a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> Theoretical analysis of the morphology of the three components is much more complex than that of two components.</p>
        <p class="otherpara">Tandem organic solar cells have shown superior power conversion efficiency (PCE). A tandem organic solar cell consists of two sub-cells. The major purposes of this device architecture are to widen the photon response range, and suppress the transmission loss and thermalization loss.<a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> It is more challenging to develop a relationship between efficiency and physical properties of active layer materials due to the great diversity of organic materials that leads to more candidate materials. To solve this problem, Min-Hsuan Lee used ML algorithms to predict the efficiency of tandem OSCs and identify sufficient bandgap combinations for tandem OSCs.<a title="Select to navigate to references" href="#cit76"><sup><span class="sup_ref">76</span></sup></a> Random Forest regression was used to perform machine learning using energy levels as input to predict the efficiency. The results indicate that energy offset in the LUMO level between the donor and the acceptor material should be optimized to improve the process of electron transfer and device performance.</p>
      
      
        
        <h3 id="sect1004"><span class="b_heading">4.6 Simulated properties</span></h3>
        <span>Film morphology controls the efficiency of organic solar cells. For further improvement of PCE, a clear understanding of the film morphology is essential. Along with the experimental work, mathematical simulation is another way to explore the film morphology and find the effect of various parameters on it. These kinds of simulations have two distinct stages: the representation stage and the mapping stage. In the first stage, a mathematical framework is developed to generate microstructures. In the second stage, the generated microstructures are related to a target property. The use of graph theory for microstructure analysis of OSCs is increasing.<a title="Select to navigate to reference" href="#cit89"><sup><span class="sup_ref">89–91</span></sup></a> For example, Ganapathysubramanian <span class="italic">et al.</span> studied morphology descriptors related to various processes in OSCs such as photon absorption, exciton diffusion, charge separation, and charge transport through a graph-based approach.<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a> High correlation between the graph-based approach and computationally intensive method was achieved. In another study, they used CNN to relate the film morphology with short-circuit current (<span class="italic">J</span><small><sub>SC</sub></small>).<a title="Select to navigate to references" href="#cit78"><sup><span class="sup_ref">78</span></sup></a> They solved the thermodynamically consistent CahnHilliard equation<a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a> for binary phase separation using an in-house finite element library.<a title="Select to navigate to references" href="#cit93"><sup><span class="sup_ref">93</span></sup></a> A total of ∼65<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 morphologies were generated. <span class="italic">J</span><small><sub>SC</sub></small>, was evaluated for each morphology using the excitonic drift-diffusion equation.<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> CNN using morphologies as input and <span class="italic">J</span><small><sub>SC</sub></small> as output showed a classification accuracy of 95.80% (<a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>).</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><a href="/image/article/2021/EE/d0ee02838j/d0ee02838j-f10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0ee02838j-f10.tif" src="/image/article/2021/EE/d0ee02838j/d0ee02838j-f10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">(a) Simple sketch of CNN architecture and (b) Confusion matrix. Adopted from <a title="Select to navigate to references" href="#cit78">ref. 78</a> Creative Commons Attribution 4.0 International License.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">MacKenzie <span class="italic">et al.</span> used the Shockley–Read–Hall based drift-diffusion model to simulate current/voltage (<span class="italic">JV</span>) curves.<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> They generated a set of 20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 devices and calculated electrical parameters such as carrier trapping rates, energetic disorder, trap densities, recombination time constants, and parasitic resistances. Simulated data were used to train the neural network. Then the trained model was used to study the effect of surfactant choice and annealing temperature on charge carrier dynamics of some well-known OSC devices.</p>
        <p class="otherpara">The solubility of the active layer materials in a specific solvent controls the film morphology and resultantly affects the performance of the device. Risko <span class="italic">et al.</span> calculated the free energy of mixing using molecular dynamics (MD) simulation.<a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a> They also used Bayesian statistics to calculate the free energy of mixing. This approach is an effective and fast way to study the large number of solvent and solvent additives.</p>
      
    
    
      
      <h2 id="sect1032"><span class="a_heading">5. Problems and future prospects</span></h2>
      <span>In spite of the increasing number of publications on machine learning, its use for organic solar cells is still not very efficient due to the complex nature of organic solar cells. The performance of OSCs depends on the processing solvent, solvent additive, crystallinity and molecular orientation of the active layer materials. Charge-separation at the donor–acceptor interface strongly depends on morphological properties. Further work is required for efficient use of ML for photovoltaic materials.</span>
      
        
        <h3 id="sect1036"><span class="b_heading">5.1. Data infrastructure</span></h3>
        <span>In many studies data from Harvard Clean Energy Project (HCEP) is used to contract the ML model for the screening of OPV materials. However, molecules reported in the literature are usually much more complicated than those in the HCEP. The differences in structures may lead to inaccurate ML predictions. Some groups have already collected data from the published literature.</span>
        <p class="otherpara">For effective training of an ML model, a large amount of data is required. Availability of data is not a problem for fields such as image recognition, <span class="italic">i.e.</span> millions of input datasets are available. However, in organic solar cells data are in hundreds or thousands. It is reported that accuracy is increased with increase of the data points (number of molecules).<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31,66,76</span></sup></a> In the case of ML models trained using descriptors related to power conversion processes, it is hard to include huge data due to time consuming DFT calculations. For limited data sets, meta-learning is a promising solution, whereby knowledge is learned within and across problems. A Bayesian framework is also a good option for limited data. A dual strategy is required to maintained balance between the availability of the data and the predictive capability of models. Instead of affecting the model precision directly, the effect of data size can be mediated by the degree of freedom (DoF) of the model, resulting in the phenomenon of association between precision and DoF. This concept is originated from the statistical bias-variance trade-off. In this regard, Zhang and Ling proposed a strategy to apply machine learning to small datasets.<a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a></p>
      
      
        
        <h3 id="sect1044"><span class="b_heading">5.2. Descriptor selection</span></h3>
        <span>Selection of molecular descriptors is a very important task. Molecular descriptors and fingerprints are fast and easy to calculate but not completely suitable for organic solar cell modelling. The microscopic properties that contain information related to photovoltaic phenomena require high accuracy quantum calculations. Their high computational cost makes it impossible to use them for large-scale fast virtual screening. A suitable trade-off between speed and accuracy is required. In this situation, accurate and easily accessible fingerprints and as well as a new generation of descriptors specifically designed for organic semiconductors are urgently needed.</span>
      
      
        
        <h3 id="sect1048"><span class="b_heading">5.3. Multidimensional design</span></h3>
        <span>Most of the models correlate the chemical structures with PCE but ignore many factors such miscibility and film morphology. The use of Flory and Huggins theory may enhance the usefulness of ML methods. Inclusion of transmission electron microscopy (TEM), atomic force microscopy (AFM), grazing-incidence small-angle X-ray scattering (GISAXS) and grazing incidence wide angle x-ray scattering (GIWAXS) data may improve the prediction accuracy.<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> Although machine learning is so successful on routine life picture analysis, images from the above techniques are very different. Compared to routine life images, images from the above techniques have featured characters, therefore, ML analysis of these images is very difficult. For example, microscopy images are associated with high-level noise and distortions.<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> In addition, information provided by images from experimental techniques strongly associates with physiochemical properties of materials, their mixing and experimental environments. All these factors make the situation much more sophisticated. Additionally, images from different techniques have different physical meanings. Therefore, different strategies will be required for different techniques. Considering this situation, machine learning analysis of experimental images should be carried out with enough knowledge of machine learning and physicochemical properties and underlying phenomena.</span>
        <p class="otherpara">In this route, the first step is collection of images, which will be manual due to the presence of various images for one compound under different experimental conditions. Automatic image extraction and sorting will be very difficult. Therefore, human assistance is essential. The second step will be task specification and analysis, for example decision of the data label or target properties. The morphology of the active layer strongly influences the FF values.<a title="Select to navigate to references" href="#cit99"><sup><span class="sup_ref">99</span></sup></a> Therefore, it will be more realistic to select FF as the target rather than PCE. Then FF along with the other factors can correlate with the PCE. Another thing to decide is the use of classification or regression. In the case of small data sets, classification can be preferred and <span class="italic">vice versa</span>. The third step will be training of the model and extraction of the pattern from the data to make a prediction. The last step will be experimental validation. It feels a hard journey to connect images to performance, however, it will be fruitful.</p>
        <p class="otherpara">Although no study is reported where experimental morphologies were used as input for machine learning analysis of organic solar cells, there are several studies that used simulated images and properties to train an ML model. Some studies are discussed in Section 4.6 and some are cited. These studies will pave the way toward the utilization of experimental morphologies for machine learning models.<a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a></p>
      
      
        
        <h3 id="sect1059"><span class="b_heading">5.4. Experimental validation</span></h3>
        <span>The review of the above mention literature has showed increasing use of machine learning in organic solar cell research. Further advances in high-throughput screening are expected. Generally, heuristic rules are used to screen the materials, and there is no guaranty that they are synthesizable (their synthesis procedures are not necessarily known). Therefore, collaboration with experimental experts is necessary. Collaboration can improve the success rate of machine prediction. Potential candidates selected from machine learning should be further manually screened on the basis of synthetic aspects. In addition, machine learning predictions should be followed by experimental validation. However, relatively few examples are present where the final candidates are validated using experiments. Sun <span class="italic">et al.</span> synthesized 10 donor materials to verify the ML results. The model has classified eight molecules into the correct category.<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> Saeki <span class="italic">et al.</span> synthesized one donor and fabricated an OSC device, and it showed a PCE of 0.53% that is much lower than the RF prediction (5.0–5.8%).<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> Min <span class="italic">et al.</span> fabricated six D/A pairs, and most of the devices showed a PCE close to the predicted values.<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a></span>
      
      
        
        <h3 id="sect1069"><span class="b_heading">5.5. Development of better software</span></h3>
        <span>Currently, some degree of programming ability is required to use the majority of machine learning tools. Therefore, ML is only accessible to those who have deep experience in computer programming and data science. However, they do not have enough knowledge of underlying processes. Sometime, it leads to the wrong interpretation of results. There are so many experimental scientists working in the field of organic solar cells. However, they do not have background of machine learning. To solve this problem, software with user-friendly graphical interface should be developed for material experts. In this way, material experts will able to explore all the powers of data driven science without the problems of unfamiliar syntax and arcane tuning parameters.</span>
      
    
    
      
      <h2 id="sect1073"><span class="a_heading">6. Summary</span></h2>
      <span>Different types of input such as molecular descriptors, molecular fingerprints, images, microscopic properties, energy levels and simulated properties are using to train machine learning models. The types of input and machine learning model (algorithm) strongly affect the performance of the machine learning model. In spite of pitfalls, machine learning is still essential to speed-up the discovery of efficient organic semi-conductors for organic solar cells. The number of publications based on machine learning is growing at an extraordinary rate. Despite of success stories, still there are many obstacles for the use of machine learning in organic solar cell research. We have mentioned some obstacles and proposed possible solutions. Further improvement will increase the use of machine learning in organic solar cell research. It will replace the experimental trial and error methods. Therefore, the use of machine learning in organic solar cells research should be encouraged. The availability of open-source tools and data sharing have enhanced the potential of ML to revolutionize materials discovery. There is a large gap between what machine learning can deliver and what currently has been achieved. Soon, machine learning will be a necessary method for organic solar cells as the existing experimental and computational techniques are.</span>
    
    
      <h2 id="sect1077"><span class="a_heading">Abbreviations</span></h2>
      <table cellspacing="5">
        <tr><td>ML</td><td>Machine learning</td></tr>
        <tr><td>BHJ</td><td>Bulk-heterojunction</td></tr>
        <tr><td>GBRT</td><td>Bradient boosting regression tree</td></tr>
        <tr><td>RF</td><td>Random forest</td></tr>
        <tr><td>SVM</td><td>Support vector machine</td></tr>
        <tr><td>ANN</td><td>Artificial neural network</td></tr>
        <tr><td>DNN</td><td>Deep neural network</td></tr>
        <tr><td>CNN</td><td>Convolutional neural network</td></tr>
        <tr><td>KRR</td><td>Kernel ridge regression</td></tr>
        <tr><td>SVR</td><td>Support vector regression</td></tr>
        <tr><td>GPR</td><td>Gaussian processes regression</td></tr>
        <tr><td>SMILES</td><td>Simplified molecular-input line-entry system</td></tr>
        <tr><td>PCE</td><td>Power conversion efficiency</td></tr>
        <tr><td>
            <span class="italic">V</span>
            <small><sub>OC</sub></small>
          </td><td>Open-circuit voltage</td></tr>
        <tr><td>
            <span class="italic">J</span>
            <small><sub>SC</sub></small>
          </td><td>Short-circuit current density</td></tr>
        <tr><td>FF</td><td>Fill factor</td></tr>
        <tr><td>HOMO</td><td>Highest occupied molecular orbitals</td></tr>
        <tr><td>LUMO</td><td>Lowest occupied molecular orbitals</td></tr>
        <tr><td>DFT</td><td>Density functional theory</td></tr>
        <tr><td>
            <span class="italic">r</span>
          </td><td>Correlation coefficient</td></tr>
        <tr><td>
            <span class="italic">R</span>
            <small><sup>2</sup></small>
          </td><td>Coefficient of determination</td></tr>
        <tr><td>HCEP</td><td>Harvard Clean Energy Project</td></tr>
        <tr><td>HOPV15</td><td>The Harvard Organic Photovoltaic Dataset</td></tr>
      </table>
    
    
      <h2 id="sect1156"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <h2 id="sect1160"><span class="a_heading">Acknowledgements</span></h2>
      <span>The authors acknowledge the support from the National Natural Science Foundation of China (No. 21672023, 21971014, and 21950410533) and the National Key Research and Development Program of China (2018YFA0901800). Jin-Liang Wang was supported by the Thousand Youth Talents Plan of China and BIT Teli Young Fellow Recruitment Program. The authors thank the Analysis &amp; Testing Center, Beijing Institute of Technology.</span>
    
    <span id="sect1162"><h2 id="sect1159"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">S. Günes, H. Neugebauer and N. S. Sariciftci, <span class="italic">Chem. Rev.</span>, 2007, <span class="bold">107</span>, 1324–1338 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/cr050149z" title="DOI Link to resource 10.1021/cr050149z">CrossRef</a>.</span></li>
      <li><span id="cit2">H. Imahori, T. Umeyama and S. Ito, <span class="italic">Acc. Chem. Res.</span>, 2009, <span class="bold">42</span>, 1809–1818 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ar900034t" title="DOI Link to resource 10.1021/ar900034t">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXlt12mt70%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit3">A. Mahmood, J.-Y. Hu, B. Xiao, A. Tang, X. Wang and E. Zhou, <span class="italic">J. Mater. Chem. A</span>, 2018, <span class="bold">6</span>, 16769–16797 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8TA06392C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8TA06392C">RSC</a>.</span></li>
      <li><span id="cit4">A. Mahmood, J. Hu, A. Tang, F. Chen, X. Wang and E. Zhou, <span class="italic">Dyes Pigm.</span>, 2018, <span class="bold">149</span>, 470–474 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.dyepig.2017.10.037" title="DOI Link to resource 10.1016/j.dyepig.2017.10.037">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhslyhtb%252FF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit5">C. Zhang, X. Song, K.-K. Liu, M. Zhang, J. Qu, C. Yang, G.-Z. Yuan, A. Mahmood, F. Liu, F. He, D. Baran and J.-L. Wang, <span class="italic">Small</span>, 2020, <span class="bold">16</span>, 1907681 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/smll.201907681" title="DOI Link to resource 10.1002/smll.201907681">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXoslegu7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit6">Q. Liu, Y. Jiang, K. Jin, J. Qin, J. Xu, W. Li, J. Xiong, J. Liu, Z. Xiao, K. Sun, S. Yang, X. Zhang and L. Ding, <span class="italic">Sci. Bulletin</span>, 2020, <span class="bold">65</span>, 272–275 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.scib.2020.01.001" title="DOI Link to resource 10.1016/j.scib.2020.01.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXptVaku7g%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit7">C. Zhu, J. Yuan, F. Cai, L. Meng, H. Zhang, H. Chen, J. Li, B. Qiu, H. Peng, S. Chen, Y. Hu, C. Yang, F. Gao, Y. Zou and Y. Li, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 2459–2466 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE00862A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE00862A">RSC</a>.</span></li>
      <li><span id="cit8">L. Zhan, S. Li, T.-K. Lau, Y. Cui, X. Lu, M. Shi, C.-Z. Li, H. Li, J. Hou and H. Chen, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 635–645 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9EE03710A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9EE03710A">RSC</a>.</span></li>
      <li><span id="cit9">J.-L. Wang, K.-K. Liu, J. Yan, Z. Wu, F. Liu, F. Xiao, Z.-F. Chang, H.-B. Wu, Y. Cao and T. P. Russell, <span class="italic">J. Am. Chem. Soc.</span>, 2016, <span class="bold">138</span>, 7687–7697 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.6b03495" title="DOI Link to resource 10.1021/jacs.6b03495">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xos12ntb4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit10">S.-S. Wan, X. Xu, Z. Jiang, J. Yuan, A. Mahmood, G.-Z. Yuan, K.-K. Liu, W. Ma, Q. Peng and J.-L. Wang, <span class="italic">J. Mater. Chem. A</span>, 2020, <span class="bold">8</span>, 4856–4867 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9TA14070K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9TA14070K">RSC</a>.</span></li>
      <li><span id="cit11">Y. Wang, Y. Wang, L. Zhu, H. Liu, J. Fang, X. Guo, F. Liu, Z. Tang, M. Zhang and Y. Li, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 1309–1317 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9EE04199K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9EE04199K">RSC</a>.</span></li>
      <li><span id="cit12">D. Hu, Q. Yang, H. Chen, F. Wobben, V. M. Le Corre, R. Singh, T. Liu, R. Ma, H. Tang, L. J. A. Koster, T. Duan, H. Yan, Z. Kan, Z. Xiao and S. Lu, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 2134–2141 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE00714E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE00714E">RSC</a>.</span></li>
      <li><span id="cit13">C. Yang, S. Zhang, J. Ren, M. Gao, P. Bi, L. Ye and J. Hou, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 2864–2869 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE01763A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE01763A">RSC</a>.</span></li>
      <li><span id="cit14">X. Wan, C. Li, M. Zhang and Y. Chen, <span class="italic">Chem. Soc. Rev.</span>, 2020, <span class="bold">49</span>, 2828–2842 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0CS00084A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0CS00084A">RSC</a>.</span></li>
      <li><span id="cit15">G. Han, T. Hu and Y. Yi, <span class="italic">Adv. Mater.</span>, 2020, <span class="bold">32</span>, 2000975 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.202000975" title="DOI Link to resource 10.1002/adma.202000975">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXnslGrsLo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit16">J.-L. Brédas, J. E. Norton, J. Cornil and V. Coropceanu, <span class="italic">Acc. Chem. Res.</span>, 2009, <span class="bold">42</span>, 1691–1699 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ar900099h" title="DOI Link to resource 10.1021/ar900099h">CrossRef</a>.</span></li>
      <li><span id="cit17">A. Wadsworth, M. Moser, A. Marks, M. S. Little, N. Gasparini, C. J. Brabec, D. Baran and I. McCulloch, <span class="italic">Chem. Soc. Rev.</span>, 2019, <span class="bold">48</span>, 1596–1625 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7CS00892A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7CS00892A">RSC</a>.</span></li>
      <li><span id="cit18">M. C. Scharber, D. Mühlbacher, M. Koppe, P. Denk, C. Waldauf, A. J. Heeger and C. J. Brabec, <span class="italic">Adv. Mater.</span>, 2006, <span class="bold">18</span>, 789–794 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.200501717" title="DOI Link to resource 10.1002/adma.200501717">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XjsV2isrw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit19">D. Padula, J. D. Simpson and A. Troisi, <span class="italic">Mater. Horiz.</span>, 2019, <span class="bold">6</span>, 343–349 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8MH01135D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8MH01135D">RSC</a>.</span></li>
      <li><span id="cit20">S. A. Lopez, B. Sanchez-Lengeling, J. de Goes Soares and A. Aspuru-Guzik, <span class="italic">Joule</span>, 2017, <span class="bold">1</span>, 857–870 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2017.10.006" title="DOI Link to resource 10.1016/j.joule.2017.10.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpvFGhu7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit21">E. O. Pyzer-Knapp, G. N. Simm and A. Aspuru Guzik, <span class="italic">Mater. Horiz.</span>, 2016, <span class="bold">3</span>, 226–233 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5MH00282F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5MH00282F">RSC</a>.</span></li>
      <li><span id="cit22">G. H. Gu, J. Noh, I. Kim and Y. Jung, <span class="italic">J. Mater. Chem. A</span>, 2019, <span class="bold">7</span>, 17096–17117 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9TA02356A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9TA02356A">RSC</a>.</span></li>
      <li><span id="cit23">C. Chen, Y. Zuo, W. Ye, X. Li, Z. Deng and S. P. Ong, <span class="italic">Adv. Energy Mater.</span>, 2020, <span class="bold">10</span>, 1903242 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201903242" title="DOI Link to resource 10.1002/aenm.201903242">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhslens7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit24">M.-H. Lee, <span class="italic">Adv. Electron. Mater.</span>, 2019, <span class="bold">5</span>, 1900573 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aelm.201900573" title="DOI Link to resource 10.1002/aelm.201900573">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhs12jtb%252FF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit25">T. Wang, C. Zhang, H. Snoussi and G. Zhang, <span class="italic">Adv. Funct. Mater.</span>, 2020, <span class="bold">30</span>, 1906041 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201906041" title="DOI Link to resource 10.1002/adfm.201906041">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXit1Sms77F" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit26">R. Hu, J. Song, Y. Liu, W. Xi, Y. Zhao, X. Yu, Q. Cheng, G. Tao and X. Luo, <span class="italic">Nano Energy</span>, 2020, <span class="bold">72</span>, 104687 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.nanoen.2020.104687" title="DOI Link to resource 10.1016/j.nanoen.2020.104687">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXlsVejsLo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit27">A. O. Oliynyk and J. M. Buriak, <span class="italic">Chem. Mater.</span>, 2019, <span class="bold">31</span>, 8243–8247 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.9b03854" title="DOI Link to resource 10.1021/acs.chemmater.9b03854">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFGls7zL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit28">T. Cova and A. Pais, <span class="italic">Front. Chem.</span>, 2019, <span class="bold">7</span>, 809 <a target="_blank" class="DOILink" href="https://doi.org/10.3389/fchem.2019.00809" title="DOI Link to resource 10.3389/fchem.2019.00809">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhslyksrjI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit29">G. R. Schleder, A. C. M. Padilha, C. M. Acosta, M. Costa and A. Fazzio, <span class="italic">J. Phys. Mater.</span>, 2019, <span class="bold">2</span>, 032001 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/2515-7639/ab084b" title="DOI Link to resource 10.1088/2515-7639/ab084b">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitFGhtrjJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit30">T. Zhou, Z. Song and K. Sundmacher, <span class="italic">Engineering</span>, 2019, <span class="bold">5</span>, 1017–1026 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.eng.2019.02.011" title="DOI Link to resource 10.1016/j.eng.2019.02.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtVektrbJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit31">E. O. Pyzer-Knapp, K. Li and A. Aspuru-Guzik, <span class="italic">Adv. Funct. Mater.</span>, 2015, <span class="bold">25</span>, 6495–6502 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201501919" title="DOI Link to resource 10.1002/adfm.201501919">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtL3I" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit32">
          S. Vieira, W. H. Lopez Pinaya and A. Mechelli, in <span class="italic">Machine Learning</span>, ed. A. Mechelli and S. Vieira, Academic Press,  2020, pp. 21–44, <small> DOI:<a class="DOILink" href="https://doi.org/10.1016/B978-0-12-815739-8.00002-X" TARGET="_BLANK" title="DOI Link to 10.1016/B978-0-12-815739-8.00002-X">10.1016/B978-0-12-815739-8.00002-X</a></small>.</span></li>
      <li><span id="cit33">Y. Iwasaki, I. Takeuchi, V. Stanev, A. G. Kusne, M. Ishida, A. Kirihara, K. Ihara, R. Sawada, K. Terashima, H. Someya, K.-i. Uchida, E. Saitoh and S. Yorozu, <span class="italic">Sci. Rep.</span>, 2019, <span class="bold">9</span>, 2751 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-019-39278-z" title="DOI Link to resource 10.1038/s41598-019-39278-z">CrossRef</a>.</span></li>
      <li><span id="cit34">B. Sanchez-Lengeling and A. Aspuru-Guzik, <span class="italic">Science</span>, 2018, <span class="bold">361</span>, 360–365 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aat2663" title="DOI Link to resource 10.1126/science.aat2663">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyitr3L" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit35">A. H. Vo, T. R. Van Vleet, R. R. Gupta, M. J. Liguori and M. S. Rao, <span class="italic">Chem. Res. Toxicol.</span>, 2020, <span class="bold">33</span>, 20–37 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Chem.%20Res.%20Toxicol.%5BJour%5D%20AND%2033%5Bvolume%5D%20AND%2020%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit36">A. Mauri, V. Consonni, M. Pavan and R. Todeschini, <span class="italic">MATCH-Commun. Math. Co.</span>, 2006, <span class="bold">56</span>, 237–248 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=MATCH-Commun.%20Math.%20Co.%5BJour%5D%20AND%2056%5Bvolume%5D%20AND%20237%5Bpage%5D%20and%202006%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit37">I. V. Tetko, J. Gasteiger, R. Todeschini, A. Mauri, D. Livingstone, P. Ertl, V. A. Palyulin, E. V. Radchenko, N. S. Zefirov, A. S. Makarenko, V. Y. Tanchuk and V. V. Prokopenko, <span class="italic">J. Comput. Aided Mol. Des.</span>, 2005, <span class="bold">19</span>, 453–463 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10822-005-8694-y" title="DOI Link to resource 10.1007/s10822-005-8694-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXhtFaht77F" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit38">H. Hong, Q. Xie, W. Ge, F. Qian, H. Fang, L. Shi, Z. Su, R. Perkins and W. Tong, <span class="italic">J. Chem. Inf. Model.</span>, 2008, <span class="bold">48</span>, 1337–1344 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci800038f" title="DOI Link to resource 10.1021/ci800038f">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXnsVehtL0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit39">H. Moriwaki, Y.-S. Tian, N. Kawashita and T. Takagi, <span class="italic">J. Cheminform.</span>, 2018, <span class="bold">10</span>, 4 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13321-018-0258-y" title="DOI Link to resource 10.1186/s13321-018-0258-y">CrossRef</a>.</span></li>
      <li><span id="cit40">C. W. Yap, <span class="italic">J. Comput. Chem.</span>, 2011, <span class="bold">32</span>, 1466–1474 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.21707" title="DOI Link to resource 10.1002/jcc.21707">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXjsF2isLc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit41">A. Kerber, R. Laue, M. Meringer and C. Ucker, <span class="italic">MATCH-Commun. Math. Co.</span>, 2004, <span class="bold">51</span>, 187–204 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXjs1ynug%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit42">D.-S. Cao, Q.-S. Xu, Q.-N. Hu and Y.-Z. Liang, <span class="italic">Bioinformatics</span>, 2013, <span class="bold">29</span>, 1092–1094 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/bioinformatics/btt105" title="DOI Link to resource 10.1093/bioinformatics/btt105">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKitr0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit43">BlueDesc, http://www.ra.cs.uni-tuebingen.de/software/bluedesc/, accessed July 12, 2020.</span></li>
      <li><span id="cit44">K. Liu, J. Feng and S. S. Young, <span class="italic">J. Chem. Inf. Model.</span>, 2005, <span class="bold">45</span>, 515–522 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci049847v" title="DOI Link to resource 10.1021/ci049847v">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXntFyqsQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit45">R. Olivares-Amaya, C. Amador-Bedolla, J. Hachmann, S. Atahan-Evrenk, R. S. Sánchez-Carrera, L. Vogt and A. Aspuru-Guzik, <span class="italic">Energy Environ. Sci.</span>, 2011, <span class="bold">4</span>, 4849–4861 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1EE02056K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1EE02056K">RSC</a>.</span></li>
      <li><span id="cit46">F. Pereira, K. Xiao, D. A. R. S. Latino, C. Wu, Q. Zhang and J. Aires-de-Sousa, <span class="italic">J. Chem. Inf. Model.</span>, 2017, <span class="bold">57</span>, 11–21 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.6b00340" title="DOI Link to resource 10.1021/acs.jcim.6b00340">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XitFKqtr%252FN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit47">M.-Y. Sui, Z.-R. Yang, Y. Geng, G.-Y. Sun, L. Hu and Z.-M. Su, <span class="italic">Sol. RRL</span>, 2019, <span class="bold">3</span>, 1900258 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/solr.201900258" title="DOI Link to resource 10.1002/solr.201900258">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitFenu7vI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit48">A. Cereto-Massagué, M. J. Ojeda, C. Valls, M. Mulero, S. Garcia-Vallvé and G. Pujadas, <span class="italic">Methods</span>, 2015, <span class="bold">71</span>, 58–63 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ymeth.2014.08.005" title="DOI Link to resource 10.1016/j.ymeth.2014.08.005">CrossRef</a>.</span></li>
      <li><span id="cit49">L. Pattanaik and C. W. Coley, <span class="italic">Chem</span>, 2020, <span class="bold">6</span>, 1204–1207 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOis7%252FE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit50">I. Muegge and P. Mukherjee, <span class="italic">Expert Opin. Drug Discovery</span>, 2016, <span class="bold">11</span>, 137–148 <a target="_blank" class="DOILink" href="https://doi.org/10.1517/17460441.2016.1117070" title="DOI Link to resource 10.1517/17460441.2016.1117070">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvFOmtLbF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit51">A. Mahmood, A. Tang, X. Wang and E. Zhou, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2019, <span class="bold">21</span>, 2128–2139 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8CP05763J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8CP05763J">RSC</a>.</span></li>
      <li><span id="cit52">A. Mahmood, J. Yang, J. Hu, X. Wang, A. Tang, Y. Geng, Q. Zeng and E. Zhou, <span class="italic">J. Phys. Chem. C</span>, 2018, <span class="bold">122</span>, 29122–29128 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.8b09336" title="DOI Link to resource 10.1021/acs.jpcc.8b09336">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitlKmsbzP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit53">G.-Z. Yuan, H. Fan, S.-S. Wan, Z. Jiang, Y.-Q. Liu, K.-K. Liu, H.-R. Bai, X. Zhu and J.-L. Wang, <span class="italic">J. Mater. Chem. A</span>, 2019, <span class="bold">7</span>, 20274–20284 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9TA06311K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9TA06311K">RSC</a>.</span></li>
      <li><span id="cit54">J.-L. Wang, K.-K. Liu, L. Hong, G.-Y. Ge, C. Zhang and J. Hou, <span class="italic">ACS Energy Lett.</span>, 2018, <span class="bold">3</span>, 2967–2976 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsenergylett.8b01808" title="DOI Link to resource 10.1021/acsenergylett.8b01808">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitFyrtL7O" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit55">K.-K. Liu, X. Xu, J.-L. Wang, C. Zhang, G.-Y. Ge, F.-D. Zhuang, H.-J. Zhang, C. Yang, Q. Peng and J. Pei, <span class="italic">J. Mater. Chem. A</span>, 2019, <span class="bold">7</span>, 24389–24399 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9TA08328F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9TA08328F">RSC</a>.</span></li>
      <li><span id="cit56">Y. Xie, W. Wang, W. Huang, F. Lin, T. Li, S. Liu, X. Zhan, Y. Liang, C. Gao, H. Wu and Y. Cao, <span class="italic">Energy Environ. Sci.</span>, 2019, <span class="bold">12</span>, 3556–3566 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9EE02939G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9EE02939G">RSC</a>.</span></li>
      <li><span id="cit57">T. Linderl, T. Zechel, M. Brendel, D. Moseguí González, P. Müller-Buschbaum, J. Pflaum and W. Brütting, <span class="italic">Adv. Energy Mater.</span>, 2017, <span class="bold">7</span>, 1700237 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201700237" title="DOI Link to resource 10.1002/aenm.201700237">CrossRef</a>.</span></li>
      <li><span id="cit58">J. Zhang, W. Liu, M. Zhang, Y. Liu, G. Zhou, S. Xu, F. Zhang, H. Zhu, F. Liu and X. Zhu, <span class="italic">iScience</span>, 2019, <span class="bold">19</span>, 883–893 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.isci.2019.08.038" title="DOI Link to resource 10.1016/j.isci.2019.08.038">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFCntL%252FL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit59">W. Sun, Y. Zheng, K. Yang, Q. Zhang, A. A. Shah, Z. Wu, Y. Sun, L. Feng, D. Chen, Z. Xiao, S. Lu, Y. Li and K. Sun, <span class="italic">Sci. Adv.</span>, 2019, <span class="bold">5</span>, eaay4275 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/sciadv.aay4275" title="DOI Link to resource 10.1126/sciadv.aay4275">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtlaksL%252FN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit60">S. Nagasawa, E. Al-Naamani and A. Saeki, <span class="italic">J. Phys. Chem. Lett.</span>, 2018, <span class="bold">9</span>, 2639–2646 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.8b00635" title="DOI Link to resource 10.1021/acs.jpclett.8b00635">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXovVyms74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit61">P. B. Jørgensen, M. Mesta, S. Shil, J. M. G. Lastra, K. W. Jacobsen, K. S. Thygesen and M. N. Schmidt, <span class="italic">J. Chem. Phys.</span>, 2018, <span class="bold">148</span>, 241735 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5023563" title="DOI Link to resource 10.1063/1.5023563">CrossRef</a>.</span></li>
      <li><span id="cit62">A. Paul, A. Furmanchuk, W.-k. Liao, A. Choudhary and A. Agrawal, <span class="italic">Mol. Inform.</span>, 2019, <span class="bold">38</span>, 1900038 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/minf.201900038" title="DOI Link to resource 10.1002/minf.201900038">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsleitbvM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit63">S.-P. Peng and Y. Zhao, <span class="italic">J. Chem. Inf. Model.</span>, 2019, <span class="bold">59</span>, 4993–5001 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.9b00732" title="DOI Link to resource 10.1021/acs.jcim.9b00732">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitFagurvE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit64">D. Padula and A. Troisi, <span class="italic">Adv. Energy Mater.</span>, 2019, <span class="bold">9</span>, 1902463 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201902463" title="DOI Link to resource 10.1002/aenm.201902463">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhs1ynsrzM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit65">Y. Wu, J. Guo, R. Sun and J. Min, <span class="italic">NPJ Comput. Mater.</span>, 2020, <span class="bold">6</span>, 120 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-020-00388-2" title="DOI Link to resource 10.1038/s41524-020-00388-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitVWrs7zL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit66">W. Sun, M. Li, Y. Li, Z. Wu, Y. Sun, S. Lu, Z. Xiao, B. Zhao and K. Sun, <span class="italic">Adv. Theory Simul.</span>, 2019, <span class="bold">2</span>, 1800116 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adts.201800116" title="DOI Link to resource 10.1002/adts.201800116">CrossRef</a>.</span></li>
      <li><span id="cit67">H. Sahu, W. Rao, A. Troisi and H. Ma, <span class="italic">Adv. Energy Mater.</span>, 2018, <span class="bold">8</span>, 1801032 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201801032" title="DOI Link to resource 10.1002/aenm.201801032">CrossRef</a>.</span></li>
      <li><span id="cit68">H. Sahu and H. Ma, <span class="italic">J. Phys. Chem. Lett.</span>, 2019, <span class="bold">10</span>, 7277–7284 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.9b02772" title="DOI Link to resource 10.1021/acs.jpclett.9b02772">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitFagtbjL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit69">Z.-W. Zhao, M. del Cueto, Y. Geng and A. Troisi, <span class="italic">Chem. Mater.</span>, 2020, <span class="bold">32</span>, 7777–7787 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.0c02325" title="DOI Link to resource 10.1021/acs.chemmater.0c02325">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Kiu7bO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit70">H. Sahu, F. Yang, X. Ye, J. Ma, W. Fang and H. Ma, <span class="italic">J. Mater. Chem. A</span>, 2019, <span class="bold">7</span>, 17480–17488 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9TA04097H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9TA04097H">RSC</a>.</span></li>
      <li><span id="cit71">J. Hachmann, R. Olivares-Amaya, A. Jinich, A. L. Appleton, M. A. Blood-Forsythe, L. R. Seress, C. Román-Salgado, K. Trepte, S. Atahan-Evrenk, S. Er, S. Shrestha, R. Mondal, A. Sokolov, Z. Bao and A. Aspuru-Guzik, <span class="italic">Energy Environ. Sci.</span>, 2014, <span class="bold">7</span>, 698–704 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3EE42756K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3EE42756K">RSC</a>.</span></li>
      <li><span id="cit72">Y. Imamura, M. Tashiro, M. Katouda and M. Hada, <span class="italic">J. Phys. Chem. C</span>, 2017, <span class="bold">121</span>, 28275–28286 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b08446" title="DOI Link to resource 10.1021/acs.jpcc.7b08446">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2mt73K" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit73">M.-H. Lee, <span class="italic">Organ. Electron.</span>, 2020, <span class="bold">76</span>, 105465 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.orgel.2019.105465" title="DOI Link to resource 10.1016/j.orgel.2019.105465">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFGms73L" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit74">M.-H. Lee, <span class="italic">Adv. Energy Mater.</span>, 2019, <span class="bold">9</span>, 1900891 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201900891" title="DOI Link to resource 10.1002/aenm.201900891">CrossRef</a>.</span></li>
      <li><span id="cit75">M.-H. Lee, <span class="italic">Adv. Intelligent Syst.</span>, 2020, <span class="bold">2</span>, 1900108 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aisy.201900108" title="DOI Link to resource 10.1002/aisy.201900108">CrossRef</a>.</span></li>
      <li><span id="cit76">M.-H. Lee, <span class="italic">Energy Technol.</span>, 2020, <span class="bold">8</span>, 1900974 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ente.201900974" title="DOI Link to resource 10.1002/ente.201900974">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXksVGlur0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit77">O. Wodo, S. Tirthapura, S. Chaudhary and B. Ganapathysubramanian, <span class="italic">Organ. Electron.</span>, 2012, <span class="bold">13</span>, 1105–1113 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.orgel.2012.03.007" title="DOI Link to resource 10.1016/j.orgel.2012.03.007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XlvFajtLc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit78">B. S. S. Pokuri, S. Ghosal, A. Kokate, S. Sarkar and B. Ganapathysubramanian, <span class="italic">NPJ Comput. Mater.</span>, 2019, <span class="bold">5</span>, 95 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-019-0231-y" title="DOI Link to resource 10.1038/s41524-019-0231-y">CrossRef</a>.</span></li>
      <li><span id="cit79">N. Majeed, M. Saladina, M. Krompiec, S. Greedy, C. Deibel and R. C. I. MacKenzie, <span class="italic">Adv. Funct. Mater.</span>, 2020, <span class="bold">30</span>, 1907259 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201907259" title="DOI Link to resource 10.1002/adfm.201907259">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitlyhtLfF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit80">L. Ye, W. Zhao, S. Li, S. Mukherjee, J. H. Carpenter, O. Awartani, X. Jiao, J. Hou and H. Ade, <span class="italic">Adv. Energy Mater.</span>, 2017, <span class="bold">7</span>, 1602000 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201602000" title="DOI Link to resource 10.1002/aenm.201602000">CrossRef</a>.</span></li>
      <li><span id="cit81">D. T. Duong, B. Walker, J. Lin, C. Kim, J. Love, B. Purushothaman, J. E. Anthony and T.-Q. Nguyen, <span class="italic">J. Polym. Sci. B Polym. Phys.</span>, 2012, <span class="bold">50</span>, 1405–1413 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/polb.23153" title="DOI Link to resource 10.1002/polb.23153">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xht1Ciur%252FE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit82">J. D. Perea, S. Langner, M. Salvador, B. Sanchez-Lengeling, N. Li, C. Zhang, G. Jarvas, J. Kontos, A. Dallos, A. Aspuru-Guzik and C. J. Brabec, <span class="italic">J. Phys. Chem. C</span>, 2017, <span class="bold">121</span>, 18153–18161 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b03228" title="DOI Link to resource 10.1021/acs.jpcc.7b03228">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFSnurfI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit83">H. Z. Jun Yuan, R. Zhang, Y. Wang, J. Hou, M. Leclerc, X. Zhan, F. Huang, F. Gao, Y. Zou and Y. Li, <span class="italic">Chem</span>, 2020, <span class="bold">6</span>, 2147–2161 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Chem%5BJour%5D%20AND%206%5Bvolume%5D%20AND%202147%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit84">Q. Yue, W. Liu and X. Zhu, <span class="italic">J. Am. Chem. Soc.</span>, 2020, <span class="bold">142</span>, 11613–11628 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.0c04084" title="DOI Link to resource 10.1021/jacs.0c04084">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtVaqs77F" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit85">J. Gao, W. Gao, X. Ma, Z. Hu, C. Xu, X. Wang, Q. An, C. Yang, X. Zhang and F. Zhang, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 958–967 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9EE04020J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9EE04020J">RSC</a>.</span></li>
      <li><span id="cit86">T. Liu, R. Ma, Z. Luo, Y. Guo, G. Zhang, Y. Xiao, T. Yang, Y. Chen, G. Li, Y. Yi, X. Lu, H. Yan and B. Tang, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 2115–2123 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE00662A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE00662A">RSC</a>.</span></li>
      <li><span id="cit87">Z. Zhou, S. Xu, J. Song, Y. Jin, Q. Yue, Y. Qian, F. Liu, F. Zhang and X. Zhu, <span class="italic">Nat. Energy</span>, 2018, <span class="bold">3</span>, 952–959 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41560-018-0234-9" title="DOI Link to resource 10.1038/s41560-018-0234-9">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhslSlu7vM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit88">G. Liu, J. Jia, K. Zhang, X. E. Jia, Q. Yin, W. Zhong, L. Li, F. Huang and Y. Cao, <span class="italic">Adv. Energy Mater.</span>, 2019, <span class="bold">9</span>, 1803657 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201803657" title="DOI Link to resource 10.1002/aenm.201803657">CrossRef</a>.</span></li>
      <li><span id="cit89">P. Du, A. Zebrowski, J. Zola, B. Ganapathysubramanian and O. Wodo, <span class="italic">NPJ Comput. Mater.</span>, 2018, <span class="bold">4</span>, 50 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-018-0108-5" title="DOI Link to resource 10.1038/s41524-018-0108-5">CrossRef</a>.</span></li>
      <li><span id="cit90">S. Pfeifer, B. S. S. Pokuri, P. Du and B. Ganapathysubramanian, <span class="italic">Mater. Disc</span>, 2018, <span class="bold">11</span>, 6–13 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mater.%20Disc%5BJour%5D%20AND%2011%5Bvolume%5D%20AND%206%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit91">R. Noruzi, S. Ghadai, O. R. Bingol, A. Krishnamurthy and B. Ganapathysubramanian, <span class="italic">Comput. Aided Des.</span>, 2020, <span class="bold">118</span>, 102771 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cad.2019.102771" title="DOI Link to resource 10.1016/j.cad.2019.102771">CrossRef</a>.</span></li>
      <li><span id="cit92">J. W. Cahn, <span class="italic">Acta Metall.</span>, 1961, <span class="bold">9</span>, 795–801 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0001-6160(61)90182-1" title="DOI Link to resource 10.1016/0001-6160(61)90182-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaF38Xnt1eqsQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit93">O. Wodo and B. Ganapathysubramanian, <span class="italic">Comput. Mater. Sci.</span>, 2012, <span class="bold">55</span>, 113–126 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2011.12.012" title="DOI Link to resource 10.1016/j.commatsci.2011.12.012">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhvFKktrk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit94">H. K. Kodali and B. Ganapathysubramanian, <span class="italic">Model. Simul. Mater. Sci. Eng.</span>, 2012, <span class="bold">20</span>, 035015 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0965-0393/20/3/035015" title="DOI Link to resource 10.1088/0965-0393/20/3/035015">CrossRef</a>.</span></li>
      <li><span id="cit95">S. Li, B. S. S. Pokuri, S. M. Ryno, A. Nkansah, C. De’Vine, B. Ganapathysubramanian and C. Risko, <span class="italic">J. Chem. Inf. Model.</span>, 2020, <span class="bold">60</span>, 1424–1431 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.9b01113" title="DOI Link to resource 10.1021/acs.jcim.9b01113">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXnslOjtA%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit96">Y. Zhang and C. Ling, <span class="italic">NPJ Comput. Mater.</span>, 2018, <span class="bold">4</span>, 25 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-018-0081-z" title="DOI Link to resource 10.1038/s41524-018-0081-z">CrossRef</a>.</span></li>
      <li><span id="cit97">A. Mahmood and J.-L. Wang, <span class="italic">Sol. RRL</span>, 2020, <span class="bold">4</span>, 2000337 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/solr.202000337" title="DOI Link to resource 10.1002/solr.202000337">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvF2ltbrO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit98">L. Jones and P. D. Nellist, <span class="italic">Microsc. Microanal.</span>, 2013, <span class="bold">19</span>, 1050–1060 <a target="_blank" class="DOILink" href="https://doi.org/10.1017/S1431927613001402" title="DOI Link to resource 10.1017/S1431927613001402">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhtV2ktbbL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit99">M. Zawodzki, R. Resel, M. Sferrazza, O. Kettner and B. Friedel, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2015, <span class="bold">7</span>, 16161–16168 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.5b04972" title="DOI Link to resource 10.1021/acsami.5b04972">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWqtrnM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit100">B. S. S. Pokuri, J. Stimes, K. O’Hara, M. L. Chabinyc and B. Ganapathysubramanian, <span class="italic">Comput. Mater. Sci.</span>, 2019, <span class="bold">163</span>, 1–10 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2019.02.030" title="DOI Link to resource 10.1016/j.commatsci.2019.02.030">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXlslGlsbg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
    </ol>
    
    
    
    
    
  <table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2021</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>