<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>The CellScan technology for in vitro studies on novel platinum complexes with organoarsenic ligands   - Dalton Transactions (RSC Publishing) DOI:10.1039/B802364F</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2008/dt/b802364f"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":563,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Eva Fischer-Fodor"/><meta name="DC.Creator" content="Natalia Moldovan"/><meta name="DC.Creator" content="Piroska Virag"/><meta name="DC.Creator" content="Olga Soritau"/><meta name="DC.Creator" content="Ioana Brie"/><meta name="DC.Creator" content="Peter Lönnecke"/><meta name="DC.Creator" content="Evamarie Hey-Hawkins"/><meta name="DC.Creator" content="Luminita Silaghi-Dumitrescu"/><meta name="DC.title" content="The CellScan technology for in vitro studies on novel platinum complexes with organoarsenic ligands  "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2008/12/07"/><meta name="DC.Identifier" scheme="doi" content="10.1039/B802364F"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="The CellScan technology for in vitro studies on novel platinum complexes with organoarsenic ligands  "/><meta name="citation_author" content="Eva Fischer-Fodor"/><meta name="citation_author" content="Natalia Moldovan"/><meta name="citation_author" content="Piroska Virag"/><meta name="citation_author" content="Olga Soritau"/><meta name="citation_author" content="Ioana Brie"/><meta name="citation_author" content="Peter Lönnecke"/><meta name="citation_author" content="Evamarie Hey-Hawkins"/><meta name="citation_author" content="Luminita Silaghi-Dumitrescu"/><meta name="citation_online_date" content="2008/10/06"/><meta name="citation_date" content="2008"/><meta name="citation_journal_title" content="Dalton Transactions"/><meta name="citation_volume" content="0"/><meta name="citation_issue" content="45"/><meta name="citation_firstpage" content="6393"/><meta name="citation_lastpage" content="6400"/><meta name="citation_doi" content="10.1039/B802364F"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2008/dt/b802364f"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2008/dt/b802364f"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2008/dt/b802364f"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/prospect.js?ver=6_2_1"> </script><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2008/dt/b802364f">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:b808208a)" href="/en/content/articlehtml/2008/dt/b808208a" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:b810487e)" href="/en/content/articlehtml/2008/dt/b810487e" target="_BLANK">Next Article</a></span></div><div class="right_head"> </div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/B802364F">10.1039/B802364F</a>
(Paper)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1477-9234/2003">Dalton Trans.</a></span>, 2008, 6393-6400<div id="chemnav"><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of compound names" onclick="toggleColour('TC');"><span alt="Show Compounds button" id="btnTC">Show Compounds</span></a><input id="inpTC" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of chem-ontology terms" onclick="toggleColour('CH');"><span alt="Show Chemical Terms button" id="btnCH">Show Chemical Terms</span></a><input id="inpCH" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of bio-ontology terms" onclick="toggleColour('AN');"><span alt="Show Biomedical Terms button" id="btnAN">Show Biomedical Terms</span></a><input id="inpAN" type="hidden" value="off"/></div></div><h1 id="sect162"><span class="title_heading">The CellScan technology for <span class="italic">in vitro</span> studies on novel <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes with organoarsenic ligands<a title="Select to navigate to footnote" href="#fn1">†</a></span></h1><p class="header_text"><span class="bold">Eva Fischer-Fodor</span>*<sup><span class="sup_ref italic">a</span></sup>, <span class="bold">Natalia Moldovan</span><sup><span class="sup_ref italic">b</span></sup>, <span class="bold">Piroska Virag</span><sup><span class="sup_ref italic">a</span></sup>, <span class="bold">Olga Soritau</span><sup><span class="sup_ref italic">a</span></sup>, <span class="bold">Ioana Brie</span><sup><span class="sup_ref italic">a</span></sup>, <span class="bold">Peter Lönnecke</span><sup><span class="sup_ref italic">c</span></sup>, <span class="bold">Evamarie Hey-Hawkins</span><sup><span class="sup_ref italic">c</span></sup><span class="bold"> and </span><span class="bold">Luminita Silaghi-Dumitrescu</span><sup><span class="sup_ref italic">b</span></sup><br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Tumor Biology Laboratory, Research Department, “I. Chiricuta” Cancer Institute, 34–36 Republici St., 400015, Cluj Napoca, Romania. E-mail: <a href="mailto:fischer.eva@iocn.ro">fischer.eva@iocn.ro</a>;  Fax: +40 264 439 260;   Tel: +40 262 450 673</span><br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Chemistry Faculty, Babes-Bolyai University, Romania</span><br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">Institut für Anorganische Chemie, Universität Leipzig, Germany</span></p><div id="art-admin"><span class="italic bold">Received 11th February 2008</span><span class="bold italic">, Accepted 22nd July 2008</span></div><p class="bold italic">First published on 6th October 2008</p><hr/><div class="abstract"><h2>Abstract</h2><p>The present paper examines one of the multiple uses of the CellScan technique, its utilization in the biological evaluation of novel compounds, in order to improve the methods concerning the prediction of their anti-proliferative effects and application as anti-cancer <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span>. The CellScan is a laser scanning static cytometer, enabling repetitive spectroscopic measurements in intact living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. The <span class="AN"><a class="simple" title="Gene Ontology link for: detection of" href="javascript:popupOBO('GO:0014865','b802364f')">detection of</a></span> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> viability and <span class="AN"><a class="simple" title="Gene Ontology link for: apoptosis" href="javascript:popupOBO('GO:0006915','b802364f')">apoptosis</a></span> is possible, based on the transformations that occur in the <span class="AN"><a class="simple" title="Gene Ontology link for: cytoplasm" href="javascript:popupOBO('GO:0005737','b802364f')">cytoplasm</a></span> matrix of tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> influenced by cytotoxic compounds. The measurement of fluorescence changes, due to this phenomenon, is possible with the CellScan system. The potential of this technology to detect the <span class="italic">in vitro</span> effects of the inhibitory molecules on tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> was demonstrated, making this method a valuable tool for chemosensitivity tests. We synthesized and fully characterised three novel <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes of tertiary <span class="TC"><a class="simple" title="Chemspider Compound link for:arsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22408.html">arsine</a></span> ligands: <span class="italic">trans</span>-[PtI<small><sub>2</sub></small>(2-<small><sup><span class="italic">i</span></sup></small>PrOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>), <span class="italic">trans</span>-[PtCl<small><sub>2</sub></small>(2-MeOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<strong>2</strong>) and <span class="italic">cis</span>-[PtCl<small><sub>2</sub></small>(2-HOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<strong>3</strong>). The three compounds are biologically active against tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> and their cytotoxicity is comparable with standard <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span>. Measurements using the CellScan technology correlate well with the results provided by other bioassay methods.</p></div><hr/><h2 id="sect306"><span class="a_heading">Introduction</span></h2><span>Metal-based chemotherapeutic agents, above all <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes, are an important class of anti-tumour compounds that have had a significant impact on the treatment of a variety of tumours for nearly 30 years.</span><p class="otherpara">The unique efficiency observed with standard <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span>-based <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span>, such as cisplatin, <span class="italic">cis</span>-diamminecyclobutanedicarboxylato-<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:platinum(ii)" target="_blank" href="http://www.chemspider.com/Chemical-Structure.94874.html">platinum(<span class="small_caps">II</span>)</a></span> (carboplatin) and 1,2-diaminocyclohexaneoxalato-<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:platinum(ii)" target="_blank" href="http://www.chemspider.com/Chemical-Structure.94874.html">platinum(<span class="small_caps">II</span>)</a></span> (oxaliplatin) in cancer treatment, fuelled the development of new <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> analogues.<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> In the last decade, it was demonstrated that several newly synthesized <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> compounds exhibit biological activity against tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>, and their anti-mitotic properties were proven by several procedures.<a title="Select to navigate to reference" href="#cit2"><sup><span class="sup_ref">2–8</span></sup></a></p><p class="otherpara">The CellScan technology is based on a spectrometric method which allows repetitive multi-parameter measurements to be performed on intact living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> exposed to different external stimuli. The equipment is designed to perform fluorescence-based cytometric studies.<a title="Select to navigate to references" href="#cit9"><sup><span class="sup_ref">9</span></sup></a> Using this straightforward and rapid method, it was possible to establish the cytotoxicity and apoptotic capacity of chemotherapeutic agents.<a title="Select to navigate to reference" href="#cit10"><sup><span class="sup_ref">10,11</span></sup></a> Our aim is to establish a procedure to study cytotoxicity, genotoxicity, and anti-proliferative effects of novel compounds.</p><p class="otherpara">We synthesized a new <span class="CH"><a class="simple" title="ChEBI Ontology link for: group" href="javascript:popupOBO('CHEBI:24433','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">group</a></span> of <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes with the general formula [PtX<small><sub>2</sub></small>L<small><sub>2</sub></small>] (X = I, Cl; L = <span class="TC"><a class="simple" title="Chemspider Compound link for:diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.92044.html">diphenylarsine</a></span> ligand) in order to determine their potential biological activity. These three <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes have organometallic ligands with structures, which lead us to presume that they exhibit biological effects: <span class="italic">trans</span>-[PtI<small><sub>2</sub></small>(2-<span class="italic"><small><sup>i</sup></small></span>PrOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>), <span class="italic">trans</span>-[PtCl<small><sub>2</sub></small>(2-MeOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<strong>2</strong>) and <span class="italic">cis</span>-[PtCl<small><sub>2</sub></small>(2-HOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<strong>3</strong>). Each structure contains two identical tertiary <span class="CH"><a class="simple" title="ChEBI Ontology link for: arsine" href="javascript:popupOBO('CHEBI:22637','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22637')">arsine</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> bonded to platin, and the only difference between these <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> being one moiety, which is <span class="CH"><a class="simple" title="ChEBI Ontology link for: isopropyl" href="javascript:popupOBO('CHEBI:30353','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30353')">isopropyl</a></span> for compound (<span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>), <span class="CH"><a class="simple" title="ChEBI Ontology link for: methyl" href="javascript:popupOBO('CHEBI:32875','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=32875')">methyl</a></span> for (<strong>2</strong>) and hydroxyl for compound (<strong>3</strong>). The molecular structures of (<span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>), (<strong>2</strong>) and (<strong>3</strong>) are analogous.</p><p class="otherpara">For this purpose, we used the CellScan system, which permits the <span class="AN"><a class="simple" title="Gene Ontology link for: detection of" href="javascript:popupOBO('GO:0014865','b802364f')">detection of</a></span> fluorescence intensity (FI) and polarization (<span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span>) of different <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> types. There is a strong correlation between the <span class="AN"><a class="simple" title="Gene Ontology link for: inhibition of cell growth" href="javascript:popupOBO('GO:0030308','b802364f')">inhibition of cell growth</a></span> induced by different chemotherapeutic agents and the resultant changes in FI and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span>. Thus, changes can be revealed by staining treated <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> with <span class="CH"><a class="simple" title="ChEBI Ontology link for: fluorescent dyes" href="javascript:popupOBO('CHEBI:51121','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51121')">fluorescent dyes</a></span> such <span class="TC"><a class="simple" title="Chemspider Compound link for:Fluorescein diacetate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.58558.html">Fluorescein diacetate</a></span> (FDA) and Rhodamine 123 (Rh123). Hyperpolarization of the fluorescence intensity detected by the CellScan system can identify apoptotic <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> within a complex <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> population; it is a marker of the early apoptotic processes.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a></p><p class="otherpara">To verify the accuracy of the new method, we correlated the results obtained in this way with those determined with customary procedures, such as the MTT (MTT = <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide" target="_blank" href="http://www.chemspider.com/Chemical-Structure.10624941.html">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</a></span>) viability test<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> and one of the standard methods for assessing <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> damage, namely, the single-<span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> gel ‘comet’ assay.<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">14,15</span></sup></a></p><p class="otherpara">Tertiary <span class="CH"><a class="simple" title="ChEBI Ontology link for: arsine" href="javascript:popupOBO('CHEBI:22637','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22637')">arsine</a></span> complexes of <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> and other metals have been recently reported,<a title="Select to navigate to reference" href="#cit16"><sup><span class="sup_ref">16–18</span></sup></a> but there is no information regarding their biological activity. Among phenylarsines, the information on the biological activity was reported for <span class="TC"><a class="simple" title="Chemspider Compound link for:phenylarsine oxide" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4614.html">phenylarsine oxide</a></span> and <span class="TC"><a class="simple" title="Chemspider Compound link for:dichlorophenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.12238.html">dichlorophenylarsine</a></span>,<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19–23</span></sup></a> and the <span class="TC"><a class="simple" title="Chemspider Compound link for:phenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.393601.html">phenylarsine</a></span> compounds attached to biomolecules like <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptides" href="javascript:popupOBO('CHEBI:16670','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptides</a></span> and enzymes were studied.<a title="Select to navigate to reference" href="#cit24"><sup><span class="sup_ref">24,25</span></sup></a> <span class="TC"><a class="simple" title="Chemspider Compound link for:Phenylarsine oxide" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4614.html">Phenylarsine oxide</a></span> derivates are well known as anti-trypanosome agents and they are used as <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span>.<a title="Select to navigate to reference" href="#cit26"><sup><span class="sup_ref">26–28</span></sup></a></p><h2 id="sect410"><span class="a_heading">Results and discussion</span></h2><h3 id="sect412"><span class="b_heading">Synthesis and characterization of Pt complexes with organoarsine ligands</span></h3><span>Complexes of the general formula [PtX<small><sub>2</sub></small>L<small><sub>2</sub></small>], (X = I, L = <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-isopropoxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.25935183.html">(2-isopropoxyphenyl)diphenylarsine</a></span>, X = Cl, L = <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-methoxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774018.html">(2-methoxyphenyl)diphenylarsine</a></span>, <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html">(2-hydroxyphenyl)diphenylarsine</a></span>) were prepared by the reaction of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: arsine" href="javascript:popupOBO('CHEBI:22637','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22637')">arsine</a></span> ligands with [PtX<small><sub>2</sub></small>(COD)] (COD = <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:1,5-cyclooctadiene" target="_blank" href="http://www.chemspider.com/Chemical-Structure.74815.html">1,5-cyclooctadiene</a></span>) in a molar ratio of 2 : 1 (<a title="Select to navigate to scheme" href="#imgsch1">Scheme 1</a>).</span><p class="otherpara">In the case of the complexes with <span class="TC"><a class="simple" title="Chemspider Compound link for:diphenylarsinophenol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html">diphenylarsinophenol</a></span> ethers (<span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and <strong>2</strong>) both reagents were dissolved in <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:dichloromethane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6104.html">dichloromethane</a></span> with stirring at room temperature, and the final products were recrystallized from <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:chloroform" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5977.html">chloroform</a></span> or from <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:dichloromethane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6104.html">dichloromethane</a></span> to obtain crystals suitable for <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: X-ray diffraction" href="javascript:popupOBO('CMO:0000156','b802364f')">X-ray diffraction</a></span>.</p><p class="otherpara">The <span class="TC"><a class="simple" title="Chemspider Compound link for:diphenylarsinophenol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html">diphenylarsinophenol</a></span> containing <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complex (<strong>3</strong>) was prepared by slow diffusion of <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvents" href="javascript:popupOBO('CHEBI:46787','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvents</a></span>; the <span class="CH"><a class="simple" title="ChEBI Ontology link for: arsine" href="javascript:popupOBO('CHEBI:22637','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22637')">arsine</a></span> ligand was dissolved in <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:toluene" target="_blank" href="http://www.chemspider.com/Chemical-Structure.1108.html">toluene</a></span> and added very slowly as a layer to a solution of [PtCl<small><sub>2</sub></small>(COD)] in <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:dichloromethane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6104.html">dichloromethane</a></span>. The final product was isolated as crystals suitable for <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: X-ray structure determination" href="javascript:popupOBO('CMO:0000156','b802364f')">X-ray structure determination</a></span>.</p><p class="otherpara">In all complexes, the geometry around the metal center is square-planar and the organoarsenic ligands are coordinated to the transition metal through the <span class="TC"><a class="simple" title="Chemspider Compound link for:arsenic" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22408.html">arsenic</a></span> atom. Similar complexes of <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> and <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:palladium" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22380.html">palladium</a></span> with O and P donor ligands have been reported in the literature.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a></p><br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch1"><img alt="General procedure for the synthesis of platinum complexes with the tertiary arsine ligands." src="/image/article/2008/DT/b802364f/b802364f-s1.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 1 </b> <span id="sch1"><span class="graphic_title">General procedure for the synthesis of <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes with the tertiary <span class="CH"><a class="simple" title="ChEBI Ontology link for: arsine" href="javascript:popupOBO('CHEBI:22637','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22637')">arsine</a></span> ligands.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div><p class="otherpara"><a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a> and <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a> present the molecular structure of the new <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> compounds.</p><br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><img alt="ORTEP representations of the molecular structure of trans-[PtI2(2-iPrOC6H4AsPh2)2] (1) (left) and trans-[PtCl2(2-MeOC6H4AsPh2)2] (2) (right); 50% ellipsoids are shown (hydrogen atoms are omitted for clarity)." src="/image/article/2008/DT/b802364f/b802364f-f1.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">ORTEP representations of the molecular structure of <span class="italic">trans</span>-[PtI<small><sub>2</sub></small>(2-<span class="italic"><small><sup>i</sup></small></span>PrOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>) (left) and <span class="italic">trans</span>-[PtCl<small><sub>2</sub></small>(2-MeOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<strong>2</strong>) (right); 50% ellipsoids are shown (<span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span> atoms are omitted for clarity).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div><br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><img alt="ORTEP representation of the molecular structure of cis-[PtCl2(2-HOC6H4AsPh2)2] (3), with both independent molecules revealed; 50% ellipsoids are shown (hydrogen atoms are omitted for clarity)." src="/image/article/2008/DT/b802364f/b802364f-f2.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">ORTEP representation of the molecular structure of <span class="italic">cis</span>-[PtCl<small><sub>2</sub></small>(2-HOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>] (<strong>3</strong>), with both independent molecules revealed; 50% ellipsoids are shown (<span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span> atoms are omitted for clarity).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div><p class="otherpara">In <span class="italic">cis</span>-[PtCl<small><sub>2</sub></small>(2-HOC<small><sub>6</sub></small>H<small><sub>4</sub></small>AsPh<small><sub>2</sub></small>)<small><sub>2</sub></small>], two independent molecules are present in the asymmetric unit (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>) and they have similar structural properties.</p><h3 id="sect499"><span class="b_heading">Biological effects</span></h3><span>In the order to study the possible anti-proliferative and anti-mitotic effects of the compounds, we established their chemosensitivity on tumour and normal <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> lines. We chose the MLS human ovarian cancer <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line, the M1–15 highly metastatic melanoma <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line, and the Hfl-1 normal human lung <span class="AN"><a class="simple" title="Cell Ontology link for: fibroblast" href="javascript:popupOBO('CL:0000057','b802364f')">fibroblast</a></span> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line. The MLS <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were cultivated in a DMEM <span class="AN"><a class="simple" title="Gene Ontology link for: cell growth" href="javascript:popupOBO('GO:0016049','b802364f')">cell growth</a></span> media, supplemented with <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glutamine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.718.html">glutamine</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotics" href="javascript:popupOBO('CHEBI:22582','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotics</a></span> and fetal bovine serum. M1–15 requires a RPMI-1640 medium, while Hfl-1 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> are cultivated in the nutrient-rich medium F-12. The <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> cultures were maintained at 37 °C under a CO<small><sub>2</sub></small>-enriched sterile atmosphere in an incubator for <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> cultures.</span><div><span id="sect516"/><span class="c_heading_indent">IC50 values. </span><span>The effectiveness of the platinum complexes against <span class="AN"><a class="simple" title="Gene Ontology link for: cell proliferation" href="javascript:popupOBO('GO:0008283','b802364f')">cell proliferation</a></span> was measured using a quantitative colorimetric assay. As described previously,<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide" target="_blank" href="http://www.chemspider.com/Chemical-Structure.10624941.html">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</a></span> (MTT) was used to assess cytotoxicity. This pale yellow salt is reduced only by the <span class="AN"><a class="simple" title="Gene Ontology link for: mitochondria" href="javascript:popupOBO('GO:0005739','b802364f')">mitochondria</a></span> of living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> to its blue formazan derivative. Using a multi-well scanning <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrophotometer" href="javascript:popupOBO('CMO:0002232','b802364f')">spectrophotometer</a></span> (ELISA reader) we measured the colour intensity, which is proportional to the number of living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>.</span><p class="otherpara">The test was performed on MLS, M1–15 and Hfl-1 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. In 96-welled flat-bottomed microculture plates, <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were suspended in a concentration of 12<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>500 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> per well, in 200 μl of <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> culture media. After 24 h incubation in the incubator, each compound was screened for <span class="italic">in vitro</span> anticancer activity by treating the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> lines with a wide range of concentrations of the platinum complex solutions. For <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong>, we used the concentration range 0.01–90 μM mL<small><sup>−1</sup></small>, for carboplatin and oxaliplatin the range was between 0.01–50 μM. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvent" href="javascript:popupOBO('CHEBI:46787','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvent</a></span> containing the upper limit 1 : 100 <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ethanol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.682.html">ethanol</a></span>–PBS (<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:phosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.1032.html">phosphate</a></span> buffered salt solution) was tested separately and displayed no cytotoxic effect, which could affect our results. Three different experiments were performed for each complex.</p><p class="otherpara">Measurements were made after 48 h of incubation with the complexes, by removing the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> culture media, staining the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> with the MTT <span class="CH"><a class="simple" title="ChEBI Ontology link for: dye" href="javascript:popupOBO('CHEBI:37958','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=37958')">dye</a></span>, and then by dissolving the formazan crystals with dimethylsulfoxyde (<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:DMSO" target="_blank" href="http://www.chemspider.com/Chemical-Structure.659.html">DMSO</a></span>). The chemosensitivity was expressed as an IC50 value, that is, the concentration of the compound causing 50% <span class="AN"><a class="simple" title="Gene Ontology link for: cell death" href="javascript:popupOBO('GO:0008219','b802364f')">cell death</a></span>. IC50 values were obtained from dose-response curves after fitting the data to a linear quadratic equation using the Graph Pad Prism biostatistics program (95% confidence intervals, <span class="italic">r</span><small><sup>2</sup></small> = 0.94, <span class="italic">P</span> &lt;0.05).</p><p class="otherpara">The IC50 values obtained are higher against normal Hfl-1 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> in all cases (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>), that is, <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong> are more toxic against MLS ovarian tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. The less toxic is <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and the most toxic is <strong>3</strong>. IC50 values of carboplatin and oxaliplatin against MLS ovarian tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> are represented for comparison with novel synthesized compounds.</p><br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><img alt="The influence of complexes 1, 2 and 3 on normal (Hfl-1) and tumour (MLS and M1–15) cell lines, expressed as IC50 values. C1 represents the platinum complex 1, C2 complex 2 and C3 complex 3, CarboPt is the abbreviation for carboplatin and OxaliPt is the abbreviation for oxaliplatin." src="/image/article/2008/DT/b802364f/b802364f-f3.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">The influence of complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong> on normal (Hfl-1) and tumour (MLS and M1–15) <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> lines, expressed as IC50 values. C1 represents the platinum complex <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, C2 complex <strong>2</strong> and C3 complex <strong>3</strong>, CarboPt is the abbreviation for carboplatin and OxaliPt is the abbreviation for oxaliplatin.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div><p class="otherpara">The cytotoxicity of the platinum complexes studied is comparable with the anti-tumour activity of carboplatin, a widely used <span class="CH"><a class="simple" title="ChEBI Ontology link for: drug" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drug</a></span> in ovarian cancer treatment. The studied compounds are all more active than oxaliplatin against ovarian tumor <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>.</p><p class="otherpara">The highly metastatic M1–15 melanoma <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line is more resistant to the complexes and the IC50 values show a lower cytotoxicity in comparison with the MLS <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>; however, the compounds are still more active against the melanoma <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line <span class="italic">versus</span> normal Hfl-1 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>, except <strong>3</strong>, which shows a lower toxicity against the highly metastatic M1–15 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>.</p><p class="otherpara">These values helped us to choose the concentrations used for the CellScan determinations.</p></div><div><span id="sect598"/><span class="c_heading_indent">CellScan <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrometry" href="javascript:popupOBO('CMO:0000228','b802364f')">spectrometry</a></span>. </span><span>In this experiment, we utilized the MLS, Hfl-1 and M1–15 <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> lines. The <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were suspended in <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b802364f')">growth</a></span> media in 6-well microplates, and were incubated for 24 h, to allow them to attach to the plate surface. The plates were treated with <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong> solutions, and also with carboplatin and oxaliplatin. As a control, we used untreated <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. In each case, we used a concentration of 5 mM. After 48 h of incubation with the cytotoxic agents, the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were stained with the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fluorescent dyes" href="javascript:popupOBO('CHEBI:51121','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51121')">fluorescent dyes</a></span> <span class="TC"><a class="simple" title="Chemspider Compound link for:Fluorescein diacetate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.58558.html">Fluorescein diacetate</a></span> and Rhodamine 123, and measurements were made with the CellScan system.</span><p class="otherpara">The CellScan apparatus is a laser scanning, static, multi-parameter cytometer<a title="Select to navigate to reference" href="#cit10"><sup><span class="sup_ref">10,30</span></sup></a> measuring both fluorescence intensity (FI) and polarization fluorescence (<span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span>) of individually fluorescent-labelled <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. The simplicity of this system provides a new perspective in clinical studies and in the basic research area. The CellScan system enables a repetitive spectroscopic measurements on intact living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> within a population under certain physiological conditions. It incorporates a unique ‘<span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b802364f')">carrier</a></span>’, which allows for repeated high-precision measurements by fixing the location of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> in the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b802364f')">carrier</a></span>. We can perform repetitive measurements on the same populations and on the same <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> after various stimulations and different set-ups of the scanner.<a title="Select to navigate to references" href="#cit31"><sup><span class="sup_ref">31</span></sup></a></p><p class="otherpara">The ‘core’ of the <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b802364f')">carrier</a></span> is a 2 mm square grid of precisely dimensioned conical apertures, 5–8 μm in diameter, spaced at 20 μm, which can hold 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>, each loaded and maintained in a separate trap (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>). Each aperture is dimensioned to hold one single <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span>; <span class="AN"><a class="simple" title="Cell Ontology link for: lymphocytes" href="javascript:popupOBO('CL:0000542','b802364f')">lymphocytes</a></span> and human tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> dimensions are compatible with the aperture diameter. We used carriers with 8 μm sized apertures. The <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were loaded in the grid using a gradient of pressure. A negative pressure of 1–5 mm H<small><sub>2</sub></small>O, maintained between the upper and the lower surfaces of the <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b802364f')">carrier</a></span>, caused a rapid fixing of the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> into the orifices. The loaded <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>, once trapped into the conical orifice, can not leave this and remain fixed during various procedures and manipulations: staining, washing, microscopic and spectrometric measurements.</p><br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><img alt="Fluorescent inverse phase microscope image of FDA stained MLS tumour cells trapped in the orifices of the cell carrier (20-fold magnitude)." src="/image/article/2008/DT/b802364f/b802364f-f4.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Fluorescent inverse phase <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: microscope image" href="javascript:popupOBO('CMO:0001824','b802364f')">microscope image</a></span> of FDA stained MLS tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> trapped in the orifices of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b802364f')">carrier</a></span> (20-fold magnitude).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div><p class="otherpara">The static He–Cd laser cytometer scans each hole and displays the spread of intensity (FI) on the <span class="italic">x</span> axis and polarization (<span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span>) on the <span class="italic">y</span> axis on the screen (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>). The illumination intensity of the laser beam is 5 μW, the intensity and polarization values measured at two wavelengths (527 and 510 nm) characterize each living <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span>. The measurements are made using a preset <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span>-counting technique, in which the same number of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> is counted for each <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span>, regardless of emissions intensity, so that the measurement has the same precision both for weakly and strongly emitting <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>.</p><br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><img alt="Histogram provided by the CellScan: the polarization and the intensity for each cell, measured at two wavelengths are represented by red and green dots. In Fig. 5a the untreated (control) MLS cells are shown, (b) and (c) represents FI and FP of MLS cells treated with complex 1, respectively." src="/image/article/2008/DT/b802364f/b802364f-f5.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">Histogram provided by the CellScan: the polarization and the intensity for each <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span>, measured at two wavelengths are represented by red and green dots. In Fig. 5a the untreated (control) MLS <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> are shown, (b) and (c) represents FI and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> of MLS <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> treated with complex <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, respectively.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div><p class="otherpara">The fluorescence polarization (<span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span>) indicates the entropy of the fluorescent molecules, that is, the relative rotational freedom. The <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> of a <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:fluorochrome" target="_blank" href="http://www.chemspider.com/Chemical-Structure.8029248.html">fluorochrome</a></span> increases with increasing viscosity or restriction of molecular rotation. <span class="AN"><a class="simple" title="Gene Ontology link for: Cell activation" href="javascript:popupOBO('GO:0001775','b802364f')">Cell activation</a></span> and <span class="AN"><a class="simple" title="Gene Ontology link for: apoptosis" href="javascript:popupOBO('GO:0006915','b802364f')">apoptosis</a></span> involve conformational changes of cytosolic enzymes and <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, that is, <span class="AN"><a class="simple" title="Gene Ontology link for: intracellular" href="javascript:popupOBO('GO:0005622','b802364f')">intracellular</a></span> matrix reorganization. The more the molecule's rotational movement is restricted, the higher the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> value, and <span class="italic">vice versa</span>. The fluorescence intensity (FI) is expressed as the number of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> counts per unit of time and the FI of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fluorochrome" href="javascript:popupOBO('CHEBI:51217','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51217')">fluorochrome</a></span> reflects <span class="AN"><a class="simple" title="Gene Ontology link for: intracellular" href="javascript:popupOBO('GO:0005622','b802364f')">intracellular</a></span> content.</p><p class="otherpara">The <span class="CH"><a class="simple" title="ChEBI Ontology link for: dyes" href="javascript:popupOBO('CHEBI:37958','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=37958')">dyes</a></span> used for the measurements are FDA and Rh123. FDA is a non-fluorescent <span class="CH"><a class="simple" title="ChEBI Ontology link for: dye" href="javascript:popupOBO('CHEBI:37958','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=37958')">dye</a></span>, which is converted to its fluorescent form by enzymatic <span class="CH"><a class="simple" title="Molecular Process Ontology link for: hydrolysis" href="javascript:popupOBO('MOP:0000619','b802364f')">hydrolysis</a></span> only in living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> and it is an <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> of <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> viability. Rh 123 is a lipophilic cationic <span class="CH"><a class="simple" title="ChEBI Ontology link for: fluorochrome" href="javascript:popupOBO('CHEBI:51217','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51217')">fluorochrome</a></span> that selectively accumulates in the <span class="AN"><a class="simple" title="Gene Ontology link for: mitochondria" href="javascript:popupOBO('GO:0005739','b802364f')">mitochondria</a></span> of living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> and whose specific uptake or release depends upon the <span class="AN"><a class="simple" title="Gene Ontology link for: mitochondrial membrane" href="javascript:popupOBO('GO:0031966','b802364f')">mitochondrial membrane</a></span> potential and it therefore reflects mitochondrial activity. A drop in the FI and an increase in the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> of Rh123 was attributed to early <span class="AN"><a class="simple" title="Gene Ontology link for: apoptosis" href="javascript:popupOBO('GO:0006915','b802364f')">apoptosis</a></span>.<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11,13</span></sup></a></p><p class="otherpara">A <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> value obtained as a result of the measurements using FDA-stained <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> gives us information about the viability of the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>, and the inhibitory effect of the treatment. The <span class="AN"><a class="simple" title="Gene Ontology link for: cell growth" href="javascript:popupOBO('GO:0016049','b802364f')">cell growth</a></span> inhibition is demonstrated if the treated <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> population displays a high <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> value and a high FI value than the untreated control and no hyperpolarization of FDA or Rh123 was observed in the resistant <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>.<a title="Select to navigate to reference" href="#cit9"><sup><span class="sup_ref">9,10</span></sup></a> In our experiment, an increase of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> was observed for the platinum complex <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong> relative to the control for both the MLS and M1–15 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>).</p><p class="otherpara">Measurements show the cytotoxic activity of the studied <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes and the fact that against MLS <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> lines they display a cytotoxic effect as high as carboplatin and oxaliplatin. The increase of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> values ratio is not so prominent in the experiment with the highly metastatic M1–15 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>.</p><p class="otherpara">The Rh123 fluorescent marker gives us information about the early apoptotic processes, which affect cytoplasmic viscosity and alterations in <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> permeability. The values from <a title="Select to navigate to table" href="#tab1">Table 1</a> show us that <strong>2</strong> displays the most significant capacity to induce <span class="AN"><a class="simple" title="Gene Ontology link for: apoptosis" href="javascript:popupOBO('GO:0006915','b802364f')">apoptosis</a></span>, in this case, the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> increase is the biggest, both for MLS and M1–15 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>, although against M1–15 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> we observed a smaller increase in <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> values. Despite its IC50 value, <strong>3</strong> does not induce <span class="AN"><a class="simple" title="Gene Ontology link for: apoptosis" href="javascript:popupOBO('GO:0006915','b802364f')">apoptosis</a></span>.</p>
<div class="table_caption"><b>Table 1</b> <span id="tab1">Fluorescence polarization values</span>
		</div>
<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/></colgroup><thead align="left"><tr align="left" valign="bottom"><th rowspan="2" align="left" valign="bottom" class="border_black">Stimulant</th><th colspan="3" align="left" valign="bottom" class="border_black"><span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> values obtained with FDA staining/a.u.</th><th colspan="2" align="left" valign="bottom" class="border_black"><span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FP" href="javascript:popupOBO('CMO:0002509','b802364f')">FP</a></span> values obtained with Rh123 staining/a.u.</th></tr><tr align="left" valign="bottom"><th align="left" valign="top" class="border_black">Hfl-1</th><th align="left" valign="top" class="border_black">MLS</th><th align="left" valign="top" class="border_black">M1–15</th><th align="left" valign="top" class="border_black">MLS</th><th align="left" valign="top" class="border_black">M1–15</th></tr></thead><tbody align="left"><tr align="left"><td valign="top">Control</td><td valign="top">0.409</td><td valign="top">0.316</td><td valign="top">0.263</td><td valign="top">0.362</td><td valign="top">0.415</td></tr><tr align="left"><td valign="top">Oxaliplatin</td><td valign="top">0.453</td><td valign="top">0.275</td><td valign="top">0.241</td><td valign="top">0.358</td><td valign="top">0.439</td></tr><tr align="left"><td valign="top">Carboplatin</td><td valign="top">0.461</td><td valign="top">0.330</td><td valign="top">0.274</td><td valign="top">0.398</td><td valign="top">0.404</td></tr><tr align="left"><td valign="top">Complex <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span></td><td valign="top">0.420</td><td valign="top">0.332</td><td valign="top">0.275</td><td valign="top">0.371</td><td valign="top">0.419</td></tr><tr align="left"><td valign="top">Complex <strong>2</strong></td><td valign="top">0.412</td><td valign="top">0.321</td><td valign="top">0.297</td><td valign="top">0.391</td><td valign="top">0.426</td></tr><tr align="left"><td valign="top">Complex <strong>3</strong></td><td valign="top">0.439</td><td valign="top">0.327</td><td valign="top">0.285</td><td valign="top">0.352</td><td valign="top">0.404</td></tr></tbody></table></div></div><hr class="hrule"/><br/><p class="otherpara"><span class="TC"><a class="simple" title="Chemspider Compound link for:Platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">Platinum</a></span> tertiary <span class="CH"><a class="simple" title="ChEBI Ontology link for: arsine" href="javascript:popupOBO('CHEBI:22637','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22637')">arsine</a></span> cytotoxicity (<span class="AN"><a class="simple" title="Gene Ontology link for: cell growth" href="javascript:popupOBO('GO:0016049','b802364f')">cell growth</a></span> inhibition) lies in a range that is comparable with other compounds that have less toxic ligands. The complexes have poor solubility and are stable in <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: phosphate buffered saline" href="javascript:popupOBO('CMO:0001751','b802364f')">phosphate buffered saline</a></span> solutions, without the release of the ligand.</p></div><div><span id="sect812"/><span class="c_heading_indent">Comet assay. </span><span>We also performed the alkaline single-<span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: gel electrophoresis" href="javascript:popupOBO('CMO:0001021','b802364f')">gel electrophoresis</a></span> or ‘comet’ assay, which detects <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> strand breaks, alkali-labile sites and cross-links in <span class="AN"><a class="simple" title="Cell Ontology link for: eukaryotic cells" href="javascript:popupOBO('CL:0000255','b802364f')">eukaryotic cells</a></span>. It is based on electrophoretic mobility and reveals changes in the tertiary <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> structure of the exposed <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. The procedure is widely used in testing novel chemicals for genotoxicity, including <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes and <span class="CH"><a class="simple" title="ChEBI Ontology link for: organometallic compounds" href="javascript:popupOBO('CHEBI:25707','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25707')">organometallic compounds</a></span>.<a title="Select to navigate to reference" href="#cit32"><sup><span class="sup_ref">32–35</span></sup></a></span><p class="otherpara">MLS tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were treated with complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong>, carboplatin and oxaliplatin at the same concentrations used for the polarization measurements. The <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were placed in six-well plates, and they were incubated in <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b802364f')">growth</a></span> media for 24 h, to allow them to attach to the plate surface. Treatment was made with <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong> solutions, and also with carboplatin and oxaliplatin. As a control, we used untreated <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. We used the same concentrations as for the static cytometry measurements and 48 h effects were considered.</p><p class="otherpara">The method used by us was described previously<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> and it was performed according to the alkaline version of the comet assay test.<a title="Select to navigate to reference" href="#cit10"><sup><span class="sup_ref">10,11</span></sup></a> <span class="AN"><a class="simple" title="Cell Ontology link for: Cells" href="javascript:popupOBO('CL:0000000','b802364f')">Cells</a></span> incorporated in agarose gel on a <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: microscope" href="javascript:popupOBO('CMO:0000953','b802364f')">microscope</a></span> slide are lysed with an alkaline lysis solution (pH &gt; 10) to denude <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleotides" href="javascript:popupOBO('CHEBI:36976','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36976')">nucleotides</a></span> containing supercoiled loops of <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span>. After this operation, gels are placed in a pH &gt; 13 <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: buffer solution" href="javascript:popupOBO('CMO:0001462','b802364f')">buffer solution</a></span> for electrophoretic migration, during which damaged <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> loops containing breaks lose their supercoils and migrate to the anode.</p><p class="otherpara">The damaged <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> display an increased electrophoretic migration of the <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> fragments by forming outlines resembling comets, in contrast to the round appearance of undamaged <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span>. The length and form of the comet ‘tail’ reveal the levels of damage. We considered five possible stages based on the distance of <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> migration ‘tail length’. The migration of the <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> fragments from the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleus" href="javascript:popupOBO('CHEBI:33252','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33252')">nucleus</a></span> is visualized by manual microscopic analysis, which is equal in sensitivity to computerized analysis.<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a></p><p class="otherpara">We evaluated the <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> damage by means of two parameters: lesion score (LS) and tail factor (TF), both expressed in arbitrary units. The lesion score represents the product of the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> ratio and stage. The tail factor quantifies the global damage in the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> by associating the average proportion of <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> migrated in the comet tail and it represents the percentage of <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> in the tail relative to the amount of <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> in the entire comet. We considered the values obtained for untreated <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> as a ‘control’ reference for each case.</p><p class="otherpara">In every case, we obtained a positive result, that is, a significant increase in LS and TF <span class="italic">versus</span> the control. The comet assay results show that <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b802364f')">DNA</a></span> damage induced by the commonly used <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span>-based <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span> and the complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and <strong>2</strong> are comparable, while compound <strong>3</strong> induces the most strand lesions among the studied compounds (<a title="Select to navigate to table" href="#tab2">Table 2</a>). Massive amounts of stage 1 and stage 2 lesions support the genotoxicity of <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span>-based <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span> described in the literature. In the case of complex <strong>3</strong>, the huge number of stage 4 lesions is confirmation of the cytotoxic effect. Complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and <strong>2</strong> show genotoxicity and an increase in stage 2 lesions is also observed. Complex <strong>2</strong> also exhibits many stage 3 lesions, which confirm its cytotoxic effect.</p>
<div class="table_caption"><b>Table 2</b> <span id="tab2">Lesion scores (LS) and tail factors (TF) for <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes and platin-based <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span></span>
		</div>
<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/></colgroup><thead align="left"><tr align="left" valign="bottom"><th align="left" valign="bottom" class="border_black">Stage</th><th align="left" valign="bottom" class="border_black">Control</th><th align="left" valign="bottom" class="border_black">Oxaliplatin</th><th align="left" valign="bottom" class="border_black">Carboplatin</th><th align="left" valign="bottom" class="border_black">Complex <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span></th><th align="left" valign="bottom" class="border_black">Complex <strong>2</strong></th><th align="left" valign="bottom" class="border_black">Complex <strong>3</strong></th></tr></thead><tbody align="left"><tr align="left"><td valign="top">0</td><td valign="top">72</td><td valign="top">2</td><td valign="top">13</td><td valign="top">20</td><td valign="top">13</td><td valign="top">19</td></tr><tr align="left"><td valign="top">1</td><td valign="top">15</td><td valign="top">58</td><td valign="top">49</td><td valign="top">47</td><td valign="top">23</td><td valign="top">17</td></tr><tr align="left"><td valign="top">2</td><td valign="top">8</td><td valign="top">29</td><td valign="top">18</td><td valign="top">9</td><td valign="top">17</td><td valign="top">7</td></tr><tr align="left"><td valign="top">3</td><td valign="top">2</td><td valign="top">3</td><td valign="top">0</td><td valign="top">2</td><td valign="top">11</td><td valign="top">1</td></tr><tr align="left"><td valign="top">4</td><td valign="top">3</td><td valign="top">8</td><td valign="top">20</td><td valign="top">22</td><td valign="top">24</td><td valign="top">56</td></tr><tr align="left"><td valign="top">Sum</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td><td valign="top">100</td></tr><tr align="left"><td valign="top">LS</td><td valign="top">35</td><td valign="top">157</td><td valign="top">165</td><td valign="top">159</td><td valign="top">186</td><td valign="top">258</td></tr><tr align="left"><td valign="top">TF</td><td valign="top">7.95</td><td valign="top">25.83</td><td valign="top">31.35</td><td valign="top">34.98</td><td valign="top">41.00</td><td valign="top">59.98</td></tr></tbody></table></div></div><hr class="hrule"/><br/></div><h2 id="sect977"><span class="a_heading">Conclusions</span></h2><span>The CellScan system has already proved its utility in a number of important biochemical, immunological, kinetic, pharmacological and physicochemical applications and is an appropriate tool for analysing newly synthesized compounds. It is a simple and rapid method to assess cytotoxicity and genotoxicity. The results obtained using this technique are confirmed by literature data and by convergence with the data acquired by other widely used methods. It requires only a small number of <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> to perform the <span class="italic">in vitro</span> studies, and the measurements can be repeated on the same <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> population because the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b802364f')">carrier</a></span> used by the system has some unique characteristics that permit the positions of living <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> during the experiment to be maintained and revealed.</span><p class="otherpara">The new <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes with tertiary <span class="CH"><a class="simple" title="ChEBI Ontology link for: arsine" href="javascript:popupOBO('CHEBI:22637','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22637')">arsine</a></span> ligands exhibit biological activity. Their activity against normal and tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> is differentiated as all are more active against tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> lines. Genotoxicity is evidenced for complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and <strong>2</strong>. Complex <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> is less toxic, while <strong>2</strong> has the greatest <span class="AN"><a class="simple" title="Gene Ontology link for: apoptosis" href="javascript:popupOBO('GO:0006915','b802364f')">apoptosis</a></span>-induction capacity and the largest gradient, in effect, against normal/malignant <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span>. We suppose that this is a consequence of the presence of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: methyl" href="javascript:popupOBO('CHEBI:32875','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=32875')">methyl</a></span> moiety, which leads to <span class="CH"><a class="simple" title="Molecular Process Ontology link for: methylation" href="javascript:popupOBO('MOP:0000364','b802364f')">methylation</a></span> of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span>. The cytotoxicity of the compounds is analogous to that of standard anticancer chemotherapeutic <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span> and it is comparable with that of other new <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> complexes that exhibit biological activity.</p><p class="otherpara">The CellScan technology was useful in the analysis of the three compounds as it has a high sensitivity and selectivity. This procedure can replace other more difficult and lengthy methods for assessing biological activity.</p><p class="otherpara">The studied compounds are also suitable for testing <span class="italic">in vivo.</span> Similar compounds with improved solubility in biologically compatible <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvents" href="javascript:popupOBO('CHEBI:46787','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvents</a></span> should be synthesized in order to develop anticancer <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span>.</p><h2 id="sect1008"><span class="a_heading">Experimental</span></h2><h3 id="sect1010"><span class="b_heading">Materials</span></h3><div><span id="sect1012"/><span class="c_heading_indent">Complex synthesis. </span><span>All reactions were carried out under an atmosphere of dry <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> using standard Schlenk or vacuum line techniques. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvents" href="javascript:popupOBO('CHEBI:46787','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvents</a></span> were purified (<span class="CH"><a class="simple" title="ChEBI Ontology link for: toluene" href="javascript:popupOBO('CHEBI:27024','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=27024')">toluene</a></span> refluxed over Na–<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:benzophenone" target="_blank" href="http://www.chemspider.com/Chemical-Structure.2991.html">benzophenone</a></span>; CH<small><sub>2</sub></small>Cl<small><sub>2</sub></small> and <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:n-hexane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.7767.html"><span class="italic">n</span>-hexane</a></span> refluxed over powdered CaH<small><sub>2</sub></small>) and distilled under <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>. [PtX<small><sub>2</sub></small>(COD)] (X = Cl, I) was prepared according to the literature procedure.<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> All chemicals were of reagent grade and were used as received: <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:1,5-cyclooctadiene" target="_blank" href="http://www.chemspider.com/Chemical-Structure.74815.html">1,5-cyclooctadiene</a></span> was purchased from Merck and H<small><sub>2</sub></small>[PtCl<small><sub>6</sub></small>]·6H<small><sub>2</sub></small>O was generously donated by Umicore AG &amp; Co KG, Hanau-Wolfgang, Germany.</span></div><div><span id="sect1034"/><span class="c_heading_indent">Biological effects. </span><span>For this purpose we performed <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> cultures on both tumour and normal <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> lines.</span><p class="otherpara">The Hfl-1 <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line was acquired from Sigma Aldrich Corporation from the European Collection of <span class="AN"><a class="simple" title="Cell Ontology link for: Cell" href="javascript:popupOBO('CL:0000000','b802364f')">Cell</a></span> Cultures; The M1–15 melanoma <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line was a generous gift from Prof. Falus Andras, Semmelweiss University, Budapest, Hungary, whereas the MLS ovarian <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> line was a donation from Dr Yael S. Schiffenbauer from MedisEl Company, Lod, Israel.</p><p class="otherpara">The <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were defrosted carefully, cultivated in special culture flasks with adequate <span class="AN"><a class="simple" title="Gene Ontology link for: cell growth" href="javascript:popupOBO('GO:0016049','b802364f')">cell growth</a></span> media, under sterile conditions. <span class="AN"><a class="simple" title="Gene Ontology link for: Growth" href="javascript:popupOBO('GO:0040007','b802364f')">Growth</a></span> conditions were assured using incubators with a stable temperature (37 °C), 5% CO<small><sub>2</sub></small> atmosphere and constant humidity.</p><p class="otherpara">Several <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> passages were performed using enzymatic procedures. Experiments were made when confluence of the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> achieved 80% on the flask surface. <span class="AN"><a class="simple" title="Cell Ontology link for: Cell" href="javascript:popupOBO('CL:0000000','b802364f')">Cell</a></span> lines were adherent, and they were seeded on 96-well <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> culture plates for the IC50 measurements, and on 6-well plates for the cytometric and comet assay tests.</p><p class="otherpara">The <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> were incubated for 24 h before adding the platinum complexes. After the treatment with sterile solutions with differing concentrations of complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong> and <span class="TC"><a class="simple" title="Chemspider Compound link for:platinum" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22381.html">platinum</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span>, respectively, the culture plates were kept in incubator for another 24 and 48 h before the measurements were taken.</p><p class="otherpara">We made 3 independent experiments for the MTT test and measurements were made directly using the 96-well plates.</p><p class="otherpara">For the cytometric and comet assay tests, a removal from the 6-well plates was made mechanically and not enzymatically to avoid accidental <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> damage, which could be caused by the trypsine. For the cytometric assay we made 3 different experiments, and for the comet assay, for each compound, we performed 3 different electrophoretic migrations on 4 separate <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: microscope" href="javascript:popupOBO('CMO:0000953','b802364f')">microscope</a></span> slides.</p><p class="otherpara"><span class="AN"><a class="simple" title="Cell Ontology link for: Cell" href="javascript:popupOBO('CL:0000000','b802364f')">Cell</a></span> cultures were performed using Dulbecco's modified eagles medium, RPMI 1640 and F-12 <span class="AN"><a class="simple" title="Gene Ontology link for: cell growth" href="javascript:popupOBO('GO:0016049','b802364f')">cell growth</a></span> media from Sigma Aldrich Corporation. <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:Phosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.1032.html">Phosphate</a></span> buffered salt solution, trypsine <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:EDTA" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5826.html">EDTA</a></span>, MTT salt, <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ethidium bromide" target="_blank" href="http://www.chemspider.com/Chemical-Structure.14034.html">ethidium bromide</a></span>, low melting point agarose, type I agarose, FDA and Rh 123 <span class="CH"><a class="simple" title="ChEBI Ontology link for: fluorescent dyes" href="javascript:popupOBO('CHEBI:51121','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51121')">fluorescent dyes</a></span> were also provided by Sigma Aldrich. NaCl p.a., NaOH p.a. and EDTA·2HCl p.a. were obtained from Merck &amp; Co. Inc., and TrizmaHCl from Fluka Analytics.</p><p class="otherpara">The supplier of all the disposables used for the biological experiments was Nalgene Nunc International.</p><p class="otherpara">The chemotherapeutical <span class="CH"><a class="simple" title="ChEBI Ontology link for: drugs" href="javascript:popupOBO('CHEBI:23888','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23888')">drugs</a></span>, carboplatin (Carboplatin) and oxaliplatin (Eloxantine), were procured from Sindan Pharma.</p></div><h3 id="sect1092"><span class="b_heading">Instrumentation</span></h3><div><span id="sect1094"/><span class="c_heading_indent">Characterization of new compounds. </span><span><span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Infrared spectra" href="javascript:popupOBO('CMO:0000818','b802364f')">Infrared spectra</a></span> were recorded on a Perkin-Elmer System 2000 <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: FTIR spectrometer" href="javascript:popupOBO('CMO:0001823','b802364f')">FTIR spectrometer</a></span> scanning between 400–4000 cm<small><sup>−1</sup></small> using KBr disks and between 200–400 cm<small><sup>−1</sup></small> using CsI plates, and also on a Bruker-Vector 22 <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrometer" href="javascript:popupOBO('CMO:0001234','b802364f')">spectrometer</a></span> scanning between 200–4000 cm<small><sup>−1</sup></small>.</span><p class="otherpara">The <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: 1H NMR spectra" href="javascript:popupOBO('CMO:0000838','b802364f')"><small><sup>1</sup></small>H NMR spectra</a></span> of the transition metal complexes were recorded on an AVANCE 300 <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrometer" href="javascript:popupOBO('CMO:0001234','b802364f')">spectrometer</a></span> (Bruker). The chemical shifts for the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: 1H NMR spectra" href="javascript:popupOBO('CMO:0000838','b802364f')"><small><sup>1</sup></small>H NMR spectra</a></span> are reported in parts per million (ppm) at 300 MHz with <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:tetramethylsilane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6156.html">tetramethylsilane</a></span> as the standard. The assignments were based on those for the unbound ligand and those for analogous complexes. The elemental analyses were recorded on a VARIO EL (Heraeus).</p><p class="otherpara">The <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: crystallographic data" href="javascript:popupOBO('CMO:0000849','b802364f')">crystallographic data</a></span> for complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and <strong>2</strong> were collected on a Siemens <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: CCD diffractometer" href="javascript:popupOBO('CMO:0000949','b802364f')">CCD diffractometer</a></span> (SMART) using MoKα radiation (<span class="italic">λ</span> = 0.71073 Å) and omega-scan rotation (<a title="Select to navigate to table" href="#tab3">Table 3</a>). Data <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b802364f')">reduction</a></span> was performed with SAINT (area-<span class="CH"><a class="simple" title="Chemical Methods Ontology link for: detector" href="javascript:popupOBO('CMO:0002244','b802364f')">detector</a></span> integration software, version 6.01 (1999), Siemens Industrial Automation, Inc., Madison, WI) including the program SADABS (G. M. Sheldrick, SADABS, Program for Scaling and Correction of Area-<span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Detector" href="javascript:popupOBO('CMO:0002244','b802364f')">Detector</a></span> data, Göttingen, 1997) for empirical absorption correction.</p>
<div class="table_caption"><b>Table 3</b> <span id="tab3"><span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Crystallographic data" href="javascript:popupOBO('CMO:0000849','b802364f')">Crystallographic data</a></span> for single-crystal X-ray structures of compounds <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong></span>
		</div>
<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup><thead align="left"><tr align="left" valign="bottom"><th align="left" valign="bottom" class="border_black">Compound</th><th align="left" valign="bottom" class="border_black"><span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span></th><th align="left" valign="bottom" class="border_black"><strong>2</strong></th><th align="left" valign="bottom" class="border_black"><strong>3</strong></th></tr></thead><tbody align="left"><tr align="left"><td valign="top">Empirical formula</td><td valign="top">C<small><sub>42</sub></small>H<small><sub>42</sub></small>As<small><sub>2</sub></small>I<small><sub>2</sub></small>O<small><sub>2</sub></small>Pt</td><td valign="top">C<small><sub>42</sub></small>H<small><sub>42</sub></small>As<small><sub>2</sub></small>Cl<small><sub>10</sub></small>O<small><sub>2</sub></small>Pt</td><td valign="top">C<small><sub>36</sub></small>H<small><sub>30</sub></small>As<small><sub>2</sub></small>Cl<small><sub>2</sub></small>O<small><sub>2</sub></small>Pt</td></tr><tr align="left"><td valign="top">Formula weight</td><td valign="top">1177.49</td><td valign="top">1278.19</td><td valign="top">910.43</td></tr><tr align="left"><td valign="top">Temperature/K</td><td valign="top">213(2)</td><td valign="top">210(2)</td><td valign="top">213(2)</td></tr><tr align="left"><td valign="top"><span class="italic">λ</span>/pm</td><td valign="top">71.073</td><td valign="top">71.073</td><td valign="top">71.073</td></tr><tr align="left"><td valign="top">Crystal system</td><td valign="top">Monoclinic</td><td valign="top">Triclinic</td><td valign="top">Triclinic</td></tr><tr align="left"><td valign="top">Space <span class="CH"><a class="simple" title="ChEBI Ontology link for: group" href="javascript:popupOBO('CHEBI:24433','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">group</a></span></td><td valign="top"><span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">c</span></td><td valign="top"><span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td><td valign="top"><span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td></tr><tr align="left"><td valign="top"><span class="italic">a</span>/pm</td><td valign="top">930.16(16)</td><td valign="top">919.1(3)</td><td valign="top">1124.10(7)</td></tr><tr align="left"><td align="left" valign="top"><span class="italic">b</span>/pm</td><td valign="top">1821.6(3)</td><td valign="top">1169.0(3)</td><td valign="top">1594.26(12)</td></tr><tr align="left"><td align="left" valign="top"><span class="italic">c</span>/pm</td><td valign="top">1215.6(2)</td><td valign="top">1344.4(4)</td><td valign="top">1861.95(12)</td></tr><tr align="left"><td align="left" valign="top"><span class="italic">α</span>/°</td><td align="left" valign="top">90</td><td align="left" valign="top">114.160(4)</td><td align="left" valign="top">83.834(8)</td></tr><tr align="left"><td align="left" valign="top"><span class="italic">β</span>/°</td><td align="left" valign="top">106.261(3)</td><td align="left" valign="top">92.344(4)</td><td align="left" valign="top">84.727(8)</td></tr><tr align="left"><td align="left" valign="top"><span class="italic">γ</span>/°</td><td align="left" valign="top">90</td><td align="left" valign="top">105.966(4)</td><td align="left" valign="top">85.694(9)</td></tr><tr align="left"><td valign="top"><span class="italic">V</span>/nm<small><sup>3</sup></small></td><td valign="top">1.9773(6)</td><td valign="top">1.2476(6)</td><td valign="top">3.2963(4)</td></tr><tr align="left"><td valign="top"><span class="italic">Z</span></td><td valign="top">2</td><td valign="top">1</td><td valign="top">4</td></tr><tr align="left"><td valign="top"><span class="italic">D</span><small><sub>calcd</sub></small>/Mg m<small><sup>−3</sup></small></td><td valign="top">1.978</td><td valign="top">1.701</td><td valign="top">1.835</td></tr><tr align="left"><td valign="top">Absorption coefficient/mm<small><sup>−1</sup></small></td><td valign="top">6.805</td><td valign="top">4.697</td><td valign="top">6.443</td></tr><tr align="left"><td valign="top"><span class="italic">F</span>(000)</td><td valign="top">1120</td><td valign="top">624</td><td valign="top">1760</td></tr><tr align="left"><td valign="top">Crystal size/mm<small><sup>3</sup></small></td><td valign="top">0.20 × 0.10 × 0.10</td><td valign="top">0.30 × 0.20 × 0.20</td><td valign="top">0.20 × 0.10 × 0.10</td></tr><tr align="left"><td valign="top">Theta range for data collection (<span class="italic">Θ</span><small><sub>min</sub></small>–<span class="italic">Θ</span><small><sub>max</sub></small>)/°</td><td valign="top">2.07–29.40</td><td valign="top">2.02–29.24</td><td valign="top">2.21–28.00</td></tr><tr align="left"><td valign="top">Index ranges</td><td align="left" valign="top">−12 ≤<span class="italic">h</span>≤ 12, −21≤<span class="italic">k</span>≤ 23, −15 ≤<span class="italic">l</span>≤ 10</td><td align="left" valign="top">−12 ≤<span class="italic">h</span>≤ 12, −15 ≤<span class="italic">k</span>≤ 15, −17 ≤<span class="italic">l</span>≤ 18</td><td align="left" valign="top">−14 ≤<span class="italic">h</span>≤ 14, −18 ≤<span class="italic">k</span>≤ 21, −24 ≤<span class="italic">l</span>≤ 24</td></tr><tr align="left"><td valign="top">Reflections collected</td><td valign="top">12<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>871</td><td valign="top">13<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>002</td><td valign="top">22<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>550</td></tr><tr align="left"><td valign="top">Independent reflections</td><td valign="top">4868 [<span class="italic">R</span><small><sub>int</sub></small> = 0.0259]</td><td valign="top">6044 [<span class="italic">R</span><small><sub>int</sub></small> = 0.0230]</td><td valign="top">14<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>616 [<span class="italic">R</span><small><sub>int</sub></small> = 0.0410]</td></tr><tr align="left"><td valign="top">Completeness to <span class="italic">Θ</span><small><sub>max</sub></small></td><td valign="top">89.3%</td><td valign="top">89.2%</td><td valign="top">91.8%</td></tr><tr align="left"><td valign="top">Absorption correction</td><td valign="top">SADABS</td><td valign="top">SADABS</td><td valign="top">Numerical</td></tr><tr align="left"><td valign="top">Min./max. transmission</td><td valign="top">0.5494/0.3431</td><td valign="top">0.4535/0.3331</td><td valign="top">0.5651/0.3590</td></tr><tr align="left"><td valign="top">Refinement method</td><td valign="top">Full-matrix least-squares on <span class="italic">F</span><small><sup>2</sup></small></td><td valign="top">Full-matrix least-squares on <span class="italic">F</span><small><sup>2</sup></small></td><td valign="top">Full-matrix least-squares on <span class="italic">F</span><small><sup>2</sup></small></td></tr><tr align="left"><td valign="top">Restraints/parameters</td><td valign="top">4868/0/225</td><td valign="top">6044/0/260</td><td valign="top">14<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>616/0/775</td></tr><tr align="left"><td valign="top">Goodness-of-fit on <span class="italic">F</span><small><sup>2</sup></small></td><td valign="top">1.041</td><td valign="top">1.080</td><td valign="top">0.706</td></tr><tr align="left"><td valign="top">Final <span class="italic">R</span> indices [<span class="italic">I</span> &gt; 2<span class="italic">σ</span>(<span class="italic">I</span>)]</td><td valign="top"><span class="italic">R</span><small><sub>1</sub></small> = 0.0271, w<span class="italic">R</span><small><sub>2</sub></small> = 0.0645</td><td valign="top"><span class="italic">R</span><small><sub>1</sub></small> = 0.0332, w<span class="italic">R</span><small><sub>2</sub></small> = 0.0789</td><td valign="top"><span class="italic">R</span><small><sub>1</sub></small> = 0.0326, w<span class="italic">R</span><small><sub>2</sub></small> = 0.0551</td></tr><tr align="left"><td valign="top"><span class="italic">R</span> indices (all data)</td><td valign="top"><span class="italic">R</span><small><sub>1</sub></small> = 0.0395, w<span class="italic">R</span><small><sub>2</sub></small> = 0.0685</td><td valign="top"><span class="italic">R</span><small><sub>1</sub></small> = 0.0384, w<span class="italic">R</span><small><sub>2</sub></small> = 0.0809</td><td valign="top"><span class="italic">R</span><small><sub>1</sub></small> = 0.0763, w<span class="italic">R</span><small><sub>2</sub></small> = 0.0605</td></tr><tr align="left"><td valign="top">Largest diffraction peak and hole/e Å<small><sup>−3</sup></small></td><td valign="top">1.014 and −1.436</td><td valign="top">2.527 and −1.441</td><td valign="top">1.134 and −1.420</td></tr></tbody></table></div></div><hr class="hrule"/><br/><p class="otherpara">The <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: crystallographic data" href="javascript:popupOBO('CMO:0000849','b802364f')">crystallographic data</a></span> for complex <strong>3</strong> were collected on a Stoe-IPDS imaging plate <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: diffractometer" href="javascript:popupOBO('CMO:0000950','b802364f')">diffractometer</a></span> (phi-scan rotation and MoKα radiation (<span class="italic">λ</span> = 0.71073 Å) and the numerical absorption correction was performed with X-RED (Version 1.22 (2001), STOE Data <span class="CH"><a class="simple" title="Molecular Process Ontology link for: Reduction" href="javascript:popupOBO('MOP:0000569','b802364f')">Reduction</a></span> Program, STOE &amp; Cie GmbH, Darmstadt).</p></div><h3 id="sect1414"><span class="b_heading">Biological effects</span></h3><span>The <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> culture laboratory is equipped with Heto Holten and Laminair laminar hoods, with Hettich centrifuges, with humidified Heto Holten Cellhouse 154 and Uniequip CO<small><sub>2</sub></small> series incubators with CO<small><sub>2</sub></small> admission.</span><p class="otherpara">The fluorescence was tested using a Desktop CellScan apparatus [MedisEl Technologies, Lod, Israel]. The <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> carriers were provided by MedisEl. The cytotoxicity was determined using a Tecan Sunrise ELISA reader with Magellan operating system. For the single-<span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> gel assay the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: electrophoresis" href="javascript:popupOBO('CMO:0001019','b802364f')">electrophoresis</a></span> was performed in a Consort <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: electrophoresis" href="javascript:popupOBO('CMO:0001019','b802364f')">electrophoresis</a></span> chamber and the power source was Consort type EV231.</p><p class="otherpara">Image analysis of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: microscope" href="javascript:popupOBO('CMO:0000953','b802364f')">microscope</a></span> slides for the comet assay test were made using a Nikon <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: fluorescence microscope" href="javascript:popupOBO('CMO:0001095','b802364f')">fluorescence microscope</a></span> with <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: CCD" href="javascript:popupOBO('CMO:0000948','b802364f')">CCD</a></span> image capture and Lucia analyzer software. The image capture of the fluorescent-marked tumour <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b802364f')">cells</a></span> on the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b802364f')">cell</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b802364f')">carrier</a></span> surface was made using a Zeiss Observer D1 <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: optical microscope" href="javascript:popupOBO('CMO:0000947','b802364f')">optical microscope</a></span> with an Axiovision Release 4.3.6/2007 software. The statistical analysis was performed using the Graph Pad Prism 5 biostatistics program.</p><h3 id="sect1433"><span class="b_heading">Preparation of complexes <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and <strong>2</strong></span></h3><div><span id="sect1437"/><span class="c_heading_indent">Synthesis of ligand solution. </span><span><span class="TC"><a class="simple" title="Chemspider Compound link for:(2-isopropoxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.25935183.html">(2-isopropoxyphenyl)diphenylarsine</a></span> (0.04 g, 0.11 mmol) for complex <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-methoxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774018.html">(2-methoxyphenyl)diphenylarsine</a></span> (0.17 g, 0.5 mmol) for complex <strong>2</strong>, in <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:dichloromethane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6104.html">dichloromethane</a></span> was added to a dichloromethane solution of [PtI<small><sub>2</sub></small>(COD)] (COD = 1, 5-cyclooctadiene) (0.03 g, 0.053 mmol) for <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span> and [PtCl<small><sub>2</sub></small>(COD)] (0.09 g, 0.025 mmol) for <strong>2</strong>. The reaction mixture was stirred at room temperature until a precipitate was formed. The precipitate was filtered and then recrystallized from <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:dichloromethane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6104.html">dichloromethane</a></span> and <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:n-hexane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.7767.html"><span class="italic">n</span>-hexane</a></span> to give the corresponding complexes.</span></div><div><span id="sect1456"/><span class="c_heading_indent"><span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>. </span><span>Yield: 0.03 g, 46%, orange crystals, mp 271 °C. Found%: C 42.4, <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> 3.5, I 12.5. Calcd% C<small><sub>42</sub></small>H<small><sub>42</sub></small>As<small><sub>2</sub></small>I<small><sub>2</sub></small>O<small><sub>2</sub></small>Pt: C 42.8, <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> 3.6, I 12.7. <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: IR" href="javascript:popupOBO('CMO:0000630','b802364f')">IR</a></span>/cm<small><sup>−1</sup></small>: 333 (Pt–I), 279 (Pt–As). <span class="italic">δ</span><small><sub>H</sub></small>/ppm (300 MHz, CDCl<small><sub>3</sub></small>, Me<small><sub>4</sub></small>Si) 1.09 (d, 12H, C<span class="italic">H</span><small><sub>3</sub></small>, <span class="italic">J</span> = 6.04 Hz), 4.50 (m, 2H, C<span class="italic">H</span>), 6.79–6.88 (m, 4H), 7.35–7.39 (m, 16H), 7.84–7.87 (m, 8H).</span></div><div><span id="sect1483"/><span class="c_heading_indent"><strong>2</strong>. </span><span>Yield: 0.15 g, 64%, pale yellow crystals, mp 274 °C. Found: C 48.4, <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> 3.4, Cl 7.1. Calcd% C<small><sub>38</sub></small>H<small><sub>34</sub></small>As<small><sub>2</sub></small>Cl<small><sub>2</sub></small>O<small><sub>2</sub></small>Pt: C 48.6, <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> 3.6, Cl 7.5%, <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: IR" href="javascript:popupOBO('CMO:0000630','b802364f')">IR</a></span>/cm<small><sup>−1</sup></small>: 334 (Pt–Cl), 282 (Pt–As). <span class="italic">δ</span><small><sub>H</sub></small>/ppm (300 MHz, CDCl<small><sub>3</sub></small>, Me<small><sub>4</sub></small>Si): 3.84 (s, 6H, C<span class="italic">H</span><small><sub>3</sub></small>), 6.89–7.04 (m, 6H), 7.38–7.43 (m, 14H), 7.82 (m, 8H).</span></div><h3 id="sect1510"><span class="b_heading">Preparation of complex <strong>3</strong></span></h3><span>A solution of <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html">(2-hydroxyphenyl)diphenylarsine</a></span> (0.03 g, 0.09 mmol) in <span class="CH"><a class="simple" title="ChEBI Ontology link for: toluene" href="javascript:popupOBO('CHEBI:27024','b802364f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=27024')">toluene</a></span> was added as a layer over a solution of [PtCl<small><sub>2</sub></small>(COD)] (0.04 g, 0.044 mmol) in <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:dichloromethane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6104.html">dichloromethane</a></span>. As a result of the slow diffusion of the two solutions, complex <strong>3</strong> was obtained (0.02 g, 49%) as pale yellow crystals suitable for <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: X-ray analysis" href="javascript:popupOBO('CMO:0000156','b802364f')">X-ray analysis</a></span>, mp 239 °C. Found: C 47.9, <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> 3.2, Cl 8.0. Calcd% C<small><sub>36</sub></small>H<small><sub>30</sub></small>As<small><sub>2</sub></small>Cl<small><sub>2</sub></small>O<small><sub>2</sub></small>Pt: C 47.5, <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> 3.3, Cl 7.8%, <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: IR" href="javascript:popupOBO('CMO:0000630','b802364f')">IR</a></span>/cm<small><sup>−1</sup></small>: 317, 325 (Pt–Cl), 277 (Pt–As). <span class="italic">δ</span><small><sub>H</sub></small>/ppm (300 MHz, CDCl<small><sub>3</sub></small>, Me<small><sub>4</sub></small>Si): 6.34 (dd, 2H, <span class="italic">J</span><small><sub>1,2</sub></small> = 6.98, <span class="italic">J</span><small><sub>1,3</sub></small> = 1.13), 6.87–7.21 (m, 6H), 7.39–7.45 (m, 12H), 7.74 (m, 8H). An OH resonance was not observed.</span>
		<h2 id="sect1548"><span class="a_heading">Acknowledgements</span></h2><span>We gratefully acknowledge financial support from the EU Marie Curie Training Site HPMT-CT-2000-00110 and the International Postgraduate Programm (IPP ‘Research at the Frontiers of Chemistry’) at the University of Leipzig (N. M.). We thank Umicore for the generous donation of H<small><sub>2</sub></small>[PtCl<small><sub>6</sub></small>]·6H<small><sub>2</sub></small>O. The authors from ‘I. Chiricuta’ Cancer Institute Cluj Napoca want to express their gratitude to Dr Yael S. Schiffenbauer for all her help and support during this study.</span><span id="sect1553"><h2 id="sect1547"><span class="a_heading">References</span></h2></span><ol type="1"><li><span id="cit1">V. T. DeVita, S. Hellman, S. Rosenberg, <span class="italic">Cancer: Principles and Practice of Oncology</span>, Lippincott, Williams and Wilkins, Philadelphia, 7th edn,  2004, ch. 15, sect. 5 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Cancer:%20Principles%20and%20Practice%20of%20Oncology%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/1_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit2">M. Fukuda, Y. Ohe, F. Kzawa, M. Oka, K. Hara and N. Saijo, <span class="italic">Anticancer Res.</span>, 1995, <span class="bold">15</span>, 393–398 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXmslWjurc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/2_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit3">C. Barbara, P. Orlandi, G. Bocci, A. Fioravanti, A. Di Paolo, G. Natale, M. Del Tacca and R. Danesi, <span class="italic">Eur. J. Pharmacol.</span>, 2006, <span class="bold">549</span>, 27–34 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ejphar.2006.08.015" title="DOI Link to resource 10.1016/j.ejphar.2006.08.015">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XhtVWqu7%252FO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/3_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit4">M. Carland, K. J. Tan, J. M. White, J. Stephenson, V. Murray, W. A. Denny and W. D. McFayden, <span class="italic">J. Inorg. Biochem.</span>, 2005, <span class="bold">99</span>, 1738–1743 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jinorgbio.2005.06.003" title="DOI Link to resource 10.1016/j.jinorgbio.2005.06.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXmvVCqu7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/4_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit5">K. Akamatsu, H. Saito, T. Tsunenari, T. Matsumoto, K. Morikawa, M. Koizumi, H. Mitsui and K. Koizumi, <span class="italic">Anticancer Res.</span>, 1993, <span class="bold">13</span>, 2261–2265 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2cXjtFGgt7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/5_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit6">M. Gay, A. M. Montaña, V. Moreno, M. J. Prieto, J. M. Pérez and C. Alonso, <span class="italic">Bioorg. Med. Chem.</span>, 2006, <span class="bold">14</span>, 1565–72 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bmc.2005.10.016" title="DOI Link to resource 10.1016/j.bmc.2005.10.016">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XmsVGhsQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/6_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>; M. Gay, A. M. Montaña, V. Moreno, M. J. Prieto, R. Llorens and L. Ferrer, <span class="italic">J. Inorg. Biochem.</span>, 2005, <span class="bold">99</span>, 2387–2394 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jinorgbio.2005.09.006" title="DOI Link to resource 10.1016/j.jinorgbio.2005.09.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXht1Olsr7M" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/6_2" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit7">M. S. Robillard, M. Bacac, H. Van Den Elst, A. Flamigni, G. A. Van Der Marel, J. H. van Boom and J. Reedijk, <span class="italic">J. Comb. Chem.</span>, 2003, <span class="bold">5</span>, 821–825 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/cc030011z" title="DOI Link to resource 10.1021/cc030011z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXnvV2qu7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/7_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit8">M. Cemazar, Z. Pipan, S. Grabner, N. Bukovec and G. Sersa, <span class="italic">Anticancer Res.</span>, 2006, <span class="bold">26</span>, 1997–2002 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XntV2nur8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/8_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit9">M. Harel, B. Gilburd, Y. S. Schiffenbauer and Y. Schoenfeld, <span class="italic">Clin. Dev. Immunol.</span>, 2005, <span class="bold">12</span>, 187–195 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Clin.%20Dev.%20Immunol.%5BJour%5D%20AND%2012%5Bvolume%5D%20AND%20187%5Bpage%5D%20and%202005%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/9_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit10">Y. S. Schiffenbauer, E. Trubniykov, B. T. Zacharia, S. Gerbat, Z. Rehavi, G. Berke and S. Chaitchik, <span class="italic">Anticancer Res.</span>, 2002, <span class="bold">22</span>, 2663–2670 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXoslyrsQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/10_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit11">S. Chaitchik, O. Soritau, E. Fischer, D. Pelau, L. Lazar and Y. S. Schiffenbauer, <span class="italic">J. Clin. Oncol.</span>, 2005, <span class="bold">23</span>, 2125 <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/11_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit12">N. Zurgil, Y. Shafran, D. Fixler and M. Deutsch, <span class="italic">Biochem. Biophys. Res. Commun.</span>, 2002, <span class="bold">290</span>, 1573–1582 <a target="_blank" class="DOILink" href="https://doi.org/10.1006/bbrc.2002.6382" title="DOI Link to resource 10.1006/bbrc.2002.6382">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XotlKrtQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/12_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit13">T. Mosmann, <span class="italic">J. Immunol. Methods</span>, 1983, <span class="bold">65</span>, 55–63 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0022-1759(83)90303-4" title="DOI Link to resource 10.1016/0022-1759(83)90303-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ABiuD1cbgsFE%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/13_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit14">R. R. Tice, E. Agurell, D. Anderson, B. Burlinson, A. Hartmann, H. Kobayashi, Y. Miyamae, E. Rojas, J. C. Ryu and Y. F. Sasaki, <span class="italic">Environ. Mol. Mutagen.</span>, 2000, <span class="bold">35</span>, 206–221 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1098-2280(2000)35:3&lt;206::AID-EM8&gt;3.0.CO;2-J" title="DOI Link to resource 10.1002/(SICI)1098-2280(2000)35:3&lt;206::AID-EM8&gt;3.0.CO;2-J">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXis1Wrurg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/14_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit15">A. R. Collins, <span class="italic">Mol. Biotechnol.</span>, 2004, <span class="bold">26</span>, 249–261 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mol.%20Biotechnol.%5BJour%5D%20AND%2026%5Bvolume%5D%20AND%20249%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/15_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit16">N. Moldovan, P. Lönnecke, I. Silaghi-Dumitrescu, L. Silaghi-Dumitrescu and E. Hey-Hawkins, <span class="italic">Inorg. Chem.</span>, 2008, <span class="bold">47</span>, 1525–31 <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/16_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit17">P. P. Phadnis, V. K. Jain, A. Klein, M. Weber and W. Kaim, <span class="italic">Inorg. Chim. Acta</span>, 2003, <span class="bold">346</span>, 119–128 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0020-1693(02)01375-0" title="DOI Link to resource 10.1016/S0020-1693(02)01375-0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXhvFSrsrk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/17_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>; P. P. Phadnis, V. K. Jain, T. Schurr, A. Klein, F. Lissner, T. Schleid and W. Kaim, <span class="italic">Inorg. Chim. Acta</span>, 2005, <span class="bold">358</span>, 2609–2617 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ica.2005.03.026" title="DOI Link to resource 10.1016/j.ica.2005.03.026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXjvFWktro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/17_2" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit18">F. Y. Kwong, C. W. Lai, M. Yu, D.-M. Tan, F. L. Lam, A. S. C. Chan and K. S. Chan, <span class="italic">Organometallics</span>, 2005, <span class="bold">24</span>, 4170–4178 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/om050237+" title="DOI Link to resource 10.1021/om050237+">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXmt1WrsLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/18_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit19">S. C. Frost and M. S. Schwalbe, <span class="italic">Biochem. J.</span>, 1990, <span class="bold">269</span>, 598–595 <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/19_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit20">R. I. Geran, N. H. Greenberg, M. M. Macdonald, A. M. Schumacher and B. J. Abbott, <span class="italic">Cancer Chemother. Rep.</span>, 1972, <span class="bold">3</span>, 1–85 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Cancer%20Chemother.%20Rep.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%201%5Bpage%5D%20and%201972%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/20_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit21">H. Takahashi, K. Suzuki and H. Namiki, <span class="italic">Cell. Struct. Funct.</span>, 2003, <span class="bold">28</span>, 123–130 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Cell.%20Struct.%20Funct.%5BJour%5D%20AND%2028%5Bvolume%5D%20AND%20123%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/21_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit22">N. Begun, <span class="italic">Am. J. Physiol.</span>, 1994, <span class="bold">267</span>, E14–23 <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/22_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit23">E. J. Verspohl, <span class="italic">Endocrin. J.</span>, 2006, <span class="bold">53</span>, 21–26 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Endocrin.%20J.%5BJour%5D%20AND%2053%5Bvolume%5D%20AND%2021%5Bpage%5D%20and%202006%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/23_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit24">A C. Schmidt, M. Neustadt and M. Otto, <span class="italic">J. Mass Spectrom.</span>, 2007, <span class="bold">42</span>, 771–80 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jms.1212" title="DOI Link to resource 10.1002/jms.1212">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/24_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit25">Z. Wang, H. Zhang, X. F. Li and X. C. Le, <span class="italic">Rapid Commun. Mass Spectrom.</span>, 2007, <span class="bold">21</span>, 3658–3666 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/rcm.3263" title="DOI Link to resource 10.1002/rcm.3263">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKktb7J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/25_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit26">J. C. Mottram and G. H. Coombs, <span class="italic">Exp. Parasitol.</span>, 1985, <span class="bold">59</span>(2), 151–60 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0014-4894(85)90067-0" title="DOI Link to resource 10.1016/0014-4894(85)90067-0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2MXitFClurk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/26_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit27">A. H. Fairlamb, G. B. Henderson and A. Cerami, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 1989, <span class="bold">86</span>, 2607–11 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL1MXitV2htro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/27_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit28">B. J. Berger and A. H. Fairlamb, <span class="italic">Antimicrob. Agents Chemother.</span>, 1994, <span class="bold">38</span>, 1298–302 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2cXksValsrc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/28_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit29">H. D. Empsall, B. L. Shaw and B. L. Turtle, <span class="italic">J. Chem. Soc., Dalton Trans.</span>, 1976, 1500 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=DT9760001500 none&amp;newsite=1" title="Link to RSC resource DOI:10.1039/DT9760001500 none">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/29_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>; C. E. Jones, B. L. Shaw and B. L. Turtle, <span class="italic">J. Chem. Soc., Dalton Trans.</span>, 1974, 992 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=DT9740000992&amp;newsite=1" title="Link to RSC resource DOI:10.1039/DT9740000992">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/29_2" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>; H. D. Empsall, P. N. Heys and B. L. Shaw, <span class="italic">J. Chem. Soc., Dalton Trans.</span>, 1978, 257 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=DT9780000257&amp;newsite=1" title="Link to RSC resource DOI:10.1039/DT9780000257">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/29_3" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit30">M. Deutsch, I. Ron, A. Weinreb, R. Tirosh and S. Chaitchik, <span class="italic">Cytometry</span>, 1996, <span class="bold">23</span>, 159–165 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1097-0320(19960201)23:2&lt;159::AID-CYTO9&gt;3.0.CO;2-F" title="DOI Link to resource 10.1002/(SICI)1097-0320(19960201)23:2&lt;159::AID-CYTO9&gt;3.0.CO;2-F">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ABymA2M7mtFM%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/30_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit31">M. Deutsch and A. Weinreb, <span class="italic">Cytometry</span>, 1994, <span class="bold">16</span>, 214–226 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cyto.990160305" title="DOI Link to resource 10.1002/cyto.990160305">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3AByqD3cbhvFE%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/31_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit32">F. S. Mackay, J. A. Woods, H. Mosely, J. Ferguson, A. Pawson, S. Parsons and P. J. Sadler, <span class="italic">Chem.–Eur. J.</span>, 2006, <span class="bold">12</span>, 3155–3161 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.200501601" title="DOI Link to resource 10.1002/chem.200501601">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xjslart7Y%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/32_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit33">J. Blasiak, J. Kovalik, E. Malecka-Panas, J. Drzewoski and M. Wojewodska, <span class="italic">Teratog. Carcinog. Mutagen.</span>, 2000, <span class="bold">20</span>, 119–131 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1520-6866(2000)20:3&lt;119::AID-TCM3&gt;3.0.CO;2-Z" title="DOI Link to resource 10.1002/(SICI)1520-6866(2000)20:3&lt;119::AID-TCM3&gt;3.0.CO;2-Z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXjsF2gsrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/33_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit34">J. Kruszewski, E. Bouzik, T. Oldak, K. Samochocka, L. Fuks, W. Lewandowski, I. Fokt and W. Priebe, <span class="italic">Teratog. Carcinog. Mutagen.</span>, 2003, <span class="bold">1</span>(Suppl.), 1–11 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/tcm.10046" title="DOI Link to resource 10.1002/tcm.10046">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/34_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit35">O. Merk and G. Speit, <span class="italic">Environ. Mol. Mutagen.</span>, 1999, <span class="bold">33</span>, 167–172 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1098-2280(1999)33:2&lt;167::AID-EM9&gt;3.0.CO;2-D" title="DOI Link to resource 10.1002/(SICI)1098-2280(1999)33:2&lt;167::AID-EM9&gt;3.0.CO;2-D">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1MXisVGku74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/35_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit36">I. Brie, M. Perde, P. Virag, E. Fischer, O. Soritau, I. D. Postescu, V. Nagy, O. Coza, I. Sicoe, N. Todor, V. Cernea and N. Ghilezan, <span class="italic">Radiother. Oncol.</span>, 2005, <span class="bold">9</span>, 96–105 <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/36_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit37">H. Kobayashi, C. Sugiyama, Y. Morikawa, M. Hayashi and T. Sofuni, <span class="italic">MMS Commun.</span>, 1995, <span class="bold">3</span>, 103–115 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=MMS%20Commun.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%20103%5Bpage%5D%20and%201995%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/37_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li><li><span id="cit38">D. Drew and J. R. Doyle, <span class="italic">Inorg. Synth.</span>, 1990, <span class="bold">28</span>, 346–349 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3MXhsVeltb8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50001067/b802364f/38_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50001067"/></a>.</span></li></ol><hr/><table><tr><td><h3>Footnote</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">Electronic supplementary information (ESI) available: POV-Ray representations, atomic coordinates and displacement parameters, anisotropic displacement parameters, bond lengths and angles for <span class="TC"><a class="simple" title="Chemspider Compound link for:(2-hydroxyphenyl)diphenylarsine" target="_blank" href="http://www.chemspider.com/Chemical-Structure.24774019.html"><strong>1</strong></a></span>, <strong>2</strong> and <strong>3</strong>. CCDC reference numbers 693566–693568. For ESI and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: crystallographic data" href="javascript:popupOBO('CMO:0000849','b802364f')">crystallographic data</a></span> in CIF or other electronic format see DOI: <a href="http://xlink.rsc.org/?DOI=b802364f" target="_blank" title="Select to resolve DOI: 10.1039/b802364f">10.1039/b802364f</a></span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2008</b></td></tr></table></div></div></div></body><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>