<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Metabolic engineering of cyanobacteria for ethanol production   - Energy &amp; Environmental Science (RSC Publishing) DOI:10.1039/B811937F</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2009/ee/b811937f"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":52,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Jason Dexter"/><meta name="DC.Creator" content="Pengcheng Fu"/><meta name="DC.title" content="Metabolic engineering of cyanobacteria for ethanol production  "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2009/07/28"/><meta name="DC.Identifier" scheme="doi" content="10.1039/B811937F"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Metabolic engineering of cyanobacteria for ethanol production  "/><meta name="citation_author" content="Jason Dexter"/><meta name="citation_author" content="Pengcheng Fu"/><meta name="citation_online_date" content="2009/06/10"/><meta name="citation_date" content="2009"/><meta name="citation_journal_title" content="Energy &amp; Environmental Science"/><meta name="citation_volume" content="2"/><meta name="citation_issue" content="8"/><meta name="citation_firstpage" content="857"/><meta name="citation_lastpage" content="864"/><meta name="citation_doi" content="10.1039/B811937F"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2009/ee/b811937f"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2009/ee/b811937f"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2009/ee/b811937f"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/prospect.js?ver=6_2_1"> </script><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2009/ee/b811937f">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:b905747a)" href="/en/content/articlehtml/2009/ee/b905747a" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:b821622n)" href="/en/content/articlehtml/2009/ee/b821622n" target="_BLANK">Next Article</a></span></div><div class="right_head"> </div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/B811937F">10.1039/B811937F</a>
(Paper)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1754-5706/2008">Energy Environ. Sci.</a></span>, 2009, <strong>2</strong>, 857-864<div id="chemnav"><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of compound names" onclick="toggleColour('TC');"><span alt="Show Compounds button" id="btnTC">Show Compounds</span></a><input id="inpTC" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of chem-ontology terms" onclick="toggleColour('CH');"><span alt="Show Chemical Terms button" id="btnCH">Show Chemical Terms</span></a><input id="inpCH" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of bio-ontology terms" onclick="toggleColour('AN');"><span alt="Show Biomedical Terms button" id="btnAN">Show Biomedical Terms</span></a><input id="inpAN" type="hidden" value="off"/></div></div><h1 id="sect172"><span class="title_heading">Metabolic engineering of cyanobacteria for <span class="TC"><a class="simple" title="Chemspider Compound link for:ethanol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.682.html">ethanol</a></span> production<a title="Select to navigate to footnote" href="#fn1">†</a></span></h1><p class="header_text">
      <span class="bold">
        
          
            Jason 
            Dexter
          
        
      </span><sup><span class="sup_ref italic">a</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Pengcheng 
            Fu
          
        
      </span>*<sup><span class="sup_ref italic">b</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Department of Molecular Biosciences &amp; Bioengineering, University of Hawaii, 1955 East-West Road, Honolulu, Hawaii 96822-2321, USA</span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Faculty of Chemical Sciences and Engineering, China University of Petroleum, 18 Fuxue Road, Changping District, Beijing, 102249, P. R. China. E-mail: <a href="mailto:pengcheng@cup.edu.cn">pengcheng@cup.edu.cn</a>;  Fax: + 86-10-6074-3667;   Tel: +86-10-8973-1283</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      11th July 2008
    </span><span class="bold italic">, Accepted 27th May 2009</span></div><p class="bold italic">First published on 
      
      
      
      
      
      10th June 2009
    </p><hr/><div class="abstract"><h2>Abstract</h2><p>Development of renewable energy is rapidly being embraced by our society and industry to achieve the nation's economic <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> goals and to help address the world's energy and global warming crises. Currently most of the bioethanol production is from the <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> of agricultural crops and residues. There is much debate concerning the cost effectiveness and energy efficiency of such biomass based <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production processes. Here, we report the creation of a <span class="italic">Synechocystis</span> sp. PCC 6803 strain that can photoautotrophically convert CO<small><sub>2</sub></small> to bioethanol. Transformation was performed using a double homologous recombination system to integrate the <span class="AN"><a class="simple" title="Gene Ontology link for: pyruvate decarboxylase" href="javascript:popupOBO('GO:0004737','b811937f')">pyruvate decarboxylase</a></span> (<span class="italic">pdc</span>) and <span class="AN"><a class="simple" title="Gene Ontology link for: alcohol dehydrogenase" href="javascript:popupOBO('GO:0004022','b811937f')">alcohol dehydrogenase</a></span> II (<span class="italic">adh</span>) <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span> from obligately <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing <span class="italic">Zymomonas mobilis</span> into the <span class="italic">Synechocystis</span> PCC 6803 <span class="AN"><a class="simple" title="Sequence Ontology link for: chromosome" href="javascript:popupOBO('SO:0000340','b811937f')">chromosome</a></span> under the control of the strong, light driven <span class="italic">psbAII</span> promoter. PCR based assay and <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production assay were used to screen for stable transformants. A computerized photobioreactor system was established for the experimental design and data acquisition for the analysis of the cyanobacterial <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> cultures and <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. The system described here shows an average yield of 5.2 mmol OD<small><sub>730</sub></small> unit<small><sup>−1</sup></small> litre<small><sup>−1</sup></small> day<small><sup>−1</sup></small> with no required <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotic" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotic</a></span>/selective agent.</p></div><hr/>
    
      <br/><table class="box_border"><tr><td>
        
          <h2 id="sect251"><span class="a_heading">Broader context</span></h2>
          <span>The development of renewable energy is rapidly being embraced by our society and industry to meet the energy <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> and emission <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b811937f')">reduction</a></span> goals. <span class="TC"><a class="simple" title="Chemspider Compound link for:Ethanol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.682.html">Ethanol</a></span> has recently surged to the forefront of biofuels technology. Most of the current <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production is from the biomass <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> of agricultural crops and residues. It remains a challenge for <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> distilleries to be environmentally friendly and economically competitive using biomass as the raw materials. The technology reported in this paper involves genetic engineering of cyanobacterial strains for direct conversion of sunlight and the greenhouse gas CO<small><sub>2</sub></small> into <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>. It will integrate <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b811937f')">photosynthesis</a></span>, emission <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b811937f')">reduction</a></span> and biofuels production in one single process. Another important advantage for our approach is that by directly using CO<small><sub>2</sub></small> and solar energy for cyanobacteria <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> and <span class="AN"><a class="simple" title="Gene Ontology link for: ethanol synthesis" href="javascript:popupOBO('GO:0006115','b811937f')">ethanol synthesis</a></span>, agricultural space can be preserved for <span class="CH"><a class="simple" title="ChEBI Ontology link for: food" href="javascript:popupOBO('CHEBI:33290','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')">food</a></span> production. This novel <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production approach thus has great potential for industrial applications for many interrelated sociological/political/environmental/economical reasons.</span>
        
      </td></tr></table><br/>
    
    
      <h2 id="sect271"><span class="a_heading">Introduction</span></h2>
      <span>Biologically produced <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> as a <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> is an attractive renewable energy concept. Bioethanol can be readily integrated with existing <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> processing/utilization platforms while offering the benefit of control over both production and utilization reactions (<span class="italic">i.e.</span> control over the entire carbon cycling of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>, as opposed to <span class="CH"><a class="simple" title="ChEBI Ontology link for: fossil fuels" href="javascript:popupOBO('CHEBI:35230','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35230')">fossil fuels</a></span> which we cannot effectively synthesize). <span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span>–gasoline hybrid <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> can be used by most internal <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: combustion" href="javascript:popupOBO('CMO:0001473','b811937f')">combustion</a></span> engines at levels up to 15% <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> and with modification they could operate on a mixture of 85% <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>–15% gasoline.<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> Currently most bioethanol production is from the <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> of agricultural crops and residues. Statistics indicate that the world <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production is 60% from sugar crops, 33% from other crops and 7% from chemical synthesis (http://attra.ncat.org/guide/a_m/biomass.html). Concerns over agricultural land/grain use for <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span>, along with the high energy input associated with traditional corn/<span class="CH"><a class="simple" title="ChEBI Ontology link for: starch" href="javascript:popupOBO('CHEBI:28017','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28017')">starch</a></span> and other biomass based <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> systems have been significant hurdles for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> community. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span> production from biomass as a primary feedstock is somewhat controversial since some experts argue that such bioprocesses are neither cost-effective nor energy-efficient.<a title="Select to navigate to reference" href="#cit2"><sup><span class="sup_ref">2,3</span></sup></a></span>
      <p class="otherpara">Alternative bioethanol production methods need to be developed so that the costs associated with the land, labor, and time of traditionally fermented crops can be circumvented. Genetic engineering of photosynthetic prokaryotes is one possibility for efficient bioethanol production which would combine the conversion of solar energy, carbon sequestration and <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production in a single organism. For example, Deng and Coleman<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a> demonstrated that through genetic engineering, <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> could be produced from the uptaken CO<small><sub>2</sub></small> by a photosynthetic organism <span class="italic">via</span> photoautotrophic <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b811937f')">metabolism</a></span>. In that experiment, cyanobacterium species <span class="italic">Synechococcus</span> sp. strain PCC 7942 was transformed with pCB4-based shuttle vectors bearing the <span class="italic">Z. mobilis</span><span class="italic">pdc</span>/<span class="italic">adh</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: gene cassette" href="javascript:popupOBO('SO:0005853','b811937f')">gene cassette</a></span>, which was first described by Ingram <span class="italic">et al.</span><a title="Select to navigate to reference" href="#cit5"><sup><span class="sup_ref">5,6</span></sup></a> The <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span> were expressed under the control of the <span class="italic">rbcLS</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: operon" href="javascript:popupOBO('SO:0000178','b811937f')">operon</a></span> promoter, both alone and in combination with the <span class="italic">E. coli lac</span> promoter. Their results showed <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yields on an order of magnitude of 54 nmol OD<small><sub>730</sub></small> unit<small><sup>−1</sup></small> litre<small><sup>−1</sup></small> day<small><sup>−1</sup></small>. Also in the same study, an <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration of approximately 5 mM was achieved after 4 weeks of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span>. More recently, the Coleman <span class="CH"><a class="simple" title="ChEBI Ontology link for: group" href="javascript:popupOBO('CHEBI:24433','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">group</a></span> filed a patent which indicates a yield of 1.7 µmol of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> per mg of <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> per hour using the CI-PL temperature inducible promoter. (Woods <span class="italic">et al.</span>, US patent 6699696, 2004).<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a></p>
      <p class="otherpara">
        <span class="italic">Synechocystis</span> sp. strain PCC 6803 (<span class="italic">Synechocystis</span>), a freshwater, non-filamentous, non-nitrogen fixing cyanobacteria capable of heterotrophic <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span>, was the first photosynthetic autotrophic organism that was completely sequenced and annotated.<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> In addition, the natural transformability of this strain has allowed for the establishment of techniques for precise <span class="AN"><a class="simple" title="Sequence Ontology link for: genome" href="javascript:popupOBO('SO:0001026','b811937f')">genome</a></span> manipulation.<a title="Select to navigate to reference" href="#cit9"><sup><span class="sup_ref">9–11</span></sup></a><span class="italic">Synechocystis</span> has thus served as a model organism for understanding fundamental physiological and <span class="AN"><a class="simple" title="Gene Ontology link for: metabolic processes" href="javascript:popupOBO('GO:0008152','b811937f')">metabolic processes</a></span> involved in O<small><sub>2</sub></small> evolving <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b811937f')">photosynthesis</a></span>. The standard method for <span class="italic">Synechocystis</span> mutation induction involves transformation with a <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid vector" href="javascript:popupOBO('SO:0000755','b811937f')">plasmid vector</a></span> that is targeted to the <span class="AN"><a class="simple" title="Sequence Ontology link for: genome" href="javascript:popupOBO('SO:0001026','b811937f')">genome</a></span> <span class="italic">via</span> double homologous recombination and selection under appropriate conditions.<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> For <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span> disruption (knockout), the vector may contain an <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotic" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotic</a></span> resistance cassette that has been inserted into the cloned <span class="AN"><a class="simple" title="Sequence Ontology link for: open reading frame" href="javascript:popupOBO('SO:0000236','b811937f')">open reading frame</a></span> (<span class="AN"><a class="simple" title="Sequence Ontology link for: ORF" href="javascript:popupOBO('SO:0000236','b811937f')">ORF</a></span>). Simultaneous genomic <span class="AN"><a class="simple" title="Sequence Ontology link for: insertion" href="javascript:popupOBO('SO:0000667','b811937f')">insertion</a></span> of both an <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotic" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotic</a></span> resistance <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span> (positive selection) and a negative selection component, such as the <span class="italic">sacB</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span> that is lethal in the presence of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:sucrose" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5768.html">sucrose</a></span>, will then allow for introduction of any desired genetic material into the <span class="AN"><a class="simple" title="Sequence Ontology link for: genome" href="javascript:popupOBO('SO:0001026','b811937f')">genome</a></span> replacing the above positive/negative selection cassette with the <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span> of interest through negative selection.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a></p>
      <p class="otherpara">The primary objective of this work is to create a strain of the freshwater cyanobacterium <span class="italic">Synechocystis</span> sp. PCC 6803 that will form the foundation for an efficient process of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. Here, we report the creation of a <span class="italic">Synechocystis</span> sp. PCC 6803 strain that can photoautotrophically convert CO<small><sub>2</sub></small> to bioethanol. Transformation was performed using a double homologous recombination system to integrate the <span class="AN"><a class="simple" title="Gene Ontology link for: pyruvate decarboxylase" href="javascript:popupOBO('GO:0004737','b811937f')">pyruvate decarboxylase</a></span> (<span class="italic">pdc</span>) and <span class="AN"><a class="simple" title="Gene Ontology link for: alcohol dehydrogenase" href="javascript:popupOBO('GO:0004022','b811937f')">alcohol dehydrogenase</a></span> II (<span class="italic">adh</span>) <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span> from obligately <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing <span class="italic">Zymomonas mobilis</span> into the <span class="italic">Synechocystis</span> 6803 <span class="AN"><a class="simple" title="Sequence Ontology link for: chromosome" href="javascript:popupOBO('SO:0000340','b811937f')">chromosome</a></span> under the control of the strong, light driven <span class="italic">psbAII</span> promoter. A PCR based assay and <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production assay were used to screen for stable transformants. A computerized photobioreactor system was established for the experimental design and data acquisition for the analysis of the cyanobacterial <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> cultures and <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. The system described here shows an average yield of 5.2 mmol OD<small><sub>730</sub></small> unit <small><sup>−1</sup></small> litre<small><sup>−1</sup></small> day <small><sup>−1</sup></small> with no required <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotic" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotic</a></span>/selective agent.</p>
    
    
      <h2 id="sect394"><span class="a_heading">Results and discussions</span></h2>
      
        <h3 id="sect396"><span class="b_heading">Comparison with the results of previous research</span></h3>
        <span>In the patent filed by Woods <span class="italic">et al.</span> (US patent 6306639, 2001),<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> the most critical alteration to their <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing system described in 1999 was the substitution of the <span class="italic">rbcLS</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: operon" href="javascript:popupOBO('SO:0000178','b811937f')">operon</a></span> endogenous promoter (from Synechococcus PCC 7942) with the temperature inducible CI-PL promoter, EMBL Accessive No. L05669, SEQ. <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: ID" href="javascript:popupOBO('CMO:0000783','b811937f')">ID</a></span>. No.7. Obviously, the first point of difference lies in the different organisms chosen to express the <span class="italic">pdc</span>/<span class="italic">adh</span> cassette. Deng and Coleman chose to use Synechococcus PCC 7942, whose <span class="AN"><a class="simple" title="Sequence Ontology link for: genome" href="javascript:popupOBO('SO:0001026','b811937f')">genome</a></span> was completely sequenced in 2004, as their model organism. We have chosen <span class="italic">Synechocystis</span> PCC 6803, which was the first cyanobacteria to be sequenced in 1996.<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> Use of <span class="italic">Synechocystis</span> thus has the advantage of having a larger body of methodological work associated with the genetics and genetic engineering strategies.</span>
        <p class="otherpara">In their work Deng and Coleman have used <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmids" href="javascript:popupOBO('SO:0000155','b811937f')">plasmids</a></span> pCB4-LRpa, pCB4-LR(TF),<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a> and pCB4-CPpa,<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> based on pCB4 as the replicating <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid vector" href="javascript:popupOBO('SO:0000755','b811937f')">plasmid vector</a></span> for expression of the <span class="italic">pdc</span>/<span class="italic">adh</span> cassette, and <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> selection <span class="italic">via</span> <span class="TC"><a class="simple" title="Chemspider Compound link for:ampicillin" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6013.html">ampicillin</a></span>. In our work, we used the non-replicating vector for direct integration of <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span> into the <span class="AN"><a class="simple" title="Sequence Ontology link for: genome" href="javascript:popupOBO('SO:0001026','b811937f')">genome</a></span> at the psbA2 genetic locus, under control of the strong, light-driven psbA2 promoter. The original pPSBAIIKS was based on pSL1180 and was designed for double homologous recombination with the <span class="italic">Synechocystis</span> 6803 psbA2 locus, thus the strains did not need to be maintained on <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotics" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotics</a></span> or other selection factors after isolation. Given the number of variables involved, it is difficult to attribute the difference in <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production observed between the Deng and Coleman effort and this work to any single factor. The combined results from both studies thus suggest that there would be much benefit in screening multiple strains of cyanobacteria using various promoter systems to determine the best combination for <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. Yet, one must also consider the additional key advantages of not requiring <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotics" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotics</a></span> for <span class="AN"><a class="simple" title="Gene Ontology link for: plasmid maintenance" href="javascript:popupOBO('GO:0006276','b811937f')">plasmid maintenance</a></span>, despite the increase in effort for genetic transformation and screening. This is because during extended photobioreactor experiments, one does not need to be concerned with the <span class="CH"><a class="simple" title="ChEBI Ontology link for: antibiotic" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">antibiotic</a></span> concentration within the culture media and its sufficiency for maintaining the <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span>.</p>
      
      
        <h3 id="sect437"><span class="b_heading">Creation of pMota for the expression of <span class="italic">pdc</span>/<span class="italic">adh</span> in <span class="italic">Synechocystis</span></span></h3>
        <span>A pararosaniline <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plate assay was used to confirm the successful creation of <span class="italic">E. coli</span> transformant pMota. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plates, as described by Ingram <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> contain the Schiff reagent, allowing for the <span class="AN"><a class="simple" title="Gene Ontology link for: detection of" href="javascript:popupOBO('GO:0014865','b811937f')">detection of</a></span> the <span class="AN"><a class="simple" title="Gene Ontology link for: alcohol dehydrogenase" href="javascript:popupOBO('GO:0004022','b811937f')">alcohol dehydrogenase</a></span> (<span class="italic">adh</span>). Therefore, only the <span class="italic">E. coli</span> that contained a <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> with the <span class="italic">adh</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span> (pLOI295 and pMota) would generate the intensely red <span class="CH"><a class="simple" title="ChEBI Ontology link for: Schiff base" href="javascript:popupOBO('CHEBI:50229','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=50229')">Schiff base</a></span> on the <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plates. <a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a> shows the <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> <span class="AN"><a class="simple" title="Sequence Ontology link for: constructs" href="javascript:popupOBO('SO:0000804','b811937f')">constructs</a></span> (2B) tested positive (in red) with the control (2A, 2C and 2D). Here 2A is a negative control plate with non-ethanogenic pPSBAIIKS. 2B is for the plates with isolated colonies expressing the <span class="italic">pdc</span>/<span class="italic">adh</span> cassette within pMota. 2C is the positive control plate with pLOI295. Plates in 2A, 2B, and 2C were incubated for 24 hours at 37 °C as given in the standard protocol for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plate assay. And 2D is for a pMota plate which was incubated for ∼2 hours. The indication reaction has not had time to diffuse throughout the LB agar. This method thus allows for rapid screening of isolated colonies of possible interest.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><img alt="Results from the pararosaniline indicator plate assay on E. coli containing various plasmid constructs. (A) Negative control plates with non-ethanogenic pPSBAIIKS. (B) Plates with isolated colonies expressing the pdc/adh cassette within pMota. (C) Positive control plate with pLOI295. (D) pMota plate." src="/image/article/2009/EE/b811937f/b811937f-f1.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Results from the pararosaniline <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plate assay on <span class="italic">E. coli</span> containing various <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> <span class="AN"><a class="simple" title="Sequence Ontology link for: constructs" href="javascript:popupOBO('SO:0000804','b811937f')">constructs</a></span>. (A) Negative control plates with non-ethanogenic pPSBAIIKS. (B) Plates with isolated colonies expressing the <span class="italic">pdc</span>/<span class="italic">adh</span> cassette within pMota. (C) Positive control plate with pLOI295. (D) pMota plate.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">The <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> <span class="AN"><a class="simple" title="Sequence Ontology link for: constructs" href="javascript:popupOBO('SO:0000804','b811937f')">constructs</a></span> tested positive using the above <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plate assay were then subjected to restriction mapping for more stringent analysis of the <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> <span class="AN"><a class="simple" title="Sequence Ontology link for: construct" href="javascript:popupOBO('SO:0000804','b811937f')">construct</a></span>, as shown in <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>. Here the ∼1 kb band located in lane 2 is some contaminant. This is not a problem, as <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> from this later <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: purification" href="javascript:popupOBO('CMO:0002231','b811937f')">purification</a></span> was not used for the initial transformation of PCC 6803 or for the construction of pMota. This batch of pPSBAIIKS will be used solely for novel <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> construction in the future, eliminating the possible interference of the 1 kb contaminant. Lane 3 shows the expected fragment sizes resulting from a SacI digest of pMota. Expected fragment sizes are observed at 5010 bp and 2276 bp. After successful restriction mapping, the <span class="AN"><a class="simple" title="Sequence Ontology link for: construct" href="javascript:popupOBO('SO:0000804','b811937f')">construct</a></span> was sequenced for final verification of pMota. Primers used for this sequencing reaction are listed in Appendix I, ESI.<a title="Select to navigate to footnote" href="#fn1">†</a> The complete pMota sequence is given in Appendix II, ESI.<a title="Select to navigate to footnote" href="#fn1">†</a></p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><img alt="Restriction digest mapping of pMota and pPSBAIIKS (control). Lanes 1 and 4 are Promega™ 1 kb step ladders. Lane two is NdeI/BamHI double digest of pPSBAIIKS with fragment sizes as expected. Lane 3 shows the expected fragment sizes resulting from a SacI digest of pMota." src="/image/article/2009/EE/b811937f/b811937f-f2.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Restriction digest mapping of pMota and pPSBAIIKS (control). Lanes 1 and 4 are Promega™ 1 kb step ladders. Lane two is NdeI/BamHI double digest of pPSBAIIKS with fragment sizes as expected. Lane 3 shows the expected fragment sizes resulting from a SacI digest of pMota.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">We found that the modification of the pMota <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> with a desired multiple cloning site should allow for simultaneous expression and analysis of other complementary <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span> products with the <span class="italic">pdc</span>/<span class="italic">adh</span> cassette. This may open up possibilities for future metabolic modifications <span class="italic">via</span> the following methods: (1) overexpression of pathway components; (2) overexpression of <span class="AN"><a class="simple" title="Gene Ontology link for: sigma factor" href="javascript:popupOBO('GO:0016987','b811937f')">sigma factor</a></span>/<span class="AN"><a class="simple" title="Gene Ontology link for: sigma factor" href="javascript:popupOBO('GO:0016987','b811937f')">sigma factor</a></span> binding <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>;<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a> (3) <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span>-silencing <span class="italic">via</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: antisense" href="javascript:popupOBO('SO:0000077','b811937f')">antisense</a></span> based <span class="AN"><a class="simple" title="Gene Ontology link for: RNAi" href="javascript:popupOBO('GO:0016246','b811937f')">RNAi</a></span>.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a></p>
      
      
        <h3 id="sect512"><span class="b_heading">Transformation and screening for stable <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production</span></h3>
        <span>After successful creation of the pMota <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span>, <span class="italic">Synechocystis</span> strains used in this study were sequentially transformed and isolated for correct construction, first with pPSBAIIKS (creating WT<small><sub>[r]</sub></small>) and then with pMota (creating WT<small><sub>[OH]</sub></small>, here the subscript [OH] denotes the <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> bearing <span class="italic">pdc</span>/<span class="italic">adh</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span>). Results from the PCR based assay, shown in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>, were used to confirm isolation of strains with homologous <span class="italic">psbA2</span> loci. The first lane in gels (a)–(e) is a 1 kb BRL ladder. Gel conditions in gels (a)–(e) were constant, 1% agarose gel in 2X <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: TAE buffer" href="javascript:popupOBO('CMO:0001750','b811937f')">TAE buffer</a></span> run at 100 V for 45 min. The last lane in each of the gels (a)–(e) is unused. Starting from the first lane after the BRL ladder, samples were run, with each set of three consecutive lanes corresponding to the same sample tested with the WT, [r], and [OH] assays respectively: (a) Sample 1 and 2 were the WT <span class="italic">Synechocystis</span> 6803 strain. As shown here, the WT psbA2 locus was intact in each of these strains and the other internal primers show no cross-reactivity with the WT psbA2 <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span> (<span class="italic">i.e.</span> the 1st lane for each sample set bears the expected 749 bp amplicon, while the 2nd and 3rd lanes corresponding to the [r] and [OH] assay show no amplification). (b) Sample 3 and 4 were for WT<small><sub>[r]</sub></small>. Here it can be seen that the WT<small><sub>[r]</sub></small> strain was completely segregated, while there was no cross reaction with the [OH] assay. In (c) it can be seen that the above WT<small><sub>[OH]</sub></small> (<span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing) isolates gave consistent results after additional bioreactor runs. As shown in both samples 5 and 6, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> cassette hybridized only with the <span class="italic">pdc</span>-specific 3′ primer, as expected. The <span class="italic">Synechocystis</span> strains were then grown under photoautotrophic conditions in a bioreactor and shown to be <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing with respect to the WT control strain. During <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> in the bioreactor, samples were taken and assayed again, yielding the results for samples 7–11. In gels (d), a 1 kb Hyperladder<small><sup>®</sup></small> (Bioline, Taunton, MA, USA) was used as a molecular weight marker. Gel (d) also shows the isolation of 5 strains (7–11) of WT<small><sub>[OH]</sub></small>, screened <span class="italic">via</span> the [OH] reaction of the PCR assay. Single bands at the expected 554 amplicon size were present for samples 7–11. These same five isolates were then tested for reactivity with the [r] reaction of the PCR assay on gel (e); showing no <span class="AN"><a class="simple" title="Gene Ontology link for: response to" href="javascript:popupOBO('GO:0014823','b811937f')">response to</a></span> the [r] primed reaction (lanes labeled 12–16 were the same samples as 7–11).</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><img alt="PCR assay results showing isolation of completely segregated transformants of WT[r] and WT[OH]." src="/image/article/2009/EE/b811937f/b811937f-f3.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">PCR assay results showing isolation of completely segregated transformants of WT<small><sub>[r]</sub></small> and WT<small><sub>[OH]</sub></small>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Colonies showing successful isolation of desired strains were then <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glycerol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.733.html">glycerol</a></span> stocked. Successful isolates bearing the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> cassette were screened using the primary seed reactor based assay, as described above. Many of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> cassette bearing isolates were shown to give reduced <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yields with continued culturing (data not shown). Typically, these strains would show significant <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b811937f')">reduction</a></span> in <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yields with each consecutive serial inoculation, such that by the second or third serial culture we were able to determine the stability of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production within the strain. One particular isolate was shown to give strong and stable <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production after repeated culturing. This isolate was designated as WT5.2<small><sub>[OH]</sub></small> which has been successfully and stably transformed with the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production cassette. As shown in <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>, the WT5.2<small><sub>[OH]</sub></small> strain shows repeatable <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production during five consecutive cultures.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><img alt="Primary seed reactor assay testing for stability of Ethanol production during repeated culturing. (A) Optical density vs. days for five consecutive batch runs; (B) Ethanol concentration (mM) vs. days for five consecutive batch runs." src="/image/article/2009/EE/b811937f/b811937f-f4.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Primary seed reactor assay testing for stability of <span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span> production during repeated culturing. (A) Optical density <span class="italic">vs.</span> days for five consecutive batch runs; (B) <span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span> concentration (mM) <span class="italic">vs.</span> days for five consecutive batch runs.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Confirmation of stability of the created <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing strains <span class="italic">via</span> the seed reactor based assay was critical for strain isolation. It was found through the seed reactor based assay that several isolates with complete segregation for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production cassette did not show stable <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production (data not shown). Typically, this could be deduced by the third serial liquid culturing of an isolate. This process has also been invaluable for the continued monitoring of strains after revival from <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glycerol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.733.html">glycerol</a></span> stock. Unfortunately, our primary seed reactor system is quite rudimentary, not allowing for pH control, gas flow rate control, and not having condensers on the off-gas outlet, resulting in continuous and unequal evaporative losses from parallel reactors. Also, our current platform can only support the simultaneous <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> of 6 such primary seed reactors. Increasing our capacity for parallel processing of primary seed reactors is critical for increasing the rate of obtaining results, ideally with added control regarding the aforementioned parameters.</p>
      
      
        <h3 id="sect598"><span class="b_heading">Creation of novel <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> pathway in <span class="italic">Synechocystis</span> metabolic network</span></h3>
        <span>The autotrophic <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> of <span class="italic">Synechocystis</span> strains encompasses two fundamental processes: (1) <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> use light as the energy source to drive the electron-transfer reactions that enable electrons from <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> to move along an <span class="AN"><a class="simple" title="Gene Ontology link for: electron transport chain" href="javascript:popupOBO('GO:0022900','b811937f')">electron transport chain</a></span> that leads to the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b811937f')">reduction</a></span> of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:NADP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5674.html">NADP</a></span> to NADPH. During the photosynthetic electron <span class="AN"><a class="simple" title="Gene Ontology link for: transport" href="javascript:popupOBO('GO:0006810','b811937f')">transport</a></span>, H<small><sup>+</sup></small> is pumped across the <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b811937f')">membrane</a></span> and the resulting <span class="CH"><a class="simple" title="ChEBI Ontology link for: proton" href="javascript:popupOBO('CHEBI:24636','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">proton</a></span>-motive force drives the formation of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span>. Thus, the light reactions couple with the production of energy and reducing power, as well as the evolution of O<small><sub>2</sub></small>. (2) <span class="AN"><a class="simple" title="Gene Ontology link for: Carbon fixation" href="javascript:popupOBO('GO:0015977','b811937f')">Carbon fixation</a></span> reactions in which the ATP and NADPH produced by the photosynthetic electron <span class="AN"><a class="simple" title="Gene Ontology link for: transport" href="javascript:popupOBO('GO:0006810','b811937f')">transport</a></span> reactions are utilized for the conversion of CO<small><sub>2</sub></small> to <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> and other <span class="CH"><a class="simple" title="ChEBI Ontology link for: metabolites" href="javascript:popupOBO('CHEBI:25212','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25212')">metabolites</a></span>. By our genetic engineering efforts, we have successfully inserted a novel <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>-producing pathway into the <span class="italic">Synechocystis</span> metabolic network. As a result, the carbon flow from <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:pyruvate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.96901.html">pyruvate</a></span> was directed to this <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> pathway for formation of the end metabolic product, <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>. It can be seen that a unique combination of three cellular functions, oxygenic <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b811937f')">photosynthesis</a></span>, <span class="AN"><a class="simple" title="Gene Ontology link for: carbon fixation" href="javascript:popupOBO('GO:0015977','b811937f')">carbon fixation</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production within the single organism, <span class="italic">Synechocystis</span> W5.2<small><sub>[OH]</sub></small>, has been achieved. This forms the foundation for the direct conversion of CO<small><sub>2</sub></small> to <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> driven by the light energy.</span>
      
      
        <h3 id="sect638"><span class="b_heading">Autotrophic photobioreactor <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span></span></h3>
        <span>The WT5.2<small><sub>[OH]</sub></small> strain was then examined under the more stringent control of a commercially based photobioreactor. <a title="Select to navigate to figure" href="#imgfig5">Fig. 5A</a> (averaging of three runs of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> experiments) shows that the WT5.2<small><sub>[OH]</sub></small> strain has a significantly reduced <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> compared to the non-ethanol-producing control. <a title="Select to navigate to figure" href="#imgfig5">Fig. 5A</a> also shows that the decrease in <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> flux into the reactor due to increased optical density of the <span class="italic">Synechocystis</span> cultures. <a title="Select to navigate to figure" href="#imgfig5">Fig. 5B</a> (averaging of three runs of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> experiments) shows that the WT5.2<small><sub>[OH]</sub></small> strain produces detectable amounts of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>, whereas the WT control strain does not.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><img alt="(A) Effect of cell density (optical density = OD730) on photon flux through the photobioreactor. (B) Ethanol concentration vs. time (hour). The results are the average of three individual experimental runs." src="/image/article/2009/EE/b811937f/b811937f-f5.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">(A) Effect of <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> density (optical density = OD<small><sub>730</sub></small>) on <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> flux through the photobioreactor. (B) <span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span> concentration <span class="italic">vs.</span> time (hour). The results are the average of three individual experimental runs.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">The <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yields from the BioFlo 110<small><sup>®</sup></small> photobioreactor (New Brunswick Scientific, Edison, NJ, USA) are significantly greater than those obtained from the primary seed reactors. This could be due to a number of factors, but it is significantly due to the increased illumination of the BioFlo 110<small><sup>®</sup></small> and the presence of a condenser at the gas outlet. Given that this system had control of critical features, such as gas flow rate, pH and a condenser on the off-gas, in comparison to the primary seed platform, the results were deemed more representative of actual <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yields and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> rate. The pH-stat control strategy also played a critical role. Without pH control, the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> may become starved by insufficient CO<small><sub>2</sub></small> supply. Multiple discreet <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: pH measurements" href="javascript:popupOBO('CMO:0002354','b811937f')">pH measurements</a></span> in the primary seed reactor have shown the pH was always greater than 10.5 at the later phase of the <span class="italic">Synechocystis</span> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> cultures. This indicates the cultures lack of CO<small><sub>2</sub></small> supply—we only sparged compressed air into the primary seed reactors, this did not appear to meet the demand. Also, with access to only a single reactor it is impossible to co-ordinate experiments with parallel variables/negative control. For this reason, analysis using the BioFlo 110<small><sup>®</sup></small> photobioreactor platform was a slow and tedious process. Despite this, the BioFlo 110<small><sup>®</sup></small> platform described here offered the most controlled system for analysis of generated cyanobacterial strains.</p>
        <p class="otherpara">Perhaps one of the most important observations from the combined BioFlo 110<small><sup>®</sup></small> runs is the fundamental consideration of limiting factors for a given photobioreactor design. The standard vat type reactor shape was shown to severely limit light flux through the reactor volume; once the OD<small><sub>730</sub></small> was increased to be around 1, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> flux at about 1 cm within the reactor was reduced by between 60% and 80%. Thus, light limitation is a critical factor that must be considered for reactor design. An additional complexity is that light availability also effects <span class="italic">pdc</span>/<span class="italic">adh</span> expression using the psbAII promoter expression system. Gas limiting has also been observed in primary seed cultures (data not shown). Thus, CO<small><sub>2</sub></small> and light availability have presented themselves as the two major limiting factors in both the culturing of <span class="italic">Synechocystis</span> 6803 and in <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. Maintenance of the pH at a given setpoint will allow for the control of CO<small><sub>2</sub></small> availability, this leaves light availability as the most critical environmental variable for analysis and control.</p>
      
      
        <h3 id="sect689"><span class="b_heading">Some issues for future study</span></h3>
        <span>
          <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a> indicates that due to the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production (which was a metabolic burden for the <span class="italic">Synechocystis</span> <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span>), the WT5.2<small><sub>[OH]</sub></small> strain could reach a final <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> density level of about half that of the wild type <span class="italic">Synechocystis</span> <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> which would direct more carbon resources towards biomass formation. It should be noted that in Fig. 5B, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration peaked around 12 mM. It then started to decrease. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span> reutilization may be possible by the <span class="italic">Synechocystis</span> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> cultures because <span class="italic">Z. mobilis</span> <span class="AN"><a class="simple" title="Gene Ontology link for: alcohol dehydrogenase" href="javascript:popupOBO('GO:0004022','b811937f')">alcohol dehydrogenase</a></span> II has been shown to be a bidirectional <span class="CH"><a class="simple" title="ChEBI Ontology link for: catalyst" href="javascript:popupOBO('CHEBI:35223','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35223')">catalyst</a></span><a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> and <span class="italic">Synechocystis</span> 6803 contains an endogenous <span class="CH"><a class="simple" title="ChEBI Ontology link for: aldehyde" href="javascript:popupOBO('CHEBI:17478','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=17478')">aldehyde</a></span> dehydrogenase (<span class="AN"><a class="simple" title="Sequence Ontology link for: ORF" href="javascript:popupOBO('SO:0000236','b811937f')">ORF</a></span>: slr0091) that may shunt <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:acetaldehyde" target="_blank" href="http://www.chemspider.com/Chemical-Structure.172.html">acetaldehyde</a></span> to <span class="AN"><a class="simple" title="Gene Ontology link for: acetate biosynthesis" href="javascript:popupOBO('GO:0019413','b811937f')">acetate biosynthesis</a></span>. This competing pathway may need to be addressed in future studies.</span>
        <p class="otherpara">Another question that arises is the possibility that <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> accumulation in the media may have direct toxic effects on the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> culture, and thus inhibit <span class="italic">Synechocystis</span> <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span>. The toxicity of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> on the <span class="italic">Synechocystis</span> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> cultures was examined by culturing <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> in parallel with the addition of various amount of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> to the culture media to make the residual <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration to vary from 0.5% to 5% (∼115–1150 mM). The experimental results indicate that <span class="italic">Synechocystis</span> can tolerate concentrations of up to 1% (∼230 mM) <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> in the media with negligible impacts on <span class="AN"><a class="simple" title="Gene Ontology link for: cell growth" href="javascript:popupOBO('GO:0016049','b811937f')">cell growth</a></span> (data not shown). It can thus be concluded that the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> accumulation in the media caused by our <span class="italic">Synechocystis</span> mutants has not reached the inhibitory level. Therefore, reduced <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> density in the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> cultures with <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>-producing <span class="italic">Synechocystis</span> mutants was not caused by the alcohol inhibition, but by the metabolic reprogramming of the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> with the <span class="AN"><a class="simple" title="Sequence Ontology link for: insertion" href="javascript:popupOBO('SO:0000667','b811937f')">insertion</a></span> of new pathway into its metabolic network.</p>
        <p class="otherpara">Several microorganisms, including <span class="italic">Clostridium</span> sp., are ethanologenic microbes, among them, the yeast <span class="italic">Saccharomyces cerevisiae</span> and facultative bacterium <span class="italic">Zymomonas mobilis</span> are considered as better candidates for industrial <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. <span class="italic">S. cerevisiae</span> is capable of fermenting <span class="CH"><a class="simple" title="ChEBI Ontology link for: sugars" href="javascript:popupOBO('CHEBI:16646','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16646')">sugars</a></span> to produce <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> anaerobically or microaerobically with high yields (over 15%). <span class="italic">Zymomonas mobilis</span> is a prokaryotic organism that produces <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>. In comparison to <span class="italic">S. cerevisiae</span>, <span class="italic">Z. mobilis</span> possesses the following advantages: 1. its higher sugar uptake and <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yield, 2. its lower biomass production, 3. its higher <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> tolerance, 4. it does not require controlled addition of oxygen during the <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span>, and 5. its amenability to genetic manipulations.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> Wild type <span class="italic">Synechocystis</span> does not possess the pathway for <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. In this paper, we report our genetic engineering efforts of creating <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>-producing cyanobacteria for the production of biofuels. So far, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yield in our engineered <span class="italic">Synechocystis</span> strains is lower than both <span class="italic">S. cerevisiae</span> and <span class="italic">Z. mobilis</span>. Since optimization strategies have not yet been implemented, and since the <span class="AN"><a class="simple" title="Gene Ontology link for: pyruvate decarboxylase" href="javascript:popupOBO('GO:0004737','b811937f')">pyruvate decarboxylase</a></span> (<span class="italic">pdc</span>) and <span class="AN"><a class="simple" title="Gene Ontology link for: alcohol dehydrogenase" href="javascript:popupOBO('GO:0004022','b811937f')">alcohol dehydrogenase</a></span> II (<span class="italic">adh</span>) enzymes are not evolved to fully function in the oxygenic photosynthetic organisms, there is no reason to assume that its performance would be optimal. Here we only intend to show that cyanobacteria <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> can stably express <span class="italic">pdc</span> and <span class="italic">adh</span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and produce <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> from CO<small><sub>2</sub></small> and light when transformed with heterologous <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span>. There exists a significant room for the improvement of Synechcystis <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production from the aspects of both molecular biology strategies and bioprocess development.</p>
      
    
    
      <h2 id="sect777"><span class="a_heading">Materials and methods</span></h2>
      
        <h3 id="sect779"><span class="b_heading">Strains/culture conditions</span></h3>
        <span>
          <span class="italic">E. coli</span> strain C600 was used for the <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> cloning. Standard LB formulation (10 g NaCl, 10 g tryptone, 5 g yeast extract per litre, pH 7) was used to culture <span class="italic">E. coli</span> in both liquid and solid (add 15 g agar per liter) media. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Antibiotics" href="javascript:popupOBO('CHEBI:22582','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22582')">Antibiotics</a></span> <span class="TC"><a class="simple" title="Chemspider Compound link for:ampicillin" target="_blank" href="http://www.chemspider.com/Chemical-Structure.6013.html">ampicillin</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: kanamycin" href="javascript:popupOBO('CHEBI:6104','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=6104')">kanamycin</a></span> were supplemented to the LB media at concentrations of 100 µg ml<small><sup>−1</sup></small> and 50 µg ml<small><sup>−1</sup></small>, respectively. Wild Type (WT) <span class="italic">Synechocystis</span> sp. strain PCC 6803 was cultured on standard BG-11 media plates, with or without the addition of: 5 mM <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glucose" target="_blank" href="http://www.chemspider.com/Chemical-Structure.96749.html">glucose</a></span>, 5% <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:sucrose" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5768.html">sucrose</a></span>, and/or either 5 µg ml<small><sup>−1</sup></small>, 25 µg ml<small><sup>−1</sup></small>, or 50 µg ml<small><sup>−1</sup></small> <span class="CH"><a class="simple" title="ChEBI Ontology link for: kanamycin" href="javascript:popupOBO('CHEBI:6104','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=6104')">kanamycin</a></span>. Plates containing <span class="italic">Synechocystis</span> strains were incubated at 30 °C under a light intensity of ∼100 microeinsteins m<small><sup>−2</sup></small> s<small><sup>−1</sup></small>. All <span class="italic">Synechocystis</span> liquid cultures were grown in standard BG-11 medium with the addition of 50 µg ml<small><sup>−1</sup></small> <span class="CH"><a class="simple" title="ChEBI Ontology link for: kanamycin" href="javascript:popupOBO('CHEBI:6104','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=6104')">kanamycin</a></span> when appropriate. A series of two seed cultures, each requiring about 4 days of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span>, were used for the inoculation of a 7.5 L photobioreactor (3 L working volume). Liquid primary seed cultures were grown at 24 °C, with a maximal surface <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> flux of ∼200 microeinsteins m<small><sup>−2</sup></small> s <small><sup>−1</sup></small> on the seed reactor wall. The seed cultures were agitated <span class="italic">via</span> magnetic stir bar, and sparged continuously with compressed air at a rate of approximately 0.5 L min<small><sup>−1</sup></small>. Primary seed cultures consisted of a working volume of 25 mL in a standard 100 mL Pyrex™ media bottle, with a two Pasteur pipettes serving as the sparge and the off-gas tubes. <span class="AN"><a class="simple" title="Sequence Ontology link for: Upstream" href="javascript:popupOBO('SO:0001635','b811937f')">Upstream</a></span> of the sparge tube was a Whatman<small><sup>®</sup></small> PFTE 0.1 µm filter and the off-gas tube was loosely capped with aluminium foil. Secondary seed cultures used for photobioreactor inoculation was nearly identical to the primary seed bottles except that the working volume was 300 mL in a standard 1 L Pyrex™ media bottle, and that the sparging pipette was replaced with a level C porosity diffuser with <span class="AN"><a class="simple" title="Gene Ontology link for: pore" href="javascript:popupOBO('GO:0015267','b811937f')">pore</a></span> size of 25–50 µm from Ace Glass Inc. Lights used were 40 W cool white fluorescent tubes. <span class="AN"><a class="simple" title="Gene Ontology link for: Cell growth" href="javascript:popupOBO('GO:0016049','b811937f')">Cell growth</a></span> in liquid culture was monitored by determination of optical density (OD) of the samples at 730 nm (OD<small><sub>730</sub></small>) using a <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrophotometer" href="javascript:popupOBO('CMO:0002232','b811937f')">spectrophotometer</a></span> (ThermoSpectronic Genesys<small><sup>®</sup></small> 10-S, Beckman Coulter, Inc., Fullerton, CA, USA). UltraVette<small><sup>®</sup></small> 1.5 ml disposable cuvettes from Plastibrand (Vineland, NJ, USA) were used for the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: OD measurement" href="javascript:popupOBO('CMO:0002039','b811937f')">OD measurement</a></span>.</span>
      
      
        <h3 id="sect827"><span class="b_heading">Creation of the transformation vector, pMota</span></h3>
        <span>All <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span>/<span class="AN"><a class="simple" title="Sequence Ontology link for: PCR product" href="javascript:popupOBO('SO:0000006','b811937f')">PCR product</a></span> cleanup kits and <span class="AN"><a class="simple" title="Gene Ontology link for: Taq DNA polymerase" href="javascript:popupOBO('GO:0003887','b811937f')">Taq DNA polymerase</a></span> were acquired from Qiagen (Valencia, CA, USA). All restriction enzymes, Vent<small><sub>R</sub></small><small><sup>®</sup></small> Polymerase and T4 <span class="AN"><a class="simple" title="Gene Ontology link for: DNA ligase" href="javascript:popupOBO('GO:0003909','b811937f')">DNA ligase</a></span> were obtained from New England Biolabs (Ipswich, MA, USA). The <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> PSBAIIKS was obtained from Dr Vermaas at Arizona State University. The <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> LOI295, containing the Z. mobilis <span class="italic">pdc</span> and <span class="italic">adhII</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: genes" href="javascript:popupOBO('SO:0000704','b811937f')">genes</a></span>, was obtained from Dr Lonnie Ingram at the University of Florida. PCR reaction was used for both the amplification of the <span class="italic">pdc</span>/<span class="italic">adhII</span> cassette from pLOI295 and for the simultaneous introduction of NdeI and BamHI sites at the 5′ and 3′ ends, respectively. These sites then allowed for subcloning <span class="italic">pdc</span>/<span class="italic">adhII</span> into the backbone of pPSBAIIKS. It resulted from the removal of <span class="italic">aphX</span>/<span class="italic">sacB</span> selection cassette <span class="italic">via</span> NdeI/BamHI dual <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b811937f')">digestion</a></span>, and in the creation of transformation vector, pMota (see Appendix III, ESI).<a title="Select to navigate to footnote" href="#fn1">†</a> The following primers were used for the above PCR reaction (restriction sites are italicised, induced mutations are in bold): <span class="AN"><a class="simple" title="Sequence Ontology link for: upstream" href="javascript:popupOBO('SO:0001635','b811937f')">upstream</a></span>: 5′-ggAgTAAg<span class="italic">CA<span class="bold">T</span>ATg</span>AgTTATACTg-3′ <span class="AN"><a class="simple" title="Sequence Ontology link for: Downstream" href="javascript:popupOBO('SO:0001633','b811937f')">Downstream</a></span>: 5′-ggATCTCgACTCTAgA<span class="italic">ggATC<span class="bold">C</span></span>-3′, with a resultant amplicon of 3117 bp. PCR reaction was carried out as follows: total reaction volume of 50 µl, 0.36 µg of pLOI295 as template, 4 units of Vent<small><sub>R</sub></small><small><sup>®</sup></small> polymerase, a final concentration of 0.5 µM for each primer, 300 µM of each <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:dNTP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.13844817.html">dNTP</a></span>. The PCR was run on the following program on an Eppendorf Mastercycler<small><sup>®</sup></small>: initial denaturation at 94 °C for 2 min, followed by 35 cycles of 10 s denaturation at 94 °C, 1 min <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: annealing" href="javascript:popupOBO('CMO:0001465','b811937f')">annealing</a></span> at 47 °C, and 3.7 min extension at 68 °C; finally, hold at 4 °C.</span>
      
      
        <h3 id="sect865"><span class="b_heading">Screening of E. coli for correct pMota <span class="AN"><a class="simple" title="Sequence Ontology link for: construct" href="javascript:popupOBO('SO:0000804','b811937f')">construct</a></span></span></h3>
        <span>After transformation of competent <span class="italic">E. coli</span> C600 <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> with the pMota ligation reaction, single colonies were analyzed on <span class="CH"><a class="simple" title="ChEBI Ontology link for: aldehyde" href="javascript:popupOBO('CHEBI:17478','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=17478')">aldehyde</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plates to verify the activity of the <span class="italic">Z. mobilis</span> <span class="AN"><a class="simple" title="Gene Ontology link for: alcohol dehydrogenase" href="javascript:popupOBO('GO:0004022','b811937f')">alcohol dehydrogenase</a></span> II <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b811937f')">enzyme</a></span>. These <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plates were formulated by the addition of 8 ml of pararosaniline (2.5 mg of the dry powder per ml of 95% <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span>; not autoclaved) and 100 mg of sodium bisulfite (unsterilized dry powder) to 400-mL batches of LB agar. Mixtures of pararosaniline and bisulfite are often referred to as Schiff reagent. It has been widely used to detect aldehydes, to detect <span class="CH"><a class="simple" title="ChEBI Ontology link for: sugars" href="javascript:popupOBO('CHEBI:16646','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16646')">sugars</a></span> on glycoproteins after <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:periodic acid" target="_blank" href="http://www.chemspider.com/Chemical-Structure.58684.html">periodic acid</a></span> <span class="CH"><a class="simple" title="Molecular Process Ontology link for: oxidation" href="javascript:popupOBO('MOP:0000568','b811937f')">oxidation</a></span>, or in a broth to test for organisms such as <span class="italic">Z. mobilis</span> which secrete aldehydes into the culture media.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> These components have been incorporated into a solid medium which is relatively nontoxic and can be used to identify clones expressing enzymes which produce aldehydes. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span> diffuses into the E. coli <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span>, where it can be converted to <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:acetaldehyde" target="_blank" href="http://www.chemspider.com/Chemical-Structure.172.html">acetaldehyde</a></span> by <span class="AN"><a class="simple" title="Gene Ontology link for: alcohol dehydrogenase" href="javascript:popupOBO('GO:0004022','b811937f')">alcohol dehydrogenase</a></span>. The leuco <span class="CH"><a class="simple" title="ChEBI Ontology link for: dye" href="javascript:popupOBO('CHEBI:37958','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=37958')">dye</a></span> serves as a sink, reacting with <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:acetaldehyde" target="_blank" href="http://www.chemspider.com/Chemical-Structure.172.html">acetaldehyde</a></span> to form a <span class="CH"><a class="simple" title="ChEBI Ontology link for: Schiff base" href="javascript:popupOBO('CHEBI:50229','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=50229')">Schiff base</a></span> which is intensely red.<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a></span>
        <p class="otherpara">
          <span class="italic">E. coli</span> colonies that yielded positive results on the <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plates <span class="italic">via</span> presence of intense red coloration were subsequently subjected to restriction mapping. Qiagen<small><sup>®</sup></small> miniprep kits were used for the preparation of <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b811937f')">DNA</a></span> from positive colonies on the <span class="CH"><a class="simple" title="ChEBI Ontology link for: indicator" href="javascript:popupOBO('CHEBI:47867','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=47867')">indicator</a></span> plates. Purified <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmids" href="javascript:popupOBO('SO:0000155','b811937f')">plasmids</a></span> were subjected to NdeI/BamHI double <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b811937f')">digestion</a></span> and SacI <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b811937f')">digestion</a></span> for verification of fragment sizes. Digests were run on 2X TAE, 1% agarose gels at 100 V for 40 min. Gels were stained with <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ethidium bromide" target="_blank" href="http://www.chemspider.com/Chemical-Structure.14034.html">ethidium bromide</a></span> and visualized on UV light box. After successful verification <span class="italic">via</span> restriction mapping, pMota was sequenced using the primer set for sequencing analysis of the <span class="italic">pdc</span>/<span class="italic">adh</span> cassette within <span class="italic">Synechocystis</span> PCC 6803. Sequencing results showed successful creation of the pMota <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> for <span class="AN"><a class="simple" title="Sequence Ontology link for: insertion" href="javascript:popupOBO('SO:0000667','b811937f')">insertion</a></span> of the <span class="italic">pdc</span>/<span class="italic">adh</span> cassette into the <span class="italic">psbA2</span> locus.</p>
      
      
        <h3 id="sect922"><span class="b_heading">Transformation and screening for stable <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production</span></h3>
        <span>Following successful creation of pMota, the mutant strains of <span class="italic">Synechocystis</span> sp. PCC 6803 were transformed and screened sequentially with pPSBAIIKS and pMota <span class="italic">via</span> the protocol described by Williams<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> and Lagarde <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> After transformation with pPSBAIIKS, a serial replating on increasing <span class="CH"><a class="simple" title="ChEBI Ontology link for: kanamycin" href="javascript:popupOBO('CHEBI:6104','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=6104')">kanamycin</a></span> concentrations were conducted for the isolation of <span class="italic">Synechocystis</span> mutant strains WT<small><sub>[r]</sub></small> (here the subscript [r] denotes the <span class="AN"><a class="simple" title="Sequence Ontology link for: plasmid" href="javascript:popupOBO('SO:0000155','b811937f')">plasmid</a></span> with resistance cassette), which were completely segregated with respect to the <span class="italic">aphX</span>/<span class="italic">sacB</span> selection cassette, as verified by a PCR based assay. The <span class="AN"><a class="simple" title="Sequence Ontology link for: constructs" href="javascript:popupOBO('SO:0000804','b811937f')">constructs</a></span> were in turn grown in liquid seed culture, under the presence of 50 µg mL<small><sup>−1</sup></small> <span class="CH"><a class="simple" title="ChEBI Ontology link for: kanamycin" href="javascript:popupOBO('CHEBI:6104','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=6104')">kanamycin</a></span>, and subsequently transformed with pMota. The transformants were then screened on BG-11 plates containing 5% <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:sucrose" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5768.html">sucrose</a></span>. Screening was performed <span class="italic">via</span> a serial streaking of single colonies coupled with a PCR based assay used for probing the <span class="italic">psbA2</span> loci. The PCR assay consisted of three PCR reactions per sample, probing for the presence of the (1) WT <span class="italic">psbA2</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span>, (2) <span class="italic">aphX</span>/<span class="italic">sacB</span> selection cassette, and (3) <span class="italic">pdc</span>/<span class="italic">adhII</span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> pathway cassette. Each of the three reactions shared a common <span class="AN"><a class="simple" title="Sequence Ontology link for: upstream" href="javascript:popupOBO('SO:0001635','b811937f')">upstream</a></span> primer that lies outside of the <span class="italic">psbA2</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span> loci, while each reaction is defined by the <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b811937f')">downstream</a></span> primer that is specific for each of the three possible genetic <span class="AN"><a class="simple" title="Sequence Ontology link for: constructs" href="javascript:popupOBO('SO:0000804','b811937f')">constructs</a></span>. The <span class="AN"><a class="simple" title="Sequence Ontology link for: upstream" href="javascript:popupOBO('SO:0001635','b811937f')">upstream</a></span> primer used in all three reactions was: 5′-gTCAgTTCCAATCTgAACATCGA-3′, with the amplicon beginning 48 bp <span class="AN"><a class="simple" title="Sequence Ontology link for: upstream" href="javascript:popupOBO('SO:0001635','b811937f')">upstream</a></span> of the <span class="italic">psbA2</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: start codon" href="javascript:popupOBO('SO:0000318','b811937f')">start codon</a></span>. The <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b811937f')">downstream</a></span> primer for probing the WT <span class="italic">psbA2</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: gene" href="javascript:popupOBO('SO:0000704','b811937f')">gene</a></span>: 5′-AATTTgTAACCgTAgTTCTgggAT-3′, and the resultant amplicon is 749 bp. For probing the <span class="italic">aphX</span>/<span class="italic">sacB</span> selection cassette, <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b811937f')">downstream</a></span> primer: 5′–TTggTgATTTTgAACTTTTgCTTgC-3′ was used, resulting in a 3.1 kb amplicon. The <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b811937f')">downstream</a></span> primer for probing the <span class="italic">pdc</span>/<span class="italic">adhII</span> cassette: 5′-TTgCAAgCgATTTCgAgATAAA-3′, resulting in a 554 bp amplicon. All PCR reactions were formulated as described in the Qiagen<small><sup>®</sup></small> Taq Polymerase Handbook in the section for long <span class="AN"><a class="simple" title="Sequence Ontology link for: PCR products" href="javascript:popupOBO('SO:0000006','b811937f')">PCR products</a></span>, modified only by the exclusion of any high fidelity polymerase. The PCR assay utilized the following cycling program: initial denaturation at 94 °C for 3 min, followed by 35 cycles of 10 s denaturation at 94 °C, 1 min <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: annealing" href="javascript:popupOBO('CMO:0001465','b811937f')">annealing</a></span> at 48 °C, and 3.5 min extension at 68 °C; a final 3 min extension at 68 °C, hold at 4 °C.</span>
        <p class="otherpara">To perform the PCR assay on a given cyanobacterial sample, FTA<small><sup>®</sup></small> cards (Whatman Inc., Piscataway, NJ, USA) were used for rapid preparation of <span class="AN"><a class="simple" title="Sequence Ontology link for: genomic DNA" href="javascript:popupOBO('SO:0000991','b811937f')">genomic DNA</a></span> for use as a template in the above PCR reaction. For testing a liquid culture, 5 µL aliquot was spotted onto the FTA<small><sup>®</sup></small> card. For testing cultures streaked on solid media, multiple colonies were lifted from the plate, streaked on the inside of a 1.5 mL tube and resuspended in 10 µL of BG-11 <span class="italic">via</span> vortexing; 5 µL aliquots were then spotted onto the FTA<small><sup>®</sup></small> card, as above. The FTA<small><sup>®</sup></small> protocol for preparation of the archived <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b811937f')">DNA</a></span> from a bacterial source was followed for preparation of the template for the PCR assay.</p>
        <p class="otherpara">A seed reactor based assay was also used for determination of stability of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> generation in consideration of the absence of selective pressure. The assay focused on screening colonies that showed to be completely segregated for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> cassette for stable <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. Seed reactors as described above were inoculated with multiple colonies from a plate of a given isolate. The <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> were grown to an OD<small><sub>730</sub></small> greater than 0.1. They were then centrifuged at 3220 × <span class="italic">g</span> for 6 min at room temperature, and resuspended in a fresh seed reactor at an initial OD<small><sub>730</sub></small> = 0.025. This would constitute the first experimental reactor in a series of five runs. Each seed reactor <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> culture was run for five days with continuous feeding of compressed air, at which point the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> were again collected by <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b811937f')">centrifugation</a></span> and used to inoculate the second experimental reactor in the series to the above OD<small><sub>730</sub></small> = 0.025. Only a subset of the total <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> biomass was used for this serial inoculation, while the <span class="AN"><a class="simple" title="Gene Ontology link for: rest" href="javascript:popupOBO('GO:0043336','b811937f')">rest</a></span> was discarded or used to make <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glycerol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.733.html">glycerol</a></span> stocks. Each day of a particular run, the OD<small><sub>730</sub></small> were recorded, and a 550 µL aliquot was taken for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration assay (the ‘before’ aliquot). The <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> were then washed by collection <span class="italic">via</span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b811937f')">centrifugation</a></span> (as above), discarding the supernatant, resuspended by vortexing of the entire pellet in 25 mL of fresh BG-11, and returned to the seed reactor. The OD<small><sub>730</sub></small> was again recorded and another aliquot was taken for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration assay (the so called ‘after’ aliquot). After isolation of a stable <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing isolate, the PCR based assay was applied a final time for complete confirmation (data not shown).</p>
      
      
        <h3 id="sect1015"><span class="b_heading">Autotrophic photobioreactor <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span></span></h3>
        <span>Exponentially growing seed cultures in the primary seed reactors were used to inoculate a secondary seed reactor, which was then grown to an OD<small><sub>730</sub></small> &gt; 0.31 and immediately used for inoculation of the autotrophic photobioreactor. For inoculation, the OD<small><sub>730</sub></small> was taken and used for determination of the seed culture volume required for the initial bioreactor OD<small><sub>730</sub></small> = 0.02 (working <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> volume was 3.1 L). <span class="AN"><a class="simple" title="Cell Ontology link for: Cells" href="javascript:popupOBO('CL:0000000','b811937f')">Cells</a></span> from this determined volume of seed culture were harvested <span class="italic">via</span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b811937f')">centrifugation</a></span> and resuspended in 50 mL of fresh BG-11 by vortexing. After inoculation of the bioreactor with this 50 mL seed, an additional 50 mL of BG-11 was used for flushing of the inoculation tube. A 7.5 L BioFlo<small><sup>®</sup></small> 110 bioreactor system (New Brunswick Scientific, Edison, NJ, USA) was used for our experiments. It has its own controllers for temperature, pH and DO adjustment. The <span class="italic">Synechocystis</span> cultivation process was monitored and controlled automatically by a Pentium II (233 MHz, Windows 98) computer equipped with an interface board PCI-MIO-16E-10 (National Instruments Corp., Austin, TX, USA). The data acquisition program was written in LabVIEW7.1 (National Instruments Corp., Austin, TX, USA). The data from Bioflo 110, including pH, agitation, temperature and dissolved oxygen concentration (DO) was acquired through the computer interface board. Six General Electric<small><sup>®</sup></small> 26 W, F26DBX/SPX41/4P light bulbs were arrayed around the reactor to give a maximal surface flux of ∼1000 microeinsteins m<small><sup>−2</sup></small> s <small><sup>−1</sup></small> on the reactor wall. The local irradiance level of the light intensity at a fixed point (1 cm away from the internal bioreactor surface) was measured using a cosine (2π) quantum meter (LI-190 <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: SA" href="javascript:popupOBO('CMO:0001305','b811937f')">SA</a></span>, LI-COR, Lincoln, NE, USA) with a spherical (4π collector) quantum micro-sensor (US-SQS/LI Heinz Walz GmbH, Effeltrich, Germany), and expressed as photosynthetic <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> flux density (PPFD). This fiber-optic micro sensor has a spherical sensor tip (with a diameter of 3 mm) that can sense light from all directions underwater. For protection, the sensor was inserted into a glass optical well that was fixed to the bioreactor head plate and submerged in the culture broth. The reactor was maintained at 27–29 °C throughout the <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> <span class="italic">via</span> an external fan circulating air around the lighting system. The reactor was sparged continuously with compressed air or N<small><sub>2</sub></small> at a rate of 1 L min<small><sup>−1</sup></small>. Additionally, pure CO<small><sub>2</sub></small> injection was conducted to maintain the pH below the set point of 8.5. This kind of pH-based CO<small><sub>2</sub></small> feeding strategy ensures that the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b811937f')">cells</a></span> will have sufficient carbon source for <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span> and/or for <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production. The agitation speed was set to 300 rpm. The condenser on the off-gas port was chilled to 8 °C <span class="italic">via</span> a thermo circulating <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> bath (C10-K20, Haake, Berlin, Germany). <span class="AN"><a class="simple" title="Gene Ontology link for: Fermentations" href="javascript:popupOBO('GO:0006113','b811937f')">Fermentations</a></span> were maintained for eight days, with sampling every eight hours. For sampling, ∼15 ml aliquot was drawn from the reactor <span class="italic">via</span> the sampling port, discarded to clear the harvesting downtube, and a second ∼15 ml aliquot, the sample, was drawn. From each sample OD<small><sub>730</sub></small> was recorded and an aliquot was taken for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration assay.</span>
      
      
        <h3 id="sect1061"><span class="b_heading"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">Ethanol</a></span> concentration assay</span></h3>
        <span>For determination of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration of a liquid culture, a 550 µl aliquot was taken, spun down at 12<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>100 × <span class="italic">g</span> for 5 min, and 500 µL (or other appropriate volume) of the supernatant was placed in a fresh 1.5 mL tube and stored at −20 °C until assay was performed. Given the linear range of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrophotometer" href="javascript:popupOBO('CMO:0002232','b811937f')">spectrophotometer</a></span> and the sensitivity of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> assay, dilution of the sample (up to 20 fold) was required during the later phase of <span class="italic">Synechocystis</span> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b811937f')">cell</a></span> cultures. In this case, an appropriate volume of BG-11 was first added to the fresh 1.5 mL tube, to which the required volume of clarified supernatant was added. This solution was used directly in the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> assay. Upon removal from −20 °C and immediately before performing the assay, the samples were spun down a second time at 12<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>100 × <span class="italic">g</span> for 5 minutes, also assisting in sample thawing. The Boehringer Mannheim/r-biopharm<small><sup>®</sup></small> (Darmstadt, Germany) enzymatic <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> detection kit was used for <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> concentration determination. For the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> assays, BG-11 medium was used as the blank.</span>
      
    
    
      <h2 id="sect1079"><span class="a_heading">Conclusions</span></h2>
      <span>The above results indicate that it is feasible to create and characterize <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> producing mutants using the light-driven psbA2 promoter system to express the <span class="italic">pdc</span>/<span class="italic">adh</span> <span class="AN"><a class="simple" title="Sequence Ontology link for: gene cassette" href="javascript:popupOBO('SO:0005853','b811937f')">gene cassette</a></span> in <span class="italic">Synechocystis</span> strains. Under defined circumstances <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> will be accumulated in the liquid media at a concentration on the order of 10 mM approximately 6 days after inoculation. During the log phase of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b811937f')">growth</a></span>, we observe <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> yields of 5.2 mmol OD<small><sub>730</sub></small> unit<small><sup>−1</sup></small> litre<small><sup>−1</sup></small> day<small><sup>−1</sup></small>.</span>
      <p class="otherpara">Production of <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> for <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> is one of the nation's and the world's major initiatives. Currently most of the bioethanol production is from the <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b811937f')">fermentation</a></span> of agricultural crops and residues. It has been argued that the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production approaches which primarily use agricultural crops or residues as a feedstock are resource costly. Novel biological solutions for the production of biofuels with the potential to revolutionize the biology-based energy production need to be developed. We have successfully established a foundation for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: ethanol" href="javascript:popupOBO('CHEBI:23982','b811937f','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23982')">ethanol</a></span> production in cyanobacteria under a photoautotrophic metabolic mode. This bioprocess has the potential to scale up for both bioenergy production and CO<small><sub>2</sub></small> utilization. The latter would prevent carbon dioxide release into the atmosphere, where it has been implicated as a major cause of global warming.</p>
    
  
    <h2 id="sect1107"><span class="a_heading">Acknowledgements</span></h2>
      <span>Plasmid PSBAIIKS was obtained from Dr Wim Vermaas at Arizona State University. Plasmid LOI295, containing the Z. mobilis <span class="italic">pdc</span> and <span class="italic">adhII</span> genes, was obtained from Dr Lonnie Ingram at the University of Florida. The authors are grateful to the two anonymous reviewers for their critical and constructive comments that have improved this manuscript.</span>
    
    <span id="sect1111"><h2 id="sect1106"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">Y. Sun and J. Cheng, Hydrolysis of lignocellulosic materials for ethanol production: a review, <span class="italic">Bioresour. Technol.</span>, 2002, <span class="bold">83</span>, 1–11 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0960-8524(01)00212-7" title="DOI Link to resource 10.1016/S0960-8524(01)00212-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38Xptleluw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit2">D. Pimentel and T. W. Patzek, Ethanol Production Using Corn, Switchgrass and Wood; Biodiesel Production Using Soybean and Sunflower, <span class="italic">Natural Resources Research</span>, 2005, <span class="bold">14</span>(1), 65–76 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Natural%20Resources%20Research%5BJour%5D%20AND%2014%5Bvolume%5D%20AND%2065%5Bpage%5D%20and%202005%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit3">T. W. Patzek, A First-Law Thermodynamic Analysis of the Corn- Ethanol Cycle, <span class="italic">Natural Resources Research</span>, 2007, <span class="bold">15</span>(4), 255–270 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Natural%20Resources%20Research%5BJour%5D%20AND%2015%5Bvolume%5D%20AND%20255%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit4">M. D. Deng and J. R. Coleman, Ethanol synthesis by genetic engineering in cyanobacteria, <span class="italic">Appl. Environ. Microbiol.</span>, 1999, <span class="bold">65</span>(2), 523–8 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BD2critVKgsA%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit5">L. O. Ingram, T. Conway, D. P. Clark, G. W. Sewell and J. F. Preston, Genetic engineering of ethanol production in <span class="italic">Escherichia coli</span>, <span class="italic">Appl. Environ. Microbiol.</span>, 1987, <span class="bold">53</span>(10), 2420–2425 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2sXmtFaqsLc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit6">L. O. Ingram, P. F. Gomez, X. Lai, M. Moniruzzaman, B. E. Wood, L. P. Yomano and S. W. York, Metabolic engineering of bacteria for ethanol production, <span class="italic">Biotechnol. Bioeng.</span>, 1998, <span class="bold">58</span>(2–3), 204–214 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1097-0290(19980420)58:2/3&lt;204::AID-BIT13&gt;3.0.CO;2-C" title="DOI Link to resource 10.1002/(SICI)1097-0290(19980420)58:2/3&lt;204::AID-BIT13&gt;3.0.CO;2-C">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXht12ks70%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit7">
          R. P. Woods, J. R. Coleman, M. de Deng, Genetically modified cyanobacteria for the production of ethanol, the constructs and method thereof, <span class="italic">US Patent</span> 6699696,  2004 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=US%20Patent%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit8">M. Ikeuchi, Complete genome sequence of a cyanobacterium <span class="italic">Synechocystis</span> sp. PCC 6803, the oxygenic photosynthetic prokaryote, <span class="italic">Tanpakushitsu Kakusan Koso</span>, 1996, <span class="bold">41</span>(16), 2579–2583 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Tanpakushitsu%20Kakusan%20Koso%5BJour%5D%20AND%2041%5Bvolume%5D%20AND%202579%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit9">S. Aoki, T. Kondo and M. Ishiura, A promoter-trap vector for clock-controlled genes in the cyanobacterium <span class="italic">Synechocystis</span> sp. PCC 6803, <span class="italic">J. Microbiol. Methods</span>, 2002, <span class="bold">49</span>(3), 265–274 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0167-7012(01)00376-1" title="DOI Link to resource 10.1016/S0167-7012(01)00376-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XhsFalu70%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit10">W. F. J. Vermaas, J. G. K. Williams, A. W. Rutherford, P. Mathis and C. J. Arntzen, Genetically engineered mutant of the cyanobacterium <span class="italic">Synechocystis</span> 6803 lacks the photosystem II chlorophyll-binding protein CP-47, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 1986, <span class="bold">83</span>(24), 9474–9477 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.83.24.9474" title="DOI Link to resource 10.1073/pnas.83.24.9474">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2sXotFOltw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit11">J. G. K. Williams, Construction of Specific Mutations in Photosystem II Phosynthetic Reaction Center by Genetic Engineering Methods in <span class="italic">Synechocystis</span> 6803, <span class="italic">Methods Enzymol.</span>, 1988, <span class="bold">167</span>, 766–778 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Methods%20Enzymol.%5BJour%5D%20AND%20167%5Bvolume%5D%20AND%20766%5Bpage%5D%20and%201988%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit12">D. Lagarde, L. Beuf and L. W. F. J. Vermaas, Increased production of zeaxanthin and other pigments by application of genetic engineering techniques to <span class="italic"><span class="italic">Synechocystis</span></span> sp. strain PCC 6803, <span class="italic">Appl. Environ. Microbiol.</span>, 2000, <span class="bold">66</span>(1), 64–72 <a target="_blank" class="DOILink" href="https://doi.org/10.1128/AEM.66.1.64-72.2000" title="DOI Link to resource 10.1128/AEM.66.1.64-72.2000">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXktlSgug%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit13">
          R. P. Woods, J. R. Coleman, M. de Deng, Genetically modified cyanobacteria for the production of ethanol, the constructs and method thereof, <span class="italic">US Patent</span> 63066639,  2001 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=US%20Patent%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202001%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit14">L. O. Ingram and T. Conway, Expression of Different Levels of Ethanologenic Enzymes from <span class="italic">Zymomonas mobilis</span> in Recombinant Strains of Escherichia coli, <span class="italic">Appl. Environ. Microbiol.</span>, 1988, <span class="bold">54</span>(2), 397–404 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL1cXht1GrsL4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit15">H. Alper, J. Moxley, E. Nevoigt, G. R. Fink and G. Stephanopoulos, Engineering yeast transcription machinery for improved ethanol tolerance and production, <span class="italic">Science</span>, 2006, <span class="bold">314</span>(5805), 1565–1568 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1131969" title="DOI Link to resource 10.1126/science.1131969">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xht1OntL%252FP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit16">C. Kemmer and P. Neubauer, Antisense RNA based down-regulation of RNaseE in E.coli, <span class="italic">Microb. Cell Fact.</span>, 2006, <span class="bold">5</span>, 38–49 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/1475-2859-5-38" title="DOI Link to resource 10.1186/1475-2859-5-38">CrossRef</a>.</span></li>
      <li><span id="cit17">S. Kinoshita, T. Kakizono, K. Kadota, K. Das and H. Taguchi, Purification of two alcohol dehydrogenases from <span class="italic">Zymomonas mobilis</span> and their properties, <span class="italic">Appl. Microbiol. Biotechnol.</span>, 1985, <span class="bold">22</span>, 249–254 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2MXlslCntLg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit18">T. Conway, G. W. Sewell, Y. A. Osman and L. O. Ingram, Cloning and sequencing of the alcohol dehydrogenase II gene from <span class="italic">Zymomonas mobilis</span>, <span class="italic">J. Bacteriol.</span>, 1987, <span class="bold">169</span>(6), 2591–2597 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2sXlsVyhtrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit19">P. Gunasekaran and K. Chandra Raj, Ethanol fermentation technology – <span class="italic">Zymomonas mobilis</span>, <span class="italic">Curr. Sci.</span>, 1999, <span class="bold">77</span>(1), 56 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1MXltVKnt7k%253D" title="Link to resource in CAS">CAS</a>.</span></li>
    </ol>
    
    
    
    
    
  <hr/><table><tr><td><h3>Footnote</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">Electronic supplementary information (ESI) available: Appendices I, II and III. See DOI: <a href="http://xlink.rsc.org/?DOI=b811937f" target="_blank" title="Select to resolve DOI: 10.1039/b811937f">10.1039/b811937f</a></span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2009</b></td></tr></table></div></div></div></body><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>