<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Characterization of aluminum, aluminum oxide and titanium dioxide nanomaterials using a combination of methods for particle surface and size analysis  ...  - RSC Advances (RSC Publishing) DOI:10.1039/C8RA00205C</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2018/ra/c8ra00205c"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":84,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};;/*! For license information please see nr-loader-spa-1.236.0.min.js.LICENSE.txt */
(()=>{"use strict";var e,t,r={5763:(e,t,r)=>{r.d(t,{P_:()=>l,Mt:()=>g,C5:()=>s,DL:()=>v,OP:()=>T,lF:()=>D,Yu:()=>y,Dg:()=>h,CX:()=>c,GE:()=>b,sU:()=>_});var n=r(8632),i=r(9567);const o={beacon:n.ce.beacon,errorBeacon:n.ce.errorBeacon,licenseKey:void 0,applicationID:void 0,sa:void 0,queueTime:void 0,applicationTime:void 0,ttGuid:void 0,user:void 0,account:void 0,product:void 0,extra:void 0,jsAttributes:{},userAttributes:void 0,atts:void 0,transactionName:void 0,tNamePlain:void 0},a={};function s(e){if(!e)throw new Error("All info objects require an agent identifier!");if(!a[e])throw new Error("Info for ".concat(e," was never set"));return a[e]}function c(e,t){if(!e)throw new Error("All info objects require an agent identifier!");a[e]=(0,i.D)(t,o),(0,n.Qy)(e,a[e],"info")}var u=r(7056);const d=()=>{const e={blockSelector:"[data-nr-block]",maskInputOptions:{password:!0}};return{allow_bfcache:!0,privacy:{cookies_enabled:!0},ajax:{deny_list:void 0,enabled:!0,harvestTimeSeconds:10},distributed_tracing:{enabled:void 0,exclude_newrelic_header:void 0,cors_use_newrelic_header:void 0,cors_use_tracecontext_headers:void 0,allowed_origins:void 0},session:{domain:void 0,expiresMs:u.oD,inactiveMs:u.Hb},ssl:void 0,obfuscate:void 0,jserrors:{enabled:!0,harvestTimeSeconds:10},metrics:{enabled:!0},page_action:{enabled:!0,harvestTimeSeconds:30},page_view_event:{enabled:!0},page_view_timing:{enabled:!0,harvestTimeSeconds:30,long_task:!1},session_trace:{enabled:!0,harvestTimeSeconds:10},harvest:{tooManyRequestsDelay:60},session_replay:{enabled:!1,harvestTimeSeconds:60,sampleRate:.1,errorSampleRate:.1,maskTextSelector:"*",maskAllInputs:!0,get blockClass(){return"nr-block"},get ignoreClass(){return"nr-ignore"},get maskTextClass(){return"nr-mask"},get blockSelector(){return e.blockSelector},set blockSelector(t){e.blockSelector+=",".concat(t)},get maskInputOptions(){return e.maskInputOptions},set maskInputOptions(t){e.maskInputOptions={...t,password:!0}}},spa:{enabled:!0,harvestTimeSeconds:10}}},f={};function l(e){if(!e)throw new Error("All configuration objects require an agent identifier!");if(!f[e])throw new Error("Configuration for ".concat(e," was never set"));return f[e]}function h(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");f[e]=(0,i.D)(t,d()),(0,n.Qy)(e,f[e],"config")}function g(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");var r=l(e);if(r){for(var n=t.split("."),i=0;i<n.length-1;i++)if("object"!=typeof(r=r[n[i]]))return;r=r[n[n.length-1]]}return r}const p={accountID:void 0,trustKey:void 0,agentID:void 0,licenseKey:void 0,applicationID:void 0,xpid:void 0},m={};function v(e){if(!e)throw new Error("All loader-config objects require an agent identifier!");if(!m[e])throw new Error("LoaderConfig for ".concat(e," was never set"));return m[e]}function b(e,t){if(!e)throw new Error("All loader-config objects require an agent identifier!");m[e]=(0,i.D)(t,p),(0,n.Qy)(e,m[e],"loader_config")}const y=(0,n.mF)().o;var w=r(385),x=r(6818);const A={buildEnv:x.Re,bytesSent:{},queryBytesSent:{},customTransaction:void 0,disabled:!1,distMethod:x.gF,isolatedBacklog:!1,loaderType:void 0,maxBytes:3e4,offset:Math.floor(w._A?.performance?.timeOrigin||w._A?.performance?.timing?.navigationStart||Date.now()),onerror:void 0,origin:""+w._A.location,ptid:void 0,releaseIds:{},session:void 0,xhrWrappable:"function"==typeof w._A.XMLHttpRequest?.prototype?.addEventListener,version:x.q4},E={};function T(e){if(!e)throw new Error("All runtime objects require an agent identifier!");if(!E[e])throw new Error("Runtime for ".concat(e," was never set"));return E[e]}function _(e,t){if(!e)throw new Error("All runtime objects require an agent identifier!");E[e]=(0,i.D)(t,A),(0,n.Qy)(e,E[e],"runtime")}function D(e){return function(e){try{const t=s(e);return!!t.licenseKey&&!!t.errorBeacon&&!!t.applicationID}catch(e){return!1}}(e)}},9567:(e,t,r)=>{r.d(t,{D:()=>i});var n=r(50);function i(e,t){try{if(!e||"object"!=typeof e)return(0,n.Z)("Setting a Configurable requires an object as input");if(!t||"object"!=typeof t)return(0,n.Z)("Setting a Configurable requires a model to set its initial properties");const r=Object.create(Object.getPrototypeOf(t),Object.getOwnPropertyDescriptors(t)),o=0===Object.keys(r).length?e:r;for(let a in o)if(void 0!==e[a])try{"object"==typeof e[a]&&"object"==typeof t[a]?r[a]=i(e[a],t[a]):r[a]=e[a]}catch(e){(0,n.Z)("An error occurred while setting a property of a Configurable",e)}return r}catch(e){(0,n.Z)("An error occured while setting a Configurable",e)}}},6818:(e,t,r)=>{r.d(t,{Re:()=>i,gF:()=>o,q4:()=>n});const n="1.236.0",i="PROD",o="CDN"},385:(e,t,r)=>{r.d(t,{FN:()=>a,IF:()=>u,Nk:()=>f,Tt:()=>s,_A:()=>o,il:()=>n,pL:()=>c,v6:()=>i,w1:()=>d});const n="undefined"!=typeof window&&!!window.document,i="undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self.navigator instanceof WorkerNavigator||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis.navigator instanceof WorkerNavigator),o=n?window:"undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis),a=""+o?.location,s=/iPad|iPhone|iPod/.test(navigator.userAgent),c=s&&"undefined"==typeof SharedWorker,u=(()=>{const e=navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);return Array.isArray(e)&&e.length>=2?+e[1]:0})(),d=Boolean(n&&window.document.documentMode),f=!!navigator.sendBeacon},1117:(e,t,r)=>{r.d(t,{w:()=>o});var n=r(50);const i={agentIdentifier:"",ee:void 0};class o{constructor(e){try{if("object"!=typeof e)return(0,n.Z)("shared context requires an object as input");this.sharedContext={},Object.assign(this.sharedContext,i),Object.entries(e).forEach((e=>{let[t,r]=e;Object.keys(i).includes(t)&&(this.sharedContext[t]=r)}))}catch(e){(0,n.Z)("An error occured while setting SharedContext",e)}}}},8e3:(e,t,r)=>{r.d(t,{L:()=>d,R:()=>c});var n=r(2177),i=r(1284),o=r(4322),a=r(3325);const s={};function c(e,t){const r={staged:!1,priority:a.p[t]||0};u(e),s[e].get(t)||s[e].set(t,r)}function u(e){e&&(s[e]||(s[e]=new Map))}function d(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"feature";if(u(e),!e||!s[e].get(t))return a(t);s[e].get(t).staged=!0;const r=[...s[e]];function a(t){const r=e?n.ee.get(e):n.ee,a=o.X.handlers;if(r.backlog&&a){var s=r.backlog[t],c=a[t];if(c){for(var u=0;s&&u<s.length;++u)f(s[u],c);(0,i.D)(c,(function(e,t){(0,i.D)(t,(function(t,r){r[0].on(e,r[1])}))}))}delete a[t],r.backlog[t]=null,r.emit("drain-"+t,[])}}r.every((e=>{let[t,r]=e;return r.staged}))&&(r.sort(((e,t)=>e[1].priority-t[1].priority)),r.forEach((e=>{let[t]=e;a(t)})))}function f(e,t){var r=e[1];(0,i.D)(t[r],(function(t,r){var n=e[0];if(r[0]===n){var i=r[1],o=e[3],a=e[2];i.apply(o,a)}}))}},2177:(e,t,r)=>{r.d(t,{c:()=>f,ee:()=>u});var n=r(8632),i=r(2210),o=r(1284),a=r(5763),s="nr@context";let c=(0,n.fP)();var u;function d(){}function f(e){return(0,i.X)(e,s,l)}function l(){return new d}function h(){u.aborted=!0,u.backlog={}}c.ee?u=c.ee:(u=function e(t,r){var n={},c={},f={},g=!1;try{g=16===r.length&&(0,a.OP)(r).isolatedBacklog}catch(e){}var p={on:b,addEventListener:b,removeEventListener:y,emit:v,get:x,listeners:w,context:m,buffer:A,abort:h,aborted:!1,isBuffering:E,debugId:r,backlog:g?{}:t&&"object"==typeof t.backlog?t.backlog:{}};return p;function m(e){return e&&e instanceof d?e:e?(0,i.X)(e,s,l):l()}function v(e,r,n,i,o){if(!1!==o&&(o=!0),!u.aborted||i){t&&o&&t.emit(e,r,n);for(var a=m(n),s=w(e),d=s.length,f=0;f<d;f++)s[f].apply(a,r);var l=T()[c[e]];return l&&l.push([p,e,r,a]),a}}function b(e,t){n[e]=w(e).concat(t)}function y(e,t){var r=n[e];if(r)for(var i=0;i<r.length;i++)r[i]===t&&r.splice(i,1)}function w(e){return n[e]||[]}function x(t){return f[t]=f[t]||e(p,t)}function A(e,t){var r=T();p.aborted||(0,o.D)(e,(function(e,n){t=t||"feature",c[n]=t,t in r||(r[t]=[])}))}function E(e){return!!T()[c[e]]}function T(){return p.backlog}}(void 0,"globalEE"),c.ee=u)},5546:(e,t,r)=>{r.d(t,{E:()=>n,p:()=>i});var n=r(2177).ee.get("handle");function i(e,t,r,i,o){o?(o.buffer([e],i),o.emit(e,t,r)):(n.buffer([e],i),n.emit(e,t,r))}},4322:(e,t,r)=>{r.d(t,{X:()=>o});var n=r(5546);o.on=a;var i=o.handlers={};function o(e,t,r,o){a(o||n.E,i,e,t,r)}function a(e,t,r,i,o){o||(o="feature"),e||(e=n.E);var a=t[o]=t[o]||{};(a[r]=a[r]||[]).push([e,i])}},3239:(e,t,r)=>{r.d(t,{bP:()=>s,iz:()=>c,m$:()=>a});var n=r(385);let i=!1,o=!1;try{const e={get passive(){return i=!0,!1},get signal(){return o=!0,!1}};n._A.addEventListener("test",null,e),n._A.removeEventListener("test",null,e)}catch(e){}function a(e,t){return i||o?{capture:!!e,passive:i,signal:t}:!!e}function s(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;window.addEventListener(e,t,a(r,n))}function c(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;document.addEventListener(e,t,a(r,n))}},4402:(e,t,r)=>{r.d(t,{Ht:()=>u,M:()=>c,Rl:()=>a,ky:()=>s});var n=r(385);const i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";function o(e,t){return e?15&e[t]:16*Math.random()|0}function a(){const e=n._A?.crypto||n._A?.msCrypto;let t,r=0;return e&&e.getRandomValues&&(t=e.getRandomValues(new Uint8Array(31))),i.split("").map((e=>"x"===e?o(t,++r).toString(16):"y"===e?(3&o()|8).toString(16):e)).join("")}function s(e){const t=n._A?.crypto||n._A?.msCrypto;let r,i=0;t&&t.getRandomValues&&(r=t.getRandomValues(new Uint8Array(31)));const a=[];for(var s=0;s<e;s++)a.push(o(r,++i).toString(16));return a.join("")}function c(){return s(16)}function u(){return s(32)}},7056:(e,t,r)=>{r.d(t,{Bq:()=>n,Hb:()=>o,oD:()=>i});const n="NRBA",i=144e5,o=18e5},7894:(e,t,r)=>{function n(){return Math.round(performance.now())}r.d(t,{z:()=>n})},7243:(e,t,r)=>{r.d(t,{e:()=>o});var n=r(385),i={};function o(e){if(e in i)return i[e];if(0===(e||"").indexOf("data:"))return{protocol:"data"};let t;var r=n._A?.location,o={};if(n.il)t=document.createElement("a"),t.href=e;else try{t=new URL(e,r.href)}catch(e){return o}o.port=t.port;var a=t.href.split("://");!o.port&&a[1]&&(o.port=a[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===a[0]?"443":"80"),o.hostname=t.hostname||r.hostname,o.pathname=t.pathname,o.protocol=a[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var s=!t.protocol||":"===t.protocol||t.protocol===r.protocol,c=t.hostname===r.hostname&&t.port===r.port;return o.sameOrigin=s&&(!t.hostname||c),"/"===o.pathname&&(i[e]=o),o}},50:(e,t,r)=>{function n(e,t){"function"==typeof console.warn&&(console.warn("New Relic: ".concat(e)),t&&console.warn(t))}r.d(t,{Z:()=>n})},2587:(e,t,r)=>{r.d(t,{N:()=>c,T:()=>u});var n=r(2177),i=r(5546),o=r(8e3),a=r(3325);const s={stn:[a.D.sessionTrace],err:[a.D.jserrors,a.D.metrics],ins:[a.D.pageAction],spa:[a.D.spa],sr:[a.D.sessionReplay,a.D.sessionTrace]};function c(e,t){const r=n.ee.get(t);e&&"object"==typeof e&&(Object.entries(e).forEach((e=>{let[t,n]=e;void 0===u[t]&&(s[t]?s[t].forEach((e=>{n?(0,i.p)("feat-"+t,[],void 0,e,r):(0,i.p)("block-"+t,[],void 0,e,r),(0,i.p)("rumresp-"+t,[Boolean(n)],void 0,e,r)})):n&&(0,i.p)("feat-"+t,[],void 0,void 0,r),u[t]=Boolean(n))})),Object.keys(s).forEach((e=>{void 0===u[e]&&(s[e]?.forEach((t=>(0,i.p)("rumresp-"+e,[!1],void 0,t,r))),u[e]=!1)})),(0,o.L)(t,a.D.pageViewEvent))}const u={}},2210:(e,t,r)=>{r.d(t,{X:()=>i});var n=Object.prototype.hasOwnProperty;function i(e,t,r){if(n.call(e,t))return e[t];var i=r();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,t,{value:i,writable:!0,enumerable:!1}),i}catch(e){}return e[t]=i,i}},1284:(e,t,r)=>{r.d(t,{D:()=>n});const n=(e,t)=>Object.entries(e||{}).map((e=>{let[r,n]=e;return t(r,n)}))},4351:(e,t,r)=>{r.d(t,{P:()=>o});var n=r(2177);const i=()=>{const e=new WeakSet;return(t,r)=>{if("object"==typeof r&&null!==r){if(e.has(r))return;e.add(r)}return r}};function o(e){try{return JSON.stringify(e,i())}catch(e){try{n.ee.emit("internal-error",[e])}catch(e){}}}},3960:(e,t,r)=>{r.d(t,{K:()=>a,b:()=>o});var n=r(3239);function i(){return"undefined"==typeof document||"complete"===document.readyState}function o(e,t){if(i())return e();(0,n.bP)("load",e,t)}function a(e){if(i())return e();(0,n.iz)("DOMContentLoaded",e)}},8632:(e,t,r)=>{r.d(t,{EZ:()=>u,Qy:()=>c,ce:()=>o,fP:()=>a,gG:()=>d,mF:()=>s});var n=r(7894),i=r(385);const o={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net"};function a(){return i._A.NREUM||(i._A.NREUM={}),void 0===i._A.newrelic&&(i._A.newrelic=i._A.NREUM),i._A.NREUM}function s(){let e=a();return e.o||(e.o={ST:i._A.setTimeout,SI:i._A.setImmediate,CT:i._A.clearTimeout,XHR:i._A.XMLHttpRequest,REQ:i._A.Request,EV:i._A.Event,PR:i._A.Promise,MO:i._A.MutationObserver,FETCH:i._A.fetch}),e}function c(e,t,r){let i=a();const o=i.initializedAgents||{},s=o[e]||{};return Object.keys(s).length||(s.initializedAt={ms:(0,n.z)(),date:new Date}),i.initializedAgents={...o,[e]:{...s,[r]:t}},i}function u(e,t){a()[e]=t}function d(){return function(){let e=a();const t=e.info||{};e.info={beacon:o.beacon,errorBeacon:o.errorBeacon,...t}}(),function(){let e=a();const t=e.init||{};e.init={...t}}(),s(),function(){let e=a();const t=e.loader_config||{};e.loader_config={...t}}(),a()}},7956:(e,t,r)=>{r.d(t,{N:()=>i});var n=r(3239);function i(e){let t=arguments.length>1&&void 0!==arguments[1]&&arguments[1],r=arguments.length>2?arguments[2]:void 0,i=arguments.length>3?arguments[3]:void 0;return void(0,n.iz)("visibilitychange",(function(){if(t)return void("hidden"==document.visibilityState&&e());e(document.visibilityState)}),r,i)}},1214:(e,t,r)=>{r.d(t,{em:()=>v,u5:()=>N,QU:()=>S,_L:()=>I,Gm:()=>L,Lg:()=>M,gy:()=>U,BV:()=>Q,Kf:()=>ee});var n=r(2177);const i="nr@original";var o=Object.prototype.hasOwnProperty,a=!1;function s(e,t){return e||(e=n.ee),r.inPlace=function(e,t,n,i,o){n||(n="");var a,s,c,u="-"===n.charAt(0);for(c=0;c<t.length;c++)d(a=e[s=t[c]])||(e[s]=r(a,u?s+n:n,i,s,o))},r.flag=i,r;function r(t,r,n,a,s){return d(t)?t:(r||(r=""),nrWrapper[i]=t,u(t,nrWrapper,e),nrWrapper);function nrWrapper(){var i,u,d,f;try{u=this,i=[...arguments],d="function"==typeof n?n(i,u):n||{}}catch(t){c([t,"",[i,u,a],d],e)}o(r+"start",[i,u,a],d,s);try{return f=t.apply(u,i)}catch(e){throw o(r+"err",[i,u,e],d,s),e}finally{o(r+"end",[i,u,f],d,s)}}}function o(r,n,i,o){if(!a||t){var s=a;a=!0;try{e.emit(r,n,i,t,o)}catch(t){c([t,r,n,i],e)}a=s}}}function c(e,t){t||(t=n.ee);try{t.emit("internal-error",e)}catch(e){}}function u(e,t,r){if(Object.defineProperty&&Object.keys)try{return Object.keys(e).forEach((function(r){Object.defineProperty(t,r,{get:function(){return e[r]},set:function(t){return e[r]=t,t}})})),t}catch(e){c([e],r)}for(var n in e)o.call(e,n)&&(t[n]=e[n]);return t}function d(e){return!(e&&e instanceof Function&&e.apply&&!e[i])}var f=r(2210),l=r(385);const h={},g=XMLHttpRequest,p="addEventListener",m="removeEventListener";function v(e){var t=function(e){return(e||n.ee).get("events")}(e);if(h[t.debugId]++)return t;h[t.debugId]=1;var r=s(t,!0);function i(e){r.inPlace(e,[p,m],"-",o)}function o(e,t){return e[1]}return"getPrototypeOf"in Object&&(l.il&&b(document,i),b(l._A,i),b(g.prototype,i)),t.on(p+"-start",(function(e,t){var n=e[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var i=(0,f.X)(n,"nr@wrapped",(function(){var e={object:function(){if("function"!=typeof n.handleEvent)return;return n.handleEvent.apply(n,arguments)},function:n}[typeof n];return e?r(e,"fn-",null,e.name||"anonymous"):n}));this.wrapped=e[1]=i}})),t.on(m+"-start",(function(e){e[1]=this.wrapped||e[1]})),t}function b(e,t){let r=e;for(;"object"==typeof r&&!Object.prototype.hasOwnProperty.call(r,p);)r=Object.getPrototypeOf(r);for(var n=arguments.length,i=new Array(n>2?n-2:0),o=2;o<n;o++)i[o-2]=arguments[o];r&&t(r,...i)}var y="fetch-",w=y+"body-",x=["arrayBuffer","blob","json","text","formData"],A=l._A.Request,E=l._A.Response,T="prototype",_="nr@context";const D={};function N(e){const t=function(e){return(e||n.ee).get("fetch")}(e);if(!(A&&E&&l._A.fetch))return t;if(D[t.debugId]++)return t;function r(e,r,n){var i=e[r];"function"==typeof i&&(e[r]=function(){var e,r=[...arguments],o={};t.emit(n+"before-start",[r],o),o[_]&&o[_].dt&&(e=o[_].dt);var a=i.apply(this,r);return t.emit(n+"start",[r,e],a),a.then((function(e){return t.emit(n+"end",[null,e],a),e}),(function(e){throw t.emit(n+"end",[e],a),e}))})}return D[t.debugId]=1,x.forEach((e=>{r(A[T],e,w),r(E[T],e,w)})),r(l._A,"fetch",y),t.on(y+"end",(function(e,r){var n=this;if(r){var i=r.headers.get("content-length");null!==i&&(n.rxSize=i),t.emit(y+"done",[null,r],n)}else t.emit(y+"done",[e],n)})),t}const O={},j=["pushState","replaceState"];function S(e){const t=function(e){return(e||n.ee).get("history")}(e);return!l.il||O[t.debugId]++||(O[t.debugId]=1,s(t).inPlace(window.history,j,"-")),t}var P=r(3239);const C={},R=["appendChild","insertBefore","replaceChild"];function I(e){const t=function(e){return(e||n.ee).get("jsonp")}(e);if(!l.il||C[t.debugId])return t;C[t.debugId]=!0;var r=s(t),i=/[?&](?:callback|cb)=([^&#]+)/,o=/(.*)\.([^.]+)/,a=/^(\w+)(\.|$)(.*)$/;function c(e,t){var r=e.match(a),n=r[1],i=r[3];return i?c(i,t[n]):t[n]}return r.inPlace(Node.prototype,R,"dom-"),t.on("dom-start",(function(e){!function(e){if(!e||"string"!=typeof e.nodeName||"script"!==e.nodeName.toLowerCase())return;if("function"!=typeof e.addEventListener)return;var n=(a=e.src,s=a.match(i),s?s[1]:null);var a,s;if(!n)return;var u=function(e){var t=e.match(o);if(t&&t.length>=3)return{key:t[2],parent:c(t[1],window)};return{key:e,parent:window}}(n);if("function"!=typeof u.parent[u.key])return;var d={};function f(){t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}function l(){t.emit("jsonp-error",[],d),t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}r.inPlace(u.parent,[u.key],"cb-",d),e.addEventListener("load",f,(0,P.m$)(!1)),e.addEventListener("error",l,(0,P.m$)(!1)),t.emit("new-jsonp",[e.src],d)}(e[0])})),t}var k=r(5763);const H={};function L(e){const t=function(e){return(e||n.ee).get("mutation")}(e);if(!l.il||H[t.debugId])return t;H[t.debugId]=!0;var r=s(t),i=k.Yu.MO;return i&&(window.MutationObserver=function(e){return this instanceof i?new i(r(e,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype),t}const z={};function M(e){const t=function(e){return(e||n.ee).get("promise")}(e);if(z[t.debugId])return t;z[t.debugId]=!0;var r=n.c,o=s(t),a=k.Yu.PR;return a&&function(){function e(r){var n=t.context(),i=o(r,"executor-",n,null,!1);const s=Reflect.construct(a,[i],e);return t.context(s).getCtx=function(){return n},s}l._A.Promise=e,Object.defineProperty(e,"name",{value:"Promise"}),e.toString=function(){return a.toString()},Object.setPrototypeOf(e,a),["all","race"].forEach((function(r){const n=a[r];e[r]=function(e){let i=!1;[...e||[]].forEach((e=>{this.resolve(e).then(a("all"===r),a(!1))}));const o=n.apply(this,arguments);return o;function a(e){return function(){t.emit("propagate",[null,!i],o,!1,!1),i=i||!e}}}})),["resolve","reject"].forEach((function(r){const n=a[r];e[r]=function(e){const r=n.apply(this,arguments);return e!==r&&t.emit("propagate",[e,!0],r,!1,!1),r}})),e.prototype=a.prototype;const n=a.prototype.then;a.prototype.then=function(){var e=this,i=r(e);i.promise=e;for(var a=arguments.length,s=new Array(a),c=0;c<a;c++)s[c]=arguments[c];s[0]=o(s[0],"cb-",i,null,!1),s[1]=o(s[1],"cb-",i,null,!1);const u=n.apply(this,s);return i.nextPromise=u,t.emit("propagate",[e,!0],u,!1,!1),u},a.prototype.then[i]=n,t.on("executor-start",(function(e){e[0]=o(e[0],"resolve-",this,null,!1),e[1]=o(e[1],"resolve-",this,null,!1)})),t.on("executor-err",(function(e,t,r){e[1](r)})),t.on("cb-end",(function(e,r,n){t.emit("propagate",[n,!0],this.nextPromise,!1,!1)})),t.on("propagate",(function(e,r,n){this.getCtx&&!r||(this.getCtx=function(){if(e instanceof Promise)var r=t.context(e);return r&&r.getCtx?r.getCtx():this})}))}(),t}const B={},F="requestAnimationFrame";function U(e){const t=function(e){return(e||n.ee).get("raf")}(e);if(!l.il||B[t.debugId]++)return t;B[t.debugId]=1;var r=s(t);return r.inPlace(window,[F],"raf-"),t.on("raf-start",(function(e){e[0]=r(e[0],"fn-")})),t}const q={},G="setTimeout",V="setInterval",X="clearTimeout",W="-start",Z="-",$=[G,"setImmediate",V,X,"clearImmediate"];function Q(e){const t=function(e){return(e||n.ee).get("timer")}(e);if(q[t.debugId]++)return t;q[t.debugId]=1;var r=s(t);return r.inPlace(l._A,$.slice(0,2),G+Z),r.inPlace(l._A,$.slice(2,3),V+Z),r.inPlace(l._A,$.slice(3),X+Z),t.on(V+W,(function(e,t,n){e[0]=r(e[0],"fn-",null,n)})),t.on(G+W,(function(e,t,n){this.method=n,this.timerDuration=isNaN(e[1])?0:+e[1],e[0]=r(e[0],"fn-",this,n)})),t}var Y=r(50);const K={},J=["open","send"];function ee(e){var t=e||n.ee;const r=function(e){return(e||n.ee).get("xhr")}(t);if(K[r.debugId]++)return r;K[r.debugId]=1,v(t);var i=s(r),o=k.Yu.XHR,a=k.Yu.MO,c=k.Yu.PR,u=k.Yu.SI,d="readystatechange",f=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],h=[],g=l._A.XMLHttpRequest.listeners,p=l._A.XMLHttpRequest=function(e){var t=new o(e);function n(){try{r.emit("new-xhr",[t],t),t.addEventListener(d,b,(0,P.m$)(!1))}catch(e){(0,Y.Z)("An error occured while intercepting XHR",e);try{r.emit("internal-error",[e])}catch(e){}}}return this.listeners=g?[...g,n]:[n],this.listeners.forEach((e=>e())),t};function m(e,t){i.inPlace(t,["onreadystatechange"],"fn-",E)}function b(){var e=this,t=r.context(e);e.readyState>3&&!t.resolved&&(t.resolved=!0,r.emit("xhr-resolved",[],e)),i.inPlace(e,f,"fn-",E)}if(function(e,t){for(var r in e)t[r]=e[r]}(o,p),p.prototype=o.prototype,i.inPlace(p.prototype,J,"-xhr-",E),r.on("send-xhr-start",(function(e,t){m(e,t),function(e){h.push(e),a&&(y?y.then(A):u?u(A):(w=-w,x.data=w))}(t)})),r.on("open-xhr-start",m),a){var y=c&&c.resolve();if(!u&&!c){var w=1,x=document.createTextNode(w);new a(A).observe(x,{characterData:!0})}}else t.on("fn-end",(function(e){e[0]&&e[0].type===d||A()}));function A(){for(var e=0;e<h.length;e++)m(0,h[e]);h.length&&(h=[])}function E(e,t){return t}return r}},7825:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.ajax},6660:(e,t,r)=>{r.d(t,{A:()=>i,t:()=>n});const n=r(3325).D.jserrors,i="nr@seenError"},3081:(e,t,r)=>{r.d(t,{gF:()=>o,mY:()=>i,t9:()=>n,vz:()=>s,xS:()=>a});const n=r(3325).D.metrics,i="sm",o="cm",a="storeSupportabilityMetrics",s="storeEventMetrics"},4649:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageAction},7633:(e,t,r)=>{r.d(t,{Dz:()=>i,OJ:()=>a,qw:()=>o,t9:()=>n});const n=r(3325).D.pageViewEvent,i="firstbyte",o="domcontent",a="windowload"},9251:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageViewTiming},3614:(e,t,r)=>{r.d(t,{BST_RESOURCE:()=>i,END:()=>s,FEATURE_NAME:()=>n,FN_END:()=>u,FN_START:()=>c,PUSH_STATE:()=>d,RESOURCE:()=>o,START:()=>a});const n=r(3325).D.sessionTrace,i="bstResource",o="resource",a="-start",s="-end",c="fn"+a,u="fn"+s,d="pushState"},7836:(e,t,r)=>{r.d(t,{BODY:()=>A,CB_END:()=>E,CB_START:()=>u,END:()=>x,FEATURE_NAME:()=>i,FETCH:()=>_,FETCH_BODY:()=>v,FETCH_DONE:()=>m,FETCH_START:()=>p,FN_END:()=>c,FN_START:()=>s,INTERACTION:()=>l,INTERACTION_API:()=>d,INTERACTION_EVENTS:()=>o,JSONP_END:()=>b,JSONP_NODE:()=>g,JS_TIME:()=>T,MAX_TIMER_BUDGET:()=>a,REMAINING:()=>f,SPA_NODE:()=>h,START:()=>w,originalSetTimeout:()=>y});var n=r(5763);const i=r(3325).D.spa,o=["click","submit","keypress","keydown","keyup","change"],a=999,s="fn-start",c="fn-end",u="cb-start",d="api-ixn-",f="remaining",l="interaction",h="spaNode",g="jsonpNode",p="fetch-start",m="fetch-done",v="fetch-body-",b="jsonp-end",y=n.Yu.ST,w="-start",x="-end",A="-body",E="cb"+x,T="jsTime",_="fetch"},5938:(e,t,r)=>{r.d(t,{W:()=>o});var n=r(5763),i=r(2177);class o{constructor(e,t,r){this.agentIdentifier=e,this.aggregator=t,this.ee=i.ee.get(e,(0,n.OP)(this.agentIdentifier).isolatedBacklog),this.featureName=r,this.blocked=!1}}},9144:(e,t,r)=>{r.d(t,{j:()=>m});var n=r(3325),i=r(5763),o=r(5546),a=r(2177),s=r(7894),c=r(8e3),u=r(3960),d=r(385),f=r(50),l=r(3081),h=r(8632);function g(){const e=(0,h.gG)();["setErrorHandler","finished","addToTrace","inlineHit","addRelease","addPageAction","setCurrentRouteName","setPageViewName","setCustomAttribute","interaction","noticeError","setUserId"].forEach((t=>{e[t]=function(){for(var r=arguments.length,n=new Array(r),i=0;i<r;i++)n[i]=arguments[i];return function(t){for(var r=arguments.length,n=new Array(r>1?r-1:0),i=1;i<r;i++)n[i-1]=arguments[i];let o=[];return Object.values(e.initializedAgents).forEach((e=>{e.exposed&&e.api[t]&&o.push(e.api[t](...n))})),o.length>1?o:o[0]}(t,...n)}}))}var p=r(2587);function m(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},m=arguments.length>2?arguments[2]:void 0,v=arguments.length>3?arguments[3]:void 0,{init:b,info:y,loader_config:w,runtime:x={loaderType:m},exposed:A=!0}=t;const E=(0,h.gG)();y||(b=E.init,y=E.info,w=E.loader_config),(0,i.Dg)(e,b||{}),(0,i.GE)(e,w||{}),(0,i.sU)(e,x),y.jsAttributes??={},d.v6&&(y.jsAttributes.isWorker=!0),(0,i.CX)(e,y),g();const T=function(e,t){t||(0,c.R)(e,"api");const h={};var g=a.ee.get(e),p=g.get("tracer"),m="api-",v=m+"ixn-";function b(t,r,n,o){const a=(0,i.C5)(e);return null===r?delete a.jsAttributes[t]:(0,i.CX)(e,{...a,jsAttributes:{...a.jsAttributes,[t]:r}}),x(m,n,!0,o||null===r?"session":void 0)(t,r)}function y(){}["setErrorHandler","finished","addToTrace","inlineHit","addRelease"].forEach((e=>h[e]=x(m,e,!0,"api"))),h.addPageAction=x(m,"addPageAction",!0,n.D.pageAction),h.setCurrentRouteName=x(m,"routeName",!0,n.D.spa),h.setPageViewName=function(t,r){if("string"==typeof t)return"/"!==t.charAt(0)&&(t="/"+t),(0,i.OP)(e).customTransaction=(r||"http://custom.transaction")+t,x(m,"setPageViewName",!0)()},h.setCustomAttribute=function(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2];if("string"==typeof e){if(["string","number"].includes(typeof t)||null===t)return b(e,t,"setCustomAttribute",r);(0,f.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t,"> was provided."))}else(0,f.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.setUserId=function(e){if("string"==typeof e||null===e)return b("enduser.id",e,"setUserId",!0);(0,f.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.interaction=function(){return(new y).get()};var w=y.prototype={createTracer:function(e,t){var r={},i=this,a="function"==typeof t;return(0,o.p)(v+"tracer",[(0,s.z)(),e,r],i,n.D.spa,g),function(){if(p.emit((a?"":"no-")+"fn-start",[(0,s.z)(),i,a],r),a)try{return t.apply(this,arguments)}catch(e){throw p.emit("fn-err",[arguments,this,"string"==typeof e?new Error(e):e],r),e}finally{p.emit("fn-end",[(0,s.z)()],r)}}}};function x(e,t,r,i){return function(){return(0,o.p)(l.xS,["API/"+t+"/called"],void 0,n.D.metrics,g),i&&(0,o.p)(e+t,[(0,s.z)(),...arguments],r?null:this,i,g),r?void 0:this}}function A(){r.e(439).then(r.bind(r,7438)).then((t=>{let{setAPI:r}=t;r(e),(0,c.L)(e,"api")})).catch((()=>(0,f.Z)("Downloading runtime APIs failed...")))}return["actionText","setName","setAttribute","save","ignore","onEnd","getContext","end","get"].forEach((e=>{w[e]=x(v,e,void 0,n.D.spa)})),h.noticeError=function(e,t){"string"==typeof e&&(e=new Error(e)),(0,o.p)(l.xS,["API/noticeError/called"],void 0,n.D.metrics,g),(0,o.p)("err",[e,(0,s.z)(),!1,t],void 0,n.D.jserrors,g)},d.il?(0,u.b)((()=>A()),!0):A(),h}(e,v);return(0,h.Qy)(e,T,"api"),(0,h.Qy)(e,A,"exposed"),(0,h.EZ)("activatedFeatures",p.T),T}},3325:(e,t,r)=>{r.d(t,{D:()=>n,p:()=>i});const n={ajax:"ajax",jserrors:"jserrors",metrics:"metrics",pageAction:"page_action",pageViewEvent:"page_view_event",pageViewTiming:"page_view_timing",sessionReplay:"session_replay",sessionTrace:"session_trace",spa:"spa"},i={[n.pageViewEvent]:1,[n.pageViewTiming]:2,[n.metrics]:3,[n.jserrors]:4,[n.ajax]:5,[n.sessionTrace]:6,[n.pageAction]:7,[n.spa]:8,[n.sessionReplay]:9}}},n={};function i(e){var t=n[e];if(void 0!==t)return t.exports;var o=n[e]={exports:{}};return r[e](o,o.exports,i),o.exports}i.m=r,i.d=(e,t)=>{for(var r in t)i.o(t,r)&&!i.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},i.f={},i.e=e=>Promise.all(Object.keys(i.f).reduce(((t,r)=>(i.f[r](e,t),t)),[])),i.u=e=>(({78:"page_action-aggregate",147:"metrics-aggregate",242:"session-manager",317:"jserrors-aggregate",348:"page_view_timing-aggregate",412:"lazy-feature-loader",439:"async-api",538:"recorder",590:"session_replay-aggregate",675:"compressor",733:"session_trace-aggregate",786:"page_view_event-aggregate",873:"spa-aggregate",898:"ajax-aggregate"}[e]||e)+"."+{78:"ac76d497",147:"3dc53903",148:"1a20d5fe",242:"2a64278a",317:"49e41428",348:"bd6de33a",412:"2f55ce66",439:"30bd804e",538:"1b18459f",590:"cf0efb30",675:"ae9f91a8",733:"83105561",786:"06482edd",860:"03a8b7a5",873:"e6b09d52",898:"998ef92b"}[e]+"-1.236.0.min.js"),i.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),e={},t="NRBA:",i.l=(r,n,o,a)=>{if(e[r])e[r].push(n);else{var s,c;if(void 0!==o)for(var u=document.getElementsByTagName("script"),d=0;d<u.length;d++){var f=u[d];if(f.getAttribute("src")==r||f.getAttribute("data-webpack")==t+o){s=f;break}}s||(c=!0,(s=document.createElement("script")).charset="utf-8",s.timeout=120,i.nc&&s.setAttribute("nonce",i.nc),s.setAttribute("data-webpack",t+o),s.src=r),e[r]=[n];var l=(t,n)=>{s.onerror=s.onload=null,clearTimeout(h);var i=e[r];if(delete e[r],s.parentNode&&s.parentNode.removeChild(s),i&&i.forEach((e=>e(n))),t)return t(n)},h=setTimeout(l.bind(null,void 0,{type:"timeout",target:s}),12e4);s.onerror=l.bind(null,s.onerror),s.onload=l.bind(null,s.onload),c&&document.head.appendChild(s)}},i.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},i.j=364,i.p="https://js-agent.newrelic.com/",(()=>{var e={364:0,953:0};i.f.j=(t,r)=>{var n=i.o(e,t)?e[t]:void 0;if(0!==n)if(n)r.push(n[2]);else{var o=new Promise(((r,i)=>n=e[t]=[r,i]));r.push(n[2]=o);var a=i.p+i.u(t),s=new Error;i.l(a,(r=>{if(i.o(e,t)&&(0!==(n=e[t])&&(e[t]=void 0),n)){var o=r&&("load"===r.type?"missing":r.type),a=r&&r.target&&r.target.src;s.message="Loading chunk "+t+" failed.\n("+o+": "+a+")",s.name="ChunkLoadError",s.type=o,s.request=a,n[1](s)}}),"chunk-"+t,t)}};var t=(t,r)=>{var n,o,[a,s,c]=r,u=0;if(a.some((t=>0!==e[t]))){for(n in s)i.o(s,n)&&(i.m[n]=s[n]);if(c)c(i)}for(t&&t(r);u<a.length;u++)o=a[u],i.o(e,o)&&e[o]&&e[o][0](),e[o]=0},r=window.webpackChunkNRBA=window.webpackChunkNRBA||[];r.forEach(t.bind(null,0)),r.push=t.bind(null,r.push.bind(r))})();var o={};(()=>{i.r(o);var e=i(3325),t=i(5763);const r=Object.values(e.D);function n(e){const n={};return r.forEach((r=>{n[r]=function(e,r){return!1!==(0,t.Mt)(r,"".concat(e,".enabled"))}(r,e)})),n}var a=i(9144);var s=i(5546),c=i(385),u=i(8e3),d=i(5938),f=i(3960),l=i(50);class h extends d.W{constructor(e,t,r){let n=!(arguments.length>3&&void 0!==arguments[3])||arguments[3];super(e,t,r),this.auto=n,this.abortHandler,this.featAggregate,this.onAggregateImported,n&&(0,u.R)(e,r)}importAggregator(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};if(this.featAggregate||!this.auto)return;const r=c.il&&!0===(0,t.Mt)(this.agentIdentifier,"privacy.cookies_enabled");let n;this.onAggregateImported=new Promise((e=>{n=e}));const o=async()=>{let t;try{if(r){const{setupAgentSession:e}=await Promise.all([i.e(860),i.e(242)]).then(i.bind(i,3228));t=e(this.agentIdentifier)}}catch(e){(0,l.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.",e)}try{if(!this.shouldImportAgg(this.featureName,t))return void(0,u.L)(this.agentIdentifier,this.featureName);const{lazyFeatureLoader:r}=await i.e(412).then(i.bind(i,8582)),{Aggregate:o}=await r(this.featureName,"aggregate");this.featAggregate=new o(this.agentIdentifier,this.aggregator,e),n(!0)}catch(e){(0,l.Z)("Downloading and initializing ".concat(this.featureName," failed..."),e),this.abortHandler?.(),n(!1)}};c.il?(0,f.b)((()=>o()),!0):o()}shouldImportAgg(r,n){return r!==e.D.sessionReplay||!1!==(0,t.Mt)(this.agentIdentifier,"session_trace.enabled")&&(!!n?.isNew||!!n?.state.sessionReplay)}}var g=i(7633),p=i(7894);class m extends h{static featureName=g.t9;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];if(super(r,n,g.t9,i),("undefined"==typeof PerformanceNavigationTiming||c.Tt)&&"undefined"!=typeof PerformanceTiming){const n=(0,t.OP)(r);n[g.Dz]=Math.max(Date.now()-n.offset,0),(0,f.K)((()=>n[g.qw]=Math.max((0,p.z)()-n[g.Dz],0))),(0,f.b)((()=>{const t=(0,p.z)();n[g.OJ]=Math.max(t-n[g.Dz],0),(0,s.p)("timing",["load",t],void 0,e.D.pageViewTiming,this.ee)}))}this.importAggregator()}}var v=i(1117),b=i(1284);class y extends v.w{constructor(e){super(e),this.aggregatedData={}}store(e,t,r,n,i){var o=this.getBucket(e,t,r,i);return o.metrics=function(e,t){t||(t={count:0});return t.count+=1,(0,b.D)(e,(function(e,r){t[e]=w(r,t[e])})),t}(n,o.metrics),o}merge(e,t,r,n,i){var o=this.getBucket(e,t,n,i);if(o.metrics){var a=o.metrics;a.count+=r.count,(0,b.D)(r,(function(e,t){if("count"!==e){var n=a[e],i=r[e];i&&!i.c?a[e]=w(i.t,n):a[e]=function(e,t){if(!t)return e;t.c||(t=x(t.t));return t.min=Math.min(e.min,t.min),t.max=Math.max(e.max,t.max),t.t+=e.t,t.sos+=e.sos,t.c+=e.c,t}(i,a[e])}}))}else o.metrics=r}storeMetric(e,t,r,n){var i=this.getBucket(e,t,r);return i.stats=w(n,i.stats),i}getBucket(e,t,r,n){this.aggregatedData[e]||(this.aggregatedData[e]={});var i=this.aggregatedData[e][t];return i||(i=this.aggregatedData[e][t]={params:r||{}},n&&(i.custom=n)),i}get(e,t){return t?this.aggregatedData[e]&&this.aggregatedData[e][t]:this.aggregatedData[e]}take(e){for(var t={},r="",n=!1,i=0;i<e.length;i++)t[r=e[i]]=A(this.aggregatedData[r]),t[r].length&&(n=!0),delete this.aggregatedData[r];return n?t:null}}function w(e,t){return null==e?function(e){e?e.c++:e={c:1};return e}(t):t?(t.c||(t=x(t.t)),t.c+=1,t.t+=e,t.sos+=e*e,e>t.max&&(t.max=e),e<t.min&&(t.min=e),t):{t:e}}function x(e){return{t:e,min:e,max:e,sos:e*e,c:1}}function A(e){return"object"!=typeof e?[]:(0,b.D)(e,E)}function E(e,t){return t}var T=i(8632),_=i(4402),D=i(4351);var N=i(7956),O=i(3239),j=i(9251);class S extends h{static featureName=j.t;constructor(e,r){let n=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,r,j.t,n),c.il&&((0,t.OP)(e).initHidden=Boolean("hidden"===document.visibilityState),(0,N.N)((()=>(0,s.p)("docHidden",[(0,p.z)()],void 0,j.t,this.ee)),!0),(0,O.bP)("pagehide",(()=>(0,s.p)("winPagehide",[(0,p.z)()],void 0,j.t,this.ee))),this.importAggregator())}}var P=i(3081);class C extends h{static featureName=P.t9;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,P.t9,r),this.importAggregator()}}var R,I=i(2210),k=i(1214),H=i(2177),L={};try{R=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(L.console=!0,-1!==R.indexOf("dev")&&(L.dev=!0),-1!==R.indexOf("nr_dev")&&(L.nrDev=!0))}catch(e){}function z(e){try{L.console&&z(e)}catch(e){}}L.nrDev&&H.ee.on("internal-error",(function(e){z(e.stack)})),L.dev&&H.ee.on("fn-err",(function(e,t,r){z(r.stack)})),L.dev&&(z("NR AGENT IN DEVELOPMENT MODE"),z("flags: "+(0,b.D)(L,(function(e,t){return e})).join(", ")));var M=i(6660);class B extends h{static featureName=M.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,M.t,i),this.skipNext=0;try{this.removeOnAbort=new AbortController}catch(e){}const o=this;o.ee.on("fn-start",(function(e,t,r){o.abortHandler&&(o.skipNext+=1)})),o.ee.on("fn-err",(function(t,r,n){o.abortHandler&&!n[M.A]&&((0,I.X)(n,M.A,(function(){return!0})),this.thrown=!0,(0,s.p)("err",[n,(0,p.z)()],void 0,e.D.jserrors,o.ee))})),o.ee.on("fn-end",(function(){o.abortHandler&&!this.thrown&&o.skipNext>0&&(o.skipNext-=1)})),o.ee.on("internal-error",(function(t){(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,o.ee)})),this.origOnerror=c._A.onerror,c._A.onerror=this.onerrorHandler.bind(this),c._A.addEventListener("unhandledrejection",(t=>{const r=function(e){let t="Unhandled Promise Rejection: ";if(e instanceof Error)try{return e.message=t+e.message,e}catch(t){return e}if(void 0===e)return new Error(t);try{return new Error(t+(0,D.P)(e))}catch(e){return new Error(t)}}(t.reason);(0,s.p)("err",[r,(0,p.z)(),!1,{unhandledPromiseRejection:1}],void 0,e.D.jserrors,this.ee)}),(0,O.m$)(!1,this.removeOnAbort?.signal)),(0,k.gy)(this.ee),(0,k.BV)(this.ee),(0,k.em)(this.ee),(0,t.OP)(r).xhrWrappable&&(0,k.Kf)(this.ee),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}onerrorHandler(t,r,n,i,o){"function"==typeof this.origOnerror&&this.origOnerror(...arguments);try{this.skipNext?this.skipNext-=1:(0,s.p)("err",[o||new F(t,r,n),(0,p.z)()],void 0,e.D.jserrors,this.ee)}catch(t){try{(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,this.ee)}catch(e){}}return!1}}function F(e,t,r){this.message=e||"Uncaught error with no additional information",this.sourceURL=t,this.line=r}let U=1;const q="nr@id";function G(e){const t=typeof e;return!e||"object"!==t&&"function"!==t?-1:e===c._A?0:(0,I.X)(e,q,(function(){return U++}))}function V(e){if("string"==typeof e&&e.length)return e.length;if("object"==typeof e){if("undefined"!=typeof ArrayBuffer&&e instanceof ArrayBuffer&&e.byteLength)return e.byteLength;if("undefined"!=typeof Blob&&e instanceof Blob&&e.size)return e.size;if(!("undefined"!=typeof FormData&&e instanceof FormData))try{return(0,D.P)(e).length}catch(e){return}}}var X=i(7243);class W{constructor(e){this.agentIdentifier=e,this.generateTracePayload=this.generateTracePayload.bind(this),this.shouldGenerateTrace=this.shouldGenerateTrace.bind(this)}generateTracePayload(e){if(!this.shouldGenerateTrace(e))return null;var r=(0,t.DL)(this.agentIdentifier);if(!r)return null;var n=(r.accountID||"").toString()||null,i=(r.agentID||"").toString()||null,o=(r.trustKey||"").toString()||null;if(!n||!i)return null;var a=(0,_.M)(),s=(0,_.Ht)(),c=Date.now(),u={spanId:a,traceId:s,timestamp:c};return(e.sameOrigin||this.isAllowedOrigin(e)&&this.useTraceContextHeadersForCors())&&(u.traceContextParentHeader=this.generateTraceContextParentHeader(a,s),u.traceContextStateHeader=this.generateTraceContextStateHeader(a,c,n,i,o)),(e.sameOrigin&&!this.excludeNewrelicHeader()||!e.sameOrigin&&this.isAllowedOrigin(e)&&this.useNewrelicHeaderForCors())&&(u.newrelicHeader=this.generateTraceHeader(a,s,c,n,i,o)),u}generateTraceContextParentHeader(e,t){return"00-"+t+"-"+e+"-01"}generateTraceContextStateHeader(e,t,r,n,i){return i+"@nr=0-1-"+r+"-"+n+"-"+e+"----"+t}generateTraceHeader(e,t,r,n,i,o){if(!("function"==typeof c._A?.btoa))return null;var a={v:[0,1],d:{ty:"Browser",ac:n,ap:i,id:e,tr:t,ti:r}};return o&&n!==o&&(a.d.tk=o),btoa((0,D.P)(a))}shouldGenerateTrace(e){return this.isDtEnabled()&&this.isAllowedOrigin(e)}isAllowedOrigin(e){var r=!1,n={};if((0,t.Mt)(this.agentIdentifier,"distributed_tracing")&&(n=(0,t.P_)(this.agentIdentifier).distributed_tracing),e.sameOrigin)r=!0;else if(n.allowed_origins instanceof Array)for(var i=0;i<n.allowed_origins.length;i++){var o=(0,X.e)(n.allowed_origins[i]);if(e.hostname===o.hostname&&e.protocol===o.protocol&&e.port===o.port){r=!0;break}}return r}isDtEnabled(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.enabled}excludeNewrelicHeader(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.exclude_newrelic_header}useNewrelicHeaderForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!1!==e.cors_use_newrelic_header}useTraceContextHeadersForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.cors_use_tracecontext_headers}}var Z=i(7825),$=["load","error","abort","timeout"],Q=$.length,Y=t.Yu.REQ,K=c._A.XMLHttpRequest;class J extends h{static featureName=Z.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,Z.t,i),(0,t.OP)(r).xhrWrappable&&(this.dt=new W(r),this.handler=(e,t,r,n)=>(0,s.p)(e,t,r,n,this.ee),(0,k.u5)(this.ee),(0,k.Kf)(this.ee),function(r,n,i,o){function a(e){var t=this;t.totalCbs=0,t.called=0,t.cbTime=0,t.end=E,t.ended=!1,t.xhrGuids={},t.lastSize=null,t.loadCaptureCalled=!1,t.params=this.params||{},t.metrics=this.metrics||{},e.addEventListener("load",(function(r){_(t,e)}),(0,O.m$)(!1)),c.IF||e.addEventListener("progress",(function(e){t.lastSize=e.loaded}),(0,O.m$)(!1))}function s(e){this.params={method:e[0]},T(this,e[1]),this.metrics={}}function u(e,n){var i=(0,t.DL)(r);i.xpid&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",i.xpid);var a=o.generateTracePayload(this.parsedOrigin);if(a){var s=!1;a.newrelicHeader&&(n.setRequestHeader("newrelic",a.newrelicHeader),s=!0),a.traceContextParentHeader&&(n.setRequestHeader("traceparent",a.traceContextParentHeader),a.traceContextStateHeader&&n.setRequestHeader("tracestate",a.traceContextStateHeader),s=!0),s&&(this.dt=a)}}function d(e,t){var r=this.metrics,i=e[0],o=this;if(r&&i){var a=V(i);a&&(r.txSize=a)}this.startTime=(0,p.z)(),this.listener=function(e){try{"abort"!==e.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==e.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof t.onload)&&"function"==typeof o.end)&&o.end(t)}catch(e){try{n.emit("internal-error",[e])}catch(e){}}};for(var s=0;s<Q;s++)t.addEventListener($[s],this.listener,(0,O.m$)(!1))}function f(e,t,r){this.cbTime+=e,t?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof r.onload||"function"!=typeof this.end||this.end(r)}function l(e,t){var r=""+G(e)+!!t;this.xhrGuids&&!this.xhrGuids[r]&&(this.xhrGuids[r]=!0,this.totalCbs+=1)}function h(e,t){var r=""+G(e)+!!t;this.xhrGuids&&this.xhrGuids[r]&&(delete this.xhrGuids[r],this.totalCbs-=1)}function g(){this.endTime=(0,p.z)()}function m(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-added",[e[1],e[2]],t)}function v(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-removed",[e[1],e[2]],t)}function b(e,t,r){t instanceof K&&("onload"===r&&(this.onload=!0),("load"===(e[0]&&e[0].type)||this.onload)&&(this.xhrCbStart=(0,p.z)()))}function y(e,t){this.xhrCbStart&&n.emit("xhr-cb-time",[(0,p.z)()-this.xhrCbStart,this.onload,t],t)}function w(e){var t,r=e[1]||{};"string"==typeof e[0]?t=e[0]:e[0]&&e[0].url?t=e[0].url:c._A?.URL&&e[0]&&e[0]instanceof URL&&(t=e[0].href),t&&(this.parsedOrigin=(0,X.e)(t),this.sameOrigin=this.parsedOrigin.sameOrigin);var n=o.generateTracePayload(this.parsedOrigin);if(n&&(n.newrelicHeader||n.traceContextParentHeader))if("string"==typeof e[0]||c._A?.URL&&e[0]&&e[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),s(i.headers,n)&&(this.dt=n),e.length>1?e[1]=i:e.push(i)}else e[0]&&e[0].headers&&s(e[0].headers,n)&&(this.dt=n);function s(e,t){var r=!1;return t.newrelicHeader&&(e.set("newrelic",t.newrelicHeader),r=!0),t.traceContextParentHeader&&(e.set("traceparent",t.traceContextParentHeader),t.traceContextStateHeader&&e.set("tracestate",t.traceContextStateHeader),r=!0),r}}function x(e,t){this.params={},this.metrics={},this.startTime=(0,p.z)(),this.dt=t,e.length>=1&&(this.target=e[0]),e.length>=2&&(this.opts=e[1]);var r,n=this.opts||{},i=this.target;"string"==typeof i?r=i:"object"==typeof i&&i instanceof Y?r=i.url:c._A?.URL&&"object"==typeof i&&i instanceof URL&&(r=i.href),T(this,r);var o=(""+(i&&i instanceof Y&&i.method||n.method||"GET")).toUpperCase();this.params.method=o,this.txSize=V(n.body)||0}function A(t,r){var n;this.endTime=(0,p.z)(),this.params||(this.params={}),this.params.status=r?r.status:0,"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var o={txSize:this.txSize,rxSize:n,duration:(0,p.z)()-this.startTime};i("xhr",[this.params,o,this.startTime,this.endTime,"fetch"],this,e.D.ajax)}function E(t){var r=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var o=0;o<Q;o++)t.removeEventListener($[o],this.listener,!1);r.aborted||(n.duration=(0,p.z)()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==r.status&&(r.status=0):_(this,t),n.cbTime=this.cbTime,i("xhr",[r,n,this.startTime,this.endTime,"xhr"],this,e.D.ajax))}}function T(e,t){var r=(0,X.e)(t),n=e.params;n.hostname=r.hostname,n.port=r.port,n.protocol=r.protocol,n.host=r.hostname+":"+r.port,n.pathname=r.pathname,e.parsedOrigin=r,e.sameOrigin=r.sameOrigin}function _(e,t){e.params.status=t.status;var r=function(e,t){var r=e.responseType;return"json"===r&&null!==t?t:"arraybuffer"===r||"blob"===r||"json"===r?V(e.response):"text"===r||""===r||void 0===r?V(e.responseText):void 0}(t,e.lastSize);if(r&&(e.metrics.rxSize=r),e.sameOrigin){var n=t.getResponseHeader("X-NewRelic-App-Data");n&&(e.params.cat=n.split(", ").pop())}e.loadCaptureCalled=!0}n.on("new-xhr",a),n.on("open-xhr-start",s),n.on("open-xhr-end",u),n.on("send-xhr-start",d),n.on("xhr-cb-time",f),n.on("xhr-load-added",l),n.on("xhr-load-removed",h),n.on("xhr-resolved",g),n.on("addEventListener-end",m),n.on("removeEventListener-end",v),n.on("fn-end",y),n.on("fetch-before-start",w),n.on("fetch-start",x),n.on("fn-start",b),n.on("fetch-done",A)}(r,this.ee,this.handler,this.dt),this.importAggregator())}}var ee=i(3614);const{BST_RESOURCE:te,RESOURCE:re,START:ne,END:ie,FEATURE_NAME:oe,FN_END:ae,FN_START:se,PUSH_STATE:ce}=ee;var ue=i(7836);const{FEATURE_NAME:de,START:fe,END:le,BODY:he,CB_END:ge,JS_TIME:pe,FETCH:me,FN_START:ve,CB_START:be,FN_END:ye}=ue;var we=i(4649);class xe extends h{static featureName=we.t;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,we.t,r),this.importAggregator()}}new class{constructor(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:(0,_.ky)(16);c._A?(this.agentIdentifier=t,this.sharedAggregator=new y({agentIdentifier:this.agentIdentifier}),this.features={},this.desiredFeatures=new Set(e.features||[]),this.desiredFeatures.add(m),Object.assign(this,(0,a.j)(this.agentIdentifier,e,e.loaderType||"agent")),this.start()):(0,l.Z)("Failed to initial the agent. Could not determine the runtime environment.")}get config(){return{info:(0,t.C5)(this.agentIdentifier),init:(0,t.P_)(this.agentIdentifier),loader_config:(0,t.DL)(this.agentIdentifier),runtime:(0,t.OP)(this.agentIdentifier)}}start(){const t="features";try{const r=n(this.agentIdentifier),i=[...this.desiredFeatures];i.sort(((t,r)=>e.p[t.featureName]-e.p[r.featureName])),i.forEach((t=>{if(r[t.featureName]||t.featureName===e.D.pageViewEvent){const n=function(t){switch(t){case e.D.ajax:return[e.D.jserrors];case e.D.sessionTrace:return[e.D.ajax,e.D.pageViewEvent];case e.D.sessionReplay:return[e.D.sessionTrace];case e.D.pageViewTiming:return[e.D.pageViewEvent];default:return[]}}(t.featureName);n.every((e=>r[e]))||(0,l.Z)("".concat(t.featureName," is enabled but one or more dependent features has been disabled (").concat((0,D.P)(n),"). This may cause unintended consequences or missing data...")),this.features[t.featureName]=new t(this.agentIdentifier,this.sharedAggregator)}})),(0,T.Qy)(this.agentIdentifier,this.features,t)}catch(e){(0,l.Z)("Failed to initialize all enabled instrument classes (agent aborted) -",e);for(const e in this.features)this.features[e].abortHandler?.();const r=(0,T.fP)();return delete r.initializedAgents[this.agentIdentifier]?.api,delete r.initializedAgents[this.agentIdentifier]?.[t],delete this.sharedAggregator,r.ee?.abort(),delete r.ee?.get(this.agentIdentifier),!1}}}({features:[J,m,S,class extends h{static featureName=oe;constructor(t,r){if(super(t,r,oe,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;const n=this.ee;let i;(0,k.QU)(n),this.eventsEE=(0,k.em)(n),this.eventsEE.on(se,(function(e,t){this.bstStart=(0,p.z)()})),this.eventsEE.on(ae,(function(t,r){(0,s.p)("bst",[t[0],r,this.bstStart,(0,p.z)()],void 0,e.D.sessionTrace,n)})),n.on(ce+ne,(function(e){this.time=(0,p.z)(),this.startPath=location.pathname+location.hash})),n.on(ce+ie,(function(t){(0,s.p)("bstHist",[location.pathname+location.hash,this.startPath,this.time],void 0,e.D.sessionTrace,n)}));try{i=new PerformanceObserver((t=>{const r=t.getEntries();(0,s.p)(te,[r],void 0,e.D.sessionTrace,n)})),i.observe({type:re,buffered:!0})}catch(e){}this.importAggregator({resourceObserver:i})}},C,xe,B,class extends h{static featureName=de;constructor(e,r){if(super(e,r,de,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;if(!(0,t.OP)(e).xhrWrappable)return;try{this.removeOnAbort=new AbortController}catch(e){}let n,i=0;const o=this.ee.get("tracer"),a=(0,k._L)(this.ee),s=(0,k.Lg)(this.ee),u=(0,k.BV)(this.ee),d=(0,k.Kf)(this.ee),f=this.ee.get("events"),l=(0,k.u5)(this.ee),h=(0,k.QU)(this.ee),g=(0,k.Gm)(this.ee);function m(e,t){h.emit("newURL",[""+window.location,t])}function v(){i++,n=window.location.hash,this[ve]=(0,p.z)()}function b(){i--,window.location.hash!==n&&m(0,!0);var e=(0,p.z)();this[pe]=~~this[pe]+e-this[ve],this[ye]=e}function y(e,t){e.on(t,(function(){this[t]=(0,p.z)()}))}this.ee.on(ve,v),s.on(be,v),a.on(be,v),this.ee.on(ye,b),s.on(ge,b),a.on(ge,b),this.ee.buffer([ve,ye,"xhr-resolved"],this.featureName),f.buffer([ve],this.featureName),u.buffer(["setTimeout"+le,"clearTimeout"+fe,ve],this.featureName),d.buffer([ve,"new-xhr","send-xhr"+fe],this.featureName),l.buffer([me+fe,me+"-done",me+he+fe,me+he+le],this.featureName),h.buffer(["newURL"],this.featureName),g.buffer([ve],this.featureName),s.buffer(["propagate",be,ge,"executor-err","resolve"+fe],this.featureName),o.buffer([ve,"no-"+ve],this.featureName),a.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"],this.featureName),y(l,me+fe),y(l,me+"-done"),y(a,"new-jsonp"),y(a,"jsonp-end"),y(a,"cb-start"),h.on("pushState-end",m),h.on("replaceState-end",m),window.addEventListener("hashchange",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("load",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("popstate",(function(){m(0,i>1)}),(0,O.m$)(!0,this.removeOnAbort?.signal)),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}}],loaderType:"spa"})})(),window.NRBA=o})();</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="B.Krause"/><meta name="DC.Creator" content="T.Meyer"/><meta name="DC.Creator" content="H.Sieg"/><meta name="DC.Creator" content="C.Kstner"/><meta name="DC.Creator" content="P.Reichardt"/><meta name="DC.Creator" content="J.Tentschert"/><meta name="DC.Creator" content="H.Jungnickel"/><meta name="DC.Creator" content="I.Estrela-Lopis"/><meta name="DC.Creator" content="A.Burel"/><meta name="DC.Creator" content="S.Chevance"/><meta name="DC.Creator" content="F.Gauffre"/><meta name="DC.Creator" content="P.Jalili"/><meta name="DC.Creator" content="J.Meijer"/><meta name="DC.Creator" content="L.Bhmert"/><meta name="DC.Creator" content="A.Braeuning"/><meta name="DC.Creator" content="A. F.Thnemann"/><meta name="DC.Creator" content="F.Emmerling"/><meta name="DC.Creator" content="V.Fessard"/><meta name="DC.Creator" content="P.Laux"/><meta name="DC.Creator" content="A.Lampen"/><meta name="DC.Creator" content="A.Luch"/><meta name="DC.title" content="Characterization of aluminum, aluminum oxide and titanium dioxide nanomaterials using a combination of methods for particle surface and size analysis  "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2018/04/17"/><meta name="DC.Identifier" scheme="doi" content="10.1039/C8RA00205C"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Characterization of aluminum, aluminum oxide and titanium dioxide nanomaterials using a combination of methods for particle surface and size analysis  "/><meta name="citation_author" content="B.Krause"/><meta name="citation_author" content="T.Meyer"/><meta name="citation_author" content="H.Sieg"/><meta name="citation_author" content="C.Kstner"/><meta name="citation_author" content="P.Reichardt"/><meta name="citation_author" content="J.Tentschert"/><meta name="citation_author" content="H.Jungnickel"/><meta name="citation_author" content="I.Estrela-Lopis"/><meta name="citation_author" content="A.Burel"/><meta name="citation_author" content="S.Chevance"/><meta name="citation_author" content="F.Gauffre"/><meta name="citation_author" content="P.Jalili"/><meta name="citation_author" content="J.Meijer"/><meta name="citation_author" content="L.Bhmert"/><meta name="citation_author" content="A.Braeuning"/><meta name="citation_author" content="A. F.Thnemann"/><meta name="citation_author" content="F.Emmerling"/><meta name="citation_author" content="V.Fessard"/><meta name="citation_author" content="P.Laux"/><meta name="citation_author" content="A.Lampen"/><meta name="citation_author" content="A.Luch"/><meta name="citation_online_date" content="2018/04/17"/><meta name="citation_date" content="2018"/><meta name="citation_journal_title" content="RSC Advances"/><meta name="citation_volume" content="8"/><meta name="citation_issue" content="26"/><meta name="citation_firstpage" content="14377"/><meta name="citation_lastpage" content="14388"/><meta name="citation_doi" content="10.1039/C8RA00205C"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2018/ra/c8ra00205c"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2018/ra/c8ra00205c"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2018/ra/c8ra00205c"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"></script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"></script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2018/ra/c8ra00205c">ViewPDFVersion</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:c8ra01147h)" href="/en/content/articlehtml/2018/ra/c8ra01147h" target="_BLANK">PreviousArticle</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:c8ra01606b)" href="/en/content/articlehtml/2018/ra/c8ra01606b" target="_BLANK">NextArticle</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script></div></div><br/><span class="oa"><img src="/content/newimages/open_access_blue.png" alt=""/> Open Access Article<br/><img src="/content/newimages/CCBY-NC.svg" alt=""/>
This Open Access Article is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/3.0/">Creative Commons Attribution-Non Commercial 3.0 Unported Licence</a></span></div><div class="article_info"> DOI:<a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/C8RA00205C">10.1039/C8RA00205C</a>
(Paper)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2046-2069/2011">RSC Adv.</a></span>, 2018, <strong>8</strong>, 14377-14388</div><h1 id="sect231"><span class="title_heading">Characterization of aluminum, aluminum oxide and titanium dioxide nanomaterials using a combination of methods for particle surface and size analysis<a title="Select to navigate to footnote" href="#fn1"></a></span></h1><p class="header_text">
<span class="bold">B. Krause</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for B.Krause (orcid.org/0000-0003-4684-2206) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-4684-2206"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">T. Meyer</span><sup><span class="sup_ref italic">b</span></sup>, 
<span class="bold">H. Sieg</span><sup><span class="sup_ref italic">c</span></sup>, 
<span class="bold">C. Kstner</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for C.Kstner (orcid.org/0000-0003-3730-7404) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-3730-7404"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">d</span></sup>, 
<span class="bold">P. Reichardt</span><sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">J. Tentschert</span><sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">H. Jungnickel</span><sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">I. Estrela-Lopis</span><sup><span class="sup_ref italic">b</span></sup>, 
<span class="bold">A. Burel</span><sup><span class="sup_ref italic">e</span></sup>, 
<span class="bold">S. Chevance</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for S.Chevance (orcid.org/0000-0001-9699-5943) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-9699-5943"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">f</span></sup>, 
<span class="bold">F. Gauffre</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for F.Gauffre (orcid.org/0000-0003-2835-3556) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-2835-3556"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">f</span></sup>, 
<span class="bold">P. Jalili</span><sup><span class="sup_ref italic">g</span></sup>, 
<span class="bold">J. Meijer</span><sup><span class="sup_ref italic">h</span></sup>, 
<span class="bold">L. Bhmert</span><sup><span class="sup_ref italic">c</span></sup>, 
<span class="bold">A. Braeuning</span><sup><span class="sup_ref italic">c</span></sup>, 
<span class="bold">A. F. Thnemann</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for A. F.Thnemann (orcid.org/0000-0002-9883-6134) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-9883-6134"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">d</span></sup>, 
<span class="bold">F. Emmerling</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for F.Emmerling (orcid.org/0000-0001-8528-0301) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-8528-0301"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">i</span></sup>, 
<span class="bold">V. Fessard</span><sup><span class="sup_ref italic">g</span></sup>, 
<span class="bold">P. Laux</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for P.Laux (orcid.org/0000-0002-0351-3392) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-0351-3392"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">A. Lampen</span><sup><span class="sup_ref italic">c</span></sup><span class="bold"> and </span>
<span class="bold">A. Luch</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for A.Luch (orcid.org/0000-0002-5866-901X) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-5866-901X"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>
<br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">German Federal Institute for Risk Assessment (BfR), Department of Chemical and Product Safety, Max-Dohrn-Strae 8-10, 10589 Berlin, Germany. E-mail: <a href="mailto:benjamin-christoph.krause@bfr.bund.de">benjamin-christoph.krause@bfr.bund.de</a></span>
<br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Institute of Medical Physics and Biophysics, University of Leipzig, Hrtelstrasse 16-18, 04275 Leipzig, Germany</span>
<br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">German Federal Institute for Risk Assessment (BfR), Department of Food Safety, Max-Dohrn-Strae 8-10, 10589 Berlin, Germany</span>
<br/><a id="affd"><sup><span class="sup_ref italic">d</span></sup></a><span class="italic">Federal Institute for Materials Research and Testing (BAM), Unter den Eichen 87, 12205 Berlin, Germany</span>
<br/><a id="affe"><sup><span class="sup_ref italic">e</span></sup></a><span class="italic">MRIC TEM BIOSIT, Universit de Rennes 1, 2 av pro Leon Bernard, France</span>
<br/><a id="afff"><sup><span class="sup_ref italic">f</span></sup></a><span class="italic">Univ Rennes, CNRS, ISCR UMR6226, F-35000 Rennes, France</span>
<br/><a id="affg"><sup><span class="sup_ref italic">g</span></sup></a><span class="italic">ANSES, French Agency for Food, Environmental and Occupational Health and Safety, Fougres Laboratory, 10B rue Claude Bourgelat, 35306, Fougres Cedex, France</span>
<br/><a id="affh"><sup><span class="sup_ref italic">h</span></sup></a><span class="italic">Felix Bloch Institute for Solid State Physics, Faculty of Physics and Geosciences, Division of Nuclear Solid State Physics, University of Leipzig, Linnstrae 5, 04103 Leipzig, Germany</span>
<br/><a id="affi"><sup><span class="sup_ref italic">i</span></sup></a><span class="italic">Federal Institute for Materials Research and Testing (BAM), Richard-Willsttter-Strae 11, 12489 Berlin, Germany</span>
</p><div id="art-admin"><span class="italic bold">Received 
8th January 2018
</span><span class="bold italic">, Accepted 22nd March 2018</span></div><p class="bold italic">First published on 17th April 2018</p><hr/><div class="abstract"><h2>Abstract</h2><p>The application of appropriate analytical techniques is essential for nanomaterial (NM) characterization. In this study, we compared different analytical techniques for NM analysis. Regarding possible adverse health effects, ionic and particulate NM effects have to be taken into account. As NMs behave quite differently in physiological media, special attention was paid to techniques which are able to determine the biosolubility and complexation behavior of NMs. Representative NMs of similar size were selected: aluminum (Al<small><sup>0</sup></small>) and aluminum oxide (Al<small><sub>2</sub></small>O<small><sub>3</sub></small>), to compare the behavior of metal and metal oxides. In addition, titanium dioxide (TiO<small><sub>2</sub></small>) was investigated. Characterization techniques such as dynamic light scattering (DLS) and nanoparticle tracking analysis (NTA) were evaluated with respect to their suitability for fast characterization of nanoparticle dispersions regarding a particle's hydrodynamic diameter and size distribution. By application of inductively coupled plasma mass spectrometry in the single particle mode (SP-ICP-MS), individual nanoparticles were quantified and characterized regarding their size. SP-ICP-MS measurements were correlated with the information gained using other characterization techniques, <span class="italic">i.e.</span> transmission electron microscopy (TEM) and small angle X-ray scattering (SAXS). The particle surface as an important descriptor of NMs was analyzed by X-ray diffraction (XRD). NM impurities and their co-localization with biomolecules were determined by ion beam microscopy (IBM) and confocal Raman microscopy (CRM). We conclude advantages and disadvantages of the different techniques applied and suggest options for their complementation. Thus, this paper may serve as a practical guide to particle characterization techniques.</p></div><hr/>
<h2 id="sect453"><span class="a_heading">Introduction</span></h2>
<span>The specific properties of NMs depend on their physicochemical characteristics. Optical properties build upon the size, the shape and the surface structure,<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1,2</span></sup></a> while higher reactivity, for example, may result from a high surface area,<a title="Select to navigate to reference" href="#cit3"><sup><span class="sup_ref">3,4</span></sup></a> specific surface coatings<a title="Select to navigate to references" href="#cit5"><sup><span class="sup_ref">5</span></sup></a> or a surface charge.<a title="Select to navigate to reference" href="#cit6"><sup><span class="sup_ref">68</span></sup></a> An important factor for increasing their activity is the self-assembly of NMs. Here, NMs associate <span class="italic">via</span> non-covalent interactions resulting in organized structures of higher-order. Different applications based on advanced functions were reported, for example formation of mesoporous TiO<small><sub>2</sub></small> mediated by ionic liquids for solar cell conversion, catalysis or electronic devices.<a title="Select to navigate to reference" href="#cit9"><sup><span class="sup_ref">911</span></sup></a> Another application is the self-assembly of biomolecules, like lipids and proteins, mediating inner-particle mesoporosity in a macroporous TiO<small><sub>2</sub></small> structure.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a></span>
<p class="otherpara">TiO<small><sub>2</sub></small> NMs are classified as granular biodurable particles (GBPs) of low toxicity.<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> They occur in the form of anatase or rutile as well as in mixtures thereof. While significant accumulation was shown in the liver of rats in the case of orally administered TiO<small><sub>2</sub></small> NMs,<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a> this was different in studies with Al<small><sup>0</sup></small> NMs in mice, in which predominant accumulation in the brain, thymus and lung was revealed.<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a> Characterization is important for both <span class="italic">in vitro</span> and <span class="italic">in vivo</span> studies. Currently, the human health risk assessment of NMs is mainly based on <span class="italic">in vivo</span> experiments in rodents.<a title="Select to navigate to reference" href="#cit16"><sup><span class="sup_ref">16,17</span></sup></a> However, due to the high number of new NMs,<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> it is not ethical or feasible to conduct such studies for each individual NM. On the other hand, <span class="italic">in vitro</span> systems proved useful, <span class="italic">e.g.</span> to generate high throughput data.<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a> Extrapolation to the <span class="italic">in vivo</span> situation remains limited, in particular due to the insufficient comparability of applied dose and particle biotransformation.<a title="Select to navigate to reference" href="#cit20"><sup><span class="sup_ref">20,21</span></sup></a> An accurate characterization of NMs in <span class="italic">in vitro</span> systems by the application of up-to-date analytical methods may therefore help to establish reliable methods for determination of nanomaterial uptake and translocation as key parameters that affect NM-related toxicity. Such an approach would therefore help to reduce the number of materials that need to be subjected to animal testing. We investigated rather soluble Al<small><sup>0</sup></small> and rather insoluble Al<small><sub>2</sub></small>O<small><sub>3</sub></small> and TiO<small><sub>2</sub></small> NMs (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>).</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f1.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Hypothesis of different behavior of soluble and insoluble NMs after uptake.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">This classification is important for NMs because even the same chemical composition can exhibit differences in physicochemical properties. Compared to bulk material, variations are much higher for NMs.<a title="Select to navigate to reference" href="#cit22"><sup><span class="sup_ref">22,23</span></sup></a></p>
<p class="otherpara">For the characterization of test materials, we applied a combination of techniques based on different measuring principles. With DLS as intensity-weighted method, we assessed the hydrodynamic diameter and the polydispersity of the materials in aqueous suspensions and cell culture media (CCM). To validate the results, the more reliable, number-based approach of NTA was applied. The two methods, DLS and NTA, were performed in two different laboratories, allowing for a direct comparison of the results achieved. For particle surface investigation, XRD was used to test whether the aluminum was already oxidized. The core diameter of Al<small><sup>0</sup></small>, Al<small><sub>2</sub></small>O<small><sub>3</sub></small> and both TiO<small><sub>2</sub></small> NMs was measured by SP-ICP-MS. These results were compared to TEM measurements. As a further technique for estimation of the core diameter, SAXS was applied. Additionally, IBM and CRM were performed to analyze the interaction of NMs with biomolecules as well as to quantify impurities in the NM composition. ToF-SIMS is capable of visualizing the formation of complexes out of Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs with components of the environmental media. This allows investigation of the behavior of NMs within physiological fluids such as CCM.</p>
<p class="otherpara">With respect to uptake, the dissolution of NMs in different media is of high importance. For example, during an artificial digestion procedure, different pH values, as well as proteins, enzymes and other compounds, mimic the oral uptake route for NMs. For Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs, an increased dissolution within the gastric environment was noticed.<a title="Select to navigate to references" href="#cit24"><sup><span class="sup_ref">24</span></sup></a> However, to properly interpret and compare the results, precise knowledge of the properties of the starting materials and of their state in CCM is required. Although there are a lot of studies dealing with silver and copper-containing NMs and their dissolution behavior in biological media,<a title="Select to navigate to reference" href="#cit25"><sup><span class="sup_ref">2527</span></sup></a> to our knowledge, no study accounts for Al<small><sup>0</sup></small> NMs. Furthermore, the dissolution behavior of aluminum-containing nanomaterials may be very different since aluminum ions already have a different complexation behavior compared to silver or copper ions. By means of the methods described earlier (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>), we were able to obtain valuable information to hypothesize the behavior of NMs in biological media and to extrapolate to the <span class="italic">in vivo</span> situation. With these assumptions, it is much easier to understand and explain future data resulting from more complex scenarios.</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f2.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Overview of key NM characteristics (colored rhombi) and methods (colored circles) used in this study for characterization. Arrows in bold imply the main method for the linked characteristic. Abbreviations: DLS  dynamic light scattering; NTA  nanoparticle tracking analysis; TEM  transmission electron microscopy; EDX  energy dispersive X-ray spectroscopy; SP-ICP-MS  single particle inductively coupled plasma mass spectrometry; SAXS  small angle X-ray scattering; XRD  X-ray diffraction; CRM  confocal Raman microscopy; ICP-MS  inductively coupled plasma mass spectrometry; IBM  ion beam microscopy; ToF-SIMS  time of flight mass spectrometry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>

<h2 id="sect511"><span class="a_heading">Experimental</span></h2>
<h3 id="sect513"><span class="b_heading">Materials and methods</span></h3>
<span>Al<small><sup>0</sup></small> NMs (mean diameter 18 nm (TEM), 99.9%) and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs (mean diameter 20 nm (TEM), 99+%) were purchased from IoLiTec Ionic Liquids Technologies GmbH, Heilbronn, Germany. TiO<small><sub>2</sub></small> NMs (NM103 and NM104, mean diameter 25 nm (TEM)) were purchased from JRC Joint Research Centre, Ispra, Italy. Bovine serum albumin (BSA) was bought from Sigma Aldrich. CCM (DMEM, high glucose (4.5 g l<small><sup>1</sup></small>), with sodium pyruvate; with <span class="small_caps">L</span>-glutamine; with 1% penicillin/streptomycin (P/S)) and fetal bovine serum were purchased from PAA Laboratories GmbH, Paching, Austria. All other chemicals used in this study were reagent grade.</span>

<h3 id="sect522"><span class="b_heading">Sample preparation</span></h3>
<span>NM dispersions were prepared following the NanoGenoTOX dispersion protocol Final protocol for producing suitable manufactured NMs exposure media (October 2011).<a title="Select to navigate to references" href="#cit28"><sup><span class="sup_ref">28</span></sup></a></span>
<p class="otherpara">For cell culture experiments, stock dispersions of NMs were diluted in DMEM with 10% fetal calf serum (FCS) to either 10 or 100 g ml<small><sup>1</sup></small>.</p>
<p class="otherpara">For ion release testing, stock dispersions were diluted in 0.05% BSA to 100 and 10 g ml<small><sup>1</sup></small> to reflect a high and a low concentration used in cell culture experiments. 1 ml was taken and centrifuged at 16000  <span class="italic">g</span> for 1 h (Hettich Zentrifuge Mikro 220R). 500 l of the supernatant were taken, 750 l of HNO<small><sub>3</sub></small> (69%) were added and Millipore water was used to fill up to 15 ml.</p>
<p class="otherpara">For IBM and CRM, NMs were centrifuged at 8000  <span class="italic">g</span> for 10 min, supernatant was taken away, mpH<small><sub>2</sub></small>O was added and the sample was vortexed. The procedure was repeated 3 times. Finally, a small drop of the sample was given on polypropylene foil for IBM and on quartz glass for CRM. The dry samples were measured.</p>

<h3 id="sect535"><span class="b_heading">Dynamic light scattering measurements</span></h3>
<span>The distributions of the hydrodynamic diameters of the NMs were determined using a Malvern Nano ZS (Malvern Inc., UK) or a Brookhaven ZetaPALS (Brookhaven Instruments Corporation, USA). A stock dispersion (2.56 mg ml<small><sup>1</sup></small> in 0.05% BSA) was diluted to the concentration of 100 g ml<small><sup>1</sup></small>. The NM dispersions in 0.05% BSA as well as in DMEM were analyzed 510 min after preparation. Thermal equilibration time was set to 60 s at 25 C. Each intensity-weighted size distribution represents the average of six individual DLS analyses, three replicates and at least three independent experiments using automatic optimization of analytical conditions and data treatment by general purpose size analysis.</span>

<h3 id="sect540"><span class="b_heading">Nanoparticle tracking analysis measurements</span></h3>
<span>NTA measurements were performed with a NanoSight LM20 and LM10 (NanoSight, Amesbury, UK), equipped with a 632 nm laser or 532 nm laser. The samples were injected into the sample chamber with sterile syringes. All measurements were performed at room temperature. The samples were diluted to a final concentration of approx. 10<small><sup>8</sup></small> particles per ml with mpH<small><sub>2</sub></small>O, depending on the NMs and media. The software used for recording and analyzing the data was NTA 2.3 and NTA 3.0. All samples were measured for 60 s at five positions. All measurements were performed with at least three independent experiments.</span>

<h3 id="sect545"><span class="b_heading">X-ray diffraction measurements</span></h3>
<span>The XRD measurements were achieved on powder samples using a D5000 diffractometer (Siemens AG, Munich, Germany) in Bragg Brentano geometry. A linear detector, a curved Ge(111) monochromator and Cu K<small><sub></sub></small> radiation (<span class="italic"></span> = 0.1542 nm) were used. The analysis was performed over the 2<span class="italic"></span> range of 10 to 90 and at a step size of 0.02 and scanning speed of 2 per step. The experiments were carried out independently three times.</span>

<h3 id="sect551"><span class="b_heading">Single particle ICP-MS measurements</span></h3>
<span>For single particle analysis of the NM solutions, a quadrupole ICP-MS (Thermo Scientific XSERIES II, Thermo Fisher Scientific, Waltham, MA, USA) with a PFA ST Nebulizer, a quartz cyclonic spray chamber and a 2.5 mm quartz O-ring-free injector (all from ESI Elemental Service &amp; Instruments GmbH, Mainz, Germany) were used. Using the time-resolved analysis (TRA) mode for data acquisition, intensities as a function of time (counts per dwell-time interval) were collected. The acquisition time for each run was set to 60 s with a dwell time (or data acquisition rate) of 3 ms. The gas flow for the plasma, the nebulizer and the auxiliary (all Ar) was set to 13 l min<small><sup>1</sup></small>, 0.89 l min<small><sup>1</sup></small> and 0.7 l min<small><sup>1</sup></small> respectively. The flow rate of the sample was 0.34 ml min<small><sup>1</sup></small>. Data were exported to a spreadsheet for further processing. For data processing, an established procedure according to Pace <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> was followed. Determination of nebulizer efficiency was performed using a described method with reference nanoparticles of known particle size.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> 60 nm gold reference nanoparticles from the U.S. National Institute of Standards and Technology (NIST, RM 8013) were used as reference nanoparticles.</span>

<h3 id="sect561"><span class="b_heading">Small angle X-ray scattering measurements</span></h3>
<span>The measurements were conducted in a flow-through capillary with a Kratky-type instrument (SAXSess, Anton Paar, Austria) at 21  1 C. The SAXSess has a low sample-detector distance of 0.309 m. Deconvolution of the SAXS curves was carried out with the SAXS-Quant software. Curve fitting was performed with the McSAS software (Monte Carlo method, version 1.0.1). The experiments were performed with 120 measurement cycles (each averaged over 10 s). NMs and controls were dispersed according to NanoGenoTOX protocol and diluted into DMEM in a sample concentration of 100 g ml<small><sup>1</sup></small>. Samples were incubated for a time of 24 and 48 h at 37 C and 5% CO<small><sub>2</sub></small> in a cell incubator until sample injection. Stock dispersions were injected directly after ultrasonication.</span>

<h3 id="sect566"><span class="b_heading">Ion beam microscopy</span></h3>
<span>Label-free IBM measurements were performed with the LIPSION Nanoprobe. A 2.25 MeV proton beam was applied by a Singletron particle accelerator. A vacuum with a pressure of 5  10<small><sup>5</sup></small> to 10<small><sup>7</sup></small> torr was applied and the beam was focused to a spot size of around 1 m. For spatial resolved element analysis, micro proton induced X-ray emission (PIXE) and micro Rutherford backscattering spectroscopy (RBS) were recorded simultaneously. The PIXE detector (Canberra, Meriden, CT, USA) consists of a high-purity Ge crystal covered with a 60 m polyethylene layer, which covers the detector for backscattered protons. RBS spectra were detected by a Canberra PIPS detector.</span>

<h3 id="sect571"><span class="b_heading">Confocal Raman microscopy</span></h3>
<span>Spectroscopic analysis of NMs was performed by CRM. A Witec alpha300 confocal Raman spectrometer (Witec GmbH, Germany) with a 532 nm laser with 30 mW power was used. Control4.1 (Witec GmbH, Germany) software was used to record and analyze the spectra. The spectra were recorded at an integration time of 0.15 s per point and with a step size of 250 nm.</span>

<h3 id="sect574"><span class="b_heading">Transmission electron microscopy</span></h3>
<span>The samples were deposited on a 400-mesh copper grid. The grids were prepared by sample adsorption and left free-standing on top of a 0.8 mg ml<small><sup>1</sup></small> solution of NMs for 20 s. Excess solution was removed by placing the grid on a filter paper and the sample was dried for 24 h. For samples in DMEM, the grid was washed by dipping the grid in a droplet of water before. Examination was performed with a JEOL 1400 transmission electron microscope, equipped with a tungsten filament and supplied with GATAN Orius 1000 camera. TEM operated at 80 kV (DMEM) or at 120 kV with magnification of 200<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000.</span>

<h3 id="sect578"><span class="b_heading">Time-of-flight mass spectrometry</span></h3>
<span>Ion images and spectra were acquired using a ToF-SIMS V instrument (ION-TOF GmbH, Mnster, Germany) with a 30 keV nano-bismuth primary ion beam source ([Bi]<span class="italic">x</span><small><sup>(<span class="italic">y</span>+)</sup></small>-cluster ion source with a BiMn emitter). The ion currents were 0.5 pA at 5 kHz using a Faraday cup. A pulse of 0.7 ns from the bunching system resulted in a mass resolution that usually exceeded 6000 (full width at half-maximum) at <span class="italic">m</span>/<span class="italic">z</span> &lt; 500 in positive ion mode. The primary ion dose was controlled below 1012 ions cm<small><sup>2</sup></small> to ensure static SIMS conditions. Charge compensation on the sample was obtained by a pulsed electron flood gun with 20 eV electrons.</span>
<p class="otherpara">The primary ion gun scanned a field of view of 500 m  500 m applying a 512  512 pixel measurement raster. Once the primary ion gun was aligned, a ToF-SIMS mass spectrum was generated by summing the detected secondary ion intensities and plotting them against the mass channels. The data were evaluated using the Surface Lab software (ION-TOF GmbH, Mnster, Germany).</p>


<h2 id="sect588"><span class="a_heading">Results and discussion</span></h2>
<span>We point to the advantages and disadvantages of the methods used and explain whether or not methods can provide complementary results to other techniques. We also analyzed possible dissolution characteristics of different matrix media (BSA and DMEM) for pristine Al<small><sup>0</sup></small> and oxide (Al<small><sub>2</sub></small>O<small><sub>3</sub></small> and TiO<small><sub>2</sub></small>) NMs. Characterization data already published on TiO<small><sub>2</sub></small> were retrieved according to JRC report.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> In conclusion, we will present data where the dissolution of NMs in complex media has an impact on particle media interactions. These results were obtained by ToF-SIMS and show the efficiency of that method for obtaining insights into elemental compositions with TEM, IBM, CRM and ToF-SIMS. An overview about the main characterization techniques and their limits is given in <a title="Select to navigate to table" href="#tab3">Table 3</a>.</span>
<p class="otherpara">A very common and frequently used method for determining the size distribution and polydispersity of NMs in solution is DLS. Fast and easy sample preparation as well as quick measurement give rapid initial indications of the sample. NTA is more appropriate for evaluating polydisperse samples with various aggregate populations, since it is based on single particle tracking as both techniques determine the hydrodynamic diameter. Due to the fixed working ranges of DLS and NTA, the used concentration of media various. Since NTA is a counting method, also proteins would be counted and would significantly lower the hydrodynamic diameter of the NM dispersion. Therefore, the particles, as well as the medium were diluted directly before measurement with mpH<small><sub>2</sub></small>O to avoid agglomeration effects.</p>
<p class="otherpara">For all particles with a primary size of approximately 20 nm, a hydrodynamic diameter between 200 and 270 nm was measured by DLS in DMEM (<a title="Select to navigate to table" href="#tab1">Table 1</a>). In contrast, different aggregate fractions of NMs in DMEM with a mean value of about 150 nm were found by means of NTA (Fig. S1<a title="Select to navigate to footnote" href="#fn1"></a>). This difference can be explained by the fact that NTA is a particle counting system, sensitive to large as well as small fractions in the sample, and DLS is an intensity-weighted system, highly responsive to the large fractions. In all cases, the NTA size distribution is asymmetrical with a steep slope on the left side (small particles) and a gentle slope on the right side, representing the fraction of agglomerates. Additionally, a calculation of number-based size distribution out of intensity-weighted DLS data was done (Table S1<a title="Select to navigate to footnote" href="#fn1"></a>).</p>

<div class="table_caption"><b>Table 1</b> <span id="tab1"><span class="italic">Z</span>-averages with standard deviation (SD) and polydispersity index (PDI) of Al<small><sup>0</sup></small>, Al<small><sub>2</sub></small>O<small><sub>3</sub></small>, NM103 and NM104 NMs in different media determined by dynamic light scattering (DLS). All results represent the average of six repeats. BSA: bovine serum albumin; DMEM: Dulbecco's modified eagle medium; FCS: fetal calf serum</span>
		</div>

<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/></colgroup>





<thead align="left">
<tr align="left" valign="bottom">
<th colspan="5" align="left" valign="bottom" class="border_black">DLS measurements and comparison</th>
</tr>
<tr align="left" valign="bottom">
<th rowspan="2"></th>
<th colspan="2" align="left" valign="bottom" class="border_black">Lab 1 (Malvern)</th>
<th colspan="2" align="left" valign="bottom" class="border_black">Lab 2 (Brookhaven)</th>
</tr>
<tr align="left" valign="bottom">
<th align="left" valign="bottom" class="border_black"><span class="italic">Z</span>-average [nm]</th>
<th align="left" valign="bottom" class="border_black">PDI</th>
<th align="left" valign="bottom" class="border_black"><span class="italic">Z</span>-average [nm]</th>
<th align="left" valign="bottom" class="border_black">PDI</th>
</tr>
</thead>
<tbody align="left">
<tr align="left">
<td colspan="5" valign="top"><span class="bold">Stock solution (0.05% BSA in H</span><small><sub><span class="bold">2</span></sub></small><span class="bold">O)</span></td>
</tr>
<tr align="left">
<td valign="top">Al<small><sup>0</sup></small> NM</td>
<td align="char" char="xb1" valign="top">250  10</td>
<td align="char" char="xb1" valign="top">0.17  0.01</td>
<td align="char" char="xb1" valign="top">270  40</td>
<td align="char" char="xb1" valign="top">0.18  0.02</td>
</tr>
<tr align="left">
<td valign="top">Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NM</td>
<td align="char" char="xb1" valign="top">170  10</td>
<td align="char" char="xb1" valign="top">0.24  0.02</td>
<td align="char" char="xb1" valign="top">210  40</td>
<td align="char" char="xb1" valign="top">0.21  0.06</td>
</tr>
<tr align="left">
<td valign="top">NM103</td>
<td align="char" char="xb1" valign="top">270  10</td>
<td align="char" char="xb1" valign="top">0.28  0.05</td>
<td align="char" char="xb1" valign="top">610  190</td>
<td align="char" char="xb1" valign="top">0.21  0.30</td>
</tr>
<tr align="left">
<td valign="top">NM104</td>
<td align="char" char="xb1" valign="top">220  10</td>
<td align="char" char="xb1" valign="top">0.26  0.03</td>
<td align="char" char="xb1" valign="top">370  90</td>
<td align="char" char="xb1" valign="top">0.15  0.03</td>
</tr>
<tr align="left">
<td colspan="5" valign="top"><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/></td>
</tr>
<tr align="left">
<td colspan="5" valign="top"><span class="bold">DMEM (with 10% FCS)</span></td>
</tr>
<tr align="left">
<td valign="top">Al<small><sup>0</sup></small> NM</td>
<td align="char" char="xb1" valign="top">200  10</td>
<td align="char" char="xb1" valign="top">0.18  0.01</td>
<td align="char" char="xb1" valign="top">220  10</td>
<td align="char" char="xb1" valign="top">0.21  0.02</td>
</tr>
<tr align="left">
<td valign="top">Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NM</td>
<td align="char" char="xb1" valign="top">70  10</td>
<td align="char" char="xb1" valign="top">0.52  0.03</td>
<td align="char" char="xb1" valign="top">230  60</td>
<td align="char" char="xb1" valign="top">0.18  0.07</td>
</tr>
<tr align="left">
<td valign="top">NM103</td>
<td align="char" char="xb1" valign="top">240  20</td>
<td align="char" char="xb1" valign="top">0.24  0.01</td>
<td align="char" char="xb1" valign="top">270  10</td>
<td align="char" char="xb1" valign="top">0.25  0.01</td>
</tr>
<tr align="left">
<td valign="top">NM104</td>
<td align="char" char="xb1" valign="top">190  10</td>
<td align="char" char="xb1" valign="top">0.28  0.02</td>
<td align="char" char="xb1" valign="top">230  10</td>
<td align="char" char="xb1" valign="top">0.18  0.02</td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><br/>
<p class="otherpara">In the case of NM103 and NM104 dispersed in 0.05% BSA water solution, a very high mean value and a large error was measured by the Brookhaven device, while NTA exhibits broad distributions with some fractions and a mean value of around 180 nm. This can be explained by unstable particle agglomerates, which are observed by the DLS.</p>
<p class="otherpara">Comparing the two DLS devices, one can observe that the Malvern DLS detects 15% smaller particles than the Brookhaven machine in 0.05% BSA water solution as well as in DMEM. This systematic deviation could be related to different detection angles of scattered intensity, 90 for Brookhaven and 173 for Malvern, since larger particles mostly scatter light at forward angles. Backscattering at 173 will not overestimate larger particles as much as measurements at 90.</p>
<p class="otherpara">In the case of NM103 and 104 in BSA and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> in DMEM, the results were not consistent for the two devices. <span class="italic">Z</span>-averaged diffusion coefficients were calculated by applying the cumulant method, which is applicable to polydisperse and non-multimodal systems. In contrast, NTA results show the presence of different particle populations. The applied cumulant algorithms are not suitable in certain cases for getting reliable DLS results. Furthermore, it should be kept in mind that the inversion of the DLS autocorrelation function is part of a poorly formulated mathematical problem. It works quite well in the case of monodisperse or low polydisperse particles. Applications of multiangle DLS and sophisticated algorithms are thus necessary to obtain trustworthy results of NM size distributions by means of DLS.<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31,32</span></sup></a></p>
<h3 id="sect704"><span class="b_heading">Investigations on primary particle size</span></h3>
<span>With the European commission's definition of a nanomaterial, that is, a nanomaterial should contain 50% or more particles in number size distribution with one or more size dimensions between 1100 nm, it is obviously necessary to obtain information about the primary particle size of the investigated particles. TEM as well as SAXS can be used to solve this problem.</span>
<p class="otherpara">Looking at advantages and disadvantages of TEM, the following issues should be considered. Since TEM measurements are performed under high vacuum, only dried samples are observed. For this reason, TEM is not representative of the sample in its solution state. In particular, agglomeration and coffee stain effect<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a> may occur during the drying process, resulting in a non-homogenously covered surface. Size distributions can be determined from TEM pictures by measuring the size of each particle using image analysis software.<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> The size of individual nanoparticles may be difficult to extract from agglomerated samples but, recently, some implementations have been proposed.<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> In addition, for irregular particles with ill-defined shapes, which dimension should be taken? In practice, size analysis can be time-consuming and TEM generally yields a poor statistical representation of the sample.<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> Electron microscopy also enables chemical and crystallographic analysis of the particles. The contrast in TEM is directly linked to the atomic number of electrons, heavier atoms giving higher contrast. This is an advantage when observing metal nanoparticles in a biological environment. However, care should be taken for mineral salts from buffers that may precipitate on the grid when drying. Aqueous washing of the grid after sample deposition might be appropriate in this case.</p>
<p class="otherpara"><a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a> shows the dispersion in water with 10% BSA. The shape of Al<small><sup>0</sup></small> NMs is globally spherical with rod-like excrescences and the primary particle size varies between 250 nm, which matches the manufacturer specification. The Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs are not spherical but have rather a needle-like shape. The width is about 10 nm, while the longest dimension varies a bit more between 2050 nm. The shape of TiO<small><sub>2</sub></small> NM104 is rod-like, with a width of approx. 10 nm and length of 2050 nm (Fig. S2<a title="Select to navigate to footnote" href="#fn1"></a>). Working with a more complex medium, such as CCM, was challenging for TEM analysis. However, we were able to image the particle in DMEM, with very little differences compared to BSA dispersion. Using relatively low voltage, the protein coating can be observed (see Fig. S2<a title="Select to navigate to footnote" href="#fn1"></a>).</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f3.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">Comparison of TEM pictures of Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs in BSA solution after applying the dispersion protocol.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">SAXS was used for characterization of the primary particle size of NMs. SAXS allows to analyze a broader variety of different sample types than most other techniques. Compared to TEM, the samples can easily be investigated <span class="italic">in situ</span>. Additionally, SAXS provides statistically more reliable data for particle size distribution quantification since more than 10<small><sup>6</sup></small> particles are typically measured, in contrast to TEM, where rarely more than a few hundred particles are counted. Size distribution of NMs can be quantified in the range of 1 to 100 nm if the shape is known from a complementary technique like TEM. The classical radius of gyration (Guinier radius) is accessible in any case.</p>
<p class="otherpara">In this study, we observed the size parameters of the particles in BSA as a stock solution and after addition in DMEM at different times of <span class="italic">t</span> = 0 h, 24 h and 48 h (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>). Since the particle cores scatter much stronger than the surrounding particle shell, the shell becomes practically invisible. Therefore, SAXS determines the size distribution of the core radii. The resulting distributions of the particles' stock solutions are shown in <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>. Their corresponding SAXS curves are displayed in the ESI (Fig. S3S5<a title="Select to navigate to footnote" href="#fn1"></a>). The accessible size range of the radii is given by the range of the scattering vector <span class="italic">q</span>: <span class="italic">R</span><small><sub>min</sub></small> = /<span class="italic">q</span><small><sub>max</sub></small> and <span class="italic">R</span><small><sub>max</sub></small> = /<span class="italic">q</span><small><sub>min</sub></small>. In the present case of the <span class="italic">q</span>-range of 0.1 nm<small><sup>1</sup></small> &lt; <span class="italic">q</span> &lt; 6 nm<small><sup>1</sup></small> corresponds to radii of 30 nm &gt; <span class="italic">R</span> &gt; 0.5 nm.</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f4.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Volume-weighted radii distributions of Al<small><sup>0</sup></small> NMs (a) and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs (b) derived from SAXS measurements. The displayed radii distribution (right panel, black bars) and cumulative fraction presentation (blue line) correspond to the NMs in stock solution (BSA). Left panel shows particle distribution in cumulative fraction presentation at different stages: in stock solution (blue line) and in DMEM after 0 h (red line), 24 h (green line) and 48 h (orange line).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">The Al<small><sup>0</sup></small> NMs showed a broad size distribution including primary particles with radii &gt; 10 nm. Since the detection limit is 30 nm (radius) in this case, bigger aggregates cannot be detected directly. However, from the steep slope of the SAXS curve (Fig. S3<a title="Select to navigate to footnote" href="#fn1"></a>) at low <span class="italic">q</span> values, it can be assumed that bigger aggregates are present. The defined characterization of these particle aggregates has to be performed using a complementary method like TEM. The inset in <a title="Select to navigate to figure" href="#imgfig4">Fig. 4a</a> shows that the radii distribution of the Al<small><sup>0</sup></small> NMs did not change significantly either after addition in DMEM or 24 h and 48 h thereafter. In contrast to the Al<small><sup>0</sup></small> NMs, the Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NM in BSA displays a distribution which consists of small primary particles and aggregates. The sample shows an amount of 75% primary particles with a mean radius of 7.1  0.5 nm. The detected aggregates display radii of &gt; 10 nm. Upon the addition in DMEM, the radii distribution shifts slightly to higher radii of 8.4  0.2 nm. These characteristics did not change significantly over the time of 48 h in DMEM.</p>
<p class="otherpara">No particles were detected in the ionic control substance AlCl<small><sub>3</sub></small>. In contrast, immediately after addition in DMEM, nanoscaled particles with sizes of 130 nm were observed. Since all curves are background-corrected with the respective solvent control, solvent effects can be excluded and the particles derive directly from the aluminum species. In conclusion, SAXS yields the size distribution of nanoparticles and its changes in DMEM.</p>
<p class="otherpara">SP-ICP-MS is another technique to determine primary particle sizes. The fundamental assumption behind this technique is that, at a sufficiently short dwell time and low particle number concentration, a pulse will represent a single particle event.</p>
<p class="otherpara">There is a direct correlation between the number of pulses and the number concentration of particles (particle number per volume). With the intensity of the pulse (<span class="italic">i.e.</span> height) and assumptions about the particle geometry, the particle size through particle mass can be determined.</p>
<p class="otherpara">Aside from single pulses, there is always a background, which originates from the ionic part of the analyzed sample. In addition to primary particle size, information about the dissolution rate of a NM sample can be achieved. While quantification is more difficult, a qualitative assessment of ions released can be inferred. For Al<small><sup>0</sup></small>, a broad distribution (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>, left) up to 200 nm is observed compared to Al<small><sub>2</sub></small>O<small><sub>3</sub></small> (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>, right). This fits very well with the data obtained from TEM analysis. Compared to the Au NIST reference material, which shows almost no dissolution, a higher background for Al<small><sup>0</sup></small> NMs was detected, indicating potential ion release.</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f5.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">Top: primary particle size distribution of Al<small><sup>0</sup></small> in 0.05% BSA determined by SP-ICP-MS; bottom: Primary particle size distribution of Al<small><sub>2</sub></small>O<small><sub>3</sub></small> in 0.05% BSA determined by SP-ICP-MS.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>

<h3 id="sect778"><span class="b_heading">Particle surface  impact on dissolution in complex media</span></h3>
<span>Al<small><sup>0</sup></small> NMs usually become quickly passivated at the surface by the formation of an oxide layer. This event is likely to change the dissolution behavior and the overall physicochemical characteristics of Al<small><sup>0</sup></small> NMs. For comparison, we also investigated Al<small><sub>2</sub></small>O<small><sub>3</sub></small> particles of similar size. To prevent the surface oxidation of Al<small><sup>0</sup></small> NMs, its processing and handling in an inert gas atmosphere was evaluated.</span>
<p class="otherpara">Investigation by XRD revealed a thin aluminum oxide layer at the surface of the Al<small><sup>0</sup></small> NMs (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>, left). This was confirmed by TEM measurements and has also already been shown in the literature for another Al<small><sup>0</sup></small> NM by TEM measurements (2.5 nm oxide layer).<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> The occurrence of an oxide layer can be explained by partial passivation of the material due to manufacturer's processing.</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f6.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">Top: XR-diffractogram of Al<small><sup>0</sup></small> NMs red: database entry for Al; blue; (green: database entry for Al<small><sub>2</sub></small>O<small><sub>3</sub></small>), space group: <span class="italic">Fm</span>3<span class="italic">m</span>, lattice constants: <span class="italic">a</span> = 4.0494 ; bottom: diffractogram of Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs (red: database entry for Al<small><sub>2</sub></small>O<small><sub>3</sub></small>), space group: <span class="italic">Fd</span>3<span class="italic">m</span>, lattice constants: <span class="italic">a</span> = 7.906 .</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">In comparison with Al<small><sup>0</sup></small> NMs, the diffractogram of Al<small><sub>2</sub></small>O<small><sub>3</sub></small> particles showed clear differences (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>, right). The diffractogram of Al<small><sup>0</sup></small> NMs showed a higher intensity for the aluminum peaks, <span class="italic">e.g.</span> at 38, 45, 66 and 78 (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>, left) as compared to Al<small><sub>2</sub></small>O<small><sub>3</sub></small> peaks, <span class="italic">e.g.</span> at 37, 46 and 67 (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>, right). This demonstrates that even though there is an Al oxide layer at the surface of the Al NMs they are not completely oxidized.</p>
<p class="otherpara">To confirm these results, we used electron energy loss spectroscopy (EELS). Here the sample becomes exposed to an electron beam with defined kinetic energy. Some electrons undergoing inelastic scattering are collected in a detector. The loss of energy reflects the chemical composition of the sample.</p>
<p class="otherpara">Analysis of Al<small><sup>0</sup></small> NMs in 0.05% BSA (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7a</a>) revealed a coreshell structure. By means of EELS, it was proven that the shell is rich in oxygen (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7b</a>), while the core consists of elemental aluminum (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7c</a>). The TEM results indicate an oxide layer of about 2 to 5 nm.</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f7.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">(a): TEM picture of Al<small><sup>0</sup></small> NMs, 200 k magnification; (b) oxygen mapping of left TEM picture; (c) Aluminum mapping of left TEM picture. (d) TEM picture of Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs, 200 k magnification; (e) aluminum mapping of left TEM picture; (f) oxygen mapping of left TEM picture.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">Elemental mapping of Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs showed a quite different picture compared to pure Al<small><sup>0</sup></small> NMs (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7d</a>). The distribution of aluminum and oxygen was homogenous over all NMs (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7e and f</a>). These results show that Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs are fully oxidized while elemental Al<small><sup>0</sup></small> NMs were passivated by an oxide layer. It is also visible because Al<small><sup>0</sup></small> NMs are dark grey to black, while Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs are white.</p>
<p class="otherpara">The results from surface investigations <span class="italic">via</span> XRD and EELS suggest a different solubility of Al<small><sup>0</sup></small> compared to Al<small><sub>2</sub></small>O<small><sub>3</sub></small>. Indeed, the thin oxide layer on Al<small><sup>0</sup></small> allows the release of ions while the fully oxidized Al<small><sub>2</sub></small>O<small><sub>3</sub></small> particle should be much more inert. Nevertheless, due to the highly specific surface, a higher solubility compared to Al<small><sub>2</sub></small>O<small><sub>3</sub></small> bulk material could be expected as more potentially ion releasing Al atoms are present on the surface.</p>

<h3 id="sect851"><span class="b_heading">Influence of particle composition and impurity patterns</span></h3>
<span>Compared to particle size and surface, the particle composition has a much higher impact on the solubility and dissolution behavior of NMs. One of the most widely used methods due to its high sensitivity, broad range for nearly all elements and detection limits down to the sub-ppb level is the ICP-MS. Unknown samples can be not only detected but also quantified. For more complex samples, microwave-assisted digestion prior to ICP-MS analysis could be performed. In this study, we confirmed the already known composition of the NMs by XRD and EELS measurements as well as ICP-MS analysis. For the assessment of particle toxicity, it is also important to take impurities into account which might alter ion release behavior. Due to element-specific RBS and PIXE, IBM became the method of choice for these investigations. The main impurities of aluminum-containing NMs were phosphorus, sulfur and chlorine, which could lead to a different ion release compared to pure materials (see Fig. S6<a title="Select to navigate to footnote" href="#fn1"></a>).</span>

<h3 id="sect855"><span class="b_heading">Solubility investigations on Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs</span></h3>
<span>To verify our hypothesis that Al<small><sup>0</sup></small> NMs are more soluble than Al<small><sub>2</sub></small>O<small><sub>3</sub></small>, we performed ion release experiments in stock dispersions and DMEM. Adjusting the pH value was not necessary due to almost neutral pH values of 7.3 in stock dispersion compared to 7.2 in DMEM. Keeping in mind that aluminum is an amphoteric material due to the aluminum aqua complexes, one can assume that dissolution of Al<small><sup>0</sup></small> NMs in complex media is highly pH-dependent. Compared to an acid, the aluminum aqua complex reacts as a base and <span class="italic">vice versa</span>. At a neutral pH value, reactivity is very low, thus only low ion release should occur. The possible aluminum aqua complexes at the different pH values are shown in Fig. S7.<a title="Select to navigate to footnote" href="#fn1"></a></span>
<p class="otherpara">The ion release for both Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs after one hour was very low, about 0.20.4% in BSA and 0.30.5% in DMEM (with 10% FCS) (<a title="Select to navigate to table" href="#tab2">Table 2</a>). As already described above, this was expected at a neutral pH value. For <span class="italic">in vitro</span> experiments, this will mean that the effects will originate mainly from particles and not from Al ions. The application of an artificial digestion procedure for mimicking <span class="italic">in vivo</span> situation showed, that the particle dissolution and complexation behavior was quite different in all three studied gastrointestinal compartments.<a title="Select to navigate to references" href="#cit24"><sup><span class="sup_ref">24</span></sup></a> After no significant changes in the saliva, the gastric environment leads to a significant increase of the dissolution rate as well as very strong agglomeration of NMs. The addition of intestinal fluid results in a nearly neutral pH value which leads to a decrease in the dissolution rate, a deagglomeration of particles and even <span class="italic">de novo</span> particle formation in ionic aluminum control.</p>

<div class="table_caption"><b>Table 2</b> <span id="tab2">Ion release of Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs compared to recovery of AlCl<small><sub>3</sub></small> in 0.05% BSA and DMEM</span>
		</div>

<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>




<thead align="left">
<tr align="left" valign="bottom">
<th colspan="2"></th>
<th align="left" valign="bottom" class="border_black">Ion release in BSA [%]</th>
<th align="left" valign="bottom" class="border_black">Ion release in DMEM [%]</th>
</tr>
</thead>
<tbody align="left">
<tr align="left">
<td rowspan="2" valign="top">Al<small><sup>0</sup></small> NMs</td>
<td valign="top">10 g ml<small><sup>1</sup></small></td>
<td align="char" char="xb1" valign="top">0.4  0.1</td>
<td align="char" char="xb1" valign="top">0.5  0.1</td>
</tr>
<tr align="left">
<td valign="top">100 g ml<small><sup>1</sup></small></td>
<td align="char" char="xb1" valign="top">0.3  0.1</td>
<td align="char" char="xb1" valign="top">0.4  0.1</td>
</tr>
<tr align="left">
<td rowspan="2" valign="top">Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs</td>
<td valign="top">10 g ml<small><sup>1</sup></small></td>
<td align="char" char="xb1" valign="top">0.4  0.1</td>
<td align="char" char="xb1" valign="top">1.4  0.1</td>
</tr>
<tr align="left">
<td valign="top">100 g ml<small><sup>1</sup></small></td>
<td align="char" char="xb1" valign="top">0.2  0.1</td>
<td align="char" char="xb1" valign="top">0.4  0.1</td>
</tr>
<tr align="left">
<td rowspan="2" valign="top">AlCl<small><sub>3</sub></small></td>
<td valign="top">10 g ml<small><sup>1</sup></small></td>
<td align="char" char="xb1" valign="top">140  9</td>
<td align="char" char="xb1" valign="top">112  4</td>
</tr>
<tr align="left">
<td valign="top">100 g ml<small><sup>1</sup></small></td>
<td align="char" char="xb1" valign="top">94  4</td>
<td align="char" char="xb1" valign="top">66  3</td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><br/>

<div class="table_caption"><b>Table 3</b> <span id="tab3">Main characterization techniques and their limits<i><small><sup style="color: red;"><a title="" href="#tab3fna">a</a></sup></small></i></span>
		</div>

<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/></colgroup>











<thead align="left">
<tr align="left" valign="bottom">
<th colspan="2" align="left" valign="bottom" class="border_black">Methods</th>
<th align="left" valign="bottom" class="border_black">TEM</th>
<th align="left" valign="bottom" class="border_black">EELS-TEM</th>
<th align="left" valign="bottom" class="border_black">XRD</th>
<th align="left" valign="bottom" class="border_black">SAXS</th>
<th align="left" valign="bottom" class="border_black">SP-ICP-MS</th>
<th align="left" valign="bottom" class="border_black">ICP-MS</th>
<th align="left" valign="bottom" class="border_black">ToF-SIMS</th>
<th align="left" valign="bottom" class="border_black">IBM, atom number %</th>
<th align="left" valign="bottom" class="border_black">CRM</th>
</tr>
</thead>
<tfoot align="left">
<tr align="left">
<th colspan="11"><a id="tab3fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Abbreviations: TEM  transmission electron microscopy; EELS  electron energy loss spectroscopy; XRD  X-ray diffraction; SAXS  small angle X-ray scattering; SP-ICP-MS  single particle inductively coupled plasma mass spectrometry; ICP-MS  inductively coupled plasma mass spectrometry; ToF-SIMS  time of flight mass spectrometry; IBM  Ion beam microscopy; CRM  confocal Raman microscopy; BSA  bovine serum albumin; DMEM  Dulbecco's modified eagle medium; ppm  parts per million.</span><a id="tab3fnb"><span class="tfootnote">b</span></a> <span class="sup_inf">Data taken from (<a title="Select to navigate to references" href="#cit30">ref. 30</a>).</span></th>
</tr>
</tfoot>
<tbody align="left">
<tr align="left">
<td rowspan="2" valign="top">Aluminum</td>
<td valign="top">Results</td>
<td valign="top">Primary particle size and shape: 250 nm, nearly spherical</td>
<td valign="top">Coreshell structure, thin (25 nm) oxide layer</td>
<td valign="top">Aluminum surface; partially oxidized</td>
<td valign="top">Particle radius: &gt; 10 nm</td>
<td valign="top">Primary particle size: 5480 nm</td>
<td valign="top">Ion release: 0.20.5%</td>
<td valign="top">Particle-amino acid agglomerates</td>
<td valign="top">Impurities: P (1%); biocorona: S (5%), protein; adsorption from DMEM; Ca<small><sub>3</sub></small>(PO<small><sub>4</sub></small>)<small><sub>2</sub></small> coating: P (3%)</td>
<td valign="top">No data available</td>
</tr>
<tr align="left">
<td valign="top">Limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">Light core of Al hinders detailed measurements</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No limits known; results show limit of 54 nm</td>
<td valign="top">Detection limit of 10 g g<small><sup>1</sup></small> in DMEM</td>
<td valign="top">No element-specific limits</td>
<td valign="top">From Al to higher atomic mass</td>
<td valign="top">Raman active molecules</td>
</tr>
<tr align="left">
<td rowspan="2" valign="top">Al<small><sub>2</sub></small>O<small><sub>3</sub></small></td>
<td valign="top">Results</td>
<td valign="top">Primary particle size and shape: 10  2050 nm, grain-like shape</td>
<td valign="top">Fully oxidized particle</td>
<td valign="top">Fully oxidized surface</td>
<td valign="top">Primary particle radius: 7.1 nm aggregates' radius: &gt; 10 nm</td>
<td valign="top">Primary particle size: 5080 nm</td>
<td valign="top">Ion release: 0.20.4%</td>
<td valign="top">Particle-amino acid agglomerates; polyoxo-aluminates</td>
<td valign="top">Impurities: S (0.2%); biocorona: S (1%), protein adsorption from DMEM; Ca<small><sub>3</sub></small>(PO<small><sub>4</sub></small>)<small><sub>2</sub></small> coating: P<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>(1%)</td>
<td valign="top">No data available</td>
</tr>
<tr align="left">
<td valign="top">Limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">Light core of Al hinders detailed measurements</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No limits known; results show limit of 50 nm</td>
<td valign="top">Detection limit of 10 ng g<small><sup>1</sup></small> in DMEM</td>
<td valign="top">No element-specific limits</td>
<td valign="top">From Al to higher atomic mass</td>
<td valign="top">Raman active molecules</td>
</tr>
<tr align="left">
<td rowspan="2" valign="top">TiO<small><sub>2</sub></small> NM103</td>
<td valign="top">Results</td>
<td valign="top">Primary particle size and shape: 20100 nm, nearly spherical<i><small><sup style="color: red;"><a title="" href="#tab3fnb">b</a></sup></small></i></td>
<td valign="top">No data available</td>
<td valign="top">Rutile; crystallite size: 20 nm<i><small><sup style="color: red;"><a title="" href="#tab3fnb">b</a></sup></small></i></td>
<td valign="top">Gyration diameter: 26 nm<i><small><sup style="color: red;"><a title="" href="#tab3fnb">b</a></sup></small></i></td>
<td valign="top">Primary particle size: 60100 nm</td>
<td valign="top">No solubility in BSA, low soluble in DMEM<i><small><sup style="color: red;"><a title="" href="#tab3fnb">b</a></sup></small></i></td>
<td valign="top">Particle-amino acid agglomerates</td>
<td valign="top">Impurities: Al (7%), S (0.6%); biocorona: S (0.5%), protein; exchange in DMEM; Ca<small><sub>3</sub></small>(PO<small><sub>4</sub></small>)<small><sub>2</sub></small> coating: P (0.5%)</td>
<td valign="top">Decrease of aliphatic/aromatic compounds on NM surface; higher protein exchange in DMEM</td>
</tr>
<tr align="left">
<td valign="top">Limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">Limit of 90 nm<a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a></td>
<td valign="top">Detection limit of 7.5 ng g<small><sup>1</sup></small> in DMEM</td>
<td valign="top">No element-specific limits</td>
<td valign="top">From Al to higher atomic mass</td>
<td valign="top">Raman active molecules</td>
</tr>
<tr align="left">
<td rowspan="2" valign="top">TiO<small><sub>2</sub></small> NM104</td>
<td valign="top">Results</td>
<td valign="top">Primary particle size and shape: 1050 nm, nearly spherical</td>
<td valign="top">No data available</td>
<td valign="top">Rutile; crystallite size: 21 nm<i><small><sup style="color: red;"><a title="" href="#tab3fnb">b</a></sup></small></i></td>
<td valign="top">Gyration diameter: 26 nm<i><small><sup style="color: red;"><a title="" href="#tab3fnb">b</a></sup></small></i></td>
<td valign="top">Primary particle size: 60100 nm</td>
<td valign="top">No solubility in BSA, low soluble in DMEM<i><small><sup style="color: red;"><a title="" href="#tab3fnb">b</a></sup></small></i></td>
<td valign="top">Particle-amino acid agglomerates</td>
<td valign="top">Impurities: Al (6%), S (0.7%); biocorona: S (1%); protein; exchange in DMEM; Ca<small><sub>3</sub></small>(PO<small><sub>4</sub></small>)<small><sub>2</sub></small> coating: P (0.8%)</td>
<td valign="top">Decrease of aliphatic/aromatic compounds on NM surface; higher protein exchange in DMEM</td>
</tr>
<tr align="left">
<td valign="top">Limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">No element-specific limits</td>
<td valign="top">Limit of 90 nm<a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a></td>
<td valign="top">Detection limit of 7.5 ng g<small><sup>1</sup></small> in DMEM</td>
<td valign="top">No element-specific limits</td>
<td valign="top">From Al to higher atomic mass</td>
<td valign="top">Raman active molecules</td>
</tr>
<tr align="left">
<td colspan="2" valign="top">General limits</td>
<td valign="top">1 nm to 2 m; only dried samples</td>
<td valign="top">Organic matrix contaminates sample</td>
<td valign="top">Only powder and crystallite samples</td>
<td valign="top"><span class="italic">r</span> = 0.530 nm (depending on the available <span class="italic">q</span>-range)</td>
<td valign="top">Only highly diluted samples, limits: Au 10 nm, Ag 20 nm</td>
<td valign="top">Sub ng g<small><sup>1</sup></small> level for most elements</td>
<td valign="top">No single particles visible, only agglomerates</td>
<td valign="top">Only dry samples under vacuum condition, detection limit: few ppm</td>
<td valign="top">Only raman active molecules observable</td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><br/>

<h3 id="sect1101"><span class="b_heading">Particle-CCM interactions</span></h3>
<span>It is of major interest to evaluate the elemental and molecular changes on the particle surface during sample preparation for cell experiments (Fig. S8 and S9<a title="Select to navigate to footnote" href="#fn1"></a>). Additionally, two steps were chosen to investigate the surface modification of particles during the sample preparation process. Firstly, the NMs were dispersed. In this step, NMs were covered with albumin corona and are referred to as BSA in the text below. The second step was to investigate the NMs under cell culture conditions, diluted in DMEM. After each step, the particles were washed three times in Millipore water. The results of the element analysis, performed by IBM, are shown in <a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>.</span>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f8.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">IBM element analysis of NM103, NM104, Al and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs. NMs as purchased and diluted in water (pure), NMs with albumin corona in H<small><sub>2</sub></small>O (BSA) and NMs under cell exposure conditions (DMEM). The graph demonstrates the ratio of atoms of several elements compared to 100 atoms of titanium or aluminum.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">The variation of the element content represents changes on the surface of NMs, <span class="italic">e.g.</span> attachment of amino acids, fatty acids, proteins and/or ions to the surface of the NMs. An increase in the Ca and P amount was observed for all studied NMs exposed to DMEM. The highest content of these elements was found in the case of Al NMs. It is suggested that calcium and phosphate ions interact strongly with the albumin corona of NMs and build a calcium phosphate layer on the particle surface. It is known that calcium phosphate has a high affinity to proteins and can increase the efficiency of uptake.<a title="Select to navigate to reference" href="#cit37"><sup><span class="sup_ref">37,38</span></sup></a></p>
<p class="otherpara">The amount of sulfur on the NMs was analyzed under different conditions. Sulfur was found in association with proteins forming a corona around the NMs. Al and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs acquire more proteins on their surface when exposed to DMEM as compared to dispersion in 0.05% BSA (Fig. S11<a title="Select to navigate to footnote" href="#fn1"></a>). In the case of Al and Al<small><sub>2</sub></small>O<small><sub>3,</sub></small> the proteins from the culture medium contribute to additional adsorption compared to the existing albumin corona of the particles alone. DMEM shows more physiologic relevant conditions. The salts might induce shielding effects on the protein, so the amount of proteins is likely to be increased. Due to the larger variety of available proteins, a more complex corona will self-assemble on the surface of the particles. In contrast, a decreased amount of proteins was found in the case of NM103 and 104 NMs exposed to DMEM. Substitution of the relatively dense BSA on the particle surface with less dense proteins from the culture media is suggested. This exchange of proteins has a stabilizing effect on TiO<small><sub>2</sub></small> NMs in DMEM and results in a strong decrease of aggregate size in DMEM media (<a title="Select to navigate to table" href="#tab1">Table 1</a>). This finding is also supported by CRM investigations (Fig. S10<a title="Select to navigate to footnote" href="#fn1"></a>).</p>
<p class="otherpara">ToF-SIMS was used to image Al NMs and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs as well as nanoparticle agglomerates in cell culture medium and to assess the chemical composition of the nanoparticle agglomerates. The analyses revealed nanoparticle-specific agglomerates, consisting of polyoxo-aluminum complexes, aluminum(<span class="small_caps">III</span>)serine and amino acid aluminate complexes (leucine and phenylalanine aluminate). These complexes were not observed when ionic AlCl<small><sub>3</sub></small> was added to the cell culture medium. ToF-SIMS images revealed a rather homogenous agglomerate distribution with only a slight accumulation of Al NMs in certain areas (see yellow circles in <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>, top) and aluminum(<span class="small_caps">III</span>)serine and polyoxo-aluminum complexes in others (see green circles in <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>, top). While the aluminates co-locate with both areas, aluminates can be found in regions with predominantly Al NMs and in areas with predominantly aluminum(<span class="small_caps">III</span>)serine and polyoxo-aluminum. <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a> shows the ToF-SIMS image for Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs (bottom). Larger agglomerate areas (green circles in <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>, bottom), where aluminum(<span class="small_caps">III</span>)serine and polyoxo-aluminum complexes were present in higher amounts, can be distinguished from areas with predominantly smaller nanoparticle agglomerates made of Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs (see purple circles in <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>, bottom).</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2018/RA/c8ra00205c/c8ra00205c-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c8ra00205c-f9.tif" src="/image/article/2018/RA/c8ra00205c/c8ra00205c-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">ToF-SIMS reconstructed ion overlay image (500 m  500 m) of Al NM (top) and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> (bottom) agglomerates of different chemical entities from a DMEM solution; yellow: Al NM, purple: Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NM, green: aluminum(<span class="small_caps">III</span>)serine; orange: phenylalanine aluminate; red: leucine aluminate; blue: polyoxo-aluminum complex.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">In DMEM with Al NMs, areas where predominantly Al NMs localize, depicted as yellow circles in <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>, top, are distinct from areas where polyoxo-aluminum complexes and aluminum(<span class="small_caps">III</span>)serine particles localize (green circles). Generally, smaller agglomerates of different chemical entities, Al NMs, aluminum(<span class="small_caps">III</span>)serine, leucine aluminate, phenylalanine aluminate and polyoxo-aluminum complexes, which do not co-localize in the same area, were observed. In addition to areas where predominantly Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs (purple) localize, Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs in DMEM show a similar pattern with areas where all chemical entities co-localize but are clearly separated from each other. This indicates a starting mineralization of the larger agglomerates, where different chemical entities co-localize and form mixed agglomerates of Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs, amino acids and aluminum salts. Further agglomerate compositions and chemical entities were detected (Fig. S1114<a title="Select to navigate to footnote" href="#fn1"></a>). For ToF-SIMS measurements for TiO<small><sub>2</sub></small> NM103 and 104 in DMEM see Fig. S1518.<a title="Select to navigate to footnote" href="#fn1"></a></p>


<h2 id="sect1160"><span class="a_heading">Conclusions</span></h2>
<span>An extensive characterization of NMs in their different states, from dispersion to <span class="italic">in vitro</span> conditions, should precede any toxicological testing. In this study, we have characterized two different types of NMs, one metal and one metal oxide. For this purpose, we used a wide range of complementary analytical techniques to characterize particle size, surface and composition in more detail. An in-depth insight into the dissolution, one of the most important determinants of NM toxicity, was achieved by investigation by the examples of Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs. A very low dissolution rate and a small percentage of ion release was observed at a neutral pH value, while higher rates for acidic environment we already reported earlier.<a title="Select to navigate to references" href="#cit24"><sup><span class="sup_ref">24</span></sup></a> A significant difference between the surface and the extent of oxidation of Al<small><sup>0</sup></small> and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NM forms was detected. In contrast to the fully oxidized Al<small><sub>2</sub></small>O<small><sub>3</sub></small> with a rather homogenous distribution of Al and O atoms, Al<small><sup>0</sup></small> NM showed a coreshell structure with an oxide layer only a few nm thick. This was proven with XRD and the EELS technique, which emphasizes the benefit of using different techniques to get reliable results. It was demonstrated with the help of ToF-SIMS that both aluminum forms are subject to a surprisingly different complexation in biological media: while Al<small><sup>0</sup></small> NMs were shown to form complexes with amino acids, Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs mostly formed polyoxo-complexes out of two or more Al<small><sub>2</sub></small>O<small><sub>3</sub></small> molecules. Based on the collected results, all of the investigation methods applied have their own benefits. Thus, with the methods described in this study focusing on size, surface and complex formation, dissolution investigations can perhaps become a bit more predictable. Right now, the recommended limit for aluminum release from food is 5 mg kg<small><sup>1</sup></small> food. The various aluminum forms, <span class="italic">e.g.</span> ions, micro- and nanomaterials, pure Al<small><sup>0</sup></small> or Al<small><sub>2</sub></small>O<small><sub>3</sub></small>, are not differentiated here. With this study, it becomes obvious that there should be differentiation for the different forms, as they may be taken up and react differently. For future analytical investigations on aluminum-containing NMs, we propose considering not only Al<small><sub>2</sub></small>O<small><sub>3</sub></small> but also Al<small><sup>0</sup></small> NMs and their interaction with the corresponding ions.</span>

<h2 id="sect1187"><span class="a_heading">Conflicts of interest</span></h2>
<span>There are no conflicts to declare.</span>


<h2 id="sect1191"><span class="a_heading">Acknowledgements</span></h2><span>This French-German bilateral project SolNanoTOX was funded by the German Research Foundation (DFG, Project ID: DFG (FKZ LA 3411/1-1 and LA 1177/9-1)) and the French National Research Agency (ANR, Project ID: ANR-13-IS10-0005). The authors would like to thank Simone Rolf from the Federal Institute for Materials Research and Testing (BAM) for the XRD measurements, Claudia Kstner from BAM for the SAXS measurements and Dr Uwe Mhle from the Fraunhofer Institute for Ceramic Technologies and Systems (IKTS) for the TEM measurements with elemental information.</span>
<span id="sect1193"><h2 id="sect1190"><span class="a_heading">Notes and references</span></h2></span><ol type="1"><li><span id="cit1">A. Moores and F. Goettmann, <span class="italic">New J. Chem.</span>, 2006, <span class="bold">30</span>, 11211132<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B604038C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B604038C">RSC</a>.</span></li><li><span id="cit2">K. L. Kelly, E. Coronado, L. L. Zhao and G. C. Schatz, <span class="italic">J. Phys. Chem. B</span>, 2003, <span class="bold">107</span>, 668677<a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp026731y" title="DOI Link to resource 10.1021/jp026731y">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38Xps1Ghur0%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit3">D. M. Brown, M. R. Wilson, W. MacNee, V. Stone and K. Donaldson, <span class="italic">Toxicol. Appl. Pharmacol.</span>, 2001, <span class="bold">175</span>, 191199<a target="_blank" class="DOILink" href="https://doi.org/10.1006/taap.2001.9240" title="DOI Link to resource 10.1006/taap.2001.9240">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXmsl2nu7o%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=11559017%5Buid%5D" title="PubMed Link to resource 11559017">PubMed</a>.</span></li><li><span id="cit4">S. Hussain, S. Boland, A. Baeza-Squiban, R. Hamel, L. C. J. Thomassen, J. A. Martens, M. A. Billon-Galland, J. Fleury-Feith, F. Moisan, J. C. Pairon and F. Marano, <span class="italic">Toxicology</span>, 2009, <span class="bold">260</span>, 142149<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.tox.2009.04.001" title="DOI Link to resource 10.1016/j.tox.2009.04.001">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXmtlOitbs%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19464580%5Buid%5D" title="PubMed Link to resource 19464580">PubMed</a>.</span></li><li><span id="cit5">Y. Zhang, N. Kohler and M. Q. Zhang, <span class="italic">Biomaterials</span>, 2002, <span class="bold">23</span>, 15531561<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0142-9612(01)00267-8" title="DOI Link to resource 10.1016/S0142-9612(01)00267-8">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XhtFykurY%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=11922461%5Buid%5D" title="PubMed Link to resource 11922461">PubMed</a>.</span></li><li><span id="cit6">A. Cockburn, R. Bradford, N. Buck, A. Constable, G. Edwards, B. Haber, P. Hepburn, J. Howlett, F. Kampers, C. Klein, M. Radomski, H. Stamm, S. Wijnhoven and T. Wildemann, <span class="italic">Food Chem. Toxicol.</span>, 2012, <span class="bold">50</span>, 22242242<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.fct.2011.12.029" title="DOI Link to resource 10.1016/j.fct.2011.12.029">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xjt1Oqs7s%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22245376%5Buid%5D" title="PubMed Link to resource 22245376">PubMed</a>.</span></li><li><span id="cit7">I. A. Rahman, P. Vejayakumaran, C. S. Sipaut, J. Ismail and C. K. Chee, <span class="italic">Mater. Chem. Phys.</span>, 2009, <span class="bold">114</span>, 328332<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.matchemphys.2008.09.068" title="DOI Link to resource 10.1016/j.matchemphys.2008.09.068">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXktlyrsQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit8">S. E. A. Gratton, P. A. Ropp, P. D. Pohlhaus, J. C. Luft, V. J. Madden, M. E. Napier and J. M. DeSimone, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2008, <span class="bold">105</span>, 1161311618<a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0801763105" title="DOI Link to resource 10.1073/pnas.0801763105">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhtVCnur%252FI" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18697944%5Buid%5D" title="PubMed Link to resource 18697944">PubMed</a>.</span></li><li><span id="cit9">Y. Zhou and M. Antonietti, <span class="italic">J. Am. Chem. Soc.</span>, 2003, <span class="bold">125</span>, 1496014961<a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja0380998" title="DOI Link to resource 10.1021/ja0380998">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXovVOisLk%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=14653710%5Buid%5D" title="PubMed Link to resource 14653710">PubMed</a>.</span></li><li><span id="cit10">S. K. Das, M. K. Bhunia and A. Bhaumik, <span class="italic">Dalton Trans.</span>, 2010, <span class="bold">39</span>, 43824390<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C000317D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C000317D">RSC</a>.</span></li><li><span id="cit11">Z. Q. Sun, T. Liao, L. Y. Sheng, L. Z. Kou, J. H. Kim and S. X. Dou, <span class="italic">Chem.Eur. J.</span>, 2016, <span class="bold">22</span>, 1135711364<a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201601546" title="DOI Link to resource 10.1002/chem.201601546">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtFWjsLjM" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27381513%5Buid%5D" title="PubMed Link to resource 27381513">PubMed</a>.</span></li><li><span id="cit12">X. N. Ren, L. Wu, J. Jin, J. Liu, Z. Y. Hu, Y. Li, T. Hasan, X. Y. Yang, G. Van Tendeloo and B. L. Su, <span class="italic">RSC Adv.</span>, 2016, <span class="bold">6</span>, 2685626862<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6RA00332J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6RA00332J">RSC</a>.</span></li><li><span id="cit13">M. Moreno-Horn and T. Gebel, <span class="italic">Crit. Rev. Toxicol.</span>, 2014, <span class="bold">44</span>, 849875<a target="_blank" class="DOILink" href="https://doi.org/10.3109/10408444.2014.938802" title="DOI Link to resource 10.3109/10408444.2014.938802">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvVOgurrP" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25257841%5Buid%5D" title="PubMed Link to resource 25257841">PubMed</a>.</span></li><li><span id="cit14">J. X. Wang, G. Q. Zhou, C. Y. Chen, H. W. Yu, T. C. Wang, Y. M. Ma, G. Jia, Y. X. Gao, B. Li, J. Sun, Y. F. Li, F. Jiao, Y. L. Zhao and Z. F. Chai, <span class="italic">Toxicol. Lett.</span>, 2007, <span class="bold">168</span>, 176185<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.toxlet.2006.12.001" title="DOI Link to resource 10.1016/j.toxlet.2006.12.001">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXltV2nug%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17197136%5Buid%5D" title="PubMed Link to resource 17197136">PubMed</a>.</span></li><li><span id="cit15">E. J. Park, H. Kim, Y. Kim and K. Choi, <span class="italic">Toxicol. Environ. Chem.</span>, 2011, <span class="bold">93</span>, 120133<a target="_blank" class="DOILink" href="https://doi.org/10.1080/02772248.2010.495191" title="DOI Link to resource 10.1080/02772248.2010.495191">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGrsLjO" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit16">Y. R. Kim, S. Y. Lee, E. J. Lee, S. H. Park, N. W. Seong, H. S. Seo, S. S. Shin, S. J. Kim, E. H. Meang, M. K. Park, M. S. Kim, C. S. Kim, S. K. Kim, S. W. Son, Y. R. Seo, B. H. Kang, B. S. Han, S. S. Aan, B. J. Lee and M. K. Kim, <span class="italic">Int. J. Nanomed.</span>, 2014, <span class="bold">9</span>, 6778<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Int.%20J.%20Nanomed.%5BJour%5D%20AND%209%5Bvolume%5D%20AND%2067%5Bpage%5D%20and%202014%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a>.</span></li><li><span id="cit17">K. Loeschner, N. Hadrup, K. Qvortrup, A. Larsen, X. Y. Gao, U. Vogel, A. Mortensen, H. R. Lam and E. H. Larsen, <span class="italic">Part. Fibre Toxicol.</span>, 2011, <span class="bold">8</span>, 18<a target="_blank" class="DOILink" href="https://doi.org/10.1186/1743-8977-8-18" title="DOI Link to resource 10.1186/1743-8977-8-18">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXpt1Grsbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21631937%5Buid%5D" title="PubMed Link to resource 21631937">PubMed</a>.</span></li><li><span id="cit18">J. H. E. Arts, M. Hadi, A. M. Keene, R. Kreiling, D. Lyon, M. Maier, K. Michel, T. Petry, U. G. Sauer, D. Warheit, K. Wiench and R. Landsiedel, <span class="italic">Regul. Toxicol. Pharmacol.</span>, 2014, <span class="bold">70</span>, 492506<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.yrtph.2014.07.025" title="DOI Link to resource 10.1016/j.yrtph.2014.07.025">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhtlartL3K" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25108058%5Buid%5D" title="PubMed Link to resource 25108058">PubMed</a>.</span></li><li><span id="cit19">A. R. Collins, B. Annangi, L. Rubio, R. Marcos, M. Dorn, C. Merker, I. Estrela-Lopis, M. R. Cimpan, M. Ibrahim, E. Cimpan, M. Ostermann, A. Sauter, N. E. Yamani, S. Shaposhnikov, S. Chevillard, V. Paget, R. Grall, J. Delic, F. G. de-Cerio, B. Suarez-Merino, V. Fessard, K. N. Hogeveen, L. M. Fjellsbo, E. R. Pran, T. Brzicova, J. Topinka, M. J. Silva, P. E. Leite, A. R. Ribeiro, J. M. Granjeiro, R. Grafstrom, A. Prina-Mello and M. Dusinska, <span class="italic">Wiley Interdiscip. Rev.: Nanomed. Nanobiotechnol.</span>, 2017, <span class="bold">9</span>, e1413<a target="_blank" class="DOILink" href="https://doi.org/10.1002/wnan.1413" title="DOI Link to resource 10.1002/wnan.1413">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27273980%5Buid%5D" title="PubMed Link to resource 27273980">PubMed</a>.</span></li><li><span id="cit20">P. Laux, C. Riebeling, A. M. Booth, J. D. Brain, J. Brunner, C. Cerrillo, O. Creutzenberg, I. Estrela-Lopis, T. Gebel, G. Johanson, H. Jungnickel, H. Kock, J. Tentschert, A. Tlili, A. Schffer, A. J. A. M. Sips, R. A. Yokel and A. Luch, <span class="italic">NanoImpact</span>, 2017, <span class="bold">6</span>, 6980<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.impact.2017.03.003" title="DOI Link to resource 10.1016/j.impact.2017.03.003">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29057373%5Buid%5D" title="PubMed Link to resource 29057373">PubMed</a>.</span></li><li><span id="cit21">J. M. Cohen, J. G. Teeguarden and P. Demokritou, <span class="italic">Part. Fibre Toxicol.</span>, 2014, <span class="bold">11</span>, 20<a target="_blank" class="DOILink" href="https://doi.org/10.1186/1743-8977-11-20" title="DOI Link to resource 10.1186/1743-8977-11-20">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24885440%5Buid%5D" title="PubMed Link to resource 24885440">PubMed</a>.</span></li><li><span id="cit22">S. Dekkers, A. G. Oomen, E. A. J. Bleeker, R. J. Vandebriel, C. Micheletti, J. Cabellos, G. Janer, N. Fuentes, S. Vazquez-Campos, T. Borges, M. J. Silva, A. Prina-Mello, D. Movia, F. Nesslany, A. R. Ribeiro, P. E. Leite, M. Groenewold, F. R. Cassee, A. J. A. M. Sips, A. Dijkzeul, T. van Teunenbroek and S. W. P. Wijnhoven, <span class="italic">Regul. Toxicol. Pharmacol.</span>, 2016, <span class="bold">80</span>, 4659<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.yrtph.2016.05.037" title="DOI Link to resource 10.1016/j.yrtph.2016.05.037">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27255696%5Buid%5D" title="PubMed Link to resource 27255696">PubMed</a>.</span></li><li><span id="cit23">A. D. Maynard, R. J. Aitken, T. Butz, V. Colvin, K. Donaldson, G. Oberdorster, M. A. Philbert, J. Ryan, A. Seaton, V. Stone, S. S. Tinkle, L. Tran, N. J. Walker and D. B. Warheit, <span class="italic">Nature</span>, 2006, <span class="bold">444</span>, 267269<a target="_blank" class="DOILink" href="https://doi.org/10.1038/444267a" title="DOI Link to resource 10.1038/444267a">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xht1SgtLjE" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17108940%5Buid%5D" title="PubMed Link to resource 17108940">PubMed</a>.</span></li><li><span id="cit24">H. Sieg, C. Kastner, B. Krause, T. Meyer, A. Burel, L. Bohmert, D. Lichtenstein, H. Jungnickel, J. Tentschert, P. Laux, A. Braeuning, I. Estrela-Lopis, F. Gauffre, V. Fessard, J. Meijer, A. Luch, A. F. Thunemann and A. Lampen, <span class="italic">Langmuir</span>, 2017, <span class="bold">33</span>, 1072610735<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.langmuir.7b02729" title="DOI Link to resource 10.1021/acs.langmuir.7b02729">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVyqu7nF" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28903564%5Buid%5D" title="PubMed Link to resource 28903564">PubMed</a>.</span></li><li><span id="cit25">J. M. Zook, S. E. Long, D. Cleveland, C. L. A. Geronimo and R. I. MacCuspie, <span class="italic">Anal. Bioanal. Chem.</span>, 2011, <span class="bold">401</span>, 19932002<a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-011-5266-y" title="DOI Link to resource 10.1007/s00216-011-5266-y">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXpsFOru7s%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21808990%5Buid%5D" title="PubMed Link to resource 21808990">PubMed</a>.</span></li><li><span id="cit26">K. Loza, J. Diendorf, C. Sengstock, L. Ruiz-Gonzalez, J. M. Gonzalez-Calbet, M. Vallet-Regi, M. Koller and M. Epple, <span class="italic">J. Mater. Chem. B</span>, 2014, <span class="bold">2</span>, 16341643<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3TB21569E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3TB21569E">RSC</a>.</span></li><li><span id="cit27">Z. Y. Wang, A. von dem Bussche, P. K. Kabadi, A. B. Kane and R. H. Hurt, <span class="italic">ACS Nano</span>, 2013, <span class="bold">7</span>, 87158727<a target="_blank" class="DOILink" href="https://doi.org/10.1021/nn403080y" title="DOI Link to resource 10.1021/nn403080y">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhsVCgtbjE" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24032665%5Buid%5D" title="PubMed Link to resource 24032665">PubMed</a>.</span></li><li><span id="cit28">A. M. Tavares, H. Louro, S. Antunes, S. Quarre, S. Simar, P. J. De Temmerman, E. Verleysen, J. Mast, K. A. Jensen, H. Norppa, F. Nesslany and M. J. Silva, <span class="italic">Toxicol. in Vitro</span>, 2014, <span class="bold">28</span>, 6069<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.tiv.2013.06.009" title="DOI Link to resource 10.1016/j.tiv.2013.06.009">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhtFaktLnI" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23811260%5Buid%5D" title="PubMed Link to resource 23811260">PubMed</a>.</span></li><li><span id="cit29">H. E. Pace, N. J. Rogers, C. Jarolimek, V. A. Coleman, C. P. Higgins and J. F. Ranville, <span class="italic">Anal. Chem.</span>, 2011, <span class="bold">83</span>, 93619369<a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac201952t" title="DOI Link to resource 10.1021/ac201952t">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCrsLvO" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22074486%5Buid%5D" title="PubMed Link to resource 22074486">PubMed</a>.</span></li><li><span id="cit30">K. RasmussenJ. MastP. J. de TemmermanE. VerleysenN. WegeneersF. van SteenJ.C. PizzolonL. de TemmermanE. van DorenK. A. JensenR. BirkedalM. LevinS. H. NielsenI. K. KoponenP. A. ClausenV. Kofored-SrensenY. KemboucheN. ThierietO. SpallaC. GiuotD. RoussetO. WitschgerS. BauB. BianchiC. MotzkusB. ShivachevL. DimowaR. NikolovaD. NihtianovaM. TarassovO. PetrovS. BakardjievaD. GillilandF. PianellaG. CecconeV. SpampinatoG. CotognoP. GibsonC. GaillardA. Mech, <span class="italic">JRC Science and Policy Reports</span> 2014, http://publications.jrc.ec.europa.eu/repository/handle/JRC86291<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=JRC%20Science%20and%20Policy%20Reports%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202014%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a>.</span></li><li><span id="cit31">M. Naiim, A. Boualem, C. Ferre, M. Jabloun, A. Jalocha and P. Ravier, <span class="italic">Soft Matter</span>, 2015, <span class="bold">11</span>, 2832<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4SM01995D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4SM01995D">RSC</a>.</span></li><li><span id="cit32">P. A. Hassan and S. K. Kulshreshtha, <span class="italic">J. Colloid Interface Sci.</span>, 2006, <span class="bold">300</span>, 744748<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jcis.2006.04.013" title="DOI Link to resource 10.1016/j.jcis.2006.04.013">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XmvFKmsbY%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16790246%5Buid%5D" title="PubMed Link to resource 16790246">PubMed</a>.</span></li><li><span id="cit33">R. D. Deegan, O. Bakajin, T. F. Dupont, G. Huber, S. R. Nagel and T. A. Witten, <span class="italic">Nature</span>, 1997, <span class="bold">389</span>, 827829<a target="_blank" class="DOILink" href="https://doi.org/10.1038/39827" title="DOI Link to resource 10.1038/39827">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2sXmvFSktrY%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit34">P. J. De Temmerman, E. Van Doren, E. Verleysen, Y. Van der Stede, M. A. D. Francisco and J. Mast, <span class="italic">J. Nanobiotechnol.</span>, 2012, <span class="bold">10</span>, 24<a target="_blank" class="DOILink" href="https://doi.org/10.1186/1477-3155-10-24" title="DOI Link to resource 10.1186/1477-3155-10-24">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhslyisbnM" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22709926%5Buid%5D" title="PubMed Link to resource 22709926">PubMed</a>.</span></li><li><span id="cit35">P. Muneesawang and C. Sirisathitkul, <span class="italic">J. Nanomater.</span>, 2015, <span class="bold">2015</span>, 8<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Nanomater.%5BJour%5D%20AND%202015%5Bvolume%5D%20AND%208%5Bpage%5D%20and%202015%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a>.</span></li><li><span id="cit36">R. J. Griffitt, A. Feswick, R. Weil, K. Hyndman, P. Carpinone, K. Powers, N. D. Denslow and D. S. Barber, <span class="italic">Environ. Toxicol.</span>, 2011, <span class="bold">26</span>, 541551<a target="_blank" class="DOILink" href="https://doi.org/10.1002/tox.20669" title="DOI Link to resource 10.1002/tox.20669">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhtFeksLbO" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21910207%5Buid%5D" title="PubMed Link to resource 21910207">PubMed</a>.</span></li><li><span id="cit37">T. T. Morgan, H. S. Muddana, E. I. Altinoglu, S. M. Rouse, A. Tabakovic, T. Tabouillot, T. J. Russin, S. S. Shanmugavelandy, P. J. Butler, P. C. Eklund, J. K. Yun, M. Kester and J. H. Adair, <span class="italic">Nano Lett.</span>, 2008, <span class="bold">8</span>, 41084115<a target="_blank" class="DOILink" href="https://doi.org/10.1021/nl8019888" title="DOI Link to resource 10.1021/nl8019888">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhtlSqtLfM" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19367837%5Buid%5D" title="PubMed Link to resource 19367837">PubMed</a>.</span></li><li><span id="cit38">G. Dordelmann, D. Kozlova, S. Karczewski, R. Lizio, S. Knauer and M. Epple, <span class="italic">J. Mater. Chem. B</span>, 2014, <span class="bold">2</span>, 72507259<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4TB00922C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4TB00922C">RSC</a>.</span></li><li><span id="cit39">S. Lee, X. Y. Bi, R. B. Reed, J. F. Ranville, P. Herckes and P. Westerhoff, <span class="italic">Environ. Sci. Technol.</span>, 2014, <span class="bold">48</span>, 1029110300<a target="_blank" class="DOILink" href="https://doi.org/10.1021/es502422v" title="DOI Link to resource 10.1021/es502422v">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhtlGgsrvP" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25122540%5Buid%5D" title="PubMed Link to resource 25122540">PubMed</a>.</span></li></ol>




<hr/><table><tr><td><h3>Footnote</h3></td></tr><tr><td><span class="sup_ref"> <span id="fn1">Electronic supplementary information (ESI) available: NTA size distributions for Al<small><sup>0</sup></small>, Al<small><sub>2</sub></small>O<small><sub>3</sub></small>, NM103, NM104, DLS number-based distribution for Al<small><sup>0</sup></small>, Al<small><sub>2</sub></small>O<small><sub>3</sub></small>, NM103, NM104, TEM measurement of TiO<small><sub>2</sub></small> NMs in DMEM, SAXS data for Al<small><sup>0</sup></small>, Al<small><sub>2</sub></small>O<small><sub>3</sub></small> and AlCl<small><sub>3</sub></small> in BSA and DMEM after 24 and 48h, impurities of used NMs determined by IBM, aluminium aqua complexes at different pH values, CRM surface investigations, colocalization pattern by IBM for Al<small><sub>2</sub></small>O<small><sub>3</sub></small>, CRM spectra displaying protein modifications, ToF-SIMS measurements of Al and Al<small><sub>2</sub></small>O<small><sub>3</sub></small> NMs in DMEM, ToF-SIMS measurements of TiO<small><sub>2</sub></small> NMs in DMEM. See DOI: <a href="http://xlink.rsc.org/?DOI=c8ra00205c" target="_blank" title="Select to resolve DOI: 10.1039/c8ra00205c">10.1039/c8ra00205c</a></span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is  The Royal Society of Chemistry 2018</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"></script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>