<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Identifying structure–absorption relationships and predicting absorption strength of non-fullerene acceptors for organic photovoltaics   - Energy &amp; Environmental Science (RSC Publishing) DOI:10.1039/D2EE00887D</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2022/ee/d2ee00887d"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":15,"applicationTime":55,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Jun Yan"/><meta name="DC.Creator" content="Xabier Rodríguez-Martínez"/><meta name="DC.Creator" content="Drew Pearce"/><meta name="DC.Creator" content="Hana Douglas"/><meta name="DC.Creator" content="Danai Bili"/><meta name="DC.Creator" content="Mohammed Azzouzi"/><meta name="DC.Creator" content="Flurin Eisner"/><meta name="DC.Creator" content="Alise Virbule"/><meta name="DC.Creator" content="Elham Rezasoltani"/><meta name="DC.Creator" content="Valentina Belova"/><meta name="DC.Creator" content="Bernhard Dörling"/><meta name="DC.Creator" content="Sheridan Few"/><meta name="DC.Creator" content="Anna A. Szumska"/><meta name="DC.Creator" content="Xueyan Hou"/><meta name="DC.Creator" content="Guichuan Zhang"/><meta name="DC.Creator" content="Hin-Lap Yip"/><meta name="DC.Creator" content="Mariano Campoy-Quiles"/><meta name="DC.Creator" content="Jenny Nelson"/><meta name="DC.title" content="Identifying structure–absorption relationships and predicting absorption strength of non-fullerene acceptors for organic photovoltaics  "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2022/07/13"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D2EE00887D"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Identifying structure–absorption relationships and predicting absorption strength of non-fullerene acceptors for organic photovoltaics  "/><meta name="citation_author" content="Jun Yan"/><meta name="citation_author" content="Xabier Rodríguez-Martínez"/><meta name="citation_author" content="Drew Pearce"/><meta name="citation_author" content="Hana Douglas"/><meta name="citation_author" content="Danai Bili"/><meta name="citation_author" content="Mohammed Azzouzi"/><meta name="citation_author" content="Flurin Eisner"/><meta name="citation_author" content="Alise Virbule"/><meta name="citation_author" content="Elham Rezasoltani"/><meta name="citation_author" content="Valentina Belova"/><meta name="citation_author" content="Bernhard Dörling"/><meta name="citation_author" content="Sheridan Few"/><meta name="citation_author" content="Anna A. Szumska"/><meta name="citation_author" content="Xueyan Hou"/><meta name="citation_author" content="Guichuan Zhang"/><meta name="citation_author" content="Hin-Lap Yip"/><meta name="citation_author" content="Mariano Campoy-Quiles"/><meta name="citation_author" content="Jenny Nelson"/><meta name="citation_online_date" content="2022/05/20"/><meta name="citation_date" content="2022"/><meta name="citation_journal_title" content="Energy &amp; Environmental Science"/><meta name="citation_volume" content="15"/><meta name="citation_issue" content="7"/><meta name="citation_firstpage" content="2958"/><meta name="citation_lastpage" content="2973"/><meta name="citation_doi" content="10.1039/D2EE00887D"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2022/ee/d2ee00887d"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2022/ee/d2ee00887d"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2022/ee/d2ee00887d"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2022/ee/d2ee00887d">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d2ee00376g)" href="/en/content/articlehtml/2022/ee/d2ee00376g" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d2ee00738j)" href="/en/content/articlehtml/2022/ee/d2ee00738j" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div><br/><span class="oa"><img src="/content/newimages/open_access_blue.png" alt=""/> Open Access Article<br/><img src="/content/newimages/CCBY-NC.svg" alt=""/>
This Open Access Article is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/3.0/">Creative Commons Attribution-Non Commercial 3.0 Unported Licence</a></span></div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D2EE00887D">10.1039/D2EE00887D</a>
(Paper)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1754-5706/2008">Energy Environ. Sci.</a></span>, 2022, <strong>15</strong>, 2958-2973</div><h1 id="sect209"><span class="title_heading">Identifying structure–absorption relationships and predicting absorption strength of non-fullerene acceptors for organic photovoltaics<a title="Select to navigate to footnote" href="#fn1">†</a></span></h1><p class="header_text">
      <span class="bold">
        
          
            Jun 
            Yan<a title="Select to navigate to footnote" href="#fn2">‡</a>
          
        
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Jun Yan (orcid.org/0000-0001-9966-4357) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-9966-4357"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Xabier 
            Rodríguez-Martínez<a title="Select to navigate to footnote" href="#fn2">‡</a>
          
        
      </span>*<sup><span class="sup_ref italic">bc</span></sup>, 
      <span class="bold">
        
          
            Drew 
            Pearce
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Hana 
            Douglas
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Danai 
            Bili
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Mohammed 
            Azzouzi
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Flurin 
            Eisner
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Flurin Eisner (orcid.org/0000-0001-5507-4961) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-5507-4961"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Alise 
            Virbule
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Elham 
            Rezasoltani
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Valentina 
            Belova
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Valentina Belova (orcid.org/0000-0002-8142-2090) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-8142-2090"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">c</span></sup>, 
      <span class="bold">
        
          
            Bernhard 
            Dörling
          
        
      </span><sup><span class="sup_ref italic">c</span></sup>, 
      <span class="bold">
        
          
            Sheridan 
            Few
          
        
      </span><sup><span class="sup_ref italic">af</span></sup>, 
      <span class="bold">
        
          
            Anna A. 
            Szumska
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Xueyan 
            Hou
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Guichuan 
            Zhang
          
        
      </span><sup><span class="sup_ref italic">d</span></sup>, 
      <span class="bold">
        
          
            Hin-Lap 
            Yip
          
        
      </span><sup><span class="sup_ref italic">de</span></sup>, 
      <span class="bold">
        
          
            Mariano 
            Campoy-Quiles
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Mariano Campoy-Quiles (orcid.org/0000-0002-8911-640X) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-8911-640X"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">c</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Jenny 
            Nelson
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Jenny Nelson (orcid.org/0000-0003-1048-1330) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-1048-1330"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">a</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Department of Physics, Imperial College London, SW7 2AZ, London, UK. E-mail: <a href="mailto:jenny.nelson@imperial.ac.uk">jenny.nelson@imperial.ac.uk</a></span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Electronic and Photonic Materials (EFM), Department of Physics, Chemistry and Biology (IFM), Linköping University, Linköping, SE 581 83, Sweden. E-mail: <a href="mailto:xabier.rodriguez.martinez@liu.se">xabier.rodriguez.martinez@liu.se</a></span>
      <br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">Instituto de Ciencia de Materiales de Barcelona, ICMAB-CSIC, Campus UAB, Bellaterra 08193, Spain. E-mail: <a href="mailto:mcampoy@icmab.es">mcampoy@icmab.es</a></span>
      <br/><a id="affd"><sup><span class="sup_ref italic">d</span></sup></a><span class="italic">Institute of Polymer Optoelectronic Materials and Devices, State Key Laboratory of Luminescent Materials and Devices, South China University of Technology, Guangzhou 510640, P. R. China</span>
      <br/><a id="affe"><sup><span class="sup_ref italic">e</span></sup></a><span class="italic">Department of Materials Science and Engineering, City University of Hong Kong, Tat Chee Avenue, Kowloon, Hong Kong</span>
      <br/><a id="afff"><sup><span class="sup_ref italic">f</span></sup></a><span class="italic">Sustainability Research Institute, School of Earth and Environment, University of Leeds, LS2 9JT, Leeds, UK</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      18th March 2022
    </span><span class="bold italic">, Accepted 20th May 2022</span></div><p class="bold italic">First published on 20th May 2022</p><hr/><div class="abstract"><h2>Abstract</h2><p>Non-fullerene acceptors (NFAs) are excellent light harvesters, yet the origin of their high optical extinction is not well understood. In this work, we investigate the absorption strength of NFAs by building a database of time-dependent density functional theory (TDDFT) calculations of ∼500 π-conjugated molecules. The calculations are first validated by comparison with experimental measurements in solution and solid state using common fullerene and non-fullerene acceptors. We find that the molar extinction coefficient (<span class="italic">ε</span><small><sub>d,max</sub></small>) shows reasonable agreement between calculation in vacuum and experiment for molecules in solution, highlighting the effectiveness of TDDFT for predicting optical properties of organic π-conjugated molecules. We then perform a statistical analysis based on molecular descriptors to identify which features are important in defining the absorption strength. This allows us to identify structural features that are correlated with high absorption strength in NFAs and could be used to guide molecular design: highly absorbing NFAs should possess a planar, linear, and fully conjugated molecular backbone with highly polarisable heteroatoms. We then exploit a random decision forest algorithm to draw predictions for <span class="italic">ε</span><small><sub>d,max</sub></small> using a computational framework based on extended tight-binding Hamiltonians, which shows reasonable predicting accuracy with lower computational cost than TDDFT. This work provides a general understanding of the relationship between molecular structure and absorption strength in π-conjugated organic molecules, including NFAs, while introducing predictive machine-learning models of low computational cost.</p></div><hr/>
    
      <br/><table class="box_border"><tr><td>
        
          <h2 id="sect367"><span class="a_heading">Broader context</span></h2>
          <span>Organic π-conjugated semiconductors (OSCs) work as the main light harvesters in organic photovoltaics (OPVs). Their synthetic versatility converts them onto suitable candidates for rational molecular design based on high-throughput screening techniques. Significant advances in the efficiency of OPVs (exceeding 19% in single junctions under 1 sun) have been made by trial-and-error with new but increasingly diverse materials, primarily non-fullerene acceptors (NFAs) and mostly owing to their high absorption strength. However, the reasons for that superior light harvesting performance remain elusive, thus preventing the molecular tailoring of NFAs with further enhanced light harvesting capabilities toward breakthrough OPV efficiencies. A statistical analysis of time-dependent density functional theory calculations and machine learning (ML) models reveal that molecular linearity, planarity, polarizability, and number of π-conjugated carbon atoms correlate strongly with the absorption strength of OSCs. A structure–absorption strength relationship is established to introduce design rules for highly absorbing OSCs. ML models, in combination with extended tight-binding Hamiltonians, are shown to predict the absorption strength of OSCs. As a result, this work contributes to an improved understanding of the absorption strength of π-conjugated organic molecules in general while suggesting ways to design highly absorbing NFAs that maximize the light harvesting capabilities for solar energy conversion.</span>
        
      </td></tr></table><br/>
    
    
      
      <h2 id="sect370"><span class="a_heading">1. Introduction</span></h2>
      <span>Organic photovoltaic (OPV) energy conversion is a promising option among next generation renewable and sustainable energy technologies for a low-carbon energy future.<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1–3</span></sup></a> OPV has shown promising potential for various applications, such as indoor photovoltaics (PV),<a title="Select to navigate to reference" href="#cit4"><sup><span class="sup_ref">4–6</span></sup></a> semi-transparent solar windows,<a title="Select to navigate to reference" href="#cit7"><sup><span class="sup_ref">7,8</span></sup></a> PV greenhouses,<a title="Select to navigate to references" href="#cit9"><sup><span class="sup_ref">9</span></sup></a> and off-grid power supply.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> Recent OPV devices based on non-fullerene acceptors (NFAs) have demonstrated certified power conversion efficiencies (PCEs) exceeding 19% in a single junction configuration,<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> approaching the efficiencies observed in inorganic semiconductor PV technologies such as crystalline silicon and perovskite solar cells, and far higher than values thought attainable in OPV when using fullerene derivatives as the electron-acceptors.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> The startling progress led by NFAs can be attributed to various advantages over fullerene derivatives, such as band-gap tunability, sharp absorption onset, high emission, high absorption, and low energy losses.<a title="Select to navigate to reference" href="#cit13"><sup><span class="sup_ref">13–15</span></sup></a> Among these advantages, the absorption strength of state-of-the-art NFAs is particularly outstanding, as exemplified in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1c</a> (a detailed list of chemical names and nomenclatures is provided in Note S1, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>).<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> For instance, Y6 shows a maximum extinction coefficient (<span class="italic">κ</span><small><sub>max</sub></small>) over 1.5 in the visible part of the electromagnetic spectrum, as compared to less than 0.75 for fullerene derivatives (PC61BM and PC71BM). A high extinction coefficient increases the chance of high quantum efficiency and photogenerated current density, and makes it possible to fabricate highly absorbing OPV films with just a few tens of nanometre-thick photoactive layers. In comparison with workhorse fullerene acceptors, OPV devices based on highly absorbing NFAs could be made comparably thinner than the former, which exponentially raises the output power per weight (<span class="italic">i.e.</span> the specific weight in W g<small><sup>−1</sup></small>) of OPV devices<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> and might be an effective route toward lower production costs (as less material could employed to achieve an equivalent PCE) and even increase device thermal stability.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> Moreover, through detailed balance between photon absorption and emission,<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19,20</span></sup></a> high absorption strength in principle should lead to high emission from the NFAs, while strong NFA emission is believed to be a key reason for NFA-based OPVs to possess low nonradiative voltage losses.<a title="Select to navigate to reference" href="#cit21"><sup><span class="sup_ref">21–25</span></sup></a> Despite the clear advantage of strong photo-absorption of NFAs over fullerene derivatives, the phenomenon has attracted much less attention than other properties of NFAs.<a title="Select to navigate to reference" href="#cit21"><sup><span class="sup_ref">21–23,25–28</span></sup></a> Conceptually, symmetry rules (<span class="italic">i.e.</span>, the Laporte rule) can explain the qualitative difference between NFAs and fullerene derivatives in terms of absorption strength, yet such rules cannot predict differences in absorption strength among structures for which the lowest transitions are symmetry allowed. The features empirically and theoretically proposed<a title="Select to navigate to reference" href="#cit29"><sup><span class="sup_ref">29,30</span></sup></a> to lead to strong absorption in π-conjugated polymers are molecular stiffness, linearity, extended π-conjugation and large molecular size. It is therefore of interest to establish whether the same (or other) molecular features are quantitatively associated or not with increased absorption strength in NFAs, while seeking molecular design rules to drive absorption and performance higher in new molecules.</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-f1.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">(a) Molecular structures of typical organic acceptors, including PC61BM, PC71BM, O-IDFBR, O-IDTBR, ITIC, IT-4F, IDIC, IEICO, IEICO-4F, Y5, Y6, and Y7. (b) Refractive index and (c) extinction coefficient of a larger set of typical organic acceptor thin films measured using VASE. (d) Experimental <span class="italic">ε</span><small><sub>d,max</sub></small> in solution <span class="italic">versus</span> calculated <span class="italic">ε</span><small><sub>d,max</sub></small> in vacuum using TDDFT of a set of ∼80 π-conjugated molecules. (e) Estimated experimental <span class="italic">ε</span><small><sub>d,max</sub></small> in film (solid state) <span class="italic">versus</span> that in solution using <a title="" href="#eqn3">eqn (3)</a>. Panel (d) contains a subset of well-known NFA molecules that are highlighted in colour. All TDDFT results in panel (d) were performed using the functional B3LYP and basis set 6-311+G(d,p), except for the ones (grey squares) taken from ref. <a title="Select to navigate to references" href="#cit69">69</a> that are based on the LRC-wPBEh functional and 6-311+G(d) basis set. We also note here that the side chains of molecules are replaced by H atoms or methyl groups in the calculations as they are computationally expensive and do not contribute to the π-conjugation, hence electronic transitions.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> The experimental data of <span class="italic">ε</span><small><sub>d,max</sub></small> in film are converted from maximum values of extinction coefficients shown in panel (c) using <a title="" href="#eqn3">eqn (3)</a>, while solution data are collected from literature, noting that different values may be present for the same material as retrieved from different sources. Grey dashed lines indicate the perfect match between x and y axis. The data required for generating panels (d) and (e) in this figure are presented in the ESI.<a title="Select to navigate to footnote" href="#fn1">†</a></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Excited state calculations based on quantum chemistry methods, such as time-dependent density functional theory (TDDFT),<a title="Select to navigate to reference" href="#cit8"><sup><span class="sup_ref">8,29,31–33</span></sup></a> Hartree–Fock method,<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a><span class="italic">ab initio</span> Monte Carlo method,<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> second order Møllier–Plesset theory (MP2),<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> and coupled cluster method,<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> have been applied to predict the electronic and optical properties of molecules. Among them, TDDFT is the most widely applied method for excited state calculations, and has shown reasonable accuracy in calculating and predicting the trends in absorption strength of organic molecules,<a title="Select to navigate to reference" href="#cit29"><sup><span class="sup_ref">29,31</span></sup></a> as also demonstrated in this work. However, the rapid scaling of computation time with molecular size has been the real obstacle limiting the applicability of TDDFT for excited state calculations on molecules with hundreds of atoms. Given the size and diverse structure of modern NFAs, faster and more efficient methods are therefore needed to establish the relationship between excited-state and molecular properties in NFAs.</p>
      <p class="otherpara">The emergence of artificial intelligence (AI) has made it possible to study quantitative structure–property relationships (QSPRs) in molecules with massively improved computational efficiency. As the most popular branch of AI, machine-learning (ML) has attracted much attention in materials science over the last decade, and has been widely applied for material property prediction and material discovery.<a title="Select to navigate to reference" href="#cit38"><sup><span class="sup_ref">38–41</span></sup></a> Recently, ML has also gained popularity in OPV scenarios,<a title="Select to navigate to reference" href="#cit42"><sup><span class="sup_ref">42–52</span></sup></a> yet existing ML studies related to OPVs have been primarily focused either on the energetics<a title="Select to navigate to reference" href="#cit42"><sup><span class="sup_ref">42,43,53–56</span></sup></a> or directly on PCE,<a title="Select to navigate to reference" href="#cit42"><sup><span class="sup_ref">42,48,57–64</span></sup></a> with little attention paid to the absorption strength of the photoactive materials.<a title="Select to navigate to reference" href="#cit65"><sup><span class="sup_ref">65,66</span></sup></a> Moreover, there are no ML studies explicitly focused on the absorption strength of NFAs beyond the identification of moieties of frequent appearance in highly absorbing molecules.<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> However, QSPR and ML models have been successfully applied to investigate the absorption strength of fluorophores or dyes typically employed in bioimaging, showing encouraging results.<a title="Select to navigate to reference" href="#cit30"><sup><span class="sup_ref">30,67,68</span></sup></a> Therefore, it is appealing to apply ML methods in combination with QSPR models to investigate the origin of the large absorption strength in state-of-the-art NFAs.</p>
      <p class="otherpara">Here, we present an experimental, TDDFT, QSPR, statistical and ML study of the absorption strength of NFAs to identify the key chemical and structural features that lead to high optical absorption in state-of-the-art NFAs. We exploit a database of nearly 500 unique organic molecules (or 3500 calculations) generated using DFT and TDDFT over several years. We obtain good quantitative agreement between TDDFT calculations of absorption strength and experimental values for state-of-the-art NFAs and fullerenes, which supports the use of TDDFT results for further statistical and QSPR modelling. Accordingly, we extract molecular information from the DFT-optimized geometries by computing nearly 6000 molecular descriptors and first looking for correlations with the absorption strength. The strongest correlations are found between experimentally measured maximum molar extinction coefficient (<span class="italic">ε</span><small><sub>d,max</sub></small>) and two main molecular descriptors from calculations: <span class="italic">λ</span><small><sub>1,p</sub></small> and C2SP2, which describe the size of the molecule in the direction of maximal atomic polarizability, and the number of sp<small><sup>2</sup></small> hybridized carbon atoms that are bound to two other carbons (C2), respectively. These quantities can be related to a few key material features leading to high absorption strength: linearity, planarity, and extension of the π-conjugation in the form of fused and closed-ring moieties, in good agreement with previous ML reports on fluorophores and dyes.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> We further identify several moieties and paired combinations thereof that are frequently found in highly absorbing NFAs, corresponding to thieno[3,2-<span class="italic">b</span>]thiophene (TT), thiophene (T), 2-(5,6-difluoro-3-oxo-2,3-dihydro-1<span class="italic">H</span>-inden-1-ylidene)malononitrile (2FIC), 2-(3-oxo-2,3-dihydro-1<span class="italic">H</span>-inden-1-ylidene)malononitrile (IC) and indaceno[1,2-<span class="italic">b</span>:5,6-<span class="italic">b</span>′]dithiophene (IDT). These form a catalogue of molecular design rules to further enhance the absorption strength of organic π-conjugated molecules, such as next-generation NFAs. We then train and test an ensemble learning method, namely a random decision forest (RF), to predict <span class="italic">ε</span><small><sub>d,max</sub></small> and provide further information about the most important features in the modelling of absorption strength in organic π-conjugated molecules. Finally, we explore the possibility to predict <span class="italic">ε</span><small><sub>d,max</sub></small> while using a cheaper molecular geometry optimization method based on semiempirical extended tight-binding (xTB) Hamiltonians instead of the expensive DFT approach. We do so by training a RF with our TDDFT database and proving its predictive properties in terms of <span class="italic">ε</span><small><sub>d,max</sub></small> when interpolated using xTB-optimized geometries. This approach shows application potential in high-throughput screening studies in combination with generative molecular models.</p>
    
    
      
      <h2 id="sect447"><span class="a_heading">2. Results and discussion</span></h2>
      
        
        <h3 id="sect450"><span class="b_heading">2.1. Experimental validation of calculated absorption strength using TDDFT</span></h3>
        <span>Quantifying how well the TDDFT derived excited state properties agree with the experimental measurements in terms of absorption strength is of utmost importance to validate our theoretical calculations and support further conclusions extracted thereof. Accordingly, we first evaluate the agreement between TDDFT calculations and experimental data in terms of the absorption strength. We compare the absorption strength of a broad catalogue (∼10 molecules) of NFA molecules and widely studied fullerene derivatives (PC61BM and PC71BM, with their molecular structures shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1a</a>) as obtained from TDDFT calculations, with a variety of optical measurements in both solution and solid state. For the most representative NFAs examined, we verify that their frontier molecular orbital energy levels as retrieved from TDDFT calculations are properly aligned, relative to those of a set of common polymer donors, for the NFAs to act as electron acceptor in a bulk heterojunction blend with those donors (Fig. S1, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). The measured refractive index (<span class="italic">n</span>) and extinction coefficient (<span class="italic">κ</span>) of those molecules in thin film obtained using our variable-angle spectroscopic ellipsometry (VASE) measurements are shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1b and c</a>. Solution state data shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1d</a> and e are collected from a variety of literature references as detailed in the ESI.<a title="Select to navigate to footnote" href="#fn1">†</a><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a></span>
        <p class="otherpara">As a metric for absorption strength, we initially consider several candidates such as the oscillator strength (<span class="italic">f</span><small><sub>osc</sub></small>), the absorption coefficient (<span class="italic">α</span>) or the imaginary part of the dielectric function (<span class="italic">ε</span><small><sub>2</sub></small>). In this work, we eventually focus on the maximum molar extinction coefficient (<span class="italic">ε</span><small><sub>d,max</sub></small>, M<small><sup>−1</sup></small> cm<small><sup>−1</sup></small>) of NFAs as it shows the best agreement between experimental and theoretical data, as we demonstrate below. <span class="italic">ε</span><small><sub>d,max</sub></small> constitutes a typical experimental measurement in solution that can also be accessed from myriad literature references. Note that the usual calculations based on single molecules using TDDFT cannot account for solid state effects as they are performed for isolated molecules in vacuum or surrounded by an isotropic medium (such as a solvent using the polarizable-continuum-solvent-model, PCM, Fig. S2, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). The derivation of the theoretical <span class="italic">ε</span><small><sub>d</sub></small> is provided in the Methods section, which results in a mathematical expression for <span class="italic">ε</span><small><sub>d,max</sub></small> as<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn1"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t1.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t1.gif"/></a></td><td class="rightEqn">(1)</td></tr></table>where <span class="italic">N</span><small><sub>A</sub></small> is the Avogadro constant, <span class="italic">e</span> the elementary charge, <span class="italic">ħ</span> the reduced Planck constant, <span class="italic">ε</span><small><sub>0</sub></small> the vacuum permittivity, <span class="italic">m</span><small><sub>0</sub></small> the electron mass, <span class="italic">n</span><small><sub>r</sub></small> is the refractive index in solution (assumed to be 1.3 of a common organic solvent throughout this study), and <span class="italic">c</span> the speed of light. <span class="italic">f</span><small><sub>osc,max</sub></small> is the oscillator strength of the strongest transition among the calculated states within the visible-IR part of the spectrum, and <span class="italic">E</span><small><sub>max</sub></small> is the energy of that transition. The brightest transition is very often the lowest-energy transition in commonly used π-conjugated molecules.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> We note here that the delta function in <a title="" href="#eqn17">eqn (17)</a> is replaced with a Gaussian distribution function with a peak intensity of <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t2.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t2.gif"/></a>, where <span class="italic">σ</span> is the Gaussian width and assumed to be 0.1 eV for a common organic pi-conjugated molecule.</p>
        <p class="otherpara">The experimental <span class="italic">ε</span><small><sub>d,max</sub></small> from solution can be obtained using the optical density (OD) measurements performed using UV-visible spectroscopy, <span class="italic">via</span><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn2"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t3.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t3.gif"/></a></td><td class="rightEqn">(2)</td></tr></table>where OD<small><sub>max</sub></small> is the maximum optical density, <span class="italic">ρ</span> is the molar concentration (M), and <span class="italic">d</span> the light path length of the cuvette (cm). Similarly, the experimental <span class="italic">ε</span><small><sub>d,max</sub></small> from film can be estimated assuming a mass concentration <span class="italic">ρ</span><small><sub>M</sub></small> in the film of 1000 g L<small><sup>−1</sup></small> (as a typical value for conjugated polymers and small molecules),<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> either from the maximum absorption coefficient <span class="italic">α</span><small><sub>cm,max</sub></small> (cm<small><sup>−1</sup></small>) or extinction coefficient (<span class="italic">κ</span><small><sub>max</sub></small>) (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1c</a>), <span class="italic">via</span><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn3"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t4.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t4.gif"/></a></td><td class="rightEqn">(3)</td></tr></table>where <span class="italic">M</span><small><sub>w</sub></small> is the molecular weight in g mol<small><sup>−1</sup></small>, and <span class="italic">λ</span><small><sub>max</sub></small> the wavelength at <span class="italic">κ</span><small><sub>max</sub></small> in centimetre.</p>
        <p class="otherpara">
          <a title="Select to navigate to figure" href="#imgfig1">Fig. 1d</a> presents the results of the comparison between experimental <span class="italic">ε</span><small><sub>d,max</sub></small> in solution and theoretical <span class="italic">ε</span><small><sub>d,max</sub></small> calculated from single molecules using TDDFT in vacuum. A brief discussion of the solvent effect on the absorption strength and the reasons why we choose a vacuum medium are provided in Fig. S2 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Despite the scattering of data points, we observe the occurrence of a monotonic relationship between solution and calculated <span class="italic">ε</span><small><sub>d,max</sub></small> with a Pearson correlation coefficient (<span class="italic">r</span>) of 0.77. Interestingly, such correlation is no longer observed when quantifying the absorption strength in terms of <span class="italic">α</span><small><sub>max</sub></small> neither when adding further data points from literature on π-conjugated fluorophores to our statistical analysis (Fig. S3a (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>), <span class="italic">r</span> = 0.30), which is believed to be caused by the differences in molecular weight; in that case, only <span class="italic">ε</span><small><sub>d,max</sub></small> is found to follow a monotonic trend (Fig. S3b, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Some of the material assumptions on refractive index and density required to obtain <span class="italic">α</span><small><sub>max</sub></small> values might be responsible for the observed mismatch. It is worth noting that, expectedly, the correlation between solid state (film) and solution (<span class="italic">r</span> = 0.66, <a title="Select to navigate to figure" href="#imgfig1">Fig. 1e</a>) or calculated <span class="italic">ε</span><small><sub>d,max</sub></small> (<span class="italic">r</span> = 0.61, <a title="Select to navigate to figure" href="#imgfig1">Fig. 1e</a>) is not as good as that from solution data <span class="italic">versus</span> calculated <span class="italic">ε</span><small><sub>d,max</sub></small> (<span class="italic">r</span> = 0.77, <a title="Select to navigate to figure" href="#imgfig1">Fig. 1d</a>, neither for <span class="italic">α</span><small><sub>max</sub></small> as shown in Fig. S4, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Such discrepancy is attributed to solid-state effects such as aggregation effects,<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a> intermolecular orientation,<a title="Select to navigate to reference" href="#cit70"><sup><span class="sup_ref">70,71</span></sup></a> and side chain interactions,<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> which are not considered in single molecule excited state calculations.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> The observed trend that a highly absorbing material in solution will produce highly absorbing films is, nonetheless, generally valid and thus solution data are relevant for devices. Since the NFAs analysed here have a rather similar number of π-electrons (<span class="italic">n</span><small><sub>π</sub></small>), the corresponding <span class="italic">ε</span><small><sub>d,max</sub></small> per π-electron (Fig. S6, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) shows a similar trend as that in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1d, e</a>, and Fig. S4 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Despite the simplicity of single molecule excited state calculations, these data show that using TDDFT calculations of the excited state to deliver <span class="italic">ε</span><small><sub>d,max</sub></small> can provide a reasonably good approximation to experimental measurements. Moreover, dealing with TDDFT calculations gives us room to correlate key molecular properties, such as molecular size and shape (aspect ratio), linearity, planarity, grafted side chain positions, or functional groups, to the absorption strength using molecular descriptors. These observations provide a foundation from molecular structures to identify the origin and further extend the high optical extinction of NFAs through chemical design rules, as we show in the upcoming sections.</p>
      
      
        
        <h3 id="sect580"><span class="b_heading">2.2. Statistical analysis of the TDDFT absorption strength dataset</span></h3>
        <span>The experimental validation of the TDDFT calculations in NFAs supports the use of such results to build an extended database of optimized molecular geometries and excited state properties. The dataset is built by collecting thousands of molecular geometries generated over the last years in our group, making up a total of 3515 calculations on small molecules and oligomers. The distribution of number of atoms in a molecule is shown in Fig. S7 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) with a majority lying between 50 and 100 atoms. This database is sufficiently diverse to allow us to detect correlations and chemical/structural design rules that could explain and/or further enhance optical absorption in conjugated small molecules.</span>
        <div>
          
          <span id="sect585"/><span class="c_heading_indent">2.2.1. Correlation analysis of molecular descriptors. </span>
          <span>In the simplest statistical analysis of our TDDFT database, we look for correlations of the absorption strength with respect to a catalogue of molecular descriptors. First, as described in Note S2 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>), we filter the pristine TDDFT database by identifying duplicate molecules (in terms of molecular weight) and selecting the lowest energy conformer (<span class="italic">i.e.</span>, optimized geometries in the ground state) among them. As a result, the curated TDDFT database employed in this work consists of 479 π-conjugated small molecules and oligomers with a distribution of moieties shown in Fig. S9 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>).</span>
          <p class="otherpara">Then, we introduce several target features related with absorption strength, starting from the maximum oscillator strength of any calculated transition (<span class="italic">f</span><small><sub>max</sub></small>); the maximum oscillator strength of any transition in the visible electromagnetic window (herein constrained between 300–1200 nm or 1–4 eV for its relevance in solar energy harvesting applications) (<span class="italic">f</span><small><sub>max,vis</sub></small>); and the sum of oscillator strengths of all transitions in the visible window, <span class="italic">f</span><small><sub>sum,vis</sub></small>. These three features are also evaluated per <span class="italic">n</span><small><sub>π</sub></small> for the molecule, <span class="italic">i.e.</span>, <span class="italic">f</span><small><sub>max</sub></small>/<span class="italic">n</span><small><sub>π</sub></small>, <span class="italic">f</span><small><sub>max,vis</sub></small>/<span class="italic">n</span><small><sub>π</sub></small> and <span class="italic">f</span><small><sub>sum,vis</sub></small>/<span class="italic">n</span><small><sub>π</sub></small>. We then consider the maximum absorption coefficient (<span class="italic">α</span><small><sub>max</sub></small>) obtained using <a title="" href="#eqn1">eqn (1) and (3)</a>; the maximum of the imaginary part of the dielectric function (<span class="italic">ε</span><small><sub>2,max</sub></small>);<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> and <span class="italic">ε</span><small><sub>d,max</sub></small>. Finally, we compute the spectral overlap between the OD (d<span class="italic">α</span>(<span class="italic">E</span>), where <span class="italic">d</span> is set to a typical film thickness value of 100 nm and <span class="italic">α</span>(<span class="italic">E</span>) derives from the Gaussian-broadened spectrum of <span class="italic">f</span> in the visible spectral range taking a standard deviation of 0.1 eV) and the AM1.5G solar photon flux spectrum (<span class="italic">Φ</span><small><sub>AM1.5G</sub></small>), namely <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t5.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t5.gif"/></a>.</p>
          <p class="otherpara">These features, together with their corresponding histograms (Fig. S14, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) in terms of Spearman's rank correlation coefficients (<span class="italic">ρ</span>), are explained in more detail in Note S2 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Molecular descriptors are calculated using up to four different open-source packages<a title="Select to navigate to reference" href="#cit73"><sup><span class="sup_ref">73–76</span></sup></a> (Note S2, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) to generate a (curated) collection of 3239 entries (including 40 electronic descriptors derived from the TDDFT calculations, namely the energy of the molecular orbitals ranging from HOMO−19 to LUMO+19). Then, we scan for statistical correlations between those descriptors and all target features introduced above, from which we consider as highly correlated descriptors those showing <span class="italic">ρ</span> ≥ 0.7 as threshold. However, since some descriptors are calculated in groups or families where weighting factors are varied among atomic masses, van der Waals volumes, electronegativities, ionization potentials or polarizabilities, we usually encounter sets of multicollinear descriptors that show very similar trends with respect to the target feature. Accordingly, to drop redundant (collinear) descriptors we classify them into clusters to select the most representative candidate of each bundle (<span class="italic">i.e.</span>, cluster). This serves us to simplify the identification of characteristic and well-correlated descriptors families. The clustering algorithm applied to analyse multicollinear descriptors based on <span class="italic">ρ</span> and <span class="italic">r</span> values is further described in Note S3 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>).</p>
          <p class="otherpara">After running the clusterization of descriptors on all target features, we identify strong correlations with molecular descriptors for <span class="italic">f</span><small><sub>max</sub></small>, <span class="italic">f</span><small><sub>sum,vis</sub></small> and <span class="italic">ε</span><small><sub>d,max</sub></small> (<span class="italic">i.e.</span>, implicitly <span class="italic">f</span><small><sub>max,vis</sub></small>). For the remaining target variables (<span class="italic">f</span><small><sub>overlap</sub></small>, <span class="italic">α</span><small><sub>max</sub></small>, <span class="italic">ε</span><small><sub>2,max</sub></small>, <span class="italic">f</span><small><sub>max</sub></small>/<span class="italic">n</span><small><sub>π</sub></small>, <span class="italic">f</span><small><sub>max,vis</sub></small>/<span class="italic">n</span><small><sub>π</sub></small> and <span class="italic">f</span><small><sub>sum,vis</sub></small>/<span class="italic">n</span><small><sub>π</sub></small>), we do not identify molecular descriptors with <span class="italic">ρ</span> above the threshold value (0.7) and they are generally below 0.6 units, see Fig. S14 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). The lack of correlation for <span class="italic">f</span><small><sub>overlap</sub></small> could be justified by the existence of a gas-to-solid shift in the corresponding absorption spectrum, which prevents proper matching of the Gaussian-broadened absorption features with the solar photon flux. Regarding <span class="italic">α</span><small><sub>max</sub></small> and <span class="italic">ε</span><small><sub>2,max</sub></small>, the estimation of these values from TDDFT calculations requires taking generalized assumptions on several materials properties (such as density or refractive index) that might be enough to disturb the underlying trends in our heterogeneous material database. For the quantities normalised by the number of pi electrons, <span class="italic">i.e. f</span><small><sub>max</sub></small>/<span class="italic">n</span><small><sub>π</sub></small>, <span class="italic">f</span><small><sub>max,vis</sub></small>/<span class="italic">n</span><small><sub>π</sub></small> and <span class="italic">f</span><small><sub>sum,vis</sub></small>/<span class="italic">n</span><small><sub>π</sub></small>, the weak correlation is expected since normalization tends to deviate from linear correlations depending on the straightness of the molecule.<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> Due to the strong correlation between size of the molecule and oscillator strength as discussed below based on C2SP2, the normalised quantity is believed to be a secondary factor, therefore not clear correlations are observed. In the successful correlation cases (<span class="italic">i.e. f</span><small><sub>max</sub></small>, <span class="italic">f</span><small><sub>sum,vis</sub></small> and <span class="italic">ε</span><small><sub>d,max</sub></small>) and with the given thresholds of 0.7 units for <span class="italic">ρ</span> and <span class="italic">r</span>, we identify a single feature cluster lead by the <span class="italic">λ</span><small><sub>1,p</sub></small> descriptor in the case of <span class="italic">f</span><small><sub>max</sub></small> and <span class="italic">ε</span><small><sub>d,max</sub></small> (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2a</a>). For <span class="italic">f</span><small><sub>sum,vis</sub></small>, a threshold <span class="italic">ρ</span> of 0.68 reveals C2SP2 as a rather descriptive molecular feature (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2b</a>). Interestingly, C2SP2 is also found in the main cluster represented by <span class="italic">λ</span><small><sub>1,p</sub></small> in <span class="italic">f</span><small><sub>max</sub></small> and <span class="italic">ε</span><small><sub>d,max</sub></small>, and we could not identify any strong correlations between the absorption strength (in any of its proposed metrics) and electronic descriptors (from HOMO−19 to LUMO+19 energy levels). Note that <span class="italic">ε</span><small><sub>d,max</sub></small> values in excess of 2.5 × 10<small><sup>5</sup></small> M<small><sup>−1</sup></small> cm<small><sup>−1</sup></small> in <a title="Select to navigate to figure" href="#imgfig2">Fig. 2a</a> and b are mostly attributed to artificially straight conjugated oligomers with &gt;10 monomers contained in our database, for which the straightness, hence high <span class="italic">ε</span><small><sub>d,max</sub></small>, are unlikely to be maintained in the experimental solid state scenario. In fact, only the exemplary and asymmetric NFA known as BDTP-4F (inset of <a title="Select to navigate to figure" href="#imgfig2">Fig. 2a</a>)<a title="Select to navigate to reference" href="#cit77"><sup><span class="sup_ref">77,78</span></sup></a> surpasses that threshold with a record <span class="italic">ε</span><small><sub>d,max</sub></small> in our NFA dataset (2.7 × 10<small><sup>5</sup></small> M<small><sup>−1</sup></small> cm<small><sup>−1</sup></small>, and 2.4 × 10<small><sup>5</sup></small> M<small><sup>−1</sup></small> cm<small><sup>−1</sup></small> measured in CHCl<small><sub>3</sub></small> solution).<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a></p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-f2.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">(a) Correlation between, as calculated from TDDFT, and <span class="italic">λ</span><small><sub>1,p</sub></small> as obtained in the database of 479 molecules. The DFT-optimized geometry of BDTP-4F is shown in the inset. (b) Correlation between <span class="italic">ε</span><small><sub>d,max</sub></small> (and <span class="italic">f</span><small><sub>sum,vis</sub></small> in the secondary axis) and C2SP2 in that same database. (c) DFT-optimized geometries of archetypal NFAs ordered by increased values of <span class="italic">λ</span><small><sub>1,p</sub></small> from bottom to top (CBM &lt; Y6 &lt; IEICO-4F &lt; NIBT). Dotted red lines tentatively indicate the overall curvature of the main conjugated backbone of the molecule. <span class="italic">λ</span><small><sub>1,p</sub></small> and C2SP2 describe the size of the molecule in the direction of maximal atomic polarizability, and the number of doubly bound carbon atoms (sp<small><sup>2</sup></small> hybridized) bound to two other carbons (C2), respectively.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">
            <span class="italic">λ</span>
            <small><sub>1,p</sub></small> is part of a bundle of three-dimensional molecular size and shape descriptors known as weighted holistic invariant molecular (WHIM) descriptors.<a title="Select to navigate to reference" href="#cit79"><sup><span class="sup_ref">79–81</span></sup></a> These can be interpreted as a generalized search for the principal axes with respect to a defined atomic property.<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> In this particular case, <span class="italic">λ</span><small><sub>1,p</sub></small> is obtained by performing a principal component analysis (PCA) on the centred atomic coordinates of the molecule using a covariance matrix (<span class="italic">s</span><small><sub><span class="italic">jk</span></sub></small>) that is weighted by the atomic polarizabilities (<span class="italic">p</span><small><sub><span class="italic">i</span></sub></small>):<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn4"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t6.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t6.gif"/></a></td><td class="rightEqn">(4)</td></tr></table>where <span class="italic">s</span><small><sub><span class="italic">jk</span></sub></small> is the weighted covariance between the <span class="italic">j</span>th and <span class="italic">k</span>th atomic coordinates; <span class="italic">A</span> is the total number of atoms; <span class="italic">p</span><small><sub><span class="italic">i</span></sub></small> is the (tabulated) polarizability of the ith atom; <span class="italic">q</span><small><sub><span class="italic">ij</span></sub></small> and <span class="italic">q</span><small><sub><span class="italic">ik</span></sub></small> represent the <span class="italic">j</span>th and <span class="italic">k</span>th coordinate of the ith atom (<span class="italic">j</span>, <span class="italic">k</span> = <span class="italic">x</span>, <span class="italic">y</span>, <span class="italic">z</span>), respectively; and <span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_0071_0304.gif" alt="[q with combining macron]"/></span> is their average value.<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> After diagonalization of the polarizability-weighted covariance matrix, the first eigenvalue (<span class="italic">λ</span><small><sub>1,p</sub></small>) quantifies the size of the molecule in the direction of maximal polarizability variance. Interestingly, the third eigenvalue (<span class="italic">λ</span><small><sub>3,p</sub></small>) approaches zero in planar molecules as a result of absence of variance in the out-of-plane (<span class="italic">z</span>) direction.<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> On the other hand, C2SP2, which is not in the WHIM group, accounts for the number of doubly bound carbon atoms (sp<small><sup>2</sup></small> hybridized, SP2) bound to two other carbons (C2), thus constituting a two-dimensional descriptor of fast computation. The correlation between C2SP2 and absorption strength can be relatively easier to understand, as C2SP2 to some extent represents the size of the conjugated molecule. Enlarging the size of the molecule increases the total number of π-electrons, which controls the total oscillator strength following the Thomas–Reiche–Kuhn rule. For the molecules that are extended along one direction, such as linear oligomers, increasing the size should enhance the oscillator strength of the first transition,<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a><span class="italic">i.e.</span> the dominant one.</p>
          <p class="otherpara">To further interpret these two magnitudes (<span class="italic">λ</span><small><sub>1,p</sub></small> and C2SP2) as the main correlated descriptors with <span class="italic">ε</span><small><sub>d,max</sub></small> and <span class="italic">f</span><small><sub>sum,vis</sub></small>, we inspect the DFT-optimized geometries of archetypal NFAs (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2c</a>). The observed trend suggests that optical extinction monotonically increases with <span class="italic">λ</span><small><sub>1,p</sub></small> (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2a</a>) in molecules having most of their polarizable atoms arranged along a main axis, <span class="italic">i.e.</span>, linear molecules. While CBM shows large torsion angles mainly affecting the 2,1,3-benzothiadiazole (BT) moieties (thus making the molecule non-planar and increasing <span class="italic">λ</span><small><sub>3,p</sub></small>, see Fig. S15, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>), Y6 shows a characteristic curved geometry that limits its <span class="italic">ε</span><small><sub>d,max</sub></small> despite showing improved planarity. The NFA with the highest <span class="italic">λ</span><small><sub>1,p</sub></small> (NIBT) shows both linearity and planarity, with most of the more polarizable atoms (mainly C and S) lying along the principal polarizable axis of the molecule. Thus, in terms of molecular geometry, the absorption strength of NFAs could be further enhanced by distributing most of the atomic polarizability along a main axis while keeping good planarity and minimizing curvature. However, <span class="italic">λ</span><small><sub>1,p</sub></small> is not the sole molecular descriptor governing absorption strength, as BDTP-4F shows <span class="italic">ca.</span> 40% lower <span class="italic">λ</span><small><sub>1,p</sub></small> (0.75 nm<small><sup>2</sup></small>) yet <span class="italic">ca.</span> 40% higher <span class="italic">ε</span><small><sub>d,max</sub></small> than NIBT (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2a</a>), which suggests that the molecular symmetry of NFAs could be another important factor affecting <span class="italic">ε</span><small><sub>d,max</sub></small>. Our preliminary investigations on this issue indicate that molecular asymmetry, as quantified by the WHIM symmetry index <span class="italic">G</span><small><sub>u</sub></small>, might drive absorption strength higher (Fig. S16a, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>), yet we require a larger NFA database including more asymmetric molecules to further explore such an observation. Also, we acknowledge that this observation might be biased by the systematic omission of side chains in the TDDFT calculations. By comparing <span class="italic">λ</span><small><sub>1,p</sub></small> in a selection of small molecule acceptors geometrically optimized with and without side chains (Fig. S17a, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>), we observe that in most cases the addition of side chains either decreases <span class="italic">λ</span><small><sub>1,p</sub></small> slightly or keeps it invariant. Still, the positive correlation of <span class="italic">λ</span><small><sub>1,p</sub></small> with respect to <span class="italic">ε</span><small><sub>d,max</sub></small> is maintained (Fig. S17b, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Furthermore, the presence of naphthalene imide derivatives in the molecular structure of NIBT could be hindering further increase of the absorption strength with <span class="italic">λ</span><small><sub>1,p</sub></small>, as suggested by our statistical analysis of frequent moieties in the selection of good light harvesters (presented in the next section). On the other hand, an increase of <span class="italic">n</span><small><sub>π</sub></small> in the molecule in the form of closed-ring conjugated moieties will systematically increase C2SP2 and accordingly <span class="italic">f</span><small><sub>sum,vis</sub></small>. These findings support the previously known design rules in terms of molecular linearity and π-conjugation enabling large oscillator strength in organic small molecules and polymers, and are consistent with a recent study on chromophores.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> In particular, trans-conjugated polymer stereoisomers are known to possess higher optical extinction due to their increased straightness and persistence length,<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> which agrees with our observations on exemplary curved (Y6) and more linear (NIBT) NFAs.</p>
          <p class="otherpara">The energy of the first optical transition (<span class="italic">E</span><small><sub>1</sub></small>) is also of practical importance in light harvesters such as NFAs as the lower energy part of the solar spectrum, down to ∼1 eV, contains a higher photon flux density. Our results show the number of heteroatoms in the molecule as the most correlated feature with (<span class="italic">ρ</span> = −0.72, Fig. S18a, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) while forming a single feature cluster, yet neither <span class="italic">λ</span><small><sub>1,p</sub></small> nor C2SP2 show strong correlations with <span class="italic">E</span><small><sub>1</sub></small>. This fact prevents the introduction of molecular design rules targeted at <span class="italic">E</span><small><sub>1</sub></small> using <span class="italic">λ</span><small><sub>1,p</sub></small> or C2SP2. However, we acknowledge a negative correlation between <span class="italic">E</span><small><sub>1</sub></small> and <span class="italic">f</span><small><sub>osc,max</sub></small> among common NFAs that suggests further room for absorption strength increase as <span class="italic">E</span><small><sub>1</sub></small> is reduced (Fig. S19, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>).</p>
        </div>
        <div>
          
          <span id="sect870"/><span class="c_heading_indent">2.2.2. Chemical insights into highly absorbing molecules. </span>
          <span>Beyond molecular descriptors, we investigate the relationship between the choice of moieties and absorption strength to provide further material design rules for highly absorbing conjugated small molecules. Our objective is to identify overrepresented moieties in the subset of high-absorbing molecules (which we arbitrarily define as those having <span class="italic">f</span><small><sub>osc,max</sub></small> &gt; 2.5, thus setting a population of size <span class="italic">p</span>) with respect to the entire molecular dataset (population of size <span class="italic">P</span>). Accordingly, we identify the molecular motifs present in the molecules by comparing their structures (as derived from SMILES notation) with those of a previously built database of moieties (also SMILES-based). This database of moieties was partly inherited from a previous work<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> and extended with further motifs present in our particular dataset (see Note S4 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) and the spreadsheet included as ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Afterwards, we consider that a discrete hypergeometric distribution is adequate to model our molecular dataset and the fragments found therein<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> to calculate the corresponding <span class="italic">Z</span>-scores as <span class="italic">Z</span> = (<span class="italic">k</span> − <span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_006b_0304.gif" alt="[k with combining macron]"/></span>)/<span class="italic">σ</span><small><sub><span class="italic">k</span></sub></small>, where <span class="italic">k</span> is the number of high-absorbing molecules containing certain moiety; is its expected value, defined as <span class="italic">pK</span>/<span class="italic">P</span> where <span class="italic">K</span> corresponds to the number of molecules in the entire dataset containing that same moiety; and <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t7.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t7.gif"/></a> is the standard deviation of the hypergeometric distribution. <span class="italic">Z</span>-scores will indicate (in units of <span class="italic">σ</span><small><sub><span class="italic">k</span></sub></small>) which moieties are overrepresented or underrepresented in the subset of high-performing molecules with respect to the expected values when looking at the entire dataset. Our results (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>) suggest that thieno[3,2-<span class="italic">b</span>]thiophene (TT), thiophene (T), 2-(5,6-difluoro-3-oxo-2,3-dihydro-1<span class="italic">H</span>-inden-1-ylidene)malononitrile (2FIC), 2-(3-oxo-2,3-dihydro-1<span class="italic">H</span>-inden-1-ylidene)malononitrile (IC), indaceno[1,2-<span class="italic">b</span>:5,6-<span class="italic">b</span>′]dithiophene (IDT), 2-methylene malononitrile, cyanide, and aniline are particularly frequent in highly absorbing molecules. Interestingly, four of those molecular fragments (TT, T, 2FIC and IC) are contained in the chemical structure of the workhorse NFA Y6 (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2c</a>). Contrarily, naphthalene imide derivatives, as typically encountered in n-type small molecules and conjugated polymers; 4<span class="italic">H</span>,8<span class="italic">H</span>-benzo[1,2-<span class="italic">c</span>:4,5-<span class="italic">c</span>′]dithiophene-4,8-dione and benzo[1,2-<span class="italic">b</span>:4,5-<span class="italic">b</span>′]dithiophene fragments are mostly underrepresented in the selection of high-performing light harvesters.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-f3.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">(a) <span class="italic">Z</span>-scores obtained from the discrete hypergeometric distribution of moieties in the highly-absorbing molecules (<span class="italic">f</span><small><sub>osc,max</sub></small> &gt; 2.5) with respect to the entire molecular dataset, for moieties activated at least 10 times. The corresponding structures of identified moieties are shown. (b) Network graph of <span class="italic">Z</span>-scores relating pairs of moieties. Source nodes are coloured in black whereas child nodes are coloured in grey. The colour of the edges corresponds to the sign of the <span class="italic">Z</span>-score (green for positive, red for negative). The width of the edges scales with the absolute value of the <span class="italic">Z</span>-score.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">We further study the existing correlation between pairs of moieties to understand in which way the different molecular fragments should (or should not) be combined to retrieve highly-absorbing molecules. Our analysis starts by creating molecular subsets determined by the presence of a given moiety, which acts as source node (coloured in black) in the network graph shown in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3b</a>. Within that subset, we identify the high-absorbing molecules (<span class="italic">f</span><small><sub>osc,max</sub></small> &gt; 2.5) and compute the <span class="italic">Z</span>-scores of their moieties (child nodes, coloured in grey in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3b</a>) with respect to the molecules of the entire molecular subset. As per the network shown in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3b</a>, the absolute <span class="italic">Z</span>-scores will determine the width of the edges connecting the nodes (moieties) and its sign the colour of the edge (green for positive <span class="italic">Z</span>-score [overrepresentation] and red for negative <span class="italic">Z</span>-score [underrepresentation]). Therefore, green and thick edges connect pairs of molecules that are more frequently found in high-absorbing molecules whereas thick and red edges indicate combinations of moieties that lead to less absorbing molecules. In this analysis, we set up 8 different source nodes corresponding to the most overrepresented moieties observed in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3a</a>. As a result, <a title="Select to navigate to figure" href="#imgfig3">Fig. 3b</a> can be interpreted as a catalogue of design rules relating pairs of moieties with high oscillator strength in π-conjugated small molecules.</p>
        </div>
      
      
        
        <h3 id="sect931"><span class="b_heading">2.3. Machine-learning modelling of the absorption strength</span></h3>
        <span>Besides providing useful chemical insights from a material design perspective, molecular descriptors can be exploited to feed regression models and draw predictions on certain target features, forming the so-called quantitative structure–property relationship (QSPR) and quantitative structure–activity relationship (QSAR) models.<a title="Select to navigate to reference" href="#cit79"><sup><span class="sup_ref">79,80,82,83</span></sup></a> In the present study, we train and test several ML models fed with molecular and electronic descriptors obtained from TDDFT calculations to predict the value of <span class="italic">ε</span><small><sub>d,max</sub></small> in conjugated small molecules and oligomers. Finally, we propose exploiting such ML model (trained with TDDFT data) to predict <span class="italic">ε</span><small><sub>d,max</sub></small> in molecules optimized using a semi-empirical quantum chemistry method, <span class="italic">i.e.</span> xTB.<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> This renders possible thanks to the geometrical similarity of the TDDFT and xTB ground state conformers, which lead to similar (geometrical) descriptors values; and the calibration of their corresponding energy levels, as per the required inputs of the ML model herein employed. Therefore, further molecular candidates beyond the pristine dataset could be geometrically optimized using solely xTB Hamiltonians and their absorption strength predicted using such ML model. This approach effectively bypasses the use of TDDFT calculations when screening the absorption strength of novel molecules, which results in less demanding computations and higher throughput. The present ML workflow will open the possibility to accelerate the screening of high-performing molecular candidates with low-to-moderate computational requirements (further discussed in Note S5, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>).</span>
        <div>
          
          <span id="sect943"/><span class="c_heading_indent">2.3.1. Modelling <span class="italic">ε</span><small><sub>d,max</sub></small> with random decision forests. </span>
          <span>From the analysis of descriptors shown in Section 2.2.1, we identified two main feature clusters represented by <span class="italic">λ</span><small><sub>1,p</sub></small> and C2SP2. We tentatively consider these two descriptors as independent variables in baseline models (such as 1-nearest neighbour and linear regression) targeted to <span class="italic">ε</span><small><sub>d,max</sub></small>. For the model training and testing, we split our pristine dataset onto two subsets, namely the training set (gathering 70% of the data, randomly selected) and the testing set (gathering the remaining 30% of the data). Such baseline models are picked according to a recently introduced catalogue of good practices in the ML field,<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> to demonstrate the requirement of more advanced regressors (namely ML) in successful data modelling. The models are scored and quantitatively compared based on workhorse fitting metrics, such as their coefficient of determination (<span class="italic">R</span><small><sup>2</sup></small>); their adjusted coefficient of determination (<span class="italic">R</span><small><sub>adj</sub></small><small><sup>2</sup></small>, which adds penalties as the number of parameters increases, see Note S2, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>); and their Pearson correlation coefficient (<span class="italic">r</span>), as retrieved in the training (fitting) and test sets. The inherent mathematical simplicity of the baseline models results in poor fitting scorings (Fig. S20 and Table S1, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) yet they suggest that feature selection procedures could end up in higher-performing models.</span>
          <p class="otherpara">Accordingly, we deploy a state-of-the-art ML method, namely a RF, to aid in both aspects: feature selection and building of <span class="italic">ε</span><small><sub>d,max</sub></small> models of higher accuracy. RFs constitute one of the simplest and most widely applied ML methods in molecular screening and data mining studies.<a title="Select to navigate to reference" href="#cit30"><sup><span class="sup_ref">30,43,46,86</span></sup></a> They are particularly appealing for their straightforward implementation through open-source Python libraries such as Scikit-Learn,<a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> and also for their inherent robustness against overfitting and fast optimization. RFs are formed by an ensemble of decision trees (estimators) that are executed in parallel and independently from each other. Decision trees serve to classify data by starting from a single root node that is subsequently divided into child nodes, the latter being chosen randomly among the input features. At every node splitting step (<span class="italic">i.e.</span>, decision making), the algorithm selects the pathway that minimizes the mean square error (MSE). Eventually, when every tree reaches its maximum extension (which is set arbitrarily <span class="italic">via</span> model hyperparameters), the predictions of all trees are averaged (ensembled), hence constituting the final predicted value of the RF. At this stage, myriad cross-validation (CV) techniques exist to evaluate the quality of the model and help in the tuning of hyperparameters. CV methods can estimate the ML model performance, evaluate potential over- or underfitting, and quantify how accurate the model is on drawing predictions on unseen data. In this work, we adopt two common cross-validation schemes, namely a repeated holdout CV; and a leave-one-out cross-validation (LOOCV). On the one hand, in a repeated holdout CV the pristine dataset is randomly split onto two distinct subsets, namely the training (here gathering 70% of the data) and testing (the remaining fraction of data, <span class="italic">i.e.</span> 30%) subsets. The model is trained and tested on the respective subsets, and the corresponding statistical metrics (<span class="italic">R</span><small><sup>2</sup></small>, <span class="italic">r</span>, MSE, <span class="italic">etc.</span>) annotated. Eventually, the process is repeated <span class="italic">k</span> times (10-fold in this work), and all metrics are averaged to evaluate the ML model performance (its CV score). On the other hand, in a LOOCV the holdout process is taken to the extreme as the testing subset consists of a single data point while the remaining data is used in the training step. The process runs recursively for all data, thus eventually all data points are used for training and testing in the LOOCV protocol. Yet being computationally expensive, a LOOCV results in a more accurate estimate of model performance.</p>
          <p class="otherpara">Table S1 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) includes the performance of an out-of-the-box RF model trained and cross-validated using 300 trees (estimators). Exemplary comparisons between the two previous baseline models (1-nearest neighbor and linear regression) and the out-of-the-box RF model are found in Fig. S20 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). The RF models indicate that scoring functions (<span class="italic">R</span><small><sup>2</sup></small>, <span class="italic">r</span>) well above 0.6–0.8 are feasible upon careful feature selection and further optimization of the RF regressor. Feature selection in RFs is usually performed by filtering variables based on their feature importance, which is a metric that accounts for how much a feature decreases the weighted variance in the node splitting steps of the decision trees. This property enables feature ranking to then apply myriad algorithms to filter out the least important variables as seen by the RF regressor. In this work, we perform a recursive feature elimination (RFE) procedure to the initial library of 3239 descriptors as described in Note S2 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). In a RFE protocol, a significant fraction of the initial population of features is dropped in successive training steps of the RF ensemble. Features are dropped based on their corresponding feature importance until reaching an arbitrarily low number of input variables, hence simplifying the original model. Our RFE analysis shows that a threshold average <span class="italic">R</span><small><sup>2</sup></small> of 0.70 is achieved using a 12-variable model (<span class="italic">R</span><small><sup>2</sup></small> = 0.70 ± 0.05, <span class="italic">r</span> = 0.84 ± 0.03), which outperforms the RF model presented earlier while including a drastic reduction in the number of variables (from 3239 to 12). The sweet spot in model accuracy and number of degrees of freedom is found for the 10-variable model, which shows the maximum average <span class="italic">R</span><small><sub>adj</sub></small><small><sup>2</sup></small> (0.67 ± 0.06).</p>
          <p class="otherpara">Notably, a threshold <span class="italic">R</span><small><sup>2</sup></small> of 0.60 is already achieved training a 3-parameter RF model (<span class="italic">R</span><small><sup>2</sup></small> = 0.63 ± 0.06, <span class="italic">R</span><small><sub>adj</sub></small><small><sup>2</sup></small> = 0.62 ± 0.06, <span class="italic">r</span> = 0.80 ± 0.03), which is particularly appealing given its simplicity. The resulting three-variable model includes one three-dimensional descriptor (<span class="italic">λ</span><small><sub>1,v</sub></small> or WHIM_45, as computed by the RDKit library, <a title="Select to navigate to figure" href="#imgfig4">Fig. 4a</a>), one two-dimensional descriptor (CIC3, as computed by PaDEL software, <a title="Select to navigate to figure" href="#imgfig4">Fig. 4b</a>) and one electronic descriptor, in this case the energy level of the second molecular orbital below the frontier HOMO (HOMO−2, <a title="Select to navigate to figure" href="#imgfig4">Fig. 4c</a>). <span class="italic">λ</span><small><sub>1,v</sub></small> refers to the first eigenvalue of the covariance matrix weighted by the atomic van der Waals volumes; thus, <span class="italic">λ</span><small><sub>1,v</sub></small> is included in the multicollinear feature cluster represented by <span class="italic">λ</span><small><sub>1,p</sub></small> that we previously and statistically identified, showing nearly perfect correlation (<span class="italic">r</span> = 0.99) with <span class="italic">λ</span><small><sub>1,p</sub></small>. Accordingly, <span class="italic">λ</span><small><sub>1,v</sub></small> can be exchanged by <span class="italic">λ</span><small><sub>1,p</sub></small> without loss of performance in the RF model. This finding confirms that the linearity of the molecule (either quantified in terms of polarizabilities or van der Waals volumes) plays a key role in determining its absorption strength in the form of <span class="italic">ε</span><small><sub>d,max</sub></small>. On the other hand, CIC3 is a graph-based, third-order neighbourhood symmetry index<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> which lacks a straightforward interpretation due to its mathematical complexity. We observe, however, that it linearly scales as log<small><sub>2</sub></small><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">A</span>, with <span class="italic">A</span> being the total number of vertices (atoms) in the graph (molecule)<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> thus likely reflecting the size of π-conjugation as per the characteristics of our dataset. The interpretation of HOMO−2 as an important descriptor is more challenging, and it is not possible to substitute it by a different descriptor without a noticeable drop in the model performance (excepting HOMO−1, which shows <span class="italic">r</span> = 0.96).</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-f4.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Correlation plots for <span class="italic">ε</span><small><sub>d,max</sub></small> and the three most important descriptors retrieved by the RF model: (a) <span class="italic">λ</span><small><sub>1,v</sub></small>; (b) CIC3; and (c) HOMO−2. (d) Holdout cross-validation run of a RF ensemble to predict <span class="italic">ε</span><small><sub>d,max</sub></small>. 70% of the data is randomly selected for training and the remaining fraction is used for testing; the process is repeated 10 times and the statistical metrics averaged. The RF model is trained with three molecular descriptors (<span class="italic">λ</span><small><sub>1,v</sub></small>; CIC3; and HOMO-2) and a Morgan fingerprint vector of 64 bits. (e) Leave-one-out cross-validation (LOOCV) of that same RF model using the optimized hyperparameter of 1200 estimators.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Interestingly, electronic descriptors (in particular) are required for the RF models to achieve their highest potential and scoring despite we have not observed strong correlations in our earlier statistical analysis. To probe it, we have performed the same RFE protocol yet skipping the set of electronic descriptors among the input features. Our results show that the top performing RF models (selecting 29 variables and getting <span class="italic">R</span><small><sup>2</sup></small> = 0.58 ± 0.06, <span class="italic">R</span><small><sub>adj</sub></small><small><sup>2</sup></small> = 0.48 ± 0.07, <span class="italic">r</span> = 0.78 ± 0.04; or selecting 9 variables to obtain <span class="italic">R</span><small><sub>adj</sub></small><small><sup>2</sup></small> = 0.52 ± 0.06, see Fig. S13, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) are yet behind the scorings recorded when the electronic descriptors are included in the list of features. Note that the performance without electronic descriptors is lower than the 3-parameter model that includes HOMO−2 as descriptor, highlighting its positive effect on the performance of the RF regressor.</p>
          <p class="otherpara">Molecular fingerprints have also been extensively exploited as input vectorial descriptors in statistical and ML models focused on feature prediction.<a title="Select to navigate to reference" href="#cit42"><sup><span class="sup_ref">42,88–90</span></sup></a> Molecular fingerprints are usually represented as bit activation vectors of arbitrary length and degree of complexity, representing the absence or presence of certain molecular (bonding) pattern, moiety, functional group, or atom. In this work, we exploit the RDKit library to generate moiety fingerprints, MACCS keys, Morgan fingerprints, path-based or topological fingerprints, E-state fingerprints, and Coulomb vectors. These fingerprints are quickly computed and serve to complement and improve the learning process of the ML models employed herein.</p>
          <p class="otherpara">To better analyse the influence of the different fingerprint vectors in improving the RF scoring, we trained and cross-validated the 3-parameter RF model previously found in combination with all fingerprint vectors generated. The results shown in Table S2 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) indicate that by adding a Morgan fingerprint vector of 64 bits to the initial set of input features the model performance can be substantially improved: <span class="italic">R</span><small><sup>2</sup></small> increases by 10% (relative), and <span class="italic">r</span> by another (relative) 5% (see <a title="Select to navigate to figure" href="#imgfig4">Fig. 4d</a>). Therefore, Morgan fingerprints are particularly suitable to fine-tune the training and prediction accuracy of <span class="italic">ε</span><small><sub>d,max</sub></small> in RF models although lacking of a straightforward physical interpretation. Additional refinement of the RF hyperparameters results in further improved models. We performed this optimization through a randomized search (in 350 iterations) of the hyperparameters controlling the number of estimators in the RF, the minimum number of samples per leaf node and the minimum number of samples required to split an internal node, which constitute the main adjustable hyperparameters of the RF algorithm. These results are shown in Table S3 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>), together with the scoring obtained in a rigorous LOOCV of the optimized RF model (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4e</a>). As an alternative ensemble of decision trees, we have also tested and optimized an Extra Trees (ET) regressor in Scikit-Learn. Its performance is, however, very close to that attained in the workhorse RF regressor (Table S3 and Fig. S21, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>).</p>
        </div>
        <div>
          
          <span id="sect1059"/><span class="c_heading_indent">2.3.2. Bypassing TDDFT calculations through machine-learning and extended tight-binding. </span>
          <span>xTB Hamiltonians have recently emerged as semi-empirical and low computational cost quantum chemistry methods.<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> These have a remarkable potential in molecular screening when implemented in multilevel workflows where xTB is exploited first to identify plausible candidates using a minimal fraction of computational resources, to then leave room for higher-level DFT methods in selected candidates.<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> In this work, we propose exploiting a ML model trained with DFT data to predict <span class="italic">ε</span><small><sub>d,max</sub></small> in molecular geometries optimized using xTB (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5a</a>). This is expected to enable faster molecular screening and geometrical optimization steps, as both being entirely run using xTB Hamiltonians; followed by absorption strength (<span class="italic">ε</span><small><sub>d,max</sub></small>) prediction in a TDDFT-trained RF model. Notably, our estimations show that the geometrical optimization step using GFN2-xTB is <span class="italic">ca.</span> 3000 times faster than using DFT with a hybrid functional (B3LYP/6-311+G(d,p)), as discussed in Note S5 and Table S4 (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>).</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-f5.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">(a) ML workflow used in this work to draw <span class="italic">ε</span><small><sub>d,max</sub></small> predictions. A RF model is trained on TDDFT data and interpolated (validated) on xTB geometries, including also their corresponding molecular descriptors. To improve the accuracy of the model, energy levels obtained using the GFN2-xTB Hamiltonian require calibration with TDDFT values (Fig. S23, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). (b) Leave-one-out interpolation of the resulting RF model using three input molecular descriptors (including calibrated energy levels) and a 64-bit Morgan fingerprint vector.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Nevertheless, the dissimilarity between xTB- and DFT-optimized molecular geometries might have a direct impact on the value of the (three-dimensional) molecular descriptors, and hence on the final accuracy of the interpolated ML model if some of those are included. Accordingly, we have first quantitatively compared both sets of molecular (non-electronic) descriptors by computing <span class="italic">r</span> in all of them and found that the median of their distributions is very close to unity in all cases (Fig. S22, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Based on this finding, we proceed by training the RF model with TDDFT-derived descriptors and exploring how well the model interpolates when fed with xTB-derived descriptors. Fig. S23a (ESI<a title="Select to navigate to footnote" href="#fn1">†</a>) shows a leave-one-out interpolation of a RF model trained using TDDFT data and interpolated on GFN2-xTB-optimized molecules, descriptors and energy levels.<a title="Select to navigate to reference" href="#cit84"><sup><span class="sup_ref">84,91,92</span></sup></a> In this kind of model validation, all TDDFT data is used in the training step excepting that for a single molecule, for which we retrieve its corresponding xTB-optimized geometry and descriptors as the sole interpolation (testing) dataset; this procedure is subsequently repeated for all molecules. Thus, the model performance is assessed by comparing the actual TDDFT-derived <span class="italic">ε</span><small><sub>d,max</sub></small> of the molecules (<span class="italic">x</span>-axis in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5b</a>) with that predicted by a RF model trained with TDDFT data and interpolated using xTB-derived descriptors (<span class="italic">y</span>-axis in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5b</a>). This is useful to evaluate whether such RF model fed with TDDFT data could be exploited to predict <span class="italic">ε</span><small><sub>d,max</sub></small> in unseen molecules that are geometrically optimized through xTB Hamiltonians.</p>
          <p class="otherpara">Our first model takes as inputs the three molecular descriptors found previously to be the most important features in the RF model together with their corresponding (64-bit) Morgan fingerprints. The scoring of the LOOCV in this preliminary model (<span class="italic">R</span><small><sup>2</sup></small> = 0.53, <span class="italic">r</span> = 0.74) is limited due to the existence of a mismatch between the absolute energy levels retrieved by either DFT (B3LYP) or GFN2-xTB methods (Fig. S23b, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). Thus, the RF model trained on TDDFT data needs proper calibration of the energy levels obtained through GFN2-xTB, which we perform using either a linear regression, a support vector regressor (SVR) or an additional RF model (Fig. S23c, ESI<a title="Select to navigate to footnote" href="#fn1">†</a>). By applying such calibration on the HOMO−2 energy levels, we obtain the champion RF model (<span class="italic">R</span><small><sup>2</sup></small> = 0.61, <span class="italic">r</span> = 0.78) shown in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5b</a> using three molecular descriptors and a 64-bit Morgan fingerprint vector. Hence, <a title="Select to navigate to figure" href="#imgfig5">Fig. 5b</a> shows that molecular databases of xTB-optimized geometries could be exploited in combination with TDDFT-trained ML models to predict the absorption strength (<span class="italic">ε</span><small><sub>d,max</sub></small>) at significantly lower computational cost and with reasonable accuracy. The statistical analysis and ML modelling framework introduced here is thus expected to show large potential in the high-throughput screening of highly absorbing molecular candidates in combination with generative models (autoencoders and neural networks) as part of future work in the group.</p>
        </div>
      
    
    
      
      <h2 id="sect1103"><span class="a_heading">3. Conclusion</span></h2>
      <span>We have demonstrated that TDDFT calculations agree reasonably well with the experimental maximum molar extinction coefficient (<span class="italic">ε</span><small><sub>d,max</sub></small>) in solution state by exploiting a database of TDDFT-optimized small molecular acceptors (NFAs) and donor oligomers collected over the years. This finding supports further analysis of the molecular dataset to identify structure–absorption relationships by means of statistical and machine-learning (ML) methods. Through the exploration of molecular descriptors, we identify two features that are strongly correlated with <span class="italic">ε</span><small><sub>d,max</sub></small>, namely the linearity and planarity of the molecule in the direction of maximum atomic polarizability variance; and the number of sp<small><sup>2</sup></small>-hybridized carbon atoms bonded to two other carbons included in the molecule. These further suggest design rules that highly absorbing organic π-conjugated molecules (such as NFAs) should follow, namely a fully conjugated, planar and linear molecular backbone with more polarisable heteroatoms. We further identify that moieties such as thieno[3,2-<span class="italic">b</span>]thiophene (TT), thiophene (T), 2-(5,6-difluoro-3-oxo-2,3-dihydro-1<span class="italic">H</span>-inden-1-ylidene)malononitrile (2FIC), 2-(3-oxo-2,3-dihydro-1<span class="italic">H</span>-inden-1-ylidene)malononitrile (IC) and indaceno[1,2-<span class="italic">b</span>:5,6-<span class="italic">b</span>′]dithiophene (IDT) appear more frequently in molecules with the highest absorption strength. Finally, we demonstrate the feasibility of random decision forests (RFs) trained with a few (3) molecular descriptors and 64-bit Morgan fingerprint vectors to predict in molecular geometries optimized by a computationally less demanding method such as extended tight-binding (xTB). This approach shows the ability to bypass thorough TDDFT calculations, thus facilitating high-throughput screening of absorption strength in organic π-conjugated molecules in combination with generative molecular models.</span>
    
    
      
      <h2 id="sect1117"><span class="a_heading">4. Outlook</span></h2>
      <span>This work was motivated by the search for molecular design rules to enable higher PCE in organic solar cells. Although maximizing light absorption for a given optical band gap is a key requirement to enable record PCE, many additional physical processes contribute to photovoltaic performance but are not considered directly in the present work, namely, exciton diffusion, charge transfer, charge separation, charge transport and charge recombination. To date, there is no holistic modelling framework nor are there sufficient data to relate these multiple processes to device performance <span class="italic">via</span> chemical structure. However, developments in AI and ML methods are likely to advance the status of models for multiple property–device performance relationships in the coming years.</span>
      <p class="otherpara">Nevertheless, understanding how light harvesting alone can be maximized by smart molecular design is significant for improving several different aspects of OPV performance. Light absorption is the primary step towards charge generation and is therefore strongly related to the macroscopic short-circuit current density of the device. According to the reciprocity relation between absorption and emission,<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a> high absorption should in principle lead to strong emission, therefore reducing the nonradiative energy losses, and benefitting the open-circuit voltage. In addition, high absorption allows the fabrication of thin devices, therefore facilitating charge extraction and enhancing fill factor.<a title="Select to navigate to references" href="#cit93"><sup><span class="sup_ref">93</span></sup></a> Moreover, based on the causality principle, high absorption strength would lead to higher refractive index, which takes the first interference maximum of electric field to lower thicknesses, resulting in large light harvesting potential in thinner devices. Therefore, designing highly absorbing organic π-conjugated molecules has the potential to enhance different aspects relating to the performance of OPVs in conjunction with the proposed predictive ML model.</p>
      <p class="otherpara">A separate aspect for future work is the impact of solid-state molecular interactions on light absorption. This paper concerns the optical absorption of isolated molecules while applications normally require thin films of molecules. Although intermolecular interactions can strongly impact the strength as well as the spectrum of thin film absorption,<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> this has been neglected in the present study due to the lack of a suitable database of computations and the lack of solid state packing information. In the future, ML approaches could be used to better understand and predict how solid state interactions affect optical absorption, and thereby improve molecular design rules. Such advances may be enabled by the growing capability in computational structure prediction as well as improved understanding of the impact of intermolecular interactions on excited state properties.</p>
    
    
      
      <h2 id="sect1127"><span class="a_heading">5. Experimental and theoretical methods</span></h2>
      <span>Excited state calculation database and experimental <span class="italic">ε</span><small><sub>d,max</sub></small> database: TDDFT results in this study are based on the functional B3LYP and were performed by present and past group members in Prof. Jenny Nelson's group at Imperial College London, making up more than 3500 entries (corresponding to 479 unique molecules). The majority of experimental solid state thin film <span class="italic">ε</span><small><sub>d,max</sub></small> values for NFAs shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1a–c</a> were measured using variable-angle spectroscopic ellipsometry (VASE) for the present study. Neat films were deposited from solution by either spin- or blade-coating on glass substrates at distinct thicknesses (typically ranging from 30 to 150 nm). Ellipsometry data were acquired at three to five angles of incidence (55°–75°) using a Sopralab GES-5E rotating polarizer spectroscopic ellipsometer (SEMILAB) coupled to a charge-coupled device (CCD) detector. Experimental solution <span class="italic">ε</span><small><sub>d,max</sub></small> were mostly collected from literature with a majority of data taken from ref. <a title="Select to navigate to references" href="#cit95">95</a>, and Y5, Y6, and Y7 measured using UV-visible spectroscopy. The complete database and sources are presented in the ESI.<a title="Select to navigate to footnote" href="#fn1">†</a></span>
      <p class="otherpara">Theoretical description of molar extinction coefficient (<span class="italic">ε</span><small><sub>d</sub></small>): to calculate the molar extinction coefficient <span class="italic">ε</span><small><sub>d</sub></small>, let us start with defining the absorption coefficient <span class="italic">α</span> in a quantum picture (we stay with SI units for the moment). The absorption coefficient for transition from state 1 to state 2 can be defined as<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19,96</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn5"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t8.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t8.gif"/></a></td><td class="rightEqn">(5)</td></tr></table>where <span class="italic">I</span> is light intensity, determined by the energy density of an electromagnetic wave <span class="italic">via</span><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn6"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t9.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t9.gif"/></a></td><td class="rightEqn">(6)</td></tr></table>where <span class="italic">n</span> is the refractive index, <span class="italic">ε</span><small><sub>0</sub></small> vacuum permittivity, <span class="italic">c</span> the speed of light, and <span class="italic">E</span><small><sub>0</sub></small> the amplitude of the electric field. For an electromagnetic wave, the rate of intensity attenuation <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t10.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t10.gif"/></a> is equal to the rate of loss of energy density from the field <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t11.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t11.gif"/></a>, and the latter is the product of transition rate <span class="italic">Γ</span><small><sub>12</sub></small> and transition energy <span class="italic">ħω</span><small><sub>12</sub></small>, and we have<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn7"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t12_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t12.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t12.gif"/></a></td><td class="rightEqn">(7)</td></tr></table>where <span class="italic">N</span> is the volume density of molecules and <span class="italic">ħ</span> the reduced Planck constant. Substituting for <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t13_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t13.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t13.gif"/></a> and <span class="italic">I</span> in the definition of <span class="italic">α</span><small><sub>12</sub></small> we get<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn8"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t14_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t14.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t14.gif"/></a></td><td class="rightEqn">(8)</td></tr></table>The transition rate <span class="italic">Γ</span><small><sub>12</sub></small> can be defined by Fermi's Golden Rule and the perturbing Hamiltonian given by <span class="italic">H</span> = <span class="italic">d</span><small><sub>12</sub></small><span class="italic">E</span><small><sub>0</sub></small> using dipole approximation, where <span class="italic">d</span><small><sub>12</sub></small> is the transition dipole moment of the transition. Considering randomly oriented transition dipoles relative to the direction of the exciting electromagnetic field, we have<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn9"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t15_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t15.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t15.gif"/></a></td><td class="rightEqn">(9)</td></tr></table>Using <span class="italic">E</span><small><sub>2</sub></small> − <span class="italic">E</span><small><sub>1</sub></small> = <span class="italic">ħω</span><small><sub>12</sub></small>, we get<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn10"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t16_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t16.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t16.gif"/></a></td><td class="rightEqn">(10)</td></tr></table>From an arbitrary transition from state <span class="italic">i</span> to state <span class="italic">j</span>, we can express above equation using oscillator strength of the transition (<span class="italic">f</span><small><sub><span class="italic">ij</span></sub></small>):<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn11"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t17_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t17.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t17.gif"/></a></td><td class="rightEqn">(11)</td></tr></table>where <span class="italic">e</span> is the elementary charge, and <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t18_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t18.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t18.gif"/></a>. Integrating over all transitions, we have<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn12"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t19_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t19.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t19.gif"/></a></td><td class="rightEqn">(12)</td></tr></table>To correlate the absorption coefficient (<span class="italic">α</span>) with the molar extinction coefficient (<span class="italic">ε</span><small><sub>d</sub></small>), we need the definition of optical density (OD) and optical depth (<span class="italic">αd</span>). Light is attenuated by passing through a depth <span class="italic">d</span> of material such that<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn13"/><span id="eqn13"><span class="italic">I</span>(<span class="italic">d</span>) = <span class="italic">I</span><small><sub>0</sub></small>e<small><sup>−<span class="italic">αd</span></sup></small> = <span class="italic">I</span><small><sub>0</sub></small>10<small><sup>−OD</sup></small></span></td><td class="rightEqn">(13)</td></tr></table>And optical density, or called sometimes absorbance is defined as.<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn14"/><span id="eqn14">OD = <span class="italic">ρε</span><small><sub>d</sub></small><span class="italic">d</span>,</span></td><td class="rightEqn">(14)</td></tr></table>where <span class="italic">ρ</span> is concentration in molar (M or mol L<small><sup>−1</sup></small>), and <span class="italic">d</span> is sample length in cm. Consequently, we have<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn15"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t20_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t20.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t20.gif"/></a></td><td class="rightEqn">(15)</td></tr></table>noting that we now write the absorption coefficient per cm to distinguish from the expression for <span class="italic">α</span> above, which we did assuming SI units, hence <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t21_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t21.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t21.gif"/></a>. <span class="italic">ρ</span> is moles of molecules per dm<small><sup>3</sup></small>. We now have<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn16"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t22_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t22.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t22.gif"/></a></td><td class="rightEqn">(16)</td></tr></table>Let us recast this in terms of photon energy <span class="italic">E</span> in eV, <span class="italic">i.e.</span><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t23_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t23.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t23.gif"/></a>, rather than angular frequency, so it is easier to consider the magnitude, and finally we have <span class="italic">ε</span><small><sub>d</sub></small> in the unit of M<small><sup>−1</sup></small> cm<small><sup>−1</sup></small>.<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn17"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t24_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t24.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t24.gif"/></a></td><td class="rightEqn">(17)</td></tr></table>This allows us to compute the theoretical <span class="italic">ε</span><small><sub>d</sub></small> using the calculated oscillator strength at different transitions. And the common method to calculate the oscillator strength is time-dependent-density-functional-theory (aka TDDFT).</p>
      <p class="otherpara">Converting complex refractive index from solid state ellipsometry measurements to <span class="italic">ε</span><small><sub>d</sub></small>: using ellipsometry measurements from film (solid state), we can extract the complex refractive index, <span class="italic">η</span><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn18"/><span id="eqn18"><span class="italic">η</span> = <span class="italic">n</span> + <span class="italic">iκ</span></span></td><td class="rightEqn">(18)</td></tr></table>where <span class="italic">n</span> is the refractive index, and <span class="italic">κ</span> the extinction coefficient. The absorption coefficient (<span class="italic">α</span><small><sub>cm</sub></small>) is then determined by<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn19"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t25_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t25.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t25.gif"/></a></td><td class="rightEqn">(19)</td></tr></table>where <span class="italic">λ</span><small><sub>cm</sub></small> is the wavelength in centimetre. Using <a title="" href="#eqn15">eqn (15)</a>, and the relationship between molar concentration <span class="italic">ρ</span> and mass concentration <span class="italic">ρ</span><small><sub>M</sub></small>, <span class="italic">i.e.</span>, <a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t26_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t26.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t26.gif"/></a>, we have<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn20"/><a href="/image/article/2022/EE/d2ee00887d/d2ee00887d-t27_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ee00887d-t27.tif" src="/image/article/2022/EE/d2ee00887d/d2ee00887d-t27.gif"/></a></td><td class="rightEqn">(20)</td></tr></table>where <span class="italic">M</span><small><sub>w</sub></small> is the molecular weight, g mol<small><sup>−1</sup></small>, and <span class="italic">ρ</span><small><sub>M</sub></small> has the unit of g L<small><sup>−1</sup></small>, and is typically assumed to be 1000 g L<small><sup>−1</sup></small>.</p>
    
    
      <h2 id="sect1296"><span class="a_heading">Author contributions</span></h2>
      <span>J. Y. and X. R.-M. contributed equally to this work and drafted the paper. J. Y. performed DFT and TDDFT calculations, absorption strength analysis, and data collection. X. R.-M. performed the statistical analysis and machine-learning study. D. P., H. D., D. B., M. A., A. V., S. F., A. A. S., and X. H. shared their DFT/TDDFT calculation results. F. E. prepared thin films of NFAs for VASE measurements. X. R.-M., V. B., and B. D. did VASE measurements. E. R. did UV-vis measurements of Y5, Y6, and Y7 in solution. G. Z. and H.-L. Y. provided Y5, Y6, and Y7. All authors gave critical review on this work. J. N. and M. C.-Q. supervised this work.</span>
    
    
      <h2 id="sect1299"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <h2 id="sect1303"><span class="a_heading">Acknowledgements</span></h2>
      <span>J. N., J. Y., D. P., M. A., F. E., and E. R. thank the European Research Council for support under the European Union's Horizon 2020 research and innovation program (Grant Agreement No. 742708 and 648901). The authors at ICMAB acknowledge financial support from the Spanish Ministry of Science and Innovation through the Severo Ochoa” Program for Centers of Excellence in R&amp;D (No. CEX2019-000917-S), and project PGC2018-095411-B-I00. E. R. is grateful to the Fonds de Recherche du Quebec-Nature et technologies (FRQNT) for a postdoctoral fellowship and acknowledges financial support from the European Cooperation in Science and Technology. M. A. thanks the Engineering and Physical Sciences Research Council (EPSRC) for support <span class="italic">via</span> doctoral studentships. F. E. thanks the Engineering and Physical Sciences Research Council (EPSRC) for support <span class="italic">via</span> the Post-Doctoral Prize Fellowship. X. R.-M. acknowledges Prof. Olle Inganäs and the Knut and Allice Wallenberg Foundation for funding of his current postdoctoral position. H.-L. Yip thanks the support from Guangdong Major Project of Basic and Applied Basic Research (2019B030302007). The TOC figure and <a title="Select to navigate to figure" href="#imgfig5">Fig. 5a</a> in the manuscript include freely available resources from http://Flaticon.com. J. Y. thank Xiaodan Ge for her support.</span>
    
    <span id="sect1309"><h2 id="sect1302"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">J. Nelson, <span class="italic">Mater. Today</span>, 2011, <span class="bold">14</span>, 462–470 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S1369-7021(11)70210-3" title="DOI Link to resource 10.1016/S1369-7021(11)70210-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXht1CqurfJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit2">G. Li, R. Zhu and Y. Yang, <span class="italic">Nat. Photonics</span>, 2012, <span class="bold">6</span>, 153–161 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nphoton.2012.11" title="DOI Link to resource 10.1038/nphoton.2012.11">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XivV2nur0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit3">A. J. Heeger, <span class="italic">Adv. Mater.</span>, 2014, <span class="bold">26</span>, 10–28 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201304373" title="DOI Link to resource 10.1002/adma.201304373">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhvVykur%252FN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24311015%5Buid%5D" title="PubMed Link to resource 24311015">PubMed</a>.</span></li>
      <li><span id="cit4">M. Mainville and M. Leclerc, <span class="italic">ACS Energy Lett.</span>, 2020, <span class="bold">5</span>, 1186–1197 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsenergylett.0c00177" title="DOI Link to resource 10.1021/acsenergylett.0c00177">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXksFGitbg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit5">H. K.-H. Lee, J. Wu, J. Barbé, S. M. Jain, S. Wood, E. M. Speller, Z. Li, F. A. Castro, J. R. Durrant and W. C. Tsoi, <span class="italic">J. Mater. Chem. A</span>, 2018, <span class="bold">6</span>, 5618–5626 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7TA10875C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7TA10875C">RSC</a>.</span></li>
      <li><span id="cit6">Y. Cui, Y. Wang, J. Bergqvist, H. Yao, Y. Xu, B. Gao, C. Yang, S. Zhang, O. Inganäs, F. Gao and J. Hou, <span class="italic">Nat. Energy</span>, 2019, <span class="bold">4</span>, 768–775 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41560-019-0448-5" title="DOI Link to resource 10.1038/s41560-019-0448-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhs1SrtrzM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit7">Y. Li, J. D. Lin, X. Che, Y. Qu, F. Liu, L. S. Liao and S. R. Forrest, <span class="italic">J. Am. Chem. Soc.</span>, 2017, <span class="bold">139</span>, 17114–17119 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.7b11278" title="DOI Link to resource 10.1021/jacs.7b11278">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSrtb%252FJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29144745%5Buid%5D" title="PubMed Link to resource 29144745">PubMed</a>.</span></li>
      <li><span id="cit8">S. Difley and T. Van Voorhis, <span class="italic">J. Chem. Theory Comput.</span>, 2011, <span class="bold">7</span>, 594–601 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ct100508y" title="DOI Link to resource 10.1021/ct100508y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXisVWqtro%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit9">C. J.-M. Emmott, J. A. Röhr, M. Campoy-Quiles, T. Kirchartz, A. Urbina, N. J. Ekins-Daukes and J. Nelson, <span class="italic">Energy Environ. Sci.</span>, 2015, <span class="bold">8</span>, 1317–1328 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4EE03132F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4EE03132F">RSC</a>.</span></li>
      <li><span id="cit10">C. J.-M. Emmott, D. Moia, P. Sandwell, N. Ekins-Daukes, M. Hösel, L. Lukoschek, C. Amarasinghe, F. C. Krebs and J. Nelson, <span class="italic">Sol. Energy Mater. Sol. Cells</span>, 2016, <span class="bold">149</span>, 284–293 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.solmat.2016.01.036" title="DOI Link to resource 10.1016/j.solmat.2016.01.036">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XitFejs7Y%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit11">L. Zhu, M. Zhang, J. Xu, C. Li, J. Yan, G. Zhou, W. Zhong, T. Hao, J. Song, X. Xue, Z. Zhou, R. Zeng, H. Zhu, C.-C. Chen, R. C.-I. MacKenzie, Y. Zou, J. Nelson, Y. Zhang, Y. Sun and F. Liu, <span class="italic">Nat. Mater.</span>, 2022, 1–8, <small> DOI:<a class="DOILink" href="https://doi.org/10.1038/s41563-022-01244-y" TARGET="_BLANK" title="DOI Link to 10.1038/s41563-022-01244-y">10.1038/s41563-022-01244-y</a></small>.</span></li>
      <li><span id="cit12">J. Zhao, Y. Li, G. Yang, K. Jiang, H. Lin, H. Ade, W. Ma and H. Yan, <span class="italic">Nat. Energy</span>, 2016, <span class="bold">1</span>, 15027 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nenergy.2015.27" title="DOI Link to resource 10.1038/nenergy.2015.27">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtVektbs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit13">P. Cheng, G. Li, X. Zhan and Y. Yang, <span class="italic">Nat. Photonics</span>, 2018, <span class="bold">12</span>, 131–142 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41566-018-0104-9" title="DOI Link to resource 10.1038/s41566-018-0104-9">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjsFWmu7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit14">Y. Wang, J. Lee, X. Hou, C. Labanti, J. Yan, E. Mazzolini, A. Parhar, J. Nelson, J. Kim and Z. Li, <span class="italic">Adv. Energy Mater.</span>, 2021, <span class="bold">11</span>, 2003002 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.202003002" title="DOI Link to resource 10.1002/aenm.202003002">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXis1OksbbI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit15">J. Hou, O. Inganäs, R. H. Friend and F. Gao, <span class="italic">Nat. Mater.</span>, 2018, <span class="bold">17</span>, 119–128 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat5063" title="DOI Link to resource 10.1038/nmat5063">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGqsbk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29358765%5Buid%5D" title="PubMed Link to resource 29358765">PubMed</a>.</span></li>
      <li><span id="cit16">J. Yuan, Y. Zhang, L. Zhou, G. Zhang, H.-L. Yip, T.-K. Lau, X. Lu, C. Zhu, H. Peng, P. A. Johnson, M. Leclerc, Y. Cao, J. Ulanski, Y. Li and Y. Zou, <span class="italic">Joule</span>, 2019, <span class="bold">3</span>, 1140–1151 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2019.01.004" title="DOI Link to resource 10.1016/j.joule.2019.01.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXnvVKnsL8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit17">M. Kaltenbrunner, M. S. White, E. D. Głowacki, T. Sekitani, T. Someya, N. S. Sariciftci and S. Bauer, <span class="italic">Nat. Commun.</span>, 2012, <span class="bold">3</span>, 1–7 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Nat.%20Commun.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202012%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit18">W. Yang, W. Wang, Y. Wang, R. Sun, J. Guo, H. Li, M. Shi, J. Guo, Y. Wu, T. Wang, G. Lu, C. J. Brabec, Y. Li and J. Min, <span class="italic">Joule</span>, 2021, <span class="bold">5</span>, 1209–1230 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2021.03.014" title="DOI Link to resource 10.1016/j.joule.2021.03.014">CrossRef</a>.</span></li>
      <li><span id="cit19">
          J. Nelson, <span class="italic">The Physics of Solar Cells</span>, Imperial College Press,  2003 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=The%20Physics%20of%20Solar%20Cells%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit20">U. Rau, <span class="italic">Phys. Rev. B</span>, 2007, <span class="bold">76</span>, 085303 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.76.085303" title="DOI Link to resource 10.1103/PhysRevB.76.085303">CrossRef</a>.</span></li>
      <li><span id="cit21">M. Azzouzi, J. Yan, T. Kirchartz, K. Liu, J. Wang, H. Wu and J. Nelson, <span class="italic">Phys. Rev. X</span>, 2018, <span class="bold">8</span>, 031055 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXltFSmtb0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit22">F. D. Eisner, M. Azzouzi, Z. Fei, X. Hou, T. D. Anthopoulos, T. J.-S. J.-S. Dennis, M. Heeney and J. Nelson, <span class="italic">J. Am. Chem. Soc.</span>, 2019, <span class="bold">141</span>, 6362–6374 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.9b01465" title="DOI Link to resource 10.1021/jacs.9b01465">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXltFCjtrc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30882218%5Buid%5D" title="PubMed Link to resource 30882218">PubMed</a>.</span></li>
      <li><span id="cit23">J. Yan, E. Rezasoltani, M. Azzouzi, F. Eisner and J. Nelson, <span class="italic">Nat. Commun.</span>, 2021, <span class="bold">12</span>, 3642 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-021-23975-3" title="DOI Link to resource 10.1038/s41467-021-23975-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhsFKgurfO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit24">A. Classen, C. L. Chochos, L. Lüer, V. G. Gregoriou, J. Wortmann, A. Osvet, K. Forberich, I. McCulloch, T. Heumüller and C. J. Brabec, <span class="italic">Nat. Energy</span>, 2020, <span class="bold">5</span>, 711–719 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41560-020-00684-7" title="DOI Link to resource 10.1038/s41560-020-00684-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvF2ksrjM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit25">X.-K. Chen, D. Qian, Y. Wang, T. Kirchartz, W. Tress, H. Yao, J. Yuan, M. Hülsbeck, M. Zhang, Y. Zou, Y. Sun, Y. Li, J. Hou, O. Inganäs, V. Coropceanu, J.-L. Bredas and F. Gao, <span class="italic">Nat. Energy</span>, 2021, <span class="bold">6</span>, 799–806 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41560-021-00843-4" title="DOI Link to resource 10.1038/s41560-021-00843-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvFOqt7vJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit26">J. Benduhn, K. Tvingstedt, F. Piersimoni, S. Ullbrich, Y. Fan, M. Tropiano, K. A.-A. McGarry, O. Zeika, M. K.-K. Riede, C. J.-J. Douglas, S. Barlow, S. R.-R. Marder, D. Neher, D. Spoltore and K. Vandewal, <span class="italic">Nat. Energy</span>, 2017, <span class="bold">2</span>, 17053 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nenergy.2017.53" title="DOI Link to resource 10.1038/nenergy.2017.53">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjt12jsbo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit27">X.-K. Chen, V. Coropceanu, J.-L. Bredas and J.-L. Bredas, <span class="italic">Nat. Commun.</span>, 2018, <span class="bold">9</span>, 5295 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-018-07707-8" title="DOI Link to resource 10.1038/s41467-018-07707-8">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisFejs7zJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30546009%5Buid%5D" title="PubMed Link to resource 30546009">PubMed</a>.</span></li>
      <li><span id="cit28">D. Qian, Z. Zheng, H. Yao, W. Tress, T. R. Hopper, S. S. Chen, S. Li, J. Liu, S. S. Chen, J. Zhang, X.-K. K. Liu, B. Gao, L. Ouyang, Y. Jin, G. Pozina, I. A. Buyanova, W. M. Chen, O. Inganäs, V. Coropceanu, J.-L. L. Bredas, H. Yan, J. Hou, F. Zhang, A. A. Bakulin and F. Gao, <span class="italic">Nat. Mater.</span>, 2018, <span class="bold">17</span>, 703–709 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41563-018-0128-z" title="DOI Link to resource 10.1038/s41563-018-0128-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGiur%252FK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30013057%5Buid%5D" title="PubMed Link to resource 30013057">PubMed</a>.</span></li>
      <li><span id="cit29">M. S. Vezie, S. Few, I. Meager, G. Pieridou, B. Dörling, R. S. Ashraf, A. R. Goñi, H. Bronstein, I. McCulloch, S. C. Hayes, M. Campoy-Quiles and J. Nelson, <span class="italic">Nat. Mater.</span>, 2016, <span class="bold">15</span>, 746–753 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat4645" title="DOI Link to resource 10.1038/nmat4645">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XotV2ntrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit30">B. Kang, C. Seok and J. Lee, <span class="italic">J. Chem. Inf. Model.</span>, 2020, <span class="bold">60</span>, 5984–5994 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.0c00698" title="DOI Link to resource 10.1021/acs.jcim.0c00698">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitFagtb%252FJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33090804%5Buid%5D" title="PubMed Link to resource 33090804">PubMed</a>.</span></li>
      <li><span id="cit31">S. Few, J. M. Frost, J. Kirkpatrick and J. Nelson, <span class="italic">J. Phys. Chem. C</span>, 2014, <span class="bold">118</span>, 8253–8261 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp412449n" title="DOI Link to resource 10.1021/jp412449n">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXksVWis7w%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit32">Y. Yi, V. Coropceanu and J.-L. Brédas, <span class="italic">J. Mater. Chem.</span>, 2011, <span class="bold">21</span>, 1479 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C0JM02467H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C0JM02467H">RSC</a>.</span></li>
      <li><span id="cit33">T. Liu and A. Troisi, <span class="italic">J. Phys. Chem. C</span>, 2011, <span class="bold">115</span>, 2406–2415 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp109130y" title="DOI Link to resource 10.1021/jp109130y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXjvFCmtQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit34">J. C. Slater, <span class="italic">Phys. Rev.</span>, 1951, <span class="bold">81</span>, 385 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRev.81.385" title="DOI Link to resource 10.1103/PhysRev.81.385">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaG3MXis1Wquw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit35">
          B. L. Hammond, W. A. Lester and P. J. Reynolds, <span class="italic">Monte Carlo Methods in Ab Initio Quantum Chemistry</span>, World Scientific,  1994, vol. 1 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Monte%20Carlo%20Methods%20in%20Ab%20Initio%20Quantum%20Chemistry%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201994%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit36">R. A. Friesner, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2005, <span class="bold">102</span>, 6648–6653 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0408036102" title="DOI Link to resource 10.1073/pnas.0408036102">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgt7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15870212%5Buid%5D" title="PubMed Link to resource 15870212">PubMed</a>.</span></li>
      <li><span id="cit37">Y. Shao, L. F. Molnar, Y. Jung, J. Kussmann, C. Ochsenfeld, S. T. Brown, A. T. B. Gilbert, L. V. Slipchenko, S. V. Levchenko, D. P. O’Neill, R. A. DiStasio Jr, R. C. Lochan, T. Wang, G. J. O. Beran, N. A. Besley, J. M. Herbert, C. Yeh Lin, T. Van Voorhis, S. Hung Chien, A. Sodt, R. P. Steele, V. A. Rassolov, P. E. Maslen, P. P. Korambath, R. D. Adamson, B. Austin, J. Baker, E. F. C. Byrd, H. Dachsel, R. J. Doerksen, A. Dreuw, B. D. Dunietz, A. D. Dutoi, T. R. Furlani, S. R. Gwaltney, A. Heyden, S. Hirata, C.-P. Hsu, G. Kedziora, R. Z. Khalliulin, P. Klunzinger, A. M. Lee, M. S. Lee, W. Liang, I. Lotan, N. Nair, B. Peters, E. I. Proynov, P. A. Pieniazek, Y. Min Rhee, J. Ritchie, E. Rosta, C. David Sherrill, A. C. Simmonett, J. E. Subotnik, H. Lee Woodcock III, W. Zhang, A. T. Bell, A. K. Chakraborty, D. M. Chipman, F. J. Keil, A. Warshel, W. J. Hehre, H. F. Schaefer III, J. Kong, A. I. Krylov, P. M. W. Gill and M. Head-Gordon, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2006, <span class="bold">8</span>, 3172–3191 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B517914A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B517914A">RSC</a>.</span></li>
      <li><span id="cit38">P. Raccuglia, K. C. Elbert, P. D.-F. Adler, C. Falk, M. B. Wenny, A. Mollo, M. Zeller, S. A. Friedler, J. Schrier and A. J. Norquist, <span class="italic">Nature</span>, 2016, <span class="bold">533</span>, 73–76 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature17439" title="DOI Link to resource 10.1038/nature17439">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XntlKhtL0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27147027%5Buid%5D" title="PubMed Link to resource 27147027">PubMed</a>.</span></li>
      <li><span id="cit39">K. T. Butler, D. W. Davies, H. Cartwright, O. Isayev and A. Walsh, <span class="italic">Nature</span>, 2018, <span class="bold">559</span>, 547–555 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41586-018-0337-2" title="DOI Link to resource 10.1038/s41586-018-0337-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtl2jt7vL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit40">J. Westermayr and P. Marquetand, <span class="italic">Chem. Rev.</span>, 2021, <span class="bold">121</span>, 9873–9926 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.0c00749" title="DOI Link to resource 10.1021/acs.chemrev.0c00749">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitlKrs7zO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33211478%5Buid%5D" title="PubMed Link to resource 33211478">PubMed</a>.</span></li>
      <li><span id="cit41">F. Häse, L. M. Roch, P. Friederich and A. Aspuru-Guzik, <span class="italic">Nat. Commun.</span>, 2020, <span class="bold">11</span>, 1–11 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-019-13993-7" title="DOI Link to resource 10.1038/s41467-019-13993-7">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31911652%5Buid%5D" title="PubMed Link to resource 31911652">PubMed</a>.</span></li>
      <li><span id="cit42">S. A. Lopez, B. Sanchez-Lengeling, J. de Goes Soares and A. Aspuru-Guzik, <span class="italic">Joule</span>, 2017, <span class="bold">1</span>, 857–870 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2017.10.006" title="DOI Link to resource 10.1016/j.joule.2017.10.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpvFGhu7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit43">H. Sahu, W. Rao, A. Troisi and H. Ma, <span class="italic">Adv. Energy Mater.</span>, 2018, <span class="bold">8</span>, 1–9 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Energy%20Mater.%5BJour%5D%20AND%208%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit44">W. K. Tatum, D. Torrejon, A. B. Resing, J. W. Onorato and C. K. Luscombe, <span class="italic">Comput. Mater. Sci.</span>, 2021, <span class="bold">197</span>, 110599 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2021.110599" title="DOI Link to resource 10.1016/j.commatsci.2021.110599">CrossRef</a>.</span></li>
      <li><span id="cit45">N. Majeed, M. Saladina, M. Krompiec, S. Greedy, C. Deibel and R. C.-I. MacKenzie, <span class="italic">Adv. Funct. Mater.</span>, 2019, 1907259 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Funct.%20Mater.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201907259%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit46">X. Rodríguez-Martínez, E. Pascual-San-José, Z. Fei, M. Heeney, R. Guimerà and M. Campoy-Quiles, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 986–994 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE02958K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE02958K">RSC</a>.</span></li>
      <li><span id="cit47">X. Rodríguez-Martínez, E. Pascual-San-José and M. Campoy-Quiles, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 3301–3322 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1EE00559F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1EE00559F">RSC</a>.</span></li>
      <li><span id="cit48">K. Kranthiraja and A. Saeki, <span class="italic">Adv. Funct. Mater.</span>, 2021, <span class="bold">31</span>, 1–11 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Funct.%20Mater.%5BJour%5D%20AND%2031%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202021%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit49">E. O. Pyzer-Knapp, C. Suh, R. Gómez-Bombarelli, J. Aguilera-Iparraguirre and A. Aspuru-Guzik, <span class="italic">Annu. Rev. Mater. Res.</span>, 2015, <span class="bold">45</span>, 195–216 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev-matsci-070214-020823" title="DOI Link to resource 10.1146/annurev-matsci-070214-020823">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWqur3L" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit50">J. Hachmann, R. Olivares-amaya, S. Atahan-evrenk, C. Amador-bedolla, R. S. Sanchez-carrera, A. Gold-parker, L. Vogt, A. M. Brockway and A. Aspuru-Guzik, <span class="italic">J. Phys. Chem. Lett.</span>, 2011, <span class="bold">2</span>, 2241–2251 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jz200866s" title="DOI Link to resource 10.1021/jz200866s">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKht7rK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit51">A. Mahmood and J.-L. Wang, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 90–105 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE02838J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE02838J">RSC</a>.</span></li>
      <li><span id="cit52">P. Malhotra, S. Biswas, F.-C. Chen and G. D. Sharma, <span class="italic">Sol. Energy</span>, 2021, <span class="bold">228</span>, 175–186 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.solener.2021.09.056" title="DOI Link to resource 10.1016/j.solener.2021.09.056">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXitFamsL%252FP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit53">A. Kuzmich, D. Padula, H. Ma and A. Troisi, <span class="italic">Energy Environ. Sci.</span>, 2017, <span class="bold">10</span>, 395–401 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6EE03654F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6EE03654F">RSC</a>.</span></li>
      <li><span id="cit54">I. Y. Kanal, S. G. Owens, J. S. Bechtel and G. R. Hutchison, <span class="italic">J. Phys. Chem. Lett.</span>, 2013, <span class="bold">4</span>, 1613–1623 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jz400215j" title="DOI Link to resource 10.1021/jz400215j">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXmsFGktro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26282968%5Buid%5D" title="PubMed Link to resource 26282968">PubMed</a>.</span></li>
      <li><span id="cit55">L. Wilbraham, E. Berardo, L. Turcani, K. E. Jelfs and M. A. Zwijnenburg, <span class="italic">J. Chem. Inf. Model.</span>, 2018, <span class="bold">58</span>, 2450–2459 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.8b00256" title="DOI Link to resource 10.1021/acs.jcim.8b00256">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2htLzE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29940733%5Buid%5D" title="PubMed Link to resource 29940733">PubMed</a>.</span></li>
      <li><span id="cit56">E. O. Pyzer-Knapp, K. Li and A. Aspuru-Guzik, <span class="italic">Adv. Funct. Mater.</span>, 2015, <span class="bold">25</span>, 6495–6502 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201501919" title="DOI Link to resource 10.1002/adfm.201501919">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtL3I" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit57">J. Hachmann, R. Olivares-Amaya, A. Jinich, A. L. Appleton, M. A. Blood-Forsythe, L. R. Seress, C. Román-Salgado, K. Trepte, S. Atahan-Evrenk, S. Er, S. Shrestha, R. Mondal, A. Sokolov, Z. Bao and A. Aspuru-Guzik, <span class="italic">Energy Environ. Sci.</span>, 2014, <span class="bold">7</span>, 698–704 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3EE42756K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3EE42756K">RSC</a>.</span></li>
      <li><span id="cit58">N. Bérubé, V. Gosselin, J. Gaudreau and M. Côté, <span class="italic">J. Phys. Chem. C</span>, 2013, <span class="bold">117</span>, 7964–7972 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp309800f" title="DOI Link to resource 10.1021/jp309800f">CrossRef</a>.</span></li>
      <li><span id="cit59">S. Nagasawa, E. Al-Naamani and A. Saeki, <span class="italic">J. Phys. Chem. Lett.</span>, 2018, <span class="bold">9</span>, 2639–2646 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.8b00635" title="DOI Link to resource 10.1021/acs.jpclett.8b00635">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXovVyms74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29733216%5Buid%5D" title="PubMed Link to resource 29733216">PubMed</a>.</span></li>
      <li><span id="cit60">Y. Huang, J. Zhang, E. S. Jiang, Y. Oya, A. Saeki, G. Kikugawa, T. Okabe, T. Okabe, F. S. Ohuchi and F. S. Ohuchi, <span class="italic">J. Phys. Chem. C</span>, 2020, <span class="bold">124</span>, 12871–12882 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.0c00517" title="DOI Link to resource 10.1021/acs.jpcc.0c00517">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXotFejurw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit61">W. Sun, M. Li, Y. Li, Z. Wu, Y. Sun, S. Lu, Z. Xiao, B. Zhao and K. Sun, <span class="italic">Adv. Theory Simul.</span>, 2019, <span class="bold">2</span>, 1–9 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Theory%20Simul.%5BJour%5D%20AND%202%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit62">R. Olivares-Amaya, C. Amador-Bedolla, J. Hachmann, S. Atahan-Evrenk, R. S. Sánchez-Carrera, L. Vogt and A. Aspuru-Guzik, <span class="italic">Energy Environ. Sci.</span>, 2011, <span class="bold">4</span>, 4849–4861 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1EE02056K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1EE02056K">RSC</a>.</span></li>
      <li><span id="cit63">H. Sahu, F. Yang, X. Ye, J. Ma, W. Fang and H. Ma, <span class="italic">J. Mater. Chem. A</span>, 2019, <span class="bold">7</span>, 17480–17488 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9TA04097H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9TA04097H">RSC</a>.</span></li>
      <li><span id="cit64">D. Padula, J. D.-D. Simpson and A. Troisi, <span class="italic">Mater. Horiz.</span>, 2019, <span class="bold">6</span>, 343–349 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8MH01135D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8MH01135D">RSC</a>.</span></li>
      <li><span id="cit65">L. Simine, T. C. Allen and P. J. Rossky, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2020, <span class="bold">117</span>, 13945–13948 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.1918696117" title="DOI Link to resource 10.1073/pnas.1918696117">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCgs73K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32513725%5Buid%5D" title="PubMed Link to resource 32513725">PubMed</a>.</span></li>
      <li><span id="cit66">J. F. Joung, M. Han, J. Hwang, M. Jeong, D. H. Choi and S. Park, <span class="italic">JACS Au</span>, 2021, <span class="bold">1</span>, 427–438 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacsau.1c00035" title="DOI Link to resource 10.1021/jacsau.1c00035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXmsFWhtr4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34467305%5Buid%5D" title="PubMed Link to resource 34467305">PubMed</a>.</span></li>
      <li><span id="cit67">S. Ye, W. Hu, X. Li, J. Zhang, K. Zhong, G. Zhang, Y. Luo, S. Mukamel and J. Jiang, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2019, <span class="bold">116</span>, 201821044 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.%5BJour%5D%20AND%20116%5Bvolume%5D%20AND%20201821044%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit68">C. Nantasenamat, C. Isarankura-Na-Ayudhya, N. Tansila, T. Naenna and V. Prachayasittikul, <span class="italic">J. Comput. Chem.</span>, 2007, <span class="bold">28</span>, 1275–1289 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.20656" title="DOI Link to resource 10.1002/jcc.20656">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXktlKlurg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17299836%5Buid%5D" title="PubMed Link to resource 17299836">PubMed</a>.</span></li>
      <li><span id="cit69">E. J. Beard, G. Sivaraman, Á. Vázquez-Mayagoitia, V. Vishwanath and J. M. Cole, <span class="italic">Sci. Data</span>, 2019, <span class="bold">6</span>, 307 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41597-019-0306-0" title="DOI Link to resource 10.1038/s41597-019-0306-0">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31804487%5Buid%5D" title="PubMed Link to resource 31804487">PubMed</a>.</span></li>
      <li><span id="cit70">S. Varghese and S. Das, <span class="italic">J. Phys. Chem. Lett.</span>, 2011, <span class="bold">2</span>, 863–873 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jz200099p" title="DOI Link to resource 10.1021/jz200099p">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXjvFaks70%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26295620%5Buid%5D" title="PubMed Link to resource 26295620">PubMed</a>.</span></li>
      <li><span id="cit71">C. L. Donley, J. Zaumseil, J. W. Andreasen, M. M. Nielsen, H. Sirringhaus, R. H. Friend and J. S. Kim, <span class="italic">J. Am. Chem. Soc.</span>, 2005, <span class="bold">127</span>, 12890–12899 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja051891j" title="DOI Link to resource 10.1021/ja051891j">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXpt1WhsrY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16159283%5Buid%5D" title="PubMed Link to resource 16159283">PubMed</a>.</span></li>
      <li><span id="cit72">P. J. Brown, D. S. Thomas, A. Köhler, J. S. Wilson, J.-S. S. Kim, C. M. Ramsdale, H. Sirringhaus and R. H. Friend, <span class="italic">Phys. Rev. B</span>, 2003, <span class="bold">67</span>, 064203 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.67.064203" title="DOI Link to resource 10.1103/PhysRevB.67.064203">CrossRef</a>.</span></li>
      <li><span id="cit73">C. W. Yap, <span class="italic">J. Comput. Chem.</span>, 2011, <span class="bold">32</span>, 1466–1474 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.21707" title="DOI Link to resource 10.1002/jcc.21707">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXjsF2isLc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21425294%5Buid%5D" title="PubMed Link to resource 21425294">PubMed</a>.</span></li>
      <li><span id="cit74">D.-S. Cao, Q.-S. Xu, Q.-N. Hu and Y.-Z. Liang, <span class="italic">Bioinformatics</span>, 2013, <span class="bold">29</span>, 1092–1094 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/bioinformatics/btt105" title="DOI Link to resource 10.1093/bioinformatics/btt105">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKitr0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23493324%5Buid%5D" title="PubMed Link to resource 23493324">PubMed</a>.</span></li>
      <li><span id="cit75">H. Moriwaki, Y.-S. Tian, N. Kawashita and T. Takagi, <span class="italic">J. Cheminform.</span>, 2018, <span class="bold">10</span>, 4 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13321-018-0258-y" title="DOI Link to resource 10.1186/s13321-018-0258-y">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29411163%5Buid%5D" title="PubMed Link to resource 29411163">PubMed</a>.</span></li>
      <li><span id="cit76">RDKit: Open-Source Cheminformatics Software.</span></li>
      <li><span id="cit77">Z. Luo, R. Ma, Y. Xiao, T. Liu, H. Sun, M. Su, Q. Guo, G. Li, W. Gao, Y. Chen, Y. Zou, X. Guo, M. Zhang, X. Lu, H. Yan and C. Yang, <span class="italic">Small</span>, 2020, <span class="bold">16</span>, 2001942 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/smll.202001942" title="DOI Link to resource 10.1002/smll.202001942">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXht1Ons7jF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32602255%5Buid%5D" title="PubMed Link to resource 32602255">PubMed</a>.</span></li>
      <li><span id="cit78">M. Y. Mehboob, M. Adnan, R. Hussain and Z. Irshad, <span class="italic">Synth. Met.</span>, 2021, <span class="bold">277</span>, 116800 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.synthmet.2021.116800" title="DOI Link to resource 10.1016/j.synthmet.2021.116800">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhtVWqt7%252FE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit79">R. Todeschini, M. Lasagni and E. Marengo, <span class="italic">J. Chemom.</span>, 1994, <span class="bold">8</span>, 263–272 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cem.1180080405" title="DOI Link to resource 10.1002/cem.1180080405">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2cXmtFentLo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit80">
          R. Todeschini and P. Gramatica, <span class="italic">3D QSAR in Drug Design</span>, Kluwer Academic Publishers, Dordrecht,  1998, pp. 355–380 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=3D%20QSAR%20in%20Drug%20Design%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201998%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit81">R. Todeschini and P. Gramatica, <span class="italic">Quant. Struct. Relationships</span>, 1997, <span class="bold">16</span>, 113–119 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/qsar.19970160203" title="DOI Link to resource 10.1002/qsar.19970160203">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2sXjsFCns74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit82">
          R. Todeschini and V. Consonni, <span class="italic">Molecular Descriptors for Chemoinformatics</span>, Wiley, 2nd edn,  2009, vol. 41 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Molecular%20Descriptors%20for%20Chemoinformatics%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit83">
          R. Todeschini and V. Consonni, <span class="italic">Handbook of Molecular Descriptors</span>, Wiley,  2000 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Handbook%20of%20Molecular%20Descriptors%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202000%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit84">C. Bannwarth, E. Caldeweyher, S. Ehlert, A. Hansen, P. Pracht, J. Seibert, S. Spicher and S. Grimme, <span class="italic">Wiley Interdiscip. Rev.: Comput. Mol. Sci.</span>, 2021, <span class="bold">11</span>, e1493 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXnt1aru74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit85">N. Artrith, K. T. Butler, F.-X. Coudert, S. Han, O. Isayev, A. Jain and A. Walsh, <span class="italic">Nat. Chem.</span>, 2021, <span class="bold">13</span>, 505–508 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41557-021-00716-z" title="DOI Link to resource 10.1038/s41557-021-00716-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXht1WqtrbL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34059804%5Buid%5D" title="PubMed Link to resource 34059804">PubMed</a>.</span></li>
      <li><span id="cit86">M. Lee, <span class="italic">Adv. Energy Mater.</span>, 2019, 1900891 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201900891" title="DOI Link to resource 10.1002/aenm.201900891">CrossRef</a>.</span></li>
      <li><span id="cit87">F. Pedregosa, R. Weiss, M. Brucher, G. Varoquaux, A. Gramfort, V. Michel, B. Thirion, O. Grisel, M. Blondel, P. Prettenhofer, R. Weiss, V. Dubourg, J. Vanderplas, A. Passos, D. Cournapeau, M. Brucher, M. Perrot and É. Duchesnay, <span class="italic">J. Mach. Learn. Res.</span>, 2011, <span class="bold">12</span>, 2825–2830 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Mach.%20Learn.%20Res.%5BJour%5D%20AND%2012%5Bvolume%5D%20AND%202825%5Bpage%5D%20and%202011%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit88">D. Rogers and M. Hahn, <span class="italic">J. Chem. Inf. Model.</span>, 2010, <span class="bold">50</span>, 742–754 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci100050t" title="DOI Link to resource 10.1021/ci100050t">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20426451%5Buid%5D" title="PubMed Link to resource 20426451">PubMed</a>.</span></li>
      <li><span id="cit89">L. H. Hall and L. B. Kier, <span class="italic">J. Chem. Inf. Comput. Sci.</span>, 1995, <span class="bold">35</span>, 1039–1045 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci00028a014" title="DOI Link to resource 10.1021/ci00028a014">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXovF2hsbc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit90">D. C. Elton, Z. Boukouvalas, M. S. Butrico, M. D. Fuge and P. W. Chung, <span class="italic">Sci. Rep.</span>, 2018, <span class="bold">8</span>, 9059 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-018-27344-x" title="DOI Link to resource 10.1038/s41598-018-27344-x">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29899464%5Buid%5D" title="PubMed Link to resource 29899464">PubMed</a>.</span></li>
      <li><span id="cit91">C. Bannwarth, S. Ehlert and S. Grimme, <span class="italic">J. Chem. Theory Comput.</span>, 2019, <span class="bold">15</span>, 1652–1671 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jctc.8b01176" title="DOI Link to resource 10.1021/acs.jctc.8b01176">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXis1entL0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30741547%5Buid%5D" title="PubMed Link to resource 30741547">PubMed</a>.</span></li>
      <li><span id="cit92">S. Grimme, C. Bannwarth and P. Shushkov, <span class="italic">J. Chem. Theory Comput.</span>, 2017, <span class="bold">13</span>, 1989–2009 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jctc.7b00118" title="DOI Link to resource 10.1021/acs.jctc.7b00118">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXmt12ltL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28418654%5Buid%5D" title="PubMed Link to resource 28418654">PubMed</a>.</span></li>
      <li><span id="cit93">F. Deledalle, T. Kirchartz, M. S. Vezie, M. Campoy-Quiles, P. S. Tuladhar, J. Nelson and J. R. Durrant, <span class="italic">Phys. Rev. X</span>, 2015, <span class="bold">5</span>, 1–13 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXmtFaitbc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit94">F. C. Spano, <span class="italic">Acc. Chem. Res.</span>, 2010, <span class="bold">43</span>, 429–439 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ar900233v" title="DOI Link to resource 10.1021/ar900233v">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXhsFKgurzL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20014774%5Buid%5D" title="PubMed Link to resource 20014774">PubMed</a>.</span></li>
      <li><span id="cit95">G. Forti, A. Nitti, P. Osw, G. Bianchi, R. Po and D. Pasini, <span class="italic">Int. J. Mol. Sci.</span>, 2020, <span class="bold">21</span>, 8085 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/ijms21218085" title="DOI Link to resource 10.3390/ijms21218085">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXisFKltbnE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33138257%5Buid%5D" title="PubMed Link to resource 33138257">PubMed</a>.</span></li>
      <li><span id="cit96">L. A.-A. Pettersson, L. S. Roman and O. Inganäs, <span class="italic">J. Appl. Phys.</span>, 1999, <span class="bold">86</span>, 487–496 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.370757" title="DOI Link to resource 10.1063/1.370757">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1MXjvV2rsrk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
    </ol>
    
    
    
    
    
  <hr/><table><tr><td><h3>Footnotes</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">Electronic supplementary information (ESI) available. See DOI: <a target="_blank" href="https://doi.org/10.1039/d2ee00887d">https://doi.org/10.1039/d2ee00887d</a></span></span></td></tr><tr><td><span class="sup_ref">‡ <span id="fn2">J. Y. and X. R.-M. contributed equally to this work.</span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2022</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>