<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Copper nanoclusters: designed synthesis, structural diversity, and multiplatform applications  - Nanoscale (RSC Publishing) DOI:10.1039/D0NR08489A</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2021/nr/d0nr08489a"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":105,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Ani Baghdasaryan"/><meta name="DC.Creator" content="Thomas Bürgi"/><meta name="DC.title" content="Copper nanoclusters: designed synthesis, structural diversity, and multiplatform applications "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2021/04/09"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D0NR08489A"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Copper nanoclusters: designed synthesis, structural diversity, and multiplatform applications "/><meta name="citation_author" content="Ani Baghdasaryan"/><meta name="citation_author" content="Thomas Bürgi"/><meta name="citation_online_date" content="2021/02/26"/><meta name="citation_date" content="2021"/><meta name="citation_journal_title" content="Nanoscale"/><meta name="citation_volume" content="13"/><meta name="citation_issue" content="13"/><meta name="citation_firstpage" content="6283"/><meta name="citation_lastpage" content="6340"/><meta name="citation_doi" content="10.1039/D0NR08489A"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2021/nr/d0nr08489a"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2021/nr/d0nr08489a"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2021/nr/d0nr08489a"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2021/nr/d0nr08489a">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d1nr90072b)" href="/en/content/articlehtml/2021/nr/d1nr90072b" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d0nr09167g)" href="/en/content/articlehtml/2021/nr/d0nr09167g" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div><br/><span class="oa"><img src="/content/newimages/open_access_blue.png" alt=""/> Open Access Article<br/><img src="/content/newimages/CCBY-NC.svg" alt=""/>
This Open Access Article is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/3.0/">Creative Commons Attribution-Non Commercial 3.0 Unported Licence</a></span></div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D0NR08489A">10.1039/D0NR08489A</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2040-3372/2009">Nanoscale</a></span>, 2021, <strong>13</strong>, 6283-6340</div><h1 id="sect336"><span class="title_heading">Copper nanoclusters: designed synthesis, structural diversity, and multiplatform applications</span></h1><p class="header_text">
      <span class="bold">
        
          
            Ani 
            Baghdasaryan
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Ani Baghdasaryan (orcid.org/0000-0003-1324-5683) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-1324-5683"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><span class="bold"> and </span>
      <span class="bold">
        
          
            Thomas 
            Bürgi
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Thomas Bürgi (orcid.org/0000-0003-0906-082X) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-0906-082X"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*
      <br/><span class="italic">Department of Physical Chemistry, University of Geneva, 30 Quai Ernest-Ansermet, 1211 Geneva 4, Switzerland. E-mail: <a href="mailto:thomas.buergi@unige.ch">thomas.buergi@unige.ch</a></span>
    </p><div id="art-admin"><span class="italic bold">Received 
      30th November 2020
    </span><span class="bold italic">, Accepted 11th February 2021</span></div><p class="bold italic">First published on 26th February 2021</p><hr/><div class="abstract"><h2>Abstract</h2><p>Atomically precise metal nanoclusters (MNCs) have gained tremendous research interest in recent years due to their extraordinary properties. The molecular-like properties that originate from the quantized electronic states provide novel opportunities for the construction of unique nanomaterials possessing rich molecular-like absorption, luminescence, and magnetic properties. The field of monolayer-protected metal nanoclusters, especially copper, with well-defined molecular structures and compositions, is relatively new, about two to three decades old. Nevertheless, the massive progress in the field illustrates the importance of such nanoobjects as promising materials for various applications. In this respect, nanocluster-based catalysts have become very popular, showing high efficiencies and activities for the catalytic conversion of chemical compounds. Biomedical applications of clusters are an active research field aimed at finding better fluorescent contrast agents, therapeutic pharmaceuticals for the treatment and prevention of diseases, the early diagnosis of cancers and other potent diseases, especially at early stages. A huge library of structures and the compositions of copper nanoclusters (CuNCs) with atomic precisions have already been discovered during last few decades; however, there are many concerns to be addressed and questions to be answered. Hopefully, in future, with the combined efforts of material scientists, inorganic chemists, and computational scientists, a thorough understanding of the unique molecular-like properties of metal nanoclusters will be achieved. This, on the other hand, will allow the interdisciplinary researchers to design novel catalysts, biosensors, or therapeutic agents using highly structured, atomically precise, and stable CuNCs. Thus, we hope this review will guide the reader through the field of CuNCs, while discussing the main achievements and improvements, along with challenges and drawbacks that one needs to face and overcome.</p></div><hr/><table><tr><td class="biogPlate"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-p1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-p1.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-p1.gif"/></a><p><span class="graphic_title">Ani Baghdasaryan</span></p></td><td><i><p>Ani Baghdasaryan received her Ph.D. in Chemistry (2020) from the University of Geneva, supervised by Prof. Thomas Burgi. While there, she worked on the designed synthesis and surface modification of metal clusters <span class="italic">via</span> ligand-exchange reactions to tune their chiroptical properties. She is currently a postdoctoral researcher at Stanford University.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-p2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-p2.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-p2.gif"/></a><p><span class="graphic_title">Thomas Burgi</span></p></td><td><i><p>Thomas Bürgi obtained his Ph.D. (1995) in Chemistry at the University of Berne (Switzerland). After a postdoc at MIT, he did his habilitation at ETH, Zürich. In 2010 he moved to the University of Geneva, where he is Professor of Physical Chemistry. His research focuses on the fundamental aspects and applications of chiral metal clusters, plasmon-based metamaterials, and the development of <span class="italic">in situ</span> spectroscopy.</p></i></td></tr></table><hr/>
    
      
      <h2 id="sect376"><span class="a_heading">1. Introduction</span></h2>
      <span>Metal nanoclusters with sizes comparable to the Fermi wavelength of electrons, <span class="italic">i.e.</span>, smaller than 2 nm, show pronounced molecular-like and size-dependent optical and luminescence properties, intrinsic magnetism, strong chiral responses, <span class="italic">etc</span>. The ultrasmall size regime breaks down the continuum of energy bands into discrete energy levels and the collective oscillations of conduction band electrons typical for plasmonic particles upon interacting with an electromagnetic field are no longer dominant. Quantized and discrete states result in rich molecular-like features in their optical absorption spectrum. Besides the above-mentioned properties, noble metal clusters are also highly stable nanoobjects. The stability, however, for atomically precise copper nanoclusters (CuNCs) is of major concern due to the high susceptibility of the metal towards oxidation. Over last few decades, by designing synthetic protocols and applying mild and inert conditions, choosing appropriate ligands with unique structures and high metal binding affinities, the stability issue is resolved to an extent, leading to the (selective) formation, crystallization and finally the determination of tens of fascinating structures and geometries.<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1–3</span></sup></a> Some structures turned out to be extraordinary not only due to the unique binding block configurations and construction moieties but also for geometries they pick up that were initially considered impossible. As for noble metals, their extraordinary stability has been investigated using several models such as the “superatom theory”<a title="Select to navigate to reference" href="#cit4"><sup><span class="sup_ref">4,5</span></sup></a> and “jellium model”.<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> According to the “superatom theory” and “jellium model”, valence electrons are delocalized and can be subjected to an external potential. Thus, clusters are considered superatoms and their angular momentum subshells can be filled according to the Aufbau principle.<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a> The superatom theory gives a series of electron shell closing numbers<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> known as “magic” numbers. The electron count for any cluster with a formula of <span class="italic">M</span><small><sub><span class="italic">n</span></sub></small>(<span class="italic">R</span>)<small><sup><span class="italic">q</span></sup></small><small><sub><span class="italic">m</span></sub></small> can be determined according to the following equation:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn1"/><span id="eqn1"><span class="italic">n</span><small><sub>e</sub></small> = <span class="italic">n</span> − <span class="italic">m</span> − <span class="italic">q</span></span></td><td class="rightEqn">(1)</td></tr></table>where <span class="italic">n</span> is the number of metal atoms (Au, Ag or Cu) in the cluster, <span class="italic">m</span> is the number of ligands (with one localized electron) and <span class="italic">q</span> is the net charge of the cluster.</span>
      <p class="otherpara">This rule has been widely used for spherical gold clusters, such as the very-well known Au<small><sub>25</sub></small>(SR)<small><sub>18</sub></small>,<a title="Select to navigate to references" href="#cit9"><sup><span class="sup_ref">9</span></sup></a> (negatively charged cluster, 8e<small><sup>−</sup></small> system: 1S<small><sup>2</sup></small> 1P<small><sup>6</sup></small>) and Au<small><sub>102</sub></small>(SR)<small><sub>44</sub></small>,<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> (neutral cluster, 58e<small><sup>−</sup></small> system: 1S<small><sup>2</sup></small> 1P<small><sup>6</sup></small> 1D<small><sup>10</sup></small> 2S<small><sup>2</sup></small> 1F<small><sup>14</sup></small> 2P<small><sup>6</sup></small> 1G<small><sup>18</sup></small>), and also applies to CuNCs. Particularly stable atomically precise spherical CuNCs are therefore expected with electron counts of 2, 8, 18, <span class="italic">etc</span>., according to <a title="" href="#eqn1">eqn (1)</a> (to be discussed in section 2.2.4). For atomically precise gold nanoclusters, all magic numbers have been discovered, whereas for CuNCs only smallest 2e<small><sup>−</sup></small> systems are known and higher systems are yet to be discovered.</p>
      <p class="otherpara">Keeping in mind the differences and common properties of atomically precise gold nanoclusters (AuNCs), we tried to summarize the main achievements in the field of CuNCs and illustrate the challenges that material scientists and inorganic chemists face on a daily basis. Thus, the purpose of this review is to explore the ever-growing field of copper nanoclusters, including chalcogenide, hydrido and pure thiolate-protected clusters. The review article focuses on the following topics: (i) the synthesis (top-down/bottom-up) of hydrophilic and hydrophobic clusters, and the differences and challenges when considering suitable ligands; (ii) size-dependent optical and luminescence properties of clusters by paying close attention to the effect of the core size and the nature of the protective ligand. In this sense, a special discussion will be dedicated to a novel aggregation-induced emission enhancement (AIE) phenomenon; (iii) structural analyses of atomically precise clusters by focusing mainly on the geometry of the metallic core and the arrangements of the ligands around the core; and (iv) exploration of the many possibilities for the application of CuNCs in biomedical, catalytic and optoelectronic fields. A key goal of this review is to present the great expansion of water-soluble CuNCs from the early 2000s and organic phase soluble and atomically precise CuNCs from 2012 until August 2020.</p>
    
    
      
      <h2 id="sect429"><span class="a_heading">2. Synthesis of CuNCs</span></h2>
      <span>The synthetic procedures are of utmost importance in order to obtain high-quality nanomaterials. The great interest in synthesizing nanomaterials comes from their unique and extraordinary applications in the different areas of modern science. Therefore, various chemical and physical methods have been developed and applied for the large-scale synthesis of metal nanoclusters with precise atomic compositions. The breakthrough in the field of nanomaterials science, starting in the early 90s, made possible the synthesis of particles down to sub-nanometre sizes. Over the last few decades, tremendous progress has been made for mainly noble metals Au and Ag protected by organic and water-soluble ligands.<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11,12</span></sup></a> Structure determination helped the understanding of the formation mechanisms and the development of new, efficient, and effective synthetic protocols for the synthesis of unique functional building blocks. However, despite the extensive research progress in the field of gold and silver nanoclusters, the studies focusing on the preparation and functionalization of other metals like earth-abundant copper are still in their infancy.<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> The problem lies in the difficulty of preparing tiny and stable copper nanoparticles and nanoclusters. Copper, being in the same group as gold and silver, shares similar properties and can be taken as a viable alternative to expensive and rare metals. Therefore, the synthetic procedures applied for the preparation of gold and silver clusters were directly applied for the synthesis of copper nanoclusters. Modifications and slight changes made in the synthetic methods and reaction conditions helped to successfully overcome the difficulty of preparing atomically precise and stable CuNCs.<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">14–16</span></sup></a></span>
      
        
        <h3 id="sect436"><span class="b_heading">2.1. Synthesis of water-soluble CuNCs</span></h3>
        <div>
          
          <span id="sect439"/><span class="c_heading_indent">2.1.1. Top-down synthesis. </span>
          <span>In the top-down synthesis of nanomaterials, the macroscopic structures (bulk matter, aggregates, particles, <span class="italic">etc</span>.) are broken up and scaled-down into small nanosized fragments that are later back-transformed into thermodynamically stable species with distinct structures, compositions, and morphologies. Apart from physical methods (mechanical milling, laser ablation and arc discharge synthesis) the wet chemical synthesis of CuNCs involves the etching or digestive ripening of larger nanoparticles<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> or nanocrystals<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> into clusters in the presence of excess ligand, by controlling the pH, temperature, and initial concentrations of reacting materials. Governed by the natural selection principle of the survival of the fittest/strongest, the most robust and thermodynamically stable clusters will be predominantly formed and dominate in the reaction mixture. An excellent example of such a selection rule in CuNCs synthesis has been reported by Amitava Patra and co-workers.<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> Starting with CuNPs, the etching of the sample in excess GSH led to the selective formation of Cu<small><sub>34–32</sub></small>(SG)<small><sub>16–13</sub></small> nanoclusters at room temperature (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>), whereas etching at relatively high temperature (70 °C) resulted in the formation of monodisperse Cu<small><sub>25</sub></small>(SG)<small><sub>20</sub></small> nanoclusters. Note that the chemical composition has been established based on mass spectrometric analysis, and no single-crystal structure information is available for the above-mentioned clusters (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1B and C</a>).</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f1.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">(A) Schematic illustration of the top-down synthesis of luminescent CuNCs. (B) (a) UV–vis absorption spectrum and (b) photoexcitation and photoluminescence spectra of red-emitting CuNCs in aqueous solution. (c) ESI-MS spectrum of CuNCs. (d) Isotopic patterns fitting to the formulas [Cu<small><sub>34</sub></small>(SG)<small><sub>16</sub></small> + 2Na<small><sup>+</sup></small> + 4H<small><sup>+</sup></small>]<small><sup>6+</sup></small> and [Cu<small><sub>34</sub></small>(SG)<small><sub>13</sub></small> + 2Na<small><sup>+</sup></small> + 4H<small><sup>+</sup></small>]<small><sup>6+</sup></small>, respectively. (C) (a) UV–vis absorption spectrum and (b) photoexcitation and photoluminescence spectra of blue-emitting CuNCs in aqueous solution. (c) ESI-MS spectrum of CuNCs obtained at 70 °C. (d) Isotopic pattern fitting to the formulas [Cu<small><sub>25</sub></small>(SG)<small><sub>20</sub></small> + 6H<small><sup>+</sup></small>]<small><sup>6+</sup></small> and [Cu<small><sub>25</sub></small>(SG)<small><sub>20</sub></small> + Na<small><sup>+</sup></small> + 7H<small><sup>+</sup></small>]<small><sup>8+</sup></small>, respectively. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit17">17</a>. Copyright 2019 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Interestingly, the bigger clusters were weakly emitting in the red, whereas the smaller ones were highly emissive in the blue spectral range. On the other hand, this showed the size-dependence of the photoluminescence properties of CuNCs (to be discussed later).</p>
          <p class="otherpara">Another reported example of the top-down method involves the utilization of ammonia as an etchant to convert non-fluorescent CuNPs into small, green, fluorescent CuNCs.<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a> The ammonia-assisted etching of the CuNCs with an average size of 3.7 nm resulted in the formation of smaller (1.2 nm) and spherical CuNCs with an absolute quantum yield of 6.6%.</p>
        </div>
        <div>
          
          <span id="sect478"/><span class="c_heading_indent">2.1.2. Bottom-up synthesis. </span>
          
            
            <span id="sect481"/><br/><span class="d_heading_indent">2.1.2.1. Template-mediated synthesis. </span>
            <span>The template-based method became a cutting-edge technology for the effective synthesis of various nanomaterials with designed structures, sizes, morphologies and properties.<a title="Select to navigate to reference" href="#cit20"><sup><span class="sup_ref">20–24</span></sup></a> The main concept of this technique is the formation of uniform nanomaterials within the pores, channels, or cavities of a nano-matrix. Thus, the target's morphology and size can be tailored by simply changing the shape of the matrix, <span class="italic">i.e.</span> cylinder, rod, sphere. In general, any porous material, including naturally occurring minerals and biological substrates, can be considered as a template. Polymer networks, hydrogels, supramolecular assemblies, biomolecules such as DNA, proteins, viruses, and other microorganisms can be used as templates. Biological substances are extensively used for nanostructure synthesis owing to their naturally occurring complex structures.</span>
            <p class="otherpara">
              <span class="bold">Polymers</span>: Several polymers with various structures were used as templates for the synthesis of CuNCs. However, the first report dates back almost two decades when Zhao <span class="italic">et al.</span> used fourth-generation (G4) poly(amidoamine) (PAMAM) dendrimers with an ethylenediamine core (G4-OH) as templates to prepare a mixture of CuNCs consisting of 4 to 64 atoms in their compositions with an average diameter of less than 1.8 nm.<a title="Select to navigate to references" href="#cit25"><sup><span class="sup_ref">25</span></sup></a> Highly stable and fluorescent (QY of 3.8% in ethanol) CuNCs were synthesized from the reduction of aqueous solutions of CuSO<small><sub>4</sub></small> and polyethyleneimine (PEI) with hydrazine hydrate upon heating at 95 °C for 19 h (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2A</a>).<a title="Select to navigate to reference" href="#cit26"><sup><span class="sup_ref">26–30</span></sup></a> When functionalized lipoic acid (LA) attached to a tunable length of polyethylene glycol (PEG) segment was used as a template, the reduction with NaBH<small><sub>4</sub></small> resulted in the formation of good-quality and highly fluorescent CuNCs (QY of 3.6% in water) with an average size of 2.5 nm.<a title="Select to navigate to references" href="#cit31"><sup><span class="sup_ref">31</span></sup></a> The formed clusters showed long-term stability when exposed to daylight and UV irradiation. However, when a similar reaction was carried out using both dihydrolipoic acid (DHLA) and poly(vinylpyrrolidone) (PVP)<a title="Select to navigate to references" href="#cit32"><sup><span class="sup_ref">32</span></sup></a> or just PVP alone<a title="Select to navigate to reference" href="#cit33"><sup><span class="sup_ref">33,34</span></sup></a> (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2B</a>) as capping agents and ascorbic acid as a reductant, the synthesis resulted in the formation of CuNCs with an average size of 2 nm. Polystyrene sulfonate (PSS)<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2C</a>) and other multidentate polymers<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> as templating scaffolds were also applied in the synthesis of CuNCs.</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f2.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">(A) Synthesis strategy for the BPEI-CuNCs, their stability under varying pH and the mechanism of the branched polyethyleneimine (BPEI)-CuNCs probe for Fe<small><sup>3+</sup></small> sensing. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit30">30</a>. Copyright 2015 Elsevier. (B) Schematic diagram of PVP-capped CuNCs (on the left) and UV–vis absorption, emission, and excitation fluorescence spectra of the probe CuNCs at pH = 6.2 in 50 mM phosphate buffer (PB) (on the right). Inset: Photograph of CuNCs probe in buffer under daylight (left) and UV-light (<span class="italic">λ</span> = 365 nm, right), respectively. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit34">34</a>. Copyright 2020 American Chemical Society. (C) A scheme depicting a plausible mechanism for the synthesis of PSS-stabilized penicillamine (PA)-CuNCs and the photographs of the PSS-PA-CuNCs synthesized under different concentrations of PSS, from 0.005 to 0.5 wt%. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit35">35</a>. Copyright 2016 Springer Nature.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">In 2012 Hui Zhang <span class="italic">et al.</span> reported the photoreductive synthesis of highly fluorescent metal nanoclusters of Cu, Ag, and Au (QYs of 2.2, 6.8, and 5.3%, respectively) in the presence of poly(methacrylic acid) functionalized with pentaerythritol tetrakis 3-mercaptopropionate (PMAA-PTMP) polymer upon UV-irradiation.<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> It was found that AuNCs were relatively stable as compared to copper and silver in the presence of foreign metal ions.</p>
            <p class="otherpara">
              <span class="bold">Proteins</span>: Proteins as structural biomolecules were widely used for the synthesis of versatile and biocompatible CuNCs (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>).<a title="Select to navigate to reference" href="#cit38"><sup><span class="sup_ref">38–59</span></sup></a> Among a huge number of accessible proteins, bovine serum albumin (BSA) gained considerable attention due to its availability, high-water solubility, and ability to bind various organic and inorganic substances in a noncovalent fashion. Due to the well-defined structure, containing charged amino acids and 35 thiol groups from Cys residues, BSA offers perfect albumin-binding sites for nanocluster formation.<a title="Select to navigate to reference" href="#cit60"><sup><span class="sup_ref">60,61</span></sup></a> The synthesis of CuNCs involves three steps till completion.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> In the first step, upon mixing the reactants, at neutral pH, –COOH carboxylate groups of the protein partially dissociate and immediately form a complex with Cu<small><sup>2+</sup></small>. The coordination complex resulted in the formation of a viscous paste. To enhance the solubility and further reduction with BSA, in the second reaction step, the pH of the medium needed to be increased to 12 by adding 1 M NaOH under heating at 55 °C.</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f3.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">(A) Illustration of the synthesis and application of the CuNCs. Republished with permission from ref. <a title="Select to navigate to references" href="#cit39">39</a>. Copyright 2014 Royal Society of Chemistry. (B) Schematic representation of reaction methods employed for the synthesis of CuNCs in cellulase solution. Republished with permission from ref. <a title="Select to navigate to references" href="#cit55">55</a>. Copyright 2016 Royal Society of Chemistry. (C) Diagrammatic illustration of the detection of dopamine using BSA CuNC synthesized in presence of H<small><sub>2</sub></small>O<small><sub>2</sub></small>. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit62">62</a>. Copyright 2019 Elsevier.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">However, as a reducing agent, BSA is relatively weak, and to completely reduce Cu<small><sup>2+</sup></small> to metallic copper, different reducing agents such as hydrogen peroxide<a title="Select to navigate to reference" href="#cit49"><sup><span class="sup_ref">49,56,62</span></sup></a> and hydrazine hydrate (N<small><sub>2</sub></small>H<small><sub>4</sub></small>·2H<small><sub>2</sub></small>O)<a title="Select to navigate to reference" href="#cit41"><sup><span class="sup_ref">41,42,45</span></sup></a> have been proposed to facilitate the last step of the reduction process. For example, the addition of a trace amount of hydrogen peroxide not only reduces the α-helix of the protein and increases the number of random and δ-structures, but also produces ˙OH radicals, which can break peptide bonds in BSA and reduce the ordered α-helix structures. Altogether, this enhances the reduction of Cu<small><sup>2+</sup></small> by the protein and leads to faster cluster formation.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> On the other hand, while using hydrazine hydrate, with decreasing the pH, the α-helix can be transformed to δ-sheets and random coil structures, thus increasing the availability of functional groups (–OH, –NH and –COOH) that can interact with the cluster.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> Other proteins such as trypsin,<a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a> human serum albumin (HSA),<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> transferrin,<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> papain,<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> cellulase<a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> and lysozyme<a title="Select to navigate to reference" href="#cit42"><sup><span class="sup_ref">42,44</span></sup></a> were also reported as effective capping agents for the preparation of fluorescent copper nanoclusters.</p>
            <p class="otherpara">
              <span class="bold">DNA</span>: DNA as a naturally occurring biostructure with variable lengths and sequences has been used for the synthesis of CuNCs (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4A</a>).<a title="Select to navigate to reference" href="#cit63"><sup><span class="sup_ref">63–77</span></sup></a> Double-stranded DNA (dsDNA) with adjustable sequences and lengths has been considered as a template for the synthesis of fluorescent copper nanoparticles/nanoclusters.<a title="Select to navigate to reference" href="#cit78"><sup><span class="sup_ref">78–82</span></sup></a> Moreover, it has been shown that at lower concentrations of metal precursor Cu<small><sup>2+</sup></small> first binds to the backbone phosphate groups of DNA <span class="italic">via</span> a nonspecific interaction. However, with increasing the concentration, the metallization of high-affinity bases is favoured. Consequently, the reduction of the pre-formed complex with ascorbic acid leads to the formation of fluorescent CuNCs.<a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> The order of mixing reagents was found to be crucial for the synthesis. In general, the reaction involves the mixing of copper salt with the ligand/template in the first step followed by reduction using the reducing agent. However, in the case of dsDNA, the yield is much higher when the first copper salt is reduced with ascorbic acid and then templated inside of DNA.<a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> The oxygen atoms (as hard Lewis bases) of phosphodiester groups and nitrogen atoms (relatively strong Lewis bases) of the nucleobases bind to Cu<small><sup>2+</sup></small> and therefore, the stabilization of a metal cation inhibits the metallization of the template. However, when the salt is first reduced to Cu<small><sup>+</sup></small>, it can undergo disproportionation and the formed Cu<small><sup>0</sup></small> can occupy and cluster inside the grooves of dsDNA. Consequently, the size of the nanoparticle/nanocluster can be tuned by changing the number of DNA base pairs, <span class="italic">i.e.</span>, much longer DNA templates resulted in the formation of nanoparticles rather than clusters. However, shorter DNA templates led to the formation of CuNCs with very low fluorescence intensity. In contrast, the PL properties can be improved by increasing the length of DNA strands to some extent.<a title="Select to navigate to reference" href="#cit72"><sup><span class="sup_ref">72,75</span></sup></a> Initially it was shown that the mismatches in nucleotides in duplex do not lead to nanocluster formation, whereas complete matches resulted in the formation of fluorescent nanoclusters.<a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> However, it was found that DNA sequences with certain mutations can successfully template CuNCs and can be used as labels for mutation detection<a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4B–F</a>). Furthermore, even the deletion of entire coding regions (exons) in some genes was observed.<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a> Qing <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> and Song <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> demonstrated the sequence-dependent formation of CuNCs and the changes in PL properties by varying the sequence of the dsDNA template. The reader is referred to recent reviews by Yi Lv and co-workers<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> and Kevin C.-W. Wu and co-workers<a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a> on DNA-templated CuNCs and applications in label-free bioassaying.</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f4.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">(A) Synthesis of the DNA-templated CuNCs. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit67">67</a>. Copyright 2017 Elsevier. (B) Schematic representation of detection strategy (Y: Single nucleotide polymorphism (SNP) site). (C) Representative TEM image and size distribution and (D) representative AFM image of Cu nanoclusters using the FULL (sequence: 5′-CCA GAT ACT CAC CGG-3′/3′-GGT CTA TGA GTG GCC-5′) duplexes as the synthetic scaffold. (E) Representative TEM image and the size distribution and (F) representative AFM image of Cu nanoclusters using the AC-mismatched (AC-MIS) duplexes (sequence: 5′-CCA GAT ACT CAC CGG-3′/3′-GGT CTA TAA GTG GCC-5′) as the synthetic scaffold. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit71">71</a>. Copyright 2012 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">The absence of grooves and base pairs in single-stranded DNA (ssDNA) excluded the formation of stable CuNCs; however, the extensive research in the field showed that highly thymine-dependent sequences of ssDNA can lead to the formation of CuNCs and the red-emissive PL properties can be improved by increasing the number of repeating thymine bases in the sequence.<a title="Select to navigate to reference" href="#cit73"><sup><span class="sup_ref">73,87–89</span></sup></a> Moreover, specific selections of single-stranded DNA (ssDNA), including random ssDNA, poly(adenine) (polyA), poly(thymine) (polyT), poly(cytosine) (polyC) and poly(guanine) (polyG) have been used for the synthesis of fluorescent copper nanoparticles (CuNPs).<a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a></p>
          
          
            
            <span id="sect573"/><br/><span class="d_heading_indent">2.1.2.2. Electrochemical synthesis. </span>
            <span>This method is based on the electrochemical reduction of copper salt and the deposition on the cathode in a typical electrochemical cell. The size, structure, and morphologies of the nanoparticles can be controlled by changing the applied currents<a title="Select to navigate to reference" href="#cit91"><sup><span class="sup_ref">91–93</span></sup></a> and using different templates within electrolyte solutions.<a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a> For instance, copper nanorods with mean diameter and length of 30 nm and 400 nm, respectively, can be synthesized in a two-electrode cell with a copper plate as the anode and a platinum cathode using a CTAB/CT<small><sub>4</sub></small>ABr/acetone/cyclohexane/water five-component system (where cyclohexane serves as a stabilizer).<a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a></span>
            <p class="otherpara">The synthesis of copper nanoclusters using the electrochemical method was first reported in 2010.<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> The synthesis of very small CuNCs (Cu<small><sub><span class="italic">n</span></sub></small> with <span class="italic">n</span> ≤ 14) in tetrabutylammonium nitrate (TBAN) as an electrolyte and protecting agent against aggregation and oxidation was done in a three-electrode conventional electrochemical cell, with a copper plate as the anode, a platinum sheet as the cathode and Ag/AgCl as the reference electrode. Cu ions produced from the soluble copper anode were reduced and spontaneously formed clusters. Recently, Vilar-Vidal <span class="italic">et al.</span> reported a selective formation of Cu<small><sub>13</sub></small> clusters in a thermostated three-electrode electrochemical cell.<a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a> In 2016 M. Arturo López-Quintela and co-workers reported the electrochemical synthesis of bare Cu<small><sub>5</sub></small> clusters that showed excellent stability against UV irradiation, heating and pH variations.<a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> In a similar study Avelino Corma and co-workers showed that the reactivity of electrochemically prepared Cu<small><sub>20</sub></small>, Cu<small><sub>8</sub></small> and Cu<small><sub>5</sub></small> clusters towards O<small><sub>2</sub></small>, and thus the oxidation of the clusters, decreased with decreasing the size of the bare clusters.<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> Unlike the Cu<small><sub>5</sub></small> cluster, bigger clusters are easily oxidized and relatively less reducible in oxygen and nitrogen environments, respectively. These studies showed that very small clusters can be prepared using electrochemical synthesis methods.</p>
          
          
            
            <span id="sect597"/><br/><span class="d_heading_indent">2.1.2.3. Water-in-oil (w/o) microemulsion. </span>
            <span>In 1993 Lisiecki <span class="italic">et al.</span> reported the first synthesis of CuNPs in w/o microemulsion using NaBH<small><sub>4</sub></small> or hydrazine as reducing agents.<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> The effect of the surfactant type and content inside the microemulsion,<a title="Select to navigate to reference" href="#cit98"><sup><span class="sup_ref">98,99</span></sup></a> and the effect of the amount of reducing agent<a title="Select to navigate to reference" href="#cit100"><sup><span class="sup_ref">100,101</span></sup></a> on the size evolution of CuNCs was studied in terms of the α parameter, which is defined by the number of reducing agent moles used for the reaction with respect to the stoichiometric amount needed for the reduction of copper cations (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>).<a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a></span>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f5.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">Schematic representation of copper nanocluster size evolution with increasing the molar percentage (<span class="italic">α</span>) of NaBH<small><sub>4</sub></small>. The clusters with <span class="italic">n</span> ≤ 13 are fluorescent, whereas big particles with <span class="italic">n</span> ≥ 309 show characteristic plasmon resonance band. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit101">101</a>. Copyright 2009 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">It was shown that with an increase in the NaBH<small><sub>4</sub></small> concentration, bigger clusters and particles were obtained, whereas small and strongly fluorescent Cu<small><sub><span class="italic">n</span></sub></small> clusters with <span class="italic">n</span> ≤ 13 atoms, were formed at lower α values.</p>
          
          
            
            <span id="sect620"/><br/><span class="d_heading_indent">2.1.2.4. Microwave-assisted synthesis. </span>
            <span>The microwave-assisted method is based on the use of microwaves as an energy source for the synthesis of copper nanomaterials.<a title="Select to navigate to reference" href="#cit102"><sup><span class="sup_ref">102–104</span></sup></a> The first preparation of CuNPs with an average size of 10 nm in ethylene glycol and NaH<small><sub>2</sub></small>PO<small><sub>2</sub></small>·H<small><sub>2</sub></small>O as a reducing agent under microwave irradiation, was described in 2004 by H.-t. Zhu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit102"><sup><span class="sup_ref">102</span></sup></a> A few years later, fluorescent CuNCs with an average size of 2 nm (containing Cu<small><sub>9</sub></small> species) were successfully prepared when NaOH was introduced into the reaction mixture upon heating at high temperatures.<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> Under harsh heating conditions, ethylene glycol undergoes dehydration and results in the formation of the aldehyde, which can reduce copper ions at high temperatures. It undergoes spontaneous ethoxylation and forms poly(ethylene)glycol (PEG), which later adsorbs on the surface of CuNCs. The surface passivation restricts the further agglomeration and oxidation of formed clusters. Recently, tannic acid<a title="Select to navigate to references" href="#cit104"><sup><span class="sup_ref">104</span></sup></a> and BSA<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> were reported as both reducing agents and protective ligands for the preparation of clusters under MW radiation.</span>
          
          
            
            <span id="sect634"/><br/><span class="d_heading_indent">2.1.2.5. Modified Brust–Schiffrin technique. </span>
            <span>The Brust–Schiffrin method is a largely applied strategy to prepare atomically precise metal nanoclusters in large quantities and narrow size distributions.<a title="Select to navigate to reference" href="#cit105"><sup><span class="sup_ref">105,106</span></sup></a> The method relies on the simultaneous growth of metal nanoclusters and the protection of their core with a thiol monolayer. The details of extensive mechanistic studies of the reduction process of the metal solution, in this case AuNCs containing protective ligand, as well as the structure and composition of intermediate species, can be found in the ref. <a title="Select to navigate to references" href="#cit107">107</a> and <a title="Select to navigate to references" href="#cit108">108</a>. Similar to gold, the [TOA][CuBr2] complex was formed as an intermediate in the synthesis of copper nanoclusters (<a title="" href="#eqn2">eqn (2) and (3)</a>), the reduction of which in the presence of 2-mercapto-5-<span class="italic">n</span>-propylpyrimidine (MMP) resulted in the formation of CuNCs:<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">14,107,109</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn2"/><span id="eqn2">[TOA]<small><sub>2</sub></small>[CuBr<small><sub>4</sub></small>] + RSH → ½RSSR + [TOA][CuBr<small><sub>2</sub></small>] + TOABr + HBr</span></td><td class="rightEqn">(2)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn3"/><span id="eqn3">[TOA][CuBr<small><sub>2</sub></small>] + BH<small><sub>4</sub></small><small><sup>−</sup></small> + RSH + RSSR → Cu<small><sub><span class="italic">x</span></sub></small>(SR)<small><sub><span class="italic">y</span></sub></small> + …</span></td><td class="rightEqn">(3)</td></tr></table></span>
            <p class="otherpara">Li <span class="italic">et al.</span> showed that the choice of reducing agent is of great importance.<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a> Cu<small><sup>0</sup></small> species are highly susceptible to oxidation in aqueous solution and thus NaBH<small><sub>4</sub></small> as a strong reducing agent (<span class="italic">E</span><small><sup>0</sup></small> = −1.24 V) decomposes rapidly and consequently does not inhibit the oxidation of the metal. On the other hand, hydrazine (N<small><sub>2</sub></small>H<small><sub>4</sub></small>) being a weaker reducing agent than NaBH<small><sub>4</sub></small> decomposes slowly but does not significantly improve the stability of the clusters. However, the authors showed that while using tetrakis(hydroxymethyl)phosphonium chloride (THPC) as a reducing agent, not only did the stability of the dihydrolipoic acid (DHLA)-stabilized cluster increase due to additional interaction with the copper core, but also the PL can be tuned from red<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a> to bright orange emission<a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> depending on the applied reaction conditions. Other ligands such as (3-mercaptopropyl)trimethoxysilane,<a title="Select to navigate to references" href="#cit112"><sup><span class="sup_ref">112</span></sup></a><span class="small_caps">D</span>-penicillamine,<a title="Select to navigate to reference" href="#cit113"><sup><span class="sup_ref">113–115</span></sup></a> cysteine,<a title="Select to navigate to reference" href="#cit116"><sup><span class="sup_ref">116–123</span></sup></a> mercaptobenzoic acid,<a title="Select to navigate to reference" href="#cit124"><sup><span class="sup_ref">124,125</span></sup></a> peptides (in particular glutathione (GSH))<a title="Select to navigate to reference" href="#cit17"><sup><span class="sup_ref">17,18,126–137</span></sup></a> and small organic molecules<a title="Select to navigate to reference" href="#cit138"><sup><span class="sup_ref">138–142</span></sup></a> were successfully applied as capping agents for the synthesis of luminescent clusters in a typical Brust–Schiffrin method (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>).</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f6.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">(A) Schematic illustration of the CuNCs/reduced graphene oxide (RGO) nanocomposite sensing platform for Hep sensing. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit126">126</a>. Copyright 2020 Elsevier. (B) Schematic representation of cysteamine (CA)-CuNCs synthesis and its sensor responses to Fe<small><sup>3+</sup></small> and I<small><sup>−</sup></small> ions. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit138">138</a>. Copyright 2020 Elsevier.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">An alternative approach to the Brust–Shiffrin method has been proposed for the synthesis of CuNCs using hydrophilic ligands other than thiols such as tannic acid,<a title="Select to navigate to reference" href="#cit143"><sup><span class="sup_ref">143–145</span></sup></a> bile acid derivatives,<a title="Select to navigate to references" href="#cit146"><sup><span class="sup_ref">146</span></sup></a> histidine,<a title="Select to navigate to references" href="#cit147"><sup><span class="sup_ref">147</span></sup></a> dopamine<a title="Select to navigate to references" href="#cit148"><sup><span class="sup_ref">148</span></sup></a> and proline.<a title="Select to navigate to references" href="#cit149"><sup><span class="sup_ref">149</span></sup></a></p>
          
        </div>
      
      
        
        <h3 id="sect692"><span class="b_heading">2.2. Synthesis of organic-phase soluble clusters</span></h3>
        <span>For the synthesis of atomically precise copper clusters, researchers followed similar procedures to those for Au clusters, which are well documented. However, considering the high susceptibility of copper towards oxidation, the reaction requires an inert atmosphere in most cases by employing Schlenk-line techniques connected to vacuum and inert gas lines. Moreover, the choice of the protecting ligand as well as its chemical structure and functionalities, the precursor of chalcogens plays a crucial role in the formation of certain cluster sizes. Although most of the reported structures had copper atoms in the +1 oxidation state or even in a mixed-valence state due to the presence of non-stoichiometric phases and electron deficiency, recently, a few “metallic” copper clusters have been reported (to be discussed later). A recent review by Didier Astruc summarized the synthesis of atomically precise CuNCs, the structures and their applications in galvanic reactions, hydrogen evolution, <span class="italic">etc</span>.<a title="Select to navigate to references" href="#cit150"><sup><span class="sup_ref">150</span></sup></a><a title="Select to navigate to table" href="#tab1">Table 1</a> summarizes all the reported atomically precise CuNCs published starting from 2010.</span>
        
<div class="table_caption"><b>Table 1</b> <span id="tab1">The table covers all the structures published starting from 2010. The papers are listed in the order of increasing cluster size</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="199.39pt"/><col align="left" width="167.50pt"/><col align="left" width="158.20pt"/><col align="left" width="68.04pt"/><col align="left" width="43.80pt"/><col align="left" width="41.37pt"/></colgroup>
              
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th rowspan="3" align="left" valign="bottom" class="border_black">Formula</th>
                  <th rowspan="3" align="left" valign="bottom" class="border_black">Ligand</th>
                  <th colspan="3" align="left" valign="bottom" class="border_black">Crystal structure</th>
                  <th rowspan="3" align="left" valign="bottom" class="border_black">Ref.</th>
                </tr>
                <tr align="left" valign="bottom">
                  <th colspan="2" align="left" valign="bottom" class="border_black">Core structure</th>
                  <th rowspan="2" align="left" valign="bottom" class="border_black">Space group</th>
                </tr>
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Geometry</th>
                  <th align="left" valign="bottom" class="border_black">Composition</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Cu<small><sub>6</sub></small>(SC<small><sub>7</sub></small>H<small><sub>4</sub></small>NO)<small><sub>6</sub></small></td>
                  <td valign="top">C<small><sub>7</sub></small>H<small><sub>5</sub></small>NOS</td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>6</sub></small></td>
                  <td valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/><span class="italic">c</span></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit151">151</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>6</sub></small>S<small><sub>6</sub></small>(SnPh)<small><sub>2</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>6</sub></small>]</td>
                  <td rowspan="3" valign="top">PhSnCl<small><sub>3</sub></small>/PPh<small><sub>2</sub></small>Et</td>
                  <td rowspan="3" valign="top">Octahedron</td>
                  <td rowspan="3" valign="top">Cu<small><sub>6</sub></small>E<small><sub>6</sub></small> (E = S, Se, Te)</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit152">152</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>6</sub></small>Se<small><sub>6</sub></small>(SnPh)<small><sub>2</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>6</sub></small>]</td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>6</sub></small>Te<small><sub>6</sub></small>(SnPh)<small><sub>2</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>6</sub></small>]</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>7</sub></small>(<span class="italic">p</span>-S–C<small><sub>6</sub></small>H<small><sub>4</sub></small>–NMe<small><sub>2</sub></small>)<small><sub>7</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>4</sub></small>]</td>
                  <td valign="top">
                    <span class="italic">p</span>-Me<small><sub>3</sub></small>SiS-C<small><sub>6</sub></small>H<small><sub>4</sub></small>-NMe<small><sub>2</sub></small>/PPh<small><sub>3</sub></small></td>
                  <td rowspan="2" valign="top">Trigonal prism</td>
                  <td rowspan="2" valign="top">Cu<small><sub>7</sub></small></td>
                  <td rowspan="2" valign="top">
                    <span class="italic">R</span>3<span class="italic">c</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit153">153</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>7</sub></small>(<span class="italic">p</span>-S–C<small><sub>6</sub></small>H<small><sub>4</sub></small>–OSiMe<small><sub>3</sub></small>)(SPh)<small><sub>6</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>4</sub></small>]</td>
                  <td valign="top">
                    <span class="italic">p</span>-Me<small><sub>3</sub></small>SiO-C<small><sub>6</sub></small>H<small><sub>4</sub></small>-SSiMe<small><sub>3</sub></small>/PhSH/PPh<small><sub>3</sub></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>7</sub></small>(H){S<small><sub>2</sub></small>P(OEt)<small><sub>2</sub></small>}<small><sub>6</sub></small>]</td>
                  <td valign="top">[NH<small><sub>4</sub></small>][S<small><sub>2</sub></small>P(OEt)<small><sub>2</sub></small>]</td>
                  <td rowspan="2" valign="top">Tetrahedron</td>
                  <td valign="top">H-Cu<small><sub>7</sub></small></td>
                  <td rowspan="2" valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit154">154</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>7</sub></small>(Br){Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>]</td>
                  <td valign="top">[NH<small><sub>4</sub></small>][Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td valign="top">Br-Cu<small><sub>7</sub></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>7</sub></small>H(S<small><sub>2</sub></small>CO<small><sup>i</sup></small>Pr)<small><sub>6</sub></small>]</td>
                  <td valign="top">KS<small><sub>2</sub></small>CO<small><sup>i</sup></small>Pr</td>
                  <td rowspan="2" valign="top">Triangular pyramid</td>
                  <td rowspan="2" valign="top">H-Cu<small><sub>7</sub></small></td>
                  <td rowspan="2" valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit155">155</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>7</sub></small>H(S<small><sub>2</sub></small>CO<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>6</sub></small>]</td>
                  <td valign="top">KS<small><sub>2</sub></small>CO<small><sup><span class="italic">n</span></sup></small>Pr</td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>7</sub></small>(H){S<small><sub>2</sub></small>CR}<small><sub>6</sub></small>] 1a–c</td>
                  <td valign="top">a: R = N<small><sup><span class="italic">n</span></sup></small>Pr<small><sub>2</sub></small></td>
                  <td rowspan="3" valign="top">Tetrahedron</td>
                  <td valign="top">Cu<small><sub>7</sub></small></td>
                  <td valign="top">2a: <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/>1<span class="italic">c</span></td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit156">156</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">[Cu<small><sub>8</sub></small>(H){S<small><sub>2</sub></small>CR}<small><sub>6</sub></small>](PF<small><sub>6</sub></small>) 2a–c</td>
                  <td valign="top">b: R = NEt<small><sub>2</sub></small></td>
                  <td valign="top">Cu<small><sub>8</sub></small></td>
                  <td valign="top">2b: <span class="italic">Pn</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/><span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">c: R = aza-15-crown-5</td>
                  <td valign="top">
                  </td>
                  <td valign="top">1c: <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>8</sub></small>(μ-H)<small><sub>6</sub></small>(μ-dppm)<small><sub>5</sub></small>](PF<small><sub>6</sub></small>)<small><sub>2</sub></small></td>
                  <td valign="top">dppm = Ph<small><sub>2</sub></small>PCH<small><sub>2</sub></small>PPh<small><sub>2</sub></small></td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>6</sub></small></td>
                  <td valign="top">
                    <span class="italic">Pna</span>2<small><sub>1</sub></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit157">157</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>8</sub></small>S<small><sub>4</sub></small>(μ<small><sub>3</sub></small>-E)<small><sub>4</sub></small>(CAAC<small><sup>Cy</sup></small>)<small><sub>4</sub></small>]</td>
                  <td rowspan="2" valign="top">CAAC<small><sup>Cy</sup></small>·HCl</td>
                  <td rowspan="2" valign="top">“Butterfly”-like</td>
                  <td rowspan="2" valign="top">Cu<small><sub>8</sub></small>E<small><sub>4</sub></small> (E = S, Se)</td>
                  <td valign="top">
                    <span class="italic">C</span>2/<span class="italic">c</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit158">158</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>8</sub></small>Se<small><sub>4</sub></small>(μ<small><sub>3</sub></small>-E)<small><sub>4</sub></small>(CAAC<small><sup>Cy</sup></small>)<small><sub>4</sub></small>]</td>
                  <td valign="top">
                    <span class="italic">C</span>2/<span class="italic">c</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>8</sub></small>(SPh)<small><sub>8</sub></small>(Ph<small><sub>3</sub></small>P)<small><sub>4</sub></small>]</td>
                  <td valign="top">PhSH</td>
                  <td valign="top">Cage</td>
                  <td valign="top">2 × Cu<small><sub>4</sub></small>S<small><sub>4</sub></small>P<small><sub>2</sub></small></td>
                  <td rowspan="2" valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit159">159</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>13</sub></small>(SePh)<small><sub>13</sub></small>(Ph<small><sub>3</sub></small>P)<small><sub>4</sub></small>]</td>
                  <td valign="top">PhSeH</td>
                  <td valign="top">Hexagonal pyramid</td>
                  <td valign="top">Cu<small><sub>4</sub></small>Se<small><sub>3</sub></small> + Cu<small><sub>9</sub></small>Se<small><sub>7</sub></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">[(CuPPh<small><sub>3</sub></small>)<small><sub>4</sub></small>(PhSn)<small><sub>18</sub></small>Cu<small><sub>10</sub></small>S<small><sub>31</sub></small>Cl<small><sub>2</sub></small>]</td>
                  <td valign="top">(PhSn)<small><sub>4</sub></small>S<small><sub>6</sub></small></td>
                  <td valign="top">Propellane-like Cu<small><sub>4</sub></small>S<small><sub>4</sub></small>/[Sn<small><sub>3</sub></small>S<small><sub>3</sub></small>] + [Sn<small><sub>2</sub></small>CuS<small><sub>3</sub></small>] six-membered rings</td>
                  <td valign="top">[Sn<small><sub>18</sub></small>Cu<small><sub>10</sub></small>S<small><sub>31</sub></small>]</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit160">160</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>11</sub></small>(TBBT)<small><sub>9</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>6</sub></small>](SbF<small><sub>6</sub></small>)<small><sub>2</sub></small></td>
                  <td valign="top">TBBT = 4-<span class="italic">tert</span>-butylbenzenethiol</td>
                  <td valign="top">Triangular bipyramid</td>
                  <td valign="top">Cu<small><sub>5</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit161">161</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>11</sub></small>(μ<small><sub>9</sub></small>-I)(μ<small><sub>3</sub></small>-I)<small><sub>3</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>](OH)</td>
                  <td valign="top">NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td rowspan="2" valign="top">Trigonal prism</td>
                  <td rowspan="2" valign="top">Cu<small><sub>11</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit162">162</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>11</sub></small>(μ<small><sub>9</sub></small>-I)(μ<small><sub>3</sub></small>-I)<small><sub>3</sub></small>(Se<small><sub>2</sub></small>PPh<small><sub>2</sub></small>)<small><sub>6</sub></small>](PF<small><sub>6</sub></small>)</td>
                  <td valign="top">NH<small><sub>4</sub></small>(Se<small><sub>2</sub></small>PPh<small><sub>2</sub></small>)</td>
                  <td valign="top">
                    <span class="italic">Pnma</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>11</sub></small>H<small><sub>2</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CPh)<small><sub>3</sub></small>]</td>
                  <td valign="top">HC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CPh</td>
                  <td rowspan="3" valign="top">Trigonal prism</td>
                  <td rowspan="3" valign="top">Cu<small><sub>11</sub></small>H<small><sub>2</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit163">163</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>11</sub></small>H<small><sub>2</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CC<small><sub>6</sub></small>H<small><sub>4</sub></small>F)<small><sub>3</sub></small>]</td>
                  <td valign="top">HC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CC<small><sub>6</sub></small>H<small><sub>4</sub></small>F</td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>11</sub></small>H<small><sub>2</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CC<small><sub>6</sub></small>H<small><sub>4</sub></small>OMe)<small><sub>3</sub></small>]</td>
                  <td valign="top">HC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CC<small><sub>6</sub></small>H<small><sub>4</sub></small>OMe</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dpppt)<small><sub>4</sub></small>]</td>
                  <td valign="top">dpppt = Ph<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>)<small><sub>5</sub></small>PPh<small><sub>2</sub></small></td>
                  <td rowspan="2" valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>P<small><sub>8</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>4<small><sub>2</sub></small>/<span class="italic">ncm</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit164">164</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>]</td>
                  <td valign="top">dppo = Ph<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>)<small><sub>8</sub></small>PPh<small><sub>2</sub></small></td>
                  <td valign="top">Cu<small><sub>12</sub></small>S<small><sub>6</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dpppt)<small><sub>4</sub></small>]</td>
                  <td valign="top">dpppt = Ph<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>)<small><sub>5</sub></small>PPh<small><sub>2</sub></small></td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>P<small><sub>8</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit165">165</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>Se<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>]</td>
                  <td valign="top">dppo = Ph<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>)<small><sub>8</sub></small>PPh<small><sub>2</sub></small></td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small>Se<small><sub>6</sub></small>P<small><sub>8</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit165">165</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dppf)<small><sub>4</sub></small>]</td>
                  <td valign="top">dppf = Ph<small><sub>2</sub></small>PCpFeCpPPh<small><sub>2</sub></small></td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small>S<small><sub>6</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0034_0304.gif" alt="[4 with combining macron]"/>2<small><sub>1</sub></small><span class="italic">c</span></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit166">166</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>8</sub></small>]</td>
                  <td valign="top">PPh<small><sub>2</sub></small>Et</td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>P<small><sub>8</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit167">167</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PEt<small><sub>3</sub></small>)<small><sub>8</sub></small>]</td>
                  <td valign="top">PEt<small><sub>3</sub></small></td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>P<small><sub>8</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit165">165</a> and <a title="Select to navigate to references" href="#cit167">167</a></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>24</sub></small>S<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>12</sub></small>]</td>
                  <td valign="top">PEt<small><sub>2</sub></small>Ph</td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Dimer of Cu<small><sub>12</sub></small>S<small><sub>6</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit165">165</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>]</td>
                  <td valign="top">PPh<small><sub>3</sub></small></td>
                  <td valign="top">Tetragonal prism</td>
                  <td valign="top">Cu<small><sub>20</sub></small>S<small><sub>10</sub></small></td>
                  <td valign="top">
                    <span class="italic">Pbca</span>
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit167">167</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(P<small><sup><span class="italic">t</span></sup></small>Bu<small><sub>3</sub></small>)<small><sub>8</sub></small>]</td>
                  <td valign="top">P<small><sup><span class="italic">t</span></sup></small>Bu<small><sub>3</sub></small></td>
                  <td valign="top">Tetragonal antiprisms</td>
                  <td valign="top">Cu<small><sub>20</sub></small>S<small><sub>10</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>4/<span class="italic">nnc</span></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit165">165</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">[Cu<small><sub>13</sub></small>(S<small><sub>2</sub></small>CN<small><sup><span class="italic">n</span></sup></small>Bu<small><sub>2</sub></small>)<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CR)<small><sub>4</sub></small>](PF<small><sub>6</sub></small>)</td>
                  <td valign="top">R = CO<small><sub>2</sub></small>Me</td>
                  <td rowspan="2" valign="top">Cuboctahedron</td>
                  <td rowspan="2" valign="top">[Cu<small><sub>13</sub></small>]<small><sup>11+</sup></small></td>
                  <td valign="top">
                    <span class="italic">C</span>2/<span class="italic">c</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit168">168</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">R = 3-FC<small><sub>6</sub></small>H<small><sub>4</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">c</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>13</sub></small>(S<small><sub>2</sub></small>CNR<small><sub>2</sub></small>)<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CR′)<small><sub>4</sub></small>](PF<small><sub>6</sub></small>) 1a–d</td>
                  <td valign="top">a: R = <small><sup><span class="italic">n</span></sup></small>Bu, R′ = CO<small><sub>2</sub></small>Me</td>
                  <td rowspan="6" valign="top">Cuboctahedron</td>
                  <td valign="top">Cu<small><sub>13</sub></small></td>
                  <td valign="top">1a: <span class="italic">C</span>2/<span class="italic">c</span></td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit168">168</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>(μ<small><sub>12</sub></small>-S){S<small><sub>2</sub></small>CNR<small><sub>2</sub></small>}<small><sub>6</sub></small>{C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CR′}<small><sub>4</sub></small>] 2a–c</td>
                  <td valign="top">b: R = <small><sup><span class="italic">n</span></sup></small>Bu, R′ = CO<small><sub>2</sub></small>Et</td>
                  <td valign="top">S-Cu<small><sub>12</sub></small></td>
                  <td valign="top">2ab: <span class="italic">P</span>2<small><sub>1</sub></small>2<small><sub>1</sub></small>2<small><sub>1</sub></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>(μ<small><sub>12</sub></small>-Cl){S<small><sub>2</sub></small>CNR<small><sub>2</sub></small>}<small><sub>6</sub></small>{C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CR′}<small><sub>4</sub></small>](PF<small><sub>6</sub></small>) 3a–e</td>
                  <td valign="top">c: R = <small><sup>i</sup></small>Pr, R′ = CO<small><sub>2</sub></small>Et</td>
                  <td valign="top">Cl-Cu<small><sub>12</sub></small></td>
                  <td valign="top">3d: <span class="italic">Pnna</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>(μ<small><sub>12</sub></small>-Br){S<small><sub>2</sub></small>CN<small><sup><span class="italic">n</span></sup></small>Bu<small><sub>2</sub></small>}<small><sub>6</sub></small>{C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CPh}<small><sub>4</sub></small>](PF<small><sub>6</sub></small>) 4e</td>
                  <td valign="top">d: R = <small><sup><span class="italic">n</span></sup></small>Pr, R′ = 3,5-(CF<small><sub>3</sub></small>)<small><sub>2</sub></small>C<small><sub>6</sub></small>H<small><sub>3</sub></small></td>
                  <td valign="top">Br-Cu<small><sub>12</sub></small></td>
                  <td valign="top">3e: <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit169">169</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">[Cu<small><sub>12</sub></small>(μ<small><sub>12</sub></small>-Cl)(μ<small><sub>3</sub></small>-Cl){S<small><sub>2</sub></small>CN<small><sup><span class="italic">n</span></sup></small>Bu<small><sub>2</sub></small>}<small><sub>6</sub></small>{C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CCO<small><sub>2</sub></small>Me}<small><sub>3</sub></small>]<small><sup>+</sup></small> 5a</td>
                  <td rowspan="2" valign="top">e: R = <small><sup><span class="italic">n</span></sup></small>Bu, R′ = Ph</td>
                  <td rowspan="2" valign="top">Cl-Cu<small><sub>12</sub></small></td>
                  <td valign="top">4e: <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">5a: <span class="italic">C</span>2/<span class="italic">c</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>12</sub></small>(SR′)<small><sub>6</sub></small>Cl<small><sub>2</sub></small>][(Cu(R′SH))<small><sub>6</sub></small>]</td>
                  <td valign="top">R′ = <small><sup><span class="italic">n</span></sup></small>Bu</td>
                  <td rowspan="2" valign="top">Cuboctahedron</td>
                  <td valign="top">[Cu<small><sub>12</sub></small>(SR′)<small><sub>6</sub></small>]<small><sup>6+</sup></small></td>
                  <td rowspan="2" valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit170">170</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>17</sub></small>(SR′′)<small><sub>6</sub></small>Cl<small><sub>13</sub></small>(THF)<small><sub>2</sub></small>(R′′SH)<small><sub>3</sub></small>][H(THF)<small><sub>2</sub></small>]<small><sub>2</sub></small></td>
                  <td valign="top">R′′ = CH<small><sub>2</sub></small>CH<small><sub>2</sub></small>Ph</td>
                  <td valign="top">[Cu<small><sub>12</sub></small>(SR′′)<small><sub>6</sub></small>]<small><sup>6+</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>14</sub></small>H<small><sub>12</sub></small>(phen)<small><sub>6</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>4</sub></small>][Cl]<small><sub>2</sub></small></td>
                  <td valign="top">Phen = 1,10-phenanthroline</td>
                  <td valign="top">Tetrahedron</td>
                  <td valign="top">[Cu<small><sub>4</sub></small>]<small><sup>4+</sup></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>2<small><sub>1</sub></small>2<small><sub>1</sub></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit171">171</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cu<small><sub>14</sub></small>(C<small><sub>2</sub></small>B<small><sub>10</sub></small>H<small><sub>10</sub></small>S<small><sub>2</sub></small>)<small><sub>6</sub></small>(CH<small><sub>3</sub></small>CN)<small><sub>8</sub></small></td>
                  <td valign="top">1,2-Dithiol-<span class="italic">o</span>-carborane</td>
                  <td valign="top">fcc</td>
                  <td valign="top">Cu<small><sub>14</sub></small></td>
                  <td valign="top">
                    <span class="italic">Fmm</span>
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit172">172</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>14</sub></small>(<span class="italic">R</span>/<span class="italic">S</span>-DPM)<small><sub>8</sub></small>](PF<small><sub>6</sub></small>)<small><sub>6</sub></small></td>
                  <td valign="top">DPM = (<span class="italic">R</span>/<span class="italic">S</span>)-2-diphenyl-2-hydroxylmethylpyrrolidine-1-propyne</td>
                  <td valign="top">Cubic</td>
                  <td valign="top">Cu<small><sub>14</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>2<small><sub>1</sub></small>2<small><sub>1</sub></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit173">173</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cu<small><sub>16</sub></small>[SC(O)OC<small><sub>6</sub></small>H<small><sub>11</sub></small>]<small><sub>16</sub></small>·2C<small><sub>3</sub></small>H<small><sub>6</sub></small>O</td>
                  <td rowspan="5" valign="top">K[SC(O)OC<small><sub>6</sub></small>H<small><sub>11</sub></small>]</td>
                  <td rowspan="5" valign="top">“Butterfly”-like</td>
                  <td rowspan="5" valign="top">Cu<small><sub>16</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="5" valign="top">
                    <a title="Select to navigate to references" href="#cit174">174</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cu<small><sub>16</sub></small>[SC(O)OC<small><sub>6</sub></small>H<small><sub>11</sub></small>]<small><sub>16</sub></small>·C<small><sub>5</sub></small>H<small><sub>8</sub></small>N<small><sub>2</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">Cu<small><sub>16</sub></small>[SC(O)OC<small><sub>6</sub></small>H<small><sub>11</sub></small>]<small><sub>16</sub></small>·C<small><sub>11</sub></small>H<small><sub>9</sub></small>N</td>
                  <td valign="top">
                    <span class="italic">P</span>2/<span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">Cu<small><sub>16</sub></small>[SC(O)OC<small><sub>6</sub></small>H<small><sub>11</sub></small>]<small><sub>16</sub></small>·C<small><sub>10</sub></small>H<small><sub>8</sub></small>N<small><sub>2</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>16</sub></small>{SC(O)OC<small><sub>6</sub></small>H<small><sub>11</sub></small>}<small><sub>16</sub></small>·C<small><sub>10</sub></small>H<small><sub>8</sub></small>N<small><sub>2</sub></small>]<small><sub>2</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>18</sub></small>H<small><sub>7</sub></small>R<small><sub>10</sub></small>I]</td>
                  <td valign="top">R = S(C<small><sub>6</sub></small>H<small><sub>4</sub></small>)PPh<small><sub>2</sub></small></td>
                  <td valign="top">Tetrahedron</td>
                  <td valign="top">Cu<small><sub>8</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit175">175</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>19</sub></small>S<small><sub>28</sub></small>(SnPh)<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>3</sub></small>][Li(THF)<small><sub>4</sub></small>]</td>
                  <td rowspan="2" valign="top">PhSnCl<small><sub>3</sub></small>/PEt<small><sub>2</sub></small>Ph</td>
                  <td rowspan="2" valign="top">Trigonal prism</td>
                  <td rowspan="2" valign="top">S-Cu<small><sub>6</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit176">176</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>19</sub></small>S<small><sub>28</sub></small>(SnPh)<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>3</sub></small>](NBu<small><sub>4</sub></small>)</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>(CCPh)<small><sub>12</sub></small>(OAc)<small><sub>6</sub></small>]</td>
                  <td valign="top">HCCPh/Ph<small><sub>2</sub></small>SiH<small><sub>2</sub></small></td>
                  <td valign="top">Tetrahedron</td>
                  <td valign="top">[Cu<small><sub>4</sub></small>]<small><sup>2+</sup></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit177">177</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>(S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>)<small><sub>9</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td valign="top">Triangular orthobicupola</td>
                  <td valign="top">Cu<small><sub>20</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit178">178</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>(Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>)<small><sub>9</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td valign="top">Cuboctahedron</td>
                  <td valign="top">Cu<small><sub>13</sub></small></td>
                  <td valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit179">179</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>}<small><sub>9</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>]</td>
                  <td valign="top">Cuboctahedron</td>
                  <td valign="top">Cu<small><sub>13</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit180">180</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>CH<small><sub>2</sub></small>Ph)<small><sub>2</sub></small>}<small><sub>9</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[S<small><sub>2</sub></small>P(C<small><sub>2</sub></small>H<small><sub>4</sub></small>Ph)<small><sub>2</sub></small>]</td>
                  <td valign="top">Triangular orthobicupola</td>
                  <td valign="top">Cu<small><sub>20</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td valign="top">Triangular orthobicupola</td>
                  <td valign="top">Cu<small><sub>20</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>2</sup></small>Bu)<small><sub>2</sub></small>}<small><sub>9</sub></small>]</td>
                  <td rowspan="2" valign="top">NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup>2</sup></small>Bu)<small><sub>2</sub></small>]</td>
                  <td valign="top">Triangular orthobicupola</td>
                  <td valign="top">Cu<small><sub>20</sub></small></td>
                  <td valign="top">
                    <span class="italic">C</span>2/<span class="italic">c</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit181">181</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>32</sub></small>H<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>2</sup></small>Bu)<small><sub>2</sub></small>}<small><sub>12</sub></small>]</td>
                  <td valign="top">Rhombohedra</td>
                  <td valign="top">Cu<small><sub>14</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>25</sub></small>H<small><sub>10</sub></small>(SPhCl<small><sub>2</sub></small>)<small><sub>18</sub></small>][PPh<small><sub>4</sub></small>]<small><sub>3</sub></small></td>
                  <td valign="top">HSPhCl<small><sub>2</sub></small></td>
                  <td valign="top">Cuboctahedron</td>
                  <td valign="top">Cu@Cu<small><sub>12</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit182">182</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>25</sub></small>H<small><sub>22</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>12</sub></small>]Cl</td>
                  <td rowspan="2" valign="top">PPh<small><sub>3</sub></small>/Ph<small><sub>2</sub></small>SiH<small><sub>2</sub></small></td>
                  <td valign="top">Icosahedron</td>
                  <td valign="top">[Cu<small><sub>13</sub></small>]<small><sup>11+</sup></small></td>
                  <td valign="top">
                    <span class="italic">Pbca</span>
                  </td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit183">183</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>18</sub></small>H<small><sub>17</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>10</sub></small>]Cl</td>
                  <td valign="top">Square antiprism</td>
                  <td valign="top">[Cu<small><sub>8</sub></small>H]<small><sup>7+</sup></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">c</span></td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">[Cu<small><sub>28</sub></small>H<small><sub>15</sub></small>(S<small><sub>2</sub></small>CNR)<small><sub>12</sub></small>]PF<small><sub>6</sub></small></td>
                  <td valign="top">NR = N<small><sup><span class="italic">n</span></sup></small>Pr<small><sub>2</sub></small></td>
                  <td rowspan="2" valign="top">Rhombicuboctahedron</td>
                  <td rowspan="2" valign="top">Cu<small><sub>24</sub></small></td>
                  <td valign="top">
                    <span class="italic">Pa</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit184">184</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">NR = aza-15-crown-5</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>29</sub></small>Cl<small><sub>4</sub></small>H<small><sub>22</sub></small>(Ph<small><sub>2</sub></small>phen)<small><sub>12</sub></small>]Cl</td>
                  <td valign="top">Ph<small><sub>2</sub></small>phen</td>
                  <td valign="top">Icosahedron</td>
                  <td valign="top">Cu<small><sub>13</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>31<span class="italic">c</span></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit185">185</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>30</sub></small>H<small><sub>18</sub></small> {S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td valign="top">Hollow icosahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="5" valign="top">
                    <a title="Select to navigate to references" href="#cit186">186</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>30</sub></small>H<small><sub>18</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td valign="top">Hollow icosahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small></td>
                  <td valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>30</sub></small>H<small><sub>18</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>}<small><sub>12</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>]</td>
                  <td valign="top">Hollow icosahedron</td>
                  <td valign="top">Cu<small><sub>12</sub></small></td>
                  <td valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[PdCu<small><sub>14</sub></small>H<small><sub>2</sub></small>{S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CPh)<small><sub>6</sub></small>]</td>
                  <td valign="top">HC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CPh/Pd(PPh<small><sub>3</sub></small>)<small><sub>2</sub></small>Cl<small><sub>2</sub></small></td>
                  <td valign="top">Icosahedron</td>
                  <td valign="top">PdH<small><sub>2</sub></small>@Cu<small><sub>14</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[PdCu<small><sub>14</sub></small>H<small><sub>2</sub></small>{S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CC<small><sub>6</sub></small>H<small><sub>4</sub></small>F)<small><sub>6</sub></small>]</td>
                  <td valign="top">HC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CC<small><sub>6</sub></small>H<small><sub>4</sub></small>F/Pd(PPh<small><sub>3</sub></small>)<small><sub>2</sub></small>Cl<small><sub>2</sub></small></td>
                  <td valign="top">Icosahedron</td>
                  <td valign="top">PdH<small><sub>2</sub></small>@Cu<small><sub>14</sub></small></td>
                  <td valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>32</sub></small>(H)<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>]</td>
                  <td valign="top">NH<small><sub>4</sub></small>[S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>]</td>
                  <td valign="top">Rhombohedra</td>
                  <td valign="top">Cu<small><sub>14</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit187">187</a> and <a title="Select to navigate to references" href="#cit188">188</a></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>32</sub></small>(PET)<small><sub>24</sub></small>H<small><sub>8</sub></small>Cl<small><sub>2</sub></small>](PPh<small><sub>4</sub></small>)<small><sub>2</sub></small></td>
                  <td valign="top">PET = 2-phenylethanethiol</td>
                  <td valign="top">Bisquare antiprism</td>
                  <td valign="top">Cu<small><sub>14</sub></small>H<small><sub>8</sub></small></td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit189">189</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>33</sub></small>(<small><sup><span class="italic">t</span></sup></small>BuC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>C)<small><sub>24</sub></small>(Mo<small><sub>4</sub></small>O<small><sub>16</sub></small>)]·BF<small><sub>4</sub></small></td>
                  <td valign="top">
                    <small><sup>
                      <span class="italic">t</span>
                    </sup></small>BuC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CH</td>
                  <td valign="top">Octahedron</td>
                  <td valign="top">Core/shell</td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">c</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit190">190</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>62</sub></small>(<small><sup><span class="italic">t</span></sup></small>BuC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>C)<small><sub>34</sub></small>(Mo<small><sub>5</sub></small>O<small><sub>19</sub></small>)<small><sub>2</sub></small>(MoO<small><sub>4</sub></small>)<small><sub>2</sub></small>(OTf)<small><sub>2</sub></small>(OH)<small><sub>4</sub></small>]·(OTf)<small><sub>2</sub></small></td>
                  <td valign="top">
                  </td>
                  <td valign="top">Cubane/octahedron</td>
                  <td valign="top">Core/shell</td>
                  <td valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">[Cu<small><sub>40</sub></small>Se<small><sub>16</sub></small>(S-C<small><sub>6</sub></small>H<small><sub>4</sub></small>-CN)<small><sub>8</sub></small>(dppm)<small><sub>8</sub></small>]</td>
                  <td valign="top">NC-C<small><sub>6</sub></small>H<small><sub>4</sub></small>-SH/dppm</td>
                  <td rowspan="2" valign="top">“Disk”-like</td>
                  <td rowspan="2" valign="top">A-B</td>
                  <td rowspan="2" valign="top">
                    <span class="italic">P</span>2<small><sub>1</sub></small>/<span class="italic">n</span></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit191">191</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dppm = Ph<small><sub>2</sub></small>PCH<small><sub>2</sub></small>PPh<small><sub>2</sub></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>43</sub></small>Al<small><sub>12</sub></small>](Cp*)<small><sub>12</sub></small></td>
                  <td valign="top">(Cp* = η<small><sup>5</sup></small>-C<small><sub>5</sub></small>Me<small><sub>5</sub></small></td>
                  <td valign="top">Icosahedron</td>
                  <td valign="top">Cu<small><sub>13</sub></small></td>
                  <td valign="top">
                    <span class="italic">Im</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit192">192</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>52</sub></small>S<small><sub>12</sub></small>(SCH<small><sub>2</sub></small>C<small><sub>6</sub></small>H<small><sub>4</sub></small><small><sup><span class="italic">t</span></sup></small>Bu)<small><sub>28</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>]</td>
                  <td valign="top">HSCH<small><sub>2</sub></small>C<small><sub>6</sub></small>H<small><sub>4</sub></small><small><sup><span class="italic">t</span></sup></small>Bu/PPh<small><sub>3</sub></small></td>
                  <td valign="top">fcc</td>
                  <td valign="top">(Cu<small><sub>2</sub></small>S)<small><sub>12</sub></small></td>
                  <td valign="top">
                    <span class="italic">C</span>2/<span class="italic">c</span></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit193">193</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>53</sub></small>(RCOO)<small><sub>10</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>C<small><sup><span class="italic">t</span></sup></small>Bu)<small><sub>20</sub></small>Cl<small><sub>2</sub></small>H<small><sub>18</sub></small>]<small><sup>+</sup></small></td>
                  <td valign="top">HC<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>C<small><sup><span class="italic">t</span></sup></small>Bu</td>
                  <td valign="top">Icosahedral/dodecahedral</td>
                  <td valign="top">Core/shell</td>
                  <td valign="top">
                    <span class="italic">Pbcn</span>
                  </td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit194">194</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cu<small><sub>74</sub></small>S<small><sub>15</sub></small>(2-PET)<small><sub>45</sub></small></td>
                  <td valign="top">2-PET = 2-phenylethanethiol</td>
                  <td valign="top">Layered</td>
                  <td valign="top">A-B-A</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit195">195</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>81</sub></small>(PhS)<small><sub>46</sub></small>(<small><sup><span class="italic">t</span></sup></small>BuNH<small><sub>2</sub></small>)<small><sub>10</sub></small>(H)<small><sub>32</sub></small>]<small><sup>3+</sup></small></td>
                  <td valign="top">PhSH/<small><sup><span class="italic">t</span></sup></small>BuNH<small><sub>2</sub></small>·BH<small><sub>3</sub></small></td>
                  <td valign="top">Planar</td>
                  <td valign="top">Cu<small><sub>17</sub></small></td>
                  <td valign="top">
                    <span class="italic">C</span>2/<span class="italic">c</span></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit196">196</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>93</sub></small>Se<small><sub>42</sub></small>(SeR)<small><sub>9</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>18</sub></small>]</td>
                  <td valign="top">R = C<small><sub>6</sub></small>H<small><sub>4</sub></small>SMe</td>
                  <td rowspan="3" valign="top">Layered</td>
                  <td valign="top">A-B-A</td>
                  <td valign="top">
                    <span class="italic">P</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0031_0304.gif" alt="[1 with combining macron]"/></td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit197">197</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>96</sub></small>Se<small><sub>45</sub></small>(SeR)<small><sub>6</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>18</sub></small>]</td>
                  <td valign="top">R = C<small><sub>6</sub></small>H<small><sub>4</sub></small>SMe</td>
                  <td valign="top">A-B-A</td>
                  <td valign="top">
                    <span class="italic">R</span><img class="charmap" src="https://www.rsc.org/images/entities/char_0033_0304.gif" alt="[3 with combining macron]"/><span class="italic">c</span></td>
                </tr>
                <tr align="left">
                  <td valign="top">[Cu<small><sub>136</sub></small>S<small><sub>56</sub></small>(SR)<small><sub>24</sub></small>(dpppt)<small><sub>10</sub></small>]</td>
                  <td valign="top">R = CH<small><sub>2</sub></small>C<small><sub>4</sub></small>H<small><sub>3</sub></small>O</td>
                  <td valign="top">A-B-C</td>
                  <td valign="top">
                    <span class="italic">C</span>2/<span class="italic">c</span></td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <div>
          
          <span id="sect2237"/><span class="c_heading_indent">2.2.1. Chalcogen-bridged CuNCs. </span>
          <span>In general, the synthesis of chalcogen-bridged, phosphine ligand-stabilized clusters involves the complexation of phosphine ligand derivatives with copper salts, which leads to the formation of reactive species. The intermediate reactive species are etched into the air- and moisture-stable copper clusters in the presence of silylated derivatives of chalcogenides (<a title="Select to navigate to scheme" href="#imgsch1">Scheme 1</a>). Note that in the following Schemes the reaction equations are not balanced (except for a few) and the equations are written in terms of the final product and the equivalent amounts of reagents used in the synthesis.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch1"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s1.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 1 </b> <span id="sch1"><span class="graphic_title">The syntheses of chalcogen-bridged CuNCs as described in ref. <a title="Select to navigate to references" href="#cit152">152</a> (Reprinted with permission, Copyright 2017 American Chemical Society), ref. <a title="Select to navigate to references" href="#cit158">158</a> (Reprinted with permission, Copyright 2019 American Chemical Society), ref. <a title="Select to navigate to references" href="#cit160">160</a> (Adapted with permission, Copyright 2019 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim), ref. <a title="Select to navigate to references" href="#cit164">164</a> and <a title="Select to navigate to references" href="#cit165">165</a> (Reprinted with permission, Copyright 2015 American Chemical Society).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The reaction conditions and initial concentrations of starting materials greatly influence the size distribution, composition, and crystallographic structures of the obtained clusters.</p>
          
            
            <span id="sect2250"/><br/><span class="d_heading_indent">2.2.1.1. Sulphur-bridged CuNCs. </span>
            <span>The pioneering work of Prof. Fenske in the field of semiconducting chalcogenide-bridged and ligand-shielded copper nanoclusters illustrated that the cluster size influences the structure. For the majority of the reported small phosphine-protected copper clusters, a [Cu<small><sub>2<span class="italic">n</span></sub></small>S<small><sub><span class="italic">n</span></sub></small>] core can be distinguished. A grand library of reported structures following the general core composition can be found in various reviews reported during the last few decades.<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1,2</span></sup></a> However, this review covers the structures of CuNCs published since 2010.</span>
            <p class="otherpara">John F. Corrigan and co-workers reported the synthesis (<a title="Select to navigate to scheme" href="#imgsch1">Scheme 1b</a>) and structures of a series of small copper(<span class="small_caps">I</span>)–chalcogenide clusters [Cu<small><sub>4</sub></small>M<small><sub>4</sub></small>(μ<small><sub>3</sub></small>-E)<small><sub>4</sub></small>(CAAC<small><sup>Cy</sup></small>)<small><sub>4</sub></small>] (M = Cu, Au, Ag; E = S, Se) stabilized by cyclic (alkyl)(amino)carbene ligands.<a title="Select to navigate to references" href="#cit158"><sup><span class="sup_ref">158</span></sup></a> Regardless of the heteroatom or the chalcogen present in the cluster, they were isostructural and isomorphous, crystalizing in the monoclinic <span class="italic">C</span>2/<span class="italic">c</span> space group. The copper atoms in the Cu<small><sub>4</sub></small> core have a nearly planar arrangement and form a square. Furthermore, the bridging of chalcogenide ligands with Cu<small><sub>4</sub></small> core atoms results in the formation of an eight-membered ring resembling a “hinged” or “butterfly” geometry. Andreas Eichhöfer and co-workers synthesized [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>] clusters protected with bidentate phosphine ligands (<a title="Select to navigate to scheme" href="#imgsch1">Scheme 1d</a>): bis(diphenylphosphino)pentane (dpppt = Ph<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>)<small><sub>5</sub></small>PPh<small><sub>2</sub></small>) or bis(diphenylphosphino)octane (dppo = Ph<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>)<small><sub>8</sub></small>PPh<small><sub>2</sub></small>).<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a></p>
            <p class="otherpara">In general, the synthetic procedure involves the reaction of copper acetate with S(SiMe<small><sub>3</sub></small>)<small><sub>2</sub></small> in toluene in the presence of bidentate phosphine ligands. The [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dpppt)<small><sub>4</sub></small>] cluster consists of a Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>P<small><sub>8</sub></small> core and crystalizes in the tetragonal space group <span class="italic">P</span>4<small><sub>2</sub></small>/<span class="italic">ncm</span>. The structure can be viewed as an octahedron of nonbonding sulphur atoms, where the twelve edges are occupied by copper atoms. The bidentate phosphine ligands bridge copper atoms at the upper and lower square planar faces (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7A</a>). A similar structure has been observed for the [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>] cluster. The only difference is that bidentate dppo ligands bridge one copper atom of the upper and one of the lower square planar face, giving rise to a helical arrangement around the Cu<small><sub>12</sub></small>S<small><sub>6</sub></small> cluster core (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7B</a>). Interestingly, both clusters in the solid state show bright red emission (from triplet states, phosphorescence) with quantum yields as high as 48% and 67% for dpppt and dppo protected clusters, respectively (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7C</a>).</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f7.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">Molecular structure of (A) [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dpppt)<small><sub>4</sub></small>] and (B) [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>] clusters. Colour codes of the elements: Cu: blue, S: yellow, P: green and C: grey. H atoms are omitted for clarity. (C) Room temperature photoluminescence excitation (PLE) and emission (PL) spectra of a suspension of freshly prepared microcrystals of [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dpppt)<small><sub>4</sub></small>] (down) and [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>] (up) in toluene, measured in the integrating sphere. The inset shows the colours of the suspensions of microcrystal sunder white LED light. Republished with permission from ref. <a title="Select to navigate to references" href="#cit164">164</a>. Copyright 2014 Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">The same group extended the synthesis of a series of copper chalcogenide clusters with varying core sizes and ligand functionalities (<a title="Select to navigate to scheme" href="#imgsch1">Scheme 1e</a> and <a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>).<a title="Select to navigate to references" href="#cit165"><sup><span class="sup_ref">165</span></sup></a> The list included structures such as [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dpppt)<small><sub>4</sub></small>], [Cu<small><sub>12</sub></small>Se<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>], [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dppf)<small><sub>4</sub></small>],<a title="Select to navigate to references" href="#cit166"><sup><span class="sup_ref">166</span></sup></a> [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>8</sub></small>],<a title="Select to navigate to references" href="#cit167"><sup><span class="sup_ref">167</span></sup></a> [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PEt<small><sub>3</sub></small>)<small><sub>8</sub></small>],<a title="Select to navigate to references" href="#cit167"><sup><span class="sup_ref">167</span></sup></a> [Cu<small><sub>24</sub></small>S<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>12</sub></small>], [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>]<a title="Select to navigate to references" href="#cit167"><sup><span class="sup_ref">167</span></sup></a> and [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(P<small><sup><span class="italic">t</span></sup></small>Bu<small><sub>3</sub></small>)<small><sub>8</sub></small>], where dppf = Ph<small><sub>2</sub></small>PCpFeCpPPh<small><sub>2</sub></small>. Note that some of the structures from the list in ref. <a title="Select to navigate to references" href="#cit165">165</a> were reported earlier. The first six compounds on the list share a similar Cu<small><sub>12</sub></small>E<small><sub>6</sub></small> (E = S, Se) core composition (building block) and tetragonal antiprism core geometries. The [Cu<small><sub>24</sub></small>S<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>12</sub></small>] cluster can be formally represented as a dimer of Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>, whereas [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(P<small><sup><span class="italic">t</span></sup></small>Bu<small><sub>3</sub></small>)<small><sub>8</sub></small>] has a core with the composition of Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>. The overall structure has an oblate shape. For the same composition with the PPh<small><sub>3</sub></small> ligand, the cluster adopts a prolate shape. Thus, [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>] and [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(P<small><sup><span class="italic">t</span></sup></small>Bu<small><sub>3</sub></small>)<small><sub>8</sub></small>] clusters can be considered as structural isomers. The crystallographic analyses of Cu–S (2.13–2.49 Å), Cu⋯Cu (2.55–3.14 Å) and S⋯S (3.64–4.62 Å) bond lengths and distances are similar in all sulphur-bridged clusters, whereas an increase was observed in the case of the selenide-bridged [Cu<small><sub>12</sub></small>Se<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>] cluster.</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f8.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">Molecular structures of [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dpppt)<small><sub>4</sub></small>] (1), [Cu<small><sub>12</sub></small>Se<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>] (2), [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(dppf)<small><sub>4</sub></small>] (3), [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>8</sub></small>] (4), [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PEt<small><sub>3</sub></small>)<small><sub>8</sub></small>] (5), [Cu<small><sub>24</sub></small>S<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>12</sub></small>] (6), [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>] (7) and [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(P<small><sup><span class="italic">t</span></sup></small>Bu<small><sub>3</sub></small>)<small><sub>8</sub></small>] (8). H atoms are omitted for clarity. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit165">165</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">Several groups reported the formation of non-stoichiometric clusters that display small variations in the total Cu/S ratio. The formation of such mixed valence states of the metal in the cluster has also been observed previously for selenide-bridged copper and silver nanoclusters.<a title="Select to navigate to reference" href="#cit198"><sup><span class="sup_ref">198,199</span></sup></a> Andreas Eichhöfer's group reported the synthesis of ternary copper–tin–chalcogenide clusters [Cu<small><sub>6</sub></small>E<small><sub>6</sub></small>(SnPh)<small><sub>2</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>6</sub></small>] (E = S, Se, Te, <a title="Select to navigate to scheme" href="#imgsch1">Scheme 1a</a>)<a title="Select to navigate to references" href="#cit152"><sup><span class="sup_ref">152</span></sup></a> and [Cu<small><sub>19</sub></small>S<small><sub>28</sub></small>(SnPh)<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>3</sub></small>]X (X = [Li(THF)<small><sub>4</sub></small>] or [NBu<small><sub>4</sub></small>], <a title="Select to navigate to scheme" href="#imgsch2">Scheme 2b</a>)<a title="Select to navigate to references" href="#cit176"><sup><span class="sup_ref">176</span></sup></a> through the reactions of CuO(O)CCH<small><sub>3</sub></small> or CuCl and PhSnCl<small><sub>3</sub></small> with E(SiMe<small><sub>3</sub></small>)<small><sub>2</sub></small> in the presence of an excess PPh<small><sub>2</sub></small>Et. In the report,<a title="Select to navigate to references" href="#cit152"><sup><span class="sup_ref">152</span></sup></a> the Cu<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>chalcogenide ratio in a series is 1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>1 regardless of the chalcogenide type. Herein, three clusters share a similar cage construction of an E<small><sub>6</sub></small> octahedron, where two trigonal faces are capped by PhSn units. The remaining six R<small><sub>3</sub></small>PCu units bridge the free edges in μ<small><sub>2</sub></small> and μ<small><sub>3</sub></small> fashion for sulphide- and selenide/telluride-bridged CuNCs, respectively. In the case of [Cu<small><sub>19</sub></small>S<small><sub>28</sub></small>(SnPh)<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>3</sub></small>]X clusters, regardless of the counter ion, the structure is composed of an inner core and an outer cluster sphere.<a title="Select to navigate to references" href="#cit176"><sup><span class="sup_ref">176</span></sup></a> The inner core is constructed from a sulphur atom coordinated with six copper atoms in a distorted trigonal prism fashion. The rest of the copper and sulphur atoms together with the remaining ligands form the outer sphere.</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch2"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s2.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 2 </b> <span id="sch2"><span class="graphic_title">The syntheses of chalcogen-bridged CuNCs as described in ref. <a title="Select to navigate to references" href="#cit166">166</a> (Reprinted with permission, Copyright 2013 American Chemical Society), ref. <a title="Select to navigate to references" href="#cit176">176</a> (Republished with permission, Copyright 2012 Royal Society of Chemistry), ref. <a title="Select to navigate to references" href="#cit193">193</a> (Republished with permission, Copyright 2016 Royal Society of Chemistry), ref. <a title="Select to navigate to references" href="#cit197">197</a> (Adapted with permission, Copyright 2010 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">Another example of such a phenomenon in the ternary systems with tin has been reported by the group of Stefanie Dehnen by the reaction of the binary organotin sulphide cluster [(PhSn)<small><sub>4</sub></small>S<small><sub>6</sub></small>] with [Cu(PPh<small><sub>3</sub></small>)<small><sub>3</sub></small>Cl] (<a title="Select to navigate to scheme" href="#imgsch1">Scheme 1c</a>).<a title="Select to navigate to references" href="#cit160"><sup><span class="sup_ref">160</span></sup></a> The resulting [(CuPPh<small><sub>3</sub></small>)<small><sub>4</sub></small>(PhSn)<small><sub>18</sub></small>Cu<small><sub>6</sub></small>S<small><sub>31</sub></small>] cluster is a pseudo dimer having a [Sn<small><sub>18</sub></small>Cu<small><sub>10</sub></small>S<small><sub>31</sub></small>] core, which is separated in two identical parts connected by a single sulphur atom (<a title="Select to navigate to figure" href="#imgfig9">Fig. 9A</a>).</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f9.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">(A) Molecular structure of the [(CuPPh<small><sub>3</sub></small>)<small><sub>4</sub></small>(PhSn)<small><sub>18</sub></small>Cu<small><sub>6</sub></small>S<small><sub>31</sub></small>] cluster in ball-stick representation (top) and with coordination polyhedra (bottom) around the Sn (blue) and Cu (orange) atoms. Organic groups are shown as transparent wires. H atoms and two disordered organic substituents are omitted for clarity. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit160">160</a>. Copyright 2019 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim. (B) Top: Molecular structure of [Cu<small><sub>52</sub></small>S<small><sub>12</sub></small>(SCH<small><sub>2</sub></small>C<small><sub>6</sub></small>H<small><sub>4</sub></small><small><sup><span class="italic">t</span></sup></small>Bu)<small><sub>28</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>] in the solid state. Bottom: Structure of the inorganic cluster core. Republished with permission from ref. <a title="Select to navigate to references" href="#cit193">193</a>. Copyright 2016 Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
            <p class="otherpara">Several larger CuNCs also fall into this category of clusters with non-stoichiometric composition. Examples involve the [Cu<small><sub>52</sub></small>S<small><sub>12</sub></small>(SCH<small><sub>2</sub></small>C<small><sub>6</sub></small>H<small><sub>4</sub></small><small><sup><span class="italic">t</span></sup></small>Bu)<small><sub>28</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>]<a title="Select to navigate to references" href="#cit193"><sup><span class="sup_ref">193</span></sup></a> (<a title="Select to navigate to scheme" href="#imgsch2">Scheme 2c</a>) and [Cu<small><sub>136</sub></small>S<small><sub>56</sub></small>(SCH<small><sub>2</sub></small>C<small><sub>4</sub></small>H<small><sub>3</sub></small>O)<small><sub>24</sub></small>(dpppt)<small><sub>10</sub></small>] clusters<a title="Select to navigate to references" href="#cit197"><sup><span class="sup_ref">197</span></sup></a> (<a title="Select to navigate to scheme" href="#imgsch2">Scheme 2d</a>) by the Fenske group. The former cluster consists of a (Cu<small><sub>2</sub></small>S)<small><sub>12</sub></small> core surrounded by eight PPh<small><sub>3</sub></small> ligands and 28 [CuSCH<small><sub>2</sub></small>C<small><sub>6</sub></small>H<small><sub>4</sub></small><small><sup><span class="italic">t</span></sup></small>Bu] subunits, where all the copper atoms are oxidized (+1) (<a title="Select to navigate to figure" href="#imgfig9">Fig. 9B</a>). [Cu<small><sub>136</sub></small>S<small><sub>56</sub></small>(SCH<small><sub>2</sub></small>C<small><sub>4</sub></small>H<small><sub>3</sub></small>O)<small><sub>24</sub></small>(dpppt)<small><sub>10</sub></small>] is so far the largest reported sulphide-bridged cluster. Being at the borderline between molecular and bulk compounds, this cluster has a layered structure with a cubic closest packing (fcc; stacking A-B-C).<a title="Select to navigate to references" href="#cit197"><sup><span class="sup_ref">197</span></sup></a> In this case, some of the copper positions are partially occupied resulting in copper content deficiency. Recently, our group reported the structure of a Cu<small><sub>74</sub></small>S<small><sub>15</sub></small>(2-PET)<small><sub>45</sub></small> (2-PET = 2-phenylethanethiol) cluster with similar Cu deficiencies.<a title="Select to navigate to references" href="#cit195"><sup><span class="sup_ref">195</span></sup></a> The reduction of a copper salt with 2-PET ligand was completed with the addition of triethylamine (TEA) as a base. The crystallization of the sample from the saturated solution resulted in the formation of dark reddish-orange crystals. The single-crystal analysis revealed that the cluster had a rod-like structure ((<a title="Select to navigate to figure" href="#imgfig10">Fig. 10A</a>) spliced into seven sulphur atom layers, organized in an A-B-A fashion (<a title="Select to navigate to figure" href="#imgfig10">Fig. 10B</a>).</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f10.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">(A) The core structure of the Cu<small><sub>74</sub></small>S<small><sub>15</sub></small>(2-PET)<small><sub>45</sub></small> cluster. Bridged-sulphur atoms are in orange, and sulphur atoms belonging to ligands are in yellow. The layers are labelled as L1 to L7 from left to right. (B) Side and top views of the layer organization of the sulphur atoms. Two types of layers (A in yellow, B in gold) organize themselves in an A-B-A pattern. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit195">195</a>. Copyright 2020 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          
          
            
            <span id="sect2552"/><br/><span class="d_heading_indent">2.2.1.2. Selenium/tellurium-bridged CuNCs. </span>
            <span>Similar to what has been discussed in the previous section, selenium- and tellurium-bridged CuNCs with and without stoichiometric compositions have been reported in the literature (<a title="Select to navigate to table" href="#tab1">Table 1</a> and <a title="Select to navigate to scheme" href="#imgsch1">Scheme 1</a>). For instance, all three types of clusters in the series [Cu<small><sub>6</sub></small>E<small><sub>6</sub></small>(SnPh)<small><sub>2</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>6</sub></small>] (E = S, Se, Te) share the same E<small><sub>6</sub></small> octahedron and have almost identical ligand binding motifs.<a title="Select to navigate to references" href="#cit152"><sup><span class="sup_ref">152</span></sup></a> On the same note, [Cu<small><sub>4</sub></small>M<small><sub>4</sub></small>(μ<small><sub>3</sub></small>-E)<small><sub>4</sub></small>(CAAC<small><sup>Cy</sup></small>)<small><sub>4</sub></small>] (M = Cu, Ag, Au; E = S, Se) clusters reported by John F. Corrigan and co-workers, have the same “butterfly”-geometries regardless of the heteroatom doping and bridging chalcogen.<a title="Select to navigate to references" href="#cit158"><sup><span class="sup_ref">158</span></sup></a> [Cu<small><sub>12</sub></small>Se<small><sub>6</sub></small>(dppo)<small><sub>4</sub></small>] cluster<a title="Select to navigate to references" href="#cit165"><sup><span class="sup_ref">165</span></sup></a> is isostructural to its sulphide-bridged analog,<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a> yet a slight variation in bond lengths was observed. The same group also reported Te-bridged [Cu<small><sub>4</sub></small>(μ<small><sub>4</sub></small>-Te)(μ<small><sub>4</sub></small>-η<small><sup>2</sup></small>-Te<small><sub>2</sub></small>)(μ-dppf)<small><sub>2</sub></small>] and [Cu<small><sub>12</sub></small>(μ<small><sub>5</sub></small>-Te)(μ<small><sub>8</sub></small>-η<small><sup>2</sup></small>-Te<small><sub>2</sub></small>)(μ-dppf)<small><sub>4</sub></small>] clusters containing telluride and ditelluride ligands in their CuTe architectures (<a title="Select to navigate to scheme" href="#imgsch2">Scheme 2a</a>).<a title="Select to navigate to references" href="#cit166"><sup><span class="sup_ref">166</span></sup></a> Large Se-bridged CuNCs have been reported back in 2010 by the group of Prof. Fenske: [Cu<small><sub>93</sub></small>Se<small><sub>42</sub></small>(Se-C<small><sub>6</sub></small>H<small><sub>4</sub></small>-SMe)<small><sub>9</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>18</sub></small>] (1) and [Cu<small><sub>96</sub></small>Se<small><sub>45</sub></small>(Se-C<small><sub>6</sub></small>H<small><sub>4</sub></small>-SMe)<small><sub>6</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>18</sub></small>] (2) (<a title="Select to navigate to scheme" href="#imgsch2">Scheme 2d</a>).<a title="Select to navigate to references" href="#cit197"><sup><span class="sup_ref">197</span></sup></a> Moreover, depending on the applied temperature conditions, the same reaction procedure can yield two different cluster compounds. Both clusters have similar layered arrangements in an A-B-A fashion, where all the copper atoms occupy identical positions inside the selenium lattices. Interestingly, compound (1) is a dimer in the solid-state. A similar A-B layered arrangement of the chalcogenide substructure was found in a [Cu<small><sub>40</sub></small>Se<small><sub>16</sub></small>(S-C<small><sub>6</sub></small>H<small><sub>4</sub></small>-CN)<small><sub>8</sub></small>(dppm)<small><sub>8</sub></small>] cluster, the structure of which was published in 2013 by Robert Langer <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit191"><sup><span class="sup_ref">191</span></sup></a> Starting from 2013, C. W. Liu together with Jean-Yves Saillard reported several selenide-CuNCs, namely [Cu<small><sub>7</sub></small>Br{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>],<a title="Select to navigate to references" href="#cit154"><sup><span class="sup_ref">154</span></sup></a> [Cu<small><sub>11</sub></small>(μ<small><sub>9</sub></small>-I)(μ<small><sub>3</sub></small>-I)<small><sub>3</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>](OH) and [Cu<small><sub>11</sub></small>(μ<small><sub>9</sub></small>-I)(μ<small><sub>3</sub></small>-I)<small><sub>3</sub></small>(Se<small><sub>2</sub></small>PPh<small><sub>2</sub></small>)<small><sub>6</sub></small>](PF<small><sub>6</sub></small>),<a title="Select to navigate to references" href="#cit162"><sup><span class="sup_ref">162</span></sup></a> [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>(Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>)<small><sub>9</sub></small>],<a title="Select to navigate to references" href="#cit179"><sup><span class="sup_ref">179</span></sup></a> [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>}<small><sub>9</sub></small>]<a title="Select to navigate to references" href="#cit180"><sup><span class="sup_ref">180</span></sup></a> and [Cu<small><sub>30</sub></small>H<small><sub>18</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>], [Cu<small><sub>30</sub></small>H<small><sub>18</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>}<small><sub>12</sub></small>],<a title="Select to navigate to references" href="#cit186"><sup><span class="sup_ref">186</span></sup></a> accompanied by extensive computational analyses of the structures. Note that in these clusters, the Se atoms are not bridged but rather are part of the ligand structure and composition (selenolates).</span>
          
        </div>
        <div>
          
          <span id="sect2667"/><span class="c_heading_indent">2.2.2. Thiolate-protected CuNCs. </span>
          <span>Thiolates are very popular and effective ligands for the preparation and passivation of the metallic cores of atomically precise gold and silver clusters. The strong sulphur–metal bond<a title="Select to navigate to references" href="#cit200"><sup><span class="sup_ref">200</span></sup></a> helps to protect the clusters against aggregation and contributes to the stability of the clusters: a property that is a prerequisite for their applications in optoelectronics and in biomedicine. Although great progress has been made on thiolate-protected gold and silver clusters with precise molecular compositions and with well-established size-property relations, atomically precise and thiolate-protected copper clusters lag behind. Despite the advancements in making atomically precise chalcogen-bridged and hydrido copper clusters, there are only a few reports on thiolate-protected copper clusters. We start the discussion with “pure”-thiolated copper clusters. Within this category, Wei Chen and co-workers in 2016 reported the synthesis and structure of a small Cu<small><sub>6</sub></small>(SR)<small><sub>6</sub></small> cluster protected with 2-mercaptobenzoxazole ligand from the reduction of a reaction mixture with sodium borohydride (<a title="Select to navigate to scheme" href="#imgsch3">Scheme 3a</a>).<a title="Select to navigate to references" href="#cit151"><sup><span class="sup_ref">151</span></sup></a></span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch3"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s3.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 3 </b> <span id="sch3"><span class="graphic_title">The syntheses of thiolate-protected CuNCs as described in ref. <a title="Select to navigate to references" href="#cit151">151</a> (Adapted with permission, Copyright 2016 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim), ref. <a title="Select to navigate to references" href="#cit170">170</a> (Reprinted with permission, Copyright 2019 American Chemical Society), and ref. <a title="Select to navigate to references" href="#cit174">174</a> (Adapted with permission, Copyright 2017 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The Cu<small><sub>6</sub></small> core has an octahedral geometry and is protected by six SR-units through a bridging bonding motif (<a title="Select to navigate to figure" href="#imgfig11">Fig. 11A</a>). In 2017 the C.W. Liu group reported the synthesis and structure of a Cu<small><sub>16</sub></small>L<small><sub>16</sub></small>·2CH<small><sub>3</sub></small>COCH<small><sub>3</sub></small> cluster (L is cyclohexylmonothiocarbonate), where the coordinated solvent molecule(s) can be easily replaced with other N-donor ligands through coordinated-solvent-replacement reactions (<a title="Select to navigate to scheme" href="#imgsch3">Scheme 3b</a>).<a title="Select to navigate to references" href="#cit174"><sup><span class="sup_ref">174</span></sup></a> Regardless of the type of coordinated solvent molecule, all structures are similar. The Cu<small><sub>16</sub></small> core can be seen as an assembly of three pentanuclear units arranged in a “butterfly”-like geometry (<a title="Select to navigate to figure" href="#imgfig11">Fig. 11B</a>). The central pentanuclear unit resembles a pyramid, whereas the other two units can be considered as butterfly motifs. The entire metallic framework is protected with cyclohexylmonothiocarbonate ligands binding in a bridging mode. Hayton and co-workers recently reported a novel synthetic strategy for the preparation of “atlas-sphere”-like [Cu<small><sub>12</sub></small>(SR′)<small><sub>6</sub></small>Cl<small><sub>12</sub></small>][(Cu(R′SH))<small><sub>6</sub></small>] (R′ = <small><sup><span class="italic">n</span></sup></small>Bu) and [Cu<small><sub>17</sub></small>(SR′′)<small><sub>6</sub></small>Cl<small><sub>13</sub></small>(THF)<small><sub>2</sub></small>(R′′SH)<small><sub>3</sub></small>][H(THF)<small><sub>2</sub></small>]<small><sub>2</sub></small> (R′′ = CH<small><sub>2</sub></small>CH<small><sub>2</sub></small>Ph) clusters (<a title="Select to navigate to scheme" href="#imgsch3">Scheme 3c</a>), both of them having a [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>]<small><sup>6+</sup></small> cuboctahedral core (<a title="Select to navigate to figure" href="#imgfig11">Fig. 11C and D</a>).<a title="Select to navigate to references" href="#cit170"><sup><span class="sup_ref">170</span></sup></a> Two other thiolate-protected copper clusters Cu<small><sub>7</sub></small> and Cu<small><sub>74</sub></small> were reported in 2013 by Olaf Fuhr and co-workers<a title="Select to navigate to references" href="#cit153"><sup><span class="sup_ref">153</span></sup></a> and in 2020 by Burgi and co-workers,<a title="Select to navigate to references" href="#cit195"><sup><span class="sup_ref">195</span></sup></a> respectively. Note that the former cluster also contains phosphine ligands in the ligand shell, whereas in the last case, the Cu<small><sub>74</sub></small> core contains 15 sulphur atoms incorporated into the metallic framework, which is further protected with 45 thiolate (in this case 2-PET: 2-phenylethylthiol) ligands (<a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>).</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig11"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f11.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f11.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 11 </b> <span id="fig11"><span class="graphic_title">(A) The total structure of a Cu<small><sub>6</sub></small>(C<small><sub>7</sub></small>H<small><sub>4</sub></small>ONS)<small><sub>6</sub></small> cluster. Colour legend: green sphere, Cu; yellow sphere, S; blue sphere, N; red sphere, O; light grey sphere, C; deep grey sphere, H. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit151">151</a>. Copyright 2016 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim. (B) Solid-state structure of a Cu<small><sub>16</sub></small>[SCOOC<small><sub>6</sub></small>H<small><sub>11</sub></small>]<small><sub>16</sub></small>·2C<small><sub>3</sub></small>H<small><sub>6</sub></small>O cluster. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit174">174</a>. Copyright 2017 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim. The core structures of [Cu<small><sub>12</sub></small>(SR′)<small><sub>6</sub></small>Cl<small><sub>12</sub></small>][(Cu(R′SH))<small><sub>6</sub></small>] (R′ = <small><sup><span class="italic">n</span></sup></small>Bu) (C) and [Cu<small><sub>17</sub></small>(SR′′)<small><sub>6</sub></small>Cl<small><sub>13</sub></small>(THF)<small><sub>2</sub></small>(R′′SH)<small><sub>3</sub></small>][H(THF)<small><sub>2</sub></small>]<small><sub>2</sub></small> (R′′ = CH<small><sub>2</sub></small>CH<small><sub>2</sub></small>Ph) clusters (D). For clarity, the organic part that is connected to the sulphur atoms and occupies one of the six square faces of the cuboctahedron has been omitted. Colour legend: orange, Cu; yellow, S; green, Cl; red, O; grey, C. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit170">170</a>. Copyright 2019 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">In 2013 T. Pradeep and co-workers reported the synthesis of a “pure” thiolate-protected cluster; no single crystal analyses were conducted, however, the molecular composition of the cluster was assigned by MALDI and ESI mass spectrometry.<a title="Select to navigate to references" href="#cit201"><sup><span class="sup_ref">201</span></sup></a></p>
          <p class="otherpara">Recently, the group of Manzhou Zhu reported the synthesis of two thiolated CuNCs with the composition of [Cu<small><sub>8</sub></small>(SPh)<small><sub>8</sub></small>(Ph<small><sub>3</sub></small>P)<small><sub>4</sub></small>]<a title="Select to navigate to references" href="#cit159"><sup><span class="sup_ref">159</span></sup></a> (1) and [Cu<small><sub>11</sub></small>(TBBT)<small><sub>9</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>6</sub></small>](SbF<small><sub>6</sub></small>)<small><sub>2</sub></small><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><a title="Select to navigate to references" href="#cit161"><sup><span class="sup_ref">161</span></sup></a> (where TBBT = 4-tert-butylbenzenethiol). Note that these are not entirely “pure” thiolate-protected clusters and have phosphine ligands contributing to the ligand shell composition. Interestingly, the ligand exchange of (1) with PhSeH ligand resulted in the drastic core transformation and formation of a new [Cu<small><sub>13</sub></small>(SePh)<small><sub>13</sub></small>(Ph<small><sub>3</sub></small>P)<small><sub>4</sub></small>] cluster with different atom-packing modes (the Se atom possesses more coordination modes with Cu (μ<small><sub>2</sub></small>, μ<small><sub>3</sub></small>, μ<small><sub>4</sub></small>, μ<small><sub>6</sub></small>) than S (μ<small><sub>2</sub></small>, μ<small><sub>3</sub></small>).</p>
        </div>
        <div>
          
          <span id="sect2774"/><span class="c_heading_indent">2.2.3. Hydrido CuNCs. </span>
          <span>A key component for future renewable energy sources is to have an effective and clean energy carrier. In this sense, hydrogen is an attractive option to consider as a universal fuel for future generations. Accordingly, materials that can store hydrogen are the focus of attention. Although the arsenal of potential hydrogen storage materials is growing rapidly, the interest in metal hydrides and especially in nanomaterial-based compounds and complex systems is increasing drastically.<a title="Select to navigate to reference" href="#cit202"><sup><span class="sup_ref">202,203</span></sup></a> In this section we will pay close attention to polyhydrido copper clusters as alternative sources in hydrogen storage and release applications. The reader is also referred to recent reviews on polyhydrido CuNCs by C.W. Liu<a title="Select to navigate to reference" href="#cit204"><sup><span class="sup_ref">204,205</span></sup></a> and Joseph P. Sadighi,<a title="Select to navigate to references" href="#cit206"><sup><span class="sup_ref">206</span></sup></a> where the authors discuss the synthetic advances, nanocluster-to-nanoparticle transformations and conversions as well as their applications in hydrogen storage/release and catalysis.</span>
          <p class="otherpara">The groups of Prof. Trevor W. Hayton and Prof. C. W. Liu revolutionized the field of copper-hydride clusters over the past few years by reporting a series of CuNCs with fascinating structures and ligand functionalities. The list extends from the small clusters to relatively large ones including those with partial Cu(0) character. <a title="Select to navigate to scheme" href="#imgsch4">Schemes 4–6</a> summarize some of the synthetic protocols used for the preparation of clusters.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch4"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s4.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 4 </b> <span id="sch4"><span class="graphic_title">The syntheses of hydrido CuNCs as described in ref. <a title="Select to navigate to references" href="#cit163">163</a> (Reprinted with permission, Copyright 2020 American Chemical Society), ref. <a title="Select to navigate to references" href="#cit171">171</a> (Adapted with permission, Copyright 2015 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim), ref. <a title="Select to navigate to references" href="#cit178">178</a>, <a title="Select to navigate to references" href="#cit179">179</a>, <a title="Select to navigate to references" href="#cit184">184</a> and <a title="Select to navigate to references" href="#cit185">185</a> (Reprinted with permission, Copyright 2016 American Chemical Society).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch5"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s5.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 5 </b> <span id="sch5"><span class="graphic_title">The syntheses of hydrido CuNCs as described in ref. <a title="Select to navigate to references" href="#cit180">180</a> (Adapted with permission, Copyright 2019 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim), ref. <a title="Select to navigate to references" href="#cit181">181</a> (Adapted with permission, Copyright 2018 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim), ref. <a title="Select to navigate to references" href="#cit182">182</a> and <a title="Select to navigate to references" href="#cit186">186</a> (Adapted with permission, Copyright 2020 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch6"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s6.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 6 </b> <span id="sch6"><span class="graphic_title">The syntheses of hydrido CuNCs as described in ref. <a title="Select to navigate to references" href="#cit154">154</a> and <a title="Select to navigate to references" href="#cit155">155</a> (Reprinted with permission, Copyright 2019 Elsevier), ref. <a title="Select to navigate to references" href="#cit156">156</a> and <a title="Select to navigate to references" href="#cit157">157</a> (Adapted with permission, Copyright 2017 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">In 2014 the Liu group reported the synthesis (<a title="Select to navigate to scheme" href="#imgsch4">Scheme 4a</a>) and structure of a novel [Cu<small><sub>28</sub></small>H<small><sub>15</sub></small>{S<small><sub>2</sub></small>CNPr<small><sub>2</sub></small>}<small><sub>12</sub></small>]PF<small><sub>6</sub></small> cluster composed of a rhombicuboctahedral framework of 24 Cu atoms.<a title="Select to navigate to references" href="#cit184"><sup><span class="sup_ref">184</span></sup></a> The Cu<small><sub>24</sub></small> core of the clusters was further enclosed by a truncated octahedron of 24 thiolate ligands (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12A</a>). The hydrides were found in the centre of the core (one central interstitial hydride); eight were outer-triangular-face-capping hydrides and six were face-truncating hydrides forming a bridge between the inner and outer copper atom shells.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig12"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f12_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f12.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f12.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 12 </b> <span id="fig12"><span class="graphic_title">(A) The molecular structure of a [Cu<small><sub>28</sub></small>H<small><sub>15</sub></small>{S<small><sub>2</sub></small>CNPr<small><sub>2</sub></small>}<small><sub>12</sub></small>] cations. The counter ions are omitted for clarity. The following colour coding has been applied for the description: carbon: grey, copper: cyan, hydrogen: red, nitrogen: pink, sulphur: yellow. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit184">184</a>. Copyright 2014 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim. (B) Ball and stick diagrams of [Cu<small><sub>29</sub></small>Cl<small><sub>4</sub></small>H<small><sub>22</sub></small>(Ph<small><sub>2</sub></small>phen)<small><sub>12</sub></small>]Cl. The Cu<small><sub>13</sub></small> centred-icosahedral core is highlighted in blue. The remaining Cu atoms are shown in dark green. The N atoms are purple, and the Cl atoms are lime green. All solvent molecules and outer sphere counter ions have been omitted for clarity. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit185">185</a>. Copyright 2016 American Chemical Society. (C) The molecular structure of the [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>] cluster. Isopropoxy groups have been omitted for clarity. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit179">179</a>. Copyright 2015 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim. (D) Crystal structures of a [Cu<small><sub>32</sub></small>H<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>] cluster. Isopropoxy groups have been omitted for clarity. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit187">187</a>. Copyright 2015 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim. (E) Crystal structure of a [Cu<small><sub>18</sub></small>H<small><sub>7</sub></small>{(S(C<small><sub>6</sub></small>H<small><sub>4</sub></small>)PPh<small><sub>2</sub></small>)}<small><sub>10</sub></small>I] cluster. Hydrogen atoms are omitted for clarity. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit175">175</a>. Copyright 2014 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim. (F) The core structure of a [Cu<small><sub>25</sub></small>H<small><sub>10</sub></small>(SPhCl<small><sub>2</sub></small>)<small><sub>18</sub></small>]<small><sup>3−</sup></small> cluster. Metal atoms within the shell and in the two layers are bridged by μ<small><sub>2</sub></small> and μ<small><sub>3</sub></small> S atoms. Colour code: dark-red and light-brown, Cu; grey, C; yellow, S; bright green, Cl; purple, P. Hydrogen atoms have been omitted for clarity. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit182">182</a>. Copyright 2019 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Hayton and co-workers in 2015 reported a novel hydride cluster compound (<a title="Select to navigate to scheme" href="#imgsch4">Scheme 4b</a>) with a composition of [Cu<small><sub>14</sub></small>H<small><sub>12</sub></small>(phen)<small><sub>6</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>4</sub></small>][X]<small><sub>2</sub></small> (X = Cl or OTf; OTf = trifluoromethanesulfonate, phen = 1,10-phenanthroline) protected with neutral donor ligands featuring a tetrahedral [Cu<small><sub>4</sub></small>]<small><sup>4+</sup></small> core.<a title="Select to navigate to references" href="#cit171"><sup><span class="sup_ref">171</span></sup></a> In that same year, they reported the first-ever [Cu<small><sub>25</sub></small>H<small><sub>22</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>12</sub></small>]Cl (1) cluster with partial Cu(0) character (to be discussed later in section 2.2.4).<a title="Select to navigate to references" href="#cit183"><sup><span class="sup_ref">183</span></sup></a> The ligand-induced structural transformations in the latter [Cu<small><sub>25</sub></small>H<small><sub>22</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>12</sub></small>]Cl (1) cluster, resulted in the formation of a new [Cu<small><sub>29</sub></small>Cl<small><sub>4</sub></small>H<small><sub>22</sub></small>(Ph<small><sub>2</sub></small>phen)<small><sub>12</sub></small>]Cl (2, Ph<small><sub>2</sub></small>phen = 4,7-diphenyl-1,10-phenanthroline) cluster in the presence of excess ligand (<a title="Select to navigate to scheme" href="#imgsch4">Scheme 4c</a>).<a title="Select to navigate to references" href="#cit185"><sup><span class="sup_ref">185</span></sup></a> Like its precursor, the Cu<small><sub>29</sub></small> cluster features a Cu<small><sub>13</sub></small> icosahedral core typical to most of the known and well-characterized monolayer-protected gold nanoclusters. The core of the cluster is connected to four triangular [Cu<small><sub>4</sub></small>(Ph<small><sub>2</sub></small>phen)<small><sub>3</sub></small>(Cl)] motifs <span class="italic">via</span> Cu–Cu bonds in a tetrahedral arrangement (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12B</a>).</p>
          <p class="otherpara">Ligand exchange induced structural transformations and the conversion from achiral [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>] (1) into chiral [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>] (2) has been observed by the group of C.W. Liu (<a title="Select to navigate to scheme" href="#imgsch4">Scheme 4d</a>).<a title="Select to navigate to references" href="#cit179"><sup><span class="sup_ref">179</span></sup></a> Unlike the transformation from a [Cu<small><sub>25</sub></small>H<small><sub>22</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>12</sub></small>]Cl cluster to a [Cu<small><sub>29</sub></small>Cl<small><sub>4</sub></small>H<small><sub>22</sub></small>(Ph<small><sub>2</sub></small>phen)<small><sub>12</sub></small>]Cl cluster,<a title="Select to navigate to references" href="#cit185"><sup><span class="sup_ref">185</span></sup></a> the ligand exchange reaction of [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>] cluster resulted in a drastic core structure transformation while converting the achiral compound (1, <a title="Select to navigate to scheme" href="#imgsch4">Scheme 4e</a>)<a title="Select to navigate to references" href="#cit178"><sup><span class="sup_ref">178</span></sup></a> with <span class="italic">C</span><small><sub>3h</sub></small> symmetry into an intrinsically chiral compound (2) with <span class="italic">C</span><small><sub>3</sub></small> symmetry. Both enantiomers can be found in the single unit cell. The structure of the left-handed enantiomer consists of three anticlockwise copper strands of ideal <span class="italic">C</span><small><sub>3</sub></small> symmetry and features an almost ideal cuboctahedral Cu<small><sub>13</sub></small> core (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12C</a>). The core is further covered by a Cu<small><sub>6</sub></small> cupola (from the top) and a capping Cu atom (from the bottom). The whole copper framework is covered by nine diselenophosphate ligands. The structure determination by neutron diffraction revealed the positions of all eleven H atoms: seven are capping μ<small><sub>3</sub></small>-H ligands and four are in interstitial μ<small><sub>5</sub></small>-H locations. Furthermore, the thermal treatment of a [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>] cluster solution in the presence of terminal alkynes leads to subsequent core transformations and the formation of [Cu<small><sub>11</sub></small>H<small><sub>2</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>6</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>CR)<small><sub>3</sub></small>] (R = Ph, C<small><sub>6</sub></small>H<small><sub>4</sub></small>F and C<small><sub>6</sub></small>H<small><sub>4</sub></small>OMe) clusters (<a title="Select to navigate to scheme" href="#imgsch4">Scheme 4f</a>).<a title="Select to navigate to references" href="#cit163"><sup><span class="sup_ref">163</span></sup></a></p>
          <p class="otherpara">Under optimized reaction conditions, ligand exchange reactions can be performed to obtain clusters with different ligand shells while preserving the composition and the structure of an initial cluster. For example, [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>}<small><sub>9</sub></small>] protected with Se-donor ligands can be obtained in good yield through the ligand exchange reaction of [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>] with NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>] or a [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(CH<small><sub>2</sub></small>CH<small><sub>2</sub></small>Ph)<small><sub>2</sub></small>}<small><sub>9</sub></small>] cluster can be produced from the ligand replacement reaction of [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>}<small><sub>9</sub></small>] with NH<small><sub>4</sub></small>[S<small><sub>2</sub></small>P(C<small><sub>2</sub></small>H<small><sub>4</sub></small>Ph)<small><sub>2</sub></small>] (<a title="Select to navigate to scheme" href="#imgsch5">Scheme 5a</a>).<a title="Select to navigate to references" href="#cit180"><sup><span class="sup_ref">180</span></sup></a> In a similar fashion [Cu<small><sub>30</sub></small>H<small><sub>18</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>] and [Cu<small><sub>30</sub></small>H<small><sub>18</sub></small>{Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>}<small><sub>12</sub></small>] clusters can be obtained from the ligand exchange of [Cu<small><sub>30</sub></small>H<small><sub>18</sub></small>{S<small><sub>2</sub></small>P(O<small><sup><span class="italic">n</span></sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>] with NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>] and NH<small><sub>4</sub></small>[Se<small><sub>2</sub></small>P(O<small><sup><span class="italic">i</span></sup></small>Bu)<small><sub>2</sub></small>], respectively (<a title="Select to navigate to scheme" href="#imgsch5">Scheme 5b</a>).<a title="Select to navigate to references" href="#cit186"><sup><span class="sup_ref">186</span></sup></a> Furthermore, All three clusters possess a hollow Cu<small><sub>12</sub></small> icosahedral core embedded inside the rhombicuboctahedral Cu<small><sub>18</sub></small> framework. Although the ligand exchange reactions are quite straightforward and are at the centre of attention in the nanocluster community, the C. W. Liu group established a direct method for the synthesis of various polynuclear copper hydrido clusters such as [Cu<small><sub>20</sub></small>H<small><sub>11</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>2</sup></small>Bu)<small><sub>2</sub></small>}<small><sub>9</sub></small>], [Cu<small><sub>32</sub></small>H<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>2</sup></small>Bu)<small><sub>2</sub></small>}<small><sub>12</sub></small>]<a title="Select to navigate to references" href="#cit181"><sup><span class="sup_ref">181</span></sup></a> (<a title="Select to navigate to scheme" href="#imgsch5">Scheme 5c</a>) and [Cu<small><sub>32</sub></small>H<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>]<a title="Select to navigate to references" href="#cit187"><sup><span class="sup_ref">187</span></sup></a> from the corresponding ammonium salts (as a ligand source) and copper complexes. Interestingly enough, those clusters having the same number of copper atoms in the composition adopt similar crystal structures and geometries regardless of the type of ligand. For instance, the [Cu<small><sub>32</sub></small>H<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>] cluster<a title="Select to navigate to references" href="#cit187"><sup><span class="sup_ref">187</span></sup></a> is composed of a distorted hexa-capped rhombohedral Cu<small><sub>14</sub></small> core sandwiched between two nest-like triangular cupola fragments of Cu atoms (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12D</a>). The twenty hydrides reside inside the structure through twelve μ<small><sub>3</sub></small>-H, six μ<small><sub>4</sub></small>-H, and two μ<small><sub>5</sub></small>-H locations. Very recently, Sanghwa Lee <span class="italic">et al.</span> reported the structure of another [Cu<small><sub>32</sub></small>(PET)<small><sub>24</sub></small>H<small><sub>8</sub></small>Cl<small><sub>2</sub></small>](PPh<small><sub>4</sub></small>)<small><sub>2</sub></small> cluster that possesses a rare edge-sharing bisquare anitiprismatic Cu<small><sub>14</sub></small>H<small><sub>8</sub></small> core capped by Cu<small><sub>7</sub></small>(PET)<small><sub>11</sub></small>Cl and Cu<small><sub>2</sub></small>PET units.<a title="Select to navigate to references" href="#cit189"><sup><span class="sup_ref">189</span></sup></a></p>
          <p class="otherpara">[Cu<small><sub>18</sub></small>H<small><sub>7</sub></small>L<small><sub>10</sub></small>I] (L = HS(C<small><sub>6</sub></small>H<small><sub>4</sub></small>)PPh<small><sub>2</sub></small>) and [Cu<small><sub>25</sub></small>H<small><sub>10</sub></small>(SPhCl<small><sub>2</sub></small>)<small><sub>18</sub></small>]<small><sup>3−</sup></small> hydride clusters were reported by Miguel A. Huertos <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit175"><sup><span class="sup_ref">175</span></sup></a> and Cunfa Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit182"><sup><span class="sup_ref">182</span></sup></a> (<a title="Select to navigate to scheme" href="#imgsch5">Scheme 5d</a>), respectively. The former structure consists of a Cu<small><sub>8</sub></small> core surrounded by the remaining 10 Cu atoms that are connected to S and P atoms (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12E</a>). The latter structure consists of a core best represented as a Cu@Cu<small><sub>12</sub></small> centered twinned cuboctahedron (ctco).<a title="Select to navigate to references" href="#cit182"><sup><span class="sup_ref">182</span></sup></a> The remaining copper atoms form a truncated tetrahedral shell, the edges of which are bridged with 18 thiolates (<a title="Select to navigate to figure" href="#imgfig12">Fig. 12F</a>). All the hydrogen atoms in hydride form were located inside the metallic core (six μ<small><sub>6</sub></small>-H and four μ<small><sub>3</sub></small>-H). Very recently, the Osman M. Bakr group reported the synthesis and full structural analyses of high-nuclearity CuNCs: [Cu<small><sub>81</sub></small>(PhS)<small><sub>46</sub></small>(<small><sup><span class="italic">t</span></sup></small>BuNH<small><sub>2</sub></small>)<small><sub>10</sub></small>(H)<small><sub>32</sub></small>]<small><sup>3+</sup></small>.<a title="Select to navigate to references" href="#cit196"><sup><span class="sup_ref">196</span></sup></a> The cluster possesses an unusual Cu<small><sub>17</sub></small> planar core, a hemispherical shell comprising a curved and a planar surface layer as well as different surface protective motifs of the ligands. This is so far the cluster with the largest number of hydrides; their presence and positions have been established by ESI-MS and DFT calculations, respectively.</p>
          <p class="otherpara">Besides the relatively large hydrido copper clusters discussed above, there are also small clusters containing less than 10 copper atoms. In this context, Prof. C.W. Liu established synthetic methods for the preparation of small hydrido copper clusters, namely Cu<small><sub>7</sub></small> and Cu<small><sub>8</sub></small> (<a title="Select to navigate to scheme" href="#imgsch6">Scheme 6a–c</a>).<a title="Select to navigate to reference" href="#cit154"><sup><span class="sup_ref">154–156</span></sup></a> Regardless of the ligand, the clusters share a common hydrogen<a title="Select to navigate to reference" href="#cit155"><sup><span class="sup_ref">155,156</span></sup></a> atom (or bromine<a title="Select to navigate to references" href="#cit154"><sup><span class="sup_ref">154</span></sup></a>) located in the middle of a tetrahedron or a pyramid. Tomoaki Tanase and co-workers reported the structure of a Cu<small><sub>8</sub></small>H<small><sub>6</sub></small> cluster, where hydrogen atoms bridge the Cu<small><sub>3</sub></small> triangular faces of a Cu<small><sub>6</sub></small> octahedral core (<a title="Select to navigate to scheme" href="#imgsch6">Scheme 6d</a>).<a title="Select to navigate to references" href="#cit157"><sup><span class="sup_ref">157</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect3122"/><span class="c_heading_indent">2.2.4. CuNCs with partial Cu(0) character. </span>
          <span>Atomically precise and monolayer-protected gold nanoclusters are interesting model structures. Some of them have a superatom electronic structure that contributes to their stability. Another feature of these clusters is their core-shell structure containing a metallic core (Au atoms are in the 0 oxidation state) protected by staple motifs of a different kind where gold atoms are formally in the +1 oxidation state. Since gold and copper are in the same group in the periodic table, they are expected to share common properties in terms of their ability to form similar core/shell geometries and ligand environments. Nonetheless, from this standpoint, the chemistry of copper nanoclusters, their preferential binding motifs and core geometries are still scarcely understood. It is common knowledge that on increasing the number of copper atoms in the structure, the ratio of Cu–Cu bonds with respect to Cu–ligand bonds increases. Hence, the oxidation state of the metal in the cluster is expected to be zero at least as concerning the cluster core. This is well accepted for nanoparticles, where the average oxidation state of the metal in the cluster is considered to be zero since the number of core atoms and hence the Cu–Cu bonds overpass the number of the copper atoms linked to the surface ligands (herein the metal atoms are oxidized). However, until lately, all the reported copper nanoclusters portray copper atoms in the +1 oxidation state in the core or in the metal–organic framework, connected either <span class="italic">via</span> Cu–Cu or Cu–E (E = S, Se, Te) or Cu–SR bonds. To date, only a few CuNCs have been reported with Cu(0) character. <a title="Select to navigate to scheme" href="#imgsch7">Schemes 7</a> and <a title="Select to navigate to scheme" href="#imgsch8">8</a> summarize the main synthetic methods for the preparation of such clusters.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch7"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s7.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 7 </b> <span id="sch7"><span class="graphic_title">The syntheses of (hydrido) CuNCs with partial Cu(0) character as described in ref. <a title="Select to navigate to references" href="#cit168">168</a> (Adapted with permission, Copyright 2016 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim), ref. <a title="Select to navigate to references" href="#cit169">169</a> and <a title="Select to navigate to references" href="#cit177">177</a> (Reprinted with permission, Copyright 2018 American Chemical Society), and ref. <a title="Select to navigate to references" href="#cit183">183</a> (Reprinted with permission, Copyright 2015 American Chemical Society).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch8"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s8.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 8 </b> <span id="sch8"><span class="graphic_title">The syntheses of CuNCs with partial Cu(0) character as described in ref. <a title="Select to navigate to references" href="#cit172">172</a> and <a title="Select to navigate to references" href="#cit192">192</a> (Adapted with permission, Copyright 2018 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim), and ref. <a title="Select to navigate to references" href="#cit194">194</a>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">In 2015 the group of Trevor. W. Hayton reported the first copper cluster, [Cu<small><sub>25</sub></small>H<small><sub>22</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>12</sub></small>]Cl, having Cu(0) character and a Cu<small><sub>13</sub></small> centred-icosahedral core (<a title="Select to navigate to scheme" href="#imgsch7">Scheme 7a</a> and <a title="Select to navigate to figure" href="#imgfig13">Fig. 13A</a>).<a title="Select to navigate to references" href="#cit183"><sup><span class="sup_ref">183</span></sup></a> The core of the cluster is connected to four triangular [Cu(PPh<small><sub>3</sub></small>)]<small><sub>3</sub></small> motifs forming a tetrahedral arrangement. What is striking about the structure is that the Cu<small><sub>13</sub></small> core is isostructural to the known M<small><sub>13</sub></small> core in atomically precise gold nanoclusters such as in the Au<small><sub>25</sub></small>(SR)<small><sub>18</sub></small> cluster.<a title="Select to navigate to references" href="#cit207"><sup><span class="sup_ref">207</span></sup></a> Based on the molecular composition, to balance the charge, two out of 25 copper atoms must be in the zero oxidation state. The electron count for the cluster gives the value of 2, thus the cluster can be considered as a <span class="italic">n</span>* = 2 superatom with a 1S<small><sup>2</sup></small> closed-shell configuration. A year later, the same group reported a Cu<small><sub>29</sub></small> cluster (the structure of the cluster has been described in the previous 2.2.3. section) as a result of the ligand exchange reaction and subsequent core transformations of the Cu<small><sub>25</sub></small> cluster in the presence of an excess ligand (<a title="Select to navigate to scheme" href="#imgsch4">Scheme 4c</a>).<a title="Select to navigate to references" href="#cit185"><sup><span class="sup_ref">185</span></sup></a> In 2018, Hayton and co-workers reported a [Cu<small><sub>20</sub></small>(CCPh)<small><sub>12</sub></small>(OAc)<small><sub>6</sub></small>] cluster as another 2-electron superatom system (<a title="Select to navigate to scheme" href="#imgsch7">Scheme 7b</a>).<a title="Select to navigate to references" href="#cit177"><sup><span class="sup_ref">177</span></sup></a> The structure of the latter can be depicted as a [Cu<small><sub>4</sub></small>]<small><sup>2+</sup></small> tetrahedral core trapped inside the [Cu<small><sub>16</sub></small>(CCPh)<small><sub>12</sub></small>(OAc)<small><sub>6</sub></small>]<small><sup>2−</sup></small>shell. Earlier on, the group of C. W. Liu reported a [Cu<small><sub>13</sub></small>(S<small><sub>2</sub></small>CN<small><sup><span class="italic">n</span></sup></small>Bu<small><sub>2</sub></small>(OAc)<small><sub>6</sub></small>](PF<small><sub>6</sub></small>) cluster bearing acetylide groups (OAc) in μ<small><sub>3</sub></small> fashion capping four triangular faces of a cuboctahedral [Cu<small><sub>13</sub></small>]<small><sup>11+</sup></small> core (<a title="Select to navigate to scheme" href="#imgsch7">Scheme 7c</a>).<a title="Select to navigate to references" href="#cit168"><sup><span class="sup_ref">168</span></sup></a> Similar core structures were observed in a series of new Cu<small><sub>13</sub></small> and Cu<small><sub>12</sub></small> clusters reported by the same group in 2018 (<a title="Select to navigate to scheme" href="#imgsch7">Scheme 7d</a>), where, in the latter, the central atom of a cuboctahedral Cu<small><sub>13</sub></small> core was replaced with S, Cl and Br leading to the formation of inverse-coordination clusters.<a title="Select to navigate to references" href="#cit169"><sup><span class="sup_ref">169</span></sup></a></p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig13"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f13_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f13.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f13.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 13 </b> <span id="fig13"><span class="graphic_title">(A) Ball and stick diagrams of [Cu<small><sub>25</sub></small>H<small><sub>22</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>12</sub></small>]Cl. The Cu<small><sub>13</sub></small>-centred-icosahedral core is highlighted in blue. The four [Cu(PPh<small><sub>3</sub></small>)]<small><sub>3</sub></small> capping motifs are shown with the Cu atoms in green and P atoms in pink. The side view with carbon atoms depicted in wireframe format. All hydrogen atoms, chloride counter ions and solvent molecules have been omitted for clarity. Bottom left: Side view showing only the Cu and P atoms. Bottom right: Top view, looking down the C<small><sub>3</sub></small> axis, showing only the Cu and P atoms. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit183">183</a>. Copyright 2015 American Chemical Society. (B) Molecular structure of the Cu<small><sub>14</sub></small>(C<small><sub>2</sub></small>B<small><sub>10</sub></small>H<small><sub>10</sub></small>S<small><sub>2</sub></small>)<small><sub>6</sub></small>(CH<small><sub>3</sub></small>CN)<small><sub>8</sub></small> cluster. Bottom left: Cu<small><sub>8</sub></small><small><sup>8+</sup></small> shell and bottom right: Cu<small><sub>8</sub></small><small><sup>8+</sup></small> shell with Cu<small><sub>6</sub></small><small><sup>4+</sup></small> core of the fcc-Cu<small><sub>14</sub></small> framework. Colour codes: brown and pink = copper; yellow = sulphur; grey = carbon; blue = nitrogen; turquoise = boron. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit172">172</a>. Copyright 2019 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Thomas C. W. Mak and co-workers very recently reported the synthesis of another Cu<small><sub>14</sub></small>(C<small><sub>2</sub></small>B<small><sub>10</sub></small>H<small><sub>10</sub></small>S<small><sub>2</sub></small>)<small><sub>6</sub></small>(CH<small><sub>3</sub></small>CN)<small><sub>8</sub></small> cluster with partial Cu(0) character having a fcc core geometry (<a title="Select to navigate to scheme" href="#imgsch8">Scheme 8a</a>).<a title="Select to navigate to references" href="#cit172"><sup><span class="sup_ref">172</span></sup></a> The six faces of the core are capped by bidentate 1,2-dithiolate-<span class="italic">o</span>-carborane ligands, whereas the eight vertices are connected to CH<small><sub>3</sub></small>CN ligands (<a title="Select to navigate to figure" href="#imgfig13">Fig. 13B</a>).</p>
          <p class="otherpara">The [Cu<small><sub>53</sub></small>(RCOO)<small><sub>10</sub></small>(C<img class="charmap" src="https://www.rsc.org/images/entities/char_e002.gif" alt="[triple bond, length as m-dash]"/>C<small><sup><span class="italic">t</span></sup></small>Bu)<small><sub>20</sub></small>Cl<small><sub>2</sub></small>H<small><sub>18</sub></small>]<small><sup>+</sup></small> cluster (<a title="Select to navigate to scheme" href="#imgsch8">Scheme 8b</a>) is another 2-electron superatom having a core–shell1–shell2–shell3 arrangement of Cu<small><sub>3</sub></small>@Cu<small><sub>10</sub></small>Cl<small><sub>2</sub></small>@Cu<small><sub>20</sub></small>@Cu<small><sub>20</sub></small> (<a title="Select to navigate to figure" href="#imgfig14">Fig. 14A</a>).<a title="Select to navigate to references" href="#cit194"><sup><span class="sup_ref">194</span></sup></a> Copper atoms in the Cu<small><sub>3</sub></small> core are in a triangular arrangement, whereas the 10 copper atoms form an icosahedron with 2 Cl atoms at the poles. The second shell of Cu<small><sub>20</sub></small> makes a pentagonal dodecahedron, which is further trapped inside a Cu<small><sub>20</sub></small> nanowheel with a diameter of 1.1 nm (shell 3, <a title="Select to navigate to figure" href="#imgfig14">Fig. 14B</a>).</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig14"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f14_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f14.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f14.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 14 </b> <span id="fig14"><span class="graphic_title">(A) Molecular structure of a Cu<small><sub>53</sub></small> cluster: the view along an approximately five-fold axis (top), and general view (bottom). Colour codes: Cu; blue, F; light green, Cl; dark yellow, O; red, C; grey. Hydrogen atoms are omitted for clarity. (B) Shell-by-shell structural analyses of the cluster framework. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit194">194</a>. Copyright 2019 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">In 2018, Jana Weßing and co-workers reported the structure of a Mackay-type M<small><sub>55</sub></small> heterometallic cluster [Cu<small><sub>43</sub></small>Al<small><sub>12</sub></small>](Cp*)<small><sub>12</sub></small> (<a title="Select to navigate to scheme" href="#imgsch8">Scheme 8c</a>) that possesses an ideal two-shell icosahedral structure.<a title="Select to navigate to references" href="#cit192"><sup><span class="sup_ref">192</span></sup></a> The detailed structure includes a body-centred Cu<small><sub>13</sub></small> icosahedron protected by a heterometallic [Cu<small><sub>30</sub></small>Al<small><sub>12</sub></small>] shell (<a title="Select to navigate to figure" href="#imgfig15">Fig. 15</a>). The structure is completed with the binding of twelve Cp* ligands to Al centres.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig15"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f15_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f15.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f15.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 15 </b> <span id="fig15"><span class="graphic_title">(A) The molecular structure of a [Cu<small><sub>43</sub></small>Al<small><sub>12</sub></small>](Cp*)<small><sub>12</sub></small> cluster in the solid state, viewed along the five-fold rotational axis. H atoms and co-crystallized solvent molecules have been omitted for clarity. Colour codes: Al; blue, Cu; orange, C; grey. (B) Naked metal cores and the underlying tetrahedral M<small><sub>10</sub></small> subunit of a cluster with highlighted M@12 M@42 M polyhedral shell geometries. The outer shell is constructed from a Cu<small><sub>30</sub></small> icosidodecahedron embedded inside an Al<small><sub>12</sub></small> icosahedron. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit192">192</a>. Copyright 2018 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The structural variety of the above-mentioned 2-electron superatoms of CuNCs is quite big as compared to gold. Besides, high electron counts have not been observed for CuNCs, which may be related to the lower stability (oxidation) of copper compared to gold.</p>
        </div>
      
    
    
      
      <h2 id="sect3277"><span class="a_heading">3. Size-dependent properties of CuNCs</span></h2>
      
        
        <h3 id="sect3280"><span class="b_heading">3.1. Optical properties</span></h3>
        <span>It is well established that depending on the reaction conditions and the initial concentrations of the reacting components, the size of the obtained clusters can be drastically varied. Consequently, with the changes in size, the physicochemical properties will also be affected. Due to the quantum confinement effects, the continuum conduction and valence bands break down into discrete energy levels in clusters (with sizes smaller than 2 nm). Thus, depending on the size of the cluster core, copper nanoclusters with well-defined compositions can possess characteristic electronic transitions in the UV–vis spectrum. The UV–vis absorption spectrum is a characteristic optical fingerprint for both qualitative (identification) and quantitative (concentration) analysis of a cluster of interest. With just a glance at the spectrum, it is possible to determine the following: (i) whether the cluster is big or small by observing the presence or absence of the plasmon peak of copper particles (in this case) at 570 nm; (ii) the optical gap, by conversion of the wavelength scale into photon energy and extrapolating the absorbance to zero value. Normally, the optical gap gets bigger by decreasing the size of the cluster. DFT calculations are required to assign the bands in the UV–vis spectrum to certain electronic transitions. Interestingly, though, compared to gold clusters,<a title="Select to navigate to references" href="#cit208"><sup><span class="sup_ref">208</span></sup></a> copper clusters do not have rich multi-band UV–vis spectra, especially in the visible region. Typically, the optical spectra of CuNCs are characterized by a smooth exponential increase in their absorbance at low energies and have multiple bands and transitions at higher energies mostly governed by the absorption of the protective ligand. This phenomenon is quite elusive and not yet understood, some of the transitions have already been assigned and marked. For instance, in trinuclear copper(<span class="small_caps">I</span>) thiolate complexes and hexanuclear copper(<span class="small_caps">I</span>) selenolate [Cu<small><sub>6</sub></small>(μ-P^P)<small><sub>4</sub></small>(μ<small><sub>3</sub></small>-SePh)<small><sub>4</sub></small>](BF<small><sub>4</sub></small>)<small><sub>2</sub></small> (P^P = dppm, (Ph<small><sub>2</sub></small>P)<small><sub>2</sub></small>NH)<a title="Select to navigate to references" href="#cit209"><sup><span class="sup_ref">209</span></sup></a> clusters the absorption bands/shoulders at <span class="italic">ca</span>. 250–290 nm in the UV–vis spectra were assigned as an intraligand (IL) n(S) → π* transition. For an octanuclear copper(<span class="small_caps">I</span>) methanediide cluster with a gyrobifastigium geometry, the absorption bands in the UV region were assigned as metal-centred metal–metal 3d → 4p transitions.<a title="Select to navigate to references" href="#cit210"><sup><span class="sup_ref">210</span></sup></a> Besides, the electronic transitions in the 300–500 nm region and below 260 nm were found to be ligand-to-metal charge transfer (LMCT) in tetranuclear, pentanuclear and hexanuclear Cu(<span class="small_caps">I</span>) clusters with aromatic ligands.<a title="Select to navigate to reference" href="#cit210"><sup><span class="sup_ref">210,211</span></sup></a> Note that depending on the ligand shell and the size of the cluster, these transitions can be shifted either way. Eichhöfer and co-workers studied the optical properties of chalcogenide clusters differing in the size and the type of protective ligand and compared the obtained experimental results with calculated absorption spectra.<a title="Select to navigate to references" href="#cit165"><sup><span class="sup_ref">165</span></sup></a> For instance, it was found that in [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>8</sub></small>], [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>] and [Cu<small><sub>24</sub></small>S<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>12</sub></small>] clusters the high energy transitions (above 2.5 eV) originated from excitations of electrons from cluster core orbitals to ligand orbitals (<a title="Select to navigate to figure" href="#imgfig16">Fig. 16A–C</a>). Such transitions are also observable for Cu<small><sub>20</sub></small> and Cu<small><sub>24</sub></small> clusters at lower energies. However, in the case of Cu<small><sub>12</sub></small>, the low energy transitions also have contributions from the cluster core too, <span class="italic">i.e.</span>, the electrons are transferred from the filled copper d-orbitals and sulphur p-orbitals to the empty p-orbitals of copper atoms. Moreover, for smaller clusters, the LUMO has contributions from the copper orbitals, whereas the HOMO mainly consists of d(Cu) and p(S) orbitals. On top of that, the low-energy excitations are very sensitive to the ligand shell and the excitation band character varies with the ligand environment even for the same core size.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig16"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f16_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f16.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f16.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 16 </b> <span id="fig16"><span class="graphic_title">A comparison of measured electronic spectra (powdered crystals in mineral oil) of (A) [Cu<small><sub>12</sub></small>S<small><sub>6</sub></small>(PPh<small><sub>2</sub></small>Et)<small><sub>8</sub></small>], (B) [Cu<small><sub>20</sub></small>S<small><sub>10</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>8</sub></small>] and (C) [Cu<small><sub>24</sub></small>S<small><sub>12</sub></small>(PEt<small><sub>2</sub></small>Ph)<small><sub>12</sub></small>] with calculated singlet excitation energies and oscillator strengths plotted as vertical green lines as well as by superimposed Gaussians of fwhm = 0.3 eV (black) to simulate the experimental spectrum. The contributions of occupied orbitals are plotted in red, and those of the unoccupied orbitals are in blue. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit165">165</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect3334"><span class="b_heading">3.2. Photoluminescence</span></h3>
        <span>Photoluminescence (PL), as a molecular property to absorb light of a particular energy and emit it at longer wavelengths, is an intriguing phenomenon that gives rise to many applications. PL is an important technique in modern spectroscopy and biology labs to study all kinds of processes. For that purpose, intrinsic (naturally occurring, <span class="italic">e.g.</span> porphyrins) and extrinsic (synthesized, such as organic molecules and quantum dots) fluorophores with high quantum yields (QYs) have been applied as dyes to detect and label biological specimens and various metal ions/small molecules in contaminated environments.<a title="Select to navigate to references" href="#cit212"><sup><span class="sup_ref">212</span></sup></a> Therefore, highly fluorescent but biocompatible and nontoxic dyes are very much required. After the discovery of monolayer-protected clusters, massive research efforts have been devoted to the preparation of nanomaterial-based fluorophores. Recent progress in this field resulted in the synthesis of various cluster-based fluorophores with high QYs, which are biocompatible.<a title="Select to navigate to references" href="#cit24"><sup><span class="sup_ref">24</span></sup></a> The PL phenomenon in CuNCs is not well understood. Perhaps, the situation is relatively easier for gold clusters due to the known compositions. However, for CuNCs the situation is often more complicated due to the unresolved structural information for water-soluble CuNCs, to say the least. Several attempts were made to obtain an in-depth understanding of the PL mechanism by studying the effect of the cluster size, ligand shell and composition. The effects of the ligand concentration (cysteine) during synthesis on the size and subsequent PL properties of obtained monodisperse clusters have been explicitly investigated by Mohammad Reza Ganjali and co-workers.<a title="Select to navigate to references" href="#cit213"><sup><span class="sup_ref">213</span></sup></a> It was found that the increase in ligand concentration results in the formation of weakly emissive and bigger clusters, whereas at moderate concentrations, highly emissive clusters are formed.</span>
        <p class="otherpara">For CuNCs soluble in organic solvents, it was found that the quantum efficiency can be increased by applying cryogenic temperatures<a title="Select to navigate to reference" href="#cit159"><sup><span class="sup_ref">159,161</span></sup></a> and modifying the ligand shell, but it also strongly depends on the crystal packing.<a title="Select to navigate to reference" href="#cit153"><sup><span class="sup_ref">153,165</span></sup></a> Note that the latter affects the electronic transitions within the cluster core.</p>
        <p class="otherpara">Occasionally, dual-emission is also observed in CuNCs, where the low-energy emission is assigned to the intraband HOMO–LUMO transitions within the sp-band, whereas the high-energy emission is attributed to the interband transition from excited states in the sp-band to the d-band.<a title="Select to navigate to reference" href="#cit109"><sup><span class="sup_ref">109,214</span></sup></a> Thus, an in-depth understanding of PL mechanisms and corresponding photophysics can be achieved only by having a complete understanding of size-structure-composition-property relationships in metal nanoclusters.</p>
      
      
        
        <h3 id="sect3347"><span class="b_heading">3.3. Aggregation/assembly-induced emission (AIE)</span></h3>
        <div>
          
          <span id="sect3350"/><span class="c_heading_indent">3.3.1. Ligand/substituent effect. </span>
          <span>Aggregation-induced emission is a novel photophysical phenomenon unearthed back in the 2000s for some organic luminophores.<a title="Select to navigate to references" href="#cit215"><sup><span class="sup_ref">215</span></sup></a> Normally, at high concentrations the luminescence is often weakened or quenched (concentration quenching); however, it was found that silole derivatives become highly emissive upon aggregation.<a title="Select to navigate to references" href="#cit215"><sup><span class="sup_ref">215</span></sup></a> In this respect, aggregation works constructively rather than destructively as stated in several cases. Thereafter, “aggregation-induced emission” has been credited in the field of organic luminophores. This topic, however, will not be elaborated on in this review, and the focus will be directed towards the extension of AIE principles in designing and synthesizing new metal nanoclusters-based AIE luminogens.<a title="Select to navigate to references" href="#cit216"><sup><span class="sup_ref">216</span></sup></a> In this review, efforts are devoted to comprehensively exploiting the prospects of CuNCs as highly fluorescent agents with expressed AIE properties. The main mechanism responsible for the AIE in both organic compounds and nanomaterials is proposed to be the restriction of intramolecular rotations (RIR). Consequently, the RIR blocks the non-radiative pathway and activates the radiative channel, thus increasing the emitting properties of clusters in the aggregated state. The literature is replete with examples of CuNCs with the AIE effect. For instance, V. Venkatesh and co-workers designed a new synthetic strategy for the self-assembly of CuNCs protected with imidazole-based bidentate ligands (abbreviated as MI).<a title="Select to navigate to references" href="#cit217"><sup><span class="sup_ref">217</span></sup></a> The effects of substituents on N1 and N3 nitrogen atoms are critical for the AIE effect. CuNCs@MI showed yellow emission at around 540 nm when excited at 340 nm (QY 11.4%, <a title="Select to navigate to figure" href="#imgfig17">Fig. 17</a>). When substituting the N1 nitrogen atom with a methyl group, bluish-green emitting CuNCs@MMI with an emission maximum at 480 nm was formed (QY 22.4%). A very weak emitting CuNPs@DMMI was formed if both N1 and N3 nitrogen atoms of MI were substituted with methyl groups.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig17"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f17_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f17.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f17.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 17 </b> <span id="fig17"><span class="graphic_title">Schematic representation showing the effect of substituents on the self-assembly and aggregation-induced emission of mercaptoimidazole capped CuNCs. Republished with permission from ref. <a title="Select to navigate to references" href="#cit217">217</a>. Copyright 2019 Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Similarly, when <span class="italic">para</span>-substituted thiophenol (substitutional groups include –NO<small><sub>2</sub></small>, –F, –Cl, –Br, –CH<small><sub>3</sub></small>, and –OCH<small><sub>3</sub></small>) was used for capping, the self-assembly process led to the formation of ribbons with different emission properties in the red region.<a title="Select to navigate to references" href="#cit140"><sup><span class="sup_ref">140</span></sup></a> By changing the electron-donating abilities of the ligand (–F to –OCH<small><sub>3</sub></small>), the PL emission peak can be tuned from 548 to 698 nm (<a title="Select to navigate to figure" href="#imgfig18">Fig. 18</a>). On the contrary, 4-nitrothiophenol leads to the random aggregation of clusters into nanoparticles with no emission enhancement.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig18"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f18_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f18.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f18.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 18 </b> <span id="fig18"><span class="graphic_title">(A) Chemical structures of the capping ligands, and the corresponding images of the sample solutions. PL images with 330 nm excitation and Tyndall scattering phenomena on the irradiation (by a laser pointer) of the self-assembled architectures composed of the CuNCs capped with different ligands. Steady-state absorption (B) and PL emission spectra (C) of the self-assembled architectures composed of the CuNCs capped with different ligands. Republished with permission from ref. <a title="Select to navigate to references" href="#cit140">140</a>. Copyright 2017 Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Jinbin Liu and co-workers demonstrated the synthesis of CuNCs assemblies embedded inside the hydrophobic core of spherical micelles formed from an amphiphilic triblock copolymer template.<a title="Select to navigate to references" href="#cit218"><sup><span class="sup_ref">218</span></sup></a> The number of encapsulated CuNCs inside the core can be controlled by adjusting the block segments of the template. The clusters are assembled into uniform structures and held together through the cross-linking of a multidentate thiol ligand. It was also shown that depending on the structure and branching of the ligand, <span class="italic">i.e.</span> whether it is mono- or multidentate, the formed cluster assemblies displayed different PL properties. Furthermore, the brightest luminescence (absolute quantum yield of 7.3%) was observed when a hexadentate thiol ligand with a star-like structure was used.</p>
          <p class="otherpara">These findings illustrated not only the effect and importance of the protective ligand in the synthesis, but also opened a new pathway for constructing highly fluorescent clusters with the AIE effect.</p>
        </div>
        <div>
          
          <span id="sect3377"/><span class="c_heading_indent">3.3.2. pH effect. </span>
          <span>Another factor that controls and induces AIE in ultrasmall metal clusters is the pH of the buffer solution in which the cluster is dissolved. The conformation and structure of the ligand shell is directly influenced by the pH of the medium. It is known that protein-templated copper nanoclusters are very sensitive to pH variations due to structural changes. This pH-stimuli-responsive property has been applied in many cases to detect pH variations in solution and in living cells by following the corresponding changes in PL (to be discussed later). The pH-guided self-assembly or aggregation of clusters in aqueous solutions induces AIE and makes the clusters highly emissive. For instance, <span class="small_caps">L</span>-cysteine-capped CuNCs formed insoluble macroscopic aggregates with stable red-coloured emission (620 nm, QY 5.4%) at pH 3.0 but became soluble with weak luminescence at pH &lt;1.5 or pH &gt;4.0.<a title="Select to navigate to references" href="#cit219"><sup><span class="sup_ref">219</span></sup></a> The addition of BSA protein or GOx enzyme caused further increases in the PL emission intensity and absolute quantum yield (QY 6.3%) due to the formation of larger aggregates. The pH-responsive PL behaviour is totally reversible, and the PL of the clusters can be quenched or enhanced back and forth without any fatigue and loss in the PL (<a title="Select to navigate to figure" href="#imgfig19">Fig. 19</a>).</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig19"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f19_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f19.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f19.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 19 </b> <span id="fig19"><span class="graphic_title">Schematic diagram of the pH-guided strategy in the fabrication of water-soluble protein/CuNCs hybrid nanostructures. The AIE behaviour of the as-prepared nanocomposite can be regulated by changing the pH of the solution. A strong emission was observed at pH 3.0 due to the aggregation of the cluster, but this further decreased significantly upon the dissolution of the composite under weakly acidic conditions (pH &gt; 3). Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit219">219</a>. Copyright 2017 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Similarly, <span class="small_caps">D</span>-penicillamine-protected CuNCs showed changes in their emission spectra when the pH increased from 4 to 6.2.<a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a> At pH 6.2 the emission intensity dropped 100-fold due to the dissolution of CuNCs lamellar structures/aggregates in aqueous solution and the formation of rather isolated clusters. On the contrary, Yuanyuan Huang <span class="italic">et al.</span> showed that the AIE enhancement of CuNCs aggregates under acidic conditions can stay unaltered when the pH is further increased to 8.<a title="Select to navigate to references" href="#cit220"><sup><span class="sup_ref">220</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect3392"/><span class="c_heading_indent">3.3.3. Solvent effect. </span>
          <span>The extent of aggregation and AIE enhancement of CuNCs can be directly controlled by introducing strongly polar solvents like ethanol or methanol,<a title="Select to navigate to reference" href="#cit18"><sup><span class="sup_ref">18,114</span></sup></a> water-soluble CuNCs or water for hydrophobic CuNCs.<a title="Select to navigate to references" href="#cit141"><sup><span class="sup_ref">141</span></sup></a> The negative charge on the surface of CuNCs (if negatively charged amino acids, peptides, proteins, <span class="italic">etc</span>. are used) and the presence of COOH and OH functional groups in the ligand structure impart water-solubility to the cluster through the formation of a thick hydration layer around the cluster. On increasing the fraction of organic solvent with respect to water, the hydrodynamic radius or solvation layer shrinks, and the clusters attract each other through cuprophilic interactions. Consequently, the intercalation of the ligand layers, as well as the restriction of ligand shell motion, vibrations and rotations in the aggregated/condensed state, leads to changes in the emission dynamics: the non-radiative pathways are deactivated, and radiative pathways are enhanced. That is exactly what happened when weakly emissive Cu<small><sub>34–32</sub></small>(SG)<small><sub>16–13</sub></small> (QY 0.03%) generated a striking orange emission when the volume fraction of ethanol was increased to 90% (36-fold enhancement in emission intensity) (<a title="Select to navigate to figure" href="#imgfig20">Fig. 20A–E</a>).<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a></span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig20"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f20_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f20.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f20.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 20 </b> <span id="fig20"><span class="graphic_title">(A) Digital photographs of Cu<small><sub>34–32</sub></small>(SG)<small><sub>16–13</sub></small> under 365 nm UV illumination (a) at ambient temperature and (b) upon freezing (temperature &lt;0 °C). (B) TEM image of aggregates with scale bar 100 and 50 nm (inset). (C) Corresponding digital photographs with volume fraction (fv) of (a) 0%, (b) 20%, (c) 40%, (d) 60%, (e) 80%, and (f) 90% under daylight conditions and UV (365 nm) illumination. (D) PL spectra of Cu<small><sub>34–32</sub></small>(SG)<small><sub>16–13</sub></small>. (E) Shifting of the emission maxima in water–EtOH mixture with fv = 0–90%. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit17">17</a>. Copyright 2019 American Chemical Society. Fluorescence spectra (F) and absorption spectra (G) of the PEI-capped Cu nanoclusters dispersed in different solvents. Photographs of the PEI-capped Cu nanoclusters dispersed in different solvents under visible light (H) and UV light (I). Panels H and I show (1) water, (2) methanol, (3) ethanol, (4) n-propanol, (5) isopropanol, (6) 1-butanol, (7) ethylene glycol, (8) acetonitrile, (9) DMF, (10) DMSO, (11) THF, (12) 1,4-dioxane, and (13) EGME. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit28">28</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Similar solvent driven AIE enhancement in GSH-CuNCs was also detected by Jingcheng Hao and co-workers.<a title="Select to navigate to references" href="#cit221"><sup><span class="sup_ref">221</span></sup></a> The addition of methanol, ethanol, <span class="italic">n</span>-propanol, 2-propanol, DMF and DMSO resulted in the formation of milky solution under daylight conditions and strong fluorescence under UV-irradiation with the order of PL intensity being <span class="italic">I</span><small><sub>DMF</sub></small> &gt; <span class="italic">I</span><small><sub>methanol</sub></small> &gt; <span class="italic">I</span><small><sub>DMSO</sub></small> &gt; <span class="italic">I</span><small><sub>ethanol</sub></small> &gt; <span class="italic">I</span><small><sub>2-propanol</sub></small> &gt; <span class="italic">I</span><small><sub><span class="italic">n</span>-propanol</sub></small>. The increase in the organic solvent content leads to the reorganization of surface ligands into amphiphilic structures to minimize the contact between the organic solvent and ligand molecules at the water/organic solvent interface.</p>
          <p class="otherpara">With the accumulation of amphiphilic structures at the water/organic solvent interface, highly ordered self-assemblies of CuNCs are formed (spherical particles in the size range of 100–500 nm).</p>
          <p class="otherpara">Yu Ling <span class="italic">et al.</span> demonstrated the solvent effect in PEI-CuNCs by adding alcohols, DMF, DMSO, THF, 1,4-dioxane, EGME and acetonitrile (<a title="Select to navigate to figure" href="#imgfig20">Fig. 20F–I</a>).<a title="Select to navigate to references" href="#cit28"><sup><span class="sup_ref">28</span></sup></a> Except for the first two solvents, strong solvent-dependent AIE enhancement was observed. Furthermore, the highest enhancement and blue shift of the emission peak was detected upon the addition of 60% THF, which was explained by the formation of hydrogen bonds and the collapse of PEI branches because of insolubility in THF as well as conformational changes in the polymer network. Altogether, this led to the aggregation of the clusters and thus, enhancement of AIE.</p>
        </div>
        <div>
          
          <span id="sect3432"/><span class="c_heading_indent">3.3.4. Synergistic effect of metal ions. </span>
          <span>The AIE of CuNCs can also be improved and enhanced by several orders of magnitude in the presence of other metals, which triggers the aggregation of the clusters in the solution. For instance, the synergistic effect of dual metals, <span class="italic">e.g.</span> Pb<small><sup>2+</sup></small> and Zr<small><sup>4+</sup></small>, has been shown to generate bright luminescence in GSH-CuNCs.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a> Alone, the metal had a lesser effect; however, when combined they facilitated the formation of aggregates in aqueous solution. The addition of 1.1 mg L<small><sup>−1</sup></small> of Al<small><sup>3+</sup></small> (ref. <a title="Select to navigate to references" href="#cit110">110</a> and <a title="Select to navigate to reference" href="#cit222">222–224</a>) or 1 mM Zn<small><sup>2+</sup></small> (ref. <a title="Select to navigate to references" href="#cit110">110</a>, <a title="Select to navigate to references" href="#cit225">225</a> and <a title="Select to navigate to references" href="#cit226">226</a>) cations were also found to trigger the aggregation of CuNCs and greatly increase the absolute quantum yield of clusters up to 27% and 6.6%, respectively (<a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a>). Moreover, benefiting from the AIE effect, the luminescence light-up detection of Zn<small><sup>2+</sup></small> levels in living cells has been successfully demonstrated (<a title="Select to navigate to figure" href="#imgfig21">Fig. 21A</a>).<a title="Select to navigate to references" href="#cit225"><sup><span class="sup_ref">225</span></sup></a> Xifan Mei and co-workers showed that the addition of Zn<small><sup>2+</sup></small> into the cluster solution leads to its binding on the cluster surface through the metal complexation with the OH groups of the ligand (<a title="Select to navigate to figure" href="#imgfig21">Fig. 21B</a>).<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a> The metal coordination leads to the formation of a Zn(OH)<small><sub><span class="italic">x</span></sub></small> layer around the cluster. With the assistance of Al<small><sup>3+</sup></small>, the luminescence of the GSH-CuNCs has been greatly enhanced due to the formation of aggregates in the form of dots with the size of 100–200 nm.<a title="Select to navigate to references" href="#cit227"><sup><span class="sup_ref">227</span></sup></a></span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig21"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f21_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f21.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f21.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 21 </b> <span id="fig21"><span class="graphic_title">(A) Schematic illustration of the preparation of GSH-capped CuNCs and the aggregation-induced PL enhancement of CuNCs in the presence of Zn<small><sup>2+</sup></small>. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit225">225</a>. Copyright 2017 Elsevier. (B) Scheme for the mechanism of the aggregation-induced fluorescence enhancement for CuNCs in the presence of Zn<small><sup>2+</sup></small>. Republished with permission from ref. <a title="Select to navigate to references" href="#cit110">110</a>. Copyright 2016 Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">From a mechanistic point of view, the coordination of metal cations with the ligand shell introduced instability and led to the instant formation of large aggregates.</p>
        </div>
        <div>
          
          <span id="sect3466"/><span class="c_heading_indent">3.3.5. Self-assembly of clusters and the contribution of metal defects. </span>
          <span>The understanding of AIE mechanisms is of great importance to portray their use in illumination and display applications. Though the real mechanism of AIE is not yet clear, RIR has been accepted as a mechanism responsible for AIE, at least to some extent when inhomogeneous aggregates are involved. However, when nanoclusters (herein CuNCs) are brought together into highly ordered, uniform, and structured assemblies/architectures, the PL enhancement is due to the assembly-induced AIE effect.<a title="Select to navigate to references" href="#cit228"><sup><span class="sup_ref">228</span></sup></a> For instance, Shuo Wang and co-workers showed that nanoribbons of CuNCs can be directly formed from the reduction of the copper ions in the presence of 2,3,5,6- tetrafluorothiophenol (TFTP) as both the reducing agent and the protecting ligand at 20 °C.<a title="Select to navigate to references" href="#cit229"><sup><span class="sup_ref">229</span></sup></a> TEM imaging revealed highly ordered nanoribbons with an approximate width of about 20 nm, a length of about 400–800 nm, and a thickness of about 15 nm, emitting strong yellow luminescence upon UV irradiation (QY 43%).</span>
          <p class="otherpara">Hao Zhang and co-workers showed that the spherical dodecanethiol (DT)-capped CuNCs with size of 1.9 nm self-assembled into ribbons (average width about 50–200 nm, length about 1–2 μm, and thickness about 13.5 nm) at 128 °C (<a title="Select to navigate to figure" href="#imgfig22">Fig. 22A and B</a>).<a title="Select to navigate to references" href="#cit139"><sup><span class="sup_ref">139</span></sup></a> The emission of CuNCs-based ribbons was greatly enhanced (absolute quantum yield of 6.5%, <a title="Select to navigate to figure" href="#imgfig22">Fig. 22C</a>). Furthermore, the emission colour can be tuned from yellow to deep blue/green just by changing the annealing temperature during the self-assembly process (thermochromic effect) (<a title="Select to navigate to figure" href="#imgfig22">Fig. 22D</a>).</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig22"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f22_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f22.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f22.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 22 </b> <span id="fig22"><span class="graphic_title">(A) TEM image of the ribbons from CuNCs self-assembly. (B) HRTEM image of the ribbons. Inset: Fourier transform image. (C) Steady-state absorption (black) and emission (red) spectra of the ribbons in chloroform. Inset: The fluorescent image with 365 nm excitation. (D) Spectra of CuNCs assemblies respectively prepared at 20, 60, 80, 110, and 120 °C. Inset: Sample excited at 365 nm at different temperatures. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit139">139</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The AIE enhancement in ribbons made of individual CuNCs is (i) due to the changes in ligand/ligand, ligand/metal, and metal/metal (cuprophilic Cu(<span class="small_caps">I</span>)⋯Cu(<span class="small_caps">I</span>)) interactions, which drastically increases the excited state relaxation dynamics. (ii) The restriction of the intramolecular vibration and rotation of capping ligands reduces the nonradiative relaxation of excited states.</p>
          <p class="otherpara">The same group also studied nanosheets of clusters and showed that metal defects on the surface of highly structured assemblies influenced the PL properties (see <a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a>).<a title="Select to navigate to references" href="#cit230"><sup><span class="sup_ref">230</span></sup></a> In short, the dangling bonds and the formation of unsaturated coordination sites on the nanosheet surface significantly altered the photophysical relaxation dynamics. When dodecanethiol (DT)-capped CuNCs assembled into a nanosheet in the presence of alcohols (ethanol and methanol), they featured pronounced AIE properties (QY 15.4%). The Cu(<span class="small_caps">I</span>)/Cu(0) ratio at the surface was found to be 1/0.32, and the presence of Cu(<span class="small_caps">I</span>) species facilitated the radiative relaxation. Moreover, at room temperature, mainly metal-defects at the surface were responsible for emission (T2 state), whereas upon lowering the temperature, the contribution of defects was negligible, and the ligand-to-metal–metal charge transfer (LMMCT)-determined triplet states (T1 state) were dominant (<a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a>). Thus, the presence of a large number of metal defects on the surface changed the LMMCT behaviour by providing a lower energy level.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig23"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f23_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f23.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f23.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 23 </b> <span id="fig23"><span class="graphic_title">Low (A) and high (B) magnification TEM images of CuNCs self-assembled nanosheets. (C) Top- and side-view cartoon image of the nanosheets. (D) Steady-state absorption (solid), excitation (dash), and emission (dot) spectra of the nanosheets in chloroform. Inset: Sample excited at 365 nm. (E) Schematic diagram of the excited state relaxation dynamics of the nanosheets. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit230">230</a>. Copyright 2017 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The discussions made in section 3.3 demonstrated the importance of the AIE mechanism/phenomenon for strong PL properties in CuNCs. Nonetheless, it is by far not the only important factor to consider since not all the CuNCs are explicit AIEgens and more studies on well-defined systems are needed to better understand the PL properties of clusters, which is important for applications.</p>
        </div>
      
      
        
        <h3 id="sect3493"><span class="b_heading">3.4. Chirality in CuNCs</span></h3>
        <span>Chirality is a symmetry property of matter and plays an important role in biology and pharmacological sciences. Since the first discovery of optical activity in glutathione-protected gold nanoclusters by Whetten and co-workers in the 2000s,<a title="Select to navigate to references" href="#cit231"><sup><span class="sup_ref">231</span></sup></a> several groups have, both experimentally and theoretically, attempted to understand the origin of chirality in small nanoclusters containing a few tens of atoms. Several theoretical calculations were conducted to individually analyse the core, the shell and overall structure to gain insight into the origin of the strong chiral responses in metal-based transitions. To that end, several mechanisms have been proposed to explain the origin of the optical activity of various passivated clusters: an intrinsically chiral metal core, dissymmetric field model and chiral footprint.<a title="Select to navigate to reference" href="#cit232"><sup><span class="sup_ref">232,233</span></sup></a> The intrinsically chiral metal core or ligand-induced chirality in metal nanoclusters, including Au<small><sub>38</sub></small>(SR)<small><sub>24</sub></small><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><a title="Select to navigate to reference" href="#cit234"><sup><span class="sup_ref">234,235</span></sup></a> and Au<small><sub>102</sub></small>(SR)<small><sub>44</sub></small>,<a title="Select to navigate to references" href="#cit236"><sup><span class="sup_ref">236</span></sup></a> has been considered as an origin of chirality, mainly due to strong Au–S bond formation and staple motif arrangements on the surface. The dissymmetric field model states that the chirality in metal nanoclusters is due to the adsorption of chiral molecules on achiral metallic cores. The chiral molecules create a dissymmetric field sensed by the metal core.<a title="Select to navigate to references" href="#cit237"><sup><span class="sup_ref">237</span></sup></a> Another model, called the “chiral footprint”, implies that when chiral molecules are adsorbed on metallic surfaces, the local environment becomes chiral.<a title="Select to navigate to references" href="#cit238"><sup><span class="sup_ref">238</span></sup></a> The induction of chirality in the cluster <span class="italic">via</span> chiral ligands has been considered as a relatively effective and controllable approach for chiral nanomaterial synthesis. In this regard, Shuang-Quan Zang and co-workers reported the synthesis of the atomically precise [Cu<small><sub>14</sub></small>(<span class="italic">R</span>/<span class="italic">S</span>-DPM)<small><sub>8</sub></small>](PF<small><sub>6</sub></small>)<small><sub>6</sub></small> cluster with strong CD and CPL properties.<a title="Select to navigate to references" href="#cit173"><sup><span class="sup_ref">173</span></sup></a> In the CD spectra, the mirror-image signals in the range of 300–460 nm contained additional signals different from the pure ligand, indicating the chirality transfer from the ligand to the core. This is the first report on CPL-active atomically precise CuNCs. More intriguingly, in solution, the clusters were non-emissive, whereas they became extremely red-emissive in the crystalline or solid phase (<a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a>) and they showed AIE enhancement.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig24"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f24_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f24.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f24.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 24 </b> <span id="fig24"><span class="graphic_title">(A) Photographs of fluorescent <span class="italic">R</span>-Cu<small><sub>14</sub></small> in solutions with different fractions of <span class="italic">n</span>-hexane. (B) UV/Vis absorption spectra, (C) emission spectra, (D) CD spectra, and (E) CPL spectra of <span class="italic">R</span>/<span class="italic">S</span>-Cu<small><sub>14</sub></small> in DCM/<span class="italic">n</span>-hexane mixtures with different (fv) values. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit173">173</a>. Copyright 2019 Wiley-VCH Verlag GmbH &amp; Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">The results were interpreted within the frame of AIE. The restriction of intramolecular rotations and vibrations due to the electrostatic interactions, the C–H⋯π interactions and hydrogen-bonding interactions, activate the radiative transitions in the aggregated state and in the crystal. Thus, the AIE enhancement originates from the restricted intramolecular rotation (RIR) mechanism, which triggers strong PL and CPL properties.</p>
      
    
    
      
      <h2 id="sect3527"><span class="a_heading">4. Applications of CuNCs</span></h2>
      
        
        <h3 id="sect3530"><span class="b_heading">4.1. Applications in sensing</span></h3>
        <span>The advantages of ultrasmall and atomically precise clusters with respect to their larger counterparts include the following: (i) the ability to tune the size-dependent properties by carefully choosing the protective ligand and/or the (bio) template; (ii) clusters are more biocompatible and environmentally friendly, some are non-toxic, however, some are quite toxic; (iii) some clusters show size-dependent and relatively strong photoluminescence. These characteristics, particularly the photoluminescence, make CuNCs attractive for applications in sensing. The number of yearly published CuNCs being applied as fluorescent probes is increasing exponentially due to the large demand for analytical tools in modern society, particularly in medicine and environmental control. <a title="Select to navigate to table" href="#tab2">Table 2</a> summarizes the recently published papers on the application of CuNCs as a new nanomaterial-based detection tool or bioassay. The easily scalable synthetic procedures facilitate real-life applications.</span>
        
<div class="table_caption"><b>Table 2</b> <span id="tab2">The cited papers are listed according to their “target” application</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="72.96pt"/><col align="left" width="60.43pt"/><col align="left" width="49.79pt"/><col align="left" width="28.76pt"/><col align="left" width="39.21pt"/><col align="left" width="67.38pt"/><col align="left" width="72.26pt"/><col align="left" width="105.83pt"/><col align="left" width="25.62pt"/></colgroup>
              
              
              
              
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Template/ligand</th>
                  <th align="left" valign="bottom" class="border_black">Cluster size (<span class="italic">d</span> nm)</th>
                  <th align="left" valign="bottom" class="border_black">Ex/Em Wavelength (nm)</th>
                  <th align="left" valign="bottom" class="border_black">QY %</th>
                  <th align="left" valign="bottom" class="border_black">Read out</th>
                  <th align="left" valign="bottom" class="border_black">Target</th>
                  <th align="left" valign="bottom" class="border_black">LOD</th>
                  <th align="left" valign="bottom" class="border_black">Linear response</th>
                  <th align="left" valign="bottom" class="border_black">Ref.</th>
                </tr>
              </thead>
              <tfoot align="left">
                <tr align="left">
                  <th colspan="9">Note, that in the third row of <a title="Select to navigate to table" href="#tab2">Table 2</a> the notations &lt;′&gt; and &lt;′′&gt; refer to the first and second emission peaks. In some cases, the excitation wavelength is not mentioned in the article, thus, a notation was used.</th>
                </tr>
              </tfoot>
              <tbody align="left">
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Metal cation detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">1.5 ± 0.4 nm</td>
                  <td valign="top">320/420</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">4.7 pM</td>
                  <td valign="top">10 pM–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit56">56</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cysteine</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">375/480</td>
                  <td valign="top">14.3</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">24 nM</td>
                  <td valign="top">0.1 μM–1 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit121">121</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">375/440</td>
                  <td valign="top">3.08</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">22 nM</td>
                  <td valign="top">40 nM–60 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit128">128</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.4 ± 0.6 nm</td>
                  <td valign="top">360/445</td>
                  <td valign="top">10.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">3.3 nM</td>
                  <td valign="top">0.01–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit239">239</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">&lt;5 nm</td>
                  <td valign="top">340/650</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">16 pM</td>
                  <td valign="top">50 pM–500 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit240">240</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Curcuma roots</td>
                  <td valign="top">2.3 ± 1.2 nm</td>
                  <td valign="top">365/440</td>
                  <td valign="top">7.2</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">0.12 nM</td>
                  <td valign="top">0.0005–25 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit241">241</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">MT</td>
                  <td rowspan="2" valign="top">&lt;5 nm</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Turn on</td>
                  <td rowspan="2" valign="top">Hg<small><sup>2+</sup></small></td>
                  <td rowspan="2" valign="top">43.8 nM</td>
                  <td valign="top">97 nM–2.3 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit242">242</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">3.1–15.6 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">570/600</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">4 pM</td>
                  <td valign="top">0.04–8 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit243">243</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PATP</td>
                  <td valign="top">2.16 ± 0.36 nm</td>
                  <td valign="top">365/602</td>
                  <td valign="top">24.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hg<small><sup>2+</sup></small></td>
                  <td valign="top">10 ppt</td>
                  <td valign="top">25 ppt–25 ppb</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit244">244</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">360/430</td>
                  <td valign="top">2.1</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">340 nM</td>
                  <td valign="top">0.5–1000 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit30">30</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TA</td>
                  <td valign="top">2.2 ± 0.5 nm</td>
                  <td valign="top">360/430</td>
                  <td valign="top">14</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">10 nM</td>
                  <td valign="top">10 nM–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit145">145</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Dopamine</td>
                  <td valign="top">8 nm</td>
                  <td valign="top">320/390</td>
                  <td valign="top">9.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">1.2 μM</td>
                  <td valign="top">5–300 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit148">148</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.2 nm</td>
                  <td valign="top">340/430</td>
                  <td valign="top">6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">25 nM</td>
                  <td valign="top">0.01–1 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit135">135</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cysteamine</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">385/467</td>
                  <td valign="top">16</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">423 nM</td>
                  <td valign="top">0–1000 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit138">138</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">1.7 ± 0.3 nm</td>
                  <td valign="top">365/422</td>
                  <td valign="top">8.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">0.3 μM</td>
                  <td valign="top">1–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit245">245</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CCCDL</td>
                  <td valign="top">2 ± 0.53 nm</td>
                  <td valign="top">365/425</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">20 nM</td>
                  <td valign="top">0.05–30 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit246">246</a>
                  </td>
                </tr>
                <tr align="left">
                  <td align="left" valign="top">—</td>
                  <td valign="top">3.5 nm</td>
                  <td valign="top">350/430</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">2 μM</td>
                  <td valign="top">0–30 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit247">247</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">ACE</td>
                  <td valign="top">30 nm</td>
                  <td valign="top">280/340</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">23.4 nM</td>
                  <td valign="top">0.2–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit248">248</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Histidine</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">385/495</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fe<small><sup>3+</sup></small></td>
                  <td valign="top">82 nM</td>
                  <td valign="top">0.1–20 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit249">249</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">1 nm</td>
                  <td valign="top">340/420</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cu<small><sup>2+</sup></small></td>
                  <td valign="top">1 nM</td>
                  <td valign="top">0.02–34 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit52">52</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">D</span>-Penicillamine</td>
                  <td valign="top">1.2 ± 0.4 nm</td>
                  <td valign="top">391/673</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Cu<small><sup>2+</sup></small></td>
                  <td valign="top">0.3 ppm</td>
                  <td valign="top">0.95–6.35 ppm</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit115">115</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">1.7 nm</td>
                  <td valign="top">380/495</td>
                  <td valign="top">7.9</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cu<small><sup>2+</sup></small></td>
                  <td valign="top">8.9 nM</td>
                  <td valign="top">22 nM–8.8 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit250">250</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/615</td>
                  <td valign="top">11.2</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Cu<small><sup>2+</sup></small></td>
                  <td valign="top">5.6 μM</td>
                  <td valign="top">15–35 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit251">251</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cytidine</td>
                  <td valign="top">1.5 ± 0.5 nm</td>
                  <td valign="top">300/380</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Cu<small><sup>2+</sup></small></td>
                  <td valign="top">0.032 μM</td>
                  <td valign="top">0.05–2 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit252">252</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">4 nm</td>
                  <td valign="top">330/615</td>
                  <td valign="top">1.2</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Cu<small><sup>2+</sup></small></td>
                  <td valign="top">0.17 μM</td>
                  <td valign="top">0.25–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit253">253</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">365/495</td>
                  <td valign="top">2.63</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cu<small><sup>2+</sup></small></td>
                  <td valign="top">0.12 μM</td>
                  <td valign="top">0.2–8 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit254">254</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2.8 ± 0.5 nm</td>
                  <td valign="top">325/410</td>
                  <td valign="top">15</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Pb<small><sup>2+</sup></small></td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit57">57</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Random dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/585</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Pb<small><sup>2+</sup></small></td>
                  <td valign="top">5 nM</td>
                  <td valign="top">5–100 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit72">72</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TBAN</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">312/408</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Pb<small><sup>2+</sup></small></td>
                  <td valign="top">4.9 μM</td>
                  <td valign="top">48 nM–77 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit95">95</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.2 ± 0.2 nm</td>
                  <td valign="top">365/606</td>
                  <td valign="top">5.3</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Pb<small><sup>2+</sup></small></td>
                  <td valign="top">1 nM</td>
                  <td valign="top">1–160 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit137">137</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/615</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Pb<small><sup>2+</sup></small></td>
                  <td valign="top">0.4 nM</td>
                  <td valign="top">1–500 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit255">255</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">MT</td>
                  <td valign="top">&lt;5 nm</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Pb<small><sup>2+</sup></small></td>
                  <td valign="top">142 nM</td>
                  <td valign="top">0.7–96 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit242">242</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">GSH</td>
                  <td rowspan="2" valign="top">2.3 nm</td>
                  <td rowspan="2" valign="top">340/585</td>
                  <td rowspan="2" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Pb<small><sup>2+</sup></small></td>
                  <td valign="top">30 μM</td>
                  <td valign="top">40–105 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit127">127</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn on</td>
                  <td valign="top">Zr<small><sup>4+</sup></small></td>
                  <td valign="top">40 μM</td>
                  <td valign="top">60–250 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">1.5 nm</td>
                  <td valign="top">370/435</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Co<small><sub>2</sub></small><small><sup>+</sup></small></td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit59">59</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">1.1 ± 0.4 nm</td>
                  <td valign="top">488/590</td>
                  <td valign="top">20</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Ca<small><sup>2+</sup></small></td>
                  <td valign="top">220 nM</td>
                  <td valign="top">2–350 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit256">256</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.8 ± 0.5 nm</td>
                  <td valign="top">354/561</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Mn<small><sup>2+</sup></small></td>
                  <td valign="top">10 μM</td>
                  <td valign="top">100–250 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit257">257</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DHLA</td>
                  <td valign="top">1.6 nm</td>
                  <td valign="top">362/627</td>
                  <td valign="top">2.8</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Zn<small><sup>2+</sup></small></td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">50–400 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit110">110</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">340/600</td>
                  <td valign="top">6.2</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Zn<small><sup>2+</sup></small></td>
                  <td valign="top">1.17 μM</td>
                  <td valign="top">4.68–2240 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit225">225</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">&lt;3 nm</td>
                  <td valign="top">440/650</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Zn<small><sup>2+</sup></small></td>
                  <td valign="top">15 nM</td>
                  <td valign="top">0.66–5.3 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit258">258</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cysteine</td>
                  <td valign="top">2.8 ± 0.6 nm</td>
                  <td valign="top">387/615</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Zn<small><sup>2+</sup></small></td>
                  <td valign="top">1 μM</td>
                  <td valign="top">1.5–750 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit259">259</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cysteine</td>
                  <td valign="top">3.5 nm</td>
                  <td valign="top">360/490</td>
                  <td valign="top">5.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cr<small><sup>6+</sup></small></td>
                  <td valign="top">43 nM</td>
                  <td valign="top">0.05–60 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit116">116</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TSA/BSA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">350/430</td>
                  <td valign="top">1.2</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cr<small><sup>6+</sup></small></td>
                  <td valign="top">3.54 nM</td>
                  <td valign="top">50–400 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit260">260</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Organic molecules</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">468/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cr<small><sup>3+</sup></small></td>
                  <td valign="top">3 μM</td>
                  <td valign="top">0–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit261">261</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">340/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cr<small><sup>3+</sup></small></td>
                  <td valign="top">0.03 μM</td>
                  <td valign="top">0–70 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit262">262</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DTT</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">360/590</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Al<small><sup>3+</sup></small></td>
                  <td valign="top">0.01 μM</td>
                  <td valign="top">0.01–7 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit263">263</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cysteamine</td>
                  <td valign="top">2.3 ± 0.5 nm</td>
                  <td valign="top">330/430</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Al<small><sup>3+</sup></small></td>
                  <td valign="top">26.7 nM</td>
                  <td valign="top">1–7 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit264">264</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/640</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Al<small><sup>3+</sup></small></td>
                  <td valign="top">62 nM</td>
                  <td valign="top">0.1–30 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit265">265</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Anion detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PSS</td>
                  <td valign="top">1.7 nm</td>
                  <td valign="top">325/665</td>
                  <td valign="top">8</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">650 nM</td>
                  <td valign="top">1–20 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit35">35</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/585</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">80 nM</td>
                  <td valign="top">0.2–2 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit82">82</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Penicillamine</td>
                  <td valign="top">2.7 ± 0.2 nm</td>
                  <td valign="top">326/580</td>
                  <td valign="top">2</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">500 nM</td>
                  <td valign="top">1–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit113">113</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cysteine</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">382/460</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">42 nM</td>
                  <td valign="top">0.2–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit117">117</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Yeast extract</td>
                  <td valign="top">2–4 nm</td>
                  <td valign="top">370/450</td>
                  <td valign="top">9.3</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">10 nM</td>
                  <td valign="top">0.02–0.8 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit266">266</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CEW</td>
                  <td valign="top">2.6 nm</td>
                  <td valign="top">465/661</td>
                  <td valign="top">6.2</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit226">226</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">365/622′′</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">4.3 nM</td>
                  <td valign="top">26–128 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit267">267</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">&lt;5 nm</td>
                  <td valign="top">360/607</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">0.5 μM</td>
                  <td valign="top">0.5–20 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit268">268</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Silk fibroin</td>
                  <td valign="top">2.8 ± 0.5 nm</td>
                  <td valign="top">326/422′′</td>
                  <td valign="top">4.9</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">S<small><sup>2−</sup></small></td>
                  <td valign="top">0.286 μM</td>
                  <td valign="top">5–110 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit269">269</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">270/430</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">I<small><sup>−</sup></small></td>
                  <td valign="top">100 nM</td>
                  <td valign="top">0–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit29">29</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cysteamine</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">385/467</td>
                  <td valign="top">16</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">I<small><sup>−</sup></small></td>
                  <td valign="top">2.02 μM</td>
                  <td valign="top">0–10 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit138">138</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TA</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">380/460</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">I<small><sup>−</sup></small></td>
                  <td valign="top">18 nM</td>
                  <td valign="top">20–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit270">270</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">ssDNA</td>
                  <td rowspan="2" valign="top">2 nm</td>
                  <td rowspan="2" valign="top">345/637</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td rowspan="2" valign="top">I<small><sup>−</sup></small></td>
                  <td rowspan="2" valign="top">15 nM</td>
                  <td valign="top">0.05–40 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit271">271</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">40–80 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">PVP</td>
                  <td valign="top">3–5 nm</td>
                  <td valign="top">365/421</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">I<small><sup>−</sup></small></td>
                  <td valign="top">1 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">10–70 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit272">272</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PVP</td>
                  <td valign="top">3.3 ± 0.4 nm</td>
                  <td valign="top">350/430</td>
                  <td valign="top">13</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">ClO<small><sup>−</sup></small></td>
                  <td valign="top">0.1 μM</td>
                  <td valign="top">1–30 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit33">33</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PVP</td>
                  <td valign="top">0.96–5.11 nm</td>
                  <td valign="top">340/416</td>
                  <td valign="top">9.4</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">ClO<small><sup>−</sup></small></td>
                  <td valign="top">19 nM</td>
                  <td valign="top">0.3–2.5 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit34">34</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">TSA</td>
                  <td rowspan="2" valign="top">3.7 ± 0.5 nm</td>
                  <td rowspan="2" valign="top">338/420</td>
                  <td rowspan="2" valign="top">13.2</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">CN<small><sup>−</sup></small></td>
                  <td valign="top">5 nM</td>
                  <td valign="top">0.01–1 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit273">273</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn off</td>
                  <td valign="top">NO<small><sub>2</sub></small><small><sup>−</sup></small></td>
                  <td valign="top">5 μM</td>
                  <td valign="top">15–50 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cys</td>
                  <td valign="top">4 nm</td>
                  <td valign="top">360/490</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">NO<small><sub>2</sub></small><small><sup>−</sup></small></td>
                  <td valign="top">95.4 nM</td>
                  <td valign="top">1–80 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit274">274</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">AA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">370/454</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">NO<small><sub>2</sub></small><small><sup>−</sup></small></td>
                  <td valign="top">3.6 nM</td>
                  <td valign="top">0.0125–125 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit275">275</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">360/430</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">PO<small><sub>4</sub></small><small><sup>3−</sup></small></td>
                  <td valign="top">9.6 nM</td>
                  <td valign="top">0.07–80 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit144">144</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PDA</td>
                  <td valign="top">53.7 ± 4 nm</td>
                  <td valign="top">340/580</td>
                  <td valign="top">2.54</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">PO<small><sub>4</sub></small><small><sup>3−</sup></small></td>
                  <td valign="top">1.5 nM</td>
                  <td valign="top">0.003–70 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit276">276</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">340/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">P<small><sub>2</sub></small>O<small><sub>7</sub></small><small><sup>4−</sup></small></td>
                  <td valign="top">5 nM</td>
                  <td valign="top">0–70 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit262">262</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/640</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">F<small><sup>−</sup></small></td>
                  <td valign="top">1 μM</td>
                  <td valign="top">2–150 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit265">265</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Hydrogen peroxide and Glucose detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Papain</td>
                  <td valign="top">2.3 ± 0.7 nm</td>
                  <td valign="top">370/620</td>
                  <td valign="top">14.3</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.2 μM</td>
                  <td valign="top">1–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit54">54</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">D</span>-Penicillamine</td>
                  <td valign="top">1 ± 0.4 nm</td>
                  <td valign="top">345/640′′</td>
                  <td valign="top">14.1</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.01 mM</td>
                  <td valign="top">0.05–2 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit114">114</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DHLA</td>
                  <td valign="top">1.8 nm</td>
                  <td valign="top">342/590</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.3 μM</td>
                  <td valign="top">1–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit111">111</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">MSA</td>
                  <td valign="top">4.5 nm</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.001 mM</td>
                  <td valign="top">0.01–1 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit142">142</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">BSA</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">335/410</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">—</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">10 μM</td>
                  <td valign="top">10 μM–1mM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit58">58</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Glucose</td>
                  <td valign="top">100 μM</td>
                  <td valign="top">100 μM–2mM</td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">PEI</td>
                  <td rowspan="2" valign="top">1.8 nm</td>
                  <td rowspan="2" valign="top">355/480</td>
                  <td rowspan="2" valign="top">3.8</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.4 μM</td>
                  <td valign="top">0.5–10 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit26">26</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Glucose</td>
                  <td valign="top">8 μM</td>
                  <td valign="top">10–100 μM</td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">Poly(thymine)</td>
                  <td rowspan="2" valign="top">1.7–5.1 nm</td>
                  <td rowspan="2" valign="top">340/615</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.55 μM</td>
                  <td valign="top">0.55–110 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit70">70</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Glucose</td>
                  <td valign="top">0.05 mM</td>
                  <td valign="top">0.05–1.6 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit73">73</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">dsDNA</td>
                  <td rowspan="2" valign="top">3–5 nm</td>
                  <td rowspan="2" valign="top">340/570</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">3 nM</td>
                  <td valign="top">10 nM–2 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit277">277</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Glucose</td>
                  <td valign="top">12 nM</td>
                  <td valign="top">50 nM–100 μM</td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">GSH</td>
                  <td rowspan="2" valign="top">2.3 nm</td>
                  <td rowspan="2" valign="top">340/585</td>
                  <td rowspan="2" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.6 μM</td>
                  <td valign="top">1–60 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit127">127</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn off</td>
                  <td valign="top">Glucose</td>
                  <td valign="top">0.37 μM</td>
                  <td valign="top">0.8–50 μM</td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">β-CD</td>
                  <td rowspan="2" valign="top">2 nm</td>
                  <td rowspan="2" valign="top">360/450</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">—</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small></td>
                  <td valign="top">0.2 μM</td>
                  <td valign="top">0.02–10 mM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit278">278</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Glucose</td>
                  <td valign="top">0.4 μM</td>
                  <td valign="top">0.04–20 mM</td>
                </tr>
                <tr align="left">
                  <td rowspan="4" valign="top">DNA</td>
                  <td rowspan="4" valign="top">—</td>
                  <td rowspan="4" valign="top">340/640</td>
                  <td rowspan="4" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small> (color.)</td>
                  <td valign="top">2.92 μM</td>
                  <td valign="top">5–2000 μM</td>
                  <td rowspan="4" valign="top">
                    <a title="Select to navigate to references" href="#cit279">279</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn off</td>
                  <td valign="top">H<small><sub>2</sub></small>O<small><sub>2</sub></small> (fluor.)</td>
                  <td valign="top">0.266 μM</td>
                  <td valign="top">0.5–2000 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn on</td>
                  <td valign="top">Glucose (color.)</td>
                  <td valign="top">15 μM</td>
                  <td valign="top">50–3000 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn off</td>
                  <td valign="top">Glucose (fluor.)</td>
                  <td valign="top">0.5 μM</td>
                  <td valign="top">1–500 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cysteine/GOx</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">330/628</td>
                  <td valign="top">5.4</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Glucose</td>
                  <td valign="top">1.5 μM</td>
                  <td valign="top">5–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit219">219</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">1.5 nm</td>
                  <td valign="top">358/625</td>
                  <td valign="top">2.1</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Glucose</td>
                  <td valign="top">32 μM</td>
                  <td valign="top">0.1–2 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit280">280</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Small molecule detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TA</td>
                  <td valign="top">3.75 ± 0.71 nm</td>
                  <td valign="top">355/430</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Ascorbic acid</td>
                  <td valign="top">0.11 μM</td>
                  <td valign="top">0.5–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit143">143</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2.7 ± 0.8 nm</td>
                  <td valign="top">330/407</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Kojic acid</td>
                  <td valign="top">0.07 μM</td>
                  <td valign="top">0.2–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit47">47</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/610</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Humic acid</td>
                  <td valign="top">0.4 mg L<small><sup>−1</sup></small></td>
                  <td valign="top">0–8 mg L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit281">281</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">325/400</td>
                  <td valign="top">15</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Picric acid</td>
                  <td valign="top">120 nM</td>
                  <td valign="top">0.8–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit51">51</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.36 ± 0.67 nm</td>
                  <td valign="top">360/625</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Picric acid</td>
                  <td valign="top">2.74 μM</td>
                  <td valign="top">9.9–43 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit282">282</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/627</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Picric acid</td>
                  <td valign="top">0.03 μM</td>
                  <td valign="top">0.1–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit283">283</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cysteamine</td>
                  <td valign="top">3–5 nm</td>
                  <td valign="top">365/480</td>
                  <td valign="top">2.3</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Picric acid</td>
                  <td valign="top">139 nM</td>
                  <td valign="top">1–80 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit284">284</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">AA</td>
                  <td valign="top">4 nm</td>
                  <td valign="top">350/430</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Picric acid</td>
                  <td valign="top">0.98 μM</td>
                  <td valign="top">2–40 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit285">285</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cys</td>
                  <td valign="top">2.2 nm</td>
                  <td valign="top">370/494</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Picric acid</td>
                  <td valign="top">0.19 μM</td>
                  <td valign="top">2.5–25 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit286">286</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cys</td>
                  <td valign="top">4 nm</td>
                  <td valign="top">360/490</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Folic acid</td>
                  <td valign="top">69.8 nM</td>
                  <td valign="top">0.1–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit274">274</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Ovalbumin</td>
                  <td valign="top">1.8 nm</td>
                  <td valign="top">348/625</td>
                  <td valign="top">3.95</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Folic acid</td>
                  <td valign="top">0.18 μM</td>
                  <td valign="top">0.5–200 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit287">287</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">355/480</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">TNT</td>
                  <td valign="top">14 pM</td>
                  <td valign="top">0–8 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit288">288</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cys</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">375/525</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">
                    <span class="italic">m</span>-DNB</td>
                  <td valign="top">0.13 μM</td>
                  <td valign="top">99 nM–1.3 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit289">289</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">&lt;3 nm</td>
                  <td valign="top">440/650</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">RDX</td>
                  <td valign="top">1.62 nM</td>
                  <td valign="top">0–0.238 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit258">258</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="4" valign="top">PEI</td>
                  <td rowspan="4" valign="top">1.8 nm</td>
                  <td rowspan="4" valign="top">355/480</td>
                  <td rowspan="4" valign="top">3.8</td>
                  <td rowspan="4" valign="top">Turn off</td>
                  <td valign="top">Sudan dyes I</td>
                  <td valign="top">65 nM</td>
                  <td valign="top">0.1–30 μM</td>
                  <td rowspan="4" valign="top">
                    <a title="Select to navigate to references" href="#cit27">27</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Sudan dyes II</td>
                  <td valign="top">70 nM</td>
                  <td valign="top">0.1–30 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Sudan dyes III</td>
                  <td valign="top">45 nM</td>
                  <td valign="top">0.1–25 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Sudan dyes IV</td>
                  <td valign="top">50 nM</td>
                  <td valign="top">0.1–25 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cysteine</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">400/495</td>
                  <td valign="top">
                  </td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Quinoline yellow</td>
                  <td valign="top">0.11 μM</td>
                  <td valign="top">0.2–5.5 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit122">122</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">AA</td>
                  <td valign="top">1.59 ± 0.23 nm</td>
                  <td valign="top">393/505</td>
                  <td valign="top">33.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Aniline yellow</td>
                  <td valign="top">1.44 μM</td>
                  <td valign="top">0–16 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit290">290</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">3–5 nm</td>
                  <td valign="top">340/570</td>
                  <td valign="top">3.5</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Ochratoxin A</td>
                  <td valign="top">5 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">0–0.1 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit291">291</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">&lt;5 nm</td>
                  <td valign="top">340/625</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Ochratoxin A</td>
                  <td valign="top">2 nM</td>
                  <td valign="top">2.5–250 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit292">292</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">His-pAT</td>
                  <td valign="top">12–25 nM</td>
                  <td valign="top">350/600</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Aflatoxin B1</td>
                  <td valign="top">0.15 pg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.46–400 pg mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit293">293</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">hpDNA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">340/575</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">MC-LR</td>
                  <td valign="top">3 pg L<small><sup>−1</sup></small></td>
                  <td valign="top">0.005–1200 μg L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit294">294</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">340/575</td>
                  <td valign="top">42</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">MC-LR</td>
                  <td valign="top">4.8 ng L<small><sup>−1</sup></small></td>
                  <td valign="top">0.01–1000 μg L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit295">295</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Bisphenol A</td>
                  <td valign="top">0.12 nM</td>
                  <td valign="top">1 nM–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit46">46</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">1–2 nm</td>
                  <td valign="top">345/598</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Melamine</td>
                  <td valign="top">95 nM</td>
                  <td valign="top">0.1–6 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit88">88</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">MBA</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Hydrazine</td>
                  <td valign="top">1.04 μM</td>
                  <td valign="top">1.04–425 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit125">125</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">6 nm</td>
                  <td valign="top">334/432</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">OPD</td>
                  <td valign="top">93 ng L<small><sup>−1</sup></small></td>
                  <td valign="top">0.15–110 μg L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit296">296</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Histidine</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">393/492</td>
                  <td valign="top">2.4</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Fluazinam</td>
                  <td valign="top">0.25 μM</td>
                  <td valign="top">1–40 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit297">297</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">&lt;5 nm</td>
                  <td valign="top">340/560</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Acetamiprid</td>
                  <td valign="top">2.37 nM</td>
                  <td valign="top">5–500 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit298">298</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.61 nm</td>
                  <td valign="top">330/430′</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Dinotefuran</td>
                  <td valign="top">7.04 μM</td>
                  <td valign="top">10–500 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit299">299</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CTAB</td>
                  <td valign="top">6.28 nm</td>
                  <td valign="top">254/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Metam sodium</td>
                  <td valign="top">0.63 mg kg<small><sup>−1</sup></small></td>
                  <td valign="top">1–100 mg kg<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit300">300</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">
                    <span class="small_caps">L</span>-Cysteine</td>
                  <td rowspan="2" valign="top">5.6 nm</td>
                  <td rowspan="2" valign="top">360/460</td>
                  <td rowspan="2" valign="top">76</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td valign="top">Biliverdin</td>
                  <td valign="top">233 nM</td>
                  <td valign="top">0.5–40 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit120">120</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Bilirubin</td>
                  <td valign="top">229 nM</td>
                  <td valign="top">1–10 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">&lt;4 nm</td>
                  <td valign="top">330/404</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Bilirubin</td>
                  <td valign="top">257 fM</td>
                  <td valign="top">10–150 pM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit60">60</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">3.3 ± 0.8 nm</td>
                  <td valign="top">320/405</td>
                  <td valign="top">6.4</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Bilirubin</td>
                  <td valign="top">6.62 nM</td>
                  <td valign="top">10–60 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit61">61</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">HSA</td>
                  <td rowspan="2" valign="top">3 ± 0.3 nm</td>
                  <td rowspan="2" valign="top">390/646</td>
                  <td rowspan="2" valign="top">3.6</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td rowspan="2" valign="top">Bilirubin</td>
                  <td valign="top">35 nM</td>
                  <td valign="top">1.25–7.5 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit301">301</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">145 nM</td>
                  <td valign="top">5–28.75 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">3.15 ± 0.38 nm</td>
                  <td valign="top">355/410</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Dopamine</td>
                  <td valign="top">0.1637 pM</td>
                  <td valign="top">0.1–0.6 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit62">62</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/565</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Dopamine</td>
                  <td valign="top">20 pM</td>
                  <td valign="top">0.1–10 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit302">302</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">325/406</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Dopamine</td>
                  <td valign="top">0.28 μM</td>
                  <td valign="top">0.5–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit303">303</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">365/640′′</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Dopamine</td>
                  <td valign="top">32 nM</td>
                  <td valign="top">0.1–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit304">304</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">D</span>-Penicillamine</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">280/650</td>
                  <td valign="top">7.16</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Histamine</td>
                  <td valign="top">30 nM</td>
                  <td valign="top">0.05–5 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit305">305</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TFTP</td>
                  <td valign="top">&lt;5 nm</td>
                  <td valign="top">325/590</td>
                  <td valign="top">43</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Histamine</td>
                  <td valign="top">60 nM</td>
                  <td valign="top">0.1–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit229">229</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2–2.5 nm</td>
                  <td valign="top">360/595</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Heparin</td>
                  <td valign="top">26 nM</td>
                  <td valign="top">0.1–10 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit126">126</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dBSA</td>
                  <td valign="top">1.7 nm</td>
                  <td valign="top">350/642</td>
                  <td valign="top">2.32</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Heparin</td>
                  <td valign="top">0.26 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">1.25–250 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit306">306</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">&lt;4 nm</td>
                  <td valign="top">330/410</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Heparin</td>
                  <td valign="top">0.0406 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">6–9 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit307">307</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CTAB</td>
                  <td valign="top">&lt;8 nm</td>
                  <td valign="top">290/480</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Carbamazepine</td>
                  <td valign="top">0.08 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.2–20 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit308">308</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">BSA</td>
                  <td rowspan="2" valign="top">&lt;5 nm</td>
                  <td rowspan="2" valign="top">325/420</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Paraoxon</td>
                  <td rowspan="2" valign="top">12.8 nM</td>
                  <td valign="top">50 nM–0.5 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit309">309</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">0.5–35 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Cysteine</td>
                  <td rowspan="2" valign="top">2 nm</td>
                  <td valign="top">368/493</td>
                  <td valign="top">5.8</td>
                  <td valign="top">Turn off</td>
                  <td rowspan="2" valign="top">Tetracycline</td>
                  <td valign="top">5.6 μM</td>
                  <td valign="top">0–20 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit310">310</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">373/595</td>
                  <td valign="top">3.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">8.4 μM</td>
                  <td valign="top">0–20 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">1–3 nm</td>
                  <td valign="top">380/600</td>
                  <td valign="top">3.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Vitamin B1</td>
                  <td valign="top">4.6 nM</td>
                  <td valign="top">20 nM–0.1 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit311">311</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">Ovalbumin</td>
                  <td rowspan="2" valign="top">3.6 ± 1.1 nm</td>
                  <td rowspan="2" valign="top">350/560</td>
                  <td rowspan="2" valign="top">5.8</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Vitamin B1</td>
                  <td valign="top">380 pM</td>
                  <td valign="top">1–1000 nM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit312">312</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn on</td>
                  <td valign="top">Doxycycline</td>
                  <td valign="top">270 nM</td>
                  <td valign="top">1–1000 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Adenosine</td>
                  <td valign="top">2.4 ± 0.3 nm</td>
                  <td valign="top">285/417</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Nitrofurantoin</td>
                  <td valign="top">30 nM</td>
                  <td valign="top">0.05–4 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit313">313</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">325/400</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">
                    <span class="small_caps">D</span>-Penicillamine</td>
                  <td valign="top">0.54 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.6–30 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit314">314</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Lysozyme</td>
                  <td valign="top">3.4 nm</td>
                  <td valign="top">335/550</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Choline</td>
                  <td valign="top">25 nM</td>
                  <td valign="top">0.1–80 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit315">315</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2.1 nm</td>
                  <td valign="top">320/640</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Rutin</td>
                  <td valign="top">0.02 μM</td>
                  <td valign="top">0.1–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit41">41</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">320/400</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Gossypol</td>
                  <td valign="top">25 nM</td>
                  <td valign="top">1–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit316">316</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cytidine</td>
                  <td valign="top">1.5 ± 0.5 nm</td>
                  <td valign="top">300/380</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hemin</td>
                  <td valign="top">0.045 μM</td>
                  <td valign="top">0.05–4 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit252">252</a>
                  </td>
                </tr>
                <tr align="left">
                  <td align="left" valign="top">—</td>
                  <td valign="top">3.5 nm</td>
                  <td valign="top">350/430</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Hemin</td>
                  <td valign="top">68 nM</td>
                  <td valign="top">0–1 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit247">247</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">AA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">390/455</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Quercetin</td>
                  <td valign="top">0.19 μM</td>
                  <td valign="top">0.7–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit317">317</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Proline</td>
                  <td valign="top">2.1 nm</td>
                  <td valign="top">373/453</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Quercetin</td>
                  <td valign="top">12 nM</td>
                  <td valign="top">0.5–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit318">318</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.8 ± 0.2 nm</td>
                  <td valign="top">350/670′′</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Acetaminophen</td>
                  <td valign="top">0.26 μM</td>
                  <td valign="top">1–80 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit319">319</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">1–2 nm</td>
                  <td valign="top">
                    <span class="italic">x</span>/547</td>
                  <td valign="top">
                  </td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Tramadol</td>
                  <td valign="top">0.8 nM</td>
                  <td valign="top">0.003–2.5 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit320">320</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">1.4 ± 0.2 nm</td>
                  <td valign="top">545/640</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Mangiferin</td>
                  <td valign="top">210 nM</td>
                  <td valign="top">3–180 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit321">321</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.4 ± 0.4 nm</td>
                  <td valign="top">360/585</td>
                  <td valign="top">3</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Creatinine</td>
                  <td valign="top">0.63 μg L<small><sup>−1</sup></small></td>
                  <td valign="top">2.5–34 μg L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit222">222</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">525/643</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Creatinine</td>
                  <td valign="top">50 nM</td>
                  <td valign="top">5–60 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit322">322</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">340/585</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Cholesterol</td>
                  <td valign="top">2.7 μM</td>
                  <td valign="top">6–80 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit127">127</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PVP</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">380/425</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">GSH</td>
                  <td valign="top">17 μM</td>
                  <td valign="top">0–140 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit323">323</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PVP</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">380/425</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">GSH</td>
                  <td valign="top">3 μM</td>
                  <td valign="top">5–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit324">324</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="3" valign="top">dsDNA</td>
                  <td rowspan="3" valign="top">—</td>
                  <td rowspan="3" valign="top">340/580</td>
                  <td rowspan="3" valign="top">—</td>
                  <td rowspan="3" valign="top">Turn off</td>
                  <td valign="top">GSH</td>
                  <td valign="top">2 μM</td>
                  <td valign="top">2–80 μM</td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit80">80</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cys</td>
                  <td valign="top">2 μM</td>
                  <td valign="top">2 μM–0.1 mM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Hcy</td>
                  <td valign="top">5 μM</td>
                  <td valign="top">5 μM–0.2 mM</td>
                </tr>
                <tr align="left">
                  <td rowspan="3" valign="top">Poly(thymine)</td>
                  <td rowspan="3" valign="top">5 nm</td>
                  <td rowspan="3" valign="top">340/615</td>
                  <td rowspan="3" valign="top">—</td>
                  <td rowspan="3" valign="top">Turn on</td>
                  <td valign="top">GSH</td>
                  <td valign="top">15 nM</td>
                  <td valign="top">0–1000 nM</td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit325">325</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cys</td>
                  <td valign="top">12.5 nM</td>
                  <td valign="top">0–1000 nM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Hcy</td>
                  <td valign="top">20 nM</td>
                  <td valign="top">0–1000 nM</td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">DNA nanoribbon</td>
                  <td rowspan="2" valign="top">0.63 nm</td>
                  <td rowspan="2" valign="top">351/594</td>
                  <td rowspan="2" valign="top">12.3</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td valign="top">GSH</td>
                  <td valign="top">0.43 μM</td>
                  <td valign="top">24–100 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit326">326</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cys</td>
                  <td valign="top">0.31 μM</td>
                  <td valign="top">24–100 μM</td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">PEI</td>
                  <td rowspan="2" valign="top">2–3 nm</td>
                  <td rowspan="2" valign="top">365/495</td>
                  <td rowspan="2" valign="top">2.63</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">GSH</td>
                  <td valign="top">0.26 μM</td>
                  <td valign="top">0.5–25 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit254">254</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn on</td>
                  <td valign="top">Cys</td>
                  <td valign="top">0.34 μM</td>
                  <td valign="top">1–25 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Ovalbumin</td>
                  <td valign="top">5.6 ± 1.1 nm</td>
                  <td valign="top">370/440</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">
                    <span class="small_caps">L</span>-Lysine</td>
                  <td valign="top">5.5 μM</td>
                  <td valign="top">10 μM –1 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit327">327</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.8 ± 0.2 nm</td>
                  <td valign="top">350/670′′</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Arginine</td>
                  <td valign="top">0.35 μM</td>
                  <td valign="top">1–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit319">319</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">Cysteine</td>
                  <td rowspan="2" valign="top">2 nm</td>
                  <td rowspan="2" valign="top">360/490</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Turn on</td>
                  <td valign="top">Tryptophan</td>
                  <td valign="top">75 nM</td>
                  <td valign="top">0.1–30 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit328">328</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Phenylalanine</td>
                  <td valign="top">840 nM</td>
                  <td valign="top">1–27 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Adenosine</td>
                  <td valign="top">3.7 ± 0.7 nm</td>
                  <td valign="top">285/380</td>
                  <td valign="top">1.34</td>
                  <td rowspan="3" valign="top">—</td>
                  <td rowspan="3" valign="top">Nucleosides</td>
                  <td rowspan="3" valign="top">20 μM</td>
                  <td rowspan="3" valign="top">20 μM–0.5 mM</td>
                  <td rowspan="3" valign="top">
                    <a title="Select to navigate to references" href="#cit329">329</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cytidine</td>
                  <td valign="top">1.5 ± 0.5 nm</td>
                  <td valign="top">300/380</td>
                  <td valign="top">0.44</td>
                </tr>
                <tr align="left">
                  <td valign="top">Guanosine</td>
                  <td valign="top">1.5 ± 0.4 nm</td>
                  <td valign="top">300/380</td>
                  <td valign="top">0.27</td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">340/615</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ATP</td>
                  <td valign="top">93 nM</td>
                  <td valign="top">0.2–50 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit330">330</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">3–4 nm</td>
                  <td valign="top">340/598</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ATP</td>
                  <td valign="top">5 pM</td>
                  <td valign="top">0.01–100 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit331">331</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">&lt;5 nm</td>
                  <td valign="top">350/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ATP</td>
                  <td valign="top">10.29 nM</td>
                  <td valign="top">100 nM–100 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit332">332</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">5.5 nm</td>
                  <td valign="top">365/605</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">ATP</td>
                  <td valign="top">0.67 μM</td>
                  <td valign="top">1–2000 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit333">333</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Histidine</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">350//456</td>
                  <td valign="top">1.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">GTP</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit147">147</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">340/600</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">NAD<small><sup>+</sup></small></td>
                  <td valign="top">0.2 nM</td>
                  <td valign="top">0.2–20 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit334">334</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">hpDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">345/635</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">NAD<small><sup>+</sup></small></td>
                  <td valign="top">5 nM</td>
                  <td valign="top">5 nM–0.5 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit335">335</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Silyl</td>
                  <td valign="top">2–6</td>
                  <td valign="top">410/695</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">H<small><sub>2</sub></small>O</td>
                  <td valign="top">0.0018% v/v</td>
                  <td valign="top">0.0062–0.19% v/v</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit112">112</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">AA</td>
                  <td valign="top">1.2 ± 0.3 nm</td>
                  <td valign="top">385/498</td>
                  <td valign="top">6.63</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Urea</td>
                  <td valign="top">0.01 mM</td>
                  <td valign="top">0.25–5 mM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit19">19</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.17 nm</td>
                  <td valign="top">340/585</td>
                  <td valign="top">13.91</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Urea</td>
                  <td valign="top">5.86 μM</td>
                  <td valign="top">20–150 μM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit336">336</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">dsDNA</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">340/596</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td valign="top">DNA aptamer</td>
                  <td valign="top">28 nM</td>
                  <td valign="top">0.05–500 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit70">70</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cocaine</td>
                  <td valign="top">0.1 μM</td>
                  <td valign="top">0.5–100 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">PAA-<span class="italic">g</span>-MEA</td>
                  <td valign="top">0.8 ± 0.2 nm</td>
                  <td valign="top">360/600</td>
                  <td valign="top">5.7</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">APDC</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit36">36</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">349/617</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Biotin</td>
                  <td valign="top">3.1 nM</td>
                  <td valign="top">10–1000 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit337">337</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">RNA detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/608</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA</td>
                  <td valign="top">1pM</td>
                  <td valign="top">1pM–10nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit75">75</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">3–4 nm</td>
                  <td valign="top">340/608</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA</td>
                  <td valign="top">10 pM</td>
                  <td valign="top">10–400 pM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit76">76</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">microRNA</td>
                  <td valign="top">8.2 fM</td>
                  <td valign="top">25–300 fM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit81">81</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">2.5 nm</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA-21</td>
                  <td valign="top">19.05 aM</td>
                  <td valign="top">100 aM–100 pM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit338">338</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/605</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA-21</td>
                  <td valign="top">18.7 pM</td>
                  <td valign="top">50–1000 pM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit339">339</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">340/580</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA-21</td>
                  <td valign="top">10 aM</td>
                  <td valign="top">0.1 fM–10 pM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit340">340</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">350/630</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA-122</td>
                  <td valign="top">44 fM</td>
                  <td valign="top">100 fM–1 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit341">341</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">400/490</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">microRNA-155</td>
                  <td valign="top">0.6 pM</td>
                  <td valign="top">1 pM–10 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit342">342</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">400/580</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA-155</td>
                  <td valign="top">11 pM</td>
                  <td valign="top">50 pm–10 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit343">343</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">1.5 nm</td>
                  <td valign="top">
                    <span class="italic">x</span>/390</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">microRNA-155</td>
                  <td valign="top">36 aM</td>
                  <td valign="top">100 aM–100 pM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit344">344</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">
                    <span class="italic">x</span>/616</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">microRNA-222</td>
                  <td valign="top">0.03 fM</td>
                  <td valign="top">0.5 fM–70 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit345">345</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Enzymatic activity detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">4-MTP</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">345/625</td>
                  <td valign="top">9</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">β-Gal</td>
                  <td valign="top">0.9 U L<small><sup>−1</sup></small></td>
                  <td valign="top">2.5–212 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit141">141</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2–5 nm</td>
                  <td valign="top">410/640</td>
                  <td valign="top">8.1</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">β-Gal</td>
                  <td valign="top">0.7 U L<small><sup>−1</sup></small></td>
                  <td valign="top">2.3–96 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit227">227</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/574</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Exo III</td>
                  <td valign="top">0.02 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.05–2 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit65">65</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">340/610</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Exo III</td>
                  <td valign="top">0.02 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.02–10 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit346">346</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/575</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">EcoRI</td>
                  <td valign="top">870 μU μL<small><sup>−1</sup></small></td>
                  <td valign="top">0.002–0.1 U μL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit347">347</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">3.58 nm</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">MMP7</td>
                  <td valign="top">5.3 pg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.01–100 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit348">348</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">350/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">MTase</td>
                  <td valign="top">0.17 mU μL<small><sup>−1</sup></small></td>
                  <td valign="top">0.1–0.2 U μL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit349">349</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/590</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Dam MTase</td>
                  <td valign="top">0.5 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.5–10 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit350">350</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">340/570</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">MNase</td>
                  <td valign="top">1 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">1–50 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit351">351</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">TA</td>
                  <td valign="top">2.04 nm</td>
                  <td valign="top">360/438</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">PPase</td>
                  <td valign="top">0.19 U L<small><sup>−1</sup></small></td>
                  <td valign="top">0.5–18 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit352">352</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.1 nm</td>
                  <td valign="top">360/615</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">PPase</td>
                  <td valign="top">1.3 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">3–40 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit223">223</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/615</td>
                  <td valign="top">6.8</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S1 nuclease</td>
                  <td valign="top">0.5 μU μL<small><sup>−1</sup></small></td>
                  <td valign="top">5 μU μL<small><sup>−1</sup></small>–2 mU μL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit353">353</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">
                    <span class="italic">x</span>/610</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">S1 nuclease</td>
                  <td valign="top">3 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">5–80 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit354">354</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">D</span>-Penicillamine</td>
                  <td valign="top">2–5 nm</td>
                  <td valign="top">302/646</td>
                  <td valign="top">11.2</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">ACP</td>
                  <td valign="top">1.3 U L<small><sup>−1</sup></small></td>
                  <td valign="top">3.8–22.8 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit220">220</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PDA</td>
                  <td valign="top">53.7 ± 4 nm</td>
                  <td valign="top">340/580</td>
                  <td valign="top">2.54</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ACP</td>
                  <td valign="top">1 mU L<small><sup>−1</sup></small></td>
                  <td valign="top">1.2 mU L<small><sup>−1</sup></small>–25 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit276">276</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">335/565</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ALP</td>
                  <td valign="top">0.1 nM</td>
                  <td valign="top">0.1–2.5 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit74">74</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">360/610</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ALP</td>
                  <td valign="top">0.15 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.5–25 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit224">224</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">&lt;5 nm</td>
                  <td valign="top">335/410</td>
                  <td valign="top">15.2</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ALP</td>
                  <td valign="top">0.1 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.5–150 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit355">355</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Histidine</td>
                  <td valign="top">3–4 nm</td>
                  <td valign="top">390/485</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">ALP</td>
                  <td valign="top">45 μU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.5–40 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit356">356</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">3.2 nm</td>
                  <td valign="top">340/575</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">ALP</td>
                  <td valign="top">6 mU L<small><sup>−1</sup></small></td>
                  <td valign="top">0.04–8 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit357">357</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">340/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">ALP</td>
                  <td valign="top">125 μU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0–62.5 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit262">262</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">4–5 nm</td>
                  <td valign="top">340/570</td>
                  <td valign="top">3.9</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">UDG</td>
                  <td valign="top">0.5 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">1 mU mL<small><sup>−1</sup></small>–0.1 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit358">358</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">2.87 nm</td>
                  <td valign="top">400/602</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">UDG</td>
                  <td valign="top">50 μU mL<small><sup>−1</sup></small></td>
                  <td valign="top">100 μU mL<small><sup>−1</sup></small>–0.01 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit359">359</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">2.5–5.3 nm</td>
                  <td valign="top">345/650</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">UDG</td>
                  <td valign="top">50 μU mL<small><sup>−1</sup></small></td>
                  <td valign="top">50 μU mL<small><sup>−1</sup></small>–2 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit360">360</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">340/520</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">T4 PNKP</td>
                  <td valign="top">0.25 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.25–25 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit361">361</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">340/620</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">T4 PNKP</td>
                  <td valign="top">0.02 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.02–20 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit362">362</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">3.3 nm</td>
                  <td valign="top">340/570</td>
                  <td valign="top">3.4</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">T4 PNKP</td>
                  <td valign="top">0.06 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.07–15 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit363">363</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/565</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">PNK</td>
                  <td valign="top">0.49 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">0–3 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit364">364</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">365/495</td>
                  <td valign="top">2.63</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">AChE</td>
                  <td valign="top">1.38 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">3–200 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit254">254</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PVP</td>
                  <td valign="top">2.72 nm</td>
                  <td valign="top">370/438′</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">AChE</td>
                  <td valign="top">0.56 U L<small><sup>−1</sup></small></td>
                  <td valign="top">2–70 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit365">365</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/600</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">AChE</td>
                  <td valign="top">0.05 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.11–2.78 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit366">366</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">340/570</td>
                  <td valign="top">11.2</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">TdT</td>
                  <td valign="top">60 mU L<small><sup>−1</sup></small></td>
                  <td valign="top">0.7–14 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit367">367</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA/RNA</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">340/570</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">RNase H</td>
                  <td valign="top">0.55 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.6 mU mL<small><sup>−1</sup></small>–30 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit368">368</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.4 nm</td>
                  <td valign="top">300/500</td>
                  <td valign="top">17.55</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">GLU</td>
                  <td valign="top">0.22 U L<small><sup>−1</sup></small></td>
                  <td valign="top">0.5–6 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit369">369</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA/Gly–Gly</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">330/415′</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">TYR</td>
                  <td valign="top">44 U L<small><sup>−1</sup></small></td>
                  <td valign="top">0.1–7 U L<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit370">370</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">2.28 ± 0.56 nm</td>
                  <td valign="top">340/590</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Thrombin</td>
                  <td valign="top">0.9 nM</td>
                  <td valign="top">1–50 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit371">371</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">350/560</td>
                  <td valign="top">14.27</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">HAase</td>
                  <td valign="top">14 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0–0.4 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit372">372</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CS/GSH</td>
                  <td valign="top">&lt;10 nm</td>
                  <td valign="top">350/574</td>
                  <td valign="top">5.2</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Lysozyme</td>
                  <td valign="top">1.6 nM</td>
                  <td valign="top">5–110 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit373">373</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">374/515</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">PKA</td>
                  <td valign="top">38 mU mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.1–6 U mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit374">374</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Protein detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">349/617</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Streptavidin</td>
                  <td valign="top">0.47 nM</td>
                  <td valign="top">1–200 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit337">337</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">340/600</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Streptavidin</td>
                  <td valign="top">0.1 nM</td>
                  <td valign="top">0.5–1000 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit375">375</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/630</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Streptavidin</td>
                  <td valign="top">20 pM</td>
                  <td valign="top">69.2 pM–17.3 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit376">376</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">1.9 nm</td>
                  <td valign="top">330/630</td>
                  <td valign="top">3.7</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Glycoprotein</td>
                  <td valign="top">2.6 nM</td>
                  <td valign="top">5–220 nM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit377">377</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">5 nm</td>
                  <td valign="top">340/585</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Actin</td>
                  <td valign="top">0.12 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0–12 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit378">378</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poly(thymine)</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/615</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Trypsin</td>
                  <td valign="top">42 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.25–1000 μg mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit379">379</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">4.5 nm</td>
                  <td valign="top">340/610</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Trypsin</td>
                  <td valign="top">2 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">2–20 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit380">380</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">2–3 nm</td>
                  <td valign="top">350/591</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Trypsin</td>
                  <td valign="top">48 pg mL<small><sup>−1</sup></small></td>
                  <td valign="top">1–1000 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit381">381</a>
                  </td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">PSS</td>
                  <td rowspan="2" valign="top">3–4 nm</td>
                  <td rowspan="2" valign="top">390/645</td>
                  <td rowspan="2" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Trypsin</td>
                  <td valign="top">20 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.1–6 μg mL<small><sup>−1</sup></small></td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit382">382</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Turn off</td>
                  <td valign="top">Cytochrome C</td>
                  <td valign="top">0.83 nM</td>
                  <td valign="top">8–680 nM</td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">3.5 nm</td>
                  <td rowspan="2" valign="top">350/430</td>
                  <td rowspan="2" valign="top">—</td>
                  <td rowspan="2" valign="top">Turn off</td>
                  <td valign="top">Cytochrome C</td>
                  <td valign="top">0.8 μM</td>
                  <td valign="top">1–8 μM</td>
                  <td rowspan="2" valign="top">
                    <a title="Select to navigate to references" href="#cit247">247</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Ferritin</td>
                  <td valign="top">16.5 nM</td>
                  <td valign="top">0–0.25 μM</td>
                </tr>
                <tr align="left">
                  <td valign="top">Aptamer</td>
                  <td valign="top">17.8 nm</td>
                  <td valign="top">332/463</td>
                  <td valign="top">8.2</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">VEGF<small><sub>165</sub></small></td>
                  <td valign="top">12 pM</td>
                  <td valign="top">10–800 pM</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit383">383</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">&lt;4 nm</td>
                  <td valign="top">330/410</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">Protamine</td>
                  <td valign="top">0.12 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">3–12 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit307">307</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">340/565</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">CEA</td>
                  <td valign="top">6.5 pg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.01–2 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit384">384</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">9 nm</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">PSA</td>
                  <td valign="top">145.7 fg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.5 pg mL<small><sup>−1</sup></small>–100 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit385">385</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">3.2 nm</td>
                  <td valign="top">340/575</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">IgG</td>
                  <td valign="top">7 pg mL<small><sup>−1</sup></small></td>
                  <td valign="top">0.05–12 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit357">357</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Temperature sensing</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">HSA</td>
                  <td valign="top">3 nm</td>
                  <td valign="top">330/414</td>
                  <td valign="top">4</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Temp</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit50">50</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">410/610</td>
                  <td valign="top">5</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Temp</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit130">130</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CLEDNN</td>
                  <td valign="top">1.7 ± 0.4 nm</td>
                  <td valign="top">373/454</td>
                  <td valign="top">7.3</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Temp</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">10–55 °C</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit134">134</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">2.4 ± 0.4 nm</td>
                  <td valign="top">400/610</td>
                  <td valign="top">4.5</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">Temp</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">20–45 °C</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit386">386</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">pH sensing</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Trypsin</td>
                  <td valign="top">2.03 ± 0.46 nm</td>
                  <td valign="top">363/455</td>
                  <td valign="top">1.1</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit39">39</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">1.6 ± 0.5 nm</td>
                  <td valign="top">320/420</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit49">49</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">BSA</td>
                  <td valign="top">2.5 nm</td>
                  <td valign="top">325/406</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit303">303</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Cysteine</td>
                  <td valign="top">2.3 nm</td>
                  <td valign="top">365/490</td>
                  <td valign="top">8.8</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit133">133</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">GSH</td>
                  <td valign="top">1–3 nm</td>
                  <td valign="top">380/600</td>
                  <td valign="top">3.6</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">4–12</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit311">311</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">PEI</td>
                  <td valign="top">2 nm</td>
                  <td valign="top">380/498</td>
                  <td valign="top">7.5</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit387">387</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CEW</td>
                  <td valign="top">2.4 ± 0.2 nm</td>
                  <td valign="top">337/417</td>
                  <td valign="top">0.98</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit388">388</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="small_caps">L</span>-Tyrosine</td>
                  <td valign="top">1–2 nm</td>
                  <td valign="top">430/477</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">pH</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">5.02–11.92</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit389">389</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="9" valign="top">
                    <span class="bold">Infections detection</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">DNA</td>
                  <td valign="top">6 nm</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn on</td>
                  <td valign="top">HBV</td>
                  <td valign="top">12 × 10<small><sup>9</sup></small> molecules</td>
                  <td valign="top">12 × 10<small><sup>9</sup></small>–12 × 10<small><sup>13</sup></small> molecules</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit390">390</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">dsDNA</td>
                  <td valign="top">&lt;4 nm</td>
                  <td valign="top">340/600</td>
                  <td align="left" valign="top">—</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">TB</td>
                  <td valign="top">5 fg μL<small><sup>−1</sup></small></td>
                  <td valign="top">10–100 fg μL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit79">79</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">CP25</td>
                  <td valign="top">2.3 ± 0.3 nm</td>
                  <td valign="top">460/630</td>
                  <td align="left" valign="top">—</td>
                  <td valign="top">Turn off</td>
                  <td valign="top">CTV</td>
                  <td valign="top">220 pg mL<small><sup>−1</sup></small></td>
                  <td valign="top">400 pg mL<small><sup>−1</sup></small>–25 ng mL<small><sup>−1</sup></small></td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit391">391</a>
                  </td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">As mentioned earlier, highly fluorescent, and biocompatible CuNCs have found applications in the sensing of metal cations such as mercury, iron and lead, not only for environmental control, but also in clinical practice. The advances in synthetic procedures have led to sensors with high sensitivity and low detection limits, even down to fM concentrations of a target material in some cases with negligible amounts of the sensor. The possible bio-applications of CuNCs are countless. In most applications, the detection of a certain molecule is based on fluorescence quenching. In the following, we describe some sensing applications, and we do this according to the target species to be detected. More detailed descriptions and applications of CuNCs can be found in the reviews by Yi Lv,<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> Chuanxi Wang,<a title="Select to navigate to references" href="#cit392"><sup><span class="sup_ref">392</span></sup></a> Bo Feng,<a title="Select to navigate to references" href="#cit393"><sup><span class="sup_ref">393</span></sup></a> Baisong Chang<a title="Select to navigate to references" href="#cit394"><sup><span class="sup_ref">394</span></sup></a> and Yuming Huang.<a title="Select to navigate to references" href="#cit395"><sup><span class="sup_ref">395</span></sup></a> Ronghua Yang<a title="Select to navigate to references" href="#cit396"><sup><span class="sup_ref">396</span></sup></a> and Erkang Wang<a title="Select to navigate to references" href="#cit397"><sup><span class="sup_ref">397</span></sup></a> summarized the recent progress in biosensing applications of CuNCs utilizing DNA scaffolds.</p>
        <div>
          
          <span id="sect6750"/><span class="c_heading_indent">4.1.1. Metal cation detection. </span>
          <span>Mercury, in the forms of metallic and organic/inorganic complexes, shows different degrees of toxicity and can perturb the nervous, digestive, and immune systems upon exposure. The normal concentration of mercury in the blood is estimated as less than 10 ng mL<small><sup>−1</sup></small>, whereas higher concentrations due to sudden inhalation or ingestion can cause serious damage and even be fatal. Thus, fast, sensitive, and selective detection tools are in high demand in clinical practice. Recently, CuNCs proved to be excellent candidates for Hg<small><sup>2+</sup></small> sensing in water samples. Yao Dou and co-workers prepared <span class="small_caps">L</span>-cysteine-stabilized and highly fluorescent CuNCs (QY 14.3%) and tested them for Hg<small><sup>2+</sup></small> detection.<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a> The designed sensor could work in the analytical range of 0.1 μM–1 mM with a detection limit of 24 nM. However, when GSH was used as a capping ligand, Tingting Luo <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> and Xue Hu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit239"><sup><span class="sup_ref">239</span></sup></a> independently prepared sensors with LODs (limits of detection) of 22 nM and 3.3 nM, respectively. It was possible to further increase the detection sensitivity and lower the LOD to 0.12 nM when CuNCs were synthesized from curcuma root extract.<a title="Select to navigate to references" href="#cit241"><sup><span class="sup_ref">241</span></sup></a> Beyond that, Hao Zhang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit243"><sup><span class="sup_ref">243</span></sup></a> prepared a Hg<small><sup>2+</sup></small> sensor with the lowest reported LOD of 4 pM. Further CuNCs with low detection limits were reported: BSA-templated CuNCs (LOD 4.7 pM),<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> DNA-templated CuNCs (LOD 16 pM),<a title="Select to navigate to references" href="#cit240"><sup><span class="sup_ref">240</span></sup></a> 4-aminothiophenol (PATP)-CuNCs (LOD 10 ppt)<a title="Select to navigate to references" href="#cit244"><sup><span class="sup_ref">244</span></sup></a> and metallothionein (MT)-stabilized CuNCs (LOD 43.8 nM).<a title="Select to navigate to references" href="#cit242"><sup><span class="sup_ref">242</span></sup></a></span>
          <p class="otherpara">Iron is an essential element for red blood cells for the transport of oxygen. Ferrous/ferric iron is a redox system, and it is involved in electron transport in bioprocesses. Under normal conditions, the iron content for a healthy person is 4–5 g and both lower and higher amounts are problematic. Iron is toxic upon overload and can cause severe diseases such as hemochromatosis. At the same time, insufficient iron levels in the blood can lead to the development of iron deficiency anaemia. In addition to clinical blood tests and attempts to prepare nanomaterial-based sensors, CuNCs have also been extensively applied in ferric ion sensing in water, urine and serum samples based on the fluorescence quenching of a sensor signal.<a title="Select to navigate to reference" href="#cit30"><sup><span class="sup_ref">30,135,138,145,148,245–249</span></sup></a> For example, a dopamine-protected CuNCs-based biosensor could properly operate in the range of 5–300 μM with a LOD of 1.2 μM.<a title="Select to navigate to references" href="#cit148"><sup><span class="sup_ref">148</span></sup></a> However, the extensive research in the field resulted in even lower LOD values, down to 10 nM, when tannic acid (TA)-protected CuNCs were implemented as a sensor.<a title="Select to navigate to references" href="#cit145"><sup><span class="sup_ref">145</span></sup></a> Note that the CuNCs-based ferric ion sensors mentioned above showed impressive sensitivities, however, none of them have made it to clinical practice so far.</p>
          <p class="otherpara">Copper is required for the proper functioning of several Cu-containing enzymes such as ceruloplasmin, cytochrome-c oxidase, <span class="italic">etc</span>. According to the “<span class="italic">WHO Guidelines for Drinking-water Quality</span>”, the allowed concentration of copper in drinking water is 2 mg L<small><sup>−1</sup></small> and the tolerable uptake is 10 mg per day per kg of body weight.<a title="Select to navigate to references" href="#cit398"><sup><span class="sup_ref">398</span></sup></a> During short- and long-term exposure from dietary supplements or drinking water, the excessive intake of copper builds up in the kidneys, brain and eyes and can cause severe damage. Apart from standard analytical techniques, CuNCs-based sensors have found important applications for detecting Cu ions in drinking water samples.<a title="Select to navigate to reference" href="#cit115"><sup><span class="sup_ref">115,250,252–254</span></sup></a> Zhihe Qing <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit251"><sup><span class="sup_ref">251</span></sup></a> prepared poly(thymine)-templated and highly fluorescent (QY 11.2%) CuNCs as a turn-on sensor for Cu<small><sup>2+</sup></small> detection and its toxicide screening. The sensor showed a good performance and could detect 5.6 μM of copper in contaminated water samples. However, the ongoing research and the race for preparing template-selective and ion-sensitive/selective sensors with excellent detecting capabilities led to a sensor with a LOD as low as 1 nM with a linear concentration range of 0.02–34 μM.<a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a></p>
          <p class="otherpara">Lead contamination became a global threat to human health and upon exposure, it can accumulate in the brain, liver, kidney, and bones. The WHO states that blood lead concentrations below 5 μg dL<small><sup>−1</sup></small> can reduce the IQ of children, cause behavioural and learning difficulties, and attack the brain and central nervous system to cause coma, convulsions and even death. The guideline value of lead in drinking water is 10 μg L<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit399"><sup><span class="sup_ref">399</span></sup></a> On top of clinically accepted blood testing, researchers have developed CuNCs-based fluorescent sensors. For example, Chuanxi Wang <span class="italic">et al.</span> reported the synthesis and application of GSH-protected CuNCs as a label-free fluorescent probe to detect Pb<small><sup>2+</sup></small> in CAL-27 cells.<a title="Select to navigate to references" href="#cit137"><sup><span class="sup_ref">137</span></sup></a> The proposed fluorescent probe showed excellent turn-off detection ability down to 1 nM with a linear response in the range of 1–160 nM. Xia Chu and co-workers prepared another sensor that was highly selective and sensitive and could operate at even lower concentrations (LOD 0.4 nM)<a title="Select to navigate to references" href="#cit255"><sup><span class="sup_ref">255</span></sup></a> than the lead ion sensors previously reported by Junhua Chen <span class="italic">et al.</span> (LOD 5 nM),<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> N. Vilar-Vidal <span class="italic">et al.</span> (LOD 4.9 μM),<a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a> Ran Liu <span class="italic">et al.</span> (LOD 142 nM)<a title="Select to navigate to references" href="#cit242"><sup><span class="sup_ref">242</span></sup></a> and Fei Qu <span class="italic">et al.</span> (LOD 30 μM).<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a></p>
          <p class="otherpara">Chromium salts have many applications in industry and vary in their toxicological profiles from one oxidation state to another. Cr(<span class="small_caps">VI</span>) is extremely toxic and is a carcinogen, whereas Cr(<span class="small_caps">III</span>) is an essential nutrient for organisms to maintain normal physiological functioning.<a title="Select to navigate to references" href="#cit400"><sup><span class="sup_ref">400</span></sup></a> Thus, for the clinical assessment and diagnosis of chromium poisoning <span class="italic">via</span> inhalation or ingestion, fast and sensitive analytical methods are required. CuNCs applications were extended towards that goal too. For instance, Malin Cui <span class="italic">et al.</span> prepared cys-CuNCs as a fluorescent probe for Cr<small><sup>+6</sup></small> with a LOD of 43 nM.<a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a> Mixed ligand shell BSA/thiosalicylic acid (TSA)-CuNCs showed much better sensitivity towards Cr<small><sup>+6</sup></small> with a LOD of 3.54 nM.<a title="Select to navigate to references" href="#cit260"><sup><span class="sup_ref">260</span></sup></a> For Cr<small><sup>+3</sup></small> detection, Ajnesh Singh <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit261"><sup><span class="sup_ref">261</span></sup></a> and Can Chen <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit262"><sup><span class="sup_ref">262</span></sup></a> developed a hybrid CuNCs@organic nanoparticle complex and poly(thymine)-CuNCs sensor with minimum detection ability of 3 μM and 30 nM, respectively.</p>
          <p class="otherpara">Aluminum is found in consumer products like antacids, food additives, cosmetics, <span class="italic">etc</span>. However, the FDA set a limit for bottled water of 0.2 mg L<small><sup>−1</sup></small>. Dithiothreitol (DTT)-protected CuNCs were suggested as an accessible sensor for the detection of aluminum in food samples.<a title="Select to navigate to references" href="#cit263"><sup><span class="sup_ref">263</span></sup></a> Under optimized conditions, the fluorescent probe exhibited a good linear relationship in the range of 0.01–7 μM with a detection limit of 0.01 μM. Later, Chanida Boonmee <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit264"><sup><span class="sup_ref">264</span></sup></a> and Jiawei Pang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit265"><sup><span class="sup_ref">265</span></sup></a> developed sensors with largely improved LODs of 26.7 nM and 62 nM, respectively.</p>
          <p class="otherpara">Zinc is an essential mineral, and it is involved in DNA synthesis and normal taste perception, supports wound healing, immune function, and reproductive health. The WHO set a provisional maximum tolerable daily intake (PMTDI) of 1.0 mg per kg of body weight.<a title="Select to navigate to references" href="#cit401"><sup><span class="sup_ref">401</span></sup></a> High doses of zinc can cause acute or chronic issues upon short or long-term exposure. Thus, the estimation and detection of zinc levels in water are of great importance.<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a> Liyun Lin <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit225"><sup><span class="sup_ref">225</span></sup></a> and Xueqing Gao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit259"><sup><span class="sup_ref">259</span></sup></a> demonstrated the applicability of GSH- and Cys-CuNCs as fluorometric probes for zinc ion detection with LODs of 1.17 μM and 1 μM, respectively. However, BSA-CuNCs proved to be a better Zn<small><sup>2+</sup></small> sensor with a LOD of 15 nM.<a title="Select to navigate to references" href="#cit258"><sup><span class="sup_ref">258</span></sup></a></p>
          <p class="otherpara">CuNCs-based sensors were also applied for the detection of Co(<span class="small_caps">II</span>),<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> Ca(<span class="small_caps">II</span>)<a title="Select to navigate to references" href="#cit256"><sup><span class="sup_ref">256</span></sup></a> and Mn(<span class="small_caps">II</span>).<a title="Select to navigate to references" href="#cit257"><sup><span class="sup_ref">257</span></sup></a> Sensing applications of metal cations using CuNCs are also summarized in the review by Jintong Song.<a title="Select to navigate to references" href="#cit212"><sup><span class="sup_ref">212</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect6841"/><span class="c_heading_indent">4.1.2. Anion detection. </span>
          <span>Hydrogen sulphide exposure is of major concern, especially in the mining industry and communities close to industrial areas. It can cause severe respiratory irritation, immunological, cardiovascular, and neurological effects. The tolerable concentration upon short-term exposure is estimated to be 2.8 mg m<small><sup>−3</sup></small>, whereas the allowed limit in water is 0.05–0.1 mg L<small><sup>−1</sup></small> (WHO, 2003).<a title="Select to navigate to references" href="#cit402"><sup><span class="sup_ref">402</span></sup></a> Considering the detection limits (about several micrograms per m<small><sup>3</sup></small>) using analytical methods (chromatographic and spectroscopic), highly sensitive and selective sensors are of great importance. Benefiting from their excellent luminescence properties, CuNCs were also applied in this area. The advances in the field resulted in the improved performances of various sensors by lowering the LOD from μM to nM concentrations for H<small><sub>2</sub></small>S in water samples. Starting in 2012,<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> a great deal of research has been conducted to develop various CuNCs for applications in sulphide ion sensing.<a title="Select to navigate to reference" href="#cit113"><sup><span class="sup_ref">113,117,268,269</span></sup></a> For instance, Po-Cheng Chen <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> have developed a paper-based device using CuNCs aggregates and were able to detect hydrogen sulphide at a concentration of 650 nM. However, a highly selective and almost 65 times more sensitive bio-sensor was prepared by Lihua Jin <span class="italic">et al.</span> (LOD 10 nM).<a title="Select to navigate to references" href="#cit266"><sup><span class="sup_ref">266</span></sup></a> Excellent analytical performance and sensitivity were achieved by using GSH-CuNCs-carbon dot nanocomposites as a sensor for sulphide ions with the lowest LOD value of 4.3 nM.<a title="Select to navigate to references" href="#cit267"><sup><span class="sup_ref">267</span></sup></a> This is, so far, the most sensitive fluorometric detection probe for H<small><sub>2</sub></small>S based on highly fluorescent CuNCs.</span>
          <p class="otherpara">Iodine is an important micronutrient for human growth and metabolism. Both iodine deficiency and surplus are harmful for the human body and can cause severe diseases (increased infant mortality and hypothyroidism in the case of deficiency, and hyperthyroidism in the case of excess of iodine). Reported fatal doses of iodine during ingestion are 200 mg to 20 g. CuNCs-based sensors were applied for iodine detection in water and urine samples.<a title="Select to navigate to references" href="#cit272"><sup><span class="sup_ref">272</span></sup></a> Very recently Yu-Ting Tai <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit138"><sup><span class="sup_ref">138</span></sup></a> reported the detection of iodine in urine samples with LOD of 2.02 μM using fluorescent CuNCs. However, much earlier, Yaping Zhong <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> reported PEI-protected CuNCs with much higher sensitivity, and a LOD value of 100 nM. TA-CuNCs as an iodine biosensor further lowered the LOD to 18 nM.<a title="Select to navigate to references" href="#cit270"><sup><span class="sup_ref">270</span></sup></a> However, ssDNA-CuNCs showed excellent analytical performance for iodine with LOD as low as 15 nM and linear ranges of 0.05–40 μM and 40–80 μM with two different slopes.<a title="Select to navigate to references" href="#cit271"><sup><span class="sup_ref">271</span></sup></a></p>
          <p class="otherpara">Hypochlorous acid (HOCl) belongs to the family of ROSs, has good microbicidal activity in the cells and is involved in the fight against pathogens and infections. Apart from being an essential chemical for biological processes, an excess of the acid can cause different diseases like atherosclerosis, arthritis and even cancer. Thus, the precise determination and sensing of hypochlorous acid concentrations in water samples or biota are crucial and of great importance. CuNCs protected with PVP polymer are found to be excellent fluorescent probes for hypochlorite detection. In an earlier report by Qin Tang <span class="italic">et al.</span>, the PVP-protected CuNCs showed high sensitivity towards ClO<small><sup>−</sup></small> and a LOD of 0.1 μM.<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a> However, very recently, Weiru Dong <span class="italic">et al.</span> applied an iodide-enhanced approach to increase the detection sensibility and surpassed the previously reported LOD value (LOD 19 nM).<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a></p>
          <p class="otherpara">Nitrite (NO<small><sub>2</sub></small><small><sup>−</sup></small>) ions are a part of the nitrogen cycle and are involved in the oxidation of haemoglobin (Hb) to methaemoglobin (metHb). The impairment of oxygen transport due to the elevated concentrations of nitrite ions in the bloodstream leads to the development of a condition called methaemoglobinaemia. It is also believed to cause gastric cancer and other severe diseases. The WHO “<span class="italic">Guidelines for Drinking-water Quality</span>” set 1 mg L<small><sup>−1</sup></small> as the maximum contaminant level for nitrite ions in drinking water.<a title="Select to navigate to references" href="#cit403"><sup><span class="sup_ref">403</span></sup></a> Considering the impact of exposure to nitrite ions (short/long term), several research groups have attempted to outperform the detection limit normally obtained by routine spectroscopic analysis and monitoring of water quality (standard LOD of 0.01 to 1 mg L<small><sup>−1</sup></small>). Three independent research groups reported the applicability of CuNCs as a sensitive probe for nitrite ions. For instance, Jinshun Cang <span class="italic">et al.</span> developed a fluorometric sensor for the detection of nitrite and cyanide anions in water samples with LODs of 5 μM and 5 nM, respectively.<a title="Select to navigate to references" href="#cit273"><sup><span class="sup_ref">273</span></sup></a> Improved sensitivity was achieved by Suqin Han <span class="italic">et al.</span> with a LOD of 95.4 nM.<a title="Select to navigate to references" href="#cit274"><sup><span class="sup_ref">274</span></sup></a> Even better sensitivity was reported for stabilizer-free CuNCs as fluorometric probes for nitrite ions.<a title="Select to navigate to references" href="#cit275"><sup><span class="sup_ref">275</span></sup></a> A remarkable sensitivity of 3.6 nM was achieved with a probe operating in a wide analytical range (12.5 nM–125 μM).</p>
          <p class="otherpara">Phosphates are essential electrolytes/minerals for the body and are involved in many processes such as bone and teeth formation. The normal serum phosphate concentration ranges from 2.5 to 4.5 mg dL<small><sup>−1</sup></small> (0.81 to 1.45 mmol L<small><sup>−1</sup></small>). High phosphate levels (hyperphosphatemia) or very low phosphate levels (hypophosphatemia) in the blood indicate certain conditions, including chronic kidney disease, which require special medical attention.<a title="Select to navigate to references" href="#cit404"><sup><span class="sup_ref">404</span></sup></a> Taking advantage of their fluorescence properties, Haiyan Cao <span class="italic">et al.</span> demonstrated the applicability of TA-CuNCs as a turn-on sensor for phosphate ions in water samples with LOD of 9.6 nM.<a title="Select to navigate to references" href="#cit144"><sup><span class="sup_ref">144</span></sup></a> Six-fold enhancement in sensitivity for phosphate ions was achieved when polydopamine-CuNCs with AIE properties were applied as a fluorometric sensor.<a title="Select to navigate to references" href="#cit276"><sup><span class="sup_ref">276</span></sup></a> The minimum detectable phosphate ion concentration in tap water and human serum was 1.5 nM. Can Chen <span class="italic">et al.</span> developed poly(thymine)-CuNCs for the detection of pyrophosphate (PPi, P<small><sub>2</sub></small>O<small><sub>7</sub></small><small><sup>4−</sup></small>) ions in human serum samples with LOD of 5 nM.<a title="Select to navigate to references" href="#cit262"><sup><span class="sup_ref">262</span></sup></a></p>
          <p class="otherpara">The applications of CuNCs were also extended towards the detection of fluoride ions.<a title="Select to navigate to references" href="#cit265"><sup><span class="sup_ref">265</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect6898"/><span class="c_heading_indent">4.1.3. Hydrogen peroxide and glucose detection. </span>
          <span>It is well known that the level of glucose, a key energy source for living organisms, is controlled by insulin-mediated signalling and any perturbation in insulin secretion may result in high levels of blood glucose, which can cause type-2 diabetes, metabolic syndrome and hypertension and put the body at cardiovascular risk. Moreover, glucose can be oxidized to hydrogen peroxide and gluconic acid, catalysed by the enzyme glucose oxidase (GOx).<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgugt1"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-t1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-t1.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-t1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td><a id="ugt1"/><span id="ugt1"/></td><td class="pushTitleLeft"> </td></tr></table></div></span>
          <p class="otherpara">On the other hand, H<small><sub>2</sub></small>O<small><sub>2</sub></small>, a reactive oxygen species (ROS), is crucial for oxidative cellular signalling, it induces intracellular stress and accelerates aging. Hence, the early diagnosis of H<small><sub>2</sub></small>O<small><sub>2</sub></small> can prevent the development of certain diseases. Therefore, the detection of hydrogen peroxide is of great importance and CuNCs-based sensors have already been reported.<a title="Select to navigate to reference" href="#cit54"><sup><span class="sup_ref">54,87,90,114,127,142,279,280</span></sup></a> For example, Lianzhe Hu <span class="italic">et al.</span> reported the synthesis of BSA-capped CuNCs to detect hydrogen peroxide and glucose with a detectable minimum concentration or limit of detection (LOD) of 10 and 100 μM, respectively.<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> Hong Qun Luo and co-workers succeeded in the preparation of PEI-CuNCs,<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a> which could operate in a linear range of 0.5–10 μM and 10–100 μM for H<small><sub>2</sub></small>O<small><sub>2</sub></small> and glucose, respectively. The sensor could detect the target at concentrations as low as 0.4 and 8 μM for H<small><sub>2</sub></small>O<small><sub>2</sub></small> and glucose, respectively. Tingyao Zhou <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> and Xuxian Su <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit219"><sup><span class="sup_ref">219</span></sup></a> reported the synthesis of DHLA-CuNCs and <span class="small_caps">L</span>-cys/GOx-CuNCs, which showed better sensitivity towards hydrogen peroxide (LOD 0.3 μM)<a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> and glucose (LOD 1.5 μM),<a title="Select to navigate to references" href="#cit219"><sup><span class="sup_ref">219</span></sup></a> respectively. However, Hai-Bo Wang <span class="italic">et al.</span> reported the synthesis of dsDNA-templated CuNCs, which is so far the most sensitive fluorescent probe for the detection of hydrogen peroxide and glucose.<a title="Select to navigate to references" href="#cit277"><sup><span class="sup_ref">277</span></sup></a> The sensor can detect H<small><sub>2</sub></small>O<small><sub>2</sub></small> and glucose with LODs of 3 nM and 12 nM, respectively with a wide dynamic range (10 nM–2 μM for hydrogen peroxide and 50 nM–100 μM for glucose). In general, the detection and PL quenching of a sensor is believed to be caused by the oxidation of CuNCs by hydrogen peroxide.</p>
          <p class="otherpara">Yaping Zhong <span class="italic">et al.</span> made use of 6-thio-β-cyclodextrin (β-CD)-templated CuNCs as a peroxidase mimetic to substitute for GOx in biocatalytic glucose oxidation reaction.<a title="Select to navigate to references" href="#cit278"><sup><span class="sup_ref">278</span></sup></a> β-CD on the cluster surface acts as a pocket for host-guest interactions and provides a catalytic site for the non-enzymatic oxidation of glucose. Within the proposed method, the glucose and H<small><sub>2</sub></small>O<small><sub>2</sub></small> detection limits were 0.4 μM and 0.2 μM, respectively.</p>
        </div>
        <div>
          
          <span id="sect6932"/><span class="c_heading_indent">4.1.4. Small molecule detection. </span>
          <span>Apart from sensing metal ions, the applications of CuNCs were extended towards the sensing of small organic molecules, the higher than normal concentration of which is directly connected to the development of diseases.</span>
          
            
            <span id="sect6936"/><br/><span class="d_heading_indent">4.1.4.1. Acid detection. </span>
            <span>Picric acid (2,4,6-trinitrophenol, TNP) is a pale yellow and odourless solid with poor solubility in water. Normally, it is safe to handle when hydrated but it becomes extremely dangerous and explosive when dry. Besides, it readily forms even more sensitive explosives when in contact with many metals (copper, lead, mercury, zinc, <span class="italic">etc</span>.) when subjected to heat, friction, or impact. Upon inhalation or absorption through the skin, it can cause severe health problems. The determination of picric acid in drinking water and its impact on the environment became an issue for communities. In addition to the common detection methods, CuNCs have attracted much interest due to the fast, effective, and sensitive detection of TNP with low LOD values. So far, most of the reported (published in 2018) fluorometric CuNCs sensors are based on fluorescence quenching due to TNP. GSH-,<a title="Select to navigate to references" href="#cit282"><sup><span class="sup_ref">282</span></sup></a> DNA-,<a title="Select to navigate to references" href="#cit283"><sup><span class="sup_ref">283</span></sup></a> cysteamine-,<a title="Select to navigate to references" href="#cit284"><sup><span class="sup_ref">284</span></sup></a> and ascorbic acid-templated<a title="Select to navigate to references" href="#cit285"><sup><span class="sup_ref">285</span></sup></a> CuNCs were reported with LODs of 2.74 μM, 0.03 μM, 139 nM, 0.98 μM, respectively. The PL quenching of the clusters due to the electrostatic interaction between the electron acceptor (TNP) and electron donor clusters was ascribed to the FRET phenomenon in BSA-CuNCs.<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> The sensor was able to detect the lowest concentration of TNP (LOD 120 nM) as compared to the other sensors, with a larger analytical range (0.8–100 μM). In another report by Krishnamoorthy Shanmugaraj <span class="italic">et al.</span>, the reduction of fluorescence for <span class="small_caps">L</span>-cys-protected CuNCs<a title="Select to navigate to references" href="#cit286"><sup><span class="sup_ref">286</span></sup></a> was ascribed to the inner filter effect.</span>
            <p class="otherpara">Folic acid (FA), a synthetic form of Vitamin B, is very crucial for early development during pregnancy; it is involved in many physiological processes promoting the formation and regeneration of blood cells and preventing neurovascular defects. The importance of FA will be described in more detail in section 4.2.1. Two reports in the literature are devoted to the detection of folic acid using <span class="small_caps">L</span>-cys (chemiluminescence (CL) method)<a title="Select to navigate to references" href="#cit274"><sup><span class="sup_ref">274</span></sup></a> and ovalbumin-protected CuNCs (fluorometric method).<a title="Select to navigate to references" href="#cit287"><sup><span class="sup_ref">287</span></sup></a> Based on the changes in the CL intensity time profiles upon the addition of FA, the deducted LOD value was 69.8 nM and the method had an application range of 0.1–10 μM.<a title="Select to navigate to references" href="#cit274"><sup><span class="sup_ref">274</span></sup></a> A much wider linear response towards FA was obtained by fluorometric detection using ovalbumin-CuNCs (0.5–200 μM); however, the method was slightly less sensitive (LOD 0.18 μM).<a title="Select to navigate to references" href="#cit287"><sup><span class="sup_ref">287</span></sup></a></p>
            <p class="otherpara">Ascorbic acid (AA), also known as vitamin C, is a vital nutrient for the body and is involved in the production of collagen, in the metabolism of proteins, and in the formation of blood vessels to heal wounds and form scar tissue, <span class="italic">etc</span>. Ascorbic acid is widely used as a common reducing agent in cluster chemistry, and also became a detection target in biosensing. Among the commonly used diagnostic methods, a novel fluorometric method based on the application of CuNCs became very popular. Hanbing Rao <span class="italic">et al.</span> reported the applicability of TA-CuNCs as a turn-on fluorometric sensor for AA.<a title="Select to navigate to references" href="#cit143"><sup><span class="sup_ref">143</span></sup></a> The sensor could operate in the linear range of 0.5–10 μM with LOD of 0.11 μM. The sensing of humic acid (found in soil and water, LOD 0.4 mg L<small><sup>−1</sup></small>)<a title="Select to navigate to references" href="#cit281"><sup><span class="sup_ref">281</span></sup></a> and kojic acid (used in cosmetics and food colouring, LOD 0.07 μM)<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> were also successfully conducted using poly(thymine)- and BSA-CuNCs, respectively.</p>
            <p class="otherpara">The applicability of CuNCs was extended towards the sensing of the explosives trinitrotoluene (TNT)<a title="Select to navigate to references" href="#cit288"><sup><span class="sup_ref">288</span></sup></a> and <span class="italic">m</span>-dinitrobenzene (<span class="italic">m</span>-DNB)<a title="Select to navigate to references" href="#cit289"><sup><span class="sup_ref">289</span></sup></a> as analogue molecules to picric acid. The LODs were 14 pM and 0.13 μM, respectively.</p>
          
          
            
            <span id="sect6967"/><br/><span class="d_heading_indent">4.1.4.2. Organic dyes and toxins detection. </span>
            <span>Sudan dyes are synthetic red azo dyes that were widely used in foodstuff and commercial manufacturing for colouring purposes. However, they are now classified as potential carcinogens and are banned from use in many countries. Nevertheless, the contamination of food with Sudan dyes continues to be a problem. Even though several tests have already been developed for these kinds of dyes (formally known as industrial solvent dyes), fast and extremely efficient and sensitive fluorometric sensors are still in demand. Yu Ling <span class="italic">et al.</span> prepared PEI-CuNCs as a sensitive and selective fluorescence probe for Sudan I–IV dyes based on the PL quenching of blue-emitting clusters (as a consequence of FRET).<a title="Select to navigate to references" href="#cit27"><sup><span class="sup_ref">27</span></sup></a> With the developed sensing principle, the LODs for Sudan I–IV dyes were 65 nM, 70 nM, 45 nM and 50 nM, respectively.</span>
            <p class="otherpara">Quinoline yellow is another artificial colouring dye used in the food and beverage market to impart an intense greenish-yellow colour. However, some countries including Canada and the USA have banned its use and other countries have agreed to reduce the limit of daily intake to several micrograms. Thus, the regular monitoring of the quinoline yellow concentrations in food and drinks is of great importance for food safety administration. Unni Sivasankaran <span class="italic">et al.</span> applied <span class="small_caps">L</span>-cys-CuNCs as a fluorometric probe to detect the amount of the dye present in soft drinks and sweets.<a title="Select to navigate to references" href="#cit122"><sup><span class="sup_ref">122</span></sup></a> The developed sensor could operate in a range of 0.2–5.5 μM with LOD of 0.11 μM.</p>
            <p class="otherpara">4-Aminoazobenzene also known as aniline yellow, is a textile dye mainly used to colour lacquer, varnish, wax products, oil stains and styrene resins. It is among the 24 prohibited textile dyes and is considered as a carcinogen. Thus, the detection of aniline yellow in water samples is of crucial importance. For monitoring the water contamination with the dye, Qiang Li <span class="italic">et al.</span> have prepared a highly luminescent (QY of 33.6%) CuNCs-based turn-off fluorometric sensor with LOD of 1.44 μM.<a title="Select to navigate to references" href="#cit290"><sup><span class="sup_ref">290</span></sup></a></p>
            <p class="otherpara">Ochratoxin A (OTA) is a natural mycotoxin produced by several fungi. The poor storage of agricultural commodities such as cereal grains, dried fruits and drinks can lead to the growth of fungi and consequently, release OTA into foodstuff. Thus, the detection of OTA toxin is of major concern for food safety regulations. Chunxia Song <span class="italic">et al.</span> have developed dsDNA-CuNCs to detect OTA in corn samples with LOD of 5 ng mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit291"><sup><span class="sup_ref">291</span></sup></a> Yue He <span class="italic">et al.</span> prepared polyT-CuNCs as a fluorometric sensor for OTA in spiked diluted red wine with a LOD of 2 nM.<a title="Select to navigate to references" href="#cit292"><sup><span class="sup_ref">292</span></sup></a></p>
            <p class="otherpara">Another foodborne toxin called aflatoxin B1 (AFB1) is produced by fungi and is found in food crops. The ingestion of products contaminated with AFB1 is potentially life-threatening. The WHO classifies AFB1 as a potent carcinogen that can affect all organ systems.<a title="Select to navigate to references" href="#cit405"><sup><span class="sup_ref">405</span></sup></a> A plethora of methods is now available to detect AFB1 and related toxins, however, more sensitive and selective tests are yet to be developed. For this reason, Ying Xiong <span class="italic">et al.</span> developed a novel fluorometric sensor for AFB1, utilizing histone-ds-poly(AT)-CuNCs.<a title="Select to navigate to references" href="#cit293"><sup><span class="sup_ref">293</span></sup></a> With the designed sensing strategy, the minimum detectable AFB1 concentration was 0.15 pg mL<small><sup>−1</sup></small> with a rather large analytical range of 0.46–400 pg mL<small><sup>−1</sup></small>.</p>
            <p class="otherpara">Microcystin-leucine-arginine (MC-LR) is the most toxic among the microcystins produced by cyanobacteria in water systems. The contamination of water ecosystems with MC-LR represents a global problem and can cause a series of related health problems. Apart from traditional methods, Yanli Zhang <span class="italic">et al.</span> have prepared novel fluorometric probes based on hairpin-CuNCs<a title="Select to navigate to references" href="#cit294"><sup><span class="sup_ref">294</span></sup></a> and dsDNA-CuNCs.<a title="Select to navigate to references" href="#cit295"><sup><span class="sup_ref">295</span></sup></a> The developed sensors were able to detect the toxin in real water samples with LODs of 3 pg L<small><sup>−1</sup></small> and 4.8 ng L<small><sup>−1</sup></small>, respectively.</p>
          
          
            
            <span id="sect6998"/><br/><span class="d_heading_indent">4.1.4.3. Industrial and agricultural products detection. </span>
            <span>Industrial safety is one of the top challenges to ensure a smooth-running operation in the workplace and limit the exposure to toxic chemicals. Although high safety standards are fundamental for every aspect of production in many countries, many industrial chemicals, even at low concentrations, remain a risk and present a danger to workers and customers. Among the potentially risky chemicals is bisphenol A, which is widely used in the production of polycarbonate, epoxy resins and plastic bottles. However, at high temperatures and pressures, the released bisphenol A from plastic bottles contaminates the water, which presents a potential risk related to cancers and cardiovascular diseases. Shuangjiao Xu <span class="italic">et al.</span> reported the applicability of BSA-CuNCs as a chemiluminescence probe to detect bisphenol A in water samples with LOD of 0.12 nM.<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a></span>
            <p class="otherpara">Melamine has many industrial applications including the production of melamine resins, laminates, glues, fertilizers and in food, for example in milk to increase the apparent protein content (melamine is a nitrogen-rich compound). However, high concentrations or exposure to melamine causes inflammatory reactions and hyperplasia in the urinary bladder in animals. Although no apparent data are available about the toxicity in humans, the WHO guidelines strongly limit the tolerable daily intake of melamine in animal and infant food.<a title="Select to navigate to references" href="#cit406"><sup><span class="sup_ref">406</span></sup></a> Therefore, it is of crucial importance to regularly monitor the amount of melamine and its analogues in food. Hong-Wei Zhu <span class="italic">et al.</span> applied poly(thymine)-CuNCs as a fluorometric probe for the determination of melamine in milk products.<a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> The designed probe showed excellent analytical performance with the highest reported sensitivity towards melamine, so far (LOD of 95 nM).</p>
            <p class="otherpara">Hydrazine is highly used in materials synthesis as a reducing agent. Its industrial applications range from the preparation of polymer foams to pesticides, it is also a precursor for many pharmaceuticals and has recently been used as a rocket fuel (as a safer alternative to hydrogen). As a chemical, it is highly toxic and can cause irritation to the eyes and skin, and damage the liver and kidneys. Thus, the detection of hydrazine in the environment is of great importance. Xiaohui Gao <span class="italic">et al.</span> studied the electrocatalytic performance of mercaptobenzoic acid (MBA)-CuNCs loaded on activated carbon to oxidize and detect hydrazine.<a title="Select to navigate to references" href="#cit125"><sup><span class="sup_ref">125</span></sup></a> From the linear curve fitting between the current density and the hydrazine concentration, the LOD value of 1.04 μM was successfully derived.</p>
            <p class="otherpara">Among the industrial amines, <span class="italic">o</span>-phenylenediamine (OPD) was widely used in cosmetics; however, it has been banned due to its toxicity, carcinogenicity, and possible disruption of genetics. Motivated by these issues, Yujun Ma <span class="italic">et al.</span> have developed GSH-CuNCs as a fluorometric probe to detect OPD in rivers and textile dye wastewater samples.<a title="Select to navigate to references" href="#cit296"><sup><span class="sup_ref">296</span></sup></a> The developed sensor was very sensitive to OPD, with a detection limit of 93 ng L<small><sup>−1</sup></small>.</p>
            <p class="otherpara">Recently, Zhifeng Cai <span class="italic">et al.</span> reported the applicability of <span class="small_caps">L</span>-histidine-CuNCs as a potent fluorometric probe for the detection of fluazinam (agricultural fungicide) in lake and tap waters.<a title="Select to navigate to references" href="#cit297"><sup><span class="sup_ref">297</span></sup></a> The sensor could operate in a wide analytical range of 1–40 μM with LOD of 0.25 μM. Kaimei Fan <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit298"><sup><span class="sup_ref">298</span></sup></a> and Yue Yang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit299"><sup><span class="sup_ref">299</span></sup></a> applied CuNCs for the detection of neonicotinoid insecticides such as acetamiprid (brand name: Assail) and dinotefuran in water and food samples. The lowest detectable amounts of acetamiprid and dinotefuran were 2.37 nM (ref. <a title="Select to navigate to references" href="#cit298">298</a>) and 7.04 μM (ref. <a title="Select to navigate to references" href="#cit299">299</a>) using dsDNA- and GSH-CuNCs-based fluorometric probes, respectively. CTAB-encapsulated CuNCs were successfully applied in the detection of metam sodium (fungicide) in real samples with a LOD of 0.63 mg kg<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit300"><sup><span class="sup_ref">300</span></sup></a></p>
          
          
            
            <span id="sect7028"/><br/><span class="d_heading_indent">4.1.4.4. Bilirubin/biliverdin detection. </span>
            <span>Bilirubin (yellowish compound, strong antioxidant) is a metabolite produced in the catabolic pathway of the heme breakdown to clear the body from waste products due to the death of aging red blood cells. The catabolism of heme proteins happens under the action of heme oxygenase to produce biliverdin (green tetrapyrrolic bile pigment), which is later reduced to bilirubin by biliverdin reductase. It is important for the body to excrete bilirubin as it is potentially toxic. High bilirubin levels indicate an unusual rate of red blood cell breakdown or that the liver is not clearing the bilirubin from the bloodstream properly. With this in mind, Sanu K. Anand <span class="italic">et al.</span> prepared <span class="small_caps">L</span>-cys-CuNCs as a point of care analysis probe for the detection of both biliverdin and bilirubin.<a title="Select to navigate to references" href="#cit120"><sup><span class="sup_ref">120</span></sup></a> The fluorescence quenching of the clusters’ emission upon the gradual addition of bile pigments was ascribed to the static quenching process. The limits of detection for biliverdin and bilirubin with the designed sensor were 233 nM and 229 nM, respectively. When the human serum albumin (HSA) was used as a templating matrix for the preparation of clusters, the resulting red emissive HSA-CuNCs could detect the bilirubin in two linear concentration ranges with LODs of 35 nM and 145 nM.<a title="Select to navigate to references" href="#cit301"><sup><span class="sup_ref">301</span></sup></a> Although this sensor significantly improved the detection sensitivity (with respect to laboratory tests), it was further greatly increased when BSA was implemented as a template. With the assistance of Fe<small><sup>3+</sup></small>, the fluorescence quenching of the system can be restored by the addition of bilirubin by taking advantage of its antioxidant properties.<a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> The minimum detectable bilirubin concentration was 6.62 nM. Without the addition of Fe<small><sup>3+</sup></small>, the BSA-CuNCs showed excellent sensing properties for bilirubin with LOD as low as 257 fM.<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a></span>
          
          
            
            <span id="sect7040"/><br/><span class="d_heading_indent">4.1.4.5. Neurotransmitter detection. </span>
            <span>Neurotransmitters are molecules used by the nervous system to transmit messages between neurons, or from neurons to muscles. Most neurotransmitters are either small amines, amino acids, or neuropeptides. Many neuromodulators, such as dopamine (catecholamine family), are monoamines. Dopamine is involved in many functions, including motor control, reward and reinforcement, and motivation. Histamine, one of the major monoamines, plays a role in metabolism, temperature control, regulating various hormones, and controlling the sleep-wake cycle, among other functions. Dopamine deficiency is related to certain medical conditions, including depression and Parkinson's disease. Thus, the detection of dopamine levels for the diagnosis of neurological diseases is of great importance. High levels of histamine are associated with allergic reactions and the disruption of normal bodily functions. Normal dopamine and histamine levels in the blood are 0 to 30 pg mL<small><sup>−1</sup></small> (195.8 pmol L<small><sup>−1</sup></small>) and 25–65 ng mL<small><sup>−1</sup></small>, respectively. For the detection of dopamine levels in both serum and urine samples, CuNCs prepared in various templating matrices have been applied as a sensing probe. Among them, BSA-CuNCs were the most applied. R. S. Aparna <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> Zhuang Miao <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit303"><sup><span class="sup_ref">303</span></sup></a> and Weijie He <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit304"><sup><span class="sup_ref">304</span></sup></a> reported the applicability of BSA-CuNCs as a sensor to detect dopamine with LODs of 0.1637 pM, 0.28 pM and 32 nM, respectively. dsDNA-CuNCs<a title="Select to navigate to references" href="#cit302"><sup><span class="sup_ref">302</span></sup></a> were also applied as a dopamine sensor, however, the sensitivity was lower as compared to BSA-CuNCs. Ailing Han <span class="italic">et al.</span> prepared 2,3,5,6-tetrafluorothiophenol (TFTP)-protected CuNCs with self-assembled induced emission (SAIE) properties (QY 43%) and tested them for histamine sensing.<a title="Select to navigate to references" href="#cit229"><sup><span class="sup_ref">229</span></sup></a> The developed sensor could detect histamine at its lowest concentration of 60 nM in the range of 0.1–10 μM. Furthermore, the histamine detection sensitivity was increased twice (LOD 30 nM) when <span class="small_caps">D</span>-penicillamine-CuNCs were applied as a fluorometric sensor with AIE enhancement properties.<a title="Select to navigate to references" href="#cit305"><sup><span class="sup_ref">305</span></sup></a></span>
          
          
            
            <span id="sect7058"/><br/><span class="d_heading_indent">4.1.4.6. Drugs, antibiotics, and other pharmaceuticals detection. </span>
            <span>Heparin (Hep) is a naturally occurring, highly sulphated polysaccharide (highly negatively charged macromolecule) that is widely used as a potent anticoagulant therapeutic drug for the treatment and prevention of thrombotic diseases and for maintaining blood fluidity in extracorporeal devices. Bleeding and heparin-induced thrombocytopenia are the main adverse reactions associated with heparin therapy. Laboratory monitoring of the antithrombotic efficacy of heparin can reduce the risk of haemorrhage. Thus, a fast and efficient clinical test for heparin is highly desired. Taking advantage of the fluorescence properties of CuNCs, they were applied as a point-of-care diagnostic sensor to monitor the heparin concentrations in serum samples. Bo Yin <span class="italic">et al.</span> made use of GSH-CuNCs attached to the surface of functionalized reduced graphene oxide (RGO) in the form of a CuNCs/RGO nanocomposite as a turn-on sensor for heparin.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a> The weakly emissive nanocomposite (due to the photoinduced electron transfer from CuNCs to RGO) became highly emissive upon the addition of Hep. The competitive interaction of heparin with RGO released the CuNCs from the composite and restored its original fluorescence. Thus, by following the PL intensity enhancement of the sensor, the concentrations of heparin can be quantitively determined. As such, with the above-mentioned sensing principle, the minimum detectable amount of Hep was found to be 26 nM.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a> In contrast, BSA-CuNCs were applied as a turn-off sensor for Hep.<a title="Select to navigate to reference" href="#cit306"><sup><span class="sup_ref">306,307</span></sup></a> The quenching of the PL exhibited a linear correlation upon the gradual increase in heparin concentration. Moreover, the sensing of Hep was very much dependent on whether denatured or native BSA was used to template CuNCs. With denatured dBSA-CuNCs as a sensor, the lowest detectable Hep concentration was 0.26 ng mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit306"><sup><span class="sup_ref">306</span></sup></a> However, the native BSA-CuNCs showed much better analytical performance with LOD of 0.0406 ng mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit307"><sup><span class="sup_ref">307</span></sup></a></span>
            <p class="otherpara">The applicability of CuNCs was extended towards the sensing of other drugs such as carbamazepine (anticonvulsant drug, used to reduce nerve impulses during seizures and nerve pain) with LOD of 0.08 μg mL<small><sup>−1</sup></small>,<a title="Select to navigate to references" href="#cit308"><sup><span class="sup_ref">308</span></sup></a> and paraoxon (ophthalmological drug against glaucoma) with LOD of 12.8 nM.<a title="Select to navigate to references" href="#cit309"><sup><span class="sup_ref">309</span></sup></a></p>
            <p class="otherpara">Antibiotics and similar drugs are used to treat bacterial infections by killing or inhibiting the growth of disease-causing microorganisms. However, the inappropriate use or overuse of antibiotic drugs can cause the microorganisms to develop resistance to the antibiotics, which is a serious problem. Several techniques have been developed and are widely applied in clinical trials to detect the residues of different antibiotics in the environment. Tetracyclines are a class of antibiotics used to treat respiratory tract infections and certain infections of the skin, eye, lymphatic, intestinal and urinary systems. Zhuosen Wang <span class="italic">et al.</span> have developed <span class="small_caps">L</span>-cys- and GSH-CuNCs as tetracycline sensors with LODs of 5.6 μM and 8.4 μM, respectively.<a title="Select to navigate to references" href="#cit310"><sup><span class="sup_ref">310</span></sup></a> Ovalbumin- and GSH-CuNCs, on the other hand, were applied as sensors for doxycycline with LOD of 270 nM (ref. <a title="Select to navigate to references" href="#cit312">312</a>) as well as for the detection of vitamin B1 with LODs of 380 pM (ref. <a title="Select to navigate to references" href="#cit312">312</a>) and 4.6 nM.<a title="Select to navigate to references" href="#cit311"><sup><span class="sup_ref">311</span></sup></a> CuNCs have also been also applied to detect nitrofurantoin (also used as a fungicide, LOD 30 nM),<a title="Select to navigate to references" href="#cit313"><sup><span class="sup_ref">313</span></sup></a><span class="small_caps">D</span>-penicillamine (LOD 0.54 μg mL<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit314"><sup><span class="sup_ref">314</span></sup></a> choline (LOD 25 nM),<a title="Select to navigate to references" href="#cit315"><sup><span class="sup_ref">315</span></sup></a> rutin (LOD 0.02 μM),<a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a> gossypol (LOD 25 nM),<a title="Select to navigate to references" href="#cit316"><sup><span class="sup_ref">316</span></sup></a> hemin (brand name: Panhematin, LODs 0.045 μM (ref. <a title="Select to navigate to references" href="#cit244">244</a>) and 68 nM (ref. <a title="Select to navigate to references" href="#cit248">248</a>)), quercetin (LODs 0.19 μM (ref. <a title="Select to navigate to references" href="#cit317">317</a>) and 12 nM (ref. <a title="Select to navigate to references" href="#cit318">318</a>)), acetaminophen (brand name: Paracetamol, LOD 0.26 μM),<a title="Select to navigate to references" href="#cit319"><sup><span class="sup_ref">319</span></sup></a> tramadol (LOD 0.8 nM)<a title="Select to navigate to references" href="#cit320"><sup><span class="sup_ref">320</span></sup></a> and mangiferin (LOD 210 nM).<a title="Select to navigate to references" href="#cit321"><sup><span class="sup_ref">321</span></sup></a></p>
          
          
            
            <span id="sect7095"/><br/><span class="d_heading_indent">4.1.4.7. Disease marker detection. </span>
            <span>Creatinine is a waste product of muscle tissue breakdown; it is filtered through the kidneys and excreted in the urine. The creatinine levels in the blood, as well as the rate of creatinine clearance, indicate the kidney function and test for kidney diseases. Clinical tests for creatinine are time-consuming, and novel, fast methods with high precision are still in demand. Roghayeh Jalili <span class="italic">et al.</span> have reported the use of GSH-CuNCs as a creatinine sensor taking advantage of the Al<small><sup>3+</sup></small>-triggered AIE of the original clusters.<a title="Select to navigate to references" href="#cit222"><sup><span class="sup_ref">222</span></sup></a> The sensing is based on the fluorescence quenching of the signal upon the addition of creatinine due to the coordination with Al<small><sup>3+</sup></small>. A linear dependence of PL <span class="italic">versus</span> concentration has been observed with LOD of 0.63 μg L<small><sup>−1</sup></small>. A LOD of 50 nM was reached when BSA-CuNCs were implemented as a turn-off sensing platform for creatinine, which is directly applicable for clinical samples.<a title="Select to navigate to references" href="#cit322"><sup><span class="sup_ref">322</span></sup></a></span>
            <p class="otherpara">Cholesterol is an essential substance for the body, and it is involved in the production of hormones and vitamin D, as well as in the digestion of food. Nevertheless, the high levels of cholesterol in the blood (mostly due to the diet) represent a silent danger and can cause heart attacks (due to the build-up of plaques that can block or narrow arteries). Thus, the analysis of cholesterol levels in the blood is a prerequisite for the assessment of heart disease. Although clinically available cholesterol tests called a lipid panel or a lipid profile are quite precise for the determination of the cholesterol levels in the blood, with the advancement of technology, novel and fast tests could be an attractive alternative. Considering the necessity to develop novel fluorometric tests, Fei Qu <span class="italic">et al.</span> developed a turn-off sensor based on the AIE of GSH-CuNCs.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a> The synergistic effect of Pb<small><sup>2+</sup></small> and Zr<small><sup>4+</sup></small> on GSH-CuNCs greatly enhanced the PL properties and the latter was applied in the detection of cholesterol with the detection limit down to 2.7 μM.</p>
          
          
            
            <span id="sect7111"/><br/><span class="d_heading_indent">4.1.4.8. Biothiol and biogenic amino acid detection. </span>
            <span>Cysteine and lysine are among 20 naturally occurring biogenic amino acids that serve as building blocks to form polypeptides and proteins. Like the other amino acids, cysteine and lysine are abundant in the L-form. Homocysteine (Hcy) is a sulphur-containing amino acid that is formed during the metabolism of methionine (Met, essential amino acid) to cysteine (Cys). These biothiols are necessary for many bodily functions. A high homocysteine level, also called hyperhomocysteinemia, can contribute to arterial damage and blood clots in blood vessels and can indicate a deficiency in vitamin B-12 or folate. The concentration of total cysteine (tCys) in serum/plasma is normally 250 μmol L<small><sup>−1</sup></small>, which is 20-fold higher than the plasma tHcy level. Hyperlysinemia is a genetic condition caused by the deficiency of the aminoadipic semialdehyde synthase enzyme that breaks down the lysine. Glutathione is a naturally occurring tripeptide composed of glutamine, cysteine, and glycine. The thiol group from the Cys is a potent reducing agent. As an antioxidant, GSH plays a role in detoxification, cellular reactions, the regulation of protein and gene expressions, <span class="italic">etc</span>. A deficiency of GSH puts the cell at risk for oxidative damage. Several clinically proved methods are used for the detection of biothiols in the blood, including chromatographic and mass spectrometric methods. However, a facile and rapid detection method is still required for faster clinical diagnosis. On this note, Tianzi Li <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit323"><sup><span class="sup_ref">323</span></sup></a> and Yu-e Shi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit324"><sup><span class="sup_ref">324</span></sup></a> implemented the PVP-CuNCs for the detection of GSH in human serum samples with LODs of 17 μM and 3 μM, respectively. Yihui Hu <span class="italic">et al.</span> have developed a dsDNA-templated CuNCs as a fluorescence probe for the detection of GSH, Cys and Hcy in biological fluids.<a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a> The LODs for GSH and Cys were 2 μM, and for Hcy it was 5 μM. Jinlan Yang <span class="italic">et al.</span> developed the UV-light-induced synthesis of PEI-CuNCs and applied them in the detection of GSH and Cys with LODs of 0.26 μM and 0.34 μM, respectively.<a title="Select to navigate to references" href="#cit254"><sup><span class="sup_ref">254</span></sup></a> However, much better sensitivity and analytical performance for biothiols were achieved when poly(thymine)-CuNCs were applied as a probe.<a title="Select to navigate to references" href="#cit325"><sup><span class="sup_ref">325</span></sup></a> The minimum detectable concentrations of GSH, Cys and Hcy were 15 nM, 12.5 nM and 20 nM, respectively. Another biothiol sensor based on DNA nanoribbon-templated CuNCs with QY as high as 12.3% has been applied for the detection of GSH and Cys.<a title="Select to navigate to references" href="#cit326"><sup><span class="sup_ref">326</span></sup></a> Within the developed sensing strategy, the lowest detectable concentrations of analytes were 0.43 and 0.31 μM for GSH and Cys, respectively. A detection platform for biogenic amino acids was also established using templated CuNCs. For example, Xiangning Bu <span class="italic">et al.</span> developed a hybrid DNA-CuNCs/carbon dots sensor for the detection of arginine with LOD of 0.35 μM.<a title="Select to navigate to references" href="#cit319"><sup><span class="sup_ref">319</span></sup></a> Mingming Zhang <span class="italic">et al.</span> have developed ovalbumin-CuNCs as a turn-on fluorescence probe to detect <span class="small_caps">L</span>-lysine with LOD of 5.5 μM for a wide analytical range (10 μM–1 mM).<a title="Select to navigate to references" href="#cit327"><sup><span class="sup_ref">327</span></sup></a> Yasaman-Sadat Borghei <span class="italic">et al.</span> applied Cys-CuNCs as a chemiluminescence probe for the detection of tryptophan and phenylalanine with LODs of 75 nM and 840 nM, respectively.<a title="Select to navigate to references" href="#cit328"><sup><span class="sup_ref">328</span></sup></a></span>
          
          
            
            <span id="sect7134"/><br/><span class="d_heading_indent">4.1.4.9. Nucleosides/nucleotides detection. </span>
            <span>Nucleosides are a structural subunit of nucleic acids, consisting of a molecule of sugar (ribose or deoxyribose) linked to a nitrogen-containing organic ring compound (pyrimidine-based: cytosine, thymine, uracil, or purine-based: adenine or guanine). Nucleotides are composed of nucleosides and phosphate groups. Apart from their intracellular functions as building blocks for the genetic code, nucleosides and nucleotides play an important role in extracellular compartments as signalling molecules to regulate various disease outcomes including inflammation, ischemia and reperfusion injury, infectious diseases, <span class="italic">etc</span>. Yong Wang <span class="italic">et al.</span> prepared adenosine, cytidine, and guanosine-protected CuNCs with blue emissions, where the QY of the clusters decreased in the above series of nucleosides.<a title="Select to navigate to references" href="#cit329"><sup><span class="sup_ref">329</span></sup></a> More importantly, by implementing the multivariate chemometrics analysis (principal component analysis (PCA) and hierarchical cluster analysis (HCA)), the authors were able to discriminate between different nucleosides. The detection limits for all three mentioned nucleosides were 20 μM for the range of 20 μM–0.5 mM.<a title="Select to navigate to references" href="#cit329"><sup><span class="sup_ref">329</span></sup></a> For the detection of nucleotides, specifically ATP and GTP, DNA,<a title="Select to navigate to reference" href="#cit330"><sup><span class="sup_ref">330,331</span></sup></a> poly(thymine),<a title="Select to navigate to references" href="#cit332"><sup><span class="sup_ref">332</span></sup></a> GSH<a title="Select to navigate to references" href="#cit333"><sup><span class="sup_ref">333</span></sup></a> and histidine<a title="Select to navigate to references" href="#cit147"><sup><span class="sup_ref">147</span></sup></a> were applied as templating ligands. The LODs for ATP using DNA as a template were 93 nM (ref. <a title="Select to navigate to references" href="#cit330">330</a>) and 5 pM;<a title="Select to navigate to references" href="#cit331"><sup><span class="sup_ref">331</span></sup></a> with poly(thymine)-CuNCs,<a title="Select to navigate to references" href="#cit332"><sup><span class="sup_ref">332</span></sup></a> and GSH-CuNCs<a title="Select to navigate to references" href="#cit333"><sup><span class="sup_ref">333</span></sup></a> the LOD values were 10.29 pM and 0.67 μM, respectively. Jia Ge <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit334"><sup><span class="sup_ref">334</span></sup></a> and Yinan Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit335"><sup><span class="sup_ref">335</span></sup></a> applied dsDNA-CuNCs and hpDNA-CuNCs for the detection of nicotinamide adenine dinucleotide (NAD<small><sup>+</sup></small>) with LODs of 0.2 nM and 5 nM, respectively.</span>
          
        </div>
        <div>
          
          <span id="sect7155"/><span class="c_heading_indent">4.1.5. RNA detection. </span>
          <span>MicroRNAs (miRNAs) are small non-coding RNA molecules that can turn off the genes (gene silencing) by inactivating messenger RNAs (mRNAs), which are necessary for translating the genetic information into proteins (central dogma). miRNAs participate in cell regulation from its development to its death. The dysregulation can have serious consequences for the body and can cause a range of diseases including cancer and heart disease. Having such a big impact, it is imperative to develop highly sensitive miRNA probes. To date, the majority of the reported CuNCs-based miRNA sensors operate in a wide dynamic range with LODs in the picomolar region. For instance, Yiting Li <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit339"><sup><span class="sup_ref">339</span></sup></a> and Yasaman-Sadat Borghei <span class="italic">et al.</span><a title="Select to navigate to reference" href="#cit342"><sup><span class="sup_ref">342,343</span></sup></a> have prepared a series of DNA-templated CuNCs that can detect the miRNAs at LODs of 18.7, 0.6 and 11 pM, respectively. Fengzhou Xu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit76"><sup><span class="sup_ref">76</span></sup></a> and Xin-Ping Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> prepared other DNA-templated CuNCs with LOD values of 10 pM and 1 pM, respectively. Poly(thymine)-CuNCs, as a fluorometric probe, were able to detect miRNA with LOD of 44 fM.<a title="Select to navigate to references" href="#cit341"><sup><span class="sup_ref">341</span></sup></a> Zhihui Dai and co-workers<a title="Select to navigate to references" href="#cit81"><sup><span class="sup_ref">81</span></sup></a> and Yuzhong Zhang and co-workers<a title="Select to navigate to references" href="#cit345"><sup><span class="sup_ref">345</span></sup></a> prepared miRNA sensors based on the electrocatalytic properties of DNA-templated CuNCs with LOD values of 8.2 fM and 0.03 fM, respectively. Furthermore, the detection sensitivity of various miRNAs has been improved by several orders of magnitude to the aM range with LODs from 36 aM (ref. <a title="Select to navigate to references" href="#cit344">344</a>) to 19.05 aM,<a title="Select to navigate to references" href="#cit338"><sup><span class="sup_ref">338</span></sup></a> and as low as 10 aM (ref. <a title="Select to navigate to references" href="#cit340">340</a>) with a wide analytical performance range.</span>
        </div>
        <div>
          
          <span id="sect7173"/><span class="c_heading_indent">4.1.6. Enzymatic activity detection. </span>
          
            
            <span id="sect7176"/><br/><span class="d_heading_indent">4.1.6.1. β-Galactosidase activity detection. </span>
            <span>β-Galactosidase or lactase (shortly called β-gal, EC 3.2.1.23) is a glycoside hydrolase enzyme that catalyses the hydrolysis of β-galactosides. It is involved in the glycolysis pathway for breaking down the glycosidic bond in lactose into galactose and glucose. β-Gal is important for energy production and is used as a marker for gene expression and target for cancers. Thus, the detection of its enzymatic activity in cells is crucial. Among other commercially available assays for the detection of β-gal activity, copper clusters were also applied as a fluorometric sensor to probe and monitor the β-gal activity, taking advantage of the AIE properties of clusters. For instance, the hydrophobicity-controlled assembly of 4-methylthiophenol-protected CuNCs with AIE characteristics was applied for the β-gal activity detection under physiological conditions.<a title="Select to navigate to references" href="#cit141"><sup><span class="sup_ref">141</span></sup></a> The highly emissive cluster assemblies (QY of 9%) as a sensor could operate in a wide analytical range of 2.5–212 U L<small><sup>−1</sup></small> with LOD of 0.9 U L<small><sup>−1</sup></small> (1 U is the amount of enzyme that catalyses the reaction of 1 μmol substrate per minute). However, when the Al<small><sup>3+</sup></small>-driven self-assembly of GSH-protected clusters (QY of 8.1%) was applied as a sensor, the analytical performance range was slightly narrower (2.3–96 U L<small><sup>−1</sup></small>), yet an improved LOD value (0.7 U L<small><sup>−1</sup></small>) was achieved.<a title="Select to navigate to references" href="#cit227"><sup><span class="sup_ref">227</span></sup></a></span>
          
          
            
            <span id="sect7187"/><br/><span class="d_heading_indent">4.1.6.2. Exonuclease III activity detection. </span>
            <span>The exonuclease III (Exo III, EC 3.1.11.2) enzyme from the exonuclease family catalyses DNA degradation and the removal of mononucleotides from the 3′-hydroxyl termini of double-stranded DNA. It is involved in the damage-control and repair of DNA strand breaks, contributing to DNA fidelity, <span class="italic">etc</span>. As a matter of fact, the detection of Exo III activity is crucial for disease diagnosis, especially in the early stages. Alternative methods for Exo III activity detection include the use of fluorescent metal nanoclusters, in particular CuNCs. A label-free fluorescence probe based on dsDNA-templated CuNCs has been achieved and applied as a turn-off fluorometric sensor for Exo III. Two groups have independently prepared dsDNA-CuNCs sensors for Exo III.<a title="Select to navigate to reference" href="#cit65"><sup><span class="sup_ref">65,346</span></sup></a> In both cases, the detection limit was 0.02 U mL<small><sup>−1</sup></small>, however, the analytical range varied significantly, <span class="italic">e.g.</span>, 0.05–2 U mL<small><sup>−1</sup></small> in ref. <a title="Select to navigate to references" href="#cit65">65</a> and 0.02–10 U mL<small><sup>−1</sup></small> in ref. <a title="Select to navigate to references" href="#cit346">346</a>.</span>
          
          
            
            <span id="sect7199"/><br/><span class="d_heading_indent">4.1.6.3. Methyltransferase activity detection. </span>
            <span>Methyltransferase enzymes (MTases, family EC 2.1.1) regulate the methylation of substrates, which is as an epigenetic modification that is very important for gene transcription as well as gene silencing/activation. Abnormal methylation events of DNA provide a basis for the development of diseases. Apart from the well-established colorimetric, immune-based techniques for the MTase activity detection, nanomaterial-based fluorometric probes recently became very popular. Jingxian Yin <span class="italic">et al.</span> prepared a biosensor for MTase activity detection based on the utilization of CuNCs.<a title="Select to navigate to references" href="#cit349"><sup><span class="sup_ref">349</span></sup></a> Dumbbell DNA with a locked circle consisting of two poly T loops and a poly (AT-TA) stem has been designed as an effective template. In the presence of MTase, the digestion of the DNA strands by exo/endonucleases has been prevented and the PL signal was observed. However, the enzymatic digestion of the DNA was not avoidable in the absence of MTase; thus, with the degradation of the template, the PL of the cluster cannot be seen. Based on the proposed detection method, the CuNCs-based sensor could operate in the linear range of 0.1–0.2 U μL<small><sup>−1</sup></small> with LOD of 0.17 mU μL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit349"><sup><span class="sup_ref">349</span></sup></a> The reverse strategy has been implemented by Dengpeng Gao <span class="italic">et al.</span> who designed another label-free and sensitive fluorescence method for the detection of DNA MTase activity.<a title="Select to navigate to references" href="#cit350"><sup><span class="sup_ref">350</span></sup></a> Herein, (AT/TA)-rich dsDNA containing the methylation-responsive sequence was used as a template for cluster formation. In the absence of DNA adenine methylation methyltransferase (Dam MTase), the relatively long (AT/TA)-rich dsDNA is an effective template and resulted in the formation of CuNCs with bright fluorescence. In the presence of Dam MTase, the specific methylation-sensitive restriction endonuclease Dpn I cleaves the methylated dsDNA and produces shorter dsDNA, which fails to template fluorescent CuNCs. In this case, the sensor could operate in a wider analytical range of 0.5–10 U mL<small><sup>−1</sup></small> with a detection limit of 0.5 U mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit350"><sup><span class="sup_ref">350</span></sup></a></span>
          
          
            
            <span id="sect7213"/><br/><span class="d_heading_indent">4.1.6.4. Inorganic pyrophosphatase activity detection. </span>
            <span>Inorganic pyrophosphatase (PPase) enzyme (EC 3.6.1.1) is capable of hydrolysing and catalysing the conversion of one pyrophosphate (PPi) to two phosphates (Pi). It is involved in lipid metabolism, calcium absorption and other biochemical transformations. Considering its huge impact on biotransformation, it has recently been shown that the expression of PPase is closely related to certain diseases. Alongside traditional detection methods, CuNCs-based probes were also examined in this field by Mingshu Ye <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit223"><sup><span class="sup_ref">223</span></sup></a> and Qing Liu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit352"><sup><span class="sup_ref">352</span></sup></a> Al<small><sup>3+</sup></small>-triggered AIE enhancement of GSH-protected CuNCs has been considered as a turn-on detection scheme for PPi activity screening.<a title="Select to navigate to references" href="#cit223"><sup><span class="sup_ref">223</span></sup></a> In the presence of PPi, the fluorescence of the CuNCs/Al<small><sup>3+</sup></small> system was quenched due to the stronger coordination between Al<small><sup>3+</sup></small> and PPi. Upon the addition of PPase, PPi was consumed by PPase, thus the native PL of the clusters was preserved. Based on this sensing principle, the PPase activity can be detected in the linear range of 3–40 mU mL<small><sup>−1</sup></small> with LOD of 1.3 mU mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit223"><sup><span class="sup_ref">223</span></sup></a> Similarly, in the presence of Fe<small><sup>3+</sup></small>, the PL of TA-CuNCs was quenched.<a title="Select to navigate to references" href="#cit352"><sup><span class="sup_ref">352</span></sup></a> The addition of PPi and coordination with iron restored the PL of the cluster. When the PPi was hydrolysed to Pi in the presence of PPase, the release of Fe<small><sup>3+</sup></small> again triggered the quenching of the cluster's PL signal. Thus, under optimized sensing conditions, the linear range for PPase activity sensing was 0.5–18 U L<small><sup>−1</sup></small> with LOD of 0.19 U L<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit352"><sup><span class="sup_ref">352</span></sup></a></span>
          
          
            
            <span id="sect7234"/><br/><span class="d_heading_indent">4.1.6.5. S1 nuclease activity detection. </span>
            <span>S1 nuclease (EC 3.1.30.1) belongs to the family of endonuclease enzymes that catalyse the endonucleolytic cleavage of ssDNA and RNA into oligo- or mononucleotides and hydrolysis loops in dsDNA. Besides their importance in various cellular processes, they are also of paramount interest for biotechnology applications. The S1 activity assay is normally based on PAGE, HPLC and ELISA techniques, which are time-consuming. Fluorometric probes based on CuNCs could offer much faster detection schemes. In this context, poly(thymine)<a title="Select to navigate to references" href="#cit353"><sup><span class="sup_ref">353</span></sup></a> and hairpin<a title="Select to navigate to references" href="#cit354"><sup><span class="sup_ref">354</span></sup></a> loop-templated CuNCs with bright PL properties were developed for the S1 nuclease activity detection. In the absence of S1 nuclease, polyT of 30-mer (T30) was able to effectively template and produce emissive clusters.<a title="Select to navigate to references" href="#cit353"><sup><span class="sup_ref">353</span></sup></a> However, in the presence of an enzyme, the digestion of the single strand led to smaller fragments and therefore cluster formation was not effective. Thus, just by monitoring the PL changes, it is possible to screen and assay the S1 nuclease activity.<a title="Select to navigate to references" href="#cit353"><sup><span class="sup_ref">353</span></sup></a> With this method, the lowest detectable amount of S1 nuclease was 0.5 μU μL<small><sup>−1</sup></small>. It was also found that the hairpin DNA template with a specific AT sequence stem and a hairpin loop can serve as a template to result in the formation of highly intense red-emitting CuNCs.<a title="Select to navigate to references" href="#cit354"><sup><span class="sup_ref">354</span></sup></a> However, in the presence of S1 nuclease, the hairpin loop was digested, leading to a significant decrease in the fluorescence intensity.<a title="Select to navigate to references" href="#cit354"><sup><span class="sup_ref">354</span></sup></a> The sensor operating range was 5–80 mU mL<small><sup>−1</sup></small> with LOD of 3 mU mL<small><sup>−1</sup></small>. These results demonstrated the applicability of CuNCs as a new, simple, low-cost, and effective probe for S1 nuclease detection.</span>
          
          
            
            <span id="sect7247"/><br/><span class="d_heading_indent">4.1.6.6. Acid phosphatase activity detection. </span>
            <span>Acid phosphatase (ACP, EC 3.1.3.2) is an enzyme that liberates phosphate groups during digestion under acidic conditions. The enzyme is synthesized in the liver, spleen, bone marrow, and prostate gland. Abnormally high serum levels of ACP indicate infection, injury, or cancer of the prostate. Thus, apart from clinically accepted ACP tests, CuNCs as novel fluorometric probes were also investigated and applied in the detection and evaluation of prostate activity and function. In this regard, Yuanyuan Huang <span class="italic">et al.</span> prepared a CuNCs-based fluorometric probe to detect the concentration of ACP in human serum and seminal plasma with LOD of 1.3 U L<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit220"><sup><span class="sup_ref">220</span></sup></a> Much better analytical performance for ACP detection was achieved when polydopamine (PDA)-CuNCs were applied as a sensor with a LOD value of 1 mU L<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit276"><sup><span class="sup_ref">276</span></sup></a></span>
          
          
            
            <span id="sect7256"/><br/><span class="d_heading_indent">4.1.6.7. Alkaline phosphatase activity detection. </span>
            <span>Alkaline phosphatase (ALP, EC 3.1.3.1) or basic phosphatase is a membrane-bound glycoprotein that catalyses the breakdown of proteins through the hydrolysis of phosphate monoesters. It plays a particularly important role in liver function and bone development. Severely abnormal levels of ALP can signify a severe underlying medical condition, typically one relating to the liver, bones, or gallbladder. In clinical practice, the diagnosis of those diseases is <span class="italic">via</span> blood-based analysis through the determination of ALP levels. CuNCs were also applied as an assay for ALP activity detection.<a title="Select to navigate to reference" href="#cit355"><sup><span class="sup_ref">355,357</span></sup></a> The sensing principle is based on the use of PPi as a natural substrate for the ALP, which in the cells/tissues hydrolyses the PPi to Pi to maintain normal levels in the bone. Considering the strong interaction of metal cations with PPi, the formation of fluorescent CuNCs is hampered even in the presence of the DNA template.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> However, in the presence of ALP, the PPi hydrolyses to Pi, which cannot strongly interact with Cu<small><sup>2+</sup></small>; thus, during the reduction process, the metal cations tend to cluster inside the DNA strands and form fluorescent CuNCs. Based on this strategy, the ALP levels can be quantitatively determined, <span class="italic">i.e.</span>, the lowest detectable concentration was 0.1 nM.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> GSH-CuNCs with Al<small><sup>3+</sup></small>-triggered AIE enhancement were also applied in the ALP activity detection.<a title="Select to navigate to references" href="#cit224"><sup><span class="sup_ref">224</span></sup></a> The addition of Al<small><sup>3+</sup></small> to the cluster solution caused aggregation and thus, PL enhancement. In the presence of PPi, due to the Al<small><sup>3+</sup></small>-PPi complexation, the Pl was quenched. However, the hydrolysis of PPi with ALP, liberated the Al<small><sup>3+</sup></small> and restored the PL intensity. The developed biosensor could operate in a wide linear range (0.5–25 mU mL<small><sup>−1</sup></small>) with LOD of 0.15 mU mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit224"><sup><span class="sup_ref">224</span></sup></a> The opposite effect was observed when poly(thymine)-CuNCs were applied in the detection of ALP.<a title="Select to navigate to references" href="#cit262"><sup><span class="sup_ref">262</span></sup></a> In brief, the quenching of PL (in the presence of Cr<small><sup>3+</sup></small> ions) was restored by the addition of PPi, due to the strong binding affinity of pyrophosphate towards the chromium ions. Finally, in the presence of ALP, PPi is hydrolysed, the Cr<small><sup>3+</sup></small>-PPi complex degrades and the release of Cr<small><sup>3+</sup></small> induces fluorescence quenching again.<a title="Select to navigate to references" href="#cit262"><sup><span class="sup_ref">262</span></sup></a> Yanling Hu <span class="italic">et al.</span> studied another ALP substrate, <span class="italic">p</span>-nitrophenylphosphate (PNPP), which was hydrolysed to <span class="italic">p</span>-nitrophenol (PNP).<a title="Select to navigate to references" href="#cit356"><sup><span class="sup_ref">356</span></sup></a> The hydrolyzation product caused significant quenching of the original fluorescence signal of the L-His-CuNCs. Based on the sensing method, the ALP activity can be straightforwardly determined with the lowest reported value of 45 μU mL<small><sup>−1</sup></small> in the analytical performance range of 0.5–40 mU mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit356"><sup><span class="sup_ref">356</span></sup></a></span>
          
          
            
            <span id="sect7286"/><br/><span class="d_heading_indent">4.1.6.8. Uracil-DNA glycosylase activity detection. </span>
            <span>DNA glycosylases are enzymes that remove the damaged nitrogenous base while keeping the sugar–phosphate backbone in DNA intact. Among different kinds of DNA glycosylases that are responsible for the excision of different bases in the DNA, uracil-DNA glycosylase (UDG, EC 3.2.2.27) specifically recognizes and removes uracil lesions in both double- and single-stranded DNAs and initiates the downstream repairing process. The abnormal expression of DNA glycosylases is closely related to various human diseases, such as human immunodeficiency, bloom syndrome, cancers, <span class="italic">etc</span>. Therefore, the detection of UDG levels and activity is very important for disease diagnosis and the proper functioning of base excision repair (BER) processes. Besides the highly applicable radiographic method, CuNCs were also exploited in this field as a low-risk and labour-free UDG assay. dsDNA strands have been shown to efficiently template the metal to form fluorescent CuNCs. However, the strand with uracil deoxyribonucleotides represents a substrate for UDG and recognition by the enzyme results in the spontaneous detachment of the double helix into ssDNAs. Consequently, the formed ssDNAs cannot effectively template the cluster, which leads to the weakening of the emission intensity. By monitoring the PL changes, the UDG activity can be quantitatively determined. As such, Manman Cao <span class="italic">et al.</span> prepared dsDNA-CuNCs for UDG activity detection.<a title="Select to navigate to references" href="#cit358"><sup><span class="sup_ref">358</span></sup></a> The turn-off fluorometric sensor could operate in a range of 1 mU mL<small><sup>−1</sup></small>–0.1 U mL<small><sup>−1</sup></small> with LOD of 0.5 mU mL<small><sup>−1</sup></small>. Poly(thymine)-CuNCs, on the other hand, manifest much better performances for UDG analysis.<a title="Select to navigate to reference" href="#cit359"><sup><span class="sup_ref">359,360</span></sup></a> Here, the developed turn-on sensors covered a vast detection ranges from 100 μU mL<small><sup>−1</sup></small> to 0.01 U mL<small><sup>−1</sup></small> (ref. <a title="Select to navigate to references" href="#cit359">359</a>) and 50 μU mL<small><sup>−1</sup></small> to 2 mU mL<small><sup>−1</sup></small> (ref. <a title="Select to navigate to references" href="#cit360">360</a>) with LOD of 50 μU mL<small><sup>−1</sup></small> (in both cases).</span>
          
          
            
            <span id="sect7304"/><br/><span class="d_heading_indent">4.1.6.9. Polynucleotide kinase phosphatase activity detection. </span>
            <span>The DNA damage caused by exposure to genotoxic agents results in the formation of DNA with a 3′-phosphate terminal. However, for the proper biochemistry of DNA, the removal of the 3′-phosphate and attachment of a hydroxyl group to the 3′-tail end of the DNA backbone is essential. T4 polynucleotide kinase phosphatase (T4 PNKP, EC 2.7.1.78) is a popular enzyme that catalyses the hydrolytic removal of the 3′-phosphoryl of polynucleotides and at the same time phosphorylates DNA at the 5′-hydroxyl end. For the T4 PNKP activity detection, poly(thymine)-templated CuNCs were exploited as a turn-off<a title="Select to navigate to references" href="#cit361"><sup><span class="sup_ref">361</span></sup></a> and a turn-on<a title="Select to navigate to references" href="#cit362"><sup><span class="sup_ref">362</span></sup></a> biosensor by using T-rich and A-rich DNA probes as a substrate for T4 PNKP. The 3′-end of the ssDNA probe (T-rich) modified with a phosphate group is hydrolysed into a 3′-hydroxyl, which can further be extended into dsDNA by the DNA polymerase in the presence of dNTPs.<a title="Select to navigate to references" href="#cit361"><sup><span class="sup_ref">361</span></sup></a> The polymerization product is unable to template and form fluorescent clusters. In the case of an A-rich probe, an additional step is required. Briefly, the formed dsDNA in the process described above is digested <span class="italic">via</span> a restriction enzyme followed by the amplification cycle, polymerization and nicking to produce T-rich ssDNA, which can later template the clusters.<a title="Select to navigate to references" href="#cit362"><sup><span class="sup_ref">362</span></sup></a> Thus, by following the changes in PL upon cluster formation, the T4 PNKP activity can be detected. The detectable minimum amounts of T4 PNKP enzyme with those sensors were 0.25 U mL<small><sup>−1</sup></small> (ref. <a title="Select to navigate to references" href="#cit361">361</a>) and 0.02 U mL<small><sup>−1</sup></small>. (ref. <a title="Select to navigate to references" href="#cit362">362</a>) Similarly, Xingxing Zhang <span class="italic">et al.</span> developed a dsDNA-CuNCs sensor based on the hybridization of two sequence-dependent ssDNA strands to form a partial dsDNA duplex.<a title="Select to navigate to references" href="#cit363"><sup><span class="sup_ref">363</span></sup></a> This DNA serves as a substrate for T4 PNKP to be further used for the synthesis of full dsDNA. With this sensor, the LOD of T4 PNKP activity detection was 0.06 U mL<small><sup>−1</sup></small>. Another report by Ruqin Yu and co-workers established a detection platform for polynucleotide kinase (PNK) activity sensing with the LOD of 0.49 U mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit364"><sup><span class="sup_ref">364</span></sup></a></span>
          
          
            
            <span id="sect7322"/><br/><span class="d_heading_indent">4.1.6.10. Acetylcholinesterase activity detection. </span>
            <span>Acetylcholinesterase (AChE, EC 3.1.1.7) catalyses the breakdown of the acetylcholine neurotransmitter into acetate and choline upon the termination of impulse transmission at cholinergic synapses. The abnormal concentrations of acetylcholine have been linked to the development of several neurodegenerative diseases such as Parkinson's disease, myasthenia gravis and Alzheimer's disease (AD). Thus, the detection of AChE activity is of great importance in clinical diagnostics. Apart from colorimetric and electrochemical detection methods, fluorometric methods have recently attracted much interest due to their simplicity and faster analysis. Taking advantage of their fluorescence properties, Jinlan Yang <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit254"><sup><span class="sup_ref">254</span></sup></a> Mengke Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit365"><sup><span class="sup_ref">365</span></sup></a> and Pengjuan Ni <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit366"><sup><span class="sup_ref">366</span></sup></a> developed CuNCs-based fluorometric probes to detect AChE activity in human serum samples. Within the proposed sensing strategies, the LODs of the sensors were 1.38 mU mL<small><sup>−1</sup></small>,<a title="Select to navigate to references" href="#cit254"><sup><span class="sup_ref">254</span></sup></a> 0.56 U L<small><sup>−1</sup></small>,<a title="Select to navigate to references" href="#cit365"><sup><span class="sup_ref">365</span></sup></a> and 0.05 mU mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit366"><sup><span class="sup_ref">366</span></sup></a></span>
          
          
            
            <span id="sect7338"/><br/><span class="d_heading_indent">4.1.6.11. Other enzyme activity detection. </span>
            <span>The potential applications of CuNCs towards the enzyme activity assay have also been explored for restriction enzyme EcoRI (LOD 870 μU μL<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit347"><sup><span class="sup_ref">347</span></sup></a> matrix metalloproteinase-7 (MMP-7, LOD 5.3 pg mL<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit348"><sup><span class="sup_ref">348</span></sup></a> micrococcal nuclease (MNase, LOD 1 mU mL<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit351"><sup><span class="sup_ref">351</span></sup></a> terminal deoxynucleotidyl transferase (TdT, LOD 60 mU L<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit367"><sup><span class="sup_ref">367</span></sup></a> reverse transcription-associated ribonuclease H (RNase H, LOD 0.55 mU mL<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit368"><sup><span class="sup_ref">368</span></sup></a> β glucuronidase (GLU, LOD 0.22 U L<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit369"><sup><span class="sup_ref">369</span></sup></a> tyrosinase (TYR, LOD 44 U L<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit370"><sup><span class="sup_ref">370</span></sup></a> thrombin (LOD 0.9 nM),<a title="Select to navigate to references" href="#cit371"><sup><span class="sup_ref">371</span></sup></a> hyaluronidase (HAase, LOD 14 mU mL<small><sup>−1</sup></small>),<a title="Select to navigate to references" href="#cit372"><sup><span class="sup_ref">372</span></sup></a> lysozyme (LOD 1.6 nM)<a title="Select to navigate to references" href="#cit373"><sup><span class="sup_ref">373</span></sup></a> and protein kinase A (PKA, LOD 38 mU mL<small><sup>−1</sup></small>).<a title="Select to navigate to references" href="#cit374"><sup><span class="sup_ref">374</span></sup></a></span>
          
        </div>
        <div>
          
          <span id="sect7362"/><span class="c_heading_indent">4.1.7. Protein detection. </span>
          <span>CuNCs, templated and protected with various biomolecules, were applied as a sensitive and specific bioassay in the studies of antigen–antibody interactions, the binding of small molecules to their receptors as well as for the accurate detection of disease-related biomarkers in clinical diagnosis and therapy.</span>
          <p class="otherpara">Streptavidin (SA)-biotin is the strongest noncovalent biological interaction known in biology. The tetrameric unit of the SA protein can bind four biotin molecules. The bioconjugation of SA to biomolecules or a labelled target (fluorophore, enzyme <span class="italic">etc.</span>), has been applied in modern detection techniques and bioassays like Western blotting, ELISA, and Flow Cytometry. Thus, taking advantage of a strong and specific SA–biotin interaction, a novel detection method has been employed based on the interaction between the CuNCs–SA–biotin system. For example, Hai-Bo Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit375"><sup><span class="sup_ref">375</span></sup></a> have shown that the interaction of SA with poly(thymine)-templated CuNCs attached to biotin can prevent the degradation and enzymolysis of DNA by Exo I. On the other hand, fluorescent CuNCs, can be used to determine the SA concentration from calibration curves. Jiepei Cao <span class="italic">et al.</span> reported the use of fluorescent CuNCs for the study of SA–biotin interactions under magnetic separations.<a title="Select to navigate to references" href="#cit337"><sup><span class="sup_ref">337</span></sup></a> Within the developed strategy, the LODs of SA and biotin were 0.47 nM and 3.1 nM, respectively. A better SA sensing platform was developed by Yue He <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit376"><sup><span class="sup_ref">376</span></sup></a> based on the utilization of poly(thymine)-CuNCs as a turn-on sensor (LOD 20 pM). Thus, as a classic model, SA-biotin bioconjugated CuNCs can be exploited as a sensitive, highly selective, and low-cost protein-binding assay.</p>
          <p class="otherpara">Li-Juan Ou <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit379"><sup><span class="sup_ref">379</span></sup></a> and Yanling Hu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit382"><sup><span class="sup_ref">382</span></sup></a> have reported the applications of poly(thymine)-CuNCs and PSS-CuNCs as a fluorometric assay for the detection of trypsin with LOD of 42 ng mL<small><sup>−1</sup></small> and 20 ng mL<small><sup>−1</sup></small>, respectively. Cytochrome c (Cyt c) is a well-known electron transfer mediator and thus a quencher, and it can quench the PL of CuNCs.<a title="Select to navigate to references" href="#cit247"><sup><span class="sup_ref">247</span></sup></a> However, in the presence of trypsin, Cyt c hydrolyses into peptide fragments and thus inhibits the quenching of PL from CuNCs.<a title="Select to navigate to references" href="#cit382"><sup><span class="sup_ref">382</span></sup></a> Better sensitivity towards trypsin was achieved when GSH-CuNCs together with Cyt c (LOD 2 ng mL<small><sup>−1</sup></small>)<a title="Select to navigate to references" href="#cit380"><sup><span class="sup_ref">380</span></sup></a> and dsDNA-CuNCs (LOD 48 pg mL<small><sup>−1</sup></small>)<a title="Select to navigate to references" href="#cit381"><sup><span class="sup_ref">381</span></sup></a> were implemented as fluorometric sensors.</p>
          <p class="otherpara">Xin-Ge Li <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit377"><sup><span class="sup_ref">377</span></sup></a> and Quanwei Song <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit378"><sup><span class="sup_ref">378</span></sup></a> prepared CuNCs-based bio probes for the detection of glycoproteins and actin, respectively. The LODs of glycoproteins and actin were 2.6 nM and 0.12 μg mL<small><sup>−1</sup></small>, respectively. CuNCs-based detection of VEGF165,<a title="Select to navigate to references" href="#cit383"><sup><span class="sup_ref">383</span></sup></a> protamine,<a title="Select to navigate to references" href="#cit307"><sup><span class="sup_ref">307</span></sup></a> ferritin,<a title="Select to navigate to references" href="#cit247"><sup><span class="sup_ref">247</span></sup></a> carcinoembryonic antigen (CEA),<a title="Select to navigate to references" href="#cit384"><sup><span class="sup_ref">384</span></sup></a> prostate-specific antigen (PSA)<a title="Select to navigate to references" href="#cit385"><sup><span class="sup_ref">385</span></sup></a> and human immunoglobulin G (IgG)<a title="Select to navigate to references" href="#cit357"><sup><span class="sup_ref">357</span></sup></a> were also successfully demonstrated with LODs of 12 pM, 0.12 ng mL<small><sup>−1</sup></small>, 16.5 nM, 6.5 pg mL<small><sup>−1</sup></small>, 145.7 fg mL<small><sup>−1</sup></small> and 7 pg mL<small><sup>−1</sup></small>, respectively.</p>
        </div>
        <div>
          
          <span id="sect7403"/><span class="c_heading_indent">4.1.8. Temperature sensing. </span>
          <span>Nanothermometry is an emerging field with a variety of applications in nanoelectronics, nanophotonics and biomedicine because of the capability to measure the temperature fluctuations at the nanoscale. Thus, a huge interest in nanomaterials with temperature sensing ability has emerged.</span>
          <p class="otherpara">Jiu-Ju Feng and co-workers established a CuNCs-based nanothermometer that displayed a temperature dependence of PL intensity in the range of 10–55 °C (<a title="Select to navigate to figure" href="#imgfig25">Fig. 25A</a>).<a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> The PL quenching was reversible and could be switched on and off for at least 6 cycles with negligible fatigue. Moreover, it was shown that the decrease in PL intensity was caused by the activation of nonradiative pathways.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig25"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f25_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f25.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f25.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 25 </b> <span id="fig25"><span class="graphic_title">(A) Schematic illustration of the formation mechanism of CuNCs with temperature-dependent fluorescence. Republished with permission from ref. <a title="Select to navigate to references" href="#cit134">134</a>. Copyright 2014 Royal Society of Chemistry. (B) PL reversibility exhibited by the HSA-CuNCs as a function of temperature. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit50">50</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">In another case, Saptarshi Mukherjee and co-workers studied the temperature dependence of the PL properties of HSA-CuNCs.<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> With increasing temperature, the PL intensity of the clusters was quenched, mostly due to thermal instability/unfolding caused in the stabilizing scaffolds of HSA protein (<a title="Select to navigate to figure" href="#imgfig25">Fig. 25B</a>). A similar effect was been observed when GSH-CuNCs were heated to 80 °C.<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> The increase in temperature led to the decrease in PL intensity because of the increase in collision rate and activation of nonradiative pathways. The decrease in temperature resulted in the complete restoration of PL properties in all mentioned cases. Furthermore, GSH-CuNCs were applied as a nanothermometer to detect intracellular temperature differences in MC3T3-E1<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> and MDA-MB-231<a title="Select to navigate to references" href="#cit386"><sup><span class="sup_ref">386</span></sup></a> cancer cells. As expected, with increasing the temperature, the red fluorescence of internalized clusters faded away.</p>
        </div>
        <div>
          
          <span id="sect7420"/><span class="c_heading_indent">4.1.9. pH sensing. </span>
          <span>CuNCs, owing to their attractive size-dependent optical properties, bright fluorescence, low toxicity, and biocompatibility, have become ideal optical probes for implementation in chemical sensors. Trypsin-stabilized fluorescent CuNCs prepared by Wei Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a> were employed as a fluorescent pH sensor. At very low pH value, the fluorescence emission peak reached its maximum; however, with increasing pH, the fluorescence emission steadily decreased and reached very low values at around pH 12 (<a title="Select to navigate to figure" href="#imgfig26">Fig. 26A and B</a>), similar to what has been observed for GSH-<a title="Select to navigate to references" href="#cit311"><sup><span class="sup_ref">311</span></sup></a> and <span class="small_caps">L</span>-tyrosine-protected CuNCs.<a title="Select to navigate to references" href="#cit389"><sup><span class="sup_ref">389</span></sup></a> Moreover, the formed precipitate in alkaline medium can be re-dissolved in acidic medium, thus making the pH sensor reversible. The sensor can work efficiently for at least 7 cycles without fatigue.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig26"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f26_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f26.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f26.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 26 </b> <span id="fig26"><span class="graphic_title">(A) Fluorescence emission spectra of CuNCs in Britton–Robinson (BR) buffers of different pH values. (B) The calibration curve of the pH values in the range of 2.02–12.14 <span class="italic">versus</span> the fluorescence intensity of the CuNCs in BR buffers. Republished with permission from ref. <a title="Select to navigate to references" href="#cit39">39</a>. Copyright 2014 Royal Society of Chemistry. (C) Photographs showing the colour changes of the CuNCs in BR buffer at different pH values under visible light (upper) and UV light (lower). (D) The fluorescence intensity of the CuNCs reversibly went up and down when alternating the pH value. Republished with permission from ref. <a title="Select to navigate to references" href="#cit49">49</a>. Copyright 2015 Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">In contrast, BSA-templated CuNCs were highly fluorescent in alkaline medium and less emissive under acidic conditions.<a title="Select to navigate to reference" href="#cit49"><sup><span class="sup_ref">49,303</span></sup></a> Again, the sensor could operate effectively for 7 cycles with no fatigue (<a title="Select to navigate to figure" href="#imgfig26">Fig. 26C and D</a>).<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> A similar phenomenon has been observed with green-emitting cys-CuNCs.<a title="Select to navigate to references" href="#cit133"><sup><span class="sup_ref">133</span></sup></a> Compared to the emission peak intensity at pH 4, about 66-fold higher intensities were observed at pH 9. As for PEI-capped CuNCs, the fluorescence emission reached its maximum at pH 13 and it was about 20-fold higher than that at pH 5.2.<a title="Select to navigate to references" href="#cit387"><sup><span class="sup_ref">387</span></sup></a></p>
          <p class="otherpara">CuNCs prepared directly from chicken egg white (CEW) demonstrated pH-reversible features mostly ascribed to conformational changes in the ovalbumin protein in CEW with pH variations from acidic to basic.<a title="Select to navigate to references" href="#cit388"><sup><span class="sup_ref">388</span></sup></a> The intense green emission was observed in strongly basic conditions, whereas at lower and mostly in acidic buffers, the PL decreased, consistent with the precipitation of the CuNCs.</p>
        </div>
        <div>
          
          <span id="sect7443"/><span class="c_heading_indent">4.1.10. Infections detection. </span>
          <span>dsDNA-templated CuNCs have been applied in the diagnosis of tuberculosis (TB).<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> The <span class="italic">in situ</span> cluster formation in the presence of the PCR product of the targeted IS6110 DNA sequence of TB is only successful if specifically designed AT-rich primers are used. With this, the formation of dimers and other unwanted hairpin structures and their contribution to PL were kept at a minimum level. The JPEG images of the products illuminated by a UV transilluminator and taken by a smartphone were analysed by ImageJ software. The grey value of the image's red channel was calculated. By applying a linear regression algorithm, the detection limit for the IS6110 sequence of TB dsDNA was found to be 5 fg μL<small><sup>−1</sup></small>, with a linear dynamic range from 10 to 100 fg μL<small><sup>−1</sup></small>.</span>
          <p class="otherpara">A colorimetric detection platform for hepatitis B virus (HBV) was established by using DNA-CuNCs.<a title="Select to navigate to references" href="#cit390"><sup><span class="sup_ref">390</span></sup></a> The designed sensor can specifically target and sense the HBV DNA with LOD of 12 × 10<small><sup>9</sup></small> molecules. Acceptable recoveries were detected from serum samples containing HBV DNA.</p>
          <p class="otherpara">Ehsan Shokri <span class="italic">et al.</span> reported the virus-detected synthesis of fluorescent CuNCs and tested its applicability in the detection of Citrus Tristeza Virus (CTV) in real samples with LOD of 220 pg mL<small><sup>−1</sup></small>.<a title="Select to navigate to references" href="#cit391"><sup><span class="sup_ref">391</span></sup></a></p>
          <p class="otherpara">Thus, dsDNA-templated CuNCs can be applied as a portable and digital diagnostic tool in combination with sensor technologies.</p>
        </div>
        <div>
          
          <span id="sect7459"/><span class="c_heading_indent">4.1.11. Other sensing applications. </span>
          <span>The applicability of CuNCs templated with various matrices has also been applied in the detection of water in organic solvents,<a title="Select to navigate to references" href="#cit112"><sup><span class="sup_ref">112</span></sup></a> urea,<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19,336</span></sup></a> DNA aptamers and cocaine,<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> as well as for ammonium pyrrolidine dithiocarbamate (APDC).<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a></span>
          <p class="otherpara">A large number of studies have been conducted for preparing water-soluble and highly fluorescent CuNCs as a novel fluorometric/colorimetric sensing platform. Note that CuNCs-based fluorometric sensors (mostly turn-off) sometimes show impressive sensitivity towards the target (LOD in the fM or aM range) and can operate in a wide analytical range. Nevertheless, the main drawback is that most of them are not atomically precise and less attention has been paid to the selectivity. The developed sensors have not been commercialized and applied in real clinical diagnosis.</p>
        </div>
      
      
        
        <h3 id="sect7468"><span class="b_heading">4.2. Applications in biomedicine</span></h3>
        <div>
          
          <span id="sect7471"/><span class="c_heading_indent">4.2.1. Imaging of cells. </span>
          <span>Folic acid (FA), also known as an artificial form of vitamin B9, is a crucial agent for the body, and its active form (folate) is involved in gene expression. High or low intakes of folate are directly related to the inhibition or development of various cancers. Thus, FA is an important cancer cell targeting agent <span class="italic">via</span> detecting its receptor protein, <span class="italic">i.e.</span>, folate receptor (FR). FR is overexpressed on the surface of cancerous cells and its levels increase with the increasing stages of the disease.</span>
          <p class="otherpara">Therefore, highly fluorescent, and FA-conjugated CuNCs have found applications in the early diagnosis of cancers <span class="italic">via</span> targeted bioimaging of FR overexpressed cancerous cells. Chuanxi Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit132"><sup><span class="sup_ref">132</span></sup></a> demonstrated the potential of FA-conjugated and GSH-protected CuNCs as a diagnostic tool to detect gastric HeLa cells (FR-positive cell line) in physiological conditions with no adverse effect on the cell line. Moreover, the clusters were highly biocompatible and showed very small cytotoxicity.</p>
          <p class="otherpara">Jun-Mei Xia <span class="italic">et al.</span> prepared FA-protected CuNCs (direct synthesis) and tested them against HeLa and A549 cells (human lung carcinoma cells, FR-negative cell line).<a title="Select to navigate to references" href="#cit407"><sup><span class="sup_ref">407</span></sup></a> No cluster-induced cytotoxicity was observed when the cells were incubated in the presence of different CuNCs concentrations. However, significant fluorescence was observed in the case of HeLa cells and almost no fluorescence for A549 cells. Since HeLa cells express higher levels of FR, it initiates faster internalization of FA-CuNCs in the cells through receptor-mediated endocytosis. Yaling Wang <span class="italic">et al.</span> demonstrated the one/two-photon fluorescence imaging of HeLa and A549 cells using bifunctional peptide (Sv)-functionalized CuNCs.<a title="Select to navigate to references" href="#cit408"><sup><span class="sup_ref">408</span></sup></a> Interestingly, the clusters were found to be mainly localized in the nucleus and less in the cytoplasm. Moreover, the two-photon (TP) images were less affected by the auto-fluorescence of the cell, thus making TP superior to one-photon (OP) imaging (<a title="Select to navigate to figure" href="#imgfig27">Fig. 27A</a>).</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig27"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f27_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f27.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f27.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 27 </b> <span id="fig27"><span class="graphic_title">(A) One- and two-photon confocal images of Sv–CuNCs for HeLa and A549 cells: (a)–(c) and (d)–(f) are OP and TP images of HeLa cells, respectively; (g)–(i) and (j)–(l) are OP and TP images of A549 cells, respectively. Differential interference contrast (top panel), fluorescence image of Sv–Cu cluster (middle panel, blue and yellow represent OP and TP emission, respectively) and merged view (bottom panel). Republished with permission from ref. <a title="Select to navigate to references" href="#cit408">408</a>. Copyright 2013 Royal Society of Chemistry. (B) Subcellular localization of CuNCs: HeLa, MDAMB-231, and A549 cells were incubated with CuNCs for 12 h at 37 °C. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit135">135</a>. Copyright 2015 American Chemical Society. (C) PET imaging of tumours in the mouse cell line xenograft model (left), mouse PDX model (middle) and human tumour tissue (right). Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit410">410</a>. Copyright 2019 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The preferential localization of CuNCs in the nucleus was also reported by Chan Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> and Markus J. Barthel <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit409"><sup><span class="sup_ref">409</span></sup></a> It was found that the cluster internalization is due to the crossing of the membrane since the endocytosis is inhibited at 4 °C. However, at 37 °C the accumulation of CuNCs in the perinuclear area additionally pinpointed the endocytotic uptake.<a title="Select to navigate to references" href="#cit409"><sup><span class="sup_ref">409</span></sup></a> Chan Wang <span class="italic">et al.</span> showed that CAL-27 cells incubated with BSA-CuNCs kept their morphology over time and at different doses of the cluster.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> As mentioned before, the fluorescence signal was located mostly in the cellular nucleus. Therefore, BSA-CuNCs displayed real potential for labelling the oligonucleotide and the detection of nucleic acids.</p>
          <p class="otherpara">Similar results were observed when 293T cells were treated and incubated with PEI-CuNCs.<a title="Select to navigate to references" href="#cit387"><sup><span class="sup_ref">387</span></sup></a> According to the MTT assay, 75% of cells stayed viable after 24 h of incubation. Confocal microscopy imaging revealed that the cells kept their normal morphology after the treatment and showed the internalization of the cluster into the cytoplasm and the nucleus. The energy-dependent, clathrin-mediated endocytosis has been proposed as an internalization mechanism/pathway. On the contrary, Hong Huang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> have shown that on fluorescence imaging of HeLa cells with CuNCs protected with a peptide (amino acid sequence: CLEDNN), most clusters were localized in the cell membrane and cytoplasmic area. A very weak blue fluorescence was detected in the nucleus. Therefore, the clusters are bio-safe and do not cause genetic disruption.</p>
          <p class="otherpara">Jigna R. Bhamore <span class="italic">et al.</span> have demonstrated that CuNCs prepared from the curcuma root extract showed pronounced cytotoxicity in RIN5F and MDAMB231 cancerous cell lines.<a title="Select to navigate to references" href="#cit241"><sup><span class="sup_ref">241</span></sup></a> The cell cycle analyses revealed that the G2/M phase was more affected in MDAMB231 cells, which suggests that CuNCs might contribute to both G1/S arrest and cellular apoptosis in cancer cells. Upon fungal cell imaging (<span class="italic">Penicillium citrinum</span> cells), the clusters were found to be localized near the nuclear region. Saptarshi Mukherjee and co-workers demonstrated that the cellular internalization and localization of GSH-CuNCs strongly depend on the cell line.<a title="Select to navigate to references" href="#cit135"><sup><span class="sup_ref">135</span></sup></a> For instance, in HeLa (malignant immortal cell line derived from cervical cancer) and MDAMB-231 (human breast adenocarcinoma) cells the clusters were preferentially localized near the nuclear region, whereas in the case of A549 (human lung carcinoma), the clusters were more distributed in the cytoplasm (<a title="Select to navigate to figure" href="#imgfig27">Fig. 27B</a>).</p>
          <p class="otherpara">Transferrin-templated CuNCs have been applied for the imaging of a human cancer cell line and a normal mouse cell line (3T3) through the detection of transferrin receptors.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> Because HeLa cells have a higher level of transferrin receptor expression, the cells treated with CuNCs showed an intense red fluorescence, whereas week fluorescence was observed in the case of 3T3 cells.</p>
          <p class="otherpara">Gyu Seong Heo <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit410"><sup><span class="sup_ref">410</span></sup></a> demonstrated the applicability of radiolabelled and peptide bioconjugated CuNCs (<small><sup>64</sup></small>Cu-CuNCs-FC131) for the sensitive and accurate detection of tumours in triple-negative breast cancer (TNBC) patient-derived xenograft mouse models and human TNBC tissues (<a title="Select to navigate to figure" href="#imgfig27">Fig. 27C</a>). <small><sup>64</sup></small>Cu-CuNCs showed weak tumour uptakes, whereas CXCR4 targeted <small><sup>64</sup></small>Cu-CuNCs-FC131 demonstrated significant tumour uptake. Considering the biocompatibility and low toxicity of clusters, the proposed strategy can be applied in clinical settings for the detection of overexpressed CXCR4 biomarkers in both mouse models and human TNBC tissues, for positron emission tomography (PET) imaging as well as for therapy. The obtained results demonstrated the advantages of radiolabelled CuNCs as a potential translational imaging and drug delivery machinery for cancer theranostics. A recent review by Rogach and co-workers summarizes the theranostic applications of CuNCs in both <span class="italic">in vitro</span> and <span class="italic">in vivo</span> platforms.<a title="Select to navigate to references" href="#cit411"><sup><span class="sup_ref">411</span></sup></a></p>
          <p class="otherpara">Moving forward, Fuping Gao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a> demonstrated the applicability of radiolabelled <small><sup>64</sup></small>Cu-BSA-CuNCs as an efficient and sensitive radiotracer for PET imaging. The bioconjugation of LHRH receptor peptide to the clusters ensured the effective deposition and clearance from the tumour <span class="italic">in vivo</span>. Receptor-targeted clusters showed higher tumour uptake properties than that of nontargeted clusters in A549 human lung tumour. Tomographic images revealed that after 0.5 h post-injection, the clusters were distributed all over the body (<a title="Select to navigate to figure" href="#imgfig28">Fig. 28A</a>). However, at higher post-injection times, a strong radioactivity signal was coming from the kidneys and bladder, thus the clusters were mainly accumulating in those organs (<a title="Select to navigate to figure" href="#imgfig28">Fig. 28B</a>). This is indicative of the cluster clearance through the renal route. Additionally, fluorescence imaging of cryosections of tissues has shown that targeted clusters were mostly localized in the tumour and kidneys, whereas nontargeted clusters were in the kidneys (<a title="Select to navigate to figure" href="#imgfig28">Fig. 28C</a>). GSH-CuNCs were applied in the imaging of MC3T3-E1<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> and MGC-803 cells;<a title="Select to navigate to references" href="#cit225"><sup><span class="sup_ref">225</span></sup></a> the latter report is based on the aggregation-induced emission (AIE) effect. In the presence of Zn<small><sup>2+</sup></small>, the aggregation of the clusters resulted in bright red emission and thus, Zn<small><sup>2+</sup></small>-triggered AIE of CuNCs has been applied as a “light-up” nanoprobe for the detection/imaging of zinc ions in living cells.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig28"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f28_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f28.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f28.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 28 </b> <span id="fig28"><span class="graphic_title">(A) <span class="italic">In vivo</span> PET images of coronal single slices on orthotopic A549 lung tumour-bearing mice after intravenous injection of 6.7 MBq of [<small><sup>64</sup></small>Cu]CuNC@BSA (top) and [<small><sup>64</sup></small>Cu]CuNC@BSA-LHRH (bottom). Images were acquired at 0.5, 1, 2, and 4 h. White arrows indicate the lung tumour. (B) The corresponding organ biodistribution of [<small><sup>64</sup></small>Cu]CuNC@BSA and [<small><sup>64</sup></small>Cu]CuNC@BSA-LHRH at 4 h after iv injection in mice bearing orthotopic A549 lung tumour. (C) Fluorescence microscopy images of tissue sections. The tissues were taken at 4 h after the injection of CF680R-conjugated [<small><sup>64</sup></small>Cu]CuNC@BSA-LHRH <span class="italic">via</span> the tail vein. Blue fluorescence shows nuclei stained with DAPI, and red fluorescence shows the location of [<small><sup>64</sup></small>Cu]CuNC@BSA-LHRH. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit40">40</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Arun Chattopadhyay and co-workers implemented the AIE-based enhancement of PL to monitor <span class="italic">in vitro</span> pH variations between cancerous (human breast adenocarcinoma cells: MCF-7) and non-cancerous (human embryonic kidney cells: HEK-293) cell lines.<a title="Select to navigate to references" href="#cit123"><sup><span class="sup_ref">123</span></sup></a> The as-synthesized clusters showed bright orange-red emission in acidic conditions (pH 4.5) and cyan-green emission in basic medium (pH 7.4, <a title="Select to navigate to figure" href="#imgfig29">Fig. 29</a>). The study demonstrated that higher expression levels of various proteins and biomolecules in MCF-7 cells triggered faster aggregation kinetics in the cells at higher pH (pH 7.4). On the contrary, HEK-293 cells lack the overexpression of such biomolecules in the intracellular environment, thus prohibiting the AIE-induced PL of the clusters.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig29"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f29_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f29.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f29.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 29 </b> <span id="fig29"><span class="graphic_title">(A) Bright-field image and (B) fluorescence image in the red channel of MCF-7 cells treated with the NC probe for 1 h in culture medium at pH 4.5. (C) Bright-field image and (D) fluorescence image acquired in the green channel of MCF-7 cells treated with the NC probe for 1 h in culture media at pH 7.8. (E) Bright-field image and (F) fluorescence image in the red channel of HEK-293 cells treated with the NC probe for 1 h in culture media at pH 4.5. (G) Bright field image and (H) fluorescence image acquired in the green channel of HEK-293 cells treated with the NC probe for 1 h in culture media at pH 7.4. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit123">123</a>. Copyright 2018 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Zhichao Liu <span class="italic">et al.</span> prepared a CuNCs-based fluorescence probe for the real-time sensing and multicolour imaging of calcium ions in neurons.<a title="Select to navigate to references" href="#cit256"><sup><span class="sup_ref">256</span></sup></a></p>
          <p class="otherpara">The intracellular calcium ion detection revealed the inhomogeneous localization and accumulation of calcium in the neuronal cytoplasm (<a title="Select to navigate to figure" href="#imgfig30">Fig. 30</a>). Besides, the inhomogeneity caused the formation of bright spots in the overlay image resulting from Ca<small><sup>2+</sup></small>-rich domains. More importantly, the study demonstrated that ROS-induced neuronal death is directly proportional to calcium ion overload. Thus, these findings provide a solid basis for the mechanistic studies of oxidative stress-related diseases using CuNCs.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig30"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f30_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f30.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f30.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 30 </b> <span id="fig30"><span class="graphic_title">Confocal fluorescence microscopy images of neurons collected from different channels after the neurons were co-incubated with a CuNC@AF660 probe (90 μg mL<small><sup>−1</sup></small>) in the presence of different concentrations of Ca<small><sup>2+</sup></small> (0, 100, 200, and 300 μM), respectively. Scale = 25 μm. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit256">256</a>. Copyright 2019 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Blue-emitting lysozyme-CuNCs (QY 18%) prepared by Rama Ghosh <span class="italic">et al.</span> have been considered as fluorescent agents for cell labelling.<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> In this regard, HeLa cells (cervical cancer) incubated with CuNCs were detected <span class="italic">via</span> fluorescence microscopy imaging, where a bright blue colour was attributed to the clusters since control cells and cells treated only with the ligand lacked intense PL properties. The clusters showed no cytotoxicity towards the cells.</p>
          <p class="otherpara">Jinbin Liu and co-workers demonstrated that CuNCs assemblies with enhanced AIE properties can be used to image HeLa cells with green fluorescence protein (GFP)-labelled lysosomes.<a title="Select to navigate to references" href="#cit218"><sup><span class="sup_ref">218</span></sup></a> Both red emission (from the CuNCs assemblies) and green emission (from GFP) were observed at the incubation time of 24 h, which means that the cluster assemblies are highly stable and keep their PL properties under a lysosomal acidic microenvironment (pH ≈ 4.5–6).</p>
          <p class="otherpara">The latest review by Baisong Chang discusses the recent advances in the bioimaging applications of CuNCs.<a title="Select to navigate to references" href="#cit394"><sup><span class="sup_ref">394</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect7571"/><span class="c_heading_indent">4.2.2. Antibacterial properties. </span>
          <span>Bacterial infections occur as a result of the proliferation of a harmful bacterial strain in the body and can cause mild to severe health conditions. The pathological activity of the bacteria regardless of the shape (rod-like, spherical, and helical) and the type (Gram-positive or Gram-negative), can be suppressed by appropriately choosing the course of the treatment. In this case, the main treatment is the administration of antibiotics. However, the misuse of a prescribed antibiotic can cause unwanted side effects including antibiotic resistance. Thus, considering the current challenges of emerging antibiotic resistance of known antibiotics in the clinical settings, ongoing research has led to the development of new nanomaterial-based drug alternatives as effective antibacterial agents. As far as CuNCs are concerned, the antibacterial properties of papain-templated CuNCs have been studied on both Gram-positive bacteria <span class="italic">S. aureus</span> and Gram-negative bacteria <span class="italic">E. coli</span>.<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> It has been shown that when CuNCs were used together with H<small><sub>2</sub></small>O<small><sub>2</sub></small>, the viabilities of cells were dramatically reduced and the proposed system showed striking antibacterial activity towards the abovementioned bacteria (<a title="Select to navigate to figure" href="#imgfig31">Fig. 31A</a>). Furthermore, CuNCs-H<small><sub>2</sub></small>O<small><sub>2</sub></small> systems have been employed in wound disinfection in order to stop the spread of the bacterial infection. It was found that after 48 h of therapy, the wounded mice treated with CuNCs-H<small><sub>2</sub></small>O<small><sub>2</sub></small> did not appear to show any erythema and edema and formed scabs simultaneously (<a title="Select to navigate to figure" href="#imgfig31">Fig. 31B</a>). This implies that the proposed antibacterial system can be used to fight the bacterial infection and its spread in wounds.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig31"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f31_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f31.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f31.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 31 </b> <span id="fig31"><span class="graphic_title">(A) The influence of CuNCs on the growth of Gram-positive (<span class="italic">S. aureus</span>) and Gram-negative (<span class="italic">E. coli</span>) bacteria. Gram-positive bacteria were recorded at 12 h and 24 h: <span class="italic">S. aureus</span> (105 cfu mL<small><sup>−1</sup></small>) co-incubated with CuNCs plus H<small><sub>2</sub></small>O<small><sub>2</sub></small> (A and E), H<small><sub>2</sub></small>O<small><sub>2</sub></small> (B and F), CuNCs (C and G), null (D and H), respectively; Gram-negative bacteria were observed at 24 h and 48 h: <span class="italic">E. coli</span> (105 cfu mL<small><sup>−1</sup></small>) co-incubated with CuNCs plus H<small><sub>2</sub></small>O<small><sub>2</sub></small> (I and M), H<small><sub>2</sub></small>O<small><sub>2</sub></small> (J and N), CuNCs (K and O), null (L and P), respectively. The concentrations of CuNCs and H<small><sub>2</sub></small>O<small><sub>2</sub></small> were 10 mg mL<small><sup>−1</sup></small> and 1 mM, respectively in all cases. (B) Images of the wounded mice treated with CuNCs plus H<small><sub>2</sub></small>O<small><sub>2</sub></small>, CuNCs, H<small><sub>2</sub></small>O<small><sub>2</sub></small> and null, respectively. Republished with permission from ref. <a title="Select to navigate to references" href="#cit54">54</a>. Copyright 2015 Royal Society of Chemistry. (C) Antimicrobial activity of GSH-CuNCs (inset: IC<small><sub>50</sub></small> value). (D) Fold growth of <span class="italic">E. coli</span>, DH5α at different concentrations of GSH-CuNCs. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit136">136</a>. Copyright 2018 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Baghdasaryan <span class="italic">et al.</span> reported the synthesis and antibacterial properties of GSH-CuNCs toward <span class="italic">E. coli</span>, DH5α cells.<a title="Select to navigate to references" href="#cit136"><sup><span class="sup_ref">136</span></sup></a> The antibacterial properties were studied at different cluster concentrations. It was found that with increasing cluster concentration, much lower bacterial growth was observed as compared to control bacteria (<a title="Select to navigate to figure" href="#imgfig31">Fig. 31C</a>). Moreover, the studies of the growth kinetics of DH5α cells treated with CuNCs revealed that at low concentrations, the clusters showed a bacteriostatic effect, whereas, at higher concentrations, there was a transition from bacteriostatic to bactericidal toward <span class="italic">E. coli</span>, DH5α cells (<a title="Select to navigate to figure" href="#imgfig31">Fig. 31D</a>).</p>
          <p class="otherpara">Hong Yan Zou <span class="italic">et al.</span> have also studied the antimicrobial properties of dopamine-capped CuNCs using Gram-positive <span class="italic">S. aureus</span> bacterial strains.<a title="Select to navigate to references" href="#cit148"><sup><span class="sup_ref">148</span></sup></a> The clusters showed dose-dependent antimicrobial properties with a minimum inhibitory concentration of 158 μg mL<small><sup>−1</sup></small>.</p>
          <p class="otherpara">The bacterial death associated with the bacterial cell membrane damage was considered as an antimicrobial mechanism for Gram-positive bacteria <span class="italic">S. aureus</span> and <span class="italic">B. subtilis</span> (Bacillus subtilis) and Gram-negative bacteria <span class="italic">E. coli</span> and <span class="italic">P. aeruginosa</span> (Pseudomonas aeruginosa).<a title="Select to navigate to references" href="#cit412"><sup><span class="sup_ref">412</span></sup></a> Moreover, TA-CuNCs possessed selective antibacterial activity against Gram-positive bacteria by inhibiting the bacterial growth.</p>
          <p class="otherpara">A combined bioimaging/drug delivery/antibacterial multimodal CuNCs-based system was recently developed by Anitha T. Simon <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit413"><sup><span class="sup_ref">413</span></sup></a> BSA-CuNCs together with the drug kanamycin loaded inside the luminescent hydroxyapatite nanoparticles (HAP) (<a title="Select to navigate to figure" href="#imgfig32">Fig. 32</a>) were applied first to image the bacteria, then to transport the drug and finally to explore their antibacterial activity against Gram-positive and Gram-negative bacteria.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig32"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f32_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f32.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f32.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 32 </b> <span id="fig32"><span class="graphic_title">[A(I–III)] Epifluorescence images of AO/PI stained control biofilm: [A(I)] AO-stained, [A(II)] PI-stained, and [A(III)] merged image. The arrows in the merged image of the control biofilm denote the regions with dead and live bacterial cells. [B(I–III)] Biofilm treated with kanamycin-loaded doped HAP NPs: [B(I)] AO-stained, [B(II)] PI-stained, and [B(III)] merged image. The arrows in the merged image of the treated biofilm denote the regions with dead and live bacterial cells. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit413">413</a>. Copyright 2019 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The drug-free nanocarrier showed no bactericidal activity, whereas kanamycin-loaded cargo exposed antibacterial properties. The system was also effective in eradicating <span class="italic">P. aeruginosa</span> biofilm.</p>
        </div>
        <div>
          
          <span id="sect7640"/><span class="c_heading_indent">4.2.3. Drug delivery. </span>
          <span>Arun Chattopadhyay and co-workers demonstrated the utility of red fluorescent CuNCs in PVA hydrogel networks for the imaging of HeLa cells (<a title="Select to navigate to figure" href="#imgfig33">Fig. 33</a>).<a title="Select to navigate to references" href="#cit32"><sup><span class="sup_ref">32</span></sup></a> Moreover, the CuNCs-PVA hydrogel network was used as a drug delivery machinery or nanocarrier to load the anticancer drug cisplatin (CP) to ensure apoptotic cell death. CuNCs generate ROS, which together with the drug greatly enhances the anticancer activity of the complex nano-carrier through the synergistic effect.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig33"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f33_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f33.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f33.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 33 </b> <span id="fig33"><span class="graphic_title">Confocal microscopic images of HeLa cells treated with the CP-loaded CuNC–hydrogel composite, recorded after 4 h of incubation. (A) Image under bright-field and (B) fluorescence image of HeLa cells under green light. Scale bar is 20 μm. (C and D) FESEM images of HeLa cells, treated with the CuNC–hydrogel composite and CP-loaded CuNC–hydrogel composite for 24 h, revealing apoptotic cell deaths. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit32">32</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">As already discussed in section 4.4.3, hydroxyapatite nanoparticles doped with fluorescent BSA-CuNCs have been designed as an effective kanamycin drug delivery carrier.<a title="Select to navigate to references" href="#cit413"><sup><span class="sup_ref">413</span></sup></a> The disruption of the bacterial cell wall after treating with the nanocarrier suggested the release of the drug from the cavity of the carrier and interaction with the bacteria through the depletion of NADH, thus causing unprevented cell damage.</p>
          <p class="otherpara">The real-time monitoring of transferrin-templated CuNCs in conjugation with the anticancer drug doxorubicin (Tf-CuNCs-DOX) was performed in cancerous and normal cells <span class="italic">in vitro</span> for drug localization and release purposes (<a title="Select to navigate to figure" href="#imgfig34">Fig. 34</a>).<a title="Select to navigate to references" href="#cit414"><sup><span class="sup_ref">414</span></sup></a> It was observed that TfR overexpressed cells showed higher drug release profiles with pronounced therapeutic activity. <span class="italic">In vivo</span> studies on Swiss albino mice bearing a tumour of Dalton's lymphoma ascites (DLA cells) demonstrated tumour volume reduction after the administration of the Tf-CuNCs-DOX nanodrug, mainly due to the targeted internalization and longer circulation time.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig34"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f34_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f34.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f34.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 34 </b> <span id="fig34"><span class="graphic_title">Scheme showing the formation of a targeted nanodrug for FRET-assisted bioimaging, targeted delivery and synergistic therapeutic activity (<span class="italic">in vitro</span> and <span class="italic">in vivo</span>). Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit414">414</a>. Copyright 2018 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
        <div>
          
          <span id="sect7661"/><span class="c_heading_indent">4.2.4. Nanotoxicology of CuNCs. </span>
          <span>Although the studies described in the previous sections mostly indicated no or low cytotoxicity of CuNCs, regardless of the size and the surface ligand composition/chemistry, Yong Chen and co-workers showed that BSA-CuNCs trigger muscle cell apoptosis and atrophy of C2C12 myotubes.<a title="Select to navigate to references" href="#cit415"><sup><span class="sup_ref">415</span></sup></a> In brief, the treatment of the muscle cells with varying concentrations of CuNCs showed a cell viability decrease by 88% when the concentration of the clusters was increased to 50 μg mL<small><sup>−1</sup></small>. Moreover, the percentages of both early and late apoptotic cells increased according to flow cytometry analyses. It was shown that with decreasing cell viability ROS generation increased drastically, associated with the decrease in the mitochondrial membrane potential and induction of oxidative stress. Thus, CuNCs induce cytotoxicity through the mitochondrial pathway, which is postulated by an increase in the protein expression ratio of Bax/Bcl-2 and caspase-3/9 activity. Moreover, the morphological impairment of myotubes with a progressive decrease in size was observed after CuNCs treatment. It was shown that CuNCs induced atrophy through the impairment of the Ser/Thr Akt kinase signalling pathway, followed by the transcription of atrophy-related genes. The mechanism of muscle cell apoptosis and atrophy is shown schematically in <a title="Select to navigate to figure" href="#imgfig35">Fig. 35</a>.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig35"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f35_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f35.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f35.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 35 </b> <span id="fig35"><span class="graphic_title">Schematic illustration of CuNC-induced cytotoxicity, ROS production, apoptosis, and atrophy in muscle cells. CAT, catalase; CuNCs, copper nanoclusters; GSH, glutathione; LDH, lactic dehydrogenase; MMP, mitochondrial membrane potential; ROS, reactive oxygen species. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit415">415</a>. Copyright 2015 John Wiley &amp; Sons, Ltd.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
      
      
        
        <h3 id="sect7671"><span class="b_heading">4.3. Applications in catalysis</span></h3>
        <span>Catalytic processes based on the use of nanomaterials as catalysts are currently a dynamic research field. The high interest in clusters is governed by their size-dependent and molecular-like properties. On decreasing the size of the clusters to the subnanometer range, the surface energy increases drastically. This, on the other hand, makes the clusters highly catalytically active for chemical transformations. The literature is replete with examples of very small clusters (Cu<small><sub><span class="italic">n</span></sub></small>, where <span class="italic">n</span> &lt; 5), that are very active and selective catalysts in many homogeneous and heterogeneous catalytic conversions.<a title="Select to navigate to references" href="#cit416"><sup><span class="sup_ref">416</span></sup></a> For instance, Cu<small><sub>3</sub></small> and Cu<small><sub>4</sub></small> clusters prepared using size-selection in combination with a cluster source were applied in methanol synthesis from CO<small><sub>2</sub></small><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><a title="Select to navigate to reference" href="#cit417"><sup><span class="sup_ref">417,418</span></sup></a> and in CO oxidation.<a title="Select to navigate to references" href="#cit419"><sup><span class="sup_ref">419</span></sup></a> Together with extensive DFT calculations, the catalytic activity of these clusters in various reactions was found to be strongly size-dependent. Cu<small><sub>5</sub></small> cluster species were reported to be highly active in propene oxidation<a title="Select to navigate to references" href="#cit420"><sup><span class="sup_ref">420</span></sup></a> and CO<small><sub>2</sub></small> decomposition.<a title="Select to navigate to references" href="#cit421"><sup><span class="sup_ref">421</span></sup></a> More thorough examples of catalytically active tiny copper clusters can be found in the review by Corma.<a title="Select to navigate to references" href="#cit422"><sup><span class="sup_ref">422</span></sup></a></span>
        <p class="otherpara">In the following (sub)sections, we restrict our discussion to the catalytic conversions utilizing copper nanoclusters (containing more than 6 metal atoms) as heterogeneous/homogeneous catalysts prepared by “wet chemical” methods.</p>
        <div>
          
          <span id="sect7690"/><span class="c_heading_indent">4.3.1. Electrocatalytic reactions. </span>
          <span>Wentao Wei <span class="italic">et al.</span> reported the synthesis of subnanometer-sized CuNCs protected with 2-mercapto-5-<span class="italic">n</span>-propylpyrimidine (MPP) ligands.<a title="Select to navigate to references" href="#cit109"><sup><span class="sup_ref">109</span></sup></a> Mass spectroscopic characterization (ESI) of the sample revealed the formation of singly-charged Cu<small><sub>8</sub></small>L<small><sub>4</sub></small> species (L is MPP ligand). CuNCs displayed high catalytic activity for oxygen reduction. The onset potentials were comparable to Au clusters and some Pt catalysts.</span>
          <p class="otherpara">Noelia Vilar-Vidal <span class="italic">et al.</span> studied the size-dependent catalytic properties of CuNCs, namely Cu<small><sub>5</sub></small>, Cu<small><sub>13</sub></small> and Cu<small><sub>20</sub></small>.<a title="Select to navigate to references" href="#cit423"><sup><span class="sup_ref">423</span></sup></a> The catalytic reduction of methylene blue (MB) to leucomethylene blue (LMB) by hydrazine was performed in the presence of CuNCs.<a title="Select to navigate to reference" href="#cit114"><sup><span class="sup_ref">114,423</span></sup></a> It was found that small clusters were catalytically very active, whereas with increasing the cluster size, the catalytic activity of the cluster was reduced. Cu<small><sub>20</sub></small> clusters did not catalyse the reaction at all. The reaction mechanism is explained as a cluster-mediated electron transfer from the donor (N<small><sub>2</sub></small>H<small><sub>4</sub></small>) to the acceptor (MB) through the conduction band of the CuNCs. Consequently, with increasing the size of the cluster, the band gap became smaller and thus, the cluster became catalytically less active, as is schematically depicted in <a title="Select to navigate to figure" href="#imgfig36">Fig. 36</a>.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig36"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f36_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f36.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f36.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 36 </b> <span id="fig36"><span class="graphic_title">Schematic energy diagram showing the catalytic activity of different CuNCs used for MB reduction by N<small><sub>2</sub></small>H<small><sub>4</sub></small>. Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit423">423</a>. Copyright 2012 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">De-en Jiang and co-workers studied the catalytic CO<small><sub>2</sub></small> reduction to (i) HCOOH or (ii) CO when using a polyhydrido [Cu<small><sub>32</sub></small>H<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>] cluster as an active catalyst.<a title="Select to navigate to references" href="#cit188"><sup><span class="sup_ref">188</span></sup></a> According to DFT calculations, in the first reaction, after CO<small><sub>2</sub></small> adsorption, the addition of a negative hydrogen to the positive carbon of CO<small><sub>2</sub></small> leads to the formation of an intermediate HCOO* species through the lattice hydride mechanism. The addition of a second hydrogen results in the formation of the final product leaving the cluster with two hydrogen vacancies. The loss of lattice hydrides is rapidly compensated <span class="italic">via</span> subsequent proton-reduction steps (<a title="Select to navigate to scheme" href="#imgsch9">Scheme 9A</a>). In the second reaction, again <span class="italic">via</span> lattice hydride mechanism (<a title="Select to navigate to scheme" href="#imgsch9">Scheme 9B</a>), the addition of a first hydride to an oxygen atom of CO<small><sub>2</sub></small> resulted in the formation of an intermediate COOH* species bonded to the cluster. The cleavage of the CO–OH bond in the intermediate species leads to the release of CO and the formation of a OH* intermediate. The release of water and an electrochemical proton-reduction process recovers the lost lattice hydrides in the catalytic cycle and brings the cluster back to its native composition. In accordance with theoretical calculations, the experimental electrocatalytic CO<small><sub>2</sub></small> reduction to HCOOH was favoured at low overpotentials.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch9"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s9.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 9 </b> <span id="sch9"><span class="graphic_title">Theoretical reaction schemes for CO<small><sub>2</sub></small> electroreduction on [Cu<small><sub>32</sub></small>H<small><sub>20</sub></small>{S<small><sub>2</sub></small>P(O<small><sup>i</sup></small>Pr)<small><sub>2</sub></small>}<small><sub>12</sub></small>] to form (A) HCOOH and (B) CO <span class="italic">via</span> six different pathways as described in ref. <a title="Select to navigate to references" href="#cit188">188</a> (Reprinted with permission, Copyright 2017 American Chemical Society). The calculated free energy difference (Δ<span class="italic">G</span>) for each elementary step is shown under each intermediate state. The most favourable reaction pathway is highlighted in blue.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Tomoaki Tanase and co-workers studied the hydrosilylation of CO<small><sub>2</sub></small> with tertiary hydrosilanes catalysed by [Cu<small><sub>8</sub></small>(μ-H)<small><sub>6</sub></small>(μ-dppm)<small><sub>5</sub></small>](PF<small><sub>6</sub></small>)<small><sub>2</sub></small>.<a title="Select to navigate to references" href="#cit157"><sup><span class="sup_ref">157</span></sup></a> The corresponding silyl formats were obtained after 48 h in acetonitrile, at 45 °C and 1 atm pressure of CO<small><sub>2</sub></small>.</p>
          <p class="otherpara">Recently Kingshuk Basu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a> prepared red-emitting cys-CuNCs and demonstrated its catalytic activity for the reduction of 4-nitrophenol to 4-aminophenol at room temperature. The catalyst can be recycled without loss in activity.</p>
          <p class="otherpara">Jiangfei Cao and co-workers studied the catalytic degradation reaction of amido black 10B (AB-10B) dye using <span class="small_caps">L</span>-cys-CuNCs supported on red globe flower carbon.<a title="Select to navigate to references" href="#cit424"><sup><span class="sup_ref">424</span></sup></a> Unlike control catalysts (carbon and clusters alone), up to 98% degradation efficiency was reached when the Fenton-like CuNCs/C-H<small><sub>2</sub></small>O<small><sub>2</sub></small> system was used as a catalyst. Several intermediates were confirmed by LC-MS analysis for the degradation reaction of AB-10B dye.</p>
        </div>
        <div>
          
          <span id="sect7761"/><span class="c_heading_indent">4.3.2. Catalytic reactions. </span>
          
            
            <span id="sect7764"/><br/><span class="d_heading_indent">4.3.2.1. Oxidation reactions. </span>
            <span>Chan Wang <span class="italic">et al.</span> studied the catalytic oxidation of styrene in the presence and absence of BSA-templated CuNCs.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> In the presence of CuNCs, the styrene oxidation resulted in several reaction products with benzaldehyde as the major product and with 70% conversion of styrene. In the absence of catalyst, the oxidation reaction occurred very slowly with 7% of styrene conversion.</span>
            <p class="otherpara">Finally, CuNCs were found to catalyse the oxidation of peroxidase substrate 3,3′,5,5′-tetramethylbenzidine (TMB) (ox-TMB)<a title="Select to navigate to references" href="#cit148"><sup><span class="sup_ref">148</span></sup></a> and ethanol.<a title="Select to navigate to references" href="#cit172"><sup><span class="sup_ref">172</span></sup></a></p>
          
          
            
            <span id="sect7773"/><br/><span class="d_heading_indent">4.3.2.2. “Click” reactions. </span>
            <span>Cu-catalysed [3 + 2] Huisgen cycloadditions of terminal alkynes and organic azides (CuAAC) are classic examples of “click” chemistry, where copper compounds are applied as heterogeneous or homogeneous catalysts in 5-membered heteroatom ring formation reactions. The review by Rajender S. Varma<a title="Select to navigate to references" href="#cit425"><sup><span class="sup_ref">425</span></sup></a> discusses the catalytic applications of Cu-based compounds (including click reactions), whereas the review by Francisco Alonso<a title="Select to navigate to references" href="#cit426"><sup><span class="sup_ref">426</span></sup></a> is focused on the “click” chemistry of nanostructured copper compounds. In this section, we will mainly consider the applications of atomically precise copper clusters as both supported and unsupported catalysts in “click” reactions. In this context, Hayton and co-workers studied the catalytic activity of supported (on silica) and unsupported [Cu<small><sub>20</sub></small>(CCPh)<small><sub>12</sub></small>(OAc)<small><sub>6</sub></small>] clusters for the [3 + 2] cycloaddition of alkynes (HCCR: R = Ph, CO<small><sub>2</sub></small>Et, and <small><sup><span class="italic">t</span></sup></small>Bu) with benzyl azide.<a title="Select to navigate to references" href="#cit177"><sup><span class="sup_ref">177</span></sup></a> Although comparable triazole yields were obtained with both homogeneous and heterogeneous catalysts, the supported cluster remained stable under applied reaction conditions, whereas marked precipitation and colour change were detected for the unsupported cluster.</span>
            <p class="otherpara">Didier Astruc and co-workers studied another set of “click” reactions when applying PEG-CuNCs as homogeneous and heterogeneous catalysts.<a title="Select to navigate to references" href="#cit427"><sup><span class="sup_ref">427</span></sup></a> According to the preliminary results, the supported cluster on a Santa Barbara Amorphous-15 (SBA-15) mesoporous silica sieve showed better catalytic performance than the unsupported one (<a title="Select to navigate to scheme" href="#imgsch10">Scheme 10</a>). Besides, the exposure of the catalyst to air led to the formation of Cu<small><sub>2</sub></small>O species on the cluster surface, which, on the other hand, boosted the catalytic activity of the cluster.</p>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch10"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s10.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 10 </b> <span id="sch10"><span class="graphic_title">CuAAC reactions catalysed by supported and unsupported CuNP-PEGs as described in ref. <a title="Select to navigate to references" href="#cit427">427</a>. Republished with permission, Copyright 2017 Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          
          
            
            <span id="sect7793"/><br/><span class="d_heading_indent">4.3.2.3. Hydrogenation of ketones. </span>
            <span>Nanfeng Zheng and co-workers studied the catalytic hydrogenation reactions of 2-hexanone and 3-hexanone to the corresponding alcohols by using a [Cu<small><sub>25</sub></small>H<small><sub>10</sub></small>(SPhCl<small><sub>2</sub></small>)<small><sub>18</sub></small>]<small><sup>3−</sup></small> cluster as an active catalyst at room temperature.<a title="Select to navigate to references" href="#cit182"><sup><span class="sup_ref">182</span></sup></a> Combined with DFT studies, a single-site hydrogenation mechanism was proposed, where HCHO (a model ketone chosen for the calculations) was interacting with the cluster through the formation of an intermediate alkoxy group with a μ<small><sub>3</sub></small>-H atom from the cluster. The neighbouring μ<small><sub>6</sub></small>-H from the cluster diffused close to the O atom of the alkoxy group to form the reaction product. Two hydrogen vacancies formed during this process were filled by the dissociation of molecular hydrogen to complete the catalytic cycle.</span>
          
          
            
            <span id="sect7805"/><br/><span class="d_heading_indent">4.3.2.4. Carbonylation reactions. </span>
            <span>The catalytic activity of the [Cu<small><sub>32</sub></small>(PET)<small><sub>24</sub></small>H<small><sub>8</sub></small>Cl<small><sub>2</sub></small>](PPh<small><sub>4</sub></small>)<small><sub>2</sub></small> cluster was studied for the carbonylation reaction of anilines.<a title="Select to navigate to references" href="#cit189"><sup><span class="sup_ref">189</span></sup></a> The reactions of substituted anilines with diisopropyl azodicarboxylate in the presence of a catalyst showed 100% conversions to carbamate products (<a title="Select to navigate to scheme" href="#imgsch11">Scheme 11</a>). Although complete aniline conversions were also reached with a copper salt and a CuPET complex as catalysts, and the yields of carbamate products were significantly reduced as compared to the cluster.</span>
            <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgsch11"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-s11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-s11.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-s11.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Scheme 11 </b> <span id="sch11"><span class="graphic_title">Carbonylation of anilines as described in ref. <a title="Select to navigate to references" href="#cit189">189</a>. Reprinted with permission, Copyright 2020 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          
        </div>
      
      
        
        <h3 id="sect7820"><span class="b_heading">4.4. Applications in optoelectronic devices</span></h3>
        <span>Metal nanoclusters, particularly CuNCs, have been considered as a new generation of light-emitting diodes (LEDs) with high colour rendering index (CRI) and wide-gamut display lighting. Although commercially available LED emitters mostly use the combination of red, green, and blue (RGB) LED phosphors, orange and yellow are also occasionally applied. The main goal is to mix or convert blue or near-violet LED-sources into white LED devices (WLEDs).</span>
        <p class="otherpara">Benefiting from the strong and tunable PL properties of CuNCs, the Rogach group has developed a method for preparing down-conversion white light-emitting devices (LEDs).<a title="Select to navigate to references" href="#cit428"><sup><span class="sup_ref">428</span></sup></a> The synthetic method involves the formation of PVP-stabilized CuNCs and further treatment with various electron-rich ligands, such as GSH, cysteine, cysteamine, <span class="italic">etc</span>. They showed that the initial QY of CuNCs reported to be 8%, can be greatly enhanced (for several folds) with the treatment of the above-mentioned ligands.</p>
        <p class="otherpara">The highest QY and stability were detected after GSH treatment (QY of 27%). This was attributed to the presence of electron-rich groups (–SH, –NH<small><sub>2</sub></small>, –COOH) of the ligand. Furthermore, the pre-treatment of the GSH-CuNCs powder with thermally-curable silicone resin and hardener, followed by mixing with green and red phosphors and finally coating the final mixture onto a commercial UV GaN chip, resulted in the fabrication of a white LED device (<a title="Select to navigate to figure" href="#imgfig37">Fig. 37A</a>). When excited at 380 nm, the PL peaks of the cluster, green and red phosphors were positioned at 434, 538 and 620 nm, respectively (<a title="Select to navigate to figure" href="#imgfig37">Fig. 37Aa</a>). Pictures of blue and white LEDs, and their emission spectra, are presented in <a title="Select to navigate to figure" href="#imgfig37">Fig. 37A(b, c and d)</a>. The typical CIE (Commission Internationale de l'Eclairage) chromaticity coordinates of the device were 0.350, 0.334 (<a title="Select to navigate to figure" href="#imgfig37">Fig. 37Ae</a>), with a high colour rendering index (CRI) value of 92 (<a title="Select to navigate to figure" href="#imgfig37">Fig. 37Af</a>), correlated colour temperature (CCT) of 4742 K and a luminous efficiency of 9.8 lm W<small><sup>−1</sup></small>.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig37"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f37_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f37.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f37.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 37 </b> <span id="fig37"><span class="graphic_title">(A) (a) PLE spectrum of powdered GSH-treated CuNCs (purple line); the PL spectra of powdered GSH-treated Cu NCs (blue line), VG61E (green line) and MPR635 (red line) phosphors, with the photographs of the respective powders. Photographs of (b) blue LED and (c) white LED. (d) Emission spectrum, (e) CIE chromaticity coordinates and (f) CRI at various Munsell codes of white LEDs. Republished with permission from ref. <a title="Select to navigate to references" href="#cit428">428</a>. Copyright 2016 Royal Society of Chemistry. (B) Fluorescence images of the LEDs from CuNCs self-assembled ribbons (a), sheets (b), and AuNCs sheets (c). (d) Fluorescence images of the WLED from CuNCs ribbons, CuNCs sheets, and AuNCs sheets. (e) The CIE chromaticity coordinates of the WLED in (d). The corresponding emission spectra are shown in (f), (g), (h), and (i). Reprinted with permission from ref. <a title="Select to navigate to references" href="#cit139">139</a>. Copyright 2015 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Going beyond, the same group fabricated another white LED by utilizing only CuNCs and no RGB phosphors.<a title="Select to navigate to references" href="#cit429"><sup><span class="sup_ref">429</span></sup></a> In brief, the orange-emitting CuNCs were prepared in water from the reduction with GSH. The latter was used as both reducing and capping ligand. The solvent (ethanol)-driven AIE effect was applied to improve the QY of the as-prepared CuNCs (in solution 24%, solid state 43%). Blue emitting CuNCs were prepared as mentioned before.<a title="Select to navigate to references" href="#cit428"><sup><span class="sup_ref">428</span></sup></a> The integration of orange and blue CuNCs onto commercially available GaN LED chips, resulted in the fabrication of all-CuNCs-based WLEDs. The device emitted white light with CIE, CRI, and CCT of (0.36, 0.31), 92, and 4163 K, respectively. In another case, Rogach and co-workers prepared a colour down-conversion device based on composite polymer/CuNC films.<a title="Select to navigate to references" href="#cit430"><sup><span class="sup_ref">430</span></sup></a> The copolymerization of chitosan with hypromellose resulted in the formation of a biocompatible graft copolymer, called hypromellose grafted chitosan (HGC). The latter was further applied as a depositing template to load GSH-CuNCs. The electrostatic interaction between the positively charged polymer network and negatively charged CuNCs resulted in the aggregation of the clusters showing enhanced AIE with QY as high as 42%. The obtained nanocomposite HGC/CuNCs film was used as a remote phosphor in a UV-LED chip with CIE, CRI and CCT of (0.493, 0.441), 70.1, and 2505 K, respectively. Another WLED with a colour coordinate at (0.32, 0.36) was fabricated by Yang and co-workers by combining blue-green-emitting CuNCs ribbons, yellow-emitting CuNCs sheets and AuNCs sheets with red emission (<a title="Select to navigate to figure" href="#imgfig37">Fig. 37B</a>).<a title="Select to navigate to references" href="#cit139"><sup><span class="sup_ref">139</span></sup></a> Note that the synthesis and AIE enhancement of these clusters were described in the section 3.3.5.</p>
        <p class="otherpara">In 2020, Qiong Jia and co-workers reported a novel orange LED by covering a commercial UV LED chip with GSH-CuNCs/Zn-HDS powder.<a title="Select to navigate to references" href="#cit369"><sup><span class="sup_ref">369</span></sup></a> The developed LED with CIE coordinate of (0.51, 0.45), CCT of 2461 and CRI of 73.5 proved to be an excellent nanocomposite for light-emitting devices.</p>
        <p class="otherpara">Nanfeng Zheng and co-workers reported the synthesis, crystal structure and applications of Cu<small><sub>53</sub></small> clusters as a precursor to CuI films for perovskite solar cells.<a title="Select to navigate to references" href="#cit194"><sup><span class="sup_ref">194</span></sup></a> In brief, the iodination of Cu<small><sub>53</sub></small> clusters at room temperature resulted in the formation of CuI films that were used as hole transport layers (HTO) for solar cells. More importantly, the developed perovskite solar cell featured a high power conversion efficiency (PCE) of 14.3%, had negligible hysteresis, and high stability (<a title="Select to navigate to figure" href="#imgfig38">Fig. 38</a>).</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig38"><a href="/image/article/2021/NR/d0nr08489a/d0nr08489a-f38_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d0nr08489a-f38.tif" src="/image/article/2021/NR/d0nr08489a/d0nr08489a-f38.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 38 </b> <span id="fig38"><span class="graphic_title">Device structure and cross-sectional SEM image of (A) Cu<small><sub>53</sub></small> and (B) CuI on perovskite films. (C) Energy-level diagram of CuI-based perovskite solar cell (PSC). (D) Best <span class="italic">J</span>–<span class="italic">V</span> curves of devices with CuI (w-CuI) and without CuI (w/o-CuI). FS = forward scan, RS = reverse scan. Adapted with permission from ref. <a title="Select to navigate to references" href="#cit194">194</a>. Copyright 2019 Wiley-VCH Verlag GmbH &amp;Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
    
    
      
      <h2 id="sect7858"><span class="a_heading">5. Summary, challenges, and outlook</span></h2>
      <span>In summary, this review is a testimony to the ever-growing interest and importance of atomically precise copper nanoclusters in the era of modern nanotechnology. As a growing and fascinating era of nanosciences, the field of atomically precise metal nanoclusters, in particular CuNCs, is of paramount importance due to its unique chemical and physical properties.<a title="Select to navigate to references" href="#cit431"><sup><span class="sup_ref">431</span></sup></a> As we have already discussed in previous sections of this review, CuNCs show excellent biocompatibility and are highly photoluminescent, which are important advantages with respect to noble metal clusters. Unlike gold clusters, CuNCs have major stability issues that limit their prospective applications. However, the list of stable, compositionally precise, and highly luminescent clusters is growing continuously. In the following, we address the challenges that remain and the knowledge gaps that need to be filled.</span>
      
        
        <h3 id="sect7863"><span class="b_heading">5.1. Synthesis</span></h3>
        <span>As we have shown in the previous sections, for the preparation of water-soluble CuNCs, both the bottom-up and top-down methods are straightforward and can be easily scaled up for mass production. The critical point here is the selection of an appropriate matrix (polymers, proteins, DNA, <span class="italic">etc</span>.) that will provide effective protection against the aggregation of the clusters and against the oxidation of the metallic core upon exposure to air. Although the water solubility imparts promising features for biomedical applications and biocompatibility, the main drawback is the control of the cluster size. The selective preparation of clusters with certain sizes and shapes, avoiding polydisperse mixtures, is uncontrollable to say at least. Normally, the synthesis of hydrophilic clusters in matrices results in the formation of polydisperse clusters from which certain sizes can be separated and collected <span class="italic">via</span> electrophoretic methods or column chromatography techniques. Note that these separation techniques require longer separation times and are not very effective, especially when neutral ligands are used as a protective shell (in the case of electrophoretic separations). This problem can somehow be overcome when top-down methods are applied. Herein, the initial mixture is etched in the presence of an excessive ligand with a greater binding affinity toward the metallic core to yield the selective formation of certain sizes. Thanks to the crystallization of some clusters, not only can their size and composition be determined, but the extraction of crystallized clusters from the mother liquor makes the separation and purification from other by-products simple. Besides, taking advantage of their solubility differences in polar and non-polar organic solvents, a mixture of two or more clusters can be separated and extracted into the corresponding solvent medium. This property is a huge advantage with respect to aqueous phase soluble clusters. Also, the knowledge of size and composition at the molecular levels is a blueprint for understanding the evolution of the structure and properties with size. This, in turn, is the basis for tuning their size-dependent properties for suitable applications. Thus, future work should be directed towards the development of synthetic procedures and protocols for selectively obtaining certain cluster sizes with high selectivities and yields. Besides, the field of CuNCs could profit more from atomically precise clusters in order to gain a better understanding of the structure-property relationship.</span>
      
      
        
        <h3 id="sect7869"><span class="b_heading">5.2. Characterization</span></h3>
        <span>A variety of analytical and spectroscopic methods have been applied for the characterization of CuNCs. The importance of UV–vis spectroscopy as a “first glance” spectroscopic technique for the analysis and characterization of the clusters has been introduced in section 3.1. We will shortly discuss other techniques, their advantages, and disadvantages for the characterization of small CuNCs.</span>
        <p class="otherpara">A relatively simple way to obtain the size distribution and the morphology of the nanoscale objects is to “see” the nanostructure using transmission electron microscopy (TEM). The basic concept relies on the use of an electron beam passing through the thin layer of the deposited sample on a coated grid. However, the imaging of the small nanoclusters of a few nanometres in size is very challenging. Intense electron beams easily damage the clusters, leading to decomposition or aggregation. That is why sometimes the size distribution observed from TEM is not in agreement with the actual size of the cluster. However, this does not throw shade on the importance of TEM as a microscopic technique. Among other characterization techniques, TEM analysis comes in handy for having a clue about their size distribution.<a title="Select to navigate to reference" href="#cit56"><sup><span class="sup_ref">56,71,110,145</span></sup></a> Although it is impossible to “see” the organic layer around the cluster core in TEM, it provides a hint about the overall core size. For copper nanoclusters, it was even more difficult and challenging to image and obtain the exact size-distribution histograms due to the low stability of the clusters in the solution or under the electron beam.<a title="Select to navigate to reference" href="#cit132"><sup><span class="sup_ref">132,136</span></sup></a> In many reported cases, the discrepancy between the observed size from the TEM microscopic imaging and alternative methods (dynamic light scattering for example)<a title="Select to navigate to references" href="#cit136"><sup><span class="sup_ref">136</span></sup></a> was of major concern. Nonetheless, it is still a powerful technique for estimating the approximate size of a monodisperse sample and the size distribution of polydisperse mixtures of clusters. Thus, future efforts need to be directed toward solving the problem of the stability of the clusters under electron beams for imaging purposes. Aberration-Corrected Scanning Transmission Electron Microscopy (AC-STEM)<a title="Select to navigate to references" href="#cit432"><sup><span class="sup_ref">432</span></sup></a> or similar techniques are more appropriate for the imaging of ultra-small clusters since they are (i) non-destructive (the applied voltages are considerably lower as compared to the conventional TEM technique), and (ii) they provide ultra-high resolution images as well as allow the determination of the number of metal atoms inside the cluster. On this note, STEM techniques have been applied in the size determination (as well as the number of metal atoms) of copper<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31,218,321,326,409,410</span></sup></a> and gold<a title="Select to navigate to reference" href="#cit433"><sup><span class="sup_ref">433,434</span></sup></a> clusters, and copper/gold bimetallic clusters<a title="Select to navigate to references" href="#cit435"><sup><span class="sup_ref">435</span></sup></a> in a more accurate and controllable way.</p>
        <p class="otherpara">X-ray photoelectron spectroscopy is a surface-sensitive analytical technique for determining the composition, oxidation state and empirical formula of the material being analysed. The concept is based on the interaction of the core electrons of the sample with an incident X-ray beam. The ejected electrons have characteristic energies for each energy level for every element present in the sample. Thus, by obtaining a survey XPS spectrum of the analyte, a complete picture of the composition and oxidation state can be extracted. XPS is extensively used for the characterization of CuNCs. Due to the difficulties in preparing and crystallizing stable copper nanoclusters, XPS has become a paramount analytical tool for determining the oxidation state of the metal inside the cluster composition. The binding energies of characteristic Cu 2p<small><sub>1/2</sub></small> and Cu 2p<small><sub>3/2</sub></small> appear at 952.3 and 932.3 eV, respectively, which are due to Cu<small><sup>0</sup></small>.<a title="Select to navigate to reference" href="#cit32"><sup><span class="sup_ref">32,33,121</span></sup></a> However, the binding energy of Cu 2p<small><sub>3/2</sub></small> for Cu<small><sup>+1</sup></small> is only 0.1 eV apart; thus, it is impossible to distinguish Cu<small><sup>+1</sup></small> from Cu<small><sup>0</sup></small> in the cluster. In contrast, Cu<small><sup>2+</sup></small> shows a satellite peak at around 942 eV and hence, the absence of this signal in the survey spectrum indicates the successful reduction of the copper ion. Depending on the ligand protecting the metallic core, specific elements can also be analysed. In the S 2p spectrum, the presence of a strong peak at 165.7 eV indicates the chemisorption of sulphur on the cluster surface. The peaks at the 284.6, 399.4, 530.9 eV can be assigned to C 1s, N 1s and O 1s, respectively.<a title="Select to navigate to reference" href="#cit18"><sup><span class="sup_ref">18,109</span></sup></a> Thus, herein, the main drawback of the technique is the inability to distinguish metallic copper from copper(<span class="small_caps">I</span>).</p>
        <p class="otherpara">Infrared (IR) spectroscopy is one of the most common and widely employed spectroscopic techniques for determining structures and identifying functional groups present in organic and hybrid organic/inorganic materials. It is especially handy in characterizing the MNCs. An indication of the successful anchoring of the thiol group on the cluster surface can be inferred by the absence of the characteristic S–H stretching vibrational mode at 2559 cm<small><sup>−1</sup></small>. Other functional groups such as carboxylates and amines can be directly tested and identified. However, with this technique, we can neither obtain information about the size of the cluster nor judge whether the sample is monodisperse or polydisperse.</p>
        <p class="otherpara">Mass spectrometry (MS) is a crucial analytical technique for the determination of the chemical composition and to check the molecular purity of the cluster sample. It furthermore allows the monitoring of ligand exchange reactions <span class="italic">in situ</span> and the analysis of the intermediate products of such processes at any time.<a title="Select to navigate to references" href="#cit436"><sup><span class="sup_ref">436</span></sup></a> Thus, it can provide insight into reaction mechanisms. Several mass spectrometry techniques based on soft and hard ionization modes have been used for the determination of cluster composition.<a title="Select to navigate to reference" href="#cit172"><sup><span class="sup_ref">172,201</span></sup></a> However, most of such analyses were conducted using matrix-assisted laser desorption/ionization (MALDI) and electrospray ionization (ESI). Both techniques are well suited for the needs in the cluster field.<a title="Select to navigate to references" href="#cit437"><sup><span class="sup_ref">437</span></sup></a> By choosing a proper MALDI matrix and ionization conditions in ESI, a mass detection of atomically precise CuNCs becomes a blueprint for assigning their compositions and following the reaction kinetics. The mass determination of organic phase-soluble clusters with precisely determined molecular compositions was more successful as compared to aqueous phase-soluble clusters (for the majority of reported cases). For instance, the chemical compositions and molecular formulas of Cu<small><sub>14</sub></small>(C<small><sub>2</sub></small>B<small><sub>10</sub></small>H<small><sub>10</sub></small>S<small><sub>2</sub></small>)<small><sub>6</sub></small>(CH<small><sub>3</sub></small>CN)<small><sub>8</sub></small>,<a title="Select to navigate to references" href="#cit172"><sup><span class="sup_ref">172</span></sup></a> [Cu<small><sub>25</sub></small>H<small><sub>22</sub></small>(PPh<small><sub>3</sub></small>)<small><sub>12</sub></small>]Cl,<a title="Select to navigate to references" href="#cit183"><sup><span class="sup_ref">183</span></sup></a> and [Cu<small><sub>81</sub></small>(PhS)<small><sub>46</sub></small>(<small><sup><span class="italic">t</span></sup></small>BuNH<small><sub>2</sub></small>)<small><sub>10</sub></small>(H)<small><sub>32</sub></small>]<small><sup>3+</sup></small> (ref. <a title="Select to navigate to references" href="#cit196">196</a>) clusters determined by ESI-MS fully agree with the composition established by single-crystal X-ray studies. However, for hydrophilic clusters produced within biomolecular templates, the determination of the composition and the chemical formula is far more challenging due to fragmentation and the consequent detection of small species rather than the original molecular ion species.<a title="Select to navigate to reference" href="#cit35"><sup><span class="sup_ref">35,136</span></sup></a> However, some examples have shown the successful determination of the composition when glutathione<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> or other biomolecules<a title="Select to navigate to reference" href="#cit50"><sup><span class="sup_ref">50,55</span></sup></a> were used as templating scaffolds.</p>
        <p class="otherpara">X-ray crystallography is arguably the most important and informative technique in the field of ligand-protected metal clusters. The X-ray structure of the cluster is of paramount importance because it provides information on the “construction” of the cluster core, the arrangement of the ligands, chirality, <span class="italic">etc</span>. However, the main challenge remains the growth of suitable crystals. Technically, the issue is often the small quantities and purity of the sample. Nevertheless, during the last decade, a large number of crystal structures of copper clusters have been solved, as evidenced by section 2.2. In this context, there is still the need to (i) improve the crystallization methods by finding the best solvent and the best conditions, and (ii) apply more sophisticated and automated “robots” for the understanding of the fundamental crystal nucleation processes.</p>
      
      
        
        <h3 id="sect7928"><span class="b_heading">5.3. Size-separation</span></h3>
        <span>Size-exclusion chromatography (SEC) was among the first methods applied for obtaining the narrow size (shape) distribution of nanomaterials when passing through a compact gel medium with various pore sizes. The SEC separation mechanism of nanomaterials is directly related to their sizes. The bigger particles/clusters elute faster in early fractions since they cannot penetrate the small pores and thus, their progress through the gel is not hindered. In contrast, smaller clusters have longer eluting times, because they can penetrate all the pores of the gel that are larger than their sizes. Thus, depending on the nature and properties of the nanomaterial, various gels, and beads compatible with organic and aqueous solvents can be applied in the SEC separation.<a title="Select to navigate to references" href="#cit438"><sup><span class="sup_ref">438</span></sup></a> Although it is easily scalable for the separation of large quantities of clusters, the method is time-consuming, and requires the use of special gels suitable for the properties of a cluster of interest. On the other hand, SEC gels can be applied in the separation of clusters, regardless of their charge state. Electrophoretic separations, on the other hand, are limited to water-soluble and charged clusters;<a title="Select to navigate to references" href="#cit136"><sup><span class="sup_ref">136</span></sup></a> agarose and polyacrylamide (PAGE) gels are widely used as matrices for horizontal/vertical electrophoresis in slab gels. These matrices act as molecular sieves for the separation of molecules based on their size and charge. The gel porosity and hardness as well as temperature affect the separation. The main disadvantages of PAGE separations are the long separation times and the comparably large efforts for the preparation and casting of the gels prior to the separations. In capillary electrophoresis (CE), the ionized species move across the anti-convective gel-free narrow capillaries (normally made of fused silica, FS).<a title="Select to navigate to references" href="#cit439"><sup><span class="sup_ref">439</span></sup></a> However, unlike PAGE, the driving force for CE is electroosmotic flow (EOF), which arises on the walls of the capillary tube due to the applied electric field. The main advantages here are the separation of charged (negatively and positively) and neutral species in a single run and very short running/separation times. Electrophoresis and CE, are quite straightforward for the separation of water-soluble clusters; however, the stability of the clusters in the buffers can be an issue in both techniques.</span>
      
      
        
        <h3 id="sect7935"><span class="b_heading">5.4. Unexplored compositions</span></h3>
        <span>Over the years, there have been impressive achievements in the field of copper nanoclusters. Nevertheless, the synthetic procedures for the preparation of CuNCs are not at the level where one can design a synthesis to obtain a certain size and composition. It is mainly based on the “trial-error” concept but it would be better to have insight into how to obtain some specific clusters. Copper clusters from low nuclearity (Cu<small><sub>6</sub></small>) to high nuclearity (Cu<small><sub>136</sub></small>) have already been discovered during the last few decades (<a title="Select to navigate to table" href="#tab1">Table 1</a>). The majority of the discovered structures do not possess metallic cores; they are rather oxidized and contain chalcogenides as part of their core. Only in the last few years have CuNCs with partial Cu(0) character been discovered, showing unique structural properties.<a title="Select to navigate to reference" href="#cit168"><sup><span class="sup_ref">168,169,172,177,183,192,194</span></sup></a> These are the only known clusters that fulfill the superatom theory with magic numbers for closed electron shells (only 2-electron systems are known for copper). In contrast, for gold nanoclusters, examples with magic electron counts from 2-electron to 58-electron systems are known and further examples are continuously being added.<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11,12</span></sup></a> We are still awaiting the discovery of CuNCs with higher magic numbers. Possibly, this issue is related to the lower stability of CuNCs as compared to their gold counterparts.</span>
      
      
        
        <h3 id="sect7944"><span class="b_heading">5.5. Stability issues</span></h3>
        <span>Biocompatibility, nontoxicity, catalytic activity, and strong photoluminescence are desired properties for any nanomaterials in view of the potential applications. However, a nanomaterial with these properties will not be suitable for applications unless it is sufficiently stable. The stability of the nanomaterials is of foremost importance and without solving stability issues, it will be difficult to proceed towards applications. Hereafter, the discussion is mostly made for small clusters but it also applies to larger particles and assemblies. Several factors need to be considered and carefully analysed to overcome the stability issue. Firstly, the choice of the matrix or protective ligands is of paramount importance. The type of the ligand (thiolate, phosphane, <span class="italic">etc</span>.) affects the stability of the cluster due to the affinity for the metal, and the overall structure of the ligand is also important. Normally, larger ligands, such as proteins, polymers, and DNA oligomers, have a greater potential to protect the cluster from aggregation due to the steric hindrance. Smaller ligands normally cannot effectively protect the core and the clusters are more prone to aggregation. On the other hand, under certain conditions (temperature, pH, solvent, <span class="italic">etc</span>.) the aggregation of these clusters imparts strong PL properties (AIE enhancement).<a title="Select to navigate to reference" href="#cit18"><sup><span class="sup_ref">18,141</span></sup></a></span>
        <p class="otherpara">Secondly, the stability of the clusters depends on the external conditions like the temperature of the medium/environment, the pH of the medium/buffer when dispersed in the aqueous phase and finally the presence of ions. It is important to establish a better understanding of the cluster's stability against such factors.</p>
      
      
        
        <h3 id="sect7952"><span class="b_heading">5.6. Size-composition-property relationships at the nanoscale level</span></h3>
        <span>The quantized nano regime in ultra-small MNCs results in rich molecular-like features in their optical absorption spectra that are strongly dependent on the cluster size. This means that by tuning the size of the cluster, the physicochemical properties can be changed in a controlled manner. This concept has been vastly utilized in the case of gold nanoclusters; however, it remains a complicated task for CuNCs. This is partially because of the difficulty in preparing a series of atomically precise CuNCs with varying sizes. On the other hand, this limits our understanding of the size-dependent properties of clusters of a similar kind. Perhaps, in the future, after having gained enough knowledge on the topic, we will be able to precisely control and tune the properties of CuNCs on demand for specific applications.</span>
      
      
        
        <h3 id="sect7956"><span class="b_heading">5.7. Challenges in applications</span></h3>
        <span>Nanomaterials with various structures, shapes, and morphologies, represent possibilities for applications due to their unique properties. CuNCs have found remarkable applications in catalysis, bioimaging and sensing as we have tried to elaborate in this review. The improvement of CuNCs needs to be the focus in order to find better, inexpensive and stable catalysts, to synthesize highly fluorescent, stable and biocompatible nanodyes that can be commercialized and applied in daily clinical diagnosis. There is certainly a long way to go, however, with the combined efforts of both experimental and theoretical studies, we will be able to synthesize, design and fabricate CuNCs-based universal catalysts for catalytic reactions. There is room for improvement in sensing probes, to ensure high sensitivity and selectivity towards toxic analytes, to ensure a safe and protected environment. Although CuNCs-based fluorescent probes sometimes show impressive sensitivity, they have never been commercialized and applied in real clinical practice. Finally, for biomedical applications, it is necessary to have biocompatible and nontoxic fluorescent dyes to detect any pathogens and disease markers to facilitate the clinical diagnosis, especially in the earlier stages of the disease. Although a vast amount of work has been done in this field, the research for the best is still on.</span>
        <p class="otherpara">We are hopeful that the field of atomically precise CuNCs will expand beyond recognition and will aid enormously in the understanding of the size-dependent properties of materials at the nanoscale.</p>
      
    
    
      <h2 id="sect7961"><span class="a_heading">Author contributions</span></h2>
      <span>All authors contributed equally.</span>
    
    
      <h2 id="sect7964"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <h2 id="sect7968"><span class="a_heading">Acknowledgements</span></h2>
      <span>Financial support from the Swiss National Science Foundation (Grant 200020_172511) and the University of Geneva is kindly acknowledged.</span>
    
    <span id="sect7970"><h2 id="sect7967"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">S. Dehnen, A. Eichhöfer and D. Fenske, <span class="italic">Eur. J. Inorg. Chem.</span>, 2002, <span class="bold">2002</span>, 279 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/1099-0682(20022)2002:2&lt;279::AID-EJIC279&gt;3.0.CO;2-H" title="DOI Link to resource 10.1002/1099-0682(20022)2002:2&lt;279::AID-EJIC279&gt;3.0.CO;2-H">CrossRef</a>.</span></li>
      <li><span id="cit2">O. Fuhr, S. Dehnen and D. Fenske, <span class="italic">Chem. Soc. Rev.</span>, 2013, <span class="bold">42</span>, 1871–1906 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2CS35252D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2CS35252D">RSC</a>.</span></li>
      <li><span id="cit3">C. Coughlan, M. Ibáñez, O. Dobrozhan, A. Singh, A. Cabot and K. M. Ryan, <span class="italic">Chem. Rev.</span>, 2017, <span class="bold">117</span>, 5865–6109 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.6b00376" title="DOI Link to resource 10.1021/acs.chemrev.6b00376">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXlvVaqtrk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28394585%5Buid%5D" title="PubMed Link to resource 28394585">PubMed</a>.</span></li>
      <li><span id="cit4">M. Walter, J. Akola, O. Lopez-Acevedo, P. D. Jadzinsky, G. Calero, C. J. Ackerson, R. L. Whetten, H. Gronbeck and H. Hakkinen, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2008, <span class="bold">105</span>, 9157–9162 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0801001105" title="DOI Link to resource 10.1073/pnas.0801001105">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXosFaksLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18599443%5Buid%5D" title="PubMed Link to resource 18599443">PubMed</a>.</span></li>
      <li><span id="cit5">
          H. Häkkinen, in <span class="italic">Frontiers of Nanoscience</span>, Elsevier Ltd, 1st edn,  2012, vol. 3, pp. 129–157 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Frontiers%20of%20Nanoscience%5BJour%5D%20AND%203%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202012%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit6">Z. Lin, T. Slee and D. M. P. Mingos, <span class="italic">Chem. Phys.</span>, 1990, <span class="bold">142</span>, 321–334 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0301-0104(90)89092-5" title="DOI Link to resource 10.1016/0301-0104(90)89092-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3cXisVSks7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit7">T. P. Martin, T. Bergmann, H. Goehlich and T. Lange, <span class="italic">J. Phys. Chem.</span>, 1991, <span class="bold">95</span>, 6421–6429 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/j100170a009" title="DOI Link to resource 10.1021/j100170a009">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3MXkvF2ks74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit8">H. Häkkinen, <span class="italic">Adv. Phys.: X</span>, 2016, <span class="bold">1</span>, 467–491 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Phys.:%20X%5BJour%5D%20AND%201%5Bvolume%5D%20AND%20467%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit9">C. Lavenn, F. Albrieux, G. Bergeret, R. Chiriac, P. Delichère, A. Tuel and A. Demessence, <span class="italic">Nanoscale</span>, 2012, <span class="bold">4</span>, 7334 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2NR32367B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2NR32367B">RSC</a>.</span></li>
      <li><span id="cit10">P. D. Jadzinsky, G. Calero, C. J. Ackerson, D. A. Bushnell and R. D. Kornberg, <span class="italic">Science</span>, 2007, <span class="bold">318</span>, 430–433 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1148624" title="DOI Link to resource 10.1126/science.1148624">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOjtb%252FP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17947577%5Buid%5D" title="PubMed Link to resource 17947577">PubMed</a>.</span></li>
      <li><span id="cit11">I. Chakraborty and T. Pradeep, <span class="italic">Chem. Rev.</span>, 2017, <span class="bold">117</span>, 8208–8271 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.6b00769" title="DOI Link to resource 10.1021/acs.chemrev.6b00769">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Cnu70%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28586213%5Buid%5D" title="PubMed Link to resource 28586213">PubMed</a>.</span></li>
      <li><span id="cit12">R. Jin, C. Zeng, M. Zhou and Y. Chen, <span class="italic">Chem. Rev.</span>, 2016, <span class="bold">116</span>, 10346–10413 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.5b00703" title="DOI Link to resource 10.1021/acs.chemrev.5b00703">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsVeksLnJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27585252%5Buid%5D" title="PubMed Link to resource 27585252">PubMed</a>.</span></li>
      <li><span id="cit13">S. Shahsavari, S. Hadian-Ghazvini, F. Hooriabad Saboor, I. Menbari Oskouie, M. Hasany, A. Simchi and A. L. Rogach, <span class="italic">Mater. Chem. Front.</span>, 2019, <span class="bold">3</span>, 2326–2356 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9QM00492K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9QM00492K">RSC</a>.</span></li>
      <li><span id="cit14">Y. Lu, W. Wei and W. Chen, <span class="italic">Chin. Sci. Bull.</span>, 2012, <span class="bold">57</span>, 41–47 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11434-011-4896-y" title="DOI Link to resource 10.1007/s11434-011-4896-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xks1ertw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit15">N. K. Das and S. Mukherjee, <span class="italic">Phys. Sci. Rev.</span>, 2018, <span class="bold">3</span>, 1–22 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.revip.2017.11.001" title="DOI Link to resource 10.1016/j.revip.2017.11.001">CrossRef</a>.</span></li>
      <li><span id="cit16">S. Sharma, K. K. Chakrahari, J.-Y. Saillard and C. W. Liu, <span class="italic">Acc. Chem. Res.</span>, 2018, <span class="bold">51</span>, 2475–2483 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.accounts.8b00349" title="DOI Link to resource 10.1021/acs.accounts.8b00349">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSmsrnN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30264984%5Buid%5D" title="PubMed Link to resource 30264984">PubMed</a>.</span></li>
      <li><span id="cit17">S. Maity, D. Bain and A. Patra, <span class="italic">J. Phys. Chem. C</span>, 2019, <span class="bold">123</span>, 2506–2515 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.8b09467" title="DOI Link to resource 10.1021/acs.jpcc.8b09467">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXkslKmsw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit18">X. Jia, J. Li and E. Wang, <span class="italic">Small</span>, 2013, <span class="bold">9</span>, 3873–3879 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/smll.201300896" title="DOI Link to resource 10.1002/smll.201300896">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXnsVyrtrg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23670847%5Buid%5D" title="PubMed Link to resource 23670847">PubMed</a>.</span></li>
      <li><span id="cit19">H.-H. Deng, K.-L. Li, Q.-Q. Zhuang, H.-P. Peng, Q.-Q. Zhuang, A.-L. Liu, X.-H. Xia and W. Chen, <span class="italic">Nanoscale</span>, 2018, <span class="bold">10</span>, 6467–6473 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7NR09449C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7NR09449C">RSC</a>.</span></li>
      <li><span id="cit20">A. Huczko, <span class="italic">Appl. Phys. A: Mater. Sci. Process.</span>, 2000, <span class="bold">70</span>, 365–376 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s003390051050" title="DOI Link to resource 10.1007/s003390051050">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXisFyrur0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit21">Y. Liu, J. Goebl and Y. Yin, <span class="italic">Chem. Soc. Rev.</span>, 2013, <span class="bold">42</span>, 2610–2653 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2CS35369E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2CS35369E">RSC</a>.</span></li>
      <li><span id="cit22">Y. Xie, D. Kocaefe, C. Chen and Y. Kocaefe, <span class="italic">J. Nanomater.</span>, 2016, <span class="bold">2016</span>, 1–10 <a target="_blank" class="DOILink" href="https://doi.org/10.1155/2016/2302595" title="DOI Link to resource 10.1155/2016/2302595">CrossRef</a>.</span></li>
      <li><span id="cit23">M. Cui, Y. Zhao and Q. Song, <span class="italic">TrAC, Trends Anal. Chem.</span>, 2014, <span class="bold">57</span>, 73–82 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.trac.2014.02.005" title="DOI Link to resource 10.1016/j.trac.2014.02.005">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXntlOgsbY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit24">Z. Wang, B. Chen and A. L. Rogach, <span class="italic">Nanoscale Horiz.</span>, 2017, <span class="bold">2</span>, 135–146 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7NH00013H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7NH00013H">RSC</a>.</span></li>
      <li><span id="cit25">M. Zhao, L. Sun and R. M. Crooks, <span class="italic">J. Am. Chem. Soc.</span>, 1998, <span class="bold">120</span>, 4877–4878 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja980438n" title="DOI Link to resource 10.1021/ja980438n">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXis12qurc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit26">Y. Ling, N. Zhang, F. Qu, T. Wen, Z. F. Gao, N. B. Li and H. Q. Luo, <span class="italic">Spectrochim. Acta, Part A</span>, 2014, <span class="bold">118</span>, 315–320 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.saa.2013.08.097" title="DOI Link to resource 10.1016/j.saa.2013.08.097">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXjt1yqsg%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24055680%5Buid%5D" title="PubMed Link to resource 24055680">PubMed</a>.</span></li>
      <li><span id="cit27">Y. Ling, J. X. Li, F. Qu, N. B. Li and H. Q. Luo, <span class="italic">Microchim. Acta</span>, 2014, <span class="bold">181</span>, 1069–1075 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-014-1214-9" title="DOI Link to resource 10.1007/s00604-014-1214-9">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXjsF2itL0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit28">Y. Ling, J. J. Wu, Z. F. Gao, N. B. Li and H. Q. Luo, <span class="italic">J. Phys. Chem. C</span>, 2015, <span class="bold">119</span>, 27173–27177 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.5b09488" title="DOI Link to resource 10.1021/acs.jpcc.5b09488">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsl2ht7bL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit29">Y. Zhong, Q. Wang, Y. He, Y. Ge and G. Song, <span class="italic">Sens. Actuators, B</span>, 2015, <span class="bold">209</span>, 147–153 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2014.11.060" title="DOI Link to resource 10.1016/j.snb.2014.11.060">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitFOns77M" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit30">J. Feng, Y. Ju, J. Liu, H. Zhang and X. Chen, <span class="italic">Anal. Chim. Acta</span>, 2015, <span class="bold">854</span>, 153–160 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2014.11.024" title="DOI Link to resource 10.1016/j.aca.2014.11.024">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitVWhsbnO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25479879%5Buid%5D" title="PubMed Link to resource 25479879">PubMed</a>.</span></li>
      <li><span id="cit31">M. Fernández-Ujados, L. Trapiella-Alfonso, J. M. Costa-Fernández, R. Pereiro and A. Sanz-Medel, <span class="italic">Nanotechnology</span>, 2013, <span class="bold">24</span>, 495601 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0957-4484/24/49/495601" title="DOI Link to resource 10.1088/0957-4484/24/49/495601">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24231856%5Buid%5D" title="PubMed Link to resource 24231856">PubMed</a>.</span></li>
      <li><span id="cit32">R. Ghosh, U. Goswami, S. S. Ghosh, A. Paul and A. Chattopadhyay, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2015, <span class="bold">7</span>, 209–222 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/am505799q" title="DOI Link to resource 10.1021/am505799q">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitVeks7zL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25475566%5Buid%5D" title="PubMed Link to resource 25475566">PubMed</a>.</span></li>
      <li><span id="cit33">Q. Tang, T. Yang and Y. Huang, <span class="italic">Microchim. Acta</span>, 2015, <span class="bold">182</span>, 2337–2343 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-015-1586-5" title="DOI Link to resource 10.1007/s00604-015-1586-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXht12msbbJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit34">W. Dong, C. Sun, M. Sun, H. Ge, A. M. Asiri, H. M. Marwani, R. Ni and S. Wang, <span class="italic">ACS Appl. Nano Mater.</span>, 2020, <span class="bold">3</span>, 312–318 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsanm.9b01958" title="DOI Link to resource 10.1021/acsanm.9b01958">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitl2hsbnF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit35">P.-C. Chen, Y.-C. Li, J.-Y. Ma, J.-Y. Huang, C.-F. Chen and H.-T. Chang, <span class="italic">Sci. Rep.</span>, 2016, <span class="bold">6</span>, 24882 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep24882" title="DOI Link to resource 10.1038/srep24882">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XmvFaktr4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27113330%5Buid%5D" title="PubMed Link to resource 27113330">PubMed</a>.</span></li>
      <li><span id="cit36">R. Gui, J. Sun, X. Cao, Y. Wang and H. Jin, <span class="italic">RSC Adv.</span>, 2014, <span class="bold">4</span>, 29083 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4RA03606A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4RA03606A">RSC</a>.</span></li>
      <li><span id="cit37">H. Zhang, X. Huang, L. Li, G. Zhang, I. Hussain, Z. Li and B. Tan, <span class="italic">Chem. Commun.</span>, 2012, <span class="bold">48</span>, 567–569 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1CC16088E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1CC16088E">RSC</a>.</span></li>
      <li><span id="cit38">S. Xu, F. Chen, M. Deng and Y. Sui, <span class="italic">RSC Adv.</span>, 2014, <span class="bold">4</span>, 15664–15670 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4RA00516C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4RA00516C">RSC</a>.</span></li>
      <li><span id="cit39">W. Wang, F. Leng, L. Zhan, Y. Chang, X. X. Yang, J. Lan and C. Z. Huang, <span class="italic">Analyst</span>, 2014, <span class="bold">139</span>, 2990–2993 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4AN00113C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4AN00113C">RSC</a>.</span></li>
      <li><span id="cit40">F. Gao, P. Cai, W. Yang, J. Xue, L. Gao, R. Liu, Y. Wang, Y. Zhao, X. He, L. Zhao, G. Huang, F. Wu, Y. Zhao, Z. Chai and X. Gao, <span class="italic">ACS Nano</span>, 2015, <span class="bold">9</span>, 4976–4986 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/nn507130k" title="DOI Link to resource 10.1021/nn507130k">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXntlCnu7Y%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25919205%5Buid%5D" title="PubMed Link to resource 25919205">PubMed</a>.</span></li>
      <li><span id="cit41">B. Wang, R. Gui, H. Jin, W. He and Z. Wang, <span class="italic">Talanta</span>, 2018, <span class="bold">178</span>, 1006–1010 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2017.08.102" title="DOI Link to resource 10.1016/j.talanta.2017.08.102">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKqt73J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29136788%5Buid%5D" title="PubMed Link to resource 29136788">PubMed</a>.</span></li>
      <li><span id="cit42">R. Ghosh, A. K. Sahoo, S. S. Ghosh, A. Paul and A. Chattopadhyay, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2014, <span class="bold">6</span>, 3822–3828 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/am500040t" title="DOI Link to resource 10.1021/am500040t">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXjtlSisro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24571718%5Buid%5D" title="PubMed Link to resource 24571718">PubMed</a>.</span></li>
      <li><span id="cit43">M. Zhao, A.-Y. Chen, D. Huang, Y. Zhuo, Y.-Q. Chai and R. Yuan, <span class="italic">Anal. Chem.</span>, 2016, <span class="bold">88</span>, 11527–11532 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.6b02770" title="DOI Link to resource 10.1021/acs.analchem.6b02770">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xhslyisr7J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27809492%5Buid%5D" title="PubMed Link to resource 27809492">PubMed</a>.</span></li>
      <li><span id="cit44">C. Wang, S. Shu, Y. Yao and Q. Song, <span class="italic">RSC Adv.</span>, 2015, <span class="bold">5</span>, 101599–101606 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5RA19421K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5RA19421K">RSC</a>.</span></li>
      <li><span id="cit45">C. Wang, C. Wang, L. Xu, H. Cheng, Q. Lin and C. Zhang, <span class="italic">Nanoscale</span>, 2014, <span class="bold">6</span>, 1775–1781 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3NR04835G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3NR04835G">RSC</a>.</span></li>
      <li><span id="cit46">S. Xu, M. Deng, Y. Sui, Y. Zhang and F. Chen, <span class="italic">RSC Adv.</span>, 2014, <span class="bold">4</span>, 44644–44649 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4RA09769F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4RA09769F">RSC</a>.</span></li>
      <li><span id="cit47">Z. Gao, R. Su, W. Qi, L. Wang and Z. He, <span class="italic">Sens. Actuators, B</span>, 2014, <span class="bold">195</span>, 359–364 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2014.01.051" title="DOI Link to resource 10.1016/j.snb.2014.01.051">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXjslyjt7k%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit48">H. Chen, L. Lin, H. Li, J. Li and J.-M. Lin, <span class="italic">ACS Nano</span>, 2015, <span class="bold">9</span>, 2173–2183 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.5b00141" title="DOI Link to resource 10.1021/acsnano.5b00141">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXitVersb8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25647180%5Buid%5D" title="PubMed Link to resource 25647180">PubMed</a>.</span></li>
      <li><span id="cit49">L. Xiaoqing, L. Ruiyi, L. Xiaohuan and L. Zaijun, <span class="italic">RSC Adv.</span>, 2015, <span class="bold">5</span>, 48835–48841 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5RA07966G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5RA07966G">RSC</a>.</span></li>
      <li><span id="cit50">S. Ghosh, N. K. Das, U. Anand and S. Mukherjee, <span class="italic">J. Phys. Chem. Lett.</span>, 2015, <span class="bold">6</span>, 1293–1298 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.5b00378" title="DOI Link to resource 10.1021/acs.jpclett.5b00378">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2gsrw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26262990%5Buid%5D" title="PubMed Link to resource 26262990">PubMed</a>.</span></li>
      <li><span id="cit51">X. Deng, X. Huang and D. Wu, <span class="italic">Anal. Bioanal. Chem.</span>, 2015, <span class="bold">407</span>, 4607–4613 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-015-8657-7" title="DOI Link to resource 10.1007/s00216-015-8657-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXms1ynu7o%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25893800%5Buid%5D" title="PubMed Link to resource 25893800">PubMed</a>.</span></li>
      <li><span id="cit52">Y. Zhong, J. Zhu, Q. Wang, Y. He, Y. Ge and C. Song, <span class="italic">Microchim. Acta</span>, 2015, <span class="bold">182</span>, 909–915 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-014-1407-2" title="DOI Link to resource 10.1007/s00604-014-1407-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGjurzI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit53">T. Zhao, X.-W. He, W.-Y. Li and Y.-K. Zhang, <span class="italic">J. Mater. Chem. B</span>, 2015, <span class="bold">3</span>, 2388–2394 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4TB02130D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4TB02130D">RSC</a>.</span></li>
      <li><span id="cit54">H. Miao, D. Zhong, Z. Zhou and X. Yang, <span class="italic">Nanoscale</span>, 2015, <span class="bold">7</span>, 19066–19072 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5NR05362E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5NR05362E">RSC</a>.</span></li>
      <li><span id="cit55">A. Singh, T. Rai and D. Panda, <span class="italic">RSC Adv.</span>, 2016, <span class="bold">6</span>, 55539–55545 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6RA09763D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6RA09763D">RSC</a>.</span></li>
      <li><span id="cit56">L. Xiaoqing, L. Ruiyi, L. Zaijun, S. Xiulan, W. Zhouping and L. Junkang, <span class="italic">New J. Chem.</span>, 2015, <span class="bold">39</span>, 5240–5248 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5NJ00831J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5NJ00831J">RSC</a>.</span></li>
      <li><span id="cit57">N. Goswami, A. Giri, M. S. Bootharaju, P. L. Xavier, T. Pradeep and S. K. Pal, <span class="italic">Anal. Chem.</span>, 2011, <span class="bold">83</span>, 9676–9680 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac202610e" title="DOI Link to resource 10.1021/ac202610e">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSqsbrK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22050123%5Buid%5D" title="PubMed Link to resource 22050123">PubMed</a>.</span></li>
      <li><span id="cit58">L. Hu, Y. Yuan, L. Zhang, J. Zhao, S. Majeed and G. Xu, <span class="italic">Anal. Chim. Acta</span>, 2013, <span class="bold">762</span>, 83–86 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2012.11.056" title="DOI Link to resource 10.1016/j.aca.2012.11.056">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhvFWhs7nL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23327949%5Buid%5D" title="PubMed Link to resource 23327949">PubMed</a>.</span></li>
      <li><span id="cit59">A. S. Patel, H. Sahoo and T. Mohanty, <span class="italic">Appl. Phys. Lett.</span>, 2014, <span class="bold">105</span>, 063112 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.4892832" title="DOI Link to resource 10.1063/1.4892832">CrossRef</a>.</span></li>
      <li><span id="cit60">R. S. Aparna, A. D. Anjali, N. John, K. Abha, S. S. Syamchand and S. George, <span class="italic">Spectrochim. Acta, Part A</span>, 2018, <span class="bold">199</span>, 123–129 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.saa.2018.03.045" title="DOI Link to resource 10.1016/j.saa.2018.03.045">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXmt1Wqsrw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29579715%5Buid%5D" title="PubMed Link to resource 29579715">PubMed</a>.</span></li>
      <li><span id="cit61">M. Jayasree, R. S. Aparna, R. R. Anjana, J. S. Anjali Devi, N. John, K. Abha, A. Manikandan and S. George, <span class="italic">Anal. Chim. Acta</span>, 2018, <span class="bold">1031</span>, 152–160 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2018.05.026" title="DOI Link to resource 10.1016/j.aca.2018.05.026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtVSmtb7F" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30119734%5Buid%5D" title="PubMed Link to resource 30119734">PubMed</a>.</span></li>
      <li><span id="cit62">R. S. Aparna, J. S. A. Devi, J. Nebu, S. S. Syamchand and S. George, <span class="italic">J. Photochem. Photobiol., A</span>, 2019, <span class="bold">379</span>, 63–71 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jphotochem.2019.04.043" title="DOI Link to resource 10.1016/j.jphotochem.2019.04.043">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXpsVymsb0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit63">C. F. Monson and A. T. Woolley, <span class="italic">Nano Lett.</span>, 2003, <span class="bold">3</span>, 359–363 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/nl034016+" title="DOI Link to resource 10.1021/nl034016+">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXhtFCgtLs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit64">A. Rotaru, S. Dutta, E. Jentzsch, K. Gothelf and A. Mokhir, <span class="italic">Angew. Chem., Int. Ed.</span>, 2010, <span class="bold">49</span>, 5665–5667 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.200907256" title="DOI Link to resource 10.1002/anie.200907256">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXps1Cqt74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20629055%5Buid%5D" title="PubMed Link to resource 20629055">PubMed</a>.</span></li>
      <li><span id="cit65">H. Zhang, Z. Lin and X. Su, <span class="italic">Talanta</span>, 2015, <span class="bold">131</span>, 59–63 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2014.07.065" title="DOI Link to resource 10.1016/j.talanta.2014.07.065">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXht1Kgs73L" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit66">X. Zhu, H. Shi, Y. Shen, B. Zhang, J. Zhao and G. Li, <span class="italic">Nano Res.</span>, 2015, <span class="bold">8</span>, 2714–2720 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s12274-015-0778-y" title="DOI Link to resource 10.1007/s12274-015-0778-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXpvV2gtbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit67">Y.-S. Borghei, M. Hosseini, M. R. Ganjali and S. Hosseinkhani, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">248</span>, 133–139 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.03.148" title="DOI Link to resource 10.1016/j.snb.2017.03.148">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXls1yisLc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit68">S. Kim, J. H. Kim, W. Y. Kwon, S. H. Hwang, B. S. Cha, J. M. Kim, S. S. Oh and K. S. Park, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 479 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3620-5" title="DOI Link to resource 10.1007/s00604-019-3620-5">CrossRef</a>.</span></li>
      <li><span id="cit69">S. Singh, M. K. Singh and P. Das, <span class="italic">Sens. Actuators, B</span>, 2018, <span class="bold">255</span>, 763–774 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.08.100" title="DOI Link to resource 10.1016/j.snb.2017.08.100">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWhtL3P" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit70">Z. Zhou, Y. Du and S. Dong, <span class="italic">Anal. Chem.</span>, 2011, <span class="bold">83</span>, 5122–5127 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac200120g" title="DOI Link to resource 10.1021/ac200120g">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXntlGjsrc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21612269%5Buid%5D" title="PubMed Link to resource 21612269">PubMed</a>.</span></li>
      <li><span id="cit71">X. Jia, J. Li, L. Han, J. Ren, X. Yang and E. Wang, <span class="italic">ACS Nano</span>, 2012, <span class="bold">6</span>, 3311–3317 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/nn3002455" title="DOI Link to resource 10.1021/nn3002455">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XjslKnu7o%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22417109%5Buid%5D" title="PubMed Link to resource 22417109">PubMed</a>.</span></li>
      <li><span id="cit72">J. Chen, J. Liu, Z. Fang and L. Zeng, <span class="italic">Chem. Commun.</span>, 2012, <span class="bold">48</span>, 1057–1059 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2CC16668B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2CC16668B">RSC</a>.</span></li>
      <li><span id="cit73">G. Liu, Y. Shao, J. Peng, W. Dai, L. Liu, S. Xu, F. Wu and X. Wu, <span class="italic">Nanotechnology</span>, 2013, <span class="bold">24</span>, 345502 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0957-4484/24/34/345502" title="DOI Link to resource 10.1088/0957-4484/24/34/345502">CrossRef</a>.</span></li>
      <li><span id="cit74">L. Zhang, J. Zhao, M. Duan, H. Zhang, J. Jiang and R. Yu, <span class="italic">Anal. Chem.</span>, 2013, <span class="bold">85</span>, 3797–3801 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac4001942" title="DOI Link to resource 10.1021/ac4001942">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXkslWhtbg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit75">X.-P. Wang, B.-C. Yin and B.-C. Ye, <span class="italic">RSC Adv.</span>, 2013, <span class="bold">3</span>, 8633 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3RA23296D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3RA23296D">RSC</a>.</span></li>
      <li><span id="cit76">F. Xu, H. Shi, X. He, K. Wang, D. He, Q. Guo, Z. Qing, L. Yan, X. Ye, D. Li and J. Tang, <span class="italic">Anal. Chem.</span>, 2014, <span class="bold">86</span>, 6976–6982 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac500955r" title="DOI Link to resource 10.1021/ac500955r">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhtVCqsrrN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24958493%5Buid%5D" title="PubMed Link to resource 24958493">PubMed</a>.</span></li>
      <li><span id="cit77">C.-A. Chen, C.-C. Wang, Y.-J. Jong and S.-M. Wu, <span class="italic">Anal. Chem.</span>, 2015, <span class="bold">87</span>, 6228–6232 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.5b00918" title="DOI Link to resource 10.1021/acs.analchem.5b00918">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXosFGhurk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25982038%5Buid%5D" title="PubMed Link to resource 25982038">PubMed</a>.</span></li>
      <li><span id="cit78">H. Qian, C. Liu and R. Jin, <span class="italic">Sci. China: Chem.</span>, 2012, <span class="bold">55</span>, 2359–2365 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11426-012-4669-8" title="DOI Link to resource 10.1007/s11426-012-4669-8">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xhs1OntrbE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit79">T. T. Tsai, C. A. Chen, N. Y. J. Ho, S. Yang and C. F. Chen, <span class="italic">ACS Sens.</span>, 2019, <span class="bold">4</span>, 2885–2892 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssensors.9b01163" title="DOI Link to resource 10.1021/acssensors.9b01163">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvVylsrvK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31576745%5Buid%5D" title="PubMed Link to resource 31576745">PubMed</a>.</span></li>
      <li><span id="cit80">Y. Hu, Y. Wu, T. Chen, X. Chu and R. Yu, <span class="italic">Anal. Methods</span>, 2013, <span class="bold">5</span>, 3577 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3AY40088C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3AY40088C">RSC</a>.</span></li>
      <li><span id="cit81">Z. Wang, L. Si, J. Bao and Z. Dai, <span class="italic">Chem. Commun.</span>, 2015, <span class="bold">51</span>, 6305–6307 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5CC01081K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5CC01081K">RSC</a>.</span></li>
      <li><span id="cit82">J. Liu, J. Chen, Z. Fang and L. Zeng, <span class="italic">Analyst</span>, 2012, <span class="bold">137</span>, 5502–5505 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2AN35885A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2AN35885A">RSC</a>.</span></li>
      <li><span id="cit83">T. Qing, Z. Qing, Z. Mao, X. He, F. Xu, L. Wen, D. He, H. Shi and K. Wang, <span class="italic">RSC Adv.</span>, 2014, <span class="bold">4</span>, 61092–61095 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4RA11551A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4RA11551A">RSC</a>.</span></li>
      <li><span id="cit84">Q. Song, Y. Shi, D. He, S. Xu and J. Ouyang, <span class="italic">Chem. – Eur. J.</span>, 2015, <span class="bold">21</span>, 2417–2422 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201405726" title="DOI Link to resource 10.1002/chem.201405726">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitVygt7nF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit85">R. Liu, C. Wang, J. Hu, Y. Su and Y. Lv, <span class="italic">TrAC, Trends Anal. Chem.</span>, 2018, <span class="bold">105</span>, 436–452 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.trac.2018.06.003" title="DOI Link to resource 10.1016/j.trac.2018.06.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtFGgt7fN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit86">C. A. Chen, S. C. Chen, M. J. A. Shiddiky, C. F. Chen and K. C. W. Wu, <span class="italic">Chem. Rec.</span>, 2020, <span class="bold">20</span>, 174–186 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/tcr.201900022" title="DOI Link to resource 10.1002/tcr.201900022">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtlOgsLjI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31298485%5Buid%5D" title="PubMed Link to resource 31298485">PubMed</a>.</span></li>
      <li><span id="cit87">Z. Mao, Z. Qing, T. Qing, F. Xu, L. Wen, X. He, D. He, H. Shi and K. Wang, <span class="italic">Anal. Chem.</span>, 2015, <span class="bold">87</span>, 7454–7460 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.5b01700" title="DOI Link to resource 10.1021/acs.analchem.5b01700">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOntr7M" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26112746%5Buid%5D" title="PubMed Link to resource 26112746">PubMed</a>.</span></li>
      <li><span id="cit88">H.-W. Zhu, W.-X. Dai, X.-D. Yu, J.-J. Xu and H.-Y. Chen, <span class="italic">Talanta</span>, 2015, <span class="bold">144</span>, 642–647 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2015.07.022" title="DOI Link to resource 10.1016/j.talanta.2015.07.022">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOksbzJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26452872%5Buid%5D" title="PubMed Link to resource 26452872">PubMed</a>.</span></li>
      <li><span id="cit89">D. Yang, Z. Guo, Y. Tang and P. Miao, <span class="italic">ACS Appl. Nano Mater.</span>, 2018, <span class="bold">1</span>, 168–174 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsanm.7b00078" title="DOI Link to resource 10.1021/acsanm.7b00078">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOiu77P" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit90">Z. Qing, X. He, D. He, K. Wang, F. Xu, T. Qing and X. Yang, <span class="italic">Angew. Chem., Int. Ed.</span>, 2013, <span class="bold">52</span>, 9719–9722 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201304631" title="DOI Link to resource 10.1002/anie.201304631">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhtFCkur7E" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23881724%5Buid%5D" title="PubMed Link to resource 23881724">PubMed</a>.</span></li>
      <li><span id="cit91">M. Raja, J. Subha, F. B. Ali and S. H. Ryu, <span class="italic">Mater. Manuf. Processes</span>, 2008, <span class="bold">23</span>, 782–785 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/10426910802382080" title="DOI Link to resource 10.1080/10426910802382080">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhtlehsLrK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit92">X. Yang, S. Chen, S. Zhao, D. Li and H. Ma, <span class="italic">J. Serb. Chem. Soc.</span>, 2003, <span class="bold">68</span>, 843–847 <a target="_blank" class="DOILink" href="https://doi.org/10.2298/JSC0311843Y" title="DOI Link to resource 10.2298/JSC0311843Y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXkt1Cq" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit93">Q. B. Zhang and Y. X. Hua, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2014, <span class="bold">16</span>, 27088–27095 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4CP03041A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4CP03041A">RSC</a>.</span></li>
      <li><span id="cit94">N. Vilar-Vidal, M. C. Blanco, M. A. López-Quintela, J. Rivas and C. Serra, <span class="italic">J. Phys. Chem. C</span>, 2010, <span class="bold">114</span>, 15924–15930 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp911380s" title="DOI Link to resource 10.1021/jp911380s">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXptlygurY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit95">N. Vilar-Vidal, J. Rivas and M. A. López-Quintela, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2014, <span class="bold">16</span>, 26427–26430 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4CP02148G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4CP02148G">RSC</a>.</span></li>
      <li><span id="cit96">S. Huseyinova, J. Blanco, F. G. Requejo, J. M. Ramallo-López, M. C. Blanco, D. Buceta and M. A. López-Quintela, <span class="italic">J. Phys. Chem. C</span>, 2016, <span class="bold">120</span>, 15902–15908 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.5b12227" title="DOI Link to resource 10.1021/acs.jpcc.5b12227">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtV2htLk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit97">P. Concepción, M. Boronat, S. García-García, E. Fernández and A. Corma, <span class="italic">ACS Catal.</span>, 2017, <span class="bold">7</span>, 3560–3568 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscatal.7b00778" title="DOI Link to resource 10.1021/acscatal.7b00778">CrossRef</a>.</span></li>
      <li><span id="cit98">I. Lisiecki and M. P. Pileni, <span class="italic">J. Am. Chem. Soc.</span>, 1993, <span class="bold">115</span>, 3887–3896 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja00063a006" title="DOI Link to resource 10.1021/ja00063a006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3sXisVWksLo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit99">J. N. Solanki, R. Sengupta and Z. V. P. Murthy, <span class="italic">Solid State Sci.</span>, 2010, <span class="bold">12</span>, 1560–1566 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.solidstatesciences.2010.06.021" title="DOI Link to resource 10.1016/j.solidstatesciences.2010.06.021">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGhtr3L" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit100">I. Lisiecki and M. P. Pileni, <span class="italic">J. Phys. Chem.</span>, 1995, <span class="bold">99</span>, 5077–5082 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/j100014a030" title="DOI Link to resource 10.1021/j100014a030">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXks1Gqtrk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit101">C. Vázquez-Vázquez, M. Bañobre-López, A. Mitra, M. A. López-Quintela and J. Rivas, <span class="italic">Langmuir</span>, 2009, <span class="bold">25</span>, 8208–8216 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/la900100w" title="DOI Link to resource 10.1021/la900100w">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19545135%5Buid%5D" title="PubMed Link to resource 19545135">PubMed</a>.</span></li>
      <li><span id="cit102">H. Zhu, C. Zhang and Y. Yin, <span class="italic">J. Cryst. Growth</span>, 2004, <span class="bold">270</span>, 722–728 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jcrysgro.2004.07.008" title="DOI Link to resource 10.1016/j.jcrysgro.2004.07.008">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXnslensrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit103">H. Kawasaki, Y. Kosaka, Y. Myoujin, T. Narushima, T. Yonezawa and R. Arakawa, <span class="italic">Chem. Commun.</span>, 2011, <span class="bold">47</span>, 7740 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1CC12346G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1CC12346G">RSC</a>.</span></li>
      <li><span id="cit104">R. S. Aparna, S. S. Syamchand and S. George, <span class="italic">J. Cluster Sci.</span>, 2017, <span class="bold">28</span>, 2223–2238 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10876-017-1221-1" title="DOI Link to resource 10.1007/s10876-017-1221-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXmvVCmtLg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit105">M. Brust, M. Walker, D. Bethell, D. J. Schiffrin and R. Whyman, <span class="italic">J. Chem. Soc., Chem. Commun.</span>, 1994, 801–802 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C39940000801&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C39940000801">RSC</a>.</span></li>
      <li><span id="cit106">M. Brust, J. Fink, D. Bethell, D. J. Schiffrin and C. Kiely, <span class="italic">J. Chem. Soc., Chem. Commun.</span>, 1995, 1655 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C39950001655&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C39950001655">RSC</a>.</span></li>
      <li><span id="cit107">P. J. G. Goulet and R. B. Lennox, <span class="italic">J. Am. Chem. Soc.</span>, 2010, <span class="bold">132</span>, 9582–9584 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja104011b" title="DOI Link to resource 10.1021/ja104011b">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXnvVKqtb4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20568767%5Buid%5D" title="PubMed Link to resource 20568767">PubMed</a>.</span></li>
      <li><span id="cit108">Y. Li, O. Zaluzhna, B. Xu, Y. Gao, J. M. Modest and Y. J. Tong, <span class="italic">J. Am. Chem. Soc.</span>, 2011, <span class="bold">133</span>, 2092–2095 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja1105078" title="DOI Link to resource 10.1021/ja1105078">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhtFKgtb0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21268580%5Buid%5D" title="PubMed Link to resource 21268580">PubMed</a>.</span></li>
      <li><span id="cit109">W. Wei, Y. Lu, W. Chen and S. Chen, <span class="italic">J. Am. Chem. Soc.</span>, 2011, <span class="bold">133</span>, 2060–2063 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja109303z" title="DOI Link to resource 10.1021/ja109303z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhsVSisL4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21280578%5Buid%5D" title="PubMed Link to resource 21280578">PubMed</a>.</span></li>
      <li><span id="cit110">D. Li, Z. Chen, Z. Wan, T. Yang, H. Wang and X. Mei, <span class="italic">RSC Adv.</span>, 2016, <span class="bold">6</span>, 34090–34095 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6RA01499B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6RA01499B">RSC</a>.</span></li>
      <li><span id="cit111">T. Zhou, Q. Yao, T. Zhao and X. Chen, <span class="italic">Talanta</span>, 2015, <span class="bold">141</span>, 80–85 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2015.03.056" title="DOI Link to resource 10.1016/j.talanta.2015.03.056">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXmtF2kurs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25966384%5Buid%5D" title="PubMed Link to resource 25966384">PubMed</a>.</span></li>
      <li><span id="cit112">M. Zhao, H. Feng, X. Zhang, H. Ao and Z. Qian, <span class="italic">Analyst</span>, 2017, <span class="bold">142</span>, 4613–4617 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7AN01542A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7AN01542A">RSC</a>.</span></li>
      <li><span id="cit113">J. Ma, P. Chen and H. Chang, <span class="italic">Nanotechnology</span>, 2014, <span class="bold">25</span>, 195502 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0957-4484/25/19/195502" title="DOI Link to resource 10.1088/0957-4484/25/19/195502">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24762432%5Buid%5D" title="PubMed Link to resource 24762432">PubMed</a>.</span></li>
      <li><span id="cit114">X. Jia, X. Yang, J. Li, D. Li and E. Wang, <span class="italic">Chem. Commun.</span>, 2014, <span class="bold">50</span>, 237–239 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3CC47771A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3CC47771A">RSC</a>.</span></li>
      <li><span id="cit115">D. Li, B. Li and S. I. Yang, <span class="italic">Anal. Methods</span>, 2015, <span class="bold">7</span>, 2278–2282 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5AY00219B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5AY00219B">RSC</a>.</span></li>
      <li><span id="cit116">M. Cui, G. Song, C. Wang and Q. Song, <span class="italic">Microchim. Acta</span>, 2015, <span class="bold">182</span>, 1371–1377 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-015-1458-z" title="DOI Link to resource 10.1007/s00604-015-1458-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXjtVals7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit117">Z. Li, S. Guo and C. Lu, <span class="italic">Analyst</span>, 2015, <span class="bold">140</span>, 2719–2725 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5AN00017C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5AN00017C">RSC</a>.</span></li>
      <li><span id="cit118">K. Basu, S. Paul, R. Jana, A. Datta and A. Banerjee, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2019, <span class="bold">7</span>, 1998–2007 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssuschemeng.8b04301" title="DOI Link to resource 10.1021/acssuschemeng.8b04301">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisFClurnM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit119">S. Bhunia, S. Kumar and P. Purkayastha, <span class="italic">ACS Omega</span>, 2019, <span class="bold">4</span>, 2523–2532 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsomega.8b03213" title="DOI Link to resource 10.1021/acsomega.8b03213">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXit1Sktro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31459491%5Buid%5D" title="PubMed Link to resource 31459491">PubMed</a>.</span></li>
      <li><span id="cit120">S. K. Anand, M. R. Mathew and K. G. Kumar, <span class="italic">J. Fluoresc.</span>, 2020, <span class="bold">30</span>, 63–70 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10895-019-02470-5" title="DOI Link to resource 10.1007/s10895-019-02470-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisVCitrjE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31858352%5Buid%5D" title="PubMed Link to resource 31858352">PubMed</a>.</span></li>
      <li><span id="cit121">X. Yang, Y. Feng, S. Zhu, Y. Luo, Y. Zhuo and Y. Dou, <span class="italic">Anal. Chim. Acta</span>, 2014, <span class="bold">847</span>, 49–54 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2014.07.019" title="DOI Link to resource 10.1016/j.aca.2014.07.019">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXht1amsr7J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25261900%5Buid%5D" title="PubMed Link to resource 25261900">PubMed</a>.</span></li>
      <li><span id="cit122">U. Sivasankaran, J. Radecki, H. Radecka and K. Girish Kumar, <span class="italic">Luminescence</span>, 2019, <span class="bold">34</span>, 243–248 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/bio.3601" title="DOI Link to resource 10.1002/bio.3601">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXjvFOjsL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30746849%5Buid%5D" title="PubMed Link to resource 30746849">PubMed</a>.</span></li>
      <li><span id="cit123">A. Dutta, U. Goswami and A. Chattopadhyay, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2018, <span class="bold">10</span>, 19459–19472 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.8b05160" title="DOI Link to resource 10.1021/acsami.8b05160">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpslaitLs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29775047%5Buid%5D" title="PubMed Link to resource 29775047">PubMed</a>.</span></li>
      <li><span id="cit124">Y.-J. Lin, P.-C. Chen, Z. Yuan, J.-Y. Ma and H.-T. Chang, <span class="italic">Chem. Commun.</span>, 2015, <span class="bold">51</span>, 11983–11986 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5CC02342D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5CC02342D">RSC</a>.</span></li>
      <li><span id="cit125">X. Gao, C. Du, C. Zhang and W. Chen, <span class="italic">ChemElectroChem</span>, 2016, <span class="bold">3</span>, 1266–1272 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/celc.201600036" title="DOI Link to resource 10.1002/celc.201600036">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XktVajtrc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit126">B. Yin, R. R. Liu, H. L. Zhai, S. H. Lu and J. Y. Mi, <span class="italic">Talanta</span>, 2020, <span class="bold">213</span>, 120838 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2020.120838" title="DOI Link to resource 10.1016/j.talanta.2020.120838">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXjs1KqtL0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32200926%5Buid%5D" title="PubMed Link to resource 32200926">PubMed</a>.</span></li>
      <li><span id="cit127">F. Qu, Q. Yang, B. Wang and J. You, <span class="italic">Talanta</span>, 2020, <span class="bold">207</span>, 120289 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2019.120289" title="DOI Link to resource 10.1016/j.talanta.2019.120289">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhs12rtrzK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31594584%5Buid%5D" title="PubMed Link to resource 31594584">PubMed</a>.</span></li>
      <li><span id="cit128">T. Luo, S. Zhang, Y. Wang, M. Wang, M. Liao and X. Kou, <span class="italic">Luminescence</span>, 2017, <span class="bold">32</span>, 1092–1099 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/bio.3296" title="DOI Link to resource 10.1002/bio.3296">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu7zK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28417589%5Buid%5D" title="PubMed Link to resource 28417589">PubMed</a>.</span></li>
      <li><span id="cit129">B. Han, X. Hu, M. Yu, T. Peng, Y. Li and G. He, <span class="italic">RSC Adv.</span>, 2018, <span class="bold">8</span>, 22748–22754 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8RA03632B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8RA03632B">RSC</a>.</span></li>
      <li><span id="cit130">C. Wang, L. Ling, Y. Yao and Q. Song, <span class="italic">Nano Res.</span>, 2015, <span class="bold">8</span>, 1975–1986 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s12274-015-0707-0" title="DOI Link to resource 10.1007/s12274-015-0707-0">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXmt1Oiu70%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit131">C. Wang and Y. Huang, <span class="italic">Nano</span>, 2013, <span class="bold">08</span>, 1350054 <a target="_blank" class="DOILink" href="https://doi.org/10.1142/S1793292013500549" title="DOI Link to resource 10.1142/S1793292013500549">CrossRef</a>.</span></li>
      <li><span id="cit132">C. Wang, H. Cheng, Y. Sun, Q. Lin and C. Zhang, <span class="italic">ChemNanoMat</span>, 2015, <span class="bold">1</span>, 27–31 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cnma.201500004" title="DOI Link to resource 10.1002/cnma.201500004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyhtr7O" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit133">T. Zhou, W. Xu, Q. Yao, T. Zhao and X. Chen, <span class="italic">Methods Appl. Fluoresc.</span>, 2015, <span class="bold">3</span>, 044002 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/2050-6120/3/4/044002" title="DOI Link to resource 10.1088/2050-6120/3/4/044002">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29148504%5Buid%5D" title="PubMed Link to resource 29148504">PubMed</a>.</span></li>
      <li><span id="cit134">H. Huang, H. Li, A.-J. Wang, S.-X. Zhong, K.-M. Fang and J.-J. Feng, <span class="italic">Analyst</span>, 2014, <span class="bold">139</span>, 6536–6541 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4AN01757A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4AN01757A">RSC</a>.</span></li>
      <li><span id="cit135">N. K. Das, S. Ghosh, A. Priya, S. Datta and S. Mukherjee, <span class="italic">J. Phys. Chem. C</span>, 2015, <span class="bold">119</span>, 24657–24664 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.5b08123" title="DOI Link to resource 10.1021/acs.jpcc.5b08123">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhs1SisLjF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit136">A. Baghdasaryan, R. Grillo, S. Roy Bhattacharya, M. Sharma, E. Reginato, H. Theraulaz, I. Dolamic, M. Dadras, S. Rudaz, E. Varesio and T. Burgi, <span class="italic">ACS Appl. Nano Mater.</span>, 2018, <span class="bold">1</span>, 4258–4267 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsanm.8b01049" title="DOI Link to resource 10.1021/acsanm.8b01049">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1KgurbJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit137">C. Wang, H. Cheng, Y. Huang, Z. Xu, H. Lin and C. Zhang, <span class="italic">Analyst</span>, 2015, <span class="bold">140</span>, 5634–5639 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5AN00741K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5AN00741K">RSC</a>.</span></li>
      <li><span id="cit138">Y. T. Tai, T. Simon, Y. Y. Chu and F. H. Ko, <span class="italic">Sens. Bio-Sens. Res.</span>, 2020, <span class="bold">27</span>, 100319 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbsr.2019.100319" title="DOI Link to resource 10.1016/j.sbsr.2019.100319">CrossRef</a>.</span></li>
      <li><span id="cit139">Z. Wu, J. Liu, Y. Gao, H. Liu, T. Li, H. Zou, Z. Wang, K. Zhang, Y. Wang, H. Zhang and B. Yang, <span class="italic">J. Am. Chem. Soc.</span>, 2015, <span class="bold">137</span>, 12906–12913 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.5b06550" title="DOI Link to resource 10.1021/jacs.5b06550">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersr3L" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26397821%5Buid%5D" title="PubMed Link to resource 26397821">PubMed</a>.</span></li>
      <li><span id="cit140">L. Ai, W. Jiang, Z. Liu, J. Liu, Y. Gao, H. Zou, Z. Wu, Z. Wang, Y. Liu, H. Zhang and B. Yang, <span class="italic">Nanoscale</span>, 2017, <span class="bold">9</span>, 12618–12627 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7NR03985A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7NR03985A">RSC</a>.</span></li>
      <li><span id="cit141">M. Zhao, Z. Qian, M. Zhong, Z. Chen, H. Ao and H. Feng, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2017, <span class="bold">9</span>, 32887–32895 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.7b09659" title="DOI Link to resource 10.1021/acsami.7b09659">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWnu7nL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28861993%5Buid%5D" title="PubMed Link to resource 28861993">PubMed</a>.</span></li>
      <li><span id="cit142">Y. Du, J. Fang, H. Wang and Y. Yang, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2017, <span class="bold">9</span>, 11035–11044 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.7b01228" title="DOI Link to resource 10.1021/acsami.7b01228">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXktVSiurc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28276247%5Buid%5D" title="PubMed Link to resource 28276247">PubMed</a>.</span></li>
      <li><span id="cit143">H. Rao, H. Ge, Z. Lu, W. Liu, Z. Chen, Z. Zhang, X. Wang, P. Zou, Y. Wang, H. He and X. Zeng, <span class="italic">Microchim. Acta</span>, 2016, <span class="bold">183</span>, 1651–1657 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-016-1794-7" title="DOI Link to resource 10.1007/s00604-016-1794-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xjt1agtbc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit144">H. Cao, Z. Chen and Y. Huang, <span class="italic">Talanta</span>, 2015, <span class="bold">143</span>, 450–456 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2015.05.024" title="DOI Link to resource 10.1016/j.talanta.2015.05.024">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXptVKnurk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26078183%5Buid%5D" title="PubMed Link to resource 26078183">PubMed</a>.</span></li>
      <li><span id="cit145">H. Cao, Z. Chen, H. Zheng and Y. Huang, <span class="italic">Biosens. Bioelectron.</span>, 2014, <span class="bold">62</span>, 189–195 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2014.06.049" title="DOI Link to resource 10.1016/j.bios.2014.06.049">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhtF2ktLbE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24999996%5Buid%5D" title="PubMed Link to resource 24999996">PubMed</a>.</span></li>
      <li><span id="cit146">J.-S. Shen, Y.-L. Chen, Q.-P. Wang, T. Yu, X.-Y. Huang, Y. Yang and H.-W. Zhang, <span class="italic">J. Mater. Chem. C</span>, 2013, <span class="bold">1</span>, 2092 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3TC00941F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3TC00941F">RSC</a>.</span></li>
      <li><span id="cit147">X. J. Zhao and C. Z. Huang, <span class="italic">New J. Chem.</span>, 2014, <span class="bold">38</span>, 3673 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4NJ00731J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4NJ00731J">RSC</a>.</span></li>
      <li><span id="cit148">H. Y. Zou, J. Lan and C. Z. Huang, <span class="italic">RSC Adv.</span>, 2015, <span class="bold">5</span>, 55832–55838 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5RA06240C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5RA06240C">RSC</a>.</span></li>
      <li><span id="cit149">B. Han, T. Peng, Y. Li, M. Yu, X. Hu and G. He, <span class="italic">Methods Appl. Fluoresc.</span>, 2018, <span class="bold">6</span>, 035015 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/2050-6120/aac8f5" title="DOI Link to resource 10.1088/2050-6120/aac8f5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsVyrsbbF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29848802%5Buid%5D" title="PubMed Link to resource 29848802">PubMed</a>.</span></li>
      <li><span id="cit150">X. Liu and D. Astruc, <span class="italic">Coord. Chem. Rev.</span>, 2018, <span class="bold">359</span>, 112–126 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ccr.2018.01.001" title="DOI Link to resource 10.1016/j.ccr.2018.01.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1Sjtrk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit151">X. H. Gao, S. J. He, C. M. Zhang, C. Du, X. Chen, W. Xing, S. L. Chen, A. Clayborne and W. Chen, <span class="italic">Adv. Sci.</span>, 2016, <span class="bold">3</span>, 1–7 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Sci.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit152">A. Eichhöfer, M. Kühn, S. Lebedkin, M. Kehry, M. M. Kappes and F. Weigend, <span class="italic">Inorg. Chem.</span>, 2017, <span class="bold">56</span>, 9330–9336 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.7b01495" title="DOI Link to resource 10.1021/acs.inorgchem.7b01495">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28731683%5Buid%5D" title="PubMed Link to resource 28731683">PubMed</a>.</span></li>
      <li><span id="cit153">R. Langer, M. Yadav, B. Weinert, D. Fenske and O. Fuhr, <span class="italic">Eur. J. Inorg. Chem.</span>, 2013, 3623–3631 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ejic.201300155" title="DOI Link to resource 10.1002/ejic.201300155">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXovFersb4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit154">C. Latouche, S. Kahlal, Y. R. Lin, J. H. Liao, E. Furet, C. W. Liu and J. Y. Saillard, <span class="italic">Inorg. Chem.</span>, 2013, <span class="bold">52</span>, 13253–13262 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ic402207u" title="DOI Link to resource 10.1021/ic402207u">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhslSgt73I" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24171461%5Buid%5D" title="PubMed Link to resource 24171461">PubMed</a>.</span></li>
      <li><span id="cit155">P. V. V. N. Kishore, D. R. Shi, J. H. Liao, A. K. Gupta and C. W. Liu, <span class="italic">Inorg. Chim. Acta</span>, 2019, <span class="bold">496</span>, 119068 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ica.2019.119068" title="DOI Link to resource 10.1016/j.ica.2019.119068">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsFyqt7rI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit156">P. K. Liao, C. S. Fang, A. J. Edwards, S. Kahlal, J. Y. Saillard and C. W. Liu, <span class="italic">Inorg. Chem.</span>, 2012, <span class="bold">51</span>, 6577–6591 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ic300135w" title="DOI Link to resource 10.1021/ic300135w">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XnvF2mt7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22663192%5Buid%5D" title="PubMed Link to resource 22663192">PubMed</a>.</span></li>
      <li><span id="cit157">K. Nakamae, M. Tanaka, B. Kure, T. Nakajima, Y. Ura and T. Tanase, <span class="italic">Chem. – Eur. J.</span>, 2017, <span class="bold">23</span>, 9457–9461 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201702071" title="DOI Link to resource 10.1002/chem.201702071">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXos1eiurk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28488381%5Buid%5D" title="PubMed Link to resource 28488381">PubMed</a>.</span></li>
      <li><span id="cit158">A. M. Polgar, A. Zhang, F. MacK, F. Weigend, S. Lebedkin, M. J. Stillman and J. F. Corrigan, <span class="italic">Inorg. Chem.</span>, 2019, <span class="bold">58</span>, 3338–3348 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.8b03399" title="DOI Link to resource 10.1021/acs.inorgchem.8b03399">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXjt1aisLc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30785278%5Buid%5D" title="PubMed Link to resource 30785278">PubMed</a>.</span></li>
      <li><span id="cit159">F. Ke, Y. Song, H. Li, C. Zhou, Y. Du and M. Zhu, <span class="italic">Dalton Trans.</span>, 2019, <span class="bold">48</span>, 13921–13924 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9DT02908G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9DT02908G">RSC</a>.</span></li>
      <li><span id="cit160">E. Dornsiepen, F. Weigend and S. Dehnen, <span class="italic">Chem. – Eur. J.</span>, 2019, <span class="bold">25</span>, 2486–2490 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201806407" title="DOI Link to resource 10.1002/chem.201806407">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFSgsL0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit161">H. Li, H. Zhai, C. Zhou, Y. Song, F. Ke, W. W. Xu and M. Zhu, <span class="italic">J. Phys. Chem. Lett.</span>, 2020, <span class="bold">11</span>, 4891–4896 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.0c01358" title="DOI Link to resource 10.1021/acs.jpclett.0c01358">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtVKqur7J" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit162">C. Latouche, J. H. Liao, Y. J. Li, R. Y. Shiu, V. Barone, S. Kahlal, C. W. Liu and J. Y. Saillard, <span class="italic">Inorg. Chem.</span>, 2017, <span class="bold">56</span>, 14135–14146 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.7b02269" title="DOI Link to resource 10.1021/acs.inorgchem.7b02269">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsleisLfO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29087198%5Buid%5D" title="PubMed Link to resource 29087198">PubMed</a>.</span></li>
      <li><span id="cit163">R. P. Brocha Silalahi, G. R. Huang, J. H. Liao, T. H. Chiu, K. K. Chakrahari, X. Wang, J. Cartron, S. Kahlal, J. Y. Saillard and C. W. Liu, <span class="italic">Inorg. Chem.</span>, 2020, <span class="bold">59</span>, 2536–2547 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.9b03501" title="DOI Link to resource 10.1021/acs.inorgchem.9b03501">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsFCjsrY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31977197%5Buid%5D" title="PubMed Link to resource 31977197">PubMed</a>.</span></li>
      <li><span id="cit164">X. X. Yang, I. Issac, S. Lebedkin, M. Kühn, F. Weigend, D. Fenske, O. Fuhr and A. Eichhöfer, <span class="italic">Chem. Commun.</span>, 2014, <span class="bold">50</span>, 11043–11045 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4CC04702H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4CC04702H">RSC</a>.</span></li>
      <li><span id="cit165">A. Eichhöfer, G. Buth, S. Lebedkin, M. Kühn and F. Weigend, <span class="italic">Inorg. Chem.</span>, 2015, <span class="bold">54</span>, 9413–9422 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.5b01146" title="DOI Link to resource 10.1021/acs.inorgchem.5b01146">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26378617%5Buid%5D" title="PubMed Link to resource 26378617">PubMed</a>.</span></li>
      <li><span id="cit166">C. B. Khadka, B. K. Najafabadi, M. Hesari, M. S. Workentin and J. F. Corrigan, <span class="italic">Inorg. Chem.</span>, 2013, <span class="bold">52</span>, 6798–6805 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ic3021854" title="DOI Link to resource 10.1021/ic3021854">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXotFyisbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23705664%5Buid%5D" title="PubMed Link to resource 23705664">PubMed</a>.</span></li>
      <li><span id="cit167">S. Dehnen, A. Schäfer, D. Fenske and R. Ahlrichs, <span class="italic">Angew. Chem., Int. Ed. Engl.</span>, 1994, <span class="bold">33</span>, 746–749 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.199407461" title="DOI Link to resource 10.1002/anie.199407461">CrossRef</a>.</span></li>
      <li><span id="cit168">K. K. Chakrahari, J. H. Liao, S. Kahlal, Y. C. Liu, M. H. Chiang, J. Y. Saillard and C. W. Liu, <span class="italic">Angew. Chem., Int. Ed.</span>, 2016, <span class="bold">55</span>, 14704–14708 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201608609" title="DOI Link to resource 10.1002/anie.201608609">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhslGitLfO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27781357%5Buid%5D" title="PubMed Link to resource 27781357">PubMed</a>.</span></li>
      <li><span id="cit169">K. K. Chakrahari, R. P. B. Silalahi, J. H. Liao, S. Kahlal, Y. C. Liu, J. F. Lee, M. H. Chiang, J. Y. Saillard and C. W. Liu, <span class="italic">Chem. Sci.</span>, 2018, <span class="bold">9</span>, 6785–6795 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8SC01508B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8SC01508B">RSC</a>.</span></li>
      <li><span id="cit170">A. W. Cook, Z. R. Jones, G. Wu, S. J. Teat, S. L. Scott and T. W. Hayton, <span class="italic">Inorg. Chem.</span>, 2019, <span class="bold">58</span>, 8739–8749 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.9b01140" title="DOI Link to resource 10.1021/acs.inorgchem.9b01140">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtFKju77O" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31198031%5Buid%5D" title="PubMed Link to resource 31198031">PubMed</a>.</span></li>
      <li><span id="cit171">T. A. D. Nguyen, B. R. Goldsmith, H. T. Zaman, G. Wu, B. Peters and T. W. Hayton, <span class="italic">Chem. – Eur. J.</span>, 2015, <span class="bold">21</span>, 5341–5344 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201500422" title="DOI Link to resource 10.1002/chem.201500422">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXkvVektLg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25702682%5Buid%5D" title="PubMed Link to resource 25702682">PubMed</a>.</span></li>
      <li><span id="cit172">Y. Li, J. Wang, P. Luo, X. Ma, X. Dong, Z. Wang, C. Du, S. Zang and T. C. W. Mak, <span class="italic">Adv. Sci.</span>, 2019, <span class="bold">1900833</span>, 1900833 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/advs.201900833" title="DOI Link to resource 10.1002/advs.201900833">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31559130%5Buid%5D" title="PubMed Link to resource 31559130">PubMed</a>.</span></li>
      <li><span id="cit173">M. Zhang, X. Dong, Z. Wang, H. Li, S. Li, X. Zhao and S. Zang, <span class="italic">Angew. Chem.</span>, 2020, <span class="bold">132</span>, 10138–10144 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ange.201908909" title="DOI Link to resource 10.1002/ange.201908909">CrossRef</a>.</span></li>
      <li><span id="cit174">A. K. Gupta, P. V. V. N. Kishore, J. H. Liao, Y. J. Chen and C. W. Liu, <span class="italic">Eur. J. Inorg. Chem.</span>, 2017, <span class="bold">2017</span>, 1989–1993 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ejic.201700070" title="DOI Link to resource 10.1002/ejic.201700070">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXlslOlu7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit175">M. A. Huertos, I. Cano, N. A. G. Bandeira, J. Benet-Buchholz, C. Bo and P. W. N. M. van Leeuwen, <span class="italic">Chem. – Eur. J.</span>, 2014, <span class="bold">20</span>, 16121–16127 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201404763" title="DOI Link to resource 10.1002/chem.201404763">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitVCiu7%252FM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25284300%5Buid%5D" title="PubMed Link to resource 25284300">PubMed</a>.</span></li>
      <li><span id="cit176">A. Eichhöfer, J. Jiang, S. Lebedkin, D. Fenske, D. G. McDonald, J. F. Corrigan and C. Y. Su, <span class="italic">Dalton Trans.</span>, 2012, <span class="bold">41</span>, 3321–3327 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2DT11840H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2DT11840H">RSC</a>.</span></li>
      <li><span id="cit177">A. W. Cook, Z. R. Jones, G. Wu, S. L. Scott and T. W. Hayton, <span class="italic">J. Am. Chem. Soc.</span>, 2018, <span class="bold">140</span>, 394–400 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.7b10960" title="DOI Link to resource 10.1021/jacs.7b10960">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvFaqsL3E" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit178">R. S. Dhayal, J. H. Liao, Y. R. Lin, P. K. Liao, S. Kahlal, J. Y. Saillard and C. W. Liu, <span class="italic">J. Am. Chem. Soc.</span>, 2013, <span class="bold">135</span>, 4704–4707 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja401576s" title="DOI Link to resource 10.1021/ja401576s">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXjslGgs74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit179">R. S. Dhayal, J.-H. Liao, X. Wang, Y.-C. Liu, M.-H. Chiang, S. Kahlal, J.-Y. Saillard and C. W. Liu, <span class="italic">Angew. Chem., Int. Ed.</span>, 2015, <span class="bold">54</span>, 13604–13608 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201506736" title="DOI Link to resource 10.1002/anie.201506736">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFaqs73I" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26387572%5Buid%5D" title="PubMed Link to resource 26387572">PubMed</a>.</span></li>
      <li><span id="cit180">P. Y. Lin, D. Y. Li, F. H. Ho, J. H. Liao, S. K. Barik and C. W. Liu, <span class="italic">J. Chin. Chem. Soc.</span>, 2019, <span class="bold">66</span>, 988–995 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jccs.201900086" title="DOI Link to resource 10.1002/jccs.201900086">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtVOjsbfP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit181">R. S. Dhayal, H. P. Chen, J. H. Liao, W. E. Van Zyl and C. W. Liu, <span class="italic">ChemistrySelect</span>, 2018, <span class="bold">3</span>, 3603–3610 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/slct.201800765" title="DOI Link to resource 10.1002/slct.201800765">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ektL0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit182">C. Sun, N. Mammen, S. Kaappa, P. Yuan, G. Deng, C. Zhao, J. Yan, S. Malola, K. Honkala, H. Häkkinen, B. K. Teo and N. Zheng, <span class="italic">ACS Nano</span>, 2019, <span class="bold">13</span>, 5975–5986 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.9b02052" title="DOI Link to resource 10.1021/acsnano.9b02052">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXptFyntLY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit183">T.-A. D. Nguyen, Z. R. Jones, B. R. Goldsmith, W. R. Buratto, G. Wu, S. L. Scott and T. W. Hayton, <span class="italic">J. Am. Chem. Soc.</span>, 2015, <span class="bold">137</span>, 13319–13324 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.5b07574" title="DOI Link to resource 10.1021/jacs.5b07574">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKqs73L" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26422670%5Buid%5D" title="PubMed Link to resource 26422670">PubMed</a>.</span></li>
      <li><span id="cit184">A. J. Edwards, R. S. Dhayal, P. K. Liao, J. H. Liao, M. H. Chiang, R. O. Piltz, S. Kahlal, J. Y. Saillard and C. W. Liu, <span class="italic">Angew. Chem., Int. Ed.</span>, 2014, <span class="bold">53</span>, 7214–7218 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201403324" title="DOI Link to resource 10.1002/anie.201403324">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXnslWhsb0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24803070%5Buid%5D" title="PubMed Link to resource 24803070">PubMed</a>.</span></li>
      <li><span id="cit185">T. A. D. Nguyen, Z. R. Jones, D. F. Leto, G. Wu, S. L. Scott and T. W. Hayton, <span class="italic">Chem. Mater.</span>, 2016, <span class="bold">28</span>, 8385–8390 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.6b03879" title="DOI Link to resource 10.1021/acs.chemmater.6b03879">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhslCgsrbJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit186">S. K. Barik, S. Huo, C. Wu, T.-H. Chiu, J. Liao, X. Wang, S. Kahlal, J.-Y. Saillard and C. W. Liu, <span class="italic">Chem. – Eur. J.</span>, 2020, <span class="bold">26</span>, 10471–10479 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.202001449" title="DOI Link to resource 10.1002/chem.202001449">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrurnP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32378767%5Buid%5D" title="PubMed Link to resource 32378767">PubMed</a>.</span></li>
      <li><span id="cit187">R. S. Dhayal, J. H. Liao, S. Kahlal, X. Wang, Y. C. Liu, M. H. Chiang, W. E. Van Zyl, J. Y. Saillard and C. W. Liu, <span class="italic">Chem. – Eur. J.</span>, 2015, <span class="bold">21</span>, 8369–8374 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201501122" title="DOI Link to resource 10.1002/chem.201501122">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXmsleqs7Y%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25899822%5Buid%5D" title="PubMed Link to resource 25899822">PubMed</a>.</span></li>
      <li><span id="cit188">Q. Tang, Y. Lee, D. Y. Li, W. Choi, C. W. Liu, D. Lee and D. E. Jiang, <span class="italic">J. Am. Chem. Soc.</span>, 2017, <span class="bold">139</span>, 9728–9736 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.7b05591" title="DOI Link to resource 10.1021/jacs.7b05591">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtVektb7M" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28640611%5Buid%5D" title="PubMed Link to resource 28640611">PubMed</a>.</span></li>
      <li><span id="cit189">S. Lee, M. S. Bootharaju, G. Deng, S. Malola, W. Baek, H. Häkkinen, N. Zheng and T. Hyeon, <span class="italic">J. Am. Chem. Soc.</span>, 2020, <span class="bold">142</span>, 13974–13981 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.0c06577" title="DOI Link to resource 10.1021/jacs.0c06577">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtl2rs7rF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32672452%5Buid%5D" title="PubMed Link to resource 32672452">PubMed</a>.</span></li>
      <li><span id="cit190">L. M. Zhang and T. C. W. Mak, <span class="italic">J. Am. Chem. Soc.</span>, 2016, <span class="bold">138</span>, 2909–2912 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.5b12103" title="DOI Link to resource 10.1021/jacs.5b12103">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XivVert7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26899875%5Buid%5D" title="PubMed Link to resource 26899875">PubMed</a>.</span></li>
      <li><span id="cit191">R. Langer, D. Fenske and O. Fuhr, <span class="italic">Z. Naturforsch., B: J. Chem. Sci.</span>, 2013, <span class="bold">68</span>, 575–580 <a target="_blank" class="DOILink" href="https://doi.org/10.5560/znb.2013-3078" title="DOI Link to resource 10.5560/znb.2013-3078">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOqsLnO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit192">J. Weßing, C. Ganesamoorthy, S. Kahlal, R. Marchal, C. Gemel, O. Cador, A. C. H. Da Silva, J. L. F. Da Silva, J. Y. Saillard and R. A. Fischer, <span class="italic">Angew. Chem., Int. Ed.</span>, 2018, <span class="bold">57</span>, 14630–14634 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201806039" title="DOI Link to resource 10.1002/anie.201806039">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29981271%5Buid%5D" title="PubMed Link to resource 29981271">PubMed</a>.</span></li>
      <li><span id="cit193">S. Bestgen, O. Fuhr, P. W. Roesky and D. Fenske, <span class="italic">Dalton Trans.</span>, 2016, <span class="bold">45</span>, 14907–14910 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6DT03311C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6DT03311C">RSC</a>.</span></li>
      <li><span id="cit194">P. Yuan, R. Chen, X. Zhang, F. Chen, J. Yan, C. Sun, D. Ou, J. Peng, S. Lin, Z. Tang, B. K. Teo, L. S. Zheng and N. Zheng, <span class="italic">Angew. Chem., Int. Ed.</span>, 2019, <span class="bold">58</span>, 835–839 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201812236" title="DOI Link to resource 10.1002/anie.201812236">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisVKqsLnP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30406951%5Buid%5D" title="PubMed Link to resource 30406951">PubMed</a>.</span></li>
      <li><span id="cit195">A. Baghdasaryan, C. Besnard, L. M. Lawson Daku, T. Delgado and T. Bürgi, <span class="italic">Inorg. Chem.</span>, 2020, <span class="bold">59</span>, 2200–2208 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.inorgchem.9b02828" title="DOI Link to resource 10.1021/acs.inorgchem.9b02828">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitVylt74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit196">R.-W. Huang, J. Yin, C. Dong, A. Ghosh, M. J. Alhilaly, X. Dong, M. N. Hedhili, E. Abou-Hamad, B. Alamer, S. Nematulloev, Y. Han, O. F. Mohammed and O. M. Bakr, <span class="italic">J. Am. Chem. Soc.</span>, 2020, <span class="bold">142</span>, 8696–8705 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.0c00541" title="DOI Link to resource 10.1021/jacs.0c00541">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32315164%5Buid%5D" title="PubMed Link to resource 32315164">PubMed</a>.</span></li>
      <li><span id="cit197">M.-L. Fu, I. Issac, D. Fenske and O. Fuhr, <span class="italic">Angew. Chem., Int. Ed.</span>, 2010, <span class="bold">49</span>, 6899–6903 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201001301" title="DOI Link to resource 10.1002/anie.201001301">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhtFGnsbbJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20680953%5Buid%5D" title="PubMed Link to resource 20680953">PubMed</a>.</span></li>
      <li><span id="cit198">K. YAMAMOTO and S. KASHIDA, <span class="italic">Solid State Ionics</span>, 1991, <span class="bold">48</span>, 241–248 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0167-2738(91)90038-D" title="DOI Link to resource 10.1016/0167-2738(91)90038-D">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK38XkvFCruw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit199">S. Chitsaz, D. Fenske and O. Fuhr, <span class="italic">Angew. Chem., Int. Ed.</span>, 2006, <span class="bold">45</span>, 8055–8059 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.200602709" title="DOI Link to resource 10.1002/anie.200602709">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XhtleksrfO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17086557%5Buid%5D" title="PubMed Link to resource 17086557">PubMed</a>.</span></li>
      <li><span id="cit200">A. H. Pakiari and Z. Jamshidi, <span class="italic">J. Phys. Chem. A</span>, 2010, <span class="bold">114</span>, 9212–9221 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp100423b" title="DOI Link to resource 10.1021/jp100423b">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXpvVGhu7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20687518%5Buid%5D" title="PubMed Link to resource 20687518">PubMed</a>.</span></li>
      <li><span id="cit201">A. Ganguly, I. Chakraborty, T. Udayabhaskararao and T. Pradeep, <span class="italic">J. Nanopart. Res.</span>, 2013, <span class="bold">15</span>, 1522 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11051-013-1522-8" title="DOI Link to resource 10.1007/s11051-013-1522-8">CrossRef</a>.</span></li>
      <li><span id="cit202">P. Ngene, A. Longo, L. Mooij, W. Bras and B. Dam, <span class="italic">Nat. Commun.</span>, 2017, <span class="bold">8</span>, 1846 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-017-02043-9" title="DOI Link to resource 10.1038/s41467-017-02043-9">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29184061%5Buid%5D" title="PubMed Link to resource 29184061">PubMed</a>.</span></li>
      <li><span id="cit203">Q. Lai, Y. Sun, T. Wang, P. Modi, C. Cazorla, U. B. Demirci, J. R. Ares Fernandez, F. Leardini and K. F. Aguey-Zinsou, <span class="italic">Adv. Sustainable Syst.</span>, 2019, <span class="bold">3</span>, 1–64 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Sustainable%20Syst.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit204">R. S. Dhayal, W. E. Van Zyl and C. W. Liu, <span class="italic">Acc. Chem. Res.</span>, 2016, <span class="bold">49</span>, 86–95 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.accounts.5b00375" title="DOI Link to resource 10.1021/acs.accounts.5b00375">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XlsVOgug%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit205">R. S. Dhayal, W. E. Van Zyl and C. W. Liu, <span class="italic">Dalton Trans.</span>, 2019, <span class="bold">48</span>, 3531–3538 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8DT04639E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8DT04639E">RSC</a>.</span></li>
      <li><span id="cit206">A. J. Jordan, G. Lalic and J. P. Sadighi, <span class="italic">Chem. Rev.</span>, 2016, <span class="bold">116</span>, 8318–8372 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.6b00366" title="DOI Link to resource 10.1021/acs.chemrev.6b00366">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1ams77N" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27454444%5Buid%5D" title="PubMed Link to resource 27454444">PubMed</a>.</span></li>
      <li><span id="cit207">M. W. Heaven, A. Dass, P. S. White, K. M. Holt and R. W. Murray, <span class="italic">J. Am. Chem. Soc.</span>, 2008, <span class="bold">130</span>, 3754–3755 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja800561b" title="DOI Link to resource 10.1021/ja800561b">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXivVSktLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18321116%5Buid%5D" title="PubMed Link to resource 18321116">PubMed</a>.</span></li>
      <li><span id="cit208">R. Jin, <span class="italic">Nanoscale</span>, 2015, <span class="bold">7</span>, 1549–1565 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4NR05794E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4NR05794E">RSC</a>.</span></li>
      <li><span id="cit209">V. W.-W. Yam, C.-H. Lam, W. K.-M. Fung and K.-K. Cheung, <span class="italic">Inorg. Chem.</span>, 2001, <span class="bold">40</span>, 3435–3442 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ic0012322" title="DOI Link to resource 10.1021/ic0012322">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXktFeku74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit210">W.-P. Leung, Y.-C. Chan and T. C. W. Mak, <span class="italic">Dalton Trans.</span>, 2014, <span class="bold">43</span>, 63–66 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3DT52833B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3DT52833B">RSC</a>.</span></li>
      <li><span id="cit211">P. C. Ford and A. Vogler, <span class="italic">Acc. Chem. Res.</span>, 1993, <span class="bold">26</span>, 220–226 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ar00028a013" title="DOI Link to resource 10.1021/ar00028a013">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3sXhsl2kt7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit212">Y. Guo, F. Cao, X. Lei, L. Mang, S. Cheng and J. Song, <span class="italic">Nanoscale</span>, 2016, <span class="bold">8</span>, 4852–4863 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6NR00145A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6NR00145A">RSC</a>.</span></li>
      <li><span id="cit213">Y.-S. Borghei, M. Hosseini, M. Khoobi and M. R. Ganjali, <span class="italic">J. Fluoresc.</span>, 2017, <span class="bold">27</span>, 529–536 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10895-016-1980-3" title="DOI Link to resource 10.1007/s10895-016-1980-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhvVyntLnP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27858297%5Buid%5D" title="PubMed Link to resource 27858297">PubMed</a>.</span></li>
      <li><span id="cit214">J. Yang, Z. Li and Q. Jia, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">297</span>, 126807 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2019.126807" title="DOI Link to resource 10.1016/j.snb.2019.126807">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsVWht7fK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit215">Y. Hong, J. W. Y. Lam and B. Z. Tang, <span class="italic">Chem. Soc. Rev.</span>, 2011, <span class="bold">40</span>, 5361 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1CS15113D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1CS15113D">RSC</a>.</span></li>
      <li><span id="cit216">N. Goswami, Q. Yao, Z. Luo, J. Li, T. Chen and J. Xie, <span class="italic">J. Phys. Chem. Lett.</span>, 2016, <span class="bold">7</span>, 962–975 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.5b02765" title="DOI Link to resource 10.1021/acs.jpclett.5b02765">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XjtFKhuro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26912457%5Buid%5D" title="PubMed Link to resource 26912457">PubMed</a>.</span></li>
      <li><span id="cit217">K. T. Prakash, N. Singh and V. Venkatesh, <span class="italic">Chem. Commun.</span>, 2019, <span class="bold">55</span>, 322–325 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8CC09109A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8CC09109A">RSC</a>.</span></li>
      <li><span id="cit218">T. Zhou, J. Zhu, L. Gong, L. Nong and J. Liu, <span class="italic">J. Am. Chem. Soc.</span>, 2019, <span class="bold">141</span>, 2852–2856 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.8b12026" title="DOI Link to resource 10.1021/jacs.8b12026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisVSqtbk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30717593%5Buid%5D" title="PubMed Link to resource 30717593">PubMed</a>.</span></li>
      <li><span id="cit219">X. Su and J. Liu, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2017, <span class="bold">9</span>, 3902–3910 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.6b13914" title="DOI Link to resource 10.1021/acsami.6b13914">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXms1Omtw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28067503%5Buid%5D" title="PubMed Link to resource 28067503">PubMed</a>.</span></li>
      <li><span id="cit220">Y. Huang, H. Feng, W. Liu, Y. Zhou, C. Tang, H. Ao, M. Zhao, G. Chen, J. Chen and Z. Qian, <span class="italic">Anal. Chem.</span>, 2016, <span class="bold">88</span>, 11575–11583 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.6b02957" title="DOI Link to resource 10.1021/acs.analchem.6b02957">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhslGhur3O" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27796092%5Buid%5D" title="PubMed Link to resource 27796092">PubMed</a>.</span></li>
      <li><span id="cit221">J. Yuan, L. Wang, Y. Wang and J. Hao, <span class="italic">Chem. – Eur. J.</span>, 2020, <span class="bold">26</span>, 3545–3554 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201905094" title="DOI Link to resource 10.1002/chem.201905094">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvFSmsrk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31821645%5Buid%5D" title="PubMed Link to resource 31821645">PubMed</a>.</span></li>
      <li><span id="cit222">R. Jalili and A. Khataee, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 29 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-3111-0" title="DOI Link to resource 10.1007/s00604-018-3111-0">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30565190%5Buid%5D" title="PubMed Link to resource 30565190">PubMed</a>.</span></li>
      <li><span id="cit223">M. Ye, Y. Yu, B. Lin, Y. Cai, Y. Cao, M. Guo and D. Zhu, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">284</span>, 36–44 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2018.12.092" title="DOI Link to resource 10.1016/j.snb.2018.12.092">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWgtw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit224">F. Geng, C. Zou, J. Liu, Q. Zhang, X. Guo, Y. Fan, H. Yu, S. Yang, Z. Liu and L. Li, <span class="italic">Anal. Chim. Acta</span>, 2019, <span class="bold">1076</span>, 131–137 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2019.05.026" title="DOI Link to resource 10.1016/j.aca.2019.05.026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKqsbbP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit225">L. Lin, Y. Hu, L. Zhang, Y. Huang and S. Zhao, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">94</span>, 523–529 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2017.03.038" title="DOI Link to resource 10.1016/j.bios.2017.03.038">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXkvFait7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28343105%5Buid%5D" title="PubMed Link to resource 28343105">PubMed</a>.</span></li>
      <li><span id="cit226">D. Li, Y. Zhao, Z. Chen, X. Mei and X. Qiu, <span class="italic">Mater. Sci. Eng., C</span>, 2017, <span class="bold">78</span>, 653–657 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.msec.2017.04.147" title="DOI Link to resource 10.1016/j.msec.2017.04.147">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXntFWlt7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28576034%5Buid%5D" title="PubMed Link to resource 28576034">PubMed</a>.</span></li>
      <li><span id="cit227">Y. Huang, H. Feng, W. Liu, S. Zhang, C. Tang, J. Chen and Z. Qian, <span class="italic">J. Mater. Chem. B</span>, 2017, <span class="bold">5</span>, 5120–5127 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7TB00901A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7TB00901A">RSC</a>.</span></li>
      <li><span id="cit228">Y. Liu, D. Yao and H. Zhang, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2018, <span class="bold">10</span>, 12071–12080 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.7b13940" title="DOI Link to resource 10.1021/acsami.7b13940">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSrtb7M" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29144119%5Buid%5D" title="PubMed Link to resource 29144119">PubMed</a>.</span></li>
      <li><span id="cit229">A. Han, L. Xiong, S. Hao, Y. Yang, X. Li, G. Fang, J. Liu, Y. Pei and S. Wang, <span class="italic">Anal. Chem.</span>, 2018, <span class="bold">90</span>, 9060–9067 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.8b01384" title="DOI Link to resource 10.1021/acs.analchem.8b01384">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1KnsrbE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29973046%5Buid%5D" title="PubMed Link to resource 29973046">PubMed</a>.</span></li>
      <li><span id="cit230">Z. Wu, H. Liu, T. Li, J. Liu, J. Yin, O. F. Mohammed, O. M. Bakr, Y. Liu, B. Yang and H. Zhang, <span class="italic">J. Am. Chem. Soc.</span>, 2017, <span class="bold">139</span>, 4318–4321 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.7b00773" title="DOI Link to resource 10.1021/jacs.7b00773">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXksFamurY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28318238%5Buid%5D" title="PubMed Link to resource 28318238">PubMed</a>.</span></li>
      <li><span id="cit231">T. G. Schaaff and R. L. Whetten, <span class="italic">J. Phys. Chem. B</span>, 2000, <span class="bold">104</span>, 2630–2641 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp993691y" title="DOI Link to resource 10.1021/jp993691y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXht1Gqu7c%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit232">C. Noguez and I. L. Garzón, <span class="italic">Chem. Soc. Rev.</span>, 2009, <span class="bold">38</span>, 757 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B800404H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B800404H">RSC</a>.</span></li>
      <li><span id="cit233">S. Knoppe and T. Bürgi, <span class="italic">Acc. Chem. Res.</span>, 2014, <span class="bold">47</span>, 1318–1326 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ar400295d" title="DOI Link to resource 10.1021/ar400295d">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXjsVSitr0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24588279%5Buid%5D" title="PubMed Link to resource 24588279">PubMed</a>.</span></li>
      <li><span id="cit234">O. Lopez-Acevedo, H. Tsunoyama, T. Tsukuda, H. Häkkinen and C. M. Aikens, <span class="italic">J. Am. Chem. Soc.</span>, 2010, <span class="bold">132</span>, 8210–8218 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja102934q" title="DOI Link to resource 10.1021/ja102934q">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXmsVCqtLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20499877%5Buid%5D" title="PubMed Link to resource 20499877">PubMed</a>.</span></li>
      <li><span id="cit235">I. Dolamic, S. Knoppe, A. Dass and T. Bürgi, <span class="italic">Nat. Commun.</span>, 2012, <span class="bold">3</span>, 798 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/ncomms1802" title="DOI Link to resource 10.1038/ncomms1802">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22531183%5Buid%5D" title="PubMed Link to resource 22531183">PubMed</a>.</span></li>
      <li><span id="cit236">S. Knoppe, O. A. Wong, S. Malola, H. Häkkinen, T. Bürgi, T. Verbiest and C. J. Ackerson, <span class="italic">J. Am. Chem. Soc.</span>, 2014, <span class="bold">136</span>, 4129–4132 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja500809p" title="DOI Link to resource 10.1021/ja500809p">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXjsVehsLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24588769%5Buid%5D" title="PubMed Link to resource 24588769">PubMed</a>.</span></li>
      <li><span id="cit237">M.-R. Goldsmith, C. B. George, G. Zuber, R. Naaman, D. H. Waldeck, P. Wipf and D. N. Beratan, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2006, <span class="bold">8</span>, 63–67 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B511563A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B511563A">RSC</a>.</span></li>
      <li><span id="cit238">V. Humblot, S. Haq, C. Muryn, W. A. Hofer and R. Raval, <span class="italic">J. Am. Chem. Soc.</span>, 2002, <span class="bold">124</span>, 503–510 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja012021e" title="DOI Link to resource 10.1021/ja012021e">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXpt1Cnsb0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=11792223%5Buid%5D" title="PubMed Link to resource 11792223">PubMed</a>.</span></li>
      <li><span id="cit239">X. Hu, W. Wang and Y. Huang, <span class="italic">Talanta</span>, 2016, <span class="bold">154</span>, 409–415 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2016.03.095" title="DOI Link to resource 10.1016/j.talanta.2016.03.095">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XlslWmtL4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27154693%5Buid%5D" title="PubMed Link to resource 27154693">PubMed</a>.</span></li>
      <li><span id="cit240">J. Li, W. Fu, J. Bao, Z. Wang and Z. Dai, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2018, <span class="bold">10</span>, 6965–6971 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.7b19055" title="DOI Link to resource 10.1021/acsami.7b19055">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ylsro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29363949%5Buid%5D" title="PubMed Link to resource 29363949">PubMed</a>.</span></li>
      <li><span id="cit241">J. R. Bhamore, B. Deshmukh, V. Haran, S. Jha, R. K. Singhal, N. Lenka, S. K. Kailasa and Z. V. P. Murthy, <span class="italic">New J. Chem.</span>, 2018, <span class="bold">42</span>, 1510–1520 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7NJ04031H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7NJ04031H">RSC</a>.</span></li>
      <li><span id="cit242">R. Liu, L. Zuo, X. Huang, S. Liu, G. Yang, S. Li and C. Lv, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 250 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3360-6" title="DOI Link to resource 10.1007/s00604-019-3360-6">CrossRef</a>.</span></li>
      <li><span id="cit243">H. Zhang, Y. Guan, X. Li, L. Lian, X. Wang, W. Gao, B. Zhu, X. Liu and D. Lou, <span class="italic">Anal. Sci.</span>, 2018, <span class="bold">34</span>, 1155–1161 <a target="_blank" class="DOILink" href="https://doi.org/10.2116/analsci.18P124" title="DOI Link to resource 10.2116/analsci.18P124">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisVChu7fF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30305592%5Buid%5D" title="PubMed Link to resource 30305592">PubMed</a>.</span></li>
      <li><span id="cit244">X. Feng, J. Zhang, J. Wang, A. Han, G. Fang, J. Liu and S. Wang, <span class="italic">Anal. Methods</span>, 2020, <span class="bold">12</span>, 3130–3136 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0AY00657B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0AY00657B">RSC</a>.</span></li>
      <li><span id="cit245">H. Huang, H. Li, J.-J. Feng, H. Feng, A.-J. Wang and Z. Qian, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">241</span>, 292–297 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2016.10.086" title="DOI Link to resource 10.1016/j.snb.2016.10.086">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhslemtLzE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit246">T. Tang, J. Ouyang, L. Hu, L. Guo, M. Yang and X. Chen, <span class="italic">Microchim. Acta</span>, 2016, <span class="bold">183</span>, 2831–2836 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-016-1935-z" title="DOI Link to resource 10.1007/s00604-016-1935-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xhtlyktr7O" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit247">D. K. Sahu, D. Singha and K. Sahu, <span class="italic">Sens. Bio-Sens. Res.</span>, 2019, <span class="bold">22</span>, 100250 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbsr.2018.100250" title="DOI Link to resource 10.1016/j.sbsr.2018.100250">CrossRef</a>.</span></li>
      <li><span id="cit248">Y. Huang, H. Zhang, X. Xu, J. Zhou, F. Lu, Z. Zhang, Z. Hu and J. Luo, <span class="italic">Spectrochim. Acta, Part A</span>, 2018, <span class="bold">202</span>, 65–69 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.saa.2018.05.030" title="DOI Link to resource 10.1016/j.saa.2018.05.030">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXps1Ohs7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29777936%5Buid%5D" title="PubMed Link to resource 29777936">PubMed</a>.</span></li>
      <li><span id="cit249">S. M. Lin, S. Geng, N. Li, S. G. Liu, N. B. Li and H. Q. Luo, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">252</span>, 912–918 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.06.079" title="DOI Link to resource 10.1016/j.snb.2017.06.079">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChsLnK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit250">Z.-C. Liu, J.-W. Qi, C. Hu, L. Zhang, W. Song, R.-P. Liang and J.-D. Qiu, <span class="italic">Anal. Chim. Acta</span>, 2015, <span class="bold">895</span>, 95–103 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2015.09.002" title="DOI Link to resource 10.1016/j.aca.2015.09.002">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2ntrbK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26454464%5Buid%5D" title="PubMed Link to resource 26454464">PubMed</a>.</span></li>
      <li><span id="cit251">Z. Qing, L. Zhu, S. Yang, Z. Cao, X. He, K. Wang and R. Yang, <span class="italic">Biosens. Bioelectron.</span>, 2016, <span class="bold">78</span>, 471–476 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2015.11.057" title="DOI Link to resource 10.1016/j.bios.2015.11.057">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvFKktLzO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26657590%5Buid%5D" title="PubMed Link to resource 26657590">PubMed</a>.</span></li>
      <li><span id="cit252">Y. Wang, T. Chen, Z. Zhang and Y. Ni, <span class="italic">RSC Adv.</span>, 2018, <span class="bold">8</span>, 9057–9062 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7RA11383H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7RA11383H">RSC</a>.</span></li>
      <li><span id="cit253">Z. Shen, C. Zhang, X. Yu, J. Li, B. Liu and Z. Zhang, <span class="italic">Microchem. J.</span>, 2019, <span class="bold">145</span>, 517–522 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.microc.2018.11.026" title="DOI Link to resource 10.1016/j.microc.2018.11.026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXit1CgsLbK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit254">J. Yang, N. Song, X. Lv and Q. Jia, <span class="italic">Sens. Actuators, B</span>, 2018, <span class="bold">259</span>, 226–232 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.12.045" title="DOI Link to resource 10.1016/j.snb.2017.12.045">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2lsrjK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit255">L. J. Ou, X. Y. Li, H. W. Liu, L. J. Li and X. Chu, <span class="italic">Anal. Sci.</span>, 2014, <span class="bold">30</span>, 723–727 <a target="_blank" class="DOILink" href="https://doi.org/10.2116/analsci.30.723" title="DOI Link to resource 10.2116/analsci.30.723">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhtlWlsL7O" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit256">Z. Liu, X. Jing, S. Zhang and Y. Tian, <span class="italic">Anal. Chem.</span>, 2019, <span class="bold">91</span>, 2488–2497 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.8b05360" title="DOI Link to resource 10.1021/acs.analchem.8b05360">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXkslOitw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit257">B. Han, R. Xiang, X. Hou, M. Yu, T. Peng, Y. Li and G. He, <span class="italic">Anal. Methods</span>, 2017, <span class="bold">9</span>, 2590–2595 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7AY00625J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7AY00625J">RSC</a>.</span></li>
      <li><span id="cit258">R. S. Aparna, J. S. Anjali Devi, R. R. Anjana, J. Nebu and S. George, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">291</span>, 298–305 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2019.04.051" title="DOI Link to resource 10.1016/j.snb.2019.04.051">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXosVensLY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit259">X. Gao, X. Zhuang, C. Tian, H. Liu, W. F. Lai, Z. Wang, X. Yang, L. Chen and A. L. Rogach, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">307</span>, 127626 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2019.127626" title="DOI Link to resource 10.1016/j.snb.2019.127626">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXht12ntg%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit260">A. Nain, Y.-T. Tseng, Y.-S. Lin, S.-C. Wei, R. P. Mandal, B. Unnikrishnan, C.-C. Huang, F.-G. Tseng and H.-T. Chang, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">321</span>, 128539 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2020.128539" title="DOI Link to resource 10.1016/j.snb.2020.128539">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtlyit7rI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit261">A. Singh, S. Kaur, A. Kaur, T. Aree, N. Kaur, N. Singh and M. S. Bakshi, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2014, <span class="bold">2</span>, 982–990 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/sc400557w" title="DOI Link to resource 10.1021/sc400557w">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhs1GntLw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit262">C. Chen, F. Geng, Y. Wang, H. Yu, L. Li, S. Yang, J. Liu and W. Huang, <span class="italic">Talanta</span>, 2019, <span class="bold">205</span>, 120132 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2019.120132" title="DOI Link to resource 10.1016/j.talanta.2019.120132">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsVSqu7rF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31450461%5Buid%5D" title="PubMed Link to resource 31450461">PubMed</a>.</span></li>
      <li><span id="cit263">X. Hu, X. Mao, X. Zhang and Y. Huang, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">247</span>, 312–318 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.03.050" title="DOI Link to resource 10.1016/j.snb.2017.03.050">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXlsVChsLY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit264">C. Boonmee, V. Promarak, T. Tuntulani and W. Ngeontae, <span class="italic">Talanta</span>, 2018, <span class="bold">178</span>, 796–804 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2017.10.006" title="DOI Link to resource 10.1016/j.talanta.2017.10.006">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhslSiu7bL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29136897%5Buid%5D" title="PubMed Link to resource 29136897">PubMed</a>.</span></li>
      <li><span id="cit265">J. Pang, Y. Lu, X. Gao, L. He, J. Sun, F. Yang, Z. Hao and Y. Liu, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 1–9 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-3127-5" title="DOI Link to resource 10.1007/s00604-018-3127-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXpvV2jtL4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30515617%5Buid%5D" title="PubMed Link to resource 30515617">PubMed</a>.</span></li>
      <li><span id="cit266">L. Jin, Z. Zhang, A. Tang, C. Li and Y. Shen, <span class="italic">Biosens. Bioelectron.</span>, 2016, <span class="bold">79</span>, 108–113 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2015.12.001" title="DOI Link to resource 10.1016/j.bios.2015.12.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXitVems7nI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26703988%5Buid%5D" title="PubMed Link to resource 26703988">PubMed</a>.</span></li>
      <li><span id="cit267">Z. Wen, S. Song, T. Hu, C. Wang, F. Qu, P. Wang and M. Yang, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 258 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3295-y" title="DOI Link to resource 10.1007/s00604-019-3295-y">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30915575%5Buid%5D" title="PubMed Link to resource 30915575">PubMed</a>.</span></li>
      <li><span id="cit268">H. Liao, L. Hu, Y. Zhang, X. Yu, Y. Liu and R. Li, <span class="italic">Microchim. Acta</span>, 2018, <span class="bold">185</span>, 143 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-2701-1" title="DOI Link to resource 10.1007/s00604-018-2701-1">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29594697%5Buid%5D" title="PubMed Link to resource 29594697">PubMed</a>.</span></li>
      <li><span id="cit269">G. Zhang, R. Wang, L. Shi, C. Zhang, Y. Zhang, Y. Zhou, C. Dong, G. Li and S. Shuang, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">279</span>, 361–368 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2018.09.100" title="DOI Link to resource 10.1016/j.snb.2018.09.100">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhvFWqt7%252FF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit270">X. Cao, X. Li, F. Liu, Y. Luo and L. Yu, <span class="italic">Luminescence</span>, 2018, <span class="bold">33</span>, 981–985 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/bio.3498" title="DOI Link to resource 10.1002/bio.3498">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjsbbM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29790654%5Buid%5D" title="PubMed Link to resource 29790654">PubMed</a>.</span></li>
      <li><span id="cit271">Z. Chen, Y. Niu, G. Cheng, L. Tong, G. Zhang, F. Cai, T. Chen, B. Liu and B. Tang, <span class="italic">Analyst</span>, 2017, <span class="bold">142</span>, 2781–2785 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7AN00595D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7AN00595D">RSC</a>.</span></li>
      <li><span id="cit272">F. Pena-Pereira, N. Capón, I. de la Calle, I. Lavilla and C. Bendicho, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">299</span>, 126979 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2019.126979" title="DOI Link to resource 10.1016/j.snb.2019.126979">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhs1agtL7K" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit273">J. Cang, C.-W. Wang, P.-C. Chen, Y.-J. Lin, Y.-C. Li and H.-T. Chang, <span class="italic">Anal. Methods</span>, 2017, <span class="bold">9</span>, 5254–5259 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7AY01300K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7AY01300K">RSC</a>.</span></li>
      <li><span id="cit274">S. Han and X. Chen, <span class="italic">Spectrochim. Acta, Part A</span>, 2019, <span class="bold">210</span>, 315–320 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.saa.2018.11.051" title="DOI Link to resource 10.1016/j.saa.2018.11.051">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitlegurzL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30472594%5Buid%5D" title="PubMed Link to resource 30472594">PubMed</a>.</span></li>
      <li><span id="cit275">X.-J. Zheng, R.-P. Liang, Z.-J. Li, L. Zhang and J.-D. Qiu, <span class="italic">Sens. Actuators, B</span>, 2016, <span class="bold">230</span>, 314–319 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2016.02.063" title="DOI Link to resource 10.1016/j.snb.2016.02.063">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XjtFOhsbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit276">Q. Du, X. Zhang, H. Cao and Y. Huang, <span class="italic">Microchim. Acta</span>, 2020, <span class="bold">187</span>, 357 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-020-04335-2" title="DOI Link to resource 10.1007/s00604-020-04335-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtVCqsr%252FO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32468344%5Buid%5D" title="PubMed Link to resource 32468344">PubMed</a>.</span></li>
      <li><span id="cit277">H. B. Wang, H. D. Zhang, Y. Chen, Y. Li and T. Gan, <span class="italic">RSC Adv.</span>, 2015, <span class="bold">5</span>, 77906–77912 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5RA14852A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5RA14852A">RSC</a>.</span></li>
      <li><span id="cit278">Y. Zhong, C. Deng, Y. He, Y. Ge and G. Song, <span class="italic">Microchim. Acta</span>, 2016, <span class="bold">183</span>, 2823–2830 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-016-1915-3" title="DOI Link to resource 10.1007/s00604-016-1915-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtlCqsLzN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit279">H. Li, Y. Lu, J. Pang, J. Sun, F. Yang, Z. Wang and Y. Liu, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 862 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-4008-2" title="DOI Link to resource 10.1007/s00604-019-4008-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXit12iu73F" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31792614%5Buid%5D" title="PubMed Link to resource 31792614">PubMed</a>.</span></li>
      <li><span id="cit280">S. Gou, Y. Shi, P. Li, H. Wang, T. Li, X. Zhuang, W. Li and Z. Wang, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2019, <span class="bold">11</span>, 6561–6567 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.8b20835" title="DOI Link to resource 10.1021/acsami.8b20835">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsVans7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30656937%5Buid%5D" title="PubMed Link to resource 30656937">PubMed</a>.</span></li>
      <li><span id="cit281">C. Ma, M. Chen, H. Liu, K. Wu, H. He and K. Wang, <span class="italic">Chin. Chem. Lett.</span>, 2018, <span class="bold">29</span>, 136–138 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cclet.2017.09.012" title="DOI Link to resource 10.1016/j.cclet.2017.09.012">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsFCltLjO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit282">R. Patel, S. Bothra, R. Kumar, G. Crisponi and S. K. Sahoo, <span class="italic">Biosens. Bioelectron.</span>, 2018, <span class="bold">102</span>, 196–203 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2017.11.031" title="DOI Link to resource 10.1016/j.bios.2017.11.031">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvVShsrvI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29145072%5Buid%5D" title="PubMed Link to resource 29145072">PubMed</a>.</span></li>
      <li><span id="cit283">H. Li, J. Chang, T. Hou, L. Ge and F. Li, <span class="italic">Talanta</span>, 2016, <span class="bold">160</span>, 475–480 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2016.07.030" title="DOI Link to resource 10.1016/j.talanta.2016.07.030">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtlaisrfJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27591641%5Buid%5D" title="PubMed Link to resource 27591641">PubMed</a>.</span></li>
      <li><span id="cit284">Z. Bao, K. Zhang, J. Jian, Z. Hu, K. Yuan, H. Shao, K. Peng, Z. Jiang, J. A. Zapien, Y. Yan, C. Zhang and H. Zhou, <span class="italic">Microchim. Acta</span>, 2018, <span class="bold">185</span>, 507 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-3049-2" title="DOI Link to resource 10.1007/s00604-018-3049-2">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30338341%5Buid%5D" title="PubMed Link to resource 30338341">PubMed</a>.</span></li>
      <li><span id="cit285">W. J. Zhang, S. G. Liu, L. Han, Y. Ling, L. L. Liao, S. Mo, H. Q. Luo and N. B. Li, <span class="italic">Anal. Methods</span>, 2018, <span class="bold">10</span>, 4251–4256 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8AY01357H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8AY01357H">RSC</a>.</span></li>
      <li><span id="cit286">K. Shanmugaraj and S. A. John, <span class="italic">New J. Chem.</span>, 2018, <span class="bold">42</span>, 7223–7229 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8NJ00789F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8NJ00789F">RSC</a>.</span></li>
      <li><span id="cit287">X. Li, X. Wu, F. Zhang, B. Zhao and Y. Li, <span class="italic">Talanta</span>, 2019, <span class="bold">195</span>, 372–380 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2018.11.067" title="DOI Link to resource 10.1016/j.talanta.2018.11.067">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisVSgtrrP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30625557%5Buid%5D" title="PubMed Link to resource 30625557">PubMed</a>.</span></li>
      <li><span id="cit288">R. S. Aparna, J. S. Anjali Devi, P. Sachidanandan and S. George, <span class="italic">Sens. Actuators, B</span>, 2018, <span class="bold">254</span>, 811–819 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.07.097" title="DOI Link to resource 10.1016/j.snb.2017.07.097">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXht1Oktb7E" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit289">G. Hambarde, S. Bothra, Y. Upadhyay, R. K. Bera and S. K. Sahoo, <span class="italic">Microchem. J.</span>, 2019, <span class="bold">147</span>, 899–904 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.microc.2019.04.016" title="DOI Link to resource 10.1016/j.microc.2019.04.016">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXnsVamsr8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit290">Q. Li, Y. Li, H. Li, X. Yan, G. Han, F. Chen, Z. Song, J. Zhang, W. Fan, C. Yi, Z. Xu, B. Tan and W. Yan, <span class="italic">Nanomaterials</span>, 2020, <span class="bold">10</span>, 1531 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/nano10081531" title="DOI Link to resource 10.3390/nano10081531">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsl2gu7jJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32759865%5Buid%5D" title="PubMed Link to resource 32759865">PubMed</a>.</span></li>
      <li><span id="cit291">C. Song, W. Hong, X. Zhang and Y. Lu, <span class="italic">Analyst</span>, 2018, <span class="bold">143</span>, 1829–1834 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8AN00158H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8AN00158H">RSC</a>.</span></li>
      <li><span id="cit292">Y. He, F. Tian, J. Zhou and B. Jiao, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 199 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3314-z" title="DOI Link to resource 10.1007/s00604-019-3314-z">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30796615%5Buid%5D" title="PubMed Link to resource 30796615">PubMed</a>.</span></li>
      <li><span id="cit293">Y. Xiong, B. Gao, K. Wu, Y. Wu, Y. Chai, X. Huang and Y. Xiong, <span class="italic">Nanoscale</span>, 2018, <span class="bold">10</span>, 19890–19897 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8NR06175K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8NR06175K">RSC</a>.</span></li>
      <li><span id="cit294">Y. Zhang, Z. Zhu, X. Teng, Y. Lai, S. Pu, P. Pang, H. Wang, C. Yang, C. J. Barrow and W. Yang, <span class="italic">Talanta</span>, 2019, <span class="bold">202</span>, 279–284 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2019.05.013" title="DOI Link to resource 10.1016/j.talanta.2019.05.013">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXptFKmurs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31171183%5Buid%5D" title="PubMed Link to resource 31171183">PubMed</a>.</span></li>
      <li><span id="cit295">Y. Zhang, Y. Lai, X. Teng, S. Pu, Z. Yang, P. Pang, H. Wang, C. Yang, W. Yang and C. J. Barrow, <span class="italic">Anal. Methods</span>, 2020, <span class="bold">12</span>, 1752–1758 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9AY02250C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9AY02250C">RSC</a>.</span></li>
      <li><span id="cit296">Y. Ma, Y. Yu, B. Lin, L. Zhang, Y. Cao and M. Guo, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 206 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3327-7" title="DOI Link to resource 10.1007/s00604-019-3327-7">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30820671%5Buid%5D" title="PubMed Link to resource 30820671">PubMed</a>.</span></li>
      <li><span id="cit297">Z. Cai, X. Yin, J. Fang, J. Zhao, T. Wu, E. Hao, F. Tian and C. Zhang, <span class="italic">Nano</span>, 2020, <span class="bold">15</span>, 2050063 <a target="_blank" class="DOILink" href="https://doi.org/10.1142/S1793292020500630" title="DOI Link to resource 10.1142/S1793292020500630">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXpvFamt7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit298">K. Fan, W. Kang, S. Qu, L. Li, B. Qu and L. Lu, <span class="italic">Talanta</span>, 2019, <span class="bold">197</span>, 645–652 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2019.01.069" title="DOI Link to resource 10.1016/j.talanta.2019.01.069">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisVWhsL4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit299">Y. Yang, Q. Wei, T. Zou, Y. Kong, L. Su, D. Ma and Y. Wang, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">321</span>, 128534 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2020.128534" title="DOI Link to resource 10.1016/j.snb.2020.128534">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtlOhurfF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit300">S. Chen, Y. Wang and L. Feng, <span class="italic">Talanta</span>, 2020, <span class="bold">210</span>, 120627 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2019.120627" title="DOI Link to resource 10.1016/j.talanta.2019.120627">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvFyhuw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit301">R. Rajamanikandan and M. Ilanchelian, <span class="italic">Mater. Sci. Eng., C</span>, 2019, <span class="bold">98</span>, 1064–1072 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.msec.2019.01.048" title="DOI Link to resource 10.1016/j.msec.2019.01.048">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvVWit7g%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit302">H.-B. Wang, H.-D. Zhang, Y. Chen, K.-J. Huang and Y.-M. Liu, <span class="italic">Sens. Actuators, B</span>, 2015, <span class="bold">220</span>, 146–153 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2015.05.055" title="DOI Link to resource 10.1016/j.snb.2015.05.055">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtVais7fF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit303">Z. Miao, W. Hou, M. Liu, Y. Zhang and S. Yao, <span class="italic">New J. Chem.</span>, 2018, <span class="bold">42</span>, 1446–1456 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7NJ03524A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7NJ03524A">RSC</a>.</span></li>
      <li><span id="cit304">W. He, R. Gui, H. Jin, B. Wang, X. Bu and Y. Fu, <span class="italic">Talanta</span>, 2018, <span class="bold">178</span>, 109–115 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2017.09.019" title="DOI Link to resource 10.1016/j.talanta.2017.09.019">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWgtrvK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29136799%5Buid%5D" title="PubMed Link to resource 29136799">PubMed</a>.</span></li>
      <li><span id="cit305">X. Zhang, Q. Liu, Z.-W. Wang, H. Xu, F.-P. An, Q. Huang, H.-B. Song and Y.-W. Wang, <span class="italic">Microchim. Acta</span>, 2020, <span class="bold">187</span>, 329 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-020-04271-1" title="DOI Link to resource 10.1007/s00604-020-04271-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXpt1ynsbs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32405710%5Buid%5D" title="PubMed Link to resource 32405710">PubMed</a>.</span></li>
      <li><span id="cit306">X.-M. Wu, F. Zhang and Y. Li, <span class="italic">J. Mater. Chem. B</span>, 2018, <span class="bold">6</span>, 5466–5475 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8TB01733F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8TB01733F">RSC</a>.</span></li>
      <li><span id="cit307">R. S. Aparna, J. S. Anjali Devi, R. R. Anjana, J. Nebu and S. George, <span class="italic">Analyst</span>, 2019, <span class="bold">144</span>, 1799–1808 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8AN01703D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8AN01703D">RSC</a>.</span></li>
      <li><span id="cit308">A. Hatefi, E. Rahimpour, M. Khoubnasabjafari, M. Edalat, V. Jouyban-Gharamaleki, S. Alvani-Alamdari, A. Nokhodchi, M. H. Pournaghi-Azar and A. Jouyban, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 194 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3278-z" title="DOI Link to resource 10.1007/s00604-019-3278-z">CrossRef</a>.</span></li>
      <li><span id="cit309">H. Bagheri, A. Afkhami, H. Khoshsafar, A. Hajian and A. Shahriyari, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">89</span>, 829–836 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2016.10.003" title="DOI Link to resource 10.1016/j.bios.2016.10.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xhsl2iurfP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27818050%5Buid%5D" title="PubMed Link to resource 27818050">PubMed</a>.</span></li>
      <li><span id="cit310">Z. Wang, C. C. Zhang, J. Gao and Q. Wang, <span class="italic">J. Lumin.</span>, 2017, <span class="bold">190</span>, 115–122 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jlumin.2017.05.038" title="DOI Link to resource 10.1016/j.jlumin.2017.05.038">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXosVWnt7c%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit311">Y. Luo, H. Miao and X. Yang, <span class="italic">Talanta</span>, 2015, <span class="bold">144</span>, 488–495 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2015.07.001" title="DOI Link to resource 10.1016/j.talanta.2015.07.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOmtb7P" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit312">K. Yang, Y. Wang, C. Lu and X. Yang, <span class="italic">J. Lumin.</span>, 2018, <span class="bold">196</span>, 181–186 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jlumin.2017.12.038" title="DOI Link to resource 10.1016/j.jlumin.2017.12.038">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXitVams7fP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit313">Y. Wang, T. Chen, Q. Zhuang and Y. Ni, <span class="italic">Talanta</span>, 2018, <span class="bold">179</span>, 409–413 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2017.11.014" title="DOI Link to resource 10.1016/j.talanta.2017.11.014">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2is7%252FM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit314">C. Ma, W. Ren, J. Tang, X. Wang, D. Ji, R. Meng, C. Zhang and Q. Wang, <span class="italic">Luminescence</span>, 2019, <span class="bold">34</span>, 767–773 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/bio.3672" title="DOI Link to resource 10.1002/bio.3672">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFOru73I" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31267664%5Buid%5D" title="PubMed Link to resource 31267664">PubMed</a>.</span></li>
      <li><span id="cit315">X. Bu, Y. Fu, H. Jin and R. Gui, <span class="italic">New J. Chem.</span>, 2018, <span class="bold">42</span>, 17323–17330 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8NJ03927E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8NJ03927E">RSC</a>.</span></li>
      <li><span id="cit316">S. Xu, K. Zhou, D. Fang and L. Ma, <span class="italic">Molecules</span>, 2018, <span class="bold">24</span>, 95 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/molecules24010095" title="DOI Link to resource 10.3390/molecules24010095">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30597835%5Buid%5D" title="PubMed Link to resource 30597835">PubMed</a>.</span></li>
      <li><span id="cit317">Z. Cai, H. Li, J. Wu, L. Zhu, X. Ma and C. Zhang, <span class="italic">RSC Adv.</span>, 2020, <span class="bold">10</span>, 8989–8993 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0RA01265C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0RA01265C">RSC</a>.</span></li>
      <li><span id="cit318">Z. Cai, R. Zhu, C. Zhang, E. Hao, J. Zhao and T. Wu, <span class="italic">Chem. Pap.</span>, 2020, <span class="bold">74</span>, 3839–3846 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11696-020-01199-x" title="DOI Link to resource 10.1007/s11696-020-01199-x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtVWit73M" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit319">X. Bu, Y. Fu, X. Jiang, H. Jin and R. Gui, <span class="italic">Microchim. Acta</span>, 2020, <span class="bold">187</span>, 154 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-020-4146-6" title="DOI Link to resource 10.1007/s00604-020-4146-6">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXjsFSnurg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32020297%5Buid%5D" title="PubMed Link to resource 32020297">PubMed</a>.</span></li>
      <li><span id="cit320">A. Yousefzadeh, J. Hassanzadeh, S. M. J. Mousavi and M. Yousefzadeh, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">286</span>, 154–162 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2019.01.155" title="DOI Link to resource 10.1016/j.snb.2019.01.155">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisFClsL8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit321">C. Muñoz-Bustos, A. Tirado-Guízar, F. Paraguay-Delgado and G. Pina-Luis, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">244</span>, 922–927 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.01.071" title="DOI Link to resource 10.1016/j.snb.2017.01.071">CrossRef</a>.</span></li>
      <li><span id="cit322">R. Rajamanikandan and M. Ilanchelian, <span class="italic">Anal. Methods</span>, 2018, <span class="bold">10</span>, 3666–3674 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8AY00827B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8AY00827B">RSC</a>.</span></li>
      <li><span id="cit323">T. Li, Z. Wang, D. Jiang, H. Wang, W. F. Lai, Y. Lv and Y. Zhai, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">290</span>, 535–543 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2019.04.033" title="DOI Link to resource 10.1016/j.snb.2019.04.033">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXntlKitbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit324">Y. Shi, F. Han, L. Xie, C. Zhang, T. Li, H. Wang, W.-F. Lai, S. Luo, W. Wei, Z. Wang and Y. Huang, <span class="italic">Microchim. Acta</span>, 2020, <span class="bold">187</span>, 38 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-4000-x" title="DOI Link to resource 10.1007/s00604-019-4000-x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitl2hu7nM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31823079%5Buid%5D" title="PubMed Link to resource 31823079">PubMed</a>.</span></li>
      <li><span id="cit325">L. Zhang, Q.-Y. Cai, J. Li, J. Ge, J.-Y. Wang, Z.-Z. Dong and Z.-H. Li, <span class="italic">Biosens. Bioelectron.</span>, 2015, <span class="bold">69</span>, 77–82 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2015.02.012" title="DOI Link to resource 10.1016/j.bios.2015.02.012">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXisVWjsLk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25703731%5Buid%5D" title="PubMed Link to resource 25703731">PubMed</a>.</span></li>
      <li><span id="cit326">X. Ouyang, M. Wang, L. Guo, C. Cui, T. Liu, Y. Ren, Y. Zhao, Z. Ge, X. Guo, G. Xie, J. Li, C. Fan and L. Wang, <span class="italic">Angew. Chem.</span>, 2020, <span class="bold">132</span>, 11934–11942 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ange.202003905" title="DOI Link to resource 10.1002/ange.202003905">CrossRef</a>.</span></li>
      <li><span id="cit327">M. Zhang, J. Qiao, S. Zhang and L. Qi, <span class="italic">Talanta</span>, 2018, <span class="bold">182</span>, 595–599 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2018.02.035" title="DOI Link to resource 10.1016/j.talanta.2018.02.035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjt12js74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29501198%5Buid%5D" title="PubMed Link to resource 29501198">PubMed</a>.</span></li>
      <li><span id="cit328">Y. S. Borghei, M. Hosseini, M. Khoobi and M. R. Ganjali, <span class="italic">Luminescence</span>, 2017, <span class="bold">32</span>, 1045–1050 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/bio.3289" title="DOI Link to resource 10.1002/bio.3289">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu7%252FO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28220689%5Buid%5D" title="PubMed Link to resource 28220689">PubMed</a>.</span></li>
      <li><span id="cit329">Y. Wang, T. Chen, Q. Zhuang and Y. Ni, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2017, <span class="bold">9</span>, 32135–32141 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.7b09768" title="DOI Link to resource 10.1021/acsami.7b09768">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVWhtrrP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28853550%5Buid%5D" title="PubMed Link to resource 28853550">PubMed</a>.</span></li>
      <li><span id="cit330">Q. Song, R. Wang, F. Sun, H. Chen, Z. Wang, N. Na and J. Ouyang, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">87</span>, 760–763 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2016.09.029" title="DOI Link to resource 10.1016/j.bios.2016.09.029">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFartb7M" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27649332%5Buid%5D" title="PubMed Link to resource 27649332">PubMed</a>.</span></li>
      <li><span id="cit331">Y. M. Wang, J. W. Liu, L. Y. Duan, S. J. Liu and J. H. Jiang, <span class="italic">Microchim. Acta</span>, 2017, <span class="bold">184</span>, 4183–4188 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-017-2337-6" title="DOI Link to resource 10.1007/s00604-017-2337-6">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXpsVChtbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit332">S. S. Zhou, L. Zhang, Q. Y. Cai, Z. Z. Dong, X. Geng, J. Ge and Z. H. Li, <span class="italic">Anal. Bioanal. Chem.</span>, 2016, <span class="bold">408</span>, 6711–6717 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00216-016-9788-1" title="DOI Link to resource 10.1007/s00216-016-9788-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1CksrrF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27457102%5Buid%5D" title="PubMed Link to resource 27457102">PubMed</a>.</span></li>
      <li><span id="cit333">X. Gao, J. Liu, X. Zhuang, C. Tian, F. Luan, H. Liu and Y. Xiong, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">308</span>, 127720 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2020.127720" title="DOI Link to resource 10.1016/j.snb.2020.127720">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Shu7k%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit334">J. Ge, Z. Z. Dong, L. Zhang, Q. Y. Cai, D. M. Bai and Z. H. Li, <span class="italic">RSC Adv.</span>, 2016, <span class="bold">6</span>, 91077–91082 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6RA17579A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6RA17579A">RSC</a>.</span></li>
      <li><span id="cit335">Y. Wang, H. Cui, Z. Cao, C. Lau and J. Lu, <span class="italic">Talanta</span>, 2016, <span class="bold">154</span>, 574–580 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2015.12.067" title="DOI Link to resource 10.1016/j.talanta.2015.12.067">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xot1KgtA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27154717%5Buid%5D" title="PubMed Link to resource 27154717">PubMed</a>.</span></li>
      <li><span id="cit336">F. Qu, B. Wang, K. Li, J. You and W. Han, <span class="italic">Microchim. Acta</span>, 2020, <span class="bold">187</span>, 457 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-020-04438-w" title="DOI Link to resource 10.1007/s00604-020-04438-w">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWms7vM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32683631%5Buid%5D" title="PubMed Link to resource 32683631">PubMed</a>.</span></li>
      <li><span id="cit337">J. Cao, W. Wang, B. Bo, X. Mao, K. Wang and X. Zhu, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">90</span>, 534–541 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2016.10.021" title="DOI Link to resource 10.1016/j.bios.2016.10.021">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhvVWitbvM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27825879%5Buid%5D" title="PubMed Link to resource 27825879">PubMed</a>.</span></li>
      <li><span id="cit338">H. Liao, Y. Zhou, Y. Chai and R. Yuan, <span class="italic">Biosens. Bioelectron.</span>, 2018, <span class="bold">114</span>, 10–14 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2018.05.011" title="DOI Link to resource 10.1016/j.bios.2018.05.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpsFyjsLw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit339">Y. Li, D. Tang, L. Zhu, J. Cai, C. Chu, J. Wang, M. Xia, Z. Cao and H. Zhu, <span class="italic">Anal. Biochem.</span>, 2019, <span class="bold">585</span>, 113346 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2019.113346" title="DOI Link to resource 10.1016/j.ab.2019.113346">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhs1yhtL3J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31401004%5Buid%5D" title="PubMed Link to resource 31401004">PubMed</a>.</span></li>
      <li><span id="cit340">Y. Wang, X. Zhang, L. Zhao, T. Bao, W. Wen, X. Zhang and S. Wang, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">98</span>, 386–391 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2017.07.009" title="DOI Link to resource 10.1016/j.bios.2017.07.009">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFGmurrE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit341">Y. Tang, M. Liu, Z. Zhao, Q. Li, X. Liang, J. Tian and S. Zhao, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 133 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3237-8" title="DOI Link to resource 10.1007/s00604-019-3237-8">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30707302%5Buid%5D" title="PubMed Link to resource 30707302">PubMed</a>.</span></li>
      <li><span id="cit342">Y.-S. Borghei, M. Hosseini and M. R. Ganjali, <span class="italic">Clin. Chim. Acta</span>, 2018, <span class="bold">483</span>, 119–125 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cca.2018.04.031" title="DOI Link to resource 10.1016/j.cca.2018.04.031">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXosFGqu7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29704472%5Buid%5D" title="PubMed Link to resource 29704472">PubMed</a>.</span></li>
      <li><span id="cit343">Y. S. Borghei, M. Hosseini and M. R. Ganjali, <span class="italic">Microchim. Acta</span>, 2017, <span class="bold">184</span>, 2671–2677 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-017-2272-6" title="DOI Link to resource 10.1007/s00604-017-2272-6">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXmvFGjsbo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit344">Y. Zhou, H. Wang, H. Zhang, Y. Chai and R. Yuan, <span class="italic">Anal. Chem.</span>, 2018, <span class="bold">90</span>, 3543–3549 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.7b05402" title="DOI Link to resource 10.1021/acs.analchem.7b05402">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXit1Wju7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29400060%5Buid%5D" title="PubMed Link to resource 29400060">PubMed</a>.</span></li>
      <li><span id="cit345">Y. Wang, W. Meng, X. Chen and Y. Zhang, <span class="italic">Microchim. Acta</span>, 2020, <span class="bold">187</span>, 4 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-4011-7" title="DOI Link to resource 10.1007/s00604-019-4011-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitlejsb7N" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit346">J. Ge, Z. Z. Dong, D. M. Bai, L. Zhang, Y. L. Hu, D. Y. Ji and Z. H. Li, <span class="italic">New J. Chem.</span>, 2017, <span class="bold">41</span>, 9718–9723 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7NJ01761H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7NJ01761H">RSC</a>.</span></li>
      <li><span id="cit347">H. Zhao, J. Dong, F. Zhou and B. Li, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">238</span>, 828–833 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2016.07.083" title="DOI Link to resource 10.1016/j.snb.2016.07.083">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1Krt7jM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit348">W. Zhuang, Y. Li, J. Chen, W. Liu and H. Huang, <span class="italic">Anal. Methods</span>, 2019, <span class="bold">11</span>, 2597–2604 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9AY00681H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9AY00681H">RSC</a>.</span></li>
      <li><span id="cit349">J. Yin, F. Liu, T. Fan, Y. Ren and Y. Jiang, <span class="italic">Sens. Actuators, B</span>, 2018, <span class="bold">276</span>, 499–506 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2018.08.140" title="DOI Link to resource 10.1016/j.snb.2018.08.140">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhs1KgtrvN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit350">D. Gao, H. Zhang, Y. Xu, Y. Liu, H. Xu and J. Cui, <span class="italic">Anal. Biochem.</span>, 2018, <span class="bold">559</span>, 5–10 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ab.2018.08.011" title="DOI Link to resource 10.1016/j.ab.2018.08.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsFChu7jK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30120924%5Buid%5D" title="PubMed Link to resource 30120924">PubMed</a>.</span></li>
      <li><span id="cit351">T. Qing, C. Long, X. Wang, K. Zhang, P. Zhang and B. Feng, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 248 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3363-3" title="DOI Link to resource 10.1007/s00604-019-3363-3">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30887121%5Buid%5D" title="PubMed Link to resource 30887121">PubMed</a>.</span></li>
      <li><span id="cit352">Q. Liu, Q. Lai, N. Li and X. Su, <span class="italic">Microchim. Acta</span>, 2018, <span class="bold">185</span>, 182 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-017-2599-z" title="DOI Link to resource 10.1007/s00604-017-2599-z">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29594686%5Buid%5D" title="PubMed Link to resource 29594686">PubMed</a>.</span></li>
      <li><span id="cit353">Z. Qing, X. He, T. Qing, K. Wang, H. Shi, D. He, Z. Zou, L. Yan, F. Xu, X. Ye and Z. Mao, <span class="italic">Anal. Chem.</span>, 2013, <span class="bold">85</span>, 12138–12143 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ac403354c" title="DOI Link to resource 10.1021/ac403354c">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhslymsLzJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24236868%5Buid%5D" title="PubMed Link to resource 24236868">PubMed</a>.</span></li>
      <li><span id="cit354">X. Peng, S.-Y. Chen, L.-J. Ou, F.-W. Luo, S.-W. Qin and A. Sun, <span class="italic">Analyst</span>, 2018, <span class="bold">143</span>, 415–419 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7AN01725A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7AN01725A">RSC</a>.</span></li>
      <li><span id="cit355">H.-B. Wang, Y. Li, Y. Chen, Z.-P. Zhang, T. Gan and Y.-M. Liu, <span class="italic">Microchim. Acta</span>, 2018, <span class="bold">185</span>, 102 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-017-2622-4" title="DOI Link to resource 10.1007/s00604-017-2622-4">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29594450%5Buid%5D" title="PubMed Link to resource 29594450">PubMed</a>.</span></li>
      <li><span id="cit356">Y. Hu, Y. He, Y. Han, Y. Ge, G. Song and J. Zhou, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 5 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-3122-x" title="DOI Link to resource 10.1007/s00604-018-3122-x">CrossRef</a>.</span></li>
      <li><span id="cit357">R. Li, Q. Liu, Y. Jin and B. Li, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">281</span>, 28–33 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2018.09.128" title="DOI Link to resource 10.1016/j.snb.2018.09.128">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhvFyktb7M" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit358">M. Cao, Y. Jin and B. Li, <span class="italic">Anal. Methods</span>, 2016, <span class="bold">8</span>, 4319–4323 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6AY00900J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6AY00900J">RSC</a>.</span></li>
      <li><span id="cit359">Y. Ling, J. Zhou, X. F. Zhang, X. H. Wang, N. B. Li and H. Q. Luo, <span class="italic">Sens. Actuators, B</span>, 2019, <span class="bold">286</span>, 46–51 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2019.01.108" title="DOI Link to resource 10.1016/j.snb.2019.01.108">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitlKqtL4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit360">G. Liu, W. He and C. Liu, <span class="italic">Talanta</span>, 2019, <span class="bold">195</span>, 320–326 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2018.11.083" title="DOI Link to resource 10.1016/j.talanta.2018.11.083">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisVSgtrvN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30625549%5Buid%5D" title="PubMed Link to resource 30625549">PubMed</a>.</span></li>
      <li><span id="cit361">Z.-Z. Dong, L. Zhang, M. Qiao, J. Ge, A.-L. Liu and Z.-H. Li, <span class="italic">Talanta</span>, 2016, <span class="bold">146</span>, 253–258 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2015.08.055" title="DOI Link to resource 10.1016/j.talanta.2015.08.055">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsVCmsrjO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26695260%5Buid%5D" title="PubMed Link to resource 26695260">PubMed</a>.</span></li>
      <li><span id="cit362">J. Ge, L. Zhang, Z. Z. Dong, Q. Y. Cai and Z. H. Li, <span class="italic">Anal. Methods</span>, 2016, <span class="bold">8</span>, 2831–2836 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6AY00306K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6AY00306K">RSC</a>.</span></li>
      <li><span id="cit363">X. Zhang, Q. Liu, Y. Jin and B. Li, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 1–9 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-3127-5" title="DOI Link to resource 10.1007/s00604-018-3127-5">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30515617%5Buid%5D" title="PubMed Link to resource 30515617">PubMed</a>.</span></li>
      <li><span id="cit364">L. Zhang, J. Zhao, H. Zhang, J. Jiang and R. Yu, <span class="italic">Biosens. Bioelectron.</span>, 2013, <span class="bold">44</span>, 6–9 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2012.12.037" title="DOI Link to resource 10.1016/j.bios.2012.12.037">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXjvVegs78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23380644%5Buid%5D" title="PubMed Link to resource 23380644">PubMed</a>.</span></li>
      <li><span id="cit365">M. Wang, L. Liu, X. Xie, X. Zhou, Z. Lin and X. Su, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">313</span>, 128023 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2020.128023" title="DOI Link to resource 10.1016/j.snb.2020.128023">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXmvFKru7g%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit366">P. Ni, Y. Sun, S. Jiang, W. Lu, Y. Wang, Z. Li and Z. Li, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">240</span>, 651–656 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2016.08.096" title="DOI Link to resource 10.1016/j.snb.2016.08.096">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFSjtLrM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit367">F. Zhou, X. Cui, A. Shang, J. Lian, L. Yang, Y. Jin and B. Li, <span class="italic">Microchim. Acta</span>, 2017, <span class="bold">184</span>, 773–779 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-016-2065-3" title="DOI Link to resource 10.1007/s00604-016-2065-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXisleiug%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit368">K. Zhang, X. Wang, C. Long, J. Xu, Z. Jiang, B. Feng, P. Zhang, J. Fei and T. Qing, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">316</span>, 128072 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2020.128072" title="DOI Link to resource 10.1016/j.snb.2020.128072">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXoslyqsbs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit369">J. Yang, Z. Li and Q. Jia, <span class="italic">Chem. Commun.</span>, 2020, <span class="bold">56</span>, 3081–3084 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0CC00389A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0CC00389A">RSC</a>.</span></li>
      <li><span id="cit370">X. Huang, H. Zhao, W. Qiu, J. Wang, L. Guo, Z. Lin, W. Pan, Y. Wu and B. Qiu, <span class="italic">Microchim. Acta</span>, 2020, <span class="bold">187</span>, 240 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-020-4186-y" title="DOI Link to resource 10.1007/s00604-020-4186-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXltlCqs7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32198661%5Buid%5D" title="PubMed Link to resource 32198661">PubMed</a>.</span></li>
      <li><span id="cit371">T. Ye, Y. Peng, M. Yuan, H. Cao, J. Yu, Y. Li and F. Xu, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 760 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3927-2" title="DOI Link to resource 10.1007/s00604-019-3927-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitFGitr7I" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31712919%5Buid%5D" title="PubMed Link to resource 31712919">PubMed</a>.</span></li>
      <li><span id="cit372">J. Yang, N. Song and Q. Jia, <span class="italic">Nanoscale</span>, 2019, <span class="bold">11</span>, 21927–21933 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9NR06036G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9NR06036G">RSC</a>.</span></li>
      <li><span id="cit373">S. Chen, Z. Huang and Q. Jia, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">319</span>, 128305 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2020.128305" title="DOI Link to resource 10.1016/j.snb.2020.128305">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXpvValsb4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit374">M. Wang, L. Wang, Q. Liu and X. Su, <span class="italic">Sens. Actuators, B</span>, 2018, <span class="bold">256</span>, 691–698 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2017.09.213" title="DOI Link to resource 10.1016/j.snb.2017.09.213">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GqtrjF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit375">H.-B. Wang, H.-D. Zhang, Y. Chen and Y.-M. Liu, <span class="italic">Biosens. Bioelectron.</span>, 2015, <span class="bold">74</span>, 581–586 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2015.07.021" title="DOI Link to resource 10.1016/j.bios.2015.07.021">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtlymtb3F" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26190469%5Buid%5D" title="PubMed Link to resource 26190469">PubMed</a>.</span></li>
      <li><span id="cit376">Y. He and B. Jiao, <span class="italic">Sens. Actuators, B</span>, 2018, <span class="bold">265</span>, 387–393 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2018.03.047" title="DOI Link to resource 10.1016/j.snb.2018.03.047">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXlt1Smurc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit377">X.-G. Li, F. Zhang, Y. Gao, Q.-M. Zhou, Y. Zhao, Y. Li, J.-Z. Huo and X.-J. Zhao, <span class="italic">Biosens. Bioelectron.</span>, 2016, <span class="bold">86</span>, 270–276 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2016.06.054" title="DOI Link to resource 10.1016/j.bios.2016.06.054">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtFSnsL7K" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit378">Q. Song, L. Yang, H. Chen, R. Zhang, N. Na and J. Ouyang, <span class="italic">Talanta</span>, 2017, <span class="bold">174</span>, 444–447 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2017.06.031" title="DOI Link to resource 10.1016/j.talanta.2017.06.031">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKhsL%252FI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit379">L. J. Ou, X. Y. Li, L. J. Li, H. W. Liu, A. M. Sun and K. J. Liu, <span class="italic">Analyst</span>, 2015, <span class="bold">140</span>, 1871–1875 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C4AN01994F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C4AN01994F">RSC</a>.</span></li>
      <li><span id="cit380">S. Zhang, C. Chen, X. Qin, Q. Zhang, J. Liu, J. Zhu, Y. Gao, L. Li and W. Huang, <span class="italic">Talanta</span>, 2018, <span class="bold">189</span>, 92–99 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.talanta.2018.06.026" title="DOI Link to resource 10.1016/j.talanta.2018.06.026">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1KqsbjF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30086981%5Buid%5D" title="PubMed Link to resource 30086981">PubMed</a>.</span></li>
      <li><span id="cit381">L. Wang, F. Shi, Y. Li and X. Su, <span class="italic">Sens. Actuators, B</span>, 2016, <span class="bold">222</span>, 945–951 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2015.09.024" title="DOI Link to resource 10.1016/j.snb.2015.09.024">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFCitLnE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit382">Y. Hu, Y. He, Y. Han, Y. Ge, G. Song and J. Zhou, <span class="italic">Microchim. Acta</span>, 2018, <span class="bold">185</span>, 383 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-2920-5" title="DOI Link to resource 10.1007/s00604-018-2920-5">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30032328%5Buid%5D" title="PubMed Link to resource 30032328">PubMed</a>.</span></li>
      <li><span id="cit383">F. M. Moghadam and M. Rahaie, <span class="italic">Biosens. Bioelectron.</span>, 2019, <span class="bold">132</span>, 186–195 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2019.02.046" title="DOI Link to resource 10.1016/j.bios.2019.02.046">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXkvVegu74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit384">M. Chen, F. Yeasmin Khusbu, C. Ma, K. Wu, H. Zhao, H. Chen and K. Wang, <span class="italic">New J. Chem.</span>, 2018, <span class="bold">42</span>, 13702–13707 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8NJ02774A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8NJ02774A">RSC</a>.</span></li>
      <li><span id="cit385">L. Zhao and Z. Ma, <span class="italic">Sens. Actuators, B</span>, 2017, <span class="bold">241</span>, 849–854 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2016.11.012" title="DOI Link to resource 10.1016/j.snb.2016.11.012">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhvVags7%252FN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit386">J. Ye, X. Dong, H. Jiang and X. Wang, <span class="italic">J. Mater. Chem. B</span>, 2017, <span class="bold">5</span>, 691–696 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6TB02751B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6TB02751B">RSC</a>.</span></li>
      <li><span id="cit387">C. Wang, Y. Yao and Q. Song, <span class="italic">Colloids Surf., B</span>, 2016, <span class="bold">140</span>, 373–381 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.colsurfb.2016.01.001" title="DOI Link to resource 10.1016/j.colsurfb.2016.01.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtFSjsrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit388">Y. Qiao, T. Xu, Y. Zhang, C. Zhang, L. Shi, G. Zhang, S. Shuang and C. Dong, <span class="italic">Sens. Actuators, B</span>, 2015, <span class="bold">220</span>, 1064–1069 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2015.06.073" title="DOI Link to resource 10.1016/j.snb.2015.06.073">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtV2ls7zM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit389">Z. Cai, C. Zhang and K. Jia, <span class="italic">Chem. Pap.</span>, 2020, <span class="bold">74</span>, 1831–1838 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11696-019-01027-x" title="DOI Link to resource 10.1007/s11696-019-01027-x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisVeqs7nF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit390">X. Mao, S. Liu, C. Yang, F. Liu, K. Wang and G. Chen, <span class="italic">Anal. Chim. Acta</span>, 2016, <span class="bold">909</span>, 101–108 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.aca.2016.01.009" title="DOI Link to resource 10.1016/j.aca.2016.01.009">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xot1ClsQ%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit391">E. Shokri, M. Hosseini, A. A. Sadeghan, A. Bahmani, N. Nasiri and S. Hosseinkhani, <span class="italic">Sens. Actuators, B</span>, 2020, <span class="bold">321</span>, 128634 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.snb.2020.128634" title="DOI Link to resource 10.1016/j.snb.2020.128634">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsFaitbbE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit392">Y. Zhao, X. Wang, J. Mi, Y. Jiang and C. Wang, <span class="italic">Part. Part. Syst. Charact.</span>, 2019, <span class="bold">36</span>, 1–19 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Part.%20Part.%20Syst.%20Charact.%5BJour%5D%20AND%2036%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit393">T. Qing, K. Zhang, Z. Qing, X. Wang, C. Long, P. Zhang, H. Hu and B. Feng, <span class="italic">Microchim. Acta</span>, 2019, <span class="bold">186</span>, 670 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-019-3747-4" title="DOI Link to resource 10.1007/s00604-019-3747-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhslCgsr7M" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit394">Y. An, Y. Ren, M. Bick, A. Dudek, E. Hong-Wang Waworuntu, J. Tang, J. Chen and B. Chang, <span class="italic">Biosens. Bioelectron.</span>, 2020, <span class="bold">154</span>, 112078 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2020.112078" title="DOI Link to resource 10.1016/j.bios.2020.112078">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXjtFeqs7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit395">X. Hu, T. Liu, Y. Zhuang, W. Wang, Y. Li, W. Fan and Y. Huang, <span class="italic">TrAC, Trends Anal. Chem.</span>, 2016, <span class="bold">77</span>, 66–75 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.trac.2015.12.013" title="DOI Link to resource 10.1016/j.trac.2015.12.013">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xktlarsw%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit396">Z. Qing, A. Bai, S. Xing, Z. Zou, X. He, K. Wang and R. Yang, <span class="italic">Biosens. Bioelectron.</span>, 2019, <span class="bold">137</span>, 96–109 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2019.05.014" title="DOI Link to resource 10.1016/j.bios.2019.05.014">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXpsVyjur8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit397">Q. Cao, J. Li and E. Wang, <span class="italic">Biosens. Bioelectron.</span>, 2019, <span class="bold">132</span>, 333–342 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2019.01.046" title="DOI Link to resource 10.1016/j.bios.2019.01.046">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXlt12jtrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit398">WHO, <span class="italic">W. H. O.</span>, 2004, 1–31 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=W.%20H.%20O.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit399">WHO, <span class="italic">W. H. O.</span>, 2019, 1–6 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=W.%20H.%20O.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit400">H. Sun, J. Brocato and M. Costa, <span class="italic">Curr. Environ. Health Rep.</span>, 2015, <span class="bold">2</span>, 295–303 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s40572-015-0054-z" title="DOI Link to resource 10.1007/s40572-015-0054-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht1agsrzL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26231506%5Buid%5D" title="PubMed Link to resource 26231506">PubMed</a>.</span></li>
      <li><span id="cit401">WHO, <span class="italic">W. H. O.</span>, 2003, 1–10 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=W.%20H.%20O.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit402">WHO, <span class="italic">W. H. O.</span>, 2003, 1–41 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=W.%20H.%20O.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit403">WHO, <span class="italic">W. H. O.</span>, 2011, 1–31 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=W.%20H.%20O.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202011%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit404">H. Komaba and M. Fukagawa, <span class="italic">Kidney Int.</span>, 2016, <span class="bold">90</span>, 753–763 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.kint.2016.03.039" title="DOI Link to resource 10.1016/j.kint.2016.03.039">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFWht7vP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27282935%5Buid%5D" title="PubMed Link to resource 27282935">PubMed</a>.</span></li>
      <li><span id="cit405">WHO, <span class="italic">W. H. O.</span>, 2018, 1–5 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=W.%20H.%20O.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit406">WHO, <span class="italic">W. H. O.</span>, 2008, 1–7 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=W.%20H.%20O.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit407">J.-M. Xia, X. Wei, X.-W. Chen, Y. Shu and J.-H. Wang, <span class="italic">Microchim. Acta</span>, 2018, <span class="bold">185</span>, 205 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00604-018-2743-4" title="DOI Link to resource 10.1007/s00604-018-2743-4">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29594762%5Buid%5D" title="PubMed Link to resource 29594762">PubMed</a>.</span></li>
      <li><span id="cit408">Y. Wang, Y. Cui, R. Liu, Y. Wei, X. Jiang, H. Zhu, L. Gao, Y. Zhao, Z. Chai and X. Gao, <span class="italic">Chem. Commun.</span>, 2013, <span class="bold">49</span>, 10724–10726 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3CC46690F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3CC46690F">RSC</a>.</span></li>
      <li><span id="cit409">M. J. Barthel, I. Angeloni, A. Petrelli, T. Avellini, A. Scarpellini, G. Bertoni, A. Armirotti, I. Moreels and T. Pellegrino, <span class="italic">ACS Nano</span>, 2015, <span class="bold">9</span>, 11886–11897 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.5b04270" title="DOI Link to resource 10.1021/acsnano.5b04270">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhslCjurjK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26512975%5Buid%5D" title="PubMed Link to resource 26512975">PubMed</a>.</span></li>
      <li><span id="cit410">G. S. Heo, Y. Zhao, D. Sultan, X. Zhang, L. Detering, H. P. Luehmann, X. Zhang, R. Li, A. Choksi, S. Sharp, S. Levingston, T. Primeau, D. E. Reichert, G. Sun, B. Razani, S. Li, K. N. Weilbaecher, F. Dehdashti, K. L. Wooley and Y. Liu, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2019, <span class="bold">11</span>, 19669–19678 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.8b22752" title="DOI Link to resource 10.1021/acsami.8b22752">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXptFyltrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31074257%5Buid%5D" title="PubMed Link to resource 31074257">PubMed</a>.</span></li>
      <li><span id="cit411">W. F. Lai, W. T. Wong and A. L. Rogach, <span class="italic">Adv. Mater.</span>, 2020, <span class="bold">1906872</span>, 1–21 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Mater.%5BJour%5D%20AND%201906872%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit412">J. Xia, W. Wang, X. Hai, S. E, Y. Shu and J. Wang, <span class="italic">Chin. Chem. Lett.</span>, 2019, <span class="bold">30</span>, 421–424 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cclet.2018.07.008" title="DOI Link to resource 10.1016/j.cclet.2018.07.008">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlajt7nL" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit413">A. T. Simon, D. Dutta, A. Chattopadhyay and S. S. Ghosh, <span class="italic">ACS Omega</span>, 2019, <span class="bold">4</span>, 4697–4706 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsomega.8b03076" title="DOI Link to resource 10.1021/acsomega.8b03076">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXktVGhtbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31459656%5Buid%5D" title="PubMed Link to resource 31459656">PubMed</a>.</span></li>
      <li><span id="cit414">U. Goswami, A. Dutta, A. Raza, R. Kandimalla, S. Kalita, S. S. Ghosh and A. Chattopadhyay, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2018, <span class="bold">10</span>, 3282–3294 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.7b15165" title="DOI Link to resource 10.1021/acsami.7b15165">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXitVelt7jE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29278317%5Buid%5D" title="PubMed Link to resource 29278317">PubMed</a>.</span></li>
      <li><span id="cit415">Y. Liu, J. Liang, Q. Wang, Y. He and Y. Chen, <span class="italic">J. Appl. Toxicol.</span>, 2016, <span class="bold">36</span>, 454–463 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jat.3263" title="DOI Link to resource 10.1002/jat.3263">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOltrvF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26594009%5Buid%5D" title="PubMed Link to resource 26594009">PubMed</a>.</span></li>
      <li><span id="cit416">A. Halder, L. A. Curtiss, A. Fortunelli and S. Vajda, <span class="italic">J. Chem. Phys.</span>, 2018, <span class="bold">148</span>, 110901 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5020301" title="DOI Link to resource 10.1063/1.5020301">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29566496%5Buid%5D" title="PubMed Link to resource 29566496">PubMed</a>.</span></li>
      <li><span id="cit417">B. Yang, C. Liu, A. Halder, E. C. Tyo, A. B. F. Martinson, S. Seifert, P. Zapol, L. A. Curtiss and S. Vajda, <span class="italic">J. Phys. Chem. C</span>, 2017, <span class="bold">121</span>, 10406–10412 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b01835" title="DOI Link to resource 10.1021/acs.jpcc.7b01835">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXntFCiu7k%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit418">C. Liu, B. Yang, E. Tyo, S. Seifert, J. Debartolo, B. Von Issendorff, P. Zapol, S. Vajda and L. A. Curtiss, <span class="italic">J. Am. Chem. Soc.</span>, 2015, <span class="bold">137</span>, 8676–8679 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.5b03668" title="DOI Link to resource 10.1021/jacs.5b03668">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOnsbjK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26115184%5Buid%5D" title="PubMed Link to resource 26115184">PubMed</a>.</span></li>
      <li><span id="cit419">C. Cheng, X. Zhang, Z. Yang and Z. Zhou, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2018, <span class="bold">10</span>, 32903–32912 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.8b12318" title="DOI Link to resource 10.1021/acsami.8b12318">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhs1als77E" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30157637%5Buid%5D" title="PubMed Link to resource 30157637">PubMed</a>.</span></li>
      <li><span id="cit420">E. Fernandez, M. Boronat and A. Corma, <span class="italic">J. Phys. Chem. C</span>, 2020, <span class="bold">124</span>, 21549–21558 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.0c06295" title="DOI Link to resource 10.1021/acs.jpcc.0c06295">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhslKqtL3M" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit421">P. López-Caballero, A. W. Hauser and M. Pilar De Lara-Castells, <span class="italic">J. Phys. Chem. C</span>, 2019, <span class="bold">123</span>, 23064–23074 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.9b06620" title="DOI Link to resource 10.1021/acs.jpcc.9b06620">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31598186%5Buid%5D" title="PubMed Link to resource 31598186">PubMed</a>.</span></li>
      <li><span id="cit422">L. Liu and A. Corma, <span class="italic">Chem. Rev.</span>, 2018, <span class="bold">118</span>, 4981–5079 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.7b00776" title="DOI Link to resource 10.1021/acs.chemrev.7b00776">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXnsFWisro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29658707%5Buid%5D" title="PubMed Link to resource 29658707">PubMed</a>.</span></li>
      <li><span id="cit423">N. Vilar-Vidal, J. Rivas and M. A. López-Quintela, <span class="italic">ACS Catal.</span>, 2012, <span class="bold">2</span>, 1693–1697 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/cs300355n" title="DOI Link to resource 10.1021/cs300355n">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhtVWhtLvP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit424">C. Xie, X. Wen, C. Xiao, S. Wei, X. Wu, S. Liu and J. Cao, <span class="italic">Water, Air, Soil Pollut.</span>, 2020, <span class="bold">231</span>, 280 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11270-020-04539-5" title="DOI Link to resource 10.1007/s11270-020-04539-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtVOlt7zF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit425">M. B. Gawande, A. Goswami, F.-X. Felpin, T. Asefa, X. Huang, R. Silva, X. Zou, R. Zboril and R. S. Varma, <span class="italic">Chem. Rev.</span>, 2016, <span class="bold">116</span>, 3722–3811 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.5b00482" title="DOI Link to resource 10.1021/acs.chemrev.5b00482">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XjsVKrsr8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26935812%5Buid%5D" title="PubMed Link to resource 26935812">PubMed</a>.</span></li>
      <li><span id="cit426">F. Alonso, Y. Moglie and G. Radivoy, <span class="italic">Acc. Chem. Res.</span>, 2015, <span class="bold">48</span>, 2516–2528 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.accounts.5b00293" title="DOI Link to resource 10.1021/acs.accounts.5b00293">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsVGjt7%252FO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26332570%5Buid%5D" title="PubMed Link to resource 26332570">PubMed</a>.</span></li>
      <li><span id="cit427">F. Fu, A. Martinez, C. Wang, R. Ciganda, L. Yate, A. Escobar, S. Moya, E. Fouquet, J. Ruiz and D. Astruc, <span class="italic">Chem. Commun.</span>, 2017, <span class="bold">53</span>, 5384–5387 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7CC02504A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7CC02504A">RSC</a>.</span></li>
      <li><span id="cit428">Z. Wang, A. S. Susha, B. Chen, C. Reckmeier, O. Tomanec, R. Zboril, H. Zhong and A. L. Rogach, <span class="italic">Nanoscale</span>, 2016, <span class="bold">8</span>, 7197–7202 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6NR00806B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6NR00806B">RSC</a>.</span></li>
      <li><span id="cit429">Z. Wang, B. Chen, A. S. Susha, W. Wang, C. J. Reckmeier, R. Chen, H. Zhong and A. L. Rogach, <span class="italic">Adv. Sci.</span>, 2016, <span class="bold">3</span>, 1600182 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/advs.201600182" title="DOI Link to resource 10.1002/advs.201600182">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27980993%5Buid%5D" title="PubMed Link to resource 27980993">PubMed</a>.</span></li>
      <li><span id="cit430">Z. Wang, Y. Shi, X. Yang, Y. Xiong, Y. Li, B. Chen, W.-F. Lai and A. L. Rogach, <span class="italic">Adv. Funct. Mater.</span>, 2018, <span class="bold">28</span>, 1802848 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201802848" title="DOI Link to resource 10.1002/adfm.201802848">CrossRef</a>.</span></li>
      <li><span id="cit431">D. Bain, S. Maity and A. Patra, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2019, <span class="bold">21</span>, 5863–5881 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8CP06188B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8CP06188B">RSC</a>.</span></li>
      <li><span id="cit432">E. P. W. Ward, I. Arslan, P. A. Midgley, A. Bleloch and J. M. Thomas, <span class="italic">Chem. Commun.</span>, 2005, 5805–5807 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B511004A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B511004A">RSC</a>.</span></li>
      <li><span id="cit433">J. Liu, N. Jian, I. Ornelas, A. J. Pattison, T. Lahtinen, K. Salorinne, H. Häkkinen and R. E. Palmer, <span class="italic">Ultramicroscopy</span>, 2017, <span class="bold">176</span>, 146–150 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ultramic.2016.11.021" title="DOI Link to resource 10.1016/j.ultramic.2016.11.021">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXkvV2hsbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit434">Z. W. Wang, O. Toikkanen, B. M. Quinn and R. E. Palmer, <span class="italic">Small</span>, 2011, <span class="bold">7</span>, 1542–1545 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/smll.201002168" title="DOI Link to resource 10.1002/smll.201002168">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXmvFWqsr0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21495183%5Buid%5D" title="PubMed Link to resource 21495183">PubMed</a>.</span></li>
      <li><span id="cit435">F. Yin, Z. W. Wang and R. E. Palmer, <span class="italic">J. Exp. Nanosci.</span>, 2012, <span class="bold">7</span>, 703–710 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/17458080.2012.710856" title="DOI Link to resource 10.1080/17458080.2012.710856">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xhslajs7zF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit436">P. Chakraborty and T. Pradeep, <span class="italic">NPG Asia Mater.</span>, 2019, <span class="bold">11</span>, 48 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41427-019-0149-3" title="DOI Link to resource 10.1038/s41427-019-0149-3">CrossRef</a>.</span></li>
      <li><span id="cit437">Y. Lu and W. Chen, <span class="italic">Anal. Chem.</span>, 2015, <span class="bold">87</span>, 10659–10667 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.5b00848" title="DOI Link to resource 10.1021/acs.analchem.5b00848">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtVarsrjM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26086315%5Buid%5D" title="PubMed Link to resource 26086315">PubMed</a>.</span></li>
      <li><span id="cit438">L. Pitkänen and A. M. Striegel, <span class="italic">TrAC, Trends Anal. Chem.</span>, 2016, <span class="bold">80</span>, 311–320 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.trac.2015.06.013" title="DOI Link to resource 10.1016/j.trac.2015.06.013">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27335508%5Buid%5D" title="PubMed Link to resource 27335508">PubMed</a>.</span></li>
      <li><span id="cit439">F. Tagliaro, G. Manetto, F. Crivellente and F. Smith, <span class="italic">Forensic Sci. Int.</span>, 1998, <span class="bold">92</span>, 75–88 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0379-0738(98)00010-3" title="DOI Link to resource 10.1016/S0379-0738(98)00010-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXjs1Sitbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
    </ol>
  <table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2021</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>