<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Recent trends in computational tools and data-driven modeling for advanced materials  - Materials Advances (RSC Publishing) DOI:10.1039/D2MA00067A</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2022/ma/d2ma00067a"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":54,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};;/*! For license information please see nr-loader-spa-1.236.0.min.js.LICENSE.txt */
(()=>{"use strict";var e,t,r={5763:(e,t,r)=>{r.d(t,{P_:()=>l,Mt:()=>g,C5:()=>s,DL:()=>v,OP:()=>T,lF:()=>D,Yu:()=>y,Dg:()=>h,CX:()=>c,GE:()=>b,sU:()=>_});var n=r(8632),i=r(9567);const o={beacon:n.ce.beacon,errorBeacon:n.ce.errorBeacon,licenseKey:void 0,applicationID:void 0,sa:void 0,queueTime:void 0,applicationTime:void 0,ttGuid:void 0,user:void 0,account:void 0,product:void 0,extra:void 0,jsAttributes:{},userAttributes:void 0,atts:void 0,transactionName:void 0,tNamePlain:void 0},a={};function s(e){if(!e)throw new Error("All info objects require an agent identifier!");if(!a[e])throw new Error("Info for ".concat(e," was never set"));return a[e]}function c(e,t){if(!e)throw new Error("All info objects require an agent identifier!");a[e]=(0,i.D)(t,o),(0,n.Qy)(e,a[e],"info")}var u=r(7056);const d=()=>{const e={blockSelector:"[data-nr-block]",maskInputOptions:{password:!0}};return{allow_bfcache:!0,privacy:{cookies_enabled:!0},ajax:{deny_list:void 0,enabled:!0,harvestTimeSeconds:10},distributed_tracing:{enabled:void 0,exclude_newrelic_header:void 0,cors_use_newrelic_header:void 0,cors_use_tracecontext_headers:void 0,allowed_origins:void 0},session:{domain:void 0,expiresMs:u.oD,inactiveMs:u.Hb},ssl:void 0,obfuscate:void 0,jserrors:{enabled:!0,harvestTimeSeconds:10},metrics:{enabled:!0},page_action:{enabled:!0,harvestTimeSeconds:30},page_view_event:{enabled:!0},page_view_timing:{enabled:!0,harvestTimeSeconds:30,long_task:!1},session_trace:{enabled:!0,harvestTimeSeconds:10},harvest:{tooManyRequestsDelay:60},session_replay:{enabled:!1,harvestTimeSeconds:60,sampleRate:.1,errorSampleRate:.1,maskTextSelector:"*",maskAllInputs:!0,get blockClass(){return"nr-block"},get ignoreClass(){return"nr-ignore"},get maskTextClass(){return"nr-mask"},get blockSelector(){return e.blockSelector},set blockSelector(t){e.blockSelector+=",".concat(t)},get maskInputOptions(){return e.maskInputOptions},set maskInputOptions(t){e.maskInputOptions={...t,password:!0}}},spa:{enabled:!0,harvestTimeSeconds:10}}},f={};function l(e){if(!e)throw new Error("All configuration objects require an agent identifier!");if(!f[e])throw new Error("Configuration for ".concat(e," was never set"));return f[e]}function h(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");f[e]=(0,i.D)(t,d()),(0,n.Qy)(e,f[e],"config")}function g(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");var r=l(e);if(r){for(var n=t.split("."),i=0;i<n.length-1;i++)if("object"!=typeof(r=r[n[i]]))return;r=r[n[n.length-1]]}return r}const p={accountID:void 0,trustKey:void 0,agentID:void 0,licenseKey:void 0,applicationID:void 0,xpid:void 0},m={};function v(e){if(!e)throw new Error("All loader-config objects require an agent identifier!");if(!m[e])throw new Error("LoaderConfig for ".concat(e," was never set"));return m[e]}function b(e,t){if(!e)throw new Error("All loader-config objects require an agent identifier!");m[e]=(0,i.D)(t,p),(0,n.Qy)(e,m[e],"loader_config")}const y=(0,n.mF)().o;var w=r(385),x=r(6818);const A={buildEnv:x.Re,bytesSent:{},queryBytesSent:{},customTransaction:void 0,disabled:!1,distMethod:x.gF,isolatedBacklog:!1,loaderType:void 0,maxBytes:3e4,offset:Math.floor(w._A?.performance?.timeOrigin||w._A?.performance?.timing?.navigationStart||Date.now()),onerror:void 0,origin:""+w._A.location,ptid:void 0,releaseIds:{},session:void 0,xhrWrappable:"function"==typeof w._A.XMLHttpRequest?.prototype?.addEventListener,version:x.q4},E={};function T(e){if(!e)throw new Error("All runtime objects require an agent identifier!");if(!E[e])throw new Error("Runtime for ".concat(e," was never set"));return E[e]}function _(e,t){if(!e)throw new Error("All runtime objects require an agent identifier!");E[e]=(0,i.D)(t,A),(0,n.Qy)(e,E[e],"runtime")}function D(e){return function(e){try{const t=s(e);return!!t.licenseKey&&!!t.errorBeacon&&!!t.applicationID}catch(e){return!1}}(e)}},9567:(e,t,r)=>{r.d(t,{D:()=>i});var n=r(50);function i(e,t){try{if(!e||"object"!=typeof e)return(0,n.Z)("Setting a Configurable requires an object as input");if(!t||"object"!=typeof t)return(0,n.Z)("Setting a Configurable requires a model to set its initial properties");const r=Object.create(Object.getPrototypeOf(t),Object.getOwnPropertyDescriptors(t)),o=0===Object.keys(r).length?e:r;for(let a in o)if(void 0!==e[a])try{"object"==typeof e[a]&&"object"==typeof t[a]?r[a]=i(e[a],t[a]):r[a]=e[a]}catch(e){(0,n.Z)("An error occurred while setting a property of a Configurable",e)}return r}catch(e){(0,n.Z)("An error occured while setting a Configurable",e)}}},6818:(e,t,r)=>{r.d(t,{Re:()=>i,gF:()=>o,q4:()=>n});const n="1.236.0",i="PROD",o="CDN"},385:(e,t,r)=>{r.d(t,{FN:()=>a,IF:()=>u,Nk:()=>f,Tt:()=>s,_A:()=>o,il:()=>n,pL:()=>c,v6:()=>i,w1:()=>d});const n="undefined"!=typeof window&&!!window.document,i="undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self.navigator instanceof WorkerNavigator||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis.navigator instanceof WorkerNavigator),o=n?window:"undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis),a=""+o?.location,s=/iPad|iPhone|iPod/.test(navigator.userAgent),c=s&&"undefined"==typeof SharedWorker,u=(()=>{const e=navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);return Array.isArray(e)&&e.length>=2?+e[1]:0})(),d=Boolean(n&&window.document.documentMode),f=!!navigator.sendBeacon},1117:(e,t,r)=>{r.d(t,{w:()=>o});var n=r(50);const i={agentIdentifier:"",ee:void 0};class o{constructor(e){try{if("object"!=typeof e)return(0,n.Z)("shared context requires an object as input");this.sharedContext={},Object.assign(this.sharedContext,i),Object.entries(e).forEach((e=>{let[t,r]=e;Object.keys(i).includes(t)&&(this.sharedContext[t]=r)}))}catch(e){(0,n.Z)("An error occured while setting SharedContext",e)}}}},8e3:(e,t,r)=>{r.d(t,{L:()=>d,R:()=>c});var n=r(2177),i=r(1284),o=r(4322),a=r(3325);const s={};function c(e,t){const r={staged:!1,priority:a.p[t]||0};u(e),s[e].get(t)||s[e].set(t,r)}function u(e){e&&(s[e]||(s[e]=new Map))}function d(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"feature";if(u(e),!e||!s[e].get(t))return a(t);s[e].get(t).staged=!0;const r=[...s[e]];function a(t){const r=e?n.ee.get(e):n.ee,a=o.X.handlers;if(r.backlog&&a){var s=r.backlog[t],c=a[t];if(c){for(var u=0;s&&u<s.length;++u)f(s[u],c);(0,i.D)(c,(function(e,t){(0,i.D)(t,(function(t,r){r[0].on(e,r[1])}))}))}delete a[t],r.backlog[t]=null,r.emit("drain-"+t,[])}}r.every((e=>{let[t,r]=e;return r.staged}))&&(r.sort(((e,t)=>e[1].priority-t[1].priority)),r.forEach((e=>{let[t]=e;a(t)})))}function f(e,t){var r=e[1];(0,i.D)(t[r],(function(t,r){var n=e[0];if(r[0]===n){var i=r[1],o=e[3],a=e[2];i.apply(o,a)}}))}},2177:(e,t,r)=>{r.d(t,{c:()=>f,ee:()=>u});var n=r(8632),i=r(2210),o=r(1284),a=r(5763),s="nr@context";let c=(0,n.fP)();var u;function d(){}function f(e){return(0,i.X)(e,s,l)}function l(){return new d}function h(){u.aborted=!0,u.backlog={}}c.ee?u=c.ee:(u=function e(t,r){var n={},c={},f={},g=!1;try{g=16===r.length&&(0,a.OP)(r).isolatedBacklog}catch(e){}var p={on:b,addEventListener:b,removeEventListener:y,emit:v,get:x,listeners:w,context:m,buffer:A,abort:h,aborted:!1,isBuffering:E,debugId:r,backlog:g?{}:t&&"object"==typeof t.backlog?t.backlog:{}};return p;function m(e){return e&&e instanceof d?e:e?(0,i.X)(e,s,l):l()}function v(e,r,n,i,o){if(!1!==o&&(o=!0),!u.aborted||i){t&&o&&t.emit(e,r,n);for(var a=m(n),s=w(e),d=s.length,f=0;f<d;f++)s[f].apply(a,r);var l=T()[c[e]];return l&&l.push([p,e,r,a]),a}}function b(e,t){n[e]=w(e).concat(t)}function y(e,t){var r=n[e];if(r)for(var i=0;i<r.length;i++)r[i]===t&&r.splice(i,1)}function w(e){return n[e]||[]}function x(t){return f[t]=f[t]||e(p,t)}function A(e,t){var r=T();p.aborted||(0,o.D)(e,(function(e,n){t=t||"feature",c[n]=t,t in r||(r[t]=[])}))}function E(e){return!!T()[c[e]]}function T(){return p.backlog}}(void 0,"globalEE"),c.ee=u)},5546:(e,t,r)=>{r.d(t,{E:()=>n,p:()=>i});var n=r(2177).ee.get("handle");function i(e,t,r,i,o){o?(o.buffer([e],i),o.emit(e,t,r)):(n.buffer([e],i),n.emit(e,t,r))}},4322:(e,t,r)=>{r.d(t,{X:()=>o});var n=r(5546);o.on=a;var i=o.handlers={};function o(e,t,r,o){a(o||n.E,i,e,t,r)}function a(e,t,r,i,o){o||(o="feature"),e||(e=n.E);var a=t[o]=t[o]||{};(a[r]=a[r]||[]).push([e,i])}},3239:(e,t,r)=>{r.d(t,{bP:()=>s,iz:()=>c,m$:()=>a});var n=r(385);let i=!1,o=!1;try{const e={get passive(){return i=!0,!1},get signal(){return o=!0,!1}};n._A.addEventListener("test",null,e),n._A.removeEventListener("test",null,e)}catch(e){}function a(e,t){return i||o?{capture:!!e,passive:i,signal:t}:!!e}function s(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;window.addEventListener(e,t,a(r,n))}function c(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;document.addEventListener(e,t,a(r,n))}},4402:(e,t,r)=>{r.d(t,{Ht:()=>u,M:()=>c,Rl:()=>a,ky:()=>s});var n=r(385);const i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";function o(e,t){return e?15&e[t]:16*Math.random()|0}function a(){const e=n._A?.crypto||n._A?.msCrypto;let t,r=0;return e&&e.getRandomValues&&(t=e.getRandomValues(new Uint8Array(31))),i.split("").map((e=>"x"===e?o(t,++r).toString(16):"y"===e?(3&o()|8).toString(16):e)).join("")}function s(e){const t=n._A?.crypto||n._A?.msCrypto;let r,i=0;t&&t.getRandomValues&&(r=t.getRandomValues(new Uint8Array(31)));const a=[];for(var s=0;s<e;s++)a.push(o(r,++i).toString(16));return a.join("")}function c(){return s(16)}function u(){return s(32)}},7056:(e,t,r)=>{r.d(t,{Bq:()=>n,Hb:()=>o,oD:()=>i});const n="NRBA",i=144e5,o=18e5},7894:(e,t,r)=>{function n(){return Math.round(performance.now())}r.d(t,{z:()=>n})},7243:(e,t,r)=>{r.d(t,{e:()=>o});var n=r(385),i={};function o(e){if(e in i)return i[e];if(0===(e||"").indexOf("data:"))return{protocol:"data"};let t;var r=n._A?.location,o={};if(n.il)t=document.createElement("a"),t.href=e;else try{t=new URL(e,r.href)}catch(e){return o}o.port=t.port;var a=t.href.split("://");!o.port&&a[1]&&(o.port=a[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===a[0]?"443":"80"),o.hostname=t.hostname||r.hostname,o.pathname=t.pathname,o.protocol=a[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var s=!t.protocol||":"===t.protocol||t.protocol===r.protocol,c=t.hostname===r.hostname&&t.port===r.port;return o.sameOrigin=s&&(!t.hostname||c),"/"===o.pathname&&(i[e]=o),o}},50:(e,t,r)=>{function n(e,t){"function"==typeof console.warn&&(console.warn("New Relic: ".concat(e)),t&&console.warn(t))}r.d(t,{Z:()=>n})},2587:(e,t,r)=>{r.d(t,{N:()=>c,T:()=>u});var n=r(2177),i=r(5546),o=r(8e3),a=r(3325);const s={stn:[a.D.sessionTrace],err:[a.D.jserrors,a.D.metrics],ins:[a.D.pageAction],spa:[a.D.spa],sr:[a.D.sessionReplay,a.D.sessionTrace]};function c(e,t){const r=n.ee.get(t);e&&"object"==typeof e&&(Object.entries(e).forEach((e=>{let[t,n]=e;void 0===u[t]&&(s[t]?s[t].forEach((e=>{n?(0,i.p)("feat-"+t,[],void 0,e,r):(0,i.p)("block-"+t,[],void 0,e,r),(0,i.p)("rumresp-"+t,[Boolean(n)],void 0,e,r)})):n&&(0,i.p)("feat-"+t,[],void 0,void 0,r),u[t]=Boolean(n))})),Object.keys(s).forEach((e=>{void 0===u[e]&&(s[e]?.forEach((t=>(0,i.p)("rumresp-"+e,[!1],void 0,t,r))),u[e]=!1)})),(0,o.L)(t,a.D.pageViewEvent))}const u={}},2210:(e,t,r)=>{r.d(t,{X:()=>i});var n=Object.prototype.hasOwnProperty;function i(e,t,r){if(n.call(e,t))return e[t];var i=r();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,t,{value:i,writable:!0,enumerable:!1}),i}catch(e){}return e[t]=i,i}},1284:(e,t,r)=>{r.d(t,{D:()=>n});const n=(e,t)=>Object.entries(e||{}).map((e=>{let[r,n]=e;return t(r,n)}))},4351:(e,t,r)=>{r.d(t,{P:()=>o});var n=r(2177);const i=()=>{const e=new WeakSet;return(t,r)=>{if("object"==typeof r&&null!==r){if(e.has(r))return;e.add(r)}return r}};function o(e){try{return JSON.stringify(e,i())}catch(e){try{n.ee.emit("internal-error",[e])}catch(e){}}}},3960:(e,t,r)=>{r.d(t,{K:()=>a,b:()=>o});var n=r(3239);function i(){return"undefined"==typeof document||"complete"===document.readyState}function o(e,t){if(i())return e();(0,n.bP)("load",e,t)}function a(e){if(i())return e();(0,n.iz)("DOMContentLoaded",e)}},8632:(e,t,r)=>{r.d(t,{EZ:()=>u,Qy:()=>c,ce:()=>o,fP:()=>a,gG:()=>d,mF:()=>s});var n=r(7894),i=r(385);const o={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net"};function a(){return i._A.NREUM||(i._A.NREUM={}),void 0===i._A.newrelic&&(i._A.newrelic=i._A.NREUM),i._A.NREUM}function s(){let e=a();return e.o||(e.o={ST:i._A.setTimeout,SI:i._A.setImmediate,CT:i._A.clearTimeout,XHR:i._A.XMLHttpRequest,REQ:i._A.Request,EV:i._A.Event,PR:i._A.Promise,MO:i._A.MutationObserver,FETCH:i._A.fetch}),e}function c(e,t,r){let i=a();const o=i.initializedAgents||{},s=o[e]||{};return Object.keys(s).length||(s.initializedAt={ms:(0,n.z)(),date:new Date}),i.initializedAgents={...o,[e]:{...s,[r]:t}},i}function u(e,t){a()[e]=t}function d(){return function(){let e=a();const t=e.info||{};e.info={beacon:o.beacon,errorBeacon:o.errorBeacon,...t}}(),function(){let e=a();const t=e.init||{};e.init={...t}}(),s(),function(){let e=a();const t=e.loader_config||{};e.loader_config={...t}}(),a()}},7956:(e,t,r)=>{r.d(t,{N:()=>i});var n=r(3239);function i(e){let t=arguments.length>1&&void 0!==arguments[1]&&arguments[1],r=arguments.length>2?arguments[2]:void 0,i=arguments.length>3?arguments[3]:void 0;return void(0,n.iz)("visibilitychange",(function(){if(t)return void("hidden"==document.visibilityState&&e());e(document.visibilityState)}),r,i)}},1214:(e,t,r)=>{r.d(t,{em:()=>v,u5:()=>N,QU:()=>S,_L:()=>I,Gm:()=>L,Lg:()=>M,gy:()=>U,BV:()=>Q,Kf:()=>ee});var n=r(2177);const i="nr@original";var o=Object.prototype.hasOwnProperty,a=!1;function s(e,t){return e||(e=n.ee),r.inPlace=function(e,t,n,i,o){n||(n="");var a,s,c,u="-"===n.charAt(0);for(c=0;c<t.length;c++)d(a=e[s=t[c]])||(e[s]=r(a,u?s+n:n,i,s,o))},r.flag=i,r;function r(t,r,n,a,s){return d(t)?t:(r||(r=""),nrWrapper[i]=t,u(t,nrWrapper,e),nrWrapper);function nrWrapper(){var i,u,d,f;try{u=this,i=[...arguments],d="function"==typeof n?n(i,u):n||{}}catch(t){c([t,"",[i,u,a],d],e)}o(r+"start",[i,u,a],d,s);try{return f=t.apply(u,i)}catch(e){throw o(r+"err",[i,u,e],d,s),e}finally{o(r+"end",[i,u,f],d,s)}}}function o(r,n,i,o){if(!a||t){var s=a;a=!0;try{e.emit(r,n,i,t,o)}catch(t){c([t,r,n,i],e)}a=s}}}function c(e,t){t||(t=n.ee);try{t.emit("internal-error",e)}catch(e){}}function u(e,t,r){if(Object.defineProperty&&Object.keys)try{return Object.keys(e).forEach((function(r){Object.defineProperty(t,r,{get:function(){return e[r]},set:function(t){return e[r]=t,t}})})),t}catch(e){c([e],r)}for(var n in e)o.call(e,n)&&(t[n]=e[n]);return t}function d(e){return!(e&&e instanceof Function&&e.apply&&!e[i])}var f=r(2210),l=r(385);const h={},g=XMLHttpRequest,p="addEventListener",m="removeEventListener";function v(e){var t=function(e){return(e||n.ee).get("events")}(e);if(h[t.debugId]++)return t;h[t.debugId]=1;var r=s(t,!0);function i(e){r.inPlace(e,[p,m],"-",o)}function o(e,t){return e[1]}return"getPrototypeOf"in Object&&(l.il&&b(document,i),b(l._A,i),b(g.prototype,i)),t.on(p+"-start",(function(e,t){var n=e[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var i=(0,f.X)(n,"nr@wrapped",(function(){var e={object:function(){if("function"!=typeof n.handleEvent)return;return n.handleEvent.apply(n,arguments)},function:n}[typeof n];return e?r(e,"fn-",null,e.name||"anonymous"):n}));this.wrapped=e[1]=i}})),t.on(m+"-start",(function(e){e[1]=this.wrapped||e[1]})),t}function b(e,t){let r=e;for(;"object"==typeof r&&!Object.prototype.hasOwnProperty.call(r,p);)r=Object.getPrototypeOf(r);for(var n=arguments.length,i=new Array(n>2?n-2:0),o=2;o<n;o++)i[o-2]=arguments[o];r&&t(r,...i)}var y="fetch-",w=y+"body-",x=["arrayBuffer","blob","json","text","formData"],A=l._A.Request,E=l._A.Response,T="prototype",_="nr@context";const D={};function N(e){const t=function(e){return(e||n.ee).get("fetch")}(e);if(!(A&&E&&l._A.fetch))return t;if(D[t.debugId]++)return t;function r(e,r,n){var i=e[r];"function"==typeof i&&(e[r]=function(){var e,r=[...arguments],o={};t.emit(n+"before-start",[r],o),o[_]&&o[_].dt&&(e=o[_].dt);var a=i.apply(this,r);return t.emit(n+"start",[r,e],a),a.then((function(e){return t.emit(n+"end",[null,e],a),e}),(function(e){throw t.emit(n+"end",[e],a),e}))})}return D[t.debugId]=1,x.forEach((e=>{r(A[T],e,w),r(E[T],e,w)})),r(l._A,"fetch",y),t.on(y+"end",(function(e,r){var n=this;if(r){var i=r.headers.get("content-length");null!==i&&(n.rxSize=i),t.emit(y+"done",[null,r],n)}else t.emit(y+"done",[e],n)})),t}const O={},j=["pushState","replaceState"];function S(e){const t=function(e){return(e||n.ee).get("history")}(e);return!l.il||O[t.debugId]++||(O[t.debugId]=1,s(t).inPlace(window.history,j,"-")),t}var P=r(3239);const C={},R=["appendChild","insertBefore","replaceChild"];function I(e){const t=function(e){return(e||n.ee).get("jsonp")}(e);if(!l.il||C[t.debugId])return t;C[t.debugId]=!0;var r=s(t),i=/[?&](?:callback|cb)=([^&#]+)/,o=/(.*)\.([^.]+)/,a=/^(\w+)(\.|$)(.*)$/;function c(e,t){var r=e.match(a),n=r[1],i=r[3];return i?c(i,t[n]):t[n]}return r.inPlace(Node.prototype,R,"dom-"),t.on("dom-start",(function(e){!function(e){if(!e||"string"!=typeof e.nodeName||"script"!==e.nodeName.toLowerCase())return;if("function"!=typeof e.addEventListener)return;var n=(a=e.src,s=a.match(i),s?s[1]:null);var a,s;if(!n)return;var u=function(e){var t=e.match(o);if(t&&t.length>=3)return{key:t[2],parent:c(t[1],window)};return{key:e,parent:window}}(n);if("function"!=typeof u.parent[u.key])return;var d={};function f(){t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}function l(){t.emit("jsonp-error",[],d),t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}r.inPlace(u.parent,[u.key],"cb-",d),e.addEventListener("load",f,(0,P.m$)(!1)),e.addEventListener("error",l,(0,P.m$)(!1)),t.emit("new-jsonp",[e.src],d)}(e[0])})),t}var k=r(5763);const H={};function L(e){const t=function(e){return(e||n.ee).get("mutation")}(e);if(!l.il||H[t.debugId])return t;H[t.debugId]=!0;var r=s(t),i=k.Yu.MO;return i&&(window.MutationObserver=function(e){return this instanceof i?new i(r(e,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype),t}const z={};function M(e){const t=function(e){return(e||n.ee).get("promise")}(e);if(z[t.debugId])return t;z[t.debugId]=!0;var r=n.c,o=s(t),a=k.Yu.PR;return a&&function(){function e(r){var n=t.context(),i=o(r,"executor-",n,null,!1);const s=Reflect.construct(a,[i],e);return t.context(s).getCtx=function(){return n},s}l._A.Promise=e,Object.defineProperty(e,"name",{value:"Promise"}),e.toString=function(){return a.toString()},Object.setPrototypeOf(e,a),["all","race"].forEach((function(r){const n=a[r];e[r]=function(e){let i=!1;[...e||[]].forEach((e=>{this.resolve(e).then(a("all"===r),a(!1))}));const o=n.apply(this,arguments);return o;function a(e){return function(){t.emit("propagate",[null,!i],o,!1,!1),i=i||!e}}}})),["resolve","reject"].forEach((function(r){const n=a[r];e[r]=function(e){const r=n.apply(this,arguments);return e!==r&&t.emit("propagate",[e,!0],r,!1,!1),r}})),e.prototype=a.prototype;const n=a.prototype.then;a.prototype.then=function(){var e=this,i=r(e);i.promise=e;for(var a=arguments.length,s=new Array(a),c=0;c<a;c++)s[c]=arguments[c];s[0]=o(s[0],"cb-",i,null,!1),s[1]=o(s[1],"cb-",i,null,!1);const u=n.apply(this,s);return i.nextPromise=u,t.emit("propagate",[e,!0],u,!1,!1),u},a.prototype.then[i]=n,t.on("executor-start",(function(e){e[0]=o(e[0],"resolve-",this,null,!1),e[1]=o(e[1],"resolve-",this,null,!1)})),t.on("executor-err",(function(e,t,r){e[1](r)})),t.on("cb-end",(function(e,r,n){t.emit("propagate",[n,!0],this.nextPromise,!1,!1)})),t.on("propagate",(function(e,r,n){this.getCtx&&!r||(this.getCtx=function(){if(e instanceof Promise)var r=t.context(e);return r&&r.getCtx?r.getCtx():this})}))}(),t}const B={},F="requestAnimationFrame";function U(e){const t=function(e){return(e||n.ee).get("raf")}(e);if(!l.il||B[t.debugId]++)return t;B[t.debugId]=1;var r=s(t);return r.inPlace(window,[F],"raf-"),t.on("raf-start",(function(e){e[0]=r(e[0],"fn-")})),t}const q={},G="setTimeout",V="setInterval",X="clearTimeout",W="-start",Z="-",$=[G,"setImmediate",V,X,"clearImmediate"];function Q(e){const t=function(e){return(e||n.ee).get("timer")}(e);if(q[t.debugId]++)return t;q[t.debugId]=1;var r=s(t);return r.inPlace(l._A,$.slice(0,2),G+Z),r.inPlace(l._A,$.slice(2,3),V+Z),r.inPlace(l._A,$.slice(3),X+Z),t.on(V+W,(function(e,t,n){e[0]=r(e[0],"fn-",null,n)})),t.on(G+W,(function(e,t,n){this.method=n,this.timerDuration=isNaN(e[1])?0:+e[1],e[0]=r(e[0],"fn-",this,n)})),t}var Y=r(50);const K={},J=["open","send"];function ee(e){var t=e||n.ee;const r=function(e){return(e||n.ee).get("xhr")}(t);if(K[r.debugId]++)return r;K[r.debugId]=1,v(t);var i=s(r),o=k.Yu.XHR,a=k.Yu.MO,c=k.Yu.PR,u=k.Yu.SI,d="readystatechange",f=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],h=[],g=l._A.XMLHttpRequest.listeners,p=l._A.XMLHttpRequest=function(e){var t=new o(e);function n(){try{r.emit("new-xhr",[t],t),t.addEventListener(d,b,(0,P.m$)(!1))}catch(e){(0,Y.Z)("An error occured while intercepting XHR",e);try{r.emit("internal-error",[e])}catch(e){}}}return this.listeners=g?[...g,n]:[n],this.listeners.forEach((e=>e())),t};function m(e,t){i.inPlace(t,["onreadystatechange"],"fn-",E)}function b(){var e=this,t=r.context(e);e.readyState>3&&!t.resolved&&(t.resolved=!0,r.emit("xhr-resolved",[],e)),i.inPlace(e,f,"fn-",E)}if(function(e,t){for(var r in e)t[r]=e[r]}(o,p),p.prototype=o.prototype,i.inPlace(p.prototype,J,"-xhr-",E),r.on("send-xhr-start",(function(e,t){m(e,t),function(e){h.push(e),a&&(y?y.then(A):u?u(A):(w=-w,x.data=w))}(t)})),r.on("open-xhr-start",m),a){var y=c&&c.resolve();if(!u&&!c){var w=1,x=document.createTextNode(w);new a(A).observe(x,{characterData:!0})}}else t.on("fn-end",(function(e){e[0]&&e[0].type===d||A()}));function A(){for(var e=0;e<h.length;e++)m(0,h[e]);h.length&&(h=[])}function E(e,t){return t}return r}},7825:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.ajax},6660:(e,t,r)=>{r.d(t,{A:()=>i,t:()=>n});const n=r(3325).D.jserrors,i="nr@seenError"},3081:(e,t,r)=>{r.d(t,{gF:()=>o,mY:()=>i,t9:()=>n,vz:()=>s,xS:()=>a});const n=r(3325).D.metrics,i="sm",o="cm",a="storeSupportabilityMetrics",s="storeEventMetrics"},4649:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageAction},7633:(e,t,r)=>{r.d(t,{Dz:()=>i,OJ:()=>a,qw:()=>o,t9:()=>n});const n=r(3325).D.pageViewEvent,i="firstbyte",o="domcontent",a="windowload"},9251:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageViewTiming},3614:(e,t,r)=>{r.d(t,{BST_RESOURCE:()=>i,END:()=>s,FEATURE_NAME:()=>n,FN_END:()=>u,FN_START:()=>c,PUSH_STATE:()=>d,RESOURCE:()=>o,START:()=>a});const n=r(3325).D.sessionTrace,i="bstResource",o="resource",a="-start",s="-end",c="fn"+a,u="fn"+s,d="pushState"},7836:(e,t,r)=>{r.d(t,{BODY:()=>A,CB_END:()=>E,CB_START:()=>u,END:()=>x,FEATURE_NAME:()=>i,FETCH:()=>_,FETCH_BODY:()=>v,FETCH_DONE:()=>m,FETCH_START:()=>p,FN_END:()=>c,FN_START:()=>s,INTERACTION:()=>l,INTERACTION_API:()=>d,INTERACTION_EVENTS:()=>o,JSONP_END:()=>b,JSONP_NODE:()=>g,JS_TIME:()=>T,MAX_TIMER_BUDGET:()=>a,REMAINING:()=>f,SPA_NODE:()=>h,START:()=>w,originalSetTimeout:()=>y});var n=r(5763);const i=r(3325).D.spa,o=["click","submit","keypress","keydown","keyup","change"],a=999,s="fn-start",c="fn-end",u="cb-start",d="api-ixn-",f="remaining",l="interaction",h="spaNode",g="jsonpNode",p="fetch-start",m="fetch-done",v="fetch-body-",b="jsonp-end",y=n.Yu.ST,w="-start",x="-end",A="-body",E="cb"+x,T="jsTime",_="fetch"},5938:(e,t,r)=>{r.d(t,{W:()=>o});var n=r(5763),i=r(2177);class o{constructor(e,t,r){this.agentIdentifier=e,this.aggregator=t,this.ee=i.ee.get(e,(0,n.OP)(this.agentIdentifier).isolatedBacklog),this.featureName=r,this.blocked=!1}}},9144:(e,t,r)=>{r.d(t,{j:()=>m});var n=r(3325),i=r(5763),o=r(5546),a=r(2177),s=r(7894),c=r(8e3),u=r(3960),d=r(385),f=r(50),l=r(3081),h=r(8632);function g(){const e=(0,h.gG)();["setErrorHandler","finished","addToTrace","inlineHit","addRelease","addPageAction","setCurrentRouteName","setPageViewName","setCustomAttribute","interaction","noticeError","setUserId"].forEach((t=>{e[t]=function(){for(var r=arguments.length,n=new Array(r),i=0;i<r;i++)n[i]=arguments[i];return function(t){for(var r=arguments.length,n=new Array(r>1?r-1:0),i=1;i<r;i++)n[i-1]=arguments[i];let o=[];return Object.values(e.initializedAgents).forEach((e=>{e.exposed&&e.api[t]&&o.push(e.api[t](...n))})),o.length>1?o:o[0]}(t,...n)}}))}var p=r(2587);function m(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},m=arguments.length>2?arguments[2]:void 0,v=arguments.length>3?arguments[3]:void 0,{init:b,info:y,loader_config:w,runtime:x={loaderType:m},exposed:A=!0}=t;const E=(0,h.gG)();y||(b=E.init,y=E.info,w=E.loader_config),(0,i.Dg)(e,b||{}),(0,i.GE)(e,w||{}),(0,i.sU)(e,x),y.jsAttributes??={},d.v6&&(y.jsAttributes.isWorker=!0),(0,i.CX)(e,y),g();const T=function(e,t){t||(0,c.R)(e,"api");const h={};var g=a.ee.get(e),p=g.get("tracer"),m="api-",v=m+"ixn-";function b(t,r,n,o){const a=(0,i.C5)(e);return null===r?delete a.jsAttributes[t]:(0,i.CX)(e,{...a,jsAttributes:{...a.jsAttributes,[t]:r}}),x(m,n,!0,o||null===r?"session":void 0)(t,r)}function y(){}["setErrorHandler","finished","addToTrace","inlineHit","addRelease"].forEach((e=>h[e]=x(m,e,!0,"api"))),h.addPageAction=x(m,"addPageAction",!0,n.D.pageAction),h.setCurrentRouteName=x(m,"routeName",!0,n.D.spa),h.setPageViewName=function(t,r){if("string"==typeof t)return"/"!==t.charAt(0)&&(t="/"+t),(0,i.OP)(e).customTransaction=(r||"http://custom.transaction")+t,x(m,"setPageViewName",!0)()},h.setCustomAttribute=function(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2];if("string"==typeof e){if(["string","number"].includes(typeof t)||null===t)return b(e,t,"setCustomAttribute",r);(0,f.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t,"> was provided."))}else(0,f.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.setUserId=function(e){if("string"==typeof e||null===e)return b("enduser.id",e,"setUserId",!0);(0,f.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.interaction=function(){return(new y).get()};var w=y.prototype={createTracer:function(e,t){var r={},i=this,a="function"==typeof t;return(0,o.p)(v+"tracer",[(0,s.z)(),e,r],i,n.D.spa,g),function(){if(p.emit((a?"":"no-")+"fn-start",[(0,s.z)(),i,a],r),a)try{return t.apply(this,arguments)}catch(e){throw p.emit("fn-err",[arguments,this,"string"==typeof e?new Error(e):e],r),e}finally{p.emit("fn-end",[(0,s.z)()],r)}}}};function x(e,t,r,i){return function(){return(0,o.p)(l.xS,["API/"+t+"/called"],void 0,n.D.metrics,g),i&&(0,o.p)(e+t,[(0,s.z)(),...arguments],r?null:this,i,g),r?void 0:this}}function A(){r.e(439).then(r.bind(r,7438)).then((t=>{let{setAPI:r}=t;r(e),(0,c.L)(e,"api")})).catch((()=>(0,f.Z)("Downloading runtime APIs failed...")))}return["actionText","setName","setAttribute","save","ignore","onEnd","getContext","end","get"].forEach((e=>{w[e]=x(v,e,void 0,n.D.spa)})),h.noticeError=function(e,t){"string"==typeof e&&(e=new Error(e)),(0,o.p)(l.xS,["API/noticeError/called"],void 0,n.D.metrics,g),(0,o.p)("err",[e,(0,s.z)(),!1,t],void 0,n.D.jserrors,g)},d.il?(0,u.b)((()=>A()),!0):A(),h}(e,v);return(0,h.Qy)(e,T,"api"),(0,h.Qy)(e,A,"exposed"),(0,h.EZ)("activatedFeatures",p.T),T}},3325:(e,t,r)=>{r.d(t,{D:()=>n,p:()=>i});const n={ajax:"ajax",jserrors:"jserrors",metrics:"metrics",pageAction:"page_action",pageViewEvent:"page_view_event",pageViewTiming:"page_view_timing",sessionReplay:"session_replay",sessionTrace:"session_trace",spa:"spa"},i={[n.pageViewEvent]:1,[n.pageViewTiming]:2,[n.metrics]:3,[n.jserrors]:4,[n.ajax]:5,[n.sessionTrace]:6,[n.pageAction]:7,[n.spa]:8,[n.sessionReplay]:9}}},n={};function i(e){var t=n[e];if(void 0!==t)return t.exports;var o=n[e]={exports:{}};return r[e](o,o.exports,i),o.exports}i.m=r,i.d=(e,t)=>{for(var r in t)i.o(t,r)&&!i.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},i.f={},i.e=e=>Promise.all(Object.keys(i.f).reduce(((t,r)=>(i.f[r](e,t),t)),[])),i.u=e=>(({78:"page_action-aggregate",147:"metrics-aggregate",242:"session-manager",317:"jserrors-aggregate",348:"page_view_timing-aggregate",412:"lazy-feature-loader",439:"async-api",538:"recorder",590:"session_replay-aggregate",675:"compressor",733:"session_trace-aggregate",786:"page_view_event-aggregate",873:"spa-aggregate",898:"ajax-aggregate"}[e]||e)+"."+{78:"ac76d497",147:"3dc53903",148:"1a20d5fe",242:"2a64278a",317:"49e41428",348:"bd6de33a",412:"2f55ce66",439:"30bd804e",538:"1b18459f",590:"cf0efb30",675:"ae9f91a8",733:"83105561",786:"06482edd",860:"03a8b7a5",873:"e6b09d52",898:"998ef92b"}[e]+"-1.236.0.min.js"),i.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),e={},t="NRBA:",i.l=(r,n,o,a)=>{if(e[r])e[r].push(n);else{var s,c;if(void 0!==o)for(var u=document.getElementsByTagName("script"),d=0;d<u.length;d++){var f=u[d];if(f.getAttribute("src")==r||f.getAttribute("data-webpack")==t+o){s=f;break}}s||(c=!0,(s=document.createElement("script")).charset="utf-8",s.timeout=120,i.nc&&s.setAttribute("nonce",i.nc),s.setAttribute("data-webpack",t+o),s.src=r),e[r]=[n];var l=(t,n)=>{s.onerror=s.onload=null,clearTimeout(h);var i=e[r];if(delete e[r],s.parentNode&&s.parentNode.removeChild(s),i&&i.forEach((e=>e(n))),t)return t(n)},h=setTimeout(l.bind(null,void 0,{type:"timeout",target:s}),12e4);s.onerror=l.bind(null,s.onerror),s.onload=l.bind(null,s.onload),c&&document.head.appendChild(s)}},i.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},i.j=364,i.p="https://js-agent.newrelic.com/",(()=>{var e={364:0,953:0};i.f.j=(t,r)=>{var n=i.o(e,t)?e[t]:void 0;if(0!==n)if(n)r.push(n[2]);else{var o=new Promise(((r,i)=>n=e[t]=[r,i]));r.push(n[2]=o);var a=i.p+i.u(t),s=new Error;i.l(a,(r=>{if(i.o(e,t)&&(0!==(n=e[t])&&(e[t]=void 0),n)){var o=r&&("load"===r.type?"missing":r.type),a=r&&r.target&&r.target.src;s.message="Loading chunk "+t+" failed.\n("+o+": "+a+")",s.name="ChunkLoadError",s.type=o,s.request=a,n[1](s)}}),"chunk-"+t,t)}};var t=(t,r)=>{var n,o,[a,s,c]=r,u=0;if(a.some((t=>0!==e[t]))){for(n in s)i.o(s,n)&&(i.m[n]=s[n]);if(c)c(i)}for(t&&t(r);u<a.length;u++)o=a[u],i.o(e,o)&&e[o]&&e[o][0](),e[o]=0},r=window.webpackChunkNRBA=window.webpackChunkNRBA||[];r.forEach(t.bind(null,0)),r.push=t.bind(null,r.push.bind(r))})();var o={};(()=>{i.r(o);var e=i(3325),t=i(5763);const r=Object.values(e.D);function n(e){const n={};return r.forEach((r=>{n[r]=function(e,r){return!1!==(0,t.Mt)(r,"".concat(e,".enabled"))}(r,e)})),n}var a=i(9144);var s=i(5546),c=i(385),u=i(8e3),d=i(5938),f=i(3960),l=i(50);class h extends d.W{constructor(e,t,r){let n=!(arguments.length>3&&void 0!==arguments[3])||arguments[3];super(e,t,r),this.auto=n,this.abortHandler,this.featAggregate,this.onAggregateImported,n&&(0,u.R)(e,r)}importAggregator(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};if(this.featAggregate||!this.auto)return;const r=c.il&&!0===(0,t.Mt)(this.agentIdentifier,"privacy.cookies_enabled");let n;this.onAggregateImported=new Promise((e=>{n=e}));const o=async()=>{let t;try{if(r){const{setupAgentSession:e}=await Promise.all([i.e(860),i.e(242)]).then(i.bind(i,3228));t=e(this.agentIdentifier)}}catch(e){(0,l.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.",e)}try{if(!this.shouldImportAgg(this.featureName,t))return void(0,u.L)(this.agentIdentifier,this.featureName);const{lazyFeatureLoader:r}=await i.e(412).then(i.bind(i,8582)),{Aggregate:o}=await r(this.featureName,"aggregate");this.featAggregate=new o(this.agentIdentifier,this.aggregator,e),n(!0)}catch(e){(0,l.Z)("Downloading and initializing ".concat(this.featureName," failed..."),e),this.abortHandler?.(),n(!1)}};c.il?(0,f.b)((()=>o()),!0):o()}shouldImportAgg(r,n){return r!==e.D.sessionReplay||!1!==(0,t.Mt)(this.agentIdentifier,"session_trace.enabled")&&(!!n?.isNew||!!n?.state.sessionReplay)}}var g=i(7633),p=i(7894);class m extends h{static featureName=g.t9;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];if(super(r,n,g.t9,i),("undefined"==typeof PerformanceNavigationTiming||c.Tt)&&"undefined"!=typeof PerformanceTiming){const n=(0,t.OP)(r);n[g.Dz]=Math.max(Date.now()-n.offset,0),(0,f.K)((()=>n[g.qw]=Math.max((0,p.z)()-n[g.Dz],0))),(0,f.b)((()=>{const t=(0,p.z)();n[g.OJ]=Math.max(t-n[g.Dz],0),(0,s.p)("timing",["load",t],void 0,e.D.pageViewTiming,this.ee)}))}this.importAggregator()}}var v=i(1117),b=i(1284);class y extends v.w{constructor(e){super(e),this.aggregatedData={}}store(e,t,r,n,i){var o=this.getBucket(e,t,r,i);return o.metrics=function(e,t){t||(t={count:0});return t.count+=1,(0,b.D)(e,(function(e,r){t[e]=w(r,t[e])})),t}(n,o.metrics),o}merge(e,t,r,n,i){var o=this.getBucket(e,t,n,i);if(o.metrics){var a=o.metrics;a.count+=r.count,(0,b.D)(r,(function(e,t){if("count"!==e){var n=a[e],i=r[e];i&&!i.c?a[e]=w(i.t,n):a[e]=function(e,t){if(!t)return e;t.c||(t=x(t.t));return t.min=Math.min(e.min,t.min),t.max=Math.max(e.max,t.max),t.t+=e.t,t.sos+=e.sos,t.c+=e.c,t}(i,a[e])}}))}else o.metrics=r}storeMetric(e,t,r,n){var i=this.getBucket(e,t,r);return i.stats=w(n,i.stats),i}getBucket(e,t,r,n){this.aggregatedData[e]||(this.aggregatedData[e]={});var i=this.aggregatedData[e][t];return i||(i=this.aggregatedData[e][t]={params:r||{}},n&&(i.custom=n)),i}get(e,t){return t?this.aggregatedData[e]&&this.aggregatedData[e][t]:this.aggregatedData[e]}take(e){for(var t={},r="",n=!1,i=0;i<e.length;i++)t[r=e[i]]=A(this.aggregatedData[r]),t[r].length&&(n=!0),delete this.aggregatedData[r];return n?t:null}}function w(e,t){return null==e?function(e){e?e.c++:e={c:1};return e}(t):t?(t.c||(t=x(t.t)),t.c+=1,t.t+=e,t.sos+=e*e,e>t.max&&(t.max=e),e<t.min&&(t.min=e),t):{t:e}}function x(e){return{t:e,min:e,max:e,sos:e*e,c:1}}function A(e){return"object"!=typeof e?[]:(0,b.D)(e,E)}function E(e,t){return t}var T=i(8632),_=i(4402),D=i(4351);var N=i(7956),O=i(3239),j=i(9251);class S extends h{static featureName=j.t;constructor(e,r){let n=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,r,j.t,n),c.il&&((0,t.OP)(e).initHidden=Boolean("hidden"===document.visibilityState),(0,N.N)((()=>(0,s.p)("docHidden",[(0,p.z)()],void 0,j.t,this.ee)),!0),(0,O.bP)("pagehide",(()=>(0,s.p)("winPagehide",[(0,p.z)()],void 0,j.t,this.ee))),this.importAggregator())}}var P=i(3081);class C extends h{static featureName=P.t9;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,P.t9,r),this.importAggregator()}}var R,I=i(2210),k=i(1214),H=i(2177),L={};try{R=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(L.console=!0,-1!==R.indexOf("dev")&&(L.dev=!0),-1!==R.indexOf("nr_dev")&&(L.nrDev=!0))}catch(e){}function z(e){try{L.console&&z(e)}catch(e){}}L.nrDev&&H.ee.on("internal-error",(function(e){z(e.stack)})),L.dev&&H.ee.on("fn-err",(function(e,t,r){z(r.stack)})),L.dev&&(z("NR AGENT IN DEVELOPMENT MODE"),z("flags: "+(0,b.D)(L,(function(e,t){return e})).join(", ")));var M=i(6660);class B extends h{static featureName=M.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,M.t,i),this.skipNext=0;try{this.removeOnAbort=new AbortController}catch(e){}const o=this;o.ee.on("fn-start",(function(e,t,r){o.abortHandler&&(o.skipNext+=1)})),o.ee.on("fn-err",(function(t,r,n){o.abortHandler&&!n[M.A]&&((0,I.X)(n,M.A,(function(){return!0})),this.thrown=!0,(0,s.p)("err",[n,(0,p.z)()],void 0,e.D.jserrors,o.ee))})),o.ee.on("fn-end",(function(){o.abortHandler&&!this.thrown&&o.skipNext>0&&(o.skipNext-=1)})),o.ee.on("internal-error",(function(t){(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,o.ee)})),this.origOnerror=c._A.onerror,c._A.onerror=this.onerrorHandler.bind(this),c._A.addEventListener("unhandledrejection",(t=>{const r=function(e){let t="Unhandled Promise Rejection: ";if(e instanceof Error)try{return e.message=t+e.message,e}catch(t){return e}if(void 0===e)return new Error(t);try{return new Error(t+(0,D.P)(e))}catch(e){return new Error(t)}}(t.reason);(0,s.p)("err",[r,(0,p.z)(),!1,{unhandledPromiseRejection:1}],void 0,e.D.jserrors,this.ee)}),(0,O.m$)(!1,this.removeOnAbort?.signal)),(0,k.gy)(this.ee),(0,k.BV)(this.ee),(0,k.em)(this.ee),(0,t.OP)(r).xhrWrappable&&(0,k.Kf)(this.ee),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}onerrorHandler(t,r,n,i,o){"function"==typeof this.origOnerror&&this.origOnerror(...arguments);try{this.skipNext?this.skipNext-=1:(0,s.p)("err",[o||new F(t,r,n),(0,p.z)()],void 0,e.D.jserrors,this.ee)}catch(t){try{(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,this.ee)}catch(e){}}return!1}}function F(e,t,r){this.message=e||"Uncaught error with no additional information",this.sourceURL=t,this.line=r}let U=1;const q="nr@id";function G(e){const t=typeof e;return!e||"object"!==t&&"function"!==t?-1:e===c._A?0:(0,I.X)(e,q,(function(){return U++}))}function V(e){if("string"==typeof e&&e.length)return e.length;if("object"==typeof e){if("undefined"!=typeof ArrayBuffer&&e instanceof ArrayBuffer&&e.byteLength)return e.byteLength;if("undefined"!=typeof Blob&&e instanceof Blob&&e.size)return e.size;if(!("undefined"!=typeof FormData&&e instanceof FormData))try{return(0,D.P)(e).length}catch(e){return}}}var X=i(7243);class W{constructor(e){this.agentIdentifier=e,this.generateTracePayload=this.generateTracePayload.bind(this),this.shouldGenerateTrace=this.shouldGenerateTrace.bind(this)}generateTracePayload(e){if(!this.shouldGenerateTrace(e))return null;var r=(0,t.DL)(this.agentIdentifier);if(!r)return null;var n=(r.accountID||"").toString()||null,i=(r.agentID||"").toString()||null,o=(r.trustKey||"").toString()||null;if(!n||!i)return null;var a=(0,_.M)(),s=(0,_.Ht)(),c=Date.now(),u={spanId:a,traceId:s,timestamp:c};return(e.sameOrigin||this.isAllowedOrigin(e)&&this.useTraceContextHeadersForCors())&&(u.traceContextParentHeader=this.generateTraceContextParentHeader(a,s),u.traceContextStateHeader=this.generateTraceContextStateHeader(a,c,n,i,o)),(e.sameOrigin&&!this.excludeNewrelicHeader()||!e.sameOrigin&&this.isAllowedOrigin(e)&&this.useNewrelicHeaderForCors())&&(u.newrelicHeader=this.generateTraceHeader(a,s,c,n,i,o)),u}generateTraceContextParentHeader(e,t){return"00-"+t+"-"+e+"-01"}generateTraceContextStateHeader(e,t,r,n,i){return i+"@nr=0-1-"+r+"-"+n+"-"+e+"----"+t}generateTraceHeader(e,t,r,n,i,o){if(!("function"==typeof c._A?.btoa))return null;var a={v:[0,1],d:{ty:"Browser",ac:n,ap:i,id:e,tr:t,ti:r}};return o&&n!==o&&(a.d.tk=o),btoa((0,D.P)(a))}shouldGenerateTrace(e){return this.isDtEnabled()&&this.isAllowedOrigin(e)}isAllowedOrigin(e){var r=!1,n={};if((0,t.Mt)(this.agentIdentifier,"distributed_tracing")&&(n=(0,t.P_)(this.agentIdentifier).distributed_tracing),e.sameOrigin)r=!0;else if(n.allowed_origins instanceof Array)for(var i=0;i<n.allowed_origins.length;i++){var o=(0,X.e)(n.allowed_origins[i]);if(e.hostname===o.hostname&&e.protocol===o.protocol&&e.port===o.port){r=!0;break}}return r}isDtEnabled(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.enabled}excludeNewrelicHeader(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.exclude_newrelic_header}useNewrelicHeaderForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!1!==e.cors_use_newrelic_header}useTraceContextHeadersForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.cors_use_tracecontext_headers}}var Z=i(7825),$=["load","error","abort","timeout"],Q=$.length,Y=t.Yu.REQ,K=c._A.XMLHttpRequest;class J extends h{static featureName=Z.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,Z.t,i),(0,t.OP)(r).xhrWrappable&&(this.dt=new W(r),this.handler=(e,t,r,n)=>(0,s.p)(e,t,r,n,this.ee),(0,k.u5)(this.ee),(0,k.Kf)(this.ee),function(r,n,i,o){function a(e){var t=this;t.totalCbs=0,t.called=0,t.cbTime=0,t.end=E,t.ended=!1,t.xhrGuids={},t.lastSize=null,t.loadCaptureCalled=!1,t.params=this.params||{},t.metrics=this.metrics||{},e.addEventListener("load",(function(r){_(t,e)}),(0,O.m$)(!1)),c.IF||e.addEventListener("progress",(function(e){t.lastSize=e.loaded}),(0,O.m$)(!1))}function s(e){this.params={method:e[0]},T(this,e[1]),this.metrics={}}function u(e,n){var i=(0,t.DL)(r);i.xpid&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",i.xpid);var a=o.generateTracePayload(this.parsedOrigin);if(a){var s=!1;a.newrelicHeader&&(n.setRequestHeader("newrelic",a.newrelicHeader),s=!0),a.traceContextParentHeader&&(n.setRequestHeader("traceparent",a.traceContextParentHeader),a.traceContextStateHeader&&n.setRequestHeader("tracestate",a.traceContextStateHeader),s=!0),s&&(this.dt=a)}}function d(e,t){var r=this.metrics,i=e[0],o=this;if(r&&i){var a=V(i);a&&(r.txSize=a)}this.startTime=(0,p.z)(),this.listener=function(e){try{"abort"!==e.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==e.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof t.onload)&&"function"==typeof o.end)&&o.end(t)}catch(e){try{n.emit("internal-error",[e])}catch(e){}}};for(var s=0;s<Q;s++)t.addEventListener($[s],this.listener,(0,O.m$)(!1))}function f(e,t,r){this.cbTime+=e,t?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof r.onload||"function"!=typeof this.end||this.end(r)}function l(e,t){var r=""+G(e)+!!t;this.xhrGuids&&!this.xhrGuids[r]&&(this.xhrGuids[r]=!0,this.totalCbs+=1)}function h(e,t){var r=""+G(e)+!!t;this.xhrGuids&&this.xhrGuids[r]&&(delete this.xhrGuids[r],this.totalCbs-=1)}function g(){this.endTime=(0,p.z)()}function m(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-added",[e[1],e[2]],t)}function v(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-removed",[e[1],e[2]],t)}function b(e,t,r){t instanceof K&&("onload"===r&&(this.onload=!0),("load"===(e[0]&&e[0].type)||this.onload)&&(this.xhrCbStart=(0,p.z)()))}function y(e,t){this.xhrCbStart&&n.emit("xhr-cb-time",[(0,p.z)()-this.xhrCbStart,this.onload,t],t)}function w(e){var t,r=e[1]||{};"string"==typeof e[0]?t=e[0]:e[0]&&e[0].url?t=e[0].url:c._A?.URL&&e[0]&&e[0]instanceof URL&&(t=e[0].href),t&&(this.parsedOrigin=(0,X.e)(t),this.sameOrigin=this.parsedOrigin.sameOrigin);var n=o.generateTracePayload(this.parsedOrigin);if(n&&(n.newrelicHeader||n.traceContextParentHeader))if("string"==typeof e[0]||c._A?.URL&&e[0]&&e[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),s(i.headers,n)&&(this.dt=n),e.length>1?e[1]=i:e.push(i)}else e[0]&&e[0].headers&&s(e[0].headers,n)&&(this.dt=n);function s(e,t){var r=!1;return t.newrelicHeader&&(e.set("newrelic",t.newrelicHeader),r=!0),t.traceContextParentHeader&&(e.set("traceparent",t.traceContextParentHeader),t.traceContextStateHeader&&e.set("tracestate",t.traceContextStateHeader),r=!0),r}}function x(e,t){this.params={},this.metrics={},this.startTime=(0,p.z)(),this.dt=t,e.length>=1&&(this.target=e[0]),e.length>=2&&(this.opts=e[1]);var r,n=this.opts||{},i=this.target;"string"==typeof i?r=i:"object"==typeof i&&i instanceof Y?r=i.url:c._A?.URL&&"object"==typeof i&&i instanceof URL&&(r=i.href),T(this,r);var o=(""+(i&&i instanceof Y&&i.method||n.method||"GET")).toUpperCase();this.params.method=o,this.txSize=V(n.body)||0}function A(t,r){var n;this.endTime=(0,p.z)(),this.params||(this.params={}),this.params.status=r?r.status:0,"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var o={txSize:this.txSize,rxSize:n,duration:(0,p.z)()-this.startTime};i("xhr",[this.params,o,this.startTime,this.endTime,"fetch"],this,e.D.ajax)}function E(t){var r=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var o=0;o<Q;o++)t.removeEventListener($[o],this.listener,!1);r.aborted||(n.duration=(0,p.z)()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==r.status&&(r.status=0):_(this,t),n.cbTime=this.cbTime,i("xhr",[r,n,this.startTime,this.endTime,"xhr"],this,e.D.ajax))}}function T(e,t){var r=(0,X.e)(t),n=e.params;n.hostname=r.hostname,n.port=r.port,n.protocol=r.protocol,n.host=r.hostname+":"+r.port,n.pathname=r.pathname,e.parsedOrigin=r,e.sameOrigin=r.sameOrigin}function _(e,t){e.params.status=t.status;var r=function(e,t){var r=e.responseType;return"json"===r&&null!==t?t:"arraybuffer"===r||"blob"===r||"json"===r?V(e.response):"text"===r||""===r||void 0===r?V(e.responseText):void 0}(t,e.lastSize);if(r&&(e.metrics.rxSize=r),e.sameOrigin){var n=t.getResponseHeader("X-NewRelic-App-Data");n&&(e.params.cat=n.split(", ").pop())}e.loadCaptureCalled=!0}n.on("new-xhr",a),n.on("open-xhr-start",s),n.on("open-xhr-end",u),n.on("send-xhr-start",d),n.on("xhr-cb-time",f),n.on("xhr-load-added",l),n.on("xhr-load-removed",h),n.on("xhr-resolved",g),n.on("addEventListener-end",m),n.on("removeEventListener-end",v),n.on("fn-end",y),n.on("fetch-before-start",w),n.on("fetch-start",x),n.on("fn-start",b),n.on("fetch-done",A)}(r,this.ee,this.handler,this.dt),this.importAggregator())}}var ee=i(3614);const{BST_RESOURCE:te,RESOURCE:re,START:ne,END:ie,FEATURE_NAME:oe,FN_END:ae,FN_START:se,PUSH_STATE:ce}=ee;var ue=i(7836);const{FEATURE_NAME:de,START:fe,END:le,BODY:he,CB_END:ge,JS_TIME:pe,FETCH:me,FN_START:ve,CB_START:be,FN_END:ye}=ue;var we=i(4649);class xe extends h{static featureName=we.t;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,we.t,r),this.importAggregator()}}new class{constructor(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:(0,_.ky)(16);c._A?(this.agentIdentifier=t,this.sharedAggregator=new y({agentIdentifier:this.agentIdentifier}),this.features={},this.desiredFeatures=new Set(e.features||[]),this.desiredFeatures.add(m),Object.assign(this,(0,a.j)(this.agentIdentifier,e,e.loaderType||"agent")),this.start()):(0,l.Z)("Failed to initial the agent. Could not determine the runtime environment.")}get config(){return{info:(0,t.C5)(this.agentIdentifier),init:(0,t.P_)(this.agentIdentifier),loader_config:(0,t.DL)(this.agentIdentifier),runtime:(0,t.OP)(this.agentIdentifier)}}start(){const t="features";try{const r=n(this.agentIdentifier),i=[...this.desiredFeatures];i.sort(((t,r)=>e.p[t.featureName]-e.p[r.featureName])),i.forEach((t=>{if(r[t.featureName]||t.featureName===e.D.pageViewEvent){const n=function(t){switch(t){case e.D.ajax:return[e.D.jserrors];case e.D.sessionTrace:return[e.D.ajax,e.D.pageViewEvent];case e.D.sessionReplay:return[e.D.sessionTrace];case e.D.pageViewTiming:return[e.D.pageViewEvent];default:return[]}}(t.featureName);n.every((e=>r[e]))||(0,l.Z)("".concat(t.featureName," is enabled but one or more dependent features has been disabled (").concat((0,D.P)(n),"). This may cause unintended consequences or missing data...")),this.features[t.featureName]=new t(this.agentIdentifier,this.sharedAggregator)}})),(0,T.Qy)(this.agentIdentifier,this.features,t)}catch(e){(0,l.Z)("Failed to initialize all enabled instrument classes (agent aborted) -",e);for(const e in this.features)this.features[e].abortHandler?.();const r=(0,T.fP)();return delete r.initializedAgents[this.agentIdentifier]?.api,delete r.initializedAgents[this.agentIdentifier]?.[t],delete this.sharedAggregator,r.ee?.abort(),delete r.ee?.get(this.agentIdentifier),!1}}}({features:[J,m,S,class extends h{static featureName=oe;constructor(t,r){if(super(t,r,oe,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;const n=this.ee;let i;(0,k.QU)(n),this.eventsEE=(0,k.em)(n),this.eventsEE.on(se,(function(e,t){this.bstStart=(0,p.z)()})),this.eventsEE.on(ae,(function(t,r){(0,s.p)("bst",[t[0],r,this.bstStart,(0,p.z)()],void 0,e.D.sessionTrace,n)})),n.on(ce+ne,(function(e){this.time=(0,p.z)(),this.startPath=location.pathname+location.hash})),n.on(ce+ie,(function(t){(0,s.p)("bstHist",[location.pathname+location.hash,this.startPath,this.time],void 0,e.D.sessionTrace,n)}));try{i=new PerformanceObserver((t=>{const r=t.getEntries();(0,s.p)(te,[r],void 0,e.D.sessionTrace,n)})),i.observe({type:re,buffered:!0})}catch(e){}this.importAggregator({resourceObserver:i})}},C,xe,B,class extends h{static featureName=de;constructor(e,r){if(super(e,r,de,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;if(!(0,t.OP)(e).xhrWrappable)return;try{this.removeOnAbort=new AbortController}catch(e){}let n,i=0;const o=this.ee.get("tracer"),a=(0,k._L)(this.ee),s=(0,k.Lg)(this.ee),u=(0,k.BV)(this.ee),d=(0,k.Kf)(this.ee),f=this.ee.get("events"),l=(0,k.u5)(this.ee),h=(0,k.QU)(this.ee),g=(0,k.Gm)(this.ee);function m(e,t){h.emit("newURL",[""+window.location,t])}function v(){i++,n=window.location.hash,this[ve]=(0,p.z)()}function b(){i--,window.location.hash!==n&&m(0,!0);var e=(0,p.z)();this[pe]=~~this[pe]+e-this[ve],this[ye]=e}function y(e,t){e.on(t,(function(){this[t]=(0,p.z)()}))}this.ee.on(ve,v),s.on(be,v),a.on(be,v),this.ee.on(ye,b),s.on(ge,b),a.on(ge,b),this.ee.buffer([ve,ye,"xhr-resolved"],this.featureName),f.buffer([ve],this.featureName),u.buffer(["setTimeout"+le,"clearTimeout"+fe,ve],this.featureName),d.buffer([ve,"new-xhr","send-xhr"+fe],this.featureName),l.buffer([me+fe,me+"-done",me+he+fe,me+he+le],this.featureName),h.buffer(["newURL"],this.featureName),g.buffer([ve],this.featureName),s.buffer(["propagate",be,ge,"executor-err","resolve"+fe],this.featureName),o.buffer([ve,"no-"+ve],this.featureName),a.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"],this.featureName),y(l,me+fe),y(l,me+"-done"),y(a,"new-jsonp"),y(a,"jsonp-end"),y(a,"cb-start"),h.on("pushState-end",m),h.on("replaceState-end",m),window.addEventListener("hashchange",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("load",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("popstate",(function(){m(0,i>1)}),(0,O.m$)(!0,this.removeOnAbort?.signal)),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}}],loaderType:"spa"})})(),window.NRBA=o})();</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Varshika Singh"/><meta name="DC.Creator" content="Santanu Patra"/><meta name="DC.Creator" content="Natarajan Arul Murugan"/><meta name="DC.Creator" content="Dana-Cristina Toncu"/><meta name="DC.Creator" content="Ashutosh Tiwari"/><meta name="DC.title" content="Recent trends in computational tools and data-driven modeling for advanced materials "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2022/05/23"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D2MA00067A"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Recent trends in computational tools and data-driven modeling for advanced materials "/><meta name="citation_author" content="Varshika Singh"/><meta name="citation_author" content="Santanu Patra"/><meta name="citation_author" content="Natarajan Arul Murugan"/><meta name="citation_author" content="Dana-Cristina Toncu"/><meta name="citation_author" content="Ashutosh Tiwari"/><meta name="citation_online_date" content="2022/03/25"/><meta name="citation_date" content="2022"/><meta name="citation_journal_title" content="Materials Advances"/><meta name="citation_volume" content="3"/><meta name="citation_issue" content="10"/><meta name="citation_firstpage" content="4069"/><meta name="citation_lastpage" content="4087"/><meta name="citation_doi" content="10.1039/D2MA00067A"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2022/ma/d2ma00067a"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2022/ma/d2ma00067a"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2022/ma/d2ma00067a"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2022/ma/d2ma00067a">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d2ma00100d)" href="/en/content/articlehtml/2022/ma/d2ma00100d" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d1ma01180d)" href="/en/content/articlehtml/2022/ma/d1ma01180d" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div><br/><span class="oa"><img src="/content/newimages/open_access_blue.png" alt=""/> Open Access Article<br/><img src="/content/newimages/CCBY.svg" alt=""/>
This Open Access Article is licensed under a <br/><a rel="license" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 Unported Licence</a></span></div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D2MA00067A">10.1039/D2MA00067A</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2633-5409/2020">Mater. Adv.</a></span>, 2022, <strong>3</strong>, 4069-4087</div><h1 id="sect182"><span class="title_heading">Recent trends in computational tools and data-driven modeling for advanced materials</span></h1><p class="header_text">
      <span class="bold">
        
          
            Varshika 
            Singh
          
        
      </span><sup><span class="sup_ref italic">abc</span></sup>, 
      <span class="bold">
        
          
            Santanu 
            Patra
          
        
      </span><sup><span class="sup_ref italic">ab</span></sup>, 
      <span class="bold">
        
          
            Natarajan Arul 
            Murugan
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Natarajan Arul Murugan (orcid.org/0000-0003-0185-5724) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-0185-5724"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">de</span></sup>, 
      <span class="bold">
        
          
            Dana-Cristina 
            Toncu
          
        
      </span><sup><span class="sup_ref italic">af</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Ashutosh 
            Tiwari
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Ashutosh Tiwari (orcid.org/0000-0001-5634-7749) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-5634-7749"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">a</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Institute of Advanced Materials, Gammalkilsvägen 18, 590 53 Ulrika, Sweden. E-mail: <a href="mailto:director@iaam.se">director@iaam.se</a>;   Tel: +46 1313-2424</span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">VBRI Innovation Centre, 7/16 Kalkaji Extn, New Delhi, 110019, India</span>
      <br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">University Institute of Biotechnology, Chandigarh University, Mohali, 140413, India</span>
      <br/><a id="affd"><sup><span class="sup_ref italic">d</span></sup></a><span class="italic">Department of Computer Science, School of Electrical Engineering and Computer Science, KTH Royal Institute of Technology, SE-100 44 Stockholm, Sweden</span>
      <br/><a id="affe"><sup><span class="sup_ref italic">e</span></sup></a><span class="italic">Department of Computational Biology, Indraprastha Institute of Information Technology, New Delhi 110020, India</span>
      <br/><a id="afff"><sup><span class="sup_ref italic">f</span></sup></a><span class="italic">School of Chemistry, University College Dublin, Belfield, Dublin 4, Ireland</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      21st January 2022
    </span><span class="bold italic">, Accepted 20th March 2022</span></div><p class="bold italic">First published on 25th March 2022</p><hr/><div class="abstract"><h2>Abstract</h2><p>The paradigm of advanced materials has grown exponentially over the last decade, with their new dimensions including digital design, dynamics, and functions. Materials modeling such as that of their properties and behavior in various environments using <span class="italic">ab initio</span> approaches, force-field methods and machine learning represents a key step in advanced research. Computational techniques and theoretical models pave the way for establishing the structure–property relationship for designing advanced materials with novel properties and improving their performances. Likewise, high accuracy and fewer computational resources of machine-learning approaches have been widely considered for materials design in the recent years. Furthermore, the information derived from materials studies needs to be properly stored and re-analyzed, making big data analysis an essential requirement for further investigations. The information thus generated has also led to the evolution of the genome of materials for the fostering of advanced materials. Thus, various theoretical and computational approaches provide useful predictions about materials properties and efficiency, ultimately leading to the substantial improvements for new-age devices.</p></div><hr/>
    
      
      <h2 id="sect270"><span class="a_heading">1. Introduction</span></h2>
      <span>Advanced materials sustain their enhanced structural and functional properties such as biological, electrical, magnetic, and optical at the macro- and nano-scales, whether they are polymers, glasses, metals, ceramics, or composites. The computational studies of such materials mainly focus on their atomic interactions, to understand their unique structure-derived properties for new technology developments.<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1–4</span></sup></a> The realistic models used in computer simulation both confirm and complement experiments. The computational methods allow the study of the structure, dynamics and properties of materials and establish useful structural relationships to assist the design of materials with improved properties.<a title="Select to navigate to references" href="#cit5"><sup><span class="sup_ref">5</span></sup></a> The behaviour of materials at the macroscopic level is generally governed by atomic interactions and its simulations facilitate a better understanding of the materials architecture.</span>
      <p class="otherpara">Material simulations of larger length and time scales under different external conditions such as temperature, pressure, pH and electric and magnetic fields have become a common tool for technological innovations.<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a><span class="italic">Ab initio</span> and Monte Carlo simulations have been widely used for studying the electronic structure and understanding the key intermolecular forces that dictate the structure and dynamic properties of materials. By applying computational methods during initial design and study, the cost and time of manufacturing, and product testing and production steps of materials are significantly reduced.<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a> Recent progress in machine learning, deep learning, the internet of things (IoT), big data, and intelligent optimization has deeply transformed the computational methodologies used for materials design and innovation, providing a better understanding of their fundamental properties and behavior.<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a></p>
      <p class="otherpara">Materials modeling offers significant information at the microscopic level and explains their compositional, structural, and multifunctional properties and, along with the generation of multidimensional datasets, enables advancements in materials discovery and application for sustainable energy and environmental technologies.<a title="Select to navigate to references" href="#cit9"><sup><span class="sup_ref">9</span></sup></a> It further provides useful insight into materials design and fabrication strategies for novel and improved properties. Nevertheless, such indispensable computational tools, offering prediction of structure and even elements, as well as interesting properties, are limited by their accuracy and therefore under continuous investigation.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a></p>
      <p class="otherpara">The quantum mechanics and computational methods allow the identification of interatomic interactions, while other data-driven techniques help with knowledge-based models to understand structural properties and their role in the functional properties of materials.<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11,12</span></sup></a> All these data and analysis procedures reduce time and cost to gain an insight into materials discovery. In 2011, the Materials Genome Initiative (MGI) announced the identification of faster, more efficient research methods that integrate theory, computation, synthesis, and characterization and accelerate new materials discovery, design, and deployment.<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> Huge datasets of materials are generated in the process, which can be used to analyze and share information among the researchers worldwide, to identify the respective features, which are responsible for the materials functions in the minute time span.<a title="Select to navigate to references" href="#cit14"><sup><span class="sup_ref">14</span></sup></a><a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a> explains how computational methodologies allow structure modeling and function prediction of materials while generating big data and speed up the process from theory to experiment. Computational approaches provide information from the microscopic level of materials, relate to their macroscopic properties, and help modeling and engineering of materials.<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a></p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f1.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">The fundamental approaches of computational procedures for structure modeling and property prediction of materials. Computational tools are faster and cheaper than the high throughput laboratory experiments for assessing the same (Inside atomic structure has been reproduced with permission from ref. <a title="Select to navigate to references" href="#cit11">11</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">In this review article, we have highlighted the cutting-edge materials research in the light of computational methods and modeling by focusing on the developments of advanced and innovative properties and applications. <span class="italic">In silico</span> studies enable design modification, and functional prototyping would otherwise imply time-consuming and expensive iteration and fabrication steps in the case of many materials. Numerous reviews on computational materials modeling have been published in the last few decades (<a title="Select to navigate to table" href="#tab1">Table 1</a>). Nevertheless, most of these reviews focus on a particular computational method or materials. This article enlists many <span class="italic">in silico</span> developments and important technological innovations in advanced materials that were not discussed or mentioned in any of the previous reviews.</p>
      
<div class="table_caption"><b>Table 1</b> <span id="tab1">Previous reviews on computational modeling of materials</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/></colgroup>
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Literature review</th>
                <th align="left" valign="bottom" class="border_black">Focus area of research</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit16">16</a>
                </td>
                <td valign="top">Commonly used methods of electronic structure calculation: <span class="italic">ab initio</span> methods, density functional theory (DFT), some semi-empirical methods</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit17">17</a>
                </td>
                <td valign="top">
                  <span class="italic">Ab initio</span> calculations of optically excited states in 2D materials</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit18">18</a>
                </td>
                <td valign="top">Fundamental topics of Hartree–Fock theory in quantum chemistry</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit19">19</a>
                </td>
                <td valign="top">Machine learning in solid-state materials science</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit20">20</a>
                </td>
                <td valign="top">Density functional theory in materials science</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit21">21</a>
                </td>
                <td valign="top">Density functional theory in materials research</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit22">22</a>
                </td>
                <td valign="top">Monte Carlo simulation of microstructural evolution in metals</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit6">6</a>
                </td>
                <td valign="top">Molecular dynamics computational method in shock wave solid simulation and biopolymers</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit23">23</a>
                </td>
                <td valign="top">Molecular dynamics in steel performance</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit24">24</a>
                </td>
                <td valign="top">Multiscale materials modeling</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit25">25</a>
                </td>
                <td valign="top">Data-driven materials science</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit26">26</a>
                </td>
                <td valign="top">Artificial intelligence in materials science</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit27">27</a>
                </td>
                <td valign="top">Artificial intelligence in materials science</td>
              </tr>
              <tr align="left">
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit28">28</a>
                </td>
                <td valign="top">Advances in modelling and simulation of halide perovskites for solar cell applications</td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
    
    
      
      <h2 id="sect366"><span class="a_heading">2. Computational modeling for materials properties</span></h2>
      <span>Materials are the essential part of technology and development and they have revolutionized the way we live today. The human life is becoming more and more comfortable thanks to various materials with useful properties. The computational modeling of materials provides information about their structures, dynamics, and properties and also how these materials respond to external variables like temperature, pressure and electric and magnetic fields. Such properties of materials to respond to external conditions make them useful for design of smart materials and sensors. Successful modeling of materials properties can be used to establish various structure–property relationships which then can be used to design materials with novel properties. Some of the useful materials properties are listed as follows: (i) optical properties, (ii) magnetic properties, (iii) thermal conduction, (iv) electronic conductivity, (v) negative thermal expansion, (vi) non-linear optical properties, (vii) energy efficiency in relevance to DSSC application, (viii) superionic conductors (NASICON like materials), and (ix) thermally and pH responsive materials. We have theories such as classical mechanics, quantum mechanics and continuum mechanics to model these properties. There are many detailed studies reported in the literature to study these properties. With the use of classical mechanics, one can follow the time evolution of any system and any thermodynamic properties can be computed from the stored positions and velocities. For example, transport properties such as the diffusion coefficient of ions in superionic materials can be computed from the root mean square displacement or from the velocity autocorrelation function.</span>
      <p class="otherpara">The response of materials to external temperature, pressure and pH can be modeled by running the simulations in appropriate ensembles such as the constant temperature constant volume ensemble and the constant pH ensemble, and the systems in these cases are connected to thermostats, barostats and acidostats. However, the electronic (<span class="italic">i.e</span>., absorption and emission spectra) and magnetic properties can not be computed from the classical mechanics but rather one needs to employ electronic structure theory which treats electrons explicitly. The response of materials to light and other fields can be modeled using time dependent density functional theory. So, depending on the system of interest and properties to be modeled one needs to choose the right level of theory. For example, we studied a hybrid system made of graphene, sodium dodecylbenzene sulfonate and glucose oxidase using force-field molecular dynamics simulations<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>). Since, this study investigated the stabilizing interactions of this hybrid system, and given that the length scale of the system was quite larger, we chose this approach which is computationally less demanding but still able to provide insight into the microscopic interactions responsible for the stability of the system. Based on the force-field molecular dynamics, it was found that the graphene–SDBS interface was stabilized by hydrophobic interactions, while the SDBS–glucose oxidase interface was stabilized by hydrophilic interactions. Further, this also explains why the GOx cannot be immobilized directly on the graphene surface.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f2.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Immobilization of glucose oxidase (GOx) on the graphene: sodium dodecylbenzene sulfonate (SDBS) surface for glucose sensing applications (reproduced with permission from ref. <a title="Select to navigate to references" href="#cit29">29</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Similarly, the temperature responsive behavior of poly(<span class="italic">N</span>-isopropylacrylamide) (PNIPAM) and its interaction with Troponin-T were modeled successfully using force-field molecular dynamics. The force-field MD was able to successfully reproduce the coil-globular structural change in PNIPAM with increasing temperature.<a title="Select to navigate to reference" href="#cit30"><sup><span class="sup_ref">30,31</span></sup></a> The pH responsive behavior of a generation-5 poly(amidoamine)-<span class="italic">graft</span>-poly(ethyleneglycol methacrylate) (PAMAM(G5)–PEGMA) based drug delivery system was studied by Zeng <span class="italic">et al.</span> using dissipative particle dynamics simulations<a title="Select to navigate to references" href="#cit32"><sup><span class="sup_ref">32</span></sup></a> (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>). The authors showed that PAMAM(G5)–PEGMA spontaneously forms unicellular micelles at normal pH and central hydrophobic core can be loaded with doxorubicin like drug molecules and the drug molecules can be released under acidic pH condition present in the cancer cell like microenvironment.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f3.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">pH responsive PAMAM(G5)–PCBA polymer for drug delivery for cancer treatment (reproduced with permission from ref. <a title="Select to navigate to references" href="#cit32">32</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Therefore, computer simulations can provide valuable insight into the working mechanism of various biosensor molecules and to understand the pH and thermally responsive behavior of smart polymeric materials for various biosensor–bioelectronic applications. Also, in recent years machine learning approaches have been applied to establish the relationships between the structural properties, electronic properties, and macroscopic properties and the power conversion efficiency of dye molecules for DSSC applications. With the established relationships, it is possible to design novel materials with improved power efficiency.</p>
    
    
      
      <h2 id="sect388"><span class="a_heading">3. Computational methods in materials research</span></h2>
      <span>Computational methods are driving advanced materials towards engineering high-performance materials. Nowadays, computational methods and tools are key components of research for the design and development of a new class of materials with advanced applications and properties. They help in planning and optimizing materials synthesis processes, including chemical reactions and required routes to follow. In addition, they also help in the analysis and interpretation of experimental data regarding materials structure and properties, offering meaningful insight into energy devices, composite materials, automobiles, and chemical and biological sciences.<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a></span>
      
        
        <h3 id="sect393"><span class="b_heading">3.1. Materials modeling</span></h3>
        <span>The materials modeling approach is routinely applied as a cost-effective and complementary tool for extracting information about materials at the microscopic level and for explaining their electrical, optical, and magnetic properties. It also provides an understanding into design principles and strategies for the fabrication of materials with novel and improved properties. Therefore, the use of materials modeling along with experimental validation opens the possibilities for designing extremely useful materials that are relevant for various applications such as energy and healthcare at the industrial level. Materials properties are highly dependent on structure and hence structure prediction is considered to be the center of computational materials research.<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> Materials of different length scales (<span class="italic">i.e.</span>, nano- and micro- materials) have been developed with better properties like superplasticity and enhanced toughness and strength.<a title="Select to navigate to reference" href="#cit35"><sup><span class="sup_ref">35–37</span></sup></a> Predicting the structure and properties of these materials using computational tools before performing experiments has become common practice in materials research. The development of reliable quantum/molecular mechanics approaches and atomistic scale calculations allows such projections. Materials modeling has been performed by three well-studied modeling approaches, <span class="italic">i.e.</span>, <span class="italic">ab initio</span> methods, density functional theory (DFT) semi-empirical methods and force-field methods.</span>
        <div>
          
          <span id="sect402"/><span class="c_heading_indent">3.1.1. 
            <span class="italic">Ab initio</span> methods. </span>
          <span>
            <span class="italic">Ab initio</span> methods are generally used for the prediction of the structure, dynamics, and properties of materials. They were initially limited to studying the chemical reactions and optical properties of molecular systems and extended to studying the optical properties of chromophores in biological environments such as green fluorescent proteins (GFPs). Since <span class="italic">ab initio</span> methods can be only applied to small sized systems, they need to be integrated with molecular mechanics approaches, leading to the development of hybrid QM/MM approaches, which won the Nobel Prize in 2013. But in recent times, these methods have become essential tools in the study of atoms and molecules in condensed matter systems and, increasingly, in modeling complex systems in materials science. <span class="italic">Ab initio</span> methods are based on the Schrödinger equation and involve many approximations such as the Born–Oppenheimer approximation. These involve quantum mechanical calculations to identify configurational disorder, lattice vibrations and electronic excitations to predict the lattice model and associated properties based on compositional structure.<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a></span>
          <p class="otherpara">The core knowledge achieved through <span class="italic">ab initio</span> methods arises from the computational solution of the electronic Schrödinger equation and helps in calculating and identifying the electronic energy, electron density, electron transfer, thermal relaxation, energy transfer, and charge recombination processes for applications in the development of cells, batteries, and other energy devices.<a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a><span class="italic">Ab initio</span> methods can also help in modeling the relatively small region of materials at the quantum chemistry level, where other modeling methods are treated on an approximation basis. In drug delivery, for example, <span class="italic">ab initio</span> molecular dynamics simulation followed by geometry optimization was performed in the gas phase for determining the binding energies of drug–polymer complexes, leading to the finding that drug molecules interact with the polymer through hydrogen bonding, while quantum mechanics/molecular mechanics simulation showed that ciprofloxacin and metronidazole release patterns follow the zero order kinetic model.<a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a></p>
          <p class="otherpara">The remarkable advances in <span class="italic">ab initio</span> methods provide benchmark accuracy, used routinely to complement experimental studies, and effectively calculate a wide variety of properties such as structure, thermochemistry, various types of spectroscopic quantities and response to external perturbations. This type of calculation has practical application in industry for relevant catalytic processes, semiconductor processing, and modeling of conductivity for microelectronics applications. In addition, <span class="italic">ab initio</span> methods provide a wealth of details that are not available from experiments and a degree of confidence in the results that are not available from a more empirical approach.<a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a></p>
          <p class="otherpara">
            <span class="italic">Ab initio</span> calculations of optically excited states in 2D materials derived from the nature of two- and multi-particle excited states such as strongly bound Wannier excitons and charged excitons gave valuable insight into many interesting properties, for instance enhanced light–matter coupling, gate-tunable photoluminescence, and unusual excitonic optical selection rules (<a title="Select to navigate to table" href="#tab2">Table 2</a>).<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a></p>
          
<div class="table_caption"><b>Table 2</b> <span id="tab2">Comparison of various computational methods used for materials study</span>
		</div>

              <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>
                
                
                
                
                <thead align="left">
                  <tr align="left" valign="bottom">
                    <th align="left" valign="bottom" class="border_black">Method</th>
                    <th align="left" valign="bottom" class="border_black">Advantage(s)</th>
                    <th align="left" valign="bottom" class="border_black">Disadvantage(s)</th>
                    <th align="left" valign="bottom" class="border_black">Application(s)</th>
                  </tr>
                </thead>
                <tbody align="left">
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Ab initio</span> methods</td>
                    <td valign="top">Provide very accurate predictions based on quantum mechanics using wave functions</td>
                    <td valign="top">Intensive computational requirements</td>
                    <td valign="top">Prediction of structures, thermodynamics, spectroscopic quantities, external force responses</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Density functional theory (DFT)</td>
                    <td valign="top">Electronic structure calculations, less extensive computational requirements</td>
                    <td valign="top">Too many approximations</td>
                    <td valign="top">Calculation of materials’ interaction properties and other kinetics calculations</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Semi-empirical methods</td>
                    <td valign="top">Faster computation</td>
                    <td valign="top">Less accurate</td>
                    <td valign="top">Modeling of the structure and dynamics of large systems</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Monte Carlo (MC) simulation</td>
                    <td valign="top">Probability calculations of different outcomes</td>
                    <td valign="top">Large computational resources required</td>
                    <td valign="top">Calculation of the thermodynamic properties of systems</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Molecular mechanics (MM)</td>
                    <td valign="top">Less complicated, fast, calculates equilibrium equations of the atomistic system</td>
                    <td valign="top">Can’t predict accurate bond making and breaking geometries</td>
                    <td valign="top">Prediction of accurate structures and energies for molecules</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Molecular dynamics (MD)</td>
                    <td valign="top">Predicts materials properties by analyzing atomic level behavior, visualizes the motion of the atoms at the nanosecond level</td>
                    <td valign="top">Based on Newtonian mechanics, lack quantum effects</td>
                    <td valign="top">Prediction of materials properties like hardness, dissolution rate, toughness, and reversibility</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Quantum mechanics (QM)</td>
                    <td valign="top">Can model chemical bonds</td>
                    <td valign="top">Require extensive computational facilities; thus applications are limited to smaller systems</td>
                    <td valign="top">Prediction of macroscopic properties of materials</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Finite element methods</td>
                    <td valign="top">Beer accuracy, faster, modeling of the entire structural system rather than components</td>
                    <td valign="top">Reproduce macrostructure levels but not the atomistic parameters</td>
                    <td valign="top">Analysis of stress, vibration, heat transfer and other physical analyses</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Probabilistic materials simulation</td>
                    <td valign="top">Predicts variance in materials properties due to variance in micro-structural properties</td>
                    <td valign="top">Limited accuracy</td>
                    <td valign="top">Machine learning, data mining, pattern recognition</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Multiscale simulation</td>
                    <td valign="top">Combines the strengths of different mechanical techniques for predicting the microscopic model and for macroscale analysis</td>
                    <td valign="top">Noise or fluctuations in microscale models</td>
                    <td valign="top">New materials design and modeling</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Data science</td>
                    <td valign="top">Hidden correlations inferred from already available large datasets</td>
                    <td valign="top">Integration of data from experimental and computational sources, data standardization</td>
                    <td valign="top">New materials design</td>
                  </tr>
                </tbody>
              </table></div></div>
            <hr class="hrule"/><br/>
        </div>
        <div>
          
          <span id="sect496"/><span class="c_heading_indent">3.1.2. Density functional theory. </span>
          <span>Density functional theory (DFT) is a quantum modeling method most commonly used for electronic structure calculations in materials science. Instead of the wavefunction as adopted in the <span class="italic">ab initio</span> electronic structure theory, DFT relies on electron density, and various properties including energy are represented as functions of this fundamental property. DFT calculations are based on molecular structures and find a local minimum state of molecules or materials.<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a></span>
          <p class="otherpara">Further, DFT gives estimations for the total energy and potential energy of molecules and organic complexes, and band gaps and optical transitions for metals, semiconductors, and, with modifications, insulators. This method also allows studying surface phenomena, such as oxidation and catalytic reactions and modeling of nanoparticles and nanocomposites. DFT has limitations in simulating more than few thousand atoms, which can be overcome by using empirical potentials. DFT is based on the Hohenberg–Kohn theorem that states that the potential of a quantum system can be identified by the functional of its electronic density and an additional constant.<a title="Select to navigate to references" href="#cit43"><sup><span class="sup_ref">43</span></sup></a> This theorem was extended for external potentials in an open set and established as a function of the electronic density for Coulomb potentials.<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a></p>
          <p class="otherpara">The basic formalism needed for the ground-state quantum chemical calculations that are typically performed in DFT analysis comprises the molecular Hamiltonian with a special set of atomic units that are convenient for simplifying the equations, the Born–Oppenheimer approximation for electronic and nuclear problems, and the Hartree and Hartree–Fock equations for the electronic problem.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> DFT limitations can be overcome by using a new method called holomorphic Hartree−Fock theory, where the equations are modified to avoid the disappearance of the solutions. When applied to the simplest chemical system H<small><sub>2</sub></small>, the properties of the solutions resulted without their disappearance with geometry.<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a></p>
          <p class="otherpara">DFT methods have been used for the modeling and simulation of new functional materials for various applications as energy storage devices, conductors, catalysts, and sensors.<a title="Select to navigate to reference" href="#cit47"><sup><span class="sup_ref">47–50</span></sup></a> Coupled with experimental study, they assisted the development of advanced materials for their respective applications in the energy, environmental and health sectors.<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> DFT calculations also helped in analyzing possibilities of using hydrogen as a fuel with new materials enabling catalysis and hydrogen production.<a title="Select to navigate to reference" href="#cit51"><sup><span class="sup_ref">51,52</span></sup></a> Nowadays, DFT is actively used for required property prediction of materials and screening before synthesis.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a></p>
          <p class="otherpara">Recently, DFT calculation of electronic properties contributing to a nanomaterials integrated “on/off” switchable aptasensor system has been reported. The DFT study on the structural contribution of the monomer of poly(<span class="italic">N</span>-isopropylacrylamide) (PNIPAM) grafted onto a graphene oxide (GO) surface and the effective adsorption of the nucleobases on it has explained the effect of PNIPAM on the adsorption of the nucleobases as well as its biomolecular integration with the GO surface. While the experimental study proved that the PNIPAM on the nanomaterials surface does not contribute much to the full electronic feature of the system due to the insulating nature of the polymer, the DFT study showed that the bandgap energy distribution before and after the functionalization of the PNIPAM monomer remained unchanged. The optimized structures of all nucleobases, GO, the PNIPAM monomer and the GO/NIPAM structure are presented in <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a></p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f4.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">The optimised structures of the nucleobases, GO, NIPAM, and GO/NIPAM (reproduced with permission from ref. <a title="Select to navigate to references" href="#cit47">47</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Likewise, DFT has also been applied for anionic engineering of heteroanionic materials,<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> mainly incorporating anions with different properties for designing heteroanionic materials (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>). A functional heteroanionic materials is computationally designed by sampling the structural phase space based on the electronic structure and, as such, the thermodynamic stability of the materials is further identified. The DFT can unravel the composition–structure–property relationships in materials science, contributing significantly to the understanding of the relationship between fundamental physical and chemical properties governing processes at the atomistic scale on one hand, and typically macroscopic properties required in engineering applications, on the other hand. For a range of materials, specifically polycrystalline bulk semiconductor materials, metals and ceramics, the physio-chemical relationships are determined in both chemical compositions and microstructures. Wisely chosen process conditions (<span class="italic">e.g.</span>, mechanical forming and annealing in metallurgy or epitaxial growth in semiconductor technology) lead to the desired microstructure.<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a></p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f5.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">Construction of the heteroanionic materials Na<small><sub>3</sub></small>MoO<small><sub>3</sub></small>F<small><sub>3</sub></small>. Computational analysis helps in predicting building units NaO<small><sub>3</sub></small>F<small><sub>3</sub></small> and MoO<small><sub>3</sub></small>F<small><sub>3</sub></small> and their arrangements in the cluster. This type of heteroanionic arrangement alters the electronic, magnetic, and optical properties of molecules (reproduced with permission from ref. <a title="Select to navigate to references" href="#cit11">11</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">By describing the interactions between atoms and electrons governed by the laws of quantum mechanics, DFT has become an accurate and efficient technique for solving the basic quantum-mechanical equations for complex many-atom and many-electron systems, such as structural materials, catalysis and surface science, nanomaterials, and biomaterials and geophysics. Despite its challenges in designing, simulations meaningfully represent physical and chemical properties using modeling processes including chemical reactions and phase transformation in materials. DFT is successfully used in alloy analysis, especially low-modulus, high-strength titanium alloys, high-strength steels, and aluminium cast alloys with improved processing and enhanced properties.<a title="Select to navigate to references" href="#cit21"><sup><span class="sup_ref">21</span></sup></a></p>
          <p class="otherpara">The information obtained by DFT contributed to the creation of various materials–property databases like the Materials Project, AFLOWLIB, OQMD (Open Quantum Materials Database), AiiDA (Automated Interactive Infrastructure and Database for Computational Science), JARVIS-DFT, OMDB (Organic Materials Database), and QM9. A useful system to predict novel 2D materials using DFT databases was generated<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> as shown in <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f6.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">Screening of 2D materials using DFT databases. Materials with reasonably high PBE (Perdew–Burke–Ernzerhof) functionals to experimental lattice constant difference were filtered and their layered counterparts were prepared manually. Both were used for various DFT calculations and 2D property prediction (redrawn with permission from,<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> inside atomic structure has been reproduced with permission from ref. <a title="Select to navigate to references" href="#cit11">11</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
        <div>
          
          <span id="sect545"/><span class="c_heading_indent">3.1.3. Semi-empirical methods. </span>
          <span>Based on the same theoretical background as <span class="italic">ab initio</span> theory, semi-empirical methods aim at reducing the computational cost by neglecting smaller integrals or approximating time-consuming two-electron integrals, which can be fitted to experimental data or replaced by analytical but approximate expressions. Unlike <span class="italic">ab initio</span> methods, they require parametrization of all elements involved in a molecular system.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a></span>
          <p class="otherpara">Semi-empirical methods represent an efficient computational tool for fast quantitative estimations of many properties, particularly useful for correlating large sets of experimental and theoretical data, for establishing trends in classes of related molecules, and for scanning a computational problem before proceeding with higher-level treatments. Compared with <span class="italic">ab initio</span> or density functional methods, semiempirical calculations are much faster, but less accurate, with less systematic, therefore harder to correct errors. They need improvement in accuracy and the range of applicability, without compromising their computational efficiency. In addition, they need new algorithms for modern computer architectures and extension to larger molecules.<a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> The well-known semi-empirical methods are the underlying theoretical approach, integral approximation, integral evaluation, and parametrization.</p>
        </div>
      
      
        
        <h3 id="sect555"><span class="b_heading">3.2. Computational simulation</span></h3>
        <span>With the advancement of supercomputers and the continuous development of efficient algorithms, computer simulations have become an integrated part of research and innovation. Though <span class="italic">in silico</span> simulation will never replace the experiment, it is playing an important role in the analysis, design, and optimization of engineering systems. Computer simulations help in validating theories and predicting expected outcomes under extreme physical conditions, by reducing the time and cost for extensive laboratory experiments, including trial and error steps.<a title="Select to navigate to reference" href="#cit56"><sup><span class="sup_ref">56–58</span></sup></a> To achieve the full potential of computational simulation, researchers are confident to get an accurate representation of reality. A model for an expected real-world system is designed and optimized for its performance under different conditions. Simulations for novel design and testing are readily done owing to the efficient computational tools available.<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> Some of the common simulation tools, along with their applied methods and applications, are listed in <a title="Select to navigate to table" href="#tab3">Table 3</a>.</span>
        
<div class="table_caption"><b>Table 3</b> <span id="tab3">Common computational tools available for applications in materials research based on various theories</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/></colgroup>
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Software</th>
                  <th align="left" valign="bottom" class="border_black">Simulation procedures</th>
                  <th align="left" valign="bottom" class="border_black">Applications</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Materials Studio</td>
                  <td valign="top">DFT, hybrid QM/MM and semi-empirical methods</td>
                  <td valign="top">Predict the atomic and molecular structures and properties of materials</td>
                </tr>
                <tr align="left">
                  <td valign="top">ABINIT</td>
                  <td valign="top">DFT</td>
                  <td valign="top">Calculate the optical, mechanical, vibrational, and other observable properties of materials</td>
                </tr>
                <tr align="left">
                  <td valign="top">NAMD</td>
                  <td valign="top">Molecular dynamics</td>
                  <td valign="top">Simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">GROMACS</td>
                  <td valign="top">Molecular dynamics</td>
                  <td valign="top">Simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">LAMMPS</td>
                  <td valign="top">Molecular dynamics</td>
                  <td valign="top">Modeling and simulation at the atomic, meso, or continuum scale</td>
                </tr>
                <tr align="left">
                  <td valign="top">Gaussian</td>
                  <td valign="top">DFT, molecular mechanics</td>
                  <td valign="top">Modeling</td>
                </tr>
                <tr align="left">
                  <td valign="top">VASP</td>
                  <td valign="top">DFT</td>
                  <td valign="top">Atomic scale materials modelling</td>
                </tr>
                <tr align="left">
                  <td valign="top">SIESTA</td>
                  <td valign="top">DFT/molecular dynamics</td>
                  <td valign="top">Electronic structure calculations and <span class="italic">ab initio</span> molecular dynamics</td>
                </tr>
                <tr align="left">
                  <td valign="top">CHARMM</td>
                  <td valign="top">QM/MM</td>
                  <td valign="top">Modeling and simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">OpenMM</td>
                  <td valign="top">Molecular dynamics</td>
                  <td valign="top">Simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">YASARA</td>
                  <td valign="top">Molecular dynamics</td>
                  <td valign="top">Molecular graphics, modeling, and simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">TINKER</td>
                  <td valign="top">Molecular mechanics Molecular dynamics</td>
                  <td valign="top">Modeling and simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">Amber</td>
                  <td valign="top">Molecular dynamics</td>
                  <td valign="top">Modeling and simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">Quantum Espresso</td>
                  <td valign="top">DFT calculations</td>
                  <td valign="top">Materials modeling and simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">CPMD</td>
                  <td valign="top">
                    <span class="italic">Ab initio</span> molecular dynamics</td>
                  <td valign="top">Modeling &amp; simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">Dalton</td>
                  <td valign="top">Electronic structure theory</td>
                  <td valign="top">Modeling and simulation</td>
                </tr>
                <tr align="left">
                  <td valign="top">DESMOND</td>
                  <td valign="top">Molecular dynamics</td>
                  <td valign="top">Simulation</td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">The most common simulation techniques for advanced materials modeling are molecular mechanics, molecular dynamics, Monte-Carlo simulation, quantum mechanical calculations, probabilistic materials simulation, and multiscale simulation techniques (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a>). They have been used in various studies to identify materials behavior under adverse conditions, such as high-temperature,<a title="Select to navigate to reference" href="#cit60"><sup><span class="sup_ref">60,61</span></sup></a> wearing and friction,<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> welding, joining,<a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> and crash simulations of composite materials,<a title="Select to navigate to reference" href="#cit64"><sup><span class="sup_ref">64,65</span></sup></a> leading to materials design.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a></p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f7.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">Various computational methods involved in materials modeling, simulation, and property prediction. Quantum mechanical methods like <span class="italic">ab initio</span> and DFT are widely used for structure prediction and calculating interatomic interactions and have been applied in the development of energy devices and sensors. Similarly, various simulation techniques have roles in atomistic, energetic calculations and are applied for new materials designing and modeling of non-homogenous materials. Big Data analysis approaches like ML and NN are used to train and predict materials structure and properties using data from various databases.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <div>
          
          <span id="sect656"/><span class="c_heading_indent">3.2.1. Monte Carlo simulations. </span>
          <span>Monte Carlo (MC) simulation is a numerical method that determines the average behavior of a stochastic system by repeated sampling, used to predict sampling and probability statistics results with the intervention of random variables, with applications in science and engineering. MC simulation defines a domain and gives the probability distribution of possible outcomes of different model assumptions. It allows the calculation of the thermodynamic properties of a system. Using a model of interatomic or intermolecular quantum interactions, the macroscopic properties of a materials can be successfully simulated.<a title="Select to navigate to references" href="#cit67"><sup><span class="sup_ref">67</span></sup></a> It models the variation for each possible configuration of the parameter and determines the average behavior of the system, such as repositioning of molecules or ions, and gives the probability of expected outcome. One major difficulty in MC simulation is that it requires large computational resources to perform repeated sampling for each configuration.<a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a></span>
          <p class="otherpara">The conventional MC simulation draws samples from a probability distribution, in most cases the classical Boltzmann distribution, to obtain thermodynamic properties and minimum energy structures,<a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> while dynamic MC simulation calculates non-equilibrium properties like crystallization<a title="Select to navigate to reference" href="#cit70"><sup><span class="sup_ref">70–72</span></sup></a> and quantum MC simulation computes various electronic properties of materials.<a title="Select to navigate to reference" href="#cit73"><sup><span class="sup_ref">73,74</span></sup></a> All these MC simulations are vastly used in materials design and engineering, with strong examples being a valuable python library employing Tree Search (MDTS) and a database for large silicon–germanium (Si–Ge) alloy structures.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a></p>
          <p class="otherpara">MC simulations of lattice models are widely used to compute the thermodynamic properties of substitutional alloys, despite their difficulty in obtaining the desired quantities. High-level algorithms that serve as interfaces between the user and a traditional MC code, allowing the user to postulate the goals and determine the free energy of an alloy phase over its entire region of stability within a specified accuracy, without requiring any user intervention during the calculations, enable the direct determination of composition–temperature phase boundaries without requiring the calculation of the whole free energy surface of the alloy system.<a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> The MC simulation of metal grain growth based on the Potts model provides important information, such as the kinetics of interface migration in atomistic ordering of f.c.c., to an LI<small><sub>2</sub></small> structure in Ni based alloy.<a title="Select to navigate to references" href="#cit22"><sup><span class="sup_ref">22</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect671"/><span class="c_heading_indent">3.2.2. Molecular mechanics simulation. </span>
          <span>The molecular mechanics (MM) method, also known as the force-field method, uses the classical type of model to predict the energy of a molecule as a function of its conformation. It allows the prediction of equilibrium geometry along with its transition states and relative energies between and within the molecules. This method applies Newtonian mechanics to the model molecular interaction to predict a minimum static energy structure of a system. Despite being able to supply the potential energy for larger molecules, the MM method is not appropriate for bond breaking reactions.<a title="Select to navigate to references" href="#cit76"><sup><span class="sup_ref">76</span></sup></a></span>
          <p class="otherpara">MM atomistic simulations are based on empirical inter-atomic potentials and can describe the small-amplitude vibrations, torsions, elastic deformation, and electrostatic interactions in materials systems.<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a> A MM model helped analyze single layer graphene sheets taking into consideration binary, ternary and quaternary interactions between the atoms: by solving the equilibrium equations of the atomistic system through the arc-length strategy, the critical and post-critical behavior of graphene under compression in zigzag and armchair directions and shear was highlighted.<a title="Select to navigate to references" href="#cit78"><sup><span class="sup_ref">78</span></sup></a> In another MM model, the equilibrium paths and critical behavior are shown in comparison with available solutions for thin plates.<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> MM nanostructure modeling has been widely applied.<a title="Select to navigate to reference" href="#cit80"><sup><span class="sup_ref">80,81</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect681"/><span class="c_heading_indent">3.2.3. Molecular dynamics simulation. </span>
          <span>Molecular dynamics (MD) simulation is a computational simulation approach used in the analysis of the physical movement of atoms and molecules. In a system, atoms and molecules are only allowed to interact for a particular time span, which gives a view of the dynamic evolution. MD simulation is a statistical mechanics method that employs Newton's law of motion to represent the movements of atoms and molecules in a defined system over a period of time. MD simulation represents a powerful simulation tool for investigating the chemical and mechanical properties of materials, along with their deformation and failure mechanisms. MD simulation is one of the mostly used and accepted methods to perform <span class="italic">in silico</span> research at the Ångström nanoscale. This is one of its advantages, but at the same time also a drawback, because it is difficult to have an experimental comparison at this scale. MD simulation is the only method allowing reproduction of deformation under extreme conditions such as high strain, which is difficult to simulate by other techniques or in laboratory experiments.<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> MD simulation has varied applications in materials science, like materials modeling by predicting minimum energy structures, complementing experimental findings, and predicting materials properties by analyzing atomic level behavior.<a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a></span>
          <p class="otherpara">MD and MC simulations are frequently used for atomic scale lattice configuration and conformation analysis based on physical potentials or force fields, derived from the Schrödinger equations. Major force fields used for MD simulation are CHARMM, AMBER, OPLS-AA, DREIDING, UFF, GROMS, OPLS, and COMPASS, which differ in several properties. MD simulation allows atoms and molecules to interact for a period and provides a trajectory view of the motion of the atoms at the nanosecond level. MD simulations along with topological concepts are used to predict materials properties like hardness, dissolution rate, toughness, and reversibility.<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> MD simulation helps in the development of an analytical model of macroscopic properties of materials to facilitate calculations of electronic band-structure, elastic properties, and piezoelectric, dielectric, and ferroelectric properties. Several applications such as thermoelectrics, catalysts, and energy devices have been designed making use of the MD simulation approach. MD analysis helps in assessing the stability and interaction of materials with various types of interfaces for the development of sensors and batteries.<a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a></p>
          <p class="otherpara">Furthermore, the MD approach helps in identifying atomic scale mechanisms controlling structure–property relationships in materials which would be experimentally exhausting to identify.<a title="Select to navigate to reference" href="#cit87"><sup><span class="sup_ref">87–91</span></sup></a> The MD approach was used to understand the structure–property relationship of supramolecular polymers and establish the internal structures of the monomers of the polymers, which are responsible for the adaptivity and stimuli-responsiveness behavior of the polymers. The simulation results showed that the re-organization of the supramolecular structure occurs due to monomer diffusion on the dynamic fiber surface. Modifying the structure of the monomers results in the variation in supramolecular configuration and their response to the respective stimulus (<a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>).<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a></p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f8.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">Obtaining structure–dynamics–property relationships for BTA(1)3+ supramolecular polymers. (A) The BTA monomer (green) diffuses along the <span class="italic">z</span>-direction, the monomer stops on accessible cores (hot spots) on the fiber surface. (B) Movement of the BTA(1)3+ core (points) onto fiber 1 at each step of the CG-MD simulation. The core is colored based on its <span class="italic">z</span>-displacement. The average configuration of the fiber during the CG-MD is shown in transparent gray (solid beads represent the monomer cores). High-density points identify regions where the diffusing monomer stations for longer times. (C) The <span class="italic">z</span>-displacement (black) of BTA (1)3+ onto fiber 1, the minimum distance between the BTA(1)3+ core, and the closest hot spot (HS) in the fiber (green) with <span class="italic">E</span><small><sub>z</sub></small> = 60 mV nm<small><sup>−1</sup></small>. (D) The average BTA(1) drift velocity (<span class="italic">vz</span>) as a function of the minimum distance from the closest HS for different intensities of <span class="italic">E</span><small><sub>z</sub></small>. (E) Relationship between the relative rate of stimuli-responsiveness (<span class="italic">m</span>) and the average exchange rate of monomers on the surface (<span class="italic">k</span><small><sub>1</sub></small>) for the different fibers. (F) Linear relationship between stimuli-responsiveness rates (<span class="italic">m</span>) and the equivalent relative strengths of core−core interaction in the different fibers (reproduced with permission from ref. <a title="Select to navigate to references" href="#cit84">84</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Optimized coarse-grained MD methods are great exploration tools in complex topological polymer structures including star-polymers and biomacromolecules such as polyelectrolytes and polymers with intrinsic stiffness, thus contributing to an improved understanding of medical applications such as non-invasive extracorporeal shock wave lithotripsy or tumor treatment.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> MD in materials microscopic-sized dynamic simulation and its subsequent application of macro-characteristics and microstructure to steel organization performance control describes the mechanical behavior of such materials, especially crack, dislocation, and grain boundaries and their interaction.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect714"/><span class="c_heading_indent">3.2.4. Quantum mechanics simulation. </span>
          <span>Quantum mechanics (QM) simulation is one of the fundamental theories in physics, which describes the nature of particles at the atomic level. QM methods accurately represent the behavior of electrons in atoms and molecules and can model chemical bonds. They help explain the macroscopic properties of materials. QM simulations are based on the Schrödinger equation and study nuclear and electronic interactions between particles to identify interatomic interactions. QM simulation methods imply obtaining the electron wave function by solving the Schrödinger equation to calculate the probability of an electron being present in a certain interval. This approach describes the nonlocalized behavior of electrons outside the nucleus and can fundamentally and accurately compute the molecular structure and materials properties. QM methods require extensive computational facilities and thus their applications are limited to smaller systems of a few hundreds of atoms.<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a></span>
          <p class="otherpara">Computational quantum mechanics brought a high demand for new techniques in theoretical condensed matter physics that facilitates the understanding of solid materials properties with details down to the atomic level, bringing theoretical studies to a higher, though practicable level. Particularly, the Exact Muffin-Tin Orbitals (EMTO) or Kohn−Sham method allows establishing a theoretical insight into the electronic structure of complex engineering materials such as binary, iron–chromium–nickel, and stainless-steel alloys. The single-electron Schrödinger equations are solved separately within the units defined around the lattice sites. The unit cells are chosen in such a way that they should give a proper description of the local surroundings for every lattice site.<a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect721"/><span class="c_heading_indent">3.2.5. Finite element method simulation. </span>
          <span>The FEM is a numerical method for finding approximate solutions of partial differential equations (PDE) within given boundaries.<a title="Select to navigate to reference" href="#cit93"><sup><span class="sup_ref">93,94</span></sup></a> As such, a structure is divided into several elements or pieces and then reconnected at nodes. These nodes hold elements together and the resultant simultaneous algebraic equations of finite nature are further solved.<a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a> The FEM can successfully reproduce macrostructure levels but not the atomistic parameters. It allows simulation of any design, product or equipment and helps in analysis of stress, vibration, heat transfer and other physical analyses, such as modeling of cracks, dislocations, grain boundaries and the evolution of phase boundaries of materials.<a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a></span>
          <p class="otherpara">In recent decades, the Crystal Plasticity Finite Element Method (CPFEM) has become an extremely versatile tool for describing the mechanical response of crystalline materials on all length scales from single crystals to engineering parts. Crystalline matter being mechanically anisotropic, its deformation depends on the direction of the mechanical loads and geometrical constraints imposed through the anisotropy of the elastic tensor and orientation dependence of the activation of the crystallographic deformation mechanisms (dislocations, twins, martensitic transformations).</p>
          <p class="otherpara">In consequence, associated mechanical phenomena such as materials strength, shape change, ductility, strain hardening, deformation-induced surface roughening, damage, wear, and abrasion are also orientation-dependent, and, for single-crystal elastic–plastic anisotropy, macroscopically directional properties are also produced when the orientation distribution (crystallographic texture) of the grains in a polycrystal is not random. Therefore, finite element approximations have become relevant for crystal mechanical modeling, as they are based on the variational solution of the equilibrium of the forces and the compatibility of the displacements using a weak form of the principle of virtual work in each finite-volume element. The entire sample volume under consideration is discretized into such elements.</p>
          <p class="otherpara">The essential step which renders the deformation kinematics of this approach a crystal plasticity formulation is the fact that the velocity gradient is written in dyadic form. This reflects the tensorial crystallographic nature of the underlying defects that lead to shear and, consequently, to both shape changes (symmetric part) and lattice rotations (skew-symmetric part), with meaningful results in solving crystal mechanical problems under complicated internal and/or external boundary conditions.<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> The FEM proves to be an extremely useful tool when the problems solved are either boundary value problems or initial-value problems or both, and the solution of most of these problems by exact methods of analysis is not possible. In the FEM problem formulation, the least squares, collocation, Rayleigh–Ritz and Galerkin methods are well-established and provide solution to many engineering problems.<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect733"/><span class="c_heading_indent">3.2.6. Probabilistic materials simulation. </span>
          <span>For materials with disordered microstructure and thus fluctuating behavior, probabilistic simulation predicts the variance of materials properties based on the variance of microstructural properties and the variance in component behavior based on the variance in materials behavior. Probabilistic numerical models applied to predict the effect of microstructural disorder of long fiber reinforced thermoplastics (LFT) on the macroscopic response of structures and structural components revealed the variability of the mechanical response of materials due to the variability in fiber orientation and fiber density.<a title="Select to navigate to references" href="#cit99"><sup><span class="sup_ref">99</span></sup></a> Probabilistic methods can also be used to determine the predicted environmental concentrations (PECs) of any materials, as they provide information regarding environmental fate, exposure, emission, and transmission characteristics.<a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a></span>
        </div>
        <div>
          
          <span id="sect739"/><span class="c_heading_indent">3.2.7. Multiscale simulation. </span>
          <span>Many materials exhibit multiscale behavior, <span class="italic">i.e.</span>, their properties on a macroscopic length scale are a result of activity on much smaller length or time scales. Multiscale simulations help in obtaining a microscopic model using computational and numerical methods from a macroscopic model. Multiscale simulations at different scales allow obtaining the efficiency of macroscopic models and relate it accurately to microscopic models. Commonly used multiscale simulation methods are concurrent and sequential methods. The concurrent method uses a combined model developed by linking several computational methods together, while the sequential method uses a string of hierarchical computational methods linked together.<a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> In the concurrent method, different scales of materials behavior are considered simultaneously; and in the sequential model, simulation is performed one by one, and the quantity calculated at one scale is used to define the parameters for the next. The sequential method is more convenient as compared to the concurrent multiscale method. Other examples of multiscale modeling methods are the quasi-continuum (QC) method,<a title="Select to navigate to references" href="#cit102"><sup><span class="sup_ref">102</span></sup></a> the macroscopic, atomistic, <span class="italic">ab initio</span> dynamics (MAAD) method,<a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> the coarse-grained molecular dynamics (CGMD) method,<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> the bridging domain method (BDM)<a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> and the bridging scale method (BSM).<a title="Select to navigate to reference" href="#cit73"><sup><span class="sup_ref">73,104</span></sup></a></span>
          <p class="otherpara">The main advantage of multiscale modeling is that it combines the strengths of different computational mechanics techniques like quantum mechanics, molecular dynamics, and FE for micro- and macroscale analysis. While MS materials modeling combines existing and emerging methods from diverse scientific disciplines to bridge the wide range of time and length scales that are inherent in a number of essential phenomena and processes in materials science and engineering, it needs to be properly linked to the electronic structure theory and statistical methods to describe the physics and chemistry that govern the properties and processes of materials under realistic temperature and pressure conditions.<a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a> The effect of MS transformation on materials science is well represented by its ability to discover new materials and their phenomena for acquiring insight into the processes that rule the materials behavior, as well as for quantitative predictions useful in materials design, completing the experimental synthesis and characterization.<a title="Select to navigate to references" href="#cit24"><sup><span class="sup_ref">24</span></sup></a></p>
        </div>
      
    
    
      
      <h2 id="sect754"><span class="a_heading">4. Data science</span></h2>
      <span>Data science methods are heavily promoted in materials science and data storage, re-shaping analysis. Materials discovery through conventional research including experimental, theoretical or computational implies long times and high costs, while data science accelerates the process from materials discovery to deployment.<a title="Select to navigate to references" href="#cit25"><sup><span class="sup_ref">25</span></sup></a> It includes data management with the help of reliable storage and retrieval databases and data analytics comprising data mining and analysis.<a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> For materials, data can include experimental information with details of test conditions. The common data science methods in materials research, <span class="italic">i.e.</span>, big data analytics, machine learning algorithms and artificial intelligence, are discussed below.</span>
      
        
        <h3 id="sect761"><span class="b_heading">4.1. Big data analysis</span></h3>
        <span>Big Data usually refers to a large dataset, structured or unstructured, that can be used to extract information about definite patterns or parameters using computational algorithms. It is defined by large magnitudes of five <span class="italic">vs.</span> volume, velocity, variety, veracity, and value.<a title="Select to navigate to reference" href="#cit106"><sup><span class="sup_ref">106,107</span></sup></a> Due to advanced computational and experimental studies, a large amount of materials data is available, leading materials research into the big data domain. Big data research of materials is relatively new and requires more robust approaches and databases for analysis, as well as huge computer data storage capability.</span>
        <p class="otherpara">The peer-reviewed journals are now demanding the materials science community to provide data availability statement for publications, allowing data accessibility and reuse by the community.<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a> Data can be submitted in standard databases, such as the NoMaD repository for materials property calculations, caNanoLab for nanomaterials in biomedical applications, the Cambridge Structural Database for crystallographic data, and PubChem for characterization of chemical substances.</p>
        <p class="otherpara">In materials research, most of the information on quantum mechanics remains unutilized and is ultimately discarded, but storing this detail helps in big data analysis and allows learning more about materials properties and processes. New tools, algorithms and approaches are required to access trends or patterns among materials and their properties, and for comparing different data. Big data approaches lead to efficient structure–property relationship analysis at different length or time scales. Progress in large materials databases and machine learning algorithms helps materials research tremendously.</p>
        <p class="otherpara">The development and incorporation of mathematical and statistical methods for the study of high dimensional, sparse, and noisy big data and extraction of patterns from them are now major challenges of materials research.<a title="Select to navigate to references" href="#cit109"><sup><span class="sup_ref">109</span></sup></a> These methods can be used for materials discovery by identification of key factors and exploration of their functionality in design and processing. Big data analysis of materials mainly involves extracting patterns and determining relationships between data with the aid of data mining, evolutionary algorithms, and other statistical methods. It is expected that, with recent advances in experiment and <span class="italic">in silico</span>, big data will result in significant materials advancements as they did for genomics, revolutionizing not only the outstanding outcomes, but also the way researchers work.<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a> The big data contribute to a significant acceleration in the design of new materials, from discovery to market release and even patenting such innovations, becoming a platform for the scientific practice of systematically extracting knowledge from materials datasets (<a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>). As such, it ultimately benefits the quality of life for the public.<a title="Select to navigate to reference" href="#cit25"><sup><span class="sup_ref">25,111</span></sup></a></p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f9.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">Materials discovery schematic, differentiating between the traditional approach, based on experimentation, theory, or computation, and the data-driven approach (reproduced with permission from ref. <a title="Select to navigate to references" href="#cit25">25</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect779"><span class="b_heading">4.2. Artificial intelligence</span></h3>
        <span>Artificial intelligence involves acquiring information and using it with reasoning and self-correction, like human intelligence with the help of computer systems. In the case of materials science, AI makes use of the data derived from theoretical and experimental works and stored in various materials databases like AFLOWLIB, JARVIS-DFT, Polymer Genome, Citrination, and Materials Innovation Network. This helps in accelerating the synthesis and optimization processes, and in identifying materials with similar behavior.<a title="Select to navigate to references" href="#cit112"><sup><span class="sup_ref">112</span></sup></a> The Materials Genome Initiative helped accelerating materials discovery and the development of databases from theoretical studies, which further catalyzed the progress. AI tools consist of machine-learning, deep-learning and other optimization techniques, and algorithms to evaluate materials performance. AI is a very active area, with many challenges, accelerating research in design, synthesis, characterization and application of molecules and materials in the near future.</span>
        <p class="otherpara">AI overcame the inherent limitations of the human cognitive ability with its primary algorithm framework, powerful hardware foundation, and excellent data mining ability. Its applications in materials science and engineering accelerated the simulation of the structure-related performance of materials, though with discrepancies from the experimental findings, towards a superior level, reduced the amount of computation in nano-clustered alloys, predicted properties for new materials in mapping the structure–property relationship, and allowed new technologies for innovative materials synthesis.<a title="Select to navigate to reference" href="#cit26"><sup><span class="sup_ref">26,113</span></sup></a></p>
        <p class="otherpara">Despite being described as the next scientific paradigm in materials discovery and optimization, the recent success of AI came from the combination with experimental research, with subsequent creation of new and creative modes of measuring properties and visualizing and interpreting data. Nevertheless, the next step in AI development requires closing the gap with a robust system that combines ML techniques with physical mechanisms within innovative scientific application.<a title="Select to navigate to reference" href="#cit27"><sup><span class="sup_ref">27,114</span></sup></a></p>
      
      
        
        <h3 id="sect788"><span class="b_heading">4.3. Machine learning</span></h3>
        <span>ML models help in predicting complex materials properties with advantageous accuracy.<a title="Select to navigate to references" href="#cit115"><sup><span class="sup_ref">115</span></sup></a> They benefit modeling and optimization for heat treatment, resistance, microstructure models, <span class="italic">etc.</span>, for materials by using data from previous experimental and theoretical studies.<a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a> ML methods use atoms as vectors for algorithms input, to design and predict materials properties with significant accuracy.<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a> ML techniques extract knowledge and insight from datasets and identify correlations and patterns. Data from both successful and failed experiments are used to train ML models and extract information, to further predict new properties of materials. These include prediction of crystalline structures, vibrational properties, transition temperatures, mechanical properties, band gaps, total energy, the enthalpy of formation, decomposition energies, <span class="italic">etc.</span> The ML approach was successfully used to discover the bandgap transition from indirect to direct and semiconductor to metal transition in silicon, by combining machine learning and <span class="italic">ab initio</span> calculations.<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a></span>
        <p class="otherpara">The most commonly applied ML approach in materials research is supervised learning that requires a good amount of quality data. Regression models like LASSO (least absolute shrinkage and selection operator), Gaussian process regression, support vector regression, artificial neural networks, and kernel ridge regression stimulate discovery of new materials with the desired structures and properties by utilizing established relationships between properties and input descriptors.<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19,119</span></sup></a> For properties with defined categories, like crystal structure types and magnetic property type, classification learning methods are used. These include support vector machines, decision trees, random forests, and Bayesian inference. All ML methods can be used for both regression and classification learning for materials physics problems.<a title="Select to navigate to reference" href="#cit12"><sup><span class="sup_ref">12,120</span></sup></a></p>
        <p class="otherpara">The high-impact research areas in ML for materials science, which simultaneously represent remarkable opportunities for research, have been identified as: validation by experiment of physics-based simulations (useful in predicting new materials properties), ML approaches tailored for materials data and applications (regarding both small and big-scale materials problems, with immediate applicability in new materials design), high-throughput data acquisition capabilities (accelerating materials synthesis, characterization and simulation), ML that makes better scientists (relevant for progress in materials characterization), and integration of physics within ML, and ML with physics-based simulations (by developing desirable methods, reducing discrepancies between simulation and experiments, and linking physics-based models).<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a></p>
        <p class="otherpara">In solid-state materials science, ML has led to numerous quantitative structure–property relationships, such as crystal graphs and chemical composition, building or pair-block polymers, band gap calculations, component and structure predictions, anticipation of materials properties (such as dielectric breakdown strength, glass transition temperature, thermal expansion coefficient, magnetocaloric effects, and interface energy).<a title="Select to navigate to references" href="#cit122"><sup><span class="sup_ref">122</span></sup></a></p>
        <p class="otherpara">The potential of coupling approaches from materials science, modeling, and simulation with data mining and machine learning offers exciting perspectives for solving challenging problems, such as decoding and computational modeling of complex structure–process–property relationships, replacement of computationally demanding submodels in multiscale simulations, and classification and interpretation of imaging data.<a title="Select to navigate to references" href="#cit107"><sup><span class="sup_ref">107</span></sup></a></p>
      
    
    
      
      <h2 id="sect808"><span class="a_heading">5. Materials Genome Initiative</span></h2>
      <span>The Materials Genome Initiative (MGI) is a US multi-agency initiative for creating resources and infrastructure to discover, manufacture, and deploy advanced materials.<a title="Select to navigate to reference" href="#cit13"><sup><span class="sup_ref">13,14</span></sup></a> Advanced materials have applications in energy, security, and human development, and are crucial for national economy. The MGI realizes that these developments, if fostered, can lead to a rapid shift in the time required to find and boost new materials. Its purpose is to facilitate “discovery, development, manufacturing, and deployment of advanced materials at least twice as fast as possible today, at a fraction of the cost”.<a title="Select to navigate to references" href="#cit123"><sup><span class="sup_ref">123</span></sup></a></span>
      <p class="otherpara">It takes more than a decade to bring a new materials to market and commercial use.<a title="Select to navigate to reference" href="#cit124"><sup><span class="sup_ref">124,125</span></sup></a> Accelerating the pace of discovery and deployment of advanced materials is the goal of the MGI, as it promotes the use of data generated through numerous mathematical modeling and statistical analysis studies. These data range from atomic coordinates and structural features to physical and other properties, and result during the development of numerous databases generated from the significant advancement of materials science. In the same way that the genome of a species or individual holds all the secrets of its existence and benefits mankind, decoding a materials genome will help in identifying every single aspect of microscopic buildup, leading to all its properties.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a></p>
      <p class="otherpara">The MGI stimulates use of data science for faster processes of discovery and storage of new materials information.<a title="Select to navigate to reference" href="#cit127"><sup><span class="sup_ref">127,128</span></sup></a> As exemplified in <a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>, data sharing can significantly curtail the materials research phase by (i) lowering the problem of data collection for specific research groups and (ii) allowing more economical progress of scientific suppositions and property projection standards.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a></p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><a href="/image/article/2022/MA/d2ma00067a/d2ma00067a-f10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ma00067a-f10.tif" src="/image/article/2022/MA/d2ma00067a/d2ma00067a-f10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">Representation to demonstrate differences between standard, single-group research (left) and new opportunities afforded by large materials databases (right) (Reproduced with permission from ref. <a title="Select to navigate to references" href="#cit127">127</a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Promotion and funding for the MGI by the US government have led to the speed up of the process for new materials discovery everywhere.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> With the launch of the MGI, other organizations and worldwide committees, like The European Commission and the Psi-k Community, started investing in and promoting materials discovery through computational modeling.<a title="Select to navigate to reference" href="#cit130"><sup><span class="sup_ref">130,131</span></sup></a> It fostered collaborations throughout Europe for developing computational methods for materials research among industrialists, theoreticians, and software developers.<a title="Select to navigate to references" href="#cit132"><sup><span class="sup_ref">132</span></sup></a></p>
    
    
      
      <h2 id="sect828"><span class="a_heading">6. Conclusions and outlook</span></h2>
      <span>With the progress in high computational facilities, hardware, and new algorithms, innovation in materials science has significantly progressed in the last decade. Initially, computational methods dealt with single to fewer atoms and molecules to study their properties and interactions, but now simulate thousands of atoms with high throughput calculations and algorithms. Materials research has now shifted to data science. One of the first computational methods in materials science was density functional theory (DFT), followed by better simulation techniques for multiatomic systems. DFT determines the electronic density of a system and predicts the energy of the system. Other simulation and molecular dynamics methods allow exploration of the structure, space, and composition of systems more efficiently. Simulation and experimental studies generated a huge amount of data, bringing data science approaches to materials science. Large databases are rapidly developed for vast information storage so that they can be further analyzed and classified using big data analysis approaches. The current theoretical methods reveal the internal properties of materials without taking into consideration their actual environment. The development of ML approaches will allow the identification of better descriptors.</span>
      <p class="otherpara">Although various modeling and simulation tools are available, materials research still requires more advanced and efficient tools for screening millions of compositions of new materials with desirable properties to reduce the time and cost of production for individual compositions. Most of the available tools lack efficiency in dealing with materials properties at varying length and time scales. Further advances in computational capabilities, the development of novel algorithms, and the availability of data storage infrastructures present promising progress for new materials discovery and a reduced time to reach market. Materials modeling is now a key step in shortening the manufacturing time of advanced materials, further boosting the industrial and economic developments through innovation of new improved applications. It helps tailor and design new materials for various structures and systems. Nevertheless, most of these modeling and designing processes are computationally extensive and expensive, needing hours to weeks for the desired results. Hence improvement and upgrading are further necessary.</p>
      <p class="otherpara">The developments of synthetic and composite materials will diversify advanced applications. Materials innovation is also critical for socio-economic development and big data and AI will accelerate the design, synthesis, and characterization of new materials. The huge amount of materials data from theoretical, <span class="italic">in silico</span> and experimental studies is stored and regularly analyzed. Further modeling and simulation of complex networks and systems will assist the prediction of materials for renewable energy and other applications. The multiscale, multi-physics modeling at several different scales and levels will encourage novel engineering applications. It is necessary to develop precise computational models for multiscale modeling and simulations, as well as new descriptors for multiscale problems in ML and more databases for materials properties. When produced, newly discovered materials will benefit from the internet of things, which enables empowered connection between manufacturers and machines, smarter business decision-making, and improved management of the global supply chain, but is not without its challenges in needs, security, standards, and human interfaces.<a title="Select to navigate to references" href="#cit120"><sup><span class="sup_ref">120</span></sup></a></p>
    
    
      <h2 id="sect836"><span class="a_heading">Conflicts of interest</span></h2>
      <span>The authors declare that there is no conflict of interest.</span>
    
  
    <h2 id="sect840"><span class="a_heading">Acknowledgements</span></h2>
      <span>The authors gratefully acknowledge the financial support from the International Association of Advanced Materials.</span>
    
    <span id="sect842"><h2 id="sect839"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">M. Bonek, A. Śliwa and J. Mikuła, Computer simulation of the relationship between selected properties of laser remelted tool steel surface layer, <span class="italic">Appl. Surf. Sci.</span>, 2016, <span class="bold">388</span>(A), 174–179 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.apsusc.2016.01.256" title="DOI Link to resource 10.1016/j.apsusc.2016.01.256">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhslWntb0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit2">J.-C. Du and J. M. Rimsza, Atomistic computer simulations of water interactions and dissolution of inorganic glasses, <span class="italic">npj Mater. Degrad.</span>, 2017, <span class="bold">1</span>(1), 16 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41529-017-0017-y" title="DOI Link to resource 10.1038/s41529-017-0017-y">CrossRef</a>.</span></li>
      <li><span id="cit3">J. Zhao, L. Wu, C. Zhan, Q. Shao, Z. Guo and L. Zhang, Overview of polymer nanocomposites: Computer simulation understanding of physical properties, <span class="italic">Polymer</span>, 2017, <span class="bold">133</span>, 272–287 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.polymer.2017.10.035" title="DOI Link to resource 10.1016/j.polymer.2017.10.035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKitLvJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit4">E. R. Homer, High-throughput simulations for insight into grain boundary structure-property relationships and other complex microstructural phenomena, <span class="italic">Comput. Mater. Sci.</span>, 2019, <span class="bold">161</span>, 244–254 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2019.01.041" title="DOI Link to resource 10.1016/j.commatsci.2019.01.041">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXjtlCntrc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit5">A. El-Azab, Why Materials Theory?, <span class="italic">Mater. Theory</span>, 2017, <span class="bold">1</span>, 1 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s41313-017-0001-5" title="DOI Link to resource 10.1186/s41313-017-0001-5">CrossRef</a>.</span></li>
      <li><span id="cit6">M. O. Steinhauser and S. Hiermaier, A Review of Computational Methods in Materials Science: Examples from Shock-Wave and Polymer Physics, <span class="italic">Int. J. Mol. Sci.</span>, 2009, <span class="bold">10</span>(12), 5135–5216 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/ijms10125135" title="DOI Link to resource 10.3390/ijms10125135">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXhsFeju7%252FO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20054467%5Buid%5D" title="PubMed Link to resource 20054467">PubMed</a>.</span></li>
      <li><span id="cit7">A. V. Shymchenko, V. V. Tereshchenko, Y. A. Ryabov, S. V. Salkutsan and A. I. Borovkov, Review of the computational approaches to advanced materials simulation in accordance with modern advanced manufacturing trends, <span class="italic">Mater. Phys. Mech.</span>, 2017, <span class="bold">32</span>, 328–352 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsFajtbnE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit8">
          A. Mosavi, T. Rabczuk and A. Varkonyi-Koczy, <span class="italic">Reviewing the Novel Machine Learning Tools for Materials Design. Recent Advances in Technology Research and Education</span>, Springer Nature,  2017 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Reviewing%20the%20Novel%20Machine%20Learning%20Tools%20for%20Materials%20Design.%20Recent%20Advances%20in%20Technology%20Research%20and%20Education%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit9">A. Ludwig, Discovery of new materials using combinatorial synthesis and high-throughput characterization of thin-film materials libraries combined with computational methods, <span class="italic">npj Comput. Mater.</span>, 2019, <span class="bold">5</span>, 70 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-019-0205-0" title="DOI Link to resource 10.1038/s41524-019-0205-0">CrossRef</a>.</span></li>
      <li><span id="cit10">T. S. Hofer and S. P. de Visser, Editorial: Quantum Mechanical/Molecular Mechanical Approaches for the Investigation of Chemical Systems – Recent Developments and Advanced Applications. Recent Developments and Advanced Applications, <span class="italic">Front. Chem.</span>, 2018, <span class="bold">6</span>, 357 <a target="_blank" class="DOILink" href="https://doi.org/10.3389/fchem.2018.00357" title="DOI Link to resource 10.3389/fchem.2018.00357">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30271768%5Buid%5D" title="PubMed Link to resource 30271768">PubMed</a>.</span></li>
      <li><span id="cit11">J. K. Harada, N. Charles, K. R. Poeppelmeier and J. M. Rondinelli, Heteroanionic Materials by Design: Progress Toward Targeted Properties, <span class="italic">Adv. Mater.</span>, 2019, <span class="bold">31</span>, 1805295 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201805295" title="DOI Link to resource 10.1002/adma.201805295">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30861235%5Buid%5D" title="PubMed Link to resource 30861235">PubMed</a>.</span></li>
      <li><span id="cit12">Y. Liu, T. Zhao, W. Ju and S. Shi, Materials discovery and design using machine learning, <span class="italic">J. Materiomics</span>, 2017, <span class="bold">3</span>(3), 159–177 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jmat.2017.08.002" title="DOI Link to resource 10.1016/j.jmat.2017.08.002">CrossRef</a>.</span></li>
      <li><span id="cit13">J. J. de Pablo, B. Jones, C. L. Kovacs, V. Ozolins and A. P. Ramirez, The materials genome initiative, the interplay of experiment, theory and computation, <span class="italic">Curr. Opin. Solid State Mater. Sci.</span>, 2014, <span class="bold">18</span>, 99–117 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cossms.2014.02.003" title="DOI Link to resource 10.1016/j.cossms.2014.02.003">CrossRef</a>.</span></li>
      <li><span id="cit14">J. J. de Pablo, N. E. Jackson, M. A. Webb, L.-Q. Chen, J. E. Moore, D. Morgan, R. Jacobs, T. Pollock, D. G. Schlom, E. S. Toberer, J. Analytis, I. Dabo, D. M. DeLongchamp, G. A. Fiete, G. M. Grason, G. Hautier, Y. Mo, K. Rajan, E. J. Reed, E. Rodriguez, V. Stevanovic, J. Suntivich, K. Thornton and J.-C. Zhao, New frontiers for the materials genome initiative, <span class="italic">npj Comput. Mater.</span>, 2019, <span class="bold">5</span>, 41 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-019-0173-4" title="DOI Link to resource 10.1038/s41524-019-0173-4">CrossRef</a>.</span></li>
      <li><span id="cit15">Y. Song, J. H. Dai and R. Yang, Application of First Principles Theory to the Design of Advanced Titanium Alloys, <span class="italic">Adv. Eng. Mater. Model.</span>, 2016, 203–228 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsV2iu7nO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit16">
          M. P. Johansson, V. R. I. Kaila and D. Sundholm, Ab Initio, Density Functional Theory, and Semi-Empirical Calculations, in <span class="italic">Biomolecular Simulations. Methods in Molecular Biology (Methods and Protocols)</span>, ed. L. Monticelli, E Salonen, Humana Press, Totowa, NJ,  2013, vol 924, pp. 3–27<small> DOI:<a class="DOILink" href="https://doi.org/10.1007/978-1-62703-017-5_1" TARGET="_BLANK" title="DOI Link to 10.1007/978-1-62703-017-5_1">10.1007/978-1-62703-017-5_1</a></small>.</span></li>
      <li><span id="cit17">K. Xie, X. Li and T. Cao, Theory and Ab Initio Calculation of Optically Excited States—Recent Advances in 2D Materials, <span class="italic">Adv. Mater.</span>, 2021,(33), 1904306 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201904306" title="DOI Link to resource 10.1002/adma.201904306">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitlejtbnM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31808581%5Buid%5D" title="PubMed Link to resource 31808581">PubMed</a>.</span></li>
      <li><span id="cit18">P. Echenique and J. L. Alonso, A mathematical and computational review of Hartree–Fock SCF methods in quantum chemistry, <span class="italic">Mol. Phys.</span>, 2007, <span class="bold">105</span>(23–24), 3057–3098 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/00268970701757875" title="DOI Link to resource 10.1080/00268970701757875">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhsVGjsbjJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit19">J. Schmidt, M. R. G. Marques, S. Botti and M. A. L. Marques, Recent advances and applications of machine learning in solid-state materials science, <span class="italic">npj Comput. Mater.</span>, 2019, <span class="bold">5</span>, 83 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-019-0221-0" title="DOI Link to resource 10.1038/s41524-019-0221-0">CrossRef</a>.</span></li>
      <li><span id="cit20">J. Neugebauer and T. Hickel, Density functional theory in materials science, <span class="italic">Wiley Interdiscip. Rev.: Comput. Mol. Sci.</span>, 2013, <span class="bold">3</span>, 438–448 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhsFKlsL3I" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit21">J. Hafner, C. Wolverton and G. Ceder, Toward Computational Materials Design: The Impact of Density Functional Theory on Materials Research, <span class="italic">MRS Bull.</span>, 2006,(31), 659–668 <a target="_blank" class="DOILink" href="https://doi.org/10.1557/mrs2006.174" title="DOI Link to resource 10.1557/mrs2006.174">CrossRef</a>.</span></li>
      <li><span id="cit22">Y. Saito, The Monte Carlo simulation of microstructural evolution in metals, <span class="italic">Mater. Sci. Eng., A</span>, 1997, <span class="bold">223</span>, 114–124 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0921-5093(97)80019-6" title="DOI Link to resource 10.1016/S0921-5093(97)80019-6">CrossRef</a>.</span></li>
      <li><span id="cit23">Y. Peng, S. Wang, Y. Zhang and Y. Gao, Simulation and Application of Molecular Dynamics in Materials Science, <span class="italic">Adv. Mater. Res.</span>, 2012, <span class="bold">572</span>, 232–236 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Mater.%20Res.%5BJour%5D%20AND%20572%5Bvolume%5D%20AND%20232%5Bpage%5D%20and%202012%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit24">E. van der Giessen, 
            <span class="italic">et al.</span>, Roadmap on multiscale materials modeling, <span class="italic">Modell. Simul. Mater. Sci. Eng.</span>, 2020, <span class="bold">28</span>, 043001 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/1361-651X/ab7150" title="DOI Link to resource 10.1088/1361-651X/ab7150">CrossRef</a>.</span></li>
      <li><span id="cit25">L. Himanen, A. Geurts, A. S. Foster and P. Rinke, Data-Driven Materials Science: Status, Challenges, and Perspectives, <span class="italic">Adv. Sci.</span>, 2019,(6), 1900808 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/advs.201900808" title="DOI Link to resource 10.1002/advs.201900808">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31728276%5Buid%5D" title="PubMed Link to resource 31728276">PubMed</a>.</span></li>
      <li><span id="cit26">X. Sha, 
            <span class="italic">et al.</span>, Artificial Intelligence to Power the Future of Materials Science and Engineering, <span class="italic">Adv. Intell. Syst.</span>, 2020,(2), 1900143 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aisy.201900143" title="DOI Link to resource 10.1002/aisy.201900143">CrossRef</a>.</span></li>
      <li><span id="cit27">B. L. DeCost, 
            <span class="italic">et al.</span>, Scientific AI in materials science: a path to a sustainable and scalable paradigm, <span class="italic">Mach. Learn.: Sci. Technol.</span>, 2020, <span class="bold">1</span>, 033001 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mach.%20Learn.:%20Sci.%20Technol.%5BJour%5D%20AND%201%5Bvolume%5D%20AND%20033001%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit28">C.-J. Yu, Advances in modelling and simulation of halide perovskites for solar cell applications, <span class="italic">J. Phys.: Energy</span>, 2019, <span class="bold">1</span>, 022001 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvFaltrvK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit29">A. O. Osikoya, O. Parlak, N. A. Murugan, E. D. Dikio, H. Moloto, L. Uzun, A. P. F. Turner and A. Tiwari, Acetylene-sourced CVD-synthesised catalytically active graphene for electrochemical biosensing. Acetylene-sourced CVD-synthesised catalytically active graphene for electrochemical biosensing, <span class="italic">Biosens. Bioelectron.</span>, 2017, <span class="bold">89</span>, 496–504 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2016.03.063" title="DOI Link to resource 10.1016/j.bios.2016.03.063">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XntlSgs7o%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27157880%5Buid%5D" title="PubMed Link to resource 27157880">PubMed</a>.</span></li>
      <li><span id="cit30">M. Ashaduzzaman, 
            <span class="italic">et al.</span>, Studies on an on/off-switchable immunosensor for troponin T, <span class="italic">Biosens. Bioelectron.</span>, 2015, <span class="bold">73</span>, 100–107 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2015.05.055" title="DOI Link to resource 10.1016/j.bios.2015.05.055">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtVejtb3N" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26056953%5Buid%5D" title="PubMed Link to resource 26056953">PubMed</a>.</span></li>
      <li><span id="cit31">O. Parlak, 
            <span class="italic">et al.</span>, Programmable bioelectronics in a stimuli-encoded 3D graphene interface, <span class="italic">Nanoscale</span>, 2016, <span class="bold">8</span>, 9976–9981 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6NR02355J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6NR02355J">RSC</a>.</span></li>
      <li><span id="cit32">S. Zeng, 
            <span class="italic">et al.</span>, Computer Simulations on a pH-Responsive Anticancer Drug Delivery System Using Zwitterion-Grafted Polyamidoamine Dendrimer Unimolecular Micelles, <span class="italic">Langmuir</span>, 2021, <span class="bold">37</span>(3), 1225–1234 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.langmuir.0c03217" title="DOI Link to resource 10.1021/acs.langmuir.0c03217">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXmvVyguw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33417464%5Buid%5D" title="PubMed Link to resource 33417464">PubMed</a>.</span></li>
      <li><span id="cit33">G. R. Schleder, A. C. M. Padilha, C. M. Acosta, M. Costa and A. Fazzio, From DFT to machine learning: recent approaches to materials science–a review, <span class="italic">J. Phys.: Mater.</span>, 2019, <span class="bold">2</span>(3), 032001 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitFGhtrjJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit34">A. R. Oganov, C. J. Pickard, Q. Zhu and R. J. Needs, Structure prediction drives materials discovery, <span class="italic">Nat. Rev. Mater.</span>, 2019, <span class="bold">4</span>, 331–348 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41578-019-0101-8" title="DOI Link to resource 10.1038/s41578-019-0101-8">CrossRef</a>.</span></li>
      <li><span id="cit35">G. M. Odegard, T. S. Gates, L. M. Nicholson and K. E. Wise, Equivalent continuum modeling of nanostructured materials, <span class="italic">Compos. Sci. Technol.</span>, 2002, <span class="bold">62</span>, 1869–1880 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0266-3538(02)00113-6" title="DOI Link to resource 10.1016/S0266-3538(02)00113-6">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38Xms1Kht7c%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit36">N. M. Ghoniemy, E. P. Busso, N. Kioussis and H. Huang, Multiscale modelling of nanomechanics and micromechanics: an overview, <span class="italic">Philos. Mag.</span>, 2003, <span class="bold">83</span>(31–34), 3475–3528 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/14786430310001607388" title="DOI Link to resource 10.1080/14786430310001607388">CrossRef</a>.</span></li>
      <li><span id="cit37">J. D. Lee, X. Wang and Y. Chen, Multiscale Computation for Nano/Micromaterials, <span class="italic">J. Eng. Mech.</span>, 2009, <span class="bold">135</span>(3), 192–202 <a target="_blank" class="DOILink" href="https://doi.org/10.1061/(ASCE)0733-9399(2009)135:3(192)" title="DOI Link to resource 10.1061/(ASCE)0733-9399(2009)135:3(192)">CrossRef</a>.</span></li>
      <li><span id="cit38">K. Yang, C. Oses and S. Curtarolo, Modeling Off-Stoichiometry Materials with a High-Throughput Ab-Initio Approach, <span class="italic">Chem. Mater.</span>, 2016, <span class="bold">28</span>(18), 6484–6492 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.6b01449" title="DOI Link to resource 10.1021/acs.chemmater.6b01449">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhtlOqtbnK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit39">J. Wang, Y. Ueda, H. Murakawa, M. G. Yuan and H. Q. Yang, Improvement in numerical accuracy and stability of 3-D FEM analysis in welding, <span class="italic">Weld. J.</span>, 1996, 129–134 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Weld.%20J.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20129%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit40">P. Patra, V. S. Seesala, S. R. Soni, R. K. Roy, S. Dhara, A. Ghosh, N. Patra and S. Pal, Biopolymeric pH-responsive fluorescent gel for <span class="italic">in vitro</span> and <span class="italic">in vivo</span> colon specific delivery of metronidazole and ciprofloxacin, <span class="italic">Eur. Polym. J.</span>, 2019, <span class="bold">114</span>, 255–264 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.eurpolymj.2019.02.037" title="DOI Link to resource 10.1016/j.eurpolymj.2019.02.037">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXktFOqsrs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit41">R. A. Friesner, 
            <span class="italic">Ab initio</span> quantum chemistry: Methodology and applications, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2005, <span class="bold">102</span>(19), 6648–6653 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0408036102" title="DOI Link to resource 10.1073/pnas.0408036102">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXksVKgt7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15870212%5Buid%5D" title="PubMed Link to resource 15870212">PubMed</a>.</span></li>
      <li><span id="cit42">G. Tirimbò and B. Baumeier, 
            <span class="italic">Ab initio</span> modeling of excitons: from perfect crystals to biomaterials, <span class="italic">Adv. Phys.: X</span>, 2021, <span class="bold">6</span>, 1, <small> DOI:<a class="DOILink" href="https://doi.org/10.1080/23746149.2021.1912638" TARGET="_BLANK" title="DOI Link to 10.1080/23746149.2021.1912638">10.1080/23746149.2021.1912638</a></small>.</span></li>
      <li><span id="cit43">E. H. Lieb, Density functionals for coulomb systems, <span class="italic">Int. J. Quantum Chem.</span>, 1983, <span class="bold">24</span>, 243–277, <small> DOI:<a class="DOILink" href="https://doi.org/10.1002/qua.560240302" TARGET="_BLANK" title="DOI Link to 10.1002/qua.560240302">10.1002/qua.560240302</a></small>.</span></li>
      <li><span id="cit44">A. Zhou, A mathematical aspect of Hohenberg-Kohn theorem, <span class="italic">Sci. China Math.</span>, 2019, <span class="bold">62</span>, 63–68 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11425-018-9337-2" title="DOI Link to resource 10.1007/s11425-018-9337-2">CrossRef</a>.</span></li>
      <li><span id="cit45">S. Nasiri, 
            <span class="italic">et al.</span>, Treating the motion of nuclei and electrons in atomic and molecular quantum mechanical calculations on an equal footing: Non-Born–Oppenheimer quantum chemistry, <span class="italic">Adv. Quantum Chem.</span>, 2020, <span class="bold">81</span>, 143–166 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/bs.aiq.2020.05.004" title="DOI Link to resource 10.1016/bs.aiq.2020.05.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhsFKmt77M" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit46">H. G. Hiscock and A. J. W. Thom, Holomorphic Hartree−Fock Theory and Configuration Interaction, <span class="italic">J. Chem. Theory Comput.</span>, 2014, <span class="bold">10</span>, 4795–4800 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ct5007696" title="DOI Link to resource 10.1021/ct5007696">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhslGktb%252FN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26584366%5Buid%5D" title="PubMed Link to resource 26584366">PubMed</a>.</span></li>
      <li><span id="cit47">A. D. Putri, B. Tri Murti, S. Kanchi, M. I. Sabela, K. Bisetty, A. Tiwari, Inamuddin and A. M. Asiri, Computational studies on the molecular insights of aptamer induced poly(<span class="italic">N</span>-isopropylacrylamide)-<span class="italic">graft</span>-graphene oxide for on/off- switchable whole-cell cancer diagnostics, <span class="italic">Sci. Rep.</span>, 2019, <span class="bold">9</span>, 7873 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-019-44378-x" title="DOI Link to resource 10.1038/s41598-019-44378-x">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31133671%5Buid%5D" title="PubMed Link to resource 31133671">PubMed</a>.</span></li>
      <li><span id="cit48">J. E. Saal, S. Kirklin, M. Aykol, B. Meredig and C. Wolverton, Materials Design and Discovery with High-Throughput Density Functional Theory: The Open Quantum Materials Database (OQMD), <span class="italic">JOM</span>, 2013, <span class="bold">65</span>(11), 1501–1509 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11837-013-0755-4" title="DOI Link to resource 10.1007/s11837-013-0755-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhsFentbzK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit49">A. Jain, Y. Shin and K. A. Persson, Computational predictions of energy materials using density functional theory, <span class="italic">Nat. Rev. Mater.</span>, 2016, <span class="bold">1</span>, 15004 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/natrevmats.2015.4" title="DOI Link to resource 10.1038/natrevmats.2015.4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtVertr4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit50">X. Wu, F. Kang, W. Duan and J. Li, Density functional theory calculations: A powerful tool to simulate and design high-performance energy storage and conversion materials, <span class="italic">Prog. Nat. Sci.: Mater. Int.</span>, 2019, <span class="bold">29</span>(3), 247–255 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.pnsc.2019.04.003" title="DOI Link to resource 10.1016/j.pnsc.2019.04.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXot1SrtLk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit51">J. Greeley, T. F. Jaramillo, J. Bonde, I. B. Chorkendorff and J. K. Nørskov, Computational high-throughput screening of electrocatalytic materials for hydrogen evolution, <span class="italic">Nat. Mater.</span>, 2006, <span class="bold">5</span>, 909–913 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat1752" title="DOI Link to resource 10.1038/nmat1752">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XhtFCqu73N" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17041585%5Buid%5D" title="PubMed Link to resource 17041585">PubMed</a>.</span></li>
      <li><span id="cit52">C. Wolverton, D. J. Siegel, A. R. Akbarzadeh and V. Ozolins, Discovery of novel hydrogen storage materials: an atomic scale computational approach, <span class="italic">J. Phys.: Condens. Matter</span>, 2008, <span class="bold">20</span>, 064228 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0953-8984/20/6/064228" title="DOI Link to resource 10.1088/0953-8984/20/6/064228">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BC3MnisVymtg%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21693890%5Buid%5D" title="PubMed Link to resource 21693890">PubMed</a>.</span></li>
      <li><span id="cit53">A. Hospital, J.-R. Goñi, M. Orozco and J.-L. Gelpí, Molecular dynamics simulations: advances and applications, <span class="italic">Adv. Appl. Bioinf. Chem.</span>, 2015, <span class="bold">8</span>, 37–47 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Appl.%20Bioinf.%20Chem.%5BJour%5D%20AND%208%5Bvolume%5D%20AND%2037%5Bpage%5D%20and%202015%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit54">K. Choudhary, 
            <span class="italic">et al.</span>, High-throughput Identification and Characterization of Two-dimensional Materials using Density functional theory, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 5179, <small> DOI:<a class="DOILink" href="https://doi.org/10.1038/s41598-017-05402-0" TARGET="_BLANK" title="DOI Link to 10.1038/s41598-017-05402-0">10.1038/s41598-017-05402-0</a></small>.</span></li>
      <li><span id="cit55">
          W. Thiel, Semiempirical Methods, Modern Methods and Algorithms of Quantum Chemistry, Proceedings, <span class="italic">NIC Series</span>, ed. J. Grotendorst, John von Neumann Institute for Computing, Julich, 2nd edn,  2000, vol. 3, pp. 261–283, ISBN 3-00-005834-6 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=NIC%20Series%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202000%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit56">
          M. O. Steinhauser, Multiscale Computational Materials Science, in <span class="italic">Computational Multiscale Modeling of Fluids and Solids</span>, Springer, Berlin, Heidelberg,  2017 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Computational%20Multiscale%20Modeling%20of%20Fluids%20and%20Solids%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit57">
          J. Meller, Molecular Dynamics, <span class="italic">Encyclopedia of life science</span>, Nature Publishing Group,  2001 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Encyclopedia%20of%20life%20science%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202001%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit58">M. C. Zwier and L. T. Chong, Reaching biological timescales with all-atom molecular dynamics simulations, <span class="italic">Curr. Opin. Pharmacol.</span>, 2010, <span class="bold">10</span>(6), 745–752 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.coph.2010.09.008" title="DOI Link to resource 10.1016/j.coph.2010.09.008">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhsVals7vN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20934381%5Buid%5D" title="PubMed Link to resource 20934381">PubMed</a>.</span></li>
      <li><span id="cit59">C. J. Roy, Review of code and simulation verification procedures for computational simulation, <span class="italic">J. Comput. Phys.</span>, 2005, <span class="bold">205</span>, 131–156 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jcp.2004.10.036" title="DOI Link to resource 10.1016/j.jcp.2004.10.036">CrossRef</a>.</span></li>
      <li><span id="cit60">M. Murugesan and D. W. Jung, Johnson Cook Material and Failure Model Parameters Estimation of AISI-1045 Medium Carbon Steel for Metal Forming Applications, <span class="italic">Materials</span>, 2019, <span class="bold">12</span>(4), 609 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/ma12040609" title="DOI Link to resource 10.3390/ma12040609">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXit1Gju7nJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30781637%5Buid%5D" title="PubMed Link to resource 30781637">PubMed</a>.</span></li>
      <li><span id="cit61">J. C. Osorio-Pinzon, S. Abolghasem and J. P. Casas-Rodriguez, Predicting the Johnson Cook constitutive model constants using temperature rise distribution in plane strain machining, <span class="italic">Int. J. Adv. Manuf. Technol.</span>, 2019, 1–16 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Int.%20J.%20Adv.%20Manuf.%20Technol.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit62">
          M. O. Robbins and M. H. Müser Johannes, <span class="italic">Computer Simulations of Friction, Lubrication, and Wear</span>, CRC Press LLC,  2001 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Computer%20Simulations%20of%20Friction,%20Lubrication,%20and%20Wear%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202001%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit63">I. Montava, R. Irles, J. Segura, J. M. Gadea and E. Juliá, Numerical Simulation of Steel Reinforced Concrete (SRC) Joints, <span class="italic">Metals</span>, 2019, <span class="bold">9</span>(2), 131 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/met9020131" title="DOI Link to resource 10.3390/met9020131">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitlemsrrP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit64">J. Sliseris, L. Yan and B. Kasal, Numerical Simulation and Experimental Verification of Hollow and Foam-Filled Flax-Fabric-Reinforced Epoxy Tubular Energy Absorbers Subjected to Crashing, <span class="italic">Mech. Compos. Mater.</span>, 2017, <span class="bold">53</span>(4), 487–496 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11029-017-9678-4" title="DOI Link to resource 10.1007/s11029-017-9678-4">CrossRef</a>.</span></li>
      <li><span id="cit65">M. Guida, F. Marulo and S. Abrate, Advances in crash dynamics for aircraft safety, <span class="italic">Prog. Aerosp. Sci.</span>, 2018, <span class="bold">98</span>, 106–123 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.paerosci.2018.03.008" title="DOI Link to resource 10.1016/j.paerosci.2018.03.008">CrossRef</a>.</span></li>
      <li><span id="cit66">T. M. Dieb, S. Ju, K. Yoshizoe, Z. Hou, J. Shiomi and K. Tsuda, MDTS: automatic complex materials design using Monte Carlo tree search, <span class="italic">Sci. Technol. Adv. Mater.</span>, 2017, <span class="bold">18</span>(1), 498–503 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/14686996.2017.1344083" title="DOI Link to resource 10.1080/14686996.2017.1344083">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28804525%5Buid%5D" title="PubMed Link to resource 28804525">PubMed</a>.</span></li>
      <li><span id="cit67">K. Binder, Atomistic modeling of materials properties by Monte Carlo Simulation, <span class="italic">Adv. Mater.</span>, 1992, <span class="bold">4</span>(9), 540–547 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.19920040904" title="DOI Link to resource 10.1002/adma.19920040904">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3sXitF2qtb4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit68">S. A. Adcock and J. A. McCammon, Molecular Dynamics: Survey of Methods for Simulating the Activity of Proteins, <span class="italic">Chem. Rev.</span>, 2006, <span class="bold">106</span>(5), 1589–1615 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/cr040426m" title="DOI Link to resource 10.1021/cr040426m">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xht1eqtro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16683746%5Buid%5D" title="PubMed Link to resource 16683746">PubMed</a>.</span></li>
      <li><span id="cit69">K. V. Zakharchenko, J. H. Los, M. I. Katsnelson and A. Fasolino, Atomistic simulations of structural and thermodynamic properties of bilayer graphene, <span class="italic">Phys. Rev. B: Condens. Matter Mater. Phys.</span>, 2010, <span class="bold">81</span>, 235439 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.81.235439" title="DOI Link to resource 10.1103/PhysRevB.81.235439">CrossRef</a>.</span></li>
      <li><span id="cit70">Y. Nie, Z. Gu, Q. Zhou, Y. Wei, T. Hao, Y. Liu, R. Liu and Z. Zhou, Controllability of Polymer Crystal Orientation Using Heterogeneous Nucleation of Deformed Polymer Loops Grafted on Two-Dimensional Nanofiller, <span class="italic">J. Phys. Chem. B</span>, 2017, <span class="bold">121</span>(27), 6685–6690 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcb.7b02861" title="DOI Link to resource 10.1021/acs.jpcb.7b02861">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWhtL%252FL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28628317%5Buid%5D" title="PubMed Link to resource 28628317">PubMed</a>.</span></li>
      <li><span id="cit71">Y. Nie, R. Zhang, K. Zheng and Z. Zhou, Nucleation details of nanohybrid shish-kebabs in polymer solutions studied by molecular simulations, <span class="italic">Polymer</span>, 2015, <span class="bold">76</span>, 1–7 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.polymer.2015.08.057" title="DOI Link to resource 10.1016/j.polymer.2015.08.057">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsVOntbbF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit72">Z.-Z. Gu, R. Yang, J. Yang, X.-Y. Qiu, R.-J. Liu, Y. Liu, Z. Zhou and Y. Nie, Dynamic Monte Carlo simulations of effects of nanoparticle on polymer crystallization in polymer solutions, <span class="italic">Comput. Mater. Sci.</span>, 2018, <span class="bold">147</span>, 217–226 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2018.02.009" title="DOI Link to resource 10.1016/j.commatsci.2018.02.009">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXivFKrt78%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit73">G. J. Wagner and W. K. Liu, Coupling of atomistic and continuum simulations using a bridging scale decomposition, <span class="italic">J. Comput. Phys.</span>, 2003, <span class="bold">190</span>, 249–274 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0021-9991(03)00273-0" title="DOI Link to resource 10.1016/S0021-9991(03)00273-0">CrossRef</a>.</span></li>
      <li><span id="cit74">J. Kolorenč and L. Mitas, Applications of Quantum Monte Carlo Methods in Condensed Systems, <span class="italic">Rep. Prog. Phys.</span>, 2001, <span class="bold">74</span>, 026502 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0034-4885/74/2/026502" title="DOI Link to resource 10.1088/0034-4885/74/2/026502">CrossRef</a>.</span></li>
      <li><span id="cit75">M. A. Van de Walle and, Self-driven lattice-model Monte Carlo simulations of alloy thermodynamic properties and phase diagrams, <span class="italic">Modell. Simul. Mater. Sci. Eng.</span>, 2002, <span class="bold">10</span>, 521–538 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0965-0393/10/5/304" title="DOI Link to resource 10.1088/0965-0393/10/5/304">CrossRef</a>.</span></li>
      <li><span id="cit76">K. Vanommeslaeghe, O. Guvench and A. D. J. MacKerell, Molecular Mechanics, <span class="italic">Curr. Pharm. Des.</span>, 2014, <span class="bold">20</span>(20), 3281–3292 <a target="_blank" class="DOILink" href="https://doi.org/10.2174/13816128113199990600" title="DOI Link to resource 10.2174/13816128113199990600">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXpvValtLs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23947650%5Buid%5D" title="PubMed Link to resource 23947650">PubMed</a>.</span></li>
      <li><span id="cit77">X. Zhang, Y. Zhao and G. Lu, Recent development in quantum mechanics/molecular mechanics modeling for materials, <span class="italic">Int. J. Multiscale Comput. Eng.</span>, 2012, <span class="bold">10</span>(1), 65–82 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Wgsr4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit78">A. Genoese, A. Genoese, N. L. Rizzi and G. Salerno, Buckling Analysis of Single-Layer Graphene Sheets Using Molecular Mechanics, <span class="italic">Front. Mater.</span>, 2019, <span class="bold">6</span>, 26 <a target="_blank" class="DOILink" href="https://doi.org/10.3389/fmats.2019.00026" title="DOI Link to resource 10.3389/fmats.2019.00026">CrossRef</a>.</span></li>
      <li><span id="cit79">
          S. P. Timoshenko and J. M. Gere, <span class="italic">Theory of Elastic Stability</span>, McGraw-Hill, New York, NY,  1963 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Theory%20of%20Elastic%20Stability%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201963%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit80">T. Chang and H. Gao, Size-dependent elastic properties of a single-walled carbon nanotube via a molecular mechanics model, <span class="italic">J. Mech. Phys. Solids</span>, 2003, <span class="bold">51</span>(6), 1059–1074 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0022-5096(03)00006-1" title="DOI Link to resource 10.1016/S0022-5096(03)00006-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXis1ars7c%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit81">R. Rafiee and R. M. Moghadam, On the modeling of carbon nanotubes: a critical review, <span class="italic">Composites, Part B</span>, 2014, <span class="bold">56</span>, 435–449 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compositesb.2013.08.037" title="DOI Link to resource 10.1016/j.compositesb.2013.08.037">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit82">W. Humphrey, A. Dalke and K. Schulten, VMD: Visual molecular dynamics, <span class="italic">J. Mol. Graphics</span>, 1996, <span class="bold">14</span>(1), 33–38 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0263-7855(96)00018-5" title="DOI Link to resource 10.1016/0263-7855(96)00018-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=8744570%5Buid%5D" title="PubMed Link to resource 8744570">PubMed</a>.</span></li>
      <li><span id="cit83">
          C. Massobrio and P. Blandin, Classical and First Principles Molecular Dynamics Simulations in Material Science: Application to Structural and Dynamical Properties of Free and Supported Clusters, in <span class="italic">Stability of Materials. NATO ASI Series (B: Physics)</span>, ed. A. Gonis, P. E. A. Turchi and J. Kudrnovský, Springer, Boston, MA,  1996, p. 355 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Stability%20of%20Materials.%20NATO%20ASI%20Series%20(B:%20Physics)%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit84">A. Torchi, D. Bochicchio and G. M. Pavan, How the Dynamics of a Supramolecular Polymer Determines Its Dynamic Adaptivity and Stimuli-Responsiveness: Structure–Dynamics–Property Relationships from Coarse-Grained Simulations, <span class="italic">J. Phys. Chem. B</span>, 2018, <span class="bold">122</span>(14), 4169–4178 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcb.8b00428" title="DOI Link to resource 10.1021/acs.jpcb.8b00428">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXltVGkt78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29543455%5Buid%5D" title="PubMed Link to resource 29543455">PubMed</a>.</span></li>
      <li><span id="cit85">M. Bauchy, Deciphering the atomic genome of glasses by topological constraint theory and molecular dynamics: A review, <span class="italic">Comput. Mater. Sci.</span>, 2019, <span class="bold">159</span>, 95–102 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.commatsci.2018.12.004" title="DOI Link to resource 10.1016/j.commatsci.2018.12.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisVylu7zP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit86">M. Ashaduzzaman, S. R. Deshpande, N. A. Murugan, Y. K. Mishra, A. P. F. Turner and A. Tiwari, On/off-switchable LSPR nano- immunoassay for troponin-T, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 44027 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep44027" title="DOI Link to resource 10.1038/srep44027">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXlvVeku7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28382946%5Buid%5D" title="PubMed Link to resource 28382946">PubMed</a>.</span></li>
      <li><span id="cit87">J. Liang, R. Tcygana and T. Malam, Molecular dynamics simulation of the structure and properties of lithium phosphate glasses, <span class="italic">J. Non-Cryst. Solids</span>, 2000, <span class="bold">263 &amp; 264</span>, 167–179 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Non-Cryst.%20Solids%5BJour%5D%20AND%20263 &amp; 264%5Bvolume%5D%20AND%20167%5Bpage%5D%20and%202000%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit88">V. Yamakov, D. Wolf, S. R. Phillpot, A. K. Mukherjee and H. Gleiter, Dislocation processes in the deformation of nanocrystalline aluminium by molecular-dynamics simulation, <span class="italic">Nat. Mater.</span>, 2002, <span class="bold">1</span>, 45–49 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat700" title="DOI Link to resource 10.1038/nmat700">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XntlWlsb0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12618848%5Buid%5D" title="PubMed Link to resource 12618848">PubMed</a>.</span></li>
      <li><span id="cit89">V. A. Bershtein, L. M. Egorova, P. N. Yakushev, P. Pissis, P. Sysel and L. Brozova, Molecular dynamics in nanostructured polyimide–silica hybrid materials and their thermal stability, <span class="italic">J. Polym. Sci., Part B: Polym. Phys.</span>, 2002, <span class="bold">40</span>(10), 1056–1069 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/polb.10162" title="DOI Link to resource 10.1002/polb.10162">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38Xjtlertr8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit90">J. Du, L. Kokou, J. L. Rygel, Y. Chen, C. G. Pantano, R. Woodman and J. Belcher, Structure of Cerium Phosphate Glasses: Molecular Dynamics Simulation, <span class="italic">J. Am. Ceram. Soc.</span>, 2011, <span class="bold">94</span>(8), 2393–2401 <a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1551-2916.2011.04514.x" title="DOI Link to resource 10.1111/j.1551-2916.2011.04514.x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhtVOqtLbF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit91">
          M. Montorsi, G. Broglia and C. Mugoni, Structural Insight into Transition Metal Oxide Containing Glasses by Molecular Dynamic Simulations, in <span class="italic">Molecular Dynamics Simulations of Disordered Materials. Springer Series in Materials Science</span>, ed. C. Massobrio, J. Du, M. Bernasconi and P. Salmon, Springer, Cham,  2015, p. 215 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Molecular%20Dynamics%20Simulations%20of%20Disordered%20Materials.%20Springer%20Series%20in%20Materials%20Science%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202015%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit92">
          L. Vitos, <span class="italic">Computational quantum mechanics for materials engineers: the EMTO method and applications, Engineering Materials and Processes</span>, Springer-Verlag London, Limited,  2007, ISBN-13: 9781846289507 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Computational%20quantum%20mechanics%20for%20materials%20engineers:%20the%20EMTO%20method%20and%20applications,%20Engineering%20Materials%20and%20Processes%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit93">T. Belytschko, R. Gracie and G. Ventura, A review of extended/generalized finite element methods for material modeling, <span class="italic">Modell. Simul. Mater. Sci. Eng.</span>, 2009, <span class="bold">17</span>, 043001 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/0965-0393/17/4/043001" title="DOI Link to resource 10.1088/0965-0393/17/4/043001">CrossRef</a>.</span></li>
      <li><span id="cit94">
          T. Belytschko, W. Liu and B. Moran, <span class="italic">Nonlinear Finite Elements for Continua and Structures</span>, John Wiley &amp; Sons,  2001 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Nonlinear%20Finite%20Elements%20for%20Continua%20and%20Structures%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202001%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit95">
          R. D. Cook, D. S. Malkus and M. E. Plesha, <span class="italic">Concepts and Applications of Finite Element Analysis</span>, John Wiley &amp; Sons,  1989 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Concepts%20and%20Applications%20of%20Finite%20Element%20Analysis%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201989%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit96">S. P. Xiao and T. Belytschko, A bridging domain method for coupling continua with molecular dynamics, <span class="italic">Comput. Methods Appl. Mech. Eng.</span>, 2004, <span class="bold">193</span>, 1645–1669 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cma.2003.12.053" title="DOI Link to resource 10.1016/j.cma.2003.12.053">CrossRef</a>.</span></li>
      <li><span id="cit97">
          F. Roters, P. Eisenlohr, T. R. Bieler and D. Raabe, <span class="italic">Crystal Plasticity Finite Element Methods in Materials Science and Engineering</span>, WILEY-VCH Verlag GmbH &amp; Co. KGaA Weinheim,  2010, ISBN 978-3-527-32447-7 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Crystal%20Plasticity%20Finite%20Element%20Methods%20in%20Materials%20Science%20and%20Engineering%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202010%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit98">
          O. Oluwole, <span class="italic">Finite Element Modeling for Materials Engineers Using MATLAB</span>, Springer London Dordrecht HeidelbergNew York,  2011, ISBN 978-0-85729-660-3 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Finite%20Element%20Modeling%20for%20Materials%20Engineers%20Using%20MATLAB%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202011%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit99">J. Hohe, H. Paul and C. Beckmann, A probabilistic elasticity model for long fiber reinforced thermoplastics with uncertain microstructure, <span class="italic">Mech. Mater.</span>, 2018, <span class="bold">122</span>, 118–132 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mechmat.2018.04.007" title="DOI Link to resource 10.1016/j.mechmat.2018.04.007">CrossRef</a>.</span></li>
      <li><span id="cit100">F. Gottschalk, R. W. Scholza and B. Nowack, Probabilistic material flow modeling for assessing the environmental exposure to compounds: Methodology and an application
to engineered nano-TiO<small><sub>2</sub></small> particles, <span class="italic">Environ. Model. Softw.</span>, 2010, <span class="bold">25</span>(3), 320–332 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.envsoft.2009.08.011" title="DOI Link to resource 10.1016/j.envsoft.2009.08.011">CrossRef</a>.</span></li>
      <li><span id="cit101">F. F. Abraham, J. Q. Broughton, N. Bernstein and E. Kaxiras, Spanning the continuum to quantum length scales in a dynamic simulation of brittle fracture, <span class="italic">Europhys. Lett.</span>, 1998, <span class="bold">44</span>, 783–787 <a target="_blank" class="DOILink" href="https://doi.org/10.1209/epl/i1998-00536-9" title="DOI Link to resource 10.1209/epl/i1998-00536-9">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXotFCntro%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit102">E. B. Tadmor, M. Ortiz and R. Phillips, Quasicontinuum analysis of defects in solids, <span class="italic">Philos. Mag. A</span>, 1996, <span class="bold">73</span>, 1529–1563 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Philos.%20Mag.%20A%5BJour%5D%20AND%2073%5Bvolume%5D%20AND%201529%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit103">R. E. Rudd and J. Q. Broughton, Coarse-grained molecular dynamics and the atomic limit of finite elements, <span class="italic">Phys. Rev. B: Condens. Matter Mater. Phys.</span>, 1998, <span class="bold">58</span>, R5893–R5896 <a target="_blank" class="DOILink" href="https://doi.org/10.1103/PhysRevB.58.R5893" title="DOI Link to resource 10.1103/PhysRevB.58.R5893">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXlsl2itLc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit104">H. S. Park and W. K. Liu, An introduction and tutorial on multiple-scale analysis in solids, <span class="italic">Comput. Methods Appl. Mech. Eng.</span>, <span class="bold">193</span>, 1733–1772 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cma.2003.12.054" title="DOI Link to resource 10.1016/j.cma.2003.12.054">CrossRef</a>.</span></li>
      <li><span id="cit105">S. Nikolov, M. Petrov, L. Lymperakis, M. Friák, C. Sachs, H.-O. Fabritius, D. Raabe and J. Neugebauer, Revealing the Design Principles of High-Performance Biological Composites Using Ab initio and Multiscale Simulations: The Example of Lobster Cuticle, <span class="italic">Adv. Mater.</span>, 2010, <span class="bold">22</span>, 519–526, <small> DOI:<a class="DOILink" href="https://doi.org/10.1002/adma.200902019" TARGET="_BLANK" title="DOI Link to 10.1002/adma.200902019">10.1002/adma.200902019</a></small>.</span></li>
      <li><span id="cit106">S. R. Kalidindi and M. De Graef, Materials Data Science: Current Status and Future Outlook, <span class="italic">Annu. Rev. Mater. Res.</span>, 2015, <span class="bold">45</span>, 171–193 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev-matsci-070214-020844" title="DOI Link to resource 10.1146/annurev-matsci-070214-020844">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWqur3F" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit107">N. Huber, S. R. Kalidindi, B. Klusemann and C. J. Cyron, Editorial: Machine Learning and Data Mining in Materials Science, <span class="italic">Front. Mater.</span>, 2020, <span class="bold">7</span>, 51, <small> DOI:<a class="DOILink" href="https://doi.org/10.3389/fmats.2020.00051" TARGET="_BLANK" title="DOI Link to 10.3389/fmats.2020.00051">10.3389/fmats.2020.00051</a></small>.</span></li>
      <li><span id="cit108">
          Dealing with data, <span class="italic">Nat. Mater.</span>,  2017, 16, 1<small> DOI:<a class="DOILink" href="https://doi.org/10.1038/nmat4841" TARGET="_BLANK" title="DOI Link to 10.1038/nmat4841">10.1038/nmat4841</a></small>.</span></li>
      <li><span id="cit109">K. Rajan, Materials Informatics: The Materials “Gene” and Big Data, <span class="italic">Annu. Rev. Mater. Res.</span>, 2015, <span class="bold">45</span>, 153–169 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev-matsci-070214-021132" title="DOI Link to resource 10.1146/annurev-matsci-070214-021132">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWqurvN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit110">
          S. Leonelli, Scientific Research and Big Data, <span class="italic">The Stanford Encyclopedia of Philosophy</span>, ed. E. N. Zalta, Summer 2020 edn, May 29, 2020, https://plato.stanford.edu/archives/sum2020/entries/science-big-data/ <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=The%20Stanford%20Encyclopedia%20of%20Philosophy%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%20%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit111">Y. Chaowei, H. Qunying, L. Zhenlong, L. Kai and H. Fei, Big Data and cloud computing: innovation opportunities and challenges, <span class="italic">Int. J. Digit. Earth</span>, 2017, <span class="bold">10</span>(1), 13–53, <small> DOI:<a class="DOILink" href="https://doi.org/10.1080/17538947.2016.1239771" TARGET="_BLANK" title="DOI Link to 10.1080/17538947.2016.1239771">10.1080/17538947.2016.1239771</a></small>.</span></li>
      <li><span id="cit112">R. K. Vasudevan, K. Choudhary and A. Mehta, 
            <span class="italic">et al.</span>, Materials Science in the AI age: high-throughput library generation, machine learning and a pathway from correlations to the underpinning physics, <span class="italic">MRS Commun.</span>, 2020, <span class="bold">9</span>(3)<small> DOI:<a class="DOILink" href="https://doi.org/10.1557/mrc.2019.95" TARGET="_BLANK" title="DOI Link to 10.1557/mrc.2019.95">10.1557/mrc.2019.95</a></small>.</span></li>
      <li><span id="cit113">Y. Xu, 
            <span class="italic">et al.</span>, Artificial intelligence: A powerful paradigm for scientific research, <span class="italic">The Innovation</span>, 2021, <span class="bold">2</span>, 100179 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.xinn.2021.100179" title="DOI Link to resource 10.1016/j.xinn.2021.100179">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34877560%5Buid%5D" title="PubMed Link to resource 34877560">PubMed</a>.</span></li>
      <li><span id="cit114">J. Kimmig, S. Zechel and U. S. Schubert, Digital Transformation in Materials Science: A Paradigm Change in Material's Development, <span class="italic">Adv. Mater.</span>, 2021, <span class="bold">33</span>, 2004940, <small> DOI:<a class="DOILink" href="https://doi.org/10.1002/adma.202004940" TARGET="_BLANK" title="DOI Link to 10.1002/adma.202004940">10.1002/adma.202004940</a></small>.</span></li>
      <li><span id="cit115">B. Kailkhura, B. Gallagher and S. Kim, 
            <span class="italic">et al.</span>, Reliable and explainable machine-learning methods for accelerated material discovery, <span class="italic">npj Comput. Mater.</span>, 2019, <span class="bold">5</span>, 108, <small> DOI:<a class="DOILink" href="https://doi.org/10.1038/s41524-019-0248-2" TARGET="_BLANK" title="DOI Link to 10.1038/s41524-019-0248-2">10.1038/s41524-019-0248-2</a></small>.</span></li>
      <li><span id="cit116">A. Suzuki, C. Shen and N. C. Kumar, Application of computational tools in alloy design, <span class="italic">MRS Bull.</span>, 2019, <span class="bold">44</span>, 247–251, <small> DOI:<a class="DOILink" href="https://doi.org/10.1557/mrs.2019.70" TARGET="_BLANK" title="DOI Link to 10.1557/mrs.2019.70">10.1557/mrs.2019.70</a></small>.</span></li>
      <li><span id="cit117">Q. Zhou, P. Tang, S. Liu, J. Pan, Q. Yan and S.-C. Zhang, Learning atoms for materials discovery, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2018, <span class="bold">115</span>(28), E6411–E6417 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.1801181115" title="DOI Link to resource 10.1073/pnas.1801181115">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCmsbrF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29946023%5Buid%5D" title="PubMed Link to resource 29946023">PubMed</a>.</span></li>
      <li><span id="cit118">Z. Shi, E. Tsymbalov, M. Dao, S. Suresh, A. Shapeev and J. Li, Deep elastic strain engineering of bandgap through machine learning, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2018, <span class="bold">116</span>(10), 4117–4122 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.1818555116" title="DOI Link to resource 10.1073/pnas.1818555116">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30770444%5Buid%5D" title="PubMed Link to resource 30770444">PubMed</a>.</span></li>
      <li><span id="cit119">X. Li, 
            <span class="italic">et al.</span>, A Comparative Assessment of Six Machine Learning Models for Prediction of Bending Force in Hot Strip Rolling Process, <span class="italic">Metals</span>, 2020, <span class="bold">10</span>, 685, <small> DOI:<a class="DOILink" href="https://doi.org/10.3390/met10050685" TARGET="_BLANK" title="DOI Link to 10.3390/met10050685">10.3390/met10050685</a></small>.</span></li>
      <li><span id="cit120">S. Chibani and F. X. Coudert, Machine learning approaches for the prediction of materials properties, <span class="italic">APL Mater.</span>, 2020, <span class="bold">8</span>, 080701, <small> DOI:<a class="DOILink" href="https://doi.org/10.1063/5.0018384" TARGET="_BLANK" title="DOI Link to 10.1063/5.0018384">10.1063/5.0018384</a></small>.</span></li>
      <li><span id="cit121">B. Meredig, Editorial: Five High-Impact Research Areas in Machine Learning for Materials Science, <span class="italic">Chem. Mater.</span>, 2019, <span class="bold">31</span>, 9579–9581 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.9b04078" title="DOI Link to resource 10.1021/acs.chemmater.9b04078">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXit1OrtL3J" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit122">D. Packwood, 
            <span class="italic">et al.</span>, Machine Learning in Materials Chemistry: An Invitation, <span class="italic">Mach. Learn. Appl.</span>, 2022, <span class="bold">8</span>, 100265 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mach.%20Learn.%20Appl.%5BJour%5D%20AND%208%5Bvolume%5D%20AND%20100265%5Bpage%5D%20and%202022%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit123">A. White, The materials genome initiative: One year on, <span class="italic">MRS Bull.</span>, 2012, <span class="bold">37</span>, 715–716 <a target="_blank" class="DOILink" href="https://doi.org/10.1557/mrs.2012.194" title="DOI Link to resource 10.1557/mrs.2012.194">CrossRef</a>.</span></li>
      <li><span id="cit124">J. F. Rodrigues, L. Florea and M. C. F. de Oliveira, 
            <span class="italic">et al.</span>, Big data and machine learning for materials science, <span class="italic">Discovery Mater.</span>, 2021, <span class="bold">1</span>, 12, <small> DOI:<a class="DOILink" href="https://doi.org/10.1007/s43939-021-00012-0" TARGET="_BLANK" title="DOI Link to 10.1007/s43939-021-00012-0">10.1007/s43939-021-00012-0</a></small>.</span></li>
      <li><span id="cit125">A. Tiwari, Advanced Materials Research and Innovation Priorities for Accomplishing the Sustainable Development Goals, <span class="italic">Adv. Mater. Lett.</span>, 2021, <span class="bold">12</span>, 1–6, <small> DOI:<a class="DOILink" href="https://doi.org/10.5185/amlett.2021.061633" TARGET="_BLANK" title="DOI Link to 10.5185/amlett.2021.061633">10.5185/amlett.2021.061633</a></small>.</span></li>
      <li><span id="cit126">S. A. M. Tofail, Additive manufacturing: scientific and technological challenges, market uptake and opportunities, <span class="italic">Mater. Today</span>, 2018, <span class="bold">21</span>, 22–37 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mattod.2017.07.001" title="DOI Link to resource 10.1016/j.mattod.2017.07.001">CrossRef</a>.</span></li>
      <li><span id="cit127">A. Jain, 
            <span class="italic">et al.</span>, Research Update: The materials genome initiative: Data sharing and the impact of collaborative ab initio databases, <span class="italic">APL Mater.</span>, 2016, <span class="bold">4</span>, 053102, <small> DOI:<a class="DOILink" href="https://doi.org/10.1063/1.4944683" TARGET="_BLANK" title="DOI Link to 10.1063/1.4944683">10.1063/1.4944683</a></small>.</span></li>
      <li><span id="cit128">A. A. Ashley, Big data are shaping the future of materials science, <span class="italic">MRS Bull.</span>, 2013, <span class="bold">38</span>, 594–595 <a target="_blank" class="DOILink" href="https://doi.org/10.1557/mrs.2013.187" title="DOI Link to resource 10.1557/mrs.2013.187">CrossRef</a>.</span></li>
      <li><span id="cit129">
          S. A. Schlueter, <span class="italic">Designing materials and devices to revolutionize and engineer the future of electronics and photonics through computationally led and data-driven approaches. Proceedings Volume 10639</span>, Micro- and Nanotechnology Sensors, Systems, and Applications X; 106390B (2018)<small> DOI:<a class="DOILink" href="https://doi.org/10.1117/12.2305496" TARGET="_BLANK" title="DOI Link to 10.1117/12.2305496">10.1117/12.2305496</a></small>.</span></li>
      <li><span id="cit130">
          Boosting materials modelling, <span class="italic">Nat. Mater.</span>,  2016, 15, 365<small> DOI:<a class="DOILink" href="https://doi.org/10.1038/nmat4619" TARGET="_BLANK" title="DOI Link to 10.1038/nmat4619">10.1038/nmat4619</a></small>.</span></li>
      <li><span id="cit131">A. Agrawal and A. Choudhary, Perspective: Materials informatics and big data: Realization of the “fourth paradigm” of science in materials science, <span class="italic">APL Mater.</span>, 2016, <span class="bold">4</span>(5), 053208 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.4946894" title="DOI Link to resource 10.1063/1.4946894">CrossRef</a>.</span></li>
      <li><span id="cit132">A. Tiwari, Priority of Materials Research for Reaching Climate Neutrality Goals, <span class="italic">Adv. Mater. Lett.</span>, 2022, <span class="bold">13</span>, 22011683, <small> DOI:<a class="DOILink" href="https://doi.org/10.5185/amlett.2022.011683" TARGET="_BLANK" title="DOI Link to 10.5185/amlett.2022.011683">10.5185/amlett.2022.011683</a></small>.</span></li>
    </ol>
    
    
    
    
    
  <table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2022</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>