<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Microalgae as biodiesel &amp; biomass feedstocks: Review &amp; analysis of the biochemistry, energetics &amp; economics   - Energy &amp; Environmental Science (RSC Publishing) DOI:10.1039/B924978H</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2010/ee/b924978h"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":87,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};;/*! For license information please see nr-loader-spa-1.236.0.min.js.LICENSE.txt */
(()=>{"use strict";var e,t,r={5763:(e,t,r)=>{r.d(t,{P_:()=>l,Mt:()=>g,C5:()=>s,DL:()=>v,OP:()=>T,lF:()=>D,Yu:()=>y,Dg:()=>h,CX:()=>c,GE:()=>b,sU:()=>_});var n=r(8632),i=r(9567);const o={beacon:n.ce.beacon,errorBeacon:n.ce.errorBeacon,licenseKey:void 0,applicationID:void 0,sa:void 0,queueTime:void 0,applicationTime:void 0,ttGuid:void 0,user:void 0,account:void 0,product:void 0,extra:void 0,jsAttributes:{},userAttributes:void 0,atts:void 0,transactionName:void 0,tNamePlain:void 0},a={};function s(e){if(!e)throw new Error("All info objects require an agent identifier!");if(!a[e])throw new Error("Info for ".concat(e," was never set"));return a[e]}function c(e,t){if(!e)throw new Error("All info objects require an agent identifier!");a[e]=(0,i.D)(t,o),(0,n.Qy)(e,a[e],"info")}var u=r(7056);const d=()=>{const e={blockSelector:"[data-nr-block]",maskInputOptions:{password:!0}};return{allow_bfcache:!0,privacy:{cookies_enabled:!0},ajax:{deny_list:void 0,enabled:!0,harvestTimeSeconds:10},distributed_tracing:{enabled:void 0,exclude_newrelic_header:void 0,cors_use_newrelic_header:void 0,cors_use_tracecontext_headers:void 0,allowed_origins:void 0},session:{domain:void 0,expiresMs:u.oD,inactiveMs:u.Hb},ssl:void 0,obfuscate:void 0,jserrors:{enabled:!0,harvestTimeSeconds:10},metrics:{enabled:!0},page_action:{enabled:!0,harvestTimeSeconds:30},page_view_event:{enabled:!0},page_view_timing:{enabled:!0,harvestTimeSeconds:30,long_task:!1},session_trace:{enabled:!0,harvestTimeSeconds:10},harvest:{tooManyRequestsDelay:60},session_replay:{enabled:!1,harvestTimeSeconds:60,sampleRate:.1,errorSampleRate:.1,maskTextSelector:"*",maskAllInputs:!0,get blockClass(){return"nr-block"},get ignoreClass(){return"nr-ignore"},get maskTextClass(){return"nr-mask"},get blockSelector(){return e.blockSelector},set blockSelector(t){e.blockSelector+=",".concat(t)},get maskInputOptions(){return e.maskInputOptions},set maskInputOptions(t){e.maskInputOptions={...t,password:!0}}},spa:{enabled:!0,harvestTimeSeconds:10}}},f={};function l(e){if(!e)throw new Error("All configuration objects require an agent identifier!");if(!f[e])throw new Error("Configuration for ".concat(e," was never set"));return f[e]}function h(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");f[e]=(0,i.D)(t,d()),(0,n.Qy)(e,f[e],"config")}function g(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");var r=l(e);if(r){for(var n=t.split("."),i=0;i<n.length-1;i++)if("object"!=typeof(r=r[n[i]]))return;r=r[n[n.length-1]]}return r}const p={accountID:void 0,trustKey:void 0,agentID:void 0,licenseKey:void 0,applicationID:void 0,xpid:void 0},m={};function v(e){if(!e)throw new Error("All loader-config objects require an agent identifier!");if(!m[e])throw new Error("LoaderConfig for ".concat(e," was never set"));return m[e]}function b(e,t){if(!e)throw new Error("All loader-config objects require an agent identifier!");m[e]=(0,i.D)(t,p),(0,n.Qy)(e,m[e],"loader_config")}const y=(0,n.mF)().o;var w=r(385),x=r(6818);const A={buildEnv:x.Re,bytesSent:{},queryBytesSent:{},customTransaction:void 0,disabled:!1,distMethod:x.gF,isolatedBacklog:!1,loaderType:void 0,maxBytes:3e4,offset:Math.floor(w._A?.performance?.timeOrigin||w._A?.performance?.timing?.navigationStart||Date.now()),onerror:void 0,origin:""+w._A.location,ptid:void 0,releaseIds:{},session:void 0,xhrWrappable:"function"==typeof w._A.XMLHttpRequest?.prototype?.addEventListener,version:x.q4},E={};function T(e){if(!e)throw new Error("All runtime objects require an agent identifier!");if(!E[e])throw new Error("Runtime for ".concat(e," was never set"));return E[e]}function _(e,t){if(!e)throw new Error("All runtime objects require an agent identifier!");E[e]=(0,i.D)(t,A),(0,n.Qy)(e,E[e],"runtime")}function D(e){return function(e){try{const t=s(e);return!!t.licenseKey&&!!t.errorBeacon&&!!t.applicationID}catch(e){return!1}}(e)}},9567:(e,t,r)=>{r.d(t,{D:()=>i});var n=r(50);function i(e,t){try{if(!e||"object"!=typeof e)return(0,n.Z)("Setting a Configurable requires an object as input");if(!t||"object"!=typeof t)return(0,n.Z)("Setting a Configurable requires a model to set its initial properties");const r=Object.create(Object.getPrototypeOf(t),Object.getOwnPropertyDescriptors(t)),o=0===Object.keys(r).length?e:r;for(let a in o)if(void 0!==e[a])try{"object"==typeof e[a]&&"object"==typeof t[a]?r[a]=i(e[a],t[a]):r[a]=e[a]}catch(e){(0,n.Z)("An error occurred while setting a property of a Configurable",e)}return r}catch(e){(0,n.Z)("An error occured while setting a Configurable",e)}}},6818:(e,t,r)=>{r.d(t,{Re:()=>i,gF:()=>o,q4:()=>n});const n="1.236.0",i="PROD",o="CDN"},385:(e,t,r)=>{r.d(t,{FN:()=>a,IF:()=>u,Nk:()=>f,Tt:()=>s,_A:()=>o,il:()=>n,pL:()=>c,v6:()=>i,w1:()=>d});const n="undefined"!=typeof window&&!!window.document,i="undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self.navigator instanceof WorkerNavigator||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis.navigator instanceof WorkerNavigator),o=n?window:"undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis),a=""+o?.location,s=/iPad|iPhone|iPod/.test(navigator.userAgent),c=s&&"undefined"==typeof SharedWorker,u=(()=>{const e=navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);return Array.isArray(e)&&e.length>=2?+e[1]:0})(),d=Boolean(n&&window.document.documentMode),f=!!navigator.sendBeacon},1117:(e,t,r)=>{r.d(t,{w:()=>o});var n=r(50);const i={agentIdentifier:"",ee:void 0};class o{constructor(e){try{if("object"!=typeof e)return(0,n.Z)("shared context requires an object as input");this.sharedContext={},Object.assign(this.sharedContext,i),Object.entries(e).forEach((e=>{let[t,r]=e;Object.keys(i).includes(t)&&(this.sharedContext[t]=r)}))}catch(e){(0,n.Z)("An error occured while setting SharedContext",e)}}}},8e3:(e,t,r)=>{r.d(t,{L:()=>d,R:()=>c});var n=r(2177),i=r(1284),o=r(4322),a=r(3325);const s={};function c(e,t){const r={staged:!1,priority:a.p[t]||0};u(e),s[e].get(t)||s[e].set(t,r)}function u(e){e&&(s[e]||(s[e]=new Map))}function d(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"feature";if(u(e),!e||!s[e].get(t))return a(t);s[e].get(t).staged=!0;const r=[...s[e]];function a(t){const r=e?n.ee.get(e):n.ee,a=o.X.handlers;if(r.backlog&&a){var s=r.backlog[t],c=a[t];if(c){for(var u=0;s&&u<s.length;++u)f(s[u],c);(0,i.D)(c,(function(e,t){(0,i.D)(t,(function(t,r){r[0].on(e,r[1])}))}))}delete a[t],r.backlog[t]=null,r.emit("drain-"+t,[])}}r.every((e=>{let[t,r]=e;return r.staged}))&&(r.sort(((e,t)=>e[1].priority-t[1].priority)),r.forEach((e=>{let[t]=e;a(t)})))}function f(e,t){var r=e[1];(0,i.D)(t[r],(function(t,r){var n=e[0];if(r[0]===n){var i=r[1],o=e[3],a=e[2];i.apply(o,a)}}))}},2177:(e,t,r)=>{r.d(t,{c:()=>f,ee:()=>u});var n=r(8632),i=r(2210),o=r(1284),a=r(5763),s="nr@context";let c=(0,n.fP)();var u;function d(){}function f(e){return(0,i.X)(e,s,l)}function l(){return new d}function h(){u.aborted=!0,u.backlog={}}c.ee?u=c.ee:(u=function e(t,r){var n={},c={},f={},g=!1;try{g=16===r.length&&(0,a.OP)(r).isolatedBacklog}catch(e){}var p={on:b,addEventListener:b,removeEventListener:y,emit:v,get:x,listeners:w,context:m,buffer:A,abort:h,aborted:!1,isBuffering:E,debugId:r,backlog:g?{}:t&&"object"==typeof t.backlog?t.backlog:{}};return p;function m(e){return e&&e instanceof d?e:e?(0,i.X)(e,s,l):l()}function v(e,r,n,i,o){if(!1!==o&&(o=!0),!u.aborted||i){t&&o&&t.emit(e,r,n);for(var a=m(n),s=w(e),d=s.length,f=0;f<d;f++)s[f].apply(a,r);var l=T()[c[e]];return l&&l.push([p,e,r,a]),a}}function b(e,t){n[e]=w(e).concat(t)}function y(e,t){var r=n[e];if(r)for(var i=0;i<r.length;i++)r[i]===t&&r.splice(i,1)}function w(e){return n[e]||[]}function x(t){return f[t]=f[t]||e(p,t)}function A(e,t){var r=T();p.aborted||(0,o.D)(e,(function(e,n){t=t||"feature",c[n]=t,t in r||(r[t]=[])}))}function E(e){return!!T()[c[e]]}function T(){return p.backlog}}(void 0,"globalEE"),c.ee=u)},5546:(e,t,r)=>{r.d(t,{E:()=>n,p:()=>i});var n=r(2177).ee.get("handle");function i(e,t,r,i,o){o?(o.buffer([e],i),o.emit(e,t,r)):(n.buffer([e],i),n.emit(e,t,r))}},4322:(e,t,r)=>{r.d(t,{X:()=>o});var n=r(5546);o.on=a;var i=o.handlers={};function o(e,t,r,o){a(o||n.E,i,e,t,r)}function a(e,t,r,i,o){o||(o="feature"),e||(e=n.E);var a=t[o]=t[o]||{};(a[r]=a[r]||[]).push([e,i])}},3239:(e,t,r)=>{r.d(t,{bP:()=>s,iz:()=>c,m$:()=>a});var n=r(385);let i=!1,o=!1;try{const e={get passive(){return i=!0,!1},get signal(){return o=!0,!1}};n._A.addEventListener("test",null,e),n._A.removeEventListener("test",null,e)}catch(e){}function a(e,t){return i||o?{capture:!!e,passive:i,signal:t}:!!e}function s(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;window.addEventListener(e,t,a(r,n))}function c(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;document.addEventListener(e,t,a(r,n))}},4402:(e,t,r)=>{r.d(t,{Ht:()=>u,M:()=>c,Rl:()=>a,ky:()=>s});var n=r(385);const i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";function o(e,t){return e?15&e[t]:16*Math.random()|0}function a(){const e=n._A?.crypto||n._A?.msCrypto;let t,r=0;return e&&e.getRandomValues&&(t=e.getRandomValues(new Uint8Array(31))),i.split("").map((e=>"x"===e?o(t,++r).toString(16):"y"===e?(3&o()|8).toString(16):e)).join("")}function s(e){const t=n._A?.crypto||n._A?.msCrypto;let r,i=0;t&&t.getRandomValues&&(r=t.getRandomValues(new Uint8Array(31)));const a=[];for(var s=0;s<e;s++)a.push(o(r,++i).toString(16));return a.join("")}function c(){return s(16)}function u(){return s(32)}},7056:(e,t,r)=>{r.d(t,{Bq:()=>n,Hb:()=>o,oD:()=>i});const n="NRBA",i=144e5,o=18e5},7894:(e,t,r)=>{function n(){return Math.round(performance.now())}r.d(t,{z:()=>n})},7243:(e,t,r)=>{r.d(t,{e:()=>o});var n=r(385),i={};function o(e){if(e in i)return i[e];if(0===(e||"").indexOf("data:"))return{protocol:"data"};let t;var r=n._A?.location,o={};if(n.il)t=document.createElement("a"),t.href=e;else try{t=new URL(e,r.href)}catch(e){return o}o.port=t.port;var a=t.href.split("://");!o.port&&a[1]&&(o.port=a[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===a[0]?"443":"80"),o.hostname=t.hostname||r.hostname,o.pathname=t.pathname,o.protocol=a[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var s=!t.protocol||":"===t.protocol||t.protocol===r.protocol,c=t.hostname===r.hostname&&t.port===r.port;return o.sameOrigin=s&&(!t.hostname||c),"/"===o.pathname&&(i[e]=o),o}},50:(e,t,r)=>{function n(e,t){"function"==typeof console.warn&&(console.warn("New Relic: ".concat(e)),t&&console.warn(t))}r.d(t,{Z:()=>n})},2587:(e,t,r)=>{r.d(t,{N:()=>c,T:()=>u});var n=r(2177),i=r(5546),o=r(8e3),a=r(3325);const s={stn:[a.D.sessionTrace],err:[a.D.jserrors,a.D.metrics],ins:[a.D.pageAction],spa:[a.D.spa],sr:[a.D.sessionReplay,a.D.sessionTrace]};function c(e,t){const r=n.ee.get(t);e&&"object"==typeof e&&(Object.entries(e).forEach((e=>{let[t,n]=e;void 0===u[t]&&(s[t]?s[t].forEach((e=>{n?(0,i.p)("feat-"+t,[],void 0,e,r):(0,i.p)("block-"+t,[],void 0,e,r),(0,i.p)("rumresp-"+t,[Boolean(n)],void 0,e,r)})):n&&(0,i.p)("feat-"+t,[],void 0,void 0,r),u[t]=Boolean(n))})),Object.keys(s).forEach((e=>{void 0===u[e]&&(s[e]?.forEach((t=>(0,i.p)("rumresp-"+e,[!1],void 0,t,r))),u[e]=!1)})),(0,o.L)(t,a.D.pageViewEvent))}const u={}},2210:(e,t,r)=>{r.d(t,{X:()=>i});var n=Object.prototype.hasOwnProperty;function i(e,t,r){if(n.call(e,t))return e[t];var i=r();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,t,{value:i,writable:!0,enumerable:!1}),i}catch(e){}return e[t]=i,i}},1284:(e,t,r)=>{r.d(t,{D:()=>n});const n=(e,t)=>Object.entries(e||{}).map((e=>{let[r,n]=e;return t(r,n)}))},4351:(e,t,r)=>{r.d(t,{P:()=>o});var n=r(2177);const i=()=>{const e=new WeakSet;return(t,r)=>{if("object"==typeof r&&null!==r){if(e.has(r))return;e.add(r)}return r}};function o(e){try{return JSON.stringify(e,i())}catch(e){try{n.ee.emit("internal-error",[e])}catch(e){}}}},3960:(e,t,r)=>{r.d(t,{K:()=>a,b:()=>o});var n=r(3239);function i(){return"undefined"==typeof document||"complete"===document.readyState}function o(e,t){if(i())return e();(0,n.bP)("load",e,t)}function a(e){if(i())return e();(0,n.iz)("DOMContentLoaded",e)}},8632:(e,t,r)=>{r.d(t,{EZ:()=>u,Qy:()=>c,ce:()=>o,fP:()=>a,gG:()=>d,mF:()=>s});var n=r(7894),i=r(385);const o={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net"};function a(){return i._A.NREUM||(i._A.NREUM={}),void 0===i._A.newrelic&&(i._A.newrelic=i._A.NREUM),i._A.NREUM}function s(){let e=a();return e.o||(e.o={ST:i._A.setTimeout,SI:i._A.setImmediate,CT:i._A.clearTimeout,XHR:i._A.XMLHttpRequest,REQ:i._A.Request,EV:i._A.Event,PR:i._A.Promise,MO:i._A.MutationObserver,FETCH:i._A.fetch}),e}function c(e,t,r){let i=a();const o=i.initializedAgents||{},s=o[e]||{};return Object.keys(s).length||(s.initializedAt={ms:(0,n.z)(),date:new Date}),i.initializedAgents={...o,[e]:{...s,[r]:t}},i}function u(e,t){a()[e]=t}function d(){return function(){let e=a();const t=e.info||{};e.info={beacon:o.beacon,errorBeacon:o.errorBeacon,...t}}(),function(){let e=a();const t=e.init||{};e.init={...t}}(),s(),function(){let e=a();const t=e.loader_config||{};e.loader_config={...t}}(),a()}},7956:(e,t,r)=>{r.d(t,{N:()=>i});var n=r(3239);function i(e){let t=arguments.length>1&&void 0!==arguments[1]&&arguments[1],r=arguments.length>2?arguments[2]:void 0,i=arguments.length>3?arguments[3]:void 0;return void(0,n.iz)("visibilitychange",(function(){if(t)return void("hidden"==document.visibilityState&&e());e(document.visibilityState)}),r,i)}},1214:(e,t,r)=>{r.d(t,{em:()=>v,u5:()=>N,QU:()=>S,_L:()=>I,Gm:()=>L,Lg:()=>M,gy:()=>U,BV:()=>Q,Kf:()=>ee});var n=r(2177);const i="nr@original";var o=Object.prototype.hasOwnProperty,a=!1;function s(e,t){return e||(e=n.ee),r.inPlace=function(e,t,n,i,o){n||(n="");var a,s,c,u="-"===n.charAt(0);for(c=0;c<t.length;c++)d(a=e[s=t[c]])||(e[s]=r(a,u?s+n:n,i,s,o))},r.flag=i,r;function r(t,r,n,a,s){return d(t)?t:(r||(r=""),nrWrapper[i]=t,u(t,nrWrapper,e),nrWrapper);function nrWrapper(){var i,u,d,f;try{u=this,i=[...arguments],d="function"==typeof n?n(i,u):n||{}}catch(t){c([t,"",[i,u,a],d],e)}o(r+"start",[i,u,a],d,s);try{return f=t.apply(u,i)}catch(e){throw o(r+"err",[i,u,e],d,s),e}finally{o(r+"end",[i,u,f],d,s)}}}function o(r,n,i,o){if(!a||t){var s=a;a=!0;try{e.emit(r,n,i,t,o)}catch(t){c([t,r,n,i],e)}a=s}}}function c(e,t){t||(t=n.ee);try{t.emit("internal-error",e)}catch(e){}}function u(e,t,r){if(Object.defineProperty&&Object.keys)try{return Object.keys(e).forEach((function(r){Object.defineProperty(t,r,{get:function(){return e[r]},set:function(t){return e[r]=t,t}})})),t}catch(e){c([e],r)}for(var n in e)o.call(e,n)&&(t[n]=e[n]);return t}function d(e){return!(e&&e instanceof Function&&e.apply&&!e[i])}var f=r(2210),l=r(385);const h={},g=XMLHttpRequest,p="addEventListener",m="removeEventListener";function v(e){var t=function(e){return(e||n.ee).get("events")}(e);if(h[t.debugId]++)return t;h[t.debugId]=1;var r=s(t,!0);function i(e){r.inPlace(e,[p,m],"-",o)}function o(e,t){return e[1]}return"getPrototypeOf"in Object&&(l.il&&b(document,i),b(l._A,i),b(g.prototype,i)),t.on(p+"-start",(function(e,t){var n=e[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var i=(0,f.X)(n,"nr@wrapped",(function(){var e={object:function(){if("function"!=typeof n.handleEvent)return;return n.handleEvent.apply(n,arguments)},function:n}[typeof n];return e?r(e,"fn-",null,e.name||"anonymous"):n}));this.wrapped=e[1]=i}})),t.on(m+"-start",(function(e){e[1]=this.wrapped||e[1]})),t}function b(e,t){let r=e;for(;"object"==typeof r&&!Object.prototype.hasOwnProperty.call(r,p);)r=Object.getPrototypeOf(r);for(var n=arguments.length,i=new Array(n>2?n-2:0),o=2;o<n;o++)i[o-2]=arguments[o];r&&t(r,...i)}var y="fetch-",w=y+"body-",x=["arrayBuffer","blob","json","text","formData"],A=l._A.Request,E=l._A.Response,T="prototype",_="nr@context";const D={};function N(e){const t=function(e){return(e||n.ee).get("fetch")}(e);if(!(A&&E&&l._A.fetch))return t;if(D[t.debugId]++)return t;function r(e,r,n){var i=e[r];"function"==typeof i&&(e[r]=function(){var e,r=[...arguments],o={};t.emit(n+"before-start",[r],o),o[_]&&o[_].dt&&(e=o[_].dt);var a=i.apply(this,r);return t.emit(n+"start",[r,e],a),a.then((function(e){return t.emit(n+"end",[null,e],a),e}),(function(e){throw t.emit(n+"end",[e],a),e}))})}return D[t.debugId]=1,x.forEach((e=>{r(A[T],e,w),r(E[T],e,w)})),r(l._A,"fetch",y),t.on(y+"end",(function(e,r){var n=this;if(r){var i=r.headers.get("content-length");null!==i&&(n.rxSize=i),t.emit(y+"done",[null,r],n)}else t.emit(y+"done",[e],n)})),t}const O={},j=["pushState","replaceState"];function S(e){const t=function(e){return(e||n.ee).get("history")}(e);return!l.il||O[t.debugId]++||(O[t.debugId]=1,s(t).inPlace(window.history,j,"-")),t}var P=r(3239);const C={},R=["appendChild","insertBefore","replaceChild"];function I(e){const t=function(e){return(e||n.ee).get("jsonp")}(e);if(!l.il||C[t.debugId])return t;C[t.debugId]=!0;var r=s(t),i=/[?&](?:callback|cb)=([^&#]+)/,o=/(.*)\.([^.]+)/,a=/^(\w+)(\.|$)(.*)$/;function c(e,t){var r=e.match(a),n=r[1],i=r[3];return i?c(i,t[n]):t[n]}return r.inPlace(Node.prototype,R,"dom-"),t.on("dom-start",(function(e){!function(e){if(!e||"string"!=typeof e.nodeName||"script"!==e.nodeName.toLowerCase())return;if("function"!=typeof e.addEventListener)return;var n=(a=e.src,s=a.match(i),s?s[1]:null);var a,s;if(!n)return;var u=function(e){var t=e.match(o);if(t&&t.length>=3)return{key:t[2],parent:c(t[1],window)};return{key:e,parent:window}}(n);if("function"!=typeof u.parent[u.key])return;var d={};function f(){t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}function l(){t.emit("jsonp-error",[],d),t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}r.inPlace(u.parent,[u.key],"cb-",d),e.addEventListener("load",f,(0,P.m$)(!1)),e.addEventListener("error",l,(0,P.m$)(!1)),t.emit("new-jsonp",[e.src],d)}(e[0])})),t}var k=r(5763);const H={};function L(e){const t=function(e){return(e||n.ee).get("mutation")}(e);if(!l.il||H[t.debugId])return t;H[t.debugId]=!0;var r=s(t),i=k.Yu.MO;return i&&(window.MutationObserver=function(e){return this instanceof i?new i(r(e,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype),t}const z={};function M(e){const t=function(e){return(e||n.ee).get("promise")}(e);if(z[t.debugId])return t;z[t.debugId]=!0;var r=n.c,o=s(t),a=k.Yu.PR;return a&&function(){function e(r){var n=t.context(),i=o(r,"executor-",n,null,!1);const s=Reflect.construct(a,[i],e);return t.context(s).getCtx=function(){return n},s}l._A.Promise=e,Object.defineProperty(e,"name",{value:"Promise"}),e.toString=function(){return a.toString()},Object.setPrototypeOf(e,a),["all","race"].forEach((function(r){const n=a[r];e[r]=function(e){let i=!1;[...e||[]].forEach((e=>{this.resolve(e).then(a("all"===r),a(!1))}));const o=n.apply(this,arguments);return o;function a(e){return function(){t.emit("propagate",[null,!i],o,!1,!1),i=i||!e}}}})),["resolve","reject"].forEach((function(r){const n=a[r];e[r]=function(e){const r=n.apply(this,arguments);return e!==r&&t.emit("propagate",[e,!0],r,!1,!1),r}})),e.prototype=a.prototype;const n=a.prototype.then;a.prototype.then=function(){var e=this,i=r(e);i.promise=e;for(var a=arguments.length,s=new Array(a),c=0;c<a;c++)s[c]=arguments[c];s[0]=o(s[0],"cb-",i,null,!1),s[1]=o(s[1],"cb-",i,null,!1);const u=n.apply(this,s);return i.nextPromise=u,t.emit("propagate",[e,!0],u,!1,!1),u},a.prototype.then[i]=n,t.on("executor-start",(function(e){e[0]=o(e[0],"resolve-",this,null,!1),e[1]=o(e[1],"resolve-",this,null,!1)})),t.on("executor-err",(function(e,t,r){e[1](r)})),t.on("cb-end",(function(e,r,n){t.emit("propagate",[n,!0],this.nextPromise,!1,!1)})),t.on("propagate",(function(e,r,n){this.getCtx&&!r||(this.getCtx=function(){if(e instanceof Promise)var r=t.context(e);return r&&r.getCtx?r.getCtx():this})}))}(),t}const B={},F="requestAnimationFrame";function U(e){const t=function(e){return(e||n.ee).get("raf")}(e);if(!l.il||B[t.debugId]++)return t;B[t.debugId]=1;var r=s(t);return r.inPlace(window,[F],"raf-"),t.on("raf-start",(function(e){e[0]=r(e[0],"fn-")})),t}const q={},G="setTimeout",V="setInterval",X="clearTimeout",W="-start",Z="-",$=[G,"setImmediate",V,X,"clearImmediate"];function Q(e){const t=function(e){return(e||n.ee).get("timer")}(e);if(q[t.debugId]++)return t;q[t.debugId]=1;var r=s(t);return r.inPlace(l._A,$.slice(0,2),G+Z),r.inPlace(l._A,$.slice(2,3),V+Z),r.inPlace(l._A,$.slice(3),X+Z),t.on(V+W,(function(e,t,n){e[0]=r(e[0],"fn-",null,n)})),t.on(G+W,(function(e,t,n){this.method=n,this.timerDuration=isNaN(e[1])?0:+e[1],e[0]=r(e[0],"fn-",this,n)})),t}var Y=r(50);const K={},J=["open","send"];function ee(e){var t=e||n.ee;const r=function(e){return(e||n.ee).get("xhr")}(t);if(K[r.debugId]++)return r;K[r.debugId]=1,v(t);var i=s(r),o=k.Yu.XHR,a=k.Yu.MO,c=k.Yu.PR,u=k.Yu.SI,d="readystatechange",f=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],h=[],g=l._A.XMLHttpRequest.listeners,p=l._A.XMLHttpRequest=function(e){var t=new o(e);function n(){try{r.emit("new-xhr",[t],t),t.addEventListener(d,b,(0,P.m$)(!1))}catch(e){(0,Y.Z)("An error occured while intercepting XHR",e);try{r.emit("internal-error",[e])}catch(e){}}}return this.listeners=g?[...g,n]:[n],this.listeners.forEach((e=>e())),t};function m(e,t){i.inPlace(t,["onreadystatechange"],"fn-",E)}function b(){var e=this,t=r.context(e);e.readyState>3&&!t.resolved&&(t.resolved=!0,r.emit("xhr-resolved",[],e)),i.inPlace(e,f,"fn-",E)}if(function(e,t){for(var r in e)t[r]=e[r]}(o,p),p.prototype=o.prototype,i.inPlace(p.prototype,J,"-xhr-",E),r.on("send-xhr-start",(function(e,t){m(e,t),function(e){h.push(e),a&&(y?y.then(A):u?u(A):(w=-w,x.data=w))}(t)})),r.on("open-xhr-start",m),a){var y=c&&c.resolve();if(!u&&!c){var w=1,x=document.createTextNode(w);new a(A).observe(x,{characterData:!0})}}else t.on("fn-end",(function(e){e[0]&&e[0].type===d||A()}));function A(){for(var e=0;e<h.length;e++)m(0,h[e]);h.length&&(h=[])}function E(e,t){return t}return r}},7825:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.ajax},6660:(e,t,r)=>{r.d(t,{A:()=>i,t:()=>n});const n=r(3325).D.jserrors,i="nr@seenError"},3081:(e,t,r)=>{r.d(t,{gF:()=>o,mY:()=>i,t9:()=>n,vz:()=>s,xS:()=>a});const n=r(3325).D.metrics,i="sm",o="cm",a="storeSupportabilityMetrics",s="storeEventMetrics"},4649:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageAction},7633:(e,t,r)=>{r.d(t,{Dz:()=>i,OJ:()=>a,qw:()=>o,t9:()=>n});const n=r(3325).D.pageViewEvent,i="firstbyte",o="domcontent",a="windowload"},9251:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageViewTiming},3614:(e,t,r)=>{r.d(t,{BST_RESOURCE:()=>i,END:()=>s,FEATURE_NAME:()=>n,FN_END:()=>u,FN_START:()=>c,PUSH_STATE:()=>d,RESOURCE:()=>o,START:()=>a});const n=r(3325).D.sessionTrace,i="bstResource",o="resource",a="-start",s="-end",c="fn"+a,u="fn"+s,d="pushState"},7836:(e,t,r)=>{r.d(t,{BODY:()=>A,CB_END:()=>E,CB_START:()=>u,END:()=>x,FEATURE_NAME:()=>i,FETCH:()=>_,FETCH_BODY:()=>v,FETCH_DONE:()=>m,FETCH_START:()=>p,FN_END:()=>c,FN_START:()=>s,INTERACTION:()=>l,INTERACTION_API:()=>d,INTERACTION_EVENTS:()=>o,JSONP_END:()=>b,JSONP_NODE:()=>g,JS_TIME:()=>T,MAX_TIMER_BUDGET:()=>a,REMAINING:()=>f,SPA_NODE:()=>h,START:()=>w,originalSetTimeout:()=>y});var n=r(5763);const i=r(3325).D.spa,o=["click","submit","keypress","keydown","keyup","change"],a=999,s="fn-start",c="fn-end",u="cb-start",d="api-ixn-",f="remaining",l="interaction",h="spaNode",g="jsonpNode",p="fetch-start",m="fetch-done",v="fetch-body-",b="jsonp-end",y=n.Yu.ST,w="-start",x="-end",A="-body",E="cb"+x,T="jsTime",_="fetch"},5938:(e,t,r)=>{r.d(t,{W:()=>o});var n=r(5763),i=r(2177);class o{constructor(e,t,r){this.agentIdentifier=e,this.aggregator=t,this.ee=i.ee.get(e,(0,n.OP)(this.agentIdentifier).isolatedBacklog),this.featureName=r,this.blocked=!1}}},9144:(e,t,r)=>{r.d(t,{j:()=>m});var n=r(3325),i=r(5763),o=r(5546),a=r(2177),s=r(7894),c=r(8e3),u=r(3960),d=r(385),f=r(50),l=r(3081),h=r(8632);function g(){const e=(0,h.gG)();["setErrorHandler","finished","addToTrace","inlineHit","addRelease","addPageAction","setCurrentRouteName","setPageViewName","setCustomAttribute","interaction","noticeError","setUserId"].forEach((t=>{e[t]=function(){for(var r=arguments.length,n=new Array(r),i=0;i<r;i++)n[i]=arguments[i];return function(t){for(var r=arguments.length,n=new Array(r>1?r-1:0),i=1;i<r;i++)n[i-1]=arguments[i];let o=[];return Object.values(e.initializedAgents).forEach((e=>{e.exposed&&e.api[t]&&o.push(e.api[t](...n))})),o.length>1?o:o[0]}(t,...n)}}))}var p=r(2587);function m(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},m=arguments.length>2?arguments[2]:void 0,v=arguments.length>3?arguments[3]:void 0,{init:b,info:y,loader_config:w,runtime:x={loaderType:m},exposed:A=!0}=t;const E=(0,h.gG)();y||(b=E.init,y=E.info,w=E.loader_config),(0,i.Dg)(e,b||{}),(0,i.GE)(e,w||{}),(0,i.sU)(e,x),y.jsAttributes??={},d.v6&&(y.jsAttributes.isWorker=!0),(0,i.CX)(e,y),g();const T=function(e,t){t||(0,c.R)(e,"api");const h={};var g=a.ee.get(e),p=g.get("tracer"),m="api-",v=m+"ixn-";function b(t,r,n,o){const a=(0,i.C5)(e);return null===r?delete a.jsAttributes[t]:(0,i.CX)(e,{...a,jsAttributes:{...a.jsAttributes,[t]:r}}),x(m,n,!0,o||null===r?"session":void 0)(t,r)}function y(){}["setErrorHandler","finished","addToTrace","inlineHit","addRelease"].forEach((e=>h[e]=x(m,e,!0,"api"))),h.addPageAction=x(m,"addPageAction",!0,n.D.pageAction),h.setCurrentRouteName=x(m,"routeName",!0,n.D.spa),h.setPageViewName=function(t,r){if("string"==typeof t)return"/"!==t.charAt(0)&&(t="/"+t),(0,i.OP)(e).customTransaction=(r||"http://custom.transaction")+t,x(m,"setPageViewName",!0)()},h.setCustomAttribute=function(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2];if("string"==typeof e){if(["string","number"].includes(typeof t)||null===t)return b(e,t,"setCustomAttribute",r);(0,f.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t,"> was provided."))}else(0,f.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.setUserId=function(e){if("string"==typeof e||null===e)return b("enduser.id",e,"setUserId",!0);(0,f.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.interaction=function(){return(new y).get()};var w=y.prototype={createTracer:function(e,t){var r={},i=this,a="function"==typeof t;return(0,o.p)(v+"tracer",[(0,s.z)(),e,r],i,n.D.spa,g),function(){if(p.emit((a?"":"no-")+"fn-start",[(0,s.z)(),i,a],r),a)try{return t.apply(this,arguments)}catch(e){throw p.emit("fn-err",[arguments,this,"string"==typeof e?new Error(e):e],r),e}finally{p.emit("fn-end",[(0,s.z)()],r)}}}};function x(e,t,r,i){return function(){return(0,o.p)(l.xS,["API/"+t+"/called"],void 0,n.D.metrics,g),i&&(0,o.p)(e+t,[(0,s.z)(),...arguments],r?null:this,i,g),r?void 0:this}}function A(){r.e(439).then(r.bind(r,7438)).then((t=>{let{setAPI:r}=t;r(e),(0,c.L)(e,"api")})).catch((()=>(0,f.Z)("Downloading runtime APIs failed...")))}return["actionText","setName","setAttribute","save","ignore","onEnd","getContext","end","get"].forEach((e=>{w[e]=x(v,e,void 0,n.D.spa)})),h.noticeError=function(e,t){"string"==typeof e&&(e=new Error(e)),(0,o.p)(l.xS,["API/noticeError/called"],void 0,n.D.metrics,g),(0,o.p)("err",[e,(0,s.z)(),!1,t],void 0,n.D.jserrors,g)},d.il?(0,u.b)((()=>A()),!0):A(),h}(e,v);return(0,h.Qy)(e,T,"api"),(0,h.Qy)(e,A,"exposed"),(0,h.EZ)("activatedFeatures",p.T),T}},3325:(e,t,r)=>{r.d(t,{D:()=>n,p:()=>i});const n={ajax:"ajax",jserrors:"jserrors",metrics:"metrics",pageAction:"page_action",pageViewEvent:"page_view_event",pageViewTiming:"page_view_timing",sessionReplay:"session_replay",sessionTrace:"session_trace",spa:"spa"},i={[n.pageViewEvent]:1,[n.pageViewTiming]:2,[n.metrics]:3,[n.jserrors]:4,[n.ajax]:5,[n.sessionTrace]:6,[n.pageAction]:7,[n.spa]:8,[n.sessionReplay]:9}}},n={};function i(e){var t=n[e];if(void 0!==t)return t.exports;var o=n[e]={exports:{}};return r[e](o,o.exports,i),o.exports}i.m=r,i.d=(e,t)=>{for(var r in t)i.o(t,r)&&!i.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},i.f={},i.e=e=>Promise.all(Object.keys(i.f).reduce(((t,r)=>(i.f[r](e,t),t)),[])),i.u=e=>(({78:"page_action-aggregate",147:"metrics-aggregate",242:"session-manager",317:"jserrors-aggregate",348:"page_view_timing-aggregate",412:"lazy-feature-loader",439:"async-api",538:"recorder",590:"session_replay-aggregate",675:"compressor",733:"session_trace-aggregate",786:"page_view_event-aggregate",873:"spa-aggregate",898:"ajax-aggregate"}[e]||e)+"."+{78:"ac76d497",147:"3dc53903",148:"1a20d5fe",242:"2a64278a",317:"49e41428",348:"bd6de33a",412:"2f55ce66",439:"30bd804e",538:"1b18459f",590:"cf0efb30",675:"ae9f91a8",733:"83105561",786:"06482edd",860:"03a8b7a5",873:"e6b09d52",898:"998ef92b"}[e]+"-1.236.0.min.js"),i.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),e={},t="NRBA:",i.l=(r,n,o,a)=>{if(e[r])e[r].push(n);else{var s,c;if(void 0!==o)for(var u=document.getElementsByTagName("script"),d=0;d<u.length;d++){var f=u[d];if(f.getAttribute("src")==r||f.getAttribute("data-webpack")==t+o){s=f;break}}s||(c=!0,(s=document.createElement("script")).charset="utf-8",s.timeout=120,i.nc&&s.setAttribute("nonce",i.nc),s.setAttribute("data-webpack",t+o),s.src=r),e[r]=[n];var l=(t,n)=>{s.onerror=s.onload=null,clearTimeout(h);var i=e[r];if(delete e[r],s.parentNode&&s.parentNode.removeChild(s),i&&i.forEach((e=>e(n))),t)return t(n)},h=setTimeout(l.bind(null,void 0,{type:"timeout",target:s}),12e4);s.onerror=l.bind(null,s.onerror),s.onload=l.bind(null,s.onload),c&&document.head.appendChild(s)}},i.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},i.j=364,i.p="https://js-agent.newrelic.com/",(()=>{var e={364:0,953:0};i.f.j=(t,r)=>{var n=i.o(e,t)?e[t]:void 0;if(0!==n)if(n)r.push(n[2]);else{var o=new Promise(((r,i)=>n=e[t]=[r,i]));r.push(n[2]=o);var a=i.p+i.u(t),s=new Error;i.l(a,(r=>{if(i.o(e,t)&&(0!==(n=e[t])&&(e[t]=void 0),n)){var o=r&&("load"===r.type?"missing":r.type),a=r&&r.target&&r.target.src;s.message="Loading chunk "+t+" failed.\n("+o+": "+a+")",s.name="ChunkLoadError",s.type=o,s.request=a,n[1](s)}}),"chunk-"+t,t)}};var t=(t,r)=>{var n,o,[a,s,c]=r,u=0;if(a.some((t=>0!==e[t]))){for(n in s)i.o(s,n)&&(i.m[n]=s[n]);if(c)c(i)}for(t&&t(r);u<a.length;u++)o=a[u],i.o(e,o)&&e[o]&&e[o][0](),e[o]=0},r=window.webpackChunkNRBA=window.webpackChunkNRBA||[];r.forEach(t.bind(null,0)),r.push=t.bind(null,r.push.bind(r))})();var o={};(()=>{i.r(o);var e=i(3325),t=i(5763);const r=Object.values(e.D);function n(e){const n={};return r.forEach((r=>{n[r]=function(e,r){return!1!==(0,t.Mt)(r,"".concat(e,".enabled"))}(r,e)})),n}var a=i(9144);var s=i(5546),c=i(385),u=i(8e3),d=i(5938),f=i(3960),l=i(50);class h extends d.W{constructor(e,t,r){let n=!(arguments.length>3&&void 0!==arguments[3])||arguments[3];super(e,t,r),this.auto=n,this.abortHandler,this.featAggregate,this.onAggregateImported,n&&(0,u.R)(e,r)}importAggregator(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};if(this.featAggregate||!this.auto)return;const r=c.il&&!0===(0,t.Mt)(this.agentIdentifier,"privacy.cookies_enabled");let n;this.onAggregateImported=new Promise((e=>{n=e}));const o=async()=>{let t;try{if(r){const{setupAgentSession:e}=await Promise.all([i.e(860),i.e(242)]).then(i.bind(i,3228));t=e(this.agentIdentifier)}}catch(e){(0,l.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.",e)}try{if(!this.shouldImportAgg(this.featureName,t))return void(0,u.L)(this.agentIdentifier,this.featureName);const{lazyFeatureLoader:r}=await i.e(412).then(i.bind(i,8582)),{Aggregate:o}=await r(this.featureName,"aggregate");this.featAggregate=new o(this.agentIdentifier,this.aggregator,e),n(!0)}catch(e){(0,l.Z)("Downloading and initializing ".concat(this.featureName," failed..."),e),this.abortHandler?.(),n(!1)}};c.il?(0,f.b)((()=>o()),!0):o()}shouldImportAgg(r,n){return r!==e.D.sessionReplay||!1!==(0,t.Mt)(this.agentIdentifier,"session_trace.enabled")&&(!!n?.isNew||!!n?.state.sessionReplay)}}var g=i(7633),p=i(7894);class m extends h{static featureName=g.t9;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];if(super(r,n,g.t9,i),("undefined"==typeof PerformanceNavigationTiming||c.Tt)&&"undefined"!=typeof PerformanceTiming){const n=(0,t.OP)(r);n[g.Dz]=Math.max(Date.now()-n.offset,0),(0,f.K)((()=>n[g.qw]=Math.max((0,p.z)()-n[g.Dz],0))),(0,f.b)((()=>{const t=(0,p.z)();n[g.OJ]=Math.max(t-n[g.Dz],0),(0,s.p)("timing",["load",t],void 0,e.D.pageViewTiming,this.ee)}))}this.importAggregator()}}var v=i(1117),b=i(1284);class y extends v.w{constructor(e){super(e),this.aggregatedData={}}store(e,t,r,n,i){var o=this.getBucket(e,t,r,i);return o.metrics=function(e,t){t||(t={count:0});return t.count+=1,(0,b.D)(e,(function(e,r){t[e]=w(r,t[e])})),t}(n,o.metrics),o}merge(e,t,r,n,i){var o=this.getBucket(e,t,n,i);if(o.metrics){var a=o.metrics;a.count+=r.count,(0,b.D)(r,(function(e,t){if("count"!==e){var n=a[e],i=r[e];i&&!i.c?a[e]=w(i.t,n):a[e]=function(e,t){if(!t)return e;t.c||(t=x(t.t));return t.min=Math.min(e.min,t.min),t.max=Math.max(e.max,t.max),t.t+=e.t,t.sos+=e.sos,t.c+=e.c,t}(i,a[e])}}))}else o.metrics=r}storeMetric(e,t,r,n){var i=this.getBucket(e,t,r);return i.stats=w(n,i.stats),i}getBucket(e,t,r,n){this.aggregatedData[e]||(this.aggregatedData[e]={});var i=this.aggregatedData[e][t];return i||(i=this.aggregatedData[e][t]={params:r||{}},n&&(i.custom=n)),i}get(e,t){return t?this.aggregatedData[e]&&this.aggregatedData[e][t]:this.aggregatedData[e]}take(e){for(var t={},r="",n=!1,i=0;i<e.length;i++)t[r=e[i]]=A(this.aggregatedData[r]),t[r].length&&(n=!0),delete this.aggregatedData[r];return n?t:null}}function w(e,t){return null==e?function(e){e?e.c++:e={c:1};return e}(t):t?(t.c||(t=x(t.t)),t.c+=1,t.t+=e,t.sos+=e*e,e>t.max&&(t.max=e),e<t.min&&(t.min=e),t):{t:e}}function x(e){return{t:e,min:e,max:e,sos:e*e,c:1}}function A(e){return"object"!=typeof e?[]:(0,b.D)(e,E)}function E(e,t){return t}var T=i(8632),_=i(4402),D=i(4351);var N=i(7956),O=i(3239),j=i(9251);class S extends h{static featureName=j.t;constructor(e,r){let n=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,r,j.t,n),c.il&&((0,t.OP)(e).initHidden=Boolean("hidden"===document.visibilityState),(0,N.N)((()=>(0,s.p)("docHidden",[(0,p.z)()],void 0,j.t,this.ee)),!0),(0,O.bP)("pagehide",(()=>(0,s.p)("winPagehide",[(0,p.z)()],void 0,j.t,this.ee))),this.importAggregator())}}var P=i(3081);class C extends h{static featureName=P.t9;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,P.t9,r),this.importAggregator()}}var R,I=i(2210),k=i(1214),H=i(2177),L={};try{R=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(L.console=!0,-1!==R.indexOf("dev")&&(L.dev=!0),-1!==R.indexOf("nr_dev")&&(L.nrDev=!0))}catch(e){}function z(e){try{L.console&&z(e)}catch(e){}}L.nrDev&&H.ee.on("internal-error",(function(e){z(e.stack)})),L.dev&&H.ee.on("fn-err",(function(e,t,r){z(r.stack)})),L.dev&&(z("NR AGENT IN DEVELOPMENT MODE"),z("flags: "+(0,b.D)(L,(function(e,t){return e})).join(", ")));var M=i(6660);class B extends h{static featureName=M.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,M.t,i),this.skipNext=0;try{this.removeOnAbort=new AbortController}catch(e){}const o=this;o.ee.on("fn-start",(function(e,t,r){o.abortHandler&&(o.skipNext+=1)})),o.ee.on("fn-err",(function(t,r,n){o.abortHandler&&!n[M.A]&&((0,I.X)(n,M.A,(function(){return!0})),this.thrown=!0,(0,s.p)("err",[n,(0,p.z)()],void 0,e.D.jserrors,o.ee))})),o.ee.on("fn-end",(function(){o.abortHandler&&!this.thrown&&o.skipNext>0&&(o.skipNext-=1)})),o.ee.on("internal-error",(function(t){(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,o.ee)})),this.origOnerror=c._A.onerror,c._A.onerror=this.onerrorHandler.bind(this),c._A.addEventListener("unhandledrejection",(t=>{const r=function(e){let t="Unhandled Promise Rejection: ";if(e instanceof Error)try{return e.message=t+e.message,e}catch(t){return e}if(void 0===e)return new Error(t);try{return new Error(t+(0,D.P)(e))}catch(e){return new Error(t)}}(t.reason);(0,s.p)("err",[r,(0,p.z)(),!1,{unhandledPromiseRejection:1}],void 0,e.D.jserrors,this.ee)}),(0,O.m$)(!1,this.removeOnAbort?.signal)),(0,k.gy)(this.ee),(0,k.BV)(this.ee),(0,k.em)(this.ee),(0,t.OP)(r).xhrWrappable&&(0,k.Kf)(this.ee),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}onerrorHandler(t,r,n,i,o){"function"==typeof this.origOnerror&&this.origOnerror(...arguments);try{this.skipNext?this.skipNext-=1:(0,s.p)("err",[o||new F(t,r,n),(0,p.z)()],void 0,e.D.jserrors,this.ee)}catch(t){try{(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,this.ee)}catch(e){}}return!1}}function F(e,t,r){this.message=e||"Uncaught error with no additional information",this.sourceURL=t,this.line=r}let U=1;const q="nr@id";function G(e){const t=typeof e;return!e||"object"!==t&&"function"!==t?-1:e===c._A?0:(0,I.X)(e,q,(function(){return U++}))}function V(e){if("string"==typeof e&&e.length)return e.length;if("object"==typeof e){if("undefined"!=typeof ArrayBuffer&&e instanceof ArrayBuffer&&e.byteLength)return e.byteLength;if("undefined"!=typeof Blob&&e instanceof Blob&&e.size)return e.size;if(!("undefined"!=typeof FormData&&e instanceof FormData))try{return(0,D.P)(e).length}catch(e){return}}}var X=i(7243);class W{constructor(e){this.agentIdentifier=e,this.generateTracePayload=this.generateTracePayload.bind(this),this.shouldGenerateTrace=this.shouldGenerateTrace.bind(this)}generateTracePayload(e){if(!this.shouldGenerateTrace(e))return null;var r=(0,t.DL)(this.agentIdentifier);if(!r)return null;var n=(r.accountID||"").toString()||null,i=(r.agentID||"").toString()||null,o=(r.trustKey||"").toString()||null;if(!n||!i)return null;var a=(0,_.M)(),s=(0,_.Ht)(),c=Date.now(),u={spanId:a,traceId:s,timestamp:c};return(e.sameOrigin||this.isAllowedOrigin(e)&&this.useTraceContextHeadersForCors())&&(u.traceContextParentHeader=this.generateTraceContextParentHeader(a,s),u.traceContextStateHeader=this.generateTraceContextStateHeader(a,c,n,i,o)),(e.sameOrigin&&!this.excludeNewrelicHeader()||!e.sameOrigin&&this.isAllowedOrigin(e)&&this.useNewrelicHeaderForCors())&&(u.newrelicHeader=this.generateTraceHeader(a,s,c,n,i,o)),u}generateTraceContextParentHeader(e,t){return"00-"+t+"-"+e+"-01"}generateTraceContextStateHeader(e,t,r,n,i){return i+"@nr=0-1-"+r+"-"+n+"-"+e+"----"+t}generateTraceHeader(e,t,r,n,i,o){if(!("function"==typeof c._A?.btoa))return null;var a={v:[0,1],d:{ty:"Browser",ac:n,ap:i,id:e,tr:t,ti:r}};return o&&n!==o&&(a.d.tk=o),btoa((0,D.P)(a))}shouldGenerateTrace(e){return this.isDtEnabled()&&this.isAllowedOrigin(e)}isAllowedOrigin(e){var r=!1,n={};if((0,t.Mt)(this.agentIdentifier,"distributed_tracing")&&(n=(0,t.P_)(this.agentIdentifier).distributed_tracing),e.sameOrigin)r=!0;else if(n.allowed_origins instanceof Array)for(var i=0;i<n.allowed_origins.length;i++){var o=(0,X.e)(n.allowed_origins[i]);if(e.hostname===o.hostname&&e.protocol===o.protocol&&e.port===o.port){r=!0;break}}return r}isDtEnabled(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.enabled}excludeNewrelicHeader(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.exclude_newrelic_header}useNewrelicHeaderForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!1!==e.cors_use_newrelic_header}useTraceContextHeadersForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.cors_use_tracecontext_headers}}var Z=i(7825),$=["load","error","abort","timeout"],Q=$.length,Y=t.Yu.REQ,K=c._A.XMLHttpRequest;class J extends h{static featureName=Z.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,Z.t,i),(0,t.OP)(r).xhrWrappable&&(this.dt=new W(r),this.handler=(e,t,r,n)=>(0,s.p)(e,t,r,n,this.ee),(0,k.u5)(this.ee),(0,k.Kf)(this.ee),function(r,n,i,o){function a(e){var t=this;t.totalCbs=0,t.called=0,t.cbTime=0,t.end=E,t.ended=!1,t.xhrGuids={},t.lastSize=null,t.loadCaptureCalled=!1,t.params=this.params||{},t.metrics=this.metrics||{},e.addEventListener("load",(function(r){_(t,e)}),(0,O.m$)(!1)),c.IF||e.addEventListener("progress",(function(e){t.lastSize=e.loaded}),(0,O.m$)(!1))}function s(e){this.params={method:e[0]},T(this,e[1]),this.metrics={}}function u(e,n){var i=(0,t.DL)(r);i.xpid&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",i.xpid);var a=o.generateTracePayload(this.parsedOrigin);if(a){var s=!1;a.newrelicHeader&&(n.setRequestHeader("newrelic",a.newrelicHeader),s=!0),a.traceContextParentHeader&&(n.setRequestHeader("traceparent",a.traceContextParentHeader),a.traceContextStateHeader&&n.setRequestHeader("tracestate",a.traceContextStateHeader),s=!0),s&&(this.dt=a)}}function d(e,t){var r=this.metrics,i=e[0],o=this;if(r&&i){var a=V(i);a&&(r.txSize=a)}this.startTime=(0,p.z)(),this.listener=function(e){try{"abort"!==e.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==e.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof t.onload)&&"function"==typeof o.end)&&o.end(t)}catch(e){try{n.emit("internal-error",[e])}catch(e){}}};for(var s=0;s<Q;s++)t.addEventListener($[s],this.listener,(0,O.m$)(!1))}function f(e,t,r){this.cbTime+=e,t?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof r.onload||"function"!=typeof this.end||this.end(r)}function l(e,t){var r=""+G(e)+!!t;this.xhrGuids&&!this.xhrGuids[r]&&(this.xhrGuids[r]=!0,this.totalCbs+=1)}function h(e,t){var r=""+G(e)+!!t;this.xhrGuids&&this.xhrGuids[r]&&(delete this.xhrGuids[r],this.totalCbs-=1)}function g(){this.endTime=(0,p.z)()}function m(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-added",[e[1],e[2]],t)}function v(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-removed",[e[1],e[2]],t)}function b(e,t,r){t instanceof K&&("onload"===r&&(this.onload=!0),("load"===(e[0]&&e[0].type)||this.onload)&&(this.xhrCbStart=(0,p.z)()))}function y(e,t){this.xhrCbStart&&n.emit("xhr-cb-time",[(0,p.z)()-this.xhrCbStart,this.onload,t],t)}function w(e){var t,r=e[1]||{};"string"==typeof e[0]?t=e[0]:e[0]&&e[0].url?t=e[0].url:c._A?.URL&&e[0]&&e[0]instanceof URL&&(t=e[0].href),t&&(this.parsedOrigin=(0,X.e)(t),this.sameOrigin=this.parsedOrigin.sameOrigin);var n=o.generateTracePayload(this.parsedOrigin);if(n&&(n.newrelicHeader||n.traceContextParentHeader))if("string"==typeof e[0]||c._A?.URL&&e[0]&&e[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),s(i.headers,n)&&(this.dt=n),e.length>1?e[1]=i:e.push(i)}else e[0]&&e[0].headers&&s(e[0].headers,n)&&(this.dt=n);function s(e,t){var r=!1;return t.newrelicHeader&&(e.set("newrelic",t.newrelicHeader),r=!0),t.traceContextParentHeader&&(e.set("traceparent",t.traceContextParentHeader),t.traceContextStateHeader&&e.set("tracestate",t.traceContextStateHeader),r=!0),r}}function x(e,t){this.params={},this.metrics={},this.startTime=(0,p.z)(),this.dt=t,e.length>=1&&(this.target=e[0]),e.length>=2&&(this.opts=e[1]);var r,n=this.opts||{},i=this.target;"string"==typeof i?r=i:"object"==typeof i&&i instanceof Y?r=i.url:c._A?.URL&&"object"==typeof i&&i instanceof URL&&(r=i.href),T(this,r);var o=(""+(i&&i instanceof Y&&i.method||n.method||"GET")).toUpperCase();this.params.method=o,this.txSize=V(n.body)||0}function A(t,r){var n;this.endTime=(0,p.z)(),this.params||(this.params={}),this.params.status=r?r.status:0,"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var o={txSize:this.txSize,rxSize:n,duration:(0,p.z)()-this.startTime};i("xhr",[this.params,o,this.startTime,this.endTime,"fetch"],this,e.D.ajax)}function E(t){var r=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var o=0;o<Q;o++)t.removeEventListener($[o],this.listener,!1);r.aborted||(n.duration=(0,p.z)()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==r.status&&(r.status=0):_(this,t),n.cbTime=this.cbTime,i("xhr",[r,n,this.startTime,this.endTime,"xhr"],this,e.D.ajax))}}function T(e,t){var r=(0,X.e)(t),n=e.params;n.hostname=r.hostname,n.port=r.port,n.protocol=r.protocol,n.host=r.hostname+":"+r.port,n.pathname=r.pathname,e.parsedOrigin=r,e.sameOrigin=r.sameOrigin}function _(e,t){e.params.status=t.status;var r=function(e,t){var r=e.responseType;return"json"===r&&null!==t?t:"arraybuffer"===r||"blob"===r||"json"===r?V(e.response):"text"===r||""===r||void 0===r?V(e.responseText):void 0}(t,e.lastSize);if(r&&(e.metrics.rxSize=r),e.sameOrigin){var n=t.getResponseHeader("X-NewRelic-App-Data");n&&(e.params.cat=n.split(", ").pop())}e.loadCaptureCalled=!0}n.on("new-xhr",a),n.on("open-xhr-start",s),n.on("open-xhr-end",u),n.on("send-xhr-start",d),n.on("xhr-cb-time",f),n.on("xhr-load-added",l),n.on("xhr-load-removed",h),n.on("xhr-resolved",g),n.on("addEventListener-end",m),n.on("removeEventListener-end",v),n.on("fn-end",y),n.on("fetch-before-start",w),n.on("fetch-start",x),n.on("fn-start",b),n.on("fetch-done",A)}(r,this.ee,this.handler,this.dt),this.importAggregator())}}var ee=i(3614);const{BST_RESOURCE:te,RESOURCE:re,START:ne,END:ie,FEATURE_NAME:oe,FN_END:ae,FN_START:se,PUSH_STATE:ce}=ee;var ue=i(7836);const{FEATURE_NAME:de,START:fe,END:le,BODY:he,CB_END:ge,JS_TIME:pe,FETCH:me,FN_START:ve,CB_START:be,FN_END:ye}=ue;var we=i(4649);class xe extends h{static featureName=we.t;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,we.t,r),this.importAggregator()}}new class{constructor(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:(0,_.ky)(16);c._A?(this.agentIdentifier=t,this.sharedAggregator=new y({agentIdentifier:this.agentIdentifier}),this.features={},this.desiredFeatures=new Set(e.features||[]),this.desiredFeatures.add(m),Object.assign(this,(0,a.j)(this.agentIdentifier,e,e.loaderType||"agent")),this.start()):(0,l.Z)("Failed to initial the agent. Could not determine the runtime environment.")}get config(){return{info:(0,t.C5)(this.agentIdentifier),init:(0,t.P_)(this.agentIdentifier),loader_config:(0,t.DL)(this.agentIdentifier),runtime:(0,t.OP)(this.agentIdentifier)}}start(){const t="features";try{const r=n(this.agentIdentifier),i=[...this.desiredFeatures];i.sort(((t,r)=>e.p[t.featureName]-e.p[r.featureName])),i.forEach((t=>{if(r[t.featureName]||t.featureName===e.D.pageViewEvent){const n=function(t){switch(t){case e.D.ajax:return[e.D.jserrors];case e.D.sessionTrace:return[e.D.ajax,e.D.pageViewEvent];case e.D.sessionReplay:return[e.D.sessionTrace];case e.D.pageViewTiming:return[e.D.pageViewEvent];default:return[]}}(t.featureName);n.every((e=>r[e]))||(0,l.Z)("".concat(t.featureName," is enabled but one or more dependent features has been disabled (").concat((0,D.P)(n),"). This may cause unintended consequences or missing data...")),this.features[t.featureName]=new t(this.agentIdentifier,this.sharedAggregator)}})),(0,T.Qy)(this.agentIdentifier,this.features,t)}catch(e){(0,l.Z)("Failed to initialize all enabled instrument classes (agent aborted) -",e);for(const e in this.features)this.features[e].abortHandler?.();const r=(0,T.fP)();return delete r.initializedAgents[this.agentIdentifier]?.api,delete r.initializedAgents[this.agentIdentifier]?.[t],delete this.sharedAggregator,r.ee?.abort(),delete r.ee?.get(this.agentIdentifier),!1}}}({features:[J,m,S,class extends h{static featureName=oe;constructor(t,r){if(super(t,r,oe,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;const n=this.ee;let i;(0,k.QU)(n),this.eventsEE=(0,k.em)(n),this.eventsEE.on(se,(function(e,t){this.bstStart=(0,p.z)()})),this.eventsEE.on(ae,(function(t,r){(0,s.p)("bst",[t[0],r,this.bstStart,(0,p.z)()],void 0,e.D.sessionTrace,n)})),n.on(ce+ne,(function(e){this.time=(0,p.z)(),this.startPath=location.pathname+location.hash})),n.on(ce+ie,(function(t){(0,s.p)("bstHist",[location.pathname+location.hash,this.startPath,this.time],void 0,e.D.sessionTrace,n)}));try{i=new PerformanceObserver((t=>{const r=t.getEntries();(0,s.p)(te,[r],void 0,e.D.sessionTrace,n)})),i.observe({type:re,buffered:!0})}catch(e){}this.importAggregator({resourceObserver:i})}},C,xe,B,class extends h{static featureName=de;constructor(e,r){if(super(e,r,de,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;if(!(0,t.OP)(e).xhrWrappable)return;try{this.removeOnAbort=new AbortController}catch(e){}let n,i=0;const o=this.ee.get("tracer"),a=(0,k._L)(this.ee),s=(0,k.Lg)(this.ee),u=(0,k.BV)(this.ee),d=(0,k.Kf)(this.ee),f=this.ee.get("events"),l=(0,k.u5)(this.ee),h=(0,k.QU)(this.ee),g=(0,k.Gm)(this.ee);function m(e,t){h.emit("newURL",[""+window.location,t])}function v(){i++,n=window.location.hash,this[ve]=(0,p.z)()}function b(){i--,window.location.hash!==n&&m(0,!0);var e=(0,p.z)();this[pe]=~~this[pe]+e-this[ve],this[ye]=e}function y(e,t){e.on(t,(function(){this[t]=(0,p.z)()}))}this.ee.on(ve,v),s.on(be,v),a.on(be,v),this.ee.on(ye,b),s.on(ge,b),a.on(ge,b),this.ee.buffer([ve,ye,"xhr-resolved"],this.featureName),f.buffer([ve],this.featureName),u.buffer(["setTimeout"+le,"clearTimeout"+fe,ve],this.featureName),d.buffer([ve,"new-xhr","send-xhr"+fe],this.featureName),l.buffer([me+fe,me+"-done",me+he+fe,me+he+le],this.featureName),h.buffer(["newURL"],this.featureName),g.buffer([ve],this.featureName),s.buffer(["propagate",be,ge,"executor-err","resolve"+fe],this.featureName),o.buffer([ve,"no-"+ve],this.featureName),a.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"],this.featureName),y(l,me+fe),y(l,me+"-done"),y(a,"new-jsonp"),y(a,"jsonp-end"),y(a,"cb-start"),h.on("pushState-end",m),h.on("replaceState-end",m),window.addEventListener("hashchange",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("load",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("popstate",(function(){m(0,i>1)}),(0,O.m$)(!0,this.removeOnAbort?.signal)),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}}],loaderType:"spa"})})(),window.NRBA=o})();</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Peter J. le B.Williams"/><meta name="DC.Creator" content="Lieve M. L.Laurens"/><meta name="DC.title" content="Microalgae as biodiesel &amp; biomass feedstocks: Review &amp; analysis of the biochemistry, energetics &amp; economics  "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2010/05/05"/><meta name="DC.Identifier" scheme="doi" content="10.1039/B924978H"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Microalgae as biodiesel &amp; biomass feedstocks: Review &amp; analysis of the biochemistry, energetics &amp; economics  "/><meta name="citation_author" content="Peter J. le B.Williams"/><meta name="citation_author" content="Lieve M. L.Laurens"/><meta name="citation_online_date" content="2010/03/22"/><meta name="citation_date" content="2010"/><meta name="citation_journal_title" content="Energy &amp; Environmental Science"/><meta name="citation_volume" content="3"/><meta name="citation_issue" content="5"/><meta name="citation_firstpage" content="554"/><meta name="citation_lastpage" content="590"/><meta name="citation_doi" content="10.1039/B924978H"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2010/ee/b924978h"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2010/ee/b924978h"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2010/ee/b924978h"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/prospect.js?ver=6_2_1"></script><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"></script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"></script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2010/ee/b924978h">ViewPDFVersion</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:b923503e)" href="/en/content/articlehtml/2010/ee/b923503e" target="_BLANK">PreviousArticle</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:b906384f)" href="/en/content/articlehtml/2010/ee/b906384f" target="_BLANK">NextArticle</a></span></div><div class="right_head"></div><div class="article_info"> DOI:<a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/B924978H">10.1039/B924978H</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1754-5706/2008">Energy Environ. Sci.</a></span>, 2010, <strong>3</strong>, 554-590<div id="chemnav"><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of compound names" onclick="toggleColour('TC');"><span alt="Show Compounds button" id="btnTC">Show Compounds</span></a><input id="inpTC" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of chem-ontology terms" onclick="toggleColour('CH');"><span alt="Show Chemical Terms button" id="btnCH">Show Chemical Terms</span></a><input id="inpCH" type="hidden" value="off"/><a class="tab" onmouseover="this.style.cursor='pointer';" title="Select to toggle highlighting of bio-ontology terms" onclick="toggleColour('AN');"><span alt="Show Biomedical Terms button" id="btnAN">Show Biomedical Terms</span></a><input id="inpAN" type="hidden" value="off"/></div></div><h1 id="sect233"><span class="title_heading">Microalgae as biodiesel &amp; biomass feedstocks: Review &amp; analysis of the biochemistry, energetics &amp; economics<a title="Select to navigate to footnote" href="#fn1"></a></span></h1><p class="header_text">
      <span class="bold">
        
          
            Peter J. le B. 
            Williams
          
          
        
      </span>*<span class="bold"> and </span>
      <span class="bold">
        
          
            Lieve M. L. 
            Laurens<a title="Select to navigate to footnote" href="#fn2"></a>
          
          
        
        
      </span>
      <br/><span class="italic">School of Ocean Sciences, University of Bangor, Menai Bridge, Anglesey, UK LL595PP</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      26th November 2009
    </span><span class="bold italic">, Accepted 18th February 2010</span></div><p class="bold italic">First published on 
      
      
      
      
      
      22nd March 2010
    </p><hr/><div class="abstract"><h2>Abstract</h2><p>Following scrutiny of present biofuels, algae are seriously considered as feedstocks for next-generation biofuels production. Their high productivity and the associated high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> yields make them attractive options. In this review, we analyse a number aspects of large-scale <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> and overall algal biomass production from a biochemical and energetic standpoint. We illustrate that the maximum conversion efficiency of total solar energy into primary photosynthetic organic products falls in the region of 10%. Biomass biochemical composition further conditions this yield: 30 and 50% of the primary product mass is lost on producing <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>. Obtained yields are one third to one tenth of the theoretical ones. Wasted energy from captured <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> is a major loss term and a major challenge in maximising mass algal production. Using irradiance data and kinetic parameters derived from reported field studies, we produce a simple model of algal biomass production and its variation with latitude and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content. An economic analysis of algal biomass production considers a number of scenarios and the effect of changing individual parameters. Our main conclusions are that: (i) the biochemical composition of the biomass influences the economics, in particular, increased <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content reduces other valuable compounds in the biomass; (ii) the biofuel only option is unlikely to be economically viable; and (iii) among the hardest problems in assessing the economics are the cost of the CO<small><sub>2</sub></small> supply and uncertain nature of <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b924978h')">downstream</a></span> processing. We conclude by considering the pressing research and development needs.</p></div><hr/><table><tr><td class="biogPlate"><img alt="Peter J. le B. Williams" src="/image/article/2010/EE/b924978h/b924978h-p1.gif"/><p><span class="graphic_title">Peter J. le B. Williams</span></p></td><td><i><p>Peter Williams is professor emeritus at University of Bangor, UK. Originally trained as an industrial biochemist, he came to the view that oceanography offered more interesting challenges. He has held posts at the Woods Hole Oceanographic Institute, Southampton University, Bigelow Lab in Maine and the University of Gothenburg, returning to the UK to become Professor of Marine Biogeochemistry at Bangor University's School of Ocean Sciences. His long term research interest has been the dynamics of organic material in the oceans, but an opportunity provided by Shell, opened up a new interest in the science behind algal biomass and biofuel production.</p></i></td></tr></table><table><tr><td class="biogPlate"><img alt="Lieve M. L. Laurens" src="/image/article/2010/EE/b924978h/b924978h-p2.gif"/><p><span class="graphic_title">Lieve M. L. Laurens</span></p></td><td><i><p>Dr Lieve Laurens is a researcher at the National Renewable Energy Laboratory (NREL) in Golden, Colorado. Prior to joining NREL, she was a Research Associate at the Centre for Applied Marine Science in the School of ocean Sciences, at the University of Bangor, UK, where she started research on biodiesel from microalgae and the work leading to this review was initiated. Her recent work focuses on the quantification and analysis of lipid yield and composition in microalgae using high-throughput spectroscopic methods. She holds a PhD from the Department of Metabolic Biology at the John Innes Centre in Norwich, UK.</p></i></td></tr></table><hr/>
    
      <br/><table class="box_border"><tr><td>
        
          <h2 id="sect293"><span class="a_heading">Broader context</span></h2>
          <span>There is a rapidly growing interest in the potential of microalgae as feedstocks for the next generation of biofuels. Working from fundamental biochemical principles, we consider the potential yields of organic production by <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. The maximum theoretical energy conversion of full <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> sunlight to organic material lies in the region of 10%. The yields are constrained by thermodynamics and stoichiometry. The yields obtained with outdoor cultures are characteristically one third to one tenth of this theoretical yield; the losses are primarily due to the inability of the photosynthetic system to process the captured <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> at the rate they are absorbed and so energy is lost. Overcoming this problem is a major challenge. The mass and, to a lesser extent, the energy yields are further reduced following the conversion of the primary photosynthetic products to the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> of biochemicals required by the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span>. With a simple light-driven model, we explore the potential economics of a number of production scenarios of algae of varying <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (1550% <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> of the cell's dry weight) at low (030), intermediate (3545) and high (4555) latitudes. The main conclusions were that (i) the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> only option is not viable, markets need to be found for the other major and minor components of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span>; (ii) high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> containing algae may not necessarily be the most favourable candidate organisms. We conclude that although the potential does appear to exist for economic production of algal biofuels, a major R &amp; D programme would be called for to convert the concept to a reality.</span>
        
      </td></tr></table><br/>
    
    
      
      <h2 id="sect307"><span class="a_heading">1. Introduction and historical background</span></h2>
      
        
        <h3 id="sect310"><span class="b_heading">1.1 History of biofuels development</span></h3>
        <span>Without question our society will need to move away from its strong dependence upon <span class="CH"><a class="simple" title="ChEBI Ontology link for: fossil fuels" href="javascript:popupOBO('CHEBI:35230','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35230')">fossil fuels</a></span> as sources of energy and to reduce the emissions of greenhouse gases. Although alternatives exist for land-based <span class="AN"><a class="simple" title="Gene Ontology link for: transport" href="javascript:popupOBO('GO:0006810','b924978h')">transport</a></span>  notably electrical power  there is an understandable reluctance to abandon liquid hydrocarbons and internal <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: combustion" href="javascript:popupOBO('CMO:0001473','b924978h')">combustion</a></span> engines for many <span class="AN"><a class="simple" title="Gene Ontology link for: transport" href="javascript:popupOBO('GO:0006810','b924978h')">transport</a></span> purposes. Further, in the case of aviation <span class="AN"><a class="simple" title="Gene Ontology link for: transport" href="javascript:popupOBO('GO:0006810','b924978h')">transport</a></span> and shipping, there is no practical alternative in the foreseeable future. These are amongst a number of reasons for the drive for the industrial development of liquid biofuels. Systems to produce biodiesel and bioethanol from crop plants (so-called first-generation biofuels) have been developed and optimised over the past several decades and are currently run as profitable businesses.</span>
        <p class="otherpara">Although the production and development of biodiesel and bioethanol has increased rapidly since the start of biofuels development and production in the 1970s, at 3040% per annum, the total energy content in both biofuels is still less than 1% of the world's energy production. The 2008 production figure for bioethanol was 2.5  10<small><sup>18</sup></small> joules and that for biodiesel an order of magnitude less by comparison to a total global energy use of 5  10<small><sup>20</sup></small> joules.</p>
        <p class="otherpara">Despite their relatively small contribution (1%) to the overall production of liquid <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span>, these first-generation biofuels have come under considerable international scrutiny and criticism. The main criticisms are the encroachment of the biofuels feedstock production on valuable crop and virgin land, and the effect biofuels have on <span class="CH"><a class="simple" title="ChEBI Ontology link for: food" href="javascript:popupOBO('CHEBI:33290','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')">food</a></span> commodity prices. These issues have raised major question marks surrounding their social benefits, as summed up in the 2008 Gallagher report to UK House of Commons.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> The executive summary of that report noted that feedstock production <span class="bold">must</span><span class="italic"> avoid agricultural land that would otherwise be used for </span><span class="CH"><a class="simple" title="ChEBI Ontology link for: food" href="javascript:popupOBO('CHEBI:33290','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')"><span class="italic">food</span></a></span><span class="italic"> production</span> and <span class="italic">The introduction of biofuels <span class="bold">should be significantly slowed</span> until adequate controls to address displacement effects are implemented and are demonstrated to be effective</span> (the embolding is ours). This mood, which although not universal, is widespread and has given rise to searches for alternative, so-called second-generation biofuels and has re-stimulated interest in mass algal biomass production.</p>
      
      
        
        <h3 id="sect337"><span class="b_heading">1.2. Algal biofuels potential</span></h3>
        <span>We have restricted the review to microalgae. Macroalgae (seaweeds) have long been grown commercially and may have a role as biofuels and although a number of the aspects of their physiology are shared with the microalgae, their mass production involves a substantially different set of considerations.</span>
        <p class="otherpara">The advantages of microalgae as a feedstock for biodiesel production, over terrestrial plants, are that there is no requirement for soil fertility and, if marine algae are used, there is no need to draw upon valuable and often scare supplies of freshwater. Thus, their negative ecological impact is much reduced as compared with higher plants.<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> The potential to produce the feedstock on waste or desert land and the reduced freshwater requirements addresses some of the concerns in the Gallagher report.</p>
        <p class="otherpara">The mass production of microalgae for <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production has a long history. Prior to the present interest in producing biofuels to mitigate CO<small><sub>2</sub></small> release, there had been an extended period of research interest motivated by security of oil supply. This was initially prompted by the 1973 oil crisis. In <span class="AN"><a class="simple" title="Gene Ontology link for: response to" href="javascript:popupOBO('GO:0014823','b924978h')">response to</a></span> the recognition of the potential vulnerability of oil supplies, Exxon Research and Engineering Company, for example, funded work of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production by algae.<a title="Select to navigate to references" href="#cit2"><sup><span class="sup_ref">2</span></sup></a> At about the same time (1978 to be precise), the U. S. Department of Energy set up the Aquatic Species Program (<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ASP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.411.html">ASP</a></span>). This program reached peak funding in the mid-1980s; after which the funding dwindled, until the program was eventually closed down in 1996. All told, just over $25 million was invested in the program. The findings, which were extensive and valuable, are collected in a major report.<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> There was a concurrent Japanese programme, which purportedly cost in excess of $100 million, but which produced very little accessible science. The main conclusion of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: ASP" href="javascript:popupOBO('CMO:0000019','b924978h')">ASP</a></span> report was that algal biofuels are a potential valuable alternative to traditional biofuels, however, considering the oil prices of the early 1990s, the calculated economics turned out to not be profitable. Nevertheless, considerable headway was made with regards to basic biology, strain selection, metabolic engineering and large-scale <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> cultures of selected strains.</p>
        <p class="otherpara">Algae have long been known to produce a great variety of lipids, hydrocarbons and other complex oils (reviewed recently in <a title="Select to navigate to reference" href="#cit4">ref. 4,5</a>). Cultured algae have been used as feeds for aquaculture applications because of their high content of nutritionally essential <span class="CH"><a class="simple" title="ChEBI Ontology link for: polyunsaturated fatty acids" href="javascript:popupOBO('CHEBI:26208','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26208')">polyunsaturated fatty acids</a></span>. A further motivation for algal culture has been the production of high value by-products such as the pigments <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:astaxanthin" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4519704.html">astaxanthin</a></span> (a <span class="CH"><a class="simple" title="ChEBI Ontology link for: food" href="javascript:popupOBO('CHEBI:33290','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')">food</a></span> colorant and <span class="CH"><a class="simple" title="ChEBI Ontology link for: antioxidant" href="javascript:popupOBO('CHEBI:22586','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=22586')">antioxidant</a></span> from <span class="italic">Haematococcus pluvialis</span>) and -carotene (a <span class="CH"><a class="simple" title="ChEBI Ontology link for: food" href="javascript:popupOBO('CHEBI:33290','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')">food</a></span> additive produced from <span class="italic">Dunaliella</span> species). These sorts of production processes have been in place since the early 1950s, so many aspects of the technology of mass algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> may be considered mature.<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> Much historical research has focused on the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> composition from either a taxonomic or nutritional standpoint. There is, however, a sharp difference between the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> of algae for nutritional and for <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> use. These nutritional products are high value: <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:astaxanthin" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4519704.html">astaxanthin</a></span> for example commands a price in the region of $3 million tonne<small><sup>1</sup></small>,<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a> compared with less than a $1000 tonne<small><sup>1</sup></small> for crude oil. Thus the economics are profoundly different. As a result, the production of biofuels will require a fundamental change in the approaches to production compared with <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutraceutical" href="javascript:popupOBO('CHEBI:50733','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=50733')">nutraceutical</a></span> and aquaculture feed-grade products and without question fresh, major challenges.</p>
        <p class="otherpara">Algal biomass can serve as a feedstock for the production of a variety of different biofuels, <span class="italic">e.g.</span> biodiesel, <span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span>, <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:methane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.291.html">methane</a></span> and bioethanol. Furthermore, its production is also being seriously considered for the removal of carbon dioxide from the flue gases of <span class="CH"><a class="simple" title="ChEBI Ontology link for: fossil fuel" href="javascript:popupOBO('CHEBI:35230','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35230')">fossil fuel</a></span> power stations. In order to maintain focus, we limit our discussion to the use of algae as feedstock for biodiesel and biomass production. There have been claims in the literature over recent years hailing algae as the solution to the global energy crisis (<span class="italic">e.g.</span><a title="Select to navigate to references" href="#cit8">ref. 8</a>). In this review we aim to present the principles behind large-scale production of algal biomass and biodiesel production along with a critical and objective review of the literature to date. As others (<span class="italic">e.g.</span><a title="Select to navigate to references" href="#cit9">ref. 9</a>), we build our review on the biochemical fundamentals of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and biomass production, and accordingly begin (Section 2) with a brief discussion of the biology and biochemical composition of microalgae, with particular regard to the major biochemical categories: <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, carbohydrates and lipids. In Section 3, starting from first principles, we establish the overall potential efficiency of photosynthetic production of algae. A major challenge in algal biofuels production is maximising this stage, as economic models for biodiesel production from seed oils have acknowledged that the feedstock cost comprises a substantial portion of the overall biodiesel cost.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> Accordingly, in Section 4 we address the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> of algae, the controls on <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> and the reported yields. These are then combined with the theoretical photosynthetic yields to give predictions of biomass yields for season and latitude. Section 5 contains a review of the harvest and processing of the biomass relevant to published processes. In Section 6, we bring together the findings of the two preceding sections in the form of an economic analysis of various scenarios for low, mid and high latitudes. Finally, in Section 7, we explore the answers to two questions: (i) can the production of biofuels from algae be economically viable; and (ii) what R &amp; D is needed to achieve a profitable outcome.</p>
      
    
    
      
      <h2 id="sect397"><span class="a_heading">2. Biology and biochemical composition of microalgae</span></h2>
      <span>In this section we introduce the aspects of algal biology and biochemistry that are relevant in the economical considerations (Section 6). The composition of the algal biomass with regards to lipids, carbohydrates and <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> will greatly determine its overall value.</span>
      <p class="otherpara">Microalgae are single <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> organisms, found in either colonies or individual <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> and are comprised of representatives of both the eukaryotes and bacteria. They are able to photosynthesise, and are found in a range of different habitats, from fresh to marine and hyper-saline environments.<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> The large number of species are generally subdivided into 10 taxonomic <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> which include the green algae (Chlorophyceae), diatoms (Bacillariophyceae), yellow-green (Xanthophyceae), golden algae (Chrysophyceae), red algae (Rhodophyceae), brown algae (Phaeophyceae), dinoflagellates (Dinophyceae), Prasinophyceae and Eustigmatophyceae.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> The blue-green algae (Cyanophyceae) were originally grouped with the eukaryotic algae; however it was subsequently realised that they belong to the bacterial domain, hence their present common name  cyanobacteria<a title="Select to navigate to footnote" href="#fn3"></a>.</p>
      <p class="otherpara">A major and significant difference between the bacteria and the eukaryotes is that the former lack discrete internal, subcellular structures, <span class="AN"><a class="simple" title="Gene Ontology link for: organelles" href="javascript:popupOBO('GO:0043226','b924978h')">organelles</a></span> (chloroplasts, <span class="AN"><a class="simple" title="Gene Ontology link for: mitochondria" href="javascript:popupOBO('GO:0005739','b924978h')">mitochondria</a></span>, nuclei). <span class="AN"><a class="simple" title="Gene Ontology link for: Organelles" href="javascript:popupOBO('GO:0043226','b924978h')">Organelles</a></span> are surrounded by <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: membranes" href="javascript:popupOBO('GO:0016020','b924978h')">membranes</a></span> whose bilayer structure requires strongly polar molecules. The triglycerides, the predominant <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> class in higher plant oils, lack the required polarity to form a sufficiently stable bilayer. The more polar phospholipids and the glycolipids are the major components of <span class="AN"><a class="simple" title="Gene Ontology link for: cell membranes" href="javascript:popupOBO('GO:0005886','b924978h')">cell membranes</a></span> and as a result, in actively growing <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span>, these two <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> are major components of algal lipids, with important consequences.</p>
      <p class="otherpara">The <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span> of organisms, particularly microorganisms, is strongly influenced by their surface-to-volume ratio. Simple, single celled organisms can be approximated to spheres, thus their <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span>, and therefore <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate, is inversely proportional to their <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> diameters (see insert in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>). The <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rates of microorganisms can be very high; whilst some algae are able to divide once every 34 h, most divide every 12 days under favourable conditions (see <a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>). Accordingly their scope for <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> is colossal and this, in major part, is the basis for the interest in their potential as biomass producers. In principle algal biomass crops may be harvested either daily or every few days (<span class="italic">e.g.</span><a title="Select to navigate to reference" href="#cit15">ref. 15, 16</a>).</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><img alt="Frequency analysis of algal growth rates (main pane), derived from data in ref. 13 and algal growth-size relationships from ref. 13, 14." src="/image/article/2010/EE/b924978h/b924978h-f1.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Frequency analysis of algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rates (main pane), derived from data in <a title="Select to navigate to references" href="#cit13">ref. 13</a> and algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>-size relationships from <a title="Select to navigate to reference" href="#cit13">ref. 13, 14</a>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
        
        <h3 id="sect447"><span class="b_heading">2.1. Major biochemical <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span>, their presence and function</span></h3>
        <span>It is conventional to consider four principal biochemical classes of molecules: carbohydrates, <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> and lipids. <a title="Select to navigate to table" href="#tab1">Table 1</a> gives the broad <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> content of these major fractions, their elemental composition and energetic properties.</span>
        
<div class="table_caption"><b>Table 1</b> <span id="tab1">Elemental composition of algal biochemical components. Data for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and fatty acids were derived from means from the analyses shown in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a> and <a title="Select to navigate to figure" href="#imgfig5">5</a>. The values for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> classes are derived from the mean <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> elemental composition and the composition of the other molecular structures in the overall molecule. That for the overall algal lipids is derived from the values for the individual <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> classes and the mean <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> class composition given in <a title="Select to navigate to table" href="#tab3">Table 3</a>. The values for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> and carbohydrates are taken from <a title="Select to navigate to references" href="#cit17">ref. 17</a>. The calorific values (as kJ g<small><sup>1</sup></small>) of lipids and their derivatives have been calculated according to an equation for higher heating values (equivalent to the bomb calorific value) = 35.17<span class="italic">C</span> + 116.25<span class="italic">H</span>  11.1<span class="italic">O</span> + 6.28<span class="italic">N</span> + 10.47<span class="italic">S</span>, where <span class="italic">C</span>, <span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html"><span class="italic">H</span></a></span>, <span class="italic">O</span>, <span class="italic">N</span>, <span class="italic">P</span> and <span class="italic">S</span> are the mass fractions of the constituent elements.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> This equation gives a good approximation for these compounds. The <span class="TC"><a class="simple" title="Chemspider Compound link for:hydrogen" target="_blank" href="http://www.chemspider.com/Chemical-Structure.762.html">hydrogen</a></span> and oxygen atoms associated with <span class="TC"><a class="simple" title="Chemspider Compound link for:phosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.1032.html">phosphate</a></span> and <span class="TC"><a class="simple" title="Chemspider Compound link for:sulfate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.1085.html">sulfate</a></span> were not included in the calculation. The calorific values for <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> are calculated from the calorific values of their component molecules, using 18 kJ per mol of <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> as the energy lost on <span class="CH"><a class="simple" title="Molecular Process Ontology link for: polymerisation" href="javascript:popupOBO('MOP:0000629','b924978h')">polymerisation</a></span>. The range given in the right hand column derives from the data compilations used to produce <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a> (only the middle 90% of the data is used)</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Biochemical component</th>
                  <th align="left" valign="bottom" class="border_black">Characteristic elemental composition</th>
                  <th align="left" valign="bottom" class="border_black">Calculated calorific value/kJ g<small><sup>1</sup></small></th>
                  <th align="left" valign="bottom" class="border_black">Range of typical <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> content (%)</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Algal lipids</td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.83</sub></small>O<small><sub>0.17</sub></small>N<small><sub>0.0031</sub></small>P<small><sub>0.006</sub></small>S<small><sub>0.0014</sub></small></td>
                  <td char="." valign="top">36.3</td>
                  <td char="" valign="top">1560</td>
                </tr>
                <tr align="left">
                  <td valign="top">Acylglycerides</td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.83</sub></small>O<small><sub>0.096</sub></small></td>
                  <td char="." valign="top">40.2</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Glycolipids</td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.79</sub></small>O<small><sub>0.24</sub></small>S<small><sub>0.0035</sub></small></td>
                  <td char="." valign="top">33.4</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Phospholipids</td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.88</sub></small>O<small><sub>0.173</sub></small>N<small><sub>0.012</sub></small>P<small><sub>0.024</sub></small></td>
                  <td char="." valign="top">35.3</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Algal fatty acid</td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.91</sub></small>O<small><sub>0.12</sub></small></td>
                  <td char="." valign="top">39.6</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Methyl" href="javascript:popupOBO('CHEBI:32875','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=32875')">Methyl</a></span> esters</td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.92</sub></small>O<small><sub>0.05</sub></small></td>
                  <td char="." valign="top">43.0</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Protein</a></span></td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.56</sub></small>O<small><sub>0.3</sub></small>N<small><sub>0.26</sub></small>S<small><sub>0.006</sub></small></td>
                  <td char="." valign="top">23.9</td>
                  <td char="" valign="top">2060</td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Nucleic acid" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">Nucleic acid</a></span></td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.23</sub></small>O<small><sub>0.74</sub></small>N<small><sub>0.40</sub></small>P<small><sub>0.11</sub></small></td>
                  <td char="." valign="top">14.8</td>
                  <td char="" valign="top">35</td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Polysaccharide" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">Polysaccharide</a></span></td>
                  <td valign="top">C<small><sub>1</sub></small>H<small><sub>1.67</sub></small>O<small><sub>0.83</sub></small></td>
                  <td char="." valign="top">17.3</td>
                  <td char="" valign="top">1050</td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <div>
          
          <span id="sect598"/><span class="c_heading_indent">(i) Carbohydrates. </span>
          <span>Microorganisms contain a wide variety of carbohydrates, both monomers and <span class="CH"><a class="simple" title="ChEBI Ontology link for: polymers" href="javascript:popupOBO('CHEBI:60027','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=60027')">polymers</a></span>. Carbohydrates serve both structural and metabolic functions and, as the early products of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>, they serve as the starting point for the synthesis of the other biochemicals. Different classes of algae produce specific types of <span class="CH"><a class="simple" title="ChEBI Ontology link for: polysaccharides" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">polysaccharides</a></span>. For example, green algae produce <span class="CH"><a class="simple" title="ChEBI Ontology link for: starch" href="javascript:popupOBO('CHEBI:28017','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28017')">starch</a></span> as an energy store, consisting of both <span class="CH"><a class="simple" title="ChEBI Ontology link for: amylose" href="javascript:popupOBO('CHEBI:28102','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28102')">amylose</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: amylopectin" href="javascript:popupOBO('CHEBI:28057','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28057')">amylopectin</a></span>, similar to higher plants. The green alga <span class="italic">Tetraselmis suecica</span>, accumulates 11 and 47% of its dry weight as <span class="CH"><a class="simple" title="ChEBI Ontology link for: starch" href="javascript:popupOBO('CHEBI:28017','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28017')">starch</a></span> in <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> replete and deplete conditions respectively.<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a> Red algae synthesise a <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: polymer" href="javascript:popupOBO('CHEBI:60027','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=60027')">polymer</a></span> known as floridean <span class="CH"><a class="simple" title="ChEBI Ontology link for: starch" href="javascript:popupOBO('CHEBI:28017','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28017')">starch</a></span>, consisting mostly of <span class="CH"><a class="simple" title="ChEBI Ontology link for: amylose" href="javascript:popupOBO('CHEBI:28102','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28102')">amylose</a></span>.<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a> A commonly found <span class="CH"><a class="simple" title="ChEBI Ontology link for: polysaccharide" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">polysaccharide</a></span> in a large number of algal species is chrysolaminarin, a linear <span class="CH"><a class="simple" title="ChEBI Ontology link for: polymer" href="javascript:popupOBO('CHEBI:60027','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=60027')">polymer</a></span> of (1  3) and (1  6) linked <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glucose" target="_blank" href="http://www.chemspider.com/Chemical-Structure.96749.html">glucose</a></span> units.<a title="Select to navigate to references" href="#cit21"><sup><span class="sup_ref">21</span></sup></a> Chrysolaminarin often accumulates in pyrenoids, which are centres of high activity of carbon assimilation in the <span class="AN"><a class="simple" title="Gene Ontology link for: chloroplast" href="javascript:popupOBO('GO:0009507','b924978h')">chloroplast</a></span>.</span>
        </div>
        <div>
          
          <span id="sect626"/><span class="c_heading_indent">(ii) <span class="CH"><a class="simple" title="ChEBI Ontology link for: Proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Proteins</a></span>. </span>
          <span>As do carbohydrates, <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> also have both structural and metabolic functions. As enzymes, they are the prime <span class="CH"><a class="simple" title="ChEBI Ontology link for: catalysts" href="javascript:popupOBO('CHEBI:35223','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35223')">catalysts</a></span> for <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span> and so facilitate <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>. Second, they serve a structural role. For example, they provide the scaffold upon which the <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules are assembled in the <span class="AN"><a class="simple" title="Gene Ontology link for: light harvesting" href="javascript:popupOBO('GO:0031992','b924978h')">light harvesting</a></span> complexes of the <span class="AN"><a class="simple" title="Gene Ontology link for: chloroplast" href="javascript:popupOBO('GO:0009507','b924978h')">chloroplast</a></span> (see Section 3.1(i)) and also are embedded in the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: membranes" href="javascript:popupOBO('GO:0016020','b924978h')">membranes</a></span>, where they serve a similar structural role, as well as a metabolic role. Furthermore, it is known that the <span class="AN"><a class="simple" title="Cell Ontology link for: structural cell" href="javascript:popupOBO('CL:0000293','b924978h')">structural cell</a></span> wall of <span class="italic">Chlamydomonas reinhardtii</span> consists primarily of cross-linked <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:hydroxy-proline" target="_blank" href="http://www.chemspider.com/Chemical-Structure.236516.html">hydroxy-proline</a></span>-rich glycoproteins.<a title="Select to navigate to references" href="#cit22"><sup><span class="sup_ref">22</span></sup></a></span>
          <p class="otherpara"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Proteins</a></span> have important commodity value as animal feed. Critical to this is their <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> composition, as a number of amino acids are dietary essentials for mammals yet they are unable to synthesise them, for example. <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a> shows the mean <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> for a number of algae. It can be seen that the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> compares favourably with <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> of high nutritional quality.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><img alt="Mean amino acid composition of algal protein derived from 28 species (9 classes) of eukaryotic algae.23 The upper bars are the standard deviations, the lower the standard errors. The low value for tryptophan will derive in part from loss of the molecule during acid hydrolysis of the protein. Characteristic values for high quality proteins are shown for comparison purposes." src="/image/article/2010/EE/b924978h/b924978h-f2.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Mean <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> composition of algal <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> derived from 28 species (9 classes) of eukaryotic algae.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> The upper bars are the standard deviations, the lower the standard errors. The low value for <span class="TC"><a class="simple" title="Chemspider Compound link for:tryptophan" target="_blank" href="http://www.chemspider.com/Chemical-Structure.1116.html">tryptophan</a></span> will derive in part from loss of the molecule during acid <span class="CH"><a class="simple" title="Molecular Process Ontology link for: hydrolysis" href="javascript:popupOBO('MOP:0000619','b924978h')">hydrolysis</a></span> of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>. Characteristic values for high quality <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> are shown for comparison purposes.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
        <div>
          
          <span id="sect663"/><span class="c_heading_indent">(iii) <span class="CH"><a class="simple" title="ChEBI Ontology link for: Nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">Nucleic acids</a></span>. </span>
          <span><span class="CH"><a class="simple" title="ChEBI Ontology link for: Nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">Nucleic acids</a></span> (<span class="AN"><a class="simple" title="Sequence Ontology link for: RNA" href="javascript:popupOBO('SO:0000356','b924978h')">RNA</a></span> and <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b924978h')">DNA</a></span>), in conjunction with <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and their monomers, provide the basis for algal division and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> comprise a small fraction of cellular biomass but the major part of the cell's <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> and the second most important site of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> (see <a title="Select to navigate to table" href="#tab2">Table 2</a>).</span>
          
<div class="table_caption"><b>Table 2</b> <span id="tab2">Calculated distribution of major elements in the four principal biochemical fractions for model low (15%), medium (25%) and high (50%) <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>-containing algae. The calculation is made using the elemental compositions and calorific values given in <a title="Select to navigate to table" href="#tab1">Table 1</a> and assumes that, as the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content varies, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acid" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acid</a></span> content remains constant at 5% and <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio remains constant at 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 (see legend to <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>). These simplifications mean that the data are mainly illustrative</span>
		</div>

              <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/></colgroup>
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                
                <thead align="left">
                  <tr align="left" valign="bottom">
                    <th rowspan="4" align="left" valign="bottom" class="border_black"></th>
                    <th rowspan="2" colspan="3" align="left" valign="bottom" class="border_black">Biochemical composition (as % total mass)</th>
                    <th colspan="18" align="left" valign="bottom" class="border_black">Percentage major element in various biochemical fractions</th>
                  </tr>
                  <tr align="left" valign="bottom">
                    <th colspan="3" align="left" valign="bottom" class="border_black">C</th>
                    <th colspan="3" align="left" valign="bottom" class="border_black"><span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span></th>
                    <th colspan="3" align="left" valign="bottom" class="border_black">O</th>
                    <th colspan="3" align="left" valign="bottom" class="border_black">N</th>
                    <th colspan="3" align="left" valign="bottom" class="border_black">P</th>
                    <th colspan="3" align="left" valign="bottom" class="border_black">S</th>
                  </tr>
                  <tr align="left" valign="bottom">
                    <th colspan="3" align="left" valign="bottom" class="border_black"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">Lipid</a></span> level</th>
                    <th colspan="18" align="left" valign="bottom" class="border_black"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">Lipid</a></span> level</th>
                  </tr>
                  <tr align="left" valign="bottom">
                    <th align="left" valign="bottom" class="border_black">Low</th>
                    <th align="left" valign="bottom" class="border_black">Medium</th>
                    <th align="left" valign="bottom" class="border_black">High</th>
                    <th align="left" valign="bottom" class="border_black">Low</th>
                    <th align="left" valign="bottom" class="border_black">Medium</th>
                    <th align="left" valign="bottom" class="border_black">High</th>
                    <th align="left" valign="bottom" class="border_black">Low</th>
                    <th align="left" valign="bottom" class="border_black">Medium</th>
                    <th align="left" valign="bottom" class="border_black">High</th>
                    <th align="left" valign="bottom" class="border_black">Low</th>
                    <th align="left" valign="bottom" class="border_black">Medium</th>
                    <th align="left" valign="bottom" class="border_black">High</th>
                    <th align="left" valign="bottom" class="border_black">Low</th>
                    <th align="left" valign="bottom" class="border_black">Medium</th>
                    <th align="left" valign="bottom" class="border_black">High</th>
                    <th align="left" valign="bottom" class="border_black">Low</th>
                    <th align="left" valign="bottom" class="border_black">Medium</th>
                    <th align="left" valign="bottom" class="border_black">High</th>
                    <th align="left" valign="bottom" class="border_black">Low</th>
                    <th align="left" valign="bottom" class="border_black">Medium</th>
                    <th align="left" valign="bottom" class="border_black">High</th>
                  </tr>
                </thead>
                <tbody align="left">
                  <tr align="left">
                    <td valign="top">Lipids</td>
                    <td char="." valign="top">15</td>
                    <td char="." valign="top">25</td>
                    <td char="." valign="top">50</td>
                    <td char="." valign="top">20</td>
                    <td char="." valign="top">33</td>
                    <td char="." valign="top">59</td>
                    <td char="." valign="top">23</td>
                    <td char="." valign="top">36</td>
                    <td char="." valign="top">63</td>
                    <td char="." valign="top">8</td>
                    <td char="." valign="top">14</td>
                    <td char="." valign="top">33</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">1</td>
                    <td char="." valign="top">2</td>
                    <td char="." valign="top">25</td>
                    <td char="." valign="top">35</td>
                    <td char="." valign="top">52</td>
                    <td char="." valign="top">9</td>
                    <td char="." valign="top">15</td>
                    <td char="." valign="top">36</td>
                  </tr>
                  <tr align="left">
                    <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Proteins</a></span></td>
                    <td char="." valign="top">48</td>
                    <td char="." valign="top">42</td>
                    <td char="." valign="top">27</td>
                    <td char="." valign="top">49</td>
                    <td char="." valign="top">41</td>
                    <td char="." valign="top">24</td>
                    <td char="." valign="top">47</td>
                    <td char="." valign="top">39</td>
                    <td char="." valign="top">22</td>
                    <td char="." valign="top">34</td>
                    <td char="." valign="top">32</td>
                    <td char="." valign="top">24</td>
                    <td char="." valign="top">90</td>
                    <td char="." valign="top">88</td>
                    <td char="." valign="top">82</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">91</td>
                    <td char="." valign="top">85</td>
                    <td char="." valign="top">64</td>
                  </tr>
                  <tr align="left">
                    <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Polysaccharides" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">Polysaccharides</a></span></td>
                    <td char="." valign="top">32</td>
                    <td char="." valign="top">28</td>
                    <td char="." valign="top">18</td>
                    <td char="." valign="top">27</td>
                    <td char="." valign="top">23</td>
                    <td char="." valign="top">13</td>
                    <td char="." valign="top">28</td>
                    <td char="." valign="top">23</td>
                    <td char="." valign="top">13</td>
                    <td char="." valign="top">52</td>
                    <td char="." valign="top">48</td>
                    <td char="." valign="top">36</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                  </tr>
                  <tr align="left">
                    <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">Nucleic acids</a></span></td>
                    <td char="." valign="top">5</td>
                    <td char="." valign="top">5</td>
                    <td char="." valign="top">5</td>
                    <td char="." valign="top">3</td>
                    <td char="." valign="top">3</td>
                    <td char="." valign="top">3</td>
                    <td char="." valign="top">3</td>
                    <td char="." valign="top">2</td>
                    <td char="." valign="top">2</td>
                    <td char="." valign="top">6</td>
                    <td char="." valign="top">6</td>
                    <td char="." valign="top">7</td>
                    <td char="." valign="top">10</td>
                    <td char="." valign="top">11</td>
                    <td char="." valign="top">15</td>
                    <td char="." valign="top">75</td>
                    <td char="." valign="top">65</td>
                    <td char="." valign="top">48</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Calorific value/kJ g<small><sup>1</sup></small></td>
                    <td char="." valign="top">23.2</td>
                    <td char="." valign="top">24.7</td>
                    <td char="." valign="top">28.5</td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                  </tr>
                  <tr align="left">
                    <td colspan="4" valign="top">Element total (as % dry weight)</td>
                    <td char="." valign="top">53</td>
                    <td char="." valign="top">55</td>
                    <td char="." valign="top">60</td>
                    <td char="." valign="top">7.2</td>
                    <td char="." valign="top">7.6</td>
                    <td char="." valign="top">8.6</td>
                    <td char="." valign="top">30</td>
                    <td char="." valign="top">29</td>
                    <td char="." valign="top">24</td>
                    <td char="." valign="top">8.7</td>
                    <td char="." valign="top">7.8</td>
                    <td char="." valign="top">5.4</td>
                    <td char="." valign="top">0.7</td>
                    <td char="." valign="top">0.81</td>
                    <td char="." valign="top">1.1</td>
                    <td char="." valign="top">0.5</td>
                    <td char="." valign="top">0</td>
                    <td char="." valign="top">0.4</td>
                  </tr>
                </tbody>
              </table></div></div>
            <hr class="hrule"/><br/>
        </div>
        <div>
          
          <span id="sect895"/><span class="c_heading_indent">(iv) Lipids. </span>
          <span>As with carbohydrates, lipids serve both as energy reserves and structural components (<span class="AN"><a class="simple" title="Gene Ontology link for: membranes" href="javascript:popupOBO('GO:0016020','b924978h')">membranes</a></span>) of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span>. The simple <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> triglycerides are important energy reserves. <span class="AN"><a class="simple" title="Gene Ontology link for: Membranes" href="javascript:popupOBO('GO:0016020','b924978h')">Membranes</a></span> are mainly constructed from phospholipids and glycolipids, where the hydrophilic polar <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> or sugar moieties and the level of saturation of the fatty acyl chains determine the fluidity of the <span class="AN"><a class="simple" title="Gene Ontology link for: membranes" href="javascript:popupOBO('GO:0016020','b924978h')">membranes</a></span>. The microalgae have the facility for rapid adaptation to new environments (<span class="italic">e.g.</span> changes in temperature), through the <span class="italic">de novo</span> synthesis and recycling of fatty acids to maintain the <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b924978h')">membrane</a></span> characteristics.</span>
          <p class="otherpara">A large fraction of the cell's <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> may be present in phospholipids. The <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b924978h')">membrane</a></span> lipids associated with the thylakoids (the internal <span class="AN"><a class="simple" title="Gene Ontology link for: chloroplast membranes" href="javascript:popupOBO('GO:0031969','b924978h')">chloroplast membranes</a></span>, also the sites for photosynthetic activity in eukaryotes, see Section 3.1(i)) contain a sulfur lipid (sulfoquinovosyldiacyl glycerol), which is the major site of the <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:sulfur" target="_blank" href="http://www.chemspider.com/Chemical-Structure.391.html">sulfur</a></span> in algal lipids.<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a><a title="Select to navigate to table" href="#tab3">Table 3</a> gives a summary of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> class distribution derived from published analyses. A number of factors give rise to the variations seen in the table. <span class="AN"><a class="simple" title="Gene Ontology link for: Storage" href="javascript:popupOBO('GO:0051235','b924978h')">Storage</a></span> lipids, which are predominantly triglycerides, may gain a greater proportion of the overall <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> fraction as the metabolic rate slows down. As a consequence, shifts in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> composition occur through the various phases of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>.</p>
          
<div class="table_caption"><b>Table 3</b> <span id="tab3">Mean ( standard error) <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> class content as a percent of total <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> of individual algal species, plus a composite from all reports for microalgae.<a title="Select to navigate to reference" href="#cit24"><sup><span class="sup_ref">2435</span></sup></a></span>
		</div>

              <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>
                
                
                
                
                <thead align="left">
                  <tr align="left" valign="bottom">
                    <th align="left" valign="bottom" class="border_black"></th>
                    <th align="left" valign="bottom" class="border_black">Simple lipids</th>
                    <th align="left" valign="bottom" class="border_black">Glycolipids</th>
                    <th align="left" valign="bottom" class="border_black">Phospholipids</th>
                  </tr>
                </thead>
                <tbody align="left">
                  <tr align="left">
                    <td colspan="4" valign="top">
                      <span class="bold">Diatoms</span>
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Chaetoceros species</span>
                    </td>
                    <td align="char" valign="top">37  16</td>
                    <td align="char" valign="top">36  8</td>
                    <td align="char" valign="top">25  8</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Phaeodactylum tricornutum</span>
                    </td>
                    <td align="char" valign="top">54  6</td>
                    <td align="char" valign="top">34  5</td>
                    <td align="char" valign="top">11  1</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Range from Borowitzka (1988)<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a></td>
                    <td align="char" valign="top">1460</td>
                    <td align="char" valign="top">1344</td>
                    <td align="char" valign="top">1047</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="bold">Green algae</span>
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Chlamydomonas species</span>
                    </td>
                    <td align="char" valign="top">48  10</td>
                    <td align="char" valign="top">44  13</td>
                    <td align="char" valign="top">6  3</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Dunaliella tertiolecta</span>
                    </td>
                    <td align="char" valign="top">7  1</td>
                    <td align="char" valign="top">67  1</td>
                    <td align="char" valign="top">25  0</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Dunalliella viridis</span>
                    </td>
                    <td align="char" valign="top">13  1</td>
                    <td align="char" valign="top">44  3</td>
                    <td align="char" valign="top">42  2</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Range from Borowitzka (1988)<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a></td>
                    <td align="char" valign="top">2166</td>
                    <td align="char" valign="top">662</td>
                    <td align="char" valign="top">1753</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="bold">Blue green algae</span>
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Range from Borowitzka (1988)<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a></td>
                    <td align="char" valign="top">1168</td>
                    <td align="char" valign="top">1241</td>
                    <td align="char" valign="top">1650</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="bold">Others</span>
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Nannochloropsis oculata</span>
                    </td>
                    <td align="char" valign="top">22  1</td>
                    <td align="char" valign="top">39  0</td>
                    <td align="char" valign="top">38  1</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">Isochrysis species</span>
                    </td>
                    <td align="char" valign="top">36  3</td>
                    <td align="char" valign="top">35  1</td>
                    <td align="char" valign="top">27  3</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                    <td valign="top">
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="bold">Composite from all reports (<span class="italic">n</span> = 46)</span>
                    </td>
                    <td align="char" valign="top">
                      <span class="bold">35  3</span>
                    </td>
                    <td align="char" valign="top">
                      <span class="bold">40  2</span>
                    </td>
                    <td align="char" valign="top">
                      <span class="bold">25  2</span>
                    </td>
                  </tr>
                </tbody>
              </table></div></div>
            <hr class="hrule"/><br/>
          <p class="otherpara">Allowing for the scatter due to differences in analytical techniques, the frequency distribution suggests a minimum <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content in the region of 15% (see <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>). There appears to be a marked difference between the total <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content of the eukaryotic and the prokaryotic algae (the cyanobacteria)  the latter containing less than the former (see <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>), likely to be due to the absence of internal <span class="AN"><a class="simple" title="Gene Ontology link for: membranes" href="javascript:popupOBO('GO:0016020','b924978h')">membranes</a></span> in the prokaryotes (see Section 2.1 (i)). So, although the cyanobacteria are easier organisms to manipulate genetically (as a result of their simpler <span class="AN"><a class="simple" title="Sequence Ontology link for: DNA" href="javascript:popupOBO('SO:0000352','b924978h')">DNA</a></span>), their potential as <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> producers would not appear to be promising.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><img alt="Frequency distribution of the lipid content (as % dry weight) of eukaryotic algae and Cyanobacteria. The values in parentheses are the number of data sets used for the analysis. Primary data and references in Appendix I (provided in the ESI)." src="/image/article/2010/EE/b924978h/b924978h-f3.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">Frequency distribution of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (as % dry weight) of eukaryotic algae and Cyanobacteria. The values in parentheses are the number of data sets used for the analysis. Primary data and references in Appendix I (provided in the ESI<a title="Select to navigate to footnote" href="#fn1"></a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> composition of algal lipids (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>) is well documented, with a high occurrence of unsaturated (and polyunsaturated) fatty acids, with half of the fatty acids having a carbon number less than C<small><sub>18</sub></small>. The majority of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: unsaturated fatty acids" href="javascript:popupOBO('CHEBI:27208','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=27208')">unsaturated fatty acids</a></span> occur in the <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b924978h')">membrane</a></span> lipids, where their main function is to maintain <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b924978h')">membrane</a></span> fluidity under different conditions. The high concentrations of <span class="CH"><a class="simple" title="ChEBI Ontology link for: unsaturated fatty acids" href="javascript:popupOBO('CHEBI:27208','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=27208')">unsaturated fatty acids</a></span> in the extracted lipids, and ultimately in the resulting <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span>, will be an important <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> quality determinant. The preponderance of the shorter chain fatty acids has significance for their potential as diesel <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span> (defined as alkyl chains of between 12 and 18 carbon atoms long). The level of unsaturation affects biodiesel properties.<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> For example, <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span> with a higher level of unsaturation of the acyl chains have a higher cloud point, which is desirable, but are also much more susceptible to <span class="CH"><a class="simple" title="Molecular Process Ontology link for: oxidation" href="javascript:popupOBO('MOP:0000568','b924978h')">oxidation</a></span>. The highly <span class="CH"><a class="simple" title="ChEBI Ontology link for: unsaturated fatty acids" href="javascript:popupOBO('CHEBI:27208','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=27208')">unsaturated fatty acids</a></span> found in algae may need to be hydrogenated to improve their potential <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> properties. Furthermore, a high level of <span class="CH"><a class="simple" title="ChEBI Ontology link for: unsaturated fatty acids" href="javascript:popupOBO('CHEBI:27208','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=27208')">unsaturated fatty acids</a></span> in a <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> increases the danger of <span class="CH"><a class="simple" title="Molecular Process Ontology link for: polymerisation" href="javascript:popupOBO('MOP:0000629','b924978h')">polymerisation</a></span> in the engine oil and can cause problems with oxidative stability of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span>.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><img alt="Upper histogram: mean fatty acid composition of various eukaryotic algal groups and Cyanobacteria. The figures in parenthesis are the numbers of analyses from which the means are derived. The upper error bars are standard deviations, the lower standard errors. Full data set and references in Appendix III (provided in the ESI). Lower histogram: data for comparison purposes for higher plant fatty acids (C. Price, Shell Global Solutions, personal communication)." src="/image/article/2010/EE/b924978h/b924978h-f4.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Upper histogram: mean <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> composition of various eukaryotic algal <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> and Cyanobacteria. The figures in parenthesis are the numbers of analyses from which the means are derived. The upper error bars are standard deviations, the lower standard errors. Full data set and references in Appendix III (provided in the ESI<a title="Select to navigate to footnote" href="#fn1"></a>). Lower histogram: data for comparison purposes for higher plant fatty acids (C. Price, Shell Global Solutions, personal communication).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
      
      
        
        <h3 id="sect1095"><span class="b_heading">2.2. Shifts in the biochemical composition with increasing <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span></span></h3>
        <span>As the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content increases, the percentage of the sum of the other components must go down. <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a> gives a summary of the reported relative proportions of the three major biochemicals  <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span>. For two reasons the shift in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content is critical. First these molecules set the level of the cell's <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span> and in conjunction with the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> determine the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate potential. Second, from an economic point of view <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> are valuable bulk components of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> (see Section 6). Thus, in designing a strategy to search and select for the economically most suitable algae, it is of prime importance to establish whether or not we can generalise over the change in <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content associated with changes in total <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content. As the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content increases, the question arises, are the carbohydrates preferentially run down and the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content held constant as long as possible. Alternatively, are the <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> preferentially run down or do the carbohydrates and <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> decrease in relative proportions. These three trajectories are shown in the triangular plot in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>. In the case of first of the three scenarios listed above, we might expect maintenance of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate with increases in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content up to 4050%, whereas in the others we would anticipate a <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate as the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content increased (see discussion below in Section 2.3). The data we have been able to assemble in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a> does not give a clear sign of any systematic compositional trend and probably a dedicated study would be needed to elicit one, if it indeed exists.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><img alt="A: triangular plot of the proportions of lipid, protein and carbohydrate content of algae, data taken from ref. 2, 27 and 3742. The red circle is the mean of the whole dataset for active growth (lipid 24.2%, protein 48.3%, carbohydrate 27.5%). The solid circles show the shift in composition from unlimited logarithmic growth (red) to N-limited growth (brown). The green line shows the trajectory on increasing lipid composition assuming the protein:carbohydrate ratio remains constant, the red line that if the carbohydrate content diminishes as the lipid content increases and the magenta line that if the protein content diminishes. Throughout this review we have used a rounded off value of 3:2 for the protein carbohydrate ratio." src="/image/article/2010/EE/b924978h/b924978h-f5.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">A: triangular plot of the proportions of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> content of algae, data taken from <a title="Select to navigate to reference" href="#cit2">ref. 2, 27 and 3742</a>. The red circle is the mean of the whole dataset for active <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> (<span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> 24.2%, <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> 48.3%, <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> 27.5%). The solid circles show the shift in composition from unlimited logarithmic <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> (red) to N-limited <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> (brown). The green line shows the trajectory on increasing <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> composition assuming the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio remains constant, the red line that if the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> content diminishes as the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content increases and the magenta line that if the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content diminishes. Throughout this review we have used a rounded off value of 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1144"><span class="b_heading">2.3. Relationship between <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> status</span></h3>
        <span>
          <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a> contains data from two papers<a title="Select to navigate to reference" href="#cit2"><sup><span class="sup_ref">2,42</span></sup></a> and an analysis of the relationship between <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate. There is scatter in the data and the absolute rates differ, however there is a significant inverse relationship between <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (<span class="italic">p</span> value for <a title="Select to navigate to references" href="#cit2">ref. 2</a> data set is 0.0006 and that for <a title="Select to navigate to references" href="#cit42">ref. 42</a> data set is 0.0014). A similar inverse relationship is also reported in <a title="Select to navigate to references" href="#cit131">ref. 131</a>. The product of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate will give a <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>-normalised production rate (in units of days<small><sup>1</sup></small>) and the relationship between this latter property and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content will take on a quadratic form, with an intermediate maximum. The first derivative of the quadratic equation will give the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content for the maximum <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production rate (mass of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> produced per mass of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> present in the culture per day). To demonstrate the outcome, and potential, of this procedure, we have processed the parameters from the two equations given in <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>. The analysis gives a maximum <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production rate at a <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content of <span class="italic">ca.</span> 15% (from the analysis in <a title="Select to navigate to references" href="#cit42">ref. 42s</a> data set) and <span class="italic">ca.</span> 30% (for <a title="Select to navigate to references" href="#cit2">ref. 2</a>). From these values, an optimum <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate for <span class="AN"><a class="simple" title="Gene Ontology link for: lipid metabolism" href="javascript:popupOBO('GO:0006629','b924978h')">lipid metabolism</a></span> may be calculated from the initial equation. Estimated maximum <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rates are valuable for process design and management, <span class="italic">e.g.</span> to allow culture dilution rates to be optimised without deleterious effect on overall <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> yield. However, bespoke data sets are needed to establish to what extent the relationship seen in <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a> is general. Overall production of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> will be more complex than this simple analysis, as there are many other considerations, but the calculation brings home that high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content alone cannot regarded as the sole, perhaps not even the major, consideration when searching for suitable strains or optimum <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> conditions.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><img alt="The variation of growth rate with lipid content. The data set from ref. 2 derives from 15 strains of freshwater and 11 strains of marine eukaryotic algae; the data used were taken during active logarithmic growth. The data from ref. 42 comes from 8 eukaryotic marine algae commonly used in aquaculture. In this case growth rate was controlled by varying the irradiance and the cultures were sampled during active growth." src="/image/article/2010/EE/b924978h/b924978h-f6.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">The variation of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate with <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content. The data set from <a title="Select to navigate to references" href="#cit2">ref. 2</a> derives from 15 strains of freshwater and 11 strains of marine eukaryotic algae; the data used were taken during active logarithmic <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>. The data from <a title="Select to navigate to references" href="#cit42">ref. 42</a> comes from 8 eukaryotic marine algae commonly used in aquaculture. In this case <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate was controlled by varying the irradiance and the cultures were sampled during active <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">It has been known since the 1940s that <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content increases during <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> limitation (see <a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a>). As <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> limitation also affects <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate, this provides an explanation for the apparent inverse relationship between <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (see <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>). <a title="Select to navigate to references" href="#cit43">Ref. 43</a> drew the broad conclusion that when the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> ran out, the organisms were forced to terminate the production of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> containing material (<span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span>) but continued to synthesise lipids and carbohydrates. Whereas it may explain the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span><span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> relationship, it cannot explain observations where <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> was controlled by light rather than inorganic <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> and where again a negative relationship was seen.<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> This, and other work, suggests that the matter may be more complex than <a title="Select to navigate to references" href="#cit43">ref. 43</a> suggested. Regardless of the biochemical mechanism, the evidence for of a negative relationship between <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (see <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>) has an important bearing on the strategy adopted for biomass production, especially when products other than the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> component are valuable (see <span class="italic">e.g.</span><a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a>).</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><img alt="The effect of nitrogen limitation on the lipid content of eukaryotic algae. The values in parenthesis are number of observations from which the mean was derived. The upper error bar is the standard deviation, the lower the standard error. Full data set and references in Appendix II (provided in the ESI)." src="/image/article/2010/EE/b924978h/b924978h-f7.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">The effect of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> limitation on the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content of eukaryotic algae. The values in parenthesis are number of observations from which the mean was derived. The upper error bar is the standard deviation, the lower the standard error. Full data set and references in Appendix II (provided in the ESI<a title="Select to navigate to footnote" href="#fn1"></a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1228"><span class="b_heading">2.4. Algal lipids as <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span></span></h3>
        <span>From their molecular composition, we can anticipate a number of properties relevant to the potential of algal lipids to serve as <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span>. A summary of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> properties of biodiesel from a microalga (heterotrophically grown <span class="italic">Chlorella</span><span class="italic"> protothecoides</span>)<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> is given in <a title="Select to navigate to table" href="#tab4">Table 4</a>. Algal oils differ from higher plant oils in their high <span class="CH"><a class="simple" title="ChEBI Ontology link for: phospholipid" href="javascript:popupOBO('CHEBI:16247','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16247')">phospholipid</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: glycolipid" href="javascript:popupOBO('CHEBI:33563','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33563')">glycolipid</a></span> concentrations. These <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> classes contain <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>, phosphorous and <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:sulfur" target="_blank" href="http://www.chemspider.com/Chemical-Structure.391.html">sulfur</a></span> that may be problematic with regards to engine performance, if present in <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span>. However, it is likely that the sulfur, phosphorus and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>-containing compounds would end up in the water-<span class="AN"><a class="simple" title="Gene Ontology link for: soluble fraction" href="javascript:popupOBO('GO:0005625','b924978h')">soluble fraction</a></span> following transesterification, so one would expect these elements to be low to non-existent in algal biodiesel. Since about 30% of the original <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> mass can be lost to the polar phase during esterification, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> class composition will also greatly affect the potential <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> yield by transesterification. Thus triglycerides have <span class="italic">a</span> &gt; 99% biodiesel yield compared to <span class="italic">a</span> &lt; 70% yield for phospholipids.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> If the relative distribution of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> classes given in <a title="Select to navigate to table" href="#tab3">Table 3</a> is used, then the overall mass yield on producing biodiesel is calculated to be about 80%. This figure does not take into consideration the consumption of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:methanol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.864.html">methanol</a></span> during esterification; if the methanol used is considered as part of the original organic supply, as it could well be, then the overall organic yield is reduced to <span class="italic">ca.</span> 72%.</span>
        
<div class="table_caption"><b>Table 4</b> <span id="tab4">Comparison of properties of biodiesel from microalgae and diesel <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> and the ASTM biodiesel standard. The data from <a title="Select to navigate to references" href="#cit44">ref. 44</a> comes from a heterotrophically grown <span class="italic">Chlorella</span><span class="italic"> protothecoides</span> culture. There are small differences between their observations and the values we calculate (given in square parentheses) from elemental composition and calorific values given in <a title="Select to navigate to table" href="#tab1">Table 1</a></span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/></colgroup>
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Properties</th>
                  <th align="left" valign="bottom" class="border_black">Micro-algal</th>
                  <th align="left" valign="bottom" class="border_black">Higher plant</th>
                  <th align="left" valign="bottom" class="border_black">Diesel <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span></th>
                  <th align="left" valign="bottom" class="border_black">ASTM biodiesel standard</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Density/kg dm<small><sup>3</sup></small></td>
                  <td align="char" valign="top">0.864</td>
                  <td align="char" valign="top">0.8770.887</td>
                  <td align="char" valign="top">0.838</td>
                  <td align="char" valign="top">0.860.90</td>
                </tr>
                <tr align="left">
                  <td valign="top">Viscosity/mm<small><sup>2</sup></small> s<small><sup>1</sup></small>@ 40 C</td>
                  <td align="char" valign="top">5.2</td>
                  <td align="char" valign="top">3.35.2</td>
                  <td align="char" valign="top">1.94.1</td>
                  <td align="char" valign="top">3.55.0</td>
                </tr>
                <tr align="left">
                  <td valign="top">Flash point/C</td>
                  <td align="char" valign="top">115</td>
                  <td align="char" valign="top"></td>
                  <td align="char" valign="top">75</td>
                  <td align="char" valign="top">Min. 100</td>
                </tr>
                <tr align="left">
                  <td valign="top">Solidifying point/C</td>
                  <td align="char" valign="top">12</td>
                  <td align="char" valign="top"></td>
                  <td align="char" valign="top">5010</td>
                  <td align="char" valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Filter plugging point/C</td>
                  <td align="char" valign="top">11</td>
                  <td align="char" valign="top"></td>
                  <td align="char" valign="top">3.0 (max. 6.7)</td>
                  <td align="char" valign="top">Summer max. 0</td>
                </tr>
                <tr align="left">
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td align="char" valign="top">Winter max. &lt;  15</td>
                </tr>
                <tr align="left">
                  <td valign="top">Acid value/mg KOH g<small><sup>1</sup></small></td>
                  <td align="char" valign="top">0.374</td>
                  <td align="char" valign="top">0.160.43</td>
                  <td align="char" valign="top">Max. 0.5</td>
                  <td align="char" valign="top">Max. 0.5</td>
                </tr>
                <tr align="left">
                  <td valign="top">Heating value/kJ g<small><sup>1</sup></small></td>
                  <td align="char" valign="top">41 [43]</td>
                  <td align="char" valign="top">39.540.3</td>
                  <td align="char" valign="top">4045</td>
                  <td align="char" valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span>/C ratio</td>
                  <td align="char" valign="top">1.8 [1.9]</td>
                  <td valign="top">
                  </td>
                  <td align="char" valign="top">1.8</td>
                  <td align="char" valign="top"></td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">The overall heating value of crude algal lipid is somewhat depressed to a value in the region of 36 kJ g<small><sup>1</sup></small> (<a title="Select to navigate to table" href="#tab1">Table 1</a>), due to the lower calorific value of the glyco- and phospholipids. However, as these fractions would be almost certainly separated off in the transesterification process, their low calorific value would have no effect on the properties of the final biodiesel. From their calculated elemental composition (based on their reported <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> composition), we determine a heating value for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> methyl esters of 43 kJ g<small><sup>1</sup></small>, slightly higher than the figure of 41 kJ g<small><sup>1</sup></small> found in <a title="Select to navigate to references" href="#cit44">ref. 44</a>. This derives from a slightly higher value in our case for the C<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span> ratio (see <a title="Select to navigate to table" href="#tab4">Table 4</a>). The difference is small and, considering the two approaches used are so different, we do not place any great significance on the variance between these two sets of values.</p>
        <p class="otherpara">The <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> characteristics will be greatly determined by the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> composition and will thus vary with species and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> conditions of the algal culture. As shown in <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>, the chain length distribution of fatty acids occurring in microalgal species is more diverse compared with that of higher plants, making it possible that certain species will be cultured for selected <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> properties. For example, strains that primarily accumulate shorter (&lt;C<small><sub>16</sub></small>) chain fatty acids may be more amenable for use in the production of jet <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span>, compared to the application of very long chain fatty acids (&gt;C<small><sub>20</sub></small>) in the lubricant market.</p>
      
    
    
      
      <h2 id="sect1375"><span class="a_heading">3. Principles and efficiency of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span></span></h2>
      <span>Since the basis of algal biomass production is directly proportional to the efficiency with which the algal <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> assimilate carbon from the atmosphere through <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>, we discuss the basic principles and efficiency of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> in this section. The aim of this section is to provide a background to the theoretical and practical yield calculations in the subsequent sections so that the limitations to mass algal production can be made clear.</span>
      <p class="otherpara"><span class="AN"><a class="simple" title="Gene Ontology link for: Photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">Photosynthesis</a></span> is the predominant process maintaining a whole <span class="AN"><a class="simple" title="Gene Ontology link for: host" href="javascript:popupOBO('GO:0018995','b924978h')">host</a></span> of elements (notably carbon, <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> and <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:sulfur" target="_blank" href="http://www.chemspider.com/Chemical-Structure.391.html">sulfur</a></span>) out of thermodynamic equilibrium, and thereby driving their global geochemical cycles. It is the basis of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: food" href="javascript:popupOBO('CHEBI:33290','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')">food</a></span> supply for most life on Earth, maintaining the biosphere; it is also the ultimate source of all <span class="CH"><a class="simple" title="ChEBI Ontology link for: fossil fuels" href="javascript:popupOBO('CHEBI:35230','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35230')">fossil fuels</a></span>. <span class="AN"><a class="simple" title="Gene Ontology link for: Photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">Photosynthesis</a></span>, in turn, is driven by <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>, which, when absorbed by <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules, give rise to a charge separation and the ejection of electrons. These electrons are driving the dissociation of <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span>, generating <span class="CH"><a class="simple" title="ChEBI Ontology link for: protons" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">protons</a></span> and oxygen. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: protons" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">protons</a></span> and the associated electrons enable the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> of carbon dioxide to organic material, whereby oxygen is essentially a waste product.</p>
      
        
        <h3 id="sect1405"><span class="b_heading">3.1. Mechanism of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span></span></h3>
        <span>The simple stoichiometry of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> may be written as:<br/><table><tr><td class="eqnText">H<small><sub>2</sub></small>O + CO<small><sub>2</sub></small>  [CH<small><sub>2</sub></small>O]<a title="Select to navigate to footnote" href="#fn4"></a> + O<small><sub>2</sub></small></td></tr></table></span>
        <p class="otherpara">However, the above simple equation implies that at least 50% of the atoms of the oxygen produced must come from the carbon dioxide, whereas both derive from <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span>, thus the more appropriate equation is:<br/><table><tr><td class="eqnText">2H<small><sub>2</sub></small>O + CO<small><sub>2</sub></small>  [CH<small><sub>2</sub></small>O] + O<small><sub>2</sub></small> + H<small><sub>2</sub></small>O</td></tr></table></p>
        <p class="otherpara">This overall reaction can be separated into two phases: (i) a set of photochemical and <span class="AN"><a class="simple" title="Gene Ontology link for: redox" href="javascript:popupOBO('GO:0016491','b924978h')">redox</a></span> reactions (conventionally called the light reaction) and ii) a sequence of enzymatic reactions, often referred to as dark reactions but better as light-independent reactions, as they occur both in the light and the dark.<br/><table><tr><td class="eqnText">Light reaction: 2H<small><sub>2</sub></small>O  4[H] + O<small><sub>2</sub></small> + energy</td></tr></table><br/><table><tr><td class="eqnText">Light-independent reactions: 4[H] + CO<small><sub>2</sub></small>  [CH<small><sub>2</sub></small>O] + H<small><sub>2</sub></small>O</td></tr></table></p>
        <p class="otherpara">(The notation [<span class="TC"><a class="simple" title="Chemspider Compound link for:H" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4575379.html">H</a></span>] refers to the combination of the reduced <span class="CH"><a class="simple" title="ChEBI Ontology link for: coenzyme" href="javascript:popupOBO('CHEBI:23354','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23354')">coenzyme</a></span> nicotinamide adenine dinucleotide phosphate (NADPH) and an electron.)</p>
        <p class="otherpara">These reactions + O<small><sub>2</sub></small> + energy are intimately connected within the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span>. The light reaction operates on very short time scales (from femtoseconds to milliseconds) whereas the light-independent reaction operates over timespans of seconds to hours. This profound mismatch of timescales gives rise to inefficiencies in fluctuating environmental circumstances, notably irradiance and temperature variations and is a major problem in maximising yields of mass algal culture.</p>
        <div>
          
          <span id="sect1446"/><span class="c_heading_indent">(i) Light, its absorption and the formation of reducing capacity. </span>
          <span>Not all incoming radiation is available for <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. The solar <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> is described by Planck's radiation distribution equation; the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> of light arriving at the surface of the planet has been attenuated by some 30% by the gases in the atmosphere and losses due to <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: light scattering" href="javascript:popupOBO('CMO:0000166','b924978h')">light scattering</a></span> and absorption by clouds. The irradiance <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> is shown in <a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>. The primary pigment involved in <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> is <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:chlorophyll a" target="_blank" href="http://www.chemspider.com/Chemical-Structure.16736115.html">chlorophyll a</a></span>, which has strong absorption bands in the regions 400450 and 650700 nm (see <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>). This delimits the useful range of incoming radiation to 400700 nm  so called photosynthetically active radiation (PAR). PAR amounts to 4550% of the total incoming radiation, the exact value mainly being determined by the moisture content of the atmosphere attenuating the infrared part of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span>.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> Since we use clear sky radiation as the basis for further discussion here, the 45% end of the spread is the more appropriate value to adopt.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><img alt="The distribution of energy (blue line) and photons flux (red line) in incoming radiation. The lower green line shows the residual energy after it has been transferred to a frequency equivalent to that of chlorophyll a at 680700 nm absorption bands." src="/image/article/2010/EE/b924978h/b924978h-f8.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">The distribution of energy (blue line) and <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> flux (red line) in incoming radiation. The lower green line shows the residual energy after it has been transferred to a frequency equivalent to that of <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll a" href="javascript:popupOBO('CHEBI:18230','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18230')">chlorophyll a</a></span> at 680700 nm absorption bands.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><img alt="The spectrum of incoming radiation (black line) and the absorption spectrum of chlorophyll and accessory photosynthetic pigments. Redrawn with permission from ref. 46." src="/image/article/2010/EE/b924978h/b924978h-f9.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">The <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> of incoming radiation (black line) and the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: absorption spectrum" href="javascript:popupOBO('CMO:0000801','b924978h')">absorption spectrum</a></span> of <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> and accessory photosynthetic pigments. Redrawn with permission from <a title="Select to navigate to references" href="#cit46">ref. 46</a>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Because of its low absorption in the range 450650 nm, <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll a" href="javascript:popupOBO('CHEBI:18230','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18230')">chlorophyll a</a></span> itself only is able to capture some 3040% of PAR. Plants have overcome this by introducing additional light capturing pigments (<span class="italic">e.g.</span> -carotene) that fill in much of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll a" href="javascript:popupOBO('CHEBI:18230','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18230')">chlorophyll a</a></span> window of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> (see <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>) so increasing the portion of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> that can be used for <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. Algae can change the quantity of these accessory pigments (subject to phylogenetic constraints) to optimise light capture, but the adaptation process is slow compared with the time scales of the photochemical reactions. The photosynthetically active parts of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span> lie at 680 and 700 nm (see <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>). The energy of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>, captured at shorter wavelengths, can be transferred to the 680700 nm region very efficiently on a quantum basis. However, there is an inevitable loss of energy resulting from the transfer from high-energy, shorter wavelength to lower energy, longer wavelength, <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>. Thus, although the resultant quantum efficiency may be more or less constant throughout the PAR <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: spectrum" href="javascript:popupOBO('CMO:0000800','b924978h')">spectrum</a></span>, there is a net loss of some 21% of the original energy; this is illustrated in <a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>.</p>
          <p class="otherpara">The broad mechanism of the conversion of the photochemical energy into metabolic energy and reducing capability is complex, although comparatively well understood. Briefly, there are two functionally separate sites of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> absorption, coupled in tandem by a chain of <span class="AN"><a class="simple" title="Gene Ontology link for: redox" href="javascript:popupOBO('GO:0016491','b924978h')">redox</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: carrier" href="javascript:popupOBO('GO:0022804','b924978h')">carrier</a></span> molecules. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> absorption elicits a charge separation of at two reaction sites, at <span class="AN"><a class="simple" title="Gene Ontology link for: photosystem II" href="javascript:popupOBO('GO:0009523','b924978h')">photosystem II</a></span> (PSII; <span class="italic"></span> = 680 nm) there is a charge separation of 1.7 V and 1.6 V at <span class="AN"><a class="simple" title="Gene Ontology link for: photosystem I" href="javascript:popupOBO('GO:0009522','b924978h')">photosystem I</a></span> (PSI; <span class="italic"></span> = 700 nm) requiring respectively 164 and 154 kJ per einstein.<a title="Select to navigate to footnote" href="#fn5">||</a> This is depicted in <a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a> as the so-called Z-scheme of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. The electron flow away from the <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules draws electrons from <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span>. Whereas the formation of electrons operates on a one-<span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span>-per-electron basis, the dissociation of <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> requires an accumulation of four electrons to effect the production of one molecule of oxygen. A tetra-<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:manganese" target="_blank" href="http://www.chemspider.com/Chemical-Structure.22372.html">manganese</a></span> complex acts as an accumulator, being oxidised stepwise at four stages, the fully oxidised form then draws the four electrons from two molecules of <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> in one step, producing a molecule of oxygen plus four <span class="CH"><a class="simple" title="ChEBI Ontology link for: protons" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">protons</a></span>. The functioning of the water splitting system is described in detail in <a title="Select to navigate to references" href="#cit48">ref. 48</a>. This whole complex of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> capturing mechanisms, charge separation, generation of metabolic energy and reducing capability, and the water splitting system is embedded in the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b924978h')">membrane</a></span> of flattened sac-like structures present in the <span class="AN"><a class="simple" title="Gene Ontology link for: chloroplast" href="javascript:popupOBO('GO:0009507','b924978h')">chloroplast</a></span>, known as thylakoids.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><img alt="Illustration of the light reactions of photosynthesis (the so-called Z-scheme). The major functional units are represented as oval shapes; photosystem II (PSII), plastoquinone (PQ), plastocyanin (PC), cytochrome b6f complex (Cyt b6f), photosystem I (PSI), ferredoxin (Fd), ferredoxin-NADP reductase (FNR) (in order of electron transport chain) and ATP synthase. P680 and P700, refer to the reaction centres of photosystem II (PSII) and I (PSI) respectively, the asterisk (*) indicates the excited state. The inset shows a schematic close-up of the light harvesting complex (LHC)." src="/image/article/2010/EE/b924978h/b924978h-f10.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">Illustration of the light reactions of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> (the so-called Z-scheme). The major functional units are represented as oval shapes; <span class="AN"><a class="simple" title="Gene Ontology link for: photosystem II" href="javascript:popupOBO('GO:0009523','b924978h')">photosystem II</a></span> (PSII), <span class="TC"><a class="simple" title="Chemspider Compound link for:plastoquinone" target="_blank" href="http://www.chemspider.com/Chemical-Structure.4938558.html">plastoquinone</a></span> (PQ), <span class="AN"><a class="simple" title="Gene Ontology link for: plastocyanin" href="javascript:popupOBO('GO:0046028','b924978h')">plastocyanin</a></span> (PC), <span class="AN"><a class="simple" title="Gene Ontology link for: cytochrome b6f complex" href="javascript:popupOBO('GO:0009512','b924978h')">cytochrome b6f complex</a></span> (Cyt b6f), <span class="AN"><a class="simple" title="Gene Ontology link for: photosystem I" href="javascript:popupOBO('GO:0009522','b924978h')">photosystem I</a></span> (PSI), <span class="CH"><a class="simple" title="ChEBI Ontology link for: ferredoxin" href="javascript:popupOBO('CHEBI:5017','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=5017')">ferredoxin</a></span> (Fd), <span class="AN"><a class="simple" title="Gene Ontology link for: ferredoxin-NADP reductase" href="javascript:popupOBO('GO:0004324','b924978h')">ferredoxin-NADP reductase</a></span> (FNR) (in order of <span class="AN"><a class="simple" title="Gene Ontology link for: electron transport chain" href="javascript:popupOBO('GO:0022900','b924978h')">electron transport chain</a></span>) and <span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span> synthase. P680 and P700, refer to the <span class="AN"><a class="simple" title="Gene Ontology link for: reaction centres" href="javascript:popupOBO('GO:0009521','b924978h')">reaction centres</a></span> of <span class="AN"><a class="simple" title="Gene Ontology link for: photosystem II" href="javascript:popupOBO('GO:0009523','b924978h')">photosystem II</a></span> (PSII) and I (PSI) respectively, the asterisk (*) indicates the excited state. The inset shows a schematic close-up of the <span class="AN"><a class="simple" title="Gene Ontology link for: light harvesting" href="javascript:popupOBO('GO:0031992','b924978h')">light harvesting</a></span> complex (LHC).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The electrons pumped by the two <span class="AN"><a class="simple" title="Gene Ontology link for: reaction centres" href="javascript:popupOBO('GO:0009521','b924978h')">reaction centres</a></span> eventually give rise to the production of the reducing agent (NADPH) used in the process of carbon assimilation. Two molecules of NADPH are being produced per four electrons transported with a free energy gain of 220 kJ mol<small><sup>1</sup></small> NADPH. At the same time <span class="CH"><a class="simple" title="ChEBI Ontology link for: protons" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">protons</a></span> are pumped across the <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b924978h')">membrane</a></span> into the inner cavity of the <span class="AN"><a class="simple" title="Gene Ontology link for: thylakoid" href="javascript:popupOBO('GO:0009579','b924978h')">thylakoid</a></span> (the lumen). This sets up a charge gradient. On their return, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protons" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">protons</a></span> spin a molecular rotor,<a title="Select to navigate to footnote" href="#fn6">**</a> which gives rise to the synthesis of <span class="TC"><a class="simple" title="Chemspider Compound link for:adenosine triphosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">adenosine triphosphate</a></span> (<span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span>), the biological energy currency. In total, three <span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span> molecules are formed per 12 <span class="CH"><a class="simple" title="ChEBI Ontology link for: protons" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">protons</a></span> transported, with a free energy gain of <span class="italic">ca.</span> 50 kJ mol<small><sup>1</sup></small> <span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span>;<a title="Select to navigate to footnote" href="#fn7"></a> along with the two NADPH molecules; the total potential energy yield is 590 kJ per 4 moles electrons transported. Although in theory this could be supplied by four <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> (two einsteins at 680 nm and two at 700 nm would yield 692 kJ), experimental observations suggest 810 <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span><a title="Select to navigate to footnote" href="#fn8"></a> are needed per four electrons transported.</p>
          <p class="otherpara">Some of the intermediates in the system have a limited lifespan, and the statistical probability of arrival of 8 <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> at a single capturing site within the required time frame is low. Nature has overcome this problem by linking together approximately 2000 <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules in a <span class="AN"><a class="simple" title="Gene Ontology link for: light harvesting" href="javascript:popupOBO('GO:0031992','b924978h')">light harvesting</a></span> complex (LHC) with a much smaller number of <span class="AN"><a class="simple" title="Gene Ontology link for: reaction centre" href="javascript:popupOBO('GO:0009521','b924978h')">reaction centre</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules. On average there are about 300 <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules per active <span class="AN"><a class="simple" title="Gene Ontology link for: reaction centre" href="javascript:popupOBO('GO:0009521','b924978h')">reaction centre</a></span>. This is termed the photosynthetic unit (see inset in <a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>). The captured energy of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> very rapidly passes through this network of <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules, reaching the <span class="AN"><a class="simple" title="Gene Ontology link for: reaction centre" href="javascript:popupOBO('GO:0009521','b924978h')">reaction centre</a></span> within a timescale of <span class="italic">ca.</span> 10<small><sup>10</sup></small> s.</p>
        </div>
        <div>
          
          <span id="sect1589"/><span class="c_heading_indent">(ii) The light-independent reactions. </span>
          <span>The reducing power (NADPH + H<small><sup>+</sup></small>) and energy (<span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span>) produced by the light reaction is used in the enzymatic light-independent part of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> to enable the incorporation of CO<small><sub>2</sub></small> into organic material and its subsequent <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> (see <a title="Select to navigate to figure" href="#imgfig11">Fig. 11</a>). The calorific value of the product is around 469 kJ mol<small><sup>1</sup></small> C. The first committed step is the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: carboxylation" href="javascript:popupOBO('MOP:0000591','b924978h')">carboxylation</a></span> of the sugar ribulose 1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>5 bisphosphate (Ru5BP) by the <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span> <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ribulose" target="_blank" href="http://www.chemspider.com/Chemical-Structure.133316.html">ribulose</a></span> bisphosphate carboxylase oxygenase (<span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span>). <span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span> exists as large heteromultimeric <span class="AN"><a class="simple" title="Gene Ontology link for: protein complex" href="javascript:popupOBO('GO:0043234','b924978h')">protein complex</a></span> (536 kDa) and is an outstandingly sluggish <span class="CH"><a class="simple" title="ChEBI Ontology link for: catalyst" href="javascript:popupOBO('CHEBI:35223','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35223')">catalyst</a></span>, fixing 210 molecules of CO<small><sub>2</sub></small> per active site per second. In addition, it has the curious property of catalysing both the carboxylation and <span class="CH"><a class="simple" title="Molecular Process Ontology link for: oxidation" href="javascript:popupOBO('MOP:0000568','b924978h')">oxidation</a></span> of its substrate. Which reaction predominates depends upon the ratio of the partial pressures of the two gases; CO<small><sub>2</sub></small> and O<small><sub>2</sub></small>. The <span class="CH"><a class="simple" title="Molecular Process Ontology link for: oxidation" href="javascript:popupOBO('MOP:0000568','b924978h')">oxidation</a></span> reaction is wasteful of energy. Most, but not all, aquatic microalgae overcome the predominance of oxygen over carbon dioxide in oceans (<span class="italic">ca.</span> 30-fold) by actively pumping in CO<small><sub>2</sub></small> and so increasing its concentration around <span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span>.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig11"><img alt="Schematic simplified representation of the Calvin-Benson cycle in three parts, (i) CO2 fixation, (ii) reduction and (iii) regeneration. The average cycling time of one round of CO2 assimilation is 100 to 500 ms. The necessary energy (ATP) and reductant (NADPH+ + H+) (not shown stoichiometrically) are originating from the photosynthetic light reactions. RuBisCO: ribulose bisphosphate carboxylase/oxygenase; RuBP: ribulose-1,5-bisphosphate; 3-PGA: 3-phosphoglycerate; 1,3-BPGA: 1,3 bis- phosphoglycerate; G3P: glyceraldehyde-3-phosphate; F6P: fructose-6-phosphate; G6P: glucose-6-phosphate; PEP: phosphoenolpyruvate." src="/image/article/2010/EE/b924978h/b924978h-f11.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 11 </b> <span id="fig11"><span class="graphic_title">Schematic simplified representation of the Calvin-Benson cycle in three parts, (i) CO<small><sub>2</sub></small> fixation, (ii) <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> and (iii) <span class="AN"><a class="simple" title="Gene Ontology link for: regeneration" href="javascript:popupOBO('GO:0031099','b924978h')">regeneration</a></span>. The average cycling time of one round of CO<small><sub>2</sub></small> assimilation is 100 to 500 ms. The necessary energy (<span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span>) and reductant (NADPH<small><sup>+</sup></small> + H<small><sup>+</sup></small>) (not shown stoichiometrically) are originating from the photosynthetic light reactions. <span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span>: <span class="AN"><a class="simple" title="Gene Ontology link for: ribulose bisphosphate carboxylase/oxygenase" href="javascript:popupOBO('GO:0016984','b924978h')">ribulose bisphosphate carboxylase/oxygenase</a></span>; RuBP: ribulose-1,5-bisphosphate; 3-PGA: <span class="TC"><a class="simple" title="Chemspider Compound link for:3-phosphoglycerate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.10669764.html">3-phosphoglycerate</a></span>; 1,3-BPGA: 1,3 bis- <span class="TC"><a class="simple" title="Chemspider Compound link for:phosphoglycerate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.10701835.html">phosphoglycerate</a></span>; G3P: glyceraldehyde-3-phosphate; F6P: <span class="TC"><a class="simple" title="Chemspider Compound link for:fructose-6-phosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.389526.html">fructose-6-phosphate</a></span>; G6P: <span class="TC"><a class="simple" title="Chemspider Compound link for:glucose-6-phosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.388417.html">glucose-6-phosphate</a></span>; PEP: phosphoenolpyruvate.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara"><span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span> plays a central role in all plant <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>, it accounts for a major fraction of all living <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>. It has long been a puzzle why an <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span>, of such crucial importance to life on Earth, has seemingly remained so inefficient since there have been a billion or more years for improvements to have evolved. It appears, however, that the <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span> is nearly perfectly optimised<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> and that the perceived inefficiency is a fallacy.</p>
        </div>
      
    
    
      <span>The first stable products of the reaction are 3-carbon organic acids. It is from these compounds that all major biochemicals (fats, fatty acids, <span class="CH"><a class="simple" title="ChEBI Ontology link for: sugars" href="javascript:popupOBO('CHEBI:16646','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16646')">sugars</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> <span class="italic">etc.</span>) are eventually formed (see <a title="Select to navigate to figure" href="#imgfig11">Fig. 11</a>). <span class="CH"><a class="simple" title="Molecular Process Ontology link for: Carboxylation" href="javascript:popupOBO('MOP:0000591','b924978h')">Carboxylation</a></span> of Ru5BP in the <span class="AN"><a class="simple" title="Gene Ontology link for: Calvin cycle" href="javascript:popupOBO('GO:0019253','b924978h')">Calvin cycle</a></span> leads to the production of two molecules of <span class="TC"><a class="simple" title="Chemspider Compound link for:3-phosphoglyceric acid" target="_blank" href="http://www.chemspider.com/Chemical-Structure.15858329.html">3-phosphoglyceric acid</a></span> (3-PGA), which is subsequently phosphorylated to 1,3-bisphosphoglycerate (1,3-BPGA) and reduced to glyceraldehyde-3-phosphate (G3P). This <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> step is where the reducing power generated during the photochemical reactions is used. These set of three carbon compounds are the building blocks for the synthesis of the basic biochemical fractions (see insert in <a title="Select to navigate to figure" href="#imgfig11">Fig. 11</a>). The biochemistry is complex<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> and incurs further demands on energy and, where acetyl-CoA in an intermediate (lipids and some amino acids), loss of carbon and oxygen, and therefore biomass, as carbon dioxide. A simplified analysis is made in Box 1 for triglycerides and <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> based on hexoses as the starting point. <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a> shows the biomass loss with increase in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production. Two matters are important to note here: first high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content is achieved at a cost of biomass loss; second, whereas theoretical calculations of photosynthetic efficiency are based on <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:hexose" target="_blank" href="http://www.chemspider.com/Chemical-Structure.23139.html">hexose</a></span> production, the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> ultimately produce biomass which comprises lipids as well as <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, this results in something in the region of a 3050% loss of the original photosynthetically produced organic material (see <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a>).</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig12"><img alt="Graph of the calculated yields of biomass (green line) from hexose for biomass with varying lipid contents. The protein:polysaccharide ratio is taken to be constant at 3:2, this assumption has a minor effect on the outcome of the calculations. The calculation is based on eqn (5), Box 1. Shown also is the hexose energy required to synthesise biomass of varying lipid content. The hexose is assumed to have a calorific value of 15.7 kJ g1." src="/image/article/2010/EE/b924978h/b924978h-f12.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 12 </b> <span id="fig12"><span class="graphic_title">Graph of the calculated yields of biomass (green line) from <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> for biomass with varying <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> contents. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="CH"><a class="simple" title="ChEBI Ontology link for: polysaccharide" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">polysaccharide</a></span> ratio is taken to be constant at 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2, this assumption has a minor effect on the outcome of the calculations. The calculation is based on <a title="" href="#eqn5">eqn (5)</a>, Box 1. Shown also is the <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> energy required to synthesise biomass of varying <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> is assumed to have a calorific value of 15.7 kJ g<small><sup>1</sup></small>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <br/><table class="box_border"><tr><td>
        
          
          <h2 id="sect1695"><span class="a_heading">Box 1: Biomass and energy losses incurred during the <span class="AN"><a class="simple" title="Gene Ontology link for: biosynthesis" href="javascript:popupOBO('GO:0009058','b924978h')">biosynthesis</a></span> of lipids and <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span></span></h2>
          <span>
            <span class="AN"><a class="simple" title="Gene Ontology link for: Lipid synthesis" href="javascript:popupOBO('GO:0008610','b924978h')"><span class="italic">Lipid synthesis</span></a></span>: The <span class="AN"><a class="simple" title="Gene Ontology link for: biosynthesis" href="javascript:popupOBO('GO:0009058','b924978h')">biosynthesis</a></span> of high calorific value compounds such as lipids from the primary products of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>  hexoses  must incur a metabolic penalty in the form of loss of mass. A simple minimum calculation may be made from the change in calorific value from 15.7 kJ g<small><sup>1</sup></small> (a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>) to 36 kJ g<small><sup>1</sup></small> (a generic algal <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>). Thus, to conserve energy there must be a mass loss of about 2.5. This, however, is a minimum calculation, as the 2nd law of thermodynamics calls for some energy, and therefore additional mass, loss. This can be estimated from the biochemical stoichiometry of the various reactions during the transformation of hexoses into lipids. If there is a negative energy balance, then as well as the mass losses during the biochemical transformations the production of CO<small><sub>2</sub></small> (see equation below), additional mass will be lost to provide the extra energy, which must come from respiration. The overall synthesis of the primary <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> product  <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:palmitic acid" target="_blank" href="http://www.chemspider.com/Chemical-Structure.960.html">palmitic acid</a></span>  from <span class="CH"><a class="simple" title="ChEBI Ontology link for: glucose" href="javascript:popupOBO('CHEBI:17234','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=17234')">glucose</a></span> may be divided into two phases: (i) the formation of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:acetyl CoA" target="_blank" href="http://www.chemspider.com/Chemical-Structure.392413.html">acetyl CoA</a></span> and (ii) and the conversion of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:acetyl CoA" target="_blank" href="http://www.chemspider.com/Chemical-Structure.392413.html">acetyl CoA</a></span> to <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:palmitic acid" target="_blank" href="http://www.chemspider.com/Chemical-Structure.960.html">palmitic acid</a></span>.<table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn1"/><span id="eqn1">4C<small><sub>6</sub></small>H<small><sub>12</sub></small>O<small><sub>6</sub></small>  8 acetyl CoA + 8CO<small><sub>2</sub></small> + 8ATP + 16 NAD(P)H</span></td><td class="rightEqn">(1)</td></tr></table><table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn2"/><span id="eqn2">8 acetyl CoA + 7ATP + 14 NAD(P)H  CH<small><sub>3</sub></small>(CH<small><sub>2</sub></small>)<small><sub>14</sub></small>COOH</span></td><td class="rightEqn">(2)</td></tr></table></span>
          <p class="otherpara">Overall:<table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn3"/><span id="eqn3">4C<small><sub>6</sub></small>H<small><sub>12</sub></small>O<small><sub>6</sub></small>  CH<small><sub>3</sub></small>(CH<small><sub>2</sub></small>)<small><sub>14</sub></small>COOH + 8CO<small><sub>2</sub></small> + ATP + 2NAD(P)H</span></td><td class="rightEqn">(3)</td></tr></table></p>
          <p class="otherpara">Thus, there is a small energy gain in the form of an ATP and two <span class="CH"><a class="simple" title="ChEBI Ontology link for: NAD(P)H" href="javascript:popupOBO('CHEBI:13392','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=13392')">NAD(P)H</a></span>; however there will be the loss of 3ATP molecules for the formation of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: triglyceride" href="javascript:popupOBO('CHEBI:17855','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=17855')">triglyceride</a></span> and additional losses when <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:palmitic acid" target="_blank" href="http://www.chemspider.com/Chemical-Structure.960.html">palmitic acid</a></span> is desaturated to form other fatty acids. These energy gains and losses are small (100200 kJ per <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:palmitic acid" target="_blank" href="http://www.chemspider.com/Chemical-Structure.960.html">palmitic acid</a></span> molecule) as compared with the overall energy present in the molecule (<span class="italic">ca.</span> 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 kJ mol<small><sup>1</sup></small>) and can be largely ignored. There will be a small energy gain if the <span class="CH"><a class="simple" title="ChEBI Ontology link for: triose" href="javascript:popupOBO('CHEBI:27137','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=27137')">triose</a></span> phosphates, rather than the <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: sugars" href="javascript:popupOBO('CHEBI:16646','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16646')">sugars</a></span> are considered to be the starting point for <span class="AN"><a class="simple" title="Gene Ontology link for: lipid biosynthesis" href="javascript:popupOBO('GO:0008610','b924978h')">lipid biosynthesis</a></span>.</p>
          <p class="otherpara">Overall, the conversion of 4 molecules of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> to 1 mole of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:palmitic acid" target="_blank" href="http://www.chemspider.com/Chemical-Structure.960.html">palmitic acid</a></span> results in a drop in calorific value from 11<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>446 to 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>117 kJ. Thus, these calculations imply that, whereas there is a small loss of energy (<span class="italic">ca.</span> 10%), there is a substantial (2.9-fold) loss of mass. Algal lipid contains a substantial amount of <span class="CH"><a class="simple" title="ChEBI Ontology link for: phospholipid" href="javascript:popupOBO('CHEBI:16247','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16247')">phospholipid</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: glycolipid" href="javascript:popupOBO('CHEBI:33563','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33563')">glycolipid</a></span>. The mass loss in making <span class="CH"><a class="simple" title="ChEBI Ontology link for: phospholipid" href="javascript:popupOBO('CHEBI:16247','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16247')">phospholipid</a></span> may be estimated to be about 2.7 and making <span class="CH"><a class="simple" title="ChEBI Ontology link for: glycolipid" href="javascript:popupOBO('CHEBI:33563','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33563')">glycolipid</a></span> 2.3  these values are not as certain as the <span class="CH"><a class="simple" title="ChEBI Ontology link for: triglyceride" href="javascript:popupOBO('CHEBI:17855','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=17855')">triglyceride</a></span> figure. If we take the mean figures for the proportions of these components of algal lipid given in <a title="Select to navigate to table" href="#tab3">Table 3</a>, a weighted mean value of 2.6-fold loss of mass is obtained for algal lipid formation from <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>; this has been used for the calculation of the mass yield of <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> with varying <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content.</p>
          <p class="otherpara">
            <span class="AN"><a class="simple" title="Gene Ontology link for: Protein synthesis" href="javascript:popupOBO('GO:0006412','b924978h')"><span class="italic">Protein synthesis</span></a></span>: The parallel calculation for the conversion of the photosynthetic products to <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> is more complex. The elemental composition of algal <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> may be calculated from the <span class="CH"><a class="simple" title="ChEBI Ontology link for: amino acid" href="javascript:popupOBO('CHEBI:33709','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33709')">amino acid</a></span> analysis as C<small><sub>1</sub></small>H<small><sub>1.56</sub></small>O<small><sub>0.30</sub></small>N<small><sub>0.26</sub></small>S<small><sub>0.006</sub></small> (see <a title="Select to navigate to table" href="#tab1">Table 1</a>); the loss of mass on production <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> from a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>, assuming no respiration of carbon, is 1.36-fold, much less than the formation of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>. The energy gain on a mass basis is much the same as the mass loss (1.33) on a carbon basis. However, there will also be metabolic work at two stages: (i) during the synthesis of the individual amino acids; and (ii) on their conversion to <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> (<a title="Select to navigate to references" href="#cit125">ref. 125</a> and personal communication) makes a very comprehensive analysis of the mass yield of synthesising the amino acids for <span class="AN"><a class="simple" title="Gene Ontology link for: protein synthesis" href="javascript:popupOBO('GO:0006412','b924978h')">protein synthesis</a></span> and obtains a value of 1.65 g g<small><sup>1</sup></small> <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>. The energy for the second reaction, which incurs a considerable <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in entropy, is estimated to be 120 kJ per <span class="CH"><a class="simple" title="ChEBI Ontology link for: peptide" href="javascript:popupOBO('CHEBI:16670','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16670')">peptide</a></span> bond.<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> The energy required per gram of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> synthesised can be estimated from the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> content of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and is small fraction (1.4 kJ g<small><sup>1</sup></small>, <span class="italic">i.e.</span> 6%) of the overall calorific value of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, thus we may take a round figure of 1.7 g for the mass of <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> requires to synthesise 1 g of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>.</p>
          <p class="otherpara">
            <span class="CH"><a class="simple" title="ChEBI Ontology link for: Nucleic acid" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')"><span class="italic">Nucleic acid</span></a></span><span class="italic"> synthesis</span>: The calculation for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> is extremely complex and, in view of their relatively small contribution to the overall mass of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span>, the calculation has not been made.</p>
          <p class="otherpara">
            <span class="italic">Calculating the mass yields</span>: given the values calculated above, the general equation for the biomass yield is<table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn4"/><span id="eqn4"><span class="italic">Y</span>= 1/(1.11<span class="italic">C</span> + 1.7<span class="italic">P</span> + 2.6<span class="italic">L</span>)</span></td><td class="rightEqn">(4)</td></tr></table></p>
          <p class="otherpara">where <span class="italic">C</span> is the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> content (assuming it to be a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>-based <span class="CH"><a class="simple" title="ChEBI Ontology link for: polysaccharide" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">polysaccharide</a></span>), <span class="italic">P</span> is the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content and <span class="italic">L</span> the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content, <span class="italic">Y</span> is the biomass yield in mass per mass <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> synthesised. Note: <span class="italic">C</span> + <span class="italic">P</span> + <span class="italic">L</span> = 1. If it is assumed that the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> to <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio is 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 and that this ratio remains constant with increase in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content, then the equation simplifies to:<table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn5"/><span id="eqn5"><span class="italic">Y</span> = 1/(1.46 + 1.14<span class="italic">L</span>)</span></td><td class="rightEqn">(5)</td></tr></table></p>
          <p class="otherpara">
            <span class="italic">Estimating mass loss from calculated the photosynthetic quotient</span> (<span class="italic">PQ</span>): If there is little metabolic work involved in the conversion of photosynthetically produced <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> to another biochemical category, then the <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> mass required for its synthesis may be simply calculated as 30  PQ/FW (g g<small><sup>1</sup></small>), where FW is the formula weight (with carbon set as 1) and 30 is the formula weight of CH<small><sub>2</sub></small>O. For <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>, it underestimates the demand 9%, in the case of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> by about 15%, reflecting the scales of the metabolic work needed.</p>
          <p class="otherpara">
            <span class="italic">Note</span>: <span class="CH"><a class="simple" title="ChEBI Ontology link for: NAD(P)H" href="javascript:popupOBO('CHEBI:13392','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=13392')">NAD(P)H</a></span> is used to refer to both <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:NADH" target="_blank" href="http://www.chemspider.com/Chemical-Structure.388299.html">NADH</a></span> and NADPH.</p>
        
      </td></tr></table><br/>
      
        
        <h3 id="sect1865"><span class="b_heading">3.2 Respiration</span></h3>
        <span>At first it may seem surprising that plants, that have the ability to create metabolic energy <span class="italic">de novo</span>, should need a supplementary energy generating process. There are a number of reasons, the two main ones being: (i) to sustain their <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span> over periods of darkness, so plants need some energy generating process independent of light; and (ii) the mismatch of time scales between the light-dependent and independent reactions in <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> can give rise to traffic jams along the <span class="AN"><a class="simple" title="Gene Ontology link for: electron transport chain" href="javascript:popupOBO('GO:0022900','b924978h')">electron transport chain</a></span> when the former reaction gets ahead of the latter. Respiration-like electron disposing reactions have evolved to alleviate this congestion. However, the penalty is wastage of the energy from the captured <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>. This is a major limitation to achieving maximum photosynthetic efficiency in other than ideal laboratory situations. <span class="AN"><a class="simple" title="Gene Ontology link for: Photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">Photosynthesis</a></span> can in most part be treated as one of two alternative pathways. By contrast respiration is more complex. In addition to the widespread so-called mitochondrial respiration, <a title="Select to navigate to references" href="#cit51">ref. 51</a> list four other forms of respiration in plants.</span>
        <p class="otherpara">In the case of microalgae, overall respiration can be treated operationally as a composite of two components: (i) a biomass-associated component, which probably represents some basic maintenance energy; and (ii) a photosynthetically-associated component, which may result from the above decongestion process but also from repair or replacement of the components of the light gathering system which suffer photo-damage. A careful analysis of these two forms of respiration for the major algae groupings has been made.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> Normally, other than at very low rates of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>, the dominant loss term is the photosynthetically associated respiration and the fraction of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> lost to concurrent respiration lies in the range 812%. As respiration continues in the dark, these numbers are roughly twice as great over a 24 h diel cycle. By comparison, respiration-associated losses in terrestrial crop plants fall in the range of 3060%.<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> The point at which the respiration and <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> curves intersect is the compensation point and the irradiance at this level is known as the compensation irradiance (<span class="italic">E</span><small><sub>c</sub></small>). Net <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> only occurs at light levels above this value. The compensation irradiance is seen as an offset on the <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>irradiance curve (see <a title="Select to navigate to figure" href="#imgfig13">Fig. 13A</a>); reported values show a wide range from 03 to 12 mE m<small><sup>2</sup></small> s<small><sup>1</sup></small> PAR (equivalent to 0.01 to 0.46 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small> of incoming solar radiation). There are thought to be differences between taxonomic groupings of algae (see <a title="Select to navigate to figure" href="#imgfig13">Fig. 13B</a>) and <a title="Select to navigate to references" href="#cit55">ref. 55</a>. The analysis in <a title="Select to navigate to references" href="#cit53">ref. 53</a> suggests that the major part of the variation can be attributed to individual <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> biomass: <span class="italic">E</span><small><sub>c</sub></small> is approximately proportional to the square root of <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> mass. The compensation irradiance takes on importance in dense algal cultures (see Section 4.4(ii) and Box 2).</p>
        <br/><table class="box_border"><tr><td>
          
            
            <h2 id="sect1906"><span class="a_heading">Box 2: Estimating the light threshold for <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> of dense algal cultures</span></h2>
            <span>In dense, well-mixed algal cultures, <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> will only occur when the daily average irradiance (<span class="italic">E</span>) through the culture is greater than the compensation irradiance (<span class="italic">E</span><small><sub>c</sub></small>) <span class="italic">i.e.</span><table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn6"/><span id="eqn6">(<span class="italic">E</span>)/<span class="italic">z</span>  <span class="italic">E</span><small><sub>c</sub></small></span></td><td class="rightEqn">(1)</td></tr></table>where <span class="italic">z</span> is the depth of the culture. The minimum incoming irradiance level that will allow <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> (<span class="italic">E</span><small><sup>min</sup></small><small><sub>0</sub></small>) may be calculated according to Sverdrup's critical depth theory.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a> Assuming the distribution of light with depth complies with Beers law, then the above integral with depth has the following solution<table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn7"/><span id="eqn7"><small><sup><span class="italic">z</span></sup></small><small><sub>0</sub></small><span class="italic">E</span> = (<span class="italic">E</span><small><sub>0</sub></small>/<span class="italic">k</span>)(1<span class="italic">e</span><small><sup><span class="italic">kz</span></sup></small>)</span></td><td class="rightEqn">(2)</td></tr></table>where <span class="italic">k</span> is the beam attenuation coefficient (extinction coefficient). In dense cultures <span class="italic">e</span><small><sup><span class="italic">kz</span></sup></small>1, thus the equation simplifies to<table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn8"/><span id="eqn8"><small><sup><span class="italic">z</span></sup></small><small><sub>0</sub></small><span class="italic">E</span> = <span class="italic">E</span><small><sub>0</sub></small>/<span class="italic">k</span></span></td><td class="rightEqn">(3)</td></tr></table></span>
            <p class="otherpara">Combining <a title="" href="#eqn6">eqn (1) and (3)</a> gives the minimum irradiance for <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span><table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn9"/><span id="eqn9"><span class="italic">E</span><small><sup>min</sup></small><small><sub>0</sub></small> = <span class="italic">E</span><small><sub>c</sub></small><span class="italic">kz</span></span></td><td class="rightEqn">(4)</td></tr></table></p>
            <p class="otherpara">For cultures with a biomass of 0.25 kg dry weight m<small><sup>3</sup></small>, the light extinction coefficient will be in the region of 50 m<small><sup>1</sup></small>. This assumes <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span>, the main light absorbing pigment, to be 2% of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> dry weight and to have a specific light attenuation of 10 m<small><sup>2</sup></small> g<small><sup>1</sup></small> (derived from data in <a title="Select to navigate to references" href="#cit128">ref. 128</a>). Thus, for a raceway 15 cm deep, using the above numbers, <a title="" href="#eqn9">eqn (4)</a> reduces to:<table><td class="leftEqn"></td><tr><td class="eqn"><span id="eqn10"/><span id="eqn10"><span class="italic">E</span><small><sup>min</sup></small><small><sub>0</sub></small> = 7.5<span class="italic">E</span><small><sub>c</sub></small></span></td><td class="rightEqn">(5)</td></tr></table></p>
            <p class="otherpara">For candidate algae, the 24 h compensation irradiance, based on total incoming radiation, has a wide scatter, generally falling in the region 0.03 to 0.3 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small> (see <a title="Select to navigate to reference" href="#cit52">ref. 52, 53 and 55</a>), with a geometric mean of 0.12 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>. This would give a range for <span class="italic">E</span><small><sup>min</sup></small><small><sub>0</sub></small> from 0.2 to 2 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>, with a median value in the vicinity of 1 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>.</p>
          
        </td></tr></table><br/>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig13"><img alt="A: Schematic of the various components of respiration and their effect on net metabolism. B: Analysis of data obtained from cultures of algae in ref. 53, which illustrates the offset due to the compensation irradiance. Gonyaulax tamarensis, a large dinoflagellate, would not be a candidate organism for mass algal culture but is included as the compensation irradiance is clearly shown with this organism." src="/image/article/2010/EE/b924978h/b924978h-f13.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 13 </b> <span id="fig13"><span class="graphic_title">A: Schematic of the various components of respiration and their effect on net <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span>. B: Analysis of data obtained from cultures of algae in <a title="Select to navigate to references" href="#cit53">ref. 53</a>, which illustrates the offset due to the compensation irradiance. <span class="italic">Gonyaulax tamarensis</span>, a large dinoflagellate, would not be a candidate organism for mass algal culture but is included as the compensation irradiance is clearly shown with this organism.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
    
    
      
        
        <h3 id="sect2018"><span class="b_heading">3.3 Overall efficiency of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span></span></h3>
        <span>The energetics of the photochemical reactions in <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and the linked enzymatic reactions associated with carbon assimilation have been covered above. We now consider the relationship between the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> flux (the irradiance) and the resultant photosynthetic rate. Characteristically the relationship is nonlinear (see <a title="Select to navigate to figure" href="#imgfig14">Fig. 14</a>). In the near-linear initial part of the curve, there is excess enzymatic activity, with light limiting the photosynthetic rate, and maximum quantum yields are obtained. The flattening off of the curve at higher irradiances is a consequence of the limitation of the rate of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> by enzymatic reactions. In this zone, the energy from the excess captured <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> has to be disposed of in some manner, either by fluorescence or by one of the respiratory decongesting mechanisms. It is in the context of this effect that the time scale mismatch of the photochemical and enzymatic parts of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> becomes significant. If the high irradiances are sustained for an hour or so, then the algae will adapt to them; by either increasing their enzymatic capacity but more commonly by reducing the capturing efficiency of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>. The latter is achieved by reducing the size of the light-collecting antenna, this lowers the slope of the initial part of the curve, such that the photosynthetic system saturates at higher irradiances. Thus, the algae can exist as two physiological types  low light adapted (high <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> content) and high light adapted (low <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> content). It would appear that the low light adapted form is the default state. The phenomenon of light adaptation has a major influence on the yields of optically dense cultures as will be discussed in a later section.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig14"><img alt="The photosynthesis versus irradiance (PvE) relationship. The dashed black line is a projection of the initial rate, and the hatched area between this and the blue curved line for photosynthesis is an indication of the photon wastage. The zone between the blue dashed line and the solid blue line for photosynthesis is that of unexpressed enzyme activity." src="/image/article/2010/EE/b924978h/b924978h-f14.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 14 </b> <span id="fig14"><span class="graphic_title">The <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> <span class="italic">versus</span> irradiance (PvE) relationship. The dashed black line is a projection of the initial rate, and the hatched area between this and the blue curved line for <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> is an indication of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> wastage. The zone between the blue dashed line and the solid blue line for <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> is that of unexpressed <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme activity" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme activity</a></span>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">It is important to stress that the following calculation of the efficiency of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> restricts its attention to the ideal circumstance. Even under these circumstances it is not a straightforward calculation and all too often the complications are overlooked or bypassed. A rigorous analysis of the thermodynamics was made in <a title="Select to navigate to references" href="#cit56">ref. 56</a>, and the energetics are concisely outlined in <a title="Select to navigate to references" href="#cit54">ref. 54</a>. The present calculation is made for the production of one mol of organic carbon (of a formula CH<small><sub>2</sub></small>O), assumed to have a heat of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: combustion" href="javascript:popupOBO('CMO:0001473','b924978h')">combustion</a></span> of 469 kJ mo1<small><sup>1</sup></small>; a quantum yield of 8 moles of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> (8 einsteins) captured per mol of oxygen produced (and CO<small><sub>2</sub></small> assimilated) is assumed. A quantum yield of 10 would give rise to a proportionately lower yield. A quantum yield of 8 (4 quanta at 680 nm and 4 at 700 nm) would have a total energy content of 1384 kJ per 8 einsteins. The consequential voltage jumps of 1.7 V (PSII) and 1.6 V (PSI) (equivalent to 1267 kJ in total) imply very efficient energy transfer at this stage. The reactions following these charge separations give rise to the formation of 3 moles of <span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span> and 2 moles of NADPH a combined yield of 590 kJ. These products of the light reaction in turn give rise to the formation of 1 mole of organic carbon with an elemental composition of CH<small><sub>2</sub></small>O. Taking this to be a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>, with a calorific value of 469 kJ mol<small><sup>1</sup></small> C, the overall efficiency of the formation of 1 mole of organic carbon from the 8 absorbed moles of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> themselves would be 34%, with the major part of the loss occurring during the light reactions (see <a title="Select to navigate to figure" href="#imgfig15">Fig. 15A and </a><a title="Select to navigate to figure" href="#imgfig15">15B</a>)</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig15"><img alt="Energy losses during photosynthesis. A. The stepwise loss of energy during the production of 1 mole of organic carbon as glucose (calorific value taken to be 469 kJ mol1 C). B. The fractional energy losses at the major steps during photosynthesis. The hatched lines are stages at which the losses are variable." src="/image/article/2010/EE/b924978h/b924978h-f15.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 15 </b> <span id="fig15"><span class="graphic_title">Energy losses during <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. A. The stepwise loss of energy during the production of 1 mole of organic carbon as <span class="CH"><a class="simple" title="ChEBI Ontology link for: glucose" href="javascript:popupOBO('CHEBI:17234','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=17234')">glucose</a></span> (calorific value taken to be 469 kJ mol<small><sup>1</sup></small> C). B. The fractional energy losses at the major steps during <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. The hatched lines are stages at which the losses are variable.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">There are also inevitable energy losses preceding and following these two sets of reactions. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">Photons</a></span> are gathered from across the whole 400700 nm range  the balance will depend upon the photosynthetic pigments present in the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span>. First, whereas the quantum yield can be remarkably constant across the PAR range, energy losses will be incurred transferring the energy to the lower frequencies of the longer wavelength <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> (see <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>). This loss is about 20%. This reduces the photosynthetic efficiency to 27% (see <a title="Select to navigate to table" href="#tab5">Table 5</a>). Second, the radiation available for <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> is only some 45% of the incoming radiation. Thus, incorporating these losses, 3908 kJ of incoming radiation is required to produce one mol of organic carbon, reducing the overall yield to 12%.</p>
        
<div class="table_caption"><b>Table 5</b> <span id="tab5">Calculation of maximum photosynthetic efficiencies. Yield terminology and symbols adopted from <a title="Select to navigate to references" href="#cit57">ref. 57</a>. CHO refers to organic material with a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>-type elemental ratio, and a calorific value of 469 kJ mol<small><sup>1</sup></small> C (15.6 kJ g<small><sup>1</sup></small>). CHON refers to the calculated carbon-normalised major element composition (C<small><sub>1</sub></small>H<small><sub>1.64</sub></small>O<small><sub>0.44</sub></small>N<small><sub>0.13</sub></small>) for a <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> containing 42% <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, 28% <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span>, 25% <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> and 5% <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acid" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acid</a></span>, with a calorific value of 541 kJ mol<small><sup>1</sup></small> C (24.7 kJ g <small><sup>1</sup></small>, data taken from <a title="Select to navigate to table" href="#tab1">Table 1</a>). This gives photosynthetic quotients (O<small><sub>2</sub></small>/CO<small><sub>2</sub></small>) of 1.1 on <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span> as a N-source and 1.3 on <span class="TC"><a class="simple" title="Chemspider Compound link for:nitrate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.918.html">nitrate</a></span>. Values are calculated on PAR except those within brackets, which are rounded off values based on total solar radiation</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/><col/></colgroup>
              
              
              
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Type of yield</th>
                  <th align="left" valign="bottom" class="border_black">Units</th>
                  <th colspan="2" align="left" valign="bottom" class="border_black">CHO</th>
                  <th colspan="2" align="left" valign="bottom" class="border_black">CHON (NH<small><sub>3</sub></small>)</th>
                  <th colspan="2" align="left" valign="bottom" class="border_black">CHON (NO<small><sub>3</sub></small>)</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Bioenergetic yield (<span class="italic"></span><small><sub>%</sub></small>)</td>
                  <td valign="top">% (kJ kJ<small><sup>1</sup></small>)</td>
                  <td align="char" char="." valign="top">26.7</td>
                  <td align="left" char="." valign="top">[12]</td>
                  <td align="char" char="." valign="top">17.7</td>
                  <td align="left" char="." valign="top">[8]</td>
                  <td align="char" char="." valign="top">14.3</td>
                  <td align="char" char="." valign="top">[6.4]</td>
                </tr>
                <tr align="left">
                  <td valign="top">Energetic yield (<span class="italic"></span><small><sub>kJ</sub></small>)</td>
                  <td valign="top">g kJ<small><sup>1</sup></small></td>
                  <td align="char" char="." valign="top">17  10<small><sup>3</sup></small></td>
                  <td align="left" char="." valign="top">[7.7  10<small><sup>3</sup></small>]</td>
                  <td align="char" char="." valign="top">7.2  10<small><sup>3</sup></small></td>
                  <td align="left" char="." valign="top">[3.2  10<small><sup>3</sup></small>]</td>
                  <td align="char" char="." valign="top">5.8  10<small><sup>3</sup></small></td>
                  <td align="char" char="." valign="top">[2.6  10<small><sup>3</sup></small>]</td>
                </tr>
                <tr align="left">
                  <td valign="top">Quantum yield (<span class="italic"></span><small><sub>E</sub></small>)</td>
                  <td valign="top">g einstein<small><sup>1</sup></small></td>
                  <td align="char" char="." valign="top">3.8</td>
                  <td align="char" char="." valign="top"></td>
                  <td align="char" char="." valign="top">2.1</td>
                  <td align="char" char="." valign="top"></td>
                  <td align="char" char="." valign="top">1.7</td>
                  <td align="char" char="." valign="top">
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">Photons</a></span>/mol C fixed</td>
                  <td valign="top">einstein mol C<small><sup>1</sup></small></td>
                  <td align="char" char="." valign="top">8</td>
                  <td align="char" char="." valign="top"></td>
                  <td align="char" char="." valign="top">10.2</td>
                  <td align="char" char="." valign="top"></td>
                  <td align="char" char="." valign="top">12.6</td>
                  <td align="char" char="." valign="top">
                  </td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">There are also energy losses associated with the metabolic conversion of the primary photosynthetic products (<span class="italic">e.g.</span> hexoses) to biomass <span class="italic">i.e.</span> lipids, <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: polysaccharides" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">polysaccharides</a></span>, <span class="italic">etc</span>. This loss will be dependent upon two factors. The biochemical composition of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> will affect both biomass and energy yields. This is discussed in Box 1 and the effect of varying <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content is illustrated in <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a>. A significant change in yield can result from the state of <span class="CH"><a class="simple" title="Molecular Process Ontology link for: oxidation" href="javascript:popupOBO('MOP:0000568','b924978h')">oxidation</a></span> of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> source  whether the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> are using <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span> (or <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:urea" target="_blank" href="http://www.chemspider.com/Chemical-Structure.1143.html">urea</a></span>) or <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrate" href="javascript:popupOBO('CHEBI:51081','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51081')">nitrate</a></span>. If <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrate" href="javascript:popupOBO('CHEBI:51081','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51081')">nitrate</a></span> is used as a <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> source as opposed to reduced sources such as <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span>, then the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrate" href="javascript:popupOBO('CHEBI:51081','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51081')">nitrate</a></span> calls for extra energy, characteristically increasing the energy demand by 2030%. A summary of photosynthetic yields calculated from the present analysis, along with the various yield terms are used in the study of the energetics of plant culture, is given in <a title="Select to navigate to table" href="#tab5">Table 5</a>.</p>
        <p class="otherpara">The above efficiencies may be regarded essentially as fixed ceilings for photosynthetic efficiency under natural conditions, as the losses are determined by the stoichiometry (metabolic losses) and the thermodynamics of the processes  for which there is limited scope for variation.</p>
        <p class="otherpara">There are two further forms of loss, which are variable, where there may be possibilities for manipulation of the photosynthetic apparatus and/or <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span>: (i) concurrent respiration (<a title="Select to navigate to figure" href="#imgfig13">Fig. 13</a>); and (ii) <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> wastage (<a title="Select to navigate to figure" href="#imgfig14">Fig. 14</a>). Taking a median estimate of the 24 h loss to respiration as 20% of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>, the overall yield of the incoming radiation is reduced to around 10%. A quantum requirement of 10, rather than 8, would reduce the overall maximum yields to 8% after respiration is considered. By comparison the estimated equivalent yield for higher plants<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> (see <a title="Select to navigate to table" href="#tab6">Table 6</a>) lies in the range 4.6 to 6%, the difference being due to the greater respiration losses in higher plants.</p>
        
<div class="table_caption"><b>Table 6</b> <span id="tab6">Observed and projected yields for crop plants and microalgae. Figures for annual insolation are calculated as for <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a>; theoretical values are given for three latitude zones, the same as in <a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a>. * allowing for respiration, ** assuming a calorific value of 20 kJ g<small><sup>1</sup></small> dry weight for the crop as a whole, *** assuming a biomass calorific value of 24.7 kJ g<small><sup>1</sup></small> dry weight (see <a title="Select to navigate to table" href="#tab5">Table 5</a>) and a respiratory loss of 20%</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/></colgroup>
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Crop</th>
                  <th align="left" valign="bottom" class="border_black">Maximum biomass yield/tonnes dry weight ha<small><sup>1</sup></small> a<small><sup>1</sup></small></th>
                  <th align="left" valign="bottom" class="border_black">Source</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td colspan="3" valign="top">
                    <span class="bold">Higher plants</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Theoretical* (C3 plants) <small><sub>%</sub></small> = 4.6%</td>
                  <td valign="top">Low 210, mid 170, high 140**</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit54">54</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Theoretical* (C4 plants) <small><sub>%</sub></small> = 6.0%</td>
                  <td valign="top">Low 270, mid 220, high 190**</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit54">54</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Sugar cane</td>
                  <td char="" valign="top">7495</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit67">6768</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Switchgrass</td>
                  <td char="" valign="top">820</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit19">19,67</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Corn (grain)</td>
                  <td char="" valign="top">834</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit66">66</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Poplar wood chips</td>
                  <td valign="top">11</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit69">69</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Soya</td>
                  <td char="" valign="top">4.65.5</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit66">66</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Rape seed</td>
                  <td char="" valign="top">4.56</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit66">66</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Oil palm</td>
                  <td valign="top">8.7</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit66">66</a>
                  </td>
                </tr>
                <tr align="left">
                  <td colspan="3" valign="top">
                    <span class="bold">Microalgae</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Theoretical <small><sub>%</sub></small> = 12% (<a title="Select to navigate to table" href="#tab5">Table 5</a>)</td>
                  <td valign="top">Low 410, mid 330, high 280***</td>
                  <td valign="top">Present account</td>
                </tr>
                <tr align="left">
                  <td valign="top">Projected raceway, algae unspecified</td>
                  <td char="" valign="top">110220</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit16">16</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Bioreactor raceway, algae unspecified</td>
                  <td valign="top">175</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit4">4</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Projected raceway, algae unspecified</td>
                  <td valign="top">127</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit4">4</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Best case raceway, algae unspecified</td>
                  <td char="" valign="top">120153</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit9">9</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Achieved bioreactor (<span class="italic">Phaeodactylum</span>)</td>
                  <td valign="top">182</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit70">70</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Achieved raceway (<span class="italic">Pleurochrysis</span>) over 10 months</td>
                  <td valign="top">60</td>
                  <td char="." valign="top">
                    <a title="Select to navigate to references" href="#cit71">71</a>
                  </td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">If we take the irradiance information used to create <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a> and the more cautious quantum requirement of 10, then, allowing for respiration, the maximum theoretical yields of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> type molecule would be 410 tonnes dry weight ha<small><sup>1</sup></small> a<small><sup>1</sup></small> (112 g m<small><sup>2</sup></small> d<small><sup>1</sup></small>) at low latitudes less than 30, falling to 280 tonnes dry weight ha<small><sup>1</sup></small> a<small><sup>1</sup></small> (75 g m<small><sup>2</sup></small> d<small><sup>1</sup></small>) at latitudes 4555. Actual yields (see <a title="Select to navigate to table" href="#tab6">Table 6</a>) fall short for a number of reasons; <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> wastage (which can be up to 80%) being the major problem, when attempting to maximise the output from mass cultures of algae. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">Photon</a></span> wastage is the principal reason why the actual yields fall short of the theoretical. This phenomenon continues to tax the ingenuity of researchers and engineers and is far from resolved. We return to this problem in the following section.</p>
      
    
    
      
      <h2 id="sect2355"><span class="a_heading">4. Production rates: background and observations</span></h2>
      <span>Building on the background information on <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>, we consider the theoretical limitations to biomass production, based on the incoming radiation and efficiency of energy transfer from <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> to biomass. We discuss two types of algal biomass production systems, closed bioreactors and open ponds.</span>
      
        
        <h3 id="sect2362"><span class="b_heading">4.1. Algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> and production</span></h3>
        <span>Initial work on mass culture of algae was carried out in ponds and tanks of various forms, frequently with some form of agitation. From these beginnings, two types of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> system have evolved: (i) raceways; (ii) photobioreactors (PBR). The former (see <a title="Select to navigate to figure" href="#imgfig16">Fig. 16</a>) are simple engineering developments of the pond/tank type system. At their simplest, they are oval in shape with depths 100300 mm and the water is kept in circulation with paddle wheels  the most energy efficient system. Bioreactors (see <a title="Select to navigate to figure" href="#imgfig17">Fig. 17</a>) are enclosed systems of various geometries, characteristically, although not invariably, with a light capturing depth of less than 100 mm, often having internal dimensions as little as 30 mm. They may take the form of narrow tubes, horizontal or inclined tubes, vertical coils or columnar structures as well as flat plate structures. Many of the financial disasters in mass culture of algae have been associated with bioreactor-based systems (see <a title="Select to navigate to references" href="#cit58">ref. 58</a>). The main problems with algal culture in bioreactors are the maintenance of the necessary turbulent flow in long lengths of narrow tubing and the possibility of inhibition of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> due to the accumulation of oxygen in the closed system. Active pumping with air-lift pumps induces turbulence and also helps to drive off the accumulating oxygen. On the positive side, bioreactors, being closed, are much less prone to contamination than open raceways. A coupling of the above systems has been used and found to be very reliable and successful. In these coupled systems, large diameter (<span class="italic">ca.</span> 300 mm) bioreactors act as a nursery stage, where a pure culture is maintained, after which the cultures are periodically flooded into large area raceways, which serve as the grow-on phase. Such systems have been successfully used for <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> of the astaxanthin-producing algae <span class="italic">Haematococcus pluvialis</span>.<a title="Select to navigate to reference" href="#cit7"><sup><span class="sup_ref">7,16</span></sup></a></span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig16"><img alt="Open ponds (image courtesy of Nature Beta Technologies Ltd, Eilat, Israel, subsidiary of Nikken Sohonsha Co. Gifu, Japan)." src="/image/article/2010/EE/b924978h/b924978h-f16.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 16 </b> <span id="fig16"><span class="graphic_title">Open ponds (image courtesy of Nature Beta Technologies Ltd, Eilat, Israel, subsidiary of Nikken Sohonsha Co. Gifu, Japan).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig17"><img alt="Photobioreactors (Greenfuels image courtesey of http://www.flickr.com/photos/jurvetson/58591531/)." src="/image/article/2010/EE/b924978h/b924978h-f17.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 17 </b> <span id="fig17"><span class="graphic_title">Photobioreactors (Greenfuels image courtesey of http://www.flickr.com/photos/jurvetson/58591531/).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Microbial production (or yield) may be regarded as a product of biomass (crop) and net <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate <span class="italic">i.e.</span>:<br/><table><tr><td class="eqnText">Production (M L<small><sup>2</sup></small> T<small><sup>1</sup></small> or M L<small><sup>3</sup></small> T<small><sup>1</sup></small>) = biomass (M L<small><sup>2</sup></small> or M L<small><sup>3</sup></small>)  growth rate (T<small><sup>1</sup></small>)</td></tr></table></p>
        <p class="otherpara">Biomass may be expressed as moles or mass of organic carbon or, more commonly in algal mass culture, as ash-free dry weight (per unit area or volume). <span class="AN"><a class="simple" title="Gene Ontology link for: Growth" href="javascript:popupOBO('GO:0040007','b924978h')">Growth</a></span> rate is subject to a number of modifying physical, chemical and biological factors. Maximising the output of these systems, with the number of interacting factors is complex, as without exception, the relationships are non-linear. Further, with probably only a single exception (the apparent Arrhenius constant in the temperature response curve), the values for the parameters cannot be anticipated from theory with any degree of certainty.</p>
      
      
        
        <h3 id="sect2404"><span class="b_heading">4.2. Controls on algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span></span></h3>
        <span><span class="AN"><a class="simple" title="Gene Ontology link for: Growth" href="javascript:popupOBO('GO:0040007','b924978h')">Growth</a></span> of microorganisms is usually modelled as a species-specific maximum <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate (<span class="italic"></span><small><sub>max</sub></small>), reduced by one or a combination of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> limiting factors  the two key ones in the case of the algae are light and inorganic <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span> (<span class="italic">e.g.</span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>, carbon or phosphorus). Characteristically the effect is non-linear, with the initially controlling factor reaching a point where it no longer affects the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate. This is modelled by the biological equivalent of the Langmuir isotherm (MichaelisMenten equation), giving broad nutrient-limited and nutrient-unlimited zones. In the case of light, there is a further zone of light inhibition (see <a title="Select to navigate to figure" href="#imgfig14">Fig. 14</a>). A number of equations have been devised to model this latter curve (see <a title="Select to navigate to reference" href="#cit57">ref. 57 and 59</a>). It is debated whether <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> is controlled by the most severe limiting factor or a product of all factors at one instance in time.</span>
        <p class="otherpara">Two additional factors affect the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate of algae; both are of considerable importance in modelling and designing <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> conditions. The first is respiration (<span class="italic">R</span>), which essentially subtracts from <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> (<span class="italic"></span>), <span class="italic">i.e.</span>:<br/><table><tr><td class="eqnText"><span class="italic"></span><small><sub>net</sub></small> = <span class="italic"></span>  <span class="italic">R</span></td></tr></table></p>
        <p class="otherpara">The second factor is temperature. The overall temperature response has three cardinal points  a temperature minimum, a temperature maximum and a temperature optimum. The zone from the temperature minimum to approaching the temperature optimum closely follows the Arrhenius equation, giving effective <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: activation" href="javascript:popupOBO('CMO:0002196','b924978h')">activation</a></span> energy of <span class="italic">ca.</span> 50<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 kJ mol<small><sup>1</sup></small>. Biologists customarily use a simple logarithmic alternative to the Arrhenius equation  the <span class="italic">Q</span><small><sub>10</sub></small>  defined as the change in rate over 10 C  the value is characteristically close to 2. Temperature is different to <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span> and light, in that it is treated as a determinant of <span class="italic"></span><small><sub>max</sub></small>, rather than a moderator. Relevant to outdoor mass algae culture (and in common with light), as well as the immediate response, algae also have some capability to adapt to the ambient temperature. There are also physiological interactions between the responses to temperature and light, as the photochemical system is essentially temperature insensitive, whereas the enzymatic system is temperature sensitive. <span class="AN"><a class="simple" title="Gene Ontology link for: Photoinhibition" href="javascript:popupOBO('GO:0010205','b924978h')">Photoinhibition</a></span>, and associated losses, is high at low temperatures and high irradiances.<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a></p>
      
      
        
        <h3 id="sect2446"><span class="b_heading">4.3. Yields</span></h3>
        <div>
          
          <span id="sect2449"/><span class="c_heading_indent">(i) Theoretical yields. </span>
          <span>From the information derived in Section 3 (summarised in <a title="Select to navigate to table" href="#tab5">Table 5</a>) and the calculated incoming radiation, it is possible to calculate maximum theoretical yields of biomass per unit area for various latitudes and times of the year. One must stress that, for a number of reasons, this particular calculation is intended to provide nothing more than a high ceiling value.</span>
          <p class="otherpara">The calculation of incoming radiation is as follows: with a clear sky, the incident solar radiation with the sun vertically overhead may be taken as 1000 W m<small><sup>2</sup></small>, of which some 50% (500 J m<small><sup>2</sup></small> s<small><sup>1</sup></small>or 2300 E m<small><sup>2</sup></small> s<small><sup>1</sup></small>) lies within the PAR region. Given this, the irradiance may be calculated for each hour angle of the day and for the cycle of the solar declination angle through the year. Finally a correction is made for reflective losses from the surface of the water using Fresnel's equation  although this correction is very small. Details and the overall calculation may be found in <a title="Select to navigate to reference" href="#cit47">ref. 47 and 61</a>. The calculated daily irradiances for various latitudes are given in <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a>. For comparison purposes, field data of total incoming radiation, derived from satellite observations, are given in <a title="Select to navigate to figure" href="#imgfig19">Fig. 19</a>. The point made by this latter figure is that local atmospheric conditions  notably cloud cover  give rise to a much more complex distribution in space and time of incident radiation than the simple calculation from which <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a> is derived.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig18"><img alt="A. Monthly average daily clear sky total isolation for various latitudes. The calculation takes into account the loss from the surface due to reflection. Calculation made following ref. 47 and 61. B. Estimated monthly averaged theoretical daily photosynthesis (as biomass dry weight, composition as in Table 5, assumed calorific value 24.7 kJ g1) through the year at various latitudes; the calculation (see text) is based on a 10% bioenergetic yield (see Table 5); no biological losses, respiration or organic excretion are taken into account. C. Variation of calculated productivity with latitude, assuming 3 and 10% bioenergetic yields (again no biological losses are taken into account) and also using the tanh relationship shown in Fig. 22. D. Monthly averaged theoretical daily productivity (as dry weight) through the year at various latitudes again using the same tanh relationship as insert C." src="/image/article/2010/EE/b924978h/b924978h-f18.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 18 </b> <span id="fig18"><span class="graphic_title">A. Monthly average daily clear sky total isolation for various latitudes. The calculation takes into account the loss from the surface due to reflection. Calculation made following <a title="Select to navigate to reference" href="#cit47">ref. 47 and 61</a>. B. Estimated monthly averaged theoretical daily <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> (as biomass dry weight, composition as in <a title="Select to navigate to table" href="#tab5">Table 5</a>, assumed calorific value 24.7 kJ g<small><sup>1</sup></small>) through the year at various latitudes; the calculation (see text) is based on a 10% bioenergetic yield (see <a title="Select to navigate to table" href="#tab5">Table 5</a>); no biological losses, respiration or organic <span class="AN"><a class="simple" title="Gene Ontology link for: excretion" href="javascript:popupOBO('GO:0007588','b924978h')">excretion</a></span> are taken into account. C. Variation of calculated productivity with latitude, assuming 3 and 10% bioenergetic yields (again no biological losses are taken into account) and also using the tanh relationship shown in <a title="Select to navigate to figure" href="#imgfig22">Fig. 22</a>. D. Monthly averaged theoretical daily productivity (as dry weight) through the year at various latitudes again using the same tanh relationship as insert C.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig19"><img alt="Global distribution of total incoming radiation through the first half of the year.62" src="/image/article/2010/EE/b924978h/b924978h-f19.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 19 </b> <span id="fig19"><span class="graphic_title">Global distribution of total incoming radiation through the first half of the year.<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Once the irradiances have been obtained, then a simple linear calculation of organic production may be made from the product of the irradiance and the energetic yield (see <a title="Select to navigate to table" href="#tab5">Table 5</a>). The curves given in <a title="Select to navigate to figure" href="#imgfig18">Fig. 18B</a> are for a ceiling value of bioenergetic yield of 10% this is obtained by correcting the 12% yield for a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> (<a title="Select to navigate to table" href="#tab5">Table 5</a>) for a 20% loss due to respiration. To transform the energy yields to mass yields (<span class="italic">i.e.</span> kJ to g dry weight) an energy/mass conversion factor is required. Commonly (see <a title="Select to navigate to references" href="#cit56">ref. 56</a>) this is calculated theoretically from elemental composition or established empirically by bomb <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: calorimetry" href="javascript:popupOBO('CMO:0000682','b924978h')">calorimetry</a></span>. These approaches do not take account of the metabolic work on transformation from hexoses (taken as the primary photosynthetic product) to other biochemicals.<a title="Select to navigate to footnote" href="#fn9"></a> A more correct value may be derived from the mass of <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> required to produce a given mass of biochemical and the calorific value of <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> itself (taken as 15.7 kJ g<small><sup>1</sup></small>, <a title="Select to navigate to reference" href="#cit63">ref. 63, 64</a>). The former can be estimated from either the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>/<span class="CH"><a class="simple" title="ChEBI Ontology link for: polysaccharide" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">polysaccharide</a></span>/<span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (see Box 1, <a title="" href="#eqn9">eqn (4)</a>) or from the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content, assuming a fixed <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>/<span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio of 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 (Box 1, <a title="" href="#eqn10">eqn (5)</a>). The 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 ratio for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> proportions comes from the analysis of data in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>.</p>
          <p class="otherpara">
            <a title="Select to navigate to figure" href="#imgfig18">Fig. 18D</a> shows the results of a calculation using a non-linear relationship between <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and radiation  the tanh relationship between <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and irradiance (see discussion in Section 4.4 (i) and <a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a>). In <a title="Select to navigate to figure" href="#imgfig18">Fig. 18C</a> the data has been integrated through the year giving the distribution of calculated annual yields with latitude, the calculations are made for a 10% bioenergetic yield, a 3% yield (an average obtained value, see <a title="Select to navigate to figure" href="#imgfig20">Fig. 20</a>) and the above tanh relationship. The calculation assumes that only light limits <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>  no consideration is given to control by other factors such as temperature or inorganic <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span>  so in this respect the rates obtained perhaps are best regarded as maximal ones, unless major improvements can be made to the light capturing system.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig20"><img alt="A. Analysis of published production rates of outdoor cultures, grouped according to type of growth system. Colours designate the type of system, crosses maximum rates, solid circles sustained rates, PBR: photobioreactors, RW: raceways. B. Average (geometric means) production rates of ponds, bioreactors and raceways. Upper line is the standard deviation, lower the standard error. C. Reported or calculated bioenergetic yield (%, see Section 3.3 and Table 5), based on photosynthetically available radiation (PAR; n = 15) and total solar radiation (TSR; n = 4). TSR is the TSR efficiencies, plus the PAR values corrected to TSR values by dividing by 2 (see Section 3.1(i)). The theoretical maximum is taken from Table 5 allowing for 20% respiration. Data and sources given in Appendix IV (provided in the ESI)." src="/image/article/2010/EE/b924978h/b924978h-f20.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 20 </b> <span id="fig20"><span class="graphic_title">A. Analysis of published production rates of outdoor cultures, grouped according to type of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> system. Colours designate the type of system, crosses maximum rates, solid circles sustained rates, PBR: photobioreactors, RW: raceways. B. Average (geometric means) production rates of ponds, bioreactors and raceways. Upper line is the standard deviation, lower the standard error. C. Reported or calculated bioenergetic yield (<span class="italic"></span><small><sub>%</sub></small>, see Section 3.3 and <a title="Select to navigate to table" href="#tab5">Table 5</a>), based on photosynthetically available radiation (PAR; <span class="italic">n</span> = 15) and total solar radiation (TSR; <span class="italic">n</span> = 4). TSR is the TSR efficiencies, plus the PAR values corrected to TSR values by dividing by 2 (see Section 3.1(i)). The theoretical maximum is taken from <a title="Select to navigate to table" href="#tab5">Table 5</a> allowing for 20% respiration. Data and sources given in Appendix IV (provided in the ESI<a title="Select to navigate to footnote" href="#fn1"></a>).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig21"><img alt="Analysis of three data sets70,72 of the seasonal changes in biomass yield in outdoor photobioreactors. The study in ref. 70 was undertaken in southern Spain at a latitude 36 48 N; the culture organism was Phaeodactylum tricornutum. The study in ref. 72 study was made in central Italy at 43 8 N; the culture organism was Tetraselmis suecica. Using a least squares analysis, the data were fitted to the tanh expression P = Pmax*tanh(a*(E  Emin)/Pmax), where P is the observed rate of photosynthesis, Pmax is the maximum photosynthetic rate, a is the initial slope, E the irradiance.47 The lines were not forced through zero, but an offset on the irradiance axis (Emin) allowed. The derived parameters are given in Table 7." src="/image/article/2010/EE/b924978h/b924978h-f21.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 21 </b> <span id="fig21"><span class="graphic_title">Analysis of three data sets<a title="Select to navigate to reference" href="#cit70"><sup><span class="sup_ref">70,72</span></sup></a> of the seasonal changes in biomass yield in outdoor photobioreactors. The study in <a title="Select to navigate to references" href="#cit70">ref. 70</a> was undertaken in southern Spain at a latitude 36 48 N; the culture organism was <span class="italic">Phaeodactylum tricornutum.</span> The study in <a title="Select to navigate to references" href="#cit72">ref. 72</a> study was made in central Italy at 43 8 N; the culture organism was <span class="italic">Tetraselmis suecica</span>. Using a least squares analysis, the data were fitted to the tanh expression <span class="italic">P</span> = <span class="italic">P</span><small><sub>max</sub></small>*tanh(<span class="italic">a</span>*(<span class="italic">E</span>  <span class="italic">E</span><small><sub>min</sub></small>)/<span class="italic">P</span><small><sub>max</sub></small>), where <span class="italic">P</span> is the observed rate of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>, <span class="italic">P</span><small><sub>max</sub></small> is the maximum photosynthetic rate, <span class="italic">a</span> is the initial slope, <span class="italic">E</span> the irradiance.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> The lines were not forced through zero, but an offset on the irradiance axis (<span class="italic">E</span><small><sub>min</sub></small>) allowed. The derived parameters are given in <a title="Select to navigate to table" href="#tab7">Table 7</a>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
        <div>
          
          <span id="sect2576"/><span class="c_heading_indent">(ii) Observed yields. </span>
          <span>
            <a title="Select to navigate to figure" href="#imgfig20">Fig. 20A</a> summarises an extensive collation of reports of area yields of algal biomass derived from various culture systems. The reported rates have been plotted against time, as there have been improvements over the years, and separated into ponds, bioreactor systems and raceways. They have been further separated into average (or sustained values) and maximum values. Two exceptionally high rates (130 and 174 g m<small><sup>2</sup></small> d<small><sup>1</sup></small>) have been omitted, as they appear close to or beyond accepted theoretical yields. Generally values above 40 g m<small><sup>2</sup></small> d<small><sup>1</sup></small> come from narrow path length (&lt;50 mm) bioreactors. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: clusters" href="javascript:popupOBO('CHEBI:33731','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33731')">clusters</a></span> of low rates (&gt;20 g m<small><sup>2</sup></small> d<small><sup>1</sup></small>) for raceways in the post 2000 period come from observations during the solar winter at latitudes greater than 30. This is not anticipated from the radiation calculations and may be due to temperature limitation arising from evaporative cooling effect on the raceways (see the following section). Furthermore, some of the data come from high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> algae, which will result in a depression of the yield (see Box 1) and <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a>.</span>
          <p class="otherpara">Ignoring the low <span class="CH"><a class="simple" title="ChEBI Ontology link for: cluster" href="javascript:popupOBO('CHEBI:33731','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33731')">cluster</a></span> (see above) there would appear a general upward drift due to improvements with time to a figure in the region of 3540 g m<small><sup>2</sup></small>d<small><sup>1</sup></small>  very much the prediction in <a title="Select to navigate to references" href="#cit65">ref. 65</a>. The means of the maximum and average/sustained rates are given in <a title="Select to navigate to figure" href="#imgfig20">Fig. 20B</a> (a geometric mean has been used to avoid biases due to the wide scatter). Where the appropriate information is available, the bioenergetic yield (<span class="italic"></span><small><sub>%</sub></small>) has been calculated (see <a title="Select to navigate to figure" href="#imgfig20">Fig. 20C</a>). The data set however is very small; some 19 sets of observations in total and predominantly from high performance photobioreactors, where the mean bioenergetic yield (based on PAR) is close to 10%; the mean for raceway observations is much lower at 2%, with these values being around 30% and 10% respectively of the theoretical yield (see <a title="Select to navigate to table" href="#tab5">Table 5</a>). The generally lower values for raceways, compared with bioreactors, may in some part be due to design limitations, but more likely that the bioreactors are far less susceptible to contamination and the populations in raceways may be growing sub-optimally.</p>
          <p class="otherpara">
            <a title="Select to navigate to table" href="#tab6">Table 6</a> sets out yield data for crop plants along with those available for microalgal cultures. In round figures, the higher plant annual yields fall generally in the range of 530 tonnes crop dry weight per hectare (sugar cane being the striking exception), the algal crops 50150 tonnes. The basis for the difference, although striking, is not wholly clear. In part, it may be that the longer growing period in the case of higher plants results in respiration taking a higher toll on the yields. It is also likely to be due to the fact that, in the case of the higher plants, the reported crop (<span class="italic">e.g.</span> grain) may only be part of the above-ground plant yield, and the below-ground yield (roots) is almost inevitably omitted. There are major uncertainties with the estimations for algal biomass yields in that there are very few reports of long time series (<span class="italic">i.e.</span> data for a full year). When better sets of data become available, we may find some downward revision of the numbers. However, this may be offset by improvements in strains and culturing techniques. We simply have to wait and see.</p>
        </div>
      
      
        
        <h3 id="sect2614"><span class="b_heading">4.4 Controls on yields</span></h3>
        <span>Broadly, the yields commonly achieved in outdoor mass culture systems lie between one tenth and one third of the theoretical value of 10% of the energy in the total incoming radiation (<span class="italic">cf.</span><a title="Select to navigate to table" href="#tab5">Table 5</a> and <a title="Select to navigate to figure" href="#imgfig20">Fig. 20C</a>). A number of factors give rise to this. In Section 4.1, light, temperature and oxygen were identified as factors that would affect <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>; further controls may be set by carbon dioxide concentrations, turbulence, <span class="CH"><a class="simple" title="ChEBI Ontology link for: inhibitors" href="javascript:popupOBO('CHEBI:35222','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35222')">inhibitors</a></span> and contamination.</span>
        <div>
          
          <span id="sect2625"/><span class="c_heading_indent">(i) Light. </span>
          <span>Given the above efficiencies, a large fraction (6090%) of the incoming radiation is not used for <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. Some 50% of the total incoming radiation is outside the range useful for <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and so cannot be used. But this is not the full story. Losses may result from light not being collected by the culture or that the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> collected by the algae are in large part wasted and their energy is lost as heat or fluorescence. The former loss can be essentially disregarded with cultures with light paths of 100 mm and biomasses of 1 kg m<small><sup>3</sup></small>, as the fraction of the light emerging from the culture is a very small proportion (&lt;10<small><sup>4</sup></small>) of that entering. The main loss is associated with the wastage of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> after absorption by the algae, which results in the culture as a whole using light very inefficiently: the algae in the first 520 mm of the light path being light-saturated or light inhibited, whereas those in remaining 9095% of the culture are severely light limited. The resolution of this problem has been, and continues to be, the major challenge to maximising the productivity of mass cultures (see Section 4.5(i)).</span>
          <p class="otherpara">There is a paucity of reported data on the seasonality of production rates for PBRs and raceways, which allows the overall relationship between daily irradiance and production rates to be explored. <a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a> and <a title="Select to navigate to figure" href="#imgfig22">22</a> show the sets of data we have been able to locate.<a title="Select to navigate to reference" href="#cit70"><sup><span class="sup_ref">70,72,73</span></sup></a> The relationship between light and areal production rates was analysed using the tanh model for the PvE relationship (<span class="italic">P</span> = <span class="italic">P</span><small><sub>max</sub></small>*tanh(<span class="italic">a</span>*<span class="italic">E</span>/<span class="italic">P</span><small><sub>max</sub></small>), where <span class="italic">P</span><small><sub>max</sub></small> is the maximum photosynthetic rate, <span class="italic">a</span> is the initial slope, <span class="italic">E</span> the irradiance; see <a title="Select to navigate to references" href="#cit47">ref. 47</a>. The fitting to the data was undertaken using a least squares analysis. We found it necessary to allow for an offset on the irradiance axis. The tanh model was chosen as it provides a direct parameterisation of the initial slope (<span class="italic">a</span>), from which the photosynthetic efficiency (<span class="italic"></span><small><sub>%</sub></small>, see <a title="Select to navigate to table" href="#tab5">Table 5</a>) may be calculated. In order to calculate the photosynthetic efficiency, a calorific value has to be prescribed for the organism, this was calculated from the reported <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content for the organism and the calorific values reported in <a title="Select to navigate to table" href="#tab1">Table 1</a>. The data have been separated into PBRs and raceways. At first sight there would appear to be a significant difference between the rates in raceways and PBRs (as seen also in <a title="Select to navigate to figure" href="#imgfig20">Fig. 20B and </a><a title="Select to navigate to figure" href="#imgfig20">C</a>) but in <a title="Select to navigate to references" href="#cit72">ref. 72</a> the same organism was grown in parallel in PBRs and raceways and the rates are essentially identical, so the apparent differences may be more due to the organism or the environmental or operating circumstances.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig22"><img alt="Analysis of two data sets71,72 of the seasonal changes in biomass yield in outdoor raceways. The study in ref. 71 was undertaken in Perth in Western Australia (latitude 31 57 S), the organism cultured: was Pleurochrysis carterae. The irradiance data were obtained from the Australian Bureau of Metrology database. Other details as Fig. 21." src="/image/article/2010/EE/b924978h/b924978h-f22.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 22 </b> <span id="fig22"><span class="graphic_title">Analysis of two data sets<a title="Select to navigate to reference" href="#cit71"><sup><span class="sup_ref">71,72</span></sup></a> of the seasonal changes in biomass yield in outdoor raceways. The study in <a title="Select to navigate to references" href="#cit71">ref. 71</a> was undertaken in Perth in Western Australia (latitude 31 57 S), the organism cultured: was <span class="italic">Pleurochrysis carterae</span>. The irradiance data were obtained from the Australian Bureau of Metrology database. Other details as <a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">The derived parameters from the analyses and the photosynthetic efficiency and its calculation are summarised in <a title="Select to navigate to table" href="#tab7">Table 7</a> and a number of features are worth noting. Most important is low spread of values for the initial slope (<span class="italic">a</span>); the mean value is 2.4  0.75 g MJ<small><sup>1</sup></small> gives a calculated photosynthetic efficiency of 5.9  1.8%; allowing for a 20% loss incurred as respiration, a value for the primary photosynthetic efficiency of 7.4% is obtained, very close to the theoretical value of 8% for the production of biomass (see <a title="Select to navigate to table" href="#tab5">Table 5</a>). The second important outcome is the apparent irradiance offsets (3.3 to 9.7 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>) seen in the data of <a title="Select to navigate to reference" href="#cit72">ref. 72 and 73</a>. This offset is not as evident, or perhaps absent, in the data from <a title="Select to navigate to references" href="#cit70">ref. 70</a>, but that data set lacks observations at the low irradiances needed to establish whether or not there is an intercept, furthermore, the plot for the 30 mm PBR certainly looks anomalous. Thresholds, equivalent to 59 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>, were reported for mass cultures in <a title="Select to navigate to references" href="#cit74">ref. 74</a> (see Fig. 14.4 and Table 14.5 in <a title="Select to navigate to references" href="#cit74">ref. 74</a>). The simple explanation for the threshold would be that it represents the compensation irradiance (see Section 3.2) but an analysis for dense cultures (see Box 2) suggests this would give rise to a maximum value of 2 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>, distinctly below the observed range of 39 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>; so it cannot be the sole explanation. <span class="TC"><a class="simple" title="Chemspider Compound link for:Water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">Water</a></span> temperature also varies with irradiance and will have an effect upon photosynthetic, and respiration rates, thus the observed offset may be a combined effect of temperature and irradiance. A temperature-parameterised model would be needed to resolve this. The data in <a title="Select to navigate to table" href="#tab7">Table 7</a> have been used as the basis of parameterisation for the production model used elsewhere is the review.</p>
          
<div class="table_caption"><b>Table 7</b> <span id="tab7">Parameters derived from the analysis of data shown in <a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a> and <a title="Select to navigate to figure" href="#imgfig22">22</a>. Sources: <a title="Select to navigate to references" href="#cit70">ref. 70</a>, organism cultured: <span class="italic">Phaeodactylum tricornutum</span>, <a title="Select to navigate to references" href="#cit72">ref. 72</a>, organism cultured: <span class="italic">Tetraselmis suecica</span>, <a title="Select to navigate to references" href="#cit71">ref. 71</a>, organism cultured: <span class="italic">Pleurochrysis carterae</span></span>
		</div>

              <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/></colgroup>
                
                
                
                
                
                
                <thead align="left">
                  <tr align="left" valign="bottom">
                    <th align="left" valign="bottom" class="border_black"><span class="AN"><a class="simple" title="Gene Ontology link for: Growth" href="javascript:popupOBO('GO:0040007','b924978h')">Growth</a></span> system</th>
                    <th align="left" valign="bottom" class="border_black">PBR</th>
                    <th align="left" valign="bottom" class="border_black">PBR</th>
                    <th align="left" valign="bottom" class="border_black">PBR</th>
                    <th align="left" valign="bottom" class="border_black">RW</th>
                    <th align="left" valign="bottom" class="border_black">RW</th>
                  </tr>
                </thead>
                <tbody align="left">
                  <tr align="left">
                    <td valign="top">Dimensions/mm diameter or depth</td>
                    <td char="." valign="top">30</td>
                    <td char="." valign="top">60</td>
                    <td char="." valign="top">35</td>
                    <td char="." valign="top">150</td>
                    <td char="." valign="top">150</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Reference</td>
                    <td valign="top">
                      <a title="Select to navigate to references" href="#cit70">70</a>
                    </td>
                    <td valign="top">
                      <a title="Select to navigate to references" href="#cit70">70</a>
                    </td>
                    <td valign="top">
                      <a title="Select to navigate to references" href="#cit72">72</a>
                    </td>
                    <td valign="top">
                      <a title="Select to navigate to references" href="#cit72">72</a>
                    </td>
                    <td valign="top">
                      <a title="Select to navigate to references" href="#cit71">71</a>
                    </td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic">P</span>
                      <small><sub>max</sub></small>/g m<small><sup>2</sup></small> d<small><sup>1</sup></small></td>
                    <td char="." valign="top">1400</td>
                    <td char="." valign="top">86</td>
                    <td char="." valign="top">20</td>
                    <td char="." valign="top">20</td>
                    <td char="." valign="top">41</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">
                      <span class="italic"></span>/g MJ<small><sup>1</sup></small></td>
                    <td char="." valign="top">1.7</td>
                    <td char="." valign="top">3.5</td>
                    <td char="." valign="top">2.9</td>
                    <td char="." valign="top">1.9</td>
                    <td char="." valign="top">2.3</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Offset/MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small></td>
                    <td char="." valign="top">4.2</td>
                    <td char="." valign="top">0.5</td>
                    <td char="." valign="top">4.6</td>
                    <td char="." valign="top">3.3</td>
                    <td char="." valign="top">9.7</td>
                  </tr>
                  <tr align="left">
                    <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">Lipid</a></span> content (% dry weight)</td>
                    <td char="." valign="top">16</td>
                    <td char="." valign="top">16</td>
                    <td char="." valign="top">26.5</td>
                    <td char="." valign="top">26.5</td>
                    <td char="." valign="top">22.5</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Calculated calorific value/kJ g<small><sup>1</sup></small></td>
                    <td char="." valign="top">23.3</td>
                    <td char="." valign="top">23.3</td>
                    <td char="." valign="top">25.0</td>
                    <td char="." valign="top">25.0</td>
                    <td char="." valign="top">24.3</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Observed photosynthetic efficiency (<span class="italic"></span><small><sub>%</sub></small>)</td>
                    <td char="." valign="top">3.9</td>
                    <td char="." valign="top">8.2</td>
                    <td char="." valign="top">7.1</td>
                    <td char="." valign="top">4.6</td>
                    <td char="." valign="top">5.5</td>
                  </tr>
                </tbody>
              </table></div></div>
            <hr class="hrule"/><br/>
        </div>
        <div>
          
          <span id="sect2806"/><span class="c_heading_indent">(ii) Temperature. </span>
          <span>The incoming radiation has a marked diurnal effect upon the temperature of the systems. <a title="Select to navigate to reference" href="#cit71">ref. 71, 75, 76</a> observed 1025 C increases in temperature during the day due to solar heating. In the case of shallow open systems there is a counter effect on temperature due to evaporative cooling. Evaporative losses at middle to low latitudes fall in the region of 2 m per annum. On a daily basis this results in a daily cooling effect equivalent to some 12 MJ m<small><sup>2</sup></small> (<span class="italic">ca.</span> 300 calories cm<small><sup>2</sup></small>). With 150 mm deep raceways this generates a potential cooling of 20 C per day. Given a <span class="italic">Q</span><small><sub>10</sub></small> of 2, this would give rise close to a 4-fold <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in metabolic rate and possibly a shift in the <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>/respiration ratio. The matter is clearly more complex than this, as there will be some moderating feedback between cooling and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: evaporation" href="javascript:popupOBO('CMO:0001574','b924978h')">evaporation</a></span>  but as <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: evaporation" href="javascript:popupOBO('CMO:0001574','b924978h')">evaporation</a></span> rates are proportional to degrees Kelvin, the feedback will be weak.</span>
          <p class="otherpara">Temperature is very likely a major factor determining the productivity of outdoor <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> systems open to the atmosphere and may limit the latitudinal extent to which they can be successfully used. Clearly supplementary heating is impractical, as the energy required would far exceed the energy gain from <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. It does not seem practical to use solar panels as heat collectors to offset the heat loss due to cooling, as the incoming radiation (1025 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>, see <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a>) is, as it should be, not too dissimilar from that associated with the evaporative associated losses (12 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>), so the heat collectors would need to be broadly of the same area as the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> systems and would almost certainly not be economic.</p>
        </div>
        <div>
          
          <span id="sect2831"/><span class="c_heading_indent">(iii) Oxygen. </span>
          <span>Oxygen is a by-product of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and it has been long known that high oxygen concentrations inhibit <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> (the so called Warburg effect). Part of this derives from the competing oxygenase activity of <span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span> (see section 3.1(ii)). Oxygen concentrations in the region of 400% saturation (<span class="italic">ca.</span> 2 mM) essentially switch off <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. Commonly encountered photosynthetic rates in mass cultures (<span class="italic">ca.</span> 700 g dry weight m<small><sup>3</sup></small> d<small><sup>1</sup></small>) result in net oxygen production rates of 30 mol m<small><sup>3</sup></small> d<small><sup>1</sup></small> or a rise in oxygen concentrations of 30 mM over the daylight period; well in excess of the 2 mM inhibitory level. The exchange of gases across the air/<span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> interface results in some 20 cm depth of <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> being re-aerated per hour. Thus with open raceway systems, provided the system is turbulent (flow rates in excess of 0.05 m s<small><sup>1</sup></small> ensure this, see the following section), accumulation of oxygen should not occur. In the case of enclosed bioreactors, oxygen may be calculated to accumulate at rate of about 4 mol m<small><sup>3</sup></small> h<small><sup>1</sup></small> (<a title="Select to navigate to references" href="#cit60">ref. 60</a> gives a figure of 6 mol m<small><sup>3</sup></small> h<small><sup>1</sup></small> for <span class="italic">Spirulina</span>), thus reaching inhibitory concentrations in 2030 min. Bioreactor systems characteristically use airlift pumps to circulate the water, which should strip off the accumulating oxygen, however, these rapid rates of oxygen production impose constraints on the maximum length of run and the minimum flow rates within the bioreactors. There is apparently a combined effect of temperature and oxygen. <a title="Select to navigate to references" href="#cit60">Ref. 60</a> reported that a fall in temperature from 35 to 25 C, accompanied with a rise in oxygen concentrations from 0.69 mM to 1.9 mM, gave rise to a substantial (60%) fall in photosynthetic rates.</span>
        </div>
        <div>
          
          <span id="sect2870"/><span class="c_heading_indent">(iv) Carbon dioxide. </span>
          <span>Many of the details for oxygen are shared with carbon dioxide, although the implications are profoundly different. Consider the example of seawater, where there is a broad chemical predictability. Seawater total inorganic carbon dioxide concentrations are approximately 2 moles m<small><sup>3</sup></small>. At daily production rates of 30 moles m<small><sup>3</sup></small> (see above), the carbon dioxide species in seawater will be exhausted within an hour of so. The rate of CO<small><sub>2</sub></small> replacement is about one tenth that of oxygen, as the driving force of the reaction is associated with the dissolved CO<small><sub>2</sub></small>, which is only a minor component of the seawater <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:carbonate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.18519.html">carbonate</a></span> system. The consequence is that re-carbonation of the water by simple exchange with the atmosphere will not sustain the rates of removal, thus the CO<small><sub>2</sub></small> needs to be replaced continuously by the addition of gaseous or liquid CO<small><sub>2</sub></small>. The supply of CO<small><sub>2</sub></small> could well be a major constraint on the siting and consequential scale of algal biomass production facilities.</span>
        </div>
        <div>
          
          <span id="sect2890"/><span class="c_heading_indent">(v) Turbulence. </span>
          <span>It had been demonstrated early on in the study of the mechanism of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> that the yield of oxygen per <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> was greater in intermittent light.<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a> The basis for this is that a single 400 <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> collector can process some 2000 excitations per second, whereas <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b924978h')">downstream</a></span> enzymatic carbon assimilation can process at best 100200 electrons per second.<a title="Select to navigate to references" href="#cit78"><sup><span class="sup_ref">78</span></sup></a> The dark period, between the flashes, allows the slower enzymatic reactions to catch up with the photochemical reaction. The optimum dark period is temperature-dependent but generally falls in the region of 50 ms. <a title="Select to navigate to references" href="#cit79">Ref. 79</a> extended the early studies in <a title="Select to navigate to references" href="#cit77">ref. 77</a> for growing cultures and showed that, to have a significant effect, the dark period needed to be 10 times greater than the light, consistent with the above generalisation on relative processing rates of the photochemical and enzymatic reactions. The gains in efficiency are lost at flash periods shorter than 10 ms. In dense cultures, where perhaps 90% of the culture is in effective darkness, turbulence cycles the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> through a dark/light cycle. There has been extensive theoretical and experimental work on this effect. <a title="Select to navigate to references" href="#cit80">Ref. 80</a> undertook a very detailed study with dense algal cultures (36 kg m<small><sup>3</sup></small>) and found the flashing light effect to cut out at about 10 Hz. <a title="Select to navigate to references" href="#cit81">Ref. 81</a> concluded that the response in outdoor tubular bioreactors saturated at frequencies greater than 1 Hz. <a title="Select to navigate to references" href="#cit74">Ref. 74</a> concluded that power consumption considerations would limit flows to 0.150.2 ms<small><sup>1</sup></small>. At these flow rates, in raceways 150 to 200 mm deep, the cross transfer frequency due to turbulent diffusion would be of the order of 0.1 Hz,<a title="Select to navigate to footnote" href="#fn10"></a> thus there is potential for improvement of production rates by increasing the turbulence in raceways. <a title="Select to navigate to references" href="#cit82">Ref. 82</a> used simple hydrofoil spoilers to induce turbulence and with a raceway system was able to achieve a <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: mixing" href="javascript:popupOBO('CMO:0001685','b924978h')">mixing</a></span> frequency of 0.5 to 1 Hz, which gave rise to a two-fold or more gain in areal production. In a subsequent paper, using the same arrangement, <a title="Select to navigate to references" href="#cit83">ref. 83</a> report sustained production rates of 40 g m<small><sup>2</sup></small> d<small><sup>1</sup></small>, with short term, maximum rates in the region of 6070 g m<small><sup>2</sup></small> d<small><sup>1</sup></small>. With the exception of a couple of questionably values, these rates are the highest achieved with raceway cultures, and are equivalent to bioenergetic efficiencies (based on total incoming radiation) of 5.5%, with a high value of 9.5%. Whether such systems can be made economic remains an open question.</span>
        </div>
      
      
        
        <h3 id="sect2943"><span class="b_heading">4.5 Maximising yields</span></h3>
        <div>
          
          <span id="sect2946"/><span class="c_heading_indent">(i) Maximising biomass yields. </span>
          <span>A full discussion of maximising the overall output and economy of algae biomass/biofuel production, as implied before, is complex and beyond the scope of the present review. We restrict our focus on maximising the utilization of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>, <span class="italic">i.e.</span> maximising the photosynthetic efficiency. This has long been seen as crucial to the success of algal biomass production.<a title="Select to navigate to reference" href="#cit15"><sup><span class="sup_ref">15,78</span></sup></a> The problem is peculiar to mass algal culture. Near theoretical yields can be achieved in short light path lengths of dilute cultures at low irradiances. Mass algal production involves optically dense cultures (biomasses 1 kg m<small><sup>3</sup></small>, giving light attenuation coefficients of 200 m<small><sup>1</sup></small>) and supra-optimal irradiances. The consequence is that 90% of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> are absorbed in the first 10 mm by algae suffering severe light inhibition and so use <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> very inefficiently  wasting perhaps some 8090%. Thus the remaining 90% or more of the culture is using <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> very efficiently, but exists in virtual darkness. <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Mixing" href="javascript:popupOBO('CMO:0001685','b924978h')">Mixing</a></span> by inducing turbulence alleviates the problem to some extent but would not appear to solve it fully. In essence, the root of the problem is that, in mass cultures, the algae are presented with a very unnatural situation and they are not equipped physiologically to deal with it.</span>
          <p class="otherpara"><span class="AN"><a class="simple" title="Gene Ontology link for: Photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">Photosynthesis</a></span> occurs as three events with badly matching time responses:<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> capture rates have timescales of 10<small><sup>6</sup></small> s or less, whereas enzymatic CO<small><sub>2</sub></small> fixation and <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> have timescales 1 s and the adaptation to changes to elevated light levels even longer timescales 10<small><sup>3</sup></small> s. Thus, the adaptive response is far too slow to resolve the problem of super-optimal irradiances. It is possible to mix the culture on time scales of fractions of a second, but not on the required microsecond time scale, so although <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: mixing" href="javascript:popupOBO('CMO:0001685','b924978h')">mixing</a></span> will alleviate the congestion in the electron chain to some extent with a consequential increase yield, it does not solve the fundamental problem. The solution lies in either speeding up the enzymatic reactions or slowing down the photochemical ones. The slow working, apparently inefficient <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span> has been an obvious target based on the supposition being that one could improve on evolution. However, the analysis made in <a title="Select to navigate to references" href="#cit49">ref. 49</a> would appear to show this assumption to be false. Even if one were able to engineer improvements in the reaction rate of <span class="AN"><a class="simple" title="Gene Ontology link for: RuBisCO" href="javascript:popupOBO('GO:0016984','b924978h')">RuBisCO</a></span>, or modify the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> to produce more, the probability is that another <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span> would then become limiting and the problem would end up processing through the complex cascade of enzymes of the dark reaction. Solving the other end of the problem, by reducing the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> capturing capacity, has long been seen as the more likely solution (<span class="italic">e.g.</span><a title="Select to navigate to reference" href="#cit15">ref. 15, 78</a>). When exposed to sustained high irradiances, algae adapt by reducing the size of the <span class="AN"><a class="simple" title="Gene Ontology link for: light harvesting" href="javascript:popupOBO('GO:0031992','b924978h')">light harvesting</a></span> antennae and so reduce the rate of collection of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>. The problem is that the time-scale of the adaptation is long as compared with the other timescales in <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and relies upon the algae being held for periods of tens of minutes to an hour or so at high irradiances. In mass culture, the algae exist in a rapidly fluctuating light regime making it impossible for them to adapt to the high irradiances. Furthermore, unless the high irradiances are sustained, the algae appear to revert back to the low light/high <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> state.<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> The solution is to manipulate the physiological control mechanism and to switch the organism permanently into the high-light adapted state. This problem has attracted research interest and headway has been made.<a title="Select to navigate to reference" href="#cit85"><sup><span class="sup_ref">8591</span></sup></a> The <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> antennae are assembled on a scaffold of so called light-harvesting <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and it is these <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> that set the size of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> antennae. It has been possible to down regulate these <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and so reduce the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> capturing ability of the algal photosynthetic system (see <span class="italic">e.g.</span><a title="Select to navigate to references" href="#cit86">ref. 86</a>). The photosynthetic unit associated with photosynthetic system II can be reduced some 3-fold from some 300 <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> molecules down to 95, but this is not achieved in the case of photosynthetic system I.<a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> The <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in the size of the light collecting antenna gives rise to 2-fold or more enhancement in <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> at high irradiances,<a title="Select to navigate to reference" href="#cit86"><sup><span class="sup_ref">86,88</span></sup></a> importantly without apparently compromising enzymatic capacity and there appears to be no penalty due to rise in respiration rate.</p>
          <p class="otherpara">The results from a simple spreadsheet model show that if the <span class="CH"><a class="simple" title="ChEBI Ontology link for: chlorophyll" href="javascript:popupOBO('CHEBI:28966','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=28966')">chlorophyll</a></span> content is reduced, a greater number of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> are delivered to the deeper parts of the culture, resulting in a more efficient <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> usage and greater areal productivity. The effect is most pronounced at intermediate irradiances and as a consequence higher latitudes. This simple model suggests that 2-fold gains in <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> can be expected, consistent with the supposition in <a title="Select to navigate to references" href="#cit15">ref. 15</a>. However, it is unlikely that one will be able wholly eliminate the problem.</p>
          <p class="otherpara">Improving productivity of the culture in this way probably negates some of the gains that can be achieved by <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: mixing" href="javascript:popupOBO('CMO:0001685','b924978h')">mixing</a></span>, so <span class="italic">a priori</span> the two approaches to improving the utilization of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> cannot be expected to be complementary. It is not possible without a sophisticated model to determine the combined effect of these two approaches. The cautious conclusion is that given these developments one could reasonably expect to approach 50% of the theoretical efficiency, giving a maximum potential bioenergetic yield in the region of 5%.</p>
        </div>
        <div>
          
          <span id="sect3011"/><span class="c_heading_indent">(ii) Maximising <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> yields. </span>
          <span>At first sight a logical strategy to improve the economy of algal biodiesel production would seem to be to obtain and grow algae of high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content. This was a major research thrust of the US Aquatic Species Program. Broadly, two lines of research were followed, the first was to isolate and screen algae for high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> producers. The second approach aimed to increase the extent of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production by identifying the rate limiting <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span> and modify the cell's genetics to increase its expression and so increase yield. The <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span> <span class="AN"><a class="simple" title="Gene Ontology link for: acetyl-CoA carboxylase" href="javascript:popupOBO('GO:0003989','b924978h')">acetyl-CoA carboxylase</a></span>, which catalyses the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: carboxylation" href="javascript:popupOBO('MOP:0000591','b924978h')">carboxylation</a></span> of acetyl-CoA to <span class="TC"><a class="simple" title="Chemspider Compound link for:malonyl-CoA" target="_blank" href="http://www.chemspider.com/Chemical-Structure.559121.html">malonyl-CoA</a></span>, is the first committed step in <span class="AN"><a class="simple" title="Gene Ontology link for: lipid synthesis" href="javascript:popupOBO('GO:0008610','b924978h')">lipid synthesis</a></span> and was regarded as the limiting reaction. It had been possible to increase the level of expression of the <span class="AN"><a class="simple" title="Gene Ontology link for: enzyme" href="javascript:popupOBO('GO:0003824','b924978h')">enzyme</a></span> two- to three-fold; however, no associated increase in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> levels was found (see <a title="Select to navigate to references" href="#cit3">ref. 3</a>, p. 138). Unfortunately this work was in its infancy when the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: ASP" href="javascript:popupOBO('CMO:0000019','b924978h')">ASP</a></span> was terminated, so the feasibility of raising <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> yields by such means has not been fully explored. An alternative approach to producing <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span>-rich <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> is to drive the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> into <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> deficiency, when <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> accumulates (see Section 2.3, <a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a>).</span>
          <p class="otherpara">There are, however, penalties in producing cultures with high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content. First, as the fraction of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> is increased some other <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> component (<span class="italic">e.g.</span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acid" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acid</a></span> or <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span>) must decrease. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> are major determinants of maximum <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rates, and it is far from clear how these biochemical <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> change as <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content changes (see <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>). The coefficient of relationships shown in <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a> implies that an increase in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content from 15 to 30% would give rise to a 50% or greater <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate. The reduced <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate is accompanied with a proportionate increase in the period the organisms would need to be held in the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> system and a consequential increase in costs. Increasing their <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content, by driving the algae into <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> deficiency, ultimately gives rise to the same problem; it extends culturing time and increases the space and capital requirements of the production plant. At the present it would appear that there is limited potential to increase the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content of the algae and for a number of reasons 35% <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content appears to be a realistic maximum target.</p>
          <p class="otherpara">An apparently unrecognised penalty of growing algae of high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content is the significant overall biomass loss that necessarily occurs when lipids are synthesised from the primary products (hexoses) of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span>. This is analysed in Box 1. There is a 50% loss in mass resulting from removal of oxygen during <span class="AN"><a class="simple" title="Gene Ontology link for: triglyceride synthesis" href="javascript:popupOBO('GO:0019432','b924978h')">triglyceride synthesis</a></span>; a further third is lost as CO<small><sub>2</sub></small>. In combination, and allowing for the different component lipids and their proportions in algae, we estimate there to be around a 2.6-fold <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in biomass during the synthesis of microalgal <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> from a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>  this effect can be seen in the yield calculations for <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> of varying <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> contents in <a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a>. If the economic analysis focuses exclusively on biodiesel and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production, then this may not be a problem. However, if the components of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> that are diminished at the expense of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> increase are an important part of the economic equation, then this loss on conversion could well be a problem, especially if they were to command a price comparable to or greater than biolipid (see Section 6). <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a>, which derives from <a title="" href="#eqn10">eqn (5)</a>, Box 1, shows the effect of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content on the total biomass and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> yields from the primary <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> photosynthetic products.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig23"><img alt="Estimates of algal areal biomass yields (both daily and annual) for a range of latitudes. The calculations are based on calculated clear sky total insolation (see Fig. 18A) coupled to a tanh model of the relationship between photosynthesis and irradiance, described in Fig. 21. The parameters used were derived from the analysis of the data in Fig. 21 and 22, summarised in Table 7. The value for the initial slope (a) was the average of the full data set, the irradiance offset (Emin) was derived from ref. 70, 72; the dataset from70 lacked the observations at the low irradiances necessary to adequately constrain the intercept. Pmax was taken from ref. 71  the maximum rates in ref. 72 are low when compared with the data shown in Fig. 20A and accordingly were not thought to be appropriate. The initial calculation is of primary photosynthetic production of a hexose, so the individual values for a and Pmax have to be scaled up to take account of the fact that the observations were of biomass, which has incurred a loss of mass on converting hexose to proteins, polysaccharides and lipid (see Box 1). The final values used were a = 3.55 (dimensionless), Emin = 5.86 MJ m2 d1, Pmax = 76.9 g m2 d1; the calculation also requires the calorific value for the primary photosynthetic product, which was taken as 15.7 kJ g1 (ref. 63, 64). The yields for various lipid concentrations are derived from primary hexose production and the lipid content using the eqn (5), Box 1. This assumes a protein to carbohydrate ratio of 3:2, the mean value found in Fig. 5. The field observation of yields (vertical bars) placed at latitude 20 come from ref. 16; the values at placed at latitude 35 come from ref. 71, 72." src="/image/article/2010/EE/b924978h/b924978h-f23.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 23 </b> <span id="fig23"><span class="graphic_title">Estimates of algal areal biomass yields (both daily and annual) for a range of latitudes. The calculations are based on calculated clear sky total insolation (see <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a>) coupled to a tanh model of the relationship between <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and irradiance, described in <a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a>. The parameters used were derived from the analysis of the data in <a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a> and <a title="Select to navigate to figure" href="#imgfig22">22</a>, summarised in <a title="Select to navigate to table" href="#tab7">Table 7</a>. The value for the initial slope (<span class="italic">a</span>) was the average of the full data set, the irradiance offset (<span class="italic">E</span><small><sub>min</sub></small>) was derived from <a title="Select to navigate to reference" href="#cit70">ref. 70, 72</a>; the dataset from<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> lacked the observations at the low irradiances necessary to adequately constrain the intercept. <span class="italic">P</span><small><sub>max</sub></small> was taken from <a title="Select to navigate to references" href="#cit71">ref. 71</a>  the maximum rates in <a title="Select to navigate to references" href="#cit72">ref. 72</a> are low when compared with the data shown in <a title="Select to navigate to figure" href="#imgfig20">Fig. 20A</a> and accordingly were not thought to be appropriate. The initial calculation is of primary photosynthetic production of a <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>, so the individual values for <span class="italic">a</span> and <span class="italic">P</span><small><sub>max</sub></small> have to be scaled up to take account of the fact that the observations were of biomass, which has incurred a loss of mass on converting <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> to <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: polysaccharides" href="javascript:popupOBO('CHEBI:18154','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18154')">polysaccharides</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> (see Box 1). The final values used were <span class="italic">a</span> = 3.55 (dimensionless), <span class="italic">E</span><small><sub>min</sub></small> = 5.86 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>, <span class="italic">P</span><small><sub>max</sub></small> = 76.9 g m<small><sup>2</sup></small> d<small><sup>1</sup></small>; the calculation also requires the calorific value for the primary photosynthetic product, which was taken as 15.7 kJ g<small><sup>1</sup></small> (<a title="Select to navigate to reference" href="#cit63">ref. 63, 64</a>). The yields for various <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> concentrations are derived from primary <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> production and the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content using the <a title="" href="#eqn10">eqn (5)</a>, Box 1. This assumes a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> to <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio of 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2, the mean value found in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>. The field observation of yields (vertical bars) placed at latitude 20 come from <a title="Select to navigate to references" href="#cit16">ref. 16</a>; the values at placed at latitude 35 come from <a title="Select to navigate to reference" href="#cit71">ref. 71, 72</a>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
        <div>
          
          <span id="sect3138"/><span class="c_heading_indent">(iii) Predicting maximum yields. </span>
          <span>We have attempted to incorporate the findings of this section into a model of algal biomass production (see <a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a>). In essence we have used the calculated clear sky irradiance values (Section 4.3(i) and <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a>) and parameters derived in major part from an analysis of the data reported in <a title="Select to navigate to references" href="#cit71">ref. 71</a> to produce biomass production rates. We chose the Moheimani data set as it comes from raceways, which it is consistent with our economic analysis (see Section 6).</span>
          <p class="otherpara">The initial calculation we make is the rate production of <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span>. In order to do this we need to back calculate the rates of <span class="AN"><a class="simple" title="Gene Ontology link for: hexose formation" href="javascript:popupOBO('GO:0019319','b924978h')">hexose formation</a></span> from the biomass-derived rates. This is done by reversing the calculation of the mass loss during the biomass. <a title="Select to navigate to references" href="#cit71">Ref. 71</a> observed a mean <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content of 36.5% of dry weight. This (from <a title="" href="#eqn10">eqn (5)</a>, Box 1) gives a figure of 1.88 for the biomass to conversion <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> and scales up the <span class="italic">P</span><small><sub>max</sub></small> value from 42 g m<small><sup>2</sup></small> d<small><sup>1</sup></small> (<a title="Select to navigate to table" href="#tab7">Table 7</a>) to 77 g m<small><sup>2</sup></small> d<small><sup>1</sup></small> and the initial slope (<span class="italic">a</span>) from 2.43 to 3.55 g MJ<small><sup>1</sup></small>. The data sets of <a title="Select to navigate to reference" href="#cit71">ref. 71, 72</a> show a mean offset on the irradiance axis of 5.86 MJ m<small><sup>2</sup></small> d<small><sup>1</sup></small>, this mean has been used for the calculation. The calculated photosynthetic <span class="CH"><a class="simple" title="ChEBI Ontology link for: hexose" href="javascript:popupOBO('CHEBI:18133','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18133')">hexose</a></span> production rates are then converted to production for biomasses of varying <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> contents (15, 35 and 50% of <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> dry weight) using <a title="" href="#eqn10">eqn (5)</a> in Box 1, assuming a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio of 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2.</p>
          <p class="otherpara">Given the above parameterisations, we can calculate the yields per unit area, and their variation with latitude from the insolation data derived in <a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a> and the results are shown in <a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a>. Shown for comparison purposes are results from three reports of seasonal productivity abstracted from published sources. It can be seen in <a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a> that there is broad agreement between the observed integrated rates and the predicted ones for algae with realistic biochemical compositions. The calculation is made from clear sky radiation data, which will overestimate the incoming radiation (<span class="italic">cf.</span><a title="Select to navigate to figure" href="#imgfig18">Fig. 18A</a> and <a title="Select to navigate to figure" href="#imgfig19">19</a>) so overestimate <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rates. There is little to be gained at this stage by making a detailed calculation based on observed insolation rates. As a counter to this, the calculation makes no attempt to anticipate improvements of the light capturing apparatus of the algae which we may expect to occur (see Section 4.5(i)).</p>
          <p class="otherpara">There is some steepening of the poleward gradient between the latitudes 3555. This may limit year-round outdoor culture systems to latitudes less than 35 (see also <a title="Select to navigate to figure" href="#imgfig19">Fig. 19</a>). This appears to be the case with commercial production facilities. The longstanding, and successful, Earthrise Farms in southern California (latitude 33 08 N) only operates its raceways for 9 months of the year, whereas the systems at Kanahoe Point on the Big Island of Hawaii (latitude 19 47 N) are operated successfully on a year-round basis, with no evident seasonal <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> of productivity.<a title="Select to navigate to reference" href="#cit16"><sup><span class="sup_ref">16,92</span></sup></a> The experimental facility at Roswell (33 N), which was part of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: ASP" href="javascript:popupOBO('CMO:0000019','b924978h')">ASP</a></span> programme, ran into problems of snow and ice during the winter period. In the model, the trend with latitude derives mainly from the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in annually integrated insolation; the effect will very likely be much greater in reality as one may expect additional strong temperature effects on <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>. This affirms the need for temperature-parameterised models of <span class="AN"><a class="simple" title="Gene Ontology link for: photosynthesis" href="javascript:popupOBO('GO:0015979','b924978h')">photosynthesis</a></span> and respiration.</p>
          <p class="otherpara">
            <a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a> shows the very marked negative effect of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content on the yields, and this questions the search for high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> species for, unless they possess uniquely high intrinsic photosynthetic rates, they will give low biomass yields. There may be an additional effect due to the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: reduction" href="javascript:popupOBO('MOP:0000569','b924978h')">reduction</a></span> in <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate at high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content, resulting in the algae failing to achieve the rates that the irradiance would allow. Thus, there would appear to be diminishing returns with increased <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (see also <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a>); this is seen when the economics are considered (see <a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a>).</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig24"><img alt="Results of a spreadsheet model of costings and economics for the production of biodiesel and associated product streams. The calculation assumes biodiesel to have a commodity value of $900 tonne1 (ca. $125 bbl1). The details of the model are given in Table 9 and the text. A. Net revenue. Lipid levels: blue 15%, green 35%, red 50%. B. The value the separate products, biodiesel at $900 tonne1, animal feed at either $500 tonne1 (as low grade protein  dark blue bars) or at $900 tonne1 (as high grade protein  pale blue bars). The net revenue calculation for anaerobic digestion involves the revenue gain from electricity production and the recycling of nitrogen (where it occurs) and phosphate minus the operating and capital costs of the anaerobic digestion plant. C. The cost of biomass ($ tonne1) and biodiesel ($ bbl1) assuming them to be the sole marketed product." src="/image/article/2010/EE/b924978h/b924978h-f24.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 24 </b> <span id="fig24"><span class="graphic_title">Results of a spreadsheet model of costings and economics for the production of biodiesel and associated product streams. The calculation assumes biodiesel to have a commodity value of $900 tonne<small><sup>1</sup></small> (<span class="italic">ca.</span> $125 bbl<small><sup>1</sup></small>). The details of the model are given in <a title="Select to navigate to table" href="#tab9">Table 9</a> and the text. A. Net revenue. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">Lipid</a></span> levels: blue 15%, green 35%, red 50%. B. The value the separate products, biodiesel at $900 tonne<small><sup>1</sup></small>, animal feed at either $500 tonne<small><sup>1</sup></small> (as low grade <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>  dark blue bars) or at $900 tonne<small><sup>1</sup></small> (as high grade <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>  pale blue bars). The net revenue calculation for anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> involves the revenue gain from electricity production and the recycling of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> (where it occurs) and <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> minus the operating and capital costs of the anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> plant. C. The cost of biomass ($ tonne<small><sup>1</sup></small>) and biodiesel ($ bbl<small><sup>1</sup></small>) assuming them to be the sole marketed product.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        </div>
      
    
    
      
      <h2 id="sect3227"><span class="a_heading">5. Harvesting and processing of microalgal biomass</span></h2>
      <span>Before considering the economics of the process (Section 6), we want to highlight the challenges associated with the harvest and processing of microalgal biomass. The methods that are most used for harvesting of algal biomass are discussed in this section, with an emphasis on the use of <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>. With regards to biomass processing, we briefly discuss the principles behind the methods relevant to the economical process we discuss in section 6.</span>
      <p class="otherpara">Out of all steps considered in an algal biomass production process, harvesting and processing of the biomass contains the largest uncertainty surrounding cost and effectiveness. The separation step in producing algal biomass as feedstock for biodiesel is difficult and inherently more expensive by comparison with oilseed crops. For algal biomass, there are three main phases in the preparation process prior to biodiesel production: (i) harvesting of the algae; (ii) disrupting the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span>; and (ii) <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> of the lipids from the biomass. The efficiency and methodology used in these three steps can have a major impact on the economics of commercial algal biofuel production. These processes have been largely avoided in past reviews, despite being identified as a high priority for the future R&amp;D.<a title="Select to navigate to reference" href="#cit93"><sup><span class="sup_ref">9395</span></sup></a></p>
      
        
        <h3 id="sect3243"><span class="b_heading">5.1 Biomass harvest</span></h3>
        <span><span class="AN"><a class="simple" title="Cell Ontology link for: Cell" href="javascript:popupOBO('CL:0000000','b924978h')">Cell</a></span> harvest we take as the initial 50200-fold concentration step of the algal cultures yielding a biomass product that can be used for further processing. The choice of harvesting methodology, likely to be a combination of individual methods, depends highly on the biomass type and requirements of the <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b924978h')">downstream</a></span> processing (Sections 5.2 through 5.4).</span>
        <p class="otherpara">Settling rate (<span class="italic">v</span><small><sub>s</sub></small>) of microalgae is a functon of two forces (drag, <span class="italic">F</span><small><sub>d</sub></small> and gravity, <span class="italic">F</span><small><sub>g</sub></small>) and the diameter (<span class="italic">r</span><small><sub>c</sub></small>) and shape of the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span>. The relationship at low Reynolds numbers for a sphere is described by Stokes' law:<br/><table><tr><td class="eqnText"><span class="italic">v</span><small><sub>s</sub></small> = <span class="italic">F</span><small><sub>g</sub></small>/<span class="italic">F</span><small><sub>d</sub></small> = 2/9<span class="italic">g</span><span class="italic">r</span><small><sub>c</sub></small><small><sup>2</sup></small> (<span class="italic"></span><small><sub>cell</sub></small>  <span class="italic"></span><small><sub>fluid</sub></small>)/<span class="italic"></span></td></tr></table></p>
        <p class="otherpara">The settling rate of <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> in suspension follows a square relationship with the radius of the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> and a linear relationship with the difference in density (<span class="italic"></span>) between the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> and the fluid and the kinematic viscosity of the fluid (<span class="italic"></span>).</p>
        <p class="otherpara"><span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">Centrifugation</a></span> is the most common method for harvesting algae from large volume cultures and is widely applicable to a variety of algal species.<a title="Select to navigate to reference" href="#cit15"><sup><span class="sup_ref">15,96,97</span></sup></a> Although this process is effective, it is also expensive and energy intensive. The operational variables, such as centrifugal force, flow rate, biomass settling rate and settling distance will determine the efficiency of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>.</p>
        <p class="otherpara">There are three different types of in-line industrial centrifuges: multi-chamber, nozzle and solid bowl. All three types are continuous in that there is a continuous flow of culture through the centrifuges. A multi-chamber centrifuge consists of a number of tubular bowls arranged coaxially, causing each chamber to collect particles of a specific size due to the distinct centrifugal forces exerted in each chamber. A solids content of 20% can be achieved, however the solids need to be removed from the bowl manually. An alternative design is the solid bowl centrifuge in which the bowl contains a disc stack, causing the biomass to collect at the outer part of the bowl. However this set-up also requires manual intervention in removing the biomass from the bowl. A third type of centrifuge is the nozzle-type centrifuge, similarly containing internally stacked discs, and in which the shape of the bowl is modified so that the conical section of the bowl has evenly spaced orifices (nozzles). The biomass is collected in a conical space on the side of the bowl where it is automatically discharged. This last type is a true continuous <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span> system with minimal manual intervention<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> and could be considered the most appropriate type to harvest the large algae cultures. These three types of centrifuges vary in the volume processed per hour, with a maximum of 50 to 100 m<small><sup>3</sup></small> h<small><sup>1</sup></small> (<a title="Select to navigate to table" href="#tab8">Table 8</a>; <a title="Select to navigate to references" href="#cit15">ref. 15</a> and centrifuge industry representatives, personal communication).</p>
        
<div class="table_caption"><b>Table 8</b> <span id="tab8">Summary of comparisons between relative harvesting throughput, applications, capital and operating costs. Settling refers to autoflocculation, changes in pH in a stagnant culture will induce settling of algal biomass, flotation refers to the separation process based on the adhesion of air bubbles to flocs of suspended algae after which these float to the surface and separation can occur. <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">Centrifugation</a></span> is subdivided into multi-chamber, nozzle and solid bowl <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>. The relative harvesting costs include reference to operating, maintenance and capital costs and are based on figures from <a title="Select to navigate to reference" href="#cit97">ref. 97, 15 and 94</a>. The capital costs presented are adjusted for inflation and converted into US dollars (USD, $)</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/></colgroup>
              
              
              
              
              
              
              
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th colspan="4" align="left" valign="bottom" class="border_black"></th>
                  <th colspan="3" align="left" valign="bottom" class="border_black">Capital cost/1000 $ m<small><sup>3</sup></small><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><i><small><sup style="color: red;"><a title="" href="#tab8fnb">b</a></sup></small></i></th>
                  <th colspan="3" align="left" valign="bottom" class="border_black">Relative costs/m<small><sup>3</sup></small></th>
                  <th colspan="2" align="left" valign="bottom" class="border_black">Cost per kg/$<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><i><small><sup style="color: red;"><a title="" href="#tab8fnc">c</a></sup></small></i></th>
                </tr>
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black"></th>
                  <th align="left" valign="bottom" class="border_black">Continuous process?</th>
                  <th align="left" valign="bottom" class="border_black">Volume processed/m<small><sup>3</sup></small> h<small><sup>1</sup></small></th>
                  <th align="left" valign="bottom" class="border_black">Final product concentration (% solids)<i><small><sup style="color: red;"><a title="" href="#tab8fna">a</a></sup></small></i></th>
                  <th align="left" valign="bottom" class="border_black">Mohn (1988)<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a></th>
                  <th align="left" valign="bottom" class="border_black">Benemann (1996)<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a></th>
                  <th align="left" valign="bottom" class="border_black">Molina Grima (2004)<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a></th>
                  <th align="left" valign="bottom" class="border_black">Mohn (1988)<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a></th>
                  <th align="left" valign="bottom" class="border_black">Benemann (1996)<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a></th>
                  <th align="left" valign="bottom" class="border_black">Molina Grima (2004)<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a></th>
                  <th align="left" valign="bottom" class="border_black">Becker (1994)<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a></th>
                  <th align="left" valign="bottom" class="border_black">Moheimani (2004)<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><i><small><sup style="color: red;"><a title="" href="#tab8fnd">d</a></sup></small></i><i><small><sup style="color: red;"><a title="" href="#tab8fne">e</a></sup></small></i></th>
                </tr>
              </thead>
              <tfoot align="left">
                <tr align="left">
                  <th>
                    <a id="tab8fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Final product concentration refers to the algal biomass dry weight in the harvested product (<span class="italic">e.g.</span> 10% = 10 g dry biomass from 100 g of centrifuged paste).</span>
                    <a id="tab8fnb"><span class="tfootnote">b</span></a> <span class="sup_inf">Capital costs presented, expressed on a 1 m<small><sup>3</sup></small> h<small><sup>1</sup></small> basis, are at best approximate since these were obtained from earlier reports and simply adjusted for inflation, not taking improved engineering into account.</span>
                    <a id="tab8fnc"><span class="tfootnote">c</span></a> <span class="sup_inf">Values given in brackets indicate the harvesting costs, for the sake of comparison the calculation uses a 1 kg m<small><sup>3</sup></small> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> concentration upon harvesting, assuming a 0.4 and 0.079 g dw L<small><sup>1</sup></small> <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> concentration for <a title="Select to navigate to reference" href="#cit37">ref. 37 and 103</a> respectively.</span>
                    <a id="tab8fnd"><span class="tfootnote">d</span></a> <span class="sup_inf">Values presented are averaged costs of biomass from cultures grown at unregulated pH, pH 9.6 or pH 9.</span>
                    <a id="tab8fne"><span class="tfootnote">e</span></a> <span class="sup_inf">The cost calculations have been adjusted for inflation and converted into today's US dollars (conversion factor 1 AUS $ = 0.81 USD).</span>
                    <a id="tab8fnf"><span class="tfootnote">f</span></a> <span class="sup_inf">The cost of harvesting using flotation was reconsidered in <a title="Select to navigate to references" href="#cit15">ref. 15</a>, where according to equipment and information available in 1996 the cost of harvesting 1 m<small><sup>3</sup></small> h<small><sup>1</sup></small> by flotation was 20% of the cost of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>. These figures did not include the cost of flocculant. Assuming a relative cost of 0.31 for <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span>, we could derive a relative harvesting figure of 0.51.</span>
                    <a id="tab8fng"><span class="tfootnote">g</span></a> <span class="sup_inf">Current <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span> engineering allows processing of up to 100 m<small><sup>3</sup></small> of culture per hour; na  not applicable.</span>
                  </th>
                </tr>
              </tfoot>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Settling/autoflocculation</td>
                  <td valign="top">N</td>
                  <td valign="top">na</td>
                  <td valign="top">13</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td char="" valign="top">0.051</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="AN"><a class="simple" title="Gene Ontology link for: Flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">Flocculation</a></span></td>
                  <td valign="top">N</td>
                  <td valign="top">na</td>
                  <td valign="top">7</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">9</td>
                  <td valign="top">0.31</td>
                  <td char="" valign="top">0.60.8</td>
                  <td valign="top"></td>
                  <td char="[" valign="top">0.61 [0.24]</td>
                  <td char="[" valign="top">1.1 [0.08]</td>
                </tr>
                <tr align="left">
                  <td valign="top">Flotation (dissolved air flotation)</td>
                  <td valign="top">Y</td>
                  <td char="" valign="top">1591900</td>
                  <td valign="top">8</td>
                  <td valign="top">33</td>
                  <td valign="top">19105</td>
                  <td valign="top"></td>
                  <td valign="top">0.91<i><small><sup style="color: red;"><a title="" href="#tab8fnf">f</a></sup></small></i></td>
                  <td char="" valign="top">0.40.6</td>
                  <td valign="top"></td>
                  <td char="[" valign="top">1.39 [0.56]</td>
                  <td char="[" valign="top">5.2 [0.41]</td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">Centrifugation</a></span></td>
                  <td valign="top">Y</td>
                  <td char="" valign="top">1520<i><small><sup style="color: red;"><a title="" href="#tab8fng">g</a></sup></small></i></td>
                  <td valign="top">1227</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">3-chamber centrifuge</td>
                  <td valign="top">N</td>
                  <td char="" valign="top">2.55</td>
                  <td valign="top">20</td>
                  <td valign="top">15</td>
                  <td valign="top">55</td>
                  <td valign="top"></td>
                  <td valign="top">1</td>
                  <td valign="top">1</td>
                  <td valign="top"></td>
                  <td char="[" valign="top">1.71 [0.68]</td>
                  <td char="[" valign="top">4.6 [0.36]</td>
                </tr>
                <tr align="left">
                  <td valign="top">nozzle centrifuge</td>
                  <td valign="top">Y</td>
                  <td valign="top">20</td>
                  <td valign="top">&lt;15</td>
                  <td valign="top">12</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">0.82</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td char="[" valign="top">3.8 [0.29]</td>
                </tr>
                <tr align="left">
                  <td valign="top">solid bowl or decanter</td>
                  <td valign="top">N</td>
                  <td valign="top">0.53</td>
                  <td valign="top">&gt;20</td>
                  <td valign="top">4</td>
                  <td valign="top"></td>
                  <td valign="top">11</td>
                  <td valign="top">0.26</td>
                  <td valign="top"></td>
                  <td valign="top">1</td>
                  <td valign="top"></td>
                  <td char="[" valign="top">4.1 [0.32]</td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">An alternative to settling is flotation of biomass, where dispersed air bubbles in the culture adhere to flocs formed during <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> and cause the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> aggregates to float. This process can operate more efficiently and rapidly than sedimentation or settling and can also achieve a higher solids fraction.<a title="Select to navigate to reference" href="#cit15"><sup><span class="sup_ref">15,97</span></sup></a> This process is potentially scalable to large-scale algal ponds.<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a> However, it is probable that an extra concentration step (possibly <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>) may be necessary because of the relatively low final product concentration.</p>
        <p class="otherpara"><span class="AN"><a class="simple" title="Gene Ontology link for: Flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">Flocculation</a></span>, in combination with <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span> or dissolved air flotation, is also widely method used for harvesting microalgal <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span>. <span class="AN"><a class="simple" title="Gene Ontology link for: Flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">Flocculation</a></span> refers to the aggregation of particles in suspension to form clumps that can overcome the Stokes' drag force and increase the settling rate. This reduces or avoids the need of energy intensive separation mechanisms like <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>. It has been suggested that this process could be adapted to be the lowest cost harvesting process.<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a> However, <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> rate and efficiency are highly dependent on cellular characteristics, such as <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> volume and <span class="AN"><a class="simple" title="Gene Ontology link for: cell wall" href="javascript:popupOBO('GO:0005618','b924978h')">cell wall</a></span> properties, or culture characteristics, such as <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> number, culture age and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> phase.<a title="Select to navigate to reference" href="#cit99"><sup><span class="sup_ref">99,100</span></sup></a></p>
        <p class="otherpara">Two mechanisms that give rise to <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> are: (i) charge neutralisation, <span class="italic">i.e.</span> the attraction between the negatively charged algal <span class="AN"><a class="simple" title="Gene Ontology link for: cell surface" href="javascript:popupOBO('GO:0009986','b924978h')">cell surface</a></span> and the positively charged flocculentl; and (ii) bridging, when larger flocculent molecules span a distance large enough to adsorb to multiple particles and hence bridge the particles.<a title="Select to navigate to reference" href="#cit101"><sup><span class="sup_ref">101,102</span></sup></a> Several different types of flocculants can be used: polyvalent metal salts, synthetic cationic <span class="CH"><a class="simple" title="ChEBI Ontology link for: polymers" href="javascript:popupOBO('CHEBI:60027','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=60027')">polymers</a></span>, high pH and bioflocculants. The low cost of metal salt flocculants make these the more popular flocculants. However, the potential toxicity of the wastewater fraction and the undesirable presence of metals in the biomass to be processed are concerns that arise and may inhibit their widespread applicability in the microalgal biomass production process. This would be particularly the case if the spent medium were to be recycled.</p>
        <p class="otherpara">The main advantage of <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> as a harvesting method is the relatively cheap nature of the process and high volume processing. However, disadvantages include the species and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> stage-dependent nature of <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> efficiency. Until now, no clear correlation has been demonstrated between <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> efficiency, dose and algal taxonomic <span class="CH"><a class="simple" title="ChEBI Ontology link for: group" href="javascript:popupOBO('CHEBI:24433','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">group</a></span>. Another disadvantage is the low solids content (<span class="italic">i.e.</span> &lt; 10%) of the biomass harvested by <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span>, which requires further concentration using other methods. As a result, <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> is often combined in a process with <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span> to increase the solids content of the harvested biomass.</p>
        <p class="otherpara">Another method for harvesting the biomass is <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: filtration" href="javascript:popupOBO('CMO:0001640','b924978h')">filtration</a></span>. The vacuum drum filters and chamber filter press units are most commonly employed for harvesting fairly large microalgae. However, because <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: filtration" href="javascript:popupOBO('CMO:0001640','b924978h')">filtration</a></span> efficiency is highly dependent on the size range of the species to be harvested, certain types of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: filtration" href="javascript:popupOBO('CMO:0001640','b924978h')">filtration</a></span> are ineffective for species in the few micrometre size range, such as <span class="italic">Scenedesmus</span>, <span class="italic">Dunaliella</span> and <span class="italic">Chlorella</span> due to the rapid clogging of the filters.<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> This species-dependent nature of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: filtration" href="javascript:popupOBO('CMO:0001640','b924978h')">filtration</a></span> process makes this less attractive compared to <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span> and <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> and will not be considered further.</p>
        <p class="otherpara">An exact economic calculation and comparison of methods for harvesting is complex and practically impossible without information on the species chosen, production system and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> stage for harvesting. We have summarised the costs published in the literature to date in <a title="Select to navigate to table" href="#tab8">Table 8</a>. This table contains the best-characterised processes and we have compared these on the basis of processing volume, species application, capital and operating costs. The purpose of the overview table is to provide relative costs estimates, not absolute comparisons. The cited literature lacked detailed information on, <span class="italic">e.g.</span> the type and energy requirements of the centrifuges used. Furthermore, species and developmental information is largely lacking in the cited reports, and together with large uncertainties on capital an operating costs it makes exact cost comparisons difficult (if not impossible). The harvesting costs are taken into consideration further in Section 6, where the economic analysis of the complete algal production process is considered.</p>
        <p class="otherpara">In summary, most of the literature on the economic considerations of harvesting processes has focused on the need for highly purified products for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: food" href="javascript:popupOBO('CHEBI:33290','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')">food</a></span> and feed industry and waste <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: purification" href="javascript:popupOBO('CMO:0002231','b924978h')">purification</a></span>. The application to harvesting biomass for biofuels will not require the stringent product purity that is the case for the nutraceutrical industry, which will make the process cheaper. It has been mentioned before<a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> that there is not a universally valid (one-size-fits-all) optimum separation process. However, the harvesting process is a major consideration in the set-up of the production process. The biomass concentration and form will affect (and most likely be dictated by) the <span class="AN"><a class="simple" title="Sequence Ontology link for: downstream" href="javascript:popupOBO('SO:0001633','b924978h')">downstream</a></span> treatment processes.</p>
      
      
        
        <h3 id="sect3548"><span class="b_heading">5.2 Biomass pre-treatment and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> fraction</span></h3>
        <span>The harvested biomass (consisting of about 520% solids, depending on the harvesting method used) has to be processed further to produce the feedstocks (<span class="italic">i.e.</span> <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> fraction) for biofuels production. In most economic analyses the exact harvest, <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> and process details are not discussed. This is largely due to the uncertainty surrounding the effectiveness, <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> yield and the costs associated with the <span class="AN"><a class="simple" title="Gene Ontology link for: catalytic" href="javascript:popupOBO('GO:0003824','b924978h')">catalytic</a></span> upgrading of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> fraction.</span>
        <p class="otherpara">Most of the economic analyses published extrapolate the oil <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> procedures from a seed oil <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> process to algal biomass. Seeds require very little pretreatment or processing and the oils (mainly triglycerides) are traditionally extracted either by pressing the oil out of the biomass or using an apolar <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvent" href="javascript:popupOBO('CHEBI:46787','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvent</a></span> to selectively extract the triglycerides from the biomass. Because of large differences between algal and seed biomass properties and oil composition, specific procedures will have to be developed, applied and costed for an algal production process.</p>
        <p class="otherpara">Pretreatment, such as drying or <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> disruption, of algal biomass has been shown to aid <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span>.<a title="Select to navigate to reference" href="#cit94"><sup><span class="sup_ref">94,104,105</span></sup></a> One advantage of processing dried biomass is better percolation of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvents" href="javascript:popupOBO('CHEBI:46787','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvents</a></span> in the biomass, leading to an increase in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span>.<a title="Select to navigate to reference" href="#cit104"><sup><span class="sup_ref">104,106,107</span></sup></a> However drying is not considered an economical option for biomass pretreatment for a biofuels production process because of the high-energy requirements of the drying process.</p>
        <p class="otherpara">Alternative pretreatment methods include <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> rupture of the harvested biomass, which could aid in the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> process by avoiding or reducing the use of <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvents" href="javascript:popupOBO('CHEBI:46787','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvents</a></span>. A range of <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> rupture techniques, mechanical, chemical and enzymatic, has been described in the literature as applied to oilseeds and algae.<a title="Select to navigate to reference" href="#cit104"><sup><span class="sup_ref">104,108112</span></sup></a> However, no such method has been adopted for algae on a large-scale. This is, most likely, because the effectiveness of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> rupture methodology depends heavily on the physical properties of the algal <span class="AN"><a class="simple" title="Gene Ontology link for: cell wall" href="javascript:popupOBO('GO:0005618','b924978h')">cell wall</a></span> and thus will change or have to be adapted with particular species used in the culture. The composition and strength of the <span class="AN"><a class="simple" title="Gene Ontology link for: cell wall" href="javascript:popupOBO('GO:0005618','b924978h')">cell wall</a></span> varies considerably across species and throughout the algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> cycle. For example, <span class="italic">Chlorella</span> sp. and <span class="italic">Scenedesmus</span> sp. both have robust <span class="AN"><a class="simple" title="Gene Ontology link for: cell walls" href="javascript:popupOBO('GO:0005618','b924978h')">cell walls</a></span> with high cellulose content, rendering the <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> difficult to digest or rupture.<a title="Select to navigate to references" href="#cit113"><sup><span class="sup_ref">113</span></sup></a> Alternatively, species like <span class="italic">Spirulina</span> (a cyanobacterium) lack a rigid <span class="AN"><a class="simple" title="Gene Ontology link for: cell wall" href="javascript:popupOBO('GO:0005618','b924978h')">cell wall</a></span> and are thus easier to disrupt,<a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a> however, cyanobacteria are characteristically low in <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content.</p>
      
      
        
        <h3 id="sect3602"><span class="b_heading">5.3. Production of biofuels</span></h3>
        <span>The lipids obtained by the above processes can then be catalytically converted to biodiesel (<span class="CH"><a class="simple" title="ChEBI Ontology link for: methyl" href="javascript:popupOBO('CHEBI:32875','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=32875')">methyl</a></span> esters of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> portion of lipids) or green diesel (deoxygenated <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> derived from fatty acids). Conventional biodiesel production occurs <span class="italic">via</span> the transesterification of pre-extracted lipids in the presence of an alcohol and a <span class="CH"><a class="simple" title="ChEBI Ontology link for: catalyst" href="javascript:popupOBO('CHEBI:35223','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35223')">catalyst</a></span>, which can be a strong acid or base to produce <span class="CH"><a class="simple" title="ChEBI Ontology link for: fatty acid" href="javascript:popupOBO('CHEBI:35366','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35366')">fatty acid</a></span> methyl esters. Green diesel refers to the production of alkanes through for example <span class="AN"><a class="simple" title="Gene Ontology link for: catalytic" href="javascript:popupOBO('GO:0003824','b924978h')">catalytic</a></span> decarboxylation of fatty acids. A detailed discussion of the methodology and efficiency of these processes falls outside of the scope of this review and can be found elsewhere.<a title="Select to navigate to references" href="#cit115"><sup><span class="sup_ref">115</span></sup></a> It is worth mentioning that the factors determining the yield and conversion efficiency, such as temperature, time of incubation and <span class="CH"><a class="simple" title="ChEBI Ontology link for: catalyst" href="javascript:popupOBO('CHEBI:35223','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=35223')">catalyst</a></span> concentration, are highly dependent on the nature and composition of the feedstocks. No systematic study has been undertaken to optimise, develop and price the cost of a biodiesel production process specifically for algal lipids or biomass. A detailed costing scheme was developed for biodiesel production from soybean oil.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> However, there are too many unknowns in the algal biodiesel process to be able to provide an estimate for the cost of this process. Furthermore, it is anticipated that this process will be species, <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>- and environmental conditions-dependent.</span>
      
      
        
        <h3 id="sect3618"><span class="b_heading">5.4. Processing of the residual biomass and co-products</span></h3>
        <span>Critical to the economic success of algal biofuel production is the use to which the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> co-products are put, as these can be as, if not more, valuable than the biodiesel itself. The process that is utilized in the cost model discussed in section 6 includes anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> as an option for residual biomass utilization.</span>
        <p class="otherpara">Anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> is a long established technology in wastewater management for the disposal of organic waste and the recouping of energy. In essence, in the absence of oxygen, bacteria turn to a variety of alternative <span class="CH"><a class="simple" title="ChEBI Ontology link for: proton" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">proton</a></span> acceptors to dispose of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protons" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">protons</a></span> the organisms abstract from their organic substrates. If available, bacteria will use <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrate" href="javascript:popupOBO('CHEBI:51081','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51081')">nitrate</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: sulfate" href="javascript:popupOBO('CHEBI:26820','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26820')">sulfate</a></span> and carbon dioxide in that order of preference. If <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrate" href="javascript:popupOBO('CHEBI:51081','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=51081')">nitrate</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: sulfate" href="javascript:popupOBO('CHEBI:26820','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26820')">sulfate</a></span> are absent, or at low concentration, carbon dioxide is the main <span class="CH"><a class="simple" title="ChEBI Ontology link for: proton" href="javascript:popupOBO('CHEBI:24636','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24636')">proton</a></span> acceptor and the products are <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:methane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.291.html">methane</a></span> and CO<small><sub>2</sub></small>. The yield of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:methane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.291.html">methane</a></span> can be high and the energy recovery likewise high, a figure of 70% may be calculated from the data in <a title="Select to navigate to references" href="#cit15">ref. 15</a>. If <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: decomposition" href="javascript:popupOBO('CMO:0001487','b924978h')">decomposition</a></span> can be driven to near completion then the process has the potential to recycle the inorganic <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> (as <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span>) and <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span>, thus saving costs and the carbon footprint associated with the HaberBosch production of <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span>. The recycling of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>, of course, could only occur if the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acid" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acid</a></span> constituents of algal biomass were put to <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>. If the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> were sold for animal feed, then there would be very limited recycling of <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>, as 8090% or so of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> resides with these molecules (see <a title="Select to navigate to table" href="#tab2">Table 2</a>). The remainder of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> is principally associated with <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span>. This class of compounds have limited nutritional value and could usefully be put to anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>, were it economic  it would also result in recycling a major part of the cell's phosphorus (see <a title="Select to navigate to table" href="#tab2">Table 2</a>).</p>
        <p class="otherpara">Despite its obvious benefits, the economics of anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> are not promising; <a title="Select to navigate to references" href="#cit15">ref. 15</a> (p. 141) <span class="italic">Thus</span>, <span class="italic">power generation</span> (<span class="italic">from anaerobic </span><span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')"><span class="italic">digestion</span></a></span>) <span class="italic">is not a major profit centre</span>. Our own analysis of recycling the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> fractions <span class="italic">via</span> anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> likewise suggests that after allowing for the offsetting of the cost of electricity from the energy derived from the methane, there is no net revenue from anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> (see <a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a>).</p>
      
      
        
        <h3 id="sect3676"><span class="b_heading">5.5 Summary</span></h3>
        <span>From the foregoing discussion it can be seen that there are a number of routes available for the separation and processing of the algal culture. In many cases their effectiveness is dependent upon specific features of the algae, which may vary between species and with their physiological and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> status. Thus, no general strategy can be specified, although some may be eliminated based on the current state of technology (<span class="italic">e.g.</span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: filtration" href="javascript:popupOBO('CMO:0001640','b924978h')">filtration</a></span>).</span>
        <p class="otherpara">Uncertainties surrounding the optimum strategy for harvest and biomass processing would not be a major concern if the separation stage made an insignificant contribution to the overall budget. However, this does not appear to be the case  in the economic analysis undertaken in the present study (Section 6), harvest and <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> accounted for 15% or so of the operating budget (50% of the labour costs has been attributed to these processes). <a title="Select to navigate to references" href="#cit15">Ref. 15</a> gives a lower figure for operating costs but attribute a higher contribution (12%) to capital costs. Presently, the estimates for the cost of harvesting and separating should be seen as best guesses, and without doubt they are major sources of uncertainty in any economic analysis. Until target organisms have been identified this may have to remain to be the case. Furthermore, advances in engineering and improvements in centrifuge technology could also alter the costing.</p>
        <p class="otherpara">Similar uncertainties surround the fate of the residual biomass. The high <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content of the residual biomass could provide an excellent source of feed. However, if the <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">extraction</a></span> process is <span class="CH"><a class="simple" title="ChEBI Ontology link for: solvent" href="javascript:popupOBO('CHEBI:46787','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=46787')">solvent</a></span>-based, prior to considering animal feed as an option, toxicology tests will have to be carried out. The carbohydrates in the residual biomass could be used for <span class="AN"><a class="simple" title="Gene Ontology link for: ethanol fermentation" href="javascript:popupOBO('GO:0019655','b924978h')">ethanol fermentation</a></span>, however, not enough information is present in the literature to date regarding the composition of individual <span class="CH"><a class="simple" title="ChEBI Ontology link for: sugars" href="javascript:popupOBO('CHEBI:16646','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16646')">sugars</a></span> and concentration in the residual biomass as well as large unknowns about the recalcitrance of microalgal structural carbohydrates. The potential of the different scenarios on the use of the residual biomass will be discussed in the following section.</p>
      
    
    
      
      <h2 id="sect3694"><span class="a_heading">6. Economics of biomass and biofuel production</span></h2>
      <span>Widely varying estimates for the cost of algal biodiesel may be found in the recent literature (see <a title="Select to navigate to table" href="#tab10">Table 10</a>). This, in large part, reflects the immaturity of the subject. The aim of this section is not to produce a definitive costing  presently there are too many uncertainties to allow this  but give a basis to examine the pattern of the economics for various management scenarios. To this end a set of objective costings is needed. There is only a limited amount of information to draw upon to <span class="AN"><a class="simple" title="Sequence Ontology link for: construct" href="javascript:popupOBO('SO:0000804','b924978h')">construct</a></span> a costing algorithm. The DOE Aquatic Species Program commissioned a report (<a title="Select to navigate to references" href="#cit116">ref. 116</a>) to produce costings; <a title="Select to navigate to references" href="#cit15">ref. 15</a> incorporated the essence of this in the 1996 Report to the Pittsburgh Energy Technology Centre. A detailed costing was prepared for a pond/raceway type system.<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a> Although the capital costs were specified, it is difficult to extract the operational costs with any certainty. <a title="Select to navigate to references" href="#cit94">Ref. 94</a> gives a costing for a tubular bioreactor and <a title="Select to navigate to references" href="#cit103">ref. 103</a> costed a raceway system. Some of the numbers in the report in <a title="Select to navigate to references" href="#cit103">ref. 103</a> are recycled from <a title="Select to navigate to reference" href="#cit15">ref. 15 and 116</a>, so it cannot be taken as a wholly independent analysis  this is also the case for a number of other reports and to some small extent the present one.</span>
      
        
        <h3 id="sect3708"><span class="b_heading">6.1 Parameterising the economic model</span></h3>
        <span>We have based our analysis upon a recent set of costings prepared for the US Department of Energy's Office of Energy Efficiency and Renewable Energy<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a> for a combined photobioreactor (PBR)/raceway system; all costings were made <span class="italic">de novo.</span> This has been updated with published costings and from discussions with colleagues in the field. From this a spreadsheet model has been generated. All the items we have been able to cost are built into the spreadsheet. However, there are a number of processes we have been unable to cost with any certainty: the processing of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> debris to separate the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> fractions and the subsequent isolation and processing of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> fraction to animal feed. There are additional uncertainties surrounding the harvesting step. Estimating the cost of the necessary CO<small><sub>2</sub></small> supply is problematic as it will be very much dependent upon the nature of the overall facility, <span class="italic">i.e.</span> whether the algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> system is an adjunct to a CO<small><sub>2</sub></small> source (<span class="italic">i.e.</span> a power station) or an isolated facility. Our solution to this problem is, rather than to put in a costing that is circumstance-specific, to leave it out of the costing algorithm but to return to it in our discussion of the analysis of scenarios. A hybrid system, such as that described in <a title="Select to navigate to references" href="#cit16">ref. 16</a>, comprising a front end PBR nursery stage and grow-on stage in raceways is used as the physical arrangement as it appears to be the most cost effective option. An outline of the costings in the model is given in <a title="Select to navigate to table" href="#tab9">Table 9</a>. Without doubt it can be added to and improved upon.</span>
        
<div class="table_caption"><b>Table 9</b> <span id="tab9">Basic structure of the spreadsheet costing model, shown for one example: a low latitude situation, with a 28 g m<small><sup>2</sup></small> d<small><sup>1</sup></small> (103 tonnes ha<small><sup>1</sup></small> a<small><sup>1</sup></small>) productivity and a <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content of 35% of dry weight. In this example, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> is separated from the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> and sold at $900 tonne<small><sup>1</sup></small> as high-grade animal feed, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> and other <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> soluble products are processed by anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>. The biodiesel, as <span class="CH"><a class="simple" title="ChEBI Ontology link for: methyl" href="javascript:popupOBO('CHEBI:32875','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=32875')">methyl</a></span> esters, is sold at $900 tonne<small><sup>1</sup></small>. The costings are intended to be for a scale where they are no longer dependent of the size of the plant (<span class="italic">i.e.</span> there is no further economy of scale), but determined, where appropriate, by the size of the culture area and the scale of processing. To convert biodiesel from mass to volume units, it is assumed to have a specific gravity of 0.88 and a barrel is taken as 159 litres. All large numbers have been rounded off to 3 significant digits; dw = dry weight</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/></colgroup>
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">ASPECT OF PRODUCTION FACILITY</th>
                  <th align="left" valign="bottom" class="border_black">UNITS</th>
                </tr>
              </thead>
              <tfoot align="left">
                <tr align="left">
                  <th>
                    <a id="tab9fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Scaling of parameters: scaled to culture area.</span>
                    <a id="tab9fnb"><span class="tfootnote">b</span></a> <span class="sup_inf">Scaled to biomass production.</span>
                    <a id="tab9fnc"><span class="tfootnote">c</span></a> <span class="sup_inf">Scaled to <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> processed.</span>
                    <a id="tab9fnd"><span class="tfootnote">d</span></a> <span class="sup_inf">Scaled to tonnage digested.</span>
                  </th>
                </tr>
              </tfoot>
              <tbody align="left">
                <tr align="left">
                  <td colspan="2" valign="top">
                    <span class="bold">Operating Parameters &amp; Assumptions</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Areal production</td>
                  <td align="char" char="." valign="top">28 g dw m<small><sup>2</sup></small> d<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Annual tonnage produced</td>
                  <td align="char" char="." valign="top">103 tonnes dw ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">Lipid</a></span> content</td>
                  <td align="char" char="." valign="top">35% dw</td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Protein</a></span><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio</td>
                  <td align="char" char="." valign="top">3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 mass</td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">Lipid</a></span> production</td>
                  <td align="char" char="." valign="top">36 tonnes dw ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Bidiesel production (at 70% biolipid production)</td>
                  <td align="char" char="." valign="top">25.2 tonnes dw ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">Carbohydrate</a></span> used for anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span></td>
                  <td align="char" char="." valign="top">100%</td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="ChEBI Ontology link for: Protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">Protein</a></span> used for anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span></td>
                  <td align="char" char="." valign="top">0%</td>
                </tr>
                <tr align="left">
                  <td valign="top">Annual tonnage used for anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span></td>
                  <td align="char" char="." valign="top">35 tonnes dw ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Electricity yield from anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span></td>
                  <td align="char" char="." valign="top">139<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 MJ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="AN"><a class="simple" title="Gene Ontology link for: Growth" href="javascript:popupOBO('GO:0040007','b924978h')">Growth</a></span> area, as % site area</td>
                  <td align="char" char="." valign="top">88%</td>
                </tr>
                <tr align="left">
                  <td valign="top">DAP costs per tonne biomass</td>
                  <td align="char" char="." valign="top">0 $ tonne<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">NH<small><sub>3</sub></small> costs per tonne biomass</td>
                  <td align="char" char="." valign="top">21 $ tonne<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td colspan="2" valign="top">
                    <span class="bold">Capital costs</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Site prep<i><small><sup style="color: red;"><a title="" href="#tab9fna">a</a></sup></small></i></td>
                  <td align="char" char="." valign="top">41<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>800 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Culture system<i><small><sup style="color: red;"><a title="" href="#tab9fna">a</a></sup></small></i></td>
                  <td align="char" char="." valign="top">148<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Engineering<i><small><sup style="color: red;"><a title="" href="#tab9fna">a</a></sup></small></i></td>
                  <td align="char" char="." valign="top">22<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Harvest<i><small><sup style="color: red;"><a title="" href="#tab9fnb">b</a></sup></small></i></td>
                  <td align="char" char="." valign="top">3570 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="CH"><a class="simple" title="Chemical Methods Ontology link for: Extraction" href="javascript:popupOBO('CMO:0001577','b924978h')">Extraction</a></span><i><small><sup style="color: red;"><a title="" href="#tab9fnc">c</a></sup></small></i></td>
                  <td align="char" char="." valign="top">4560 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Anaerobic digester @ $225 tonne<small><sup>1</sup></small> annum<small><sup>1</sup></small> throughput<i><small><sup style="color: red;"><a title="" href="#tab9fnd">d</a></sup></small></i></td>
                  <td align="char" char="." valign="top">8730 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="TC"><a class="simple" title="Chemspider Compound link for:TOTAL" target="_blank" href="http://www.chemspider.com/Chemical-Structure.48392.html">TOTAL</a></span> CAPITAL COSTS</td>
                  <td align="char" char="." valign="top">228<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td colspan="2" valign="top">
                    <span class="bold">Running costs</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Labour<i><small><sup style="color: red;"><a title="" href="#tab9fna">a</a></sup></small></i></td>
                  <td align="char" char="." valign="top">4430 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="AN"><a class="simple" title="Gene Ontology link for: Growth" href="javascript:popupOBO('GO:0040007','b924978h')">Growth</a></span>  power requirement  electricity @ $0.07/kWh<i><small><sup style="color: red;"><a title="" href="#tab9fna">a</a></sup></small></i></td>
                  <td align="char" char="." valign="top">7020 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Harvest  power requirement  electricity @ $0.07/kWh<i><small><sup style="color: red;"><a title="" href="#tab9fna">a</a></sup></small></i></td>
                  <td align="char" char="." valign="top">2660 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Dewatering  power requirement  electricity @ $0.07/kWh<i><small><sup style="color: red;"><a title="" href="#tab9fna">a</a></sup></small></i></td>
                  <td align="char" char="." valign="top">105 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="TC"><a class="simple" title="Chemspider Compound link for:Diammonium phosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.16280567.html">Diammonium phosphate</a></span> (DAP)<i><small><sup style="color: red;"><a title="" href="#tab9fnb">b</a></sup></small></i></td>
                  <td align="char" char="." valign="top">0 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="TC"><a class="simple" title="Chemspider Compound link for:Ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">Ammonia</a></span><i><small><sup style="color: red;"><a title="" href="#tab9fnb">b</a></sup></small></i></td>
                  <td align="char" char="." valign="top">2210 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Transesterification  <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:methanol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.864.html">methanol</a></span><i><small><sup style="color: red;"><a title="" href="#tab9fnc">c</a></sup></small></i></td>
                  <td align="char" char="." valign="top">1650 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Transesterification  power requirement  natural gas<i><small><sup style="color: red;"><a title="" href="#tab9fnc">c</a></sup></small></i></td>
                  <td align="char" char="." valign="top">166 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Waste  energy value @ $0.07 kWh<small><sup>1</sup></small><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><i><small><sup style="color: red;"><a title="" href="#tab9fnd">d</a></sup></small></i></td>
                  <td align="char" char="." valign="top">1120 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Anaerobic digester operating costs<i><small><sup style="color: red;"><a title="" href="#tab9fnd">d</a></sup></small></i></td>
                  <td align="char" char="." valign="top">2720 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Captial costs @10% <span class="italic">per annum</span></td>
                  <td align="char" char="." valign="top">22<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>800 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top"><span class="TC"><a class="simple" title="Chemspider Compound link for:TOTAL" target="_blank" href="http://www.chemspider.com/Chemical-Structure.48392.html">TOTAL</a></span> COSTS</td>
                  <td align="char" char="." valign="top">42<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>700 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td colspan="2" valign="top">
                    <span class="bold">Product costs &amp; revenues</span>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Biomass cost</td>
                  <td align="char" char="." valign="top">0.41 $ kg<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Biodiesel cost</td>
                  <td align="char" char="." valign="top">1.69 $ kg<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Biodiesel cost</td>
                  <td align="char" char="." valign="top">237 $ bbl<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Revenue from biodiesel (@ $900 tonne<small><sup>1</sup></small></td>
                  <td align="char" char="." valign="top">22<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>700 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Revenue from pure <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> @ $900 tonne<small><sup>1</sup></small></td>
                  <td align="char" char="." valign="top">36<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>200 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Total revenue</td>
                  <td align="char" char="." valign="top">58<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>900 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">Total expenditure</td>
                  <td align="char" char="." valign="top">42<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>700 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
                <tr align="left">
                  <td valign="top">NET REVENUE</td>
                  <td align="char" char="." valign="top">16<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>200 $ ha<small><sup>1</sup></small> a<small><sup>1</sup></small></td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">Recently a comprehensive and thorough review of the economics of algal biomass production report has been prepared for the British Columbian Innovation Council.<a title="Select to navigate to references" href="#cit119"><sup><span class="sup_ref">119</span></sup></a> These authors worked largely, although not entirely, from fresh estimates. Their analysis was not available when our spreadsheet was generated. We are encouraged to find that their capital costs ($209<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>500 ha<small><sup>1</sup></small>, converted from Canadian dollars using a Can/US exchange rate of 0.8) are almost identical to our estimate ($209<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 ha<small><sup>1</sup></small>). Their estimate for operating costs, with the capital repayment costs removed, was $20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>425 per year; our estimate was $26<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>375; however, the similarity in these values is almost certainly fortuitous, since there is considerable variance in the estimates for several major individual items between the two budgets. Their energy costs are about 15% of ours; (deriving in part from the low cost of hydropower in British Columbia) whilst their labour costs are around four times greater (indicating that ours may need revisiting). There is, however, no significant change to the economic pattern seen in <a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a> if their labour costs are substituted into our economic analysis.</p>
        <p class="otherpara">In the economic analysis we consider three <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> levels (15, 35 and 50% <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> as dry weight) and three latitudes: low (030), mid (3545) and high (4555) latitude bands using the production rates shown in <a title="Select to navigate to figure" href="#imgfig23">Fig. 23</a>. We have explored three scenarios surrounding the exploitation of the non-lipid fraction: (i) the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> part of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> debris is sold as animal feed (60% <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, taken from the 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> to <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio used throughout this work, see legend to <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>); (ii) all of the non-lipid <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> debris (plus the <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glycerol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.733.html">glycerol</a></span> and other <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> soluble products from the transesterification process) is put to anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>, with the production of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:methane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.291.html">methane</a></span> (used to generate energy  in the process the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> are recycled); (iii) the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> from the algae and the glycerol and other <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> soluble products are digested anaerobically when all the <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> will be recycled, but only the small fraction of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> in <span class="CH"><a class="simple" title="ChEBI Ontology link for: nucleic acids" href="javascript:popupOBO('CHEBI:33696','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33696')">nucleic acids</a></span> and phospholipids, about 1015% of the total <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> (see <a title="Select to navigate to table" href="#tab2">Table 2</a>). The <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> is sold as high grade <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>-rich material, which could have a market as supplements for low <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> animal feeds such as corn meal (15% <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>).</p>
        <p class="otherpara">Assigning cash values to these various products (algal lipid, <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> plus <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> mix) has to be a matter of speculation as no market exits for them as yet. Animal feeding trials on algal preparations in general have been underway for some time.<a title="Select to navigate to references" href="#cit120"><sup><span class="sup_ref">120</span></sup></a> However, they will need to be extended to look at, amongst other things, the possible effects of materials used during harvesting and processing and it will be some time before a product value will be established. We have approached the problem by setting a starting figure from the expected chemical property of the product. The <span class="CH"><a class="simple" title="ChEBI Ontology link for: Food" href="javascript:popupOBO('CHEBI:33290','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33290')">Food</a></span> and Agriculture Organization (FAO) statistics<a title="Select to navigate to references" href="#cit132"><sup><span class="sup_ref">132</span></sup></a> for the prices of animal feed show a broad correlation with <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content, thus given a <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content we can derive an objective estimate of price. For feed with 60% <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content a value of $750 tonne<small><sup>1</sup></small> is obtained. An alternative calculation may be made <span class="italic">pro rata</span> from the price of soya meal (<span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> content 45%; median commodity value <span class="italic">ca.</span> $400 tonne<small><sup>1</sup></small>); this gives a lower figure of $533 tonne<small><sup>1</sup></small>. We have conservatively used a rounded down value of the lower figure  $500 tonne<small><sup>1</sup></small>. For the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>-only material the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>/price analysis is not regarded to be reliable as it exceeds the available data, a figure of $890 may be calculated <span class="italic">pro rata</span> from soya, this has been rounded off to $900 for the purpose of calculation. It's important to note that these values are only used as starting values for scenario analysis, so their exact values are not critical. Somewhat more critical is the price we assign to algal biodiesel. We have drawn a mean value of $916  73 tonne<small><sup>1</sup></small> from recent analyses by market specialists<a title="Select to navigate to footnote" href="#fn11">||||</a> and have rounded this down to $900 tonne<small><sup>1</sup></small>  this approximates to $125/barrel. Whether algal biodiesel will be more valuable than traditional biodiesel remains to be determined and will depend on the market when its <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> properties are better established, as well as social attitudes towards higher plant biodiesel (see highlighted comment in the Gallagher report, Section 1). As a starting point we have used the above value, plus two additional values: $550 tonne<small><sup>1</sup></small> and $700 tonne<small><sup>1</sup></small>, these are rounded off values for $75/barrel and $100/barrel.</p>
        <p class="otherpara">The required <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> and phosphorus, are assumed to be supplied as a mixture of di-<span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ammonium phosphate" target="_blank" href="http://www.chemspider.com/Chemical-Structure.140090.html">ammonium phosphate</a></span> and <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span> (costed at $325 tonne<small><sup>1</sup></small> and $300 tonne<small><sup>1</sup></small>, respectively) matched to the calculated <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> and phosphorous content for <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> of low (15%), medium (35%) and high (50%) <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content (see <a title="Select to navigate to table" href="#tab2">Table 2</a>). When all the non-lipid fraction is put to anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> cost is set to zero (this assumes 100% recovery). In the scenario where the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> fraction is not digested but sold, the <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> addition is set to zero and <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span> alone is added to satisfy the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> requirement.</p>
        <p class="otherpara">The model assumes that the cell's <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> ratio is 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>2 and does not vary with <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content. This assumption needs verification as it has a marked effect on some aspects of the costing. Furthermore, the separation of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> fraction will have influence on the costing, however, no data on this separation step was found in the literature. There are three losses that need to be taken into consideration: (i) the production of <span class="AN"><a class="simple" title="Gene Ontology link for: extracellular" href="javascript:popupOBO('GO:0005576','b924978h')">extracellular</a></span> dissolved organic material during <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>; (ii) an allowance for operational downtime; and (iii) losses during harvesting. There is a substantial literature surrounding the losses due to the release of organic material (a figure of 5% is used) and downtime is assumed to be 1 week <span class="italic">per annum</span> (a 2% loss). Losses on harvesting are presently not established. A round figure of 10% is taken for the combined loss from these three sources, although we suspect that this may be an underestimate.</p>
        <p class="otherpara">The spreadsheet (see <a title="Select to navigate to table" href="#tab9">Table 9</a>) scales the costings, where appropriate, to the tonnage <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> material produced, the tonnage of <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> processed and the tonnage put to anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>. The capital and operating costs of anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> were supplied by a UK manufacturer. The figures are not too dissimilar from costings (inflation adjusted) used by <a title="Select to navigate to reference" href="#cit15">ref. 15 and 67</a>. Anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> is assumed to lose 25% of the calorific value of the feedstock during the production of <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:methane" target="_blank" href="http://www.chemspider.com/Chemical-Structure.291.html">methane</a></span>, the latter is then converted to electricity assuming 35% efficiency, waste heat from the generators is not put to any specific use in our analysis, although it certainly would be in the plant design. The digester is assumed to enable 100% recycling of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> in the feedstock for use in the culture media. All of these assumptions need confirmation.</p>
      
      
        
        <h3 id="sect4153"><span class="b_heading">6.2 Analysis of scenarios</span></h3>
        <span>An illustration of the three end-product management scenarios is given in <a title="Select to navigate to figure" href="#imgfig24">Fig. 24A</a>. A general trend of reducing revenue with increasing <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content is apparent. Furthermore, only at a <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content of 50% of the cell's dry weight does the revenue of biodiesel exceed that from selling the algal <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>. However at 50% <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> levels, the assumption in the productivity model that sufficiently high <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rates can be sustained and only light limits <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>, becomes uncertain (see <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>).</span>
        <p class="otherpara">A summary of a matrix analysis for Scenarios 1 and 3 is shown in <a title="Select to navigate to figure" href="#imgfig25">Fig. 25</a>. Given a value of $125 bbl<small><sup>1</sup></small> for biodiesel, positive economic solutions can be found if the value of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> + <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> product (Scenario 1) is above $350 tonne<small><sup>1</sup></small>, and for the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>-only scenario (Scenario 3) above values of $600 tonne<small><sup>1</sup></small>. The analyses are not particularly sensitive to lowering of oil prices  an analysis for $350 tonne<small><sup>1</sup></small> (nominally $50 bbl<small><sup>1</sup></small>) is much the same as the analysis for $550 tonne<small><sup>1</sup></small>. However, a rise of the same scale to from $100 to $125 bbl<small><sup>1</sup></small> would make all the scenarios in <a title="Select to navigate to figure" href="#imgfig25">Fig. 25</a> profitable on the costings used.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig25"><img alt="A matrix of the outcome of analyses of economic Scenarios 1 and 3 for the net revenue resulting from various commodity prices for the algal products." src="/image/article/2010/EE/b924978h/b924978h-f25.gif"/></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 25 </b> <span id="fig25"><span class="graphic_title">A matrix of the outcome of analyses of economic Scenarios 1 and 3 for the net revenue resulting from various commodity prices for the algal products.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> (Scenario 2), even allowing for savings arising from the recycling of energy and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span>, is not a net source of revenue (<a title="Select to navigate to figure" href="#imgfig24">Fig. 24B</a>). This was also the earlier conclusion in <a title="Select to navigate to references" href="#cit15">ref. 15</a>. The costs of anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> come from temperate latitudes, where a considerable proportion of the energy yield has to be used to maintain the temperature of the digester; one may expect these circumstances to be very different at low latitudes. This may make the anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, along with <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span>, a more attractive proposition. This would be essential if the intention were for biodiesel to be the sole commodity. Whether the alternative of fermenting the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> and <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:glycerol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.733.html">glycerol</a></span> to <span class="RTC"><a class="simple" title="(Routine) Chemspider Compound link for:ethanol" target="_blank" href="http://www.chemspider.com/Chemical-Structure.682.html">ethanol</a></span> offers better economic rewards would be worth exploring.</p>
        <p class="otherpara">The analysis makes the point that the co-products, notably <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, have the potential to be very valuable. Perhaps the most important finding is that the simple aim to increase <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> yields using high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> containing algae may not necessarily be a sound economic strategy if the biochemical and other losses associated with <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> and biodiesel production are taken into consideration, gains from high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> production only appear at biodiesel prices above $125 bbl<small><sup>1</sup></small>.</p>
        <p class="otherpara">It is appropriate at this point to recognise that no costing has been made for CO<small><sub>2</sub></small> supply. At liquid CO<small><sub>2</sub></small> prices of $10 tonne<small><sup>1</sup></small> the pattern in <a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a> remains much the same. At $50 tonne<small><sup>1</sup></small>, the price determined from a recent study in <a title="Select to navigate to references" href="#cit121">ref. 121</a> for carbon capture and sequestration, profitable scenarios were only found at the higher price levels, at $100 tonne<small><sup>1</sup></small> profitable scenarios appear only with biodiesel prices of $125 and above.</p>
        <p class="otherpara">There is no doubt that our analysis can be improved upon and that there are uncertainties associated with many of the numbers used in the calculations. Large uncertainties surround the costing of the plant  notably the harvesting and subsequent treatment of the biomass and the eventual market values of the various biomass components. Uncertainties over the estimated areal production rates will condition the reliability of our economic analysis. There is active ongoing research and this is one area where one can anticipate substantial advances in the not too distant future. The present model assumes that light alone limits production, the algae are presumed not to enter <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrient" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrient</a></span> limitation, it this respect it may be providing optimistic estimates. On the other side of the coin, the sense exists that there is potential to improve upon present yields. Potential gains in productivity could arise from modifications of the cell's light capturing apparatus (see Section 4.5). We have chosen not to make allowance for these potential improvements. By far the majority of algal farms to date have focused on high value products, so the incentive to undertake expensive R &amp; D to increase yields has been low. This investment will need to be made to increase sustained algal biomass production rates if we are to use algae for relatively low value products such as biodiesel and animal feed.</p>
      
      
        
        <h3 id="sect4215"><span class="b_heading">6.3 Comparison with other economic analyses</span></h3>
        <span>We have compared the costings published in the literature. <a title="Select to navigate to table" href="#tab10">Table 10</a> gives a collation of published costings. At first sight they seem disturbingly disparate: estimates for biodiesel costs range for $280$7500 tonne<small><sup>1</sup></small> ($39$1050 bbl<small><sup>1</sup></small>) and biomass from $110$30<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>400 tonne<small><sup>1</sup></small>. <a title="Select to navigate to references" href="#cit117">Ref. 117</a> contains his own estimate and summarises earlier work  the average of these estimates in the region $15<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 tonne<small><sup>1</sup></small>. Part of the reason for the high costs in the latter study are the conservative assumed yield rates (15 to 20 g dry weight m<small><sup>2</sup></small> d<small><sup>1</sup></small>) used in the analysis, but also in Borowitzka's study an apparently high operational cost. The estimate in <a title="Select to navigate to references" href="#cit103">ref. 103</a>, which again is high, likewise contains high values for manpower and harvesting, which seems out of line with other costings. Further, the numbers must also be considered in context  the high estimate reported in <a title="Select to navigate to references" href="#cit94">ref. 94</a> derives from an advanced PBR system, intended for a high value product. The costings in <a title="Select to navigate to references" href="#cit123">ref. 123</a> are based on an extremely cautious assumption of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> yields (2.714 g dry weight m<small><sup>2</sup></small> d<small><sup>1</sup></small>). We find it difficult to see how systems can be run in profit with biomass yields below 20 g dry weight m<small><sup>2</sup></small> d<small><sup>1</sup></small>. The careful analysis in <a title="Select to navigate to references" href="#cit119">ref. 119</a> also stands out as high-cost estimates; however, they made their analysis for the latitude of British Columbia (50N), where biomass yields would around half those from latitudes of 030. If we consider the remaining analyses and disregard the high cost, non-viable, versions of the present report and the worst case scenario of the analysis in <a title="Select to navigate to references" href="#cit95">ref. 95</a>, the spread is still large but reduced to $110$600 tonne<small><sup>1</sup></small> for biomass and $280$2050 tonne<small><sup>1</sup></small> ($39$315 bbl<small><sup>1</sup></small>) for biodiesel. Despite the uncertainties, the broad patterns in the economics are probably correct, although it would be unrealistic to expect the exact values to be.</span>
        
<div class="table_caption"><b>Table 10</b> <span id="tab10">A comparison of published costings for algal biomass and biodiesel production. We have only taken the median value from <a title="Select to navigate to references" href="#cit122">ref. 122</a>. To convert biodiesel from mass to volume units, it is assumed to have a specific gravity of 0.88 and a barrel is taken as 159 litres. The estimates have been adjusted to 2007 prices, the currency conversions used are: $ Aus/$ US = 0.81, $ Can/$ US = 0.8 and /$ US = 1.28</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/><col/><col/><col/></colgroup>
              
              
              
              
              
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th rowspan="3" align="left" valign="bottom" class="border_black">Source</th>
                  <th rowspan="3" align="left" valign="bottom" class="border_black">Organism</th>
                  <th align="left" valign="bottom" class="border_black">Assumed productivity</th>
                  <th align="left" valign="bottom" class="border_black">Assumed <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content</th>
                  <th colspan="3" align="left" valign="bottom" class="border_black">Biomass costs</th>
                  <th colspan="3" align="left" valign="bottom" class="border_black">Biodiesel costs</th>
                </tr>
                <tr align="left" valign="bottom">
                  <th rowspan="2" align="left" valign="bottom" class="border_black">g dw/m<small><sup>2</sup></small> d<small><sup>1</sup></small></th>
                  <th rowspan="2" align="left" valign="bottom" class="border_black">% dw</th>
                  <th colspan="3" align="left" valign="bottom" class="border_black">$ tonne<small><sup>1</sup></small></th>
                  <th colspan="3" align="left" valign="bottom" class="border_black">$ tonne<small><sup>1</sup></small> [$ bbl<small><sup>1</sup></small>]</th>
                </tr>
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">PBR</th>
                  <th align="left" valign="bottom" class="border_black">RW</th>
                  <th align="left" valign="bottom" class="border_black">Hybrid</th>
                  <th align="left" valign="bottom" class="border_black">PBR</th>
                  <th align="left" valign="bottom" class="border_black">RW</th>
                  <th align="left" valign="bottom" class="border_black">Hybrid</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td rowspan="3" valign="top">Borowitzka (1992)<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">20</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>6800</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Various</td>
                  <td valign="top">3.230 </td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">35070<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Mean (+StDev)</td>
                  <td valign="top">(14.4  8.2)</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">(1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>4600  8200)</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">Benemann &amp; Oswald (1996)<a title="Select to navigate to references" href="#cit15"><sup><span class="sup_ref">15</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">30</td>
                  <td char="." valign="top">50</td>
                  <td valign="top"></td>
                  <td char="." valign="top">241</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">490 [69]</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">60</td>
                  <td char="." valign="top">50</td>
                  <td valign="top"></td>
                  <td char="." valign="top">148</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">300 [42]</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Molina Grima <span class="italic">et al.</span> (2004)<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a></td>
                  <td valign="top">
                    <span class="italic">Phaeodactylum</span>
                  </td>
                  <td valign="top">Not given</td>
                  <td valign="top">Not given</td>
                  <td valign="top">30<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>400</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">Moheimani (2005)<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a></td>
                  <td valign="top">
                    <span class="italic">Dunaliella</span>
                  </td>
                  <td char="." valign="top">20</td>
                  <td valign="top">Not given</td>
                  <td valign="top"></td>
                  <td valign="top">6100</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">
                    <span class="italic">Pleurochrysis</span>
                  </td>
                  <td valign="top">1620</td>
                  <td valign="top">Not given</td>
                  <td valign="top"></td>
                  <td valign="top">47006900</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                  <td valign="top">
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">van Harmelen &amp; Oonk (2006)<a title="Select to navigate to references" href="#cit122"><sup><span class="sup_ref">122</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">27</td>
                  <td char="." valign="top">30</td>
                  <td valign="top"></td>
                  <td char="." valign="top">370</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">1200 [168]</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">Chisti (2007)<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">72</td>
                  <td char="." valign="top">30</td>
                  <td char="." valign="top">470</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">1600 [224]</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">35</td>
                  <td char="." valign="top">30</td>
                  <td valign="top"></td>
                  <td char="." valign="top">600</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">2050 [287]</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Huntley &amp; Redalje (2007)<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td valign="top">1960</td>
                  <td char="." valign="top">40</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">110360</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">280910 [39127]</td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Carlsson <span class="italic">et al.</span> (2007)<a title="Select to navigate to references" href="#cit123"><sup><span class="sup_ref">123</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td valign="top">2.714</td>
                  <td valign="top">Not given</td>
                  <td valign="top"></td>
                  <td valign="top">200015<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td rowspan="2" valign="top">Alabi <span class="italic">et al.</span> (2009)<a title="Select to navigate to references" href="#cit119"><sup><span class="sup_ref">119</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">15.3</td>
                  <td char="." valign="top">25</td>
                  <td valign="top">5900</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">9.4</td>
                  <td char="." valign="top">15</td>
                  <td valign="top"></td>
                  <td valign="top">2140</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td rowspan="3" valign="top">Pienkos &amp; Darzins (2009)<a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a></td>
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">10</td>
                  <td char="." valign="top">15</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">7500 [1050]</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">25</td>
                  <td char="." valign="top">25</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">2250 [315]</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Not specified</td>
                  <td char="." valign="top">50</td>
                  <td char="." valign="top">50</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">750 [[105]</td>
                  <td valign="top"></td>
                </tr>
                <tr align="left">
                  <td colspan="10" valign="top"></td>
                </tr>
                <tr align="left">
                  <td valign="top">Present account</td>
                  <td valign="top">Not specified</td>
                  <td valign="top">1837</td>
                  <td valign="top">1550</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">360650</td>
                  <td valign="top"></td>
                  <td valign="top"></td>
                  <td valign="top">9003500 [126490]</td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">Thus, given the most favourable form of culture  a photobioreactor/raceway hybrid  we can envisage a number of scenarios that would appear to have the potential to yield net profit.</p>
      
    
    
      
      <h2 id="sect4542"><span class="a_heading">7 Prospects and research needs</span></h2>
      <span>In summing up the review we explore the answers to two questions that arose through the course of the review:</span>
      <p class="otherpara">(1) Can the production of algal biomass (or biolipid) be made profitable?</p>
      <p class="otherpara">(2) What R &amp; D is needed to achieve these aims?</p>
      
        
        <h3 id="sect4550"><span class="b_heading">7.1 Can the production of algal biomass (or biolipid) be made profitable?</span></h3>
        <span>The evidence from our analysis is a conditioned yes. First, in our analysis in Section 6 we found that, with prices for biodiesel up to $1400 tonne<small><sup>1</sup></small> ($200/barrel), we are only able to find profitable outcomes if all the major biochemical products are processed and marketed. We are unable to find economic solutions for the marketing of biodiesel alone and the associated recycling of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span> and energy in the non-lipid fractions by anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>. Our finding is in agreement with <a title="Select to navigate to references" href="#cit122">ref. 122</a>, in which it was concluded that the <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> only option was not economically plausible  other revenue sources were needed to bring the system into profit. If the first option is adopted, <span class="italic">i.e.</span> to market other biomass products, notably <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> as animal feed, in addition to biolipid, the market for <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> will control the overall scale of production. This has implications, potentially setting a limit to the scale of production.</span>
        <p class="otherpara">There are broadly four areas of uncertainty in the economic analysis. First, we have not been able to obtain costings for anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> at low latitudes. This could extend the economic viability. A second major uncertainty relates to the cost of harvesting. This is probably the most complex of the problems, as the properties of the alga used, its nutritional and physiological status, will determine the strategy that can be used for harvesting. Harvesting costs will have a major, if not overriding, effect on strain selection. Estimates of harvesting costs (capital and operating costs), as a percentage of total costs, vary considerably in different economic analyses. The analysis in <a title="Select to navigate to references" href="#cit103">ref. 103</a> gave a range of 4050% for the harvest and processing costs for the calcium carbonate-containing coccolithophorid <span class="italic">Pleurochrysis carterae</span> and 63% for <span class="italic">Dunaliella salina</span>. The analysis in <a title="Select to navigate to references" href="#cit15">ref. 15</a> gave range of 2025%, and the analysis in <a title="Select to navigate to references" href="#cit117">ref. 117</a> analysis implies a similar figure: 1525% of the capital costs and 20% of the operating costs were associated with harvesting. We derive a figure in the region of 10% in the present analysis, similar to the estimate of 12% in <a title="Select to navigate to references" href="#cit119">ref. 119</a>. Much of this variability derives not from the uncertainty in the costings themselves, over which there is reasonable consistency, but in the uncertainty over what procedures will be used. In particular, whether or not and to what extent <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>, which has high capital and operating costs, will be used; <span class="italic">i.e.</span> whether it has to be used for primary separation (which seems unlikely) or final concentration. Doubling the harvesting costs in our analysis gives rise to a 1015% fall in revenue of the more profitable scenarios at low latitudes, but a greater (1530%) loss at high latitudes.</p>
        <p class="otherpara">A third area of uncertainty deals with the future market prices for the various commodities  notably the value and acceptability of algal <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> as an animal feed and, of course, the market price of biodiesel. If legislation for minimum additions of biodiesel to <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span> persists and the ecological concerns over higher plant <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuels" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuels</a></span> continues or rises, then this may inflate the value and so price of algal biodiesel.</p>
        <p class="otherpara">The fourth major uncertainty lies in the prediction of biomass yields. As discussed in Section 6, yields have a pronounced effect upon the economics. If, for example, the yields are reduced to two-thirds from about 3040 g dry weight m<small><sup>2</sup></small> d<small><sup>1</sup></small> (100140 tonnes ha<small><sup>1</sup></small> a<small><sup>1</sup></small>) to 2025 g dry weight m<small><sup>2</sup></small> d<small><sup>1</sup></small> (6793 tonnes ha<small><sup>1</sup></small> a<small><sup>1</sup></small>) then we find it difficult to produce economic scenarios. Although predicting yields is a complex matter, we would see in many respects that producing models of algal production is a more tractable problem than many of the others above and the potential exists to produce robust models in the not too distant future.</p>
      
      
        
        <h3 id="sect4598"><span class="b_heading">7.2 What R &amp; D is needed to achieve these aims?</span></h3>
        <span>A number of reports have considered the R &amp; D required to produce algal biomass economically, and on an industrial scale.<a title="Select to navigate to reference" href="#cit3"><sup><span class="sup_ref">3,5,15</span></sup></a> The list of R &amp; D needs is very long and there is no need to detail these at length. The aim here is to give emphasis to the R &amp; D requirements that this review has indicated to be of greatest importance and urgency: optimising yields, advancing our understanding of harvesting and processing strategies and resolving the problems of scaling up.</span>
        <div>
          
          <span id="sect4604"/><span class="c_heading_indent">(i) Optimising yields. </span>
          <span>This may be subdivided into two broad problems  increasing production rates and maximising the economics of product yields.</span>
          <p class="otherpara">In Section 3 we developed the broad basis for the theoretical photosynthetic yields  a figure in the region of 10% of the energy in incident irradiation incorporated into organic material is a widely accepted value. This maximum yield, we point out, is constrained by stoichiometry and thermodynamics and, for all practical purposes, may be considered to be immovable. Actual yields fall short of this, often being in the region of one third to one tenth of the theoretical maximum (see Section 4.3(ii)). As far as we can determine, there is no fundamental barrier to achieving the theoretical yields, however finding solutions that are economically viable present severe practical challenges. The losses are principally due to wastage of the energy of the collected <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> at high <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> flux rates. This derives mainly from a mismatch between the processing rates of the photochemical and enzymatic reactions. The problem was discussed at length in Section 4.5(i) and its probable resolution lies not in increasing the rates of the dark reaction, there seems limited capability to do this, but by slowing down the photochemical reaction by reducing the <span class="CH"><a class="simple" title="ChEBI Ontology link for: photon" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photon</a></span> capturing capability of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span>. There has been work done on this and progress made. This will require modifying the control of the cell's light capturing apparatus, by some intrusive procedure  essentially locking the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> into the high light adapted state. It is very unlikely to come from screening of natural species but rather through genetic engineering. In <a title="Select to navigate to references" href="#cit15">ref. 15</a> it was noted that the type of strain needed would be as <span class="italic">as common as hen's teeth</span> in Nature, as they would have little to no competitive capability. A major issue is whether this could be achieved by classical plant breeding techniques  induced mutations (so called accelerated evolution techniques) or by direct manipulation of the <span class="AN"><a class="simple" title="Sequence Ontology link for: genome" href="javascript:popupOBO('SO:0001026','b924978h')">genome</a></span>. Some major players in the field, sensitive to the negative public reaction towards genetically modified organisms (GMOs), have set their face against going down the latter route. This, as other issues that surround the use of GMOs, will need to be debated.</p>
          <p class="otherpara">In a growing microalgal culture there is a complex of interactions between the organism and the physical and chemical environments, involving a number of positive and negative feedbacks (see Section 4.4). Commonly used, simple first order empirical models based on energetics, such as that employed in Section 4.5(iii), cannot incorporate these feedbacks in any explicit way and thus can only be considered as first order solutions. Fundamental physiological models of algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> exist,<a title="Select to navigate to references" href="#cit124"><sup><span class="sup_ref">124</span></sup></a> but do not appear to be extensively used, as yet, in the field of mass production of algae. They will be needed to optimise production rates  further they have the potential to serve as an exploratory tool  to determine the environmental criteria necessary for economic production. One matter that needs urgent consideration is the apparent irradiance threshold (see <a title="Select to navigate to figure" href="#imgfig22">Fig. 22</a>, and text in Section 4.4(ii)). We conclude that this cannot be solely attributable to the compensation irradiance, but there may also be a secondary effect of light mediated through temperature. It may set an upper latitude boundary for year-round production. In <a title="Select to navigate to references" href="#cit122">ref. 122</a> a latitude boundary of 35 was suggested, they used a mean annual temperature of 15 C as a criterion. Such latitudinal boundaries would exclude year-round production in the EU, Russia and Canada, and geographically limit it in the US and China. Most of Africa, the Middle East, Central and South America, India, South-East Asia and Australia would however be well placed to develop such technologies. There is a pressing need to establish whether or not this apparent irradiance threshold is a general phenomenon and, if it is we need to gain a better understanding of its physiological basis.</p>
          <p class="otherpara">Many of the previous analyses of R &amp; D needs have pointed to the requirement to isolate or develop high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> producing algae. In relation to this, a pressing matter is to determine whether the apparent negative relationship between <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> content and <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate exists (see <a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a> and the associated discussion in Section 2.3). This brings to the fore the need to distinguish between high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="italic">containing</span> algae and high <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> <span class="italic">yielding</span> algae. <span class="CH"><a class="simple" title="ChEBI Ontology link for: Lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">Lipid</a></span> content by itself would seem of limited value, if it has to be achieved at the expense of <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate. This is particularly the case when products other than lipids are part of the overall economic equation (see Section 6 and <a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a>).</p>
          <p class="otherpara">Following on from the above, as soon as the economics are extended beyond the production of lipids and biodiesel, the value and the dynamics of the quantitative distribution of the other major biochemicals within the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> become a major issue. A collation of the reported proportions of the major biochemical fractions (lipids, carbohydrates and <span class="CH"><a class="simple" title="ChEBI Ontology link for: proteins" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">proteins</a></span>) was analysed in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a> and no pattern was evident. This may well be the case, but the compilation came from very disparate studies and in many cases required data conversions and any pattern may have been lost or concealed within the resultant noise. Network models for the flows of material during <span class="AN"><a class="simple" title="Gene Ontology link for: metabolism" href="javascript:popupOBO('GO:0008152','b924978h')">metabolism</a></span> have been developed for higher plants (see <a title="Select to navigate to reference" href="#cit50">ref. 50 and 125</a>). These could be adapted to microalgae. However, these models will need dedicated studies on microalgae in order to calibrate them. Once they are available, one could use these models as powerful predictive tools.</p>
        </div>
        <div>
          
          <span id="sect4642"/><span class="c_heading_indent">(ii) Advancing our understanding of strategies for harvesting and processing of algal biomass. </span>
          <span>Reviews and reports invariably point to processing, and particularly harvesting, as a major area of uncertainty that has major economic implications (as discussed in section 5.1). The present review reaffirms this. The point has been made on a number of occasions that <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span>, although conceptually and operationally simple, may not be an option as a primary form of separation on cost grounds. However, advances in <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span> engineering may improve the economics. Similarly, the current status of <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: filtration" href="javascript:popupOBO('CMO:0001640','b924978h')">filtration</a></span> and micro-straining technology does not allow for a viable option for harvesting algae on a large scale. Again, technological advances in <span class="AN"><a class="simple" title="Gene Ontology link for: membrane" href="javascript:popupOBO('GO:0016020','b924978h')">membrane</a></span> and systems engineering could change this supposition. Furthermore, the strong species-dependent nature of the <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: filtration" href="javascript:popupOBO('CMO:0001640','b924978h')">filtration</a></span> process makes it difficult to draw conclusions about the general applicability of this process. One is left with some form of <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span>, agglutination or <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: co-precipitation" href="javascript:popupOBO('CMO:0001690','b924978h')">co-precipitation</a></span> in combination with a <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: centrifugation" href="javascript:popupOBO('CMO:0001047','b924978h')">centrifugation</a></span> step to harvest <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span>. To provide some strategic basis we need a much more detailed understanding of the surface properties of the <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> and how this varies with species, physiology and culture conditions. As we start from a position of very little existing knowledge in the case of microalgae, it will call for a major research effort, however without some basic understanding we will be committed to continue to fumble our way through the problem by trial and error.</span>
          <p class="otherpara">Once a concentrated slurry of <span class="AN"><a class="simple" title="Cell Ontology link for: cells" href="javascript:popupOBO('CL:0000000','b924978h')">cells</a></span> has been produced we have a number of processing options. Three scenarios, which involved overall various ways of handling the major biochemicals, were considered in Section 6 (see <a title="Select to navigate to figure" href="#imgfig24">Fig. 24</a>). These raise major R&amp;D questions; the cost, practicality and utility of separating the <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: lipid" href="javascript:popupOBO('CHEBI:18059','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=18059')">lipid</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> fractions from one another, and the further separation of valuable, more specialised biochemical, <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutraceutical" href="javascript:popupOBO('CHEBI:50733','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=50733')">nutraceutical</a></span> and potential <span class="CH"><a class="simple" title="ChEBI Ontology link for: pharmaceutical" href="javascript:popupOBO('CHEBI:52217','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=52217')">pharmaceutical</a></span> products. In the analysis in Section 6, <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> was relegated to a low value commodity; research is needed to determine whether the bulk <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> fraction or a sub-fraction can be put to better use than simply recouping a fraction of its energy by anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>. Compositional analysis of the algal biomass will determine whether the <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> fraction can be used to generate a second biofuels stream; <span class="italic">e.g.</span> bioethanol after <span class="AN"><a class="simple" title="Gene Ontology link for: fermentation" href="javascript:popupOBO('GO:0006113','b924978h')">fermentation</a></span> of the structural <span class="CH"><a class="simple" title="ChEBI Ontology link for: sugars" href="javascript:popupOBO('CHEBI:16646','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=16646')">sugars</a></span>. However, current published <span class="CH"><a class="simple" title="ChEBI Ontology link for: carbohydrate" href="javascript:popupOBO('CHEBI:23008','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=23008')">carbohydrate</a></span> information does not allow predictions of their potential yield.</p>
          <p class="otherpara">Major questions surround the application of anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> to the processing of waste algal debris. Whereas we have extensive knowledge of the application of anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> to human and animal waste and a growing knowledge of the treatment of domestic waste, by comparison we have a very limited knowledge of the <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> of microalgal debris. We are, however, not without experience of applying anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> to algal material, as this was undertaken as part of the high <span class="CH"><a class="simple" title="Molecular Process Ontology link for: oxidation" href="javascript:popupOBO('MOP:0000568','b924978h')">oxidation</a></span> rate ponds used in some of the US west coast sewage treatment plants. How easy it would be to transfer this knowledge to the residual <span class="AN"><a class="simple" title="Cell Ontology link for: cell" href="javascript:popupOBO('CL:0000000','b924978h')">cell</a></span> material from mass algal culture needs to be explored. We need insight on a number of matters surrounding anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span>: (i) the speed and completeness with which <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> (presumably as <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span>) and particularly <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> are released and therefore recycled; (ii) the energetic budget and whether or not the net energy yield improves with the increasing ambient temperature at lower latitudes; and (iii) the fraction that remains undigested over some economic time scale and whether it has value as a fertiliser or soil conditioner.</p>
          <p class="otherpara">There remains the question whether, rather than separating and processing the individual biochemical fractions, there is value in subjecting algal biomass to some form of thermal upgrading for producing biofuels. Work on this was recently reviewed in <a title="Select to navigate to references" href="#cit115">ref. 115</a>; however, a great deal more will be required before we can establish the practicalities and their relative economics.</p>
        </div>
        <div>
          
          <span id="sect4693"/><span class="c_heading_indent">(iii) Resolving the problems of scaling up. </span>
          <span>There are two levels of scaling up: (i) from the laboratory to the pilot plant/modest production plant scale, (ii) from the pilot plant scale to a scale that would make a <span class="italic">significant</span> and <span class="italic">sustainable</span> contribution to global liquid hydrocarbon <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> production. The first is essentially scaling up from gram to tonnage production and is associated with the R&amp;D needs outlined above. The second brings along a number of special considerations. To make a significant contribution to global <span class="CH"><a class="simple" title="ChEBI Ontology link for: fuel" href="javascript:popupOBO('CHEBI:33292','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33292')">fuel</a></span> production one needs to consider the question of sustainable production approaching the near gigatonne scale. This, not surprisingly, brings along a <span class="AN"><a class="simple" title="Gene Ontology link for: host" href="javascript:popupOBO('GO:0018995','b924978h')">host</a></span> of problems: political, social and economic as well as scientific. There are a number of potential show stoppers to sustained production on this scale, three of them merit early consideration and research. Algal mass culture, in addition to land and sunlight, requires, at the minimum, significant quantities of <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> and <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span>. A first possible show stopper is the availability of the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> and phosphorus in particular, required for algal mass culture. Of the three major <span class="CH"><a class="simple" title="ChEBI Ontology link for: nutrients" href="javascript:popupOBO('CHEBI:33284','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=33284')">nutrients</a></span>, <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> might be argued not to be a problem as, given energy, it can be generated from atmospheric <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>. However even if the energy were derived from within the algal production facility, and thereby carbon neutral, nitrous oxide, a powerful greenhouse gas, is a by-product of the HaberBosch process (used for <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span> synthesis) and its release into the atmosphere will need to be incorporated into any life cycle analysis. In contrast to <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span>, supplies of <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> would need to come from very finite reserves. At present rates of exploitation the global reserves are estimated to have a lifespan no more than 50100 years.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a> One-off use of <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> on a gigatonne scale of algal lipid production would add significantly to the present demand and shorten the lifespan of the global phosphorus reserves. This returns us again to the problem of the recycling of <span class="CH"><a class="simple" title="ChEBI Ontology link for: phosphate" href="javascript:popupOBO('CHEBI:26020','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=26020')">phosphate</a></span> within the overall scheme of production and the successful, or otherwise, use of anaerobic <span class="AN"><a class="simple" title="Gene Ontology link for: digestion" href="javascript:popupOBO('GO:0007586','b924978h')">digestion</a></span> to achieve this.</span>
          <p class="otherpara">A second possible show stopper is the availability and delivery of significant quantities of CO<small><sub>2</sub></small>. As discussed in Section 4.4(iv), in order to achieve rapidly growing dense cultures, carbon dioxide has to be added to the culture in some way, as the natural exchange across the atmosphere is too slow. It may be possible to achieve this by active aeration relying upon the ambient concentration in air, although this would be costly or it may be necessary to actively separate the CO<small><sub>2</sub></small> from the atmosphere by liquefaction and add pure CO<small><sub>2</sub></small> to the culture. Neither is likely to be a low cost solution.</p>
          <p class="otherpara">The third potential show stopper, not surprisingly, is <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span>. As soon as one anticipates production plants of considerable scales, local <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> supplies are unlikely to be adequate and so <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> (for a number of reasons most probably seawater) will need to be pumped onto the culturing site and after use disposed of in some manner. The engineering and energy costs for pumping could be considerable and almost certainly one would need to resort to recycling <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> to some extent. Recycling of the spent culture medium is embedded in the conceptual scheme in <a title="Select to navigate to references" href="#cit95">ref. 95</a>. Recycling of used <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> brings along a number of problems, the spent culture medium will contain soluble organic products released during algal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span>; these will accumulate and may have a positive or negative effect upon the <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> rate of subsequent cultures. Similarly any residual additives, for example material added to induce <span class="AN"><a class="simple" title="Gene Ontology link for: flocculation" href="javascript:popupOBO('GO:0000128','b924978h')">flocculation</a></span> will be present or will have to be removed from the spent medium. Biological contaminants, <span class="italic">e.g.</span> competitors and predators, and, perhaps more problematic, pathogens such as viruses if not removed will be recycled. This may call for further treatment <span class="italic">e.g.</span> sterilisation of the recycled <span class="TC"><a class="simple" title="Chemspider Compound link for:water" target="_blank" href="http://www.chemspider.com/Chemical-Structure.937.html">water</a></span> with inevitable cost implications. There is experience of these matters from the existing production plants, whether this is sufficient to plan large-scale production facilities or whether further systematic R &amp; D is needed will require careful consideration.</p>
          <p class="otherpara">In conclusion, the production of biodiesel and other algal biomass components on an industrial scale does appear to have the potential to be a sound economic proposition. It would, however, be unwise to underestimate the scale of R &amp; D required to make this a reality. The diversity of knowledge required is such that it is unlikely that a single institution or corporation would have the resources to undertake the work exclusively in house. The review <span class="italic">National </span><span class="italic">Algal</span><span class="italic"> Biofuels Technology Roadmap</span> prepared as part of the US Department of Energy Biomass Program makes a strong play for openness in the R &amp; D and the value of collaborative work and exchange of information between industry and academic centres. This steer should be noted.</p>
          <p class="otherpara">It is widely accepted that it will need considerable political will and social change if we are to find solutions to the problem of greenhouse gas emissions. The same will and changes in attitude and approach will be needed if we are to contemplate putting algal biomass and biofuel production on a scale where it is a significant part of the overall equation. James Lovelock once observed that it characteristically takes at least 25 years for new major technologies to come on line, this broadly matches the timescale of many of the major political aspirations and promises surrounding the tackling the problem of CO<small><sub>2</sub></small> emissions. Thus, that the task may seem daunting should not deter us.</p>
        </div>
      
    
  
    <h2 id="sect4752"><span class="a_heading">Acknowledgements</span></h2>
      <span>We have valued the discussions with many of our colleagues, in particular P. W. has appreciated inputs and guidance from Ian Archibald (Shell Global Solutions), John Beardall (Monash University), Ian Joint (Plymouth Marine Laboratory), Phillip Williamson (University of East Anglia), Dylan Evans (Anglesey Sea Zoo) and David Bowers (School of Ocean Sciences, University of Bangor) and L. L. has appreciated input from Martia Ferreira-Novio (Universidade de Santiago de Compostela, Spain) in Section 5.1. Al Darzins and Andy Aden (NREL) are gratefully acknowledged for their comments on the manuscript. We thank Jasvir Atwal, Account Manager of ICIS-ICIS Pricing &amp; ICIS News for providing a copy of their publication Market. P. W. also thanks Jrg Schwender of the Biology Department of the Brookhaven National Laboratory for help with the calculations of the conversion efficiency of protein given in Box 1. We thank Carolyn Walburton for help with preparing the images and Alice Bowers for digitising the Pedroni <span class="italic">et al.</span> data used in <a title="Select to navigate to figure" href="#imgfig20">Fig. 20</a> and <a title="Select to navigate to figure" href="#imgfig21">21</a>. Finally, much of the data present in the Appendices and P. W.s understanding of the subject are an outcome of contracts from Shell and the company is thanked for their support and for their willingness to allow this material into the public domain.</span>
    
    <span id="sect4758"><h2 id="sect4751"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">P. J. le B. Williams, <span class="italic">Nature</span>, 2007, <span class="bold">450</span>, 478<a target="_blank" class="DOILink" href="https://doi.org/10.1038/450478a" title="DOI Link to resource 10.1038/450478a">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCrtbfO" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/1_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit2">N. S. Shifrin and S. W. Chisholm, <span class="italic">J. Phycol.</span>, 1981, <span class="bold">17</span>, 374384<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.0022-3646.1981.00374.x" title="DOI Link to resource 10.1111/j.0022-3646.1981.00374.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL38XkvVKrtw%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/2_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit3">
          J. Sheehan, T. Dunahay, J. R. Benemann and P. Roessler, <span class="italic">A Look Back at the U.S Department of Energy's Aquatic Species Program-Biodiesel from Algae</span>, National Renewable Energy Laboratory, Golden,  1998<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=A%20Look%20Back%20at%20the%20U.S%20Department%20of%20Energy's%20Aquatic%20Species%20Program-Biodiesel%20from%20Algae%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201998%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/3_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit4">Y. Chisti, <span class="italic">Biotechnol. Adv.</span>, 2007, <span class="bold">25</span>, 294306<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.biotechadv.2007.02.001" title="DOI Link to resource 10.1016/j.biotechadv.2007.02.001">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXjslGksbo%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/4_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit5">Q. Hu, M. Sommerfeld, E. Jarvis, M. Ghirardi, M. Posewitz, M. Seibert and A. Darzins, <span class="italic">Plant J.</span>, 2008, <span class="bold">54</span>, 621639<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1365-313X.2008.03492.x" title="DOI Link to resource 10.1111/j.1365-313X.2008.03492.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXmvFKgsrs%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/5_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit6">
          A. Richmond, in <span class="italic">Handbook of Microalgal Culture</span>, Blackwell,  2004, p. 566<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Handbook%20of%20Microalgal%20Culture%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/6_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit7">
          G. R. Cysewski and R. T. Lorenz, in <span class="italic">Hangbook of Microalgal Culture</span>, Blackwells,  2004, pp. 281288<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Hangbook%20of%20Microalgal%20Culture%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/7_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit8">Y. Chisti, <span class="italic">Trends Biotechnol.</span>, 2008, <span class="bold">26</span>, 126131<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.tibtech.2007.12.002" title="DOI Link to resource 10.1016/j.tibtech.2007.12.002">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXitVOktL4%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/8_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit9">K. M. Weyer, D. R. Bush, A. Darzins and B. D. Willson, <span class="italic">Bioenerg. Res.</span>, 2010<small>DOI:<a class="DOILink" href="https://doi.org/10.1007/s121550099046-x" TARGET="_BLANK" title="DOI Link to 10.1007/s121550099046-x">10.1007/s121550099046-x</a></small><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/9_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit10">M. J. Haas, A. J. McAloon, W. C. Yee and T. A. Foglia, <span class="italic">Bioresour. Technol.</span>, 2006, <span class="bold">97</span>, 671678<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.biortech.2005.03.039" title="DOI Link to resource 10.1016/j.biortech.2005.03.039">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXht12jurrO" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/10_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit11">
          P. G. Falkowski and J. A. Raven, <span class="italic">Aquatic Photosynthesis</span>, Princeton University Press,  2007<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Aquatic%20Photosynthesis%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/11_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit12">
          C. Van den Hoek, D. G. Mann, and H. M. Jahns., <span class="italic">Algae. An Introduction to Phycology</span>, Cambridge University Press, Cambridge,  1995<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Algae.%20An%20Introduction%20to%20Phycology%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201995%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/12_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit13">M. J. Furnas, <span class="italic">J. Plankton Res.</span>, 1990, <span class="bold">12</span>, 11171151<a target="_blank" class="DOILink" href="https://doi.org/10.1093/plankt/12.6.1117" title="DOI Link to resource 10.1093/plankt/12.6.1117">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/13_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit14">G. Sarthou, K. R. Timmermans, S. Blain and P. Treguer, <span class="italic">J. Sea Res.</span>, 2005, <span class="bold">53</span>, 2542<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.seares.2004.01.007" title="DOI Link to resource 10.1016/j.seares.2004.01.007">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXjsFSi" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/14_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit15">
          J. R. Benemann, W. J. Oswald, <span class="italic">Systems and Economic Analysis of Microalgae Ponds for Conversion of CO<small><sub>2</sub></small> to Biomass</span>, Final Report, Pittsburgh Energy Technology Center,  1996<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Systems%20and%20Economic%20Analysis%20of%20Microalgae%20Ponds%20for%20Conversion%20of%20CO2%20to%20Biomass%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/15_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit16">M. E. Huntley and D. G. Redalje, <span class="italic">Mitigation Adapt. Strategies Global Change</span>, 2007, <span class="bold">12</span>, 573608<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mitigation%20Adapt.%20Strategies%20Global%20Change%5BJour%5D%20AND%2012%5Bvolume%5D%20AND%20573%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/16_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit17">P. J. l. B. Williams and J. E. Robertson, <span class="italic">J. Plankton Res.</span>, 1991, <span class="bold">13</span>, 153169<a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/17_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit18">S. Ringen, J. Lanum and F. P. Miknis, <span class="italic">Fuel</span>, 1979, <span class="bold">58</span>, 6971<a target="_blank" class="DOILink" href="https://doi.org/10.1016/0016-2361(79)90056-5" title="DOI Link to resource 10.1016/0016-2361(79)90056-5">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaE1MXkslKnsr0%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/18_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit19">G. C. Dismukes, D. Carrieri, N. Bennette, G. M. Ananyev and M. C. Posewitz, <span class="italic">Biotechnology</span>, 2008, <span class="bold">19</span>, 235240<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Biotechnology%5BJour%5D%20AND%2019%5Bvolume%5D%20AND%20235%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/19_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit20">D. A. McCracken and J. R. Cain, <span class="italic">New Phytol.</span>, 1981, <span class="bold">88</span>, 6771<a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL3MXlsVOitLw%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/20_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit21">P. Roessler, <span class="italic">J. Phycol.</span>, 1987, <span class="bold">23</span>, 494498<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1529-8817.1987.tb02536.x" title="DOI Link to resource 10.1111/j.1529-8817.1987.tb02536.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL1cXptFCrsQ%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/21_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit22">K. Roberts, M. Gurney-Smith and G. J. Hills, <span class="italic">J. Ultrastruct. Res.</span>, 1972, <span class="bold">40</span>, 599613<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0022-5320(72)80046-7" title="DOI Link to resource 10.1016/S0022-5320(72)80046-7">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaE3sXhtleisbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/22_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit23">Y. K. Chau, C. L. Chuecas and J. P. Riley, <span class="italic">J. Mar. Biol. Assoc. U. K.</span>, 1967, <span class="bold">47</span>, 543554<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Mar.%20Biol.%20Assoc.%20U.%20K.%5BJour%5D%20AND%2047%5Bvolume%5D%20AND%20543%5Bpage%5D%20and%201967%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/23_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit24">D. Lpez Alonso, E. H. Belarbi, J. M. Fernandez Sevilla, J. Rodriguez Ruiz and E. M. Molina Grima, <span class="italic">Phytochemistry</span>, 2000, <span class="bold">54</span>, 461471<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0031-9422(00)00084-4" title="DOI Link to resource 10.1016/S0031-9422(00)00084-4">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/24_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit25">D. Lpez Alonso and F. Garcia Maroto, <span class="italic">Biotechnol. Adv.</span>, 2000, <span class="bold">18</span>, 481497<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0734-9750(00)00048-3" title="DOI Link to resource 10.1016/S0734-9750(00)00048-3">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/25_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit26">S. M. Budge and C. C. Parrish, <span class="italic">Phytochemistry</span>, 1999, <span class="bold">52</span>, 561566<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0031-9422(99)00241-1" title="DOI Link to resource 10.1016/S0031-9422(99)00241-1">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1MXntlOjsL4%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/26_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit27">J. P. Fidalgo, A. Cid, E. Torres, A. Sukenik and C. Herrero, <span class="italic">Aquaculture</span>, 1998, <span class="bold">166</span>, 105116<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0044-8486(98)00278-6" title="DOI Link to resource 10.1016/S0044-8486(98)00278-6">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXkslyksLw%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/27_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit28">Z. Gombos and N. Murata, <span class="italic">Plant Cell
Physiol.</span>, 1991, <span class="bold">32</span>, 7377<a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3MXhs1Snsbo%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/28_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit29">F. J. L. Gordillo, M. Goutx, F. L. Figueroa and F. X. Niell, <span class="italic">J. Appl. Phycol.</span>, 1998, <span class="bold">10</span>, 135144<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2010%5Bvolume%5D%20AND%20135%5Bpage%5D%20and%201998%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/29_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit30">P. A. Hodgson, R. J. Henderson, J. R. Sargent and J. W. Leftley, <span class="italic">J. Appl. Phycol.</span>, 1991, <span class="bold">3</span>, 169181<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%20169%5Bpage%5D%20and%201991%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/30_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit31">M. Kates and B. E. Volcani, <span class="italic">Biochim. Biophys. Acta</span>, 1966, <span class="bold">116</span>, 264278<a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaF28Xos1arsQ%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/31_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit32">M. G. Kleinschmidt and V. A. McMahon, <span class="italic">Plant Physiol.</span>, 1970, <span class="bold">46</span>, 286289<a target="_blank" class="DOILink" href="https://doi.org/10.1104/pp.46.2.286" title="DOI Link to resource 10.1104/pp.46.2.286">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaE3MXhtVersw%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/32_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit33">A. T. Lombardi and P. J. Wangersky, <span class="italic">Hydrobiologia</span>, 1995, <span class="bold">306</span>, 16<a target="_blank" class="DOILink" href="https://doi.org/10.1007/BF00007853" title="DOI Link to resource 10.1007/BF00007853">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXovVWksrg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/33_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit34">F. Pernet, R. Tremblay, E. Demers and M. Roussy, <span class="italic">Aquaculture</span>, 2003, <span class="bold">221</span>, 393406<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0044-8486(03)00030-9" title="DOI Link to resource 10.1016/S0044-8486(03)00030-9">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXitlGhsro%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/34_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit35">H. Tatsuzawa, E. Takizawa, M. Wada and Y. Yamamoto, <span class="italic">J. Phycol.</span>, 1996, <span class="bold">32</span>, 598601<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.0022-3646.1996.00598.x" title="DOI Link to resource 10.1111/j.0022-3646.1996.00598.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28XlvVWisbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/35_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit36">T. W. Ryan, L. G. Dodge and T. J. Callahan, <span class="italic">J. Am. Oil Chem. Soc.</span>, 1984, <span class="bold">61</span>, 16101619<a target="_blank" class="DOILink" href="https://doi.org/10.1007/BF02541645" title="DOI Link to resource 10.1007/BF02541645">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2cXmtFeqsbo%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/36_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit37">
          E. W. Becker, <span class="italic">Microalgae  Biotechnology and Microbiology</span>, Cambridge University Press, Cambridge,  1994<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Microalgae%20%20Biotechnology%20and%20Microbiology%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201994%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/37_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit38">M. R. Brown, <span class="italic">J. Exp. Mar. Biol. Ecol.</span>, 1991, <span class="bold">145</span>, 7999<a target="_blank" class="DOILink" href="https://doi.org/10.1016/0022-0981(91)90007-J" title="DOI Link to resource 10.1016/0022-0981(91)90007-J">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3MXltFehsbk%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/38_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit39">M. R. Brown, G. A. Dunstan, S. J. Norwood and K. A. Miller, <span class="italic">J. Phycol.</span>, 1996, <span class="bold">32</span>, 6473<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.0022-3646.1996.00064.x" title="DOI Link to resource 10.1111/j.0022-3646.1996.00064.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28Xjslagt74%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/39_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit40">T. R. Larson and T. A. V. Rees, <span class="italic">J. Phycol.</span>, 1996, <span class="bold">32</span>, 388393<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.0022-3646.1996.00388.x" title="DOI Link to resource 10.1111/j.0022-3646.1996.00388.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28XktlygurY%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/40_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit41">S. O. Loureno, E. Barbarino, J. Mancini-Filho, K. P. Schinke and E. Aidar, <span class="italic">Phycologia</span>, 2002, <span class="bold">41</span>, 158168<a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/41_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit42">P. A. Thompson, P. J. Harrison and J. N. C. Whyte, <span class="italic">J. Phycol.</span>, 1990, <span class="bold">26</span>, 278288<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.0022-3646.1990.00278.x" title="DOI Link to resource 10.1111/j.0022-3646.1990.00278.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3MXhs1Sgtbk%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/42_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit43">P. G. Roessler, <span class="italic">J. Phycol.</span>, 1990, <span class="bold">26</span>, 393399<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.0022-3646.1990.00393.x" title="DOI Link to resource 10.1111/j.0022-3646.1990.00393.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3MXit1Ghsbw%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/43_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit44">H. Xu, X. Miao and Q. Wu, <span class="italic">J. Technol.</span>, 2006, <span class="bold">126</span>, 499507<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Technol.%5BJour%5D%20AND%20126%5Bvolume%5D%20AND%20499%5Bpage%5D%20and%202006%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/44_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit45">N. Nagle and P. R. Lemke, <span class="italic">Appl. Biochem. Biotechnol.</span>, 1990, <span class="bold">2425</span>, 355361<a target="_blank" class="DOILink" href="https://doi.org/10.1007/BF02920259" title="DOI Link to resource 10.1007/BF02920259">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3cXkt1aitrc%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/45_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit46">
          C. K. Mathews and K. E. Van Holde, <span class="italic">Biochemistry</span>, Benjamin/Cummings Publishing Company, Redwood City,  1990<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Biochemistry%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201990%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/46_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit47">
          J. T. O. Kirk, <span class="italic">Light and Photosynthesis in Aquatic Ecosystems</span>, Cambridge University Press,  1994<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Light%20and%20Photosynthesis%20in%20Aquatic%20Ecosystems%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201994%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/47_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit48">W. Lubitz, E. J. Reijerse and J. Messinger, <span class="italic">Energy Environ. Sci.</span>, 2008, <span class="bold">1</span>, 1531<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=b808792j&amp;newsite=1" title="Link to RSC resource DOI:10.1039/b808792j">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/48_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit49">G. G. B. Tcherkez, G. D. Farquhar and T. J. Andrews, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2006, <span class="bold">103</span>, 72467251<a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0600605103" title="DOI Link to resource 10.1073/pnas.0600605103">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XkslOkur8%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/49_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit50">D. K. Allen, I. G. L. Libourel and Y. Shachar-Hill, <span class="italic">Plant, Cell Environ.</span>, 2009, <span class="bold">32</span>, 12411257<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1365-3040.2009.01992.x" title="DOI Link to resource 10.1111/j.1365-3040.2009.01992.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGhtrfJ" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/50_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit51">
          J. A. Raven and J. Beardall, in <span class="italic">Respiration in Aquatic Ecosystems</span>, ed. P. A. d. Giorgio and P. J. le B. Williams, Oxford University Press,  2005<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Respiration%20in%20Aquatic%20Ecosystems%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202005%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/51_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit52">C. Langdon, <span class="italic">J. Plankton Res.</span>, 1987, <span class="bold">9</span>, 459482<a target="_blank" class="DOILink" href="https://doi.org/10.1093/plankt/9.3.459" title="DOI Link to resource 10.1093/plankt/9.3.459">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/52_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit53">C. Langdon, <span class="italic">Mar. Sci. Symp.</span>, 1993, <span class="bold">197</span>, 6978<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mar.%20Sci.%20Symp.%5BJour%5D%20AND%20197%5Bvolume%5D%20AND%2069%5Bpage%5D%20and%201993%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/53_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit54">X. G. Zhu, S. P. Long and D. R. Ort, <span class="italic">Curr. Opin. Biotechnol.</span>, 2008, <span class="bold">19</span>, 153159<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.copbio.2008.02.004" title="DOI Link to resource 10.1016/j.copbio.2008.02.004">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXlslKisL4%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/54_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit55">K. Richardson, J. Beardall and J. A. Raven, <span class="italic">New Phytol.</span>, 1983, <span class="bold">93</span>, 157191<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1469-8137.1983.tb03422.x" title="DOI Link to resource 10.1111/j.1469-8137.1983.tb03422.x">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/55_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit56">J. Pirt, <span class="italic">New Phytol.</span>, 1986, <span class="bold">102</span>, 337<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1469-8137.1986.tb00794.x" title="DOI Link to resource 10.1111/j.1469-8137.1986.tb00794.x">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/56_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit57">E. Molina Grima, J. M. Fernandez Sevilla, J. A. Sanchez Perez and P. Garcia Camacho, <span class="italic">J. Biotechnol.</span>, 1996, <span class="bold">45</span>, 5969<a target="_blank" class="DOILink" href="https://doi.org/10.1016/0168-1656(95)00144-1" title="DOI Link to resource 10.1016/0168-1656(95)00144-1">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28XovFOltg%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/57_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit58">
          M. Tredici, in <span class="italic">Handbook of Microalgal Culture</span>, ed. A. Richmond, Blackwell,  2004, pp. 178214<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Handbook%20of%20Microalgal%20Culture%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/58_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit59">J. Huisman and F. J. Weissing, <span class="italic">Ecology</span>, 1994, <span class="bold">75</span>, 507520<a target="_blank" class="DOILink" href="https://doi.org/10.2307/1939554" title="DOI Link to resource 10.2307/1939554">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/59_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit60">G. Torzillo, T. Goksan, C. Faraloni, J. Kopecky and J. Masojidek, <span class="italic">J. Appl. Phycol.</span>, 2003, <span class="bold">15</span>, 127136<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2015%5Bvolume%5D%20AND%20127%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/60_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit61">NASA, http://edmall.gsfc.nasa.gov/inv99Project.Site/Pages/science-briefs/ed-stickler/ed-irradiance.html,  2009.</span></li>
      <li><span id="cit62">NASA, http://eosweb.larc.nasa.gov/sse/, <span class="italic">Atmospheric Science Data Center</span>, NASA Surface meteorology and Solar Energy: Global/Regional Data,  2009<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Atmospheric%20Science%20Data%20Center%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/62_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit63">E. S. Domalski, <span class="italic">J. Phys. Ref. Data</span>, 1972, <span class="bold">1</span>, 221277<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Phys.%20Ref.%20Data%5BJour%5D%20AND%201%5Bvolume%5D%20AND%20221%5Bpage%5D%20and%201972%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/63_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit64">M. S. Kharasch, <span class="italic">Bur. Stand. J. Res.</span>, 1929, <span class="bold">2</span>, 359<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Bur.%20Stand.%20J.%20Res.%5BJour%5D%20AND%202%5Bvolume%5D%20AND%20359%5Bpage%5D%20and%201929%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/64_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit65">J. C. Goldman, <span class="italic">Water Res.</span>, 1979, <span class="bold">13</span>, 119136<a target="_blank" class="DOILink" href="https://doi.org/10.1016/0043-1354(79)90083-6" title="DOI Link to resource 10.1016/0043-1354(79)90083-6">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaE1MXhtlGjs7g%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/65_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit66">
          G. Fischer, H. van Velthuizen, M. Shah, F. Nachtergaele, <span class="italic">Global Agro-Ecological Assessment for Agriculture in the 21st Century: Methodology and Results</span>, International Institute for Applied Systems Analysis (IIASA), Vienna,  2002<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Global%20Agro-Ecological%20Assessment%20for%20Agriculture%20in%20the%2021st%20Century:%20Methodology%20and%20Results%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202002%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/66_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit67">
          D. L. Klass, <span class="italic">Biomass for Renewable Energy, Fuels, and Chemicals</span>, Elsvier Incorporation,  1998<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Biomass%20for%20Renewable%20Energy,%20Fuels,%20and%20Chemicals%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201998%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/67_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit68">I. C. Macedo, J. E. A. Seabra and J. E. A. R. Silvac, <span class="italic">Biomass Bioenergy</span>, 2008, <span class="bold">32</span>, 582595<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.biombioe.2007.12.006" title="DOI Link to resource 10.1016/j.biombioe.2007.12.006">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXnt1OhtLw%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/68_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit69">H. S. Kheshgi, R. C. Prince and G. Marland, <span class="italic">Annu. Rev. Energy Environ.</span>, 2000, <span class="bold">25</span>, 199244<a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev.energy.25.1.199" title="DOI Link to resource 10.1146/annurev.energy.25.1.199">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/69_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit70">F. G. Acin Fernndez, F. Garcia Camacho, J. A. Sanchez Perez, J. M. Fernandez Sevilla and E. Molina-Grima, <span class="italic">Biotechnol. Bioeng.</span>, 1998, <span class="bold">58</span>, 605616<a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1097-0290(19980620)58:6&lt;605::AID-BIT6&gt;3.0.CO;2-M" title="DOI Link to resource 10.1002/(SICI)1097-0290(19980620)58:6&lt;605::AID-BIT6&gt;3.0.CO;2-M">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXisFeksL8%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/70_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit71">N. R. Moheimani and M. A. Borowitzka, <span class="italic">J. Appl. Phycol.</span>, 2006, <span class="bold">18</span>, 703712<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2018%5Bvolume%5D%20AND%20703%5Bpage%5D%20and%202006%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/71_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit72">
          P. M. Pedroni, G. Lamenti, G. Prosperi, L. Ritorto, G. Scolla, F. Capuano, M. Valdiserri, in <span class="italic">Greenhouse Gas Control Technologies 7. Proceedings of the 7th International Conference on Greenhouse Gas Control Technologies 5 September 2004</span>, Vancouver, Canada, ed. E. S. Rubin, Keith, D. W., Gilboy, C. F., Wilson, T., Morris, T., Gale, J.Thambimuthu, K., Elsevier Science Ltd, Oxford,  2005, pp. 10371042.<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Greenhouse%20Gas%20Control%20Technologies%207.%20Proceedings%20of%20the%207th%20International%20Conference%20on%20Greenhouse%20Gas%20Control%20Technologies%205%20September%202004%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202005%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/72_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit73">N. R. Moheimani and M. A. Borowitzka, <span class="italic">Biotechnol. Bioeng.</span>, 2007, <span class="bold">96</span>, 2736<a target="_blank" class="DOILink" href="https://doi.org/10.1002/bit.21169" title="DOI Link to resource 10.1002/bit.21169">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXjsV2mtQ%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/73_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit74">
          W. J. Oswald, in <span class="italic">Micro-Algal Biotechnology</span>, ed. M. A. Borowitzka and L. J. Borowitzka, Cambridge University Press, Cambridge,  1988, pp. 357394<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Micro-Algal%20Biotechnology%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201988%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/74_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit75">M. R. Tredici and G. C. Zittelli, <span class="italic">Biotechnol. Bioeng.</span>, 1998, <span class="bold">57</span>, 187197<a target="_blank" class="DOILink" href="https://doi.org/10.1002/(SICI)1097-0290(19980120)57:2&lt;187::AID-BIT7&gt;3.0.CO;2-J" title="DOI Link to resource 10.1002/(SICI)1097-0290(19980120)57:2&lt;187::AID-BIT7&gt;3.0.CO;2-J">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXntFyn" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/75_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit76">G. C. Zittelli, V. Tomasello, E. Pinzani and M. R. Tredici, <span class="italic">J. Appl. Phycol.</span>, 1996, <span class="bold">8</span>, 293301<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%208%5Bvolume%5D%20AND%20293%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/76_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit77">R. Emerson and W. Arnold, <span class="italic">J. Gen. Physiol.</span>, 1932, <span class="bold">15</span>, 391420<a target="_blank" class="DOILink" href="https://doi.org/10.1085/jgp.15.4.391" title="DOI Link to resource 10.1085/jgp.15.4.391">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaA38Xjs1OitA%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/77_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit78">R. Radmer and B. Kok, <span class="italic">BioScience</span>, 1977, <span class="bold">27</span>, 599605<a target="_blank" class="DOILink" href="https://doi.org/10.2307/1297655" title="DOI Link to resource 10.2307/1297655">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/78_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit79">
          B. Kok, in <span class="italic">Algal Culture from Laboratory to Pilot Plant</span>, ed. J. S. Burlew, Carnegie Institution of Washington Publication No. 600, Washington D.C,  1953, pp. 6384<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Algal%20Culture%20from%20Laboratory%20to%20Pilot%20Plant%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201953%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/79_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit80">J. U. Grobbelaar, L. Nedbal and V. Tichy, <span class="italic">J. Appl. Phycol.</span>, 1996, <span class="bold">8</span>, 335343<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%208%5Bvolume%5D%20AND%20335%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/80_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit81">E. Molina Grima, F. G. Acien Fernandez, F. Garcia Camacho, F. Camacho Rubio and Y. Chisti, <span class="italic">J. Appl. Phycol.</span>, 2000, <span class="bold">12</span>, 365367<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2012%5Bvolume%5D%20AND%20365%5Bpage%5D%20and%202000%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/81_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit82">E. A. Laws, K. L. Terry, J. Wickman and M. S. Chalup, <span class="italic">Biotechnol. Bioeng.</span>, 1983, <span class="bold">25</span>, 23192335<a target="_blank" class="DOILink" href="https://doi.org/10.1002/bit.260251004" title="DOI Link to resource 10.1002/bit.260251004">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BD1cznvFeisQ%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/82_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit83">E. A. Laws, S. Taguchi, J. Hirata and L. Pang, <span class="italic">Biotechnol. Bioeng.</span>, 1986, <span class="bold">28</span>, 191197<a target="_blank" class="DOILink" href="https://doi.org/10.1002/bit.260280207" title="DOI Link to resource 10.1002/bit.260280207">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BD1czos12htg%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/83_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit84">J. C. Merchuk and X. Wu, <span class="italic">J. Appl. Phycol.</span>, 2003, <span class="bold">15</span>, 163169<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2015%5Bvolume%5D%20AND%20163%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/84_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit85">A. Melis, J. Neidhardt and J. R. Benemann, <span class="italic">J. Appl. Phycol.</span>, 1999, <span class="bold">10</span>, 515529<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2010%5Bvolume%5D%20AND%20515%5Bpage%5D%20and%201999%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/85_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit86">J. H. Mussgnug, S. T. Hall, J. Rupprecht, A. Foo, V. Klassen, A. McDowall, P. Schenk, O. Kruse and B. Hankamer, <span class="italic">J. Plant Technol.</span>, 2007, <span class="bold">5</span>, 802814<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Plant%20Technol.%5BJour%5D%20AND%205%5Bvolume%5D%20AND%20802%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/86_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit87">Y. Nakajima and R. Ueda, <span class="italic">J. Appl. Phycol.</span>, 1997, <span class="bold">9</span>, 503510<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%209%5Bvolume%5D%20AND%20503%5Bpage%5D%20and%201997%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/87_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit88">J. E. W. Polle, J. R. Benemann, A. Tankaka and A. Melis, <span class="italic">Planta</span>, 2000, <span class="bold">211</span>, 335344<a target="_blank" class="DOILink" href="https://doi.org/10.1007/s004250000279" title="DOI Link to resource 10.1007/s004250000279">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXlsVWqs7g%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/88_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit89">J. E. W. Polle, S. Kanakagiri, E. Jin, T. Masuda and A. Melis, <span class="italic">Int. J. Hydrogen Energy</span>, 2002, <span class="bold">27</span>, 12571264<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0360-3199(02)00116-7" title="DOI Link to resource 10.1016/S0360-3199(02)00116-7">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38Xnt1Kisbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/89_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit90">J. E. W. Polle, S. Kanakagiri and A. Melis, <span class="italic">Planta</span>, 2003, <span class="bold">217</span>, 4959<a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXjt1Omtr4%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/90_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit91">S. D. Tetal, M. Mitra and A. Melis, <span class="italic">Planta</span>, 2007, <span class="bold">225</span>, 813829<a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00425-006-0392-z" title="DOI Link to resource 10.1007/s00425-006-0392-z">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/91_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit92">M. Olaizola, <span class="italic">J. Appl. Phycol.</span>, 2000, <span class="bold">12</span>, 499506<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2012%5Bvolume%5D%20AND%20499%5Bpage%5D%20and%202000%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/92_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit93">
          J. R. Benemann, <span class="italic">Biofixation of CO<small><sub>2</sub></small> and Greenhouse Gas Abatement with Microalgae  Technology Roadmap</span>, National Energy Technology Laboratory,  2003<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Biofixation%20of%20CO2%20and%20Greenhouse%20Gas%20Abatement%20with%20Microalgae%20%20Technology%20Roadmap%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/93_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit94">
          E. Molina Grima, F. G. Acien Fernandez and A. Rogbies Medina, in <span class="italic">Handbook of Microalgal Culture</span>, ed. A. Richmond, Blackwells,  2004, pp. 215251<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Handbook%20of%20Microalgal%20Culture%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/94_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit95">P. T. Pienkos and A. Darzins, <span class="italic">Biofuels, Bioprod. Biorefin.</span>, 2009, <span class="bold">3</span>, 431440<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Biofuels,%20Bioprod.%20Biorefin.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%20431%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/95_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit96">F. H. Mohn, <span class="italic">Arch. Hydrobiol.</span>, 1978, <span class="bold">1</span>, 228253<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Arch.%20Hydrobiol.%5BJour%5D%20AND%201%5Bvolume%5D%20AND%20228%5Bpage%5D%20and%201978%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/96_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit97">
          F. H. Mohn, in <span class="italic">Algal Biomass</span>, ed. G. Shelef, Soeder, C. J., Elsevier, Amsterdam,  1980, pp. 547471<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Algal%20Biomass%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201980%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/97_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit98">
          L. Svarovsky, in <span class="italic">Solid-Liquid Separation</span>, ed. L. Svarovsky, Butterworth-Heinemann, Oxford,  2001, pp. 129<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Solid-Liquid%20Separation%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202001%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/98_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit99">L. M. Lubian, <span class="italic">Aquacult. Eng.</span>, 1989, <span class="bold">8</span>, 257265<a target="_blank" class="DOILink" href="https://doi.org/10.1016/0144-8609(89)90013-7" title="DOI Link to resource 10.1016/0144-8609(89)90013-7">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/99_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit100">J. Morales, J. Delanoue and G. Picard, <span class="italic">Aquacult. Eng.</span>, 1985, <span class="bold">4</span>, 257270<a target="_blank" class="DOILink" href="https://doi.org/10.1016/0144-8609(85)90018-4" title="DOI Link to resource 10.1016/0144-8609(85)90018-4">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/100_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit101">N. Levy, S. Magdassi and Y. Baror, <span class="italic">Water Res.</span>, 1992, <span class="bold">26</span>, 249254<a target="_blank" class="DOILink" href="https://doi.org/10.1016/0043-1354(92)90225-S" title="DOI Link to resource 10.1016/0043-1354(92)90225-S">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK38XhsFymsrk%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/101_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit102">H. Salehizadeh and S. A. Shojaosadati, <span class="italic">Biotechnol. Adv.</span>, 2001, <span class="bold">19</span>, 371385<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0734-9750(01)00071-4" title="DOI Link to resource 10.1016/S0734-9750(01)00071-4">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXms1OksL8%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/102_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit103">
          N. R. Moheimani, Ph D Thesis, Murdoch University, Australia,  2005<a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/103_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit104">S. J. Lee, B. D. Yoon and H. M. Oh, <span class="italic">Biotechnol. Tech.</span>, 1998, <span class="bold">12</span>, 553556<a target="_blank" class="DOILink" href="https://doi.org/10.1023/A:1008811716448" title="DOI Link to resource 10.1023/A:1008811716448">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXltl2nurk%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/104_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit105">M. M. Mendes-Pinto, M. F. J. Raposo, J. Bowen, A. J. Young and R. Morais, <span class="italic">J. Appl. Phycol.</span>, 2001, <span class="bold">13</span>, 1924<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2013%5Bvolume%5D%20AND%2019%5Bpage%5D%20and%202001%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/105_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit106">G. Ahlgren and L. Merino, <span class="italic">Arch. Hydrobiol.</span>, 1991, <span class="bold">121</span>, 295306<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Arch.%20Hydrobiol.%5BJour%5D%20AND%20121%5Bvolume%5D%20AND%20295%5Bpage%5D%20and%201991%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/106_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit107">M. L. Eltgroth, R. L. Watwood and G. V. Wolfe, <span class="italic">J. Phycol.</span>, 2005, <span class="bold">41</span>, 10001009<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1529-8817.2005.00128.x" title="DOI Link to resource 10.1111/j.1529-8817.2005.00128.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2nsb3O" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/107_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit108">J. Darbyshire, <span class="italic">Top. Enzyme Ferment. Biotechnol.</span>, 1981, <span class="bold">5</span>, 147186<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Top.%20Enzyme%20Ferment.%20Biotechnol.%5BJour%5D%20AND%205%5Bvolume%5D%20AND%20147%5Bpage%5D%20and%201981%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/108_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit109">J. Fleurence, <span class="italic">J. Appl. Phycol.</span>, 1999, <span class="bold">11</span>, 313314<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%2011%5Bvolume%5D%20AND%20313%5Bpage%5D%20and%201999%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/109_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit110">G. E. Napolitano, <span class="italic">J. Phycol.</span>, 1994, <span class="bold">30</span>, 943950<a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.0022-3646.1994.00943.x" title="DOI Link to resource 10.1111/j.0022-3646.1994.00943.x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXkt1Chs7s%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/110_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit111">A. Rosenthal, D. L. Pyle and K. Niranjan, <span class="italic">Enzyme Microb. Technol.</span>, 1996, <span class="bold">19</span>, 402420<a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0141-0229(96)80004-F" title="DOI Link to resource 10.1016/S0141-0229(96)80004-F">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28XmtVynt7s%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/111_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit112">K. H. Wiltshire, M. Boersma, A. Moeller and H. Buhtz, <span class="italic">Aquat. Ecol.</span>, 2000, <span class="bold">34</span>, 119126<a target="_blank" class="DOILink" href="https://doi.org/10.1023/A:1009911418606" title="DOI Link to resource 10.1023/A:1009911418606">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXmvVCqsrg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/112_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit113">D. S. Domozych, K. D. Stewart and K. R. Mattox, <span class="italic">J. Mol. Evol.</span>, 1980, <span class="bold">15</span>, 112<a target="_blank" class="DOILink" href="https://doi.org/10.1007/BF01732578" title="DOI Link to resource 10.1007/BF01732578">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL3cXhvFCktb0%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/113_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit114">T. Ogawa and G. Terui, <span class="italic">J. Ferment. Technol.</span>, 1970, <span class="bold">48</span>, 361367<a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/114_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit115">B. Smith, H. C. Greenwell and A. Whiting, <span class="italic">Energy Environ. Sci.</span>, 2009, <span class="bold">2</span>, 262271<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=b814123a&amp;newsite=1" title="Link to RSC resource DOI:10.1039/b814123a">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/115_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit116">
          J. C. Weissman and R. P. Goebel, <span class="italic">Design and Analysis of Microalgal Open Pond Systems for the Purpose of Producing Fuels</span>, Solar Energy Research Institution, Golden,  1987<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Design%20and%20Analysis%20of%20Microalgal%20Open%20Pond%20Systems%20for%20the%20Purpose%20of%20Producing%20Fuels%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201987%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/116_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit117">M. A. Borowitzka, <span class="italic">J. Appl. Phycol.</span>, 1992, <span class="bold">4</span>, 267279<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Appl.%20Phycol.%5BJour%5D%20AND%204%5Bvolume%5D%20AND%20267%5Bpage%5D%20and%201992%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/117_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit118">
          C. B. Raleigh, M. E. Huntley, <span class="italic">Subtask 3B-2/3: Feedstock Processing and Microbial Biomass Assessment</span>. DE-FC3604GO14248, Hawaii Natural Energy Institute,  2008<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Subtask%203B-2/3:%20Feedstock%20Processing%20and%20Microbial%20Biomass%20Assessment%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/118_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit119">
          A. O. Alabi, M. Tampier, E. Bibeau, <span class="italic">Microalgae Technologies and Processes for Biofuels/Bioenergy Production in British Columbia: Current Technology, Suitability and Barriers to Implementation: Final Report</span>, British Columbia Innovation Council, Victoria, British Columbia,  2009<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Microalgae%20Technologies%20and%20Processes%20for%20Biofuels/Bioenergy%20Production%20in%20British%20Columbia:%20Current%20Technology,%20Suitability%20and%20Barriers%20to%20Implementation:%20Final%20Report%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/119_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit120">A. P. Singh, C. A. Avramis, J. K. G. Kramerand and A. G. Marangoni, <span class="italic">J. Dairy Res.</span>, 2004, <span class="bold">71</span>, 6673<a target="_blank" class="DOILink" href="https://doi.org/10.1017/S0022029903006642" title="DOI Link to resource 10.1017/S0022029903006642">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXhvFWjt7g%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/120_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit121">S. Paltsev, J. M. Reilly, H. D. Jacoby, A. C. Gurgel, G. E. Metcalf, A. P. Sokolov and J. F. Hola, <span class="italic">Clim. Policy</span>, 2008, <span class="bold">8</span>, 395420<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Clim.%20Policy%5BJour%5D%20AND%208%5Bvolume%5D%20AND%20395%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/121_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit122">
          T. van Harmelen, H. Oonk, <span class="italic">Microalgae Biofixation Processes: Application and Potential Contributions to Greenhouse Gas Mitigation</span>, TNO, Appeldoorn,  2006<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Microalgae%20Biofixation%20Processes:%20Application%20and%20Potential%20Contributions%20to%20Greenhouse%20Gas%20Mitigation%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202006%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/122_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit123">
          A. S. Carlsson, J. B. Van Beilen, R. Moeller, D. Clayton, <span class="italic">Micro-and Macro Algae: Utility for Industrial Applications</span>, CPL Press, Newbury,  2007<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Micro-and%20Macro%20Algae:%20Utility%20for%20Industrial%20Applications%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/123_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit124">K. J. Flynn, <span class="italic">J. Plankton Res.</span>, 2001, <span class="bold">23</span>, 977997<a target="_blank" class="DOILink" href="https://doi.org/10.1093/plankt/23.9.977" title="DOI Link to resource 10.1093/plankt/23.9.977">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/124_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit125">J. Schwender, <span class="italic">Curr. Opin. Biotechnol.</span>, 2008, <span class="bold">19</span>, 131137<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.copbio.2008.02.006" title="DOI Link to resource 10.1016/j.copbio.2008.02.006">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXlslKis7k%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/125_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit126">D. Cordell, J.-O. Dangert and S. White, <span class="italic">Global Environ. Change</span>, 2009, <span class="bold">19</span>, 292305<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Global%20Environ.%20Change%5BJour%5D%20AND%2019%5Bvolume%5D%20AND%20292%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/126_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit127">H. U. Sverdrup, <span class="italic">J. Conseil Int. Exploration Mer</span>, 1953, <span class="bold">18</span>, 287295<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Conseil%20Int.%20Exploration%20Mer%5BJour%5D%20AND%2018%5Bvolume%5D%20AND%20287%5Bpage%5D%20and%201953%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/127_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit128">T. R. Parsons and M. Takahashi, <span class="italic">Limnol. Oceanogr.</span>, 1973, <span class="bold">18</span>, 511515<a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/128_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit129">
          E. Gallagher, <span class="italic">The Gallagher Review of the Indirect Effects of Biofuels Production</span>,  2008, Renewable Fuels Agency<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=The%20Gallagher%20Review%20of%20the%20Indirect%20Effects%20of%20Biofuels%20Production%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/129_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit130">
          M. A. Borowitzka in <span class="italic">Micro-Algal Biotechnology</span>, ed. M. A. Borowitzka and L. J. Borowitzka, Cambridge University Press, Cambridge,  1988, pp. 257287<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Micro-Algal%20Biotechnology%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201988%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/130_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit131">L. Rodolfi, G. C. Graziella, N. Bassi, G. Padovani, N. Bionini, G. Bionini and M. R. Tredici, <span class="italic">Biotechnol. Bioeng.</span>, 2009, <span class="bold">102</span>, 100112<a target="_blank" class="DOILink" href="https://doi.org/10.1002/bit.22033" title="DOI Link to resource 10.1002/bit.22033">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXitlSk" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50003858/b924978h/131_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50003858"/></a>.</span></li>
      <li><span id="cit132">
          www.fao.org/es/esc/prices
        .</span></li>
    </ol>
    
    
    
    
    
  <hr/><table><tr><td><h3>Footnotes</h3></td></tr><tr><td><span class="sup_ref"> <span id="fn1">Electronic supplementary information (ESI) available: Appendices IIV. See DOI: <a href="http://xlink.rsc.org/?DOI=b924978h" target="_blank" title="Select to resolve DOI: 10.1039/b924978h">10.1039/b924978h</a></span></span></td></tr><tr><td><span class="sup_ref"> <span id="fn2">Current address: National Bioenergy Center, National Renewable Energy Laboratory, 1617 Cole Blvd, Golden, CO 80401, USA</span></span></td></tr><tr><td><span class="sup_ref"> <span id="fn3">Strictly the term alga should be limited to the eukaryotic phototrophs, since cyanobacteria (formerly known as blue-green algae) are a form of bacteria, with very different genetics and evolutionary history. It is however very cumbersome to qualify every reference to these two <span class="CH"><a class="simple" title="ChEBI Ontology link for: groups" href="javascript:popupOBO('CHEBI:24433','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=24433')">groups</a></span> of single-celled phototrophs as algae and cyanobacteria, or as micro-photoautotrophs so, unless there is need to be specific, for convenience the two are referred in the text simply as algae.</span></span></td></tr><tr><td><span class="sup_ref"> <span id="fn4">The notation [CH<small><sub>2</sub></small>O] is commonly used as shorthand for organic material in biology  in the present context its use is restricted to organic material with the same elemental ratio as monosaccharides.</span></span></td></tr><tr><td><span class="sup_ref">|| <span id="fn5">An einstein is defined as a mole of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span>, of unprescribed wavelength.</span></span></td></tr><tr><td><span class="sup_ref">** <span id="fn6">It has been held that Nature never evolved a wheel  it however did a billion or more years ago and on the nanoscale! Only recently have we developed the skills to observe these molecular mechanisms.</span></span></td></tr><tr><td><span class="sup_ref"> <span id="fn7">Depending upon the circumstances the <span class="italic">G</span> for the <span class="CH"><a class="simple" title="Molecular Process Ontology link for: hydrolysis" href="javascript:popupOBO('MOP:0000619','b924978h')">hydrolysis</a></span> of <span class="TC"><a class="simple" title="Chemspider Compound link for:ATP" target="_blank" href="http://www.chemspider.com/Chemical-Structure.5742.html">ATP</a></span> varies from 45 to 55 kJ mol<small><sup>1</sup></small>. A mean figure of 50 kJ mol<small><sup>1</sup></small> is used in the present account.</span></span></td></tr><tr><td><span class="sup_ref"> <span id="fn8">There have been two schools of thought over the number of <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> required per molecule of O<small><sub>2</sub></small> split: either 36 <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> or 810 <span class="CH"><a class="simple" title="ChEBI Ontology link for: photons" href="javascript:popupOBO('CHEBI:30212','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=30212')">photons</a></span> (see <a title="Select to navigate to references" href="#cit56">ref. 56</a>). Present consensus favours the latter.</span></span></td></tr><tr><td><span class="sup_ref"> <span id="fn9">Bomb <span class="CH"><a class="simple" title="Chemical Methods Ontology link for: calorimetry" href="javascript:popupOBO('CMO:0000682','b924978h')">calorimetry</a></span> involves the conversion the <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> in the molecule to dinitrogen, whereas the starting and end <span class="CH"><a class="simple" title="ChEBI Ontology link for: nitrogen" href="javascript:popupOBO('CHEBI:25555','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=25555')">nitrogen</a></span> compound in microalgal <span class="AN"><a class="simple" title="Gene Ontology link for: growth" href="javascript:popupOBO('GO:0040007','b924978h')">growth</a></span> is commonly <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span>. Strictly a correction should be made to the reported bomb calorimitry values for the heat of formation of <span class="TC"><a class="simple" title="Chemspider Compound link for:ammonia" target="_blank" href="http://www.chemspider.com/Chemical-Structure.217.html">ammonia</a></span> from dinitrogen (46 kJ mol<small><sup>1</sup></small>) otherwise the <span class="italic">G</span> of the reaction will be overestimated. The error is small, about 0.4 kJ g<small><sup>1</sup></small> in the case of <span class="CH"><a class="simple" title="ChEBI Ontology link for: protein" href="javascript:popupOBO('CHEBI:36080','b924978h','http://www.ebi.ac.uk/chebi/searchId.do?chebiId=36080')">protein</a></span> and 0.2 or less in the case of algal biomass, and is customarily ignored.</span></span></td></tr><tr><td><span class="sup_ref"> <span id="fn10">This is calculated as the transverse timescale = <span class="italic">h</span><small><sup>3</sup></small>/(0.06<span class="italic">k</span><small><sub><span class="italic">z</span></sub></small>) = <span class="italic">h</span><small><sup>3</sup></small>/(0.06<span class="italic">u</span>*<span class="italic">xh</span>) = <span class="italic">h</span><small><sup>2</sup></small>/(0.06  0.05  <span class="italic">u</span>)  30<span class="italic">h</span><small><sup>2</sup></small>/<span class="italic">u</span>; where <span class="italic">h</span> is the depth and <span class="italic">u</span> is the velocity, all in SI units.</span></span></td></tr><tr><td><span class="sup_ref">|||| <span id="fn11">Values for Europe, Asia and USA from ICIS pricing and for Europe from Kingsman (http://www.kingsman.com).</span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is  The Royal Society of Chemistry 2010</b></td></tr></table></div></div></div></body><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>