<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Deep learning for molecular design—a review of the state of the art  - Molecular Systems Design &amp; Engineering (RSC Publishing) DOI:10.1039/C9ME00039A</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2019/me/c9me00039a"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":69,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Daniel C. Elton"/><meta name="DC.Creator" content="Zois Boukouvalas"/><meta name="DC.Creator" content="Mark D. Fuge"/><meta name="DC.Creator" content="Peter W. Chung"/><meta name="DC.title" content="Deep learning for molecular design—a review of the state of the art "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2019/08/05"/><meta name="DC.Identifier" scheme="doi" content="10.1039/C9ME00039A"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Deep learning for molecular design—a review of the state of the art "/><meta name="citation_author" content="Daniel C. Elton"/><meta name="citation_author" content="Zois Boukouvalas"/><meta name="citation_author" content="Mark D. Fuge"/><meta name="citation_author" content="Peter W. Chung"/><meta name="citation_online_date" content="2019/05/22"/><meta name="citation_date" content="2019"/><meta name="citation_journal_title" content="Molecular Systems Design &amp; Engineering"/><meta name="citation_volume" content="4"/><meta name="citation_issue" content="4"/><meta name="citation_firstpage" content="828"/><meta name="citation_lastpage" content="849"/><meta name="citation_doi" content="10.1039/C9ME00039A"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2019/me/c9me00039a"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2019/me/c9me00039a"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2019/me/c9me00039a"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2019/me/c9me00039a">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:c8me00116b)" href="/en/content/articlehtml/2019/me/c8me00116b" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:c9me00050j)" href="/en/content/articlehtml/2019/me/c9me00050j" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div> </div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/C9ME00039A">10.1039/C9ME00039A</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2058-9689/2016">Mol. Syst. Des. Eng.</a></span>, 2019, <strong>4</strong>, 828-849</div><h1 id="sect239"><span class="title_heading">Deep learning for molecular design—a review of the state of the art</span></h1><p class="header_text">
      <span class="bold">
        
          
            Daniel C. 
            Elton<a title="Select to navigate to footnote" href="#fn1">†</a>
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Daniel C. Elton (orcid.org/0000-0003-0249-1387) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-0249-1387"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Zois 
            Boukouvalas
          
          
        
      </span><sup><span class="sup_ref italic">ab</span></sup>, 
      <span class="bold">
        
          
            Mark D. 
            Fuge
          
          
        
      </span><sup><span class="sup_ref italic">a</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Peter W. 
            Chung
          
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Department of Mechanical Engineering, University of Maryland, College Park, Maryland, 20740, USA. E-mail: <a href="mailto:daniel.elton@nih.gov">daniel.elton@nih.gov</a></span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Department of Mathematics and Statistics, American University, Washington, D.C., 20016, USA</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      18th March 2019
    </span><span class="bold italic">, Accepted 22nd May 2019</span></div><p class="bold italic">First published on 22nd May 2019</p><hr/><div class="abstract"><h2>Abstract</h2><p>In the space of only a few years, deep generative modeling has revolutionized how we think of artificial creativity, yielding autonomous systems which produce original images, music, and text. Inspired by these successes, researchers are now applying deep generative modeling techniques to the generation and optimization of molecules—in our review we found 45 papers on the subject published in the past two years. These works point to a future where such systems will be used to generate lead molecules, greatly reducing resources spent downstream synthesizing and characterizing bad leads in the lab. In this review we survey the increasingly complex landscape of models and representation schemes that have been proposed. The four classes of techniques we describe are recursive neural networks, autoencoders, generative adversarial networks, and reinforcement learning. After first discussing some of the mathematical fundamentals of each technique, we draw high level connections and comparisons with other techniques and expose the pros and cons of each. Several important high level themes emerge as a result of this work, including the shift away from the SMILES string representation of molecules towards more sophisticated representations such as graph grammars and 3D representations, the importance of reward function design, the need for better standards for benchmarking and testing, and the benefits of adversarial training and reinforcement learning over maximum likelihood based training.</p></div><hr/><table><tr><td class="biogPlate"><a href="/image/article/2019/ME/c9me00039a/c9me00039a-p1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-p1.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-p1.gif"/></a><p><span class="graphic_title">Daniel C. Elton</span></p></td><td><i><p>Dr. Daniel C. Elton received a B.S. degree in physics from Rensselaer Polytechnic Institute in 2009 and a Ph.D. in physics from Stony Brook University in 2016. He worked as a Postdoctoral Research Associate and later an Assistant Research Scientist at the University of Maryland, College Park between 2017–2019 where he focused on applications of deep learning to molecular design and discovery. In January 2019 he moved to work as a contractor Staff Scientist at the National Institutes of Health. His current work focuses on applications of deep learning and AI to detection and segmentation in medical images.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2019/ME/c9me00039a/c9me00039a-p2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-p2.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-p2.gif"/></a><p><span class="graphic_title">Zois Boukouvalas</span></p></td><td><i><p>Dr. Zois Boukouvalas received his B.S. degree in Mathematics from the University of Patras, Greece, an M.S. degree in Applied and Computational Mathematics from the Rochester Institute of Technology, and a Ph.D. degree in Applied Mathematics from University of Maryland Baltimore County in 2017. Since 2017 he has worked as a Postdoctoral Research Associate at the University of Maryland, College Park, and in August 2019 he will start as an Assistant Professor in the Department of Mathematics and Statistics at American University. His research interests include blind source separation and machine learning for big data problems.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2019/ME/c9me00039a/c9me00039a-p3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-p3.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-p3.gif"/></a><p><span class="graphic_title">Mark D. Fuge</span></p></td><td><i><p>Dr. Mark D. Fuge is an Assistant Professor of Mechanical Engineering at the University of Maryland, College Park. His staff and students study fundamental scientific and mathematical questions behind how humans and computers can work together to design better complex engineered systems, from the molecular scale to systems as large as aircraft and ships, by using tools from Applied Mathematics and Computer Science. He received his Ph.D. from UC Berkeley and has received a DARPA Young Faculty Award, a National Defense Science and Engineering Graduate Fellowship, and has prior/current support from NSF, NIH, DARPA, ONR, and Lockheed Martin.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2019/ME/c9me00039a/c9me00039a-p4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-p4.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-p4.gif"/></a><p><span class="graphic_title">Peter W. Chung</span></p></td><td><i><p>Dr. Peter W. Chung is an Associate Professor in the Department of Mechanical Engineering at the University of Maryland in College Park. He serves as the Division Lead of the Mechanics, Materials, and Manufacturing Division within the department and is also the Lead of the Energetics Group in the Center for Engineering Concepts Development.</p></i></td></tr></table><hr/>
    
      <br/><table class="box_border"><tr><td>
        
          <h2 id="sect311"><span class="a_heading">Design, System, Application</span></h2>
          <span>In this review we discuss the use of deep learning techniques to perform molecular generation and optimization, a new field which has proved to be a fertile area for development in the past three years. The broad categories of techniques which we discuss are recursive neural networks, variational autoencoders, generative adversarial networks, and reinforcement learning. These techniques can be used for generating a diverse set of leads for high throughput screening. The latent spaces of generative models can be used for optimization of molecules when coupled with a property predicting module. Alternatively, a pure reinforcement learning approach to optimization can be taken. The majority of work so far has been focused on drug design but there are numerous other application areas now appearing in the literature such as metal organic frameworks, organic LEDs, organic solar cells, and energetic materials. While currently genetic algorithms can often compete with deep learning based methods for molecular optimization, the field is rapidly developing and there are many avenues open for improvement. The current literature points to a future where deep generative modeling techniques will find utility not just for generating and optimizing molecules but also for materials and nanoscale systems.</span>
        
      </td></tr></table><br/>
    
    
      <span>The average cost to bring a new drug to market is now well over one billion USD,<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> with an average time from discovery to market of 13 years.<a title="Select to navigate to references" href="#cit2"><sup><span class="sup_ref">2</span></sup></a> Outside of pharmaceuticals the average time from discovery to commercial production can be even longer, for instance for energetic molecules it is 25 years.<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> A critical first step in molecular discovery is generating a pool of candidates for computational study or synthesis and characterization. This is a daunting task because the space of possible molecules is enormous—the number of potential drug-like compounds has been estimated to be between 10<small><sup>23</sup></small> and 10<small><sup>60</sup></small>,<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a> while the number of all compounds that have been synthesized is on the order of 10<small><sup>8</sup></small>. Heuristics, such as Lipinski's “rule of five” for pharmaceuticals<a title="Select to navigate to references" href="#cit5"><sup><span class="sup_ref">5</span></sup></a> can help narrow the space of possibilities, but the task remains daunting. High throughput screening (HTS)<a title="Select to navigate to references" href="#cit6"><sup><span class="sup_ref">6</span></sup></a> and high throughput virtual screening (HTVS)<a title="Select to navigate to references" href="#cit7"><sup><span class="sup_ref">7</span></sup></a> techniques have made larger parts of chemical space accessible to computational and experimental study. Machine learning has been shown to be capable of yielding rapid and accurate property predictions for many properties of interest and is being integrated into screening pipelines, since it is orders of magnitude faster than traditional computational chemistry methods.<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> Techniques for the interpretation and “inversion” of a machine learning model can illuminate structure–property relations that have been learned by the model which can in turn be used to guide the design of new lead molecules.<a title="Select to navigate to reference" href="#cit9"><sup><span class="sup_ref">9,10</span></sup></a> However even with these new techniques bad leads still waste limited supercomputer and laboratory resources, so minimizing the number of bad leads generated at the start of the pipeline remains a key priority. The focus of this review is on the use of deep learning techniques for the targeted generation of molecules and guided exploration of chemical space. We note that machine learning (and more broadly artificial intelligence) is having an impact on accelerating other parts of the chemical discovery pipeline as well, <span class="italic">via</span> machine learning accelerated <span class="italic">ab initio</span> simulation,<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> machine learning based reaction prediction,<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11,12</span></sup></a> deep learning based synthesis planning,<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a> and the development of high-throughput “self-driving” robotic laboratories.<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">14,15</span></sup></a></span>
      <p class="otherpara">Deep neural networks, which are often defined as networks with more than three layers, have been around for many decades but until recently were difficult to train and fell behind other techniques for classification and regression. By most accounts, the deep learning revolution in machine learning began in 2012, when deep neural network based models began to win several different competitions for the first time. First came a demonstration by Cireşan <span class="italic">et al.</span> of how deep neural networks could achieve near-human performance on the task of handwritten digit classification.<a title="Select to navigate to references" href="#cit16"><sup><span class="sup_ref">16</span></sup></a> Next came groundbreaking work by Krizhevsky <span class="italic">et al.</span> which showed how deep convolutional networks achieved superior performance on the 2010 ImageNet image classification challenge.<a title="Select to navigate to references" href="#cit17"><sup><span class="sup_ref">17</span></sup></a> Finally, around the same time in 2012, a multitask neural network developed by Dahl <span class="italic">et al.</span> won the “Merck Molecular Activity Challenge” to predict the molecular activities of molecules at 15 different sites in the body, beating out more traditional machine learning approaches such as boosted decision trees.<a title="Select to navigate to references" href="#cit18"><sup><span class="sup_ref">18</span></sup></a> One of the key technical advances published that year and used by both Krizhevsky <span class="italic">et al.</span> and Dahl <span class="italic">et al.</span> was a novel regularization trick called “dropout”.<a title="Select to navigate to reference" href="#cit18"><sup><span class="sup_ref">18,19</span></sup></a> Another important technical advance was the efficient implementation of neural network training on graphics processing units (GPUs). By 2015 better hardware, deeper networks, and a variety of further technical advances had reduced error rates on the ImageNet challenge by a factor of 3 compared to the Krizhevsky's 2012 result.<a title="Select to navigate to references" href="#cit20"><sup><span class="sup_ref">20</span></sup></a></p>
      <p class="otherpara">In addition to the tasks of classification and regression, deep neural networks began to be used for generation of images, audio, and text, giving birth to the field of “deep generative modeling”. Two key technical advances in deep generative modeling were the variational autoencoder (Kingma <span class="italic">et al.</span>, 2013 (<a title="Select to navigate to references" href="#cit21">ref. 21</a>)) and generative adversarial networks (Goodfellow <span class="italic">et al.</span> 2014 (<a title="Select to navigate to references" href="#cit22">ref. 22</a>)). The first work demonstrating deep generative modeling of molecules was the “molecular autoencoder” work of Gómez-Bombarelli <span class="italic">et al.</span> which appeared on the arXiv in October 2016 and was published in <span class="italic">ACS Central Science</span> in 2018.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> Since then, there has been an explosion of advancements in deep generative modeling of molecules using several different deep learning architectures and many variations thereof, as shown in <a title="Select to navigate to table" href="#tab2">Table 2</a>. In addition to new architectures, new representation schemes, many of which are graph based, have been introduced as alternatives to the SMILES representation used by Gómez-Bombarelli <span class="italic">et al.</span> The growing complexity of the landscape of architectures and representations and the lack of agreement upon standards for benchmarking and comparing different approaches has prompted us to write this review.</p>
      <p class="otherpara">While much of the work so far has focused on deep generative modeling for drug molecules,<a title="Select to navigate to references" href="#cit24"><sup><span class="sup_ref">24</span></sup></a> there are many other application domains which are benefiting from the application of deep learning to lead generation and screening, such as organic light emitting diodes,<a title="Select to navigate to references" href="#cit25"><sup><span class="sup_ref">25</span></sup></a> organic solar cells,<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a> energetic materials,<a title="Select to navigate to reference" href="#cit10"><sup><span class="sup_ref">10,27</span></sup></a> electrochromic devices,<a title="Select to navigate to references" href="#cit28"><sup><span class="sup_ref">28</span></sup></a> polymers,<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> polypeptides,<a title="Select to navigate to reference" href="#cit30"><sup><span class="sup_ref">30–32</span></sup></a> and metal organic frameworks.<a title="Select to navigate to reference" href="#cit33"><sup><span class="sup_ref">33,34</span></sup></a></p>
      <p class="otherpara">Our review touches on four major issues we have observed in the field. The first is the importance and opportunities for improvement by using different molecular representations. Recent efforts have begun to depart from the use of simplified molecular-input line-entry system (SMILES) strings towards representations that are “closer to the chemical structure” and offer improved chemical accuracy, such as graph grammar based methods. The second issue is architecture selection. We discuss the pros and cons underlying different choices of model architecture and present some of their key mathematical details to better illuminate how different approaches relate to each other. This leads us to highlight the advantages of adversarial training and reinforcement learning over maximum likelihood based training. We also touch on techniques for molecular optimization using generative models, which has grown in popularity recently. The third major issue is the approaches for quantitatively evaluating different approaches for molecular generation and optimization. Fourth, and finally, we discuss is reward function design, which is crucial for the practical application of methods which use reinforcement learning. We contribute by offering novel overview of how to engineer reward function to generate a set of leads which is chemically stable, diverse, novel, has good properties, and is synthesizable.</p>
      <p class="otherpara">There are reasons to be skeptical about whether today's deep generative models can outperform traditional computational approaches to lead generation and optimization. Traditional approaches are fundamentally combinatorial in nature and involve mixing scaffolds, functional groups, and fragments known to be relevant to the problem at hand (for a review, see Pirard <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a>). A naive combinatorial approach to molecular generation leads to most molecules being unstable or impossible to synthesize, so details about chemical bonding generally must be incorporated. One approach is to have an algorithm perform virtual chemical reactions, either from a list of known reactions, or using <span class="italic">ab initio</span> methods for reaction prediction.<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> Another popular approach is to use genetic algorithms with custom transformation rules which are known to maintain chemical stability.<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> One of the latest genetic algorithm based approaches (“Grammatical Evolution”) can match the performance of the deep learning approaches for molecular optimization under some metrics.<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> Deep generative modeling of molecules has made rapid progress in just a few years and there are reasons to expect this progress to continue, not just with better hardware and data, but due to new architectures and approaches. For instance, generative adversarial networks and deep reinforcement learning (which may be combined or used separately) have both seen technical advancements recently.</p>
    
    
      
      <h2 id="sect372"><span class="a_heading">1 Molecular representation</span></h2>
      <span>The molecular representation refers to the digital encoding used for each molecule that serves as input for training the deep learning model. A representation scheme must capture essential structural information about each molecule. Creating an appropriate representation from a molecular structure is called featurization. Two important properties that are desirable (but not required) for representations are uniqueness and invertibility. Uniqueness means that each molecular structure is associated with a single representation. Invertibility means that each representation is associated with a single molecule (a one-to-one mapping). Most representations used for molecular generation are invertible, but many are non-unique. There are many reasons for non-uniqueness, including the representation not being invariant to the underlying physical symmetries of rotation, translation, and permutation of atomic indexes.</span>
      <p class="otherpara">Another factor one should consider when choosing a representation is the whether it is a character sequence or tensor. Some methods only work with sequences, while others only work with tensor. Sequences may be converted into tensors using one-hot encoding. Another choice is whether to use a representation based on the 3D coordinates of the molecule or a representation based on the 2D connectivity graph. Molecules are fundamentally three dimensional quantum mechanical objects, typically visualized as consisting of nuclei with well-defined positions surrounded by many electrons which are described by a complex-valued wavefunction. Fundamentally, all properties of a molecule can be predicted using quantum mechanics given only the relative coordinates of the nuclei and the type and ionization state of each atom. Equilibrium coordinates can be determined from the 2D graph <span class="italic">via</span> energy minimization. This point may explain the success of machine learning based property prediction at predicting high level properties from 2D graphs, as opposed to 3D structures.<a title="Select to navigate to reference" href="#cit8"><sup><span class="sup_ref">8,27,39</span></sup></a> In so much as properties are related to equilibrium structure, machine learning infers this from the 2D graph. In this section, we review both 3D and 2D representation schemes that have been developed recently (<a title="Select to navigate to table" href="#tab1">Table 1</a>).</p>
      
<div class="table_caption"><b>Table 1</b> <span id="tab1">Different representation schemes</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom">
                </th>
                <th align="left" valign="bottom" class="border_black">Method</th>
                <th align="left" valign="bottom" class="border_black">Unique?</th>
                <th align="left" valign="bottom" class="border_black">Invertible?</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td rowspan="3" valign="top">3D</td>
                <td valign="top">Raw voxels</td>
                <td valign="top">✗</td>
                <td valign="top">✓</td>
              </tr>
              <tr align="left">
                <td valign="top">Smoothed voxels</td>
                <td valign="top">✗</td>
                <td valign="top">✓</td>
              </tr>
              <tr align="left">
                <td valign="top">Tensor field networks</td>
                <td valign="top">✗</td>
                <td valign="top">✗</td>
              </tr>
              <tr align="left">
                <td rowspan="7" valign="top">2D graph</td>
                <td valign="top">SMILES</td>
                <td valign="top">✗</td>
                <td valign="top">✓</td>
              </tr>
              <tr align="left">
                <td valign="top">Canonical SMILES</td>
                <td valign="top">✓</td>
                <td valign="top">✓</td>
              </tr>
              <tr align="left">
                <td valign="top">InChI</td>
                <td valign="top">✓</td>
                <td valign="top">✓</td>
              </tr>
              <tr align="left">
                <td valign="top">MACCS keys</td>
                <td valign="top">✓</td>
                <td valign="top">✗</td>
              </tr>
              <tr align="left">
                <td valign="top">Tensors</td>
                <td valign="top">✗</td>
                <td valign="top">✓</td>
              </tr>
              <tr align="left">
                <td valign="top">Chemception images</td>
                <td valign="top">✓</td>
                <td valign="top">✓</td>
              </tr>
              <tr align="left">
                <td valign="top">Fingerprinting</td>
                <td valign="top">✓</td>
                <td valign="top">✗</td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
        
        <h3 id="sect437"><span class="b_heading">1.1 Representations of 3D geometry</span></h3>
        <span>Trying to implement machine learning directly with nuclear coordinates introduces a number of issues. The main issue is that coordinates are not invariant to molecular translation, rotation, and permutation of atomic indexing. While machine learning directly on coordinates is possible, it is much better to remove invariances to create a more compact representation (by removing degrees of freedom) and develop a scheme to obtain a unique representation for each molecule. One approach that uses 3Dcoordinates uses a 3D grid of voxels and specifies the nuclear charge contained within each voxel, thus creating a consistent representation. Nuclear charge (<span class="italic">i.e.</span> atom type) is typically specified by a one-hot vector of dimension equal to the number of atom types in the dataset. This scheme leads to a very high dimensional sparse representation, since the vast majority of voxels will not contain a nuclear charge. While sparse representations are considered desirable in some contexts, here sparsity leads to very large training datasets. This issue can be mitigated <span class="italic">via</span> spatial smoothing (blurring) by placing spherical Gaussians or a set of decaying concentric waves around each atomic nuclei.<a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a> Alternatively, the van der Waals radius may be used.<a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a> Amidi <span class="italic">et al.</span> use this type of approach for predictive modeling with 3D convolutional neural networks (CNNs),<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> while Kuzminykh <span class="italic">et al.</span> and Skalic <span class="italic">et al.</span> use this approach with a CNN-based autoencoder for generative modeling.<a title="Select to navigate to reference" href="#cit40"><sup><span class="sup_ref">40,41</span></sup></a></span>
        <p class="otherpara">Besides high dimensionality and sparsity, another issue with 3D voxelized representations is they do not capture invariances to translation, rotation, and reflection, which are hard for present-day deep learning based architectures to learn. Capturing such invariances is important for property prediction, since properties are invariant to such transformations. It is also important for creating compact representations of molecules for generative modeling. One way to deal with such issues is to always align the molecular structure along a principal axis as determined by principle component analysis to ensure a unique orientation.<a title="Select to navigate to reference" href="#cit40"><sup><span class="sup_ref">40,42</span></sup></a> Approaches which generate feature vectors from 3D coordinates that are invariant to translation and rotation are wavelet transform invariants,<a title="Select to navigate to references" href="#cit43"><sup><span class="sup_ref">43</span></sup></a> solid harmonic wavelet scattering transforms,<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> and tensor field networks.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> All of these methods incur a loss of information about 3D structure and are not easy to invert, so their utility for generative modeling may be limited (deep learning models learn to generate these representations, but if they cannot be unambiguously related to a 3D structure they are not very useful). Despite their issues with invertibility, tensor field networks have been suggested to have utility for generative modeling since it was shown they can accurately predict the location of missing atoms in molecules where one atom was removed.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> We expect future work on 3D may proceed in the direction of developing invertible representations that are based on the internal (relative) coordinates of the molecule.</p>
      
      
        
        <h3 id="sect456"><span class="b_heading">1.2 Representations of molecular graphs</span></h3>
        <div>
          
          <span id="sect459"/><span class="c_heading_indent">1.2.1 SMILES and string-based representations. </span>
          <span>Most generative modeling so far has not been done with coordinates but instead has worked with molecular graphs. A molecule can be considered as an undirected graph <img class="charmap" src="https://www.rsc.org/images/entities/char_e112.gif" alt="[capital G, script]"/> with a set of edges <img class="charmap" src="https://www.rsc.org/images/entities/char_e140.gif" alt="[scr E, script letter E]"/> and set of vertices <img class="charmap" src="https://www.rsc.org/images/entities/char_e149.gif" alt="[scr V, script letter V]"/>. The obvious disadvantage of such graphs is that information about bond lengths and 3D conformation is lost. For some properties one may wish to predict, the specific details of a molecule's 3D conformations may be important. For instance, when packing in a crystal or binding to a receptor, molecules will find the most energetically favorable conformation, and details of geometry often have a big effect. Despite this, graph representations have been remarkably successful for a variety of generative modeling and property prediction tasks. If a 3D structure is desired from a graph representation, molecular graphs can be embedded in 3D using distance geometry methods (for instance as implemented in the <span class="italic">OpenBabel</span> toolkit<a title="Select to navigate to reference" href="#cit46"><sup><span class="sup_ref">46,47</span></sup></a>). After coordinate embedding, the most energetically favorable conformation of the molecule can be obtained by doing energy minimization with classical force fields or quantum mechanical simulation.</span>
          <p class="otherpara">There are several ways to represent graphs for machine learning. The most popular way is the SMILES string representation.<a title="Select to navigate to references" href="#cit48"><sup><span class="sup_ref">48</span></sup></a> SMILES strings are a non-unique representation which encode the molecular graph into a sequence of ASCII characters using a depth-first graph traversal. SMILES are typically first converted into a one-hot based representation. Generative models then produce a categorical distribution for each element, often with a softmax function, which is sampled. Since standard multinomial sampling procedures are non-differentiable, sampling can be avoided during training or a Gumbel-softmax can be used.<a title="Select to navigate to reference" href="#cit49"><sup><span class="sup_ref">49,50</span></sup></a></p>
          <p class="otherpara">Many deep generative modeling techniques have been developed specifically for sequence generation, most notably recurrent neural networks (RNNs), which can be used for SMILES generation. The non-uniqueness of SMILES arises from a fundamental ambiguity about which atom to start the SMILES string construction on, which means that every molecule with <span class="italic">N</span> heavy (non-hydrogen) atoms can have at least <span class="italic">N</span> equivalent SMILES string representations. There is additional non-uniqueness due to different conventions on whether to include charge information in resonance structures such as nitro groups and azides. The <span class="italic">MolVS</span><a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> or <span class="italic">RDKit</span><a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a> cheminformatics packages can be used to standardize SMILES, putting them in a canonical form. However, Bjerrum <span class="italic">et al.</span> have pointed out that the latent representations obtained from canonical SMILES may be less useful because they become more related to specific grammar rules of canonical SMILES rather than the chemical structure of the underlying molecule.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> This is considered an issue for interpretation and optimization since it is better if latent spaces encode underlying chemical properties and capture notions of chemical similarity rather than SMILES syntax rules. Bjerrum <span class="italic">et al.</span> have suggested SMILES enumeration (training on all SMILES representations of each molecule), rather than using canonical SMILES, as a better solution to the non-uniqueness issue.<a title="Select to navigate to reference" href="#cit53"><sup><span class="sup_ref">53,54</span></sup></a> An approach similar to SMILES enumeration is used in computer vision applications to obtain rotational invariance—image datasets are often “augmented” by including many rotated versions of each image. Another approach to obtain better latent representations explored by Bjerrum <span class="italic">et al.</span> is to input both enumerated SMILES and Chemception-like image arrays (discussed below) into a single “heteroencoder” framework.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a></p>
          <p class="otherpara">In addition to SMILES strings, Gómez-Bombarelli <span class="italic">et al.</span> have tried InChI strings<a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> with their variational autoencoder, but found they led to inferior performance in terms of the decoding rate and the subjective appearance of the molecules generated.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> Interestingly, Winter <span class="italic">et al.</span> show that more physically meaningful latent spaces can be obtained by training a variational autoencoder to translate between InChI to SMILES.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> There is an intuitive explanation for this—the model must learn to extract the underlying chemical structures which are encoded in different ways by the two representations.</p>
          <p class="otherpara">SMILES based methods often struggle to achieve a high percentage of valid SMILES. As a possible solution to this, Kusner <span class="italic">et al.</span> proposed decomposing SMILES into a sequence of rules from a context free grammar (CFG).<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> The rules of the context-free grammar impose constraints based on the grammar of SMILES strings.<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> Because the construction of SMILES remains probabilistic, the rate of valid SMILES generation remains below 100%, even when CFGs are employed and additional semantic constraints are added on top.<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> Despite the issues inherent with using SMILES, we expect it will continue to a popular representation since most datasets store molecular graphs using SMILES as the native format, and since architectures developed for sequence generation (<span class="italic">i.e.</span> for natural language or music) can be readily adopted. Looking longer term, we expect a shift towards methods which work directly with the graph and construct molecules according to elementary operations which maintain chemical valence rules.</p>
          <p class="otherpara">Li <span class="italic">et al.</span> have developed a conditional graph generation procedure which obtains a very high rate of valid chemical graphs (91%) but lower negative log likelihood scores compared to a traditional SMILES based RNN model.<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> Another more recent work by Li <span class="italic">et al.</span> uses a deep neural network to decide on graph generation steps (append, connect, or terminate).<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> Efficient algorithms for graph and tree enumeration have been previously developed in a more pure computer science context. Recent work has looked at how such techniques can be used for molecular graph generation,<a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> and likely will have utility for deep generative models as well.</p>
        </div>
        <div>
          
          <span id="sect499"/><span class="c_heading_indent">1.2.2 Image-based representations. </span>
          <span>Most small molecules are easily represented as 2D images (with some notable exceptions like cubane). Inspired by the success of Google's Inception-ResNet deep convolutional neural network (CNN) architecture for image recognition, Goh <span class="italic">et al.</span> developed “Chemception”, a deep CNN which predicts molecular properties using custom-generated images of the molecular graph.<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> The Chemception framework takes a SMILES string in and produces an 80 × 80 greyscale image which is actually an array of integers, where empty space is ‘0’, bonds are ‘2’ and atoms are represented by their atomic number.<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> Bjerrum <span class="italic">et al.</span> extend this idea, producing “images” with five color channels which encode a variety of molecular features, some which have been compressed to few dimensions using PCA.<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a></span>
        </div>
        <div>
          
          <span id="sect508"/><span class="c_heading_indent">1.2.3 Tensor representations. </span>
          <span>Another approach to storing the molecular graph is to store the vertex type (atom type), edge type (bond type), and connectivity information in multidimensional arrays (tensors). In the approach used by de Cao &amp; Kipf,<a title="Select to navigate to reference" href="#cit50"><sup><span class="sup_ref">50,63</span></sup></a> each atom is a vertex <span class="italic">v</span><small><sub><span class="italic">i</span></sub></small> ∈ <img class="charmap" src="https://www.rsc.org/images/entities/char_e149.gif" alt="[scr V, script letter V]"/> which may be represented by a one-hot vector <span class="bold"><span class="italic">x</span></span><small><sub><span class="italic">i</span></sub></small> ∈ {0, 1}<small><sup>|<span class="italic"><img class="charmap" align="top" src="https://www.rsc.org/images/entities/i_char_e520.gif" alt="[scr A, script letter A]"/></span>|</sup></small> which indicates the atom type, out of |<img class="charmap" src="https://www.rsc.org/images/entities/char_e520.gif" alt="[scr A, script letter A]"/>| possible atom types. Each bond is represented by an edge (<span class="italic">v</span><small><sub><span class="italic">i</span></sub></small>, <span class="italic">v</span><small><sub><span class="italic">j</span></sub></small>) which is associated with a one-hot vector <span class="bold"><span class="italic">y</span></span><small><sub><span class="italic">i</span></sub></small> ∈ {0, 1}<small><sup><span class="italic">Y</span></sup></small> representing the type of bond out of <span class="italic">Y</span> possible bond types. The vertex and edge information can be stored in a vertex feature matrix <span class="italic">X</span> = [<span class="bold"><span class="italic">x</span></span><small><sub>1</sub></small>,…, <span class="bold"><span class="italic">x</span></span><small><sub><span class="italic">N</span></sub></small>]<small><sup><span class="italic">T</span></sup></small> ∈ <img class="charmap" src="https://www.rsc.org/images/entities/char_e175.gif" alt="[Doublestruck R]"/><small><sup><span class="italic">N</span>×|<img class="charmap" align="top" src="https://www.rsc.org/images/entities/char_e520.gif" alt="[scr A, script letter A]"/>|</sup></small> and an adjacency tensor <span class="italic">A</span> ∈ <img class="charmap" src="https://www.rsc.org/images/entities/char_e175.gif" alt="[Doublestruck R]"/><small><sup><span class="italic">N</span>×<span class="italic">N</span>×<span class="italic">Y</span></sup></small> where <span class="italic">A</span><small><sub><span class="italic">ij</span></sub></small> ∈ <img class="charmap" src="https://www.rsc.org/images/entities/char_e175.gif" alt="[Doublestruck R]"/><small><sup><span class="italic">Y</span></sup></small>. Simonovsky <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> use a similar approach—they take a vertex feature matrix <span class="italic">X</span> and concatenate the adjacency tensor <span class="italic">A</span> with a traditional adjacency matrix where connections are indicated by a ‘1’. As with SMILES, adjacency matrices suffer from non-uniqueness—for a molecule with <span class="italic">N</span> atoms, there are <span class="italic">N</span>! equivalent adjacency matrices representing the same molecular graph, each corresponding to a different re-ordering of the atoms/nodes. This makes it challenging to compute objective functions, which require checking if two adjacency matrix representations correspond to the same underlying graph (the “graph isomorphism” problem, which takes <span class="italic">N</span><small><sup>4</sup></small> operations in the worse case). Simonovsky <span class="italic">et al.</span> use an approximate graph matching algorithm to do this, but it is still computationally expensive.</span>
        </div>
        <div>
          
          <span id="sect566"/><span class="c_heading_indent">1.2.4 Other graph-based representations. </span>
          <span>Another approach is to train an RNN or reinforcement learning agent to operate directly on the molecular graph, adding new atoms and bonds in each action step from a list of predefined possible actions. This approach is taken with the graph convolutional policy network<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> and in recent work using pure deep reinforcement learning to generate molecules.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> Because these methods work directly on molecular graphs with rules which ensure that basic atom valence is satisfied, they generate 100% chemically valid molecules.</span>
          <p class="otherpara">Finally, when limited to small datasets one may elect to do generative modeling with compact feature vectors based on fingerprinting methods or descriptors. There are many choices (Coulomb matrices, bag of bonds, sum over bonds, descriptor sets, graph convolutional fingerprints, <span class="italic">etc.</span>) which we have previously tested for regression,<a title="Select to navigate to reference" href="#cit27"><sup><span class="sup_ref">27,67</span></sup></a> but they are generally not invertible unless a very large database with a look-up table has been constructed (in this context, by invertible we mean the complete molecular graph can be reconstructed without loss). As an example of how it may be done, Kadurin <span class="italic">et al.</span> use 166 bit Molecular ACCess System (MACCS) keys<a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a> for molecular representation with adversarial autoencoders.<a title="Select to navigate to reference" href="#cit69"><sup><span class="sup_ref">69,70</span></sup></a> In MACCS keys, also called MACCS fingerprints, each bit is associated with a specific structural pattern or question about structure. To associate molecules to MACCS keys one must search for molecules with similar or identical MACCS keys in a large chemical database. Fortunately several large online chemical databases have application programming interfaces (APIs) which allow for MACCS-based queries, for instance <span class="italic">PubChem</span>, which contains 72 million compounds.</p>
        </div>
      
    
    
      
      <h2 id="sect579"><span class="a_heading">2 Deep learning architectures</span></h2>
      <span>In this section we summarize the mathematical foundations of several popular deep learning architectures and expose some of their pros and cons. High level diagrams of the major architectures are shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>. A basic familiarity with machine learning concepts is assumed.</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2019/ME/c9me00039a/c9me00039a-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-f1.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Bird's eye views of three popular frameworks for generative modeling using SMILES strings, with possible variations shown with dashed lines.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
        
        <h3 id="sect586"><span class="b_heading">2.1 Recurrent neural networks (RNNs)</span></h3>
        <span>We discuss recurrent neural network sequence models first because they are fundamental to molecular generation—most VAE and GAN implementations include an RNN for sequence generation. In what follows, a sequence of length <span class="italic">T</span> will be denoted as <span class="italic">S</span><small><sub>1:<span class="italic">T</span></sub></small> = (<span class="italic">S</span><small><sub>1</sub></small>,…, <span class="italic">S</span><small><sub><span class="italic">T</span></sub></small>), <span class="italic">S</span><small><sub>T</sub></small> ∈ <img class="charmap" src="https://www.rsc.org/images/entities/char_e149.gif" alt="[scr V, script letter V]"/>, where <img class="charmap" src="https://www.rsc.org/images/entities/char_e149.gif" alt="[scr V, script letter V]"/> is the set of tokens, also called the vocabulary. For the purpose of this section we assume the sequences in question are SMILES strings, as they are by far the most widely used. As discussed previously in the context of SMILES the “tokens” are the different characters which are used to specify atom types, bond types, parentheses, and the start and stop points of rings. The first step in sequence modeling is typically one-hot encoding of the sequence's tokens, in which each token is represented as a unique <span class="italic">N</span> dimensional vector where one element is 1 and the rest are 0 (where <span class="italic">N</span> is the number of tokens in the vocabulary).</span>
        <p class="otherpara">Recurrent neural networks (RNNs) are the most popular models for sequence modeling and generation. We will not go into detail of their architecture, since it is well described elsewhere.<a title="Select to navigate to reference" href="#cit120"><sup><span class="sup_ref">120,121</span></sup></a> An important detail to note however is that the type of RNN unit that is typically used for generating molecules is either the long short term memory (LSTM) unit,<a title="Select to navigate to references" href="#cit122"><sup><span class="sup_ref">122</span></sup></a> or a newer more computationally efficient variant called the gated recurrent unit (GRU).<a title="Select to navigate to references" href="#cit123"><sup><span class="sup_ref">123</span></sup></a> Both LSTMs and GRUs contain a memory cell which alleviates the exploding and vanishing gradient problems that can occur when training RNNs to predict long-term dependencies.<a title="Select to navigate to reference" href="#cit122"><sup><span class="sup_ref">122,123</span></sup></a></p>
        <p class="otherpara">Sequence models are often trained to predict just a single missing token in a sequence, as trying to predict more than one token leads to a combinatorial explosion of possibilities. Any machine learning model trained to predict the next character in an input sequence can be run in “generative mode” or “autoregressive mode” by concatenating the predicted token to the input sequence and feeding the new sequence back into the model. However, this type of autoregressive generation scheme typically fails because the model was trained to predict on the data distribution and not its own generative distribution, and therefore each prediction contains at least a small error. As the network is run recursively, these errors rapidly compound, leading to rapid degradation in the quality of the generated sequences. This problem is known as “exposure bias”.<a title="Select to navigate to references" href="#cit124"><sup><span class="sup_ref">124</span></sup></a> The Data as Demonstrator (DaD) algorithm tries to solve the problem of exposure bias by running a model recursively during training and comparing the output to the training data during training.<a title="Select to navigate to references" href="#cit125"><sup><span class="sup_ref">125</span></sup></a> DaD was extended to sequence generation with RNNs by Bengio <span class="italic">et al.</span>, who called the method “scheduled sampling”.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a> While research continues in this direction, issues have been raised about the lack of a firm mathematical foundation for such techniques, with some suggesting they do not properly approximate maximum likelihood.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a></p>
        <p class="otherpara">Better generative models can be obtained by training using maximum likelihood maximization on the sequence space rather than next-character prediction. In maximum likelihood training a model <span class="italic">π</span><small><sub><span class="italic">θ</span></sub></small> parametrized by <span class="italic">θ</span> is trained with the following differentiable loss:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn1"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t1.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t1.gif"/></a></td><td class="rightEqn">(1)</td></tr></table>Here <span class="italic">Z</span> is the set of training sequences which are assumed to each be of length <span class="italic">T</span>. This expression is proportional to the negative cross entropy of the model distribution and the training data distribution (maximizing likelihood is equivalent to minimizing cross entropy). MLE training can be done with standard gradient descent techniques and backpropagation through time to compute the gradient of the loss. In practice though this type of training fails to generate valid SMILES, likely because of strict long term dependencies such as closing parentheses and rings. The “teacher forcing” training procedure<a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> is an important ingredient which was found to be necessary to include in the molecular autoencoder VAE to capture such long term dependencies—otherwise the generation rate of valid SMILES was found to be near 0%.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> In teacher forcing, instead of sampling from the model's character distribution to get the next character, the right character is given directly to the model during training.<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a></p>
        <p class="otherpara">In the context of SMILES strings generation, to generate each character the output layer usually gives probabilities <span class="italic">p</span><small><sub><span class="italic">i</span></sub></small> for every possible SMILES string token. When running in generative mode, the typical method is to use a multinomial sampler to sample this distribution, while in training mode one typically just chooses the token with the highest probability. Using a multinomial sampler captures the model's true distribution, but because MLE training tends to focus on optimizing the peaks of the distribution and doesn't always capture the tails of distributions well. So called “thermal” rescaling can be used to sample further away from the peaks of the distribution by rescaling the probabilities as:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn2"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t2.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t2.gif"/></a></td><td class="rightEqn">(2)</td></tr></table>where <span class="italic">T</span> is a sampling temperature. Alternatively, if a softmax layer is used to generate the final output of a neural network, a temperature parameter can be built directly into it. Another alternative is the “freezing function”:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn3"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t3.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t3.gif"/></a></td><td class="rightEqn">(3)</td></tr></table></p>
        <p class="otherpara">Sampling at low <span class="italic">T</span> leads to the generation of molecules which are only slight variations on molecules seen in the training data. Generation at high <span class="italic">T</span> leads to greater diversity but also higher probability of nonsensical results.</p>
        <div>
          
          <span id="sect641"/><span class="c_heading_indent">2.1.1 Optimization with RNNs using reinforcement learning. </span>
          <span>Neil <span class="italic">et al.</span> introduced a simple method for repeated MLE which biases generation towards molecules with good properties, which they call <span class="italic">HillClimb-MLE</span>.<a title="Select to navigate to references" href="#cit78"><sup><span class="sup_ref">78</span></sup></a> Starting with a model that has been trained <span class="italic">via</span> MLE on the training data, they generate a large set of SMILES sequences. They then calculate a reward function <span class="italic">R</span>(<span class="italic">S</span>) for each sequence <span class="italic">S</span> generated and find the subset of <span class="italic">N</span>′ generated molecules with the highest rewards. This subset is used to retrain the model with MLE, and the process is repeated. Each time a new subset of <span class="italic">N</span>′ generated molecules is determined, it is concatenated on the previous set, so the amount of data being used for MLE grows with each iteration. As this process is repeated the model begins to generate molecules which return higher and higher values from <span class="italic">R</span>(<span class="italic">S</span>).</span>
          <p class="otherpara">A more common technique is to use <span class="italic">reinforcement learning</span> after MLE pretraining to fine tune the generator to produce molecules with high reward. The problem of sequence generation can be recast as a reinforcement learning problem with a discrete action space. At each timestep time <span class="italic">t</span>, the current state of the “environment” is the sequence generated so far is (<span class="italic">s</span><small><sub>0</sub></small>,..., <span class="italic">s</span><small><sub><span class="italic">t</span></sub></small>) and the action <span class="italic">a</span> is next token to be chosen, <span class="italic">a</span> = <span class="italic">s</span><small><sub><span class="italic">t</span>+1</sub></small>. The goal of reinforcement learning is to maximize the expected return <span class="italic">G</span><small><sub><span class="italic">T</span></sub></small> for all possible start states <span class="italic">s</span><small><sub>0</sub></small>. The return function <a href="/image/article/2019/ME/c9me00039a/c9me00039a-t4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t4.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t4.gif"/></a> simply sums the rewards over the length of time the agent is active, which is called an episode. Mathematically the optimization problem reinforcement learning tries to solve is expressed as:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn4"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t5.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t5.gif"/></a></td><td class="rightEqn">(4)</td></tr></table>where <span class="italic">θ</span> are the parameters of the model. In our case, one episode corresponds to the generation of one molecule, there is only one start state, (the ‘GO’ character) and <span class="italic">R</span><small><sub><span class="italic">T</span></sub></small> = 0 until the end-of-sequence (‘EOS’) token is generated or the max string length is reached. If <span class="italic">T</span> denotes the max length of the SMILES string then only <span class="italic">R</span><small><sub><span class="italic">T</span></sub></small> is non-zero and therefore <span class="italic">G</span><small><sub><span class="italic">T</span></sub></small> = <span class="italic">R</span><small><sub><span class="italic">T</span></sub></small> for all <span class="italic">T</span>. The state transition is deterministic (<span class="italic">i.e.</span><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t6.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t6.gif"/></a> for the next state <span class="italic">S</span><small><sub>1:<span class="italic">T</span>+1</sub></small> if the current state is <span class="italic">S</span><small><sub>1:<span class="italic">T</span></sub></small> and the action <span class="italic">a</span> = <span class="italic">s</span><small><sub><span class="italic">t</span>+1</sub></small>, while for other states <span class="italic">s</span>′′, <a href="/image/article/2019/ME/c9me00039a/c9me00039a-t7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t7.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t7.gif"/></a>). Because of these simplifications, <a title="" href="#eqn4">eqn (4)</a> assumes a simple form:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn5"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t8.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t8.gif"/></a></td><td class="rightEqn">(5)</td></tr></table>Here the policy model <span class="italic">π</span><small><sub><span class="italic">θ</span></sub></small>(<span class="italic">a</span>|<span class="italic">s</span>) gives the probability for choosing the next action given the current state. In our case:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn6"/><span id="eqn6"><span class="italic">π</span><small><sub><span class="italic">θ</span></sub></small>(<span class="italic">a</span><small><sub><span class="italic">t</span></sub></small>|<span class="italic">s</span><small><sub><span class="italic">t</span></sub></small>) = <span class="italic">π</span><small><sub><span class="italic">θ</span></sub></small>(<span class="italic">s</span><small><sub><span class="italic">t</span> + 1</sub></small>|<span class="italic">S</span><small><sub>1:<span class="italic">T</span></sub></small>)</span></td><td class="rightEqn">(6)</td></tr></table></p>
          <p class="otherpara">There are many reinforcement learning methods, but broadly speaking they can be broken into value learning and policy learning methods. Most work so far has used variants of the REINFORCE algorithm,<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> a type of policy learning method which falls into the class of policy gradient methods. It can be shown that for a run of length <span class="italic">T</span> the gradient of <span class="italic">J</span>(<span class="italic">θ</span>) (<a title="" href="#eqn4">eqn (4)</a>) is:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn7"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t9.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t9.gif"/></a></td><td class="rightEqn">(7)</td></tr></table></p>
          <p class="otherpara">Computing the exact expectation of <span class="italic">G</span><small><sub><span class="italic">T</span></sub></small> for all possible action sequences is impossible, so instead the <span class="italic">G</span><small><sub><span class="italic">T</span></sub></small> from a single run is used before each gradient update. This is sometimes referred to as a “Monte-Carlo” type approach. Fortunately, this process can be parallelized by calculating multiple gradient updates on different threads before applying them. Neil <span class="italic">et al.</span> recently tested several newer reinforcement learning algorithms—advantage actor-critic (AAC) and proximal policy optimization (PPO), where they report superior performance over REINFORCE (PPO &gt; AAC &gt; REINFORCE). Interestingly, they find <span class="italic">Hillclimb-MLE</span> is competitive with and occasionally superior to PPO.<a title="Select to navigate to references" href="#cit78"><sup><span class="sup_ref">78</span></sup></a></p>
          <p class="otherpara">Olivecrona <span class="italic">et al.</span> argue that policy learning methods are a more natural choice for molecular optimization because they can start with a pre-trained generative model, while value-function learning based methods cannot.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> Additionally, most policy learning methods have been proven to lead to an optimal policy and the resulting generative models are faster to sample.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> In contrast, Zhou <span class="italic">et al.</span> argue that value function learning methods are superior in part because policy gradient methods suffer from issues with high variance in the gradient estimates during training.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a></p>
          <p class="otherpara">Empirically it has been found that using RL after MLE can cause the generated model to “drift” too far, causing important information about viable chemical structures learned during MLE to be lost. This can take the form of highly unstable structures being generated or invalid SMILES. One solution is to “augment” the reward function with the likelihood:<a title="Select to navigate to reference" href="#cit74"><sup><span class="sup_ref">74,131</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn8"/><span id="eqn8"><span class="italic">R</span>′(<span class="italic">S</span>) = [<span class="italic">σR</span>(<span class="italic">S</span>) + log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span><small><sub>prior</sub></small>(<span class="italic">S</span>) − log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span><small><sub>current</sub></small>(<span class="italic">S</span>)]<small><sup>2</sup></small></span></td><td class="rightEqn">(8)</td></tr></table></p>
          <p class="otherpara">Other possibilities are explored by Olivecrona <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> Fundamentally, whether “drift” during RL training becomes an issue depends on the details of the reward function—if the reward function is good, drift should be in a good direction. Recently Zhou <span class="italic">et al.</span> sought approaches that circumvent MLE when training. In their RL based approach for molecular optimization, they do not use an RNN or any pre-trained generative model and instead use pure RL training.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> The RL agent works directly on constructing molecular graphs, taking actions such as atom/bond addition and atom removal. The particular approach they use is deep-<span class="italic">Q</span> learning, which incorporates several recent innovations that were developed at <span class="italic">DeepMind</span> and elsewhere.<a title="Select to navigate to references" href="#cit132"><sup><span class="sup_ref">132</span></sup></a> Jaques <span class="italic">et al.</span> have also explored the application of deep <span class="italic">Q</span>-learning and <span class="italic">G</span>-learning to molecular optimization.<a title="Select to navigate to references" href="#cit131"><sup><span class="sup_ref">131</span></sup></a> Reinforcement learning is a rapidly developing field, and there remain many recent advancements such as new attention mechanisms which have not yet been tested in the domain of molecular optimization.</p>
          <p class="otherpara">To give a flavor of what applications have been demonstrated, we will briefly present some representative works using RNNs. Bjerrum &amp; Threlfall explore using an architecture consisting of 256 LSTM cells followed by a “time-distributed dense” output layer.<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> Their network achieved a SMILES validity rates of 98% and the property distributions for the properties tested matched the property distributions found in the training data (some key properties they looked at were synthetic accessibility score, molecular weight, log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span>, and total polar surface area). Popova <span class="italic">et al.</span> have shown how an RNN trained for generation can be further trained with reinforcement learning to generate molecules targeted towards a specific biological function - in their case they focused on the decree to which molecules bind and inhibit the JAK2 enzyme, for which much empirical data exists. They showed how their system could be used to either maximize or minimize inhibition with JAK2 and also independently discovered 793 commercially available compounds found in the ZINC database.<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> In a similar vein, Segler <span class="italic">et al.</span> fine tune an RNN to generate a “focused library” of molecules which are likely to target the 5-HT<small><sub>2A</sub></small> receptor.<a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> Finally, Olivecrona <span class="italic">et al.</span> show how a generative model can be fine tuned to generate analogs of a particular drug (Celecoxib) or molecules which bind to the type 2 dopamine receptor.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a></p>
        </div>
      
      
        
        <h3 id="sect800"><span class="b_heading">2.2 Autoencoders</span></h3>
        <span>In 2006 Hinton and Salakhutdinov showed how advances in computing power allowed for the training of a deep autoencoder which was capable of beating other methods for document classification.<a title="Select to navigate to references" href="#cit133"><sup><span class="sup_ref">133</span></sup></a> The particular type of neural network they used was a stack of restricted Boltzmann machines, an architecture which would later be called a “deep Boltzmann machine”.<a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> While deep Boltzmann machines are theoretically powerful, they are computationally expensive to train and impractical for many tasks. In 2013 Kingma <span class="italic">et al.</span> introduced the variational autoencoder (VAE),<a title="Select to navigate to references" href="#cit21"><sup><span class="sup_ref">21</span></sup></a> which was used in 2016 by Bombarelli <span class="italic">et al.</span> to create the first machine learning based generative model for molecules.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a></span>
        <div>
          
          <span id="sect810"/><span class="c_heading_indent">2.2.1 Variational autoencoders (VAEs). </span>
          <span>VAEs are derived mathematically from the theory of variational inference and are only called autoencoders because the resulting architecture has the same high level structure as a classical autoencoder. VAEs are fundamentally a latent variable model <span class="italic">p</span>(<span class="bold"><span class="italic">x</span></span>,<span class="bold"><span class="italic">z</span></span>) = <span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>|<span class="bold"><span class="italic">z</span></span>)<span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>) which consists of latent variables <span class="bold"><span class="italic">z</span></span> drawn from a pre-specified prior <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>) and passed into a decoder <span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>|<span class="bold"><span class="italic">z</span></span>) parametrized by parameters <span class="italic">θ</span>. To apply maximum likelihood learning to such a model we like to maximize the probability of each observed datapoint <a href="/image/article/2019/ME/c9me00039a/c9me00039a-t10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t10.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t10.gif"/></a> for all datapoints in our training data. However for complicated models with many parameters <span class="italic">θ</span> (like neural networks) this integral is intractable to compute. The method of variational inference instead maximizes a lower bound on log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">p</span>(<span class="bold"><span class="italic">x</span></span>):<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn9"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t11.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t11.gif"/></a></td><td class="rightEqn">(9)</td></tr></table>where <span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>) is an estimate of posterior distribution <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>) = <span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>|<span class="bold"><span class="italic">z</span></span>)<span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>)/<span class="italic">p</span>(<span class="bold"><span class="italic">x</span></span>). The right hand side of <a title="" href="#eqn9">eqn (9)</a> is called the “negative variational free energy” or “evidence lower bound” (ELBO) and can be written as:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn10"/><span id="eqn10"><span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_e144.gif" alt="[script L]"/></span><small><sub><span class="italic">θ</span>,<span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>) = <img class="charmap" src="https://www.rsc.org/images/entities/char_e168.gif" alt="[Doublestruck E]"/><small><sub><span class="bold"><span class="italic">z</span></span>∼<span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>)</sub></small>[log<span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>)] − <span class="italic">D</span><small><sub>KL</sub></small>(<span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>),<span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>))</span></td><td class="rightEqn">(10)</td></tr></table>Here we encounter the Kullback–Leibler (KL) divergence:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn11"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t12_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t12.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t12.gif"/></a></td><td class="rightEqn">(11)</td></tr></table></span>
          <p class="otherpara">After several manipulations, <a title="" href="#eqn10">eqn (10)</a> can be written as<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn12"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t13_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t13.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t13.gif"/></a></td><td class="rightEqn">(12)</td></tr></table>where <span class="italic">H</span> is the (differentiable) entropy. The loss function for the variational autoencoder for examples <span class="bold"><span class="italic">x</span></span> in our training dataset <span class="italic">Z</span> is:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn13"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t14_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t14.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t14.gif"/></a></td><td class="rightEqn">(13)</td></tr></table></p>
          <p class="otherpara">In a VAE, during training first <span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>) (the encoder) generates a <span class="bold"><span class="italic">z</span></span>. Then the decoder <span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>|<span class="bold"><span class="italic">z</span></span>) model attempts to recover <span class="bold"><span class="italic">x</span></span>. Training is done using backpropagation and the loss function (<a title="" href="#eqn13">eqn (13))</a> which tries to maximize <span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_e144.gif" alt="[script L]"/></span>(<span class="bold"><span class="italic">x</span></span>,<span class="italic">θ</span>,<span class="italic">ϕ</span>). This corresponds to maximizing the chance of reconstruction <span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>|<span class="bold"><span class="italic">z</span></span>) (the first term) but also minimizing the KL-divergence between <span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>) and the prior distribution <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>). Typically the prior is chosen to be a set of independent unit normal distributions and the encoder is assumed to be a factorized multidimensional normal distribution:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn14"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t15_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t15.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t15.gif"/></a></td><td class="rightEqn">(14)</td></tr></table></p>
          <p class="otherpara">The encoder <span class="italic">q</span><small><sub>θ</sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>) is typically a neural network with parameters <span class="italic">ϕ</span> used to find the mean and variance functions in <img class="charmap" src="https://www.rsc.org/images/entities/char_e52d.gif" alt="[scr N, script letter N]"/>(<span class="bold"><span class="italic">z</span></span>,<span class="italic">μ</span>(<span class="bold"><span class="italic">x</span></span>), diag(<span class="italic">σ</span><small><sup>2</sup></small>(<span class="bold"><span class="italic">x</span></span>))). The resulting “Gaussian VAE” has the advantage that the KL-divergence can be computed analytically. The parameters <span class="italic">θ</span> and <span class="italic">ϕ</span> in the decoder and encoder are all learned <span class="italic">via</span> backpropagation. There are several important innovations which have been developed to streamline backpropagation and training which are described in detail elsewhere.<a title="Select to navigate to reference" href="#cit21"><sup><span class="sup_ref">21,121,135</span></sup></a></p>
          <p class="otherpara">There are several reasons that variational autoencoders perform better than classical autoencoders. Since the latent distribution is probabilistic, this introduces noise which intuitively can be seen as a type of regularization that forces the VAE to learn more robust representations. Additionally, specifying that the latent space must be a Gaussian leads to a much smoother latent space which makes optimization much easier and also leads to fewer “holes” in the distribution corresponding to invalid or bad molecules. VAEs therefore are useful for interpolation between points corresponding to molecules in the training data.</p>
          <p class="otherpara">In the molecular autoencoder of Gómez-Bombarelli <span class="italic">et al.</span> each SMILES <span class="bold"><span class="italic">x</span></span> is converted to a one-hot representation and a convolutional neural network is used to find the parameters of the Gaussian distribution <span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>).<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> The decoder in the molecular autoencoder is an RNN, but in contrast to pure RNN models, where high rates of valid SMILES generation have been reported (94–98%),<a title="Select to navigate to reference" href="#cit72"><sup><span class="sup_ref">72,75,78</span></sup></a> the molecular autoencoder generates far fewer valid SMILES. The valid SMILES rate was found to vary greatly between ≈75% for points near known molecules to only 4% for randomly selected latent points.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> Kusner <span class="italic">et al.</span> report an average valid decoding rate of only 0.7% using a similar VAE architecture.<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> These low decoding rates are not a fatal issue however simply because a validity checker (such as found in RDKit) can easily be used to throw out invalid SMILES during generation. However, the low rate of validity suggests fundamental issues in quality of the learned latent representation. As mentioned previously, higher rates of SMILES validity have been achieved by representing SMILES in terms of rules from a context-free grammar (CFG).<a title="Select to navigate to reference" href="#cit57"><sup><span class="sup_ref">57,58</span></sup></a> Kusner <span class="italic">et al.</span> achieved somewhat higher rates of SMILES generation using a CFG (7.2%, as described in the supplementary information of Kusner <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a>). Further work by Dai <span class="italic">et al.</span> added additional “semantic” constraints on top of a CFG yielding a higher rate of valid SMILES (43.5%).<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> Janz <span class="italic">et al.</span> recently proposed using Bayesian active learning as a method of forcing models to learn what makes a sequence valid, and incorporating this into RNNs in VAEs could lead to higher valid decoding rates.<a title="Select to navigate to references" href="#cit136"><sup><span class="sup_ref">136</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect1022"/><span class="c_heading_indent">2.2.2 Adversarial autoencoders (AAEs). </span>
          <span>Adversarial autoencoders are similar to variational autoencoders, but differ in the means by which they regularize the latent distribution by forcing it to conform to the prior <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>).<a title="Select to navigate to references" href="#cit137"><sup><span class="sup_ref">137</span></sup></a> Instead of minimizing KL-divergence metric to enforce the generator to output a latent distribution corresponding to a prespecified prior (usually a normal distribution), they use adversarial training with a discriminator <span class="italic">D</span> whose job is to distinguish the generator's latent distribution from the prior. The discriminator outputs a probability <span class="italic">p</span> ∈ (0, 1) which predicts the probability samples it sees are from the prior. The objective of the discriminator is <span class="italic">maximize</span> the following:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn15"/><span id="eqn15"><span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_e144.gif" alt="[script L]"/></span><small><sub>adv</sub></small> = <img class="charmap" src="https://www.rsc.org/images/entities/char_e168.gif" alt="[Doublestruck E]"/><small><sub><span class="bold"><span class="italic">x</span></span>∼<span class="italic">p</span><small><sub>d</sub></small></sub></small>[log<span class="italic">D</span>(<span class="italic">q</span><small><sub>Θ</sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>))] + <img class="charmap" src="https://www.rsc.org/images/entities/char_e168.gif" alt="[Doublestruck E]"/><small><sub><span class="bold"><span class="italic">x</span></span>∼<span class="italic">p</span><small><sub><span class="italic">z</span></sub></small></sub></small>[log(1 − <span class="italic">D</span>(<span class="bold"><span class="italic">z</span></span>))]</span></td><td class="rightEqn">(15)</td></tr></table></span>
          <p class="otherpara">The overall objective function for the AAE to <span class="italic">minimize</span> can be expressed as<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn16"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t16_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t16.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t16.gif"/></a></td><td class="rightEqn">(16)</td></tr></table></p>
        </div>
        <div>
          
          <span id="sect1064"/><span class="c_heading_indent">2.2.3 Supervised VAEs/AAEs for property prediction &amp; optimization. </span>
          <span>In supervised VAEs, target properties <span class="italic">y</span> for each molecule are incorporated into the generator in addition to the SMILES strings or other molecular representation. <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a> shows several different ways this can be done, representing the generative models as graphical models. Everything we discuss in this section can also be applied to AAEs,<a title="Select to navigate to references" href="#cit137"><sup><span class="sup_ref">137</span></sup></a> but we restrict our discussion to VAEs for simplicity.</span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2019/ME/c9me00039a/c9me00039a-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-f2.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Different deep learning approaches visualized as graphical models. Figure adapted from Kang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a> Solid lines represent explicit conditional dependencies while dashed lines represent implicit conditional dependencies (arising from the relationship between <span class="italic">X</span> and <span class="italic">Y</span> inherent in the training data) for which disentanglement may be desired. a.) Regression (property prediction) b.) direct generative model c.) autoencoder d.) supervised autoencoder, type 1 (<a title="Select to navigate to references" href="#cit23">ref. 23</a>) e.) supervised autoencoder, type 2 (<a title="Select to navigate to reference" href="#cit88">ref. 88 and 111</a>) f.) supervised/semisupervised autoencoder, type 3.<a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a></span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">In the work by Gómez-Bombarelli <span class="italic">et al.</span> they attached a neural network (multilayer perceptron) to the latent layer and jointly trained the neural network to predict property values <span class="italic">y</span> and the VAE to minimize reconstruction loss. One unique property they optimize after training such a system is the predicted HOMO–LUMO gap, which is important for determining a molecule's utility in organic solar cells. The advantage of supervised VAEs is that the generator learns a good latent representation both for property prediction and reconstruction. With the property predictor trained, it becomes possible to do property optimization in the latent space, by either using Gaussian process optimization or gradient ascent. Interestingly, in supervised VAEs a particular direction in the latent space always became correlated with the property value <span class="italic">y</span>, while this was never observed in unsupervised VAEs. When one desires to do optimization, Gómez-Bombarelli <span class="italic">et al.</span> argue for using a Gaussian process model as the property predictor instead of a neural network, because it generates a smoother landscape.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> The specific procedure they used was to first train a VAE using supervised training with a neural network property predictor and then train a Gaussian process model separately using the latent space representations of the training molecules as input. They then use the Gaussian process model for optimization, and they showed it was superior to a genetic optimization algorithm and random search in the latent space. Since that work, several other researchers have used Gaussian process regression to perform optimization in the latent space of a VAE.<a title="Select to navigate to reference" href="#cit99"><sup><span class="sup_ref">99,110,138</span></sup></a></p>
          <p class="otherpara">Two other types of supervised VAEs are shown in <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>, which we call “type 2” and “type 3”. Unlike the autoencoder frame work discussed in the previous section, these two types of autoencoder can be used for conditional generation. In “type 3” supervised VAEs the ELBO term in the objective function (<a title="" href="#eqn12">eqn (12)</a>) becomes:<a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn17"/><span id="eqn17"><img class="charmap" src="https://www.rsc.org/images/entities/char_e168.gif" alt="[Doublestruck E]"/><small><sub><span class="bold"><span class="italic">z</span></span>∼<span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>,<span class="bold"><span class="italic">y</span></span>)</sub></small>[log<span class="italic">p</span><small><sub><span class="italic">θ</span></sub></small>(<span class="bold"><span class="italic">x</span></span>|<span class="bold"><span class="italic">y</span></span>,<span class="bold"><span class="italic">z</span></span>)] − <span class="italic">D</span><small><sub>KL</sub></small>[<span class="italic">q</span><small><sub><span class="italic">ϕ</span></sub></small>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">x</span></span>,<span class="bold"><span class="italic">y</span></span>)∥<span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>,<span class="bold"><span class="italic">y</span></span>)]</span></td><td class="rightEqn">(17)</td></tr></table></p>
          <p class="otherpara">Kang <span class="italic">et al.</span> assume that the property values have a Gaussian distribution. Type 3 VAEs are particularly useful when <span class="italic">y</span> is known for only some of the training data (a semi-supervised setting). In semi-supervised VAEs, the generator is tasked with predict <span class="italic">y</span> and is trained on the molecules where <span class="italic">y</span> is known and makes a best guess prediction for the rest. In effect, when <span class="italic">y</span> is not known, it becomes just another latent variable and a different objective function is used (for details, see Kang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a>).</p>
          <p class="otherpara">In type 2 VAEs, property data <span class="bold"><span class="italic">y</span></span> is embedded directly into the latent space during training.<a title="Select to navigate to reference" href="#cit88"><sup><span class="sup_ref">88,111</span></sup></a> Supervised and semi-supervised VAEs can both be used for conditional sampling, and thus are sometimes called “conditional VAEs”. In the traditional way of doing conditional sampling, <span class="bold"><span class="italic">y</span></span> is specified and then one samples from the prior <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>). Then one samples from the generator <span class="italic">p</span><small><sub>θ</sub></small>(<span class="bold"><span class="italic">x</span></span>|<span class="bold"><span class="italic">y</span></span>,<span class="bold"><span class="italic">z</span></span>). In the case of type 1 and type 2 VAEs, however, there is an issue pointed out by Polykovskiy <span class="italic">et al.</span> which they call “entanglement”.<a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> The issue is that when sampling we assumed that <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>) is independent of <span class="italic">p</span>(<span class="bold"><span class="italic">y</span></span>). However, the two distributions are actually “entangled” by the implicit relationship between <span class="bold"><span class="italic">x</span></span> and <span class="bold"><span class="italic">y</span></span> which is in the training data (this is indicated by a dashed line in <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a>). For consistency, one should be sampling from <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">y</span></span>). Polykovskiy <span class="italic">et al.</span> developed two “disentanglement” approaches to ameliorate this issue: learning <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">y</span></span>) and forcing all <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>|<span class="bold"><span class="italic">y</span></span>) to match <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>).<a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a></p>
          <p class="otherpara">When generating molecules with an RNN, we previously discussed sampling from the model's distribution by simply running the model and taking either the token with the maximum probability or using a multinomial sampler at each step of the sequence generation. When sampling from the generator of a conditional VAE, we wish to know what the model says is the likely molecule given <span class="bold"><span class="italic">y</span></span> and <span class="bold"><span class="italic">z</span></span>, since we are interested in focusing on the molecules the model predicts are most likely to be associated with a particular set of properties:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn18"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t17_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t17.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t17.gif"/></a></td><td class="rightEqn">(18)</td></tr></table></p>
          <p class="otherpara">Taking the most likely token at each step (the “greedy” method) is only a rough approximation to <span class="bold"><span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/b_i_char_0078_0302.gif" alt="[x with combining circumflex]"/></span></span>. Unfortunately, completing the optimization in <a title="" href="#eqn18">eqn (18)</a> is a computationally intractable problem because the space of sequences grows exponentially with the length of the sequence. However, a variation on the greedy method called “beam search” can be used to get an approximation of <span class="bold"><span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/b_i_char_0078_0302.gif" alt="[x with combining circumflex]"/></span></span>.<a title="Select to navigate to reference" href="#cit89"><sup><span class="sup_ref">89,139</span></sup></a> In brief, beam search keeps the top <span class="italic">K</span> most likely (sub)sequences at each step of the generation.</p>
        </div>
      
      
        
        <h3 id="sect1205"><span class="b_heading">2.3 Generative adversarial networks (GANs)</span></h3>
        <span>The key idea underlying GANs is to introduce a discriminator network whose job is to distinguish whether the molecule it is looking at was generated by the generative model or came from the training data. In GAN training, the objective of the generative model becomes to try to fool the discriminator rather than maximizing likelihood. There are theoretical arguments and growing empirical evidence showing that GAN models can overcome some of the well known weaknesses of maximum likelihood based training. However, there are also many technical difficulties which plague GAN training and getting GANs to work well typically requires careful hyperparameter tuning and implementation of several non-obvious “tricks”.<a title="Select to navigate to references" href="#cit140"><sup><span class="sup_ref">140</span></sup></a> GANs are a rapidly evolving research area, and given space limitations we can only touch on several of the key developments here.</span>
        <p class="otherpara">The original paper on GANs (Goodfellow <span class="italic">et al.</span> 2014) introduced the following objective function:<a title="Select to navigate to references" href="#cit22"><sup><span class="sup_ref">22</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn19"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t18_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t18.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t18.gif"/></a></td><td class="rightEqn">(19)</td></tr></table>Here <span class="italic">p</span><small><sub>d</sub></small>(<span class="bold"><span class="italic">x</span></span>) is the data distribution. This form of the objective function has a nice interpretation as a two person minimax game. However, this objective function is rarely used in practice for a few reasons. Firstly, as noted in the original paper, this objective function does not provide a very strong gradient signal when training starts because then log(1 − <span class="italic">D</span>(<span class="italic">G</span>(<span class="bold"><span class="italic">z</span></span>))) saturates (goes to negative infinity) and the numerical derivative becomes impossible to calculate. Still, understanding this objective function can help understand how generative modeling with GAN training can be superior to maximum likelihood based generative modeling. For a fixed <span class="italic">G</span>, the optimal discriminator is:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn20"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t19_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t19.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t19.gif"/></a></td><td class="rightEqn">(20)</td></tr></table></p>
        <p class="otherpara">If we assume <a href="/image/article/2019/ME/c9me00039a/c9me00039a-t20_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t20.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t20.gif"/></a>, then the objective function <img class="charmap" src="https://www.rsc.org/images/entities/char_e522.gif" alt="[capital script C]"/>(<span class="italic">G</span>) for the generator can be expressed as:<a title="Select to navigate to references" href="#cit22"><sup><span class="sup_ref">22</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn21"/><span id="eqn21"><img class="charmap" src="https://www.rsc.org/images/entities/char_e522.gif" alt="[capital script C]"/>(<span class="italic">G</span>) = −log(4) + 2<span class="italic">D</span><small><sub>JS</sub></small>(<span class="italic">p</span><small><sub>d</sub></small>,<span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small>)</span></td><td class="rightEqn">(21)</td></tr></table>where <span class="italic">D</span><small><sub>JS</sub></small>(<span class="italic">p</span><small><sub>d</sub></small>,<span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small>) is the Jensen–Shannon divergence:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn22"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t21_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t21.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t21.gif"/></a></td><td class="rightEqn">(22)</td></tr></table>Here <span class="italic">D</span><small><sub>KL</sub></small>(<span class="italic">p</span>,<span class="italic">q</span>) is the Kullback–Leibler (KL) divergence. Maximizing the log-likelihood is equivalent to minimizing the forward KL divergence <span class="italic">D</span><small><sub>KL</sub></small>(<span class="italic">p</span><small><sub>d</sub></small>,<span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small>).<a title="Select to navigate to references" href="#cit135"><sup><span class="sup_ref">135</span></sup></a> To better understand what this means, we can rewrite the equation for KL divergence (<a title="" href="#eqn11">eqn (11))</a> in a slightly different way:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn23"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t22_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t22.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t22.gif"/></a></td><td class="rightEqn">(23)</td></tr></table></p>
        <p class="otherpara">This shows us that KL divergence captures the difference between <span class="italic">p</span><small><sub>d</sub></small> and <span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small> weighted by <span class="italic">p</span><small><sub>d</sub></small>. Thus one of the weaknesses of maximum likelihood is that <span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small> may have significant deviations from <span class="italic">p</span><small><sub>d</sub></small> when <span class="italic">p</span><small><sub>d</sub></small> ≈ 0. To summarize, the forward KL divergence (<span class="italic">D</span><small><sub>KL</sub></small>(<span class="italic">p</span><small><sub>d</sub></small>,<span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small>)) punishes models that underestimate the data distribution, while the reverse KL divergence (<span class="italic">D</span><small><sub>KL</sub></small>(<span class="italic">p</span><small><sub>d</sub></small>,<span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small>)) punishes models that overestimate the data distribution. Therefore we see that <a title="" href="#eqn21">eqn (21)</a>, which contains both forward and backwards KL terms, takes a more “balanced” approach than maximum likelihood, which only optimizes forward KL divergence. Optimizing reverse KL divergence directly requires knowing an explicit distribution for <span class="italic">p</span><small><sub>d</sub></small>, which usually is not available. In effect, the discriminator component of the GAN works to learn <span class="italic">p</span><small><sub>d</sub></small>, and thus GANs provide a way of overcoming this issue.</p>
        <p class="otherpara">As noted before, the GAN objective function given in <a title="" href="#eqn19">eqn (19)</a>, however, does not provide a good gradient signal when training first starts since typically <span class="italic">p</span><small><sub>d</sub></small> and <span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small> have little overlap to begin with. Empirically, this occurs because data distributions typically lie on a low dimensional manifold in a high dimensional space, and the location of this manifold is not known in advance. The Wasserstein GAN (WGAN) is widely accepted to provide a better metric for measuring the distance between <span class="italic">p</span><small><sub>d</sub></small> and <span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small> than the original GAN objective function and results in faster and more stable training.<a title="Select to navigate to references" href="#cit141"><sup><span class="sup_ref">141</span></sup></a> The WGAN is based on the Wasserstein metric (also called the “Earth mover's distance”) which can be informally understood by imagining the two probability distributions <span class="italic">p</span><small><sub>d</sub></small> and <span class="italic">p</span><small><sub><span class="italic">θ</span><small><sub><span class="italic">G</span></sub></small></sub></small> to be piles of dirt, and the distance between them to be the number of buckets of dirt that need to be moved to transform one to the other, times the sum of the distances each bucket must be moved. Mathematically this is expressed as:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn24"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t23_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t23.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t23.gif"/></a></td><td class="rightEqn">(24)</td></tr></table></p>
        <p class="otherpara">Π(<span class="italic">x</span>,<span class="italic">y</span>) can be understood to be the optimal “transport plan” explaining how much probability mass is moved from <span class="italic">x</span> to <span class="italic">y</span>. Another feature of the WGAN is the introduction of a “Lipschitz constraint” which clamps the weights of the discriminator to lie in a fixed interval. The Lipschitz constraint has been found to result in a more reliable gradient signal for the generator and improve training stability. Many other types of GAN objective function have been developed which we do not have room to discuss here. For a review of the major GAN objective functions and latest techniques, see Kurach <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit142"><sup><span class="sup_ref">142</span></sup></a></p>
        <p class="otherpara">Several papers have emerged so far applying GANs to molecular generation—Guimaraes <span class="italic">et al.</span> (ORGAN),<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> Sanchez-Lengeling <span class="italic">et al.</span> (ORGANIC),<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a> De Cao &amp; Kipf (MolGAN),<a title="Select to navigate to reference" href="#cit50"><sup><span class="sup_ref">50,63</span></sup></a> and Putin <span class="italic">et al.</span> (RANC, ATNC).<a title="Select to navigate to reference" href="#cit104"><sup><span class="sup_ref">104,105</span></sup></a> The Objective-Reinforced GAN (ORGAN) of Guimares <span class="italic">et al.</span> uses the SMILES molecular representation and an RNN (LSTM) generator and a CNN discriminator.<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> The architecture of the ORGAN is taken from the SeqGAN of Yu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit143"><sup><span class="sup_ref">143</span></sup></a> and uses a WGAN. In ORGAN, the GAN objective function is modified by adding an additional “objective reinforcement” term to the generator RNN's reward function which biases the RNN to produce molecules with a certain objective property or set of objective properties. Typically the objective function returns a value <span class="italic">R</span>(<span class="italic">S</span>) ∈ [0, 1]. The reward for a SMILES string <span class="italic">S</span> becomes a mixture of two terms:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn25"/><span id="eqn25"><span class="italic">R</span>(<span class="italic">S</span>) = <span class="italic">λD</span>(<span class="italic">S</span>) + (1 − <span class="italic">λ</span>)<span class="italic">R</span>(<span class="italic">S</span>)</span></td><td class="rightEqn">(25)</td></tr></table>where <span class="italic">λ</span> ∈ [0, 1] is a tunable hyperparameter which sets the mixing between rewards for fooling the discriminator and maximizing the objective function. The proof of concept of the ORGAN was demonstrated by optimizing three quick-to-evaluate metrics which can be computed with RDKit—druglikeliness, synthesizability, and solubility. Proof of concept applications of the ORGAN have been demonstrated in two domains – firstly for drug design it as shown how ORGAN can be used to maximize Lapinksi's rule of five metric as well as the quantitative estimate of drug likeliness metric. The second application for which ORGAN was demonstrated is maximizing the power conversion efficiency (PCE) of molecules for use in organic photovoltaics, where PCE is estimated using a machine learning based predictor that was previously developed.<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a></p>
        <div>
          
          <span id="sect1378"/><span class="c_heading_indent">2.3.1 The perfect discriminator problem and training instabilities. </span>
          <span>GAN optimization is a saddle point optimization problem, and such problems are known to be inherently unstable. If gradients for one part of the optimization dominate, optimizers can run or “spiral” away from the saddle point so that either the generator or the discriminator achieves a perfect score. The traditional approach to avoiding the perfect discriminator problem, taken by Guimares <span class="italic">et al.</span> and others, is to do additional MLE pretraining with the generator to strengthen it and then do <span class="italic">m</span> gradient updates for the generator for every one gradient update for the discriminator. In this method, <span class="italic">m</span> must be tuned to balance the discriminator and generator training. A different, more dynamic method for balancing the discriminator and generator was invented by Kardurin <span class="italic">et al.</span> in their work on the DruGAN AAE.<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> They introduce a hyperparameter 0.5 &lt; <span class="italic">p</span> &lt; 1 which sets the desired “discriminator power”. Then, after each training step, if the discriminator correctly labels samples from the generator with probability less than <span class="italic">p</span>, the discriminator is trained, otherwise the generator is trained. Clearly <span class="italic">p</span> should be larger than 0.5 since the discriminator should do better than random chance in order to challenge the generator to improve. Empirically, they found <span class="italic">p</span> = 3/5 to be a good value.</span>
          <p class="otherpara">Putin <span class="italic">et al.</span> show that the ORGANIC model<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a> with its default hyperparameters suffers from a perfect discriminator problem during training, leading to a plateauing of the generator's loss.<a title="Select to navigate to references" href="#cit104"><sup><span class="sup_ref">104</span></sup></a> To help solve these issues, Putin <span class="italic">et al.</span> implemented a differentiable neural computer (DNC) as the generator.<a title="Select to navigate to references" href="#cit104"><sup><span class="sup_ref">104</span></sup></a> The DNC (Graves <span class="italic">et al.</span>, 2016)<a title="Select to navigate to references" href="#cit144"><sup><span class="sup_ref">144</span></sup></a> is an extension of the neural Turing machine (Graves <span class="italic">et al.</span> 2014)<a title="Select to navigate to references" href="#cit145"><sup><span class="sup_ref">145</span></sup></a> that contains a differentiable memory cell. A memory cell allows the generator to memorize key SMILES motifs, which results in a much “stronger” generator. They found that the discriminator never achieves a perfect score when training against a DNC. The strength of the DNC is also shown by the fact that it has a higher rate of valid SMILES generation <span class="italic">vs.</span> the ORGAN RNN generator (76% <span class="italic">vs.</span> 24%) and generates SMILES that are on average twice as long as the SMILES generated by ORGAN. In a subsequent work, Putin <span class="italic">et al.</span> also introduced the adversarial threshold neural computer, another architecture with a DNC generator.<a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a></p>
          <p class="otherpara">Another issue with GANs is mode collapse, where the generator only generates a narrow range of samples. In the context of molecules, an example might be a generator that only generates molecules with carbon rings and less than 20 atoms.</p>
        </div>
      
    
    
      
      <h2 id="sect1406"><span class="a_heading">3 Metrics and reward functions</span></h2>
      <span>A key issue in deep generative modeling research is how to quantitatively compare the performance of different generative models. More generally a decline in rigor in the field of deep learning as a whole has been noted by Sculley, Rahimi and others.<a title="Select to navigate to references" href="#cit146"><sup><span class="sup_ref">146</span></sup></a> While the recent growth in the number of researchers in the field has obvious benefits, the increased competition that can result from such rapid growth disincentivizes taking time for careful tuning and rigorous evaluation of new methods with previous ones. Published comparisons are often subtly biased towards demonstrating superior performance for technically novel methods <span class="italic">vs.</span> older more conventional methods. A study by Lucic <span class="italic">et al.</span> for instance found that in the field of generative adversarial networks better hyperparameter tuning and training lead to most recently proposed methods reaching very similar results.<a title="Select to navigate to reference" href="#cit140"><sup><span class="sup_ref">140,147</span></sup></a> Similarly, Melis <span class="italic">et al.</span> found that with proper hyperparameter tuning a conventional LSTM architecture could beat several more recently proposed methods for natural language modeling.<a title="Select to navigate to references" href="#cit148"><sup><span class="sup_ref">148</span></sup></a> At the same time, there is a reproducibility crisis afflicting deep learning—codes published side-by-side with papers often give different results than what was published,<a title="Select to navigate to references" href="#cit142"><sup><span class="sup_ref">142</span></sup></a> and in the field of reinforcement learning it has been found that codes which purport to do the same thing will give different results.<a title="Select to navigate to references" href="#cit147"><sup><span class="sup_ref">147</span></sup></a> The fields of deep learning and deep generative modeling are still young however, and much work is currently underway on developing new standards and techniques for rigorously comparing different methods. In this section we will discuss several of the recently proposed metrics for comparing generative models and the closely related topic of reward function design for molecular optimization.</span>
      
        
        <h3 id="sect1418"><span class="b_heading">3.1 Metrics for scoring generative models</span></h3>
        <span>Theis <span class="italic">et al.</span> discuss three separate approaches—log-likelihood, estimating the divergence metric between the training data distribution <span class="italic">p</span>(<span class="bold"><span class="italic">x</span></span>) and the model's distribution <span class="italic">q</span>(<span class="bold"><span class="italic">x</span></span>), and human rating by visual inspection (also called the “visual Turing test”).<a title="Select to navigate to reference" href="#cit149"><sup><span class="sup_ref">149,150</span></sup></a> Interestingly, they show that these three methodologies measure different things, so good performance under one does not imply good performance under another.<a title="Select to navigate to references" href="#cit150"><sup><span class="sup_ref">150</span></sup></a></span>
        <p class="otherpara">The “inception score” (IS) uses a third-party neural network which has been trained in a supervised manner to do classification.<a title="Select to navigate to references" href="#cit149"><sup><span class="sup_ref">149</span></sup></a> In the original IS, Google's Inception network trained on ImageNet was used as the third-party network. IS computes the divergence between the distribution of classes predicted by the third-party neural network on generated molecules with the distribution of classes predicted for the dataset used to train the neural network. The main weakness of IS is that much information about the quality of images is lost by focusing only on classification labels. The Fréchet Inception Distance (FID) builds off the IS by comparing latent vectors obtained from a late-stage layer of a third-party classification network instead of the predictions.<a title="Select to navigate to references" href="#cit151"><sup><span class="sup_ref">151</span></sup></a> Inspired by this, Preuer <span class="italic">et al.</span> created the Fréchet ChemNet Distance metric for evaluating models that generate molecules.<a title="Select to navigate to references" href="#cit152"><sup><span class="sup_ref">152</span></sup></a> Unfortunately, there is a lack of agreement on how to calculate the FID—some report the score by comparing training data with generated data, while others report the score comparing a hold out test set with the generated data.<a title="Select to navigate to references" href="#cit142"><sup><span class="sup_ref">142</span></sup></a> Comparing with test data gives a more useful metric which measures generalization ability, and is advocated as a standard by Kurach <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit142"><sup><span class="sup_ref">142</span></sup></a></p>
        <p class="otherpara">In the world of machine learning for molecular property prediction, <span class="italic">MoleculeNet</span> provides a benchmark to compare the utility of different regression modeling techniques across a wide range of property prediction problems.<a title="Select to navigate to references" href="#cit153"><sup><span class="sup_ref">153</span></sup></a> Inspired by <span class="italic">MoleculeNet</span>, Polykovskiy and collaborators have introduced the MOlecular SEtS (MOSES) package to make it easier to build and test generative models.<a title="Select to navigate to references" href="#cit154"><sup><span class="sup_ref">154</span></sup></a> To compare the output of generative models, they provide functions to compute Fréchet ChemNet Distance, internal diversity, as well as several metrics which are of general importance for pharmaceuticals: molecular weight, log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span>, synthetic accessibility score, and the quantitative estimation of drug-likeness. In a similar vein, Benhenda <span class="italic">et al.</span> have released the <span class="italic">DiversityNet</span> benchmark, which was also (as the name suggests) inspired by <span class="italic">MoleculeNet</span>.<a title="Select to navigate to references" href="#cit155"><sup><span class="sup_ref">155</span></sup></a> Finally, another Python software package called <span class="italic">GuacaMol</span> has also been released which contains 5 general purpose benchmarking methods and 20 “optimization specific” benchmarking methods for drug discovery.<a title="Select to navigate to references" href="#cit156"><sup><span class="sup_ref">156</span></sup></a> One unique feature of <span class="italic">GuacaMol</span> is the ability to compute KL-divergences between the distributions from generated molecules and training molecules for a variety of physio chemical descriptors.</p>
        <p class="otherpara">Recently in the context of generative modeling of images with GANs, Im <span class="italic">et al.</span> have shown significant pitfalls to using the Inception Distance metric.<a title="Select to navigate to references" href="#cit157"><sup><span class="sup_ref">157</span></sup></a> As an alternative, they suggest using the same type of divergence metrics that are used during GAN training. This method has been explored recently to quantify generalization performance of GANS<a title="Select to navigate to references" href="#cit158"><sup><span class="sup_ref">158</span></sup></a> and could be of use to the molecular modeling community.</p>
      
      
        
        <h3 id="sect1456"><span class="b_heading">3.2 Reward function design</span></h3>
        <span>A good reward function is often important for molecular generation and essential for molecular optimization. The pioneering molecular autoencoder work resulted in molecules which were difficult to synthesize or contained highly labile (reactive or unstable) groups such as enamines, hemiaminals, and enol ethers which would rapidly break apart in the body and thus were not viable drugs.<a title="Select to navigate to references" href="#cit159"><sup><span class="sup_ref">159</span></sup></a> Since then, the development of better reward functions has greatly helped to mitigate such issues, but low diversity and novelty remains an issue.<a title="Select to navigate to reference" href="#cit160"><sup><span class="sup_ref">160–162</span></sup></a> After reviewing the work that has been done so far on reward function design, we conclude that good reward functions should lead to generated molecules which meet the following desiderata:</span>
        <p class="otherpara">1. <span class="bold">Diversity</span>—the set of molecules generated is diverse enough to be interesting.</p>
        <p class="otherpara">2. <span class="bold">Novelty</span>—the set of molecules does not simply reproduce molecules in the training set.</p>
        <p class="otherpara">3. <span class="bold">Stability</span>—the molecules are stable in the target environment and not highly reactive.</p>
        <p class="otherpara">4. <span class="bold">Synthesizability</span>—the molecules can actually be synthesized.</p>
        <p class="otherpara">5. <span class="bold">Non-triviality</span>—the molecules are not degenerate or trivial solutions to maximizing the reward function.</p>
        <p class="otherpara">6. <span class="bold">Good properties</span>—the molecules have the properties desired for the application at hand.</p>
        <div>
          
          <span id="sect1474"/><span class="c_heading_indent">3.2.1 Diversity &amp; novelty. </span>
          <span>A diversity metric is a key component of any reward function, especially when using a GAN, where it helps counter the issue of mode collapse to a non-diverse subset. Given a molecular similarity metric between two molecules <span class="italic">T</span>(<span class="italic">x</span><small><sub>1</sub></small>,<span class="italic">x</span><small><sub>2</sub></small>) ∈ [0, 1] the diversity of a generated set <img class="charmap" src="https://www.rsc.org/images/entities/char_e112.gif" alt="[capital G, script]"/> can be defined quite simply as:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn26"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t24_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t24.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t24.gif"/></a></td><td class="rightEqn">(26)</td></tr></table></span>
          <p class="otherpara">A popular metric is the Tanimoto similarity between two extended-connectivity fingerprint bit vectors.<a title="Select to navigate to references" href="#cit154"><sup><span class="sup_ref">154</span></sup></a> Since the diversity of a single molecule does not make sense, diversity rewards are calculated on mini-batches during mini-batch stochastic gradient descent training. <a title="" href="#eqn26">Eqn (26)</a> is called “internal diversity”. An alternative which compares the diversity of the generated set with the diversity of the training data is the nearest neighbor similarity (SNN) metric:<a title="Select to navigate to references" href="#cit154"><sup><span class="sup_ref">154</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn27"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t25_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t25.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t25.gif"/></a></td><td class="rightEqn">(27)</td></tr></table></p>
          <p class="otherpara">Of course, too much diversity can also be an issue. One option is to use the following negative reward which biases the generator towards matching the diversity of the training data:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn28"/><span id="eqn28"><span class="italic">R</span><small><sub>diversity mismatch</sub></small> = −|<span class="italic">r</span><small><sup>generated</sup></small><small><sub>diversity</sub></small> − <span class="italic">r</span><small><sup>training</sup></small><small><sub>diversity</sub></small>|</span></td><td class="rightEqn">(28)</td></tr></table></p>
          <p class="otherpara">Another diversity measure that has been employed is uniqueness, which aims to reduce the number of repeat molecules. The uniqueness reward <span class="italic">R</span><small><sub>uniqueness</sub></small> ∈ [0, 1] is defined as:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn29"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t26_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t26.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t26.gif"/></a></td><td class="rightEqn">(29)</td></tr></table>where |set(<img class="charmap" src="https://www.rsc.org/images/entities/char_e112.gif" alt="[capital G, script]"/>)| is the number of unique molecules in the generated batch <img class="charmap" src="https://www.rsc.org/images/entities/char_e112.gif" alt="[capital G, script]"/> and |<img class="charmap" src="https://www.rsc.org/images/entities/char_e112.gif" alt="[capital G, script]"/>| is the total number of molecules.</p>
          <p class="otherpara">Novelty is just as important as diversity since a generator which just generates the training dataset over and over is of no practical utility. Guimaraes <span class="italic">et al.</span> define the “soft novelty” for a single molecule as:<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn30"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t27_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t27.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t27.gif"/></a></td><td class="rightEqn">(30)</td></tr></table></p>
          <p class="otherpara">When measuring the novelty of molecules generated post-training to get an overall novelty measure for the model, it is important to do so on a hold-out test set <img class="charmap" src="https://www.rsc.org/images/entities/char_e533.gif" alt="[scr T, script letter T]"/>. Then one can look at how many molecules in a set of generated molecules <img class="charmap" src="https://www.rsc.org/images/entities/char_e112.gif" alt="[capital G, script]"/> appear in <img class="charmap" src="https://www.rsc.org/images/entities/char_e533.gif" alt="[scr T, script letter T]"/> and use a novelty reward such as:<a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn31"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t28_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t28.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t28.gif"/></a></td><td class="rightEqn">(31)</td></tr></table>which gives the fraction of generated molecules not appearing in the test set. The diversity of the generated molecules and how they compare to the diversity of the training set can also be visualized by generating fingerprint vectors (which typically have dimensionalities of <span class="italic">d</span> &gt; 100) and then projecting them into two dimensions using dimensionality reduction techniques. The resulting 2D point cloud can then be compared with the corresponding points from the training set and/or a hold out test set. There are many possible dimensionality reduction techniques to choose from—Yoshikawa <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit161"><sup><span class="sup_ref">161</span></sup></a> use the ISOMAP method,<a title="Select to navigate to references" href="#cit163"><sup><span class="sup_ref">163</span></sup></a> Merk <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit81"><sup><span class="sup_ref">81</span></sup></a> use multidimensional scaling, and Segler <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> use t-SNE projection.<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a></p>
          <p class="otherpara">Interpolation between training molecules may be a useful way to generate molecules post-training which are novel, but not too novel as to be unstable or outside the intended property manifold. In the domain of image generation, GANs seem to excel at interpolation <span class="italic">vs.</span> VAEs, for reasons that are not yet fully understood. For instance with GANs trained on natural images, interpolation can be done between a <span class="bold"><span class="italic">z</span></span> point corresponding to a frowning man to a point <span class="bold"><span class="italic">z</span></span>′ corresponding to a smiling woman, and all of the intervening points result in images which make sense.<a title="Select to navigate to references" href="#cit165"><sup><span class="sup_ref">165</span></sup></a> Empirically most real world high dimensional datasets are found to lie on a low density manifold.<a title="Select to navigate to references" href="#cit166"><sup><span class="sup_ref">166</span></sup></a> Ideally, the dimensionality of the latent space <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>) used in a GAN, VAE, or AAE will correspond to the dimensionality of this manifold. If the dimensionality of <span class="italic">p</span>(<span class="bold"><span class="italic">z</span></span>) is higher than the intrinsic dimensionality of the data manifold, then interpolation can end up going “off manifold” into so-called “dead zones”. For high dimensional latent spaces with a Gaussian prior, most points will lie on a thin spherical shell. In such cases it has been found empirically that better results can be found by doing spherical linear interpolation or <span class="italic">slerp</span>.<a title="Select to navigate to references" href="#cit167"><sup><span class="sup_ref">167</span></sup></a> The equation for <span class="italic">slerp</span> interpolation between two vectors <span class="bold"><span class="italic">v</span></span><small><sub>1</sub></small> and <span class="bold"><span class="italic">v</span></span><small><sub>2</sub></small> is<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn32"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t29_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t29.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t29.gif"/></a></td><td class="rightEqn">(32)</td></tr></table>where <span class="italic">θ</span> = arccos(<span class="bold"><span class="italic">v</span></span><small><sub>1</sub></small>·<span class="bold"><span class="italic">v</span></span><small><sub>2</sub></small>) and 0 ≤ <span class="italic">t</span> ≤ 1 is the interpolation parameter.</p>
          <p class="otherpara">Another option for generating molecules close to training molecules but not too close is to have a reward for being similar to the training data but not too similar. Olivecrona <span class="italic">et al.</span> use a reward function <span class="italic">R</span><small><sub>s</sub></small>(<span class="italic">S</span>) ∈ [−1, 1] of the form:<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn33"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t30_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t30.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t30.gif"/></a></td><td class="rightEqn">(33)</td></tr></table>here <span class="italic">S</span> is the input SMILES and <span class="italic">T</span> is the target SMILES, and Sim(<span class="italic">S</span>,<span class="italic">T</span>) ∈ [0, 1] is similarity scoring function which computes fingerprint-based similarity between the two molecules. <span class="italic">k</span> is a tunable cutoff parameter which sets the maximum similarity accepted. This type of reward can be particularly useful for generating focused libraries of molecules very similar to a single target molecule or a small set of “actives” which are known to bind to a receptor. Note that most generative models can be run so as to generate molecules close to a given molecule. For instance, with RNNs, one can do “fragment growing”, which allows molecular designers to explore molecules which share a predefined scaffold.<a title="Select to navigate to references" href="#cit73"><sup><span class="sup_ref">73</span></sup></a> Similarly, with reinforcement learning one can simply start the agent with a particular molecule and let it add or remove bonds. Finally, with a VAE one can find the latent representation for a given molecule and then inject a small amount of Gaussian noise to generate “nearby” molecules.<a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect1583"/><span class="c_heading_indent">3.2.2 Stability and synthesizability. </span>
          <span>Enforcement of synthesizability has thus far mainly been done using the synthetic accessibility (SA) score developed by Ertl &amp; Schuffenhauer,<a title="Select to navigate to references" href="#cit168"><sup><span class="sup_ref">168</span></sup></a> although other synthesizability scoring functions exist.<a title="Select to navigate to reference" href="#cit169"><sup><span class="sup_ref">169,170</span></sup></a> The model underlying the SA score was designed and fit specifically to match scores from synthetic chemists on a set of drug molecules, and therefore may be of limited applicability to other types of molecules. When using SA score as a reward in their molecular autoencoder, Gómez-Bombarelli <span class="italic">et al.</span> found that it still produced a large number of molecules with unrealistically large carbon rings. Therefore, they added an additional “RingPenalty” term to penalize rings with more than six carbons. In the ORGANIC GAN code, Sanchez-Lengeling <span class="italic">et al.</span> added several penalty terms to the original SA score function, and also developed an additional reward for chemical symmetry, based on the observation that symmetric molecules are typically easier to synthesize.<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a></span>
          <p class="otherpara">For drug molecules, the use of scoring functions developed to estimate how “drug-like” or “natural” a compound is can also help improve the synthesizability, stability, and usefulness of the generated molecules.<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> Examples of such functions include Lipinski's Rule of Five score,<a title="Select to navigate to references" href="#cit5"><sup><span class="sup_ref">5</span></sup></a> the natural product-likeness score,<a title="Select to navigate to references" href="#cit171"><sup><span class="sup_ref">171</span></sup></a> the quantitative estimate of drug-likeness,<a title="Select to navigate to references" href="#cit172"><sup><span class="sup_ref">172</span></sup></a> and the Muegge metrics.<a title="Select to navigate to reference" href="#cit173"><sup><span class="sup_ref">173,174</span></sup></a> Another option of particular utility to drug discovery is to apply medicinal chemistry filters either during training or post-training to tag unstable, toxic, or unsynthesizable molecules. For drug molecules, catalogs of problematic functional groups to avoid have been developed in order to limit the probability of unwanted side-effects.<a title="Select to navigate to references" href="#cit175"><sup><span class="sup_ref">175</span></sup></a> For energetic molecules and other niche domains an analogous set of functional groups to avoid has yet to be developed.</p>
          <p class="otherpara">Many software packages exist for checking molecule's stability and synthesizability which may be integrated into training or as a post-training filter. For example Popova <span class="italic">et al.</span> use the ChemAxon structure checker software to do a validity check on the generated molecules.<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> Bjerrum <span class="italic">et al.</span> use the Wiley ChemPlanner software post-training to find synthesis routes for 25–55% of the molecules generated by their RNN.<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> Finally, Sumita <span class="italic">et al.</span> check for previously discovered synthetic routes for their generated molecules using a SciFinder literature search.<a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a></p>
          <p class="otherpara">It is worth mentioning that deep learning is now being used for the prediction of chemical reactions<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11,12</span></sup></a> and synthesis planning. Segler <span class="italic">et al.</span> trained a deep reinforcement learning system on a large set of reactions that have been published in organic chemistry and demonstrated a system that could predict chemical reactions and plan synthetic routes at the level of expert chemists.<a title="Select to navigate to references" href="#cit13"><sup><span class="sup_ref">13</span></sup></a></p>
        </div>
        <div>
          
          <span id="sect1610"/><span class="c_heading_indent">3.2.3 Rewards for good properties. </span>
          <span>Because they are called often during training, reward functions should be quick to compute, and therefore fast property estimators are called for. Examples of property estimation functions which are fast to evaluate are the estimated octanol–water partition coefficient (log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span>), and the quantitative measure of drug-likeness (QED),<a title="Select to navigate to references" href="#cit172"><sup><span class="sup_ref">172</span></sup></a> both of which can be found in the open source package RDKit.<a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a></span>
          <p class="otherpara">Since physics based prediction is usually very computationally intensive, a popular approach is to train a property predicting machine learning model ahead of time. There is now an enormous literature on deep learning for property prediction demonstrating success in multiple areas.<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a> Impressive results have been obtained, such as systems which can predict molecular energies at DFT accuracy,<a title="Select to navigate to references" href="#cit176"><sup><span class="sup_ref">176</span></sup></a> and highly accurate systems which can transfer between many types of molecules.<a title="Select to navigate to references" href="#cit177"><sup><span class="sup_ref">177</span></sup></a> While the literature on quantum property prediction is perhaps the most developed, success has been seen in other areas, such as calculating high level properties of energetic molecular crystals (such as sensitivity and detonation velocity).<a title="Select to navigate to reference" href="#cit10"><sup><span class="sup_ref">10,27</span></sup></a> Many predictive models are now published for “off the shelf” use, for instance a collection of predictive models for ADME (absorption, distribution, metabolism, and excretion) called “SwissADME” was recently published.<a title="Select to navigate to references" href="#cit176"><sup><span class="sup_ref">176</span></sup></a></p>
          <p class="otherpara">It has also been demonstrated that traditional physics-based simulations can be used—Sumita <span class="italic">et al.</span> optimize absorption wavelength by converting SMILES strings into 3D structures using RDKit and then calculating their UV-VIS absorption spectra on-the-fly with time-dependent density functional theory (TD-DFT). Instead of the obvious reward function −<span class="italic">α</span>|<span class="italic">λ</span>* − <span class="italic">λ</span>|, where <span class="italic">λ</span>* is the target wavelength, they used the following:<a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn34"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t31_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t31.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t31.gif"/></a></td><td class="rightEqn">(34)</td></tr></table></p>
          <p class="otherpara">From the molecules generated by their RNN, Sumita <span class="italic">et al.</span> selected six molecules for synthesis and found that 5/6 exhibited the desired absorption profiles.<a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a></p>
          <p class="otherpara">A reward function which has been used by several different researchers to generate drug molecules is:<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn35"/><span id="eqn35"><span class="italic">J</span>(<span class="italic">S</span>) = log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span>(<span class="italic">S</span>) − SA(<span class="italic">S</span>) − Ring Penalty(<span class="italic">S</span>)</span></td><td class="rightEqn">(35)</td></tr></table></p>
          <p class="otherpara">Yang <span class="italic">et al.</span> add an additional penalty for generating invalid SMILES which could be used more broadly:<a title="Select to navigate to references" href="#cit76"><sup><span class="sup_ref">76</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn36"/><a href="/image/article/2019/ME/c9me00039a/c9me00039a-t32_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: c9me00039a-t32.tif" src="/image/article/2019/ME/c9me00039a/c9me00039a-t32.gif"/></a></td><td class="rightEqn">(36)</td></tr></table></p>
          <p class="otherpara">In the context of training the ORGAN architecture, Guimaraes <span class="italic">et al.</span> found that rotating the reward function metric from epoch to epoch had some advantages to using all metrics at once.<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> In other words, in one epoch the rewards may just be for diversity, while in the next they would just be for synthesizability, and so on. This idea could likely be explored further.</p>
        </div>
      
    
    
      
      <h2 id="sect1655"><span class="a_heading">4 Prospective and future directions</span></h2>
      <span>In this review we have tried to summarize the current state of the art for generative modeling of molecules using deep learning. The current literature is composed of a rich array of representation strategies and model architectures. As in many areas of generative modeling and deep learning, the present day work is largely empirical in nature. As our mathematical understanding of the landscape of generative models improves, so too will our ability to select the best approaches to a particular problem. There are many promising new techniques and architectures from deep generative modeling and optimization more broadly which are ripe to be transferred to the world of molecules. For example, for sequence modeling the Maximum Likelihood Augmented Discrete GAN (MaliGAN) has been shown to be superior to the SeqGAN on which ORGAN is based.<a title="Select to navigate to references" href="#cit178"><sup><span class="sup_ref">178</span></sup></a> With RNNs, recently developed attention mechanisms and external memory cells offer a possible avenue to improve SMILES string generation.<a title="Select to navigate to references" href="#cit179"><sup><span class="sup_ref">179</span></sup></a></span>
      <p class="otherpara">It is worth noting that the latest genetic algorithm based methods can still compete with today's deep learning based methods. Yoshikawa <span class="italic">et al.</span> developed a genetic algorithm which makes edits to SMILES and uses population-based evolutionary selection to generate molecules with high binding affinity as calculated <span class="italic">via</span> docking (rDock).<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> They compared their method with three other deep-learning based methods (CVAE,<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> GVAE,<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> and ChemTS<a title="Select to navigate to references" href="#cit76"><sup><span class="sup_ref">76</span></sup></a>) for optimizing the “penalized log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span> score” (<a title="" href="#eqn35">eqn (35))</a>. They found that with computer time fixed to eight hours, their method performed better or comparable to the deep learning methods. In a similar vein, Jensen found that a genetic algorithm performed better than a SMILES based RNN + Bayesian optimization, the ChemTS RNN, and a CVAE with 100x lower computational cost.<a title="Select to navigate to references" href="#cit180"><sup><span class="sup_ref">180</span></sup></a> It appears that genetic algorithms can explore chemical space in a very efficient manner.</p>
      <p class="otherpara">In our discussion of GANs we highlighted an important way in which GANs are superior to maximum likelihood based methods—namely that they can avoid the “filling in” problem which occurs with maximum likelihood where the model's distribution ends up non-zero where the data distribution is zero. Another point is that the theorems on which the maximum likelihood methodology is based only hold true in the limit of infinite samples.<a title="Select to navigate to references" href="#cit181"><sup><span class="sup_ref">181</span></sup></a> In general it appears that GANs can be trained with far fewer samples than maximum likelihood based methods—this can be seen by looking at the <span class="italic">N</span><small><sub>train</sub></small> values in <a title="Select to navigate to table" href="#tab2">Table 2</a>. In addition to their benefits, we also touched on several difficulties with GANs—small starting gradient, training instabilities, the perfect discriminator problem, and mode collapse. However, we also cited possible remedies for each of these issues and we expect more remedies to be developed in the future.</p>
      
<div class="table_caption"><b>Table 2</b> <span id="tab2">For works that trained models separately on multiple datasets we report only the largest dataset used. Several of these datasets are described in <a title="Select to navigate to table" href="#tab3">Table 3</a>, which lists the major publicly available datasets. Other datasets are “HCEP”, the Harvard Clean Energy Project dataset of lead molecules for organic photovoltaic, “PSC”, a dataset of monomer repeat units for polymer solar cells, “MCF-7”, a database of anti-cancer molecules, and “L1000”, a database of molecules and gene expression profiles</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/></colgroup>
            
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Architecture</th>
                <th align="left" valign="bottom" class="border_black">Representation</th>
                <th align="left" valign="bottom" class="border_black">
                  <span class="italic">N</span>
                  <small><sub>train</sub></small>
                </th>
                <th align="left" valign="bottom" class="border_black">Dataset(s)</th>
                <th align="left" valign="bottom" class="border_black">Citation(s)</th>
              </tr>
            </thead>
            <tfoot align="left">
              <tr align="left">
                <th colspan="5">
                  <span>Acronyms used are: AAE = adversarial autoencoder, ANC = adversarial neural computer, ATNC = adversarial threshold neural computer, BMI = Bayesian model inversion, CAAE = constrained AAE, CCM-AAE = constant-curvature Riemannian manifold AAE, CFG = context free grammar, CVAE = constrained VAE, ECC = edge-conditioned graph convolutions,<a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> GAN = generative adversarial network, GCPN = graph convolutional policy network, GVAE = grammar VAE, JT-VAE = junction tree VAE, MHG = molecular hypergraph grammar, RG = reduced graph, RNN = recurrent neural network, sGAN = stacked GAN, SD-VAE = syntax-directed VAE, SSVAE = semi-supervised VAE, VAE = variational autoencoder.</span>
                  <a id="tab2fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Filtered to isolate likely bioactive compounds.</span>
                </th>
              </tr>
            </tfoot>
            <tbody align="left">
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>611<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>889</td>
                <td valign="top">ZINC</td>
                <td valign="top">Bjerrum, 2017 (<a title="Select to navigate to references" href="#cit72">ref. 72</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">541<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>555</td>
                <td valign="top">ChEMBL<i><small><sup style="color: red;"><a title="" href="#tab2fna">a</a></sup></small></i></td>
                <td valign="top">Gupta, 2017 (<a title="Select to navigate to references" href="#cit73">ref. 73</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">350<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>419</td>
                <td valign="top">DRD2</td>
                <td valign="top">Olivecrona, 2017 (<a title="Select to navigate to references" href="#cit74">ref. 74</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>400<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ChEMBL</td>
                <td valign="top">Segler, 2017 (<a title="Select to navigate to references" href="#cit75">ref. 75</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Yang, 2017 (<a title="Select to navigate to references" href="#cit76">ref. 76</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">200<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Cherti, 2017 (<a title="Select to navigate to references" href="#cit77">ref. 77</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>735<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>442</td>
                <td valign="top">ChEMBL</td>
                <td valign="top">Neil, 2018 (<a title="Select to navigate to references" href="#cit78">ref. 78</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>500<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ChEMBL</td>
                <td valign="top">Popova, 2018 (<a title="Select to navigate to references" href="#cit79">ref. 79</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">13<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">PubChemQC</td>
                <td valign="top">Sumita, 2018 (<a title="Select to navigate to references" href="#cit80">ref. 80</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">541<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>555</td>
                <td valign="top">ChEMBL<i><small><sup style="color: red;"><a title="" href="#tab2fna">a</a></sup></small></i></td>
                <td valign="top">Merk, 2018 (<a title="Select to navigate to references" href="#cit81">ref. 81</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">541<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>555</td>
                <td valign="top">ChEMBL<i><small><sup style="color: red;"><a title="" href="#tab2fna">a</a></sup></small></i></td>
                <td valign="top">Merk, 2018 (<a title="Select to navigate to references" href="#cit82">ref. 82</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">509<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ChEMBL</td>
                <td valign="top">Ertl, 2018 (<a title="Select to navigate to references" href="#cit83">ref. 83</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">GDB-13</td>
                <td valign="top">Arús-Pous, 2018 (<a title="Select to navigate to references" href="#cit84">ref. 84</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">SMILES</td>
                <td valign="top">163<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Zheng, 2019 (<a title="Select to navigate to references" href="#cit85">ref. 85</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">RG + SMILES</td>
                <td valign="top">798<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>243</td>
                <td valign="top">ChEMBL</td>
                <td valign="top">Pogány, 2018 (<a title="Select to navigate to references" href="#cit86">ref. 86</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">RNN</td>
                <td valign="top">Graph operations</td>
                <td valign="top">130<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>830</td>
                <td valign="top">ChEMBL</td>
                <td valign="top">Li, 2018 (<a title="Select to navigate to references" href="#cit59">ref. 59</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">249<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC/QM9</td>
                <td valign="top">Gómez-Bombarelli, 2016 (<a title="Select to navigate to references" href="#cit23">ref. 23</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>200<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ChEMBL</td>
                <td valign="top">Blaschke, 2018 (<a title="Select to navigate to references" href="#cit87">ref. 87</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">500<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Lim, 2018 (<a title="Select to navigate to references" href="#cit88">ref. 88</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">300<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Kang, 2018 (<a title="Select to navigate to references" href="#cit89">ref. 89</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">190<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Harel, 2018 (<a title="Select to navigate to references" href="#cit90">ref. 90</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>211<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>352</td>
                <td valign="top">ChEMBL23</td>
                <td valign="top">Sattarov, 2019 (<a title="Select to navigate to references" href="#cit91">ref. 91</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GVAE</td>
                <td valign="top">CFG (SMILES)</td>
                <td valign="top">200<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Kusner, 2017 (<a title="Select to navigate to references" href="#cit92">ref. 92</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GVAE</td>
                <td valign="top">CFG (custom)</td>
                <td valign="top">3989</td>
                <td valign="top">PSC</td>
                <td valign="top">Jørgensen, 2018 (<a title="Select to navigate to reference" href="#cit26">ref. 26 and 93</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">SD-VAE</td>
                <td valign="top">CFG (custom)</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Dai, 2018 (<a title="Select to navigate to references" href="#cit58">ref. 58</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">JT-VAE</td>
                <td valign="top">Graph operations</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Jin, 2018 (<a title="Select to navigate to references" href="#cit94">ref. 94</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">JT-VAE</td>
                <td valign="top">Graph operations</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Jin, 2019 (<a title="Select to navigate to references" href="#cit95">ref. 95</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">CVAE</td>
                <td valign="top">Graph</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC/CEPDB</td>
                <td valign="top">Liu, 2018 (<a title="Select to navigate to references" href="#cit96">ref. 96</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">MHG-VAE</td>
                <td valign="top">Graph (MHG)</td>
                <td valign="top">220<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>011</td>
                <td valign="top">ZINC</td>
                <td valign="top">Kajino, 2018 (<a title="Select to navigate to references" href="#cit97">ref. 97</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">Graph</td>
                <td valign="top">72<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC + PubChem</td>
                <td valign="top">Winter, 2018 (<a title="Select to navigate to references" href="#cit98">ref. 98</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">Graph</td>
                <td valign="top">10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC/QM9</td>
                <td valign="top">Samanta, 2018 (<a title="Select to navigate to references" href="#cit99">ref. 99</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">Graph (tensors)</td>
                <td valign="top">10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Samanta, 2018 (<a title="Select to navigate to references" href="#cit100">ref. 100</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">Graph (tensors)</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC/QM9</td>
                <td valign="top">Simonovsky, 2018 (<a title="Select to navigate to references" href="#cit64">ref. 64</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">CVAE</td>
                <td valign="top">Graph (tensors)</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Ma, 2018 (<a title="Select to navigate to references" href="#cit101">ref. 101</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE</td>
                <td valign="top">3D wave transform</td>
                <td valign="top">48<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Kuzminykh, 2018 (<a title="Select to navigate to references" href="#cit40">ref. 40</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">CVAE</td>
                <td valign="top">3D density</td>
                <td valign="top">192<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>813<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>983</td>
                <td valign="top">ZINC</td>
                <td valign="top">Skalic, 2019 (<a title="Select to navigate to references" href="#cit41">ref. 41</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">VAE + RL</td>
                <td valign="top">MPNN + graph ops</td>
                <td valign="top">133<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>885</td>
                <td valign="top">QM9</td>
                <td valign="top">Kearnes, 2019 (<a title="Select to navigate to references" href="#cit102">ref. 102</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GAN</td>
                <td valign="top">SMILES</td>
                <td valign="top">5000</td>
                <td valign="top">GBD-17</td>
                <td valign="top">Guimaraes, 2017 (<a title="Select to navigate to references" href="#cit103">ref. 103</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GAN (ANC)</td>
                <td valign="top">SMILES</td>
                <td valign="top">15<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC/CHEMDIV</td>
                <td valign="top">Putin, 2018 (<a title="Select to navigate to references" href="#cit104">ref. 104</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GAN (ATNC)</td>
                <td valign="top">SMILES</td>
                <td valign="top">15<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC/CHEMDIV</td>
                <td valign="top">Putin, 2018 (<a title="Select to navigate to references" href="#cit105">ref. 105</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GAN</td>
                <td valign="top">Graph (tensors)</td>
                <td valign="top">133<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>885</td>
                <td valign="top">QM9</td>
                <td valign="top">De Cao, 2018 (<a title="Select to navigate to reference" href="#cit50">ref. 50 and 63</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GAN</td>
                <td valign="top">MACCS (166 bit)</td>
                <td valign="top">6252</td>
                <td valign="top">MCF-7</td>
                <td valign="top">Kadurin, 2017 (<a title="Select to navigate to references" href="#cit70">ref. 70</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">sGAN</td>
                <td valign="top">MACCS (166 bit)</td>
                <td valign="top">20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">L1000</td>
                <td valign="top">Méndez-Lucio, 2017 (<a title="Select to navigate to references" href="#cit106">ref. 106</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">CycleGAN</td>
                <td valign="top">Graph operations</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Maziarka, 2019 (<a title="Select to navigate to references" href="#cit107">ref. 107</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">AAE</td>
                <td valign="top">MACCS (166 bit)</td>
                <td valign="top">6252</td>
                <td valign="top">MCF-7</td>
                <td valign="top">Kadurin, 2017 (<a title="Select to navigate to references" href="#cit69">ref. 69</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">AAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">15<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">HCEP</td>
                <td valign="top">Sanchez-Lengeling, 2017 (<a title="Select to navigate to references" href="#cit108">ref. 108</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">CCM-AAE</td>
                <td valign="top">Graph (tensors)</td>
                <td valign="top">133<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>885</td>
                <td valign="top">QM9</td>
                <td valign="top">Grattarola, 2018 (<a title="Select to navigate to references" href="#cit109">ref. 109</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">BMI</td>
                <td valign="top">SMILES</td>
                <td valign="top">16<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>674</td>
                <td valign="top">PubChem</td>
                <td valign="top">Ikebata, 2017 (<a title="Select to navigate to references" href="#cit110">ref. 110</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">CAAE</td>
                <td valign="top">SMILES</td>
                <td valign="top">1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>800<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">Polykovskiy, 2018 (<a title="Select to navigate to references" href="#cit111">ref. 111</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">GCPN</td>
                <td valign="top">Graph</td>
                <td valign="top">250<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">ZINC</td>
                <td valign="top">You, 2018 (<a title="Select to navigate to references" href="#cit65">ref. 65</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">Pure RL</td>
                <td valign="top">Graph</td>
                <td valign="top">n/a</td>
                <td valign="top">n/a</td>
                <td valign="top">Zhou, 2018 (<a title="Select to navigate to references" href="#cit66">ref. 66</a>)</td>
              </tr>
              <tr align="left">
                <td valign="top">Pure RL</td>
                <td valign="top">Fragments</td>
                <td valign="top">n/a</td>
                <td valign="top">n/a</td>
                <td valign="top">Ståhl, 2019 (<a title="Select to navigate to references" href="#cit112">ref. 112</a>)</td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
<div class="table_caption"><b>Table 3</b> <span id="tab3">Some publicly available datasets</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Dataset</th>
                <th align="left" valign="bottom" class="border_black">Description</th>
                <th align="left" valign="bottom" class="border_black">
                  <span class="italic">N</span>
                </th>
                <th align="left" valign="bottom" class="border_black">URL/citation</th>
              </tr>
            </thead>
            <tfoot align="left">
              <tr align="left">
                <th colspan="4">
                  <a id="tab3fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Also contains numerous conformers for each molecule, for a total of 4855 structures.</span>
                </th>
              </tr>
            </tfoot>
            <tbody align="left">
              <tr align="left">
                <td valign="top">GDB-13</td>
                <td valign="top">Combinatorially generated library</td>
                <td align="char" char="." valign="top">977<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>468<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>314</td>
                <td valign="top">
                  http://gdb.unibe.ch/downloads/
                  <a title="Select to navigate to references" href="#cit113"><sup><span class="sup_ref">113</span></sup></a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">ZINC15</td>
                <td valign="top">Commercially available compounds</td>
                <td align="char" char="." valign="top">&gt;750<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">
                  http://zinc15.docking.org
                  <a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">GDB-17</td>
                <td valign="top">Combinatorially generated library</td>
                <td align="char" char="." valign="top">50<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">
                  http://gdb.unibe.ch/downloads/
                  <a title="Select to navigate to references" href="#cit115"><sup><span class="sup_ref">115</span></sup></a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">eMolecules</td>
                <td valign="top">Commercially available compounds</td>
                <td align="char" char="." valign="top">18<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">
                  https://reaxys.emolecules.com/
                </td>
              </tr>
              <tr align="left">
                <td valign="top">SureChEMBL</td>
                <td valign="top">Compounds obtained from chemical patents</td>
                <td align="char" char="." valign="top">17<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">
                  https://www.surechembl.org/search/
                </td>
              </tr>
              <tr align="left">
                <td valign="top">PubChemQC</td>
                <td valign="top">Compounds from PubChem with property data from quantum chemistry (DFT) calculations</td>
                <td align="char" char="." valign="top">3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>981<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>230</td>
                <td valign="top">
                  http://pubchemqc.riken.jp/
                  <a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">ChEMBL</td>
                <td valign="top">A curated database of bioactive molecules</td>
                <td align="char" char="." valign="top">2<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">
                  https://www.ebi.ac.uk/chembl/
                </td>
              </tr>
              <tr align="left">
                <td valign="top">SuperNatural</td>
                <td valign="top">A curated database of natural products</td>
                <td align="char" char="." valign="top">2<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">
                  http://bioinformatics.charite.de/supernatural/
                </td>
              </tr>
              <tr align="left">
                <td valign="top">QM9</td>
                <td valign="top">Stable small CHONHF organic molecules taken from GDB-17 with properties calculated from <span class="italic">ab initio</span> density functional theory</td>
                <td align="char" char="." valign="top">133<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>885</td>
                <td valign="top">
                  http://quantum-machine.org/datasets/
                  <a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">BNPAH</td>
                <td valign="top">B, <span class="italic">N</span>-substituted polycyclic aromatic hydrocarbons with properties calculated from <span class="italic">ab initio</span> density functional theory</td>
                <td align="char" char="." valign="top">33<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                <td valign="top">
                  https://moldis.tifrh.res.in/datasets.html
                  <a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">DrugBank</td>
                <td valign="top">FDA drugs and other drugs available internationally</td>
                <td align="char" char="." valign="top">10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>500</td>
                <td valign="top">
                  https://www.drugbank.ca/
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Energetics</td>
                <td valign="top">Energetic molecules and simulation data collected from public domain literature</td>
                <td align="char" char="." valign="top">417</td>
                <td valign="top">
                  https://git.io/energeticmols
                  <a title="Select to navigate to references" href="#cit27"><sup><span class="sup_ref">27</span></sup></a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">HOPV15</td>
                <td valign="top">Harvard organic photovoltaic dataset</td>
                <td align="char" char="." valign="top">350<i><small><sup style="color: red;"><a title="" href="#tab3fna">a</a></sup></small></i></td>
                <td valign="top">
                  https://figshare.com/articles/HOPV15_Dataset/1610063/4 (<a title="Select to navigate to references" href="#cit119">ref. 119</a>)</td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      <p class="otherpara">There are several major trends we have observed which present day practitioners and those entering the field should be cognizant of:</p>
      <p class="otherpara">
        <span class="bold">New representation methods</span>. SMILES based techniques are quickly being replaced with techniques that work directly with chemical graphs and three dimensional chemical structures. Directly working with chemical graphs, either by using chemistry-preserving graph operations or tensor representations avoids the problems associated with requiring deep generative models to learn SMILES syntax. At the same time, there is also growing interest in generative models which can generate 3D equilibrium structures, since in drug design and other specialized applications the 3D geometry of molecules is important.</p>
      <p class="otherpara">
        <span class="bold">Better reward functions</span>. As mentioned earlier, reward function design is a critical component to molecular generation and optimization. We expect future work will use more sophisticated reward functions which combine multiple objectives into a single reward function. Using multiple reward functions and multi-objective reinforcement learning is also a promising approach.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a></p>
      <p class="otherpara">
        <span class="bold">Pure reinforcement learning approaches</span>. The deep reinforcement learning work of Zhou <span class="italic">et al.</span> demonstrated superior molecular optimization performance when compared with the Junction Tree VAE,<a title="Select to navigate to references" href="#cit94"><sup><span class="sup_ref">94</span></sup></a> ORGAN,<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> and Graph Convolutional Policy Network<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> approaches when optimizing the log<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><span class="italic">P</span> and QED metrics.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> The work of Zhou <span class="italic">et al.</span> is notable as it is the first to take a pure RL approach with no pretrained generator. We believe much future work in molecular optimization will proceed in this direction since many application areas have limited training data available.</p>
      <p class="otherpara">
        <span class="bold">Hierarchical modeling</span>. Hierarchical representation schemes will allow for efficient generative modeling of large molecules (such as proteins<a title="Select to navigate to reference" href="#cit182"><sup><span class="sup_ref">182,183</span></sup></a>) as well as complex systems such as polymers, metal organic frameworks, and molecular crystals. Generative modeling techniques will also be useful not just for optimizing molecules but also optimizing the structures and systems in which they are embedded. GANs have recently been applied to the generation of crystal structures<a title="Select to navigate to references" href="#cit184"><sup><span class="sup_ref">184</span></sup></a> and microstructures.<a title="Select to navigate to reference" href="#cit185"><sup><span class="sup_ref">185–187</span></sup></a> Hierarchical GANs<a title="Select to navigate to references" href="#cit188"><sup><span class="sup_ref">188</span></sup></a> may be useful for the generation of many-molecule complexes or for the simultaneous optimization of both material and geometry in nanomaterials and metamaterials.</p>
      <p class="otherpara">
        <span class="bold">Closing the loop</span>. After the synthesis and characterization of new lead compounds the data obtained can be fed back to improve the models used, a process called “closing the loop”. More work is needed to develop workflows and methods to interface and integrate generative models into laboratory platforms to allow for rapid feedback and cycling. A key challenge is developing useful software and cyberinfrastructure for computational screening and data management.<a title="Select to navigate to references" href="#cit189"><sup><span class="sup_ref">189</span></sup></a> The potential for efficiency improvements <span class="italic">via</span> automated AI-assisted synthesis planning and “self-driving” robotic laboratories is quite profound.<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">14,15,190–192</span></sup></a></p>
    
    
      <h2 id="sect2205"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <h2 id="sect2209"><span class="a_heading">Acknowledgements</span></h2>
      <span>Support for this work is gratefully acknowledged from the U.S. Office of Naval Research under grant number N00014-17-1-2108 and from the Energetics Technology Center under project number 2044-001. Partial support is also acknowledged from the Center for Engineering Concepts Development in the Department of Mechanical Engineering at the University of Maryland, College Park. We thank Dr. Ruth M. Doherty, Dr. William Wilson, and Dr. Andrey Gorlin for their input and for proofreading the manuscript.</span>
    
    <span id="sect2211"><h2 id="sect2208"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">J. A. DiMasi, H. G. Grabowski and R. W. Hansen, <span class="italic">J. Health. Med. Econ.</span>, 2016, <span class="bold">47</span>, 20–33 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jhealeco.2016.01.012" title="DOI Link to resource 10.1016/j.jhealeco.2016.01.012">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26928437%5Buid%5D" title="PubMed Link to resource 26928437">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/1_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit2">S. M. Paul, D. S. Mytelka, C. T. Dunwiddie, C. C. Persinger, B. H. Munos, S. R. Lindborg and A. L. Schacht, <span class="italic">Nat. Rev. Drug Discovery</span>, 2010, <span class="bold">9</span>, 203–214 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nrd3078" title="DOI Link to resource 10.1038/nrd3078">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXitFemsbg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20168317%5Buid%5D" title="PubMed Link to resource 20168317">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/2_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit3">A. Homburg, <span class="italic">Propellants, Explos., Pyrotech.</span>, 2017, <span class="bold">42</span>, 851–853 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/prep.201780831" title="DOI Link to resource 10.1002/prep.201780831">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/3_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit4">P. G. Polishchuk, T. I. Madzhidov and A. Varnek, <span class="italic">J. Comput.-Aided Mol. Des.</span>, 2013, <span class="bold">27</span>, 675–679 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10822-013-9672-4" title="DOI Link to resource 10.1007/s10822-013-9672-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWjtbjM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23963658%5Buid%5D" title="PubMed Link to resource 23963658">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/4_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit5">C. A. Lipinski, F. Lombardo, B. W. Dominy and P. J. Feeney, <span class="italic">Adv. Drug Delivery Rev.</span>, 1997, <span class="bold">23</span>, 3–25 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/S0169-409X(96)00423-1" title="DOI Link to resource 10.1016/S0169-409X(96)00423-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/5_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit6">R. Macarron, M. N. Banks, D. Bojanic, D. J. Burns, D. A. Cirovic, T. Garyantes, D. V. S. Green, R. P. Hertzberg, W. P. Janzen, J. W. Paslay, U. Schopfer and G. S. Sittampalam, <span class="italic">Nat. Rev. Drug Discovery</span>, 2011, <span class="bold">10</span>, 188–195 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nrd3368" title="DOI Link to resource 10.1038/nrd3368">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/6_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit7">E. O. Pyzer-Knapp, C. Suh, R. Gómez-Bombarelli, J. Aguilera-Iparraguirre and A. Aspuru-Guzik, <span class="italic">Annu. Rev. Mater. Res.</span>, 2015, <span class="bold">45</span>, 195–216 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev-matsci-070214-020823" title="DOI Link to resource 10.1146/annurev-matsci-070214-020823">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWqur3L" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/7_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit8">K. T. Butler, D. W. Davies, H. Cartwright, O. Isayev and A. Walsh, <span class="italic">Nature</span>, 2018, <span class="bold">559</span>, 547–555 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41586-018-0337-2" title="DOI Link to resource 10.1038/s41586-018-0337-2">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtl2jt7vL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30046072%5Buid%5D" title="PubMed Link to resource 30046072">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/8_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit9">P. Raccuglia, K. C. Elbert, P. D. F. Adler, C. Falk, M. B. Wenny, A. Mollo, M. Zeller, S. A. Friedler, J. Schrier and A. J. Norquist, <span class="italic">Nature</span>, 2016, <span class="bold">533</span>, 73–76 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature17439" title="DOI Link to resource 10.1038/nature17439">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XntlKhtL0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27147027%5Buid%5D" title="PubMed Link to resource 27147027">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/9_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit10">
          B. C. Barnes, D. C. Elton, Z. Boukouvalas, D. E. Taylor, W. D. Mattson, M. D. Fuge and P. W. Chung,  2018, arXiv e-prints:1807.06156.</span></li>
      <li><span id="cit11">D. Fooshee, A. Mood, E. Gutman, M. Tavakoli, G. Urban, F. Liu, N. Huynh, D. V. Vranken and P. Baldi, <span class="italic">Mol. Syst. Des. Eng.</span>, 2018, <span class="bold">3</span>, 442–452 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7ME00107J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7ME00107J">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/11_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit12">P. Schwaller, T. Gaudin, D. Lányi, C. Bekas and T. Laino, <span class="italic">Chem. Sci.</span>, 2018, <span class="bold">9</span>, 6091–6098 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8SC02339E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8SC02339E">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/12_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit13">M. H. S. Segler, M. Preuss and M. P. Waller, <span class="italic">Nature</span>, 2018, <span class="bold">555</span>, 604–610 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature25978" title="DOI Link to resource 10.1038/nature25978">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXmsVGqt7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29595767%5Buid%5D" title="PubMed Link to resource 29595767">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/13_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit14">A. B. Henson, P. S. Gromski and L. Cronin, <span class="italic">ACS Cent. Sci.</span>, 2018, <span class="bold">4</span>, 793–804 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.8b00176" title="DOI Link to resource 10.1021/acscentsci.8b00176">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1ertLvJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30062108%5Buid%5D" title="PubMed Link to resource 30062108">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/14_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit15">L. M. Roch, F. Häse, C. Kreisbeck, T. Tamayo-Mendoza, L. P. E. Yunker, J. E. Hein and A. Aspuru-Guzik, <span class="italic">Sci. Robot.</span>, 2018, <span class="bold">3</span>, eaat5559 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/scirobotics.aat5559" title="DOI Link to resource 10.1126/scirobotics.aat5559">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/15_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit16">
          D. Cireşan, U. Meier and J. Schmidhuber, <span class="italic">2012 IEEE Conference on Computer Vision and Pattern Recognition</span>,  2012, pp. 3642–3649 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=2012%20IEEE%20Conference%20on%20Computer%20Vision%20and%20Pattern%20Recognition%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202012%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit17">
          A. Krizhevsky, I. Sutskever and G. E. Hinton, in <span class="italic">Advances in Neural Information Processing Systems 25</span>, ed. F. Pereira, C. J. C. Burges, L. Bottou and K. Q. Weinberger, Curran Associates, Inc.,  2012, pp. 1097–1105 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Advances%20in%20Neural%20Information%20Processing%20Systems%2025%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202012%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit18">
          G. E. Dahl, N. Jaitly and R. Salakhutdinov,  2014, arXiv e-prints:1406.1231.</span></li>
      <li><span id="cit19">
          G. E. Hinton, N. Srivastava, A. Krizhevsky, I. Sutskever and R. R. Salakhutdinov,  2012, arXiv e-prints:1207.0580.</span></li>
      <li><span id="cit20">A. Krizhevsky, I. Sutskever and G. E. Hinton, <span class="italic">Commun. ACM</span>, 2017, <span class="bold">60</span>, 84–90 <a target="_blank" class="DOILink" href="https://doi.org/10.1145/3065386" title="DOI Link to resource 10.1145/3065386">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/20_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit21">
          D. P. Kingma and M. Welling,  2013, arXiv e-prints: 1312.6114.</span></li>
      <li><span id="cit22">
          I. Goodfellow, J. Pouget-Abadie, M. Mirza, B. Xu, D. Warde-Farley, S. Ozair, A. Courville and Y. Bengio, in <span class="italic">Advances in Neural Information Processing Systems 27</span>, ed. Z. Ghahramani, M. Welling, C. Cortes, N. D. Lawrence and K. Q. Weinberger, Curran Associates, Inc.,  2014, pp. 2672–2680 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Advances%20in%20Neural%20Information%20Processing%20Systems%2027%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202014%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit23">R. Gómez-Bombarelli, J. N. Wei, D. Duvenaud, J. M. Hernández-Lobato, B. Sánchez-Lengeling, D. Sheberla, J. Aguilera-Iparraguirre, T. D. Hirzel, R. P. Adams and A. Aspuru-Guzik, <span class="italic">ACS Cent. Sci.</span>, 2018, <span class="bold">4</span>, 268–276 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.7b00572" title="DOI Link to resource 10.1021/acscentsci.7b00572">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29532027%5Buid%5D" title="PubMed Link to resource 29532027">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/23_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit24">E. J. Griffen, A. G. Dossetter, A. G. Leach and S. Montague, <span class="italic">Drug Discovery Today</span>, 2018, <span class="bold">23</span>, 1373–1384 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.drudis.2018.03.011" title="DOI Link to resource 10.1016/j.drudis.2018.03.011">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29577971%5Buid%5D" title="PubMed Link to resource 29577971">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/24_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit25">R. Gómez-Bombarelli, J. Aguilera-Iparraguirre, T. D. Hirzel, D. Duvenaud, D. Maclaurin, M. A. Blood-Forsythe, H. S. Chae, M. Einzinger, D.-G. Ha, T. Wu, G. Markopoulos, S. Jeon, H. Kang, H. Miyazaki, M. Numata, S. Kim, W. Huang, S. I. Hong, M. Baldo, R. P. Adams and A. Aspuru-Guzik, <span class="italic">Nat. Mater.</span>, 2016, <span class="bold">15</span>, 1120–1127 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat4717" title="DOI Link to resource 10.1038/nmat4717">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27500805%5Buid%5D" title="PubMed Link to resource 27500805">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/25_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit26">P. B. Jørgensen, M. Mesta, S. Shil, J. M. G. Lastra, K. W. Jacobsen, K. S. Thygesen and M. N. Schmidt, <span class="italic">J. Chem. Phys.</span>, 2018, <span class="bold">148</span>, 241735 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5023563" title="DOI Link to resource 10.1063/1.5023563">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29960358%5Buid%5D" title="PubMed Link to resource 29960358">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/26_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit27">D. C. Elton, Z. Boukouvalas, M. S. Butrico, M. D. Fuge and P. W. Chung, <span class="italic">Sci. Rep.</span>, 2018, <span class="bold">8</span>, 9059 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-018-27344-x" title="DOI Link to resource 10.1038/s41598-018-27344-x">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29899464%5Buid%5D" title="PubMed Link to resource 29899464">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/27_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit28">B. C. Rinderspacher and J. M. Elward, <span class="italic">Mol. Syst. Des. Eng.</span>, 2018, <span class="bold">3</span>, 485–495 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7ME00126F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7ME00126F">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/28_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit29">H. Li, C. R. Collins, T. G. Ribelli, K. Matyjaszewski, G. J. Gordon, T. Kowalewski and D. J. Yaron, <span class="italic">Mol. Syst. Des. Eng.</span>, 2018, <span class="bold">3</span>, 496–508 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7ME00131B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7ME00131B">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/29_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit30">D. Nagarajan, T. Nagarajan, N. Roy, O. Kulkarni, S. Ravichandran, M. Mishra, D. Chakravortty and N. Chandra, <span class="italic">J. Biol. Chem.</span>, 2017, <span class="bold">293</span>, 3492–3509 <a target="_blank" class="DOILink" href="https://doi.org/10.1074/jbc.M117.805499" title="DOI Link to resource 10.1074/jbc.M117.805499">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29259134%5Buid%5D" title="PubMed Link to resource 29259134">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/30_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit31">A. T. Müller, J. A. Hiss and G. Schneider, <span class="italic">J. Chem. Inf. Model.</span>, 2018, <span class="bold">58</span>, 472–479 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.7b00414" title="DOI Link to resource 10.1021/acs.jcim.7b00414">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29355319%5Buid%5D" title="PubMed Link to resource 29355319">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/31_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit32">F. Grisoni, C. S. Neuhaus, G. Gabernet, A. T. Müller, J. A. Hiss and G. Schneider, <span class="italic">ChemMedChem</span>, 2018, <span class="bold">13</span>, 1300–1302 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cmdc.201800204" title="DOI Link to resource 10.1002/cmdc.201800204">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtVaisb%252FI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29679519%5Buid%5D" title="PubMed Link to resource 29679519">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/32_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit33">X. Shen, T. Zhang, S. Broderick and K. Rajan, <span class="italic">Mol. Syst. Des. Eng.</span>, 2018, <span class="bold">3</span>, 826–838 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8ME00014J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8ME00014J">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/33_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit34">Y. He, E. D. Cubuk, M. D. Allendorf and E. J. Reed, <span class="italic">J. Phys. Chem. Lett.</span>, 2018, <span class="bold">9</span>, 4562–4569 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.8b01707" title="DOI Link to resource 10.1021/acs.jpclett.8b01707">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsVShsrbP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30052453%5Buid%5D" title="PubMed Link to resource 30052453">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/34_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit35">B. Pirard, <span class="italic">Expert Opin. Drug Discovery</span>, 2011, <span class="bold">6</span>, 225–231 <a target="_blank" class="DOILink" href="https://doi.org/10.1517/17460441.2011.554394" title="DOI Link to resource 10.1517/17460441.2011.554394">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22647201%5Buid%5D" title="PubMed Link to resource 22647201">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/35_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit36">L.-P. Wang, A. Titov, R. McGibbon, F. Liu, V. S. Pande and T. J. Martínez, <span class="italic">Nat. Chem.</span>, 2014, <span class="bold">6</span>, 1044–1048 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nchem.2099" title="DOI Link to resource 10.1038/nchem.2099">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGitbnI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25411881%5Buid%5D" title="PubMed Link to resource 25411881">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/36_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit37">J. Besnard, G. F. Ruda, V. Setola, K. Abecassis, R. M. Rodriguiz, X.-P. Huang, S. Norval, M. F. Sassano, A. I. Shin, L. A. Webster, F. R. C. Simeons, L. Stojanovski, A. Prat, N. G. Seidah, D. B. Constam, G. R. Bickerton, K. D. Read, W. C. Wetsel, I. H. Gilbert, B. L. Roth and A. L. Hopkins, <span class="italic">Nature</span>, 2012, <span class="bold">492</span>, 215–220 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature11691" title="DOI Link to resource 10.1038/nature11691">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhvVamurfN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23235874%5Buid%5D" title="PubMed Link to resource 23235874">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/37_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit38">N. Yoshikawa, K. Terayama, M. Sumita, T. Homma, K. Oono and K. Tsuda, <span class="italic">Chem. Lett.</span>, 2018, <span class="bold">47</span>, 1431–1434 <a target="_blank" class="DOILink" href="https://doi.org/10.1246/cl.180665" title="DOI Link to resource 10.1246/cl.180665">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXitlKnsLvP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/38_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit39">A. Daina, O. Michielin and V. Zoete, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 2717 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-017-02902-x" title="DOI Link to resource 10.1038/s41598-017-02902-x">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28578429%5Buid%5D" title="PubMed Link to resource 28578429">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/39_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit40">D. Kuzminykh, D. Polykovskiy, A. Kadurin, A. Zhebrak, I. Baskov, S. Nikolenko, R. Shayakhmetov and A. Zhavoronkov, <span class="italic">Mol. Pharmaceutics</span>, 2018, 4378–4385 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.molpharmaceut.7b01134" title="DOI Link to resource 10.1021/acs.molpharmaceut.7b01134">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjtFKrurc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29473756%5Buid%5D" title="PubMed Link to resource 29473756">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/40_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit41">M. Skalic, J. Jiménez Luna, D. Sabbadin and G. De Fabritiis, <span class="italic">J. Chem. Inf. Model.</span>, 2019, <span class="bold">59</span>(3), 1205–1214 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.8b00706" title="DOI Link to resource 10.1021/acs.jcim.8b00706">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXjtVShu78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30762364%5Buid%5D" title="PubMed Link to resource 30762364">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/41_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit42">A. Amidi, S. Amidi, D. Vlachakis, V. Megalooikonomou, N. Paragios and E. I. Zacharaki, <span class="italic">PeerJ</span>, 2018, <span class="bold">6</span>, e4750 <a target="_blank" class="DOILink" href="https://doi.org/10.7717/peerj.4750" title="DOI Link to resource 10.7717/peerj.4750">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29740518%5Buid%5D" title="PubMed Link to resource 29740518">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/42_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit43">M. Hirn, S. Mallat and N. Poilvert, <span class="italic">Multiscale Model. Simul.</span>, 2017, <span class="bold">15</span>, 827–863 <a target="_blank" class="DOILink" href="https://doi.org/10.1137/16M1075454" title="DOI Link to resource 10.1137/16M1075454">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/43_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit44">
          M. Eickenberg, G. Exarchakis, M. Hirn and S. Mallat, in <span class="italic">Advances in Neural Information Processing Systems 30</span>, ed. I. Guyon, U. V. Luxburg, S. Bengio, H. Wallach, R. Fergus, S. Vishwanathan and R. Garnett, Curran Associates, Inc.,  2017, pp. 6540–6549 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Advances%20in%20Neural%20Information%20Processing%20Systems%2030%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/44_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit45">
          N. Thomas, T. Smidt, S. Kearnes, L. Yang, L. Li, K. Kohlhoff and P. Riley, <span class="italic">Tensor field networks: Rotation- and translation-equivariant neural networks for 3D point clouds</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Tensor%20field%20networks:%20Rotation-%20and%20translation-equivariant%20neural%20networks%20for%203D%20point%20clouds%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit46">N. M. O'Boyle, M. Banck, C. A. James, C. Morley, T. Vandermeersch and G. R. Hutchison, <span class="italic">J. Cheminf.</span>, 2011, <span class="bold">3</span>, 33 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%203%5Bvolume%5D%20AND%2033%5Bpage%5D%20and%202011%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/46_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit47">
          <span class="italic">The Open Babel Package</span>, http://www.openbabel.org <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=The%20Open%20Babel%20Package%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%20%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/47_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit48">D. Weininger, <span class="italic">J. Chem. Inf. Model.</span>, 1988, <span class="bold">28</span>(1), 31–36 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci00057a005" title="DOI Link to resource 10.1021/ci00057a005">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL1cXnsVeqsA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/48_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit49">
          E. Jang, S. Gu and B. Poole,  2016, arXiv e-prints:1611.01144.</span></li>
      <li><span id="cit50">
          N. De Cao and T. Kipf, <span class="italic">ICML 2018 workshop on Theoretical Foundations and Applications of Deep Generative Models</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ICML%202018%20workshop%20on%20Theoretical%20Foundations%20and%20Applications%20of%20Deep%20Generative%20Models%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit51">
          M. Swain, <span class="italic">MolVS</span>, https://github.com/mcs07/MolVS <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=MolVS%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%20%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/51_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit52">
          G. Landrum, <span class="italic">RDKit: Open-source cheminformatics</span>, http://www.rdkit.org <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=RDKit:%20Open-source%20cheminformatics%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%20%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/52_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit53">
          E. J. Bjerrum and B. Sattarov,  2018, arXiv e-prints:1806.09300.</span></li>
      <li><span id="cit54">
          E. J. Bjerrum,  2017, arXiv e-prints:1703.07076.</span></li>
      <li><span id="cit55">S. Heller, A. McNaught, S. Stein, D. Tchekhovskoi and I. Pletnev, <span class="italic">J. Cheminf.</span>, 2013, <span class="bold">5</span>, 7 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXlvVyhtLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/55_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit56">R. Winter, F. Montanari, F. Noé and D.-A. Clevert, <span class="italic">Chem. Sci.</span>, 2019, <span class="bold">10</span>, 1692–1701 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8SC04175J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8SC04175J">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/56_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit57">
          M. J. Kusner, B. Paige and J. M. Hernández-Lobato,  2017, arXiv e-prints:1703.01925.</span></li>
      <li><span id="cit58">
          H. Dai, Y. Tian, B. Dai, S. Skiena and L. Song,  2018, arXiv e-prints:1802.08786.</span></li>
      <li><span id="cit59">
          Y. Li, O. Vinyals, C. Dyer, R. Pascanu and P. Battaglia,  2018, arXiv e-prints:1803.03324.</span></li>
      <li><span id="cit60">Y. Li, L. Zhang and Z. Liu, <span class="italic">J. Cheminf.</span>, 2018, <span class="bold">10</span>, 33 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%2010%5Bvolume%5D%20AND%2033%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/60_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit61">M. Suzuki, H. Nagamochi and T. Akutsu, <span class="italic">J. Cheminf.</span>, 2014, <span class="bold">6</span>, 31 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%206%5Bvolume%5D%20AND%2031%5Bpage%5D%20and%202014%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/61_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit62">
          G. B. Goh, C. Siegel, A. Vishnu, N. O. Hodas and N. Baker,  2017, arXiv e-prints:1706.06689.</span></li>
      <li><span id="cit63">
          N. De Cao and T. Kipf,  2018, arXiv e-prints:1805.11973.</span></li>
      <li><span id="cit64">
          M. Simonovsky and N. Komodakis,  2018, arXiv e-prints:1802.03480.</span></li>
      <li><span id="cit65">
          J. You, B. Liu, R. Ying, V. Pande and J. Leskovec,  2018, arXiv e-prints:-1806.02473.</span></li>
      <li><span id="cit66">
          Z. Zhou, S. Kearnes, L. Li, R. N. Zare and P. Riley,  2018, arXiv e-prints:1810.08678.</span></li>
      <li><span id="cit67">
          Z. Boukouvalas, D. C. Elton, P. W. Chung and M. D. Fuge,  2018, arXiv e-prints:1811.00628.</span></li>
      <li><span id="cit68">J. L. Durant, B. A. Leland, D. R. Henry and J. G. Nourse, <span class="italic">J. Chem. Inf. Comput. Sci.</span>, 2002, <span class="bold">42</span>, 1273–1280 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci010132r" title="DOI Link to resource 10.1021/ci010132r">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XmvFKktLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12444722%5Buid%5D" title="PubMed Link to resource 12444722">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/68_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit69">A. Kadurin, A. Aliper, A. Kazennov, P. Mamoshina, Q. Vanhaelen, K. Khrabrov and A. Zhavoronkov, <span class="italic">Oncotarget</span>, 2016, <span class="bold">8</span>(7), 10883–10890 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Oncotarget%5BJour%5D%20AND%208%5Bvolume%5D%20AND%2010883%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/69_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit70">A. Kadurin, S. Nikolenko, K. Khrabrov, A. Aliper and A. Zhavoronkov, <span class="italic">Mol. Pharmaceutics</span>, 2017, <span class="bold">14</span>, 3098–3104 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.molpharmaceut.7b00346" title="DOI Link to resource 10.1021/acs.molpharmaceut.7b00346">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFGitbbF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28703000%5Buid%5D" title="PubMed Link to resource 28703000">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/70_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit71">
          M. Simonovsky and N. Komodakis, <span class="italic">2017 IEEE Conference on Computer Vision and Pattern Recognition</span>, CVPR 2017, Honolulu, HI, USA, July 21–26,  2017, vol. 2017, pp. 29–38 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=2017%20IEEE%20Conference%20on%20Computer%20Vision%20and%20Pattern%20Recognition%5BJour%5D%20AND%20vol. 2017%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit72">
          E. J. Bjerrum and R. Threlfall,  2017, arXiv e-prints:1705.04612.</span></li>
      <li><span id="cit73">A. Gupta, A. T. Müller, B. J. H. Huisman, J. A. Fuchs, P. Schneider and G. Schneider, <span class="italic">Mol. Inf.</span>, 2017, <span class="bold">37</span>, 1700111 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/minf.201700111" title="DOI Link to resource 10.1002/minf.201700111">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29095571%5Buid%5D" title="PubMed Link to resource 29095571">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/73_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit74">M. Olivecrona, T. Blaschke, O. Engkvist and H. Chen, <span class="italic">J. Cheminf.</span>, 2017, <span class="bold">9</span>, 48 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%209%5Bvolume%5D%20AND%2048%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/74_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit75">M. H. S. Segler, T. Kogej, C. Tyrchan and M. P. Waller, <span class="italic">ACS Cent. Sci.</span>, 2017, <span class="bold">4</span>, 120–131 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.7b00512" title="DOI Link to resource 10.1021/acscentsci.7b00512">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29392184%5Buid%5D" title="PubMed Link to resource 29392184">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/75_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit76">X. Yang, J. Zhang, K. Yoshizoe, K. Terayama and K. Tsuda, <span class="italic">Sci. Technol. Adv. Mater.</span>, 2017, <span class="bold">18</span>, 972–976 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/14686996.2017.1401424" title="DOI Link to resource 10.1080/14686996.2017.1401424">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsFamu7rO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29435094%5Buid%5D" title="PubMed Link to resource 29435094">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/76_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit77">
          M. Cherti, B. Kégl and A. Kazakçı, <span class="italic">International Conference on Learning Representations, workshop track</span>, Toulon, France,  2017 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations,%20workshop%20track%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit78">
          D. Neil, M. Segler, L. Guasch, M. Ahmed, D. Plumbley, M. Sellwood and N. Brown, <span class="italic">International Conference on Learning Representations</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit79">M. Popova, O. Isayev and A. Tropsha, <span class="italic">Sci. Adv.</span>, 2018, <span class="bold">4</span>, eaap7885 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/sciadv.aap7885" title="DOI Link to resource 10.1126/sciadv.aap7885">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30050984%5Buid%5D" title="PubMed Link to resource 30050984">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/79_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit80">M. Sumita, X. Yang, S. Ishihara, R. Tamura and K. Tsuda, <span class="italic">ACS Cent. Sci.</span>, 2018, <span class="bold">4</span>, 1126–1133 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.8b00213" title="DOI Link to resource 10.1021/acscentsci.8b00213">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsFaqtrnN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30276245%5Buid%5D" title="PubMed Link to resource 30276245">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/80_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit81">D. Merk, L. Friedrich, F. Grisoni and G. Schneider, <span class="italic">Mol. Inf.</span>, 2018, <span class="bold">37</span>, 1700153 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/minf.201700153" title="DOI Link to resource 10.1002/minf.201700153">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29319225%5Buid%5D" title="PubMed Link to resource 29319225">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/81_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit82">D. Merk, F. Grisoni, L. Friedrich and G. Schneider, <span class="italic">Communications Chemistry</span>, 2018, <span class="bold">1</span>, 68 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s42004-018-0068-1" title="DOI Link to resource 10.1038/s42004-018-0068-1">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/82_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit83">
          P. Ertl, R. Lewis, E. Martin and V. Polyakov,  2017, arXiv e-prints:1712.07449.</span></li>
      <li><span id="cit84">
          J. Arús-Pous, T. Blaschke, S. Ulander, J.-L. Reymond, H. Chen and O. Engkvist, <span class="italic">ChemRxiv preprint</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ChemRxiv%20preprint%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit85">S. Zheng, X. Yan, Q. Gu, Y. Yang, Y. Du, Y. Lu and J. Xu, <span class="italic">J. Cheminf.</span>, 2019, <span class="bold">11</span>, 5 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%2011%5Bvolume%5D%20AND%205%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/85_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit86">P. Pogány, N. Arad, S. Genway and S. D. Pickett, <span class="italic">J. Chem. Inf. Model.</span>, 2018, <span class="bold">59</span>(3), 1136–1146 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.8b00626" title="DOI Link to resource 10.1021/acs.jcim.8b00626">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30525594%5Buid%5D" title="PubMed Link to resource 30525594">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/86_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit87">T. Blaschke, M. Olivecrona, O. Engkvist, J. Bajorath and H. Chen, <span class="italic">Mol. Inf.</span>, 2017, <span class="bold">37</span>, 1700123 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/minf.201700123" title="DOI Link to resource 10.1002/minf.201700123">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29235269%5Buid%5D" title="PubMed Link to resource 29235269">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/87_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit88">J. Lim, S. Ryu, J. W. Kim and W. Y. Kim, <span class="italic">J. Cheminf.</span>, 2018, <span class="bold">10</span>, 31 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%2010%5Bvolume%5D%20AND%2031%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/88_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit89">S. Kang and K. Cho, <span class="italic">J. Chem. Inf. Model.</span>, 2018, <span class="bold">59</span>(1), 43–52 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.8b00263" title="DOI Link to resource 10.1021/acs.jcim.8b00263">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30016587%5Buid%5D" title="PubMed Link to resource 30016587">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/89_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit90">S. Harel and K. Radinsky, <span class="italic">Mol. Pharmaceutics</span>, 2018, <span class="bold">15</span>, 4406–4416 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.molpharmaceut.8b00474" title="DOI Link to resource 10.1021/acs.molpharmaceut.8b00474">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSrsb%252FE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30063142%5Buid%5D" title="PubMed Link to resource 30063142">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/90_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit91">B. Sattarov, I. I. Baskin, D. Horvath, G. Marcou, E. J. Bjerrum and A. Varnek, <span class="italic">J. Chem. Inf. Model.</span>, 2019, <span class="bold">59</span>(3), 1182–1196 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.8b00751" title="DOI Link to resource 10.1021/acs.jcim.8b00751">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXjtlCisbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/91_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit92">
          M. J. Kusner, B. Paige and J. M. Hernández-Lobato,  2017, arXiv e-prints:1703.01925.</span></li>
      <li><span id="cit93">P. B. Jørgensen, M. N. Schmidt and O. Winther, <span class="italic">Mol. Inf.</span>, 2018, <span class="bold">37</span>, 1700133 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/minf.201700133" title="DOI Link to resource 10.1002/minf.201700133">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29405647%5Buid%5D" title="PubMed Link to resource 29405647">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/93_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit94">
          W. Jin, R. Barzilay and T. S. Jaakkola, <span class="italic">International Conference on Learning Representations</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit95">
          W. Jin, K. Yang, R. Barzilay and T. Jaakkola, <span class="italic">International Conference on Learning Representations</span>,  2019 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit96">
          Q. Liu, M. Allamanis, M. Brockschmidt and A. L. Gaunt,  2018, arXiv e-prints:1805.09076.</span></li>
      <li><span id="cit97">
          H. Kajino,  2018, arXiv e-prints:1803.03324.</span></li>
      <li><span id="cit98">
          R. Winter, F. Montanari, F. Noé and D.-A. Clevert, <span class="italic">ChemRxiv preprint</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ChemRxiv%20preprint%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit99">
          B. Samanta, A. De, N. Ganguly and M. Gomez-Rodriguez,  2018, arXiv e-prints:1802.05283.</span></li>
      <li><span id="cit100">
          B. Samanta, A. De, G. Jana, P. K. Chattaraj, N. Ganguly and M. Gomez-Rodriguez,  2018, arXiv e-prints:1802.05283.</span></li>
      <li><span id="cit101">
          T. Ma, J. Chen and C. Xiao, in <span class="italic">Advances in Neural Information Processing Systems 32</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Advances%20in%20Neural%20Information%20Processing%20Systems%2032%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit102">
          S. M. Kearnes, L. Li and P. Riley,  2019, arXiv e-prints:1904.08915.</span></li>
      <li><span id="cit103">
          G. Lima Guimaraes, B. Sanchez-Lengeling, C. Outeiral, P. L. Cunha Farias and A. Aspuru-Guzik,  2017, arXiv e-prints:1705.10843.</span></li>
      <li><span id="cit104">E. Putin, A. Asadulaev, Y. Ivanenkov, V. Aladinskiy, B. Sanchez-Lengeling, A. Aspuru-Guzik and A. Zhavoronkov, <span class="italic">J. Chem. Inf. Model.</span>, 2018, <span class="bold">58</span>, 1194–1204 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.7b00690" title="DOI Link to resource 10.1021/acs.jcim.7b00690">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpsVChtrs%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29762023%5Buid%5D" title="PubMed Link to resource 29762023">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/104_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit105">E. Putin, A. Asadulaev, Q. Vanhaelen, Y. Ivanenkov, A. V. Aladinskaya, A. Aliper and A. Zhavoronkov, <span class="italic">Mol. Pharmaceutics</span>, 2018, <span class="bold">15</span>(10), 4386–4397 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.molpharmaceut.7b01137" title="DOI Link to resource 10.1021/acs.molpharmaceut.7b01137">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXlt1Kks7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29569445%5Buid%5D" title="PubMed Link to resource 29569445">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/105_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit106">
          O. Méndez-Lucio, B. Baillif, D.-A. Clevert, D. Rouquié and J. Wichard, <span class="italic">ChemRxiv preprint</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ChemRxiv%20preprint%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit107">
          L. Maziarka, A. Pocha, J. Kaczmarczyk, K. Rataj and M. Warchoł, <span class="italic">Mol-CycleGAN - a generative model for molecular optimization</span>,  2019 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mol-CycleGAN%20-%20a%20generative%20model%20for%20molecular%20optimization%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit108">
          B. Sanchez-Lengeling, C. Outeiral, G. L. Guimaraes and A. Aspuru-Guzik, <span class="italic">ChemRxiv preprint</span>,  2017 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ChemRxiv%20preprint%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit109">
          D. Grattarola, L. Livi and C. Alippi,  2018, arXiv e-prints:1812.04314.</span></li>
      <li><span id="cit110">H. Ikebata, K. Hongo, T. Isomura, R. Maezono and R. Yoshida, <span class="italic">J. Comput.-Aided Mol. Des.</span>, 2017, <span class="bold">31</span>, 379–391 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10822-016-0008-z" title="DOI Link to resource 10.1007/s10822-016-0008-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXktVGlu7w%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28281211%5Buid%5D" title="PubMed Link to resource 28281211">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/110_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit111">D. Polykovskiy, A. Zhebrak, D. Vetrov, Y. Ivanenkov, V. Aladinskiy, P. Mamoshina, M. Bozdaganyan, A. Aliper, A. Zhavoronkov and A. Kadurin, <span class="italic">Mol. Pharmaceutics</span>, 2018, <span class="bold">15</span>(10), 4398–4405 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.molpharmaceut.8b00839" title="DOI Link to resource 10.1021/acs.molpharmaceut.8b00839">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhs1ensL%252FM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30180591%5Buid%5D" title="PubMed Link to resource 30180591">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/111_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit112">
          N. Ståhl, G. Falkman, A. Karlsson, G. Mathiason and J. Bostrom, <span class="italic">ChemRxiv e-print</span>,  2019 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ChemRxiv%20e-print%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit113">L. C. Blum and J.-L. Reymond, <span class="italic">J. Am. Chem. Soc.</span>, 2009, <span class="bold">131</span>, 8732–8733 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja902302h" title="DOI Link to resource 10.1021/ja902302h">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXmvFWru7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19505099%5Buid%5D" title="PubMed Link to resource 19505099">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/113_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit114">T. Sterling and J. J. Irwin, <span class="italic">J. Chem. Inf. Model.</span>, 2015, <span class="bold">55</span>, 2324–2337 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.5b00559" title="DOI Link to resource 10.1021/acs.jcim.5b00559">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhs1OhurbF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26479676%5Buid%5D" title="PubMed Link to resource 26479676">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/114_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit115">L. Ruddigkeit, R. van Deursen, L. C. Blum and J.-L. Reymond, <span class="italic">J. Chem. Inf. Model.</span>, 2012, <span class="bold">52</span>, 2864–2875 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci300415d" title="DOI Link to resource 10.1021/ci300415d">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhsFClsL3J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23088335%5Buid%5D" title="PubMed Link to resource 23088335">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/115_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit116">M. Nakata and T. Shimazaki, <span class="italic">J. Chem. Inf. Model.</span>, 2017, <span class="bold">57</span>, 1300–1308 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.7b00083" title="DOI Link to resource 10.1021/acs.jcim.7b00083">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXntFKnt7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28481528%5Buid%5D" title="PubMed Link to resource 28481528">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/116_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit117">R. Ramakrishnan, P. O. Dral, M. Rupp and O. A. von Lilienfeld, <span class="italic">Sci. Data</span>, 2014, <span class="bold">1</span>, 14022 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Sci.%20Data%5BJour%5D%20AND%201%5Bvolume%5D%20AND%2014022%5Bpage%5D%20and%202014%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/117_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit118">S. Chakraborty, P. Kayastha and R. Ramakrishnan, <span class="italic">J. Chem. Phys.</span>, 2019, <span class="bold">150</span>, 114106 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5088083" title="DOI Link to resource 10.1063/1.5088083">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/118_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit119">S. A. Lopez, E. O. Pyzer-Knapp, G. N. Simm, T. Lutzow, K. Li, L. R. Seress, J. Hachmann and A. Aspuru-Guzik, <span class="italic">Sci. Data</span>, 2016, <span class="bold">3</span>, 160086 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/sdata.2016.86" title="DOI Link to resource 10.1038/sdata.2016.86">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27676312%5Buid%5D" title="PubMed Link to resource 27676312">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/119_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit120">
          A. Géron, <span class="italic">Hands-On Machine Learning with Scikit-Learn and TensorFlow: Concepts, Tools, and Techniques to Build Intelligent Systems</span>, O'Reilly Media, Inc.,  2017, 1st edn <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Hands-On%20Machine%20Learning%20with%20Scikit-Learn%20and%20TensorFlow:%20Concepts,%20Tools,%20and%20Techniques%20to%20Build%20Intelligent%20Systems%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/120_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit121">
          I. Goodfellow, Y. Bengio and A. Courville, <span class="italic">Deep Learning</span>, MIT Press,  2016 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Deep%20Learning%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/121_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit122">S. Hochreiter and J. Schmidhuber, <span class="italic">Neural Comput.</span>, 1997, <span class="bold">9</span>, 1735–1780 <a target="_blank" class="DOILink" href="https://doi.org/10.1162/neco.1997.9.8.1735" title="DOI Link to resource 10.1162/neco.1997.9.8.1735">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADyaK1c%252FhvVahsQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/122_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit123">
          K. Cho, B. van Merrienboer, C. Gulcehre, D. Bahdanau, F. Bougares, H. Schwenk and Y. Bengio, <span class="italic">Proceedings of the 2014 Conference on Empirical Methods in Natural Language Processing (EMNLP)</span>,  2014, pp. 1724–1734 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proceedings%20of%20the%202014%20Conference%20on%20Empirical%20Methods%20in%20Natural%20Language%20Processing%20(EMNLP)%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202014%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit124">
          M. Ranzato, S. Chopra, M. Auli and W. Zaremba,  2015, arXiv:abs/1511.06732.</span></li>
      <li><span id="cit125">
          A. Venkatraman, M. Hebert and J. A. Bagnell, <span class="italic">Proceedings of the Twenty-Ninth AAAI Conference on Artificial Intelligence</span>,  2015, pp. 3024–3030 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proceedings%20of%20the%20Twenty-Ninth%20AAAI%20Conference%20on%20Artificial%20Intelligence%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202015%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit126">
          S. Bengio, O. Vinyals, N. Jaitly and N. Shazeer, <span class="italic">Proceedings of the 28th International Conference on Neural Information Processing Systems</span>, Cambridge, MA, USA,  2015, vol. 1, pp. 1171–1179 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proceedings%20of%20the%2028th%20International%20Conference%20on%20Neural%20Information%20Processing%20Systems%5BJour%5D%20AND%20vol. 1%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202015%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit127">
          F. Huszár,  2015, arXiv e-prints:1511.05101.</span></li>
      <li><span id="cit128">R. J. Williams and D. Zipser, <span class="italic">Neural Comput.</span>, 1989, <span class="bold">1</span>, 270–280 <a target="_blank" class="DOILink" href="https://doi.org/10.1162/neco.1989.1.2.270" title="DOI Link to resource 10.1162/neco.1989.1.2.270">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/128_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit129">
          R. Gómez-Bombarelli, <span class="italic">Broad Institute Models, Inference, &amp; Algorithms talk: “Deep learning chemical space”</span>,  2017, https://www.youtube.com/watch?v=ieZhnnvjyWU <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Broad%20Institute%20Models,%20Inference,%20&amp;%20Algorithms%20talk:%20“Deep%20learning%20chemical%20space”%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/129_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit130">
          R. J. Williams, <span class="italic">Machine Learning</span>,  1992, vol. 8, pp. 229–256 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Machine%20Learning%5BJour%5D%20AND%20vol. 8%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201992%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/130_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit131">
          N. Jaques, S. Gu, D. Bahdanau, J. M. Hernádez-Lobato, R. E. Turner and D. Eck, <span class="italic">International Conference on Machine Learning</span>,  2017 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Machine%20Learning%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit132">V. Mnih, K. Kavukcuoglu, D. Silver, A. A. Rusu, J. Veness, M. G. Bellemare, A. Graves, M. Riedmiller, A. K. Fidjeland, G. Ostrovski, S. Petersen, C. Beattie, A. Sadik, I. Antonoglou, H. King, D. Kumaran, D. Wierstra, S. Legg and D. Hassabis, <span class="italic">Nature</span>, 2015, <span class="bold">518</span>, 529–533 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature14236" title="DOI Link to resource 10.1038/nature14236">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXjsVagur0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25719670%5Buid%5D" title="PubMed Link to resource 25719670">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/132_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit133">G. Hinton and R. Salakhutdinov, <span class="italic">Science</span>, 2006, <span class="bold">313</span>, 504–507 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1127647" title="DOI Link to resource 10.1126/science.1127647">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xnt1KntrY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16873662%5Buid%5D" title="PubMed Link to resource 16873662">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/133_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit134">
          R. Salakhutdinov and G. Hinton, <span class="italic">Proceedings of the Twelth International Conference on Artificial Intelligence and Statistics</span>, Hilton Clearwater Beach Resort, Clearwater Beach, Florida USA,  2009, pp. 448–455 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proceedings%20of%20the%20Twelth%20International%20Conference%20on%20Artificial%20Intelligence%20and%20Statistics%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit135">
          P. Mehta, M. Bukov, C.-H. Wang, A. G. R. Day, C. Richardson, C. K. Fisher and D. J. Schwab,  2018, arXiv e-prints:1803.08823.</span></li>
      <li><span id="cit136">
          D. Janz, J. van der Westhuizen and J. M. Hernández-Lobato,  2017, arXiv e-prints:1708.04465.</span></li>
      <li><span id="cit137">
          A. Makhzani, J. Shlens, N. Jaitly and I. Goodfellow, <span class="italic">International Conference on Learning Representations</span>,  2016 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit138">
          R.-R. Griffiths and J. M. Hernández-Lobato,  2017, arXiv e-prints:1709.05501.</span></li>
      <li><span id="cit139">
          I. Sutskever, O. Vinyals and Q. V. Le,  2014, arXiv e-prints: 1409.3215.</span></li>
      <li><span id="cit140">
          M. Lucic, K. Kurach, M. Michalski, S. Gelly and O. Bousquet,  2017, arXiv e-prints:1711.10337.</span></li>
      <li><span id="cit141">
          M. Arjovsky, S. Chintala and L. Bottou,  2017, arXiv e-prints: 1701.07875.</span></li>
      <li><span id="cit142">
          K. Kurach, M. Lucic, X. Zhai, M. Michalski and S. Gelly,  2018, arXiv e-prints:1807.04720.</span></li>
      <li><span id="cit143">
          L. Yu, W. Zhang, J. Wang and Y. Yu, <span class="italic">Proceedings of the Thirty-First AAAI Conference on Artificial Intelligence</span>, February 4–9,  2017, San Francisco, California, USA.,  2017, pp. 2852–2858 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proceedings%20of%20the%20Thirty-First%20AAAI%20Conference%20on%20Artificial%20Intelligence%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017 2017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit144">A. Graves, G. Wayne, M. Reynolds, T. Harley, I. Danihelka, A. Grabska-Barwińska, S. G. Colmenarejo, E. Grefenstette, T. Ramalho, J. Agapiou, A. P. Badia, K. M. Hermann, Y. Zwols, G. Ostrovski, A. Cain, H. King, C. Summerfield, P. Blunsom, K. Kavukcuoglu and D. Hassabis, <span class="italic">Nature</span>, 2016, <span class="bold">538</span>, 471–476 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature20101" title="DOI Link to resource 10.1038/nature20101">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27732574%5Buid%5D" title="PubMed Link to resource 27732574">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/144_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit145">
          A. Graves, G. Wayne and I. Danihelka,  2014, arXiv e-prints: 1807.06156.</span></li>
      <li><span id="cit146">
          D. Sculley, J. Snoek, A. Wiltschko and A. Rahimi, <span class="italic">Sixth International Conference on Learning Representations - Workshop Track</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Sixth%20International%20Conference%20on%20Learning%20Representations%20-%20Workshop%20Track%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit147">
          P. Henderson, R. Islam, P. Bachman, J. Pineau, D. Precup and D. Meger, <span class="italic">Thirthy-Second AAAI Conference On Artificial Intelligence</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Thirthy-Second%20AAAI%20Conference%20On%20Artificial%20Intelligence%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit148">
          G. Melis, C. Dyer and P. Blunsom,  2017, arXiv e-prints:1707.05589.</span></li>
      <li><span id="cit149">
          T. Salimans, I. J. Goodfellow, W. Zaremba, V. Cheung, A. Radford and X. Chen,  2016, arXiv e-prints:1606.03498.</span></li>
      <li><span id="cit150">
          L. Theis, A. van den Oord and M. Bethge, <span class="italic">International Conference on Learning Representations</span>,  2016 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit151">
          M. Heusel, H. Ramsauer, T. Unterthiner, B. Nessler and S. Hochreiter, <span class="italic">GANs Trained
by a Two Time-Scale Update Rule Converge to a Local Nash Equilibrium</span>,  2017 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=GANs%20Trained&#xA;by%20a%20Two%20Time-Scale%20Update%20Rule%20Converge%20to%20a%20Local%20Nash%20Equilibrium%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit152">K. Preuer, P. Renz, T. Unterthiner, S. Hochreiter and G. Klambauer, <span class="italic">J. Chem. Inf. Model.</span>, 2018, <span class="bold">58</span>, 1736–1741 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.8b00234" title="DOI Link to resource 10.1021/acs.jcim.8b00234">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsFejsLrL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30118593%5Buid%5D" title="PubMed Link to resource 30118593">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/152_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit153">Z. Wu, B. Ramsundar, E. N. Feinberg, J. Gomes, C. Geniesse, A. S. Pappu, K. Leswing and V. Pande, <span class="italic">Chem. Sci.</span>, 2018, <span class="bold">9</span>, 513–530 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7SC02664A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7SC02664A">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/153_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit154">
          D. Polykovskiy, A. Zhebrak, B. Sanchez-Lengeling, S. Golovanov, O. Tatanov, S. Belyaev, R. Kurbanov, A. Artamonov, V. Aladinskiy, M. Veselov, A. Kadurin, S. Nikolenko, A. Aspuru-Guzik and A. Zhavoronkov,  2018, arXiv e-prints:: arXiv:1811.12823.</span></li>
      <li><span id="cit155">
          M. Benhenda, E. J. Bjerrum, H. Yi and C. Zaveri, <span class="italic">Authorea preprint</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Authorea%20preprint%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit156">N. Brown, M. Fiscato, M. H. Segler and A. C. Vaucher, <span class="italic">J. Chem. Inf. Model.</span>, 2019, <span class="bold">59</span>(3), 1096–1108 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.8b00839" title="DOI Link to resource 10.1021/acs.jcim.8b00839">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXltVWrsbY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30887799%5Buid%5D" title="PubMed Link to resource 30887799">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/156_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit157">
          D. J. Im, A. H. Ma, G. W. Taylor and K. Branson, <span class="italic">International Conference on Learning Representations</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit158">
          I. Gulrajani, C. Raffel and L. Metz, <span class="italic">International Conference on Learning Representations</span>,  2019 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit159">
          D. Lowe, <span class="italic">Calculating A Few Too Many New Compounds</span>,  2016, http://blogs.sciencemag.org/pipeline/archives/2016/11/08/calculating-a-few-too-many-new-compounds <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Calculating%20A%20Few%20Too%20Many%20New%20Compounds%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/159_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit160">
          M. Benhenda,  2017, arXiv e-prints:1703.01925.</span></li>
      <li><span id="cit161">
          N. Yoshikawa, K. Terayama, T. Honma, K. Oono and K. Tsuda,  2018, arXiv e-prints:1804.02134.</span></li>
      <li><span id="cit162">J. Panteleev, H. Gao and L. Jia, <span class="italic">Bioorg. Med. Chem. Lett.</span>, 2018, <span class="bold">28</span>, 2807–2815 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bmcl.2018.06.046" title="DOI Link to resource 10.1016/j.bmcl.2018.06.046">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1yrsrfL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30122222%5Buid%5D" title="PubMed Link to resource 30122222">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/162_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit163">J. B. Tenenbaum, <span class="italic">Science</span>, 2000, <span class="bold">290</span>, 2319–2323 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.290.5500.2319" title="DOI Link to resource 10.1126/science.290.5500.2319">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BD3M%252Fnt1yitQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=11125149%5Buid%5D" title="PubMed Link to resource 11125149">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/163_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit164">L. van der Maaten and G. Hinton, <span class="italic">J. Mach. Learn. Res.</span>, 2008, <span class="bold">9</span>, 2579–2605 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Mach.%20Learn.%20Res.%5BJour%5D%20AND%209%5Bvolume%5D%20AND%202579%5Bpage%5D%20and%202008%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/164_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit165">
          A. Radford, L. Metz and S. Chintala, <span class="italic">International Conference on Learning Representations</span>,  2016 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=International%20Conference%20on%20Learning%20Representations%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit166">P. Domingos, <span class="italic">Commun. ACM</span>, 2012, <span class="bold">55</span>, 78–87 <a target="_blank" class="DOILink" href="https://doi.org/10.1145/2347736.2347755" title="DOI Link to resource 10.1145/2347736.2347755">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/166_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit167">
          T. White,  2016, arXiv e-prints:1609.04468.</span></li>
      <li><span id="cit168">P. Ertl and A. Schuffenhauer, <span class="italic">J. Cheminf.</span>, 2009, <span class="bold">1</span>, 8 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%201%5Bvolume%5D%20AND%208%5Bpage%5D%20and%202009%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/168_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit169">Y. Podolyan, M. A. Walters and G. Karypis, <span class="italic">J. Chem. Inf. Model.</span>, 2010, <span class="bold">50</span>, 979–991 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci900301v" title="DOI Link to resource 10.1021/ci900301v">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXnt1Cntbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20536191%5Buid%5D" title="PubMed Link to resource 20536191">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/169_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit170">Y. Fukunishi, T. Kurosawa, Y. Mikami and H. Nakamura, <span class="italic">J. Chem. Inf. Model.</span>, 2014, <span class="bold">54</span>, 3259–3267 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci500568d" title="DOI Link to resource 10.1021/ci500568d">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOjs7%252FK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25420000%5Buid%5D" title="PubMed Link to resource 25420000">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/170_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit171">P. Ertl, S. Roggo and A. Schuffenhauer, <span class="italic">J. Chem. Inf. Model.</span>, 2008, <span class="bold">48</span>, 68–74 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci700286x" title="DOI Link to resource 10.1021/ci700286x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtlCqtLfP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18034468%5Buid%5D" title="PubMed Link to resource 18034468">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/171_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit172">G. R. Bickerton, G. V. Paolini, J. Besnard, S. Muresan and A. L. Hopkins, <span class="italic">Nat. Chem.</span>, 2012, <span class="bold">4</span>, 90–98 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nchem.1243" title="DOI Link to resource 10.1038/nchem.1243">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktLk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22270643%5Buid%5D" title="PubMed Link to resource 22270643">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/172_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit173">I. Muegge, S. L. Heald and D. Brittelli, <span class="italic">J. Med. Chem.</span>, 2001, <span class="bold">44</span>, 1841–1846 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jm015507e" title="DOI Link to resource 10.1021/jm015507e">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXjsVGisL4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/173_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit174">I. Muegge, <span class="italic">Med. Res. Rev.</span>, 2003, <span class="bold">23</span>, 302–321 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/med.10041" title="DOI Link to resource 10.1002/med.10041">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXjsVWitLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12647312%5Buid%5D" title="PubMed Link to resource 12647312">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/174_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit175">A. Kalgutkar, I. Gardner, R. Obach, C. Shaffer, E. Callegari, K. Henne, A. Mutlib, D. Dalvie, J. Lee, Y. Nakai, J. O'Donnell, J. Boer and S. Harriman, <span class="italic">Curr. Drug Metab.</span>, 2005, <span class="bold">6</span>, 161–225 <a target="_blank" class="DOILink" href="https://doi.org/10.2174/1389200054021799" title="DOI Link to resource 10.2174/1389200054021799">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXkvF2is7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15975040%5Buid%5D" title="PubMed Link to resource 15975040">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/175_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit176">F. A. Faber, L. Hutchison, B. Huang, J. Gilmer, S. S. Schoenholz, G. E. Dahl, O. Vinyals, S. Kearnes, P. F. Riley and O. A. von Lilienfeld, <span class="italic">J. Chem. Theory Comput.</span>, 2017, <span class="bold">13</span>, 5255–5264 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jctc.7b00577" title="DOI Link to resource 10.1021/acs.jctc.7b00577">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWhu7vL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28926232%5Buid%5D" title="PubMed Link to resource 28926232">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/176_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit177">L. Cheng, M. Welborn, A. S. Christensen and T. F. Miller III, <span class="italic">J. Chem. Phys.</span>, 2019, <span class="bold">150</span>, 131103 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5088393" title="DOI Link to resource 10.1063/1.5088393">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30954042%5Buid%5D" title="PubMed Link to resource 30954042">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/177_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit178">
          T. Che, Y. Li, R. Zhang, R. D. Hjelm, W. Li, Y. Song and Y. Bengio, <span class="italic">Maximum-Likelihood Augmented Discrete Generative Adversarial Networks</span>,  2017 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Maximum-Likelihood%20Augmented%20Discrete%20Generative%20Adversarial%20Networks%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202017%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit179">C. Olah and S. Carter, <span class="italic">Distill</span>, 2016, <span class="bold">1</span> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Distill%5BJour%5D%20AND%201%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/179_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit180">J. H. Jensen, <span class="italic">Chem. Sci.</span>, 2019, <span class="bold">10</span>, 3567–3572 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8SC05372C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8SC05372C">RSC</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/180_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit181">
          Z. Boukouvalas,  2018, arXiv e-prints: 1801.08600.</span></li>
      <li><span id="cit182">
          E. C. Alley, G. Khimulya, S. Biswas, M. AlQuraishi and G. M. Church,  2019, bioRxiv e-prints:10.1101/589333v1.</span></li>
      <li><span id="cit183">
          N. Anand and P. Huang, in <span class="italic">Advances in Neural Information Processing Systems 31</span>, ed. S. Bengio, H. Wallach, H. Larochelle, K. Grauman, N. Cesa-Bianchi and R. Garnett, Curran Associates, Inc.,  2018, pp. 7494–7505 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Advances%20in%20Neural%20Information%20Processing%20Systems%2031%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit184">
          A. Nouira, N. Sokolovska and J.-C. Crivello,  2018, arXiv e-prints:1810.11203.</span></li>
      <li><span id="cit185">
          X. Li, Z. Yang, L. C. Brinson, A. Choudhary, A. Agrawal and W. Chen, <span class="italic">Volume 2B: 44th Design Automation Conference</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Volume%202B:%2044th%20Design%20Automation%20Conference%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit186">
          R. Singh, V. Shah, B. Pokuri, S. Sarkar, B. Ganapathysubramanian and C. Hegde,  2018, arXiv e-prints:1811.09669.</span></li>
      <li><span id="cit187">
          Z. Yang, X. Li, L. C. Brinson, A. N. Choudhary, W. Chen and A. Agrawal,  2018, arXiv e-prints:1805.02791.</span></li>
      <li><span id="cit188">
          W. Chen, A. Jeyaseelan and M. D. Fuge, <span class="italic">ASME 2016 International Design Engineering Technical Conferences and Computers and Information in Engineering Conference</span>, Quebec City, Canada,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ASME%202016%20International%20Design%20Engineering%20Technical%20Conferences%20and%20Computers%20and%20Information%20in%20Engineering%20Conference%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit189">J. Hachmann, M. A. F. Afzal, M. Haghighatlari and Y. Pal, <span class="italic">Mol. Simul.</span>, 2018, <span class="bold">44</span>, 921–929 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/08927022.2018.1471692" title="DOI Link to resource 10.1080/08927022.2018.1471692">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtVSktbfP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/189_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit190">S. K. Saikin, C. Kreisbeck, D. Sheberla, J. S. Becker and A. Aspuru-Guzik, <span class="italic">Expert Opin. Drug Discovery</span>, 2018, <span class="bold">14</span>, 1–4 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/17425255.2018.1414181" title="DOI Link to resource 10.1080/17425255.2018.1414181">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29205069%5Buid%5D" title="PubMed Link to resource 29205069">PubMed</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/190_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit191">P. S. Gromski, A. B. Henson, J. M. Granda and L. Cronin, <span class="italic">Nat. Rev. Chem.</span>, 2019, <span class="bold">3</span>, 119–128 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41570-018-0066-y" title="DOI Link to resource 10.1038/s41570-018-0066-y">CrossRef</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/191_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
      <li><span id="cit192">D. P. Tabor, L. M. Roch, S. K. Saikin, C. Kreisbeck, D. Sheberla, J. H. Montoya, S. Dwaraknath, M. Aykol, C. Ortiz, H. Tribukait, C. Amador-Bedolla, C. J. Brabec, B. Maruyama, K. A. Persson and A. Aspuru-Guzik, <span class="italic">Nat. Rev. Mater.</span>, 2018, <span class="bold">3</span>, 5–20 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41578-018-0005-z" title="DOI Link to resource 10.1038/s41578-018-0005-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht1Cjs73F" title="Link to resource in CAS">CAS</a> <a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000437/c9me00039a/192_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000437"/></a>.</span></li>
    </ol>
    
    
    
    
    
  <hr/><table><tr><td><h3>Footnote</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">Present address: National Institutes of Health Clinical Center, Bethesda, Maryland, USA.</span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2019</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>