<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Opportunities and challenges for machine learning to select combination of donor and acceptor materials for efficient organic solar cells   - Journal of Materials Chemistry C (RSC Publishing) DOI:10.1039/D2TC03276G</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2022/tc/d2tc03276g"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":76,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};;/*! For license information please see nr-loader-spa-1.236.0.min.js.LICENSE.txt */
(()=>{"use strict";var e,t,r={5763:(e,t,r)=>{r.d(t,{P_:()=>l,Mt:()=>g,C5:()=>s,DL:()=>v,OP:()=>T,lF:()=>D,Yu:()=>y,Dg:()=>h,CX:()=>c,GE:()=>b,sU:()=>_});var n=r(8632),i=r(9567);const o={beacon:n.ce.beacon,errorBeacon:n.ce.errorBeacon,licenseKey:void 0,applicationID:void 0,sa:void 0,queueTime:void 0,applicationTime:void 0,ttGuid:void 0,user:void 0,account:void 0,product:void 0,extra:void 0,jsAttributes:{},userAttributes:void 0,atts:void 0,transactionName:void 0,tNamePlain:void 0},a={};function s(e){if(!e)throw new Error("All info objects require an agent identifier!");if(!a[e])throw new Error("Info for ".concat(e," was never set"));return a[e]}function c(e,t){if(!e)throw new Error("All info objects require an agent identifier!");a[e]=(0,i.D)(t,o),(0,n.Qy)(e,a[e],"info")}var u=r(7056);const d=()=>{const e={blockSelector:"[data-nr-block]",maskInputOptions:{password:!0}};return{allow_bfcache:!0,privacy:{cookies_enabled:!0},ajax:{deny_list:void 0,enabled:!0,harvestTimeSeconds:10},distributed_tracing:{enabled:void 0,exclude_newrelic_header:void 0,cors_use_newrelic_header:void 0,cors_use_tracecontext_headers:void 0,allowed_origins:void 0},session:{domain:void 0,expiresMs:u.oD,inactiveMs:u.Hb},ssl:void 0,obfuscate:void 0,jserrors:{enabled:!0,harvestTimeSeconds:10},metrics:{enabled:!0},page_action:{enabled:!0,harvestTimeSeconds:30},page_view_event:{enabled:!0},page_view_timing:{enabled:!0,harvestTimeSeconds:30,long_task:!1},session_trace:{enabled:!0,harvestTimeSeconds:10},harvest:{tooManyRequestsDelay:60},session_replay:{enabled:!1,harvestTimeSeconds:60,sampleRate:.1,errorSampleRate:.1,maskTextSelector:"*",maskAllInputs:!0,get blockClass(){return"nr-block"},get ignoreClass(){return"nr-ignore"},get maskTextClass(){return"nr-mask"},get blockSelector(){return e.blockSelector},set blockSelector(t){e.blockSelector+=",".concat(t)},get maskInputOptions(){return e.maskInputOptions},set maskInputOptions(t){e.maskInputOptions={...t,password:!0}}},spa:{enabled:!0,harvestTimeSeconds:10}}},f={};function l(e){if(!e)throw new Error("All configuration objects require an agent identifier!");if(!f[e])throw new Error("Configuration for ".concat(e," was never set"));return f[e]}function h(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");f[e]=(0,i.D)(t,d()),(0,n.Qy)(e,f[e],"config")}function g(e,t){if(!e)throw new Error("All configuration objects require an agent identifier!");var r=l(e);if(r){for(var n=t.split("."),i=0;i<n.length-1;i++)if("object"!=typeof(r=r[n[i]]))return;r=r[n[n.length-1]]}return r}const p={accountID:void 0,trustKey:void 0,agentID:void 0,licenseKey:void 0,applicationID:void 0,xpid:void 0},m={};function v(e){if(!e)throw new Error("All loader-config objects require an agent identifier!");if(!m[e])throw new Error("LoaderConfig for ".concat(e," was never set"));return m[e]}function b(e,t){if(!e)throw new Error("All loader-config objects require an agent identifier!");m[e]=(0,i.D)(t,p),(0,n.Qy)(e,m[e],"loader_config")}const y=(0,n.mF)().o;var w=r(385),x=r(6818);const A={buildEnv:x.Re,bytesSent:{},queryBytesSent:{},customTransaction:void 0,disabled:!1,distMethod:x.gF,isolatedBacklog:!1,loaderType:void 0,maxBytes:3e4,offset:Math.floor(w._A?.performance?.timeOrigin||w._A?.performance?.timing?.navigationStart||Date.now()),onerror:void 0,origin:""+w._A.location,ptid:void 0,releaseIds:{},session:void 0,xhrWrappable:"function"==typeof w._A.XMLHttpRequest?.prototype?.addEventListener,version:x.q4},E={};function T(e){if(!e)throw new Error("All runtime objects require an agent identifier!");if(!E[e])throw new Error("Runtime for ".concat(e," was never set"));return E[e]}function _(e,t){if(!e)throw new Error("All runtime objects require an agent identifier!");E[e]=(0,i.D)(t,A),(0,n.Qy)(e,E[e],"runtime")}function D(e){return function(e){try{const t=s(e);return!!t.licenseKey&&!!t.errorBeacon&&!!t.applicationID}catch(e){return!1}}(e)}},9567:(e,t,r)=>{r.d(t,{D:()=>i});var n=r(50);function i(e,t){try{if(!e||"object"!=typeof e)return(0,n.Z)("Setting a Configurable requires an object as input");if(!t||"object"!=typeof t)return(0,n.Z)("Setting a Configurable requires a model to set its initial properties");const r=Object.create(Object.getPrototypeOf(t),Object.getOwnPropertyDescriptors(t)),o=0===Object.keys(r).length?e:r;for(let a in o)if(void 0!==e[a])try{"object"==typeof e[a]&&"object"==typeof t[a]?r[a]=i(e[a],t[a]):r[a]=e[a]}catch(e){(0,n.Z)("An error occurred while setting a property of a Configurable",e)}return r}catch(e){(0,n.Z)("An error occured while setting a Configurable",e)}}},6818:(e,t,r)=>{r.d(t,{Re:()=>i,gF:()=>o,q4:()=>n});const n="1.236.0",i="PROD",o="CDN"},385:(e,t,r)=>{r.d(t,{FN:()=>a,IF:()=>u,Nk:()=>f,Tt:()=>s,_A:()=>o,il:()=>n,pL:()=>c,v6:()=>i,w1:()=>d});const n="undefined"!=typeof window&&!!window.document,i="undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self.navigator instanceof WorkerNavigator||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis.navigator instanceof WorkerNavigator),o=n?window:"undefined"!=typeof WorkerGlobalScope&&("undefined"!=typeof self&&self instanceof WorkerGlobalScope&&self||"undefined"!=typeof globalThis&&globalThis instanceof WorkerGlobalScope&&globalThis),a=""+o?.location,s=/iPad|iPhone|iPod/.test(navigator.userAgent),c=s&&"undefined"==typeof SharedWorker,u=(()=>{const e=navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);return Array.isArray(e)&&e.length>=2?+e[1]:0})(),d=Boolean(n&&window.document.documentMode),f=!!navigator.sendBeacon},1117:(e,t,r)=>{r.d(t,{w:()=>o});var n=r(50);const i={agentIdentifier:"",ee:void 0};class o{constructor(e){try{if("object"!=typeof e)return(0,n.Z)("shared context requires an object as input");this.sharedContext={},Object.assign(this.sharedContext,i),Object.entries(e).forEach((e=>{let[t,r]=e;Object.keys(i).includes(t)&&(this.sharedContext[t]=r)}))}catch(e){(0,n.Z)("An error occured while setting SharedContext",e)}}}},8e3:(e,t,r)=>{r.d(t,{L:()=>d,R:()=>c});var n=r(2177),i=r(1284),o=r(4322),a=r(3325);const s={};function c(e,t){const r={staged:!1,priority:a.p[t]||0};u(e),s[e].get(t)||s[e].set(t,r)}function u(e){e&&(s[e]||(s[e]=new Map))}function d(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:"",t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"feature";if(u(e),!e||!s[e].get(t))return a(t);s[e].get(t).staged=!0;const r=[...s[e]];function a(t){const r=e?n.ee.get(e):n.ee,a=o.X.handlers;if(r.backlog&&a){var s=r.backlog[t],c=a[t];if(c){for(var u=0;s&&u<s.length;++u)f(s[u],c);(0,i.D)(c,(function(e,t){(0,i.D)(t,(function(t,r){r[0].on(e,r[1])}))}))}delete a[t],r.backlog[t]=null,r.emit("drain-"+t,[])}}r.every((e=>{let[t,r]=e;return r.staged}))&&(r.sort(((e,t)=>e[1].priority-t[1].priority)),r.forEach((e=>{let[t]=e;a(t)})))}function f(e,t){var r=e[1];(0,i.D)(t[r],(function(t,r){var n=e[0];if(r[0]===n){var i=r[1],o=e[3],a=e[2];i.apply(o,a)}}))}},2177:(e,t,r)=>{r.d(t,{c:()=>f,ee:()=>u});var n=r(8632),i=r(2210),o=r(1284),a=r(5763),s="nr@context";let c=(0,n.fP)();var u;function d(){}function f(e){return(0,i.X)(e,s,l)}function l(){return new d}function h(){u.aborted=!0,u.backlog={}}c.ee?u=c.ee:(u=function e(t,r){var n={},c={},f={},g=!1;try{g=16===r.length&&(0,a.OP)(r).isolatedBacklog}catch(e){}var p={on:b,addEventListener:b,removeEventListener:y,emit:v,get:x,listeners:w,context:m,buffer:A,abort:h,aborted:!1,isBuffering:E,debugId:r,backlog:g?{}:t&&"object"==typeof t.backlog?t.backlog:{}};return p;function m(e){return e&&e instanceof d?e:e?(0,i.X)(e,s,l):l()}function v(e,r,n,i,o){if(!1!==o&&(o=!0),!u.aborted||i){t&&o&&t.emit(e,r,n);for(var a=m(n),s=w(e),d=s.length,f=0;f<d;f++)s[f].apply(a,r);var l=T()[c[e]];return l&&l.push([p,e,r,a]),a}}function b(e,t){n[e]=w(e).concat(t)}function y(e,t){var r=n[e];if(r)for(var i=0;i<r.length;i++)r[i]===t&&r.splice(i,1)}function w(e){return n[e]||[]}function x(t){return f[t]=f[t]||e(p,t)}function A(e,t){var r=T();p.aborted||(0,o.D)(e,(function(e,n){t=t||"feature",c[n]=t,t in r||(r[t]=[])}))}function E(e){return!!T()[c[e]]}function T(){return p.backlog}}(void 0,"globalEE"),c.ee=u)},5546:(e,t,r)=>{r.d(t,{E:()=>n,p:()=>i});var n=r(2177).ee.get("handle");function i(e,t,r,i,o){o?(o.buffer([e],i),o.emit(e,t,r)):(n.buffer([e],i),n.emit(e,t,r))}},4322:(e,t,r)=>{r.d(t,{X:()=>o});var n=r(5546);o.on=a;var i=o.handlers={};function o(e,t,r,o){a(o||n.E,i,e,t,r)}function a(e,t,r,i,o){o||(o="feature"),e||(e=n.E);var a=t[o]=t[o]||{};(a[r]=a[r]||[]).push([e,i])}},3239:(e,t,r)=>{r.d(t,{bP:()=>s,iz:()=>c,m$:()=>a});var n=r(385);let i=!1,o=!1;try{const e={get passive(){return i=!0,!1},get signal(){return o=!0,!1}};n._A.addEventListener("test",null,e),n._A.removeEventListener("test",null,e)}catch(e){}function a(e,t){return i||o?{capture:!!e,passive:i,signal:t}:!!e}function s(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;window.addEventListener(e,t,a(r,n))}function c(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2],n=arguments.length>3?arguments[3]:void 0;document.addEventListener(e,t,a(r,n))}},4402:(e,t,r)=>{r.d(t,{Ht:()=>u,M:()=>c,Rl:()=>a,ky:()=>s});var n=r(385);const i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";function o(e,t){return e?15&e[t]:16*Math.random()|0}function a(){const e=n._A?.crypto||n._A?.msCrypto;let t,r=0;return e&&e.getRandomValues&&(t=e.getRandomValues(new Uint8Array(31))),i.split("").map((e=>"x"===e?o(t,++r).toString(16):"y"===e?(3&o()|8).toString(16):e)).join("")}function s(e){const t=n._A?.crypto||n._A?.msCrypto;let r,i=0;t&&t.getRandomValues&&(r=t.getRandomValues(new Uint8Array(31)));const a=[];for(var s=0;s<e;s++)a.push(o(r,++i).toString(16));return a.join("")}function c(){return s(16)}function u(){return s(32)}},7056:(e,t,r)=>{r.d(t,{Bq:()=>n,Hb:()=>o,oD:()=>i});const n="NRBA",i=144e5,o=18e5},7894:(e,t,r)=>{function n(){return Math.round(performance.now())}r.d(t,{z:()=>n})},7243:(e,t,r)=>{r.d(t,{e:()=>o});var n=r(385),i={};function o(e){if(e in i)return i[e];if(0===(e||"").indexOf("data:"))return{protocol:"data"};let t;var r=n._A?.location,o={};if(n.il)t=document.createElement("a"),t.href=e;else try{t=new URL(e,r.href)}catch(e){return o}o.port=t.port;var a=t.href.split("://");!o.port&&a[1]&&(o.port=a[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===a[0]?"443":"80"),o.hostname=t.hostname||r.hostname,o.pathname=t.pathname,o.protocol=a[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var s=!t.protocol||":"===t.protocol||t.protocol===r.protocol,c=t.hostname===r.hostname&&t.port===r.port;return o.sameOrigin=s&&(!t.hostname||c),"/"===o.pathname&&(i[e]=o),o}},50:(e,t,r)=>{function n(e,t){"function"==typeof console.warn&&(console.warn("New Relic: ".concat(e)),t&&console.warn(t))}r.d(t,{Z:()=>n})},2587:(e,t,r)=>{r.d(t,{N:()=>c,T:()=>u});var n=r(2177),i=r(5546),o=r(8e3),a=r(3325);const s={stn:[a.D.sessionTrace],err:[a.D.jserrors,a.D.metrics],ins:[a.D.pageAction],spa:[a.D.spa],sr:[a.D.sessionReplay,a.D.sessionTrace]};function c(e,t){const r=n.ee.get(t);e&&"object"==typeof e&&(Object.entries(e).forEach((e=>{let[t,n]=e;void 0===u[t]&&(s[t]?s[t].forEach((e=>{n?(0,i.p)("feat-"+t,[],void 0,e,r):(0,i.p)("block-"+t,[],void 0,e,r),(0,i.p)("rumresp-"+t,[Boolean(n)],void 0,e,r)})):n&&(0,i.p)("feat-"+t,[],void 0,void 0,r),u[t]=Boolean(n))})),Object.keys(s).forEach((e=>{void 0===u[e]&&(s[e]?.forEach((t=>(0,i.p)("rumresp-"+e,[!1],void 0,t,r))),u[e]=!1)})),(0,o.L)(t,a.D.pageViewEvent))}const u={}},2210:(e,t,r)=>{r.d(t,{X:()=>i});var n=Object.prototype.hasOwnProperty;function i(e,t,r){if(n.call(e,t))return e[t];var i=r();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,t,{value:i,writable:!0,enumerable:!1}),i}catch(e){}return e[t]=i,i}},1284:(e,t,r)=>{r.d(t,{D:()=>n});const n=(e,t)=>Object.entries(e||{}).map((e=>{let[r,n]=e;return t(r,n)}))},4351:(e,t,r)=>{r.d(t,{P:()=>o});var n=r(2177);const i=()=>{const e=new WeakSet;return(t,r)=>{if("object"==typeof r&&null!==r){if(e.has(r))return;e.add(r)}return r}};function o(e){try{return JSON.stringify(e,i())}catch(e){try{n.ee.emit("internal-error",[e])}catch(e){}}}},3960:(e,t,r)=>{r.d(t,{K:()=>a,b:()=>o});var n=r(3239);function i(){return"undefined"==typeof document||"complete"===document.readyState}function o(e,t){if(i())return e();(0,n.bP)("load",e,t)}function a(e){if(i())return e();(0,n.iz)("DOMContentLoaded",e)}},8632:(e,t,r)=>{r.d(t,{EZ:()=>u,Qy:()=>c,ce:()=>o,fP:()=>a,gG:()=>d,mF:()=>s});var n=r(7894),i=r(385);const o={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net"};function a(){return i._A.NREUM||(i._A.NREUM={}),void 0===i._A.newrelic&&(i._A.newrelic=i._A.NREUM),i._A.NREUM}function s(){let e=a();return e.o||(e.o={ST:i._A.setTimeout,SI:i._A.setImmediate,CT:i._A.clearTimeout,XHR:i._A.XMLHttpRequest,REQ:i._A.Request,EV:i._A.Event,PR:i._A.Promise,MO:i._A.MutationObserver,FETCH:i._A.fetch}),e}function c(e,t,r){let i=a();const o=i.initializedAgents||{},s=o[e]||{};return Object.keys(s).length||(s.initializedAt={ms:(0,n.z)(),date:new Date}),i.initializedAgents={...o,[e]:{...s,[r]:t}},i}function u(e,t){a()[e]=t}function d(){return function(){let e=a();const t=e.info||{};e.info={beacon:o.beacon,errorBeacon:o.errorBeacon,...t}}(),function(){let e=a();const t=e.init||{};e.init={...t}}(),s(),function(){let e=a();const t=e.loader_config||{};e.loader_config={...t}}(),a()}},7956:(e,t,r)=>{r.d(t,{N:()=>i});var n=r(3239);function i(e){let t=arguments.length>1&&void 0!==arguments[1]&&arguments[1],r=arguments.length>2?arguments[2]:void 0,i=arguments.length>3?arguments[3]:void 0;return void(0,n.iz)("visibilitychange",(function(){if(t)return void("hidden"==document.visibilityState&&e());e(document.visibilityState)}),r,i)}},1214:(e,t,r)=>{r.d(t,{em:()=>v,u5:()=>N,QU:()=>S,_L:()=>I,Gm:()=>L,Lg:()=>M,gy:()=>U,BV:()=>Q,Kf:()=>ee});var n=r(2177);const i="nr@original";var o=Object.prototype.hasOwnProperty,a=!1;function s(e,t){return e||(e=n.ee),r.inPlace=function(e,t,n,i,o){n||(n="");var a,s,c,u="-"===n.charAt(0);for(c=0;c<t.length;c++)d(a=e[s=t[c]])||(e[s]=r(a,u?s+n:n,i,s,o))},r.flag=i,r;function r(t,r,n,a,s){return d(t)?t:(r||(r=""),nrWrapper[i]=t,u(t,nrWrapper,e),nrWrapper);function nrWrapper(){var i,u,d,f;try{u=this,i=[...arguments],d="function"==typeof n?n(i,u):n||{}}catch(t){c([t,"",[i,u,a],d],e)}o(r+"start",[i,u,a],d,s);try{return f=t.apply(u,i)}catch(e){throw o(r+"err",[i,u,e],d,s),e}finally{o(r+"end",[i,u,f],d,s)}}}function o(r,n,i,o){if(!a||t){var s=a;a=!0;try{e.emit(r,n,i,t,o)}catch(t){c([t,r,n,i],e)}a=s}}}function c(e,t){t||(t=n.ee);try{t.emit("internal-error",e)}catch(e){}}function u(e,t,r){if(Object.defineProperty&&Object.keys)try{return Object.keys(e).forEach((function(r){Object.defineProperty(t,r,{get:function(){return e[r]},set:function(t){return e[r]=t,t}})})),t}catch(e){c([e],r)}for(var n in e)o.call(e,n)&&(t[n]=e[n]);return t}function d(e){return!(e&&e instanceof Function&&e.apply&&!e[i])}var f=r(2210),l=r(385);const h={},g=XMLHttpRequest,p="addEventListener",m="removeEventListener";function v(e){var t=function(e){return(e||n.ee).get("events")}(e);if(h[t.debugId]++)return t;h[t.debugId]=1;var r=s(t,!0);function i(e){r.inPlace(e,[p,m],"-",o)}function o(e,t){return e[1]}return"getPrototypeOf"in Object&&(l.il&&b(document,i),b(l._A,i),b(g.prototype,i)),t.on(p+"-start",(function(e,t){var n=e[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var i=(0,f.X)(n,"nr@wrapped",(function(){var e={object:function(){if("function"!=typeof n.handleEvent)return;return n.handleEvent.apply(n,arguments)},function:n}[typeof n];return e?r(e,"fn-",null,e.name||"anonymous"):n}));this.wrapped=e[1]=i}})),t.on(m+"-start",(function(e){e[1]=this.wrapped||e[1]})),t}function b(e,t){let r=e;for(;"object"==typeof r&&!Object.prototype.hasOwnProperty.call(r,p);)r=Object.getPrototypeOf(r);for(var n=arguments.length,i=new Array(n>2?n-2:0),o=2;o<n;o++)i[o-2]=arguments[o];r&&t(r,...i)}var y="fetch-",w=y+"body-",x=["arrayBuffer","blob","json","text","formData"],A=l._A.Request,E=l._A.Response,T="prototype",_="nr@context";const D={};function N(e){const t=function(e){return(e||n.ee).get("fetch")}(e);if(!(A&&E&&l._A.fetch))return t;if(D[t.debugId]++)return t;function r(e,r,n){var i=e[r];"function"==typeof i&&(e[r]=function(){var e,r=[...arguments],o={};t.emit(n+"before-start",[r],o),o[_]&&o[_].dt&&(e=o[_].dt);var a=i.apply(this,r);return t.emit(n+"start",[r,e],a),a.then((function(e){return t.emit(n+"end",[null,e],a),e}),(function(e){throw t.emit(n+"end",[e],a),e}))})}return D[t.debugId]=1,x.forEach((e=>{r(A[T],e,w),r(E[T],e,w)})),r(l._A,"fetch",y),t.on(y+"end",(function(e,r){var n=this;if(r){var i=r.headers.get("content-length");null!==i&&(n.rxSize=i),t.emit(y+"done",[null,r],n)}else t.emit(y+"done",[e],n)})),t}const O={},j=["pushState","replaceState"];function S(e){const t=function(e){return(e||n.ee).get("history")}(e);return!l.il||O[t.debugId]++||(O[t.debugId]=1,s(t).inPlace(window.history,j,"-")),t}var P=r(3239);const C={},R=["appendChild","insertBefore","replaceChild"];function I(e){const t=function(e){return(e||n.ee).get("jsonp")}(e);if(!l.il||C[t.debugId])return t;C[t.debugId]=!0;var r=s(t),i=/[?&](?:callback|cb)=([^&#]+)/,o=/(.*)\.([^.]+)/,a=/^(\w+)(\.|$)(.*)$/;function c(e,t){var r=e.match(a),n=r[1],i=r[3];return i?c(i,t[n]):t[n]}return r.inPlace(Node.prototype,R,"dom-"),t.on("dom-start",(function(e){!function(e){if(!e||"string"!=typeof e.nodeName||"script"!==e.nodeName.toLowerCase())return;if("function"!=typeof e.addEventListener)return;var n=(a=e.src,s=a.match(i),s?s[1]:null);var a,s;if(!n)return;var u=function(e){var t=e.match(o);if(t&&t.length>=3)return{key:t[2],parent:c(t[1],window)};return{key:e,parent:window}}(n);if("function"!=typeof u.parent[u.key])return;var d={};function f(){t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}function l(){t.emit("jsonp-error",[],d),t.emit("jsonp-end",[],d),e.removeEventListener("load",f,(0,P.m$)(!1)),e.removeEventListener("error",l,(0,P.m$)(!1))}r.inPlace(u.parent,[u.key],"cb-",d),e.addEventListener("load",f,(0,P.m$)(!1)),e.addEventListener("error",l,(0,P.m$)(!1)),t.emit("new-jsonp",[e.src],d)}(e[0])})),t}var k=r(5763);const H={};function L(e){const t=function(e){return(e||n.ee).get("mutation")}(e);if(!l.il||H[t.debugId])return t;H[t.debugId]=!0;var r=s(t),i=k.Yu.MO;return i&&(window.MutationObserver=function(e){return this instanceof i?new i(r(e,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype),t}const z={};function M(e){const t=function(e){return(e||n.ee).get("promise")}(e);if(z[t.debugId])return t;z[t.debugId]=!0;var r=n.c,o=s(t),a=k.Yu.PR;return a&&function(){function e(r){var n=t.context(),i=o(r,"executor-",n,null,!1);const s=Reflect.construct(a,[i],e);return t.context(s).getCtx=function(){return n},s}l._A.Promise=e,Object.defineProperty(e,"name",{value:"Promise"}),e.toString=function(){return a.toString()},Object.setPrototypeOf(e,a),["all","race"].forEach((function(r){const n=a[r];e[r]=function(e){let i=!1;[...e||[]].forEach((e=>{this.resolve(e).then(a("all"===r),a(!1))}));const o=n.apply(this,arguments);return o;function a(e){return function(){t.emit("propagate",[null,!i],o,!1,!1),i=i||!e}}}})),["resolve","reject"].forEach((function(r){const n=a[r];e[r]=function(e){const r=n.apply(this,arguments);return e!==r&&t.emit("propagate",[e,!0],r,!1,!1),r}})),e.prototype=a.prototype;const n=a.prototype.then;a.prototype.then=function(){var e=this,i=r(e);i.promise=e;for(var a=arguments.length,s=new Array(a),c=0;c<a;c++)s[c]=arguments[c];s[0]=o(s[0],"cb-",i,null,!1),s[1]=o(s[1],"cb-",i,null,!1);const u=n.apply(this,s);return i.nextPromise=u,t.emit("propagate",[e,!0],u,!1,!1),u},a.prototype.then[i]=n,t.on("executor-start",(function(e){e[0]=o(e[0],"resolve-",this,null,!1),e[1]=o(e[1],"resolve-",this,null,!1)})),t.on("executor-err",(function(e,t,r){e[1](r)})),t.on("cb-end",(function(e,r,n){t.emit("propagate",[n,!0],this.nextPromise,!1,!1)})),t.on("propagate",(function(e,r,n){this.getCtx&&!r||(this.getCtx=function(){if(e instanceof Promise)var r=t.context(e);return r&&r.getCtx?r.getCtx():this})}))}(),t}const B={},F="requestAnimationFrame";function U(e){const t=function(e){return(e||n.ee).get("raf")}(e);if(!l.il||B[t.debugId]++)return t;B[t.debugId]=1;var r=s(t);return r.inPlace(window,[F],"raf-"),t.on("raf-start",(function(e){e[0]=r(e[0],"fn-")})),t}const q={},G="setTimeout",V="setInterval",X="clearTimeout",W="-start",Z="-",$=[G,"setImmediate",V,X,"clearImmediate"];function Q(e){const t=function(e){return(e||n.ee).get("timer")}(e);if(q[t.debugId]++)return t;q[t.debugId]=1;var r=s(t);return r.inPlace(l._A,$.slice(0,2),G+Z),r.inPlace(l._A,$.slice(2,3),V+Z),r.inPlace(l._A,$.slice(3),X+Z),t.on(V+W,(function(e,t,n){e[0]=r(e[0],"fn-",null,n)})),t.on(G+W,(function(e,t,n){this.method=n,this.timerDuration=isNaN(e[1])?0:+e[1],e[0]=r(e[0],"fn-",this,n)})),t}var Y=r(50);const K={},J=["open","send"];function ee(e){var t=e||n.ee;const r=function(e){return(e||n.ee).get("xhr")}(t);if(K[r.debugId]++)return r;K[r.debugId]=1,v(t);var i=s(r),o=k.Yu.XHR,a=k.Yu.MO,c=k.Yu.PR,u=k.Yu.SI,d="readystatechange",f=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],h=[],g=l._A.XMLHttpRequest.listeners,p=l._A.XMLHttpRequest=function(e){var t=new o(e);function n(){try{r.emit("new-xhr",[t],t),t.addEventListener(d,b,(0,P.m$)(!1))}catch(e){(0,Y.Z)("An error occured while intercepting XHR",e);try{r.emit("internal-error",[e])}catch(e){}}}return this.listeners=g?[...g,n]:[n],this.listeners.forEach((e=>e())),t};function m(e,t){i.inPlace(t,["onreadystatechange"],"fn-",E)}function b(){var e=this,t=r.context(e);e.readyState>3&&!t.resolved&&(t.resolved=!0,r.emit("xhr-resolved",[],e)),i.inPlace(e,f,"fn-",E)}if(function(e,t){for(var r in e)t[r]=e[r]}(o,p),p.prototype=o.prototype,i.inPlace(p.prototype,J,"-xhr-",E),r.on("send-xhr-start",(function(e,t){m(e,t),function(e){h.push(e),a&&(y?y.then(A):u?u(A):(w=-w,x.data=w))}(t)})),r.on("open-xhr-start",m),a){var y=c&&c.resolve();if(!u&&!c){var w=1,x=document.createTextNode(w);new a(A).observe(x,{characterData:!0})}}else t.on("fn-end",(function(e){e[0]&&e[0].type===d||A()}));function A(){for(var e=0;e<h.length;e++)m(0,h[e]);h.length&&(h=[])}function E(e,t){return t}return r}},7825:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.ajax},6660:(e,t,r)=>{r.d(t,{A:()=>i,t:()=>n});const n=r(3325).D.jserrors,i="nr@seenError"},3081:(e,t,r)=>{r.d(t,{gF:()=>o,mY:()=>i,t9:()=>n,vz:()=>s,xS:()=>a});const n=r(3325).D.metrics,i="sm",o="cm",a="storeSupportabilityMetrics",s="storeEventMetrics"},4649:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageAction},7633:(e,t,r)=>{r.d(t,{Dz:()=>i,OJ:()=>a,qw:()=>o,t9:()=>n});const n=r(3325).D.pageViewEvent,i="firstbyte",o="domcontent",a="windowload"},9251:(e,t,r)=>{r.d(t,{t:()=>n});const n=r(3325).D.pageViewTiming},3614:(e,t,r)=>{r.d(t,{BST_RESOURCE:()=>i,END:()=>s,FEATURE_NAME:()=>n,FN_END:()=>u,FN_START:()=>c,PUSH_STATE:()=>d,RESOURCE:()=>o,START:()=>a});const n=r(3325).D.sessionTrace,i="bstResource",o="resource",a="-start",s="-end",c="fn"+a,u="fn"+s,d="pushState"},7836:(e,t,r)=>{r.d(t,{BODY:()=>A,CB_END:()=>E,CB_START:()=>u,END:()=>x,FEATURE_NAME:()=>i,FETCH:()=>_,FETCH_BODY:()=>v,FETCH_DONE:()=>m,FETCH_START:()=>p,FN_END:()=>c,FN_START:()=>s,INTERACTION:()=>l,INTERACTION_API:()=>d,INTERACTION_EVENTS:()=>o,JSONP_END:()=>b,JSONP_NODE:()=>g,JS_TIME:()=>T,MAX_TIMER_BUDGET:()=>a,REMAINING:()=>f,SPA_NODE:()=>h,START:()=>w,originalSetTimeout:()=>y});var n=r(5763);const i=r(3325).D.spa,o=["click","submit","keypress","keydown","keyup","change"],a=999,s="fn-start",c="fn-end",u="cb-start",d="api-ixn-",f="remaining",l="interaction",h="spaNode",g="jsonpNode",p="fetch-start",m="fetch-done",v="fetch-body-",b="jsonp-end",y=n.Yu.ST,w="-start",x="-end",A="-body",E="cb"+x,T="jsTime",_="fetch"},5938:(e,t,r)=>{r.d(t,{W:()=>o});var n=r(5763),i=r(2177);class o{constructor(e,t,r){this.agentIdentifier=e,this.aggregator=t,this.ee=i.ee.get(e,(0,n.OP)(this.agentIdentifier).isolatedBacklog),this.featureName=r,this.blocked=!1}}},9144:(e,t,r)=>{r.d(t,{j:()=>m});var n=r(3325),i=r(5763),o=r(5546),a=r(2177),s=r(7894),c=r(8e3),u=r(3960),d=r(385),f=r(50),l=r(3081),h=r(8632);function g(){const e=(0,h.gG)();["setErrorHandler","finished","addToTrace","inlineHit","addRelease","addPageAction","setCurrentRouteName","setPageViewName","setCustomAttribute","interaction","noticeError","setUserId"].forEach((t=>{e[t]=function(){for(var r=arguments.length,n=new Array(r),i=0;i<r;i++)n[i]=arguments[i];return function(t){for(var r=arguments.length,n=new Array(r>1?r-1:0),i=1;i<r;i++)n[i-1]=arguments[i];let o=[];return Object.values(e.initializedAgents).forEach((e=>{e.exposed&&e.api[t]&&o.push(e.api[t](...n))})),o.length>1?o:o[0]}(t,...n)}}))}var p=r(2587);function m(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:{},m=arguments.length>2?arguments[2]:void 0,v=arguments.length>3?arguments[3]:void 0,{init:b,info:y,loader_config:w,runtime:x={loaderType:m},exposed:A=!0}=t;const E=(0,h.gG)();y||(b=E.init,y=E.info,w=E.loader_config),(0,i.Dg)(e,b||{}),(0,i.GE)(e,w||{}),(0,i.sU)(e,x),y.jsAttributes??={},d.v6&&(y.jsAttributes.isWorker=!0),(0,i.CX)(e,y),g();const T=function(e,t){t||(0,c.R)(e,"api");const h={};var g=a.ee.get(e),p=g.get("tracer"),m="api-",v=m+"ixn-";function b(t,r,n,o){const a=(0,i.C5)(e);return null===r?delete a.jsAttributes[t]:(0,i.CX)(e,{...a,jsAttributes:{...a.jsAttributes,[t]:r}}),x(m,n,!0,o||null===r?"session":void 0)(t,r)}function y(){}["setErrorHandler","finished","addToTrace","inlineHit","addRelease"].forEach((e=>h[e]=x(m,e,!0,"api"))),h.addPageAction=x(m,"addPageAction",!0,n.D.pageAction),h.setCurrentRouteName=x(m,"routeName",!0,n.D.spa),h.setPageViewName=function(t,r){if("string"==typeof t)return"/"!==t.charAt(0)&&(t="/"+t),(0,i.OP)(e).customTransaction=(r||"http://custom.transaction")+t,x(m,"setPageViewName",!0)()},h.setCustomAttribute=function(e,t){let r=arguments.length>2&&void 0!==arguments[2]&&arguments[2];if("string"==typeof e){if(["string","number"].includes(typeof t)||null===t)return b(e,t,"setCustomAttribute",r);(0,f.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t,"> was provided."))}else(0,f.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.setUserId=function(e){if("string"==typeof e||null===e)return b("enduser.id",e,"setUserId",!0);(0,f.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e,"> was provided."))},h.interaction=function(){return(new y).get()};var w=y.prototype={createTracer:function(e,t){var r={},i=this,a="function"==typeof t;return(0,o.p)(v+"tracer",[(0,s.z)(),e,r],i,n.D.spa,g),function(){if(p.emit((a?"":"no-")+"fn-start",[(0,s.z)(),i,a],r),a)try{return t.apply(this,arguments)}catch(e){throw p.emit("fn-err",[arguments,this,"string"==typeof e?new Error(e):e],r),e}finally{p.emit("fn-end",[(0,s.z)()],r)}}}};function x(e,t,r,i){return function(){return(0,o.p)(l.xS,["API/"+t+"/called"],void 0,n.D.metrics,g),i&&(0,o.p)(e+t,[(0,s.z)(),...arguments],r?null:this,i,g),r?void 0:this}}function A(){r.e(439).then(r.bind(r,7438)).then((t=>{let{setAPI:r}=t;r(e),(0,c.L)(e,"api")})).catch((()=>(0,f.Z)("Downloading runtime APIs failed...")))}return["actionText","setName","setAttribute","save","ignore","onEnd","getContext","end","get"].forEach((e=>{w[e]=x(v,e,void 0,n.D.spa)})),h.noticeError=function(e,t){"string"==typeof e&&(e=new Error(e)),(0,o.p)(l.xS,["API/noticeError/called"],void 0,n.D.metrics,g),(0,o.p)("err",[e,(0,s.z)(),!1,t],void 0,n.D.jserrors,g)},d.il?(0,u.b)((()=>A()),!0):A(),h}(e,v);return(0,h.Qy)(e,T,"api"),(0,h.Qy)(e,A,"exposed"),(0,h.EZ)("activatedFeatures",p.T),T}},3325:(e,t,r)=>{r.d(t,{D:()=>n,p:()=>i});const n={ajax:"ajax",jserrors:"jserrors",metrics:"metrics",pageAction:"page_action",pageViewEvent:"page_view_event",pageViewTiming:"page_view_timing",sessionReplay:"session_replay",sessionTrace:"session_trace",spa:"spa"},i={[n.pageViewEvent]:1,[n.pageViewTiming]:2,[n.metrics]:3,[n.jserrors]:4,[n.ajax]:5,[n.sessionTrace]:6,[n.pageAction]:7,[n.spa]:8,[n.sessionReplay]:9}}},n={};function i(e){var t=n[e];if(void 0!==t)return t.exports;var o=n[e]={exports:{}};return r[e](o,o.exports,i),o.exports}i.m=r,i.d=(e,t)=>{for(var r in t)i.o(t,r)&&!i.o(e,r)&&Object.defineProperty(e,r,{enumerable:!0,get:t[r]})},i.f={},i.e=e=>Promise.all(Object.keys(i.f).reduce(((t,r)=>(i.f[r](e,t),t)),[])),i.u=e=>(({78:"page_action-aggregate",147:"metrics-aggregate",242:"session-manager",317:"jserrors-aggregate",348:"page_view_timing-aggregate",412:"lazy-feature-loader",439:"async-api",538:"recorder",590:"session_replay-aggregate",675:"compressor",733:"session_trace-aggregate",786:"page_view_event-aggregate",873:"spa-aggregate",898:"ajax-aggregate"}[e]||e)+"."+{78:"ac76d497",147:"3dc53903",148:"1a20d5fe",242:"2a64278a",317:"49e41428",348:"bd6de33a",412:"2f55ce66",439:"30bd804e",538:"1b18459f",590:"cf0efb30",675:"ae9f91a8",733:"83105561",786:"06482edd",860:"03a8b7a5",873:"e6b09d52",898:"998ef92b"}[e]+"-1.236.0.min.js"),i.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),e={},t="NRBA:",i.l=(r,n,o,a)=>{if(e[r])e[r].push(n);else{var s,c;if(void 0!==o)for(var u=document.getElementsByTagName("script"),d=0;d<u.length;d++){var f=u[d];if(f.getAttribute("src")==r||f.getAttribute("data-webpack")==t+o){s=f;break}}s||(c=!0,(s=document.createElement("script")).charset="utf-8",s.timeout=120,i.nc&&s.setAttribute("nonce",i.nc),s.setAttribute("data-webpack",t+o),s.src=r),e[r]=[n];var l=(t,n)=>{s.onerror=s.onload=null,clearTimeout(h);var i=e[r];if(delete e[r],s.parentNode&&s.parentNode.removeChild(s),i&&i.forEach((e=>e(n))),t)return t(n)},h=setTimeout(l.bind(null,void 0,{type:"timeout",target:s}),12e4);s.onerror=l.bind(null,s.onerror),s.onload=l.bind(null,s.onload),c&&document.head.appendChild(s)}},i.r=e=>{"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},i.j=364,i.p="https://js-agent.newrelic.com/",(()=>{var e={364:0,953:0};i.f.j=(t,r)=>{var n=i.o(e,t)?e[t]:void 0;if(0!==n)if(n)r.push(n[2]);else{var o=new Promise(((r,i)=>n=e[t]=[r,i]));r.push(n[2]=o);var a=i.p+i.u(t),s=new Error;i.l(a,(r=>{if(i.o(e,t)&&(0!==(n=e[t])&&(e[t]=void 0),n)){var o=r&&("load"===r.type?"missing":r.type),a=r&&r.target&&r.target.src;s.message="Loading chunk "+t+" failed.\n("+o+": "+a+")",s.name="ChunkLoadError",s.type=o,s.request=a,n[1](s)}}),"chunk-"+t,t)}};var t=(t,r)=>{var n,o,[a,s,c]=r,u=0;if(a.some((t=>0!==e[t]))){for(n in s)i.o(s,n)&&(i.m[n]=s[n]);if(c)c(i)}for(t&&t(r);u<a.length;u++)o=a[u],i.o(e,o)&&e[o]&&e[o][0](),e[o]=0},r=window.webpackChunkNRBA=window.webpackChunkNRBA||[];r.forEach(t.bind(null,0)),r.push=t.bind(null,r.push.bind(r))})();var o={};(()=>{i.r(o);var e=i(3325),t=i(5763);const r=Object.values(e.D);function n(e){const n={};return r.forEach((r=>{n[r]=function(e,r){return!1!==(0,t.Mt)(r,"".concat(e,".enabled"))}(r,e)})),n}var a=i(9144);var s=i(5546),c=i(385),u=i(8e3),d=i(5938),f=i(3960),l=i(50);class h extends d.W{constructor(e,t,r){let n=!(arguments.length>3&&void 0!==arguments[3])||arguments[3];super(e,t,r),this.auto=n,this.abortHandler,this.featAggregate,this.onAggregateImported,n&&(0,u.R)(e,r)}importAggregator(){let e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{};if(this.featAggregate||!this.auto)return;const r=c.il&&!0===(0,t.Mt)(this.agentIdentifier,"privacy.cookies_enabled");let n;this.onAggregateImported=new Promise((e=>{n=e}));const o=async()=>{let t;try{if(r){const{setupAgentSession:e}=await Promise.all([i.e(860),i.e(242)]).then(i.bind(i,3228));t=e(this.agentIdentifier)}}catch(e){(0,l.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.",e)}try{if(!this.shouldImportAgg(this.featureName,t))return void(0,u.L)(this.agentIdentifier,this.featureName);const{lazyFeatureLoader:r}=await i.e(412).then(i.bind(i,8582)),{Aggregate:o}=await r(this.featureName,"aggregate");this.featAggregate=new o(this.agentIdentifier,this.aggregator,e),n(!0)}catch(e){(0,l.Z)("Downloading and initializing ".concat(this.featureName," failed..."),e),this.abortHandler?.(),n(!1)}};c.il?(0,f.b)((()=>o()),!0):o()}shouldImportAgg(r,n){return r!==e.D.sessionReplay||!1!==(0,t.Mt)(this.agentIdentifier,"session_trace.enabled")&&(!!n?.isNew||!!n?.state.sessionReplay)}}var g=i(7633),p=i(7894);class m extends h{static featureName=g.t9;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];if(super(r,n,g.t9,i),("undefined"==typeof PerformanceNavigationTiming||c.Tt)&&"undefined"!=typeof PerformanceTiming){const n=(0,t.OP)(r);n[g.Dz]=Math.max(Date.now()-n.offset,0),(0,f.K)((()=>n[g.qw]=Math.max((0,p.z)()-n[g.Dz],0))),(0,f.b)((()=>{const t=(0,p.z)();n[g.OJ]=Math.max(t-n[g.Dz],0),(0,s.p)("timing",["load",t],void 0,e.D.pageViewTiming,this.ee)}))}this.importAggregator()}}var v=i(1117),b=i(1284);class y extends v.w{constructor(e){super(e),this.aggregatedData={}}store(e,t,r,n,i){var o=this.getBucket(e,t,r,i);return o.metrics=function(e,t){t||(t={count:0});return t.count+=1,(0,b.D)(e,(function(e,r){t[e]=w(r,t[e])})),t}(n,o.metrics),o}merge(e,t,r,n,i){var o=this.getBucket(e,t,n,i);if(o.metrics){var a=o.metrics;a.count+=r.count,(0,b.D)(r,(function(e,t){if("count"!==e){var n=a[e],i=r[e];i&&!i.c?a[e]=w(i.t,n):a[e]=function(e,t){if(!t)return e;t.c||(t=x(t.t));return t.min=Math.min(e.min,t.min),t.max=Math.max(e.max,t.max),t.t+=e.t,t.sos+=e.sos,t.c+=e.c,t}(i,a[e])}}))}else o.metrics=r}storeMetric(e,t,r,n){var i=this.getBucket(e,t,r);return i.stats=w(n,i.stats),i}getBucket(e,t,r,n){this.aggregatedData[e]||(this.aggregatedData[e]={});var i=this.aggregatedData[e][t];return i||(i=this.aggregatedData[e][t]={params:r||{}},n&&(i.custom=n)),i}get(e,t){return t?this.aggregatedData[e]&&this.aggregatedData[e][t]:this.aggregatedData[e]}take(e){for(var t={},r="",n=!1,i=0;i<e.length;i++)t[r=e[i]]=A(this.aggregatedData[r]),t[r].length&&(n=!0),delete this.aggregatedData[r];return n?t:null}}function w(e,t){return null==e?function(e){e?e.c++:e={c:1};return e}(t):t?(t.c||(t=x(t.t)),t.c+=1,t.t+=e,t.sos+=e*e,e>t.max&&(t.max=e),e<t.min&&(t.min=e),t):{t:e}}function x(e){return{t:e,min:e,max:e,sos:e*e,c:1}}function A(e){return"object"!=typeof e?[]:(0,b.D)(e,E)}function E(e,t){return t}var T=i(8632),_=i(4402),D=i(4351);var N=i(7956),O=i(3239),j=i(9251);class S extends h{static featureName=j.t;constructor(e,r){let n=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,r,j.t,n),c.il&&((0,t.OP)(e).initHidden=Boolean("hidden"===document.visibilityState),(0,N.N)((()=>(0,s.p)("docHidden",[(0,p.z)()],void 0,j.t,this.ee)),!0),(0,O.bP)("pagehide",(()=>(0,s.p)("winPagehide",[(0,p.z)()],void 0,j.t,this.ee))),this.importAggregator())}}var P=i(3081);class C extends h{static featureName=P.t9;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,P.t9,r),this.importAggregator()}}var R,I=i(2210),k=i(1214),H=i(2177),L={};try{R=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(L.console=!0,-1!==R.indexOf("dev")&&(L.dev=!0),-1!==R.indexOf("nr_dev")&&(L.nrDev=!0))}catch(e){}function z(e){try{L.console&&z(e)}catch(e){}}L.nrDev&&H.ee.on("internal-error",(function(e){z(e.stack)})),L.dev&&H.ee.on("fn-err",(function(e,t,r){z(r.stack)})),L.dev&&(z("NR AGENT IN DEVELOPMENT MODE"),z("flags: "+(0,b.D)(L,(function(e,t){return e})).join(", ")));var M=i(6660);class B extends h{static featureName=M.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,M.t,i),this.skipNext=0;try{this.removeOnAbort=new AbortController}catch(e){}const o=this;o.ee.on("fn-start",(function(e,t,r){o.abortHandler&&(o.skipNext+=1)})),o.ee.on("fn-err",(function(t,r,n){o.abortHandler&&!n[M.A]&&((0,I.X)(n,M.A,(function(){return!0})),this.thrown=!0,(0,s.p)("err",[n,(0,p.z)()],void 0,e.D.jserrors,o.ee))})),o.ee.on("fn-end",(function(){o.abortHandler&&!this.thrown&&o.skipNext>0&&(o.skipNext-=1)})),o.ee.on("internal-error",(function(t){(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,o.ee)})),this.origOnerror=c._A.onerror,c._A.onerror=this.onerrorHandler.bind(this),c._A.addEventListener("unhandledrejection",(t=>{const r=function(e){let t="Unhandled Promise Rejection: ";if(e instanceof Error)try{return e.message=t+e.message,e}catch(t){return e}if(void 0===e)return new Error(t);try{return new Error(t+(0,D.P)(e))}catch(e){return new Error(t)}}(t.reason);(0,s.p)("err",[r,(0,p.z)(),!1,{unhandledPromiseRejection:1}],void 0,e.D.jserrors,this.ee)}),(0,O.m$)(!1,this.removeOnAbort?.signal)),(0,k.gy)(this.ee),(0,k.BV)(this.ee),(0,k.em)(this.ee),(0,t.OP)(r).xhrWrappable&&(0,k.Kf)(this.ee),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}onerrorHandler(t,r,n,i,o){"function"==typeof this.origOnerror&&this.origOnerror(...arguments);try{this.skipNext?this.skipNext-=1:(0,s.p)("err",[o||new F(t,r,n),(0,p.z)()],void 0,e.D.jserrors,this.ee)}catch(t){try{(0,s.p)("ierr",[t,(0,p.z)(),!0],void 0,e.D.jserrors,this.ee)}catch(e){}}return!1}}function F(e,t,r){this.message=e||"Uncaught error with no additional information",this.sourceURL=t,this.line=r}let U=1;const q="nr@id";function G(e){const t=typeof e;return!e||"object"!==t&&"function"!==t?-1:e===c._A?0:(0,I.X)(e,q,(function(){return U++}))}function V(e){if("string"==typeof e&&e.length)return e.length;if("object"==typeof e){if("undefined"!=typeof ArrayBuffer&&e instanceof ArrayBuffer&&e.byteLength)return e.byteLength;if("undefined"!=typeof Blob&&e instanceof Blob&&e.size)return e.size;if(!("undefined"!=typeof FormData&&e instanceof FormData))try{return(0,D.P)(e).length}catch(e){return}}}var X=i(7243);class W{constructor(e){this.agentIdentifier=e,this.generateTracePayload=this.generateTracePayload.bind(this),this.shouldGenerateTrace=this.shouldGenerateTrace.bind(this)}generateTracePayload(e){if(!this.shouldGenerateTrace(e))return null;var r=(0,t.DL)(this.agentIdentifier);if(!r)return null;var n=(r.accountID||"").toString()||null,i=(r.agentID||"").toString()||null,o=(r.trustKey||"").toString()||null;if(!n||!i)return null;var a=(0,_.M)(),s=(0,_.Ht)(),c=Date.now(),u={spanId:a,traceId:s,timestamp:c};return(e.sameOrigin||this.isAllowedOrigin(e)&&this.useTraceContextHeadersForCors())&&(u.traceContextParentHeader=this.generateTraceContextParentHeader(a,s),u.traceContextStateHeader=this.generateTraceContextStateHeader(a,c,n,i,o)),(e.sameOrigin&&!this.excludeNewrelicHeader()||!e.sameOrigin&&this.isAllowedOrigin(e)&&this.useNewrelicHeaderForCors())&&(u.newrelicHeader=this.generateTraceHeader(a,s,c,n,i,o)),u}generateTraceContextParentHeader(e,t){return"00-"+t+"-"+e+"-01"}generateTraceContextStateHeader(e,t,r,n,i){return i+"@nr=0-1-"+r+"-"+n+"-"+e+"----"+t}generateTraceHeader(e,t,r,n,i,o){if(!("function"==typeof c._A?.btoa))return null;var a={v:[0,1],d:{ty:"Browser",ac:n,ap:i,id:e,tr:t,ti:r}};return o&&n!==o&&(a.d.tk=o),btoa((0,D.P)(a))}shouldGenerateTrace(e){return this.isDtEnabled()&&this.isAllowedOrigin(e)}isAllowedOrigin(e){var r=!1,n={};if((0,t.Mt)(this.agentIdentifier,"distributed_tracing")&&(n=(0,t.P_)(this.agentIdentifier).distributed_tracing),e.sameOrigin)r=!0;else if(n.allowed_origins instanceof Array)for(var i=0;i<n.allowed_origins.length;i++){var o=(0,X.e)(n.allowed_origins[i]);if(e.hostname===o.hostname&&e.protocol===o.protocol&&e.port===o.port){r=!0;break}}return r}isDtEnabled(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.enabled}excludeNewrelicHeader(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.exclude_newrelic_header}useNewrelicHeaderForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!1!==e.cors_use_newrelic_header}useTraceContextHeadersForCors(){var e=(0,t.Mt)(this.agentIdentifier,"distributed_tracing");return!!e&&!!e.cors_use_tracecontext_headers}}var Z=i(7825),$=["load","error","abort","timeout"],Q=$.length,Y=t.Yu.REQ,K=c._A.XMLHttpRequest;class J extends h{static featureName=Z.t;constructor(r,n){let i=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(r,n,Z.t,i),(0,t.OP)(r).xhrWrappable&&(this.dt=new W(r),this.handler=(e,t,r,n)=>(0,s.p)(e,t,r,n,this.ee),(0,k.u5)(this.ee),(0,k.Kf)(this.ee),function(r,n,i,o){function a(e){var t=this;t.totalCbs=0,t.called=0,t.cbTime=0,t.end=E,t.ended=!1,t.xhrGuids={},t.lastSize=null,t.loadCaptureCalled=!1,t.params=this.params||{},t.metrics=this.metrics||{},e.addEventListener("load",(function(r){_(t,e)}),(0,O.m$)(!1)),c.IF||e.addEventListener("progress",(function(e){t.lastSize=e.loaded}),(0,O.m$)(!1))}function s(e){this.params={method:e[0]},T(this,e[1]),this.metrics={}}function u(e,n){var i=(0,t.DL)(r);i.xpid&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",i.xpid);var a=o.generateTracePayload(this.parsedOrigin);if(a){var s=!1;a.newrelicHeader&&(n.setRequestHeader("newrelic",a.newrelicHeader),s=!0),a.traceContextParentHeader&&(n.setRequestHeader("traceparent",a.traceContextParentHeader),a.traceContextStateHeader&&n.setRequestHeader("tracestate",a.traceContextStateHeader),s=!0),s&&(this.dt=a)}}function d(e,t){var r=this.metrics,i=e[0],o=this;if(r&&i){var a=V(i);a&&(r.txSize=a)}this.startTime=(0,p.z)(),this.listener=function(e){try{"abort"!==e.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==e.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof t.onload)&&"function"==typeof o.end)&&o.end(t)}catch(e){try{n.emit("internal-error",[e])}catch(e){}}};for(var s=0;s<Q;s++)t.addEventListener($[s],this.listener,(0,O.m$)(!1))}function f(e,t,r){this.cbTime+=e,t?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof r.onload||"function"!=typeof this.end||this.end(r)}function l(e,t){var r=""+G(e)+!!t;this.xhrGuids&&!this.xhrGuids[r]&&(this.xhrGuids[r]=!0,this.totalCbs+=1)}function h(e,t){var r=""+G(e)+!!t;this.xhrGuids&&this.xhrGuids[r]&&(delete this.xhrGuids[r],this.totalCbs-=1)}function g(){this.endTime=(0,p.z)()}function m(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-added",[e[1],e[2]],t)}function v(e,t){t instanceof K&&"load"===e[0]&&n.emit("xhr-load-removed",[e[1],e[2]],t)}function b(e,t,r){t instanceof K&&("onload"===r&&(this.onload=!0),("load"===(e[0]&&e[0].type)||this.onload)&&(this.xhrCbStart=(0,p.z)()))}function y(e,t){this.xhrCbStart&&n.emit("xhr-cb-time",[(0,p.z)()-this.xhrCbStart,this.onload,t],t)}function w(e){var t,r=e[1]||{};"string"==typeof e[0]?t=e[0]:e[0]&&e[0].url?t=e[0].url:c._A?.URL&&e[0]&&e[0]instanceof URL&&(t=e[0].href),t&&(this.parsedOrigin=(0,X.e)(t),this.sameOrigin=this.parsedOrigin.sameOrigin);var n=o.generateTracePayload(this.parsedOrigin);if(n&&(n.newrelicHeader||n.traceContextParentHeader))if("string"==typeof e[0]||c._A?.URL&&e[0]&&e[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),s(i.headers,n)&&(this.dt=n),e.length>1?e[1]=i:e.push(i)}else e[0]&&e[0].headers&&s(e[0].headers,n)&&(this.dt=n);function s(e,t){var r=!1;return t.newrelicHeader&&(e.set("newrelic",t.newrelicHeader),r=!0),t.traceContextParentHeader&&(e.set("traceparent",t.traceContextParentHeader),t.traceContextStateHeader&&e.set("tracestate",t.traceContextStateHeader),r=!0),r}}function x(e,t){this.params={},this.metrics={},this.startTime=(0,p.z)(),this.dt=t,e.length>=1&&(this.target=e[0]),e.length>=2&&(this.opts=e[1]);var r,n=this.opts||{},i=this.target;"string"==typeof i?r=i:"object"==typeof i&&i instanceof Y?r=i.url:c._A?.URL&&"object"==typeof i&&i instanceof URL&&(r=i.href),T(this,r);var o=(""+(i&&i instanceof Y&&i.method||n.method||"GET")).toUpperCase();this.params.method=o,this.txSize=V(n.body)||0}function A(t,r){var n;this.endTime=(0,p.z)(),this.params||(this.params={}),this.params.status=r?r.status:0,"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var o={txSize:this.txSize,rxSize:n,duration:(0,p.z)()-this.startTime};i("xhr",[this.params,o,this.startTime,this.endTime,"fetch"],this,e.D.ajax)}function E(t){var r=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var o=0;o<Q;o++)t.removeEventListener($[o],this.listener,!1);r.aborted||(n.duration=(0,p.z)()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==r.status&&(r.status=0):_(this,t),n.cbTime=this.cbTime,i("xhr",[r,n,this.startTime,this.endTime,"xhr"],this,e.D.ajax))}}function T(e,t){var r=(0,X.e)(t),n=e.params;n.hostname=r.hostname,n.port=r.port,n.protocol=r.protocol,n.host=r.hostname+":"+r.port,n.pathname=r.pathname,e.parsedOrigin=r,e.sameOrigin=r.sameOrigin}function _(e,t){e.params.status=t.status;var r=function(e,t){var r=e.responseType;return"json"===r&&null!==t?t:"arraybuffer"===r||"blob"===r||"json"===r?V(e.response):"text"===r||""===r||void 0===r?V(e.responseText):void 0}(t,e.lastSize);if(r&&(e.metrics.rxSize=r),e.sameOrigin){var n=t.getResponseHeader("X-NewRelic-App-Data");n&&(e.params.cat=n.split(", ").pop())}e.loadCaptureCalled=!0}n.on("new-xhr",a),n.on("open-xhr-start",s),n.on("open-xhr-end",u),n.on("send-xhr-start",d),n.on("xhr-cb-time",f),n.on("xhr-load-added",l),n.on("xhr-load-removed",h),n.on("xhr-resolved",g),n.on("addEventListener-end",m),n.on("removeEventListener-end",v),n.on("fn-end",y),n.on("fetch-before-start",w),n.on("fetch-start",x),n.on("fn-start",b),n.on("fetch-done",A)}(r,this.ee,this.handler,this.dt),this.importAggregator())}}var ee=i(3614);const{BST_RESOURCE:te,RESOURCE:re,START:ne,END:ie,FEATURE_NAME:oe,FN_END:ae,FN_START:se,PUSH_STATE:ce}=ee;var ue=i(7836);const{FEATURE_NAME:de,START:fe,END:le,BODY:he,CB_END:ge,JS_TIME:pe,FETCH:me,FN_START:ve,CB_START:be,FN_END:ye}=ue;var we=i(4649);class xe extends h{static featureName=we.t;constructor(e,t){let r=!(arguments.length>2&&void 0!==arguments[2])||arguments[2];super(e,t,we.t,r),this.importAggregator()}}new class{constructor(e){let t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:(0,_.ky)(16);c._A?(this.agentIdentifier=t,this.sharedAggregator=new y({agentIdentifier:this.agentIdentifier}),this.features={},this.desiredFeatures=new Set(e.features||[]),this.desiredFeatures.add(m),Object.assign(this,(0,a.j)(this.agentIdentifier,e,e.loaderType||"agent")),this.start()):(0,l.Z)("Failed to initial the agent. Could not determine the runtime environment.")}get config(){return{info:(0,t.C5)(this.agentIdentifier),init:(0,t.P_)(this.agentIdentifier),loader_config:(0,t.DL)(this.agentIdentifier),runtime:(0,t.OP)(this.agentIdentifier)}}start(){const t="features";try{const r=n(this.agentIdentifier),i=[...this.desiredFeatures];i.sort(((t,r)=>e.p[t.featureName]-e.p[r.featureName])),i.forEach((t=>{if(r[t.featureName]||t.featureName===e.D.pageViewEvent){const n=function(t){switch(t){case e.D.ajax:return[e.D.jserrors];case e.D.sessionTrace:return[e.D.ajax,e.D.pageViewEvent];case e.D.sessionReplay:return[e.D.sessionTrace];case e.D.pageViewTiming:return[e.D.pageViewEvent];default:return[]}}(t.featureName);n.every((e=>r[e]))||(0,l.Z)("".concat(t.featureName," is enabled but one or more dependent features has been disabled (").concat((0,D.P)(n),"). This may cause unintended consequences or missing data...")),this.features[t.featureName]=new t(this.agentIdentifier,this.sharedAggregator)}})),(0,T.Qy)(this.agentIdentifier,this.features,t)}catch(e){(0,l.Z)("Failed to initialize all enabled instrument classes (agent aborted) -",e);for(const e in this.features)this.features[e].abortHandler?.();const r=(0,T.fP)();return delete r.initializedAgents[this.agentIdentifier]?.api,delete r.initializedAgents[this.agentIdentifier]?.[t],delete this.sharedAggregator,r.ee?.abort(),delete r.ee?.get(this.agentIdentifier),!1}}}({features:[J,m,S,class extends h{static featureName=oe;constructor(t,r){if(super(t,r,oe,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;const n=this.ee;let i;(0,k.QU)(n),this.eventsEE=(0,k.em)(n),this.eventsEE.on(se,(function(e,t){this.bstStart=(0,p.z)()})),this.eventsEE.on(ae,(function(t,r){(0,s.p)("bst",[t[0],r,this.bstStart,(0,p.z)()],void 0,e.D.sessionTrace,n)})),n.on(ce+ne,(function(e){this.time=(0,p.z)(),this.startPath=location.pathname+location.hash})),n.on(ce+ie,(function(t){(0,s.p)("bstHist",[location.pathname+location.hash,this.startPath,this.time],void 0,e.D.sessionTrace,n)}));try{i=new PerformanceObserver((t=>{const r=t.getEntries();(0,s.p)(te,[r],void 0,e.D.sessionTrace,n)})),i.observe({type:re,buffered:!0})}catch(e){}this.importAggregator({resourceObserver:i})}},C,xe,B,class extends h{static featureName=de;constructor(e,r){if(super(e,r,de,!(arguments.length>2&&void 0!==arguments[2])||arguments[2]),!c.il)return;if(!(0,t.OP)(e).xhrWrappable)return;try{this.removeOnAbort=new AbortController}catch(e){}let n,i=0;const o=this.ee.get("tracer"),a=(0,k._L)(this.ee),s=(0,k.Lg)(this.ee),u=(0,k.BV)(this.ee),d=(0,k.Kf)(this.ee),f=this.ee.get("events"),l=(0,k.u5)(this.ee),h=(0,k.QU)(this.ee),g=(0,k.Gm)(this.ee);function m(e,t){h.emit("newURL",[""+window.location,t])}function v(){i++,n=window.location.hash,this[ve]=(0,p.z)()}function b(){i--,window.location.hash!==n&&m(0,!0);var e=(0,p.z)();this[pe]=~~this[pe]+e-this[ve],this[ye]=e}function y(e,t){e.on(t,(function(){this[t]=(0,p.z)()}))}this.ee.on(ve,v),s.on(be,v),a.on(be,v),this.ee.on(ye,b),s.on(ge,b),a.on(ge,b),this.ee.buffer([ve,ye,"xhr-resolved"],this.featureName),f.buffer([ve],this.featureName),u.buffer(["setTimeout"+le,"clearTimeout"+fe,ve],this.featureName),d.buffer([ve,"new-xhr","send-xhr"+fe],this.featureName),l.buffer([me+fe,me+"-done",me+he+fe,me+he+le],this.featureName),h.buffer(["newURL"],this.featureName),g.buffer([ve],this.featureName),s.buffer(["propagate",be,ge,"executor-err","resolve"+fe],this.featureName),o.buffer([ve,"no-"+ve],this.featureName),a.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"],this.featureName),y(l,me+fe),y(l,me+"-done"),y(a,"new-jsonp"),y(a,"jsonp-end"),y(a,"cb-start"),h.on("pushState-end",m),h.on("replaceState-end",m),window.addEventListener("hashchange",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("load",m,(0,O.m$)(!0,this.removeOnAbort?.signal)),window.addEventListener("popstate",(function(){m(0,i>1)}),(0,O.m$)(!0,this.removeOnAbort?.signal)),this.abortHandler=this.#e,this.importAggregator()}#e(){this.removeOnAbort?.abort(),this.abortHandler=void 0}}],loaderType:"spa"})})(),window.NRBA=o})();</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="PrateekMalhotra"/><meta name="DC.Creator" content="KanupriyaKhandelwal"/><meta name="DC.Creator" content="SubhayanBiswas"/><meta name="DC.Creator" content="Fang-ChungChen"/><meta name="DC.Creator" content="Ganesh D.Sharma"/><meta name="DC.title" content="Opportunities and challenges for machine learning to select combination of donor and acceptor materials for efficient organic solar cells  "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2022/12/08"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D2TC03276G"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Opportunities and challenges for machine learning to select combination of donor and acceptor materials for efficient organic solar cells  "/><meta name="citation_author" content="PrateekMalhotra"/><meta name="citation_author" content="KanupriyaKhandelwal"/><meta name="citation_author" content="SubhayanBiswas"/><meta name="citation_author" content="Fang-ChungChen"/><meta name="citation_author" content="Ganesh D.Sharma"/><meta name="citation_online_date" content="2022/10/19"/><meta name="citation_date" content="2022"/><meta name="citation_journal_title" content="Journal of Materials Chemistry C"/><meta name="citation_volume" content="10"/><meta name="citation_issue" content="47"/><meta name="citation_firstpage" content="17781"/><meta name="citation_lastpage" content="17811"/><meta name="citation_doi" content="10.1039/D2TC03276G"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2022/tc/d2tc03276g"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2022/tc/d2tc03276g"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2022/tc/d2tc03276g"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"></script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"></script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2022/tc/d2tc03276g">ViewPDFVersion</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d2tc90242g)" href="/en/content/articlehtml/2022/tc/d2tc90242g" target="_BLANK">PreviousArticle</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d2tc03227a)" href="/en/content/articlehtml/2022/tc/d2tc03227a" target="_BLANK">NextArticle</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script></div></div></div><div class="article_info"> DOI:<a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D2TC03276G">10.1039/D2TC03276G</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2050-7534/2013">J. Mater. Chem. C</a></span>, 2022, <strong>10</strong>, 17781-17811</div><h1 id="sect221"><span class="title_heading">Opportunities and challenges for machine learning to select combination of donor and acceptor materials for efficient organic solar cells<a title="Select to navigate to footnote" href="#fn1"></a></span></h1><p class="header_text">
      <span class="bold">
        
          
            Prateek 
            Malhotra
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for PrateekMalhotra (orcid.org/0000-0003-0816-3712) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-0816-3712"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Kanupriya 
            Khandelwal
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for KanupriyaKhandelwal (orcid.org/0000-0003-4550-2867) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0003-4550-2867"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Subhayan 
            Biswas
          
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Fang-Chung 
            Chen
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Fang-ChungChen (orcid.org/0000-0002-4131-3893) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-4131-3893"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">bc</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Ganesh D. 
            Sharma
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Ganesh D.Sharma (orcid.org/0000-0002-1717-0116) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-1717-0116"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">ad</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Department of Physics, The LNM Institute of Information Technology, Jamdoli, Jaipur, 302031, Rajasthan, India. E-mail: <a href="mailto:gdsharma273@gmail.com">gdsharma273@gmail.com</a>; <a href="mailto:gdsharma@lnmiit.ac.in">gdsharma@lnmiit.ac.in</a></span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Department of Photonics, National Yang Ming Chiao Tung University, Hsinchu, 30010, Taiwan</span>
      <br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">Center for Emergent Functional Matter Science, National Yang Ming Chiao Tung University, Hsinchu, 30010, Taiwan</span>
      <br/><a id="affd"><sup><span class="sup_ref italic">d</span></sup></a><span class="italic">Department of Electronics and Communication Engineering, The LNM Institute of Information Technology, Jamdoli, Jaipur, 302031, Rajasthan, India</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      5th August 2022
    </span><span class="bold italic">, Accepted 19th October 2022</span></div><p class="bold italic">First published on 19th October 2022</p><hr/><div class="abstract"><h2>Abstract</h2><p>Organic solar cells (OSCs) have witnessed significant improvement in power conversion efficiency (PCE) in the last decade. The structural flexibility of organic semiconductors provides vast search space for potential candidates of OSCs, but discovering new materials from search space with traditional approaches such as DFT is computationally expensive and time-consuming. Machine learning (ML) is extensively used in OSCs to accelerate productivity and materials discovery. ML is gaining more attention due to the availability of large datasets, improved algorithms, and exponentially growing computational power. In this review, current progress, opportunity, and challenges for ML in OSCs have been identified. Given the rapid advances in this field, impactful techniques that have been useful in extracting meaningful insights are discussed. Finally, we elaborate upon the bottlenecks of the ML workflow with respect to data size, model interpretability, and extrapolation.</p></div><hr/><table><tr><td class="biogPlate"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-p1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-p1.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-p1.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Prateek Malhotra</span>
                </span></p></td><td><i><p>Prateek Malhotra is a PhD scholar in the Department of Physics at The LNM Institute of Information Technology, India, under the supervision of Prof. Ganesh D. Sharma and Dr Subhayan Biswas. His research focuses on developing high-efficiency organic solar cells and machine learning applications for the prediction of photovoltaic parameters of organic solar cells.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-p2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-p2.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-p2.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Kanupriya Khandelwal</span>
                </span></p></td><td><i><p>Kanupriya Khandelwal is a PhD Scholar in the Department of Physics at The LNM Institute of Information Technology, Jaipur, India, under the supervision of Prof. G. D. Sharma and Dr Subhayan Biswas. Her main research interests are in the optoelectronic properties of organic materials for application in photovoltaic devices. She is currently working on semitransparent organic solar cells and their BIPV (Building Integrated Photovoltaics) application.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-p3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-p3.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-p3.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Subhayan Biswas</span>
                </span></p></td><td><i><p>Dr Subhayan Biswas is an associate professor in the Department of Physics at The LNM Institute of Information Technology, Jaipur, India. After completing his PhD from the Indian Association for the Cultivation of Science, Kolkata, India, he did post-doctoral research in Inha University (South Korea), National Sun Yat-Sen University &amp; National Taiwan University (Taiwan) and Toyama University (Japan). His research field is nanostructured solar cells including dye-sensitized solar cells, quantum-dot solar cells and organic solar cells.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-p4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-p4.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-p4.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Fang-Chung Chen</span>
                </span></p></td><td><i><p>Prof. Fang-Chung Chen is currently a Professor and Chairman of Department of Photonics (DoP), National Yang Ming Chiao Tung University (NYCU). He joined DoP, National Chiao Tung University (NCTU, now NYCU) in February 2004. He was also the chairman of Degree Program of Flat Panel Display Technology, NCTU. He has published more than 130 journal papers, 100 conference papers, and 5 book chapters, and owns 14 patents. Prof. Chen is a Fellow of the Royal Society of Chemistry (FRSC). He is also the recipient of the 2021 IoT Innovation Award (Pen Wen Yuan Foundation), 2020 Y. Z. Hsu Scientific Paper Award and Award for Junior Research Investigators of Academia Sinica 2008, which is one of the most important awards for junior research investigators in all research fields in Taiwan. His h-index is 48 (Google Scholar). His research interests include flexible solar cells, organic electronics and materials, perovskite electronics, plasmonic materials and low-dimensional nanomaterials.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-p5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-p5.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-p5.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Ganesh D. Sharma</span>
                </span></p></td><td><i><p>Prof. Ganesh D. Sharma has been working as a Senior Professor of Physics, Electronics Communication and Engineering and Dean (Research and development) at The LNM Institute of Information Technology, Jaipur (Raj.), India, since December 2014. Prof. Sharma obtained his PhD degree from Indian Institute of Technology in 1985 and after that he joined JNV University, Jodhpur (Raj), India, as Assistant Professor and subsequently as Professor. His area of research is organic solar cells, nanocrystalline organicinorganic hybrid solar cells and fuel cells, and organic nanomaterials for energy conversion. He has published more than 330 research papers. He had research collaboration with different international and national research organizations.</p></i></td></tr></table><hr/>
    
      
      <h2 id="sect329"><span class="a_heading">1. Introduction</span></h2>
      <span>Organic solar cells (OSCs) have come a long way in their quest to become a viable alternative to more expensive conventional solar cell technology.<a title="Select to navigate to reference" href="#cit1"><sup><span class="sup_ref">1,2</span></sup></a> Being flexible, lightweight, eco-friendly, and semi-transparent, OSCs hold promising potential for various applications.<a title="Select to navigate to reference" href="#cit3"><sup><span class="sup_ref">38</span></sup></a> With the advent of non-fullerene small molecule acceptors (NFSMAs),<a title="Select to navigate to reference" href="#cit9"><sup><span class="sup_ref">913</span></sup></a> especially Y-series NFSMAs,<a title="Select to navigate to reference" href="#cit14"><sup><span class="sup_ref">1416</span></sup></a> bulk heterojunction (BHJ) based OSCs have already reached power conversion efficiencies (PCEs) in the range of 1819% for single junction binary and ternary systems<a title="Select to navigate to reference" href="#cit17"><sup><span class="sup_ref">1723</span></sup></a> and are exceeding 20% for tandem configuration.<a title="Select to navigate to references" href="#cit24"><sup><span class="sup_ref">24</span></sup></a> Sharp absorption onset, bandgap tunability, high absorption, and low energy losses are some of the benefits of NFSMAs over fullerene derivatives that have propelled them to the forefront of technological advancement and potential commercialization of OSCs at low cost.<a title="Select to navigate to reference" href="#cit25"><sup><span class="sup_ref">2528</span></sup></a></span>
      <p class="otherpara">The most efficient OSCs are based on the concept of BHJ, in which the active layer is sandwiched between the anode and the cathode. The hole transport layer (HTL) is inserted between the BHJ layer and the anode, and the electron transport layer (ETL) is inserted between the BHJ layer and the cathode to enhance the charge collection. The BHJ active layer is a mix of donor (p-type organic semiconductor) and acceptor (n-type organic semiconductor), which provides an appropriate phase separation for efficient exciton dissociation into free charge carriers and interpenetrating network pathways for charge transport towards respective electrodes. In general, the energy conversion in OSCs based on the bulk heterojunction active layer is accomplished by the following main consecutive steps: (i) absorption of photons by the BHJ active layer and exciton generation, (ii) exciton diffusion to D/A interfaces, (iii) exciton dissociation, (iv) transport of electrons and holes towards the cathode and anode, and (v) charge extraction. The step (vi) charge recombination, would lead to a decrease in device efficiency; these processes are presented in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>. Considering all the processes, the PCE of the OSC depends on many properties of the organic semiconductors such as the optical absorption profile and molar extinction coefficient, charge carrier mobility, frontier energy levels of donor and acceptor, and electronhole (exciton) binding energy (<span class="italic">E</span><small><sub>bind</sub></small>) and it is very important to construct more precise models using all appropriate and easily available descriptors.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f1.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Photocurrent generation in BHJ-OSCs. (1) Exciton generation, (2) exciton diffusion, (3) exciton dissociation, (4) charge transportation, (5) charge extraction, and (6) charge recombination.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Although OSCs have attained PCEs in the range of 1920%, one faces new challenges in exploring numerous possible material combinations together with weight ratios between donor and acceptor and processing conditions. Once a market-competitive PCE is realized in the laboratory, the next step must be transferring this technology for manufacturing, which is challenging and time-consuming.<a title="Select to navigate to reference" href="#cit29"><sup><span class="sup_ref">29,30</span></sup></a> This demands a new research method that can rapidly explore enormous parameter space, ideally <span class="italic">via</span> industry-relevant methods. Many experimental methods, including screen printing, doctor blade, and slot-die, have been explored to cover the enormous composition space.<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">3134</span></sup></a></p>
      <p class="otherpara">Most efficient BHJ-OSCs employ the ADA structured NFSMAs as the acceptor and DA copolymers as the donor. The structural diversity of these materials is of significant interest for materials scientists; however, it is not possible to completely survey the huge molecular space. Quantum chemical calculations and molecular dynamics simulations can provide an approximate evaluation of the optical and electrochemical properties of new materials;<a title="Select to navigate to reference" href="#cit35"><sup><span class="sup_ref">3537</span></sup></a> however, the complex relationship between the structure and these properties of materials used for OSCs has hindered the efficient assessment of materials.</p>
      <p class="otherpara">Machine learning (ML) is a subfield of rapidly developing artificial intelligence (AI) technology that seeks to create programs capable of learning from large data sets by employing various algorithms and statistical techniques. Such programs can then be used to do things like explore hidden patterns in data, build predictive models, and create guidelines for future research. In recent years, many publications on the application of ML techniques in materials science have been published, as these techniques have been widely used in material research to determine the properties and functions of existing materials or to discover new materials with more desirable functions. ML is getting a lot of attention these days because of its potential to boost output and aid in discovering new materials.<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> ML has been used extensively in fields including property prediction and material discovery in the field of OSCs.<a title="Select to navigate to reference" href="#cit39"><sup><span class="sup_ref">3943</span></sup></a> The number of publications detailing the use of machine learning for the analysis or screening of data obtained experimentally or computationally has also expanded dramatically over the past couple of years. Since non-fullerene small molecule acceptors were discovered recently, earlier ML studies were performed entirely on polymer donors, and the acceptor was fullerene. The majority of datasets now being used for the study include polymer donors and small molecule non-fullerene acceptors. With an increase in data size, model performance is also improved.</p>
      <p class="otherpara">Many high-performance active materials for OSCs are yet to be discovered, and material synthesis followed by device fabrication is expensive and time-consuming. We want to understand the functions that link material properties with structure. At the device fabrication stage, there are a number of tunable parameters such as the D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A ratio,<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> alignment between frontier molecular orbitals (FMOs),<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> miscibility,<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> film processing (spin-casting speed),<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> calcination temperature and duration,<a title="Select to navigate to references" href="#cit48"><sup><span class="sup_ref">48</span></sup></a> active layer thickness<a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a>, and processing additives.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> Due to these tunable parameters, the study becomes more complex. To select the most optimal parameter combination, the one-variable-at-a-time method (Edisonian approach) is most often used.<a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a> In this trial-and-error approach, a lot of time and expensive materials are consumed without a guarantee of reaching the most optimal set of parameters. Such processes require a long time because they have to investigate the vast chemical space of materials. To counteract this, machine learning (ML) models have allowed researchers to investigate chemical space and its properties more effectively, saving time and money. The availability of large training datasets, superior algorithms, and ever-increasing processing power has excited the interest of materials scientists in ML. Photovoltaic parameters such as power conversion efficiency (PCE), short circuit current density (<span class="italic">J</span><small><sub>SC</sub></small>), open circuit voltage (<span class="italic">V</span><small><sub>OC</sub></small>), fill factor (FF), non-radiative voltage loss, and frontier molecular orbitals (FMO) have been used as target variables in many ML related studies for OSCs.</p>
      <p class="otherpara">ML models are implemented for photovoltaic property prediction in OSCs using inputs such as molecular properties (MP), molecular descriptors (MD), fingerprints (FP), FMO, and molecular images. For the studies on OSCs, calculated descriptors/fingerprints of donor and acceptor are concatenated, and the total number of input descriptors for the ML model becomes very high and demands suitable feature engineering. Even after performing feature engineering, the remaining number of input descriptors are still high and cause degraded results by ML model. This is termed as curse of dimensionality. Here comes the use of dimensionality reduction techniques such as principal component analysis (PCA), which is commonly used in the community. ML has granted the OSC research community entirely new abilities from predicting photovoltaic properties,<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> quantitative structureproperty relationship (QSPR),<a title="Select to navigate to reference" href="#cit51"><sup><span class="sup_ref">51,52</span></sup></a> design of experiments (DOE),<a title="Select to navigate to reference" href="#cit47"><sup><span class="sup_ref">47,53</span></sup></a> novel polymer/NFA discovery,<a title="Select to navigate to reference" href="#cit54"><sup><span class="sup_ref">54,55</span></sup></a> and robotization of labs.<a title="Select to navigate to reference" href="#cit48"><sup><span class="sup_ref">48,56</span></sup></a> Most of the studies in the field are regression problems, and only a few are done for classification.</p>
      <p class="otherpara">This review article is written to offer a synopsis of ML applications in OSCs. Section 2 describes the ML workflow, from data gathering to novel materials discovery. In addition, available datasets and their categories are also discussed. In Section 3, research papers on ML implementation in OSCs are analyzed and reviewed. The review is grouped on the basis of the acceptor type: (1) fullerene acceptor (FA), (2) non-fullerene acceptor (NFA), and (3) mix of FA and NFA. In Section 4, problems and future scope are discussed.</p>
    
    
      
      <h2 id="sect375"><span class="a_heading">2. Machine learning workflow</span></h2>
      <span>To begin any ML project, professionals in the relevant field are consulted to help define the aims and targets of the models. This is the most critical phase since the target must be learnable from accessible information such as microscopic properties, molecular descriptors, molecular fingerprints, and DFT calculated descriptors. When the prediction objective is chosen incorrectly, it might lead to models with spuriously significant errors or models that aren't generalizable. After setting the research objectives, ML is implemented by systematically implementing the process summarized as (1) data gathering, (2) data preparation, (3) model training, (4) model evaluation, and (5) discovering and testing novel materials. <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a> represents a high-level overview of the steps involved in the ML workflow.</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f2.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Workflow of ML in OSCs.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
        
        <h3 id="sect382"><span class="b_heading">2.1. Data gathering</span></h3>
        <span>For ML application in OSCs, input data are required to train the ML model, whether the task is prediction of photovoltaic properties or clustering of molecules into different categories. Early research in OSCs was dominated by fullerene acceptor-based solar cells because of high electron mobility due to their isotropic nature; thus, plenty of data could be easily collected from the literature for ML applications.</span>
        <p class="otherpara">Prof. Alan Aspuru-Guzik and his team run the clean energy project. They created the Harvard clean energy project (HCEP) dataset consisting of 2.3 million compounds derived from 150 million DFT calculations to find high-efficiency OSC materials.<a title="Select to navigate to reference" href="#cit57"><sup><span class="sup_ref">5759</span></sup></a> All these molecules are generated using 26 molecular building blocks. The properties include FMO, photovoltaic performance parameters (<span class="italic">J</span><small><sub>sc</sub></small>, <span class="italic">V</span><small><sub>oc</sub></small>, PCE), and stoichiometric formulas, and various studies have been performed using this dataset.<a title="Select to navigate to reference" href="#cit59"><sup><span class="sup_ref">59,60</span></sup></a> For high-throughput virtual screening, the HCEP dataset has been used by several research groups. Compounds in the HCEP dataset are considerably different and structurally simple from those found in the real world. Therefore, potential candidates for high-performance OSCs are less likely to be discovered from the HCEP dataset.</p>
        <p class="otherpara">In 2006, the Scharber model<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> was developed to predict the PCE of fullerene based OSCs and the model has been used in various high-throughput screenings.<a title="Select to navigate to reference" href="#cit61"><sup><span class="sup_ref">6163</span></sup></a> Since this model considers only a few electronic parameters and no other structural descriptors are considered, the performance of this model is relatively poor. Brabec <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> proposed a modification to the Scharber model by taking the absorption of non-fullerene acceptors (NFAs) into account. In 2018, Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> created a FA based dataset of 280 OSCs with small molecule donors, and the same dataset was used by Goharimanesh <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> in 2022 for interpretable ML models. Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> in 2018 created a FA based dataset of 1200 OSCs with polymer donors for PCE prediction, and the same approach was used by Wei <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit67"><sup><span class="sup_ref">67</span></sup></a> for 500 NFA based OSCs. In 2019, Chen<a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a> used the 1200 dataset by Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> for virtual screening of conjugated polymers. In 2019, Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> and Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> created FA datasets of 249 and 300 OSCs with small molecule donors. Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> used their earlier dataset (# = 300)<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> to unravel correlations between device performance parameters and molecular properties. In 2019, Lee<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> also developed a dataset of 124 fullerene derivative-based ternary OSCs for predicting PCE, and further in 2020, Lee<a title="Select to navigate to references" href="#cit73"><sup><span class="sup_ref">73</span></sup></a> used the same dataset for <span class="italic">V</span><small><sub>oc</sub></small> prediction of ternary OSCs.</p>
        <p class="otherpara">In 2015, Zhan <span class="italic">et al.</span> introduced efficient NFSMAs by synthesis of ITIC.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> In 2017, Zhao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> presented the first NFA-based BHJ organic solar cell that outperformed the state-of-the-art solar cells with fullerenes as acceptors. Now NFA-based OSCs have crossed efficiency over 19%.<a title="Select to navigate to reference" href="#cit17"><sup><span class="sup_ref">17,18</span></sup></a> NFAs are used much more in recent studies due to their tunable absorption spectra, easily adjustable FMO, solubility, photostability, and thermal stability.<a title="Select to navigate to reference" href="#cit12"><sup><span class="sup_ref">12,7678</span></sup></a> With the generation of new datasets for NFA-based OSCs, interest in NFA-based datasets is gathering momentum across the globe in ML applications.</p>
        <p class="otherpara">Aspuru Guzik <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> in 2017 created a dataset of 51<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 NFAs with PCDTBT as the donor material. In 2020, Min <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> created a dataset of 565 polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA based OSCs for the discovery of new D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations. In 2020, Lee<a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a> created a dataset of 135 NFA-based OSCs for PCE prediction, and later in 2022, Lee<a title="Select to navigate to references" href="#cit81"><sup><span class="sup_ref">81</span></sup></a> used the same dataset (# = 135) to identify the correlation between FMO and open circuit voltage of OSCs. In 2021, Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> created a dataset of 566 polymer donor<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA based OSCs and later in the same year increased their dataset to 1318<a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> for comparison. Later in 2022, Saeki <span class="italic">et al.</span> used their earlier dataset (# = 1318) for machine learning-assisted polymer design<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> and used failure data to improve PCE prediction.<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> Liu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a> created a dataset of 157 non-fullerene ternary OSCs to predict PCE by using FMO of donor, acceptor, and the third component as input descriptors. Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> created datasets of 351 polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA OSCs for optimization of donoracceptor combination. Sharma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> manually collected a dataset of 154 polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA OSCs for prediction of non-radiative voltage losses. To predict the PCE of P3HT donor-based OSCs, Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> created a dataset of 283 NFAs from the literature with single donor P3HT. Similarly, Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a> conducted a study on 265 NFAs with single donor PTB7-Th. Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a> collected a dataset of 164 NFSMAs from the literature with PBDB-T as the donor. In 2022, Lu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit91"><sup><span class="sup_ref">91</span></sup></a> created a dataset of 717 NFA based organic solar cells for PCE prediction.</p>
        <p class="otherpara">Aspuru-Guzik <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a> published a collection of 350 organic small molecules and polymers from the literature that were used as p-type materials in OSCs. This model has been widely used to train QSPR models.<a title="Select to navigate to reference" href="#cit54"><sup><span class="sup_ref">54,9395</span></sup></a> To build structureproperty relationship, Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> in 2019 developed a dataset of 1719 OSCs with a mix of FA and NFA. For multicomponent materials optimization in BHJ OSCs, Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> in 2019 gathered a dataset of 320 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations for PCE prediction. In 2019, Kettle <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> created a dataset of 1900 OSCs with corresponding device structure, performance, and stability. Kettle <span class="italic">et al.</span> later used their dataset to study enhancing stability<a title="Select to navigate to references" href="#cit99"><sup><span class="sup_ref">99</span></sup></a> and understanding the trade-offs between device performance, stability, and environmental impact.<a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a> To study the effect of increasing descriptor set size, Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> in 2020 created a dataset of OSCs with a mix of FA and NFA. Later in 2022, Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> used the same dataset (# = 566) for PCE prediction of completely new molecule families (extrapolation) by exploring different cross validation techniques. In 2020, Lee<a title="Select to navigate to references" href="#cit102"><sup><span class="sup_ref">102</span></sup></a> gathered the first dataset for 70 tandem OSCs (both conventional and inverted) for PCE prediction. In 2021, Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> gathered a dataset of 1758 donor materials (mix of polymers and small molecules) and tested their novel fingerprinting technique for expressing 6180 different fragments (bits). In 2022, Hutchinson <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit104"><sup><span class="sup_ref">104</span></sup></a> created a dataset of 84 OSCs and used simplified time-dependent density functional theory (sTD-DFT) for speeding up the calculations by 23 times. A summary of datasets used for ML studies in OSCs is given in <a title="Select to navigate to table" href="#tab1">Table 1</a>.</p>
        
<div class="table_caption"><b>Table 1</b> <span id="tab1">Summary of datasets used for ML studies in OSCs</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/><col/></colgroup>
              
              
              
              
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Source</th>
                  <th align="left" valign="bottom" class="border_black">Donor</th>
                  <th align="left" valign="bottom" class="border_black">Acceptor</th>
                  <th align="left" valign="bottom" class="border_black">Data size</th>
                  <th align="left" valign="bottom" class="border_black">Descriptors</th>
                  <th align="left" valign="bottom" class="border_black">Method</th>
                  <th align="left" valign="bottom" class="border_black">Year published</th>
                  <th align="left" valign="bottom" class="border_black">Ref.</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM</td>
                  <td valign="top">FA</td>
                  <td valign="top">280</td>
                  <td valign="top">MP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2018</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit50">50 and 65</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">FA</td>
                  <td valign="top">1200</td>
                  <td valign="top">MP, FP</td>
                  <td valign="top">Regression, classification</td>
                  <td valign="top">2018</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit66">66 and 68</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM</td>
                  <td valign="top">FA</td>
                  <td valign="top">249</td>
                  <td valign="top">MP, FP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2019</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit69">69</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM</td>
                  <td valign="top">FA</td>
                  <td valign="top">300</td>
                  <td valign="top">MP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2019</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit70">70 and 71</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">FA</td>
                  <td valign="top">124</td>
                  <td valign="top">FMO</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2019</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit72">72 and 73</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">HCEP</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">51<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000</td>
                  <td valign="top">Scharber model</td>
                  <td valign="top">Regression, classification</td>
                  <td valign="top">2017</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit55">55, 61, 105 and 106</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">500</td>
                  <td valign="top">FP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2019</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit67">67</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">565</td>
                  <td valign="top">FP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2020</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit79">79</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">135</td>
                  <td valign="top">MP, FMO</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2020</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit80">80, 81</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">566</td>
                  <td valign="top">MP, FP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit82">82</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">1318</td>
                  <td valign="top">MP, FP, MD</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit83">8385</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">157</td>
                  <td valign="top">FMO</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit86">86</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">154</td>
                  <td valign="top">MP, FP, MD</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit87">87</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">P3HT</td>
                  <td valign="top">NFA</td>
                  <td valign="top">283</td>
                  <td valign="top">MD</td>
                  <td valign="top">Regression, classification</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit88">88</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">PTB7-Th</td>
                  <td valign="top">NFA</td>
                  <td valign="top">265</td>
                  <td valign="top">MD, DFT</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2022</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit89">89</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">PBDBT</td>
                  <td valign="top">NFA</td>
                  <td valign="top">164</td>
                  <td valign="top">MD</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit90">90</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">351</td>
                  <td valign="top">MP, FP, DFT</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit51">51</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">NFA</td>
                  <td valign="top">717</td>
                  <td valign="top">MD</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2022</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit91">91</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">HOPV</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">FA, NFA</td>
                  <td valign="top">350</td>
                  <td valign="top">MP, DFT</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2016</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit54">54 and 9295</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">FA, NFA</td>
                  <td valign="top">1719</td>
                  <td valign="top">FP, MD, Image</td>
                  <td valign="top">Classification</td>
                  <td valign="top">2019</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit96">96</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">FA, NFA</td>
                  <td valign="top">320</td>
                  <td valign="top">MP, FP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2019</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit97">97</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">FA, NFA</td>
                  <td valign="top">1900</td>
                  <td valign="top">Device structure</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2019</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit98">98100</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top">FA, NFA</td>
                  <td valign="top">566</td>
                  <td valign="top">MP, FP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2020</td>
                  <td valign="top">
                    <a title="Select to navigate to reference" href="#cit46">46 and 101</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top"></td>
                  <td valign="top">FA, NFA</td>
                  <td valign="top">70</td>
                  <td valign="top">FMO</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2020</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit102">102</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">SM, polymer</td>
                  <td valign="top"></td>
                  <td valign="top">1758</td>
                  <td valign="top">FP</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2021</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit103">103</a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Literature</td>
                  <td valign="top">Polymer</td>
                  <td valign="top">FA, NFA</td>
                  <td valign="top">84</td>
                  <td valign="top">MP, DFT</td>
                  <td valign="top">Regression</td>
                  <td valign="top">2022</td>
                  <td valign="top">
                    <a title="Select to navigate to references" href="#cit104">104</a>
                  </td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
      
      
        
        <h3 id="sect760"><span class="b_heading">2.2. Data preparation</span></h3>
        <span>After gathering a dataset, suitable set of descriptors are required for prediction of a specific target property. Descriptors used for machine learning applications should be easily accessible so that predictions for unexplored materials or combinations could be quick after saving the model. The solution is to use descriptors that are directly calculated from the chemical structure. In OSC studies, the commonly used input descriptors are microscopic properties (MP), molecular descriptors (MD), molecular fingerprints (FP), frontier molecular orbitals (FMO), and images.</span>
        <p class="otherpara">From photon absorption to transport of charge carriers to their respective electrodes, there are a number of microscopic properties that highly influence the PCE of OSCs.<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> Some of the microscopic properties that have been used in ML studies for OSCs are charge carrier mobility, optical bandgap, electronhole binding energy, and many more. However, they are expensive to calculate but provide more realistic and accurate measures for organic materials.</p>
        <p class="otherpara">To convert the chemical structure of OSC materials into a machine-readable format, simplified molecular input line entry system (SMILES) strings<a title="Select to navigate to references" href="#cit107"><sup><span class="sup_ref">107</span></sup></a> are used. Using SMILES strings, molecular descriptors and molecular fingerprints are generated by using various open-source packages. A comprehensive set of chemical descriptors must be used instead of SMILES strings to describe molecules in order to create a powerful machine learning model.</p>
        <p class="otherpara">A molecular descriptor is a machine-readable representation of the information contained within a molecule. Atomic count, atomic type, and molecular weight are all examples of zero-dimensional (0D) descriptors because they make no inferences about topology or atom connectivity. Chemical fragment kinds and their count can be described using one-dimensional (1D) descriptors. Alternatively, two-dimensional (2D) descriptors characterize molecular topology and chemistry. Three-dimensional (3D) descriptors, which take into account conformational information like molecular volume and partial surface charges, capture geometrical information as well.<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a></p>
        <p class="otherpara">A molecular fingerprint is the representation of some known structural property of a molecule. When the expected data structure is available, the corresponding bit is set to 1 (ON); otherwise, it is left at 0 (OFF). More bits mean more structural information and could be further used for comparing the structural similarity of two molecules.</p>
        <p class="otherpara">FMOs (D<small><sub>HOMO</sub></small>, D<small><sub>LUMO</sub></small>, A<small><sub>HOMO</sub></small>, A<small><sub>LUMO</sub></small>) of donors and acceptors have a direct impact on the charge carrier dynamics in OSCs. With these values, LUMO<small><sub>offset</sub></small>, HOMO<small><sub>offset</sub></small>, and Donor<small><sub>HOMO</sub></small>Acceptor<small><sub>LUMO</sub></small> are also taken into account to predict better results. For prediction of ternary OSC photovoltaic properties, FMOs of donor, acceptor, and the third component are simultaneously used as input descriptors.<a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a> Similarly, for tandem OSCs, FMOs for both cells (bottom sub-cell near ITO and top sub-cell near the metal electrode) are used as input descriptors for ML models.<a title="Select to navigate to references" href="#cit102"><sup><span class="sup_ref">102</span></sup></a></p>
        <p class="otherpara">The material structure image or the structure encoded image could be directly used by convolutional neural network (CNN) models for materials property prediction or device property prediction. Models can achieve high accuracy by using images as input, but much larger datasets are required.</p>
        <p class="otherpara">Since not all material qualities are relevant for all performance indicators, the list of descriptors that will uniquely define the data will depend on the purpose or information to be retrieved. Before feeding input data into ML models for training, raw data need to be transformed into data that can be effectively used for supervised learning and this process is called feature engineering. The steps involved in the feature engineering process are (1) imputation (handling missing values), (2) encoding categorical variables, (3) handling outliers, (4) scaling (minmax scaling and standardization scaling), and (5) creating new descriptors (from domain knowledge). With the increase in the number of descriptors, the number of data points should be also increased for effectively training a ML model. Since in OSCs, large experimental datasets are not available, high dimensional datasets may confuse ML models, and dimensionality reduction becomes crucial. Input descriptors that are highly correlated are not relevant for the model and are usually dropped based on high correlation coefficient and variance threshold. The recursive feature elimination (RFE) technique is preferably used for this purpose. Principal component analysis (PCA)<a title="Select to navigate to references" href="#cit109"><sup><span class="sup_ref">109</span></sup></a> is a well-known instance of feature extraction, a technique used to reduce dimensionality by constructing a smaller collection of new descriptors from the original list. After applying such feature engineering techniques, the dataset becomes ready to train ML models.</p>
      
      
        
        <h3 id="sect785"><span class="b_heading">2.3. Model training</span></h3>
        <span>ML implementations are classified into three categories: supervised learning, unsupervised learning, and reinforcement learning.</span>
        <p class="otherpara">In supervised learning, the model learns the functional mapping between input descriptors and output values. Both regression and classification problems come under supervised learning and is by far the most common type of ML in the field of OSCs. Unsupervised learning models are used to draw inferences from datasets that consist of input data but do not have labeled responses. Lastly, reinforcement learning is a method that simulates the way in which people learn by interacting with their surroundings. This method allows an algorithm to improve its performance on a variety of tasks by receiving feedback in the form of rewards or punishments.</p>
        <p class="otherpara">The goal of classification is to predict class labels from the input data. In binary classification, there are only two class labels; if class labels are more than two, it is called multiclass classification. Classification of organic compounds with a high PCE in OSCs has been used in many studies. Because classification methods provide discrete results, a metric is required to compare the distinct classes. Classification metrics measure a model's performance and tell you how excellent or poor the classification is, but each evaluates it differently.</p>
        <p class="otherpara">In the regression task, we try to predict a continuous dependent variable using a set of independent variables. For example, photovoltaic properties, such as PCE values, are mostly predicted. Prediction error is used to quantify the success of a model in regression situations. Prediction error, also known as residuals, is the discrepancy between the observed and expected values. The goal of the regression model is to find the best line fit that minimizes the discrepancy between the predicted and observed values. Researchers are actively working to develop a more precise model based on a variety of usability considerations. To gauge the success of a new regression model, it is customary to benchmark it against previously developed models using a set of accepted metrics. Most commonly, these ML architectures for OSC studies are directly accessed from the scikit-learn python package.<a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a></p>
        <p class="otherpara">Quantitative structureproperty relationship (QSPR) models are being used to understand the hidden relationship between materials and their photovoltaic properties.<a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a> A relationship between the structure of materials and their target property could be obtained by generating feature importance<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> and visualization of decision tree.<a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> Various ML methodologies have been used specifically on OSCs for QSPR.<a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> Model training is done by selecting a suitable cross-validation method for getting an optimal set of hyperparameters for the selected ML model. Generally, K fold cross-validation is used, but in cases like OSCs where available experimental datasets are not large, leave-one-out-cross validation (LOOCV) is often used. There are currently a plethora of resources available to help with the creation of ML models for use in the materials science field. There has been an explosion of specialized open-source ML software libraries for materials research such as scikit-learn,<a title="Select to navigate to references" href="#cit112"><sup><span class="sup_ref">112</span></sup></a> TensorFlow,<a title="Select to navigate to references" href="#cit113"><sup><span class="sup_ref">113</span></sup></a> and PyTorch.<a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a></p>
      
      
        
        <h3 id="sect801"><span class="b_heading">2.4. Evaluation metrics</span></h3>
        <span>A model's performance may be monitored and measured using metrics (during training and testing). In any machine learning pipeline, performance indicators, also called evaluation metrics, play a significant role. Evaluation metrics are used to quantify performance and compare ML models for classification or regression. We have restricted our focus to supervised learning evaluation metrics.</span>
        <p class="otherpara">The accuracy (A) metric simply measures how often the classifier predicted correctly. The Confusion Matrix is a table-based representation of the ground-truth labels <span class="italic">versus</span> the model's predictions. For each row in the confusion matrix, instances in a predicted class are represented, whereas occurrences in a real class are represented in each column. True Positive (TP) denotes the number of positive class samples correctly classified by the model. True Negative (TN) signifies the number of negative class samples classified correctly by the model. False Positive (FP) denotes the number of negative class samples predicted incorrectly by the model. False Negative (FN) signifies the number of positive class samples predicted incorrectly by the model.</p>
        <p class="otherpara">The Confusion Matrix is not technically a performance statistic, but it serves as a foundation for evaluating other metrics such as precision (P) and recall (R). When False Positives are of more concern than False Negatives, precision is helpful. Precision is the ratio of True Positives to total number of predicted positives. Recall or Sensitivity is a useful metric in cases where False Negative is of higher concern than False Positive. Recall is the ratio of true positives to total number of actual positives. The F1 score is the harmonic mean of precision and recall. The F1 score is primarily used for comparing different classifier models. The false positive rate (FPR) is defined as the ratio of False Positives to the total number of actual negatives. Specificity is the ratio of true negatives to total number of actual negatives. All major classification evaluation metrics are given in <a title="Select to navigate to table" href="#tab2">Table 2</a>.</p>
        
<div class="table_caption"><b>Table 2</b> <span id="tab2">Metrics and their equation used for evaluating the performance of classification and regression ML models</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/></colgroup>
              
              
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Classification metric</th>
                  <th align="left" valign="bottom" class="border_black">Equation</th>
                  <th align="left" valign="bottom" class="border_black">Regression metric</th>
                  <th align="left" valign="bottom" class="border_black">Equation</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Accuracy (A)</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t1.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t1.gif"/></a>
                  </td>
                  <td valign="top">r</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t2.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t2.gif"/></a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Precision (P)</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t3.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t3.gif"/></a>
                  </td>
                  <td valign="top">R<small><sup>2</sup></small></td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t4.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t4.gif"/></a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Recall (R)/sensitivity/true positive rate (TPR)</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t5.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t5.gif"/></a>
                  </td>
                  <td valign="top">MAE</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t6.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t6.gif"/></a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">F1</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t7.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t7.gif"/></a>
                  </td>
                  <td valign="top">MAPE</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t8.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t8.gif"/></a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">FPR</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t9.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t9.gif"/></a>
                  </td>
                  <td valign="top">MSE</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t10.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t10.gif"/></a>
                  </td>
                </tr>
                <tr align="left">
                  <td valign="top">Specificity/true negative rate (TNR)</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t11.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t11.gif"/></a>
                  </td>
                  <td valign="top">RMSE</td>
                  <td valign="top">
                    <a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-t12_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-t12.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-t12.gif"/></a>
                  </td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">The receiver operating characteristic (ROC) curve is a probability curve that shows how well a classification model works at all classification thresholds by plotting two parameters: the true positive rate (TPR) and the false positive rate (FPR). The area under the ROC curve (AUC) is a measure of how well a model can tell the difference between two classes. The better the model can guess 0's are 0's and 1's are 1's, the higher the AUC.</p>
        <p class="otherpara">Some of the most common measures of evaluation for regression models are outlined here. The Pearson correlation coefficient (<span class="italic">r</span>) is the most widespread measure for measuring linear correlation. It measures the strength and direction of the relationship between two variables and ranges between 1.0 and +1.0. The <span class="italic">R</span><small><sup>2</sup></small> score is defined as the proportion of variance in a dependent variable that is predictable from the independent variables. It is also known as coefficient of determination. Mean absolute error (MAE) is one of the most used evaluation metrics and is simply calculated as absolute difference between actual and predicted values. Mean absolute percentage error (MAPE) is the percentage equivalent of MAE normalized by true observations. Mean squared error (MSE) is similar to MAE, but the error is squared here. Root mean squared error (RMSE) is the most widely used evaluation metric. It is simply the square root of MSE. Equations for these evaluation metrics are provided in <a title="Select to navigate to table" href="#tab2">Table 2</a>. In <a title="Select to navigate to table" href="#tab2">Table 2</a>, <span class="italic">x</span><small><sub><span class="italic">i</span></sub></small> denotes the actual value, <span class="italic"><img class="charmap" src="https://www.rsc.org/images/entities/i_char_0078_0304.gif" alt="[x with combining macron]"/></span> denotes the mean of actual values, <span class="italic">y</span><small><sub><span class="italic">i</span></sub></small> denotes the predicted value, and <span class="italic"></span> denotes the mean of predicted values.</p>
      
      
        
        <h3 id="sect882"><span class="b_heading">2.5. Discovery</span></h3>
        <span>Structural flexibility of organic semiconductors provides vast chemical search space for OSC materials, and ML is being utilized for this purpose with full potential. Trained ML models are used for high-throughput screening of novel OSC materials that the models have never seen during the training phase. High-performing materials predicted by the ML models are screened out, and their predicted property is validated experimentally.</span>
      
    
    
      
      <h2 id="sect886"><span class="a_heading">3. Review of ML in OSC research</span></h2>
      
        
        <h3 id="sect889"><span class="b_heading">3.1. FA based OSCs</span></h3>
        <span>In 2018, Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> gathered a dataset of 280 small molecule OSCs with 270 distinct small molecule donors and two distinct acceptors (PC<small><sub>61</sub></small>BM and PC<small><sub>71</sub></small>BM). Five ML models (LR, k-NN, ANN, RF, GB) for PCE prediction were employed with 13 microscopic properties as inputs for ML models. The 13 microscopic properties used as inputs for ML models are (1) number of conjugated atoms in the main conjugation path of the donor molecule (<span class="italic">N</span><small><sup>D</sup></small><small><sub>atom</sub></small>), (2) polarizability, (3) energetic difference of LUMO and LUMO1 (<span class="italic"></span><small><sub>L</sub></small>), (4) energetic difference of HOMO and HOMO1 (<span class="italic"></span><small><sub>H</sub></small>), (5) vertical ionization potential of donor molecules (IP(<span class="italic"></span>)), (6) reorganization energy for holes in donor molecules (<span class="italic"></span><small><sub>h</sub></small>), (7) holeelectron binding energy in donor molecules (<span class="italic">E</span><small><sub>bind</sub></small>), (8) energetic difference between LUMO of donor and LUMO of acceptor (<span class="italic">E</span><small><sup>DA</sup></small><small><sub>LL</sub></small>), (9) energetic difference between HOMO of donor and LUMO of acceptor (<span class="italic">E</span><small><sup>DA</sup></small><small><sub>HL</sub></small>), (10) energy of transition to singlet excited state with largest oscillator strength (<span class="italic">E</span><small><sub>g</sub></small>), (11) change in dipole moment in going from ground state to first excited state for donor molecules (<span class="italic"></span><small><sub>ge</sub></small>), (12) energy of electronic transition to lowest-lying triplet state (<span class="italic">E</span><small><sub>T1</sub></small>), and (13) energetic difference between LUMO and LUMO+1 of acceptor (<span class="italic"></span><small><sup>A</sup></small><small><sub>L</sub></small>). The GB model performed the best for the test set (<span class="italic">r</span> = 0.79) (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3(a)</a>) and using LOOCV (<span class="italic">r</span> = 0.76) (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3(b)</a>). As shown in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3(c and d)</a>, descriptor importance revealed that for both ensemble techniques GB and RF, out of the 13 descriptors, holeelectron binding energy (<span class="italic">E</span><small><sub>bind</sub></small>)<a title="Select to navigate to references" href="#cit115"><sup><span class="sup_ref">115</span></sup></a> is the most important descriptor.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f3.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">(a) Theoretically predicted <span class="italic">versus</span> experimental PCE for the testing set (30 molecules). (b) All data points using the LOO cross validation technique for the GB model. (c) The descriptor importance for the GB. (d) The descriptor importance for the RF. Descriptors are in the following order: (1) <span class="italic">N</span><small><sup>D</sup></small><small><sub>atom</sub></small>, (2) polarizability, (3) <span class="italic">L</span>, (4) <span class="italic">H</span>, (5) IP(<span class="italic"></span>), (6) <span class="italic">h</span>, (7) Ebind, (8) <span class="italic">E</span><small><sup>DA</sup></small><small><sub>LL</sub></small>, (9) <span class="italic">E</span><small><sup>DA</sup></small><small><sub>HL</sub></small>, (10) energy of transition to singlet excited state with largest oscillator strength (<span class="italic">E</span><small><sub>g</sub></small>), (11) change in dipole moment in going from <span class="italic"></span><small><sub>ge</sub></small>, (12) ET1, and (13) <span class="italic"></span><small><sup>A</sup></small><small><sub>L</sub></small>. Reproduced with permission<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> Copyright, 2018, John Wiley and Sons.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> designed a polymer for OPV using machine learning for the first time. Despite the advances in ML, data-driven approaches for OPV are not performing that well. In this work, the authors have used supervised algorithms (RF and ANN) for screening potential polymerfullerene OPV. The authors conducted a study on 1200 polymerfullerene data collected from the literature (500 papers). Using the random forest model, they achieved a correlation coefficient (<span class="italic">r</span>) of 0.6 to 0.7. Various parameters have been used in this study such as <span class="italic">M</span><small><sub>w</sub></small>, <span class="italic">E</span><small><sub>g</sub></small>, FMO, and fingerprints (MACCS<a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a> and ECFP6<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a>). In the next step the polymer design scheme is used as depicted in <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>. By using the 2.3 million HCEP dataset, 1000 molecules were selected, and MACCS fingerprints were used to train the classification model for PCE. The model gave an accuracy of 48%. Based on synthetic feasibility, they selected one polymer with predicted PCE (5% to 5.8%) but got only 0.53% PCE experimentally. The reasons for poor results by the authors are (1) less dataset and (2) poor performance of the Scharber model. Non-availability of exact processing conditions, including film thickness, solvent, solvent additive, p/n blend ratio, and thermal solvent annealing, is a significant issue for this study. The authors also focused on the unavailability of essential data such as (1) purity of the polymer, (2) surface free energy, (3) polymer orientation, and (4) BHJ morphology associated with miscibility.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f4.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Scheme of polymer design by combining the RF screening and manual screening/modification. The picked-up molecule or polymer in each stage is shown. Reproduced with permission<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> Copyright  2018 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Chen<a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a> used a dataset of 1203 polymerfullerene OSCs by Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> for virtual screening using RF and SVM models. Using RDKit,<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a> morgan fingerprints<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a> were calculated as input descriptors, and the use of morgan radius further improved the model accuracy. Predictions of these two models were further ensembled to achieve more accurate results (<span class="italic">r</span> = 0.653).</p>
        <p class="otherpara">Using the concepts of design of experiments (DoE),<a title="Select to navigate to references" href="#cit119"><sup><span class="sup_ref">119</span></sup></a> one may test and optimize several variables at the same time. Minimizing experimental effort while maximizing experimental information output is meant by DoE, a strategy for conducting experiments. In 2018, Buriak <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> demonstrated the ML model and DOE integration to create meaningful multidimensional maps for insights on PCE. For DoE on the PCDTBT<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>PCBM solar cell, four initial parameters with four levels each were chosen: (1) donor weight percentage, (2) total solution concentration, (3) bulk heterojunction spin cast speed, and (4) processing additives. A complete analysis would require 4<small><sup>4</sup></small> = 256 experiments; by using fractional factorial design only 16 experiments were performed. With variance analysis of all the parameters, the contribution of the processing additive was low and hence was dropped for further analysis. These data were fit using support vector machine (SVM) with radial basis function (RBF) kernel to determine PCE. Three-dimensional maps were generated to visualize PCE approximation in three-parameter space.</p>
        <p class="otherpara">Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> in 2018 used the Scharber model on a dataset of 249 organic donoracceptor pairs, where acceptors are limited to only C<small><sub>60</sub></small>, PC<small><sub>61</sub></small>BM, or PC<small><sub>71</sub></small>BM. High accuracy was attained using the Tanimoto similarity index between donors and the Euclidean distance of electronic properties for PCE predictions. Prediction results were obtained utilizing both topological and electronic descriptors with KRR (<span class="italic">r</span> = 0.68) and k-NN (<span class="italic">r</span> = 0.61) algorithms. The authors concluded that the Scharber model performs poorly when studied on a collected dataset with <span class="italic">r</span> = 0.38 for <span class="italic">V</span><small><sub>oc</sub></small>, 0.17 for <span class="italic">J</span><small><sub>sc</sub></small>, and 0.18 for PCE. Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> (2019) created a dataset of 300 OSCs (small molecule donor and fullerene acceptor). More than 10K molecules were generated for creating a search space by using 32 unique molecular building blocks (donor, acceptor, -spacer, and end-capping groups) and 17K DFT calculations. Five machine learning models were trained on 250 data points using 10-fold cross-validation, and the results were compared. The best results were achieved by GBRT (<span class="italic">r</span> = 0.80) and the worst results by the Scharber model (<span class="italic">r</span> = 0.14). The authors gave the following reasons for the failure of the Scharber model: (1) oversimplified model and (2) the limitations of DFT in estimating FMO. The authors identified important moieties using the GBRT model. They filtered out 126 potential molecules with predicted PCE &gt; 8% and suggested synthesizing materials and fabricating the devices.</p>
        <p class="otherpara">Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> used an earlier dataset of 300 fullerene-based OSCs<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> and used two ML models (RF and GBRT) to predict <span class="italic">J</span><small><sub>sc</sub></small>, <span class="italic">V</span><small><sub>oc</sub></small>, FF, and PCE. Special attention is given to <span class="italic">J</span><small><sub>sc</sub></small> and <span class="italic">V</span><small><sub>oc</sub></small> prediction because of potential commercial applications: water splitting for high solar to fuel energy conversion (high <span class="italic">V</span><small><sub>oc</sub></small> is required) and solar-window application (high <span class="italic">J</span><small><sub>sc</sub></small> is required). 13 molecular properties were used as input descriptors for ML models and also inspected visually by using decision tree for all target variables (<span class="italic">J</span><small><sub>sc</sub></small>, <span class="italic">V</span><small><sub>oc</sub></small>, FF, and PCE). The GBRT model gave the best results for all four cases <span class="italic">J</span><small><sub>sc</sub></small> (<span class="italic">r</span> = 0.66), <span class="italic">V</span><small><sub>oc</sub></small> (<span class="italic">r</span> = 0.67), FF (<span class="italic">r</span> = 0.71), and PCE (<span class="italic">r</span> = 0.78).</p>
        <p class="otherpara">Ternary OSCs are made by introducing a third component into the binary blend. Adding a third component in the blend increases light harvesting, reduces energy loss, and creates more D/A interfaces, facilitating exciton dissociation more effectively. In 2019, Lee<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> created a dataset of 124 fullerene derivative-based ternary OSCs (published during 20122019). The dataset was manually collected from the literature (&gt;100 papers) and consisted of FMOs of donor, acceptor, and the third component, and their corresponding PCE. Using this dataset, five ML models (RF, GB, k-NN, LR, and SVR) were compared to predict PCE, and the best results were achieved with RF (<span class="italic">R</span><small><sup>2</sup></small> = 0.77) followed by GB (<span class="italic">R</span><small><sup>2</sup></small> = 0.73). The authors also studied a two-class classification model on the same dataset, class1 (PCE &gt; 9%) and class2 (PCE &lt; 9%). Again, the RF model performed the best with a test accuracy of 0.76, and with further hyperparameter optimization, the test accuracy improved to 0.855. Lee<a title="Select to navigate to references" href="#cit73"><sup><span class="sup_ref">73</span></sup></a> used an earlier dataset<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> (# = 124) for the prediction of open-circuit voltage (<span class="italic">V</span><small><sub>oc</sub></small>) in ternary OSCs using ML models (RF and SVR). FMOs of donors, acceptors, and the third component were taken as input descriptors. The RF model performed the best with <span class="italic">R</span><small><sup>2</sup></small> = 0.77 for the test set and was further used to generate feature importance scores. According to the model, HOMO and LUMO of donor are the two most important factors that influence <span class="italic">V</span><small><sub>oc</sub></small>. Using a contour plot, the author visualized optimal energy level alignment rules for donor and the third component.</p>
        <p class="otherpara">Small datasets are not ideal for training a neural network; in the case of OSCs, large experimental datasets are not available. MacKenzie <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit120"><sup><span class="sup_ref">120</span></sup></a> generated a dataset of 20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 devices using the ShockleyReadHall based drift-diffusion model using gpvdm. For all the devices, dark and light currentvoltage curves were simulated using randomly assigned electrical parameters such as trap densities, recombination time constants, carrier trapping states, energetic disorder, and parasitic resistance. They demonstrated their method for getting optimal surfactant choice and annealing temperature in terms of charge carrier dynamics in P3HT, PBTZT-stat-BDTT-8, and PTB7 based OSCs with PCBM acceptors. All these electrical parameters were included in the output layer node of the neural network as depicted in <a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>. After model training the neural network is used to predict electrical parameters using dark and light currentvoltage curves as input.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f5.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">A diagram of the neural network used to extract material parameters from the data within this paper; the actual network used had ten times more neurons in each hidden layer than the diagram depicts but otherwise the same structure. Visible on the left hand side of the image are the experimental (or simulated) data, with the red dots on the curves representing the points at which the curves were sampled to form input vectors for the neural network. (b) In the diagram a light and a dark JV curve are each being sampled at 6 places to provide 12 data points to the neural networks 12 input nodes. Any number or combination of experimental measurements can be placed on the input to the network; one simply has to extend the number of input neurons, and retrain the network. The neural network itself has red input nodes, blue hidden layers, and green output nodes. Each output node corresponds to a device/material parameter such as charge carrier mobility or trap density. Inset: A single neuron. Reproduced with permission<a title="Select to navigate to references" href="#cit120"><sup><span class="sup_ref">120</span></sup></a> Copyright  2019 WILEY-VCH Verlag GmbH &amp; Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Goharimanesh <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> in 2022 used 240 small molecule OSC data from the dataset of Ma <span class="italic">et al</span>.<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> The performance of six ML models (LR, k-NN, ANN, RF, GB, and XGB) was compared for the prediction of PCE. Expensive quantum chemical descriptors were used as input (calculated with the Gaussian 09 package), and ANN performed the best (<span class="italic">r</span> = 0.79) on the training set. The authors also designed a new technique for mapping structureproperty relationships by combining the Taguchi design of experiments (TDOE) approach and ML. The complete workflow of this study using the TDOE approach is represented in <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>. A PCE map represented model interpretability for the following descriptors studied: energetic difference between LUMO+1 and LUMO of donor (<span class="italic"></span><small><sub>L</sub></small>), energetic difference between HOMO and HOMO1 of donor (<span class="italic"></span><small><sub>H</sub></small>), optical gap (<span class="italic"></span><small><sub>HL</sub></small>), hole reorganization energy (<span class="italic"></span><small><sub>h</sub></small>), exciton binding energy (<span class="italic">E</span><small><sub>b</sub></small>), and LUMO band offset (<span class="italic"></span><small><sub>LUMO</sub></small>).</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f6.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">Steps of the research in this paper: (A) data construction, (B) feature extraction, (C) ML prediction, (D) SHAP mappings, (E) optimization by TDOE, and (F) rational insights. Reproduced with permission<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> Copyright  2022 Elsevier Ltd.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1078"><span class="b_heading">3.2. NFA based OSCs</span></h3>
        <span>Aspuru Guzik <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> in 2017 created a dataset of 51<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 NFAs based on tetraazabenzodifluoranthenes (BFIs), diketopyrrolo-pyrroles (DPPs), perylene diimides (PDIs), benzothiadiazole (BT), and fluoranthene-fused imides. Since the Scharber model was used to predict the PCE of NFAs and poly[<span class="italic">N</span>-90-heptadecanyl-2,7-carbazole-<span class="italic">alt</span>-5,5-(40,70-di-2-thienyl-20,10,30-benzothiadiazole)] (PCDTBT) as the donor material, poor results were obtained while comparing true and predicted PCE (<span class="italic">r</span> = 0.43 and <span class="italic">R</span><small><sup>2</sup></small> = 0.11).</span>
        <p class="otherpara">To discover novel active materials for OSCs, generative models<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a> are being used effectively. Generative models generate new molecules by creating a latent space from molecular information and then manipulating the latent space. For OSCs, novel polymer donors<a title="Select to navigate to reference" href="#cit54"><sup><span class="sup_ref">54,63</span></sup></a> and NFAs<a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> are generated using generative models. Hada <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> developed a model for high throughput screening of thiophene-based DA polymers. They automated the process in three steps: (1) polymer generation, (2) orbital energy estimation by Hckel-based model, and (3) photovoltaic property calculation. Using donor and acceptor units, 380 polymers were generated and studied, followed by identification of promising acceptor units for photovoltaic materials. The results are not significant in terms of PCE because the Scharber model solely uses orbital energies.</p>
        <p class="otherpara">In recent years, CNNs have established themselves as the standard framework for computer vision tasks. Then the training model is utilized for regression or classification task. Because of its capacity to build and evaluate features in image-like inputs, CNN has become extremely popular. Zhao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> in 2019 used CNN for the generation of new acceptors and their structureproperty prediction. The authors have demonstrated the generation of new molecules by CNN and controlling the diversity of the molecules by variation in the number of convolution layers. Post molecule generation, various descriptors are used for property prediction such as extended connectivity fingerprints, molecular graphs, bag of bonds, coulomb matrix, and SMILES strings. The attention mechanism is adopted to get character-specific weights in SMILES strings. The model is used for prediction of HOMO, LUMO, and PCE. The authors have used a dataset from Aspuru Guzik <span class="italic">et al.</span>'s (# = 51<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000) dataset.<a title="Select to navigate to references" href="#cit61"><sup><span class="sup_ref">61</span></sup></a> For this study, all the molecules with PCE &lt; 0.5% were removed, and 24<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 molecules were chosen randomly. Out of this, 20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 were used for training, 2000 for evaluation, and 2000 for testing. The donor used for this study is fixed, and the Scharber model was used for calculating PCE.</p>
        <p class="otherpara">In 2019, Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a> used CNN on the HCEP dataset for the prediction of PCE. Without any prior transformation, the CNN model uses chemical structure images of donor materials as input. The dataset is classified into two categories of PCE (04.9% and 59.9%). The classification model achieved an accuracy of over 90%. The impact of data size on model accuracy is also studied. The major drawbacks of this study are (1) donor molecules being used today are much more structurally complex and bigger in size compared to those in the HCEP dataset and (2) PCE is calculated using the Scharber model, and the Scharber model estimates PCE using DFT-calculated energy levels.</p>
        <p class="otherpara">Using a similar approach as in ref. <a title="Select to navigate to references" href="#cit66">66</a>, Wei <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit67"><sup><span class="sup_ref">67</span></sup></a> performed a similar study with a 500 polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>SMA dataset using ECFP6 fingerprints. The authors studied two new polymers (BO2FC8 and BO2FEH) and using the random forest model they predicted PCE (11.2% and 10.9%). They also validated the results with PCE of 11.0% and 6.4% from experimental findings. Interestingly the former gave consistent results with the ML model, while the latter results did not agree with the ML model.</p>
        <p class="otherpara">Min <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> used five ML models, linear regression (LR), multiple linear regression (MLR), boosted regression trees (BRT), random forest (RF), and artificial neural network (ANN), on 565 polymer donor<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA pairs to predict PCE. The dataset was manually collected from the literature (274 papers). 477 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs were used for training and the best results were achieved by BRT (<span class="italic">r</span> = 0.71) and RF (<span class="italic">r</span> = 0.7), while the other models performed poorly (LR (<span class="italic">r</span> = 0.54), MLR (<span class="italic">r</span> = 059), and ANN (<span class="italic">r</span> = 0.6)). Based on the structural fragment in donor and acceptor in 565 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs, the authors created a search space dataset of 3<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>76<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs and predicted their PCE using the trained BRT and RF model. The complete workflow is depicted in <a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a>. From the predictions of both models, it was observed that for RF, 12.27% of D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs gave a PCE of greater than 11%, while for BRT, 14.15% D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs gave a PCE of greater than 11%. Six D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs were selected for experimental validation with donors (PM6 and PBDB-T) and acceptors (Y-ThCN, Y-ThCH3, and Y-PhCl). These acceptors were selected for experimental validation because they are easily synthesizable with a one-step method. The PCE values obtained in experiments were quite similar to those predicted by BRT and RF.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f7.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">Workflow of building, application, and evaluations of machine learning methods in this work. (a) Scheme of collecting experimental data and converting chemical structures to digitized data. (b) Scheme of machine training, prediction, and method evaluation. Reproduced with permission<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> Copyright  2020, The Authors, Springer Nature.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Lee<a title="Select to navigate to references" href="#cit80"><sup><span class="sup_ref">80</span></sup></a> created a dataset of 135 donor<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA pairs from literature reviews<a title="Select to navigate to reference" href="#cit122"><sup><span class="sup_ref">122,123</span></sup></a> consisting of 117 unique NFA and 30 unique donor materials. FMOs and bandgaps were used as input descriptors to predict PCE using RF and GB regression models. For the test set, the RF model gave the best results (<span class="italic">R</span><small><sup>2</sup></small> = 0.80), and the acceptor bandgap was given the highest feature importance. Buriak <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit124"><sup><span class="sup_ref">124</span></sup></a> in 2020 used the same ML-DOE approach as in ref. <a title="Select to navigate to references" href="#cit47">47</a> for an all-small-molecule organic solar cell with the donor DRCN5T and acceptors ITIC, IT-M, and IT4F. Their work focused on active layer optimization of OSCs, and four initial parameters were used for DoE: (1) solution concentration of donor and acceptor ink, (2) donor fraction, (3) temperature, and (4) annealing time. Maps were derived for power conversion efficiency to visualize the effect of parameters and find the most optimal combination of parameters. It is important to note that fitting functions like RBF are good for interpolation only and should not be applied for extrapolation tasks.</p>
        <p class="otherpara">Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> manually collected a dataset of 566 polymer donor<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>non-fullerene acceptors and used material property (<span class="italic">E</span><small><sub>g</sub></small>, HOMO, LUMO, <span class="italic">M</span><small><sub>w</sub></small>) and ECFP6 fingerprints for prediction of PCE. The dataset was collected until 2018 and does not include high-performing NFA of Y6 or ternary solar cells. The best result (<span class="italic">r</span> = 0.85  0.02) was obtained using 5-fold CV and <span class="italic">r</span> = (0.85  0.02) with LOOCV. From their earlier polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>FA dataset (# = 1203)<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> and current polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA dataset (# = 566), donor and acceptor units were extracted, and a total of 2<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>00<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>932 DA polymers were virtually generated as shown in <a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>. For all generated DA polymers, PCE was predicted with acceptors ITIC and IT-4F. Using the RF model on ECFP6 fingerprints, the authors revealed that feature importance of ECFP6 fingerprints was more than polymer molecular properties, and the model gave similar results when polymer molecular features were removed. Interestingly for IT4F, the 20 polymers with the highest predicted PCE are analogs of PBDB-T. From top predictions, they selected second-rank polymer PBDT(SBO)TzH and prepared a device with IT4F. Its predicted efficiency was 10.5%, but it achieved 3.42% PCE in the experiment, which is quite low. This unsatisfactory performance of the polymer is due to the poor solubility, which is also evident from high PDI. They modified PBDT(SBO)TzH and synthesized PBDTTzH, PBDTTzEH, PBDTTzBO, and PBDTTzHD to improve the solubility. PBDTTzEH achieved a PCE of 7.5%. Although the ML workflow did not suggest additives, after using DPE as an additive (0.5 vol%), PCE further improved to 10.1%.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f8.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">Scheme of polymer screening for polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA OPVs. The number (#) of polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>FA<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> and polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA (present work) data points was 1203 and 566, respectively. Donor (D) and acceptor (A) units were extracted from these data after removing overlaps and a total of 200<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>932 new DA polymers were virtually generated. Polymer screening by the RF model constructed on the supervised ML of polymer<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA was applied using MP(n) and FP(n,p) as the descriptors. Reproduced with permission<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> Copyright  2021 Wiley-VCH GmbH.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> increased their dataset for study from 566<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> to 1318. It is important to note that Y6 and its derivatives are included in the increased dataset. The authors studied the effect of dataset size by using ECFP6 fingerprints, Mordred descriptors, and molecular properties and applying a 5-fold CV. The authors studied the effect of categorizing different NFAs and observed that merging all groups into one dataset improves the overall result. The authors also studied the effect of increasing number of data points on ML predictions, and the results are depicted in <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a>. The results reveal that <span class="italic">r</span> = 0.25 for # = 50 and <span class="italic">r</span> = 0.79 for # = 200, and saturates at <span class="italic">r</span> = 0.82 for # = &gt;300. For # = 1318, observed <span class="italic">r</span> = 0.84.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f9.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">(a) Effect of the number of data points (#) of NFA-OPV on the correlation coefficient of the RF model (5-fold CV). (b) Plots of predicted and experimental PCE obtained for each #. The color corresponds to the plot in panel (a). Reproduced with permission<a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> Copyright  2021 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> developed three new polymers (PBDTTzBO, PFSBDTTzBO, and PFBDTTzBO) and predicted their PCE with NFAs (IT-4F and Y6). Using the RF model trained on the 1318 dataset,<a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> predicted PCEs of these three polymers with IT-4F (9.93, 11.35, and 11.47%) were found to be in good agreement with experimental validation (5.24, 7.35, and 10.30%). On the other hand, for Y6, an inverse trend was observed between predicted values (9.20, 12.29, and 12.20%) and experimental validation (11.98, 1.57, and 6.53%). The authors mentioned that an inverse relationship in predicted and experimental values of Y6 PCE is due to a small training set of Y6 acceptors (# = 46) in the complete dataset (# = 1318).</p>
        <p class="otherpara">Liu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a> in 2021 created a dataset of 157 non-fullerene ternary OSCs (published during 20152020). Five ML models (RF, XGBoost, KNN, Decision tree, SVM, Ridge regression, MLP) were used to predict ternary OSCs' PCE and compare their results. FMOs of donor, acceptor, and the third component were used as input descriptors. The RF model performed the best for this task. Moreover, fine-tuned classification models were also compared, and the RF model again gave the best results indicating its suitability for such application. The study outcomes make it clear that (1) LUMO of NFA can slightly lower <span class="italic">V</span><small><sub>oc</sub></small> but significantly enhance <span class="italic">J</span><small><sub>sc</sub></small> and (2) <span class="italic">V</span><small><sub>oc</sub></small> could be optimized by shifting the LUMO of the third component slightly up in ternary OSCs.</p>
        <p class="otherpara">Compared with other photovoltaic technologies such as perovskite solar cells, OSCs have lower <span class="italic">V</span><small><sub>oc</sub></small> and higher non-radiative voltage loss. Sharma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> created a dataset of 154 unique D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations with their corresponding non-radiative voltage loss. Combinations of molecular descriptors, fingerprints, FMO, and optical bandgap (<span class="italic">E</span><small><sub>g</sub></small>) were used as inputs for ML models to predict non-radiative voltage loss. Two molecular descriptors (RDKit<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a> and Mordred<a title="Select to navigate to references" href="#cit125"><sup><span class="sup_ref">125</span></sup></a>) and four fingerprints (CDK,<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a> MACCS,<a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a> PubChem,<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a> and Morgan<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a>) were used in this study. Four ML models (RF, GBR, SVR, and ANN) were used to predict non-radiative voltage loss. The trend of PCE with non-radiative voltage loss was analyzed, and RF obtained the best results by using a combination of FMO, optical bandgap, and RDKit descriptors as input (<span class="italic">r</span> = 0.857). The ML workflow for non-radiative voltage loss is depicted in <a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f10.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">Machine learning workflow for the prediction of %<span class="italic">V</span><small><sub>NR</sub></small>. Data for 154 unique D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations with reported %<span class="italic">V</span><small><sub>NR</sub></small> are collected from the literature, having 46 distinct donors and 79 distinct acceptors. Reported FMO and <span class="italic">E</span><small><sub>g</sub></small> values are taken from the literature and then transformed by median values for distinct donors and acceptors. SMILES codes of donor and acceptor molecules are generated by using ChemDraw software. SMILES codes are then used to generate molecular descriptor datasets and molecular fingerprint datasets. Finally, the datasets are scaled and fed into ML models for the prediction of %<span class="italic">V</span><small><sub>NR</sub></small>. Reproduced with permission<a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> Copyright  2021 International Solar Energy Society. Published by Elsevier Ltd. All rights reserved.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> designed a ML model for P3HT<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>NFA based OSCs (# = 283). More than 3000 easily synthesizable small molecule acceptors were designed using various building blocks. A dataset of 764 small molecule organic semiconductors was collected from the literature, and their corresponding molecular descriptors were calculated using OCHEM<a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> and Chemdes.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> KNIME<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> and Weka<a title="Select to navigate to references" href="#cit131"><sup><span class="sup_ref">131</span></sup></a> open-source platforms were used to perform ML studies. The authors used a two-step screening approach as demonstrated in <a title="Select to navigate to figure" href="#imgfig11">Fig. 11</a>. In the first step, FMOs were predicted using the LR model with # = 764 dataset, followed by the screening of molecules whose FMO aligns properly with P3HT. After screening out 500 NFAs, in the second step their PCE is predicted using the SVM regression model. Further for selection of suitable green solvents, the RF model was trained on a dataset of 252 solvents to predict their corresponding Hansen solubility parameters (HSP).</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig11"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f11_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f11.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f11.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 11 </b> <span id="fig11"><span class="graphic_title">Brief description of the whole study. (a) Model training. (b) Library generation and screening of potential candidates. Reproduced with permission<a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> Copyright, 2021, Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">In 2021, Brabec <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit48"><sup><span class="sup_ref">48</span></sup></a> created an autonomous materials and device application platform (AMANDA Line One) for high-throughput screening and device fabrication. The study was conducted on PM6/Y6 combination for evaluation of PCE, <span class="italic">J</span><small><sub>sc</sub></small>, <span class="italic">V</span><small><sub>oc</sub></small>, FF, and photostability using Gaussian process regressor (GPR). With this method, the authors reported screening of PM6<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>Y6 OSCs with 100 processing conditions within 70 hours (including the photostability test) and the highest PCE of 14% with a fully automated fabrication process. The processing parameters include concentration, D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A ratio, active layer annealing temperature and time, spin speed, solvent additive variations in materials and volume, electron transport layer (ETL) material variation, and ETL annealing temperature and time. With this study, photovoltaic parameters and device stability could be predicted using GPR with high accuracy. Also, the PCE and burn-in losses could be improved by using devices with a medium thermal annealing temperature and a thin active layer.</p>
        <p class="otherpara">Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a> collected a dataset of 265 NFAs with the PTB7-Th donor. ML models (RF, k-NN, LR, and SVM) were used for the prediction of FMO, PCE, and absorption maxima. Input descriptors were calculated using Gaussian 09, OCHEM,<a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> and Chemdes.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> The RF model gave the best results for predicting PCE on the test set (<span class="italic">r</span> = 0.93). Easily synthesizable building blocks were used to construct more than 5000 novel small molecule acceptors (SMAs). Over 1700 small molecule acceptors were discarded since they didnt have matching energy levels in common with the PBT7-Th. Blue-shifted SMAs were not evaluated any further. Based on the predicted UV/visible absorption maxima, the total number of SMAs was whittled down to 2350. The SMAs were then further scrutinized based on the predicted PCE. Molecular dynamics (MD) simulations were used to investigate more than 100 SMAs with more than 13% PCE as shown in <a title="Select to navigate to figure" href="#imgfig12">Fig. 12</a>. The FloryHuggins parameter was used to investigate the mixing behavior of PBT7-Th<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>SMA blends. A total of 15 SMAs were chosen because of their ability to combine well with PBT7-Th.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig12"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f12_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f12.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f12.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 12 </b> <span id="fig12"><span class="graphic_title">The screening pipeline to screen the designed SMAs for PBT7-Th<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>SMA-based OSCs. Reproduced with permission<a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a> Copyright, 2022, Royal Society of Chemistry.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Ma <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> created a reliable QSPR model that integrates chemical descriptors and device requirements and achieves a promising inverse optimization for unexplored potential D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations. The dataset, consisting of 351 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations (44 polymer donors and 195 NFAs), was collected from the literature. Device specifications, 19 electronic properties (using DFT), and six fingerprints were used as inputs for ML models. For developing the QSPR model, different combinations of these input descriptors were studied. With various combinations of these descriptors, sixty ML models were created with ridge regression, GB, SVR, ANN, and voting ensemble approach (combines predictions of all regression models). Since CDK fingerprints performed the best out of all, it was used for further analysis. The voting model obtained the best results for PCE prediction (<span class="italic">r</span> &gt; 0.8). This QSPR model was further integrated with high-throughput screening by creating a search space of 19<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>42<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>785 D/A pairs, and D/A pairs with PCE greater than 14% were screened out. The complete workflow is demonstrated in <a title="Select to navigate to figure" href="#imgfig13">Fig. 13</a>.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig13"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f13_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f13.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f13.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 13 </b> <span id="fig13"><span class="graphic_title">Workflow of screening and optimization of D/A pairs and device parameters. Reproduced with permission<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> Copyright  2021 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Vak <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit49"><sup><span class="sup_ref">49</span></sup></a> in 2021 developed a novel research method that enables quick screening of vast parameter space using methods relevant to industries (roll-to-roll processing). The PM6<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>Y6<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>IT4F ternary system was chosen for the experiment, and 2218 OSCs were fabricated by varying the ratios. This study used two new terminologies: DD (deposition density) and TDD (total deposition density), which refer to the quantity deposited per unit area (in g cm<small><sup>2</sup></small>) of each substance and their total (the sum of PM6, Y6, and IT-4F) quantities. As a result, they give information on coating thickness and material composition ratios at various coating positions indirectly. The RF model was trained on the 2218 dataset, and for the prediction set a high-resolution 3D space based on DD for PM6, Y6, and IT4F was created (80<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>00<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 rows). Experimental confirmation of the prediction led to a PCE of 10.2% in the expected thickness range for the high efficiency formulation (PM6<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>Y6<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>IT-4F = 1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>1.08<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>0.27).</p>
        <p class="otherpara">Lu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit91"><sup><span class="sup_ref">91</span></sup></a> created a dataset of 717 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs of OSCs from the literature (379 papers, published during 20152020). In their dataset, donors are both small molecules and polymers (# = 192), while all acceptors are NFAs (# = 377). Input descriptors were calculated using Dragon 7 software<a title="Select to navigate to references" href="#cit132"><sup><span class="sup_ref">132</span></sup></a> (5270 descriptors for each donor and acceptor). As demonstrated in <a title="Select to navigate to figure" href="#imgfig14">Fig. 14</a>, feature elimination was employed to remove unnecessary features. For the prediction of PCE, four ML algorithms (XGBoost, Decision tree, KNN, and RF) were compared, and XGBoost performed the best (<span class="italic">r</span> = 0.79). The physical meaning of the significant features was studied by adopting the SHAP approach to better characterize the relationship between key features and PCE. Using high-throughput screening on 76<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>814 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations using XGBoost, 10 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations with high PCE were screened. For ensuring the properties of high PCE screened out D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs, photoelectric properties were calculated using DFT, TDFT, and Marcus charge transfer theory.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig14"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f14_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f14.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f14.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 14 </b> <span id="fig14"><span class="graphic_title">The flowchart of the strategy integrating machine learning (ML) and density functional theory (DFT) for the screening of promising D/A combinations. Reproduced with permission<a title="Select to navigate to references" href="#cit91"><sup><span class="sup_ref">91</span></sup></a> Copyright  2022 The Authors. Published by Elsevier Ltd.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Banerji <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> in 2022 used the CNN model to predict HOMO/LUMO levels using the HCEP dataset. SMILES strings are converted to 2D RGB images of chemical structures and are used as input for the CNN model. Transfer learning is employed to overcome the poor performance of the deep learning model on a small dataset. The model is initially trained on the HCEP dataset and then fine-tuned (retrained) on the HOPV dataset using a small learning rate. The results are also compared with a use-case dataset (commercially available 26 polymer donors with experimentally measured and DFT estimated FMO) consisting of commercially available donor materials. An illustration of the complete workflow is provided in <a title="Select to navigate to figure" href="#imgfig15">Fig. 15</a>.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig15"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f15_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f15.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f15.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 15 </b> <span id="fig15"><span class="graphic_title">(a) Illustration of the convolutional neural network, showing the preprocessing step, the convolutional network (with rectified linear unit (ReLU) activation function), max-pooling, global averaging, and fully connected layers (with activation functions). (b) Examples of (left) a molecular image, (middle) the output of two filters after the first max-pooling layer, and (right) the output of the model. Reproduced with permission<a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> Copyright  2022 The Authors. Advanced Theory and Simulations published by Wiley-VCH GmbH.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">In 2022, Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit133"><sup><span class="sup_ref">133</span></sup></a> manually created a dataset of 29 OSCs from the literature employing Y6 and its derivative as an acceptor material and only two donor materials (PBDB-T or PBDB-TF). With LOOCV, the random forest model shows promising results and is further used to screen potential acceptor molecules. Acceptor molecules were split into three parts, end acceptor unit (A1), donor unit (D1), and core acceptor unit (A2), and further encoded with the one-hot-encoding approach. From the dataset all A1D1A2 permutations were created as an acceptor search space for potential acceptor molecules with high PCE. With both the donors, the number of D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations were 1296, and PCE predicted by the RF model was greater than 17% for 25 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations. The model predicts that molecules with high PCE possess side chains of medium length. Five high-performing acceptors discovered by ML are shown in <a title="Select to navigate to figure" href="#imgfig16">Fig. 16</a>. The study on five high-performing acceptor molecules with different donors reveals that the difference in the photoelectric properties of these molecules is mainly because of FMO and electrostatic potential.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig16"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f16_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f16.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f16.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 16 </b> <span id="fig16"><span class="graphic_title">Five typical machine learning predicted high-performance acceptor molecules with different acceptor units at the end groups (highlighted in red) and their predicted PCE. Reproduced with permission<a title="Select to navigate to references" href="#cit133"><sup><span class="sup_ref">133</span></sup></a> Copyright 2022 The Authors. Advanced Science published by Wiley-VCH GmbH.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Hutchison <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> discovered novel NFAs by screening 5426 NFAs using a genetic algorithm and various sequences, symmetry and building blocks were analyzed. PBDB-T-SF is used as the donor material for this study because of its strong absorption in UV and high-energy visible regions. Out of 5426 NFAs, 1087 predicted PCE greater than 18%, and 159 predicted PCE greater than 20%. From the study, the ADDDA or AADAA sequence was found to be best for high PCE. Current terminal acceptor units used in common NFAs were determined to be top performers in the GA candidates when picking individual building blocks.</p>
        <p class="otherpara">Solubility of active materials is one of the crucial factors governing the PCE of OSCs, and it depends on the length of the solubilizing side alkyl chain. In studies with experimentation validation of ML models, it is found that PCE predictions are overestimated, and the materials are not highly soluble.<a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> Unavailability of failure data (failed experiments with low or 0 PCE) can be considered as the major cause of overestimated predictions of ML models.<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> Recently Saeki <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> used artificially generated failure data for training the random forest model on NFA based OSCs. To create the failure dataset (# = 875) the authors replaced three 2-ethylhexyl (EH) with methyl (Me) in the previously reported dataset (# = 1295)<a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> as shown in <a title="Select to navigate to figure" href="#imgfig17">Fig. 17(a)</a>. DA polymers with Me group solubility will be low, and the PCE is assumed to be zero. Using the RF model with Mordred descriptors on the experimental + failure dataset achieved <span class="italic">r</span> = 0.84 as shown in <a title="Select to navigate to figure" href="#imgfig17">Fig. 17(b)</a>, and most of the important features revealed by the model belong to polymer donors (<a title="Select to navigate to figure" href="#imgfig17">Fig. 17(c)</a>). The model was also tested for new sets of polymers (<a title="Select to navigate to figure" href="#imgfig17">Fig. 17e and f</a> (# = 19<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>613) and <a title="Select to navigate to figure" href="#imgfig17">Fig. 17(g and h)</a> (# = 2<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>00<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>932)). The study revealed that for both datasets with inclusion of failure data, the prediction range widens, and predictions of insoluble polymers get shifted to lower PCE, making model predictions more realistic. To check the potential of this model, the authors performed experimental validation by designing and synthesizing 12 new polymers with four types of backbone and different alkyl chains as shown in <a title="Select to navigate to figure" href="#imgfig17">Fig. 17(i)</a>. For each polymer, predicted values with and without failure data are compared with the experimental value as shown in <a title="Select to navigate to figure" href="#imgfig17">Fig. 17(j)</a>. The results reveal that inclusion of failure data gives more realistic predictions and holds potential for discovery of high-performance materials for OSCs.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig17"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f17_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f17.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f17.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 17 </b> <span id="fig17"><span class="graphic_title">(a) Procedure of generating failure data of polymers. (b) Correlation of PCE<small><sub>pred</sub></small><span class="italic">vs.</span> PCE<small><sub>exptl</sub></small> calculated by RF CV using failure data. The <span class="italic">r</span> and # values are appended. (c) Ranking of normalized importance of the RF model. The orange and blue bars are polymer and NFA-related parameters, respectively. (d) PCE<small><sub>exptl</sub></small><span class="italic">vs.</span> ETA_dBeta, the number one-ranked parameter by importance in the RF model. The red arrows represent a trend. (e and f) PCE<small><sub>pred</sub></small> distribution of 19<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>613 virtually generated polymers using IT-4F as the NFA. The PCE<small><sub>pred</sub></small> was predicted by the RF (e) without and (f) with failure data. The categories of soluble/insoluble polymers were labeled by another RF model. (g and h) PCE<small><sub>pred</sub></small> distribution of 200<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>932 virtually generated polymers using Y6 as the NFA. The PCE<small><sub>pred</sub></small> was predicted by the RF (g) without and (h) with failure data. The inset in (g) is the PCE<small><sub>exptl</sub></small> histogram for the dataset. The chemical structure of Y6 is superimposed on (h). (i) Chemical structures of the synthesized polymers. (j) Predicted PCE of twelve polymers (P1P3 assume IT-4F as the NFA; P4 assumes Y6 as the NFA) calculated by the RF model without failure data (meshed blue) and with failure data (solid green). Reproduced with permission<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> Copyright  2022 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect1287"><span class="b_heading">3.3. Mix based OSCs</span></h3>
        <span>The HOPV dataset<a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a> published in 2016 is a collection of 350 organic small molecules and polymers manually collected from the literature that were used as p-type materials in OSCs. This model has been widely used to train QSPR models. Using the Scharber model, <span class="italic">J</span><small><sub>sc</sub></small>, <span class="italic">V</span><small><sub>oc</sub></small>, and PCE were calculated, while FMOs and bandgaps were calculated using the DFT functionals BP86, B3LYP, M06-2x, and PBE0. Many research papers have used this dataset.<a title="Select to navigate to reference" href="#cit54"><sup><span class="sup_ref">54,9395</span></sup></a></span>
        <p class="otherpara">Aspuru Guzik <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit93"><sup><span class="sup_ref">93</span></sup></a> (2016) calculated <span class="italic">J</span><small><sub>sc</sub></small>, <span class="italic">V</span><small><sub>oc</sub></small>, and PCE using the Scharber model. The authors calibrated quantum chemical calculations using the Gaussian approach to improvise the results. The study was conducted with the HOPV dataset on HOMO, LUMO, bandgap, PCE, <span class="italic">V</span><small><sub>oc</sub></small>, and <span class="italic">J</span><small><sub>sc</sub></small>. The results in terms of Pearson's <span class="italic">r</span> were improved after the calibration was applied.</p>
        <p class="otherpara">Balasubramanian <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> generated novel polymer donors for OSCs using the transfer learning scheme based on long short-term memory (LSTM). Since the SMILES notation is a chemical language, LSTM can understand this language's grammar and vocabulary. Novel polymer donor SMILES strings were generated with the LSTM model established on a large dataset (1 million) from the GDB17 chemical database<a title="Select to navigate to references" href="#cit135"><sup><span class="sup_ref">135</span></sup></a> and were further tuned on a smaller (1400 conjugated molecules) dataset.<a title="Select to navigate to reference" href="#cit66"><sup><span class="sup_ref">66,92</span></sup></a> A total of 1000 molecules were generated, out of which 90% were valid SMILES strings with an average Tanimoto similarity of 0.42  0.010. The authors also created a response surface plot for PCE using PCA and found that newly generated polymer donors lie in the region of high PCE.</p>
        <p class="otherpara">Russo <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a> demonstrated the pre-screening potential of machine learning in accurately predicting OSCs material properties and bypassing the need for expensive DFT calculations. For the HOPV dataset, the authors have used molecular signature descriptors (chemical fragments of donors and acceptors) and one-hot descriptors to make their model interpretable. The model could predict DFT calculated PCE with a standard error of 0.5.</p>
        <p class="otherpara">Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> in 2019 gathered a dataset of 1719 donor materials (mix of polymers and small molecules) from the literature with fullerene or non-fullerene acceptors. To build structureproperty relationships, ASCII stings, fingerprints, descriptors, and images were used as input for the ML model. For the classification study, two classes were created, low performance (PCE in the range 02.99) and high performance (PCE higher than 3%). The results reveal that fingerprints with length over 1000 bits perform much better than other descriptors. 81.76% accuracy was obtained in predicting PCE by the RF model using fingerprints as input. The authors also verified the potential of the ML model by synthesizing 10 new donor materials for OSCs, and the predicted PCE values were in good agreement with the experimental values.</p>
        <p class="otherpara">Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit97"><sup><span class="sup_ref">97</span></sup></a> studied the optimization of multicomponent materials for BHJ OSCs. For this purpose, a dataset of 320 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations (262 unique donors and 76 unique acceptors) was manually collected from the literature, and PCE was predicted with different ML models (KRR, SVR, k-NN, and Gaussian process regressor (GPR)). KRR with LOOCV gave the best results (<span class="italic">r</span> = 0.78). Model performance was also verified on recently published D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations and impressive results were obtained.</p>
        <p class="otherpara">Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> in 2020 created a dataset of 566 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs (513 distinct donors and 33 distinct acceptors) and studied the effect of increasing the descriptor set on prediction of PCE using K-nearest neighbours (KNN), kernel ridge regressor (KRR), and support vector regression (SVR). The authors divided the descriptors used in this study into five groups: (1) molecular topology, (2) properties related to molecular size, (3) properties related to molecular energy levels, (4) absorption properties, and (5) mixing properties. Fingerprints were computed using the RDKit package,<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a> and miscibility properties were calculated using SwiddADME.<a title="Select to navigate to references" href="#cit136"><sup><span class="sup_ref">136</span></sup></a> The authors concluded that excited state and miscibility-related properties do not improve the model's performance since the information they carry is already encoded within the structural fingerprint. It is important to remember that not all machine learning models have the exact computing cost. Some low-cost methods, such as simple physical descriptors and structural information, already yield good results.</p>
        <p class="otherpara">Usually, ML models perform well on the class of compounds already used in the training set. However, they do not perform well on a new class of compounds (extrapolation) that have not been seen in the training stage. Troisi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> in 2020 used leave-one-group-out (LOGO) cross-validation to predict the PCE of NFA materials from completely new families and accelerate materials discovery. The dataset used in this study consists of 566 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A pairs. The authors have shown that LOGO significantly improves PCE prediction of unseen materials above and below the median efficiency. The authors also concluded that physical descriptors provide much more reliable results than fingerprints when extrapolating to new chemical families.</p>
        <p class="otherpara">Tandem OSCs can simultaneously address narrow absorption window and thermalization problems by stacking two or more cells with a complementary absorption range.<a title="Select to navigate to references" href="#cit137"><sup><span class="sup_ref">137</span></sup></a> Lee<a title="Select to navigate to references" href="#cit102"><sup><span class="sup_ref">102</span></sup></a> created a dataset of 70 tandem OSCs (37 are conventional and 33 are inverted) to predict PCE with RF and SVR. FMOs for both cells (bottom sub-cell near ITO and top sub-cell near the metal electrode) were used as input descriptors. The RF model performed the best (<span class="italic">R</span><small><sup>2</sup></small> = 0.69) on the test set. Using feature importance, FMO of the donor (bottom sub-cell near ITO) was found to be the most crucial feature in predicting the PCE of tandem OSCs. The best results were achieved by RF followed by XGBoost. Feature importance by the RF model suggests high importance to LUMO of acceptor, while relatively low importance is assigned to HMO and LUMO of the third component. The authors also performed a two-class classification study with class1 (PCE &lt; 16%) and class2 (PCE &gt; 16%). Again the RF model performed the best with an accuracy score of 0.97 on the test set.</p>
        <p class="otherpara">Kettle <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit99"><sup><span class="sup_ref">99</span></sup></a> in 2020 used a dataset of 1850 OSCs (20112019) with corresponding device characteristics, performance, and stability data. The stability and performance of OSCs are estimated using the sequential minimal optimization regression (SMOreg) model. The dataset is acquired from their earlier work.<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> The key attributes related to OSC degradation were identified using the initial efficiency (Eo) and time taken to reach 80% of the initial value (T80). The dataset was separated for tests conducted under ISOS-L and ISOS-D. The choice of light spectrum and active materials were found to be the key parameters to increase stability for ISOS-L testing, while material-depending attributes and encapsulation were found to be the key parameters for ISOS-D testing.</p>
        <p class="otherpara">In 2020, Brabec <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> introduced robotization of the lab for OSC studies. With their novel robotized film creation technology, 6048 films can be produced per day. This automated experimentation platform is integrated with an active learning technique (Bayesian optimization) for 4D parameter space (PTB7-Th or PBQ-QF, P3HT, oIDTBR, PCBM) of quaternary OSC blends. High-throughput experimentation is done by using the ChemOS software package. The study demonstrates lower stability for PTB7-Th rich blends than P3HT and PBQ-QF rich blends. Moreover, the active learning technique is able to find stability maxima and minima with a 93% reduction of sample. This study demonstrates that with robotization, 2000 combinations can be screened with less than 10 mg material, and complex active layer parameter optimization problems can be solved. The automated platform and workflow of high throughput experimentation are represented in <a title="Select to navigate to figure" href="#imgfig18">Fig. 18</a>.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig18"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f18_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f18.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f18.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 18 </b> <span id="fig18"><span class="graphic_title">(a) Representations of the three polymer donors and the two small molecule acceptors. Note that the first quaternary system consists of P3HT, PBQ-QF, PCBM, and oIDTBr () and the second consists of P3HT, PTB7-Th, PCBM, and oIDTBr (). (b) Side view of the automated platform for ink formulation, coating, and characterization. (c) Experimental workflow with the two approaches adopted in this study: conventional high-throughput experimentation <span class="italic">via</span> grid and the self-driving approach with the ChemOS software package. Reproduced with permission<a title="Select to navigate to references" href="#cit138"><sup><span class="sup_ref">138</span></sup></a> Copyright,  2020 The Authors. Published by WILEY-VCH Verlag GmbH &amp; Co. KGaA, Weinheim.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">By commonly used fingerprints like MACCS<a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a> and Pubchem,<a title="Select to navigate to references" href="#cit139"><sup><span class="sup_ref">139</span></sup></a> we cannot correlate each bit of fingerprint with a specific substructure. To overcome this limitation of fingerprints, Sun <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> designed the La FREMD fingerprint for expressing 6180 different fragments (bits). Integration of the La FREMD fingerprint and ML allows rapid design and screening of donor materials for high-efficiency OSCs. This framework is used on a dataset of 1758 experimentally reported donor materials. Using the RF model, 15 most important fragments were identified for high PCE and were compared based on the newly described variable frequency difference (FD). As shown in <a title="Select to navigate to figure" href="#imgfig19">Fig. 19</a>, after identifying important building blocks, a library of 18<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>960 small molecule donors is created with their combination. For rapid screening from this library, four ML models (ANN, GBRT, RF, and SVR) were trained on the donor material database (1758) with daylight fingerprint as the input descriptor. The GBRT model gave the lowest RMSE (2.05) and 6337 molecules predicted PCE greater than 8%. 20 promising molecules were selected based on molecular symmetry with Y6 as the acceptor. For this purpose a new dataset of 44 OSCs with the Y6 or Y6 derivative was used to train the GBRT model for PCE prediction and test it on 20 promising molecules selected earlier. 5 out of 20 promising molecules predicted PCE greater than 15%.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig19"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f19_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f19.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f19.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 19 </b> <span id="fig19"><span class="graphic_title">Scheme of the AI design framework for developing high-performance OPV donor materials. Reproduced with permission<a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> Copyright  2021 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Hutchinson <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit104"><sup><span class="sup_ref">104</span></sup></a> created a model for NFA based OSCs using 47 input descriptors to predict FF, <span class="italic">J</span><small><sub>sc</sub></small>, <span class="italic">V</span><small><sub>oc</sub></small>, and PCE. To speed up the calculation, they used sTD-DFT<a title="Select to navigate to references" href="#cit140"><sup><span class="sup_ref">140</span></sup></a> which is 23 times faster than TD-DFT. A dataset of 84 D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A combinations was used for modeling, out of which 6 are polymer donors and 66 are NFAs and 2 are FAs. The results reveal that the sTD-DFT based model can predict PCE with RMSE = 1.60  0.04%.</p>
        <p class="otherpara">A methodology is required to rapidly identify optimal device configurations that optimize net energy output while minimizing the environmental impact of OPVs. Kettle <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a> created a dataset of 1580 OSC devices (20112020) with reported PCE and stability data. The objective is to find optimal OSC materials and device architectures that are environmentally friendly. For each device, the net energy (<span class="italic">E</span><small><sub>Net</sub></small>) is also calculated, which is a function of time taken to degrade to 80% of its stabilized initial output power (<span class="italic">T</span><small><sub>80</sub></small>), time taken to drop to 80% of initial power at <span class="italic">t</span> = 0 (<span class="italic">T</span><small><sub>S80</sub></small>), and embodied energy (<span class="italic">E</span><small><sub>Emb</sub></small>). The SMOreg model is used to analyze <span class="italic">E</span><small><sub>Net</sub></small> by using structural components of each device. GA clustering was used to determine the optimal material sets for OPV ENet output. This study uses the GenClust++ clustering method. By using the genetic search method clustering, eco-friendly device configurations are obtained. The complete workflow is depicted in <a title="Select to navigate to figure" href="#imgfig20">Fig. 20</a>. The active layer materials DRCN7T, DR3TSBDT, ZnPc, PDPP4T-2F, PCDTBT (donors), and IT-4F, C61 (acceptors) show promise for delivering positive net energy. The authors concluded that trained and validated models could accurately predict the efficiency, stability, and embodied energy of an OPV.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig20"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f20_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f20.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f20.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 20 </b> <span id="fig20"><span class="graphic_title">Schematic of how material and energy costs are acquired and stages of analysis using ML GAs. Reproduced with permission<a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a> Copyright  2022 American Chemical Society.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">OPV performance can be improved in a variety of ways by better understanding how smart molecular design can increase light harvesting on its own. The device's macroscopic short-circuit current density is closely tied to light absorption, which is the first stage in generating an electrical charge. High absorption should, in principle, lead to strong emission due to the reciprocity relationship between absorption and emission, lowering nonradiative energy losses and benefiting open-circuit voltage. For identifying structureabsorption relationship, Nelson <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit141"><sup><span class="sup_ref">141</span></sup></a> investigated 500 unique organic molecules with DFT and TDDFT to study absorption strength and unravel structural features that lead to high absorption strength. The authors found that the absorption strength calculated experimentally and by TDDFT is in good agreement for NFAs and fullerene and suggested the use of TDDFT calculations for further modeling. From DFT optimized geometries, 6000 molecular descriptors were created, and the highest correlation with absorption strength was given by <span class="italic"></span><small><sub>d,max</sub></small> (experimentally measured maximum molar extinction coefficient), <span class="italic"></span><small><sub>1,p</sub></small> (size of the molecule in the direction of maximum atomic polarizability), and C2SP2 (number of sp<small><sup>2</sup></small> hybridized carbon atoms bound with two other carbons). TDDFT and ML studies found that molecular linearity, planarity, polarizability and number of -conjugated carbon atoms also correlate strongly with absorption strength. Moreover, the authors created RF model using LOOCV to predict ed, max by using semiempirical extended tight-binding (xTB) Hamiltonians which is 3000 times faster than using DFT. The workflow of this study along with prediction results is represented in <a title="Select to navigate to figure" href="#imgfig21">Fig. 21</a>.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig21"><a href="/image/article/2022/TC/d2tc03276g/d2tc03276g-f21_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2tc03276g-f21.tif" src="/image/article/2022/TC/d2tc03276g/d2tc03276g-f21.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 21 </b> <span id="fig21"><span class="graphic_title">(a) ML workflow used in this work to draw ed,max predictions. A RF model is trained on TDDFT data and interpolated (validated) on xTB geometries, including also their corresponding molecular descriptors. To improve the accuracy of the model, energy levels obtained using the GFN2-xTB Hamiltonian require calibration with TDDFT values. (b) Leave-one-out interpolation of the resulting RF model using three input molecular descriptors (including calibrated energy levels) and a 64-bit Morgan fingerprint vector. Reproduced with permission<a title="Select to navigate to references" href="#cit141"><sup><span class="sup_ref">141</span></sup></a> Copyright, 2022, Royal Society of Chemistry (RSC).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
    
    
      
      <h2 id="sect1396"><span class="a_heading">4. Future scope and challenges</span></h2>
      <span>With the growing interest in ML applications for materials science, significant research is also conducted on OSCs using ML on available experimental data. With continuous development in algorithms and enhancement of computational power, we may anticipate the trend of ML-OSC research to continue.</span>
      <p class="otherpara">As the PCE has achieved acceptable levels in recent years, the focus has switched from efficiency improvement to stability, which remains an obstacle to the widespread use of the technology. There is still a lot we do not know about how environmental stress variables like temperature, humidity, and oxygen affect the degrading behavior of materials and, by extension, the durability of solar cells under the conditions they will be used or stored.<a title="Select to navigate to references" href="#cit142"><sup><span class="sup_ref">142</span></sup></a></p>
      <p class="otherpara">Various problems faced by researchers in this direction are discussed here.</p>
      
        
        <h3 id="sect1403"><span class="b_heading">4.1. Selection of a suitable training set</span></h3>
        <span>The same material is being used in the literature with different names. The solution is to use the canonical SMILES string after gathering the dataset.</span>
      
      
        
        <h3 id="sect1407"><span class="b_heading">4.2. Dataset size</span></h3>
        <span>OSC datasets are pretty small, and k-fold cross-validation gives poor results. The solution is to use leave-one-out cross-validation for small datasets.</span>
      
      
        
        <h3 id="sect1411"><span class="b_heading">4.3. Extrapolation</span></h3>
        <span>For OSCs, the available dataset for training ML models is small and chemical structures are also large and complex. Thus, ML models often struggle in extrapolation tasks and give poor results on the dataset that is never seen during the training. The solution is to use a scaffold-based train-test split or leave one group out cross-validation.<a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> If large datasets are available, feature engineering can improve the extrapolation ability of ML models.</span>
      
      
        
        <h3 id="sect1416"><span class="b_heading">4.4. Simple input descriptors</span></h3>
        <span>To quickly make predictions for previously untested materials or combinations after saving a machine learning model, it is important to have quick access to the descriptors used in the model. Input descriptors should be easy to calculate and should be sensitive to the target variable. For example, simple input descriptors such as Number of Nitrogen and Oxygen or Number of valence electrons the molecule has could be easily obtained using RDKit. The problem can be solved by employing descriptors that are computed immediately from a chemical structure or SMILES string.</span>
      
      
        
        <h3 id="sect1420"><span class="b_heading">4.5. Inclusion of failure data</span></h3>
        <span>During training, the model learns from training data, and usually failure data are not reported in publications and hence are not included in experimental datasets. Because of unavailability of the failure dataset, models predictions are usually overestimated and highly deviated. With the inclusion of the failure dataset, the model can understand device physics in a much better way and avoid overestimated prediction.</span>
      
      
        
        <h3 id="sect1424"><span class="b_heading">4.6. Active learning techniques</span></h3>
        <span>Active learning techniques can guide OSC experimentalists to find most suitable active materials from vast search space in a few number of experiments. Active learning could be also used for finding the most optimal device fabrication parameters such as ratio of donor and acceptor in the BHJ active layer, thickness of the active layer, annealing temperature of the active layer, <span class="italic">etc.</span></span>
      
      
        
        <h3 id="sect1429"><span class="b_heading">4.7. Experimental validation</span></h3>
        <span>To verify the results predicted by ML models, experimental validation is required. Various studies have demonstrated that ML models predict photovoltaic properties on the test set with high accuracy. But when predicted on novel materials, the results deviate by a considerable margin. The reason for this is that a number of experimental parameters such as D<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>:<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>A ratio, spin speed, solvent, processing temperature, <span class="italic">etc.</span>, are not usually taken as input while training.</span>
        <p class="otherpara">The research of ML in the field of OSCs is still in its early stage. While many models and/or algorithms have been developed, very few new materials, processes or structures are demonstrated successful. With the increasing volume of data accumulated and research experience, the accuracy of models is anticipated to be improved substantially. We foresee that AI can truly aid scientists in gathering knowledge of various domains and to develop promising materials and experimental parameters for OSCs in the near future.</p>
      
    
    
      
      <h2 id="sect1435"><span class="a_heading">5. Conclusion and perspectives</span></h2>
      <span>Tremendous efforts have been devoted to developing ML models, which can learn from the existing performance data of OSCs, to help the researchers to predict and explore the properties of new organic materials and devices. The accuracy from the ML prediction has been improved substantially in many respects of OSCs, including efficiency and stability. ML models now can also assist and simplify the optimization of experimental parameters for device fabrication.</span>
      <p class="otherpara">On the other hand, as the data size of OSCs is relatively small, it is very critical to develop ML algorithms and/or design workflows that can be trained from a small-sized dataset. Further, new descriptors and structural fingerprints can improve models' accuracy and help develop new materials. Although many reports of ML models demonstrate extrapolation ability, accurate predictions are still very challenging. Such prediction ability is more important because the results can indeed save many research resources and accelerate the development progress. Most current screening functions of the ML models can only handle chemical structures built from known blocks and units. Finally, the automation from material design, synthesis work, experimental data collection to result analysis is worthy of attention. With a complete research automated loop, we can explore much larger materials space in a limited time. The introduction of ML models and AI to the development of OSCs is still in its infancy. We witnessed the beginning of this field, and only very limited materials are developed using the ML models currently. We anticipate that more organic materials will be synthesized, and the performance of OSCs can eventually be pushed to the limit with the assistance from ML models.</p>
    
    
      <h2 id="sect1440"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <span id="sect1444"><h2 id="sect1443"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">O. Almora, D. Baran, G. C. Bazan, C. Berger, C. I. Cabrera, K. R. Catchpole, S. Erten-Ela, F. Guo, J. Hauch, A. W. Y. Ho-Baillie, T. J. Jacobsson, R. A. J. Janssen, T. Kirchartz, N. Kopidakis, Y. Li, M. A. Loi, R. R. Lunt, X. Mathew, M. D. McGehee, J. Min, D. B. Mitzi, M. K. Nazeeruddin, J. Nelson, A. F. Nogueira, U. W. Paetzold, N. Park, B. P. Rand, U. Rau, H. J. Snaith, E. Unger, L. Vaillant-Roca, H. Yip and C. J. Brabec, Device Performance of Emerging Photovoltaic Materials (Version 2), <span class="italic">Adv. Energy Mater.</span>, 2021, <span class="bold">11</span>, 2102526<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.202102526" title="DOI Link to resource 10.1002/aenm.202102526">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXisVKjs7zN" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/1_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit2">A. Karki, A. J. Gillett, R. H. Friend and T. Nguyen, The Path to 20% Power Conversion Efficiencies in Nonfullerene Acceptor Organic Solar Cells, <span class="italic">Adv. Energy Mater.</span>, 2021, <span class="bold">11</span>, 2003441<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.202003441" title="DOI Link to resource 10.1002/aenm.202003441">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXis1WrsrvE" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/2_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit3">L. Duan and A. Uddin, Progress in Stability of Organic Solar Cells, <span class="italic">Adv. Sci.</span>, 2020, <span class="bold">7</span>, 1903259<a target="_blank" class="DOILink" href="https://doi.org/10.1002/advs.201903259" title="DOI Link to resource 10.1002/advs.201903259">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvFart7rK" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32537401%5Buid%5D" title="PubMed Link to resource 32537401">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/3_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit4">J. Wu, M. Gao, Y. Chai, P. Liu, B. Zhang, J. Liu and L. Ye, Towards a bright future: The versatile applications of organic solar cells, <span class="italic">Mater. Rep. Energy</span>, 2021, <span class="bold">1</span>, 100062<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mater.%20Rep.%20Energy%5BJour%5D%20AND%201%5Bvolume%5D%20AND%20100062%5Bpage%5D%20and%202021%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/4_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit5">K. Khandelwal, S. Biswas, A. Mishra and G. D. Sharma, Semitransparent organic solar cells: From molecular design to structure-performance relationships, <span class="italic">J. Mater. Chem. C</span>, 2022, <span class="bold">10</span>, 1343<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1TC04569E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1TC04569E">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/5_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit6">S. Lee, D. Jeong, C. Kim, C. Lee, H. Kang, H. Y. Woo and B. J. Kim, Eco-friendly polymer solar cells: Advances in green-solvent processing and material design, <span class="italic">ACS Nano</span>, 2020, <span class="bold">14</span>, 1449314527<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.0c07488" title="DOI Link to resource 10.1021/acsnano.0c07488">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33103903%5Buid%5D" title="PubMed Link to resource 33103903">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/6_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit7">Y. Cui, L. Hong and J. Hou, Organic Photovoltaic Cells for Indoor Applications: Opportunities and Challenges, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2020, <span class="bold">12</span>, 3881538828<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.0c10444" title="DOI Link to resource 10.1021/acsami.0c10444">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsFWgt73L" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32805933%5Buid%5D" title="PubMed Link to resource 32805933">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/7_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit8">E. Ravishankar, R. E. Booth, C. Saravitz, H. Sederoff, H. W. Ade and B. T. OConnor, Achieving Net Zero Energy Greenhouses by Integrating Semitransparent Organic Solar Cells, <span class="italic">Joule</span>, 2020, <span class="bold">4</span>, 490506<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2019.12.018" title="DOI Link to resource 10.1016/j.joule.2019.12.018">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXnslemtrg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/8_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit9">J. Hou, O. Inganas, R. H. Friend and F. Gao, Organic solar cells based on non-fullerene acceptors, <span class="italic">Nat. Mater.</span>, 2018, <span class="bold">17</span>, 119128<a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmat5063" title="DOI Link to resource 10.1038/nmat5063">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGqsbk%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29358765%5Buid%5D" title="PubMed Link to resource 29358765">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/9_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit10">A. Wadsworth, M. Moser, A. Marks, M. S. Little, N. Gasparini, C. J. Brabec, D. Baran and I. McCulloch, Critical review of the molecular design progress in non-fullerene electron acceptors towards commercially viable organic solar cells, <span class="italic">Chem. Soc. Rev.</span>, 2019, <span class="bold">48</span>, 15961625<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7CS00892A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7CS00892A">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/10_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit11">F. Zhao, H. Zhang, R. Zhang, J. Yuan, D. He, Y. Zou and F. Gao, Emerging Approaches in Enhancing the Efficiency and Stability in Non-Fullerene Organic Solar Cells, <span class="italic">Adv. Energy Mater.</span>, 2020, <span class="bold">10</span>, 2002746<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.202002746" title="DOI Link to resource 10.1002/aenm.202002746">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXit1KhsbjJ" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/11_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit12">A. Armin, W. Li, O. J. Sandberg, Z. Xiao, L. Ding, J. Nelson, D. Neher, K. Vandewal, S. Shoaee, T. Wang, H. Ade, T. Heumller, C. Brabec and P. Meredith, A History and Perspective of Non-Fullerene Electron Acceptors for Organic Solar Cells, <span class="italic">Adv. Energy Mater.</span>, 2021, <span class="bold">11</span>, 142<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Energy%20Mater.%5BJour%5D%20AND%2011%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202021%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/12_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit13">D. Luo, W. Jang, D. D. Babu, M. S. Kim, D. H. Wang and A. K. K. Kyaw, Recent progress in organic solar cells based on non-fullerene acceptors: materials to devices, <span class="italic">J. Mater. Chem. A</span>, 2022, <span class="bold">10</span>, 32553295<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1TA10707K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1TA10707K">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/13_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit14">J. Yuan, Y. Zhang, L. Zhou, G. Zhang, H. L. Yip, T. K. Lau, X. Lu, C. Zhu, H. Peng, P. A. Johnson, M. Leclerc, Y. Cao, J. Ulanski, Y. Li and Y. Zou, Single-Junction Organic Solar Cell with over 15% Efficiency Using Fused-Ring Acceptor with Electron-Deficient Core, <span class="italic">Joule</span>, 2019, <span class="bold">3</span>, 11401151<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2019.01.004" title="DOI Link to resource 10.1016/j.joule.2019.01.004">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXnvVKnsL8%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/14_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit15">Q. Wei, W. Liu, M. Leclerc, J. Yuan, H. Chen and Y. Zou, A-DAD-A non-fullerene acceptors for high-performance organic solar cells, <span class="italic">Sci. China: Chem.</span>, 2020, <span class="bold">63</span>, 13521366<a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11426-020-9799-4" title="DOI Link to resource 10.1007/s11426-020-9799-4">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsVGqsb%252FK" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/15_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit16">B. Lu, J. Wang, Z. Zhang, J. Wang, X. Yuan, Y. Ding, Y. Wang and Y. Yao, Recent progress of Y-series electron acceptors for organic solar cells, <span class="italic">Nano Sel.</span>, 2021, <span class="bold">2</span>, 20292039<a target="_blank" class="DOILink" href="https://doi.org/10.1002/nano.202100036" title="DOI Link to resource 10.1002/nano.202100036">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38Xmt1SrsrY%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/16_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit17">Y. Cui, Y. Xu, H. Yao, P. Bi, L. Hong, J. Zhang, Y. Zu, T. Zhang, J. Qin, J. Ren, Z. Chen, C. He, X. Hao, Z. Wei and J. Hou, Single-Junction Organic Photovoltaic Cell with 19% Efficiency, <span class="italic">Adv. Mater.</span>, 2021, <span class="bold">33</span>, 2102420<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.202102420" title="DOI Link to resource 10.1002/adma.202102420">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvFGms73L" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34464466%5Buid%5D" title="PubMed Link to resource 34464466">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/17_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit18">L. Zhu, M. Zhang, J. Xu, C. Li, J. Yan, G. Zhou, W. Zhong, T. Hao, J. Song, X. Xue, Z. Zhou, R. Zeng, H. Zhu, C. C. Chen, R. C. I. MacKenzie, Y. Zou, J. Nelson, Y. Zhang, Y. Sun and F. Liu, Single-junction organic solar cells with over 19% efficiency enabled by a refined double-fibril network morphology, <span class="italic">Nat. Mater.</span>, 2022, <span class="bold">21</span>, 656663<a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41563-022-01244-y" title="DOI Link to resource 10.1038/s41563-022-01244-y">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38Xht1Wis7zL" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35513501%5Buid%5D" title="PubMed Link to resource 35513501">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/18_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit19">C. Li, J. Zhou, J. Song, J. Xu, H. Zhang, X. Zhang, J. Guo, L. Zhu, D. Wei, G. Han, J. Min, Y. Zhang, Z. Xie, Y. Yi, H. Yan, F. Gao, F. Liu and Y. Sun, Non-fullerene acceptors with branched side chains and improved molecular packing to exceed 18% efficiency in organic solar cells, <span class="italic">Nat. Energy</span>, 2021, <span class="bold">6</span>, 605613<a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41560-021-00820-x" title="DOI Link to resource 10.1038/s41560-021-00820-x">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvFShtbzO" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/19_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit20">Y. Wei, Z. Chen, G. Lu, N. Yu, C. Li, J. Gao, X. Gu, X. Hao, G. Lu, Z. Tang, J. Zhang, Z. Wei, X. Zhang and H. Huang, Binary Organic Solar Cells Breaking 19% via Manipulating Vertical Component Distribution, <span class="italic">Adv. Mater.</span>, 2022, <span class="bold">34</span>, 2204718<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.202204718" title="DOI Link to resource 10.1002/adma.202204718">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhvVChurfM" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35747988%5Buid%5D" title="PubMed Link to resource 35747988">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/20_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit21">W. Feng, S. Wu, H. Chen, L. Meng, F. Huang, H. Liang, J. Zhang, Z. Wei, X. Wan, C. Li, Z. Yao and Y. Chen, Tuning Morphology of Active Layer by using a Wide Bandgap Oligomer-Like Donor Enables Organic Solar Cells with Over 18% Efficiency, <span class="italic">Adv. Energy Mater.</span>, 2022, <span class="bold">12</span>, 2104060<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.202104060" title="DOI Link to resource 10.1002/aenm.202104060">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XntVahur0%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/21_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit22">C. He, Y. Pan, Y. Ouyang, Q. Shen, Y. Gao, K. Yan, J. Fang, Y. Chen, C.-Q. Ma, J. Min, C. Zhang, L. Zuo and H. Chen, Manipulating the D:A interfacial energetics and intermolecular packing for 19.2% efficiency organic photovoltaics, <span class="italic">Energy Environ. Sci.</span>, 2022, <span class="bold">15</span>, 25372544<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D2EE00595F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D2EE00595F">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/22_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit23">L. Zhan, S. Li, Y. Li, R. Sun, J. Min, Z. Bi, W. Ma, Z. Chen, G. Zhou, H. Zhu, M. Shi, L. Zuo and H. Chen, Desired open-circuit voltage increase enables efficiencies approaching 19% in symmetric-asymmetric molecule ternary organic photovoltaics, <span class="italic">Joule</span>, 2022, <span class="bold">6</span>, 662675<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2022.02.001" title="DOI Link to resource 10.1016/j.joule.2022.02.001">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38Xlt12gsbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/23_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit24">Z. Zheng, J. Wang, P. Bi, J. Ren, Y. Wang, Y. Yang, X. Liu, S. Zhang and J. Hou, Tandem Organic Solar Cell with 20.2% Efficiency, <span class="italic">Joule</span>, 2022, <span class="bold">6</span>, 171184<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2021.12.017" title="DOI Link to resource 10.1016/j.joule.2021.12.017">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38Xlt1ymsrg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/24_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit25">Y. Lin, Q. He, F. Zhao, L. Huo, J. Mai, X. Lu, C. J. Su, T. Li, J. Wang, J. Zhu, Y. Sun, C. Wang and X. Zhan, A Facile Planar Fused-Ring Electron Acceptor for As-Cast Polymer Solar Cells with 8.71% Efficiency, <span class="italic">J. Am. Chem. Soc.</span>, 2016, <span class="bold">138</span>, 29732976<a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.6b00853" title="DOI Link to resource 10.1021/jacs.6b00853">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XjtVCis74%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26909887%5Buid%5D" title="PubMed Link to resource 26909887">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/25_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit26">Y. Lin, F. Zhao, Q. He, L. Huo, Y. Wu, T. C. Parker, W. Ma, Y. Sun, C. Wang, D. Zhu, A. J. Heeger, S. R. Marder and X. Zhan, High-Performance Electron Acceptor with Thienyl Side Chains for Organic Photovoltaics, <span class="italic">J. Am. Chem. Soc.</span>, 2016, <span class="bold">138</span>, 49554961<a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.6b02004" title="DOI Link to resource 10.1021/jacs.6b02004">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XkvVahsb0%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27015115%5Buid%5D" title="PubMed Link to resource 27015115">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/26_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit27">S. Dai, F. Zhao, Q. Zhang, T. K. Lau, T. Li, K. Liu, Q. Ling, C. Wang, X. Lu, W. You and X. Zhan, Fused nonacyclic electron acceptors for efficient polymer solar cells, <span class="italic">J. Am. Chem. Soc.</span>, 2017, <span class="bold">139</span>, 13361343<a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.6b12755" title="DOI Link to resource 10.1021/jacs.6b12755">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXkslWqsg%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28059503%5Buid%5D" title="PubMed Link to resource 28059503">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/27_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit28">J. Wang and X. Zhan, Fused-Ring Electron Acceptors for Photovoltaics and beyond, <span class="italic">Acc. Chem. Res.</span>, 2021, <span class="bold">54</span>, 132143<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.accounts.0c00575" title="DOI Link to resource 10.1021/acs.accounts.0c00575">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXisVyrurvE" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33284599%5Buid%5D" title="PubMed Link to resource 33284599">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/28_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit29">A. Harillo-Baos, X. Rodrguez-Martnez and M. Campoy-Quiles, Efficient Exploration of the Composition Space in Ternary Organic Solar Cells by Combining High-Throughput Material Libraries and Hyperspectral Imaging, <span class="italic">Adv. Energy Mater.</span>, 2020, <span class="bold">10</span>, 1902417<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201902417" title="DOI Link to resource 10.1002/aenm.201902417">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/29_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit30">X. Rodrguez-Martnez, S. Sevim, X. Xu, C. Franco, P. Pamies-Puig, L. Crcoles-Guija, R. Rodriguez-Trujillo, F. J. del Campo, D. Rodriguez San Miguel, A. J. deMello, S. Pan, D. B. Amabilino, O. Ingans, J. Puigmart-Luis and M. Campoy-Quiles, Microfluidic-Assisted Blade Coating of Compositional Libraries for Combinatorial Applications: The Case of Organic Photovoltaics, <span class="italic">Adv. Energy Mater.</span>, 2020, <span class="bold">10</span>, 2001308<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.202001308" title="DOI Link to resource 10.1002/aenm.202001308">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/30_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit31">R. Po, A. Bernardi, A. Calabrese, C. Carbonera, G. Corso and A. Pellegrino, From lab to fab: how must the polymer solar cell materials design change?  an industrial perspective, <span class="italic">Energy Environ. Sci.</span>, 2014, <span class="bold">7</span>, 925943<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3EE43460E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3EE43460E">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/31_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit32">J. E. Carl, M. Helgesen, O. Hagemann, M. Hsel, I. M. Heckler, E. Bundgaard, S. A. Gevorgyan, R. R. Sndergaard, M. Jrgensen, R. Garca-Valverde, S. Chaouki-Almagro, J. A. Villarejo and F. C. Krebs, Overcoming the Scaling Lag for Polymer Solar Cells, <span class="italic">Joule</span>, 2017, <span class="bold">1</span>, 274289<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2017.08.002" title="DOI Link to resource 10.1016/j.joule.2017.08.002">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/32_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit33">P. Meredith and A. Armin, Scaling of next generation solution processed organic and perovskite solar cells, <span class="italic">Nat. Commun.</span>, 2018, <span class="bold">9</span>, 5261<a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-018-05514-9" title="DOI Link to resource 10.1038/s41467-018-05514-9">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXisVygs7%252FO" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30532028%5Buid%5D" title="PubMed Link to resource 30532028">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/33_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit34">A. S. Gertsen, M. F. Castro, R. R. Sndergaard and J. W. Andreasen, Scalable fabrication of organic solar cells based on non-fullerene acceptors, <span class="italic">Flexible Printed Electron.</span>, 2020, <span class="bold">5</span>, 014004<a target="_blank" class="DOILink" href="https://doi.org/10.1088/2058-8585/ab5f57" title="DOI Link to resource 10.1088/2058-8585/ab5f57">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhtFGrsbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/34_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit35">Y. T. Fu, C. Risko and J. L. Brdas, Intermixing at the Pentacene-Fullerene Bilayer Interface: A Molecular Dynamics
Study, <span class="italic">Adv. Mater.</span>, 2013, <span class="bold">25</span>, 878882<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201203412" title="DOI Link to resource 10.1002/adma.201203412">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhsleisrnF" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23184531%5Buid%5D" title="PubMed Link to resource 23184531">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/35_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit36">Y. Shin, J. Liu, J. J. Quigley, H. Luo and X. Lin, Combinatorial design of copolymer donor materials for bulk heterojunction solar cells, <span class="italic">ACS Nano</span>, 2014, <span class="bold">8</span>, 60896096<a target="_blank" class="DOILink" href="https://doi.org/10.1021/nn5014066" title="DOI Link to resource 10.1021/nn5014066">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXotVelurg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24835665%5Buid%5D" title="PubMed Link to resource 24835665">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/36_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit37">Y. Imamura, M. Suganuma and M. Hada, Computational Study on the Search for Non-Fullerene Acceptors, Examination of Interface Geometry, and Investigation of Electron Transfer, <span class="italic">J. Phys. Chem. C</span>, 2019, <span class="bold">123</span>, 1767817685<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.9b02933" title="DOI Link to resource 10.1021/acs.jpcc.9b02933">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXht1CqsbrE" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/37_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit38">C. Chen, Y. Zuo, W. Ye, X. Li, Z. Deng and S. P. Ong, A Critical Review of Machine Learning of Energy Materials, <span class="italic">Adv. Energy Mater.</span>, 2020, <span class="bold">10</span>, 1903242<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201903242" title="DOI Link to resource 10.1002/aenm.201903242">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhslens7o%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/38_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit39">A. Mahmood and J. L. Wang, Machine learning for high performance organic solar cells: current scenario and future prospects, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 90105<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE02838J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE02838J">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/39_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit40">X. Rodrguez-Martnez, E. Pascual-San-Jos and M. Campoy-Quiles, Accelerating organic solar cell material's discovery: high-throughput screening and big data, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 33013322<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1EE00559F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1EE00559F">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/40_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit41">Z. Zhao, Y. Geng, A. Troisi and H. Ma, Performance Prediction and Experimental Optimization Assisted by Machine Learning for Organic Photovoltaics, <span class="italic">Adv. Intell. Syst.</span>, 2022, <span class="bold">4</span>, 2100261<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aisy.202100261" title="DOI Link to resource 10.1002/aisy.202100261">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/41_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit42">T. Lu, M. Li, W. Lu and T. Y. Zhang, Recent progress in the data-driven discovery of novel photovoltaic materials, <span class="italic">J. Mater. Inf.</span>, 2022, <span class="bold">2</span>, 7<a target="_blank" class="DOILink" href="https://doi.org/10.20517/jmi.2022.07" title="DOI Link to resource 10.20517/jmi.2022.07">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/42_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit43">A. Mahmood, A. Irfan and J.-L. Wang, Machine Learning for Organic Photovoltaic Polymers: A Minireview, <span class="italic">Chin. J. Polym. Sci.</span>, 2022, <span class="bold">40</span>, 870876<a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10118-022-2782-5" title="DOI Link to resource 10.1007/s10118-022-2782-5">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhslSltr3I" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/43_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit44">X. Rodrguez-Martnez, E. Pascual-San-Jos, Z. Fei, M. Heeney, R. Guimer and M. Campoy-Quiles, Predicting the photocurrentcomposition dependence in organic solar cells, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 986994<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0EE02958K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0EE02958K">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/44_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit45">M. C. Scharber, D. Mhlbacher, M. Koppe, P. Denk, C. Waldauf, A. J. Heeger and C. J. Brabec, Design Rules for Donors in Bulk-Heterojunction Solar CellsTowards 10% Energy-Conversion Efficiency, <span class="italic">Adv. Mater.</span>, 2006, <span class="bold">18</span>, 789794<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.200501717" title="DOI Link to resource 10.1002/adma.200501717">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XjsV2isrw%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/45_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit46">Z. Zhao, M. del Cueto, Y. Geng and A. Troisi, Effect of Increasing the Descriptor Set on Machine Learning Prediction of Small Molecule-Based Organic Solar Cells, <span class="italic">Chem. Mater.</span>, 2020, <span class="bold">32</span>, 77777787<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.0c02325" title="DOI Link to resource 10.1021/acs.chemmater.0c02325">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Kiu7bO" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/46_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit47">B. Cao, L. A. Adutwum, A. O. Oliynyk, E. J. Luber, B. C. Olsen, A. Mar and J. M. Buriak, How to optimize materials and devices via design of experiments and machine learning: Demonstration using organic photovoltaics, <span class="italic">ACS Nano</span>, 2018, <span class="bold">12</span>, 74347444<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.8b04726" title="DOI Link to resource 10.1021/acsnano.8b04726">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlCksrjO" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30027732%5Buid%5D" title="PubMed Link to resource 30027732">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/47_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit48">X. Du, L. Ler, T. Heumueller, J. Wagner, C. Berger, T. Osterrieder, J. Wortmann, S. Langner, U. Vongsaysy, M. Bertrand, N. Li, T. Stubhan, J. Hauch and C. J. Brabec, Elucidating the Full Potential of OPV Materials Utilizing a High-Throughput Robot-Based Platform and Machine Learning, <span class="italic">Joule</span>, 2021, <span class="bold">5</span>, 495506<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2020.12.013" title="DOI Link to resource 10.1016/j.joule.2020.12.013">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXitlSiurs%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/48_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit49">N. G. An, J. Y. Kim and D. Vak, Machine learning-assisted development of organic photovoltaics via high-throughput in situ formulation, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 34383446<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1EE00641J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1EE00641J">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/49_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit50">H. Sahu, W. Rao, A. Troisi and H. Ma, Toward Predicting Efficiency of Organic Solar Cells via Machine Learning and Improved Descriptors, <span class="italic">Adv. Energy Mater.</span>, 2018, <span class="bold">8</span>, 1801032<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201801032" title="DOI Link to resource 10.1002/aenm.201801032">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/50_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit51">Y. Wen, Y. Liu, B. Yan, T. Gaudin, J. Ma and H. Ma, Simultaneous Optimization of Donor/Acceptor Pairs and Device Specifications for Nonfullerene Organic Solar Cells Using a QSPR Model with Morphological Descriptors, <span class="italic">J. Phys. Chem. Lett.</span>, 2021, <span class="bold">12</span>, 49804986<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.1c01099" title="DOI Link to resource 10.1021/acs.jpclett.1c01099">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhtFWqtbfE" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34015223%5Buid%5D" title="PubMed Link to resource 34015223">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/51_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit52">S. Kar, N. Sizochenko, L. Ahmed, V. S. Batista and J. Leszczynski, Quantitative structure-property relationship model leading to virtual screening of fullerene derivatives: Exploring structural attributes critical for photoconversion efficiency of polymer solar cell acceptors, <span class="italic">Nano Energy</span>, 2016, <span class="bold">26</span>, 677691<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.nanoen.2016.06.011" title="DOI Link to resource 10.1016/j.nanoen.2016.06.011">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xht12ns7bF" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/52_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit53">E. Abbasi Jannat Abadi, H. Sahu, S. M. Javadpour and M. Goharimanesh, Interpretable machine learning for developing high-performance organic solar cells, Mater, <span class="italic">Today Energy</span>, 2022, <span class="bold">25</span>, 100969<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mtener.2022.100969" title="DOI Link to resource 10.1016/j.mtener.2022.100969">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhtlOjurvF" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/53_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit54">J. Munshi, W. Chen, T. Chien and G. Balasubramanian, Transfer Learned Designer Polymers for Organic Solar Cells, <span class="italic">J. Chem. Inf. Model.</span>, 2021, <span class="bold">61</span>, 134142<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.0c01157" title="DOI Link to resource 10.1021/acs.jcim.0c01157">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXkvVCntA%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33410685%5Buid%5D" title="PubMed Link to resource 33410685">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/54_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit55">S. P. Peng and Y. Zhao, Convolutional Neural Networks for the Design and Analysis of Non-Fullerene Acceptors, <span class="italic">J. Chem. Inf. Model.</span>, 2019, <span class="bold">59</span>, 49935001<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.9b00732" title="DOI Link to resource 10.1021/acs.jcim.9b00732">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitFagurvE" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31710212%5Buid%5D" title="PubMed Link to resource 31710212">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/55_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit56">S. Langner, F. Hse, J. D. Perea, T. Stubhan, J. Hauch, L. M. Roch, T. Heumueller, A. Aspuru-Guzik and C. J. Brabec, Beyond Ternary OPV: High-Throughput Experimentation and Self-Driving Laboratories Optimize Multicomponent Systems, <span class="italic">Adv. Mater.</span>, 2020, <span class="bold">32</span>, 1907801<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201907801" title="DOI Link to resource 10.1002/adma.201907801">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXivFygsro%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32049386%5Buid%5D" title="PubMed Link to resource 32049386">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/56_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit57">R. Olivares-Amaya, C. Amador-Bedolla, J. Hachmann, S. Atahan-Evrenk, R. S. Snchez-Carrera, L. Vogt and A. Aspuru-Guzik, Accelerated computational discovery of high-performance materials for organic photovoltaics by means of cheminformatics, <span class="italic">Energy Environ. Sci.</span>, 2011, <span class="bold">4</span>, 48494861<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1EE02056K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1EE02056K">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/57_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit58">J. Hachmann, R. Olivares-Amaya, S. Atahan-Evrenk, C. Amador-Bedolla, R. S. Snchez-Carrera, A. Gold-Parker, L. Vogt, A. M. Brockway and A. Aspuru-Guzik, The harvard clean energy project: Large-scale computational screening and design of organic photovoltaics on the world community grid, <span class="italic">J. Phys. Chem. Lett.</span>, 2011, <span class="bold">2</span>, 22412251<a target="_blank" class="DOILink" href="https://doi.org/10.1021/jz200866s" title="DOI Link to resource 10.1021/jz200866s">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKht7rK" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/58_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit59">J. Hachmann, R. Olivares-Amaya, A. Jinich, A. L. Appleton, M. A. Blood-Forsythe, L. R. Seress, C. Romn-Salgado, K. Trepte, S. Atahan-Evrenk, S. Er, S. Shrestha, R. Mondal, A. Sokolov, Z. Bao and A. Aspuru-Guzik, Lead candidates for high-performance organic photovoltaics from high-throughput quantum chemistry-the Harvard Clean Energy Project, <span class="italic">Energy Environ. Sci.</span>, 2014, <span class="bold">7</span>, 698704<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3EE42756K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3EE42756K">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/59_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit60">E. O. Pyzer-Knapp, K. Li and A. Aspuru-Guzik, Learning from the Harvard Clean Energy Project: The Use of Neural Networks to Accelerate Materials Discovery, <span class="italic">Adv. Funct. Mater.</span>, 2015, <span class="bold">25</span>, 64956502<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201501919" title="DOI Link to resource 10.1002/adfm.201501919">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtL3I" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/60_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit61">S. A. Lopez, B. Sanchez-Lengeling, J. de Goes Soares and A. Aspuru-Guzik, Design Principles and Top Non-Fullerene Acceptor Candidates for Organic Photovoltaics, <span class="italic">Joule</span>, 2017, <span class="bold">1</span>, 857870<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.joule.2017.10.006" title="DOI Link to resource 10.1016/j.joule.2017.10.006">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXpvFGhu7o%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/61_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit62">C. Zanlorenzi and L. Akcelrud, Theoretical studies for forecasting the power conversion efficiencies of polymer-based organic photovoltaic cells, <span class="italic">J. Polym. Sci., Part B: Polym. Phys.</span>, 2017, <span class="bold">55</span>, 919927<a target="_blank" class="DOILink" href="https://doi.org/10.1002/polb.24338" title="DOI Link to resource 10.1002/polb.24338">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXlslelsbs%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/62_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit63">Y. Imamura, M. Tashiro, M. Katouda and M. Hada, Automatic High-Throughput Screening Scheme for Organic Photovoltaics: Estimating the Orbital Energies of Polymers from Oligomers and Evaluating the Photovoltaic Characteristics, <span class="italic">J. Phys. Chem. C</span>, 2017, <span class="bold">121</span>, 2827528286<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.7b08446" title="DOI Link to resource 10.1021/acs.jpcc.7b08446">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvV2mt73K" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/63_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit64">N. Li, I. McCulloch and C. J. Brabec, Analyzing the efficiency, stability and cost potential for fullerene-free organic photovoltaics in one figure of merit, <span class="italic">Energy Environ. Sci.</span>, 2018, <span class="bold">11</span>, 13551361<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8EE00151K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8EE00151K">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/64_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit65">E. Abbasi Jannat Abadi, H. Sahu and S. M. Javadpour, M. Goharimanesh, Interpretable machine learning for developing high-performance organic solar cells, <span class="italic">Mater. Today Energy</span>, 2022, <span class="bold">25</span>, 100969<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mtener.2022.100969" title="DOI Link to resource 10.1016/j.mtener.2022.100969">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhtlOjurvF" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/65_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit66">S. Nagasawa, E. Al-Naamani and A. Saeki, Computer-Aided Screening of Conjugated Polymers for Organic Solar Cell: Classification by Random Forest, <span class="italic">J. Phys. Chem. Lett.</span>, 2018, <span class="bold">9</span>, 26392646<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.8b00635" title="DOI Link to resource 10.1021/acs.jpclett.8b00635">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXovVyms74%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29733216%5Buid%5D" title="PubMed Link to resource 29733216">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/66_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit67">Y.-C. Lin, Y.-J. Lu, C.-S. Tsao, A. Saeki, J.-X. Li, C.-H. Chen, H.-C. Wang, H.-C. Chen, D. Meng, K.-H. Wu, Y. Yang and K.-H. Wei, Enhancing photovoltaic performance by tuning the domain sizes of a small-molecule acceptor by side-chain-engineered polymer donors, <span class="italic">J. Mater. Chem. A</span>, 2019, <span class="bold">7</span>, 0723082<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Mater.%20Chem.%20A%5BJour%5D%20AND%207%5Bvolume%5D%20AND%20072%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/67_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit68">F.-C. Chen, and Virtual Screening of Conjugated Polymers for Organic Photovoltaic Devices Using Support Vector Machines and Ensemble Learning, <span class="italic">Int. J. Polym. Sci.</span>, 2019, <span class="bold">2019</span>, 4538514, <small>DOI:<a class="DOILink" href="https://doi.org/10.1155/2019/4538514" TARGET="_BLANK" title="DOI Link to 10.1155/2019/4538514">10.1155/2019/4538514</a></small><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/68_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit69">D. Padula, J. D. Simpson and A. Troisi, Combining electronic and structural features in machine learning models to predict organic solar cells properties, <span class="italic">Mater. Horiz.</span>, 2019, <span class="bold">6</span>, 343349<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8MH01135D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8MH01135D">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/69_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit70">H. Sahu, F. Yang, X. Ye, J. Ma, W. Fang and H. Ma, Designing promising molecules for organic solar cells via machine learning assisted virtual screening, <span class="italic">J. Mater. Chem. A</span>, 2019, <span class="bold">7</span>, 1748017488<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9TA04097H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9TA04097H">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/70_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit71">H. Sahu and H. Ma, Unraveling Correlations between Molecular Properties and Device Parameters of Organic Solar Cells Using Machine Learning, <span class="italic">J. Phys. Chem. Lett.</span>, 2019, <span class="bold">10</span>, 72777284<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.9b02772" title="DOI Link to resource 10.1021/acs.jpclett.9b02772">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitFagtbjL" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31702163%5Buid%5D" title="PubMed Link to resource 31702163">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/71_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit72">M. Lee, Insights from Machine Learning Techniques for Predicting the Efficiency of Fullerene Derivatives-Based Ternary Organic Solar Cells at Ternary Blend Design, <span class="italic">Adv. Energy Mater.</span>, 2019, <span class="bold">9</span>, 1900891<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201900891" title="DOI Link to resource 10.1002/aenm.201900891">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/72_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit73">M.-H. Lee, A Machine LearningBased Design Rule for Improved Open-Circuit Voltage in Ternary Organic Solar Cells, <span class="italic">Adv. Intell. Syst.</span>, 2010, <span class="bold">2</span>, 1900108<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aisy.201900108" title="DOI Link to resource 10.1002/aisy.201900108">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/73_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit74">Y. Lin, J. Wang, Z. G. Zhang, H. Bai, Y. Li, D. Zhu and X. Zhan, An electron acceptor challenging fullerenes for efficient polymer solar cells, <span class="italic">Adv. Mater.</span>, 2015, <span class="bold">27</span>, 11701174<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201404317" title="DOI Link to resource 10.1002/adma.201404317">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXltVGqsw%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25580826%5Buid%5D" title="PubMed Link to resource 25580826">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/74_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit75">W. Zhao, S. Li, H. Yao, S. Zhang, Y. Zhang, B. Yang and J. Hou, Molecular Optimization Enables over 13% Efficiency in Organic Solar Cells, <span class="italic">J. Am. Chem. Soc.</span>, 2017, <span class="bold">139</span>, 71487151<a target="_blank" class="DOILink" href="https://doi.org/10.1021/jacs.7b02677" title="DOI Link to resource 10.1021/jacs.7b02677">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCgtbo%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28513158%5Buid%5D" title="PubMed Link to resource 28513158">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/75_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit76">P. Cheng, G. Li, X. Zhan and Y. Yang, Next-generation organic photovoltaics based on non-fullerene acceptors, <span class="italic">Nat. Photonics</span>, 2018, <span class="bold">12</span>, 131142<a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41566-018-0104-9" title="DOI Link to resource 10.1038/s41566-018-0104-9">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjsFWmu7s%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/76_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit77">J. Wang, P. Xue, Y. Jiang, Y. Huo and X. Zhan, The principles, design and applications of fused-ring electron acceptors, <span class="italic">Nat. Rev. Chem.</span>, 2022, <span class="bold">6</span>, 614634<a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41570-022-00409-2" title="DOI Link to resource 10.1038/s41570-022-00409-2">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/77_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit78">C. Yan, S. Barlow, Z. Wang, H. Yan, A. K. Y. Jen, S. R. Marder and X. Zhan, Non-fullerene acceptors for organic solar cells, <span class="italic">Nat. Rev. Mater.</span>, 2018, <span class="bold">3</span>, 18003<a target="_blank" class="DOILink" href="https://doi.org/10.1038/natrevmats.2018.3" title="DOI Link to resource 10.1038/natrevmats.2018.3">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXis1Slur8%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/78_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit79">Y. Wu, J. Guo, R. Sun and J. Min, Machine learning for accelerating the discovery of high-performance donor/acceptor pairs in non-fullerene organic solar cells, <span class="italic">npj Comput. Mater.</span>, 2020, <span class="bold">6</span>, 120<a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-020-00388-2" title="DOI Link to resource 10.1038/s41524-020-00388-2">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitVWrs7zL" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/79_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit80">M. H. Lee, Robust random forest based non-fullerene organic solar cells efficiency prediction, <span class="italic">Org. Electron.</span>, 2020, <span class="bold">76</span>, 105465<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.orgel.2019.105465" title="DOI Link to resource 10.1016/j.orgel.2019.105465">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFGms73L" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/80_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit81">M. H. Lee, Identifying correlation between the open-circuit voltage and the frontier orbital energies of non-fullerene organic solar cells based on interpretable machine-learning approaches, <span class="italic">Sol. Energy</span>, 2022, <span class="bold">234</span>, 360367<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.solener.2022.02.010" title="DOI Link to resource 10.1016/j.solener.2022.02.010">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XntFeis78%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/81_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit82">K. Kranthiraja and A. Saeki, Experiment-Oriented Machine Learning of Polymer:Non-Fullerene Organic Solar Cells, <span class="italic">Adv. Funct. Mater.</span>, 2021, <span class="bold">31</span>, 2011168<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.202011168" title="DOI Link to resource 10.1002/adfm.202011168">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXltFOrtLs%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/82_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit83">Y. Miyake and A. Saeki, Machine Learning-Assisted Development of Organic Solar Cell Materials: Issues, Analyses, and Outlooks, <span class="italic">J. Phys. Chem. Lett.</span>, 2021, <span class="bold">12</span>, 1239112401<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.1c03526" title="DOI Link to resource 10.1021/acs.jpclett.1c03526">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXivVehsL7F" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34939806%5Buid%5D" title="PubMed Link to resource 34939806">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/83_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit84">K. Kranthiraja and A. Saeki, Machine Learning-Assisted Polymer Design for Improving the Performance of Non-Fullerene Organic Solar Cells, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2022, <span class="bold">14</span>, 2893628944<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.2c06077" title="DOI Link to resource 10.1021/acsami.2c06077">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhsFSntrrK" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35696604%5Buid%5D" title="PubMed Link to resource 35696604">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/84_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit85">Y. Miyake, K. Kranthiraja, F. Ishiwari and A. Saeki, Improved Predictions of Organic Photovoltaic Performance through Machine Learning Models Empowered by Artificially Generated Failure Data, <span class="italic">Chem. Mater.</span>, 2022, <span class="bold">34</span>, 69126920<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemmater.2c01294" title="DOI Link to resource 10.1021/acs.chemmater.2c01294">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhvV2iu77F" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/85_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit86">T. Hao, S. Leng, Y. Yang, W. Zhong, M. Zhang, L. Zhu, J. Song, J. Xu, G. Zhou, Y. Zou, Y. Zhang and F. Liu, Capture the high-efficiency non-fullerene ternary organic solar cells formula by machine-learning-assisted energy-level alignment optimization, <span class="italic">Patterns</span>, 2021, <span class="bold">2</span>, 100333<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.patter.2021.100333" title="DOI Link to resource 10.1016/j.patter.2021.100333">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhvF2nurnN" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34553173%5Buid%5D" title="PubMed Link to resource 34553173">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/86_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit87">P. Malhotra, S. Biswas, F.-C. Chen and G. D. Sharma, Prediction of non-radiative voltage losses in organic solar cells using machine learning, <span class="italic">Sol. Energy</span>, 2021, <span class="bold">228</span>, 175186<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.solener.2021.09.056" title="DOI Link to resource 10.1016/j.solener.2021.09.056">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXitFamsL%252FP" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/87_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit88">A. Mahmood and J.-L. Wang, A time and resource efficient machine learning assisted design of non-fullerene small molecule acceptors for P3HT-based organic solar cells and green solvent selection, <span class="italic">J. Mater. Chem. A</span>, 2021, <span class="bold">9</span>, 1568415695<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1TA04742F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1TA04742F">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/88_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit89">A. Mahmood, A. Irfan and J.-L. Wang, Machine learning and molecular dynamics simulation-assisted evolutionary design and discovery pipeline to screen efficient small molecule acceptors for PTB7-Th-based organic solar cells with over 15% efficiency, <span class="italic">J. Mater. Chem. A</span>, 2022, <span class="bold">10</span>, 41704180<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1TA09762H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1TA09762H">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/89_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit90">A. Mahmood, A. Irfan and J. Wang, Developing Efficient Small Molecule Acceptors with sp<small><sup>2</sup></small>-Hybridized Nitrogen at Different Positions by Density Functional Theory Calculations, Molecular Dynamics Simulations and Machine Learning, <span class="italic">Chem.  Eur. J.</span>, 2022, <span class="bold">28</span>, e202103712<a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.202103712" title="DOI Link to resource 10.1002/chem.202103712">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXis1KhsrbL" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34767281%5Buid%5D" title="PubMed Link to resource 34767281">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/90_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit91">X. Liu, Y. Shao, T. Lu, D. Chang, M. Li and W. Lu, Accelerating the discovery of high-performance donor/acceptor pairs in photovoltaic materials via machine learning and density functional theory, <span class="italic">Mater. Des.</span>, 2022, <span class="bold">216</span>, 110561<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.matdes.2022.110561" title="DOI Link to resource 10.1016/j.matdes.2022.110561">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38Xhtlanur7O" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/91_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit92">S. A. Lopez, E. O. Pyzer-Knapp, G. N. Simm, T. Lutzow, K. Li, L. R. Seress, J. Hachmann and A. Aspuru-Guzik, The Harvard organic photovoltaic dataset, <span class="italic">Sci. Data</span>, 2016, <span class="bold">3</span>, 160086<a target="_blank" class="DOILink" href="https://doi.org/10.1038/sdata.2016.86" title="DOI Link to resource 10.1038/sdata.2016.86">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27676312%5Buid%5D" title="PubMed Link to resource 27676312">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/92_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit93">E. O. Pyzer-Knapp, G. N. Simm and A. Aspuru Guzik, A Bayesian approach to calibrating high-throughput virtual screening results and application to organic photovoltaic materials, <span class="italic">Mater. Horiz.</span>, 2016, <span class="bold">3</span>, 226233<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5MH00282F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5MH00282F">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/93_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit94">A. Paul, A. Furmanchuk, W. Liao, A. Choudhary and A. Agrawal, Property Prediction of Organic Donor Molecules for Photovoltaic Applications Using Extremely Randomized Trees, <span class="italic">Mol. Inf.</span>, 2019, <span class="bold">38</span>, 1900038<a target="_blank" class="DOILink" href="https://doi.org/10.1002/minf.201900038" title="DOI Link to resource 10.1002/minf.201900038">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhsleitbvM" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31503423%5Buid%5D" title="PubMed Link to resource 31503423">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/94_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit95">N. Meftahi, M. Klymenko, A. J. Christofferson, U. Bach, D. A. Winkler and S. P. Russo, Machine learning property prediction for organic photovoltaic devices, <span class="italic">npj Comput. Mater.</span>, 2020, <span class="bold">6</span>, 166<a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-020-00429-w" title="DOI Link to resource 10.1038/s41524-020-00429-w">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXotlGitLk%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/95_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit96">W. Sun, Y. Zheng, K. Yang, Q. Zhang, A. A. Shah, Z. Wu, Y. Sun, L. Feng, D. Chen, Z. Xiao, S. Lu, Y. Li and K. Sun, Machine learningassisted molecular design and efficiency prediction for high-performance organic photovoltaic materials, <span class="italic">Sci. Adv.</span>, 2019, <span class="bold">5</span>, eaay4275<a target="_blank" class="DOILink" href="https://doi.org/10.1126/sciadv.aay4275" title="DOI Link to resource 10.1126/sciadv.aay4275">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtlaksL%252FN" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31723607%5Buid%5D" title="PubMed Link to resource 31723607">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/96_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit97">D. Padula and A. Troisi, Concurrent Optimization of Organic DonorAcceptor Pairs through Machine Learning, <span class="italic">Adv. Energy Mater.</span>, 2019, <span class="bold">9</span>, 1902463<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.201902463" title="DOI Link to resource 10.1002/aenm.201902463">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhs1ynsrzM" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/97_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit98">T. W. David, H. Anizelli, P. Tyagi, C. Gray, W. Teahan and J. Kettle, Using Large Datasets of Organic Photovoltaic Performance Data to Elucidate Trends in Reliability Between 2009 and 2019, <span class="italic">IEEE J. Photovoltaics</span>, 2019, <span class="bold">9</span>, 17681773<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=IEEE%20J.%20Photovoltaics%5BJour%5D%20AND%209%5Bvolume%5D%20AND%201768%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/98_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit99">T. W. David, H. Anizelli, T. J. Jacobsson, C. Gray, W. Teahan and J. Kettle, Enhancing the stability of organic photovoltaics through machine learning, <span class="italic">Nano Energy</span>, 2020, <span class="bold">78</span>, 105342<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.nanoen.2020.105342" title="DOI Link to resource 10.1016/j.nanoen.2020.105342">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSmu77P" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/99_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit100">T. W. David and J. Kettle, Design for a Sustainability Approach to Organic Solar Cell Design: the Use of Machine Learning to Quantify the Trade-off between Performance, Stability, and Environmental Impact, <span class="italic">J. Phys. Chem. C</span>, 2022, <span class="bold">126</span>, 47744784<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcc.1c10114" title="DOI Link to resource 10.1021/acs.jpcc.1c10114">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XmtFKmt7s%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/100_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit101">Z.-W. Zhao, M. del Cueto and A. Troisi, Limitations of machine learning models when predicting compounds with completely new chemistries: possible improvements applied to the discovery of new non-fullerene acceptors, <span class="italic">Digital Discovery</span>, 2022, <span class="bold">1</span>, 266276<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D2DD00004K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D2DD00004K">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/101_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit102">M.-H. Lee, Performance and Matching Band Structure Analysis of Tandem Organic Solar Cells Using Machine Learning Approaches, <span class="italic">Energy Technol.</span>, 2020, <span class="bold">8</span>, 1900974<a target="_blank" class="DOILink" href="https://doi.org/10.1002/ente.201900974" title="DOI Link to resource 10.1002/ente.201900974">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXksVGlur0%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/102_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit103">W. Sun, Y. Zheng, Q. Zhang, K. Yang, H. Chen, Y. Cho, J. Fu, O. Odunmbaku, A. A. Shah, Z. Xiao, S. Lu, S. Chen, M. Li, B. Qin, C. Yang, T. Frauenheim and K. Sun, Artificial Intelligence Designer for Highly-Efficient Organic Photovoltaic Materials, <span class="italic">J. Phys. Chem. Lett.</span>, 2021, <span class="bold">12</span>, 88478854<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.1c02554" title="DOI Link to resource 10.1021/acs.jpclett.1c02554">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvFKhs7bJ" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34494851%5Buid%5D" title="PubMed Link to resource 34494851">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/103_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit104">B. L. Greenstein and G. R. Hutchison, Organic Photovoltaic Efficiency Predictor: Data-Driven Models for Non-Fullerene Acceptor Organic Solar Cells, <span class="italic">J. Phys. Chem. Lett.</span>, 2022, <span class="bold">13</span>, 42354243<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.2c00866" title="DOI Link to resource 10.1021/acs.jpclett.2c00866">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38Xht1SgsL7O" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35522056%5Buid%5D" title="PubMed Link to resource 35522056">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/104_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit105">W. Sun, M. Li, Y. Li, Z. Wu, Y. Sun, S. Lu, Z. Xiao, B. Zhao and K. Sun, The Use of Deep Learning to Fast Evaluate Organic Photovoltaic Materials, <span class="italic">Adv. Theory Simul.</span>, 2019, <span class="bold">2</span>, 1800116<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adts.201800116" title="DOI Link to resource 10.1002/adts.201800116">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/105_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit106">G. J. Moore, O. Bardagot and N. Banerji, Deep Transfer Learning: A Fast and Accurate Tool to Predict the Energy Levels of Donor Molecules for Organic Photovoltaics, <span class="italic">Adv. Theory Simul.</span>, 2022, <span class="bold">5</span>, 2100511<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adts.202100511" title="DOI Link to resource 10.1002/adts.202100511">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XpsVOksLc%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/106_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit107">D. Weininger, SMILES, a chemical language and information system. 1. Introduction to methodology and encoding rules, <span class="italic">J. Chem. Inf. Model.</span>, 1988, <span class="bold">28</span>, 3136<a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci00057a005" title="DOI Link to resource 10.1021/ci00057a005">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL1cXnsVeqsA%253D%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/107_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit108">A. H. Vo, T. R. Van Vleet, R. R. Gupta, M. J. Liguori and M. S. Rao, An Overview of Machine Learning and Big Data for Drug Toxicity Evaluation, <span class="italic">Chem. Res. Toxicol.</span>, 2020, <span class="bold">33</span>, 2037<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Chem.%20Res.%20Toxicol.%5BJour%5D%20AND%2033%5Bvolume%5D%20AND%2020%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/108_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit109">J. Schmidt, M. R. G. Marques, S. Botti and M. A. L. Marques, Recent advances and applications of machine learning in solid-state materials science, <span class="italic">npj Comput. Mater.</span>, 2019, <span class="bold">5</span>, 83<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=npj%20Comput.%20Mater.%5BJour%5D%20AND%205%5Bvolume%5D%20AND%2083%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/109_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit110">F. Pedregosa, G. Varoquaux, A. Gramfort, V. Michel, B. Thirion, O. Grisel, M. Blondel, A. Mller, J. Nothman, G. Louppe, P. Prettenhofer, R. Weiss, V. Dubourg, J. Vanderplas, A. Passos, D. Cournapeau, M. Brucher, M. Perrot and . Duchesnay, Scikit-learn: Machine Learning in Python, <span class="italic">J. Mach. Learn. Res.</span>, 2012, <span class="bold">12</span>, 28252830<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Mach.%20Learn.%20Res.%5BJour%5D%20AND%2012%5Bvolume%5D%20AND%202825%5Bpage%5D%20and%202012%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/110_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit111">X. Rodrguez-Martnez, E. Pascual-San-Jos and M. Campoy-Quiles, Accelerating organic solar cell material's discovery: high-throughput screening and big data, <span class="italic">Energy Environ. Sci.</span>, 2021, <span class="bold">14</span>, 33013322<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1EE00559F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1EE00559F">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/111_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit112">F. Pedregosa, G. Varoquaux, A. Gramfort, V. Michel, B. Thirion, O. Grisel, M. Blondel, P. Prettenhofer, R. Weiss, V. Dubourg, J. Vanderplas, A. Passos, D. Cournapeau, M. Brucher, M. Perrot and . Duchesnay, Scikit-learn: Machine learning in Python, <span class="italic">J. Mach. Learn. Res.</span>, 2011, <span class="bold">12</span>, 28252830<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Mach.%20Learn.%20Res.%5BJour%5D%20AND%2012%5Bvolume%5D%20AND%202825%5Bpage%5D%20and%202011%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/112_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit113">
          M. Abadi, P. Barham, J. Chen, Z. Chen, A. Davis, J. Dean, M. Devin, S. Ghemawat, G. Irving, M. Isard, M. Kudlur, J. Levenberg, R. Monga, S. Moore, D. G. Murray, B. Steiner, P. Tucker, V. Vasudevan, P. Warden, M. Wicke, Y. Yu and X. Zheng, TensorFlow: A system for large-scale machine learning, in <span class="italic">Proc. 12th USENIX Symp. Oper. Syst. Des. Implementation, OSDI 2016</span>,  2016, pp. 265283, https://github.com/<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proc.%2012th%20USENIX%20Symp.%20Oper.%20Syst.%20Des.%20Implementation,%20OSDI%202016%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/113_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit114">
          A. Paszke, S. Gross, F. Massa, A. Lerer, J. Bradbury, G. Chanan, T. Killeen, Z. Lin, N. Gimelshein, L. Antiga, A. Desmaison, A. Kpf, E. Yang, Z. DeVito, M. Raison, A. Tejani, S. Chilamkurthy, B. Steiner, L. Fang, J. Bai and S. Chintala, PyTorch: An imperative style, high-performance deep learning library, in: <span class="italic">Adv. Neural Inf. Process. Syst.</span>,  2019. https://arxiv.org/abs/1912.01703<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Adv.%20Neural%20Inf.%20Process.%20Syst.%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/114_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit115">J. L. Bredas, Mind the gap!, <span class="italic">Mater. Horiz.</span>, 2014, <span class="bold">1</span>, 1719<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3MH00098B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3MH00098B">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/115_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit116">J. L. Durant, B. A. Leland, D. R. Henry and J. G. Nourse, Reoptimization of MDL keys for use in drug discovery, <span class="italic">J. Chem. Inf. Comput. Sci.</span>, 2002, <span class="bold">42</span>, 12731280<a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci010132r" title="DOI Link to resource 10.1021/ci010132r">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD38XmvFKktLY%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12444722%5Buid%5D" title="PubMed Link to resource 12444722">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/116_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit117">D. Rogers and M. Hahn, Extended-Connectivity Fingerprints, <span class="italic">J. Chem. Inf. Model.</span>, 2010, <span class="bold">50</span>, 742754<a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci100050t" title="DOI Link to resource 10.1021/ci100050t">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20426451%5Buid%5D" title="PubMed Link to resource 20426451">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/117_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit118">G. Landrum, RDKit, <span class="italic">Open-Source Cheminformatics</span>, 2016<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Open-Source%20Cheminformatics%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/118_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit119">
          R. Fisher, <span class="italic">The Design of Experiments (1935), Edinburgh Oliver Boyd</span>, 1935<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=The%20Design%20of%20Experiments%20(1935),%20Edinburgh%20Oliver%20Boyd%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%20%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/119_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit120">N. Majeed, M. Saladina, M. Krompiec, S. Greedy, C. Deibel and R. C. I. MacKenzie, Using Deep Machine Learning to Understand the Physical Performance Bottlenecks in Novel Thin-Film Solar Cells, <span class="italic">Adv. Funct. Mater.</span>, 2020, <span class="bold">30</span>, 1907259<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.201907259" title="DOI Link to resource 10.1002/adfm.201907259">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitlyhtLfF" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/120_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit121">B. Sanchez-Lengeling and A. Aspuru-Guzik, Inverse molecular design using machine learning:Generative models for matter engineering, <span class="italic">Science</span>, 2018, <span class="bold">361</span>, 360365<a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aat2663" title="DOI Link to resource 10.1126/science.aat2663">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyitr3L" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30049875%5Buid%5D" title="PubMed Link to resource 30049875">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/121_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit122">H. Sun, F. Chen and Z. Chen, Recent progress on non-fullerene acceptors for organic photovoltaics, <span class="italic">Mater. Today</span>, 2019, <span class="bold">24</span>, 94118<a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mattod.2018.09.004" title="DOI Link to resource 10.1016/j.mattod.2018.09.004">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhvVOhsLrP" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/122_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit123">W. Xu and F. Gao, The progress and prospects of non-fullerene acceptors in ternary blend organic solar cells, <span class="italic">Mater. Horiz.</span>, 2018, <span class="bold">5</span>, 206221<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7MH00958E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7MH00958E">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/123_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit124">A. Kirkey, E. J. Luber, B. Cao, B. C. Olsen and J. M. Buriak, Optimization of the Bulk Heterojunction of All-Small-Molecule Organic Photovoltaics Using Design of Experiment and Machine Learning Approaches, <span class="italic">ACS Appl. Mater. Interfaces</span>, 2020, <span class="bold">12</span>, 5459654607<a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsami.0c14922" title="DOI Link to resource 10.1021/acsami.0c14922">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitlyrs7jM" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33226763%5Buid%5D" title="PubMed Link to resource 33226763">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/124_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit125">H. Moriwaki, Y.-S. Tian, N. Kawashita and T. Takagi, Mordred: a molecular descriptor calculator, <span class="italic">J. Cheminf.</span>, 2018, <span class="bold">10</span>, 4<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%2010%5Bvolume%5D%20AND%204%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/125_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit126">C. Steinbeck, Y. Han, S. Kuhn, O. Horlacher, E. Luttmann and E. Willighagen, The Chemistry Development Kit (CDK): An open-source Java library for chemo- and bioinformatics, <span class="italic">J. Chem. Inf. Comput. Sci.</span>, 2003, <span class="bold">43</span>, 493500<a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci025584y" title="DOI Link to resource 10.1021/ci025584y">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXhtVaktbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12653513%5Buid%5D" title="PubMed Link to resource 12653513">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/126_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit127">S. Kim, P. A. Thiessen, E. E. Bolton, J. Chen, G. Fu, A. Gindulyte, L. Han, J. He, S. He, B. A. Shoemaker, J. Wang, B. Yu, J. Zhang and S. H. Bryant, PubChem substance and compound databases, <span class="italic">Nucleic Acids Res.</span>, 2016, <span class="bold">44</span>, D1202D1213<a target="_blank" class="DOILink" href="https://doi.org/10.1093/nar/gkv951" title="DOI Link to resource 10.1093/nar/gkv951">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bE" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26400175%5Buid%5D" title="PubMed Link to resource 26400175">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/127_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit128">I. Sushko, S. Novotarskyi, R. Krner, A. K. Pandey, M. Rupp, W. Teetz, S. Brandmaier, A. Abdelaziz, V. V. Prokopenko, V. Y. Tanchuk, R. Todeschini, A. Varnek, G. Marcou, P. Ertl, V. Potemkin, M. Grishina, J. Gasteiger, C. Schwab, I. I. Baskin, V. A. Palyulin, E. V. Radchenko, W. J. Welsh, V. Kholodovych, D. Chekmarev, A. Cherkasov, J. Aires-De-Sousa, Q. Y. Zhang, A. Bender, F. Nigsch, L. Patiny, A. Williams, V. Tkachenko and I. V. Tetko, Online
chemical modeling environment (OCHEM): Web platform for data storage, model development and publishing of chemical information, <span class="italic">J. Comput.-Aided Mol. Des.</span>, 2011, <span class="bold">25</span>, 533554<a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10822-011-9440-2" title="DOI Link to resource 10.1007/s10822-011-9440-2">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXosFCgsrg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21660515%5Buid%5D" title="PubMed Link to resource 21660515">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/128_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit129">J. Dong, D.-S. Cao, H.-Y. Miao, S. Liu, B.-C. Deng, Y.-H. Yun, N.-N. Wang, A.-P. Lu, W.-B. Zeng and A. F. Chen, ChemDes: an integrated web-based platform for molecular descriptor and fingerprint computation, <span class="italic">J. Cheminf.</span>, 2015, <span class="bold">7</span>, 60<a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Cheminf.%5BJour%5D%20AND%207%5Bvolume%5D%20AND%2060%5Bpage%5D%20and%202015%5Bpdat%5D" title="Search PubMed for this citation">SearchPubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/129_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit130">
          M. R. Berthold, N. Cebron, F. Dill, G. Di Fatta, T. R. Gabriel, F. Georg, T. Meinl, P. Ohl, C. Sieb and B. Wiswedel, KNIME: The konstanz information miner, <span class="italic">4th Int. Ind. Simul. Conf. 2006</span>,  2006, 11, 5861<small>DOI:<a class="DOILink" href="https://doi.org/10.1145/1656274.1656280" TARGET="_BLANK" title="DOI Link to 10.1145/1656274.1656280">10.1145/1656274.1656280</a></small><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/130_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit131">M. Hall, E. Frank, G. Holmes, B. Pfahringer, P. Reutemann and I. H. Witten, The WEKA data mining software, <span class="italic">ACM SIGKDD Explor. Newsl.</span>, 2009, <span class="bold">11</span>, 1018<a target="_blank" class="DOILink" href="https://doi.org/10.1145/1656274.1656278" title="DOI Link to resource 10.1145/1656274.1656278">CrossRef</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/131_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit132">A. Mauri, V. Consonni, M. Pavan and R. Todeschini, DRAGON software: An easy approach to molecular descriptor calculations, <span class="italic">MATCH</span>, 2006, <span class="bold">56</span>, 237248<a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtVWhtbg%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/132_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit133">Q. Zhang, Y. J. Zheng, W. Sun, Z. Ou, O. Odunmbaku, M. Li, S. Chen, Y. Zhou, J. Li, B. Qin and K. Sun, High-Efficiency Non-Fullerene Acceptors Developed by Machine Learning and Quantum Chemistry, <span class="italic">Adv. Sci.</span>, 2022, <span class="bold">9</span>, 2104742<a target="_blank" class="DOILink" href="https://doi.org/10.1002/advs.202104742" title="DOI Link to resource 10.1002/advs.202104742">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhtVWgurvN" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34989179%5Buid%5D" title="PubMed Link to resource 34989179">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/133_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit134">B. L. Greenstein, D. C. Hiener and G. R. Hutchison, Computational Evolution of High-Performing Unfused Non-Fullerene Acceptors for Organic Solar Cells, <span class="italic">J. Chem. Phys.</span>, 2022, <span class="bold">156</span>, 174107<a target="_blank" class="DOILink" href="https://doi.org/10.1063/5.0087299" title="DOI Link to resource 10.1063/5.0087299">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38Xht1WgsrnK" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35525667%5Buid%5D" title="PubMed Link to resource 35525667">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/134_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit135">L. Ruddigkeit, R. Van Deursen, L. C. Blum and J. L. Reymond, Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17, <span class="italic">J. Chem. Inf. Model.</span>, 2012, <span class="bold">52</span>, 28642875<a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci300415d" title="DOI Link to resource 10.1021/ci300415d">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhsFClsL3J" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23088335%5Buid%5D" title="PubMed Link to resource 23088335">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/135_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit136">A. Daina, O. Michielin and V. Zoete, SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 42717<a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep42717" title="DOI Link to resource 10.1038/srep42717">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28256516%5Buid%5D" title="PubMed Link to resource 28256516">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/136_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit137">Z. Shi, Y. Bai, X. Chen, R. Zeng and Z. Tan, Tandem structure: a breakthrough in power conversion efficiency for highly efficient polymer solar cells, <span class="italic">Sustainable Energy Fuels</span>, 2019, <span class="bold">3</span>, 910934<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8SE00601F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8SE00601F">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/137_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit138">S. Langner, F. Hse, J. D. Perea, T. Stubhan, J. Hauch, L. M. Roch, T. Heumueller, A. Aspuru-Guzik and C. J. Brabec, Beyond Ternary OPV: High-Throughput Experimentation and Self-Driving Laboratories Optimize Multicomponent Systems, <span class="italic">Adv. Mater.</span>, 2020, <span class="bold">32</span>, 1907801<a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.201907801" title="DOI Link to resource 10.1002/adma.201907801">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXivFygsro%253D" title="Link to resource in CAS">CAS</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32049386%5Buid%5D" title="PubMed Link to resource 32049386">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/138_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit139">
          E. E. Bolton, Y. Wang, P. A. Thiessen and S. H. Bryant, PubChem: Integrated Platform of Small Molecules and Biological Activities, in <span class="italic">Annu. Rep. Comput. Chem.</span>, Elsevier BV,  2008, pp. 217241<small>DOI:<a class="DOILink" href="https://doi.org/10.1016/S1574-1400(08)00012-1" TARGET="_BLANK" title="DOI Link to 10.1016/S1574-1400(08)00012-1">10.1016/S1574-1400(08)00012-1</a></small><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/139_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit140">M. de Wergifosse and S. Grimme, Nonlinear-response properties in a simplified time-dependent density functional theory (sTD-DFT) framework: Evaluation of the first hyperpolarizability, <span class="italic">J. Chem. Phys.</span>, 2018, <span class="bold">149</span>, 024108<a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5037665" title="DOI Link to resource 10.1063/1.5037665">CrossRef</a><a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30007395%5Buid%5D" title="PubMed Link to resource 30007395">PubMed</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/140_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit141">J. Yan, X. Rodrguez-Martnez, D. Pearce, H. Douglas, D. Bili, M. Azzouzi, F. Eisner, A. Virbule, E. Rezasoltani, V. Belova, B. Drling, S. Few, A. A. Szumska, X. Hou, G. Zhang, H.-L. Yip, M. Campoy-Quiles and J. Nelson, Identifying structureabsorption relationships and predicting absorption strength of non-fullerene acceptors for organic photovoltaics, <span class="italic">Energy Environ. Sci.</span>, 2022, <span class="bold">15</span>, 29582973<a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D2EE00887D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D2EE00887D">RSC</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/141_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
      <li><span id="cit142">C. Yan, J. Qin, Y. Wang, G. Li and P. Cheng, Emerging Strategies toward Mechanically Robust Organic Photovoltaics: Focus on Active Layer, <span class="italic">Adv. Energy Mater.</span>, 2022, <span class="bold">12</span>, 2201087<a target="_blank" class="DOILink" href="https://doi.org/10.1002/aenm.202201087" title="DOI Link to resource 10.1002/aenm.202201087">CrossRef</a><a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhsVSnu7%252FP" title="Link to resource in CAS">CAS</a><a target="_blank" class="URLLink" href="/en/Content/OpenURL/50000925/d2tc03276g/142_1" title="Link via OpenURL Resolver"><img height="14px" alt="" src="/en/Image/GetOpenURLImage/50000925"/></a>.</span></li>
    </ol>
    
    
    
    
    
  <hr/><table><tr><td><h3>Footnote</h3></td></tr><tr><td><span class="sup_ref"> <span id="fn1">Electronic supplementary information (ESI) available. See DOI: <a target="_blank" href="https://doi.org/10.1039/d2tc03276g">https://doi.org/10.1039/d2tc03276g</a></span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is  The Royal Society of Chemistry 2022</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"></script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>