<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Million-atom molecular dynamics simulations reveal the interfacial interactions and assembly of plant PSII-LHCII supercomplex   - RSC Advances (RSC Publishing) DOI:10.1039/D2RA08240C</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2023/ra/d2ra08240c"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":52,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Ruichao Mao"/><meta name="DC.Creator" content="Han Zhang"/><meta name="DC.Creator" content="Lihua Bie"/><meta name="DC.Creator" content="Lu-Ning Liu"/><meta name="DC.Creator" content="Jun Gao"/><meta name="DC.title" content="Million-atom molecular dynamics simulations reveal the interfacial interactions and assembly of plant PSII-LHCII supercomplex  "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2023/02/27"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D2RA08240C"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Million-atom molecular dynamics simulations reveal the interfacial interactions and assembly of plant PSII-LHCII supercomplex  "/><meta name="citation_author" content="Ruichao Mao"/><meta name="citation_author" content="Han Zhang"/><meta name="citation_author" content="Lihua Bie"/><meta name="citation_author" content="Lu-Ning Liu"/><meta name="citation_author" content="Jun Gao"/><meta name="citation_online_date" content="2023/02/27"/><meta name="citation_date" content="2023"/><meta name="citation_journal_title" content="RSC Advances"/><meta name="citation_volume" content="13"/><meta name="citation_issue" content="10"/><meta name="citation_firstpage" content="6699"/><meta name="citation_lastpage" content="6712"/><meta name="citation_doi" content="10.1039/D2RA08240C"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2023/ra/d2ra08240c"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2023/ra/d2ra08240c"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2023/ra/d2ra08240c"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2023/ra/d2ra08240c">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d2ra08324h)" href="/en/content/articlehtml/2023/ra/d2ra08324h" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d2ra07112f)" href="/en/content/articlehtml/2023/ra/d2ra07112f" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div><br/><span class="oa"><img src="/content/newimages/open_access_blue.png" alt=""/> Open Access Article<br/><img src="/content/newimages/CCBY.svg" alt=""/>
This Open Access Article is licensed under a <br/><a rel="license" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 Unported Licence</a></span></div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D2RA08240C">10.1039/D2RA08240C</a>
(Paper)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2046-2069/2011">RSC Adv.</a></span>, 2023, <strong>13</strong>, 6699-6712</div><h1 id="sect233"><span class="title_heading">Million-atom molecular dynamics simulations reveal the interfacial interactions and assembly of plant PSII-LHCII supercomplex<a title="Select to navigate to footnote" href="#fn1">†</a></span></h1><p class="header_text">
<span class="bold">Ruichao Mao</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Ruichao Mao (orcid.org/0000-0002-5494-7465) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-5494-7465"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">Han Zhang</span><sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">Lihua Bie</span><sup><span class="sup_ref italic">a</span></sup>, 
<span class="bold">Lu-Ning Liu</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Lu-Ning Liu (orcid.org/0000-0002-8884-4819) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-8884-4819"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">bc</span></sup><span class="bold"> and </span>
<span class="bold">Jun Gao</span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Jun Gao (orcid.org/0000-0002-6692-1828) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-6692-1828"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">a</span></sup>
<br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Hubei Key Laboratory of Agricultural Bioinformatics, College of Informatics, Huazhong Agricultural University, Wuhan 430070, Hubei, China. E-mail: <a href="mailto:gaojun@mail.hzau.edu.cn">gaojun@mail.hzau.edu.cn</a></span>
<br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Institute of Systems, Molecular and Integrative Biology, University of Liverpool, Liverpool L69 7ZB, UK. E-mail: <a href="mailto:luning.liu@liverpool.ac.uk">luning.liu@liverpool.ac.uk</a></span>
<br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">Frontiers Science Center for Deep Ocean Multispheres and Earth System, College of Marine Life Sciences, Ocean University of China, Qingdao 266003, China</span>
</p><div id="art-admin"><span class="italic bold">Received 
26th December 2022
</span><span class="bold italic">, Accepted 7th February 2023</span></div><p class="bold italic">First published on 27th February 2023</p><hr/><div class="abstract"><h2>Abstract</h2><p>Protein–protein interface interactions dictate efficient excitation energy transfer from light-harvesting antennas to the photosystem II (PSII) core. In this work, we construct a 1.2 million atom-scale model of plant C<small><sub>2</sub></small>S<small><sub>2</sub></small>-type PSII-LHCII supercomplex and perform microsecond-scale molecular dynamics (MD) simulations to explore the interactions and assembly mechanisms of the sizeable PSII-LHCII supercomplex. We optimize the nonbonding interactions of the PSII-LHCII cryo-EM structure using microsecond-scale MD simulations. Binding free energy calculations with component decompositions reveal that hydrophobic interactions predominantly drive antenna–core association and the antenna–antenna interactions are relatively weak. Despite the positive electrostatic interaction energies, hydrogen bonds and salt bridges mainly provide directional or anchoring forces for interface binding. Analysis of the roles of small intrinsic subunits of PSII suggests that LHCII and CP26 first interact with small intrinsic subunits and then bind to the core proteins, whereas CP29 adopts a one-step binding process to the PSII core without the assistance of other factors. Our study provides insights into the molecular underpinnings of the self-organization and regulation of plant PSII-LHCII. It lays the framework for deciphering the general assembly principles of photosynthetic supercomplexes and possibly other macromolecular structures. The finding also has implications for repurposing photosynthetic systems to enhance photosynthesis.</p></div><hr/>
<h2 id="sect300"><span class="a_heading">Introduction</span></h2>
<span>Photosynthesis is the primary source of energy for most life on Earth. It is estimated that photosynthesis produces more than 100 billion tons of dry biomass annually, which is equivalent to 100 times the weight of the current total human population on Earth and equal to an average energy storage rate of about 100 TW.<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> The high-efficiency energy conversion is governed by the physiological coordination and arrangement of photosynthetically active chromoprotein complexes, which were generated during more than 3.5 billion years of evolution.<a title="Select to navigate to references" href="#cit2"><sup><span class="sup_ref">2</span></sup></a> Understanding the assembly and energy transfer mechanisms of photosynthetic protein complexes is pivotal for advancing our knowledge of the fundamentals of photosynthesis and underpinning the development of artificial photosynthesis to enhance and modulate energy conversion.<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a></span>
<p class="otherpara">Photosystem II (PSII) is the only enzyme that catalyzes water splitting in oxygenic photosynthesis and has been a primary target in designing biomimetic photosynthetic systems.<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a> Our knowledge about the assembly principles of the PSII core has been greatly improved by studies on high-resolution structures (up to 1.9 Å) and functions.<a title="Select to navigate to reference" href="#cit5"><sup><span class="sup_ref">5–7</span></sup></a> Moreover, substantial developments have been made recently in PSII-based hybrid systems and the utilization of PSII assemblies for photocatalytic water splitting.<a title="Select to navigate to reference" href="#cit3"><sup><span class="sup_ref">3,8,9</span></sup></a> For example, a hybrid natural-artificial photosynthetic platform was constructed by wiring a PSII core and a silicon photoelectrochemical cell, which could perform the water-splitting process under solar irradiation.<a title="Select to navigate to references" href="#cit8"><sup><span class="sup_ref">8</span></sup></a></p>
<p class="otherpara">Plant PSII dimeric core associates with membrane-spinning light-harvesting antenna complexes (LHCII) to form PSII-LHCII supercomplexes. The dynamic interactions between LHCII and PSII have been the major challenge in the high-resolution structural characterization of PSII-LHCII supercomplexes.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> This has been drastically improved by the technological advances of cryo-electron microscopy (cryo-EM), demonstrating the power in revolving large biomolecular assemblies.<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11–18</span></sup></a> However, the resolved 3D structures reflect the static states of multiprotein supercomplexes under specific experimental conditions.<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a> They could not delineate dynamic protein associations and the assembly process of photosynthetic PSII-LHCII supercomplexes. Indeed, cryo-EM structures have suggested a higher degree of flexibility of atomic positions at the antenna-PSII core interface and dynamic associations of LHCII with the PSII core,<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> which are fundamental for state transitions and PSII repair.<a title="Select to navigate to reference" href="#cit20"><sup><span class="sup_ref">20–22</span></sup></a> Moreover, PSII-LHCII undertakes a stepwise assembly pathway, and the attachment of LHCII to the dimeric PSII core appears to occur at the final step of PSII-LHCII assembly.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> Some protein subunits, including PsbW, PsbZ, and PsbH, were suggested to be involved in the antenna-PSII core assembly.<a title="Select to navigate to reference" href="#cit11"><sup><span class="sup_ref">11,23–26</span></sup></a> The detailed protein–protein interactions and assembly mechanisms that govern the association of LHCII with the PSII core remain elusive.</p>
<p class="otherpara">Molecular dynamics (MD) simulations provide a powerful toolkit to investigate <span class="italic">in silico</span> the dynamics of biomolecular assemblies with atomic resolutions at a time scale ranging from femtoseconds to milliseconds.<a title="Select to navigate to reference" href="#cit19"><sup><span class="sup_ref">19,27</span></sup></a> Large-scale MD simulations have been applied in studying the structures and functions of photosynthetic macromolecular complexes and organelles at different scales.<a title="Select to navigate to reference" href="#cit17"><sup><span class="sup_ref">17–19,28–30</span></sup></a> Here, we construct a million-atom scale model of plant C<small><sub>2</sub></small>S<small><sub>2</sub></small>-type PSII-LHCII supercomplex embedded in the solvated membrane, based on the cryo-EM structure,<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> and perform microsecond-scale MD simulations to study the protein interactions and assembly process of the sizeable PSII-LHCII supercomplex in a near cellular environment. Our results indicate the general binding principles and atomistic details of the PSII-LHCII assembly.</p>

<h2 id="sect326"><span class="a_heading">Methods</span></h2>
<h3 id="sect328"><span class="b_heading">Initial model</span></h3>
<span>The cryo-EM structure of spinach PSII-LHCII (PDB ID: <a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>)<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> was selected as the initial model for simulating the interactions between LHCII proteins and the PSII core. According to the previous studies,<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31–33</span></sup></a> the PSII-LHCII complex was embedded in a pre-equilibrated lipid bilayer consisting of single-component 1-palmitoyl-2-oleoyl-<span class="italic">sn-glycero</span>-3-phosphocholine (POPC) to mimics native thylakoid membranes. The system was then neutralized by the addition of counter ions (202 sodium ions) and solvated in a double-shell water box containing 277<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>727 TIP3P water molecules. Our simulation box (240.5 Å × 350.0 Å × 155.5 Å) contains a total of 1<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>159<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>431 atoms, and the solvated complexes with lipid bilayers are shown in Fig. S1.<a title="Select to navigate to footnote" href="#fn1">†</a></span>
<p class="otherpara">Histidine residues in the system were singly protonated on Nε, except those coordinating to non-heme FE, HEM or CLA <span class="italic">via</span> Nε (and were thus protonated on Nδ). In addition, the two disulfide bonds contained in each monomer (C112–C135 of PsbO and C17–C26 of PsbTn) were explicitly considered. The AMBER ff14SB force field parameter set<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> was selected for standard amino acids residues. For the ten cofactors (PHO, BCR, PL9, LHG, SQD, LMG, DGD, LUT, XAT, NEX) (full names are shown in Table S1<a title="Select to navigate to footnote" href="#fn1">†</a>), the generalized Amber force field (GAFF) parameter set was adopted.<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> The parameters of CLA and CHL were taken from those developed by Ceccarelli <span class="italic">et al.</span> for bacteriochlorophyll <span class="italic">a</span>.<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> The parameters of HEM were taken from the AMBER parameter database.<a title="Select to navigate to references" href="#cit37"><sup><span class="sup_ref">37</span></sup></a> The parameters of POPC were taken from previous work.<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> For the non-heme Fe clusters, we obtained the parameters using the MCPB.py program.<a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a> The atomic charges of the cofactors were determined by fitting the electrostatic potential around these molecules by using the RESP model.<a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a> The atomic charges of the OEX complexes were assigned according to the redox states of its atoms in the dark-adapted (S1) state as follows: Mn1–Mn2, +3; Mn3–Mn4, +4; O1–O5, −2, Ca, +2. According to Ogata <span class="italic">et al.</span>,<a title="Select to navigate to references" href="#cit41"><sup><span class="sup_ref">41</span></sup></a> the equilibrium bond lengths, bond angles, and torsion angles were set as the average value of OEX in the two monomers in the cryo-EM structure. The force constants for bond lengths, bond angles, and torsion angles were appropriately set at 1500 kcal mol<small><sup>−1</sup></small> Å<small><sup>−2</sup></small>, 500 kcal mol<small><sup>−1</sup></small> Å<small><sup>−2</sup></small>, and 100 kcal mol<small><sup>−1</sup></small> Å<small><sup>−2</sup></small>, respectively, to maintain the configuration of OEX during the simulations.</p>

<h3 id="sect356"><span class="b_heading">Molecular dynamic simulations</span></h3>
<span>Energy minimization of the whole system was performed with a three-step procedure: (1) the system was minimized with 20<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 steps with the steepest descent algorithm by freezing the PSII-LHCII complex to smooth the contact between the phospholipid membrane and the protein. (2) 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 steps of energy minimization were performed for the whole system with restraints on the protein backbone and heavy atoms of cofactors (100 kcal mol<small><sup>−1</sup></small> Å<small><sup>−2</sup></small>) (3) 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 steps of energy minimization was performed without any restraints. The whole system was then slowly heated to 300 K within 60 ps under the restraint (10 kcal mol<small><sup>−1</sup></small> Å<small><sup>−2</sup></small>) to the protein backbone and heavy atoms of cofactors. After these optimization and heating procedures, the constraints were released, and 10 ns equilibrium simulations were performed under the <span class="italic">NPT</span> ensemble at 300 K to balance the dimensions and density of the system. Finally, a 1 μs production MD simulation was performed using the GPU implementation of PMEMD from the AMBER16 software package.<a title="Select to navigate to references" href="#cit42"><sup><span class="sup_ref">42</span></sup></a> Atomic coordinates of all atoms were recorded every 1 ps. Temperature is controlled here using the Langevin thermostat<a title="Select to navigate to references" href="#cit43"><sup><span class="sup_ref">43</span></sup></a> while pressure is controlled using the anisotropic Berendsen barostat.<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> Covalent bonds involving hydrogen atoms were constrained using the SHAKE algorithm.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> A non-bonding cutoff of 10 Å was applied to van der Waals interactions, and the particle mesh Ewald (PME) method<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> was used to deal with long-range electrostatics.</span>

<h3 id="sect369"><span class="b_heading">Binding free energy analysis and identification of hot spots</span></h3>
<span>The structure and function of proteins are extremely sensitive to the surrounding environment. For biomolecular in aqueous environments, a general molecular mechanics Poisson–Boltzmann surface area (MM/PBSA) approach is used, which replaced the explicit solvent with an implicit continuous solvent. However, for membrane proteins, the effect of the membrane environment must be properly considered to ensure accuracy. In this work, the implicit membrane model developed by Luo <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> was used to examine the binding free energy between the antennas and the PSII core complex. Unfortunately, the implicit membrane model does not support energy decomposition. So, the classical implicit solvent model approach was also applied.</span>
<p class="otherpara">The MM/PBSA method calculates the binding free energy of the complex (Δ<span class="italic">G</span><small><sub>bind</sub></small>) by eqn <a title="" href="#eqn1">(1)</a>. In eqn <a title="" href="#eqn1">(1)</a>, −<span class="italic">T</span>ΔS is the contribution of entropy to the system's free energy, which mainly involves the energy changes caused by conformational changes such as translation and rotation of the protein. Given that there is no significant change in protein subunits conformation, and the analysis mainly focuses on the electrostatic and hydrophobic contributions of the system rather than the absolute binding free energy, the contribution of the entropy (−<span class="italic">T</span>ΔS) was not calculated as that in recent literature.<a title="Select to navigate to reference" href="#cit48"><sup><span class="sup_ref">48–50</span></sup></a> Δ<span class="italic">G</span><small><sub>gas</sub></small> is the sum of the electrostatic interaction energy Δ<span class="italic">G</span><small><sub>elec</sub></small> and the van der Waals interaction energy Δ<span class="italic">G</span><small><sub>vdW</sub></small> in vacuum (eqn <a title="" href="#eqn2">(2)</a>), representing the contribution of the molecular potential energy. The solvation energy includes the electrostatic solvation energy Δ<span class="italic">G</span><small><sub>PB</sub></small> and the non-polar solvation energy Δ<span class="italic">G</span><small><sub>np</sub></small> (eqn <a title="" href="#eqn3">(3)</a>).<table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn1"/>
<span id="eqn1">Δ<span class="italic">G</span><small><sub>bind</sub></small> = Δ<span class="italic">G</span><small><sub>gas</sub></small> + Δ<span class="italic">G</span><small><sub>sol</sub></small> − <span class="italic">T</span>ΔS</span>
</td><td class="rightEqn">(1)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn2"/>
<span id="eqn2">Δ<span class="italic">G</span><small><sub>gas</sub></small> = Δ<span class="italic">G</span><small><sub>elec</sub></small> + Δ<span class="italic">G</span><small><sub>vdW</sub></small></span>
</td><td class="rightEqn">(2)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn3"/>
<span id="eqn3">Δ<span class="italic">G</span><small><sub>sol</sub></small> = Δ<span class="italic">G</span><small><sub>PB</sub></small> + Δ<span class="italic">G</span><small><sub>np</sub></small></span>
</td><td class="rightEqn">(3)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn4"/>
<span id="eqn4"><span class="italic">G</span><small><sub>np</sub></small> = <span class="italic">γ</span> × SASA + <span class="italic">b</span></span>
</td><td class="rightEqn">(4)</td></tr></table></p>
<p class="otherpara">For the solvated PSII-LHCII complexes with lipid bilayers, the binding free energy of the protein–protein interfaces was calculated using the MM/PBSA approach with the MMPBSA.py module in AmberTools20. For the implicit membrane model, the heterogeneous dielectric membrane model is used here because it describes the membrane environment more accurately than the single dielectric membrane model.<a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> The spline fitting was adopted, and the implicit membrane thickness was obtained by calculating the average explicit membrane thickness of the last 800 ns trajectory. This was accomplished by calculating the location of the center of mass of the N and P atoms in the phosphatidylcholine headgroups at the top of the membrane, performing the same calculation on the bottom of the membrane, and taking the difference between them. Periodic boundary conditions were used here so that the value of the electrostatic solvation energy Δ<span class="italic">G</span><small><sub>PB</sub></small> is always 0, as it is part of the vacuum electrostatic interaction energy Δ<span class="italic">G</span><small><sub>elec</sub></small>. Other PB settings are consistent with Greene <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit47"><sup><span class="sup_ref">47</span></sup></a> The internal dielectric constant settings for protein in the implicit membrane model and the implicit solvent model will be discussed in the next section. Δ<span class="italic">G</span><small><sub>np</sub></small> is calculated by eqn <a title="" href="#eqn4">(4)</a>, where the values of constant <span class="italic">γ</span> and constant <span class="italic">b</span> are set to 0.005 kcal mol<small><sup>−1</sup></small> Å<small><sup>−2</sup></small> and 0.92 kcal mol<small><sup>−1</sup></small>, respectively. Solvent accessible surface area (SASA) was calculated using a water probe radius of 1.4 Å.</p>
<p class="otherpara">The free energy contributions of protein residues could be divided into polar (Δ<span class="italic">G</span><small><sub>polar</sub></small>) and nonpolar (Δ<span class="italic">G</span><small><sub>nonpolar</sub></small>) interactions according to eqn <a title="" href="#eqn5">(5)</a> where each part is the sum of two energy terms, as shown in eqn <a title="" href="#eqn6">(6)</a> and <a title="" href="#eqn7">(7)</a>. In the following analysis, Δ<span class="italic">G</span><small><sub>polar</sub></small> is considered as the contribution of electrostatic interactions, while Δ<span class="italic">G</span><small><sub>nonpolar</sub></small> is considered as the contribution of hydrophobic interactions.<a title="Select to navigate to reference" href="#cit48"><sup><span class="sup_ref">48,49</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn5"/>
<span id="eqn5">Δ<span class="italic">G</span><small><sub>bind</sub></small> = Δ<span class="italic">G</span><small><sub>polar</sub></small> + Δ<span class="italic">G</span><small><sub>nonpolar</sub></small></span>
</td><td class="rightEqn">(5)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn6"/>
<span id="eqn6">Δ<span class="italic">G</span><small><sub>polar</sub></small> = Δ<span class="italic">G</span><small><sub>elec</sub></small> + Δ<span class="italic">G</span><small><sub>PB</sub></small></span>
</td><td class="rightEqn">(6)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn7"/>
<span id="eqn7">Δ<span class="italic">G</span><small><sub>nonpolar</sub></small> = Δ<span class="italic">G</span><small><sub>vdW</sub></small> + Δ<span class="italic">G</span><small><sub>np</sub></small></span>
</td><td class="rightEqn">(7)</td></tr></table></p>
<p class="otherpara">According to the previous literature,<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> the hot spots are identified as those whose absolute free energy value is larger than 1.5 kcal mol<small><sup>−1</sup></small>. 8000 snapshots were extracted at intervals of 100 ps along the trajectory. It should be mentioned that since the PSII-LHCII complex is a dimer, the binding free energy values are discussed using the average of the two monomers.</p>


<h2 id="sect488"><span class="a_heading">Results</span></h2>
<h3 id="sect490"><span class="b_heading">Construction of the structural model of PSII-LHCII for MD simulations</span></h3>
<span>The structures of several PSII-LHCII complexes from different species have been solved.<a title="Select to navigate to reference" href="#cit12"><sup><span class="sup_ref">12–14,52</span></sup></a> In this work, the cryo-EM structure from spinach PSII-LHCII (PDB ID: <a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>) was selected as the initial model for simulating the interactions between antenna proteins and the PSII core.<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> As shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>, the spinach C<small><sub>2</sub></small>S<small><sub>2</sub></small>-type PSII-LHCII supercomplex forms a homodimer with two-fold symmetry. Each monomer comprises a PSII core, a major peripheral light-harvesting complex (LHCII), and two minor chlorophyll-binding proteins of 26 and 29 kDa (CP26, CP29). The PSII core contains four large intrinsic subunits (D1, D2, CP43 and CP47), four extrinsic subunits (PsbO, PsbP, PsbQ, PsbTn) on the luminal surface (Fig. S1<a title="Select to navigate to footnote" href="#fn1">†</a>), and twelve small membrane-spanning subunits (PsbE, PsbF, PsbH, PsbI, PsbJ, PsbK, PsbL, PsbM, PsbTc, PsbW, PsbX and PsbZ). Three light-harvesting complexes are bound to the core complex through the LHCII–core, CP26–core, and CP29–core interfaces, which display different protein contacts. Here, we define the three interfaces as <span class="italic">S</span><small><sub>LHCII/core</sub></small>, <span class="italic">S</span><small><sub>CP26/core</sub></small>, and <span class="italic">S</span><small><sub>CP29/core</sub></small>, respectively. In addition, LHCII also forms interfaces with CP26 and CP29, defined as <span class="italic">S</span><small><sub>LHCII/CP26</sub></small> and <span class="italic">S</span><small><sub>LHCII/CP29</sub></small>, respectively. This PSII-LHCII structure was embedded in a pre-equilibrated lipid bilayer consisting of single-component 1-palmitoyl-2-oleoyl-<span class="italic">sn-glycero</span>-3-phosphocholine (POPC) to mimic native thylakoid membranes (Fig. S1<a title="Select to navigate to footnote" href="#fn1">†</a>).<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31–33</span></sup></a> The model construction and the MD simulation process of the PSII-LHCII complex are shown in ESI Video 1.<a title="Select to navigate to footnote" href="#fn1">†</a></span>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2023/RA/d2ra08240c/d2ra08240c-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ra08240c-f1.tif" src="/image/article/2023/RA/d2ra08240c/d2ra08240c-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">Validation of the quality of MD simulation. (A) Stromal view on the PSII-LHCII dimer. For clarity, only the subunits of the right monomer are highlighted, and different subunits are colored and labeled differently (the four extrinsic subunits are not shown). Among them, the 4 large core subunits are shown using the surface model and the remaining parts are shown using the cartoon model. The positions of three antennae–core interfaces are indicated in the left monomer. (B and C) Ramachandran plots for cryo-EM and MD structures, respectively. (D and E) RMSD values of proteins associated with the antennae–core interfaces (for CP26 and PsbW, only the RMSDs of non-loop regions were considered due to the high volatility of the loop regions). The RMSDs are obtained using the average of the two monomers and are colored as in (A).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">The conformations from the 1 μs MD production run were compared to the cryo-EM structure to check the quality of MD simulations. The average structure was obtained over a 1 μs trajectory, and the snapshot with the smallest RMSD value from the average structure was adopted as the representative structure. We found that the representative structure has only slight differences with the cryo-EM structure (Fig. S2<a title="Select to navigate to footnote" href="#fn1">†</a>). First, Ramachandran plot analysis from PROCHECK<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> revealed that the MD structure has a wider conformational distribution space and most of the backbone in the allowed regions of Ramachandran plot, compared with the cryo-EM structure (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1B and C</a>). The number of residues in disallowed regions was reduced from 0.6% to 0.2%, and the number in generously allowed regions was reduced from 0.6% to 0.3% (<a title="Select to navigate to table" href="#tab1">Table 1</a>). The number of residues in the most favored regions was similar (88.1% for MD structure and 88.3% for cryo-EM structure). These results confirmed the good quality of the cryo-EM structure and indicated that MD simulations only slightly improved the stereochemistry of the cryo-EM structure. Then, we examined the quality of nonbonded interactions using ERRAT program.<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a> The structures were evaluated based on the quality factor, which depends on the statistics of nonbonded atomic interactions in the 3-D protein structure. The results showed that the overall quality factor of the MD structure (90.2%) was greater than that of the cryo-EM structure (79.8%) (<a title="Select to navigate to table" href="#tab1">Table 1</a>), indicating that MD simulations did improve the quality of nonbonded interactions from the PSII-LHCII cryo-EM structure. Since the antenna–core binding mainly depends on nonbonded interactions between the protein interfaces, the improvement in the accuracy of nonbonded interactions here is very important for the subsequent accurate calculation of protein interface affinities.</p>

<div class="table_caption"><b>Table 1</b> <span id="tab1">Validation of cryo-EM and MD structures using PROCHECK and ERRAT program</span>
		</div>

<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/></colgroup>






<thead align="left">
<tr align="left" valign="bottom">
<th rowspan="2" align="left" valign="bottom" class="border_black">Structure</th>
<th colspan="4" align="left" valign="bottom" class="border_black">Ramachandran plot statistics (%)</th>
<th align="left" valign="bottom" class="border_black">ERRAT (%)</th>
</tr>
<tr align="left" valign="bottom">
<th align="left" valign="bottom" class="border_black">Most favored</th>
<th align="left" valign="bottom" class="border_black">Additionally allowed</th>
<th align="left" valign="bottom" class="border_black">Generously allowed</th>
<th align="left" valign="bottom" class="border_black">Disallowed</th>
<th align="left" valign="bottom" class="border_black">Overall quality factor</th>
</tr>
</thead>
<tbody align="left">
<tr align="left">
<td valign="top">Cryo-EM</td>
<td align="char" char="." valign="top">88.3</td>
<td align="char" char="." valign="top">10.5</td>
<td align="char" char="." valign="top">0.6</td>
<td align="char" char="." valign="top">0.6</td>
<td align="char" char="." valign="top">79.8</td>
</tr>
<tr align="left">
<td valign="top">MD</td>
<td align="char" char="." valign="top">88.1</td>
<td align="char" char="." valign="top">11.4</td>
<td align="char" char="." valign="top">0.3</td>
<td align="char" char="." valign="top">0.2</td>
<td align="char" char="." valign="top">90.2</td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><br/>

<h3 id="sect559"><span class="b_heading">Conformational fluctuation analysis</span></h3>
<span>To verify whether the antenna–core interfaces reach equilibrium, we calculated the RMSD values of three antennas (LHCII, CP29, CP26) and seven PSII core subunits (4 large intrinsic subunits D1, D2, Cp43, Cp47, and 3 small intrinsic subunits PsbW, PsbZ, PsbH). As shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1D and E</a>, after 200 ns of simulation, the RMSD values of all the proteins at the antenna–core interfaces remained consistent, indicating that the <span class="italic">S</span><small><sub>LHCII/core</sub></small>, <span class="italic">S</span><small><sub>CP26/core</sub></small>, and <span class="italic">S</span><small><sub>CP29/core</sub></small> interfaces reached stable states. Likewise, RMSD analysis showed that the pigments also reached equilibrium (Fig. S3<a title="Select to navigate to footnote" href="#fn1">†</a>). Accordingly, the MD simulation trajectories after 200 ns were used for subsequent statistical analysis.</span>
<p class="otherpara">The RMSD values of the peripheral antennas LHCII, CP29, and CP26 (ranging from 1.50 Å to 3.07 Å, <a title="Select to navigate to figure" href="#imgfig1">Fig. 1D</a>) were generally more significant than those of the core proteins (ranging from 0.58 Å to 2.31 Å, <a title="Select to navigate to figure" href="#imgfig1">Fig. 1E</a>), suggesting that the light-harvesting antennas possess larger conformational fluctuations than the PSII core subunits, consistent with the experimentally observed high mobility of antenna proteins.<a title="Select to navigate to references" href="#cit10"><sup><span class="sup_ref">10</span></sup></a> To further characterize the fluctuation features of protein residues, we investigated root mean square fluctuation (RMSF) of the protein backbone C<small><sub>α</sub></small> atoms around their average positions. The volatility of extrinsic subunits (average RMSF value of 1.91 Å, <a title="Select to navigate to figure" href="#imgfig2">Fig. 2A</a>) and antenna proteins (average RMSF value of 1.71 Å, <a title="Select to navigate to figure" href="#imgfig2">Fig. 2B</a>) was markedly higher than that of the core complex (average RMSF value of 1.16 Å, <a title="Select to navigate to figure" href="#imgfig2">Fig. 2B</a>), consistent with RMSD results (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>). In addition, the fluctuation of pigments was strongly correlated with the fluctuation of binding proteins (Fig. S4<a title="Select to navigate to footnote" href="#fn1">†</a>). For individual antennas, CP26 has a notably higher RMSF value (average value: 2.22 Å) than LHCII monomer adjacent to the core (LHCII(A), average value: 1.36 Å) and CP29 (average value: 1.45 Å). The greater fluctuation was mainly ascribed to the loop regions of CP26, which is away from the <span class="italic">S</span><small><sub>CP26/core</sub></small> interface; likewise, the fluctuating regions of CP29 and LHCII(A) are also not at the antenna–core interfaces (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2C–E</a>). Consequently, although the peripheral antennas showed overall conformational fluctuations, <span class="italic">S</span><small><sub>LHCII/core</sub></small>, <span class="italic">S</span><small><sub>CP26/core</sub></small>, and <span class="italic">S</span><small><sub>CP29/core</sub></small> exhibited relatively stable conformations (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2B</a>), presumably favoring the association of antennas with the PSII core and energy transfer. The analysis also verified the reliability of MD simulations using the constructed structural model.</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2023/RA/d2ra08240c/d2ra08240c-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ra08240c-f2.tif" src="/image/article/2023/RA/d2ra08240c/d2ra08240c-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">RMSF fluctuations of the PSII-LHCII backbone. (A) Side view and (B) stromal view on the PSII-LHCII dimer. All protein residues were colored according to the RMSF values of C<small><sub>α</sub></small> atoms capped at 2.5 Å: red represents a high RMSF value and blue represents a small RMSF value. (C–E) RMSF values of LHCII(A), CP29 and CP26 residues, respectively. Grey boxes and arrows indicate the regions with higher fluctuation values.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>

<h3 id="sect593"><span class="b_heading">Analysis of the interface binding free energies</span></h3>
<span>To study the binding characteristics at different antenna–core interfaces, the binding free energies of the three antenna–core interfaces (<span class="italic">S</span><small><sub>LHCII/core</sub></small>, <span class="italic">S</span><small><sub>CP26/core</sub></small>, and <span class="italic">S</span><small><sub>CP29/core</sub></small>) were calculated using the MM/PBSA approach.<a title="Select to navigate to reference" href="#cit48"><sup><span class="sup_ref">48,51,55</span></sup></a> As the choice of internal dielectric constant values has a significant impact on the results,<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> we examined a set of values using <span class="italic">S</span><small><sub>LHCII/core</sub></small> as an example (Table S2<a title="Select to navigate to footnote" href="#fn1">†</a>) to determine an appropriate internal dielectric constant. For the implicit membrane model,<a title="Select to navigate to reference" href="#cit47"><sup><span class="sup_ref">47,57</span></sup></a> a value of 20 is appropriate when the ligand is charged, according to recommended settings in the literature.<a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a> Therefore, we tested values around 20, including 10, 15, 20, 25, and 30. The change in binding free energy was examined with internal dielectric constant values of 1–8, often used for the implicit solvent model.<a title="Select to navigate to reference" href="#cit58"><sup><span class="sup_ref">58,59</span></sup></a> The results showed that both models' free energy decreased with the internal dielectric constant increased, and only the electrostatic interaction energy changed (Table S2<a title="Select to navigate to footnote" href="#fn1">†</a>). In addition, the electrostatic interaction energy was always positive, whereas the contribution of hydrophobic interactions remained negative. These results suggest that the choice of internal dielectric constants values does not have remarkable effects on qualitative determination of binding affinity. Therefore, the common setting, namely the internal dielectric constant value of 20 for the implicit membrane model and 1 for the implicit solvent model, was used to investigate the binding free energy of protein interfaces.</span>
<p class="otherpara">Based on the heterogeneous implicit membrane model, free energies of the three antenna–core protein interfaces were obtained from equilibrium trajectories (<span class="italic">i.e.</span>, 200–1000 ns) (<a title="Select to navigate to table" href="#tab2">Table 2</a>). To verify the convergence of the free energy calculations, we performed convergence tests for <span class="italic">S</span><small><sub>LHCII/core</sub></small> (−58.0 ± 7.2 kcal mol<small><sup>−1</sup></small>, −57.0 ± 7.9 kcal mol<small><sup>−1</sup></small>, −59.4 ± 7.2 kcal mol<small><sup>−1</sup></small>, −56.4 ± 13.6 kcal mol<small><sup>−1</sup></small>), <span class="italic">S</span><small><sub>CP26/core</sub></small> (2.3 ± 5.5 kcal mol<small><sup>−1</sup></small>, 1.5 ± 5.6 kcal mol<small><sup>−1</sup></small>, 2.0 ± 5.6 kcal mol<small><sup>−1</sup></small>, 3.2 ± 6.4 kcal mol<small><sup>−1</sup></small>) and <span class="italic">S</span><small><sub>CP29/core</sub></small> (−111.6 ± 9.6 kcal mol<small><sup>−1</sup></small>, −112.7 ± 9.7 kcal mol<small><sup>−1</sup></small>, −116.1 ± 9.4 kcal mol<small><sup>−1</sup></small>, −116.1 ± 9.8 kcal mol<small><sup>−1</sup></small>) using trajectories of 800–1000 ns, 600–1000 ns, 400–1000 ns and 200–1000 ns, respectively. In addition, non-equilibrium trajectories (0–50 ns) were also used to calculate the binding free energies of <span class="italic">S</span><small><sub>LHCII/core</sub></small> (−2.7 ± 14.8 kcal mol<small><sup>−1</sup></small>), <span class="italic">S</span><small><sub>CP26/core</sub></small> (12.2 ± 5.1 kcal mol<small><sup>−1</sup></small>) and <span class="italic">S</span><small><sub>CP29/core</sub></small> (−84.5 ± 8.7 kcal mol<small><sup>−1</sup></small>), the results are clearly different from the above calculations using equilibrium trajectories, indicating that a long simulation scale is necessary for accurate free energy calculations of the antenna–core interface.</p>

<div class="table_caption"><b>Table 2</b> <span id="tab2">Binding free energy of different antenna–core interfaces (kcal mol<small><sup>−1</sup></small>). Δ<span class="italic">G</span><small><sub>vdW</sub></small>, the van der Waals interaction energy; Δ<span class="italic">G</span><small><sub>np</sub></small>, the non-polar solvation energy; Δ<span class="italic">G</span><small><sub>elec</sub></small>, the electrostatic interaction energy; Δ<span class="italic">G</span><small><sub>nonpolar</sub></small>, the contribution of hydrophobic interactions; Δ<span class="italic">G</span><small><sub>polar</sub></small>, the contribution of electrostatic interactions; Δ<span class="italic">G</span><small><sub>bind</sub></small>, the binding free energy</span>
		</div>

<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/></colgroup>







<thead align="left">
<tr align="left" valign="bottom">
<th align="left" valign="bottom" class="border_black">Interface</th>
<th align="left" valign="bottom" class="border_black">Δ<span class="italic">G</span><small><sub>vdW</sub></small></th>
<th align="left" valign="bottom" class="border_black">Δ<span class="italic">G</span><small><sub>np</sub></small></th>
<th align="left" valign="bottom" class="border_black">Δ<span class="italic">G</span><small><sub>elec</sub></small><i><small><sup style="color: red;"><a title="" href="#tab2fna">a</a></sup></small></i></th>
<th align="left" valign="bottom" class="border_black">Δ<span class="italic">G</span><small><sub>nonpolar</sub></small><i><small><sup style="color: red;"><a title="" href="#tab2fnb">b</a></sup></small></i></th>
<th align="left" valign="bottom" class="border_black">Δ<span class="italic">G</span><small><sub>polar</sub></small><i><small><sup style="color: red;"><a title="" href="#tab2fnc">c</a></sup></small></i></th>
<th align="left" valign="bottom" class="border_black">Δ<span class="italic">G</span><small><sub>bind</sub></small><i><small><sup style="color: red;"><a title="" href="#tab2fnd">d</a></sup></small></i></th>
</tr>
</thead>
<tfoot align="left">
<tr align="left">
<th colspan="7"><a id="tab2fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Includes the electrostatic solvation energy, see methods.</span><a id="tab2fnb"><span class="tfootnote">b</span></a> <span class="sup_inf">Formula <a title="" href="#eqn6">(6)</a>.</span><a id="tab2fnc"><span class="tfootnote">c</span></a> <span class="sup_inf">Formula <a title="" href="#eqn7">(7)</a>.</span><a id="tab2fnd"><span class="tfootnote">d</span></a> <span class="sup_inf">Formula <a title="" href="#eqn5">(5)</a>.</span></th>
</tr>
</tfoot>
<tbody align="left">
<tr align="left">
<td valign="top"><span class="italic">S</span><small><sub>LHCII/core</sub></small></td>
<td align="char" char="xb1" valign="top">−103.5 ± 12.5</td>
<td align="char" char="xb1" valign="top">−11.4 ± 1.0</td>
<td align="char" char="xb1" valign="top">58.5 ± 2.5</td>
<td align="char" char="." valign="top">−114.9</td>
<td align="char" char="." valign="top">58.5</td>
<td align="char" char="xb1" valign="top">−56.4 ± 13.6</td>
</tr>
<tr align="left">
<td valign="top"><span class="italic">S</span><small><sub>CP26/core</sub></small></td>
<td align="char" char="xb1" valign="top">−46.5 ± 5.3</td>
<td align="char" char="xb1" valign="top">−6.2 ± 0.5</td>
<td align="char" char="xb1" valign="top">55.9 ± 3.0</td>
<td align="char" char="." valign="top">−52.7</td>
<td align="char" char="." valign="top">55.9</td>
<td align="char" char="xb1" valign="top">3.2 ± 6.4</td>
</tr>
<tr align="left">
<td valign="top"><span class="italic">S</span><small><sub>CP29/core</sub></small></td>
<td align="char" char="xb1" valign="top">−167.4 ± 8.5</td>
<td align="char" char="xb1" valign="top">−18.5 ± 0.7</td>
<td align="char" char="xb1" valign="top">69.8 ± 3.8</td>
<td align="char" char="." valign="top">−185.9</td>
<td align="char" char="." valign="top">69.8</td>
<td align="char" char="xb1" valign="top">−116.1 ± 9.8</td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><br/>
<p class="otherpara"><span class="italic">S</span><small><sub>CP29/core</sub></small> (−116.1 kcal mol<small><sup>−1</sup></small>) has the largest total free energy among the three interfaces, which is double of <span class="italic">S</span><small><sub>LHCII/core</sub></small> (−56.4 kcal mol<small><sup>−1</sup></small>), indicating the strong binding affinity at <span class="italic">S</span><small><sub>CP29/core</sub></small>. The electrostatic interaction energies (Δ<span class="italic">G</span><small><sub>elec</sub></small>) of <span class="italic">S</span><small><sub>LHCII/core</sub></small>, <span class="italic">S</span><small><sub>CP26/core</sub></small>, and <span class="italic">S</span><small><sub>CP29/core</sub></small> were 58.5 kcal mol<small><sup>−1</sup></small>, 55.9 kcal mol<small><sup>−1</sup></small> and 69.8 kcal mol<small><sup>−1</sup></small>. In contrast, their hydrophobic interaction energies (Δ<span class="italic">G</span><small><sub>np</sub></small>) were −114.9 kcal mol<small><sup>−1</sup></small>, −52.7 kcal mol<small><sup>−1</sup></small>, and −185.9 kcal mol<small><sup>−1</sup></small>, respectively. As described in Methods (eqn <a title="" href="#eqn6">(6)</a> and <a title="" href="#eqn7">(7)</a>), the binding free energies can be divided into Δ<span class="italic">G</span><small><sub>polar</sub></small> and Δ<span class="italic">G</span><small><sub>nonpolar</sub></small> (hydrophobic interaction). Our analysis suggested that Δ<span class="italic">G</span><small><sub>nonpolar</sub></small> provides the dominant contribution to the binding of the antenna to the PSII core. Meanwhile, the Δ<span class="italic">G</span><small><sub>polar</sub></small> values of all interfaces are positive, indicating that polar or electrostatic interactions between the antenna and PSII provide repulsive forces unfavorable for binding. This conclusion was also supported by the calculation using the implicit water model (Table S3<a title="Select to navigate to footnote" href="#fn1">†</a>). In contrast, the binding free energies of the two antenna–antenna interfaces, <span class="italic">S</span><small><sub>LHCII/CP26</sub></small> and <span class="italic">S</span><small><sub>LHCII/CP29,</sub></small> were both positive values (Table S4<a title="Select to navigate to footnote" href="#fn1">†</a>), indicative of the very weak antenna–antenna association (<a title="Select to navigate to table" href="#tab2">Table 2</a>).</p>

<h3 id="sect773"><span class="b_heading">Free energy decomposition and hot spot analysis</span></h3>
<span>Based on the identification of hydrogen bonds (HBs) and salt bridges (SBs) at the antenna–core interface in the cryo-EM structure,<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> electrostatic interactions were previously assumed to be the main driving force mediating the binding of antennae to the PSII core.<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> However, our free energy calculation indicated that the total electrostatic interactions act as repulsive forces. To address this contradiction, we examined the contributions of individual residues to the total free energies and HB/SB interactions formed by the interface residues. The free energy contributions were decomposed into polar and non-polar fractions to evaluate the contributions of electrostatic and hydrophobic interactions. Residues with an absolute value of binding free energy greater than 1.5 kcal mol<small><sup>−1</sup></small> were identified as hot spots (Tables S5 and S6<a title="Select to navigate to footnote" href="#fn1">†</a>), according to previous work.<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a></span>
<p class="otherpara">For <span class="italic">S</span><small><sub>LHCII/core</sub></small>, 23 amino acids were identified as hot spots (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3E</a> and Table S5<a title="Select to navigate to footnote" href="#fn1">†</a>). Among them, 20 hot spots had negative free energy values, and their total free energy contribution was −42.22 kcal mol<small><sup>−1</sup></small>, accounting for 65% of the total binding free energies (−64.5 kcal mol<small><sup>−1</sup></small>). Three residues in LHCII, E175, K179, and D215, have positive free energy contributions. All Δ<span class="italic">G</span><small><sub>nonpolar</sub></small> of residues are negative, but Δ<span class="italic">G</span><small><sub>polar</sub></small> of 13 residues are positive. For <span class="italic">S</span><small><sub>CP26/core</sub></small>, only 5 amino acids were identified as hot spots (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3E</a> and Table S5<a title="Select to navigate to footnote" href="#fn1">†</a>), consistent with the relatively low interface affinity (<a title="Select to navigate to table" href="#tab2">Table 2</a>). Interestingly, Δ<span class="italic">G</span><small><sub>polar</sub></small> of most of the residues are positive (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3E</a> and Table S5<a title="Select to navigate to footnote" href="#fn1">†</a>). A large number of hot spots at <span class="italic">S</span><small><sub>CP29/core</sub></small> (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3J</a> and Table S6<a title="Select to navigate to footnote" href="#fn1">†</a>), compared with those at <span class="italic">S</span><small><sub>LHCII/core</sub></small> and <span class="italic">S</span><small><sub>CP26/core</sub></small>, is consistent with its higher interface affinity (<a title="Select to navigate to table" href="#tab2">Table 2</a>). 26 of the 29 hot spots contribute negative free energies. Their total free energy contribution was −79.54 kcal mol<small><sup>−1</sup></small>, accounting for 68% of the total binding free energy. In contrast, 19 hot spots have positive Δ<span class="italic">G</span><small><sub>polar</sub></small> contribution. The free energy decomposition indicated that the Δ<span class="italic">G</span><small><sub>polar</sub></small> contribution (<span class="italic">i.e.</span> electrostatic interactions) is primarily positive, representing unfavorable interactions, consistent with our free energy analysis (<a title="Select to navigate to table" href="#tab2">Table 2</a>). It should be noted that the residues with hydrogen bonds may contribute negative Δ<span class="italic">G</span><small><sub>polar</sub></small>; we will discuss it in the next section.</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2023/RA/d2ra08240c/d2ra08240c-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ra08240c-f3.tif" src="/image/article/2023/RA/d2ra08240c/d2ra08240c-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">Binding free energy and corresponding component contribution of hot spots on <span class="italic">S</span><small><sub>LHCII/core</sub></small>, <span class="italic">S</span><small><sub>CP26/core</sub></small> and <span class="italic">S</span><small><sub>CP29/core</sub></small> and their localization. (A and B) Localization of hot spots on LHCII and the core complex (at the <span class="italic">S</span><small><sub>LHCII/core</sub></small>), respectively. (C and D) Localization of hot spots on CP26 and the core complex (at the <span class="italic">S</span><small><sub>CP26/core</sub></small>), respectively. The protein surface map colored according to the free energy contribution of each hot spot. The yellow dotted lines in panels (B) and (D) correspond to the positions of the small intrinsic subunits PsbH and PsbZ, respectively. The unit of energy is kcal mol<small><sup>−1</sup></small>. (E) Binding free energy and corresponding component contribution of hot spots, including hydrophobic (red), electrostatic (blue) and total (yellow) contribution. Color scheme is presented as the same as shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>. The dotted box lines with different colors represent different subunits, which are CP43 (tan), PsbW (purple), LHCII (lime), PsbZ (silver) and CP26 (pink). Hot spots that participate in the formation of hydrogen bonds or salt bridges are marked with *. See Table S5<a title="Select to navigate to footnote" href="#fn1">†</a> for more details. (F and H) Localization of CP29 and corresponding hot spots. (G and I) Are the localization of core complex (at the <span class="italic">S</span><small><sub>CP29/core</sub></small>) and corresponding hot spots. The yellow dotted lines in panel (i) correspond to the positions of the small intrinsic subunits PsbH. (J) Binding free energy and corresponding component contribution of hot spots. The dotted box lines with different colors represent different subunits, which are CP47 (ochre), D1 (ice blue), PsbH (mauve), PsbL (green), and CP29 (cyan), and the colors correspond to <a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>. See Table S6<a title="Select to navigate to footnote" href="#fn1">†</a> for more details.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">We also determined the spatial distribution of hot spots. At <span class="italic">S</span><small><sub>LHCII/core</sub></small>, 12 hot spots are located in LHCII, including 4 hot spots (L164, L166, E175, K179) at the stromal side and 8 hot spots (F81, L113, L85, V90, G89, L213, P216, D215) at the luminal side. Eleven hot spots are located in the core (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3A and B</a>), including 3 hot spots (L127, G126, and S125) at the stromal side and 8 near the luminal side. At <span class="italic">S</span><small><sub>CP26/core</sub></small>, 3 hot spots are located at CP26, and 1 hot spot is in the core complex, all of which are near the luminal side or the stromal side (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3C and D</a>). At <span class="italic">S</span><small><sub>CP29/core</sub></small>, all hot spots are concentrated at the stromal side: 16 hot spots were localized in the core complex, and 13 were localized at CP29 (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3F–I</a>). Moreover, the hot spots in the core complex are widely distributed in multiple protein chains: 7 of the 16 hot spots are located at PsbH, 4 hot spots are at CP47, 3 hot spots are at D1, and the remaining one is localized at PsbL (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3I</a>). At CP29, all the 13 hot spots are localized at Motif II (Pro42–Phe87) of the N-terminal long loop chain, implying its role of CP29–core complex binding. Collectively, the hot spots are mainly distributed at the luminal or stromal side, indicating that the main driving force in these interfaces' binding originates from the luminal or stromal side.</p>

<h3 id="sect849"><span class="b_heading">Hydrogen bonds/salt bridges across the antenna–core interfaces</span></h3>
<span>To further determine the effects of electrostatic interactions on the interfacial affinity, HBs/SBs at the antenna–core interfaces were analyzed. The distance for determining the formation of HBs was set to 3.5 Å, and the corresponding angle was 135°.<a title="Select to navigate to reference" href="#cit32"><sup><span class="sup_ref">32,61</span></sup></a> SB analysis was performed using a homemade python script, and a cutoff distance of 6 Å was used to detect SBs between the basic nitrogen and acidic oxygens.<a title="Select to navigate to references" href="#cit62"><sup><span class="sup_ref">62</span></sup></a> HBs/SBs with an occupancy greater than 30% during MD simulations were considered stable. In addition to the HBs/SBs characterized in the cryo-EM structure (Table S7<a title="Select to navigate to footnote" href="#fn1">†</a>), 25 additional HB/SB interactions were identified between the antenna–core interfaces during the course of 1 μs MD simulations at room temperature (<a title="Select to navigate to table" href="#tab3">Tables 3</a> and <a title="Select to navigate to table" href="#tab4">4</a>). Intriguingly, many of these HBs/SBs have been reported in other PSII-LHCII structures at higher resolutions,<a title="Select to navigate to reference" href="#cit13"><sup><span class="sup_ref">13,52</span></sup></a> confirming the reliability of the structural model achieved from MD simulations.</span>

<div class="table_caption"><b>Table 3</b> <span id="tab3">Hydrogen bonds and salt bridges at <span class="italic">S</span><small><sub>LHCII/core</sub></small> and <span class="italic">S</span><small><sub>CP26/core</sub></small></span>
		</div>

<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/></colgroup>







<thead align="left">
<tr align="left" valign="bottom">
<th rowspan="2"> </th>
<th colspan="2" align="left" valign="bottom" class="border_black">Hydrogen bond</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Distance (Å)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Occupancy (%)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">PSII subunits</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Cryo-EM structures<i><small><sup style="color: red;"><a title="" href="#tab3fna">a</a></sup></small></i></th>
</tr>
<tr align="left" valign="bottom">
<th align="left" valign="bottom" class="border_black">Donor</th>
<th align="left" valign="bottom" class="border_black">Acceptor</th>
</tr>
</thead>
<tfoot align="left">
<tr align="left">
<th colspan="7"><a id="tab3fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Interfacial HBs/SBs present in the cryo-EM structure of spinach (PDB ID: <a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, 3.2 Å), pea (PDB ID: <a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a>, 2.7 Å) and arabidopsis (PDB ID: <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a>, 2.79 Å).</span></th>
</tr>
</tfoot>
<tbody align="left">
<tr align="left">
<td rowspan="9" valign="top"><span class="italic">S</span><small><sub>LHCII/core</sub></small></td>
<td valign="top">VAL_90@N</td>
<td valign="top">ASN_103@Oδ</td>
<td align="char" char="xb1" valign="top">2.96 ± 0.16</td>
<td align="char" char="." valign="top">64</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASN_88@Nδ</td>
<td valign="top">PRO_97@O</td>
<td align="char" char="xb1" valign="top">2.88 ± 0.14</td>
<td align="char" char="." valign="top">38</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASN_103@Nδ</td>
<td valign="top">LEU_113@O</td>
<td align="char" char="xb1" valign="top">2.92 ± 0.15</td>
<td align="char" char="." valign="top">95</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">SER_205@N</td>
<td valign="top">ALA_214@O</td>
<td align="char" char="xb1" valign="top">2.92 ± 0.15</td>
<td align="char" char="." valign="top">66</td>
<td valign="top">CP43</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASN_103@Nδ</td>
<td valign="top">VAL_90@O</td>
<td align="char" char="xb1" valign="top">2.94 ± 0.15</td>
<td align="char" char="." valign="top">88</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">GLY_126@N</td>
<td valign="top">GLU_175@Oε</td>
<td align="char" char="xb1" valign="top">3.03 ± 0.19</td>
<td align="char" char="." valign="top">88</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">TRP_107@Nε</td>
<td valign="top">TYR_112@Oη</td>
<td align="char" char="xb1" valign="top">3.14 ± 0.17</td>
<td align="char" char="." valign="top">32</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">SER_101@N</td>
<td valign="top">ASN_88@O</td>
<td align="char" char="xb1" valign="top">2.99 ± 0.16</td>
<td align="char" char="." valign="top">37</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">SER_125@Oγ</td>
<td valign="top">GLU_175@Oε</td>
<td align="char" char="xb1" valign="top">2.64 ± 0.11</td>
<td align="char" char="." valign="top">34</td>
<td valign="top">PsbW</td>
<td> </td>
</tr>
<tr align="left">
<td rowspan="4" valign="top"><span class="italic">S</span><small><sub>CP26/core</sub></small></td>
<td valign="top">ARG_32@Nη</td>
<td valign="top">SER_143@O</td>
<td align="char" char="xb1" valign="top">2.90 ± 0.17</td>
<td align="char" char="." valign="top">37</td>
<td valign="top">CP43</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">TRP_33@Nε</td>
<td valign="top">GLY_38@O</td>
<td align="char" char="xb1" valign="top">2.92 ± 0.16</td>
<td align="char" char="." valign="top">61</td>
<td valign="top">PsbZ</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">SER_59@Oγ</td>
<td valign="top">LEU_231@O</td>
<td align="char" char="xb1" valign="top">2.78 ± 0.17</td>
<td align="char" char="." valign="top">31</td>
<td valign="top">PsbZ</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
<tr align="left">
<td valign="top">LYS_37@Nζ</td>
<td valign="top">LEU_39@O</td>
<td align="char" char="xb1" valign="top">2.88 ± 0.15</td>
<td align="char" char="." valign="top">40</td>
<td valign="top">PsbZ</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/><col/></colgroup>







<thead align="left">
<tr align="left" valign="bottom">
<th rowspan="2"> </th>
<th colspan="2" align="left" valign="bottom" class="border_black">Salt bridge</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Distance (Å)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Occupancy (%)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">PSII subunits</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Cryo-EM structures<i><small><sup style="color: red;"><a title="" href="#tab3fna">a</a></sup></small></i></th>
</tr>
<tr align="left" valign="bottom">
<th align="left" valign="bottom" class="border_black">Acidic</th>
<th align="left" valign="bottom" class="border_black">Basic</th>
</tr>
</thead>
<tbody align="left">
<tr align="left">
<td rowspan="2" valign="top"><span class="italic">S</span><small><sub>CP26/core</sub></small></td>
<td valign="top">ASP_41</td>
<td valign="top">LYS_37</td>
<td align="char" char="xb1" valign="top">3.99 ± 0.92</td>
<td align="char" char="." valign="top">47</td>
<td valign="top">PsbZ</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
<tr align="left">
<td valign="top">ASP_44</td>
<td valign="top">LYS_37</td>
<td align="char" char="xb1" valign="top">5.05 ± 1.00</td>
<td align="char" char="." valign="top">42</td>
<td valign="top">PsbZ</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><br/>

<div class="table_caption"><b>Table 4</b> <span id="tab4">Hydrogen bonds and salt bridges at <span class="italic">S</span><small><sub>CP29/core</sub></small></span>
		</div>

<div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/></colgroup>






<thead align="left">
<tr align="left" valign="bottom">
<th colspan="2" align="left" valign="bottom" class="border_black">Hydrogen bond</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Distance (Å)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Occupancy (%)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Proteins in PSII</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Cryo-EM structures<i><small><sup style="color: red;"><a title="" href="#tab4fna">a</a></sup></small></i></th>
</tr>
<tr align="left" valign="bottom">
<th align="left" valign="bottom" class="border_black">Donor</th>
<th align="left" valign="bottom" class="border_black">Acceptor</th>
</tr>
</thead>
<tfoot align="left">
<tr align="left">
<th colspan="6"><a id="tab4fna"><span class="tfootnote">a</span></a> <span class="sup_inf">Interfacial HBs/SBs also present in the cryo-EM structure of spinach (PDB ID: <a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, 3.2 Å), pea (PDB ID: <a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a>, 2.7 Å) and arabidopsis (PDB ID: <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a>, 2.79 Å).</span></th>
</tr>
</tfoot>
<tbody align="left">
<tr align="left">
<td valign="top">TYR_48@N</td>
<td valign="top">GLY_31@O</td>
<td align="char" char="xb1" valign="top">2.94 ± 0.14</td>
<td align="char" char="." valign="top">99</td>
<td valign="top">PsbH</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
<tr align="left">
<td valign="top">ASN_56@Nδ</td>
<td valign="top">GLU_8@O</td>
<td align="char" char="xb1" valign="top">2.92 ± 0.15</td>
<td align="char" char="." valign="top">96</td>
<td valign="top">PsbL</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">GLN_47@Nε</td>
<td valign="top">TYR_30@O</td>
<td align="char" char="xb1" valign="top">2.92 ± 0.16</td>
<td align="char" char="." valign="top">81</td>
<td valign="top">PsbH</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a></td>
</tr>
<tr align="left">
<td valign="top">GLN_55@N</td>
<td valign="top">ASN_7@Oδ</td>
<td align="char" char="xb1" valign="top">2.99 ± 0.18</td>
<td align="char" char="." valign="top">81</td>
<td valign="top">PsbL</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">LEU_46@N</td>
<td valign="top">VAL_33@O</td>
<td align="char" char="xb1" valign="top">2.94 ± 0.14</td>
<td align="char" char="." valign="top">87</td>
<td valign="top">PsbH</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">TYR_50@Oη</td>
<td valign="top">SER_132@Oγ</td>
<td align="char" char="xb1" valign="top">2.85 ± 0.16</td>
<td align="char" char="." valign="top">59</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASN_56@Nδ</td>
<td valign="top">ASN_5@O</td>
<td align="char" char="xb1" valign="top">2.84 ± 0.12</td>
<td align="char" char="." valign="top">56</td>
<td valign="top">PsbL</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASN_56@N</td>
<td valign="top">PRO_6@O</td>
<td align="char" char="xb1" valign="top">2.98 ± 0.16</td>
<td align="char" char="." valign="top">48</td>
<td valign="top">PsbL</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">GLN_55@Nε</td>
<td valign="top">THR_10@O</td>
<td align="char" char="xb1" valign="top">3.03 ± 0.19</td>
<td align="char" char="." valign="top">34</td>
<td valign="top">CP47</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">ASN_60@Nδ</td>
<td valign="top">ARG_476@O</td>
<td align="char" char="xb1" valign="top">2.90 ± 0.14</td>
<td align="char" char="." valign="top">50</td>
<td valign="top">CP47</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
<tr align="left">
<td valign="top">GLN_55@Nε</td>
<td valign="top">ASN_230@Oδ</td>
<td align="char" char="xb1" valign="top">2.97 ± 0.17</td>
<td align="char" char="." valign="top">45</td>
<td valign="top">D1</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">ASN_56@Nδ</td>
<td valign="top">PRO_6@O</td>
<td align="char" char="xb1" valign="top">2.92 ± 0.16</td>
<td align="char" char="." valign="top">33</td>
<td valign="top">PsbL</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">SER_84@Oγ</td>
<td valign="top">GLU_29@Oε</td>
<td align="char" char="xb1" valign="top">2.68 ± 0.13</td>
<td align="char" char="." valign="top">32</td>
<td valign="top">PsbH</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">VAL_33@N</td>
<td valign="top">LEU_46@O</td>
<td align="char" char="xb1" valign="top">2.91 ± 0.14</td>
<td align="char" char="." valign="top">99</td>
<td valign="top">PsbH</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">ASN_230@Nδ</td>
<td valign="top">ASN_56@Oδ</td>
<td align="char" char="xb1" valign="top">2.90 ± 0.15</td>
<td align="char" char="." valign="top">95</td>
<td valign="top">D1</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">GLU_229@N</td>
<td valign="top">LEU_57@O</td>
<td align="char" char="xb1" valign="top">3.02 ± 0.17</td>
<td align="char" char="." valign="top">83</td>
<td valign="top">D1</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
<tr align="left">
<td valign="top">GLN_9@Nε</td>
<td valign="top">ASN_56@Oδ</td>
<td align="char" char="xb1" valign="top">2.90 ± 0.14</td>
<td align="char" char="." valign="top">88</td>
<td valign="top">PsbL</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
<tr align="left">
<td valign="top">ASN_230@N</td>
<td valign="top">GLN_55@O</td>
<td align="char" char="xb1" valign="top">3.03 ± 0.16</td>
<td align="char" char="." valign="top">59</td>
<td valign="top">D1</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">GLN_223@Nε</td>
<td valign="top">TYR_48@Oη</td>
<td align="char" char="xb1" valign="top">3.07 ± 0.18</td>
<td align="char" char="." valign="top">48</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASN_14@Nδ</td>
<td valign="top">LEU_53@O</td>
<td align="char" char="xb1" valign="top">2.97 ± 0.17</td>
<td align="char" char="." valign="top">40</td>
<td valign="top">CP47</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">ARG_230@Nη</td>
<td valign="top">ASN_60@Oδ</td>
<td align="char" char="xb1" valign="top">2.94 ± 0.16</td>
<td align="char" char="." valign="top">77</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ARG_230@Nη</td>
<td valign="top">ASP_51@Oδ</td>
<td align="char" char="xb1" valign="top">2.80 ± 0.11</td>
<td align="char" char="." valign="top">48</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">LYS_137@Nζ</td>
<td valign="top">GLN_81@Oε</td>
<td align="char" char="xb1" valign="top">2.84 ± 0.13</td>
<td align="char" char="." valign="top">86</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ARG_476@Nη</td>
<td valign="top">LEU_53@O</td>
<td align="char" char="xb1" valign="top">2.98 ± 0.18</td>
<td align="char" char="." valign="top">30</td>
<td valign="top">CP47</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a></td>
</tr>
<tr align="left">
<td valign="top">ARG_230@Nε</td>
<td valign="top">ASP_51@Oδ</td>
<td align="char" char="xb1" valign="top">3.08 ± 0.18</td>
<td align="char" char="." valign="top">30</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/><col/><col/><col/></colgroup>






<thead align="left">
<tr align="left" valign="bottom">
<th colspan="2" align="left" valign="bottom" class="border_black">Salt bridge</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Distance (Å)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Occupancy (%)</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Proteins in PSII</th>
<th rowspan="2" align="left" valign="bottom" class="border_black">Cryo-EM structures<i><small><sup style="color: red;"><a title="" href="#tab4fna">a</a></sup></small></i></th>
</tr>
<tr align="left" valign="bottom">
<th align="left" valign="bottom" class="border_black">Acidic</th>
<th align="left" valign="bottom" class="border_black">Basic</th>
</tr>
</thead>
<tbody align="left">
<tr align="left">
<td valign="top">GLU_85</td>
<td valign="top">LYS_32</td>
<td align="char" char="xb1" valign="top">3.80 ± 0.45</td>
<td align="char" char="." valign="top">95</td>
<td valign="top">PsbH</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">ASP_49</td>
<td valign="top">LYS_130</td>
<td align="char" char="xb1" valign="top">5.34 ± 0.39</td>
<td align="char" char="." valign="top">55</td>
<td valign="top">CP47</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a></td>
</tr>
<tr align="left">
<td valign="top">GLU_71</td>
<td valign="top">ARG_127</td>
<td align="char" char="xb1" valign="top">4.32 ± 0.93</td>
<td align="char" char="." valign="top">43</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASP_51</td>
<td valign="top">ARG_230</td>
<td align="char" char="xb1" valign="top">4.47 ± 0.29</td>
<td align="char" char="." valign="top">71</td>
<td valign="top">CP47</td>
<td> </td>
</tr>
<tr align="left">
<td valign="top">ASP_477</td>
<td valign="top">LYS_59</td>
<td align="char" char="xb1" valign="top">5.35 ± 0.33</td>
<td align="char" char="." valign="top">39</td>
<td valign="top">CP47</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=7OUI" title="View structure 7OUI on PDB web page" target="_blank">7OUI</a></td>
</tr>
<tr align="left">
<td valign="top">ASP_483</td>
<td valign="top">LYS_59</td>
<td align="char" char="xb1" valign="top">4.07 ± 1.23</td>
<td align="char" char="." valign="top">79</td>
<td valign="top">CP47</td>
<td valign="top"><a href="http://xlink.rsc.org/?pdb=3JCU" title="View structure 3JCU on PDB web page" target="_blank">3JCU</a>, <a href="http://xlink.rsc.org/?pdb=5XNL" title="View structure 5XNL on PDB web page" target="_blank">5XNL</a></td>
</tr>
</tbody>
</table></div></div>
<hr class="hrule"/><br/>
<p class="otherpara">The total numbers and spatial locations of HBs/SBs found at <span class="italic">S</span><small><sub>LHCII/core</sub></small>, <span class="italic">S</span><small><sub>CP26/core</sub></small>, and <span class="italic">S</span><small><sub>CP29/core</sub></small> are consistent with those of the hot spots identified. The numbers of HBs/SBs are 25/6 for <span class="italic">S</span><small><sub>CP29/core</sub></small>, 9/0 for <span class="italic">S</span><small><sub>LHCII/core</sub></small>, and 4/2 for <span class="italic">S</span><small><sub>CP26/core</sub></small> (<a title="Select to navigate to table" href="#tab3">Table 3</a>). Eight of the 9 HBs at <span class="italic">S</span><small><sub>LHCII/core</sub></small> are mediated by PsbW, and 7 are distributed at the luminal side (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4A</a> and <a title="Select to navigate to table" href="#tab3">Table 3</a>). Three of the 4 HBs and all SBs at <span class="italic">S</span><small><sub>CP26/core</sub></small> are mediated by PsbZ, and 3 HBs/SBs are located at the stromal side (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4B</a> and <a title="Select to navigate to table" href="#tab3">Table 3</a>). All the 25 HBs and 6 SBs at <span class="italic">S</span><small><sub>CP29/core</sub></small> are distributed at the stromal side and are mediated by Motif II of CP29. Residues of Motif II form HBs/SBs with multiple protein subunits of the PSII core (PsbH: 5/1; PsbL: 6/0; CP47: 10/5; D1: 4/0) (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4C</a> and <a title="Select to navigate to table" href="#tab4">Table 4</a>).</p>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2023/RA/d2ra08240c/d2ra08240c-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ra08240c-f4.tif" src="/image/article/2023/RA/d2ra08240c/d2ra08240c-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Hydrogen bond and salt bridge interactions at <span class="italic">S</span><small><sub>LHCII/core</sub></small> (a), <span class="italic">S</span><small><sub>CP26/core</sub></small> (b), and <span class="italic">S</span><small><sub>CP29/core</sub></small> (c). Hydrogen bonds and salt bridges are represented by yellow and orange dashed lines, respectively. Colour scheme is presented as the same as shown in <a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">The differences between average binding free energies of the HB/SB-involved hot spots and all hot spots were −0.38 kcal mol<small><sup>−1</sup></small> for <span class="italic">S</span><small><sub>LHCII/core</sub></small>, −0.04 kcal mol<small><sup>−1</sup></small> for <span class="italic">S</span><small><sub>CP26/core</sub></small>, and −0.66 kcal mol<small><sup>−1</sup></small> for <span class="italic">S</span><small><sub>CP29/core</sub></small> (Fig. S5<a title="Select to navigate to footnote" href="#fn1">†</a>), which accounted for 26%, 1.6%, and 27% of the average binding free energies of all hot spots, respectively. This suggests that the formation of HBs/SBs predominately modulates hot spots' binding free energy contribution and enhances their binding ability. Moreover, the change in binding free energies was dominated by the reduction in electrostatic interaction energies (Fig. S5<a title="Select to navigate to footnote" href="#fn1">†</a>). At <span class="italic">S</span><small><sub>LHCII/core</sub></small>, 8 of the 23 hot spots were involved in HB formation (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3A–E</a>, residues marked with *), and 5 hot spots, including V90 and L113 (LHCII), S125 and G126 (PsbW), and S205 (CP43), provide negative electrostatic interaction energies. At <span class="italic">S</span><small><sub>CP26/core</sub></small>, 3 of the 5 hot spots (W33, L39, L231) were involved in HB formation and W33 contributes negative electrostatic interaction energies (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3C–E</a>). At <span class="italic">S</span><small><sub>CP29/core</sub></small>, 19 of the 29 hot spots were involved in HB/SB formation (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3J</a>, residues marked with *), and 8 hot spots, including N230 (D1), R127 and R230 (CP47), V33 (PsbH), Q9 (PsbL), as well as L46, N56, and K59 (CP29), contribute negative electrostatic interaction energy. Overall, our results revealed that HBs/SBs are favorable for the binding of the interfaces, which is consistent to a certain extent with the previous study<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> that indicated that electrostatic interactions were the main driving force to mediate the binding of antennae to the PSII core. However, our analysis showed that hydrogen bonds contribute only about 20% of the binding energies (Fig. S5<a title="Select to navigate to footnote" href="#fn1">†</a>). In contrast, most of the binding energies come from Δ<span class="italic">G</span><small><sub>nonpolar</sub></small> (hydrophobic interaction). It is likely that hydrogen bonds mainly provide directional interactions and the anchoring of the interface.</p>

<h3 id="sect1390"><span class="b_heading">Roles of PSII small intrinsic subunits in antenna–core association</span></h3>
<span>To study the roles of small intrinsic subunits in antenna–core binding, the binding free energies at the antenna–core interface in the absence of PsbW, PsbZ, and PsbH were calculated (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5A</a>). The binding affinities at the antenna–core interfaces were significantly weakened in the absence of PsbW, PsbZ, and PsbH (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5B</a>). The hydrophobic interaction energies of the ΔPsbW, ΔPsbZ, and ΔPsbH systems were decreased by 66.5%, 52.3%, and 32.9%, respectively, and the changes in electrostatic interaction energies were negligible. These results indicate that the small intrinsic subunits of PSII (PsbW, PsbZ, and PsbH) contribute about 30% to 60% of the binding free energies for antenna–core binding, corroborating the experimental findings that peripheral antenna content was reduced in the absence of PsbW and PsbZ.<a title="Select to navigate to reference" href="#cit24"><sup><span class="sup_ref">24,25</span></sup></a></span>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2023/RA/d2ra08240c/d2ra08240c-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ra08240c-f5.tif" src="/image/article/2023/RA/d2ra08240c/d2ra08240c-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">Binding free energy changes caused by the deletion of key subunits and a schematic diagram of the inferred assembly process. (A) Display of the small intrinsic subunits PsbW, PsbZ, PsbH, and Motif II of the N-terminal loop region of CP29, the colors of protein subunits correspond to <a title="Select to navigate to figure" href="#imgfig1">Fig. 1a</a>, and the Motif II is shown in yellow. (B) Affinity changes of the antenna–core interface in the absence of PsbW, PsbZ, PsbH and Motif II.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
<p class="otherpara">Most of the hot spots and residues involved in HB/SB formation of CP29 were identified on Motif II of CP29, which represents a long loop region (Pro42–Phe87) and is responsible for binding to the core proteins.<a title="Select to navigate to references" href="#cit11"><sup><span class="sup_ref">11</span></sup></a> In the absence of Motif II, the binding free energy increased from −116.1 ± 9.8 kcal mol<small><sup>−1</sup></small> to 56.6 ± 4.1 (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5B</a>), implicating that CP29 can not bind to core proteins without Motif II. Moreover, CP29 interacts with multiple subunits including PsbH, CP47, D1, and PsbL. This differs from LHCII and CP26, which interact with core proteins <span class="italic">via</span> small intrinsic subunits, <span class="italic">i.e.</span>, PsbW and PsbZ. The absence of PsbH only led to a relatively small percentage (32.9%) of binding free energy lose, suggesting that the function of PsbH in binding with CP29 and the core may not be as significant as expected.</p>


<h2 id="sect1405"><span class="a_heading">Discussion</span></h2>
<span>High-resolution structures of PSII-LHCII supercomplexes have advanced our knowledge about the self-organization of photosynthetic antenna-reaction center systems. However, the relatively low resolution at highly dynamic regions in the systems, particularly the antenna–core interface, has raised many questions in understanding the PSII-LHCII assembly. Standing on the cryo-EM structure, we developed a reliable million-atom-scale model of plant C<small><sub>2</sub></small>S<small><sub>2</sub></small>-type PSII-LHCII supercomplex embedded in a solvated membrane using <span class="italic">in silico</span> simulations. Microsecond-scale MD simulations on the 1.1 megadalton multi-protein ensemble allow us to comprehensively analyze the inter-protein interactions and binding free energies at the antenna–core and antenna–antenna interfaces, providing very suggestive details of the stepwise assembly of the PSII-LHCII supercomplex.</span>
<p class="otherpara">In our study, the binding free energy calculations with corresponding component decompositions revealed that the antenna–core binding is a competitive process between electrostatic energy and hydrophobic energy, in which the electrostatic interactions provide mainly repulsive energy. In contrast, the hydrophobic contributions serve as the main driving force to facilitate antenna–core association (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>). This may imply a common mechanism underlying the protein–protein interactions and assembly of membrane macromolecular complexes, in which hydrophobic interactions play an essential role. In addition, compared to the cryo-EM structure, MD simulations reveal the HBs/SBs interactions at the antenna–core interface more comprehensively (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>). Electrostatic interaction energy analysis further indicated that the formation of HBs/SBs enhances the binding affinity of hot spots and is thought to play a positioning role during antenna–core assembly.</p>
<h3 id="sect1414"><span class="b_heading">Possible assembly pathways of PSII-LHCII, PSII-CP26 and PSII-CP29</span></h3>
<span>Our MD simulations data supported a stepwise assembly of the PSII–LHCII complex (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>). The small intrinsic proteins PsbW and PsbZ bind to the periphery of the PSII core.<a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> They may play critical roles in mediating the binding of the antenna to the PSII core. Their absence, especially PsbW, could significantly weaken the hydrophobic interactions at the antenna–core interface (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>). The LHCII–core binding may undertake two steps: LHCII first binds to PsbW, due to its strong interaction with PsbW, to form the LHCII–PsbW assembly intermediate, and then binds to the PSII core as one assembly unit (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>). Previous experimental results supported that PsbW and LHCII co-occur with the PSII core.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> The CP26–core binding may also undertake two steps and form the CP26–PsbZ assembly intermediate before binding to the core. A previous experimental study has consistently shown that PsbZ could be released together with CP26 from the PSII complex.<a title="Select to navigate to references" href="#cit23"><sup><span class="sup_ref">23</span></sup></a> For CP29, <span class="italic">S</span><small><sub>CP29/core</sub></small> interaction analysis showed that binding to the PSII core was not strongly dependent on specific small intrinsic subunits (such as PsbH). Instead, CP29 forms extensive interactions with multiple core subunits of PSII through its Motif II to facilitate stable binding to the core. Therefore, it is more likely that the anchoring of CP29 to the PSII core represents a one-step process without the assistance of other factors and any assembly intermediate formation (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>).</span>
<br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2023/RA/d2ra08240c/d2ra08240c-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d2ra08240c-f6.tif" src="/image/article/2023/RA/d2ra08240c/d2ra08240c-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">Schematic diagram of the antenna–core assembly process of the C<small><sub>2</sub></small>S<small><sub>2</sub></small>-type PSII-LHCII. LHCII is represented in green, CP26 in pink, CP29 in blue, PsbW in purple, and PsbZ in gray, these colors correspond to <a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>. Motif II of CP29 is shown with a yellow coil. Brown and maroon show dimerized PSII cores. Arrows with numbers represent potential assembly steps.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>

<h3 id="sect1431"><span class="b_heading">Weak interactions of the antenna–antenna interfaces support an independent assembly</span></h3>
<span>Our data also indicated weaker binding affinity at the antenna–antenna interfaces than at the antenna–core interfaces, suggesting independent association/disassociation of individual antenna complexes LHCII, CP29, and CP26. Interestingly, under low-light conditions, two other antennas, M-LHCII and L-LHCII, could attach to the C<small><sub>2</sub></small>S<small><sub>2</sub></small>-type PSII-LHCII supercomplex.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> They do not interact directly with the PSII core but are bridged by CP29, ultimately forming the C<small><sub>2</sub></small>S<small><sub>2</sub></small>M<small><sub>2</sub></small>L<small><sub>2</sub></small>-type PSII-LHCII complex.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> These periphery antennae belong to the Lhcb protein family with similar sequence and structural features, suggesting that they may have similar interaction characteristics. It is known that LHCII can migrate between PSI and PSII to balance the distribution of excitation energy between PSI and PSII in state transitions.<a title="Select to navigate to reference" href="#cit20"><sup><span class="sup_ref">20,21</span></sup></a> We speculate the interfacial affinities between M-LHCII/L-LHCII and CP29 are relatively weak, which might be essential for driving the dynamic association/dissociation of M-LHCII/L-LHCII to the PSII core in state transitions.<a title="Select to navigate to reference" href="#cit64"><sup><span class="sup_ref">64,65</span></sup></a> Given the relatively strong interaction with the core, S-LHCII might be difficult for detach from the PSII core. This is consistent with previous experimental results, which showed that only L-LHCII freely diffuse between PSI and PSII in state transitions<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> and that M-LHCII may detach from the PSII-LHCII supercomplex under specific physiological conditions.<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> The detailed mechanisms underlying the dynamic PSII-LHCII assembly in state transitions remain to be elucidated. It has been presumed that phosphorylation of LHCII N-terminus sites could alter the binding affinity of LHCII to PSII, resulting in its disassociation from PSII.<a title="Select to navigate to reference" href="#cit20"><sup><span class="sup_ref">20,66,67</span></sup></a> As electrostatic interactions provide repulsion between protein interfaces and protein residues at the stromal side of PSII-LHCII exhibits a negative electrostatic potential (Fig. S6<a title="Select to navigate to footnote" href="#fn1">†</a>), phosphorylation of LHCII N-terminus likely leads to an increase in the negative electrostatic potential and the repulsive force at the protein–protein interfaces, which drive the disassociation of LHCII from PSII.</span>


<h2 id="sect1448"><span class="a_heading">Conclusions</span></h2>
<span>Our structural model and simulations approach provide the framework for characterizing the molecular mechanisms of photosynthetic complexes' structural and functional dynamics. A better understanding of the antenna–photosynthetic reaction center assembly and interactions will inform strategies to enhance photosynthetic energy transfer and design efficient artificial photosynthetic systems.</span>

<h2 id="sect1451"><span class="a_heading">Author contributions</span></h2>
<span>JG and LNL: idea conceptualization. JG and LNL: project administration, and funding acquisition. RM, LB, HZ, and JG: methodology and validation. RM, LB, JG and LNL: article writing.</span>

<h2 id="sect1454"><span class="a_heading">Conflicts of interest</span></h2>
<span>There are no conflicts to declare.</span>


<h2 id="sect1458"><span class="a_heading">Acknowledgements</span></h2><span>This work was supported by the National Natural Science Foundation of China (No. 21873034, 32070109), the National Key R&amp;D Program of China (No. 2021YFA0909600, 2017YFB0203405), Fundamental Research Funds for the Central Universities (No. 2662020XXPY02), the Royal Society (URF\R\180030 to LNL), the Biotechnology and Biological Sciences Research Council (BBSRC) (BB/V009729/1 and BB/R003890/1 to LNL).</span>
<span id="sect1460"><h2 id="sect1457"><span class="a_heading">References</span></h2></span><ol type="1"><li><span id="cit1">J. Barber, <span class="italic">Chem. Soc. Rev.</span>, 2009, <span class="bold">38</span>, 185–196 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B802262N&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B802262N">RSC</a>.</span></li><li><span id="cit2">L. Wu, L. Zhang, W. Tu, R. Sun and C. Yang, <span class="italic">J. Plant Physiol.</span>, 2020, <span class="bold">251</span>, 153189 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jplph.2020.153189" title="DOI Link to resource 10.1016/j.jplph.2020.153189">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWkt7%252FK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32526555%5Buid%5D" title="PubMed Link to resource 32526555">PubMed</a>.</span></li><li><span id="cit3">M. Xuan and J. Li, <span class="italic">Natl. Sci. Rev.</span>, 2021, <span class="bold">8</span>, nwab051 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nsr/nwab051" title="DOI Link to resource 10.1093/nsr/nwab051">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXislWjsr7L" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34691712%5Buid%5D" title="PubMed Link to resource 34691712">PubMed</a>.</span></li><li><span id="cit4">P. Cao, X. W. Pan, X. D. Su, Z. F. Liu and M. Li, <span class="italic">Curr. Opin. Struct. Biol.</span>, 2020, <span class="bold">63</span>, 49–57 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbi.2020.03.007" title="DOI Link to resource 10.1016/j.sbi.2020.03.007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXotVensbo%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32389895%5Buid%5D" title="PubMed Link to resource 32389895">PubMed</a>.</span></li><li><span id="cit5">J. R. Shen, <span class="italic">Annu. Rev. Plant Biol.</span>, 2015, <span class="bold">66</span>, 23–48 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev-arplant-050312-120129" title="DOI Link to resource 10.1146/annurev-arplant-050312-120129">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtVajtbvO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25746448%5Buid%5D" title="PubMed Link to resource 25746448">PubMed</a>.</span></li><li><span id="cit6">L. X. Shi, M. Hall, C. Funk and W. P. SchröDer, <span class="italic">Biochim. Biophys. Acta</span>, 2016, <span class="bold">1817</span>, 13–25 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bbabio.2011.08.008" title="DOI Link to resource 10.1016/j.bbabio.2011.08.008">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21907181%5Buid%5D" title="PubMed Link to resource 21907181">PubMed</a>.</span></li><li><span id="cit7">C. Pagliano, G. Saracco and J. Barber, <span class="italic">Photosynth. Res.</span>, 2013, <span class="bold">116</span>, 167–188 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11120-013-9803-8" title="DOI Link to resource 10.1007/s11120-013-9803-8">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Clt77N" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23417641%5Buid%5D" title="PubMed Link to resource 23417641">PubMed</a>.</span></li><li><span id="cit8">W. Wang, H. Wang, Q. Zhu, W. Qin, G. Han, J. R. Shen, X. Zong and C. Li, <span class="italic">Angew. Chem.</span>, 2016, <span class="bold">128</span>, 9375–9379 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ange.201604091" title="DOI Link to resource 10.1002/ange.201604091">CrossRef</a>.</span></li><li><span id="cit9">O. Yehezkeli, R. Tel-Vered, J. Wasserman, A. Trifonov, D. Michaeli, R. Nechushtai and I. Willner, <span class="italic">Nat. Commun.</span>, 2012, <span class="bold">3</span>, 742 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/ncomms1741" title="DOI Link to resource 10.1038/ncomms1741">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22415833%5Buid%5D" title="PubMed Link to resource 22415833">PubMed</a>.</span></li><li><span id="cit10">F. Azadi-Chegeni, M. E. Ward, G. Perin, D. Simionato, T. Morosinotto, M. Baldus and A. Pandit, <span class="italic">Biophys. J.</span>, 2021, <span class="bold">120</span>, 270–283 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bpj.2020.11.2265" title="DOI Link to resource 10.1016/j.bpj.2020.11.2265">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXisF2gt7vE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33285116%5Buid%5D" title="PubMed Link to resource 33285116">PubMed</a>.</span></li><li><span id="cit11">X. Wei, X. Su, P. Cao, X. Liu, W. Chang, M. Li, X. Zhang and Z. Liu, <span class="italic">Nature</span>, 2016, <span class="bold">534</span>, 69–74 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature18020" title="DOI Link to resource 10.1038/nature18020">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XotFOktLg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27251276%5Buid%5D" title="PubMed Link to resource 27251276">PubMed</a>.</span></li><li><span id="cit12">X. Sheng, A. Watanabe, A. J. Li, E. Kim, C. H. Song, K. Murata, D. F. Song, J. Minagawa and Z. F. Liu, <span class="italic">Nat. Plants</span>, 2019, <span class="bold">5</span>, 1320–1330 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41477-019-0543-4" title="DOI Link to resource 10.1038/s41477-019-0543-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisVyktbjI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31768031%5Buid%5D" title="PubMed Link to resource 31768031">PubMed</a>.</span></li><li><span id="cit13">X. Su, J. Ma, X. Wei, P. Cao, D. Zhu, W. Chang, Z. Liu, X. Zhang and M. Li, <span class="italic">Science</span>, 2017, <span class="bold">357</span>, 815–820 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aan0327" title="DOI Link to resource 10.1126/science.aan0327">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKntrrJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28839073%5Buid%5D" title="PubMed Link to resource 28839073">PubMed</a>.</span></li><li><span id="cit14">P. Cao, X. Su, X. Pan, Z. Liu, W. Chang and M. Li, <span class="italic">Biochim. Biophys. Acta</span>, 2018, <span class="bold">1859</span>, 633–644 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bbabio.2018.03.007" title="DOI Link to resource 10.1016/j.bbabio.2018.03.007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXltlSktLk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29550213%5Buid%5D" title="PubMed Link to resource 29550213">PubMed</a>.</span></li><li><span id="cit15">R. Croce and H. van Amerongen, <span class="italic">Science</span>, 2020, <span class="bold">369</span>, eaay2058 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aay2058" title="DOI Link to resource 10.1126/science.aay2058">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhs1GrsL%252FJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32820091%5Buid%5D" title="PubMed Link to resource 32820091">PubMed</a>.</span></li><li><span id="cit16">X. Sheng, Z. Liu, E. Kim and J. Minagawa, <span class="italic">Plant Cell Physiol.</span>, 2021, <span class="bold">62</span>, 1108–1120 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/pcp/pcab072" title="DOI Link to resource 10.1093/pcp/pcab072">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhslWgurvL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34038564%5Buid%5D" title="PubMed Link to resource 34038564">PubMed</a>.</span></li><li><span id="cit17">P. Cao, L. Bracun, A. Yamagata, B. M. Christianson, T. Negami, B. Zou, T. Terada, D. P. Canniffe, M. Shirouzu, M. Li and L. N. Liu, <span class="italic">Nat. Commun.</span>, 2022, <span class="bold">13</span>, 1977 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-022-29563-3" title="DOI Link to resource 10.1038/s41467-022-29563-3">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XpvVahsb4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=35418573%5Buid%5D" title="PubMed Link to resource 35418573">PubMed</a>.</span></li><li><span id="cit18">L. Bracun, A. Yamagata, B. M. Christianson, T. Terada, D. P. Canniffe, M. Shirouzu and L. N. Liu, <span class="italic">Sci Adv</span>, 2021, <span class="bold">7</span>, eabf8864 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/sciadv.abf8864" title="DOI Link to resource 10.1126/sciadv.abf8864">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvVSht77N" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34134992%5Buid%5D" title="PubMed Link to resource 34134992">PubMed</a>.</span></li><li><span id="cit19">N. Liguori, R. Croce, S. J. Marrink and S. Thallmair, <span class="italic">Photosynth. Res.</span>, 2020, <span class="bold">144</span>, 273–295 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11120-020-00741-y" title="DOI Link to resource 10.1007/s11120-020-00741-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXntFOksLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32297102%5Buid%5D" title="PubMed Link to resource 32297102">PubMed</a>.</span></li><li><span id="cit20">A. V. Vener, <span class="italic">Biochim. Biophys. Acta</span>, 2007, <span class="bold">1767</span>, 449–457 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bbabio.2006.11.007" title="DOI Link to resource 10.1016/j.bbabio.2006.11.007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXmt1ygsLY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17184728%5Buid%5D" title="PubMed Link to resource 17184728">PubMed</a>.</span></li><li><span id="cit21">J. F. Allen, <span class="italic">Physiol. Plant.</span>, 2017, <span class="bold">161</span>, 28–44 <a target="_blank" class="DOILink" href="https://doi.org/10.1111/ppl.12577" title="DOI Link to resource 10.1111/ppl.12577">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXovVSltLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28393369%5Buid%5D" title="PubMed Link to resource 28393369">PubMed</a>.</span></li><li><span id="cit22">S. JäRvi, M. Suorsa and E. M. Aro, <span class="italic">Biochim. Biophys. Acta</span>, 2015, <span class="bold">1847</span>, 900–909 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bbabio.2015.01.006" title="DOI Link to resource 10.1016/j.bbabio.2015.01.006">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25615587%5Buid%5D" title="PubMed Link to resource 25615587">PubMed</a>.</span></li><li><span id="cit23">A. Rokka, M. Suorsa, A. N. Battchikova and E. M. Aro, <span class="italic">Biochem. J.</span>, 2005, <span class="bold">388</span>, 159–168 <a target="_blank" class="DOILink" href="https://doi.org/10.1042/BJ20042098" title="DOI Link to resource 10.1042/BJ20042098">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXktVKntro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15638811%5Buid%5D" title="PubMed Link to resource 15638811">PubMed</a>.</span></li><li><span id="cit24">J. G. Garcia-Cerdan, L. Kovacs, T. Toth, S. Kereiche, E. Aseeva, E. J. Boekema, F. Mamedov, C. Funk and W. P. Schroder, <span class="italic">Plant J.</span>, 2011, <span class="bold">65</span>, 368–381 <a target="_blank" class="DOILink" href="https://doi.org/10.1111/j.1365-313X.2010.04429.x" title="DOI Link to resource 10.1111/j.1365-313X.2010.04429.x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXisVensbg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21265891%5Buid%5D" title="PubMed Link to resource 21265891">PubMed</a>.</span></li><li><span id="cit25">M. Swiatek, <span class="italic">Plant Cell</span>, 2001, <span class="bold">13</span>, 1347–1368 <a target="_blank" class="DOILink" href="https://doi.org/10.1105/TPC.010001" title="DOI Link to resource 10.1105/TPC.010001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3MXkvVakt74%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit26">Y. Mohammadi, A. R. Haghighi and S. G. Sehrigh, <span class="italic">Int. J. BIOautom.</span>, 2016, <span class="bold">20</span>, 351–358 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXoslWiur4%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit27">L. Nierzwicki and G. Palermo, <span class="italic">Front. Mol. Biosci.</span>, 2021, <span class="bold">8</span>, 641208 <a target="_blank" class="DOILink" href="https://doi.org/10.3389/fmolb.2021.641208" title="DOI Link to resource 10.3389/fmolb.2021.641208">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhtFCitbnN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33884260%5Buid%5D" title="PubMed Link to resource 33884260">PubMed</a>.</span></li><li><span id="cit28">J. R. Perilla, B. C. Goh, C. K. Cassidy, B. Liu, R. C. Bernardi, T. Rudack, H. Yu, Z. Wu and K. Schulten, <span class="italic">Curr. Opin. Struct. Biol.</span>, 2015, <span class="bold">31</span>, 64–74 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.sbi.2015.03.007" title="DOI Link to resource 10.1016/j.sbi.2015.03.007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXlsFemsb4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25845770%5Buid%5D" title="PubMed Link to resource 25845770">PubMed</a>.</span></li><li><span id="cit29">A. Singharoy, C. Maffeo, K. H. Delgado-Magnero, D. J. K. Swainsbury, M. Sener, U. Kleinekathofer, J. W. Vant, J. Nguyen, A. Hitchcock, B. Isralewitz, I. Teo, D. E. Chandler, J. E. Stone, J. C. Phillips, T. V. Pogorelov, M. I. Mallus, C. Chipot, Z. Luthey-Schulten, D. P. Tieleman, C. N. Hunter, E. Tajkhorshid, A. Aksimentiev and K. Schulten, <span class="italic">Cell</span>, 2019, <span class="bold">179</span>, 1098–1111 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cell.2019.10.021" title="DOI Link to resource 10.1016/j.cell.2019.10.021">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXitFymtb%252FN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31730852%5Buid%5D" title="PubMed Link to resource 31730852">PubMed</a>.</span></li><li><span id="cit30">M. Faulkner, I. Szabó, S. L. Weetman, F. Sicard, R. G. Huber, P. J. Bond, E. Rosta and L.-N. Liu, <span class="italic">Sci. Rep.</span>, 2020, <span class="bold">10</span>, 17501 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-020-74536-5" title="DOI Link to resource 10.1038/s41598-020-74536-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitFSgu7vK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33060756%5Buid%5D" title="PubMed Link to resource 33060756">PubMed</a>.</span></li><li><span id="cit31">L. Zhang, D. A. Silva, H. Zhang, A. Yue, Y. Yan and X. Huang, <span class="italic">Nat. Commun.</span>, 2014, <span class="bold">5</span>, 4170 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/ncomms5170" title="DOI Link to resource 10.1038/ncomms5170">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2mu7jN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24954746%5Buid%5D" title="PubMed Link to resource 24954746">PubMed</a>.</span></li><li><span id="cit32">R. Mao, X. Wang and J. Gao, <span class="italic">Front. Chem.</span>, 2021, <span class="bold">9</span>, 764107 <a target="_blank" class="DOILink" href="https://doi.org/10.3389/fchem.2021.764107" title="DOI Link to resource 10.3389/fchem.2021.764107">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXislSls73E" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34671594%5Buid%5D" title="PubMed Link to resource 34671594">PubMed</a>.</span></li><li><span id="cit33">N. Sakashita, H. C. Watanabe, T. Ikeda, K. Saito and H. Ishikita, <span class="italic">Biochemistry</span>, 2017, <span class="bold">56</span>, 3049–3057 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.biochem.7b00220" title="DOI Link to resource 10.1021/acs.biochem.7b00220">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXotVOjt78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28534615%5Buid%5D" title="PubMed Link to resource 28534615">PubMed</a>.</span></li><li><span id="cit34">J. A. Maier, C. Martinez, K. Kasavajhala, L. Wickstrom, K. E. Hauser and C. Simmerling, <span class="italic">J. Chem. Theory Comput.</span>, 2015, <span class="bold">11</span>, 3696–3713 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jctc.5b00255" title="DOI Link to resource 10.1021/acs.jctc.5b00255">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFequ7rN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26574453%5Buid%5D" title="PubMed Link to resource 26574453">PubMed</a>.</span></li><li><span id="cit35">J. Wang, R. M. Wolf, J. W. Caldwell, P. A. Kollman and D. A. Case, <span class="italic">J. Comput. Chem.</span>, 2004, <span class="bold">25</span>, 1157–1174 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.20035" title="DOI Link to resource 10.1002/jcc.20035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15116359%5Buid%5D" title="PubMed Link to resource 15116359">PubMed</a>.</span></li><li><span id="cit36">M. Ceccarelli, P. Procacci and M. Marchi, <span class="italic">J. Comput. Chem.</span>, 2003, <span class="bold">24</span>, 129–142 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.10198" title="DOI Link to resource 10.1002/jcc.10198">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXlt1CqsA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12497594%5Buid%5D" title="PubMed Link to resource 12497594">PubMed</a>.</span></li><li><span id="cit37">D. A. Giammona, PhD thesis, University of California,  1984.</span></li><li><span id="cit38">P. M. Kasson, E. Lindahl and V. S. Pande, <span class="italic">PLoS Comput. Biol.</span>, 2010, <span class="bold">6</span>, e1000829 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pcbi.1000829" title="DOI Link to resource 10.1371/journal.pcbi.1000829">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20585620%5Buid%5D" title="PubMed Link to resource 20585620">PubMed</a>.</span></li><li><span id="cit39">P. F. Li and K. M. Merz, <span class="italic">J. Chem. Inf. Model.</span>, 2016, <span class="bold">56</span>, 599–604 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.5b00674" title="DOI Link to resource 10.1021/acs.jcim.5b00674">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XjtFCjtbo%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26913476%5Buid%5D" title="PubMed Link to resource 26913476">PubMed</a>.</span></li><li><span id="cit40">C. I. Bayly, P. Cieplak, W. D. Cornell and P. A. Kollman, <span class="italic">J. Phys. Chem.</span>, 1993, <span class="bold">97</span>, 10269–10280 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/j100142a004" title="DOI Link to resource 10.1021/j100142a004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3sXlvVyqsLs%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit41">K. Ogata, M. Hatakeyama, Y. Sakamoto and S. Nakamura, <span class="italic">J. Phys. Chem. B</span>, 2019, <span class="bold">123</span>, 6444–6452 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcb.9b04838" title="DOI Link to resource 10.1021/acs.jpcb.9b04838">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSnurbJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31268711%5Buid%5D" title="PubMed Link to resource 31268711">PubMed</a>.</span></li><li><span id="cit42">D. A. Case, T. E. Cheatham, T. Darden, H. Gohlke, R. Luo, K. M. Merz, A. Onufriev, C. Simmerling, B. Wang and R. J. Woods, <span class="italic">J. Comput. Chem.</span>, 2005, <span class="bold">26</span>, 1668–1688 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.20290" title="DOI Link to resource 10.1002/jcc.20290">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16200636%5Buid%5D" title="PubMed Link to resource 16200636">PubMed</a>.</span></li><li><span id="cit43">S. E. Feller, Y. Zhang, R. W. Pastor and B. R. Brooks, <span class="italic">J. Chem. Phys.</span>, 1995, <span class="bold">103</span>, 4613–4621 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.470648" title="DOI Link to resource 10.1063/1.470648">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXotVentLo%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit44">H. J. C. P. Berendsen, J. P. M. V. Postma, W. F. V. Gunsteren, A. D. Dinola and J. R. Haak, <span class="italic">J. Chem. Phys.</span>, 1984, <span class="bold">81</span>, 3684–3690 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.448118" title="DOI Link to resource 10.1063/1.448118">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit45">S. Miyamoto and P. A. Kollman, <span class="italic">J. Comput. Chem.</span>, 1992, <span class="bold">13</span>, 952–962 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jcc.540130805" title="DOI Link to resource 10.1002/jcc.540130805">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK38Xlslykt7o%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit46">U. Essmann, L. Perera, M. L. Berkowitz, T. Darden, H. Lee and L. G. Pedersen, <span class="italic">J. Chem. Phys.</span>, 1995, <span class="bold">103</span>, 8577–8593 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.470117" title="DOI Link to resource 10.1063/1.470117">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit47">D. Greene, R. Qi, R. Nguyen, T. Qiu and R. Luo, <span class="italic">J. Chem. Inf. Model.</span>, 2019, <span class="bold">59</span>, 3041–3056 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.9b00363" title="DOI Link to resource 10.1021/acs.jcim.9b00363">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKgt7jK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31145610%5Buid%5D" title="PubMed Link to resource 31145610">PubMed</a>.</span></li><li><span id="cit48">L. Zhang, R. Tang, S. Bai, N. K. Connors, L. H. Lua, Y. P. Chuan, A. P. Middelberg and Y. Sun, <span class="italic">PLoS One</span>, 2014, <span class="bold">9</span>, e107313 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0107313" title="DOI Link to resource 10.1371/journal.pone.0107313">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25215874%5Buid%5D" title="PubMed Link to resource 25215874">PubMed</a>.</span></li><li><span id="cit49">L. Zhang, R. Tang, S. Bai, N. K. Connors, L. H. Lua, Y. P. Chuan, A. P. Middelberg and Y. Sun, <span class="italic">J. Phys. Chem. B</span>, 2013, <span class="bold">117</span>, 5411–5421 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp311170w" title="DOI Link to resource 10.1021/jp311170w">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXmtVyhtLc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23586433%5Buid%5D" title="PubMed Link to resource 23586433">PubMed</a>.</span></li><li><span id="cit50">R. Mao, L. Bie, M. Xu, X. Wang and J. Gao, <span class="italic">Phys. Chem. Chem. Phys.</span>, 2021, <span class="bold">23</span>, 12549–12558 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1CP01045J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1CP01045J">RSC</a>.</span></li><li><span id="cit51">V. Lafont, M. Schaefer, R. H. Stote, D. Altschuh and A. Dejaegere, <span class="italic">Proteins</span>, 2007, <span class="bold">67</span>, 418–434 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/prot.21259" title="DOI Link to resource 10.1002/prot.21259">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXjvVSgsrc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17256770%5Buid%5D" title="PubMed Link to resource 17256770">PubMed</a>.</span></li><li><span id="cit52">A. T. Graca, M. Hall, K. Persson and W. P. Schroder, <span class="italic">Sci. Rep.</span>, 2021, <span class="bold">11</span>, 15534 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-021-94914-x" title="DOI Link to resource 10.1038/s41598-021-94914-x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhslKgsLzI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34330992%5Buid%5D" title="PubMed Link to resource 34330992">PubMed</a>.</span></li><li><span id="cit53">R. A. Laskowski, M. W. Macarthur, D. S. Moss and J. M. Thornton, <span class="italic">J. Appl. Crystallogr.</span>, 1993, <span class="bold">26</span>, 283–291 <a target="_blank" class="DOILink" href="https://doi.org/10.1107/S0021889892009944" title="DOI Link to resource 10.1107/S0021889892009944">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="Link to resource in CAS">CAS</a>.</span></li><li><span id="cit54">C. Colovos and T. O. Yeates, <span class="italic">Protein Sci.</span>, 1993, <span class="bold">2</span>, 1511–1519 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/pro.5560020916" title="DOI Link to resource 10.1002/pro.5560020916">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2cXhvVKltbk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=8401235%5Buid%5D" title="PubMed Link to resource 8401235">PubMed</a>.</span></li><li><span id="cit55">H. J. Bai, W. Z. Ma, S. Y. Liu and L. H. Lai, <span class="italic">Proteins</span>, 2008, <span class="bold">70</span>, 1323–1331 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/prot.21625" title="DOI Link to resource 10.1002/prot.21625">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXisFyrtL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17876833%5Buid%5D" title="PubMed Link to resource 17876833">PubMed</a>.</span></li><li><span id="cit56">W. Wang and P. A. Kollman, <span class="italic">J. Mol. Biol.</span>, 2000, <span class="bold">303</span>, 567–582 <a target="_blank" class="DOILink" href="https://doi.org/10.1006/jmbi.2000.4057" title="DOI Link to resource 10.1006/jmbi.2000.4057">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXotlyit7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=11054292%5Buid%5D" title="PubMed Link to resource 11054292">PubMed</a>.</span></li><li><span id="cit57">L. Xiao, J. Diao, D. Greene, J. Wang and R. Luo, <span class="italic">J. Chem. Theory Comput.</span>, 2017, <span class="bold">13</span>, 3398–3412 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jctc.7b00382" title="DOI Link to resource 10.1021/acs.jctc.7b00382">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXovVGgu7o%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28564540%5Buid%5D" title="PubMed Link to resource 28564540">PubMed</a>.</span></li><li><span id="cit58">G. Archontis and T. Simonson, <span class="italic">Biophys. J.</span>, 2005, <span class="bold">88</span>, 3888–3904 <a target="_blank" class="DOILink" href="https://doi.org/10.1529/biophysj.104.055996" title="DOI Link to resource 10.1529/biophysj.104.055996">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXksl2jsbw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15821163%5Buid%5D" title="PubMed Link to resource 15821163">PubMed</a>.</span></li><li><span id="cit59">S. Genheden and U. Ryde, <span class="italic">Expert Opin. Drug Discovery</span>, 2015, <span class="bold">10</span>, 449–461 <a target="_blank" class="DOILink" href="https://doi.org/10.1517/17460441.2015.1032936" title="DOI Link to resource 10.1517/17460441.2015.1032936">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXntFGktr8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25835573%5Buid%5D" title="PubMed Link to resource 25835573">PubMed</a>.</span></li><li><span id="cit60">H. J. Liu and R. E. Blankenship, <span class="italic">Biochim. Biophys. Acta</span>, 2019, <span class="bold">1860</span>, 148079 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bbabio.2019.148079" title="DOI Link to resource 10.1016/j.bbabio.2019.148079">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvVSgsbzF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31518567%5Buid%5D" title="PubMed Link to resource 31518567">PubMed</a>.</span></li><li><span id="cit61">F. Guerra, M. Siemers, C. Mielack and A. N. Bondar, <span class="italic">J. Phys. Chem. B</span>, 2018, <span class="bold">122</span>, 4625–4641 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpcb.8b00649" title="DOI Link to resource 10.1021/acs.jpcb.8b00649">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXmt1Wlt7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29589763%5Buid%5D" title="PubMed Link to resource 29589763">PubMed</a>.</span></li><li><span id="cit62">M. Gur, E. Taka, S. Z. Yilmaz, C. Kilinc, U. Aktas and M. Golcuk, <span class="italic">J. Chem. Phys.</span>, 2020, <span class="bold">153</span>, 075101 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/5.0011141" title="DOI Link to resource 10.1063/5.0011141">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhs1ert7zO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32828084%5Buid%5D" title="PubMed Link to resource 32828084">PubMed</a>.</span></li><li><span id="cit63">Y. Lu, <span class="italic">Front. Plant Sci.</span>, 2016, <span class="bold">7</span>, 168 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Front.%20Plant%20Sci.%5BJour%5D%20AND%207%5Bvolume%5D%20AND%20168%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li><li><span id="cit64">K. Qin, A. R. Fernie and Y. Zhang, <span class="italic">Biomolecules</span>, 2021, <span class="bold">11</span>, 1839 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/biom11121839" title="DOI Link to resource 10.3390/biom11121839">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB38XhtVCjtbk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34944483%5Buid%5D" title="PubMed Link to resource 34944483">PubMed</a>.</span></li><li><span id="cit65">L. s. Kovács, J. Damkjær, S. Kereïche, C. Ilioaia, A. V. Ruban, E. J. Boekema, S. Jansson and P. Horton, <span class="italic">Plant Cell</span>, 2006, <span class="bold">18</span>, 3106–3120 <a target="_blank" class="DOILink" href="https://doi.org/10.1105/tpc.106.045641" title="DOI Link to resource 10.1105/tpc.106.045641">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17114352%5Buid%5D" title="PubMed Link to resource 17114352">PubMed</a>.</span></li><li><span id="cit66">P. Galka, S. Santabarbara, T. T. H. Khuong, H. Degand, P. Morsomme, R. C. Jennings, E. J. Boekema and S. Caffarri, <span class="italic">Plant Cell</span>, 2012, <span class="bold">24</span>, 2963–2978 <a target="_blank" class="DOILink" href="https://doi.org/10.1105/tpc.112.100339" title="DOI Link to resource 10.1105/tpc.112.100339">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhtlChsbzI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22822202%5Buid%5D" title="PubMed Link to resource 22822202">PubMed</a>.</span></li><li><span id="cit67">S. Rinalducci, M. R. Larsen, S. Mohammed and L. Zolla, <span class="italic">J. Proteome Res.</span>, 2006, <span class="bold">5</span>, 973–982 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/pr050476n" title="DOI Link to resource 10.1021/pr050476n">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xis1Omtrc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16602705%5Buid%5D" title="PubMed Link to resource 16602705">PubMed</a>.</span></li></ol>




<hr/><table><tr><td><h3>Footnote</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">Electronic supplementary information (ESI) available. See DOI: <a target="_blank" href="https://doi.org/10.1039/d2ra08240c">https://doi.org/10.1039/d2ra08240c</a></span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2023</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>