<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Chemical data intelligence for sustainable chemistry  - Chemical Society Reviews (RSC Publishing) DOI:10.1039/D1CS00477H</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2021/cs/d1cs00477h"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":55,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Jana M. Weber"/><meta name="DC.Creator" content="Zhen Guo"/><meta name="DC.Creator" content="Chonghuan Zhang"/><meta name="DC.Creator" content="Artur M. Schweidtmann"/><meta name="DC.Creator" content="Alexei A. Lapkin"/><meta name="DC.title" content="Chemical data intelligence for sustainable chemistry "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2021/11/01"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D1CS00477H"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Chemical data intelligence for sustainable chemistry "/><meta name="citation_author" content="Jana M. Weber"/><meta name="citation_author" content="Zhen Guo"/><meta name="citation_author" content="Chonghuan Zhang"/><meta name="citation_author" content="Artur M. Schweidtmann"/><meta name="citation_author" content="Alexei A. Lapkin"/><meta name="citation_online_date" content="2021/09/14"/><meta name="citation_date" content="2021"/><meta name="citation_journal_title" content="Chemical Society Reviews"/><meta name="citation_volume" content="50"/><meta name="citation_issue" content="21"/><meta name="citation_firstpage" content="12013"/><meta name="citation_lastpage" content="12036"/><meta name="citation_doi" content="10.1039/D1CS00477H"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2021/cs/d1cs00477h"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2021/cs/d1cs00477h"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2021/cs/d1cs00477h"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2021/cs/d1cs00477h">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d1cs00465d)" href="/en/content/articlehtml/2021/cs/d1cs00465d" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d0cs00031k)" href="/en/content/articlehtml/2021/cs/d0cs00031k" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div><br/><span class="oa"><img src="/content/newimages/open_access_blue.png" alt=""/> Open Access Article<br/><img src="/content/newimages/CCBY.svg" alt=""/>
This Open Access Article is licensed under a <br/><a rel="license" href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 Unported Licence</a></span></div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D1CS00477H">10.1039/D1CS00477H</a>
(Review Article)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/1460-4744/1972">Chem. Soc. Rev.</a></span>, 2021, <strong>50</strong>, 12013-12036</div><h1 id="sect226"><span class="title_heading">Chemical data intelligence for sustainable chemistry</span></h1><p class="header_text">
      <span class="bold">
        
          
            Jana M. 
            Weber
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Jana M. Weber (orcid.org/0000-0002-2867-0087) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-2867-0087"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">ab</span></sup>, 
      <span class="bold">
        
          
            Zhen 
            Guo
          
          
        
      </span><sup><span class="sup_ref italic">bc</span></sup>, 
      <span class="bold">
        
          
            Chonghuan 
            Zhang
          
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Artur M. 
            Schweidtmann
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Artur M. Schweidtmann (orcid.org/0000-0001-8885-6847) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-8885-6847"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">d</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Alexei A. 
            Lapkin
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Alexei A. Lapkin (orcid.org/0000-0001-7621-0889) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-7621-0889"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">abc</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">Department of Chemical Engineering and Biotechnology, University of Cambridge, West Cambridge Site, Philippa Fawcett Drive, Cambridge CB3 0AS, UK. E-mail: <a href="mailto:aal35@cam.ac.uk">aal35@cam.ac.uk</a></span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Chemical Data Intelligence (CDI) Pte Ltd, Robinson Road, #02-00, 068898, Singapore</span>
      <br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">Cambridge Centre for Advanced Research and Education in Singapore, CARES Ltd. 1 CREATE Way, CREATE Tower #05-05, 138602, Singapore</span>
      <br/><a id="affd"><sup><span class="sup_ref italic">d</span></sup></a><span class="italic">Department of Chemical Engineering, Delft University of Technology, Van der Maasweg 9, Delft 2629 HZ, The Netherlands</span>
    </p><div id="art-admin"><span class="italic bold">Received 
      19th June 2021
    </span></div><p class="bold italic">First published on 14th September 2021</p><hr/><div class="abstract"><h2>Abstract</h2><p>This study highlights new opportunities for optimal reaction route selection from large chemical databases brought about by the rapid digitalisation of chemical data. The chemical industry requires a transformation towards more sustainable practices, eliminating its dependencies on fossil fuels and limiting its impact on the environment. However, identifying more sustainable process alternatives is, at present, a cumbersome, manual, iterative process, based on chemical intuition and modelling. We give a perspective on methods for automated discovery and assessment of competitive sustainable reaction routes based on renewable or waste feedstocks. Three key areas of transition are outlined and reviewed based on their state-of-the-art as well as bottlenecks: (i) data, (ii) evaluation metrics, and (iii) decision-making. We elucidate their synergies and interfaces since only together these areas can bring about the most benefit. The field of chemical data intelligence offers the opportunity to identify the inherently more sustainable reaction pathways and to identify opportunities for a circular chemical economy. Our review shows that at present the field of data brings about most bottlenecks, such as data completion and data linkage, but also offers the principal opportunity for advancement.</p></div><hr/><table><tr><td class="biogPlate"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-p1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-p1.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-p1.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Jana M. Weber</span>
                </span></p></td><td><i><p>Jana M. Weber's research focuses on machine learning and complexity science for the development of sustainable process solutions. Jana is currently pursuing her Ph.D. at the Department of Chemical Engineering and Biotechnology at the University of Cambridge (UK). Previously, she studied environmental engineering at RWTH Aachen University (Germany) and Aalto University (Finland). In 2020, Jana was recognized as one of the Top 50 Women in Engineering – Sustainability in the UK.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-p2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-p2.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-p2.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Zhen Guo</span>
                </span></p></td><td><i><p>Dr Zhen Guo obtained his Bachelor and Master degree in chemistry in Wuhan University, China, and his PhD in chemical engineering in Nanyang Technological University, Singapore. He focuses on development of in silico solutions for industrial challenges through cheminformatics, data mining, and machine learning.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-p3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-p3.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-p3.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Chonghuan Zhang</span>
                </span></p></td><td><i><p>Chonghuan Zhang received his Bachelor's degree in engineering from The University of Sydney (Australia) in 2017 and Master's degree in Chemical Engineering and Biotechnology from University of Cambridge (UK) in 2018. He is currently a PhD candidate at University of Cambridge (UK) and he focuses on applying data informatics to chemistry and synthetic biology.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-p4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-p4.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-p4.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Artur M. Schweidtmann</span>
                </span></p></td><td><i><p>Artur M. Schweidtmann is an assistant professor for chemical engineering at Delft University of Technology and co-director of a lab within the TU Delft AI Labs Programme. His research focuses on the combination of artificial intelligence and chemical engineering. He received his Master of Science from RWTH Aachen University in 2017 and defended his Ph.D. from RWTH in 2021, both in Chemical Engineering. During his studies, he spent the academic year 2013/2014 at Carnegie Mellon University as a visiting student via the DAAD ISAP program. He performed his Master thesis at the University of Cambridge.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-p5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-p5.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-p5.gif"/></a><p><span class="graphic_title">
                  <span class="bold">Alexei A. Lapkin</span>
                </span></p></td><td><i><p>Alexei Lapkin studied chemistry at Novosibirsk State University and obtained his PhD in Chemical Engineering from the University of Bath. He then held academic positions at Universities of Bath and Warwick, before being elected to his current position of Professor of Sustainable Reaction Engineering @ Cambridge in 2013. His research is focusing on process intensification and developing sustainable chemical technologies supported by artificial intelligence and robotics (lapkingroup.com). He is a co-director of EPSRC Centre for Doctoral Training “Syntech” and a director of ERDF co-funded Innovation Centre in Digital Molecular Technologies “iDMT”.</p></i></td></tr></table><hr/>
    
      
      <h2 id="sect325"><span class="a_heading">1. Introduction</span></h2>
      <span>Chemical industries worldwide heavily rely on non-renewable fossil feedstocks, which results in linear economy models, <span class="italic">i.e.</span> extract-make-use-dispose schemes.<a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> This contributes to chemical production becoming the central driver of global oil consumption by the year 2030.<a title="Select to navigate to references" href="#cit2"><sup><span class="sup_ref">2</span></sup></a> To tackle the problem, research efforts shifted towards studying the utilization of renewable feedstocks within chemical industries, such as cellulose,<a title="Select to navigate to references" href="#cit3"><sup><span class="sup_ref">3</span></sup></a> lignin,<a title="Select to navigate to references" href="#cit4"><sup><span class="sup_ref">4</span></sup></a> chitin,<a title="Select to navigate to references" href="#cit5"><sup><span class="sup_ref">5</span></sup></a> or bio-wastes.<a title="Select to navigate to reference" href="#cit6"><sup><span class="sup_ref">6–8</span></sup></a> In recent decades, the scientific community developed new reactions and engineering techniques to produce various value-added chemical compounds from renewable feedstocks.<a title="Select to navigate to reference" href="#cit5"><sup><span class="sup_ref">5,9–11</span></sup></a> However, one of the main problems when developing sustainable processes is the lack of access to information on all multiple co-existing options, hindering a systematic way to shape early, but key process decisions.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> Novel process routes based on renewable or waste feedstock are in fierce competition with the petrochemical-based market,<a title="Select to navigate to reference" href="#cit13"><sup><span class="sup_ref">13–16</span></sup></a> where companies operate at economies of scale and have optimised both processes and supply chains for over a century.<a title="Select to navigate to reference" href="#cit17"><sup><span class="sup_ref">17,18</span></sup></a> Thus, a shift in industrial techniques, if not enforced through strict regulation, can only happen if sustainable alternatives are equally good or even economically superior solutions. Yet, even early-stage schemes of novel reaction routes require process modelling, pre-collected data, and last but not least chemical intuition, making them a long and manual selection process. Thus, there is a need for a systematic and fast tool to identify the most promising reaction routes. The three key aspects to develop such a chemical data intelligence tool are (i) data, (ii) assessment metrics, and (iii) decision-making approaches (see <a title="Select to navigate to figure" href="#imgfig1">Fig. 1</a>) and will be discussed throughout this study.</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f1.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">The three main areas: data, metrics, and decision-making directly influence the search for sustainable reaction pathways in reaction network, but also influence each other. To enable large-scale systematic searches and rankings, automation within and between all areas is key.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">A systematic picture of the available knowledge on reaction data can be illustrated through a network of chemical reactions, where species are connected with one another through chemical reactions – products and reactants of each reaction are connected. <a title="Select to navigate to figure" href="#imgfig2">Fig. 2</a> illustrates how the evolving reaction network can connect feedstock molecules (<span class="italic">e.g.</span> from biomass) to target molecules (<span class="italic">e.g.</span> drug compounds) over a sequence of reactions involving intermediate molecules (<span class="italic">e.g.</span> chemical commodities). The increase in electronic data recordings and thus, data availability, has paved the way for rapid progress on reaction networks mined from large chemical databases, sometimes called the chemical universe or the network of organic chemistry (NOC). Fialkowski <span class="italic">et al.</span> first introduced the study of organic synthesis reactions with a network representation based on the Beilstein Database.<a title="Select to navigate to references" href="#cit19"><sup><span class="sup_ref">19</span></sup></a> Then, studies on the topology and growth of the network,<a title="Select to navigate to reference" href="#cit20"><sup><span class="sup_ref">20–22</span></sup></a> synthesis planning through the network,<a title="Select to navigate to reference" href="#cit23"><sup><span class="sup_ref">23,24</span></sup></a> and applications to One-Pot-Reactions have followed.<a title="Select to navigate to references" href="#cit25"><sup><span class="sup_ref">25</span></sup></a> In our previous works, we have highlighted the potential of the NOC for process route selection and for the identification of strategic molecules for sustainable supply chains.<a title="Select to navigate to reference" href="#cit26"><sup><span class="sup_ref">26–28</span></sup></a> With rapid increases in digitalisation, it is worthwhile to revisit the NOC and identify future avenues for chemical reaction data. Information extraction and information representation play key roles, where tools such as natural language processing (NLP) can lead to more complete datasets and ontological representation, or knowledge graphs, allow machines to better “understand” the data.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f2.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">Reaction networks connect feedstock molecules with target molecules. A sequence of reactions, a reaction route or pathway, is required to connect feedstocks with desired target molecules through intermediate species. Note that an intermediate species is the product of one reaction and the reactant of another in a chemical supply chain, rather than a transient species in a single reaction step. Thus, they are retrievable (as reactant or product) from the database of choice. The best sequence can be identified from large sets of chemical reactions through optimisation of the resulting reaction network. Note that the reaction network is illustrated as a bipartite network here, but multiple other representations are possible.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">The combination of large sets of chemical reaction data and decision-making algorithms is a prerequisite for fast and systematic assessment of reaction pathways. Reaction Network Flux Analysis (RNFA), which was inspired by the analysis of metabolic networks,<a title="Select to navigate to references" href="#cit29"><sup><span class="sup_ref">29</span></sup></a> was first introduced by Voll and Marquardt who identified optimal pathways for bio renewables processing based on a literature review.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> In RNFA, a steady state of the material flows in the system is assumed and the system is optimised for the production of the desired products from available feedstock molecules subject to mass balance constraints.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> The approach was further extended by advanced metrics and supply chain considerations,<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31,32</span></sup></a> to represent process networks,<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a> and was adopted for the assessment of routes to biopolymers.<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> An alternative to the steady state model for reaction network optimisation was presented in ref. <a title="Select to navigate to references" href="#cit35">35</a>. Methodologies are evolving quickly, yet their full potential will only be realised if connected to large chemical databases.</p>
      <p class="otherpara">To evaluate routes in reaction networks, appropriate criteria must be used. The twelve principles of green chemistry,<a title="Select to navigate to references" href="#cit36"><sup><span class="sup_ref">36</span></sup></a> the “productivity scheme”,<a title="Select to navigate to reference" href="#cit37"><sup><span class="sup_ref">37,38</span></sup></a> and their extension towards green engineering,<a title="Select to navigate to references" href="#cit39"><sup><span class="sup_ref">39</span></sup></a> the “improvement scheme”,<a title="Select to navigate to references" href="#cit40"><sup><span class="sup_ref">40</span></sup></a> have established a common understanding of environmental considerations in chemical engineering. Metrics, such as the environmental impact factor (E-factor), atom economy (AE), or energy requirements enable us to assess environmental considerations within chemical processes.<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a> The field is going through a transition from green to sustainable chemistry, which requires the consideration of wider system boundaries. This has been actively discussed and incorporated in life cycle assessment (LCA) literature.<a title="Select to navigate to reference" href="#cit41"><sup><span class="sup_ref">41–43</span></sup></a> Sustainability criteria should be able to simulate the system boundaries (<span class="italic">e.g.</span> demand/supply outside the network) and should be retrievable in an automated manner on early process development stage data sources. Wider chemical reaction systems have previously been analysed based on exergetic efficiencies and sets of chemical heuristics.<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a></p>
      <p class="otherpara">Only together can the areas of data, metrics, and decision-making make the most use of chemical data intelligence and enable practitioners to plan the most sustainable reaction routes. In this work, we explore the potential of semantic data for rich and structured chemical knowledge. Advances in the fields such as NLP and recommendation systems are further reviewed as they promise to tackle the challenge of data scarcity. For sustainability aspects in chemical reactions, we elucidate the importance, as well as challenges, of system thinking. We research a navigation system for chemical space similar to Google Maps, showing us the most sustainable pathways in the entanglement of chemical reactions.</p>
      <p class="otherpara">We provide a roadmap with our recommendations for the development of a systematic early-stage sustainability assessment tool in <a title="Select to navigate to figure" href="#imgfig3">Fig. 3</a>. Within the three research fields, we identify impact opportunities and provide action steps and approximate time frames. The foundation for the recommendations is explained throughout this work in the detailed sections on data, metrics, and decision-making.</p>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f3.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">Roadmap towards systematic early-stage sustainability assessment. Impact opportunities and action points in the three areas data, metrics, and decision-making (d-m) are outlined. Action points are grouped into timeline categories (&lt;3, 3–5, and 5–10 years) and are explained in the boxes below.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara"># 1 Data impact opportunities and action points. The first opportunity is the development of a chemical big open linked data (BOLD) structure. Emphasis lays on the coverage of freely available datasets where single entries are recorded with a uniform resource identifier, enabling linkage to the chemical context and to adjacent fields, such as substance emissions and market values of the molecules. As second impact opportunity, we strongly recommend the inclusion of biological data. This includes molecular transformations from systems biology, purified enzymatic reactions and whole cell transformation, as well as biological feedstocks as primary raw material and as secondary raw material from waste streams. Lastly, we emphasise the importance of complete data structures. While for novel publications, journal standards can enable the recording of stoichiometry, yield, and reaction conditions, the body of already published reactions needs to be revisited in order to withdraw such information in an electronic standard. NLP can gather previously stated information and predictive models can be utilised for the fractions where original sources do not contain the information.</p>
      <p class="otherpara"># 2 Metrics impact opportunities and action points. The first opportunity is the transition from green to sustainable metrics. Herein, we recognise the importance to focus on the integration of resource-based assessment, <span class="italic">e.g.</span> exergy, which complements emission-based metrics, as well as social and economic assessment. Furthermore, only cross-domain standards, <span class="italic">e.g.</span> on allocation and the system boundary, can lead to a successful tool as the chemical sectors spans multiple domains. Last, but not least, we recommend to focus on early-stage metrics, as only early-stage decision-making can lead to inherently sustainable pathways. The second impact opportunity is the measurement of circularity potentials. In order to evaluate the sustainability of a pathway, knowledge on potential uses of waste streams generated throughout the process are indispensable. Considerations on possible upstream treatment and on the stability of molecular properties during multiple cycles of reuse are of interest. The last impact opportunity is the prediction of molecular properties relevant for sustainability assessment, <span class="italic">e.g.</span> prediction of chemical exergies from molecular structures.</p>
      <p class="otherpara"># 3 Decision-making impact opportunities and action points. Future decision-making systems for early-stage sustainability assessment are required to evolve at two fronts: increasing model accuracy and decreasing model complexity. On the one hand, it is desirable to derive modelling frameworks which take for instance solvents, separations, upstream treatment, material circularity, and sequential manufacturing into account. On the other hand, linear models or heuristic, solving strategies for non-linear models, are essential to facilitate large-scale, and thus, systematic assessment. Last, but not least, sustainability is a dynamic systems problem, which requires a decision-making support able to interact with the dynamic nature of the system. While metrics might be able to map current market prices and data can show the demand for or supply of certain materials at regarded geographic locations, the envisioned decision-making framework should work beyond these static snapshots of the system, <span class="italic">e.g.</span> following agent-based modelling approaches.</p>
      <p class="otherpara">Enabling the three areas to evolve together will be the key aspect for well-reasoned reaction pathway development, tackling the different aspects of sustainability. We would like to stress the following interfaces between the areas in particular.</p>
      <p class="otherpara"># Interface 1: data is the foundation for metrics and decision-making. Accessible and well-documented databases are a prerequisite to further development of metrics. Molecular properties such as chemical exergy, thermodynamic properties or toxicity values are key aspects of sustainability assessment. Further development of computational tools to automatically generate metrics for given systems are needed. Additionally, data on reaction structures, <span class="italic">i.e.</span> the stoichiometric relationships, are essential to formulate mass balances as physical constraints of the decision-making formalism. We anticipate that only clear communication of data needs from the metrics and decision-making community will enable sufficiently quick extensions to the current data sources to be developed. Alongside, conversations between the communities should include ontology development to define the characterisation and relationships of data.</p>
      <p class="otherpara"># Interface 2: co-development of metrics and decision-making. Throughout this article, we argue that sustainability is a systems science. For a future sustainable chemical supply chain, it is essential to evaluate proposed reactions within their environment, rather than detached from it. Thus, the metrics required to assess the sustainability of novel reaction pathways need to capture wider system interactions considered in the decision-making approach. Here, the foundations of assessing the greenness or the sustainability of reactions are provided by the metrics community, but new requirements for metrics will arise from within the systems modelling community. We anticipate large benefits if both domains come together.</p>
      <p class="otherpara"># Interface 3: defining decision-making environments through regional and dynamic data. Data on the price of molecules illustrates the economic interaction of a system with its environment across the system's boundary. This interaction is further defined by data on the availability and the demand of each molecule, thus mapping the chemical supply chain. However, market prices and supply chains as well as the energy price are temporal and spatially fluctuating. In the long term, modelling novel reaction pathways within a circular chemical supply chain requires dynamic interactive environment descriptions, rather than static snapshots. Here, joint research is needed to develop dynamic environments based on suitable data sources. Future advanced tools may even include regional policy insights, as well as costs associated to infrastructure and personal.</p>
    
    
      
      <h2 id="sect390"><span class="a_heading">2. Data</span></h2>
      <span>Large volumes of open and big data have revolutionised many fields of our modern society.<a title="Select to navigate to references" href="#cit44"><sup><span class="sup_ref">44</span></sup></a> While enormous improvements have driven developments in areas such as computer vision, or language recognition, chemical data has yet to overcome urgent challenges, such as establishing openly accessible data with standardised representations and improving quality inconsistencies of existing data.<a title="Select to navigate to references" href="#cit45"><sup><span class="sup_ref">45</span></sup></a> FAIR scientific data – findable, accessible, interpretable, and reusable<a title="Select to navigate to references" href="#cit46"><sup><span class="sup_ref">46</span></sup></a> – evolving into BOLD concepts<a title="Select to navigate to reference" href="#cit47"><sup><span class="sup_ref">47–49</span></sup></a> will push chemical discovery and is essential for cross-disciplinary tasks, such as sustainability assessments.</span>
      
        <h3 id="sect398"><span class="b_heading">Databases and accessibility</span></h3>
        <span>To facilitate large-scale reaction route screening, access to large reaction databases is the stepping stone. When aiming for more sustainable process routes, it is worthwhile to discuss the extraction of reactions from conventional chemical transformation as well as biosynthetic conversion strategies and alternative chemical conversion strategies.</span>
        <div>
          <span id="sect401"/><span class="c_heading_indent">Conventional chemical reaction databases. </span>
          <span>There exists a variety of chemical reaction databases with different sizes and accessibility rights as well as distinct coverage of the chemical space. <a title="Select to navigate to table" href="#tab1">Table 1</a> outlines a selection of common databases for organic reactions. CASREACT<a title="Select to navigate to footnote" href="#fn1">†</a> and Reaxys<small><sup>TM</sup></small><a title="Select to navigate to footnote" href="#fn2">‡</a> (in the following called Reaxys) are by far the largest databases for chemical reactions. They include scientific literature and a selection of patents, but require users to buy licenses to work with large-scale data. The database called Pistachio<a title="Select to navigate to footnote" href="#fn3">§</a> developed by the company NextMove stores reaction from US patents and has released a publicly available subset CC-Zero<a title="Select to navigate to footnote" href="#fn4">¶</a> of over one million reactions. SPRESI<a title="Select to navigate to footnote" href="#fn5">||</a> is another database for organic reactions, which provides a subset of 500<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 reactions as a free app. The USPTO<a title="Select to navigate to footnote" href="#fn6">**</a> database is the smallest, yet it is entirely openly accessible. Open reaction databases are gaining increasing momentum. One recent example is the open reaction database,<a title="Select to navigate to footnote" href="#fn7">††</a> which is a multi-institution initiative to aid machine learning (ML) tasks in chemistry/chemical engineering by providing structured and freely available reaction data. The project sits on GitHub and its launch is planned for early 2021. Regarding data coverage, Thakkar <span class="italic">et al.</span> have outlined the differences in data coverage from multiple sources including a dataset based on electronic notebooks from AstraZeneca.<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> They studied reaction templates within different databases and found that only 2% of templates were common in all considered data sources.<a title="Select to navigate to references" href="#cit50"><sup><span class="sup_ref">50</span></sup></a> Notably, the development of chemical databases is a rapidly developing field. Content breadth and depth are being constantly reviewed and further developed. Reaxys, for instance, now covers supplier information on price, supplier geolocation, packages sizes and much more.</span>
          
<div class="table_caption"><b>Table 1</b> <span id="tab1">Selection of large databases recording chemical reactions</span>
		</div>

              <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/></colgroup>
                
                
                
                <thead align="left">
                  <tr align="left" valign="bottom">
                    <th align="left" valign="bottom" class="border_black">Database</th>
                    <th align="left" valign="bottom" class="border_black">Size</th>
                    <th align="left" valign="bottom" class="border_black">Accessibility</th>
                  </tr>
                </thead>
                <tbody align="left">
                  <tr align="left">
                    <td valign="top">CASREACT</td>
                    <td valign="top">&gt;128 million reactions</td>
                    <td valign="top">Proprietary</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Reaxys</td>
                    <td valign="top">&gt;46 million reactions</td>
                    <td valign="top">Proprietary</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Pistachio</td>
                    <td valign="top">&gt;9 million, patent literature based</td>
                    <td valign="top">Proprietary</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">SPRESI</td>
                    <td valign="top">&gt;4.6 million reactions</td>
                    <td valign="top">Proprietary</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">USPTO</td>
                    <td valign="top">&gt;3.3 million, patent literature based</td>
                    <td valign="top">Open</td>
                  </tr>
                </tbody>
              </table></div></div>
            <hr class="hrule"/><br/>
        </div>
        <div>
          <span id="sect463"/><span class="c_heading_indent">Biosynthetic reaction databases. </span>
          <span>A hybrid system of biosynthetic and conventional chemical synthesis opens up opportunities for efficient (bio)chemical pathways search. Enzymatic reactions can lead to more efficient reaction pathways with reduced operational costs as synthetic biology can enable shortcuts and flexible design of supply chains (see <a title="Select to navigate to figure" href="#imgfig4">Fig. 4</a>), improving redox efficiency. Moderate temperatures/pressure and the avoidance of metal catalysts or hazardous solvents can ease synthesis and lower operational costs.<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> Additionally, enzymatic reactions are well suited to utilise and further functionalise the biological structures in renewable feedstock. Advantages of metabolic alternatives have encouraged synthetic biologists to find biosynthetic routes that produce bulk chemicals and industrial chemicals such as ethanol,<a title="Select to navigate to references" href="#cit52"><sup><span class="sup_ref">52</span></sup></a> benzoic acid,<a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> toluene,<a title="Select to navigate to references" href="#cit54"><sup><span class="sup_ref">54</span></sup></a><span class="italic">etc.</span>, and active pharmaceutical ingredients of pharmaceuticals such as flavonoid<a title="Select to navigate to references" href="#cit55"><sup><span class="sup_ref">55</span></sup></a> and tryptophan.<a title="Select to navigate to references" href="#cit56"><sup><span class="sup_ref">56</span></sup></a> A metabolic map for the production of bio-based chemicals was summarised by Lee <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit57"><sup><span class="sup_ref">57</span></sup></a></span>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f4.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">Illustration of enzymatic process pathways and conventional process pathways based on fossil feedstocks. In conventional processes, feedstocks are first broken into smaller building blocks and then reassembled and functionalised step by step. Renewable feedstock, however, is often already highly functionalised and different enzymatic transformations can make use of this for direct transformation into different stages of the conventional supply chain.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Databases such as Kyoto Encyclopaedia of Genes and Genomes (KEGG),<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><a title="Select to navigate to footnote" href="#fn8">‡‡</a> Rhea,<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><a title="Select to navigate to footnote" href="#fn9">§§</a> and the Enzyme Catalytic-mechanism Database<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a><img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/><a title="Select to navigate to footnote" href="#fn10">¶¶</a> open up opportunities to obtain bio-information for metabolic reaction networks. The most comprehensive biological database is KEGG with currently almost 13<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 recorded reactions.<a title="Select to navigate to references" href="#cit58"><sup><span class="sup_ref">58</span></sup></a> However, in comparison to conventional chemical databases, enzymatic databases are relatively small and sparse at present. Synthetic biologists are actively working towards the prediction of metabolic reaction behaviours to populate the databases.<a title="Select to navigate to reference" href="#cit61"><sup><span class="sup_ref">61,62</span></sup></a></p>
        </div>
        <div>
          <span id="sect493"/><span class="c_heading_indent">Alternative conversion strategies. </span>
          <span>Further opportunities for discovering new reaction pathways include electrochemical or photochemical transformations. Electrocatalytic hydrogenations may hydrogenate molecules in water under ambient conditions and thus replace conventional hydrogenation steps typically requiring elevated pressures.<a title="Select to navigate to references" href="#cit63"><sup><span class="sup_ref">63</span></sup></a> Besides, electrochemistry may also unlock entirely new synthetic pathways as novel molecular transformations are observed.<a title="Select to navigate to references" href="#cit64"><sup><span class="sup_ref">64</span></sup></a> Harnisch and Urban illustrate the concept of an electrobiorefinery, where they anticipate that the synergies between microbial and electrochemical conversations are likely to impact, amongst others, enlarging product portfolios and exploiting new feedstocks.<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> In particular, they outline electrochemistry for decomposition of bio-based feedstocks, <span class="italic">e.g.</span> lignin pretreatment, to provide chemical feedstocks, <span class="italic">e.g.</span> H<small><sub>2</sub></small>, CH<small><sub>4</sub></small>, or C<small><sub>1</sub></small>- or C<small><sub>2</sub></small>-compounds, but also to electrochemically steer fermentation, <span class="italic">e.g.</span> CO<small><sub>2</sub></small> can be used as carbon source for fermentation cultures.<a title="Select to navigate to references" href="#cit65"><sup><span class="sup_ref">65</span></sup></a> Another promising conversion strategy is photochemistry. Research efforts are focusing, for instance, on the utilisation of sunlight to produce CO, ethanol, or methane from CO<small><sub>2</sub></small> in aqueous solutions, or on solar-driven organic synthesis, where the target is to obtain high-value products.<a title="Select to navigate to references" href="#cit66"><sup><span class="sup_ref">66</span></sup></a> He and Janáky state that utilising solar energy and CO<small><sub>2</sub></small> resources can be expected to yield both fuels and value-added chemicals. They list a range of possible chemical products in their work and compare the performance of different photochemical conversion strategies.<a title="Select to navigate to references" href="#cit67"><sup><span class="sup_ref">67</span></sup></a> In large chemical databases, such as Reaxys, one can specify reaction types, also including electro- or photochemical transformations. This allows specific inclusion of alternative conversion strategies and potentially enlarges the toolbox for the development of more sustainable chemical reaction pathways.</span>
        </div>
      
      
        <h3 id="sect512"><span class="b_heading">Data formats</span></h3>
        <span>Linking data sources is often essential for decision-making, but requires rethinking of existing data practices.<a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a> A stepwise approach for data formats, the 5-star plan<a title="Select to navigate to footnote" href="#fn11">||||</a> (<span class="italic">cf.</span><a title="Select to navigate to figure" href="#imgfig5">Fig. 5</a>), was suggested by Tim Berners-Lee, the inventor of the worldwide web. The plan describes a trajectory for data formats starting by open data in any format and resulting in the semantic web.<a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> In the semantic web, data is accessible both for humans and machines as data is stored with structure and context, generating meaningful content. The content is made comparable between sources through ontologies.<a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> Ontologies exploit triple relationships, <span class="italic">e.g.</span> Acetone 〈is used as〉 solvent is broken into two concepts “molecule” and “solvent” and their relationship is given by 〈is used as〉. These generate metadata structures, <span class="italic">i.e.</span> reusable knowledge representation.<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> Additional terminology for data structures is explained in <a title="Select to navigate to table" href="#tab2">Table 2</a>.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f5.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">5-Star plan for open data suggested by Tim Berners-Lee (based on https://5stardata.info/en/).</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        
<div class="table_caption"><b>Table 2</b> <span id="tab2">Explanation of specific terminology in data representations</span>
		</div>

            <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/></colgroup>
              
              
              <thead align="left">
                <tr align="left" valign="bottom">
                  <th align="left" valign="bottom" class="border_black">Term</th>
                  <th align="left" valign="bottom" class="border_black">Description</th>
                </tr>
              </thead>
              <tbody align="left">
                <tr align="left">
                  <td valign="top">Knowledge graph</td>
                  <td valign="top">A knowledge graph uses data in graph structure. Data entities and their semantic types and properties are linked with each other. Knowledge graphs can allow machines and humans to reason from the data.</td>
                </tr>
                <tr align="left">
                  <td valign="top">Metadata</td>
                  <td valign="top">Metadata is data about data. One example is structural metadata, which provides schemes and order to data.</td>
                </tr>
                <tr align="left">
                  <td valign="top">Ontology</td>
                  <td valign="top">An ontology provides a uniform approach to describe data semantically. It is a specific conceptualization in a format that allows for reasoning and inference.<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a></td>
                </tr>
                <tr align="left">
                  <td valign="top">Semantics</td>
                  <td valign="top">Semantics provide methods to include meaning to information constructs. Adding meaningful tags to pieces of data brings about better readability as it is an abstraction of what the stored piece of data resembles in the real world.</td>
                </tr>
                <tr align="left">
                  <td valign="top">Semantic Web</td>
                  <td valign="top">Tim Berners-Lee, who is the inventor of the World Wide Web, defines the Semantic Web not as a separate one, but as an extension of it. His vision is a web of information which can be processed by computers and which brings structure to the meaningful content of web pages.<a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a></td>
                </tr>
                <tr align="left">
                  <td valign="top">Unified resource identifier</td>
                  <td valign="top">Unified resource identifiers are used to identify any object, <span class="italic">e.g.</span> people, books, or places, in web technologies. In semantic web technologies, they help linking objects from different sources.</td>
                </tr>
              </tbody>
            </table></div></div>
          <hr class="hrule"/><br/>
        <p class="otherpara">Scientific practices to record data openly and in machine-readable formats lack behind. Large chemical databases such as CAS<a title="Select to navigate to footnote" href="#fn12">***</a> are fundamental sources of chemical information, however, they do not fulfil the requirements of the semantic web when it comes to information access and representation.<a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> This is due to current ways of publishing, <span class="italic">e.g.</span> providing PDF files, which support human readability, but are ill-suited for data mining and analysis.<a title="Select to navigate to reference" href="#cit68"><sup><span class="sup_ref">68,70</span></sup></a> The project open research knowledge graph has made it their mission to change the document-centric information flow in the scientific community into knowledge-based information flows.<a title="Select to navigate to reference" href="#cit71"><sup><span class="sup_ref">71–73</span></sup></a> The final product is envisioned to be a “structured, interlinked and semantically rich knowledge graph”.<a title="Select to navigate to references" href="#cit73"><sup><span class="sup_ref">73</span></sup></a></p>
        <p class="otherpara">Chemical data often lacks relational information and is stored in diverse formats. For example, molecule representations range from structural chemical formulas over numeric descriptors to string representations, <span class="italic">e.g.</span> SMILES or SMARTS. The chemical mark-up language was introduced to offer semantics for chemical data.<a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> It allows the integration of various entities, <span class="italic">e.g.</span> molecules, spectra, and reactions in mark-up text for electronic use. Adding relationships to the entities, an ontological structure of information emerges. A few early ontologies for chemical engineering have been developed.<a title="Select to navigate to reference" href="#cit75"><sup><span class="sup_ref">75,76</span></sup></a> OntoCAPE, for example, defines an ontology for chemical processes.<a title="Select to navigate to reference" href="#cit77"><sup><span class="sup_ref">77,78</span></sup></a> OntoCAPE introduces chemical primitives such as system property, physical dimensions or units to define a system. System properties can have numerous values. The authors illustrate this through the system property “temperature” where different values can be distinguished from each other by the system property “time” both given in their respective units, here degrees Kelvin and time in hours, to record a temperature profile over time.<a title="Select to navigate to references" href="#cit77"><sup><span class="sup_ref">77</span></sup></a> The extension of such formalisms to broader domains of chemical applications brings about the potential to gather chemical data in a structured and standardised way even across different subject areas.</p>
        <p class="otherpara">Linked data is much needed for sustainability assessment as it allows for holistic and cross-disciplinary assessment.<a title="Select to navigate to references" href="#cit79"><sup><span class="sup_ref">79</span></sup></a> The development of knowledge graph technologies potentially enables efficient data handling, including conditional data queries within or between subject areas, and more accurate data inference, due to high contextualising. The group of Prof. Kraft optimises an eco-industrial park, integrating water networks, waste streams, and energy links, to minimise environmental impacts through semantic web technologies.<a title="Select to navigate to reference" href="#cit75"><sup><span class="sup_ref">75,80–82</span></sup></a> Similar structures of semantic web will be essential to enable sustainability assessment within reaction networks, where temperatures, yields, solvents, and reaction stoichiometry should be recorded and linked with each reaction.<a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> An entity-based data format would allow us to link further information (<span class="italic">e.g.</span> data on waste streams and their compositions, or on regional availabilities of renewable energy) in a modular way, paving the way for more holistic considerations in future reaction planning.</p>
      
      
        <h3 id="sect583"><span class="b_heading">Data completion</span></h3>
        <span>While text-mining has enabled gathering large-scale chemical information, such as properties and structures of molecules, connections between reactants and products, to populate electronic chemical databases (<span class="italic">cf.</span> Section Databases and accessibility), the methods used in creating the datasets sometimes lack accuracy and miss-classify important pieces of information. Missing stoichiometric data and incorrect recording of multi-step reactions as single-step reactions prevents mass balances within a reaction to be calculated and are major hurdles for decision-making based on automatically generated process options. Furthermore, records of reagents, solvents and catalysts are often inconsistent, <span class="italic">e.g.</span> they are sometimes absent or incorrectly recorded as a reactant and no information about required quantities is provided. This is challenging for sustainability assessments because the use of reagents, solvents, and catalysts has a significant influence on environmental impacts.<a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> Additionally, in some cases, clear identification of specific chemical species within the databases is problematic as mixtures of enantiomers are recorded as pure compounds or entries simply state “mixture out of C3 to C6 hydrocarbons”, without reference to their molecular structures or a database registry numbers. The same problem exists in the identification of complex feedstocks without exact structure and composition, such as lignin, chitin, or cellulose. Inconsistency in temperature and pressure recordings makes an energetic analysis of processes difficult. With rapid developments in the field of NLP and high throughput experiments (HTE), it is expected that data quality will quickly improve and some of these current hurdles for algorithmic use of chemical information will be overcome.</span>
        <div>
          <span id="sect589"/><span class="c_heading_indent">Information extraction from scientific literature. </span>
          <span>NLP describes a range of computational techniques to analyse and represent natural text in a human-like manner for a variety of applications.<a title="Select to navigate to references" href="#cit85"><sup><span class="sup_ref">85</span></sup></a> One application especially relevant to this review is information extraction, the task of gaining structured knowledge from text. NLP aims (i) to aid human–human communication, <span class="italic">e.g.</span> translation tasks, (ii) human–machine communication, <span class="italic">e.g.</span> conversational agents such as Apple's Siri, or (iii) to bring about benefit for machines and humans, <span class="italic">e.g.</span> through learning from large amounts of data.<a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a> According to Hirschberg and Manning, NLP has seen an immense boost within the last few years due to: an increase in computational power, large availability of linguistic data, successful ML algorithms, <span class="italic">e.g.</span> the transformer model,<a title="Select to navigate to references" href="#cit87"><sup><span class="sup_ref">87</span></sup></a> and a better understanding of human languages.<a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a> Information extraction from scientific literature brings about great benefits, not only to fill in gaps in data but also to keep track of the ever growing body of literature.<a title="Select to navigate to references" href="#cit86"><sup><span class="sup_ref">86</span></sup></a></span>
          <p class="otherpara">The last decade has shown immense progress of NLP techniques within chemistry and related fields, making it a promising avenue to overcome data completion tasks. Jessop <span class="italic">et al.</span> have developed the Open-Source Chemistry Analysis Routines (OSCAR) software to read entities and information in chemistry publications.<a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> OSCAR4, a library onto which text-mining tools for chemistry can be built, was released and the authors illustrated that OSCAR may also be applied to other areas of physical sciences as customisation through different dictionaries is possible.<a title="Select to navigate to references" href="#cit88"><sup><span class="sup_ref">88</span></sup></a> Such transdisciplinary data systems are of increasing importance as sustainability assessment requires a variety of data. Krallinger <span class="italic">et al.</span> present a review on the access of chemical information through text mining techniques and especially value chemical entity recognition as well as the interlinkage to biological data.<a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a> NLP techniques have also gained a foothold in related fields, such as nanotechnology,<a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a> medical/clinical text documents,<a title="Select to navigate to references" href="#cit91"><sup><span class="sup_ref">91</span></sup></a> and biomedical texts.<a title="Select to navigate to references" href="#cit92"><sup><span class="sup_ref">92</span></sup></a> However, linking data from scientific publications of different chemical subject areas is potentially problematic as definitions and reporting standards may vary. Automated assessment frameworks of data quality for linked open data<a title="Select to navigate to reference" href="#cit93"><sup><span class="sup_ref">93,94</span></sup></a> may offer the potential to identify pieces of misleading information between communities.</p>
        </div>
        <div>
          <span id="sect611"/><span class="c_heading_indent">Data acquisition through high-throughput experiments. </span>
          <span>In HTE multiple reactions are performed in parallel to quickly answer a specific chemical question.<a title="Select to navigate to references" href="#cit95"><sup><span class="sup_ref">95</span></sup></a> High-throughput virtual experiments (HTVE) are often utilised to examine material or drug leads at a large scale, when experimental searching becomes impractical due to high cost or technical issues. They are commonly used in combination; experimental data can be used to calibrate HTVE, while results from HTVE guide further experimental explorations.<a title="Select to navigate to reference" href="#cit96"><sup><span class="sup_ref">96–98</span></sup></a> This iterative approach has proven to be very powerful, especially when combined with ML guided exploration algorithms.<a title="Select to navigate to reference" href="#cit95"><sup><span class="sup_ref">95,99</span></sup></a> Eyke <span class="italic">et al.</span> employed data produced by HTE to train a ML model, and the ML model predicted reaction outcomes and selected the most informative experimental region for HTE to explore further.<a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> Chen and Visco built a support vector machine (SVM) model on the basis of experimental data and molecular descriptors, which they trained for the identification of drug candidates<a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> and Li <span class="italic">et al.</span> demonstrated that artificial neural networks (ANNs) trained on density functional theory (DFT) data were able to capture complex absorbate–metal interaction, providing guidance on the design of bimetallic catalysts.<a title="Select to navigate to references" href="#cit100"><sup><span class="sup_ref">100</span></sup></a></span>
          <p class="otherpara">To further accelerate data collection speed, robotic platforms for chemical experiments have become the focus of many studies.<a title="Select to navigate to reference" href="#cit101"><sup><span class="sup_ref">101–104</span></sup></a> For instance in 2004, King <span class="italic">et al.</span> reported an automatic experimental system called “robot scientist” that was able to independently conduct an entire research cycle, including planning, testing, analysis and re-run if hypothesis and results were inconsistent.<a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> In the platform assembled by Coley <span class="italic">et al.</span>, synthetic routes were proposed by a retrosynthesis software and organic synthesis was conducted in flow reactors, automatically configured by a robotic arm.<a title="Select to navigate to references" href="#cit104"><sup><span class="sup_ref">104</span></sup></a> In Cronin's group, a robotic platform was constructed along with a standardized architecture for organic synthesis, called the Chemputer.<a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a> By taking advantage of various smart hardware and programming languages, Chemputer system showed the potential to standardize the whole automatic experimental process, from conducting synthesis to generating reports.</p>
          <p class="otherpara">Advanced data analysis techniques, such as the transformer-based model developed by Schwaller <span class="italic">et al.</span>, perform well over data from HTE, but are still not feasible for processing historical experimental data, which often suffers from high inconsistencies.<a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> With the continuous improvements in HTE, HTVE, automation, and data analysis, highly consistent and reliable experimental data may quickly expand, bringing to light more reliable data standards in larger regions of the chemical space.</p>
        </div>
      
      
        <h3 id="sect632"><span class="b_heading">Data inference</span></h3>
        <span>Some data is never reported in primary research publications or patents, but data may be augmented through data inference – a cheap and fast alternative to experimental studies. In this section, we first outline recommender systems as a general way to complete data matrixes and then highlight developments for data inference of the specific contents as illustrated in <a title="Select to navigate to figure" href="#imgfig6">Fig. 6</a>. Relevant for reaction route search are: firstly, missing reaction conditions, <span class="italic">e.g.</span> temperature, pressure, and reagents, secondly, reaction outcomes, <span class="italic">e.g.</span> yield is not always available and thirdly, reaction structures; <span class="italic">e.g.</span> reactants and/or products and the reaction stoichiometry are missing.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f6.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">Illustration of data completion issues for basic chemical reaction information. A set of necessary reaction information for early-stage assessment is shown in the top line. Relevant pieces of information are grouped into reaction structure, reaction conditions, and reaction outcomes in the second line. A, B, C, and D represent molecules and the table below represents structural information of the molecules. Temperatures and pressures can be named either as explicit numbers, as ranges or referred to as ambient. Solvents, catalysts, and reagents are often named; however, their quantity remains unknown. Note that a combination of these issues is most often the case.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <div>
          <span id="sect641"/><span class="c_heading_indent">Recommender systems. </span>
          <span>Recommender systems have been an effective approach to deal with information overload and are seen most promising in problems related to “over-choice” of options. A standard recommender system solves a problem of a set of <span class="italic">n</span> users and <span class="italic">m</span> items, which it generally recommends to the users according to their preference. The relationships between users and items are commonly represented in a <span class="italic">n</span> × <span class="italic">m</span> matrix, being the core element of the recommender system.<a title="Select to navigate to references" href="#cit107"><sup><span class="sup_ref">107</span></sup></a> The matrices are often very sparse as little information is originally available and the aim is to predict what the missing cells will be.<a title="Select to navigate to references" href="#cit107"><sup><span class="sup_ref">107</span></sup></a> The entries may range from single bits to unstructured text.<a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a> One prominent example of a recommender system has been the Netflix challenge, where a price of one million US$ was awarded to the team to first model the dataset and predict new ratings to a specified accuracy.<a title="Select to navigate to references" href="#cit109"><sup><span class="sup_ref">109</span></sup></a> Matrix completion methods or graph recommender systems have been applied for such problems.<a title="Select to navigate to reference" href="#cit107"><sup><span class="sup_ref">107,110,111</span></sup></a> For a deeper understanding of recommender systems and their current trends and challenges as well as the underlying deep-learning strategies we refer the reader to these surveys.<a title="Select to navigate to reference" href="#cit107"><sup><span class="sup_ref">107,112,113</span></sup></a></span>
          <p class="otherpara">Within the domain of chemistry, there are early works that recognise the impact of recommender systems for both experimental and computational data. Savage <span class="italic">et al.</span> recommend candidate molecules as reactants for the synthesis of desired products. They formulate the problem as link prediction over a graph base, where links represent reactant–product relationships and provide chemical knowledge in the form of molecular fingerprints.<a title="Select to navigate to references" href="#cit114"><sup><span class="sup_ref">114</span></sup></a> In 2020, Jirasek <span class="italic">et al.</span> have shown an application of a recommender system for the prediction of binary activity coefficients.<a title="Select to navigate to references" href="#cit115"><sup><span class="sup_ref">115</span></sup></a> Other examples within chemical engineering include the use of recommender systems to predict drug side effects,<a title="Select to navigate to references" href="#cit116"><sup><span class="sup_ref">116</span></sup></a> to estimate the relevance of chemical compounds to form crystals,<a title="Select to navigate to references" href="#cit117"><sup><span class="sup_ref">117</span></sup></a> and for material choices in polymerisation experiments.<a title="Select to navigate to references" href="#cit118"><sup><span class="sup_ref">118</span></sup></a></p>
        </div>
        <div>
          <span id="sect662"/><span class="c_heading_indent">Inference of reaction outcomes. </span>
          <span>For reaction outcomes, we focus on the inference for yield data. At present, yield records are far from sufficient in most databases. For instance, an exemplary dataset from Reaxys database with 17 million reaction records contains around nine million reactions without any yield information. Additionally, among reaction records with yield information, the yields of only a few products are listed.</span>
          <p class="otherpara">Data-driven approaches are promising avenues for yield predictions. Through advances in HTE techniques, detailed and structured experimental data became more easily available. Simultaneously, ML methods evolved and machine-readable representation of molecules and reactions is under constant development. One of the earliest approaches of ML for yield prediction was presented by Kito <span class="italic">et al.</span> and predicted the selectivity of catalytic oxidative dehydrogenation reaction products by using an ANN.<a title="Select to navigate to references" href="#cit119"><sup><span class="sup_ref">119</span></sup></a></p>
          <p class="otherpara">Two trends emerged for yield prediction afterwards. On the one hand, models are based on more accurate, but expensive inputs through descriptors based on DFT and focus on specific reaction types, on the other hand, models aim to identify more generic relationships for multiple reaction types based on cheaper molecular representations. DFT-based descriptors were employed by Yada <span class="italic">et al.</span> for tungsten-catalyzed epoxidation of alkenes by using a linear function ensembled in a logistic regression model,<a title="Select to navigate to references" href="#cit120"><sup><span class="sup_ref">120</span></sup></a> by Estrada <span class="italic">et al.</span> for palladium-catalyzed Buchwald–Hartwig cross-coupling reactions,<a title="Select to navigate to references" href="#cit121"><sup><span class="sup_ref">121</span></sup></a> and by Fu <span class="italic">et al.</span> to predict yields of Pd-catalyzed Suzuki–Miyaura reactions in the microfluidic system.<a title="Select to navigate to references" href="#cit122"><sup><span class="sup_ref">122</span></sup></a> While Yada <span class="italic">et al.</span> only trained their model on 14 data points, Estrada <span class="italic">et al.</span> worked with a set of 4140 reaction results with the aid of high-throughput screening.</p>
          <p class="otherpara">Eyke <span class="italic">et al.</span> utilised reaction fingerprints by concatenation of Morgan fingerprints to guide their experimental design for two specific types of reactions through an ANN.<a title="Select to navigate to references" href="#cit96"><sup><span class="sup_ref">96</span></sup></a> Sandfort <span class="italic">et al.</span> present a broader approach through their structural-based platform for reactivity prediction in organic chemistry.<a title="Select to navigate to references" href="#cit123"><sup><span class="sup_ref">123</span></sup></a> The idea was to use molecular fingerprints as the only type of inputs for ML models to solve all kinds of reaction predictions. While no universally applicable fingerprints for all applications were found, only focusing on C–N cross-coupling reactions, a comparable accuracy to the work of Estrada <span class="italic">et al.</span> was achieved. Skoraczyński <span class="italic">et al.</span> showed reaction examples where subtle changes in molecular structure or reaction conditions led to distinct reaction results, and thus, argued that general descriptors for diverse sets of organic reactions are difficult to set.<a title="Select to navigate to references" href="#cit124"><sup><span class="sup_ref">124</span></sup></a> A very recent development in the area is an algorithm based on NLP.<a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> Their model consists of an encoder and a regression layer to predict yields and is based on reaction SMILES as inputs. On a dataset based on HTE reactions for Buchwald–Hartwig reactions and Suzuki–Miyaura reactions a high prediction accuracy was achieved, while for a generic dataset, the open-source USPTO, poorer accuracy was obtained. While the prediction of accurate yields for diverse reaction types remains a challenge at present, the new methods bring about promising outlooks.</p>
        </div>
        <div>
          <span id="sect686"/><span class="c_heading_indent">Inference of reaction conditions. </span>
          <span>Methods for prediction of reaction conditions, also known as reaction context,<a title="Select to navigate to references" href="#cit125"><sup><span class="sup_ref">125</span></sup></a> have evolved from specific methods only valid for certain reaction types and reaction context towards more holistic approaches. The reaction context is made out of discrete decisions, <span class="italic">e.g.</span> catalyst and solvents, and continuous decisions, <span class="italic">e.g.</span> temperature, pressure, and pH-value, which influence one another. Marcou <span class="italic">et al.</span> predicted catalysts and solvents for the Michael reaction through formulating binary classification problems on a set of 198 reactions.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a> The prediction of both catalysts and solvents was correct for only eight out of 52 reactions from an external validation set.<a title="Select to navigate to references" href="#cit126"><sup><span class="sup_ref">126</span></sup></a> Lin <span class="italic">et al.</span> focused on catalyst recommendations for deprotecting reactions and demonstrate their work on catalytic hydrogenation reactions.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a> They used the methodology of condensed graph of reactions to reduce a reaction to a single graph, allowing for descriptors and fingerprints, and employed similarity searches to suggest catalysts.<a title="Select to navigate to references" href="#cit127"><sup><span class="sup_ref">127</span></sup></a> Gao <span class="italic">et al.</span> also recognises such approaches based on chemical similarity, however highlight computational costs in sufficiently large databases.<a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> Segler and Waller aim for conditions recommendations for different types of reactions utilising a knowledge graph.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> The graph consists of two node types (<span class="italic">i.e.</span> reactions and molecules) and a variety of edge types, describing the reaction conditions, <span class="italic">e.g.</span> 〈is reactant in〉, 〈is catalyst in〉, 〈is solvent in〉. New conditions are predicted through node and link completion tasks.<a title="Select to navigate to references" href="#cit129"><sup><span class="sup_ref">129</span></sup></a> The group by Jensen also aimed for a generic model to predict reaction context, here catalysts, solvents, reagents, and temperature.<a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> They trained an ANN on a dataset of about 10 million single-step and single-product reactions and allowed for hierarchical ANN structures which take interdependencies between the reaction contexts into account. In 69.6% of the time, a close match to the recorded conditions is found within the top 10 prediction outcomes.<a title="Select to navigate to references" href="#cit128"><sup><span class="sup_ref">128</span></sup></a> Also, the optimal selection of reaction conditions can be solved as an inverse design problem by using an optimisation algorithm to change the inputs of an reaction outcome model.</span>
        </div>
        <div>
          <span id="sect706"/><span class="c_heading_indent">Inference of reaction structures. </span>
          <span>To evaluate reaction synthetic routes, masses of products, reactants, and side-products need to be quantified. These can only be computed when all substances and their stoichiometric coefficients are known. However, three main aspects of current data recording hinder this analysis: (i) stoichiometric coefficients are lacking, (ii) reaction co-participants are missing, and (iii) multiple reaction steps are integrated into a single reaction entry.</span>
          <p class="otherpara">In the current literature, there exists a limited number of methods for reaction structure completion. Firstly, reaction templates can be utilised. Grzybowski <span class="italic">et al.</span> manually curated around 100<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 reaction rules with complete understanding of reaction participants and stoichiometry.<a title="Select to navigate to reference" href="#cit24"><sup><span class="sup_ref">24,25</span></sup></a> Their templates are now linked with the commercial software SYNTHIA<a title="Select to navigate to footnote" href="#fn13">†††</a> and can guide retrosynthesis and analyse carbon efficiency based on mass conservation.<a title="Select to navigate to references" href="#cit130"><sup><span class="sup_ref">130</span></sup></a> However, manual curation of reaction rules is far away from exploring the entire chemical space. Secondly, atom mapping, which relies on the rearrangement of atoms in chemical transformations, is promising to tackle this problem. The completion of an exemplary reaction is shown in <a title="Select to navigate to figure" href="#imgfig7">Fig. 7</a>, where in (a) stoichiometric coefficients are absent and mass balances are not obeyed and in (b) atom mapping describes the exact transformation and reveals the missing species on the product side. The existing atom mapping methods, described in recent reviews,<a title="Select to navigate to reference" href="#cit131"><sup><span class="sup_ref">131,132</span></sup></a> often convert molecules into graphs and compare the most common subgraphs. However, this results in an NP-hard problem where computational time increases exponentially with the number of atoms in molecules. Jaworski <span class="italic">et al.</span> utilise graph-theoretical considerations and chose 20 chemical rules/heuristics to correct mapping of reactions.<a title="Select to navigate to references" href="#cit133"><sup><span class="sup_ref">133</span></sup></a> This method attempts to complete the stoichiometry, firstly, by adding small molecules such as acetaldehyde, ammonia, and others to balance the reactions and, secondly, by fitting reactions into popular reaction templates and adding the missing parts. Only if such attempts fail, atom mapping is employed. The work by Schwaller <span class="italic">et al.</span> utilised NLP to infer reaction structures.<a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> A neural network (transformer) was trained on a set of mapped reactions and showed to be able to complete the mapping task quicker and with confidence scores.<a title="Select to navigate to references" href="#cit134"><sup><span class="sup_ref">134</span></sup></a> Despite the aforementioned improvements in atom mapping, inferring complete reaction structure remains a challenge at present as a precise prediction of functional transformations is required.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f7.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">The principle of atom mapping and how it can aid in the completion of reaction structures is outlined. The sample reaction is retrieved from Reaxys with Reaxys reaction ID: 615722. In (a) participants and stoichiometric coefficients are missing and in (b) the reaction is balanced with help species HCl and stoichiometry coefficients. The recording structure from C:1 to Cl:19 denotes the type of atom and its identifier.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Notably, computational capacities have increased immensely over the last few decades, and new computational approaches, such as graphics processing units (GPU) or quantum computing, are promising avenues for complex computational tasks. GPU computing has enormously advanced the field of deep learning in the areas such as computer vision and speech recognition.<a title="Select to navigate to references" href="#cit135"><sup><span class="sup_ref">135</span></sup></a> Quantum computing has shown to speed different search algorithms<a title="Select to navigate to references" href="#cit136"><sup><span class="sup_ref">136</span></sup></a> and its potential for complex optimisation tasks, such as energy system optimisation, has recently been highlighted.<a title="Select to navigate to references" href="#cit137"><sup><span class="sup_ref">137</span></sup></a> However, it has been argued that the power of quantum computing is limited, especially when it comes to NP-hard problems.<a title="Select to navigate to references" href="#cit138"><sup><span class="sup_ref">138</span></sup></a> Nevertheless, new computing approaches provide promising avenues for computationally expensive algorithms such as large scale atom mapping challenges.</p>
          <p class="otherpara">Besides said data-driven techniques to infer missing species and stoichiometry, it is worthwhile to discuss the automated generation of entire reaction networks based on chemical rules, which also leads to stoichiometric relationships of reaction networks.<a title="Select to navigate to reference" href="#cit139"><sup><span class="sup_ref">139,140</span></sup></a> For instance, the rule network input generator (RING) can construct complex networks based on a set of input reactants in their SMILES representation and a set of defined reaction rules and constraints.<a title="Select to navigate to reference" href="#cit139"><sup><span class="sup_ref">139,140</span></sup></a> RING showed to reproduce mechanisms reported in the literature for systems such as dehydration of fructose to produce HMF or acid-catalysed hydrolysis of HMF to levulinic acid.<a title="Select to navigate to references" href="#cit139"><sup><span class="sup_ref">139</span></sup></a> Most notably, Marvin <span class="italic">et al.</span> combine the network generation method RING with a mixed-integer linear programming (MILP) model for the optimisation of pathways towards biofuel-gasoline blends.<a title="Select to navigate to references" href="#cit141"><sup><span class="sup_ref">141</span></sup></a></p>
          <p class="otherpara">To generate reaction structures for large-scale networks, encoding all permitted reaction rules and constraints may be tiresome, as theoretically literature and databases can easily provide this information. However, rule-based network generation can be of particular interest in sparse regions of chemical knowledge. Here, methods such as RING can significantly contribute.</p>
          <p class="otherpara">With all types of prediction algorithms, it is worthwhile to keep the stochastic nature of the results in mind. While information from the literature often contains measurement uncertainties, introducing predictive algorithms adds model uncertainties. In Section 4 on decision-making, we shortly sketch the influence of uncertainties on finding optimal solutions.</p>
        </div>
      
    
    
      
      <h2 id="sect738"><span class="a_heading">3. Metrics</span></h2>
      <span>The assessment of sustainability through metric values is not trivial. As a system challenge, it calls for use of large data sets. At the same time, it is strongly biased by our subjective view of the dynamic concept of sustainability. Metrics to assess sustainability are diverse and difficult to set. Within the United Nations framework of the Sustainable Development Goals (SDGs), there exist 231 unique indicators to measure sustainability in the 17 dimensions of the SDGs. This illustrates the necessary diversity in dimensions and indicators utilised to assess sustainability. Additionally, strong interconnectivity between dimensions has been acknowledged by many,<a title="Select to navigate to reference" href="#cit142"><sup><span class="sup_ref">142–146</span></sup></a> possibly leading to synergetic effects, but also to trade-offs.<a title="Select to navigate to reference" href="#cit143"><sup><span class="sup_ref">143,144</span></sup></a></span>
      <p class="otherpara">There are limited possibilities to measure the sustainability of reaction routes to such extent as for instance the SDGs would indicate. Yet, the transition from green to sustainable chemistry has identified key aspects which should be considered. While the well-established principles of green chemistry<a title="Select to navigate to reference" href="#cit36"><sup><span class="sup_ref">36,38,147,148</span></sup></a> and green engineering<a title="Select to navigate to reference" href="#cit37"><sup><span class="sup_ref">37–39</span></sup></a> lead the way towards more environmental-friendly practices, sustainable chemistry requires the expansion of system boundaries.<a title="Select to navigate to reference" href="#cit149"><sup><span class="sup_ref">149–152</span></sup></a> On the reaction network level, this means considering entire supply chains as well as moving away from purely environmental concerns towards implicit inclusion of societal and economic ones, their trade-offs, and synergies. The European Technology Platform for Sustainable Chemistry, SusChem,<a title="Select to navigate to footnote" href="#fn14">‡‡‡</a> focuses on projects which combine all three sectors, and the International Sustainable Chemistry Collaborative Centre, ISC3,<a title="Select to navigate to footnote" href="#fn15">§§§</a> highlights <span class="italic">inter alia</span> systems thinking, ethical and social responsibility, and circularity as key characteristics for chemists to focus on.<a title="Select to navigate to references" href="#cit153"><sup><span class="sup_ref">153</span></sup></a></p>
      
        <h3 id="sect756"><span class="b_heading">Systems thinking</span></h3>
        <span>In the search for a more sustainable future, systems thinking and systems modelling are powerful tools. A system can be defined as a whole made out of interlinked, possibly nested, subsystems.<a title="Select to navigate to references" href="#cit154"><sup><span class="sup_ref">154</span></sup></a> Sustainability is often referred as a system which sustains itself, making the notion of systems thinking even more relevant for the discussion of sustainability assessment. In the following, we will visit the importance of system boundaries with regard to LCA and to circularity.</span>
        <div>
          <span id="sect760"/><span class="c_heading_indent">System boundaries and life cycle assessment. </span>
          <span>System boundaries describe the interfaces between the system and its exterior, the environment. The work by Nabavi <span class="italic">et al.</span> describes how system boundaries influence the assessment of sustainability aspects in dynamic systems.<a title="Select to navigate to references" href="#cit155"><sup><span class="sup_ref">155</span></sup></a> While system dynamic models should have a broad boundary including all variables considered important,<a title="Select to navigate to references" href="#cit156"><sup><span class="sup_ref">156</span></sup></a> it is essential to set boundaries somewhere for practical reasons.<a title="Select to navigate to references" href="#cit155"><sup><span class="sup_ref">155</span></sup></a> However, the implications of having set these exact boundaries will play an important role throughout the entire modelling process, in particular when dealing with complex sustainability considerations.</span>
          <p class="otherpara">Modelling chemical reactions from a systems perspective also strongly relies on the chosen boundaries. LCA is a common systems-based method to quantify environmental impacts on life cycle inventories.<a title="Select to navigate to reference" href="#cit41"><sup><span class="sup_ref">41–43</span></sup></a> In LCA, a functional unit, <span class="italic">e.g.</span> one kg of the desired product, is taken as reference, and boundaries are drawn, <span class="italic">e.g.</span> cradle–gate–grave, often as wide as possible to follow the materials and energy flows. Associated with these are the environmental burdens which can be summed up for impact categories, such as the cumulative energy demand (CED), the global warming potential (GWP), human toxicity, or land use. Social or socio-economic life cycle assessment (S-LCA) incorporates social indicators and describes impacts such as working hours and local employment.<a title="Select to navigate to reference" href="#cit157"><sup><span class="sup_ref">157,158</span></sup></a> LCA requires the specification of a system boundary on various levels, the ultimate one being the one between nature and the technical system.<a title="Select to navigate to references" href="#cit159"><sup><span class="sup_ref">159</span></sup></a> The scope of investigation requires further boundaries as the temporal and the geographic dimensions. Additionally, boundaries are chosen when deciding for metrics of interest (will impacts on aquatic life be within or outside the system boundary?). The tool named “strategic life cycle management” utilises sustainability principles as system boundaries and aims to provide an even wider overview.<a title="Select to navigate to references" href="#cit160"><sup><span class="sup_ref">160</span></sup></a></p>
          <p class="otherpara">A reaction network is a subsystem, which is in exchange with its environment over a system boundary, see <a title="Select to navigate to figure" href="#imgfig8">Fig. 8</a>. Decision-making requires the assignment of assessment aspects to flows that are in exchange with the environment, <span class="italic">e.g.</span> what are the monetary values of mass flows, or what is the availability or the demand of mass flows in the geographic context. The choice of system boundary therewith strongly frames the problem and assessment aspects at the system boundary, <span class="italic">e.g.</span> how useful, toxic, expensive streams crossing the boundary are, strongly impact the results of any study. As of now, it is difficult to associate these aspects to large quantities of molecules as necessary for large-scale network data. However, semantic web and knowledge graph technologies are envisioned for scientific and chemical data. They can lead the community towards a future where assessment may be easily associated with a large diversity of chemical species.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f8.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">Exchange of mass and energy at the system boundary from reaction networks (systems) and the economy, society and planetary boundaries. Mass and energy exchange with the environment can be assessed in multiple dimensions, <span class="italic">e.g.</span> the value of a mass flow leaving the system can be determined based on its monetary value, the demand for it or its environmental impact. Assessment criteria are influenced by the wider environment, <span class="italic">e.g.</span> energy markets and availability of renewable sources vary at different geographic locations and in time.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">At present, a chemical reaction network is commonly built based on: (i) one or more feedstock molecules of interest, and (ii) one or more product molecules of interest. Reactions connecting feedstock(s) and product(s) are then introduced manually from literature review or automatically from electronic databases or reaction generators, sometimes constrained by a maximum size of reaction steps. Some queries are open-ended on the product or the feedstock side, allowing queries such as, which is the best feedstock to produce product <span class="italic">X</span>, or which valuable products can be produced from feedstock <span class="italic">Y.</span> The system boundary can now be further specified, <span class="italic">e.g.</span> which species can be exchanged with the environment. Assessment metrics describe the exchange at the system boundary.</p>
        </div>
        <div>
          <span id="sect786"/><span class="c_heading_indent">System boundaries and circular economy. </span>
          <span>Nowadays, many industries strive for different levels of circularity of their supply chains.<a title="Select to navigate to reference" href="#cit161"><sup><span class="sup_ref">161,162</span></sup></a> By industrial symbiosis companies can exchange material flows, allowing by-products from one industrial process to become the feedstock of another and thereby closing material loops.<a title="Select to navigate to references" href="#cit163"><sup><span class="sup_ref">163</span></sup></a> In contrast to closing the loop through technology, biodegradable products close the biological loop.<a title="Select to navigate to references" href="#cit163"><sup><span class="sup_ref">163</span></sup></a></span>
          <p class="otherpara">Including circularity in system modelling requires careful evaluation of system boundaries. <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a> outlines that the aspects of circularity may lay within the system boundary, <span class="italic">e.g.</span> recycle streams within multi-step reactions, or may lay outside of current system boundary, <span class="italic">e.g.</span> similarly to BASF's Verbund system<a title="Select to navigate to footnote" href="#fn16">¶¶¶</a> companies or geographical regions can exchange material flows. While internal system circularity, see <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a> (left), influences the necessary exchange of flow quantities, external circularity, see <a title="Select to navigate to figure" href="#imgfig9">Fig. 9</a> (right), influences the assessment metrics. Utilising a material as system input, which is an output from another system can contribute to the overall reduction of waste and minimisation of raw material use, <span class="italic">e.g.</span> substituting for either fossil or renewable feedstocks. This should be taken into account when evaluation reaction pathways.</p>
          <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig9"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f9_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f9.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f9.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 9 </b> <span id="fig9"><span class="graphic_title">Circularity within one system (left) and between multiple systems (right). Circularity within one system affects the flow quantity exchanged at the system boundary, <span class="italic">e.g.</span> less solvent is needed as input and generated as output if solvent recovery takes place. Circularity between multiple systems should affect the assessment of exchanged flow quantity, <span class="italic">e.g.</span> output flows that can be used at input flows for other processes should be preferred over waste output flows.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
          <p class="otherpara">Molecular circularity indicators are required to inform on alternative utilisation possibilities. The Ellen McArthur Foundation has shaped the discussion on circularity indicators, introducing the material circularity indicator to aid assessing material flows both at product and at company level<a title="Select to navigate to references" href="#cit164"><sup><span class="sup_ref">164</span></sup></a> and its tool Circulytics,<a title="Select to navigate to footnote" href="#fn17">||||||</a> which measures circularity for businesses. Additionally, the World Business Council for Sustainable Development has introduced the circular transition indicators as quantitative framework to measure sustainability for businesses. With respect to material circularity in chemical engineering, Razza <span class="italic">et al.</span> provide metrics for biobased and biodegradable products, emphasising the biological cycle (extraction of renewable feedstock – use – composting or biodegradation in soil) in contrast to technological recycle steps.<a title="Select to navigate to references" href="#cit165"><sup><span class="sup_ref">165</span></sup></a> Most notable, Lokesh <span class="italic">et al.</span> have extended common green chemistry metrics towards capturing circularity aspects.<a title="Select to navigate to references" href="#cit166"><sup><span class="sup_ref">166</span></sup></a> We note the need of a waste stream database, which records large waste streams over various industries and maps functional molecules and pretreatment to the streams, allowing circularity assessment of chemical loops.</p>
          <p class="otherpara">The knowledge on the demand and supply (the usefulness) of material streams across the boundary will allow for the design of circular chemical solutions. For example, in view of a circular chemical supply chain, it is impossible to measure and compare the circularity of two competing reaction pathway options without considering their respective environments. One reaction pathway may seem less competitive due to a high amount of waste material generated throughout the processes. If, however, such material is in demand across the system boundary, the reaction pathway becomes increasingly competitive and the entire system more circular. Note that quantifying the circularity is not a molecular property and hardly a reaction property, but rather a system property, solvable only by data-based descriptions of the environment of a system.</p>
        </div>
      
      
        <h3 id="sect816"><span class="b_heading">Sustainability metrics for reaction networks from large databases</span></h3>
        <span>Ideally, a detailed analysis of social and environmental impacts in an (S-)LCA should be performed to evaluate reaction routes, yet, early-stage reaction data is at present not sufficient for such scopes of analysis. Sustainability assessment is ideally performed early on, as it allows for cheaper and faster implementations of process improvements<a title="Select to navigate to reference" href="#cit167"><sup><span class="sup_ref">167–169</span></sup></a> and for designing inherently sustainable pathways.<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> However, detailed process knowledge is often only available at the end of the process development pipeline.<a title="Select to navigate to reference" href="#cit170"><sup><span class="sup_ref">170,171</span></sup></a> Streamlining methods to estimate LCA impacts of molecules in early stages try to circumvent this problem.<a title="Select to navigate to reference" href="#cit169"><sup><span class="sup_ref">169,170,172–175</span></sup></a> Yet, only a few take differences in process conditions into account<a title="Select to navigate to reference" href="#cit167"><sup><span class="sup_ref">167,168,176</span></sup></a> and the possibility of completely different reaction routes within the reaction network are not discussed at present.</span>
        <p class="otherpara">The literature on reaction network optimisation has employed various metrics – much simpler than a full LCA – to assess the sustainability of reaction routes. The metrics utilised by Voll and Marquardt cover mass balances, energy, and cost criteria.<a title="Select to navigate to references" href="#cit30"><sup><span class="sup_ref">30</span></sup></a> Zhang <span class="italic">et al.</span> utilised the enthalpy of reaction as energy criteria, however conclude that it is not sensitive enough; they recommend the inclusion of separation processes for better performance.<a title="Select to navigate to references" href="#cit34"><sup><span class="sup_ref">34</span></sup></a> In later works, some considerations of environmental impacts were included through energy consumption, resource consumption, emission impact, and toxicity potential or the CED and the GWP in the objective function.<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31,32</span></sup></a> While the previous metrics of assessment were built on manually curated data, large-scale reaction network optimisation can only work with metrics obtainable for millions of molecules and reactions in an automated manner.</p>
        <div>
          <span id="sect829"/><span class="c_heading_indent">Mass-based evaluation. </span>
          <span>Most early metrics within the field of green chemistry are mass-based evaluations of the reactions. The most influential ones are the AE,<a title="Select to navigate to references" href="#cit177"><sup><span class="sup_ref">177</span></sup></a> the <span class="italic">E</span> factor,<a title="Select to navigate to references" href="#cit178"><sup><span class="sup_ref">178</span></sup></a> and the reaction mass efficiency (RME).<a title="Select to navigate to references" href="#cit38"><sup><span class="sup_ref">38</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn1"/><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-t1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-t1.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-t1.gif"/></a></td><td class="rightEqn">(1)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn2"/><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-t2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-t2.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-t2.gif"/></a></td><td class="rightEqn">(2)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn3"/><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-t3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-t3.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-t3.gif"/></a></td><td class="rightEqn">(3)</td></tr></table><a title="" href="#eqn1">Eqn (1)–(3)</a> require knowledge on the following factors: all participating species and their molecular weight must be known, the reaction stoichiometry is required, and differentiation between products and waste needs to be enabled. If circularity is a premise for sustainable processes, we will need to reassess the binary classification into waste and product for such metrics. <a title="" href="#eqn3">Eqn (3)</a> also requires information about product yield. More detailed mass-based metrics, <span class="italic">e.g.</span> the mass intensity (MI) or process mass intensity (PMI), include the use of solvents, catalysts, and other substances, leading to a more holistic assessment of the reactions.<a title="Select to navigate to references" href="#cit150"><sup><span class="sup_ref">150</span></sup></a><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn4"/><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-t4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-t4.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-t4.gif"/></a></td><td class="rightEqn">(4)</td></tr></table><table><td class="leftEqn"> </td><tr><td class="eqn"><span id="eqn5"/><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-t5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-t5.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-t5.gif"/></a></td><td class="rightEqn">(5)</td></tr></table>As discussed in the section on Inference of reaction structures, stoichiometry and participating species are often not known and require computationally expensive atom mapping for completion. The section on Inference of reaction outcomes explains that predictions of yield are at present impossible for generic sets of reaction. Furthermore, the differentiation between waste and product is complicated in the context of larger system boundaries and circular economy, where waste streams are seen as potential feedstocks. Improvements in information extraction and data inference can make much more data available in the future. Evolving towards a linked data structure will make evaluations of molecules at system boundaries much easier in reaction networks. <a title="" href="#eqn4">Eqn (4) and (5)</a> require additional information on the masses of all involved materials, which also necessitates advanced information extraction techniques for chemical entity recognition.</span>
          <p class="otherpara">Assuming reaction data is available, some automated tools can be used to determine mass-based metrics in reaction synthesis plans, <span class="italic">e.g.</span> the environmental assessment tool for organic synthesis (EATOS),<a title="Select to navigate to footnote" href="#fn18">****</a> the American Chemical Society PMI (prediction) calculator,<a title="Select to navigate to footnote" href="#fn19">††††</a> or the Andraos’ algorithm.<a title="Select to navigate to reference" href="#cit179"><sup><span class="sup_ref">179,180</span></sup></a> The EATOS and Andraos’ method were found most rigorous for material efficiency metrics<a title="Select to navigate to references" href="#cit180"><sup><span class="sup_ref">180</span></sup></a> and the Green star<a title="Select to navigate to reference" href="#cit181"><sup><span class="sup_ref">181,182</span></sup></a> and University of Toronto green chemistry initiative method<a title="Select to navigate to references" href="#cit183"><sup><span class="sup_ref">183</span></sup></a> were recommended for environmental and hazard impacts in introductory analysis.<a title="Select to navigate to references" href="#cit184"><sup><span class="sup_ref">184</span></sup></a> Key challenges for any of the given algorithms at present are: firstly, approximations if data is missing, <span class="italic">e.g.</span> general scaling factors for the required masses of organic solvents and aqueous washes,<a title="Select to navigate to references" href="#cit180"><sup><span class="sup_ref">180</span></sup></a> secondly, the comparability to biotransformation synthesis,<a title="Select to navigate to references" href="#cit180"><sup><span class="sup_ref">180</span></sup></a> thirdly, the evaluation of only linear synthesis trees or synthesis networks,<a title="Select to navigate to reference" href="#cit185"><sup><span class="sup_ref">185,186</span></sup></a> and last but not least, the integration of recycles for solvents and catalysts.<a title="Select to navigate to references" href="#cit179"><sup><span class="sup_ref">179</span></sup></a> Simplified algorithms for linear and tree cases were introduced<a title="Select to navigate to reference" href="#cit179"><sup><span class="sup_ref">179,186</span></sup></a> and applied in a reaction network.<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a></p>
        </div>
        <div>
          <span id="sect876"/><span class="c_heading_indent">Exergy-based evaluation. </span>
          <span>Exergy is the maximum amount of work, which can be extracted from a system when the system is brought to thermodynamic equilibrium with components of the natural environment through reversible processes.<a title="Select to navigate to references" href="#cit187"><sup><span class="sup_ref">187</span></sup></a> It is a measure of energy quality as it quantifies the ability of a form of energy to do physical work. Exergy destruction is proportional to the entropy generated due to irreversible processes.<a title="Select to navigate to references" href="#cit188"><sup><span class="sup_ref">188</span></sup></a> Thus, exergy destruction is a measure of degradation of both energy and material in a system.<a title="Select to navigate to references" href="#cit189"><sup><span class="sup_ref">189</span></sup></a></span>
          <p class="otherpara">Exergetic analysis has been linked to both, the environmental and the economic aspects of sustainability. From the environmental perspective, the concept of exergy has been positively highlighted as it takes the natural environment into account as a reference state.<a title="Select to navigate to references" href="#cit190"><sup><span class="sup_ref">190</span></sup></a> Ao <span class="italic">et al.</span> however stress that before widely accepting exergy as an environmental impact indicator, more work needs to be done.<a title="Select to navigate to references" href="#cit191"><sup><span class="sup_ref">191</span></sup></a> From the economical perspective, it has been noted that exergy can be strongly linked to costs through exergoeconomics.<a title="Select to navigate to references" href="#cit192"><sup><span class="sup_ref">192</span></sup></a> Labour and capital costs for processes can be included in exergy evaluation<a title="Select to navigate to references" href="#cit193"><sup><span class="sup_ref">193</span></sup></a> and it may be the most useful function for solving cost-optimisation problems.<a title="Select to navigate to references" href="#cit194"><sup><span class="sup_ref">194</span></sup></a> For more information on exergy as a process and/or sustainability indicator, we refer the reader to the reviews by Dewulf <span class="italic">et al.</span> and Romero and Linares.<a title="Select to navigate to reference" href="#cit190"><sup><span class="sup_ref">190,193</span></sup></a></p>
          <p class="otherpara">In the context of reaction network optimisation, in our previous work, we utilised an exergy assessment for ranking reaction routes.<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a> To describe a reaction we included both the physical and chemical exergy of participating species and evaluated further the exergy requirements for process heating and separation.<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a> Exergetic analysis was applied to rank 15 reaction route options after <span class="italic">a priori</span> removal of reactions with insufficient data.</p>
          <p class="otherpara">Exergy-based analysis at large scales requires automated retrieval of thermodynamic data. Physical exergies can be computed based on specific heat capacities retrieved from the software COSMOtherm RS, while the computation of chemical exergies pose a larger challenge. Approaches utilising linear regression models for specific types of molecules, <span class="italic">e.g.</span> solid or liquid fuels,<a title="Select to navigate to reference" href="#cit195"><sup><span class="sup_ref">195–198</span></sup></a> more advanced ML models,<a title="Select to navigate to reference" href="#cit199"><sup><span class="sup_ref">199–202</span></sup></a> and group contribution techniques<a title="Select to navigate to references" href="#cit203"><sup><span class="sup_ref">203</span></sup></a> have been proposed. Promising for large-scale data, an atomic contribution model was shown to provide a generic framework to provide simple, yet relatively accurate estimations of the standard molar chemical exergies.<a title="Select to navigate to references" href="#cit204"><sup><span class="sup_ref">204</span></sup></a> An alternative is a prediction of Gibb's free energy of formation for compounds, <span class="italic">e.g.</span> through the Joback method as in ref. <a title="Select to navigate to references" href="#cit26">26</a>, from which the chemical exergy can be calculated based on the tabulated exergies of elements.<a title="Select to navigate to reference" href="#cit187"><sup><span class="sup_ref">187,205</span></sup></a> In the future, we expect graph convolutional neural networks to predict necessary properties to a high accuracy.<a title="Select to navigate to reference" href="#cit206"><sup><span class="sup_ref">206,207</span></sup></a></p>
        </div>
        <div>
          <span id="sect905"/><span class="c_heading_indent">Early-stage assessment. </span>
          <span>To a certain extent, simple chemical rules can substitute the computation of data-intensive metrics at present. Especially for large scale datasets, manual data curation from simulations and/or experiments is not an option. In our previous work, we have hence introduced a few simple chemical heuristics, which can be utilised to provide a rough filtration of reaction routes.<a title="Select to navigate to references" href="#cit26"><sup><span class="sup_ref">26</span></sup></a> For instance, datasets may be screened for reactions that have a minimum number of records, making them more reliable, or which report a yield value above a certain threshold, making them more efficient. Further heuristics utilise the chemical structure of the materials and may be applied for example to prevent aromatics or certain heteroatoms. We outline an extended list of example heuristics in <a title="Select to navigate to table" href="#tab3">Table 3</a>. While efficiency potentials may contribute to the environmental and economic dimensions, toxicity potentials shed light on social and environmental issues, and the reliability of the data can bring advantages in social and economic perspectives through faster and safer process development. Note that one heuristic can also cover multiple potentials.</span>
          
<div class="table_caption"><b>Table 3</b> <span id="tab3">Outline of possible heuristics for large-scale screening. Note that all heuristics are independent from stoichiometry and need to be adjusted based on the problem formulation. The list is by no means complete and the functions and potentials listed are exemplary</span>
		</div>

              <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/><col/></colgroup>
                
                
                
                <thead align="left">
                  <tr align="left" valign="bottom">
                    <th align="left" valign="bottom" class="border_black">Heuristic</th>
                    <th align="left" valign="bottom" class="border_black">Function</th>
                    <th align="left" valign="bottom" class="border_black">Potential</th>
                  </tr>
                </thead>
                <tbody align="left">
                  <tr align="left">
                    <td valign="top">Carbon counts</td>
                    <td valign="top">Remove reactions with large variation of carbon counts</td>
                    <td valign="top">Efficiency</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Catalysts</td>
                    <td valign="top">Remove reactions using undesired catalysts</td>
                    <td valign="top">Efficiency, toxicity</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Fragments</td>
                    <td valign="top">Preserve/prevent selected fragments throughout the route</td>
                    <td valign="top">Efficiency, toxicity</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Number of records</td>
                    <td valign="top">Remove reactions with few records</td>
                    <td valign="top">Reliability</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Publication year</td>
                    <td valign="top">Remove old reactions</td>
                    <td valign="top">Reliability</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Reaction type</td>
                    <td valign="top">Remove all reactions which are not of desired reaction type</td>
                    <td valign="top">Reliability</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Reagents</td>
                    <td valign="top">Remove reactions using undesired reagents</td>
                    <td valign="top">Toxicity</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Similarity</td>
                    <td valign="top">Guarantee smooth structural transition along a reaction route</td>
                    <td valign="top">Efficiency, reliability</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Solvents</td>
                    <td valign="top">Remove reactions using undesired solvents</td>
                    <td valign="top">Efficiency, toxicity</td>
                  </tr>
                  <tr align="left">
                    <td valign="top">Yield</td>
                    <td valign="top">Remove reactions with yields lower than threshold</td>
                    <td valign="top">Efficiency</td>
                  </tr>
                </tbody>
              </table></div></div>
            <hr class="hrule"/><br/>
        </div>
      
    
    
      
      <h2 id="sect963"><span class="a_heading">4. Decision making</span></h2>
      <span>Optimisation algorithms have proven to be a reliable tool for optimal decision-making in complex problems, in particular, in complex network structures. Within reaction network optimisation, decisions on the sequence of reactions from feedstock molecules to target species are required. There commonly exists a variety of reaction sequence possibilities to connect different molecules, see <a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>, and appropriate algorithms can make decisions based on metrics discussed in the previous section. Strategies to solve the optimisation formalism depend on the underlying network structure of the problem. The characteristic for the problem of reaction network optimisation however is the number of products and reactants which connect to one reaction, <span class="italic">cf.</span><a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>, and which can lead to complex and cyclic network structures.</span>
      <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig10"><a href="/image/article/2021/CS/d1cs00477h/d1cs00477h-f10_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1cs00477h-f10.tif" src="/image/article/2021/CS/d1cs00477h/d1cs00477h-f10.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 10 </b> <span id="fig10"><span class="graphic_title">Illustration of reaction routes in chemical reaction networks. The bipartite reaction network represents reactions as bar nodes (r1 to r4) and molecules as circular nodes. Decision-making is required to decide between two alternatives reaction sequences (r1 and r3 <span class="italic">vs.</span> r2 and r4), which connect the same feedstock molecule to the same target molecule. However, different co-reactants are required and different co-products/waste are generated. The directed and weighted molecule network and reaction network are projections of the bipartite network and can function as simplification for shortest path search algorithms. The quantities on the edges illustrate possible weighting schemes.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      <p class="otherpara">Decision-making in network structures has been broadly explored in many different fields. Examples are vehicle routing or navigation systems,<a title="Select to navigate to reference" href="#cit208"><sup><span class="sup_ref">208,209</span></sup></a> in tree searches (chemical application of retrosynthesis),<a title="Select to navigate to reference" href="#cit210"><sup><span class="sup_ref">210–212</span></sup></a> and in supply chain optimisation and task scheduling (chemical application of batch plants in process industries).<a title="Select to navigate to reference" href="#cit213"><sup><span class="sup_ref">213–215</span></sup></a> In the following, we will shortly review a selection of related fields and highlight their similarities and differences to reaction network optimisation. We will then visit recent literature on reaction network optimisation, which takes the fully connected structure of chemical reaction networks into account. In <a title="Select to navigate to table" href="#tab4">Table 4</a> we explain domain-specific terminology.</p>
      
<div class="table_caption"><b>Table 4</b> <span id="tab4">Explanation of specific terminology in decision-making</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col/><col/></colgroup>
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Term</th>
                <th align="left" valign="bottom" class="border_black">Description</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td valign="top">Big O notation</td>
                <td valign="top">The big O notation refers to the time or memory needed to run an algorithm. It is a theoretical measure of the asymptotic behaviour of an algorithm.</td>
              </tr>
              <tr align="left">
                <td valign="top">Constraints</td>
                <td valign="top">Constraints determine the feasible space in which variables can lay. They impose limitations, <span class="italic">e.g.</span> that material flows cannot be less than zero.</td>
              </tr>
              <tr align="left">
                <td valign="top">Deterministic optimisation</td>
                <td valign="top">Deterministic optimisation means all algorithms based on a rigorous mathematical approach, which will lead to the same solution space when run multiple times with the same system parameters.</td>
              </tr>
              <tr align="left">
                <td valign="top">Heuristic function</td>
                <td valign="top">A heuristic function approximates certain parts of a problem in order to solve a problem more quickly. Precision is traded for speed.</td>
              </tr>
              <tr align="left">
                <td valign="top">Linear programming</td>
                <td valign="top">Linear programming (LP) problems consist out of a linear objective function and linear constraints.</td>
              </tr>
              <tr align="left">
                <td valign="top">Mixed-integer programming</td>
                <td valign="top">In mixed-integer programming discrete variables are added to the continuous variables used within the objective function and the constraints.</td>
              </tr>
              <tr align="left">
                <td valign="top">Objective function</td>
                <td valign="top">The objective function describes the value to be optimised. It is a real-value function, by general convention to be minimised over alternative system variables.</td>
              </tr>
              <tr align="left">
                <td valign="top">Relaxation</td>
                <td valign="top">A relaxation is an underestimation of a more complicated system to a simpler system. In optimisation, relaxations can transform hard problems into approximated, yet solvable ones.</td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      
        <h3 id="sect1014"><span class="b_heading">Decision-making in network structures</span></h3>
        <span>Many decision-making problems can be represented by different network structures where fluxes or connections are optimised. The bipartite reaction network may be approximated by graph projections to reactions or molecules, see <a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a>, which in turn allow for certain search strategies.</span>
        <p class="otherpara">Navigation systems, such as Google Maps, explore the shortest path between two endpoints in a weighted network, where the weighting is the distance or the time required to travel between the points. The algorithm behind navigation systems is often based on the Dijkstra algorithm invented by Edsger Dijkstra in 1959. The algorithm works on a weighted graph, visits each node of the graph, and updates a table on the shortest distances to all others from a selected starting node. Its time complexity is <span class="italic">O</span>(|<span class="italic">E</span>| + |<span class="italic">V</span>|log|<span class="italic">V</span>|) where <span class="italic">E</span> is the number of edges and <span class="italic">V</span> is the number of vertices.<a title="Select to navigate to references" href="#cit208"><sup><span class="sup_ref">208</span></sup></a> An extension of the Dijkstra algorithm is the A* (A-star) algorithm, which changes the way the algorithm selects the next node to visit. While in the Dijkstra algorithm, this has been done based on the cost between the start node and the next node, the A* algorithm adds a heuristic function to this process, which estimates the cost from the node to be chosen to the target node.<a title="Select to navigate to references" href="#cit209"><sup><span class="sup_ref">209</span></sup></a></p>
        <p class="otherpara">While these routing algorithms bring about benefit in implementation and scale-up, they are not directly applicable to reaction networks. <a title="Select to navigate to figure" href="#imgfig10">Fig. 10</a> illustrates simplifications of an illustrative reaction network to a directed and weighted network either focusing on molecules or on reactions. The shortest pathway search may be able to regard parts of sustainability considerations in the weights, however, lacks the systems perspective, where co-products/waste and the source of all co-reactants are regarded as inherently connected to the chosen route. Furthermore, the meaning of weights in reaction networks depends on the case study, as they could be the emissions produced, the costs generated, chemical similarity, or any valid combination of our understanding of sustainability.</p>
        <p class="otherpara">Tree-based network searches are utilised in the field of automated retrosynthesis planning. The retrosynthetic analysis describes the task of transforming the structure of a synthetic target molecule into known and simpler starting molecules by constructing a sequence of molecule deconstructions.<a title="Select to navigate to references" href="#cit216"><sup><span class="sup_ref">216</span></sup></a> Traditionally, iterative cycles of logical analysis and perception were applied by chemists to the target compound and the available data space.<a title="Select to navigate to references" href="#cit217"><sup><span class="sup_ref">217</span></sup></a> In automated, or computer-aided, retrosynthesis, an algorithm proposes the most suitable synthesis route. The search space resembles a tree with molecules as nodes and reactions as edges, is intractable large, and the decision-making task is the identification of the most suitable branches within the tree. Solving retrosynthetic trees has been largely inspired by tree problems in games such as Chess or Go, however, retrosynthetic trees are considerably different as they are usually shallower (∼10–20 steps) but the branching factor is higher (around 200 options at each node).<a title="Select to navigate to references" href="#cit210"><sup><span class="sup_ref">210</span></sup></a></p>
        <p class="otherpara">ML-based techniques have significantly impacted tree-based retrosynthetic analysis.<a title="Select to navigate to reference" href="#cit218"><sup><span class="sup_ref">218–220</span></sup></a> Segler <span class="italic">et al.</span> demonstrated a Monte Carlo Tree Search (MCTS) with three deep neural networks; two ANNs for reaction rule extraction and one as a reinforcement framework.<a title="Select to navigate to references" href="#cit210"><sup><span class="sup_ref">210</span></sup></a> Kishimoto <span class="italic">et al.</span> investigate two common search techniques within the domain; MCTS and depth-first proof-number (DFPN) search. They find that the enhanced MCTS by Segler <span class="italic">et al.</span> outperforms common DFPN, however propose a new DFPN with heuristic rules for edge initialisation, which outperforms Segler's algorithm regarding time complexity and delivers equivalent success rates.<a title="Select to navigate to references" href="#cit211"><sup><span class="sup_ref">211</span></sup></a> Coley <span class="italic">et al.</span> utilise molecular similarity to inform on edge choices<a title="Select to navigate to references" href="#cit221"><sup><span class="sup_ref">221</span></sup></a> and demonstrate how a learned synthetic complexity metric can assist to scan the exponentially increasing search space.<a title="Select to navigate to references" href="#cit212"><sup><span class="sup_ref">212</span></sup></a> After further development, Segler <span class="italic">et al.</span> present their algorithm based on ANNs and symbolic artificial intelligence, which showed to produce routes that chemists found on average equivalent to the literature reported routes.<a title="Select to navigate to references" href="#cit222"><sup><span class="sup_ref">222</span></sup></a> Their algorithm has recently been commercialised by Elsevier and Pending.AI as Reaxys Predictive Retrosynthesis.</p>
        <p class="otherpara">Despite differences in network structure, quick searching strategies from the field of computer-assisted retrosynthesis will become immensely valuable when data and metric hurdles are overcome. For sustainability consideration, the task in reaction networks is truly the optimisation of the entire system, including co-products and co-reactants, rather than one synthetic pathway. While retrosynthetic analysis aims towards any known, simple, and cheap starting molecules, starting molecules fulfilling sustainability considerations largely constrain the search space for sustainable pathway identification. Thus, the network topology resembles two branching trees, which meet in the middle, see the description of the “forward-backward” network built by ref. <a title="Select to navigate to references" href="#cit223">223</a> and the overall aim is to optimise the entire system. Nevertheless, techniques from the field of ML-based retrosynthesis will inspire the development of new methods to handle large reaction networks. To take advantage of the full potential of ML-based techniques for decision-making in the chemical domain, the need for explainable artificial intelligence has been emphasised in a recent review article on drug discovery.<a title="Select to navigate to references" href="#cit224"><sup><span class="sup_ref">224</span></sup></a> They identify the current lack of an open-community platform but highlight the potential of explainable artificial intelligence for the discovery of novel bioactive compounds.<a title="Select to navigate to references" href="#cit224"><sup><span class="sup_ref">224</span></sup></a> Similarly, for computational tools to identify novel reaction pathways, we would expect a faster uptake within the community if solution strategies are comprehensible by chemists and chemical engineers.</p>
        <p class="otherpara">Other network systems, from which algorithms can be explored, are batch/job scheduling problems. Here multiple inputs and multiple outputs are taken into account per batch and the sequential manner of performed reactions is regarded. In process industries, consumer products are produced by sequential processing of chemical and physical tasks (in our case a chemical reaction, but generally any kind of task). Tasks require different process units and different storage facilities for in and outputs, which constrain the solution space.<a title="Select to navigate to references" href="#cit213"><sup><span class="sup_ref">213</span></sup></a> In single machine batching and scheduling problems, a set of jobs need to be processed by one machine, where jobs of similar type can be processed together and jobs from different families separately.<a title="Select to navigate to references" href="#cit215"><sup><span class="sup_ref">215</span></sup></a> The capacity of the machine as well as processing time and heating or cooling requirements strongly constrain the feasible region.<a title="Select to navigate to references" href="#cit215"><sup><span class="sup_ref">215</span></sup></a> If working with multiple pieces of equipment, problems are further constrained by sequential requirements, <span class="italic">e.g.</span> some pieces of equipment are always used before others, and by equipment interference problems, <span class="italic">e.g.</span> certain tasks cannot be performed simultaneously.<a title="Select to navigate to references" href="#cit214"><sup><span class="sup_ref">214</span></sup></a></p>
        <p class="otherpara">While reaction networks and batch scheduling jobs exhibit similarities in their network structure, it is worthwhile to note their divergence in problem specifics, such as interference constraints, a large variety of different types of tasks, and task-specific constraints such as cooling/heating. However, algorithms from the mature field of batch/task scheduling will come in beneficial when automated large-scale reaction network optimisation develops from the conceptional early-stage design towards different implementation levels, considering supply chains and production planning. Very promising concepts for this are integrated decision-making strategies.<a title="Select to navigate to references" href="#cit225"><sup><span class="sup_ref">225</span></sup></a></p>
      
      
        <h3 id="sect1058"><span class="b_heading">Pathway optimisation in integrated biorefineries</span></h3>
        <span>Identifying the most promising pathway alternatives for the production of chemicals from renewable feedstock has been the focus of superstructure optimisation for integrated biorefineries. A superstructure describes a network of technologies, in particular, a process diagram with all hypothetically useful units and connections.<a title="Select to navigate to reference" href="#cit226"><sup><span class="sup_ref">226,227</span></sup></a> The advantage of optimising the superstructure <span class="italic">e.g.</span> of processes and streams in a biorefinery is that complex interactions between different design choices are considered. However, a rich structure is necessary requiring much data and often leading to large-scale, non-convex, mixed-integer, nonlinear programming models.<a title="Select to navigate to references" href="#cit226"><sup><span class="sup_ref">226</span></sup></a></span>
        <p class="otherpara">Superstructure problems can be formulated by distinct programming models (<span class="italic">i.e.</span> disjunctive programming).<a title="Select to navigate to references" href="#cit228"><sup><span class="sup_ref">228</span></sup></a> One approach is a formulation as mixed-integer-nonlinear programming (MINLP) problem.<a title="Select to navigate to reference" href="#cit226"><sup><span class="sup_ref">226,227</span></sup></a> Giuliano <span class="italic">et al.</span> optimise a superstructure for levulinic acid, succinic acid, and ethanol product from lignocellulosic biomass.<a title="Select to navigate to references" href="#cit227"><sup><span class="sup_ref">227</span></sup></a> In their approach, rigorous process models account for significant nonlinearities leading to the MINLP formulation. Their problem is linearised to a MILP problem through variable discretisation methods. Kong <span class="italic">et al.</span> optimise a superstructure including heat integration and utility plant design by an MINLP problem to which they propose a set of solution methods to speed up the computation.<a title="Select to navigate to references" href="#cit226"><sup><span class="sup_ref">226</span></sup></a> Nonlinearities are introduced by processing unit models, where outlet material flows and outlet temperature are nonlinear functions as well as heat and electricity requirements. Alternatively, Garcia and You describe their product and process network by an NLP.<a title="Select to navigate to references" href="#cit229"><sup><span class="sup_ref">229</span></sup></a> Nonconvex terms are caused through economic considerations such as capital expenditures. They utilise a piecewise linear approximation, leading to an easier solvable MILP problem.<a title="Select to navigate to references" href="#cit229"><sup><span class="sup_ref">229</span></sup></a> Some works formulate the interdependencies through linear models.<a title="Select to navigate to reference" href="#cit223"><sup><span class="sup_ref">223,230</span></sup></a></p>
        <p class="otherpara">Additionally, most works handle contradicting objective functions through a multi-objective framework. Andiappan <span class="italic">et al.</span> formulate a multi-objective optimisation of the superstructure for an integrated biorefinery, addressing possible trade-offs between economic and environmental objectives through two approaches. A bi-level formulation maximises the gross profit on the upper level, subject to the minimisation of the environmental burden and the reaction heat on the lower level. Alternatively, fuzzy optimisation is extended by introducing upper and lower bounds for the factor lambda accounting for the satisfaction of all three objectives.<a title="Select to navigate to references" href="#cit230"><sup><span class="sup_ref">230</span></sup></a> Garcia and You utilise the epsilon constraint method to allow for multiple objectives.<a title="Select to navigate to references" href="#cit229"><sup><span class="sup_ref">229</span></sup></a></p>
      
      
        <h3 id="sect1079"><span class="b_heading">Early-stage pathways optimisation in reaction networks</span></h3>
        <span>In contrast to rich superstructures with rigorous unit operation models, technologies, and utility integration, stand early-stage evaluation methods. Most promising reaction pathways are estimated at an early-stage without rigorous process models of different technologies. One example of such an early stage approach is Bao <span class="italic">et al.</span>'s short-cut method for the preliminary synthesis of process technology pathways.<a title="Select to navigate to references" href="#cit231"><sup><span class="sup_ref">231</span></sup></a> They propose a chemical species/conversion operator diagram which they optimise through an NLP model. Nonlinearities are introduced through entering and leaving species flowrates in conversion operators and through annualised costs of conversion. Instead of rigorous models, they assess various conversion technologies through characteristics such as yield and cost.<a title="Select to navigate to references" href="#cit231"><sup><span class="sup_ref">231</span></sup></a> Further early-stage methods will be discussed in the following three sections.</span>
        <div>
          <span id="sect1085"/><span class="c_heading_indent">Reaction network flux analysis. </span>
          <span>Optimal reaction pathways for the conversion of renewable feedstocks are often examined by the approximate method RNFA.<a title="Select to navigate to reference" href="#cit30"><sup><span class="sup_ref">30,232</span></sup></a> The RNFA is inspired by earlier works on metabolic networks<a title="Select to navigate to reference" href="#cit29"><sup><span class="sup_ref">29,233</span></sup></a> and models mass flows and reactions through linear balance equations for all components. Hereby, sink and source terms represent supply and demand. To model the reactions, all participating species and the stoichiometry of all reactions need to be known. The RNFA does not account for mixing and separation.</span>
          <p class="otherpara">While the core of the problem formulation lays in an LP formulation for mass balances that can be efficiently solved in polynomial time (<span class="italic">e.g.</span>, using state-of-the-art solvers like CPLEX),<a title="Select to navigate to reference" href="#cit234"><sup><span class="sup_ref">234,235</span></sup></a> integers have been introduced to account for the activity of fluxes, resulting in a MILP problem.<a title="Select to navigate to references" href="#cit232"><sup><span class="sup_ref">232</span></sup></a> Also, alternative optima were identified through the integer constraints,<a title="Select to navigate to reference" href="#cit30"><sup><span class="sup_ref">30,232,236</span></sup></a> while at present CPLEX can already account for alternative solutions in LPs without manual extension to MILP formulations. In the work of Besler <span class="italic">et al.</span> knowledge on active fluxes has also been used to describe non-flux-related costs for reaction pathways, <span class="italic">e.g.</span> toxicity. In the work by Dahmen and Marquardt the RNFA is combined with a model for computer-aided molecular design for mixtures, which resulted in an NLP model. Nonlinearities were introduced through the consideration of mixtures, which requires the solver to recompute properties of compositions at each step.<a title="Select to navigate to references" href="#cit237"><sup><span class="sup_ref">237</span></sup></a></p>
          <p class="otherpara">Notably, the RNFA can lead to degenerated solutions when components are consumed and generated in cycles (<span class="italic">e.g.</span>, equilibrium reactions or protecting groups). Similarly, huge recycle streams can occur as separation is simplified. The RNFA has been successfully applied to identify optimal reaction pathways for biofuel and biopolymer synthesis.<a title="Select to navigate to reference" href="#cit31"><sup><span class="sup_ref">31,34</span></sup></a></p>
        </div>
        <div>
          <span id="sect1101"/><span class="c_heading_indent">Process network flux analysis. </span>
          <span>An extension of the RNFA is the process network flux analysis (PNFA)<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a> where pseudo-components and -reactions are introduced to resemble mixing and separation fluxes. For this, all possible mixtures and potential separation tasks are identified a priori, modelled through short-cut methods, and included as pseudo-components and -reactions. The PNFA resembles the superstructure optimisation problems as it aims to include more detailed process knowledge. Operating cost or energy demand of separations are considered through pre-computed energy demands. Besides, binary variables are introduced for all equipment using big-M formulations that are active when the respective flux is greater than zero, allowing the estimation of the number of process units. The investment costs are considered through binary variables and nonlinear cost correlations. Multiple objectives are taken into consideration by the epsilon constraint method. The overall problem results in an MINLP problem that can be solved using deterministic global solvers like BARON<a title="Select to navigate to reference" href="#cit238"><sup><span class="sup_ref">238,239</span></sup></a> or MAiNGO.<a title="Select to navigate to references" href="#cit240"><sup><span class="sup_ref">240</span></sup></a> However, solving nonlinear programs is often NP-hard and thus limited to small problem instances. The PNFA, formulated using GAMS<a title="Select to navigate to references" href="#cit241"><sup><span class="sup_ref">241</span></sup></a> and solved by BARON, has been successfully used for biofuels production<a title="Select to navigate to references" href="#cit33"><sup><span class="sup_ref">33</span></sup></a> also including the biomass supply chain<a title="Select to navigate to references" href="#cit32"><sup><span class="sup_ref">32</span></sup></a> and for pathway considerations for biofuel product design.<a title="Select to navigate to references" href="#cit242"><sup><span class="sup_ref">242</span></sup></a></span>
        </div>
        <div>
          <span id="sect1111"/><span class="c_heading_indent">Petri net optimisation. </span>
          <span>An alternative modelling approach for the optimisation of pathways in reaction networks is a Petri net. A Petri net explicitly takes the reaction sequence into account which can be an important factor during optimisation. Petri nets were first introduced by Carl Adam Petri<a title="Select to navigate to references" href="#cit243"><sup><span class="sup_ref">243</span></sup></a> and are a type of directed bipartite network. In bipartite networks, two node types exist and links can only connect nodes of different types. In Petri nets, the node types are places (resembling molecules) and transitions (resembling reactions). Their input and output relations are shown by links, called arcs and an incidence matrix, which records the stoichiometry. A flow between places <span class="italic">via</span> transitions is given by a marking of places with tokens. Such a marking describes a state of a Petri net. Tokens change from one place to another through the firing of transitions, leading to a change in state.<a title="Select to navigate to reference" href="#cit244"><sup><span class="sup_ref">244,245</span></sup></a></span>
          <p class="otherpara">The Petri net optimisation (PNO) problem determines an optimal sequence of firing certain transitions and a formulation for reaction route optimisation was presented by ref. <a title="Select to navigate to references" href="#cit35">35</a> after an extension of the formulation from ref. <a title="Select to navigate to references" href="#cit246">246</a>. Petri nets have been used to model chemical and biological reaction networks,<a title="Select to navigate to reference" href="#cit243"><sup><span class="sup_ref">243,245,247,248</span></sup></a> while the use of PNO in chemical engineering has to date mostly focused on batch scheduling.<a title="Select to navigate to reference" href="#cit246"><sup><span class="sup_ref">246,249</span></sup></a> The MILP problem has higher model complexity than the LP core of the RNFA. Working with a PNO formulation allows to have a more detailed analysis of the solution space, <span class="italic">e.g.</span> the reaction sequence is considered, degenerated solutions are prevented, the maximum size of reaction steps is controlled, and non-flux dependent costs can be introduced.<a title="Select to navigate to references" href="#cit35"><sup><span class="sup_ref">35</span></sup></a> The number of continuous variables is higher by a factor of the number of reaction steps and the MILP formulation introduces binary variables per reaction and reaction step. Additionally, the number of constraints is higher, due to constraints on the firing of the transitions in sequence.</p>
        </div>
      
      
        <h3 id="sect1124"><span class="b_heading">Uncertainty in decision-making</span></h3>
        <span>Data underlying decision-making algorithms often bring about uncertainties, <span class="italic">e.g.</span> through experiments and measurements, through data inference to build complete datasets, through real-life scenarios of market prices, and through dynamic changes in supply and demand. To account for uncertainties in key parameters, deterministic models, which describe parameter uncertainties by bounds of anticipated derivations, or stochastic programming, which takes probability distribution functions for parameters into account, are applied.<a title="Select to navigate to references" href="#cit250"><sup><span class="sup_ref">250</span></sup></a></span>
        <p class="otherpara">The field of optimisation under uncertainty already contains well-established methods, <span class="italic">e.g.</span> stochastic programming, robust optimisation, or fuzzy programming, which can be applied on reaction networks with uncertain data.<a title="Select to navigate to reference" href="#cit250"><sup><span class="sup_ref">250,251</span></sup></a> While stochastic programming approaches generate comprehensive solutions based on probabilities, they are often computationally expensive. Robust programming defines uncertainties as inequality constraints and is often a good alternative if probability distributions are not known.<a title="Select to navigate to references" href="#cit251"><sup><span class="sup_ref">251</span></sup></a> For pathways selection in integrated biorefineries, some works have integrated uncertainties. Morales-Rodriguez <span class="italic">et al.</span> have applied stochastic process optimisation for lignocellulosic ethanol production and Kasaš <span class="italic">et al.</span> outline a strategy based on stochastic programming merging four distinct solution techniques for a bioethanol product case study.<a title="Select to navigate to reference" href="#cit252"><sup><span class="sup_ref">252,253</span></sup></a> Tay <span class="italic">et al.</span> and Tang <span class="italic">et al.</span> solve MINLP problems for integrated biorefineries using robust optimisation.<a title="Select to navigate to reference" href="#cit251"><sup><span class="sup_ref">251,254</span></sup></a> Uncertainties in the aforementioned studies cover amongst others the market price, supply of biomass, and demand for products as well as technological constraints.</p>
        <p class="otherpara">Including uncertainties during decision-making brings about benefits as it avoids non-optimal or infeasible solutions, but requires models that inform on uncertainties within the prediction task.<a title="Select to navigate to references" href="#cit250"><sup><span class="sup_ref">250</span></sup></a> For reaction networks, this means that uncertainties for key parameters, <span class="italic">e.g.</span> stoichiometry and helper species, or reaction conditions need to be collected during data inference stages.</p>
      
    
    
      
      <h2 id="sect1142"><span class="a_heading">5. Conclusions and perspective</span></h2>
      <span>The identification of sustainable reactions is a highly complex and interdisciplinary challenge. In this review we present the first multidisciplinary perspective, integrating the fields of data, metrics, and decision-making to guide and accelerate further developments. We highlight synergies between the fields and potential for future developments.</span>
      <p class="otherpara">Currently, the field of data brings about most bottlenecks, and therewith greatest potential for advancement. Data is, at present, incomplete, lacking information necessary to perform mass balances over large numbers of reactions. Furthermore, enabling linkages of various data sources, <span class="italic">e.g.</span> regional waste stream compositions, pretreatment options, or end-of-life use, is essential when dealing with questions of sustainability. For the field of sustainability metrics, we envision, that molecular property prediction, <span class="italic">e.g.</span> by graph convolutional networks, will allow more accurate evaluations of different environmental metrics and that linked and accessible data sources will allow assessments across the system boundary. In the area of decision-making, we highlight the importance of the structure of reaction networks (multiple in- and outputs, circular interactions) and the scalability of the previously suggested algorithms as main factors of importance. Methods in the field are well-established and most likely to evolve further through smarter heuristics or ML-guided approximations, enabling solution of system-level problems.</p>
      <p class="otherpara">Our findings elucidate the interface between the three areas. This allows scientists to take into account possible improvements within other fields so that we will jointly work towards more sustainable use of present resources. This also highlights the need for targeted interdisciplinary funding across the three domains. Through such targeted interventions society will achieve a faster transition towards developing truly sustainable solutions. The contribution of this work, while conceptual, provides a roadmap towards systematic reaction pathway planning based on rapid digitalisation of chemical data.</p>
    
    
      <h2 id="sect1150"><span class="a_heading">Conflicts of interest</span></h2>
      <span>Jana Weber, Zhen Guo and Alexei Lapkin are founders of Chemical Data Intelligence (CDI) Pte Ltd. The company is exploiting know-how and developed codes on analysis of large reaction networks.</span>
    
  
    <h2 id="sect1154"><span class="a_heading">Acknowledgements</span></h2>
      <span>J. M. W. gratefully acknowledges the Department of Chemical Engineering and Biotechnology at the University of Cambridge for funding her PhD scholarship. AMS is supported by the TU Delft AI Labs Programme. C. Z. acknowledges funding of his PhD scholarship by Cambridge Trust and Chinese Scholarship Council. This work was supported by National Research Foundation (NRF), Prime Minister's Office, Singapore under its Campus for Research Excellence and Technological Enterprise (CREATE) program: Cambridge Centre for Advanced Research and Education in Singapore Ltd (CARES, C4T project) (AAL &amp; ZG). CDI Pte Ltd is a spin off company of CARES Ltd.</span>
    
    <span id="sect1156"><h2 id="sect1153"><span class="a_heading">References</span></h2></span><ol type="1">
      <li><span id="cit1">J. H. Clark, <span class="italic">Curr. Opin. Green Sustainable Chem.</span>, 2017, <span class="bold">8</span>, 10–13 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cogsc.2017.07.008" title="DOI Link to resource 10.1016/j.cogsc.2017.07.008">CrossRef</a>.</span></li>
      <li><span id="cit2">A. Kätelhön, R. Meys, S. Deutz, S. Suh and A. Bardow, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2019, <span class="bold">166</span>, 11187–11194 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.1821029116" title="DOI Link to resource 10.1073/pnas.1821029116">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31085651%5Buid%5D" title="PubMed Link to resource 31085651">PubMed</a>.</span></li>
      <li><span id="cit3">S. Wang, G. Dai, H. Yang and Z. Luo, <span class="italic">Prog. Energy Combust. Sci.</span>, 2017, <span class="bold">62</span>, 33–86 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.pecs.2017.05.004" title="DOI Link to resource 10.1016/j.pecs.2017.05.004">CrossRef</a>.</span></li>
      <li><span id="cit4">Z. Sun, B. Fridrich, A. De Santi, S. Elangovan and K. Barta, <span class="italic">Chem. Rev.</span>, 2018, <span class="bold">118</span>, 614–678 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.7b00588" title="DOI Link to resource 10.1021/acs.chemrev.7b00588">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXotFGqsg%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29337543%5Buid%5D" title="PubMed Link to resource 29337543">PubMed</a>.</span></li>
      <li><span id="cit5">A. D. Sadiq, X. Chen, N. Yan and J. Sperry, <span class="italic">ChemSusChem</span>, 2018, <span class="bold">11</span>, 532–535 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cssc.201702356" title="DOI Link to resource 10.1002/cssc.201702356">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXlt1Cjsw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29247474%5Buid%5D" title="PubMed Link to resource 29247474">PubMed</a>.</span></li>
      <li><span id="cit6">Z. Guo, N. Yan and A. A. Lapkin, <span class="italic">Curr. Opin. Chem. Eng.</span>, 2019, <span class="bold">26</span>, 148–156 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.coche.2019.09.010" title="DOI Link to resource 10.1016/j.coche.2019.09.010">CrossRef</a>.</span></li>
      <li><span id="cit7">M. Alexandri, R. Schneider, H. Papapostolou, D. Ladakis, A. Koutinas and J. Venus, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2019, <span class="bold">7</span>, 6569–6579 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssuschemeng.8b04874" title="DOI Link to resource 10.1021/acssuschemeng.8b04874">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXjsFaiu7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit8">M. Arshadi, T. M. Attard, R. M. Lukasik, M. Brncic, A. M. Da Costa Lopes, M. Finell, P. Geladi, L. N. Gerschenson, F. Gogus, M. Herrero, A. J. Hunt, E. Ibáñez, B. Kamm, I. Mateos-Aparicio, A. Matias, N. E. Mavroudis, E. Montoneri, A. R. C. Morais, C. Nilsson, E. H. Papaioannou, A. Richel, P. Rupérez, B. Škrbić, M. B. Solarov, J. Švarc-Gajić, K. W. Waldron and F. J. Yuste-Córdoba, <span class="italic">Green Chem.</span>, 2016, <span class="bold">18</span>, 6160–6204 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6GC01389A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6GC01389A">RSC</a>.</span></li>
      <li><span id="cit9">H. Yang, G. Gözaydln, R. R. Nasaruddin, J. R. G. Har, X. Chen, X. Wang and N. Yan, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2019, <span class="bold">7</span>, 5532–5542 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssuschemeng.8b06853" title="DOI Link to resource 10.1021/acssuschemeng.8b06853">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFejsbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit10">L. Y. Jia, M. Raad, S. Hamieh, J. Toufaily, T. Hamieh, M. M. Bettahar, G. Mauviel, M. Tarrighi, L. Pinard and A. Dufour, <span class="italic">Green Chem.</span>, 2017, <span class="bold">19</span>, 5442–5459 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7GC02309J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7GC02309J">RSC</a>.</span></li>
      <li><span id="cit11">A. V. Bridgwater, <span class="italic">Biomass Bioenergy</span>, 2012, <span class="bold">38</span>, 68–94 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.biombioe.2011.01.048" title="DOI Link to resource 10.1016/j.biombioe.2011.01.048">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xit1SrtL4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit12">A. D. Patel, K. Meesters, H. Den Uil, E. De Jong, K. Blok and M. K. Patel, <span class="italic">Energy Environ. Sci.</span>, 2012, <span class="bold">5</span>, 8430–8444 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C2EE21581K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C2EE21581K">RSC</a>.</span></li>
      <li><span id="cit13">B. E. Dale, <span class="italic">J. Chem. Technol. Biotechnol.</span>, 2003, <span class="bold">78</span>, 1093–1103 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/jctb.850" title="DOI Link to resource 10.1002/jctb.850">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3sXnslKhsrk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit14">P. F. H. Harmsen, M. M. Hackmann and H. L. Bos, <span class="italic">Biofuels, Bioprod. Biorefin.</span>, 2014, <span class="bold">8</span>, 306–324 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/bbb.1468" title="DOI Link to resource 10.1002/bbb.1468">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXit1OktA%253D%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit15">H. Storz and K. Vorlop, <span class="italic">Appl. Agric. For. Res.</span>, 2013, <span class="bold">63</span>, 321–332 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Appl.%20Agric.%20For.%20Res.%5BJour%5D%20AND%2063%5Bvolume%5D%20AND%20321%5Bpage%5D%20and%202013%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit16">J. B. McKinlay, C. Vieille and J. G. Zeikus, <span class="italic">Appl. Microbiol. Biotechnol.</span>, 2007, <span class="bold">76</span>, 727–740 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00253-007-1057-y" title="DOI Link to resource 10.1007/s00253-007-1057-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXpslarur0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17609945%5Buid%5D" title="PubMed Link to resource 17609945">PubMed</a>.</span></li>
      <li><span id="cit17">L. G. Papageorgiou, <span class="italic">Comput. Chem. Eng.</span>, 2009, <span class="bold">33</span>, 1931–1938 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2009.06.014" title="DOI Link to resource 10.1016/j.compchemeng.2009.06.014">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXht1Oht7jI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit18">T. Verbeek and A. Mah, <span class="italic">Econ. Geogr.</span>, 2020, <span class="bold">96</span>, 363–387 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/00130095.2020.1794809" title="DOI Link to resource 10.1080/00130095.2020.1794809">CrossRef</a>.</span></li>
      <li><span id="cit19">M. Fialkowski, K. J. M. Bishop, V. A. Chubukov, C. J. Campbell and B. A. Grzybowski, <span class="italic">Angew. Chem.</span>, 2005, <span class="bold">117</span>, 7429–7435 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ange.200502272" title="DOI Link to resource 10.1002/ange.200502272">CrossRef</a>.</span></li>
      <li><span id="cit20">K. J. M. Bishop, R. Klajn and B. A. Grzybowski, <span class="italic">Angew. Chem., Int. Ed.</span>, 2006, <span class="bold">45</span>, 5348–5354 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.200600881" title="DOI Link to resource 10.1002/anie.200600881">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28Xoslels7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16835857%5Buid%5D" title="PubMed Link to resource 16835857">PubMed</a>.</span></li>
      <li><span id="cit21">P. M. Jacob and A. Lapkin, <span class="italic">React. Chem. Eng.</span>, 2018, <span class="bold">3</span>, 102–118 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7RE00129K&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7RE00129K">RSC</a>.</span></li>
      <li><span id="cit22">E. J. Llanos, W. Leal, D. H. Luu, J. Jost, P. F. Stadler and G. Restrepo, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2019, <span class="bold">116</span>, 12660–12665 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.1816039116" title="DOI Link to resource 10.1073/pnas.1816039116">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXht1ersrnO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31186353%5Buid%5D" title="PubMed Link to resource 31186353">PubMed</a>.</span></li>
      <li><span id="cit23">B. A. Grzybowski, K. J. M. Bishop, B. Kowalczyk and C. E. Wilmer, <span class="italic">Nat. Chem.</span>, 2009, <span class="bold">1</span>, 31–36 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nchem.136" title="DOI Link to resource 10.1038/nchem.136">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXktlSltbc%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21378798%5Buid%5D" title="PubMed Link to resource 21378798">PubMed</a>.</span></li>
      <li><span id="cit24">S. Szymkuć, E. P. Gajewska, T. Klucznik, K. Molga, P. Dittwald, M. Startek, M. Bajczyk and B. A. Grzybowski, <span class="italic">Angew. Chem., Int. Ed.</span>, 2016, <span class="bold">55</span>, 5904–5937 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201506101" title="DOI Link to resource 10.1002/anie.201506101">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27062365%5Buid%5D" title="PubMed Link to resource 27062365">PubMed</a>.</span></li>
      <li><span id="cit25">C. M. Gothard, S. Soh, N. A. Gothard, B. Kowalczyk, Y. Wei, B. Baytekin and B. A. Grzybowski, <span class="italic">Angew. Chem., Int. Ed.</span>, 2012, <span class="bold">51</span>, 7922–7927 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201202155" title="DOI Link to resource 10.1002/anie.201202155">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhtVamtLvF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22807071%5Buid%5D" title="PubMed Link to resource 22807071">PubMed</a>.</span></li>
      <li><span id="cit26">P. M. Jacob, P. Yamin, C. Perez-Storey, M. Hopgood and A. A. Lapkin, <span class="italic">Green Chem.</span>, 2017, <span class="bold">19</span>, 140–152 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C6GC02482C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C6GC02482C">RSC</a>.</span></li>
      <li><span id="cit27">A. A. Lapkin, P. K. Heer, P. M. Jacob, M. Hutchby, W. Cunningham, S. D. Bull and M. G. Davidson, <span class="italic">Faraday Discuss.</span>, 2017, <span class="bold">202</span>, 483–496 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7FD00073A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7FD00073A">RSC</a>.</span></li>
      <li><span id="cit28">J. M. Weber, P. Lió and A. A. Lapkin, <span class="italic">React. Chem. Eng.</span>, 2019, <span class="bold">4</span>, 1969–1981 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9RE00213H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9RE00213H">RSC</a>.</span></li>
      <li><span id="cit29">J. D. Orth, I. Thiele and B. O. Palsson, <span class="italic">Nat. Biotechnol.</span>, 2010, <span class="bold">28</span>, 245–248 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nbt.1614" title="DOI Link to resource 10.1038/nbt.1614">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXivV2rtL4%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20212490%5Buid%5D" title="PubMed Link to resource 20212490">PubMed</a>.</span></li>
      <li><span id="cit30">A. Voll and W. Marquardt, <span class="italic">AIChE J.</span>, 2012, <span class="bold">58</span>, 1788–1801 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aic.12704" title="DOI Link to resource 10.1002/aic.12704">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXpvVCisbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit31">K. Ulonska, A. Voll and W. Marquardt, <span class="italic">Energy Fuels</span>, 2016, <span class="bold">30</span>, 445–456 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.energyfuels.5b02460" title="DOI Link to resource 10.1021/acs.energyfuels.5b02460">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXitVSksrrN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit32">K. Ulonska, A. König, M. Klatt, A. Mitsos and J. Viell, <span class="italic">Ind. Eng. Chem. Res.</span>, 2018, <span class="bold">57</span>, 6980–6991 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.iecr.8b00245" title="DOI Link to resource 10.1021/acs.iecr.8b00245">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXovVOisrs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit33">K. Ulonska, M. Skiborowski, A. Mitsos and J. Viell, <span class="italic">AIChE J.</span>, 2016, <span class="bold">62</span>, 3096–3108 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aic.15305" title="DOI Link to resource 10.1002/aic.15305">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xoslaltro%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit34">D. Zhang, E. A. Del Rio-Chanona and N. Shah, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2017, <span class="bold">5</span>, 4388–4398 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssuschemeng.7b00429" title="DOI Link to resource 10.1021/acssuschemeng.7b00429">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXlvVGrtrw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit35">J. M. Weber, A. M. Schweidtmann, E. Nolasco and A. A. Lapkin, <span class="italic">Eur. Symp. Comput. Aided Process Eng.</span>, 2020, <span class="bold">48</span>, 1843–1848 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Eur.%20Symp.%20Comput.%20Aided%20Process%20Eng.%5BJour%5D%20AND%2048%5Bvolume%5D%20AND%201843%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit36">
          P. T. Anastas and J. C. Warner, <span class="italic">Green Chemistry: Theory and Practice</span>, Oxford University Press, Oxford,  1998, pp. 29–56 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Green%20Chemistry:%20Theory%20and%20Practice%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201998%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit37">S. L. Y. Tang, R. L. Smith and M. Poliakoff, <span class="italic">Green Chem.</span>, 2005, <span class="bold">7</span>, 761–762 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B513020B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B513020B">RSC</a>.</span></li>
      <li><span id="cit38">F. G. Calvo-Flores, <span class="italic">ChemSusChem</span>, 2009, <span class="bold">2</span>, 905–919 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cssc.200900128" title="DOI Link to resource 10.1002/cssc.200900128">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXht1yltbjK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19780101%5Buid%5D" title="PubMed Link to resource 19780101">PubMed</a>.</span></li>
      <li><span id="cit39">P. T. Anastas and J. B. Zimmermann, <span class="italic">Environ. Sci. Technol.</span>, 2003, 94–101 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/es032373g" title="DOI Link to resource 10.1021/es032373g">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=12666905%5Buid%5D" title="PubMed Link to resource 12666905">PubMed</a>.</span></li>
      <li><span id="cit40">S. Y. Tang, R. A. Bourne, R. L. Smith and M. Poliakoff, <span class="italic">Green Chem.</span>, 2008, <span class="bold">10</span>, 268–269 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B719469M&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B719469M">RSC</a>.</span></li>
      <li><span id="cit41">ISO 14044, 2006, Environmental management — Life cycle assessment — Requirements and guidelines, International Organization for Standardization, available from https://www.iso.org/standard/38498.html.</span></li>
      <li><span id="cit42">L. Jacquemin, P. Y. Pontalier and C. Sablayrolles, <span class="italic">Int. J. Life Cycle Assess.</span>, 2012, <span class="bold">17</span>, 1028–1041 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11367-012-0432-9" title="DOI Link to resource 10.1007/s11367-012-0432-9">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhtFCnu7rO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit43">M. A. Curran, <span class="italic">Curr. Opin. Chem. Eng.</span>, 2013, <span class="bold">2</span>, 273–277 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.coche.2013.02.002" title="DOI Link to resource 10.1016/j.coche.2013.02.002">CrossRef</a>.</span></li>
      <li><span id="cit44">
          V. Kapur, Hydrocarbon Processing, 2015.</span></li>
      <li><span id="cit45">C. W. Coley, N. S. Eyke and K. F. Jensen, <span class="italic">Angew. Chem., Int. Ed.</span>, 2020, <span class="bold">59</span>, 23414–23436 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201909989" title="DOI Link to resource 10.1002/anie.201909989">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtFCjtL7M" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31553509%5Buid%5D" title="PubMed Link to resource 31553509">PubMed</a>.</span></li>
      <li><span id="cit46">M. D. Wilkinson, M. Dumontier, Ij. J. Aalbersberg, G. Appleton, M. Axton, A. Baak, N. Blomberg, J. W. Boiten, L. B. da Silva Santos, P. E. Bourne, J. Bouwman, A. J. Brookes, T. Clark, M. Crosas, I. Dillo, O. Dumon, S. Edmunds, C. T. Evelo, R. Finkers, A. Gonzalez-Beltran, A. J. G. Gray, P. Groth, C. Goble, J. S. Grethe, J. Heringa, P. A. C. t Hoen, R. Hooft, T. Kuhn, R. Kok, J. Kok, S. J. Lusher, M. E. Martone, A. Mons, A. L. Packer, B. Persson, P. Rocca-Serra, M. Roos, R. van Schaik, S. A. Sansone, E. Schultes, T. Sengstag, T. Slater, G. Strawn, M. A. Swertz, M. Thompson, J. Van Der Lei, E. Van Mulligen, J. Velterop, A. Waagmeester, P. Wittenburg, K. Wolstencroft, J. Zhao and B. Mons, <span class="italic">Sci. Data</span>, 2016, <span class="bold">3</span>, 160018 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/sdata.2016.18" title="DOI Link to resource 10.1038/sdata.2016.18">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26978244%5Buid%5D" title="PubMed Link to resource 26978244">PubMed</a>.</span></li>
      <li><span id="cit47">M. Lnenicka and J. Komarkova, <span class="italic">Gov. Inf. Q</span>, 2019, <span class="bold">36</span>, 129–144 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.giq.2018.11.004" title="DOI Link to resource 10.1016/j.giq.2018.11.004">CrossRef</a>.</span></li>
      <li><span id="cit48">M. Janssen and G. Kuk, <span class="italic">J. Organ. Comput. Electron. Commer</span>, 2016, <span class="bold">26</span>, 3–13 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/10919392.2015.1124005" title="DOI Link to resource 10.1080/10919392.2015.1124005">CrossRef</a>.</span></li>
      <li><span id="cit49">
          M. Janssen, R. Matheus and A. Zuiderwijk, International Conference on Electronic Government, 2015, vol. 9248, pp. 79–90.</span></li>
      <li><span id="cit50">A. Thakkar, T. Kogej, J. L. Reymond, O. Engkvist and E. J. Bjerrum, <span class="italic">Chem. Sci.</span>, 2020, <span class="bold">11</span>, 154–168 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9SC04944D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9SC04944D">RSC</a>.</span></li>
      <li><span id="cit51">Y.-S. Ko, J. W. Kim, J. A. Lee, T. Han, G. B. Kim, J. E. Park and S. Y. Lee, <span class="italic">Chem. Soc. Rev.</span>, 2020, <span class="bold">49</span>, 4615–4636 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0CS00155D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0CS00155D">RSC</a>.</span></li>
      <li><span id="cit52">K. C. Thomas and W. M. Ingledew, <span class="italic">J. Ind. Microbiol.</span>, 1992, <span class="bold">10</span>, 61–68 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/BF01583635" title="DOI Link to resource 10.1007/BF01583635">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK38XmtlaitL8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit53">N. J. H. Averesch and J. O. Krömer, <span class="italic">Front. Bioeng. Biotechnol.</span>, 2018, <span class="bold">6</span>, 32 <a target="_blank" class="DOILink" href="https://doi.org/10.3389/fbioe.2018.00032" title="DOI Link to resource 10.3389/fbioe.2018.00032">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29632862%5Buid%5D" title="PubMed Link to resource 29632862">PubMed</a>.</span></li>
      <li><span id="cit54">C. Fischer-Romero, B. J. Tindall and F. Jüttner, <span class="italic">Int. J. Syst. Bacteriol.</span>, 1996, <span class="bold">46</span>, 183–188 <a target="_blank" class="DOILink" href="https://doi.org/10.1099/00207713-46-1-183" title="DOI Link to resource 10.1099/00207713-46-1-183">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADyaK287jvV2ktA%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=8573493%5Buid%5D" title="PubMed Link to resource 8573493">PubMed</a>.</span></li>
      <li><span id="cit55">T. Fehér, A. G. Planson, P. Carbonell, A. Fernández-Castané, I. Grigoras, E. Dariy, A. Perret and J. L. Faulon, <span class="italic">Biotechnol. J.</span>, 2014, <span class="bold">9</span>, 1446–1457 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/biot.201400055" title="DOI Link to resource 10.1002/biot.201400055">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25224453%5Buid%5D" title="PubMed Link to resource 25224453">PubMed</a>.</span></li>
      <li><span id="cit56">V. E. Balderas-Hernández, A. Sabido-Ramos, P. Silva, N. Cabrera-Valladares, G. Hernández-Chávez, J. L. Báez-Viveros, A. Martínez, F. Bolívar and G. Gosset, <span class="italic">Microb. Cell Fact.</span>, 2009, <span class="bold">8</span>, 1–12 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/1475-2859-8-19" title="DOI Link to resource 10.1186/1475-2859-8-19">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19341482%5Buid%5D" title="PubMed Link to resource 19341482">PubMed</a>.</span></li>
      <li><span id="cit57">S. Y. Lee, H. U. Kim, T. U. Chae, J. S. Cho, J. W. Kim, J. H. Shin, D. I. Kim, Y. S. Ko, W. D. Jang and Y. S. Jang, <span class="italic">Nat. Catal.</span>, 2019, <span class="bold">2</span>, 18–33 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41929-018-0212-4" title="DOI Link to resource 10.1038/s41929-018-0212-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtFGisb%252FI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit58">M. Kanehisa and S. Goto, <span class="italic">Nucleic Acids Res.</span>, 2000, <span class="bold">28</span>, 27–30 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nar/28.1.27" title="DOI Link to resource 10.1093/nar/28.1.27">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD3cXhvVGqu74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=10592173%5Buid%5D" title="PubMed Link to resource 10592173">PubMed</a>.</span></li>
      <li><span id="cit59">A. Morgat, T. Lombardot, K. B. Axelsen, L. Aimo, A. Niknejad, N. Hyka-Nouspikel, E. Coudert, M. Pozzato, M. Pagni, S. Moretti, S. Rosanoff, J. Onwubiko, L. Bougueleret, I. Xenarios, N. Redaschi and A. Bridge, <span class="italic">Nucleic Acids Res.</span>, 2017, <span class="bold">45</span>, 415–418 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nar/gkw990" title="DOI Link to resource 10.1093/nar/gkw990">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27789701%5Buid%5D" title="PubMed Link to resource 27789701">PubMed</a>.</span></li>
      <li><span id="cit60">N. Nagano, <span class="italic">Nucleic Acids Res.</span>, 2005, <span class="bold">33</span>, 407–412 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nar/gki080" title="DOI Link to resource 10.1093/nar/gki080">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15608227%5Buid%5D" title="PubMed Link to resource 15608227">PubMed</a>.</span></li>
      <li><span id="cit61">M. Oh, T. Yamada, M. Hattori, S. Goto and M. Kanehisa, <span class="italic">J. Chem. Inf. Model.</span>, 2007, <span class="bold">47</span>, 1702–1712 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci700006f" title="DOI Link to resource 10.1021/ci700006f">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXls1SmtLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17516640%5Buid%5D" title="PubMed Link to resource 17516640">PubMed</a>.</span></li>
      <li><span id="cit62">Y. Moriya, D. Shigemizu, M. Hattori, T. Tokimatsu, M. Kotera, S. Goto and M. Kanehisa, <span class="italic">Nucleic Acids Res.</span>, 2010, <span class="bold">38</span>, 138–143 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/nar/gkq318" title="DOI Link to resource 10.1093/nar/gkq318">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20435670%5Buid%5D" title="PubMed Link to resource 20435670">PubMed</a>.</span></li>
      <li><span id="cit63">J. E. Matthiesen, J. M. Carraher, M. Vasiliu, D. A. Dixon and J. P. Tessonnier, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2016, <span class="bold">4</span>, 3575–3585 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssuschemeng.6b00679" title="DOI Link to resource 10.1021/acssuschemeng.6b00679">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XntVOgs7g%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit64">M. J. Orella, Y. Román-Leshkov and F. R. Brushett, <span class="italic">Curr. Opin. Chem. Eng.</span>, 2018, <span class="bold">20</span>, 159–167 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.coche.2018.05.002" title="DOI Link to resource 10.1016/j.coche.2018.05.002">CrossRef</a>.</span></li>
      <li><span id="cit65">F. Harnisch and C. Urban, <span class="italic">Angew. Chem., Int. Ed.</span>, 2018, <span class="bold">57</span>, 10016–10023 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.201711727" title="DOI Link to resource 10.1002/anie.201711727">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjs1Cms7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29235724%5Buid%5D" title="PubMed Link to resource 29235724">PubMed</a>.</span></li>
      <li><span id="cit66">V. Balzani, G. Pacchioni, M. Prato and A. Zecchina, <span class="italic">Rend. Lincei</span>, 2019, <span class="bold">30</span>, 443–452 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s12210-019-00836-2" title="DOI Link to resource 10.1007/s12210-019-00836-2">CrossRef</a>.</span></li>
      <li><span id="cit67">J. He and C. Janáky, <span class="italic">ACS Energy Lett.</span>, 2020, <span class="bold">5</span>, 1996–2014 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsenergylett.0c00645" title="DOI Link to resource 10.1021/acsenergylett.0c00645">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXptlGgs7k%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32566753%5Buid%5D" title="PubMed Link to resource 32566753">PubMed</a>.</span></li>
      <li><span id="cit68">S. J. Lusher, R. McGuire, R. C. Van Schaik, C. D. Nicholson and J. De Vlieg, <span class="italic">Drug Discovery Today</span>, 2014, <span class="bold">19</span>, 859–868 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.drudis.2013.12.004" title="DOI Link to resource 10.1016/j.drudis.2013.12.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXivVKhtw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=24361338%5Buid%5D" title="PubMed Link to resource 24361338">PubMed</a>.</span></li>
      <li><span id="cit69">T. Berners-Lee, J. Hendler and O. Lassila, <span class="italic">Sci. Am.</span>, 2001, <span class="bold">284</span>, 34–43 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/scientificamerican0501-34" title="DOI Link to resource 10.1038/scientificamerican0501-34">CrossRef</a>.</span></li>
      <li><span id="cit70">P. Murray-Rust, <span class="italic">Nature</span>, 2008, <span class="bold">451</span>, 648–651 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/451648a" title="DOI Link to resource 10.1038/451648a">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXhs1KntrY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18256659%5Buid%5D" title="PubMed Link to resource 18256659">PubMed</a>.</span></li>
      <li><span id="cit71">S. Auer and S. Mann, <span class="italic">Ser. Libr.</span>, 2019, <span class="bold">76</span>, 35–41 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/0361526X.2019.1540272" title="DOI Link to resource 10.1080/0361526X.2019.1540272">CrossRef</a>.</span></li>
      <li><span id="cit72">
          M. Y. Jaradeh, A. Oelen, K. E. Farfar, M. Prinz, J. D’Souza, G. Kismihok, M. Stocker and S. Auer, <span class="italic">Proceedings of the 10th International Conference on Knowledge Capture</span>,  2019, pp. 243–246 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Proceedings%20of%20the%2010th%20International%20Conference%20on%20Knowledge%20Capture%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit73">
          S. Auer, A. Kasprzik, V. Kovtun, M. Stocker, M. Prinz and M. E. Vidal, Proceedings of the 8th International Conference on Web Intelligence, Mining and Semantics, 2018, pp. 1–6.</span></li>
      <li><span id="cit74">P. Murray-Rust and H. S. Rzepa, <span class="italic">J. Chem. Inf. Comput. Sci.</span>, 1999, <span class="bold">39</span>, 928–942 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci990052b" title="DOI Link to resource 10.1021/ci990052b">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1MXlvV2jt7c%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit75">F. Farazi, J. Akroyd, S. Mosbach, P. Buerger, D. Nurkowski, M. Salamanca and M. Kraft, <span class="italic">J. Chem. Inf. Model.</span>, 2019, <span class="bold">60</span>, 108–120 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.9b00960" title="DOI Link to resource 10.1021/acs.jcim.9b00960">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31846323%5Buid%5D" title="PubMed Link to resource 31846323">PubMed</a>.</span></li>
      <li><span id="cit76">A. Menon, N. B. Krdzavac and M. Kraft, <span class="italic">Curr. Opin. Chem. Eng.</span>, 2019, <span class="bold">26</span>, 33–37 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.coche.2019.08.004" title="DOI Link to resource 10.1016/j.coche.2019.08.004">CrossRef</a>.</span></li>
      <li><span id="cit77">J. Morbach, A. Yang and W. Marquardt, <span class="italic">Eng. Appl. Artif. Intell.</span>, 2007, <span class="bold">20</span>, 147–161 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.engappai.2006.06.010" title="DOI Link to resource 10.1016/j.engappai.2006.06.010">CrossRef</a>.</span></li>
      <li><span id="cit78">J. Morbach, A. Wiesner and W. Marquardt, <span class="italic">Comput. Chem. Eng.</span>, 2009, <span class="bold">33</span>, 1546–1556 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2009.01.019" title="DOI Link to resource 10.1016/j.compchemeng.2009.01.019">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXpvFentbs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit79">A. Eibeck, M. Q. Lim and M. Kraft, <span class="italic">Comput. Chem. Eng.</span>, 2019, <span class="bold">131</span>, 106586 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2019.106586" title="DOI Link to resource 10.1016/j.compchemeng.2019.106586">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvFyhu7zE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit80">M. Pan, J. Sikorski, C. A. Kastner, J. Akroyd, S. Mosbach, R. Lau and M. Kraft, <span class="italic">Energy Procedia</span>, 2015, <span class="bold">75</span>, 1536–1541 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.egypro.2015.07.313" title="DOI Link to resource 10.1016/j.egypro.2015.07.313">CrossRef</a>.</span></li>
      <li><span id="cit81">M. Kraft and A. Eibeck, <span class="italic">Chem. Ing. Tech.</span>, 2020, <span class="bold">92</span>, 967–977 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/cite.202000002" title="DOI Link to resource 10.1002/cite.202000002">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXovFymtr0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit82">L. Zhou, C. Zhang, I. A. Karimi and M. Kraft, <span class="italic">Energy Procedia</span>, 2017, <span class="bold">142</span>, 2953–2958 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.egypro.2017.12.313" title="DOI Link to resource 10.1016/j.egypro.2017.12.313">CrossRef</a>.</span></li>
      <li><span id="cit83">P. M. Jacob, T. Lan, J. M. Goodman and A. A. Lapkin, <span class="italic">J. Cheminform.</span>, 2017, <span class="bold">9</span>, 1–12 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13321-016-0187-6" title="DOI Link to resource 10.1186/s13321-016-0187-6">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28316652%5Buid%5D" title="PubMed Link to resource 28316652">PubMed</a>.</span></li>
      <li><span id="cit84">P. Yaseneva, P. Hodgson, J. Zakrzewski, S. Falß, R. E. Meadows and A. A. Lapkin, <span class="italic">React. Chem. Eng.</span>, 2016, <span class="bold">1</span>, 229–238 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5RE00048C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5RE00048C">RSC</a>.</span></li>
      <li><span id="cit85">
          E. D. Liddy, <span class="italic">Encyclopedia of Library and Information Science</span>, Marcel Decker, Inc, NY, USA, 2nd edn,  2001 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Encyclopedia%20of%20Library%20and%20Information%20Science%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202001%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit86">J. Hirschberg and C. D. Manning, <span class="italic">Science</span>, 2015, <span class="bold">349</span>, 261–266 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aaa8685" title="DOI Link to resource 10.1126/science.aaa8685">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFKktL%252FO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26185244%5Buid%5D" title="PubMed Link to resource 26185244">PubMed</a>.</span></li>
      <li><span id="cit87">
          A. Vaswani, N. Shazeer, N. Parmar, J. Uszkoreit, L. Jones, A. N. Gomez, Ł. Kaiser and I. Polosukhin, 2017, arXiv:1706.03762.</span></li>
      <li><span id="cit88">D. M. Jessop, S. E. Adams, E. L. Willighagen, L. Hawizy and P. Murray-Rust, <span class="italic">J. Cheminform.</span>, 2011, <span class="bold">3</span>, 1–12 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/1758-2946-3-1" title="DOI Link to resource 10.1186/1758-2946-3-1">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21214931%5Buid%5D" title="PubMed Link to resource 21214931">PubMed</a>.</span></li>
      <li><span id="cit89">M. Krallinger, O. Rabal, A. Lourenço, J. Oyarzabal and A. Valencia, <span class="italic">Chem. Rev.</span>, 2017, <span class="bold">117</span>, 7673–7761 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.chemrev.6b00851" title="DOI Link to resource 10.1021/acs.chemrev.6b00851">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXntVyru78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28475312%5Buid%5D" title="PubMed Link to resource 28475312">PubMed</a>.</span></li>
      <li><span id="cit90">N. A. Lewinski and B. T. McInnes, <span class="italic">Beilstein J. Nanotechnol.</span>, 2015, <span class="bold">6</span>, 1439–1449 <a target="_blank" class="DOILink" href="https://doi.org/10.3762/bjnano.6.149" title="DOI Link to resource 10.3762/bjnano.6.149">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXht1SlsrbL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26199848%5Buid%5D" title="PubMed Link to resource 26199848">PubMed</a>.</span></li>
      <li><span id="cit91">P. M. Nadkarni, L. Ohno-Machado and W. W. Chapman, <span class="italic">J. Am. Med. Inform. Assoc.</span>, 2011, <span class="bold">18</span>, 544–551 <a target="_blank" class="DOILink" href="https://doi.org/10.1136/amiajnl-2011-000464" title="DOI Link to resource 10.1136/amiajnl-2011-000464">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21846786%5Buid%5D" title="PubMed Link to resource 21846786">PubMed</a>.</span></li>
      <li><span id="cit92">
          M. Neumann, D. King, I. Beltagy and W. Ammar, 2019, arXiv:1902.07669.</span></li>
      <li><span id="cit93">A. Zaveri, A. Rula, A. Maurino, R. Pietrobon, J. Lehmann and S. Auer, <span class="italic">Semant. Web</span>, 2016, <span class="bold">7</span>, 63–93 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Semant.%20Web%5BJour%5D%20AND%207%5Bvolume%5D%20AND%2063%5Bpage%5D%20and%202016%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit94">J. Debattista, S. Auer and C. Lange, <span class="italic">J. Data Inf. Qual.</span>, 2016, <span class="bold">8</span>, 1–32 <a target="_blank" class="DOILink" href="https://doi.org/10.1145/2992786" title="DOI Link to resource 10.1145/2992786">CrossRef</a>.</span></li>
      <li><span id="cit95">S. M. Mennen, C. Alhambra, C. L. Allen, M. Barberis, S. Berritt, T. A. Brandt, A. D. Campbell, J. Castañón, A. H. Cherney, M. Christensen, D. B. Damon, J. Eugenio De Diego, S. García-Cerrada, P. García-Losada, R. Haro, J. Janey, D. C. Leitch, L. Li, F. Liu, P. C. Lobben, D. W. C. Macmillan, J. Magano, E. McInturff, S. Monfette, R. J. Post, D. Schultz, B. J. Sitter, J. M. Stevens, I. I. Strambeanu, J. Twilton, K. Wang and M. A. Zajac, <span class="italic">Org. Process Res. Dev.</span>, 2019, <span class="bold">23</span>, 1213–1242 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.oprd.9b00140" title="DOI Link to resource 10.1021/acs.oprd.9b00140">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXptFyhsrc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit96">N. S. Eyke, W. H. Green and K. F. Jensen, <span class="italic">React. Chem. Eng.</span>, 2020, <span class="bold">5</span>, 1963–1972 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0RE00232A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0RE00232A">RSC</a>.</span></li>
      <li><span id="cit97">E. O. Pyzer-Knapp, G. N. Simm and A. A. Guzik, <span class="italic">Mater. Horiz.</span>, 2016, <span class="bold">3</span>, 226–233 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C5MH00282F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C5MH00282F">RSC</a>.</span></li>
      <li><span id="cit98">J. J. F. Chen and D. P. Visco, <span class="italic">Chem. Eng. Sci.</span>, 2017, <span class="bold">159</span>, 31–42 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ces.2016.02.037" title="DOI Link to resource 10.1016/j.ces.2016.02.037">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xjs12qtbk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit99">E. Danielson, J. H. Golden, E. W. McFarland, C. M. Reaves, W. H. Weinberg and X. Di Wu, <span class="italic">Nature</span>, 1997, <span class="bold">389</span>, 944–948 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/40099" title="DOI Link to resource 10.1038/40099">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK2sXntValsr0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit100">Z. Li, S. Wang, W. S. Chin, L. E. Achenie and H. Xin, <span class="italic">J. Mater. Chem. A</span>, 2017, <span class="bold">5</span>, 24131–24138 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7TA01812F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7TA01812F">RSC</a>.</span></li>
      <li><span id="cit101">R. D. King, K. E. Whelan, F. M. Jones, P. G. K. Reiser, C. H. Bryant, S. H. Muggleton, D. B. Kell and S. G. Oliver, <span class="italic">Nature</span>, 2004, <span class="bold">427</span>, 247–252 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature02236" title="DOI Link to resource 10.1038/nature02236">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKjuw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=14724639%5Buid%5D" title="PubMed Link to resource 14724639">PubMed</a>.</span></li>
      <li><span id="cit102">A. M. Schweidtmann, A. D. Clayton, N. Holmes, E. Bradford, R. A. Bourne and A. A. Lapkin, <span class="italic">Chem. Eng. J.</span>, 2018, <span class="bold">352</span>, 277–282 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cej.2018.07.031" title="DOI Link to resource 10.1016/j.cej.2018.07.031">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXht12jur3O" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit103">C. Houben and A. A. Lapkin, <span class="italic">Curr. Opin. Chem. Eng.</span>, 2015, <span class="bold">9</span>, 1–7 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.coche.2015.07.001" title="DOI Link to resource 10.1016/j.coche.2015.07.001">CrossRef</a>.</span></li>
      <li><span id="cit104">C. W. Coley, D. A. Thomas, J. A. M. Lummiss, J. N. Jaworski, C. P. Breen, V. Schultz, T. Hart, J. S. Fishman, L. Rogers, H. Gao, R. W. Hicklin, P. P. Plehiers, J. Byington, J. S. Piotti, W. H. Green, A. John Hart, T. F. Jamison and K. F. Jensen, <span class="italic">Science</span>, 2019, <span class="bold">365</span>, 1–9 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aax1566" title="DOI Link to resource 10.1126/science.aax1566">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31395756%5Buid%5D" title="PubMed Link to resource 31395756">PubMed</a>.</span></li>
      <li><span id="cit105">S. Steiner, J. Wolf, S. Glatzel, A. Andreou, J. M. Granda, G. Keenan, T. Hinkley, G. Aragon-Camarasa, P. J. Kitson, D. Angelone and L. Cronin, <span class="italic">Science</span>, 2019, <span class="bold">363</span>, 1–8 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aav2211" title="DOI Link to resource 10.1126/science.aav2211">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30498165%5Buid%5D" title="PubMed Link to resource 30498165">PubMed</a>.</span></li>
      <li><span id="cit106">P. Schwaller, A. C. Vaucher, T. Laino and J.-L. Reymond, <span class="italic">Mach. Learn. Sci. Technol.</span>, 2021, <span class="bold">2</span>, 015016 <a target="_blank" class="DOILink" href="https://doi.org/10.1088/2632-2153/abc81d" title="DOI Link to resource 10.1088/2632-2153/abc81d">CrossRef</a>.</span></li>
      <li><span id="cit107">D. Jannach, P. Resnick, A. Tuzhilin and M. Zanker, <span class="italic">Commun. ACM</span>, 2016, <span class="bold">59</span>, 94–102 <a target="_blank" class="DOILink" href="https://doi.org/10.1145/2891406" title="DOI Link to resource 10.1145/2891406">CrossRef</a>.</span></li>
      <li><span id="cit108">P. Resnick and H. R. Varian, <span class="italic">Commun. ACM</span>, 1997, <span class="bold">40</span>, 56–58 <a target="_blank" class="DOILink" href="https://doi.org/10.1145/245108.245121" title="DOI Link to resource 10.1145/245108.245121">CrossRef</a>.</span></li>
      <li><span id="cit109">A. Feuerverger, Y. He and S. Khatri, <span class="italic">State Sci.</span>, 2012, <span class="bold">27</span>, 202–231 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=State%20Sci.%5BJour%5D%20AND%2027%5Bvolume%5D%20AND%20202%5Bpage%5D%20and%202012%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit110">E. J. Candès and B. Recht, <span class="italic">Found. Comput. Math.</span>, 2009, <span class="bold">9</span>, 717–772 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10208-009-9045-5" title="DOI Link to resource 10.1007/s10208-009-9045-5">CrossRef</a>.</span></li>
      <li><span id="cit111">
          V. Kalofolias, X. Bresson, M. Bronstein and P. Vandergheynst, 2014, arXiv1408.1717.2014.</span></li>
      <li><span id="cit112">G. Adomavicius and A. Tuzhilin, <span class="italic">IEEE Trans. Knowl. Data Eng.</span>, 2005, <span class="bold">17</span>, 734–749 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=IEEE%20Trans.%20Knowl.%20Data%20Eng.%5BJour%5D%20AND%2017%5Bvolume%5D%20AND%20734%5Bpage%5D%20and%202005%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit113">S. Zhang, L. Yao, A. Sun and Y. Tay, <span class="italic">ACM Comput. Surv.</span>, 2019, <span class="bold">52</span>, 1–38 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=ACM%20Comput.%20Surv.%5BJour%5D%20AND%2052%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit114">
          J. Savage, A. Kishimoto, B. Buesser, E. DIaz-Aviles and C. Alzate, Proc. 11th ACM Conf. Recomm. Syst., 2017, pp. 210–214.</span></li>
      <li><span id="cit115">F. Jirasek, R. A. S. Alves, J. Damay, R. A. Vandermeulen, R. Bamler, M. Bortz, S. Mandt, M. Kloft and H. Hasse, <span class="italic">J. Phys. Chem. Lett.</span>, 2020, <span class="bold">11</span>, 981–985 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jpclett.9b03657" title="DOI Link to resource 10.1021/acs.jpclett.9b03657">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtlWhu70%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31964142%5Buid%5D" title="PubMed Link to resource 31964142">PubMed</a>.</span></li>
      <li><span id="cit116">W. Zhang, H. Zou, L. Luo, Q. Liu, W. Wu and W. Xiao, <span class="italic">Neurocomputing</span>, 2016, <span class="bold">173</span>, 979–987 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.neucom.2015.08.054" title="DOI Link to resource 10.1016/j.neucom.2015.08.054">CrossRef</a>.</span></li>
      <li><span id="cit117">A. Seko, H. Hayashi and I. Tanaka, <span class="italic">J. Chem. Phys.</span>, 2018, <span class="bold">148</span>, 241719 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.5016210" title="DOI Link to resource 10.1063/1.5016210">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29960357%5Buid%5D" title="PubMed Link to resource 29960357">PubMed</a>.</span></li>
      <li><span id="cit118">N. H. Park, D. Y. Zubarev, J. L. Hedrick, V. Kiyek, C. Corbet and S. Lottier, <span class="italic">Macromolecules</span>, 2020, <span class="bold">53</span>, 10847–10854 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.macromol.0c02127" title="DOI Link to resource 10.1021/acs.macromol.0c02127">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXisVWntbrJ" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit119">S. Kite, T. Hattori and Y. Murakami, <span class="italic">Appl. Catal., A</span>, 1994, <span class="bold">114</span>, 173–178 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0926-860X(94)80169-X" title="DOI Link to resource 10.1016/0926-860X(94)80169-X">CrossRef</a>.</span></li>
      <li><span id="cit120">A. Yada, K. Nagata, Y. Ando, T. Matsumura, S. Ichinoseki and K. Sato, <span class="italic">Chem. Lett.</span>, 2018, <span class="bold">47</span>, 284–287 <a target="_blank" class="DOILink" href="https://doi.org/10.1246/cl.171130" title="DOI Link to resource 10.1246/cl.171130">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXjt1Sru7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit121">J. G. Estrada, D. T. Ahneman, R. P. Sheridan, S. D. Dreher and A. G. Doyle, <span class="italic">Science</span>, 2018, <span class="bold">362</span>, 6416 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.aat8763" title="DOI Link to resource 10.1126/science.aat8763">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30442777%5Buid%5D" title="PubMed Link to resource 30442777">PubMed</a>.</span></li>
      <li><span id="cit122">Z. Fu, X. Li, Z. Wang, Z. Li, X. Liu, X. Wu, J. Zhao, X. Ding, X. Wan, F. Zhong, D. Wang, X. Luo, K. Chen, H. Liu, J. Wang, H. Jiang and M. Zheng, <span class="italic">Org. Chem. Front.</span>, 2020, <span class="bold">7</span>, 2269–2277 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0QO00544D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0QO00544D">RSC</a>.</span></li>
      <li><span id="cit123">F. Sandfort, F. Strieth-Kalthoff, M. Kühnemund, C. Beecks and F. Glorius, <span class="italic">Chem</span>, 2020, <span class="bold">6</span>, 1379–1390 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOis7zP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit124">G. Skoraczyñski, P. DIttwald, B. Miasojedow, S. Szymkuc, E. P. Gajewska, B. A. Grzybowski and A. Gambin, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 1–9 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-016-0028-x" title="DOI Link to resource 10.1038/s41598-016-0028-x">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28127051%5Buid%5D" title="PubMed Link to resource 28127051">PubMed</a>.</span></li>
      <li><span id="cit125">A. A. Lapkin, A. Voutchkova and P. Anastas, <span class="italic">Chem. Eng. Process. Process Intensif.</span>, 2011, <span class="bold">50</span>, 1027–1034 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cep.2011.06.005" title="DOI Link to resource 10.1016/j.cep.2011.06.005">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhsFKrs7rE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit126">G. Marcou, J. Aires De Sousa, D. A. R. S. Latino, A. De Luca, D. Horvath, V. Rietsch and A. Varnek, <span class="italic">J. Chem. Inf. Model.</span>, 2015, <span class="bold">55</span>, 239–250 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ci500698a" title="DOI Link to resource 10.1021/ci500698a">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXnvFGnsQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25588070%5Buid%5D" title="PubMed Link to resource 25588070">PubMed</a>.</span></li>
      <li><span id="cit127">A. I. Lin, T. I. Madzhidov, O. Klimchuk, R. I. Nugmanov, I. S. Antipin and A. Varnek, <span class="italic">J. Chem. Inf. Model.</span>, 2016, <span class="bold">56</span>, 2140–2148 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jcim.6b00319" title="DOI Link to resource 10.1021/acs.jcim.6b00319">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhslemsL%252FM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27783508%5Buid%5D" title="PubMed Link to resource 27783508">PubMed</a>.</span></li>
      <li><span id="cit128">H. Gao, T. J. Struble, C. W. Coley, Y. Wang, W. H. Green and K. F. Jensen, <span class="italic">ACS Cent. Sci.</span>, 2018, <span class="bold">4</span>, 1465–1476 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.8b00357" title="DOI Link to resource 10.1021/acscentsci.8b00357">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXit1aru77J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30555898%5Buid%5D" title="PubMed Link to resource 30555898">PubMed</a>.</span></li>
      <li><span id="cit129">M. H. S. Segler and M. P. Waller, <span class="italic">Chem. – Eur. J.</span>, 2017, <span class="bold">23</span>, 6118–6128 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/chem.201604556" title="DOI Link to resource 10.1002/chem.201604556">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXislersw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27862477%5Buid%5D" title="PubMed Link to resource 27862477">PubMed</a>.</span></li>
      <li><span id="cit130">B. A. Grzybowski, S. Szymkuć, E. P. Gajewska, K. Molga, P. Dittwald, A. Wołos and T. Klucznik, <span class="italic">Chem</span>, 2018, <span class="bold">4</span>, 390–398 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXktlyitLc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit131">W. L. Chen, D. Z. Chen and K. T. Taylor, <span class="italic">Wiley Interdiscip. Rev.: Comput. Mol. Sci.</span>, 2013, <span class="bold">3</span>, 560–593 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yksrbM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit132">G. A. Preciat Gonzalez, L. R. P. El Assal, A. Noronha, I. Thiele, H. S. Haraldsdóttir and R. M. T. Fleming, <span class="italic">J. Cheminform.</span>, 2017, <span class="bold">9</span>, 1–15 <a target="_blank" class="DOILink" href="https://doi.org/10.1186/s13321-016-0187-6" title="DOI Link to resource 10.1186/s13321-016-0187-6">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28316652%5Buid%5D" title="PubMed Link to resource 28316652">PubMed</a>.</span></li>
      <li><span id="cit133">W. Jaworski, S. Szymkuć, B. Mikulak-Klucznik, K. Piecuch, T. Klucznik, M. Kaźmierowski, J. Rydzewski, A. Gambin and B. A. Grzybowski, <span class="italic">Nat. Commun.</span>, 2019, <span class="bold">10</span>, 1–11 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-018-07882-8" title="DOI Link to resource 10.1038/s41467-018-07882-8">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXoslGmu7Y%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30602773%5Buid%5D" title="PubMed Link to resource 30602773">PubMed</a>.</span></li>
      <li><span id="cit134">P. Schwaller, B. Hoover, J.-L. Reymond, H. Strobelt and T. Laino, <span class="italic">Sci. Adv.</span>, 2021, <span class="bold">7</span>, eabe4166 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/sciadv.abe4166" title="DOI Link to resource 10.1126/sciadv.abe4166">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33827815%5Buid%5D" title="PubMed Link to resource 33827815">PubMed</a>.</span></li>
      <li><span id="cit135">Y. Lecun, Y. Bengio and G. Hinton, <span class="italic">Nature</span>, 2015, <span class="bold">521</span>, 436–444 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature14539" title="DOI Link to resource 10.1038/nature14539">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXht1WlurzP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26017442%5Buid%5D" title="PubMed Link to resource 26017442">PubMed</a>.</span></li>
      <li><span id="cit136">E. Rieffel and W. Polak, <span class="italic">ACM Comput. Surv.</span>, 2000, <span class="bold">32</span>, 300–335 <a target="_blank" class="DOILink" href="https://doi.org/10.1145/367701.367709" title="DOI Link to resource 10.1145/367701.367709">CrossRef</a>.</span></li>
      <li><span id="cit137">A. Ajagekar and F. You, <span class="italic">Energy</span>, 2019, <span class="bold">179</span>, 76–89 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.energy.2019.04.186" title="DOI Link to resource 10.1016/j.energy.2019.04.186">CrossRef</a>.</span></li>
      <li><span id="cit138">J. Preskill, <span class="italic">Quantum</span>, 2018, <span class="bold">2</span>, 79 <a target="_blank" class="DOILink" href="https://doi.org/10.22331/q-2018-08-06-79" title="DOI Link to resource 10.22331/q-2018-08-06-79">CrossRef</a>.</span></li>
      <li><span id="cit139">S. Rangarajan, A. Bhan and P. Daoutidis, <span class="italic">Ind. Eng. Chem. Res.</span>, 2010, <span class="bold">49</span>, 10459–10470 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ie100546t" title="DOI Link to resource 10.1021/ie100546t">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXmvFKmtLw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit140">S. Rangarajan, A. Bhan and P. Daoutidis, <span class="italic">Comput. Chem. Eng.</span>, 2012, <span class="bold">45</span>, 114–123 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2012.06.008" title="DOI Link to resource 10.1016/j.compchemeng.2012.06.008">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhtVKnsL%252FP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit141">W. A. Marvin, S. Rangarajan and P. Daoutidis, <span class="italic">Energy Fuels</span>, 2013, <span class="bold">27</span>, 3585–3594 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ef4003318" title="DOI Link to resource 10.1021/ef4003318">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXmtVygsrc%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit142">J. M. Weber, C. P. Lindenmeyer, P. Liò and A. A. Lapkin, <span class="italic">Int. J. Sustainable High. Educ.</span>, 2021, <span class="bold">22</span>, 25–41 <a target="_blank" class="DOILink" href="https://doi.org/10.1108/IJSHE-06-2020-0209" title="DOI Link to resource 10.1108/IJSHE-06-2020-0209">CrossRef</a>.</span></li>
      <li><span id="cit143">K. Vladimirova and D. Le Blanc, <span class="italic">Sustainable Dev.</span>, 2016, <span class="bold">24</span>, 254–271 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/sd.1626" title="DOI Link to resource 10.1002/sd.1626">CrossRef</a>.</span></li>
      <li><span id="cit144">
          K. Vladimirova and D. Le Blanc, How well are the links between education and other sustainable development goals covered in UN flagship reports? A contribution to the study of the science-policy interface on education in the UN system, New York, USA, 2015.</span></li>
      <li><span id="cit145">R. Costanza, L. Daly, L. Fioramonti, E. Giovannini, I. Kubiszewski, L. F. Mortensen, K. E. Pickett, K. V. Ragnarsdottir, R. De Vogli and R. Wilkinson, <span class="italic">Ecol. Econ.</span>, 2016, <span class="bold">130</span>, 350–355 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ecolecon.2016.07.009" title="DOI Link to resource 10.1016/j.ecolecon.2016.07.009">CrossRef</a>.</span></li>
      <li><span id="cit146">D. Le Blanc, <span class="italic">Sustainable Dev.</span>, 2015, <span class="bold">23</span>, 176–187 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/sd.1582" title="DOI Link to resource 10.1002/sd.1582">CrossRef</a>.</span></li>
      <li><span id="cit147">
          R. A. Sheldon, I. Arends and U. Hanefeld, <span class="italic">Green chemistry and catalysis</span>, John Wiley &amp; Sons,  2007 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Green%20chemistry%20and%20catalysis%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202007%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit148">R. A. Sheldon, <span class="italic">Green Chem.</span>, 2014, <span class="bold">16</span>, 950–963 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3GC41935E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3GC41935E">RSC</a>.</span></li>
      <li><span id="cit149">M. A. Gonzalez and R. L. Smith, <span class="italic">Environ. Prog.</span>, 2003, <span class="bold">22</span>, 269–276 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ep.670220415" title="DOI Link to resource 10.1002/ep.670220415">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXhtlWit70%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit150">C. Jiménez-González, D. J. C. Constable and C. S. Ponder, <span class="italic">Chem. Soc. Rev.</span>, 2012, <span class="bold">41</span>, 1485–1498 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C1CS15215G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C1CS15215G">RSC</a>.</span></li>
      <li><span id="cit151">P. Marion, B. Bernela, A. Piccirilli, B. Estrine, N. Patouillard, J. Guilbot and F. Jérôme, <span class="italic">Green Chem.</span>, 2017, <span class="bold">19</span>, 4973–4989 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C7GC02006F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C7GC02006F">RSC</a>.</span></li>
      <li><span id="cit152">
          A. A. Lapkin, <span class="italic">Handbook of Green Chemistry</span>,  2010, pp. 1–16 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Handbook%20of%20Green%20Chemistry%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202010%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit153">
          K. Kümmerer, A.-K. Amsel, D. Bartkowiak, A. Bazzanella, C. Blum and C. Cinquemani, Key Characteristics of Sustainable Chemistry, 2021.</span></li>
      <li><span id="cit154">
          Y. Merali and P. Allen, <span class="italic">SAGE Handbook of Complexity and Management</span>,  2011, pp. 31–52 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=SAGE%20Handbook%20of%20Complexity%20and%20Management%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202011%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit155">E. Nabavi, K. A. Daniell and H. Najafi, <span class="italic">J. Clean. Prod.</span>, 2017, <span class="bold">140</span>, 312–323 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jclepro.2016.03.032" title="DOI Link to resource 10.1016/j.jclepro.2016.03.032">CrossRef</a>.</span></li>
      <li><span id="cit156">
          J. D. Sternam, <span class="italic">System Dynamics: Systems Thinking and Modeling for a Complex World</span>,  2002 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=System%20Dynamics:%20Systems%20Thinking%20and%20Modeling%20for%20a%20Complex%20World%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202002%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit157">C. Benoît, G. A. Norris, S. Valdivia, A. Ciroth, A. Moberg, U. Bos, S. Prakash, C. Ugaya and T. Beck, <span class="italic">Int. J. Life Cycle Assess</span>, 2010, <span class="bold">15</span>, 156–163 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11367-009-0147-8" title="DOI Link to resource 10.1007/s11367-009-0147-8">CrossRef</a>.</span></li>
      <li><span id="cit158">A. Jørgensen, A. Le Bocq, L. Nazarkina and M. Hauschild, <span class="italic">Int. J. Life Cycle Assess</span>, 2008, <span class="bold">13</span>, 96–103 <a target="_blank" class="DOILink" href="https://doi.org/10.1065/lca2007.11.367" title="DOI Link to resource 10.1065/lca2007.11.367">CrossRef</a>.</span></li>
      <li><span id="cit159">A.-M. Tillman, T. Ekvall, H. Baumann and T. Rydberg, <span class="italic">J. Clean. Prod.</span>, 1994, <span class="bold">2</span>, 21–29 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0959-6526(94)90021-3" title="DOI Link to resource 10.1016/0959-6526(94)90021-3">CrossRef</a>.</span></li>
      <li><span id="cit160">H. Ny, J. P. MacDonald, G. Broman, R. Yamamoto and K. H. Robèrt, <span class="italic">J. Ind. Ecol.</span>, 2006, <span class="bold">10</span>, 61–77 <a target="_blank" class="DOILink" href="https://doi.org/10.1162/108819806775545349" title="DOI Link to resource 10.1162/108819806775545349">CrossRef</a>.</span></li>
      <li><span id="cit161">W. R. Stahel, <span class="italic">Nature</span>, 2016, <span class="bold">531</span>, 435–438 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/531435a" title="DOI Link to resource 10.1038/531435a">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XkvVCkurw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27008952%5Buid%5D" title="PubMed Link to resource 27008952">PubMed</a>.</span></li>
      <li><span id="cit162">J. Kirchherr, D. Reike and M. Hekkert, <span class="italic">Resour., Conserv. Recycl.</span>, 2017, <span class="bold">127</span>, 221–232 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.resconrec.2017.09.005" title="DOI Link to resource 10.1016/j.resconrec.2017.09.005">CrossRef</a>.</span></li>
      <li><span id="cit163">
          B. Dittrich-Krämer, C. Bunte, A. Kircherer and T. Schaffrannek, <span class="italic">Global Goals Yearbook</span>,  2018 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Global%20Goals%20Yearbook%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit164">
          Ellen MacArthur Foundation, Circularity Indicators An approach to measuring circularity, 2019.</span></li>
      <li><span id="cit165">F. Razza, C. Briani, T. Breton and D. Marazza, <span class="italic">Resour., Conserv. Recycl.</span>, 2020, <span class="bold">159</span>, 104753 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.resconrec.2020.104753" title="DOI Link to resource 10.1016/j.resconrec.2020.104753">CrossRef</a>.</span></li>
      <li><span id="cit166">K. Lokesh, A. S. Matharu, I. K. Kookos, D. Ladakis, A. Koutinas, P. Morone and J. Clark, <span class="italic">Green Chem.</span>, 2020, <span class="bold">22</span>, 803–813 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9GC02992C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9GC02992C">RSC</a>.</span></li>
      <li><span id="cit167">P. Karka, S. Papadokonstantakis and A. Kokossis, <span class="italic">Int. J. Life Cycle Assess</span>, 2019, <span class="bold">24</span>, 1675–1700 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11367-019-01591-0" title="DOI Link to resource 10.1007/s11367-019-01591-0">CrossRef</a>.</span></li>
      <li><span id="cit168">P. Karka, S. Papadokonstantakis, K. Hungerbühler and A. Kokossis, <span class="italic">Comput. - Aided Chem. Eng.</span>, 2014, <span class="bold">34</span>, 543–548 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitFKqtrzE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit169">G. Wernet, S. Papadokonstantakis, S. Hellweg and K. Hungerbühler, <span class="italic">Green Chem.</span>, 2009, <span class="bold">11</span>, 1826–1831 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B905558D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B905558D">RSC</a>.</span></li>
      <li><span id="cit170">R. Calvo-Serrano, M. González-Miquel, S. Papadokonstantakis and G. Guillén-Gosálbez, <span class="italic">Comput. Chem. Eng.</span>, 2018, <span class="bold">108</span>, 179–193 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2017.09.010" title="DOI Link to resource 10.1016/j.compchemeng.2017.09.010">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsFCkurrM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit171">P. Karka, S. Papadokonstantakis and A. Kokossis, <span class="italic">Comput. - Aided Chem. Eng.</span>, 2019, <span class="bold">46</span>, 97–102 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtVeksL3F" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit172">R. G. Hunt, T. K. Boguski, K. Weitz and A. Sharma, <span class="italic">Int. J. Life Cycle Assess.</span>, 1998, <span class="bold">3</span>, 36–42 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/BF02978450" title="DOI Link to resource 10.1007/BF02978450">CrossRef</a>.</span></li>
      <li><span id="cit173">A. Marvuglia, M. Kanevski and E. Benetto, <span class="italic">Environ. Int.</span>, 2015, <span class="bold">83</span>, 72–85 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.envint.2015.05.011" title="DOI Link to resource 10.1016/j.envint.2015.05.011">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtFSqtL%252FF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26101085%5Buid%5D" title="PubMed Link to resource 26101085">PubMed</a>.</span></li>
      <li><span id="cit174">G. Wernet, S. Hellweg, U. Fischer, S. Papadokonstantakis and K. Hungerbühler, <span class="italic">Environ. Sci. Technol.</span>, 2008, <span class="bold">42</span>, 6717–6722 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/es7022362" title="DOI Link to resource 10.1021/es7022362">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXpt1yns74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18800554%5Buid%5D" title="PubMed Link to resource 18800554">PubMed</a>.</span></li>
      <li><span id="cit175">R. Song, A. A. Keller and S. Suh, <span class="italic">Environ. Sci. Technol.</span>, 2017, <span class="bold">51</span>, 10777–10785 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.est.7b02862" title="DOI Link to resource 10.1021/acs.est.7b02862">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtlaktrbM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28809480%5Buid%5D" title="PubMed Link to resource 28809480">PubMed</a>.</span></li>
      <li><span id="cit176">J. Kleinekorte, L. Kröger, K. Leonhard and A. Bardow, <span class="italic">Comput. - Aided Chem. Eng.</span>, 2019, <span class="bold">46</span>, 1447–1452 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Comput.%20-%20Aided%20Chem.%20Eng.%5BJour%5D%20AND%2046%5Bvolume%5D%20AND%201447%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit177">B. M. Trost, <span class="italic">Science</span>, 1991, <span class="bold">254</span>, 1471–1477 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/science.1962206" title="DOI Link to resource 10.1126/science.1962206">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK38XlvVegsQ%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=1962206%5Buid%5D" title="PubMed Link to resource 1962206">PubMed</a>.</span></li>
      <li><span id="cit178">R. A. Sheldon, <span class="italic">Green Chem.</span>, 2007, <span class="bold">9</span>, 1273–1283 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=B713736M&amp;newsite=1" title="Link to RSC resource DOI:10.1039/B713736M">RSC</a>.</span></li>
      <li><span id="cit179">J. Andraos, <span class="italic">Org. Process Res. Dev.</span>, 2005, <span class="bold">9</span>, 149–163 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/op049803n" title="DOI Link to resource 10.1021/op049803n">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXht1yit7Y%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit180">J. Andraos, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2016, <span class="bold">4</span>, 1917–1933 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssuschemeng.5b01554" title="DOI Link to resource 10.1021/acssuschemeng.5b01554">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xjs1Cgtro%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit181">M. G. T. C. Ribeiro, D. A. Costa and A. A. S. C. Machado, <span class="italic">Green Chem. Lett. Rev.</span>, 2010, <span class="bold">3</span>, 149–159 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/17518251003623376" title="DOI Link to resource 10.1080/17518251003623376">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhtlSntrjE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit182">R. C. C. Duarte, M. G. T. C. Ribeiro and A. A. S. C. Machado, <span class="italic">J. Chem. Educ.</span>, 2015, <span class="bold">92</span>, 1024–1034 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ed5004096" title="DOI Link to resource 10.1021/ed5004096">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXktFamsL0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit183">S. M. Mercer, J. Andraos and P. G. Jessop, <span class="italic">J. Chem. Educ.</span>, 2012, <span class="bold">89</span>, 215–220 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ed200249v" title="DOI Link to resource 10.1021/ed200249v">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXhsFOjtrvO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit184">J. Andraos, M. L. Mastronardi, L. B. Hoch and A. Hent, <span class="italic">ACS Sustainable Chem. Eng.</span>, 2016, <span class="bold">4</span>, 1934–1945 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssuschemeng.5b01555" title="DOI Link to resource 10.1021/acssuschemeng.5b01555">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XitFeru7w%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit185">J. Andraos and A. Hent, <span class="italic">J. Chem. Educ.</span>, 2015, <span class="bold">92</span>, 1820–1830 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jchemed.5b00058" title="DOI Link to resource 10.1021/acs.jchemed.5b00058">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2iu7bF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit186">J. Andraos, <span class="italic">Org. Process Res. Dev.</span>, 2006, <span class="bold">10</span>, 212–240 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/op0501904" title="DOI Link to resource 10.1021/op0501904">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XhtlCnu78%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit187">
          J. Szargut, D. Morris and F. Steward, <span class="italic">Energy analysis of thermal chemical, and metallurgical processes</span>, Hemisphere Publishing, New York, USA,  1988 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Energy%20analysis%20of%20thermal%20chemical,%20and%20metallurgical%20processes%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201988%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit188">
          A. Bejan, G. Tsatsaronis and M. J. Moran, <span class="italic">Thermal design and optimization</span>, John Wiley &amp; Sons, New York,  1995 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Thermal%20design%20and%20optimization%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201995%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit189">K. Kaygusuz and S. Bilgen, <span class="italic">Energy Sources, Part A</span>, 2009, <span class="bold">31</span>, 287–298 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/15567030701715401" title="DOI Link to resource 10.1080/15567030701715401">CrossRef</a>.</span></li>
      <li><span id="cit190">J. C. Romero and P. Linares, <span class="italic">Renewable Sustainable. Energy Rev.</span>, 2014, <span class="bold">33</span>, 427–442 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.rser.2014.02.012" title="DOI Link to resource 10.1016/j.rser.2014.02.012">CrossRef</a>.</span></li>
      <li><span id="cit191">Y. Ao, L. Gunnewiek and M. A. Rosen, <span class="italic">Int. J. Green Energy</span>, 2008, <span class="bold">5</span>, 87–104 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/15435070701839439" title="DOI Link to resource 10.1080/15435070701839439">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXisFCks7o%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit192">G. Tsatsaronis, <span class="italic">Chem. Eng. Technol.</span>, 1996, <span class="bold">19</span>, 163–169 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ceat.270190210" title="DOI Link to resource 10.1002/ceat.270190210">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK28XjtFGjtrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit193">J. Dewulf, H. Van Langenhove, B. Muys, S. Bruers, B. R. Bakshi, G. F. Grubb, D. M. Paulus and E. Sciubba, <span class="italic">Environ. Sci. Technol.</span>, 2008, <span class="bold">42</span>, 2221–2232 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/es071719a" title="DOI Link to resource 10.1021/es071719a">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1cXisVCnsbY%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18504947%5Buid%5D" title="PubMed Link to resource 18504947">PubMed</a>.</span></li>
      <li><span id="cit194">S. Bilgen, S. Keleş and K. Kaygusuz, <span class="italic">Energy</span>, 2012, <span class="bold">41</span>, 380–385 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.energy.2012.03.001" title="DOI Link to resource 10.1016/j.energy.2012.03.001">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XlvFOgu74%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit195">G. Song, L. Shen and J. Xiao, <span class="italic">Ind. Eng. Chem. Res.</span>, 2011, <span class="bold">50</span>, 9758–9766 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ie200534n" title="DOI Link to resource 10.1021/ie200534n">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXpt1Cjtrg%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit196">Y. Zhang, W. Zhao, B. Li, H. Zhang, B. Jiang and C. Ke, <span class="italic">Energy</span>, 2016, <span class="bold">106</span>, 400–407 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.energy.2016.03.040" title="DOI Link to resource 10.1016/j.energy.2016.03.040">CrossRef</a>.</span></li>
      <li><span id="cit197">G. Song, J. Xiao, H. Zhao and L. Shen, <span class="italic">Energy</span>, 2012, <span class="bold">40</span>, 164–173 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.energy.2012.02.016" title="DOI Link to resource 10.1016/j.energy.2012.02.016">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XkvVOjsr4%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit198">J. H. Shieh and L. T. Fan, <span class="italic">Energy Sources</span>, 1982, <span class="bold">6</span>, 1–46 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/00908318208946020" title="DOI Link to resource 10.1080/00908318208946020">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaL38Xhs1Gmsro%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit199">S. Sharifian, M. Madadkhani, M. Rahimi, M. Mir and A. Baghban, <span class="italic">Pet. Sci. Technol.</span>, 2019, <span class="bold">37</span>, 2174–2181 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/10916466.2018.1496100" title="DOI Link to resource 10.1080/10916466.2018.1496100">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXht1Oqtb%252FF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit200">F. Gharagheizi and M. Mehrpooya, <span class="italic">Energy Convers. Manage.</span>, 2007, <span class="bold">48</span>, 2453–2460 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.enconman.2007.04.005" title="DOI Link to resource 10.1016/j.enconman.2007.04.005">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXosVSlurY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit201">Y. W. Huang, M. Q. Chen, Y. Li and J. Guo, <span class="italic">Energy</span>, 2016, <span class="bold">114</span>, 1164–1175 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.energy.2016.08.090" title="DOI Link to resource 10.1016/j.energy.2016.08.090">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsV2qu7%252FM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit202">M. Mir, M. Kamyab, M. Janghorban Lariche, R. Razavi and A. Baghban, <span class="italic">Pet. Sci. Technol.</span>, 2018, <span class="bold">36</span>, 1022–1029 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/10916466.2018.1458123" title="DOI Link to resource 10.1080/10916466.2018.1458123">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXos1Ontr8%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit203">F. Gharagheizi, P. Ilani-Kashkouli and R. C. Hedden, <span class="italic">Energy</span>, 2018, <span class="bold">158</span>, 924–935 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.energy.2018.05.186" title="DOI Link to resource 10.1016/j.energy.2018.05.186">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtFyntL%252FE" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit204">R. Haghbakhsh and S. Raeissi, <span class="italic">Fluid Phase Equilib.</span>, 2020, <span class="bold">507</span>, 112397 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.fluid.2019.112397" title="DOI Link to resource 10.1016/j.fluid.2019.112397">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXit1Kqt7fF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit205">R. Rivero and M. Garfias, <span class="italic">Energy</span>, 2006, <span class="bold">31</span>, 3310–3326 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.energy.2006.03.020" title="DOI Link to resource 10.1016/j.energy.2006.03.020">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XptlSksLk%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit206">A. M. Schweidtmann, J. G. Rittig, A. König, M. Grohe, A. Mitsos and M. Dahmen, <span class="italic">Energy Fuels</span>, 2020, <span class="bold">34</span>, 11395–11407 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.energyfuels.0c01533" title="DOI Link to resource 10.1021/acs.energyfuels.0c01533">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOisrjM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit207">C. W. Coley, W. Jin, L. Rogers, T. F. Jamison, T. S. Jaakkola, W. H. Green, R. Barzilay and K. F. Jensen, <span class="italic">Chem. Sci.</span>, 2019, <span class="bold">10</span>, 370–377 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8SC04228D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8SC04228D">RSC</a>.</span></li>
      <li><span id="cit208">
          D. R. Lanning, G. K. Harrell and J. Wang, Proc. 2014 ACM Southeast Reg. Conf., 2014, pp. 1–3.</span></li>
      <li><span id="cit209">H. Mehta, P. Kanani and P. Lande, <span class="italic">Int. J. Comput. Appl.</span>, 2019, <span class="bold">178</span>, 41–46 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Int.%20J.%20Comput.%20Appl.%5BJour%5D%20AND%20178%5Bvolume%5D%20AND%2041%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit210">
          M. Segler, M. Preuß and M. P. Waller, 2017, arXiv:1702.00020.</span></li>
      <li><span id="cit211">
          A. Kishimoto, B. Buesser, B. Chen and A. Botea, <span class="italic">33rd Conference on Neural Information Processing Systems</span>,  2019 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=33rd%20Conference%20on%20Neural%20Information%20Processing%20Systems%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit212">C. W. Coley, W. H. Green and K. F. Jensen, <span class="italic">Acc. Chem. Res.</span>, 2018, <span class="bold">51</span>, 1281–1289 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.accounts.8b00087" title="DOI Link to resource 10.1021/acs.accounts.8b00087">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXosFKhsb0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29715002%5Buid%5D" title="PubMed Link to resource 29715002">PubMed</a>.</span></li>
      <li><span id="cit213">K. Neumann, C. Schwindt and N. Trautmann, <span class="italic">Or Spectr.</span>, 2002, <span class="bold">24</span>, 251–279 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s00291-002-0100-8" title="DOI Link to resource 10.1007/s00291-002-0100-8">CrossRef</a>.</span></li>
      <li><span id="cit214">A. S. Manne, <span class="italic">Oper. Res.</span>, 1959, <span class="bold">8</span>, 219–223 <a target="_blank" class="DOILink" href="https://doi.org/10.1287/opre.8.2.219" title="DOI Link to resource 10.1287/opre.8.2.219">CrossRef</a>.</span></li>
      <li><span id="cit215">G. Dobson and R. S. Nambimadom, <span class="italic">Oper. Res.</span>, 2001, <span class="bold">49</span>, 52–65 <a target="_blank" class="DOILink" href="https://doi.org/10.1287/opre.49.1.52.11189" title="DOI Link to resource 10.1287/opre.49.1.52.11189">CrossRef</a>.</span></li>
      <li><span id="cit216">E. J. Corey, <span class="italic">Angew. Chem., Int. Ed. Engl.</span>, 1991, <span class="bold">30</span>, 455–465 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/anie.199104553" title="DOI Link to resource 10.1002/anie.199104553">CrossRef</a>.</span></li>
      <li><span id="cit217">
          E. J. Corey, <span class="italic">The logic of chemical synthesis</span>,  1991 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=The%20logic%20of%20chemical%20synthesis%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201991%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit218">M. Koch, T. Duigou and J. L. Faulon, <span class="italic">ACS Synth. Biol.</span>, 2020, <span class="bold">9</span>, 157–168 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acssynbio.9b00447" title="DOI Link to resource 10.1021/acssynbio.9b00447">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisVWltbvJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31841626%5Buid%5D" title="PubMed Link to resource 31841626">PubMed</a>.</span></li>
      <li><span id="cit219">J. S. Schreck, C. W. Coley and K. J. M. Bishop, <span class="italic">ACS Cent. Sci.</span>, 2019, <span class="bold">5</span>, 970–981 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.9b00055" title="DOI Link to resource 10.1021/acscentsci.9b00055">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtVKiu7jP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31263756%5Buid%5D" title="PubMed Link to resource 31263756">PubMed</a>.</span></li>
      <li><span id="cit220">X. Wang, Y. Qian, H. Gao, C. W. Coley, Y. Mo, R. Barzilay and K. F. Jensen, <span class="italic">Chem. Sci.</span>, 2020, <span class="bold">11</span>, 10959–10972 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0SC04184J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0SC04184J">RSC</a>.</span></li>
      <li><span id="cit221">C. W. Coley, L. Rogers, W. H. Green and K. F. Jensen, <span class="italic">ACS Cent. Sci.</span>, 2017, <span class="bold">3</span>, 1237–1245 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acscentsci.7b00355" title="DOI Link to resource 10.1021/acscentsci.7b00355">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSmurfN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29296663%5Buid%5D" title="PubMed Link to resource 29296663">PubMed</a>.</span></li>
      <li><span id="cit222">M. H. S. Segler, M. Preuss and M. P. Waller, <span class="italic">Nature</span>, 2018, <span class="bold">555</span>, 604–610 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature25978" title="DOI Link to resource 10.1038/nature25978">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXmsVGqt7c%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29595767%5Buid%5D" title="PubMed Link to resource 29595767">PubMed</a>.</span></li>
      <li><span id="cit223">V. Pham and M. El-Halwagi, <span class="italic">AIChE J.</span>, 2012, <span class="bold">58</span>, 1212–1221 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aic.12640" title="DOI Link to resource 10.1002/aic.12640">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXnslSmtbw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit224">J. Jiménez-Luna, F. Grisoni and G. Schneider, <span class="italic">Nat. Mach. Intell.</span>, 2020, <span class="bold">2</span>, 573–584 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s42256-020-00236-4" title="DOI Link to resource 10.1038/s42256-020-00236-4">CrossRef</a>.</span></li>
      <li><span id="cit225">L. S. Dias and M. G. Ierapetritou, <span class="italic">Comput. Chem. Eng.</span>, 2017, <span class="bold">106</span>, 826–835 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2017.02.006" title="DOI Link to resource 10.1016/j.compchemeng.2017.02.006">CrossRef</a>.</span></li>
      <li><span id="cit226">L. Kong, S. M. Sen, C. A. Henao, J. A. Dumesic and C. T. Maravelias, <span class="italic">Comput. Chem. Eng.</span>, 2016, <span class="bold">91</span>, 68–84 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2016.02.013" title="DOI Link to resource 10.1016/j.compchemeng.2016.02.013">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28Xjs1eltb0%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit227">A. Giuliano, R. Cerulli, M. Poletto, G. Raiconi and D. Barletta, <span class="italic">Ind. Eng. Chem. Res.</span>, 2016, <span class="bold">55</span>, 10699–10717 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.iecr.6b01454" title="DOI Link to resource 10.1021/acs.iecr.6b01454">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhsFCmsrbP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit228">I. Grossmann, <span class="italic">Optim. Eng.</span>, 2002, <span class="bold">3</span>, 227–252 <a target="_blank" class="DOILink" href="https://doi.org/10.1023/A:1021039126272" title="DOI Link to resource 10.1023/A:1021039126272">CrossRef</a>.</span></li>
      <li><span id="cit229">D. J. Garcia and F. You, <span class="italic">AIChE J.</span>, 2015, <span class="bold">61</span>, 530–554 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aic.14666" title="DOI Link to resource 10.1002/aic.14666">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWit7nI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit230">V. Andiappan, A. S. Y. Ko, V. W. S. Lau, L. Y. Ng, R. T. L. Ng, N. G. Chemmangattuvalappil and D. K. S. Ng, <span class="italic">AIChE J.</span>, 2015, <span class="bold">61</span>, 132–146 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aic.14616" title="DOI Link to resource 10.1002/aic.14616">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXitFWjs73I" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit231">B. Bao, D. K. S. Ng, D. H. S. Tay, A. Jiménez-Gutiérrez and M. M. El-Halwagi, <span class="italic">Comput. Chem. Eng.</span>, 2011, <span class="bold">35</span>, 1374–1383 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2011.04.013" title="DOI Link to resource 10.1016/j.compchemeng.2011.04.013">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3MXpt1aksrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit232">A. Besler, A. Harwardt and W. Marquardt, <span class="italic">Comput. - Aided Chem. Eng.</span>, 2009, <span class="bold">26</span>, 243–248 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXhtFais7rP" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit233">C. H. Schilling and B. O. Palsson, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 1998, <span class="bold">95</span>, 4193–4198 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.95.8.4193" title="DOI Link to resource 10.1073/pnas.95.8.4193">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADyaK1cXis1Oht7g%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=9539712%5Buid%5D" title="PubMed Link to resource 9539712">PubMed</a>.</span></li>
      <li><span id="cit234">
          N. Karmarkar, Proc. sizteenth Annu. ACM Symp. Theory Comput., 1984, pp. 302–311.</span></li>
      <li><span id="cit235">
          P. Gács and L. Lovász, <span class="italic">Mathematical Programming at Oberwolfach</span>, Springer Berlin, Heidelberg,  1981, pp. 61–68 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mathematical%20Programming%20at%20Oberwolfach%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201981%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit236">M. Hechinger, A. Voll and W. Marquardt, <span class="italic">Comput. Chem. Eng.</span>, 2010, <span class="bold">34</span>, 1909–1918 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2010.07.035" title="DOI Link to resource 10.1016/j.compchemeng.2010.07.035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXht12ntrvN" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit237">M. Dahmen and W. Marquardt, <span class="italic">Energy Fuels</span>, 2017, <span class="bold">31</span>, 4096–4121 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.energyfuels.7b00118" title="DOI Link to resource 10.1021/acs.energyfuels.7b00118">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXjsVOnurs%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit238">M. Tawarmalani and N. V. Sahinidis, <span class="italic">Math. Program., Ser. B</span>, 2005, <span class="bold">103</span>, 225–249 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10107-005-0581-8" title="DOI Link to resource 10.1007/s10107-005-0581-8">CrossRef</a>.</span></li>
      <li><span id="cit239">M. R. Kılınç and N. V. Sahinidis, <span class="italic">Optim. Methods Software</span>, 2018, <span class="bold">33</span>, 540–562 <a target="_blank" class="DOILink" href="https://doi.org/10.1080/10556788.2017.1350178" title="DOI Link to resource 10.1080/10556788.2017.1350178">CrossRef</a>.</span></li>
      <li><span id="cit240">
          D. Bongartz, J. Najman, S. Sass and A. Mitsos, MAiNGO: McCormick based algorithm for mixes integer nonlinear global optimization, Technical report, 2018.</span></li>
      <li><span id="cit241">
          M. R. Bussieck and A. Meeraus, <span class="italic">Modeling languages in mathematical optimization</span>, Springer, Boston, MA,  2004, pp. 137–157 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Modeling%20languages%20in%20mathematical%20optimization%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202004%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit242">A. König, L. Neidhardt, J. Viell, A. Mitsos and M. Dahmen, <span class="italic">Comput. Chem. Eng.</span>, 2020, <span class="bold">134</span>, 106712 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2019.106712" title="DOI Link to resource 10.1016/j.compchemeng.2019.106712">CrossRef</a>.</span></li>
      <li><span id="cit243">
          C. A. Petri, PhD thesis, Technische Hochschule Darmstadt,  1962.</span></li>
      <li><span id="cit244">M. Ghaeli, P. A. Bahri, P. Lee and T. Gu, <span class="italic">Comput. Chem. Eng.</span>, 2005, <span class="bold">29</span>, 249–259 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2004.08.025" title="DOI Link to resource 10.1016/j.compchemeng.2004.08.025">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXhtVykt7fI" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit245">J. L. Peterson, <span class="italic">ACM Comput. Surv.</span>, 1977, <span class="bold">9</span>, 223–252 <a target="_blank" class="DOILink" href="https://doi.org/10.1145/356698.356702" title="DOI Link to resource 10.1145/356698.356702">CrossRef</a>.</span></li>
      <li><span id="cit246">E. C. Yamalidou and J. C. Kantor, <span class="italic">Comput. Chem. Eng.</span>, 1991, <span class="bold">15</span>, 503–519 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/0098-1354(91)85029-T" title="DOI Link to resource 10.1016/0098-1354(91)85029-T">CrossRef</a>.</span></li>
      <li><span id="cit247">I. Koch, <span class="italic">Mol. Inform.</span>, 2010, <span class="bold">29</span>, 838–843 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/minf.201000086" title="DOI Link to resource 10.1002/minf.201000086">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2rtr3F" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27464348%5Buid%5D" title="PubMed Link to resource 27464348">PubMed</a>.</span></li>
      <li><span id="cit248">C. Chaouiya, <span class="italic">Brief. Bioinform.</span>, 2007, <span class="bold">8</span>, 210–219 <a target="_blank" class="DOILink" href="https://doi.org/10.1093/bib/bbm029" title="DOI Link to resource 10.1093/bib/bbm029">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGgsb3J" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=17626066%5Buid%5D" title="PubMed Link to resource 17626066">PubMed</a>.</span></li>
      <li><span id="cit249">T. Gu, P. A. Bahri and G. Cai, <span class="italic">Int. J. Appl. Math. Comput. Sci.</span>, 2003, <span class="bold">13</span>, 527–536 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Int.%20J.%20Appl.%20Math.%20Comput.%20Sci.%5BJour%5D%20AND%2013%5Bvolume%5D%20AND%20527%5Bpage%5D%20and%202003%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit250">L. T. Biegler and I. E. Grossmann, <span class="italic">Comput. Chem. Eng.</span>, 2004, <span class="bold">28</span>, 1169–1192 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2003.11.003" title="DOI Link to resource 10.1016/j.compchemeng.2003.11.003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2cXjslemsrY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit251">D. H. S. Tay, D. K. S. Ng and R. R. Tan, <span class="italic">Environ. Prog. Sustainable Energy</span>, 2013, <span class="bold">32</span>, 384–389 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/ep.10632" title="DOI Link to resource 10.1002/ep.10632">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XivFyhu7s%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit252">R. Morales-Rodriguez, A. S. Meyer, K. V. Gernaey and G. Sin, <span class="italic">Comput. Chem. Eng.</span>, 2012, <span class="bold">42</span>, 115–129 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.compchemeng.2011.12.004" title="DOI Link to resource 10.1016/j.compchemeng.2011.12.004">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XnvVOmtLo%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit253">M. Kasaŝ, Z. Kravanja and Z. N. Pintariĉ, <span class="italic">Comput. - Aided Chem. Eng.</span>, 2011, <span class="bold">29</span>, 407–411 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Comput.%20-%20Aided%20Chem.%20Eng.%5BJour%5D%20AND%2029%5Bvolume%5D%20AND%20407%5Bpage%5D%20and%202011%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit254">M. C. Tang, M. W. S. Chin, K. M. Lim, Y. S. Mun, R. T. L. Ng, D. H. S. Tay and D. K. S. Ng, <span class="italic">Clean Technol. Environ. Policy</span>, 2013, <span class="bold">15</span>, 783–799 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s10098-013-0582-x" title="DOI Link to resource 10.1007/s10098-013-0582-x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SltbnO" title="Link to resource in CAS">CAS</a>.</span></li>
    </ol>
    
    
    
    
    
  <hr/><table><tr><td><h3>Footnotes</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">https://www.cas.org/support/documentation/reactions.</span></span></td></tr><tr><td><span class="sup_ref">‡ <span id="fn2">https://supportcontent.elsevier.com/RightNow%20Next%20Gen/Reaxys/New_RX_FactSheet_Jul_2018.pdf.</span></span></td></tr><tr><td><span class="sup_ref">§ <span id="fn3">https://www.nextmovesoftware.com/pistachio.html.</span></span></td></tr><tr><td><span class="sup_ref">¶ <span id="fn4">https://nextmovesoftware.com/blog/2014/02/27/unleashing-over-a-million-reactions-into-the-wild/.</span></span></td></tr><tr><td><span class="sup_ref">|| <span id="fn5">https://www.deepmatter.io/spresi/.</span></span></td></tr><tr><td><span class="sup_ref">** <span id="fn6">https://figshare.com/articles/dataset/Chemical_reactions_from_US_patents_1976-Sep2016_/5104873.</span></span></td></tr><tr><td><span class="sup_ref">†† <span id="fn7">https://github.com/open-reaction-database.</span></span></td></tr><tr><td><span class="sup_ref">‡‡ <span id="fn8">https://www.kegg.jp/kegg/kegg1a.html.</span></span></td></tr><tr><td><span class="sup_ref">§§ <span id="fn9">https://www.rhea-db.org/statistics.</span></span></td></tr><tr><td><span class="sup_ref">¶¶ <span id="fn10">http://ezcatdb.cbrc.jp/EzCatDB/.</span></span></td></tr><tr><td><span class="sup_ref">|||| <span id="fn11">https://5stardata.info/en/.</span></span></td></tr><tr><td><span class="sup_ref">*** <span id="fn12">https://www.cas.org/support/documentation/cas-databases.</span></span></td></tr><tr><td><span class="sup_ref">††† <span id="fn13">https://www.sigmaaldrich.com/chemistry/chemical-synthesis/synthesis-software.html.</span></span></td></tr><tr><td><span class="sup_ref">‡‡‡ <span id="fn14">http://www.suschem.org/about.</span></span></td></tr><tr><td><span class="sup_ref">§§§ <span id="fn15">https://www.isc3.org/en/about-isc3.html.</span></span></td></tr><tr><td><span class="sup_ref">¶¶¶ <span id="fn16">https://www.basf.com/global/en/who-we-are/strategy/verbund.html.</span></span></td></tr><tr><td><span class="sup_ref">|||||| <span id="fn17">https://www.ellenmacarthurfoundation.org/resources/apply/circulytics-measuring-circularity.</span></span></td></tr><tr><td><span class="sup_ref">**** <span id="fn18">http://www.metzger.chemie.uni-oldenburg.de/eatos/english.html (accessed 25.02.2021).</span></span></td></tr><tr><td><span class="sup_ref">†††† <span id="fn19">https://www.acs.org/content/acs/en/greenchemistry/research-innovation/tools-for-green-chemistry.html (accessed 25.02.2021).</span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2021</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>