<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" xmlns:art="http://www.rsc.org/schema/rscart38" xmlns:epub="http://www.idpf.org/2007/ops" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:rsc="urn:rsc.org" xml:lang="en" lang="en"><head><!--Google Tag Manager--><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-56FZ7G');</script><!--End Google Tag Manager--><title>Artificial intelligence-powered microfluidics for nanomedicine and materials synthesis  - Nanoscale (RSC Publishing) DOI:10.1039/D1NR06195J</title><link rel="canonical" href="https://pubs.rsc.org/en/content/articlehtml/2021/nr/d1nr06195j"/><meta http-equiv="content-type" content="application/xhtml+xml; charset=utf-8"/><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"NRJS-aaa897feae8feeca979","applicationID":"1313546638","transactionName":"M1wANxQFCEcDVU0PWgoWLzUlSyVbDEJcCEEnVgwXFAsKWAdEFgdHEFABDwMMElkO","queueTime":0,"applicationTime":68,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).init={privacy:{cookies_enabled:true},ajax:{deny_list:["bam.nr-data.net"]},distributed_tracing:{enabled:true}};(window.NREUM||(NREUM={})).loader_config={agentID:"1386013924",accountID:"2851366",trustKey:"1029994",xpid:"Vg4CUFVVDhABV1BRAgUBUFcJ",licenseKey:"NRJS-aaa897feae8feeca979",applicationID:"1313546638"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(31),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(32),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError";if(!c.disabled){var l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(14),t(13),"addEventListener"in window&&t(7),c.xhrWrappable&&t(15),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})}},{}],3:[function(t,e,n){var r=t("loader");r.disabled||(r.features.ins=!0)},{}],4:[function(t,e,n){function r(){U++,L=g.hash,this[u]=y.now()}function o(){U--,g.hash!==L&&i(0,!0);var t=y.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+g,e])}function a(t,e){t.on(e,function(){this[e]=y.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,g=w.location,y=t("loader");if(w[v]&&y.xhrWrappable&&!y.disabled){var x=t(11),b=t(12),E=t(9),R=t(7),O=t(14),T=t(8),S=t(15),P=t(10),M=t("ee"),C=M.get("tracer"),N=t(23);t(17),y.features.spa=!0;var L,U=0;M.on(u,r),b.on(p,r),P.on(p,r),M.on(d,o),b.on(l,o),P.on(l,o),M.buffer([u,d,"xhr-resolved"]),R.buffer([u]),O.buffer(["setTimeout"+c,"clearTimeout"+s,u]),S.buffer([u,"new-xhr","send-xhr"+s]),T.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),x.buffer([u]),b.buffer(["propagate",p,l,"executor-err","resolve"+s]),C.buffer([u,"no-"+u]),P.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,m+s),a(T,m+"-done"),a(P,"new-jsonp"),a(P,"jsonp-end"),a(P,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,N(!0)),w[v]("load",i,N(!0)),w[v]("popstate",function(){i(0,U>1)},N(!0))}},{}],5:[function(t,e,n){function r(){var t=new PerformanceObserver(function(t,e){var n=t.getEntries();s(v,[n])});try{t.observe({entryTypes:["resource"]})}catch(e){}}function o(t){if(s(v,[window.performance.getEntriesByType(w)]),window.performance["c"+p])try{window.performance[h](m,o,!1)}catch(t){}else try{window.performance[h]("webkit"+m,o,!1)}catch(t){}}function i(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var a=t("ee"),s=t("handle"),c=t(14),f=t(13),u=t(6),d=t(23),p="learResourceTimings",l="addEventListener",h="removeEventListener",m="resourcetimingbufferfull",v="bstResource",w="resource",g="-start",y="-end",x="fn"+g,b="fn"+y,E="bstTimer",R="pushState",O=t("loader");if(!O.disabled){O.features.stn=!0,t(9),"addEventListener"in window&&t(7);var T=NREUM.o.EV;a.on(x,function(t,e){var n=t[0];n instanceof T&&(this.bstStart=O.now())}),a.on(b,function(t,e){var n=t[0];n instanceof T&&s("bst",[n,e,this.bstStart,O.now()])}),c.on(x,function(t,e,n){this.bstStart=O.now(),this.bstType=n}),c.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),this.bstType])}),f.on(x,function(){this.bstStart=O.now()}),f.on(b,function(t,e){s(E,[e,this.bstStart,O.now(),"requestAnimationFrame"])}),a.on(R+g,function(t){this.time=O.now(),this.startPath=location.pathname+location.hash}),a.on(R+y,function(t){s("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),u()?(s(v,[window.performance.getEntriesByType("resource")]),r()):l in window.performance&&(window.performance["c"+p]?window.performance[l](m,o,d(!1)):window.performance[l]("webkit"+m,o,d(!1))),document[l]("scroll",i,d(!1)),document[l]("keypress",i,d(!1)),document[l]("click",i,d(!1))}}},{}],6:[function(t,e,n){e.exports=function(){return"PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver}},{}],7:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t("wrap-function")(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1];if(null!==n&&("function"==typeof n||"object"==typeof n)){var r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],8:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=i(arguments),e={};o.emit(n+"before-start",[t],e);var a;e[m]&&e[m].dt&&(a=e[m].dt);var s=r.apply(this,t);return o.emit(n+"start",[t,a],s),s.then(function(t){return o.emit(n+"end",[null,t],s),t},function(t){throw o.emit(n+"end",[t],s),t})})}var o=t("ee").get("fetch"),i=t(32),a=t(31);e.exports=o;var s=window,c="fetch-",f=c+"body-",u=["arrayBuffer","blob","json","text","formData"],d=s.Request,p=s.Response,l=s.fetch,h="prototype",m="nr@context";d&&p&&l&&(a(u,function(t,e){r(d[h],e,f),r(p[h],e,f)}),r(s,"fetch",c),o.on(c+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(c+"done",[null,e],n)}else o.emit(c+"done",[t],n)}))},{}],9:[function(t,e,n){var r=t("ee").get("history"),o=t("wrap-function")(r);e.exports=r;var i=window.history&&window.history.constructor&&window.history.constructor.prototype,a=window.history;i&&i.pushState&&i.replaceState&&(a=i),o.inPlace(a,["pushState","replaceState"],"-")},{}],10:[function(t,e,n){function r(t){function e(){f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}function n(){f.emit("jsonp-error",[],l),f.emit("jsonp-end",[],l),t.removeEventListener("load",e,c(!1)),t.removeEventListener("error",n,c(!1))}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var d=s(a),p="function"==typeof d.parent[d.key];if(p){var l={};u.inPlace(d.parent,[d.key],"cb-",l),t.addEventListener("load",e,c(!1)),t.addEventListener("error",n,c(!1)),f.emit("new-jsonp",[t.src],l)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(d);return e?e[1]:null}function a(t,e){var n=t.match(l),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(p);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t(23),f=t("ee").get("jsonp"),u=t("wrap-function")(f);if(e.exports=f,o()){var d=/[?&](?:callback|cb)=([^&#]+)/,p=/(.*)\.([^.]+)/,l=/^(\w+)(\.|$)(.*)$/,h=["appendChild","insertBefore","replaceChild"];Node&&Node.prototype&&Node.prototype.appendChild?u.inPlace(Node.prototype,h,"dom-"):(u.inPlace(HTMLElement.prototype,h,"dom-"),u.inPlace(HTMLHeadElement.prototype,h,"dom-"),u.inPlace(HTMLBodyElement.prototype,h,"dom-")),f.on("dom-start",function(t){r(t[0])})}},{}],11:[function(t,e,n){var r=t("ee").get("mutation"),o=t("wrap-function")(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],12:[function(t,e,n){function r(t){var e=i.context(),n=s(t,"executor-",e,null,!1),r=new f(n);return i.context(r).getCtx=function(){return e},r}var o=t("wrap-function"),i=t("ee").get("promise"),a=t("ee").getOrSetContext,s=o(i),c=t(31),f=NREUM.o.PR;e.exports=i,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){i.emit("propagate",[null,!o],a,!1,!1),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var a=e.apply(f,arguments),s=f.resolve(a);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&i.emit("propagate",[t,!0],n,!1,!1),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),o.wrapInPlace(f.prototype,"then",function(t){return function(){var e=this,n=o.argsToArray.apply(this,arguments),r=a(e);r.promise=e,n[0]=s(n[0],"cb-",r,null,!1),n[1]=s(n[1],"cb-",r,null,!1);var c=t.apply(this,n);return r.nextPromise=c,i.emit("propagate",[e,!0],c,!1,!1),c}}),i.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this,null,!1),t[1]=s(t[1],"resolve-",this,null,!1)}),i.on("executor-err",function(t,e,n){t[1](n)}),i.on("cb-end",function(t,e,n){i.emit("propagate",[n,!0],this.nextPromise,!1,!1)}),i.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=i.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],13:[function(t,e,n){var r=t("ee").get("raf"),o=t("wrap-function")(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],14:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t("wrap-function")(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],15:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){x.push(t),m&&(E?E.then(a):w?w(a):(R=-R,O.data=R))}function a(){for(var t=0;t<x.length;t++)r([],x[t]);x.length&&(x=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(7);var f=t("ee"),u=f.get("xhr"),d=t("wrap-function")(u),p=t(23),l=NREUM.o,h=l.XHR,m=l.MO,v=l.PR,w=l.SI,g="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],x=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new h(t);try{u.emit("new-xhr",[e],e),e.addEventListener(g,o,p(!1))}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(h,b),b.prototype=h.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),m){var E=v&&v.resolve();if(!w&&!v){var R=1,O=document.createTextNode(R);new m(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===g||a()})},{}],16:[function(t,e,n){function r(t){if(!s(t))return null;var e=window.NREUM;if(!e.loader_config)return null;var n=(e.loader_config.accountID||"").toString()||null,r=(e.loader_config.agentID||"").toString()||null,f=(e.loader_config.trustKey||"").toString()||null;if(!n||!r)return null;var h=l.generateSpanId(),m=l.generateTraceId(),v=Date.now(),w={spanId:h,traceId:m,timestamp:v};return(t.sameOrigin||c(t)&&p())&&(w.traceContextParentHeader=o(h,m),w.traceContextStateHeader=i(h,v,n,r,f)),(t.sameOrigin&&!u()||!t.sameOrigin&&c(t)&&d())&&(w.newrelicHeader=a(h,m,v,n,r,f)),w}function o(t,e){return"00-"+e+"-"+t+"-01"}function i(t,e,n,r,o){var i=0,a="",s=1,c="",f="";return o+"@nr="+i+"-"+s+"-"+n+"-"+r+"-"+t+"-"+a+"-"+c+"-"+f+"-"+e}function a(t,e,n,r,o,i){var a="btoa"in window&&"function"==typeof window.btoa;if(!a)return null;var s={v:[0,1],d:{ty:"Browser",ac:r,ap:o,id:t,tr:e,ti:n}};return i&&r!==i&&(s.d.tk=i),btoa(JSON.stringify(s))}function s(t){return f()&&c(t)}function c(t){var e=!1,n={};if("init"in NREUM&&"distributed_tracing"in NREUM.init&&(n=NREUM.init.distributed_tracing),t.sameOrigin)e=!0;else if(n.allowed_origins instanceof Array)for(var r=0;r<n.allowed_origins.length;r++){var o=h(n.allowed_origins[r]);if(t.hostname===o.hostname&&t.protocol===o.protocol&&t.port===o.port){e=!0;break}}return e}function f(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.enabled}function u(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.exclude_newrelic_header}function d(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&NREUM.init.distributed_tracing.cors_use_newrelic_header!==!1}function p(){return"init"in NREUM&&"distributed_tracing"in NREUM.init&&!!NREUM.init.distributed_tracing.cors_use_tracecontext_headers}var l=t(28),h=t(18);e.exports={generateTracePayload:r,shouldGenerateTrace:s}},{}],17:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<p;r++)t.removeEventListener(d[r],this.listener,!1);return e.protocol&&"data"===e.protocol?void g("Ajax/DataUrl/Excluded"):void(e.aborted||(n.duration=a.now()-this.startTime,this.loadCaptureCalled||4!==t.readyState?null==e.status&&(e.status=0):i(this,t),n.cbTime=this.cbTime,s("xhr",[e,n,this.startTime,this.endTime,"xhr"],this)))}}function o(t,e){var n=c(e),r=t.params;r.hostname=n.hostname,r.port=n.port,r.protocol=n.protocol,r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.parsedOrigin=n,t.sameOrigin=n.sameOrigin}function i(t,e){t.params.status=e.status;var n=v(e,t.lastSize);if(n&&(t.metrics.rxSize=n),t.sameOrigin){var r=e.getResponseHeader("X-NewRelic-App-Data");r&&(t.params.cat=r.split(", ").pop())}t.loadCaptureCalled=!0}var a=t("loader");if(a.xhrWrappable&&!a.disabled){var s=t("handle"),c=t(18),f=t(16).generateTracePayload,u=t("ee"),d=["load","error","abort","timeout"],p=d.length,l=t("id"),h=t(24),m=t(22),v=t(19),w=t(23),g=t(25).recordSupportability,y=NREUM.o.REQ,x=window.XMLHttpRequest;a.features.xhr=!0,t(15),t(8),u.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,e.loadCaptureCalled=!1,e.params=this.params||{},e.metrics=this.metrics||{},t.addEventListener("load",function(n){i(e,t)},w(!1)),h&&(h>34||h<10)||t.addEventListener("progress",function(t){e.lastSize=t.loaded},w(!1))}),u.on("open-xhr-start",function(t){this.params={method:t[0]},o(this,t[1]),this.metrics={}}),u.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid);var n=f(this.parsedOrigin);if(n){var r=!1;n.newrelicHeader&&(e.setRequestHeader("newrelic",n.newrelicHeader),r=!0),n.traceContextParentHeader&&(e.setRequestHeader("traceparent",n.traceContextParentHeader),n.traceContextStateHeader&&e.setRequestHeader("tracestate",n.traceContextStateHeader),r=!0),r&&(this.dt=n)}}),u.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=m(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"!==t.type||o.loadCaptureCalled||(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<p;s++)e.addEventListener(d[s],this.listener,w(!1))}),u.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),u.on("xhr-load-added",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),u.on("xhr-load-removed",function(t,e){var n=""+l(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),u.on("xhr-resolved",function(){this.endTime=a.now()}),u.on("addEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-added",[t[1],t[2]],e)}),u.on("removeEventListener-end",function(t,e){e instanceof x&&"load"===t[0]&&u.emit("xhr-load-removed",[t[1],t[2]],e)}),u.on("fn-start",function(t,e,n){e instanceof x&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),u.on("fn-end",function(t,e){this.xhrCbStart&&u.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)}),u.on("fetch-before-start",function(t){function e(t,e){var n=!1;return e.newrelicHeader&&(t.set("newrelic",e.newrelicHeader),n=!0),e.traceContextParentHeader&&(t.set("traceparent",e.traceContextParentHeader),e.traceContextStateHeader&&t.set("tracestate",e.traceContextStateHeader),n=!0),n}var n,r=t[1]||{};"string"==typeof t[0]?n=t[0]:t[0]&&t[0].url?n=t[0].url:window.URL&&t[0]&&t[0]instanceof URL&&(n=t[0].href),n&&(this.parsedOrigin=c(n),this.sameOrigin=this.parsedOrigin.sameOrigin);var o=f(this.parsedOrigin);if(o&&(o.newrelicHeader||o.traceContextParentHeader))if("string"==typeof t[0]||window.URL&&t[0]&&t[0]instanceof URL){var i={};for(var a in r)i[a]=r[a];i.headers=new Headers(r.headers||{}),e(i.headers,o)&&(this.dt=o),t.length>1?t[1]=i:t.push(i)}else t[0]&&t[0].headers&&e(t[0].headers,o)&&(this.dt=o)}),u.on("fetch-start",function(t,e){this.params={},this.metrics={},this.startTime=a.now(),this.dt=e,t.length>=1&&(this.target=t[0]),t.length>=2&&(this.opts=t[1]);var n,r=this.opts||{},i=this.target;if("string"==typeof i?n=i:"object"==typeof i&&i instanceof y?n=i.url:window.URL&&"object"==typeof i&&i instanceof URL&&(n=i.href),o(this,n),"data"!==this.params.protocol){var s=(""+(i&&i instanceof y&&i.method||r.method||"GET")).toUpperCase();this.params.method=s,this.txSize=m(r.body)||0}}),u.on("fetch-done",function(t,e){if(this.endTime=a.now(),this.params||(this.params={}),"data"===this.params.protocol)return void g("Ajax/DataUrl/Excluded");this.params.status=e?e.status:0;var n;"string"==typeof this.rxSize&&this.rxSize.length>0&&(n=+this.rxSize);var r={txSize:this.txSize,rxSize:n,duration:a.now()-this.startTime};s("xhr",[this.params,r,this.startTime,this.endTime,"fetch"],this)})}},{}],18:[function(t,e,n){var r={};e.exports=function(t){if(t in r)return r[t];if(0===(t||"").indexOf("data:"))return{protocol:"data"};var e=document.createElement("a"),n=window.location,o={};e.href=t,o.port=e.port;var i=e.href.split("://");!o.port&&i[1]&&(o.port=i[1].split("/")[0].split("@").pop().split(":")[1]),o.port&&"0"!==o.port||(o.port="https"===i[0]?"443":"80"),o.hostname=e.hostname||n.hostname,o.pathname=e.pathname,o.protocol=i[0],"/"!==o.pathname.charAt(0)&&(o.pathname="/"+o.pathname);var a=!e.protocol||":"===e.protocol||e.protocol===n.protocol,s=e.hostname===document.domain&&e.port===n.port;return o.sameOrigin=a&&(!e.hostname||s),"/"===o.pathname&&(r[t]=o),o}},{}],19:[function(t,e,n){function r(t,e){var n=t.responseType;return"json"===n&&null!==e?e:"arraybuffer"===n||"blob"===n||"json"===n?o(t.response):"text"===n||""===n||void 0===n?o(t.responseText):void 0}var o=t(22);e.exports=r},{}],20:[function(t,e,n){function r(){}function o(t,e,n,r){return function(){return u.recordSupportability("API/"+e+"/called"),i(t+e,[f.now()].concat(s(arguments)),n?null:this,r),n?void 0:this}}var i=t("handle"),a=t(31),s=t(32),c=t("ee").get("tracer"),f=t("loader"),u=t(25),d=NREUM;"undefined"==typeof window.newrelic&&(newrelic=d);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",h=l+"ixn-";a(p,function(t,e){d[e]=o(l,e,!0,"api")}),d.addPageAction=o(l,"addPageAction",!0),d.setCurrentRouteName=o(l,"routeName",!0),e.exports=newrelic,d.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(h+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){m[e]=o(h,e)}),newrelic.noticeError=function(t,e){"string"==typeof t&&(t=new Error(t)),u.recordSupportability("API/noticeError/called"),i("err",[t,f.now(),!1,e])}},{}],21:[function(t,e,n){function r(t){if(NREUM.init){for(var e=NREUM.init,n=t.split("."),r=0;r<n.length-1;r++)if(e=e[n[r]],"object"!=typeof e)return;return e=e[n[n.length-1]]}}e.exports={getConfiguration:r}},{}],22:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],23:[function(t,e,n){var r=!1;try{var o=Object.defineProperty({},"passive",{get:function(){r=!0}});window.addEventListener("testPassive",null,o),window.removeEventListener("testPassive",null,o)}catch(i){}e.exports=function(t){return r?{passive:!0,capture:!!t}:!!t}},{}],24:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],25:[function(t,e,n){function r(t,e){var n=[a,t,{name:t},e];return i("storeMetric",n,null,"api"),n}function o(t,e){var n=[s,t,{name:t},e];return i("storeEventMetrics",n,null,"api"),n}var i=t("handle"),a="sm",s="cm";e.exports={constants:{SUPPORTABILITY_METRIC:a,CUSTOM_METRIC:s},recordSupportability:r,recordCustom:o}},{}],26:[function(t,e,n){function r(){return s.exists&&performance.now?Math.round(performance.now()):(i=Math.max((new Date).getTime(),i))-a}function o(){return i}var i=(new Date).getTime(),a=i,s=t(33);e.exports=r,e.exports.offset=a,e.exports.getLastTimestamp=o},{}],27:[function(t,e,n){function r(t,e){var n=t.getEntries();n.forEach(function(t){"first-paint"===t.name?l("timing",["fp",Math.floor(t.startTime)]):"first-contentful-paint"===t.name&&l("timing",["fcp",Math.floor(t.startTime)])})}function o(t,e){var n=t.getEntries();if(n.length>0){var r=n[n.length-1];if(f&&f<r.startTime)return;var o=[r],i=a({});i&&o.push(i),l("lcp",o)}}function i(t){t.getEntries().forEach(function(t){t.hadRecentInput||l("cls",[t])})}function a(t){var e=navigator.connection||navigator.mozConnection||navigator.webkitConnection;if(e)return e.type&&(t["net-type"]=e.type),e.effectiveType&&(t["net-etype"]=e.effectiveType),e.rtt&&(t["net-rtt"]=e.rtt),e.downlink&&(t["net-dlink"]=e.downlink),t}function s(t){if(t instanceof w&&!y){var e=Math.round(t.timeStamp),n={type:t.type};a(n),e<=h.now()?n.fid=h.now()-e:e>h.offset&&e<=Date.now()?(e-=h.offset,n.fid=h.now()-e):e=h.now(),y=!0,l("timing",["fi",e,n])}}function c(t){"hidden"===t&&(f=h.now(),l("pageHide",[f]))}if(!("init"in NREUM&&"page_view_timing"in NREUM.init&&"enabled"in NREUM.init.page_view_timing&&NREUM.init.page_view_timing.enabled===!1)){var f,u,d,p,l=t("handle"),h=t("loader"),m=t(30),v=t(23),w=NREUM.o.EV;if("PerformanceObserver"in window&&"function"==typeof window.PerformanceObserver){u=new PerformanceObserver(r);try{u.observe({entryTypes:["paint"]})}catch(g){}d=new PerformanceObserver(o);try{d.observe({entryTypes:["largest-contentful-paint"]})}catch(g){}p=new PerformanceObserver(i);try{p.observe({type:"layout-shift",buffered:!0})}catch(g){}}if("addEventListener"in document){var y=!1,x=["click","keydown","mousedown","pointerdown","touchstart"];x.forEach(function(t){document.addEventListener(t,s,v(!1))})}m(c)}},{}],28:[function(t,e,n){function r(){function t(){return e?15&e[n++]:16*Math.random()|0}var e=null,n=0,r=window.crypto||window.msCrypto;r&&r.getRandomValues&&(e=r.getRandomValues(new Uint8Array(31)));for(var o,i="xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx",a="",s=0;s<i.length;s++)o=i[s],"x"===o?a+=t().toString(16):"y"===o?(o=3&t()|8,a+=o.toString(16)):a+=o;return a}function o(){return a(16)}function i(){return a(32)}function a(t){function e(){return n?15&n[r++]:16*Math.random()|0}var n=null,r=0,o=window.crypto||window.msCrypto;o&&o.getRandomValues&&Uint8Array&&(n=o.getRandomValues(new Uint8Array(t)));for(var i=[],a=0;a<t;a++)i.push(e().toString(16));return i.join("")}e.exports={generateUuid:r,generateSpanId:o,generateTraceId:i}},{}],29:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],30:[function(t,e,n){function r(t){function e(){t(s&&document[s]?document[s]:document[i]?"hidden":"visible")}"addEventListener"in document&&a&&document.addEventListener(a,e,o(!1))}var o=t(23);e.exports=r;var i,a,s;"undefined"!=typeof document.hidden?(i="hidden",a="visibilitychange",s="visibilityState"):"undefined"!=typeof document.msHidden?(i="msHidden",a="msvisibilitychange"):"undefined"!=typeof document.webkitHidden&&(i="webkitHidden",a="webkitvisibilitychange",s="webkitVisibilityState")},{}],31:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],32:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],33:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?f(t,c,a):a()}function n(n,r,o,i,a){if(a!==!1&&(a=!0),!l.aborted||i){t&&a&&t(n,r,o);for(var s=e(o),c=m(n),f=c.length,u=0;u<f;u++)c[u].apply(s,r);var p=d[y[n]];return p&&p.push([x,n,r,s]),s}}function i(t,e){g[t]=m(t).concat(e)}function h(t,e){var n=g[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return g[t]||[]}function v(t){return p[t]=p[t]||o(n)}function w(t,e){l.aborted||u(t,function(t,n){e=e||"feature",y[n]=e,e in d||(d[e]=[])})}var g={},y={},x={on:i,addEventListener:i,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:s,aborted:!1};return x}function i(t){return f(t,c,a)}function a(){return new r}function s(){(d.api||d.feature)&&(l.aborted=!0,d=l.backlog={})}var c="nr@context",f=t("gos"),u=t(31),d={},p={},l=e.exports=o();e.exports.getOrSetContext=i,l.backlog=d},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!T++){var t=O.info=NREUM.info,e=m.getElementsByTagName("script")[0];if(setTimeout(f.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return f.abort();c(E,function(e,n){t[e]||(t[e]=n)});var n=a();s("mark",["onload",n+O.offset],null,"api"),s("timing",["load",n]);var r=m.createElement("script");0===t.agent.indexOf("http://")||0===t.agent.indexOf("https://")?r.src=t.agent:r.src=l+"://"+t.agent,e.parentNode.insertBefore(r,e)}}function o(){"complete"===m.readyState&&i()}function i(){s("mark",["domContent",a()+O.offset],null,"api")}var a=t(26),s=t("handle"),c=t(31),f=t("ee"),u=t(29),d=t(21),p=t(23),l=d.getConfiguration("ssl")===!1?"http":"https",h=window,m=h.document,v="addEventListener",w="attachEvent",g=h.XMLHttpRequest,y=g&&g.prototype,x=!1;NREUM.o={ST:setTimeout,SI:h.setImmediate,CT:clearTimeout,XHR:g,REQ:h.Request,EV:h.Event,PR:h.Promise,MO:h.MutationObserver};var b=""+location,E={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1216.min.js"},R=g&&y&&y[v]&&!/CriOS/.test(navigator.userAgent),O=e.exports={offset:a.getLastTimestamp(),now:a,origin:b,features:{},xhrWrappable:R,userAgent:u,disabled:x};if(!x){t(20),t(27),m[v]?(m[v]("DOMContentLoaded",i,p(!1)),h[v]("load",r,p(!1))):(m[w]("onreadystatechange",o),h[w]("onload",r)),s("mark",["firstbyte",a.getLastTimestamp()],null,"api");var T=0}},{}],"wrap-function":[function(t,e,n){function r(t,e){function n(e,n,r,c,f){function nrWrapper(){var i,a,u,p;try{a=this,i=d(arguments),u="function"==typeof r?r(i,a):r||{}}catch(l){o([l,"",[i,a,c],u],t)}s(n+"start",[i,a,c],u,f);try{return p=e.apply(a,i)}catch(h){throw s(n+"err",[i,a,h],u,f),h}finally{s(n+"end",[i,a,p],u,f)}}return a(e)?e:(n||(n=""),nrWrapper[p]=e,i(e,nrWrapper,t),nrWrapper)}function r(t,e,r,o,i){r||(r="");var s,c,f,u="-"===r.charAt(0);for(f=0;f<e.length;f++)c=e[f],s=t[c],a(s)||(t[c]=n(s,u?c+r:r,o,c,i))}function s(n,r,i,a){if(!h||e){var s=h;h=!0;try{t.emit(n,r,i,e,a)}catch(c){o([c,n,r,i],t)}h=s}}return t||(t=u),n.inPlace=r,n.flag=p,n}function o(t,e){e||(e=u);try{e.emit("internal-error",t)}catch(n){}}function i(t,e,n){if(Object.defineProperty&&Object.keys)try{var r=Object.keys(t);return r.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(i){o([i],n)}for(var a in t)l.call(t,a)&&(e[a]=t[a]);return e}function a(t){return!(t&&t instanceof Function&&t.apply&&!t[p])}function s(t,e){var n=e(t);return n[p]=t,i(t,n,u),n}function c(t,e,n){var r=t[e];t[e]=s(r,n)}function f(){for(var t=arguments.length,e=new Array(t),n=0;n<t;++n)e[n]=arguments[n];return e}var u=t("ee"),d=t(32),p="nr@original",l=Object.prototype.hasOwnProperty,h=!1;e.exports=r,e.exports.wrapFunction=s,e.exports.wrapInPlace=c,e.exports.argsToArray=f},{}]},{},["loader",2,17,5,3,4]);</script><meta charset="UTF-8"/><meta name="robots" content="index, follow"/><meta name="DC.Creator" content="Linbo Liu"/><meta name="DC.Creator" content="Mingcheng Bi"/><meta name="DC.Creator" content="Yunhua Wang"/><meta name="DC.Creator" content="Junfeng Liu"/><meta name="DC.Creator" content="Xiwen Jiang"/><meta name="DC.Creator" content="Zhongbin Xu"/><meta name="DC.Creator" content="Xingcai Zhang"/><meta name="DC.title" content="Artificial intelligence-powered microfluidics for nanomedicine and materials synthesis "/><meta name="DC.publisher" content="Royal Society of Chemistry"/><meta name="DC.Date" content="2021/12/02"/><meta name="DC.Identifier" scheme="doi" content="10.1039/D1NR06195J"/><meta name="DC.Language" content="en"/><meta name="citation_title" content="Artificial intelligence-powered microfluidics for nanomedicine and materials synthesis "/><meta name="citation_author" content="Linbo Liu"/><meta name="citation_author" content="Mingcheng Bi"/><meta name="citation_author" content="Yunhua Wang"/><meta name="citation_author" content="Junfeng Liu"/><meta name="citation_author" content="Xiwen Jiang"/><meta name="citation_author" content="Zhongbin Xu"/><meta name="citation_author" content="Xingcai Zhang"/><meta name="citation_online_date" content="2021/11/09"/><meta name="citation_date" content="2021"/><meta name="citation_journal_title" content="Nanoscale"/><meta name="citation_volume" content="13"/><meta name="citation_issue" content="46"/><meta name="citation_firstpage" content="19352"/><meta name="citation_lastpage" content="19366"/><meta name="citation_doi" content="10.1039/D1NR06195J"/><meta name="citation_pdf_url" content="https://pubs.rsc.org/en/content/articlepdf/2021/nr/d1nr06195j"/><meta name="citation_abstract_html_url" content="https://pubs.rsc.org/en/content/articlelanding/2021/nr/d1nr06195j"/><meta name="citation_fulltext_html_url" content="https://pubs.rsc.org/en/content/articlehtml/2021/nr/d1nr06195j"/><link rel="shortcut icon" href=""/><link type="text/css" rel="stylesheet" href="/content/stylesheets/rschtml2.css?ver=6_2_1"/><link href="https://www.rsc-cdn.org/oxygen/assets/webfonts/fonts.min.css" rel="stylesheet" type="text/css"/><link type="text/css" rel="stylesheet" href="/content/stylesheets/pubs-ui.min.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/><script type="text/javascript" src="/content/scripts/JQueryPlugins.min.js"> </script><script type="text/javascript" src="/content/scripts/GetAnchorText.js"> </script><script type="text/javascript">
    
      $(function() {
      $("table.tgroup tfoot th").attr("colspan", "100");
      $("table.tgroup.rtable").each( function (idx, el) {
      var tw = $(this).width();
      $(this).parent().css("min-width", tw+"px");
      });

      });
      
    </script><!--6_2_1--></head><body class="oxy-ui pubs-ui ahtml-page"><!--Google Tag Manager (noscript)--><noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-56FZ7G" height="0" width="0" style="display:none;visibility:hidden"> </iframe></noscript><!--End Google Tag Manager (noscript)--><div class="viewport autopad"><div class="pnl pnl--drop autopad"><span id="top"/><div id="wrapper"><div class="left_head"><a class="simple" href="/"><img class="rsc-logo" border="0" src="/content/NewImages/royal-society-of-chemistry-logo.png" alt="Royal Society of Chemistry"/></a><br/><span class="btnContainer"><a class="btn btn--tiny btn--primary" target="_blank" title="Link to PDF version" href="/en/content/articlepdf/2021/nr/d1nr06195j">View PDF Version</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to previous article (id:d1nr05478c)" href="/en/content/articlehtml/2021/nr/d1nr05478c" target="_BLANK">Previous Article</a></span><span class="btnContainer"><a class="btn btn--tiny btn--nobg" title="Link to next article (id:d1nr04308k)" href="/en/content/articlehtml/2021/nr/d1nr04308k" target="_BLANK">Next Article</a></span></div><div class="right_head"><div id="crossmark_container"><div id="crossmark-content"><a id="open-crossmark" href="#" data-target="crossmark"><img style="max-width:100px" id="crossmark-logo" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_square.svg" alt="Check for updates"/></a><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"> </script></div></div> </div><div class="article_info"> DOI: <a target="_blank" title="Link to landing page via DOI" href="https://doi.org/10.1039/D1NR06195J">10.1039/D1NR06195J</a>
(Minireview)
<span class="italic"><a title="Link to journal home page" href="https://doi.org/10.1039/2040-3372/2009">Nanoscale</a></span>, 2021, <strong>13</strong>, 19352-19366</div><h1 id="sect167"><span class="title_heading">Artificial intelligence-powered microfluidics for nanomedicine and materials synthesis</span></h1><p class="header_text">
      <span class="bold">
        
          
            Linbo 
            Liu<a title="Select to navigate to footnote" href="#fn1">†</a>
          
          
        
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Mingcheng 
            Bi<a title="Select to navigate to footnote" href="#fn1">†</a>
          
          
        
      </span><sup><span class="sup_ref italic">b</span></sup>, 
      <span class="bold">
        
          
            Yunhua 
            Wang
          
        
      </span><sup><span class="sup_ref italic">a</span></sup>, 
      <span class="bold">
        
          
            Junfeng 
            Liu
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Junfeng Liu (orcid.org/0000-0002-7922-9275) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0002-7922-9275"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span><sup><span class="sup_ref italic">b</span></sup>, 
      <span class="bold">
        
          
            Xiwen 
            Jiang
          
        
      </span>*<sup><span class="sup_ref italic">c</span></sup>, 
      <span class="bold">
        
          
            Zhongbin 
            Xu
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Zhongbin Xu (orcid.org/0000-0001-7225-8420) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-7225-8420"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">b</span></sup><span class="bold"> and </span>
      <span class="bold">
        
          
            Xingcai 
            Zhang
          
          
        
      </span><span class="orcid"><a target="_blank" title="Select to open ORCID record for Xingcai Zhang (orcid.org/0000-0001-7114-1095) in a new window" id="connect-orcid-link" href="http://orcid.org/0000-0001-7114-1095"><img id="orcid-id-logo" src="/content/NewImages/orcid_16x16.png" alt="ORCID logo"/></a></span>*<sup><span class="sup_ref italic">ad</span></sup>
      <br/><a id="affa"><sup><span class="sup_ref italic">a</span></sup></a><span class="italic">John A. Paulson School of Engineering and Applied Science, Harvard University, Cambridge, MA 02138, USA</span>
      <br/><a id="affb"><sup><span class="sup_ref italic">b</span></sup></a><span class="italic">Institute of Process Equipment, College of Energy Engineering, Zhejiang University, Hangzhou 310027, P.R. China</span>
      <br/><a id="affc"><sup><span class="sup_ref italic">c</span></sup></a><span class="italic">College of Biological Science and Engineering, Fuzhou university, Fuzhou 350108, P.R. China</span>
      <br/><a id="affd"><sup><span class="sup_ref italic">d</span></sup></a><span class="italic">School of Engineering, Massachusetts Institute of Technology, Cambridge, MA 02139, USA. E-mail: <a href="mailto:xingcai@mit.edu">xingcai@mit.edu</a></span>
    </p><div id="art-admin"><span class="italic bold">Received 
      20th September 2021
    </span><span class="bold italic">, Accepted 29th October 2021</span></div><p class="bold italic">First published on 9th November 2021</p><hr/><div class="abstract"><h2>Abstract</h2><p>Artificial intelligence (AI) is an emerging technology with great potential, and its robust calculation and analysis capabilities are unmatched by traditional calculation tools. With the promotion of deep learning and open-source platforms, the threshold of AI has also become lower. Combining artificial intelligence with traditional fields to create new fields of high research and application value has become a trend. AI has been involved in many disciplines, such as medicine, materials, energy, and economics. The development of AI requires the support of many kinds of data, and microfluidic systems can often mine object data on a large scale to support AI. Due to the excellent synergy between the two technologies, excellent research results have emerged in many fields. In this review, we briefly review AI and microfluidics and introduce some applications of their combination, mainly in nanomedicine and material synthesis. Finally, we discuss the development trend of the combination of the two technologies.</p></div><hr/><table><tr><td class="biogPlate"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-p1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-p1.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-p1.gif"/></a><p><span class="graphic_title">Linbo Liu</span></p></td><td><i><p>Linbo Liu received his Ph.D. in mechanical engineering from Jiangsu Key Laboratory for Design and Manufacture of Micro-Nano Biomedical Instruments, Southeast University, China, in 2021. He has been working as a visiting scholar in the School of Engineering and Applied Sciences and Department of Physics at Harvard University since 2018. His research interests include droplet microfluidics, inertial microfluidics, dielectrophoresis, and molecular diagnostic instruments.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-p2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-p2.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-p2.gif"/></a><p><span class="graphic_title">Mingcheng Bi</span></p></td><td><i><p>Mingcheng Bi is currently a postgraduate candidate at the Energy Engineering College at Zhejiang University (ZJU), Hangzhou, China. He completed his Bachelor's degree at Dalian University of Technology (DLUT), Dalian, China in 2020. His current research interests include artificial intelligence and polymer processing engineering, especially microwave foaming materials.</p></i></td></tr></table><table><tr><td class="biogPlate"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-p3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-p3.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-p3.gif"/></a><p><span class="graphic_title">Xingcai Zhang</span></p></td><td><i><p>Dr Xingcai Zhang is a Harvard/MIT Research Fellow with expertise in sustainable nature/bio-derived/inspired materials/technologies for advanced biomed/energy/environment applications. He has published more than 100 peer-reviewed papers in top journals, including <span class="italic">Nat. Rev. Mater.</span>, <span class="italic">Nat. Nanotechnol.</span>, <span class="italic">Matter</span>, <span class="italic">Nat. Commun.</span>, <span class="italic">Sci. Adv.</span>, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, <span class="italic">Natl. Sci. Rev.</span>, <span class="italic">J. Am. Chem. Soc.</span>, <span class="italic">Angew. Chem. Int. Ed.</span>, <span class="italic">Adv. Mater.</span>, <span class="italic">Mater. Today</span>, <span class="italic">Prog. Mater. Sci.</span>, <span class="italic">Chem. Soc. Rev.</span>, and much more. He has served as a science writer/editor/advisory board member for Nature, Springer, Elsevier, Materials Today, American Chemical Society (ACS), Royal Society of Chemistry, and Wiley. He has been acknowledged by Nature VP for the publication of Nature Outlook Tea. He is a Nature Nano Ambassador and a member of multiple societies. Dr Zhang received multiple awards including the Nature Nano Award.</p></i></td></tr></table><hr/>
    
      
      <h2 id="sect277"><span class="a_heading">1. Introduction</span></h2>
      <span>Artificial intelligence (AI) has become a hot topic in recent years, which can be used to research and develop theories, methods, technologies, and application systems for simulating and extending human intelligence. The concept of AI was first proposed by McCarthy <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit1"><sup><span class="sup_ref">1</span></sup></a> in 1956, but it failed to show its potential due to algorithm limitation and insufficient data in the early stage. In recent years, with the emergence of a variety of neural network models represented by Convolutional Neural Networks (CNN) and a large number of algorithms,<a title="Select to navigate to reference" href="#cit2"><sup><span class="sup_ref">2–11</span></sup></a> as well as the development of open-source frameworks such as TensorFlow,<a title="Select to navigate to references" href="#cit12"><sup><span class="sup_ref">12</span></sup></a> AI has achieved leapfrog development. It has gained incomparable computing and analytical capabilities of traditional computing tools and rapidly become an emerging technology with great potential. In the past few years, the wave of artificial intelligence has rapidly swept across various industries, significantly impacting them.<a title="Select to navigate to reference" href="#cit13"><sup><span class="sup_ref">13–34</span></sup></a></span>
      <p class="otherpara">The rapid development of AI has brought new influence in many fields, and microfluidic technology is one of the beneficiaries. Microfluidics is a system that can process or manipulate a small number of fluids (10<small><sup>−9</sup></small> to 10<small><sup>−18</sup></small> L) in a microscale structure and has found diverse applications.<a title="Select to navigate to reference" href="#cit35"><sup><span class="sup_ref">35–37</span></sup></a> It can be traced back to the Miniaturized Total Analysis System (μ-TAS) proposed by Manz and Harison <span class="italic">et al.</span> in the 1990s. Since the fluid in the microfluidic system is at the micro/nano-level, it produces completely different behaviors from macro-fluids, such as high heat transfer efficiency, large laminar flow, and significant surface effect,<a title="Select to navigate to reference" href="#cit38"><sup><span class="sup_ref">38–40</span></sup></a> and it can precisely control nanoliter-scale liquids. Therefore, microfluidics has brought a new perspective to many traditional disciplines. In nearly 30 years of history, microfluidics has developed by leaps and bounds, showing great potential in many fields such as biology, new materials, chemistry, and energy,<a title="Select to navigate to reference" href="#cit41"><sup><span class="sup_ref">41–48</span></sup></a> and has become an interdisciplinary discipline involving engineering, physics, micro-processing, <span class="italic">etc</span>. Besides, microfluidic systems have the advantages of high throughput, high integration, high sensitivity, low power consumption, and can produce many data,<a title="Select to navigate to reference" href="#cit49"><sup><span class="sup_ref">49–54</span></sup></a> including size, shape, structure, composition, <span class="italic">etc</span>. Microfluidic systems often cannot mine and process these data well.</p>
      <p class="otherpara">However, the large amount of complex data generated by microfluidics is a fertile ground for AI. The core of AI is data support. The rapid acquisition of valuable information from a large amount of data mined by microfluidic systems can provide a good foundation for machine learning and other AI algorithms. The training process of the AI algorithm model directly determines its ability to process data.<a title="Select to navigate to reference" href="#cit55"><sup><span class="sup_ref">55–58</span></sup></a> Moreover, the AI model trained by the microfluidics can better feedback the microfluidic system, so that it can better mine data and lead the microfluidic system towards automation and intelligence. And the artificial intelligence model can be verified and adjusted. Therefore, these two emerging technologies can be mutually beneficial, creating new research in many fields (<a title="Select to navigate to table" href="#tab1">Table 1</a>).</p>
      
<div class="table_caption"><b>Table 1</b> <span id="tab1">Application of artificial intelligence-powered microfluidics</span>
		</div>

          <div class="rtable__wrapper"><div class="rtable__inner"><table class="tgroup rtable" border="0"><colgroup><col align="left" width="101.93pt"/><col align="left" width="111.33pt"/><col align="left" width="132.43pt"/><col align="left" width="151.38pt"/><col align="left" width="25.15pt"/></colgroup>
            
            
            
            
            
            <thead align="left">
              <tr align="left" valign="bottom">
                <th align="left" valign="bottom" class="border_black">Microfluidic technology</th>
                <th align="left" valign="bottom" class="border_black">AI algorithm</th>
                <th align="left" valign="bottom" class="border_black">Applications</th>
                <th align="left" valign="bottom" class="border_black">Advantages</th>
                <th align="left" valign="bottom" class="border_black">Ref.</th>
              </tr>
            </thead>
            <tbody align="left">
              <tr align="left">
                <td valign="top">Microfluidic chip</td>
                <td valign="top">Deep convolutional neural network</td>
                <td valign="top">Intelligent image-activated cell sorting</td>
                <td valign="top">High integration and automation</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit53">53</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microfluidic cytometer</td>
                <td valign="top">Convolutional neural network and extreme learning machine</td>
                <td valign="top">Lensless blood cell counting system</td>
                <td valign="top">High resolution, low cost, and high throughput</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit68">68</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">A microfluidic chip and mobile lensless blood cell image acquisition device</td>
                <td valign="top">Convolutional neural network</td>
                <td valign="top">An entire mobile lensless microfluidic blood image acquisition and analysis system</td>
                <td valign="top">Equipment miniaturization and high accuracy</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit69">69</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">3D microfluidic devices</td>
                <td valign="top">Convolutional neural networks backward Kalman filtering and multiple hypothesis tracking</td>
                <td valign="top">Multicell migration tracking</td>
                <td valign="top">Migration tracking and high accuracy</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit70">70</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Digital microfluidic (DMF)</td>
                <td valign="top">Convolutional neural network</td>
                <td valign="top">Digital microfluidic Isolation of Single Cells for -Omics</td>
                <td valign="top">Unique levels of selectivity, context, and accountability</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit71">71</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microfluidic microscopy</td>
                <td valign="top">Support vector machine</td>
                <td valign="top">Cancer screening</td>
                <td valign="top">Cost-effective and label-free</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit73">73</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Single-cell microfluidic method and quantitative deformability cytometry (q-DC)</td>
                <td valign="top">A <span class="italic">k</span>-nearest neighbor machine learning algorithm</td>
                <td valign="top">Predicting cancer cell invasion</td>
                <td valign="top">Classification of cancer cell types</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit74">74</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">A microfluidic device consisted of luer-lock inlet and outlet ports</td>
                <td valign="top">Random-forest decision trees</td>
                <td valign="top">Risk stratification of cancer patients</td>
                <td valign="top">High accuracy and operability</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit75">75</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">A microfluidic blood–brain niche (μBBN) chip</td>
                <td valign="top">Neural network, random forest, logistic regression, <span class="italic">etc.</span></td>
                <td valign="top">Identify the possibility of tumor cells extravasating into the brain metastatic niche</td>
                <td valign="top">Precise</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit76">76</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">2D light scattering microfluidic cytometry</td>
                <td valign="top">Deep learning</td>
                <td valign="top">Detection of leukemia cells</td>
                <td valign="top">Label-free, high efficiency, and high automation</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit77">77</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">A microfluidic device with channels and mazes</td>
                <td valign="top">Support vector machine</td>
                <td valign="top">Diagnosis of sepsis</td>
                <td valign="top">High accuracy</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit78">78</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">PDMS microfluidic channel</td>
                <td valign="top">Extreme learning machine</td>
                <td valign="top">Microfluidic imaging cytometer</td>
                <td valign="top">Continuous high throughput flowing cell recognition and counting</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit79">79</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Optofluidic time-stretch microscope with microfluidic device</td>
                <td valign="top">Support vector machine</td>
                <td valign="top">Detection of cellular drug responses</td>
                <td valign="top">High throughput and label-free</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit82">82</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Extreme-throughput, label-free, whole-blood imaging flow cytometer</td>
                <td valign="top">A deep convolutional autoencoder</td>
                <td valign="top">
                  <span class="italic">Ex vivo</span> drug-susceptibility testing of leukemia</td>
                <td valign="top">Highly simple, rapid, and cost-effective</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit83">83</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microfluidic chip with both physical stability and optical transparency</td>
                <td valign="top">Convolutional neural network</td>
                <td valign="top">Optofluidic time-stretch quantitative phase imaging</td>
                <td valign="top">Highly reliable, high-throughput label-free</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit84">84</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Vitro microfluidic cancer models</td>
                <td valign="top">Custom image analytic algorithms based on machine learning</td>
                <td valign="top">Recapitulation of the tumor microenvironment</td>
                <td valign="top">Sustain tumor fragments for multiple days, permit real-time, high-resolution imaging of the interaction and provide automated and quantitative assessment of experimental results</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit90">90</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">16-Channel microfluidic reactor</td>
                <td valign="top">Black box optimization</td>
                <td valign="top">Optimization of quantum dot reaction conditions</td>
                <td valign="top">Fast and flexible</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit98">98</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Integrated silicon microreactor</td>
                <td valign="top">Black-box algorithms</td>
                <td valign="top">Optimize the yield and the throughput of a Knoevenagel condensation reaction</td>
                <td valign="top">
                </td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit101">101</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microreactor</td>
                <td valign="top">Artificial neural network</td>
                <td valign="top">Rapid data analysis in combinatorial nanoparticle syntheses</td>
                <td valign="top">Extract condition–property relations rapidly from the combinatorial synthesis data</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit102">102</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Tubular micro-reactor</td>
                <td valign="top">Artificial neural network</td>
                <td valign="top">Study the fluid dynamics of VOC oxidation over the nano-catalyst under the different operating conditions</td>
                <td valign="top">The model can be employed for the complex simultaneous oxidation of VOCs in the plug reactor without involving in the kinetic mechanism.</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit103">103</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microreactor</td>
                <td valign="top">Genetic algorithm</td>
                <td valign="top">Autonomously driven materials-evolution robotic platform</td>
                <td valign="top">Discover complex nano-constructs using desired spectroscopic responses</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit104">104</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Oscillatory microfluidic platforms</td>
                <td valign="top">Bayesian optimization</td>
                <td valign="top">Self-driving platform for metal nanoparticle synthesis</td>
                <td valign="top">Time- and resource-saving</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit105">105</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microfluidic reactors</td>
                <td valign="top">Ensemble neural networks and Bayesian optimization</td>
                <td valign="top">Self-driven multistep quantum dot synthesis</td>
                <td valign="top">Rapid, autonomous and low cost</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit107">107</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microfluidic reactors</td>
                <td valign="top">Artificial neural network</td>
                <td valign="top">Identify relationships between variables affecting drug nanoprecipitation</td>
                <td valign="top">
                </td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit108">108</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microfluidic devices</td>
                <td valign="top">Artificial neural network</td>
                <td valign="top">Predict PLGA particle sizes produced by different microfluidic systems either individually or jointly merged</td>
                <td valign="top">Rapid and economical</td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit109">109</a>
                </td>
              </tr>
              <tr align="left">
                <td valign="top">Microfluidic high-throughput experimental</td>
                <td valign="top">Bayesian optimization with a deep neural network</td>
                <td valign="top">Produce silver nanoparticles with the desired absorbance spectrum</td>
                <td valign="top">
                </td>
                <td valign="top">
                  <a title="Select to navigate to references" href="#cit110">110</a>
                </td>
              </tr>
            </tbody>
          </table></div></div>
        <hr class="hrule"/><br/>
      <p class="otherpara">Artificial intelligence and microfluidics have excellent synergy.<a title="Select to navigate to references" href="#cit51"><sup><span class="sup_ref">51</span></sup></a> Therefore, in this minireview, we provide a brief overview of how the combination of microfluidic technology and artificial intelligence can help in other fields and mainly introduce the applications in nanomedicine and material synthesis. Finally, we discuss the future development trend of the combination of the two technologies.</p>
    
    
      
      <h2 id="sect503"><span class="a_heading">2. Applications in nanomedicine</span></h2>
      <span>Nanomedicine is an emerging science that uses the technology and principles of nanoscience to treat, detect, and diagnose biological systems.<a title="Select to navigate to references" href="#cit59"><sup><span class="sup_ref">59</span></sup></a> Many biological mechanisms in the human body and the pathogenic mechanisms of viruses and bacteria occur on the nanoscale.<a title="Select to navigate to references" href="#cit60"><sup><span class="sup_ref">60</span></sup></a> It uses nanoparticles to enter the human body to provide imaging feedback to specific areas, or use high-throughput equipment to quickly detect blood and other samples <span class="italic">in vitro</span> to diagnose diseases, and use nanoparticles as drug carriers for disease treatment, which solves many problems that are difficult to be solved by traditional medical technology.<a title="Select to navigate to reference" href="#cit61"><sup><span class="sup_ref">61–63</span></sup></a> Besides, empowered by AI, nanomedicine is increasingly used in cell identification and classification, disease diagnosis, drug sensitivity testing, disease treatment, <span class="italic">etc</span>.</span>
      
        
        <h3 id="sect512"><span class="b_heading">2.1 Cell detection and recognition</span></h3>
        <span>A complete blood count (CBC) is one of the most commonly used laboratory tests in medicine. It reflects the health of the human body by counting cells or substances such as red blood cells, white blood cells, and hemoglobin in the blood. Some common diseases can be diagnosed by complete blood count, such as leukemia, inflammation, and scurvy.<a title="Select to navigate to reference" href="#cit64"><sup><span class="sup_ref">64–67</span></sup></a> The traditional complete blood count method usually uses a high-power microscope for manual counting, which is often time-consuming and labor-intensive, and has a small amount of processing. Although the flow cytometers based on microfluidic technology have high throughput, they are usually large in size and expensive. Recently, a portable and low-cost technology for the timely detection of blood cell counts has been developed. In 2016, Huang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit68"><sup><span class="sup_ref">68</span></sup></a> reported a single-frame super-resolution processing system for lensless blood cell counting based on machine learning (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1A</a>). They built a miniaturized lensless imaging system by integrating a microfluidic channel on a miniature image sensor. To solve the problem of low throughput of multi-frame super-resolution processing and a large amount of storage space, they developed a microfluidic lensless cell counting system that can achieve high single-cell image quality without the limitation of throughput through super-resolution processing based on extreme learning machine (ELM) and convolutional neural network (CNN). The results show that the cell resolution of CNNSR had been increased by four times, the results are in good agreement with the commercial flow cytometer, and single-cell level analysis could be performed. In 2019, Liao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit69"><sup><span class="sup_ref">69</span></sup></a> built a mobile CNN lensless sensor blood cell collection and analysis system (<a title="Select to navigate to figure" href="#imgfig1">Fig. 1B</a>). In the system, a pure integer quantization algorithm is proposed, which maintains high analysis accuracy while making the equipment miniaturized. Compared with the traditional floating-point method, the area is reduced by 45% while the calculation accuracy is only reduced by 0.56%. What's more, experimental results show that when the frequency is 100 MHz, the classification speed can reach 17.9 fps. These results show that the ingenious combination of artificial intelligence and lensless microfluidic imaging technology can pave the way for the miniaturization, portability, and real-time detection of complete blood count instruments.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig1"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f1_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f1.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f1.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 1 </b> <span id="fig1"><span class="graphic_title">(A) Schematic diagram of the lens-free cell counting system and model structure of the extreme learning machine. Reproduced from ref. <a title="Select to navigate to references" href="#cit68">68</a> with permission from Molecular Diversity Preservation International, 2016. (B) Cell division network structure in lensless imaging systems and comparison between the microfluidic chip and a coin. Reproduced from ref. <a title="Select to navigate to references" href="#cit69">69</a> with permission from Molecular Diversity Preservation International, 2019.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">Identifying different cell types and exploring the relationship between cell genes, compositions, structures and morphologies and cell physiology are important in biomedical research. In recent years, more and more technologies have emerged to identify and screen different cells, and conduct research. In 2018, Nao Nitta <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit53"><sup><span class="sup_ref">53</span></sup></a> proposed intelligent image-activated cell sorting (IACS) technology, which combined high-throughput cell microscopy, cell focusing, and sorting technology to achieve real-time automated operation of cell data collection, data processing, decision-making and driving. The system has high flexibility and high scalability. When this technology uses a five-layer deep CNN to screen platelets with high components, the high classification purity of aggregation can reach 79.5%, and even single platelets and double platelets can be distinguished. The whole process can be completed in 32 milliseconds (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2A</a>). Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit70"><sup><span class="sup_ref">70</span></sup></a> proposed an automatic image analysis system to vascular endothelial cells’ migration in a three-dimensional microfluidic device. The cell detection algorithm based on CNN obtained by using five-fold cross-validation achieved 87.4% accuracy and 91.2% recall rate in cell candidate detection, and the association accuracy reached 86.4% when a multi-hypothesis Kalman filter was used to correlate detected candidate cells (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2B</a>). Regarding single-cell detection, Julian Lamanna <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit71"><sup><span class="sup_ref">71</span></sup></a> have launched a Digital microfluidic Isolation of Single Cells for -Omics (DISCO) platform, which uses lasers to lyse cells and collects and analyzes the contents of target cells on the microfluidic platform, and then uses artificial intelligence-driven image processing technology to analyze the contents (<a title="Select to navigate to figure" href="#imgfig2">Fig. 2C</a>). To verify the selectivity of the system, a mixture of human glioblastoma cells and murine melanoma cells transfected with different substances was inoculated and cultured and analyzed on a microfluidic device. The results showed that the positioning rate for the correct species was more than 80% and the positioning rate for the wrong species was less than 8%. These methods can help people better understand the role of different cells and evaluate the capabilities of different components of the cell, and master the cell mechanism.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig2"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f2_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f2.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f2.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 2 </b> <span id="fig2"><span class="graphic_title">(A) Probability distribution of representative platelet aggregation, white blood cell, and single platelet images obtained by IACS using CNN. Reproduced from ref. <a title="Select to navigate to references" href="#cit53">53</a> with permission from Cell Press, 2018. (B) Example of cell tracking after one day of migration, where green is the previous day. Reproduced from ref. <a title="Select to navigate to references" href="#cit70">70</a> with permission from Spie-Soc Photo-Optical Instrumentation Engineers, 2018. (C) The structure of the CNN used by the DISCO platform for cell localization and segmentation and examples of artificial intelligence processing images. Reproduced from ref. <a title="Select to navigate to references" href="#cit71">71</a> with permission from Nature Research, 2020.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect542"><span class="b_heading">2.2 Diagnosis of disease</span></h3>
        <span>Cancer is one of the most life-threatening diseases. According to statistics, there were 18.1 million new cancer cases and 9.6 million cancer deaths in 2018.<a title="Select to navigate to references" href="#cit72"><sup><span class="sup_ref">72</span></sup></a> The mortality rate of cancer can be reduced through early detection and treatment. In cancer detection, the cytopathological examination is critical, and the physical characteristics of cells are usually used as biomarkers for cancer diagnosis. But most cancer screenings are time-consuming and expensive. In recent years, many studies have been devoted to solving these problems. In 2016, Veerendra Kalyan Jagannadh <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit73"><sup><span class="sup_ref">73</span></sup></a> developed a low-cost and label-free method for microfluidic microscopy to identify cancer cells (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3A</a>). This technology is based on principal component analysis (PCA) for feature extraction, support vector machine (SVM) for classification, and combined microfluidic microscopy with artificial intelligence. The “malignant cell surveillance system” suitable for cancer screening has been demonstrated for the first time, and the observation accuracy is up to 98.01%. In terms of cancer cell invasion ability to predict, Kendra D. Nyberg <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit74"><sup><span class="sup_ref">74</span></sup></a> used quantitative deformable cell count (Q-DC) to measure 6 physical phenotypes (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3B</a>) at the rate of 100 cells per second, including elastic modulus, maximum strain, cell fluidity and size, transit time, and entry time. And using the <span class="italic">k</span>-nearest neighbor machine learning algorithm classified cell lines, the accuracy of cancer cell line classification is nearly doubled compared with single parameter analysis (up to 96%). They used a set of experimental data of human ovarian cancer cells overexpressing tumor suppressor microRNAs to train a multiple linear regression model to generate a physical phenotype model of invasion and successfully validated the model with human cancer cells of breast and ovarian cancer. This allows people to better predict the invasion of cancer cells through the physical phenotype of single cells. By stratifying the risk of cancer patients, appropriate treatment can be given to different degrees of cancer. In 2018, Michael S. Manak <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit75"><sup><span class="sup_ref">75</span></sup></a> reported a phenotypic biomarker assay for living primary cells. Unlike conventional formalin fixation histochemical biomarker analysis, this method can measure previously unavailable phenotypic biomarkers for living cells. This technology combines machine vision analysis and random forest decision trees, uses a microfluidic device as a cell measurement environment, and uses dynamic live cell biomarkers to stain prostate cancer and breast cancer tissue samples. The results are highly sensitive and specific. It divided cancer patients into three levels, which successfully provided a method for the clinical application of risk stratification (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3C</a>). In 2019, C. Ryan Oliver <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit76"><sup><span class="sup_ref">76</span></sup></a> built a platform to quantitatively analyze the aggressiveness of cancer cells (<a title="Select to navigate to figure" href="#imgfig3">Fig. 3D</a>). They used the microfluidic blood–brain niche (μBBN) chip to describe the extravasation of cancer cells, combined with advanced live-cell imaging algorithms and artificial intelligence algorithms such as neural network, naive Bayes, random forest tree, logistic regression, knowledge network, and stochastic gradient descent, and successfully identified cancer cells with brain metastasis phenotypes with an accuracy of up to 0.87. It can be seen that the combination of artificial intelligence and microfluidic technology provides a new, convenient, and cheap method for cancer diagnosis and classification, which is of great significance for humans to fight cancer.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig3"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f3_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f3.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f3.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 3 </b> <span id="fig3"><span class="graphic_title">(A) Label-free cancer cell identification and screening framework. Reproduced from ref. <a title="Select to navigate to references" href="#cit73">73</a> with permission from Springer Heidelberg, 2017. (B) Comparison of the Q-DC microfluidic device with US penny, schematic diagram of the Q-DC physical phenotype, six physics are obtained through microfluidic contraction to denature cells, parameters and the confusion matrix of Q-DC's predictive ability for cell classification. Reproduced from ref. <a title="Select to navigate to references" href="#cit74">74</a> with permission from Oxford University Press, 2018 (C) Artificial intelligence algorithm integrates biomarkers to score and predict cells and patients. Reproduced from ref. <a title="Select to navigate to references" href="#cit75">75</a> with permission from Nature Research, 2018. (D) Flow chart for the identification of the potential of tumor cells to extravasate into the brain metastatic niche. Reproduced from ref. <a title="Select to navigate to references" href="#cit76">76</a> with permission from Royal Society Chemistry, 2019.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect566"><span class="b_heading">2.3 Drug-susceptibility testing</span></h3>
        <span>Drug sensitivity testing can help people accurately and effectively formulate disease treatment strategies and use drugs for treatment. In the past few years, high-content screening based on multivariate single-cell imaging has been proven effective in drug discovery, which can assess drug-induced phenotypic variation in gene expression, protein localization,<a title="Select to navigate to reference" href="#cit80"><sup><span class="sup_ref">80,81</span></sup></a> and cytoskeletal structure. However, high-content screening requires fluorescent labels, most of which are analyses of inactivated cells, which will increase the cost and time-consumption of the experiment. These disadvantages hinder large-scale analysis. To overcome these shortcomings, in 2017, Hirofumi Kobayashi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit82"><sup><span class="sup_ref">82</span></sup></a> proposed a label-free method that only uses high-throughput brightfield imaging and machine learning algorithms to evaluate cellular drug responses (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4A</a>). They used the anticancer drug paclitaxel as a model drug, and MCF-7 as the model cell line to induce cell morphological changes. The cells were collected through an optical flow time stretching microscope at a high throughput of up to 10<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 cells per second bright-field images and researchers could use unlabeled cell images and SVM to identify their morphological changes, the discrimination accuracy rate can reach 92%. These studies have laid the foundation for unlabeled drug screening in pharmaceutical science and industry. In 2018, Hirofumi Kobayashi <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit83"><sup><span class="sup_ref">83</span></sup></a> proposed a liquid biopsy method for leukemia drug sensitivity test <span class="italic">in vitro</span>, which overcomes the disadvantages of poor clinical applicability and time-consuming and labor-consuming techniques in the past methods (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4B and C</a>) and uses a high-throughput whole blood imaging flow cytometer (&gt;1 million cells per second) and a deep convolution autoencoder for label-free detection. It can be used to analyze drug-treated whole blood cells within 24 hours. The drug sensitivity of whole blood leukocytes is obtained based on image recognition. In addition, based on the dose-dependent morphological changes, they verified through experiments that the drug sensitivity of the human chronic myeloid leukemia cell line K562 resistant strain is the same as the detection result of the traditional method. This method paves establishing of a drug susceptibility group for all available anticancer drugs for each patient and can improve the survival rate of leukemia patients. In 2020, Wu <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit84"><sup><span class="sup_ref">84</span></sup></a> proposed a high-throughput (up to 15<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 cells per second), high-accuracy frequency-shifted optical fluid time-stretch quantitative phase imaging technology, using this technology to obtain a large number of images of white blood cells and cancer cells. And they used more than 30<img class="charmap" src="https://www.rsc.org/images/entities/char_2009.gif" alt="[thin space (1/6-em)]"/>000 images to train a convolutional neural network (CNN), according to the images to display the intelligent classification of various types of leukemia cells and white blood cells, with an average accuracy of 96.2% (<a title="Select to navigate to figure" href="#imgfig4">Fig. 4D</a>). Based on this, they evaluated the efficacy of anti-cancer drugs on leukemia cells, and through CNN analysis, the drug-induced differentiation of cells can be identified in image recognition. This technology can monitor and evaluate the efficacy of drug-treated leukemia cells in a label-free manner. Through the combination of microfluidics and artificial intelligence technology, these technologies have successfully provided a basis for selecting disease drugs and brought hope to leukemia patients.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig4"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f4_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f4.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f4.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 4 </b> <span id="fig4"><span class="graphic_title">(A) Schematic diagram of the optical flow time stretching microscope and machine learning-assisted image analysis. Reproduced from ref. <a title="Select to navigate to references" href="#cit82">82</a> with permission from Nature Publishing, 2017. (B) Intelligent whole blood imaging flow cytometer drug sensitivity test process and principle diagram. (C) Comparison of the exported leukocyte susceptibility test images of healthy people and patients with acute lymphoblastic leukemia from an intelligent whole blood imaging flow cytometer. Reproduced from ref. <a title="Select to navigate to references" href="#cit83">83</a> with permission from Royal Society Chemistry, 2019. (D) Intelligent evaluation results of leukemia treatment. Reproduced from ref. <a title="Select to navigate to references" href="#cit84">84</a> with permission from Optical Society America, 2020.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect586"><span class="b_heading">2.4 Treatment of diseases</span></h3>
        <span>Cancer treatment is limited by the complexity of the microenvironment in which tumor cells are located. The regeneration of the tumor microenvironment is the key to explore the tumorigenesis mechanism, targeted therapy, and immunotherapy and evaluate chemotherapy.<a title="Select to navigate to reference" href="#cit85"><sup><span class="sup_ref">85–88</span></sup></a> With the emergence of microfluidic technology, the ability to model the tumor microenvironment <span class="italic">in vitro</span> has been greatly improved, and the “cancer on a chip” platform has developed rapidly. In 2016, Jose M. Ayuso <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit89"><sup><span class="sup_ref">89</span></sup></a> proposed a microfluidic device that can model and study the tumor microenvironment in real-time and is easy to use (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5A and B</a>). The device can simulate the three-dimensional structure of multicellular spheroids to generate visible, live tumor chips. It is easy to observe cells in different areas in the simulated environment and their response to different drugs in real-time and evaluate the ability of drugs to penetrate several layers of cells to improve the effective rate of drug treatment. In 2018, N. Moore <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit90"><sup><span class="sup_ref">90</span></sup></a> combined the microfluidic technology with artificial intelligence and reported a microfluidic model of an <span class="italic">in vitro</span> immune tumor dynamic environment for tumor biopsy (<a title="Select to navigate to figure" href="#imgfig5">Fig. 5C and D</a>). They put live tumor slices (up to 12) in the model to simulate the interaction of tumor cells and tumor-infiltrating lymphocytes (TILs) in real-time and dynamically, and can obtain high-resolution images in real-time. On this basis, they use the machine learning custom image analysis algorithm for the quantitative evaluation of experimental results. They used the system to demonstrate the interaction of mouse tumor cells with anti-PD-1 immune checkpoint inhibitors and to evaluate the response of human tumor cells to immune checkpoint inhibitors. These studies show that the development of <span class="italic">in vitro</span> modeling of the tumor microenvironment provides a powerful platform for <span class="italic">in vitro</span> experiments of cancer and other diseases, allowing researchers to better develop drugs and treat cancer more effectively.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig5"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f5_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f5.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f5.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 5 </b> <span id="fig5"><span class="graphic_title">(A) The principle of “tumor slice” <span class="italic">in vivo</span>, in which the red area simulates the area of tumor necrosis. (B) Cells in the microdevice are observed under the microscope (stained). Reproduced from ref. <a title="Select to navigate to references" href="#cit89">89</a> with permission from Nature Publishing Group, 2016. (C) Image analytic mapping of the tumor using automated segmentation technology. (D) Recognition map of the tumor infiltration process. Reproduced from ref. <a title="Select to navigate to references" href="#cit90">90</a> with permission from Royal Society Chemistry, 2018.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
    
    
      
      <h2 id="sect606"><span class="a_heading">3. Applications in materials synthesis</span></h2>
      <span>Due to their good mechanical and optical properties, nanomaterials have an extensive range of applications and broad development prospects. Traditional preparation methods have limited the synthesis of some nanoparticles due to expensive equipment, low yields, and high requirements for reaction conditions. And it is difficult to produce nanoparticles with more complex structures. With the emergence of microfluidic technology, it has become an excellent synthesis platform for nanoparticles due to its excellent droplet manipulation ability, excellent efficient heat transfer and mass transfer rates. It has been widely used in the synthesis of nanoparticles, such as composite materials, metal particles, nanofibers, and quantum dots,<a title="Select to navigate to reference" href="#cit91"><sup><span class="sup_ref">91–100</span></sup></a> opening a new direction in nanoparticle synthesis. However, during the synthesis of nanomaterials, many parameters need to be controlled, including temperature, pressure, reagent concentration, flow rate, <span class="italic">etc</span>. These parameters are directly related to the performance of the prepared nanoparticles, and it is challenging to adjust these parameters manually, which also limits the development of nanomaterials. In recent years, with the development of artificial intelligence technology, people have begun to combine it with microfluidic technology to synthesize nanomaterials, optimize and predict the synthesis of nanomaterials, and reduce the required cost while preparing high-quality nanoparticles.</span>
      
        
        <h3 id="sect612"><span class="b_heading">3.1 Synthetic optimization</span></h3>
        <span>The reaction parameters during material synthesis can determine the performance of the product. Finding the optimal conditions for material synthesis has always been one of the key research contents in chemical synthesis. Although microfluidic technology has greatly optimized material synthesis, it is still challenging to manually adjust a large number of reaction parameters and make them optimal. As early as in 2010, Jonathan P. McMullen <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit101"><sup><span class="sup_ref">101</span></sup></a> began to consider using artificial intelligence to adjust response conditions and obtain optimal results. They proposed an automated microfluidic system for the online optimization of chemical synthesis and carried out 46 oxidation experiments of benzoic acid using benzoyl formaldehyde. The simplex method was used for multi-parameter optimization, and the optimal reaction conditions were successfully determined when the yield of benzoyl formaldehyde was 80%, which provided a way for optimizing chemical synthesis (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6A</a>). In 2012, Yuuichi Orimoto <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit102"><sup><span class="sup_ref">102</span></sup></a> trained 1600 different neural networks to construct an integrated neural network to determine the relationship between reaction conditions and the properties of produced nanomaterials, and successfully extracted the basic model of the conditional-attribute relationship of CdSe nanoparticles. The key parameters of control synthesis were extracted by data interpolation and sensitivity analysis of the neural network, which guided the optimization of the properties of nanoparticles. In 2020, Amin Sokhansanj <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit103"><sup><span class="sup_ref">103</span></sup></a> developed an intelligent hybrid model (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6B</a>) using computational fluid dynamics analysis and artificial neural networks to simulate the simultaneous catalytic oxidation of benzene and toluene. The result showed that at 350 °C, using nano-supported cobalt oxide for catalysis, the maximum conversion rates of benzene and toluene are 89.74% and 82.37%, which are consistent with the experimental results. This model can not only avoid the more complicated kinetic mechanism in the reaction process, solve the reaction rate more accurately, but also can explore the influence of reaction conditions on the catalytic oxidation reaction. This work provides a useful tool for studying the hydrodynamics of VOC oxidation on the nano-catalyst under different operating conditions. In the same year, Daniel Salley <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit104"><sup><span class="sup_ref">104</span></sup></a> proposed an autonomously driven material evolution robot platform (<a title="Select to navigate to figure" href="#imgfig6">Fig. 6C</a>). They used genetic algorithms to synthesize designated nanoparticles. After analyzing the synthesized nanoparticles, they optimized the synthesis again. This method can not only synthesize known structures can also be used to discover more complex nanostructures. Experiments have proved that the system can produce spherical nanoparticles and rod-shaped particles. Finally, octahedral nanoparticles are synthesized by optimizing rod-shaped particles as seeds. In 2021, Tao <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit105"><sup><span class="sup_ref">105</span></sup></a> reported a platform for the automatic synthesis of nanoparticles.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig6"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f6_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f6.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f6.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 6 </b> <span id="fig6"><span class="graphic_title">(A) Operation flow chart of the automated microfluidic system and the best reaction conditions for maximum benzaldehyde yield obtained by the simplex method. Reproduced from ref. <a title="Select to navigate to references" href="#cit101">101</a> with permission from American Chemistry Society, 2010. (B) CFD-ANN hybrid modeling architecture for studying the simultaneous catalytic oxidation of benzene and toluene in the air on nano-catalysts. Reproduced from ref. <a title="Select to navigate to references" href="#cit103">103</a> with permission from Elsevier, 2020. (C) Intelligent evolution of gold nanoparticles. Reproduced from ref. <a title="Select to navigate to references" href="#cit104">104</a> with permission from Nature Publishing Group, 2020.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
        <p class="otherpara">They combined the oscillating microfluidic platform with machine learning for the first time and used the oscillating microfluidic platform to synthesize nanoparticles of different dimensions. They used Bayesian optimization to analyze the nanoparticles and only a few experiments were performed to obtain the optimal nanoparticle synthesis formula. Furthermore, the relationship between the reaction conditions and the properties of the nanoparticles was also obtained through the ML analysis. It provides support for the automatic synthesis technology of nanoparticles. These studies provide a more convenient method for optimizing the conditions of materials synthesis, which can greatly reduce costs and increase the yield of target products. Furthermore, it can provide a theory for people to discover new structures.</p>
        <p class="otherpara">In recent years, quantum dots have attracted attention due to their unique optical and electrical properties, and are very popular in fields such as solar cells, displays, and cell imaging. However, the synthesis of quantum dots involves a series of processes such as the nucleation and growth of crystals, which have a complex reaction mechanism. This leads to a large dispersion of the performance of the produced quantum dots, which limits their application to a certain extent. Recently, more and more scientists have researched this. In 2019, Oleksandr Voznyy <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit106"><sup><span class="sup_ref">106</span></sup></a> optimized the synthesis of monodisperse PbS nanoparticles (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7A</a>), using Bayesian optimization implemented by neural networks to establish a continuous numerical model of the reaction parameter space, and using its optimized synthesis parameter results to synthesize PbS nanoparticles, whose half-width at half-maximum are 55 meV and 24 meV at 950 nm and 1500 nm (the best values previously announced are 75 and 26 meV). In 2020, Wang <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit98"><sup><span class="sup_ref">98</span></sup></a> used feedback control for the first time to synthesize high-throughput nanomaterials in a continuous flow parallel reactor. They developed a 16-channel microfluidic reactor to produce light-emitting quantum dots, which solved the problem of low throughput required for industrial production. The reactor is based on the real-time monitoring of the photoluminescence characteristics of quantum dots and uses a black-box algorithm for feedback control to self-optimize product quality. It is used to produce CsPbBr<small><sub>3</sub></small> quantum dots with a yield of 1L h<small><sup>−1</sup></small>. With the help of artificial intelligence, these studies have a more thorough understanding of the nucleation and growth mechanism of quantum dots, and can better control and optimize reaction conditions to obtain high-quality quantum dots quickly and in large quantities (<a title="Select to navigate to figure" href="#imgfig7">Fig. 7B</a>). In 2021, to find the optimal synthesis formula for inorganic lead halide perovskite quantum dots, Kameel Abdel-Latif <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit107"><sup><span class="sup_ref">107</span></sup></a> developed an autonomous microfluidic synthesis strategy using integrated neural networks and Bayesian optimization. The process for determining the optimal synthesis route to the continuous manufacturing of ideal quantum dots is realized, and the tedious steps in the middle are omitted, saving a lot of time.</p>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig7"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f7_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f7.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f7.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 7 </b> <span id="fig7"><span class="graphic_title">(A) The effects of oleylamine and chloride on PbS synthesis predicted by the machine learning model are the same as experimental results. Reproduced from ref. <a title="Select to navigate to references" href="#cit106">106</a> with permission from American Chemical Society, 2019. (B) Self-optimization and anti-interference performance of microfluidic reactor for nanoparticle synthesis. Reproduced from ref. <a title="Select to navigate to references" href="#cit98">98</a> with permission from Royal Society Chemistry, 2020.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
      
        
        <h3 id="sect650"><span class="b_heading">3.2 Synthetic prediction</span></h3>
        <span>It is essential to accurately predict the properties of synthetic materials. During the synthesis of materials, the performance of the products may vary greatly due to the different reaction conditions. However, in microfluidic synthesis, there are many reaction conditions involved, and a small change may lead to a large change in the properties of the product. Therefore, a lot of research is devoted to this. In 2009, Hany S. M. Ali <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit108"><sup><span class="sup_ref">108</span></sup></a> trained a neural network using 53 samples prepared under different processing conditions and created a model to identify the relationship between variables that affect drug precipitation. They used this model to input the reaction conditions such as the angle and inner diameter of the microreactor during prednisolone nanoprecipitation, and successfully predicted the resulting particle size, which was consistent with the experimental data (<a title="Select to navigate to figure" href="#imgfig8">Fig. 8A</a>). In 2020, Safa A. Damiati <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit109"><sup><span class="sup_ref">109</span></sup></a> developed an artificial neural network prediction model for droplet and particle size generated by three different microfluidic systems to control the size uniformity of synthesized PLGA particles (<a title="Select to navigate to figure" href="#imgfig8">Fig. 8B</a>), which can predict the size of PLGA particles and quickly and effectively produce PLGA particles with adjustable size. To synthesize nanomaterials with specific properties, Flore Mekki-Berrada <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit110"><sup><span class="sup_ref">110</span></sup></a> proposed a two-step framework for a machine learning-driven high-throughput microfluidic platform in 2020 (<a title="Select to navigate to figure" href="#imgfig8">Fig. 8C</a>). They combined Bayesian optimization of the Gaussian process with a deep neural network. After sufficient data training, it can be used to predict the reaction synthesis of accessible color palettes and successfully optimize the synthesis of nanomaterials. In 2020, Jonathan A. Fine <span class="italic">et al.</span><a title="Select to navigate to references" href="#cit111"><sup><span class="sup_ref">111</span></sup></a> trained a neural network to use a combination of Fourier transform infrared and mass spectroscopy data to predict the functional groups of molecules. After experimental verification, the neural network model can predict and recognize compound mixtures containing different functional groups (<a title="Select to navigate to figure" href="#imgfig8">Fig. 8D</a>). Artificial intelligence not only helps to understand the relationship between reaction conditions and product properties but also provides a basis for optimizing the properties of target products and quickly synthesizing materials with specific properties in large quantities.</span>
        <br/><div class="image_table"><table><tr><td colspan="3" class="imgHolder" id="imgfig8"><a href="/image/article/2021/NR/d1nr06195j/d1nr06195j-f8_hi-res.gif" title="Select to open image in new window" onclick="open(this.href, &#34;_blank&#34;, &#34;toolbar=1,scrollbars=yes,resizable=1&#34;); return false;"><img alt="image file: d1nr06195j-f8.tif" src="/image/article/2021/NR/d1nr06195j/d1nr06195j-f8.gif"/></a></td></tr><tr><td class="pushTitleRight"> </td><td class="image_title"><b>Fig. 8 </b> <span id="fig8"><span class="graphic_title">(A) The predicted particle size obtained by the artificial neural network is almost the same as the observed size. Reproduced from ref. <a title="Select to navigate to references" href="#cit108">108</a> with permission from Elsevier, 2009. (B) The schematic diagram of the artificial neural network is used to predict the size of PLGA particles. Reproduced from ref. <a title="Select to navigate to references" href="#cit109">109</a> with permission from Nature Research, 2020. (C) The two-step algorithm framework of the high-throughput microfluidic platform. Reproduced from ref. <a title="Select to navigate to references" href="#cit110">110</a> with permission from Nature Research, 2021. (D) Algorithm schematic diagram and prediction examples for functional group prediction classification. Reproduced from ref. <a title="Select to navigate to references" href="#cit111">111</a> with permission from Royal Society of Chemistry, 2020.</span></span></td><td class="pushTitleLeft"> </td></tr></table></div>
      
    
    
      
      <h2 id="sect672"><span class="a_heading">4. Summary and outlook</span></h2>
      <span>With the development of AI, more and more microfluidics scholars have begun to devote themselves to the study of AI, aiming to create new ideas for their research fields. The advantages of the two technologies complement each other as if they are a natural pair. Although microfluidic systems have advantages of small scale and high throughput, they often produce a large amount of complex data which are difficult to be processed manually, while artificial intelligence technology is efficient and has powerful computing and analysis capabilities, but needs a large amount of data to train the model. Their characteristics determine that they will inevitably be combined to bring a new and rapidly developing direction to nanomedicine and material synthesis. In the field of nanomedicine, these two technologies are helping medical workers gain a deeper understanding of human cells and their disease-causing mechanisms. It has made some traditional detection instruments intelligent and miniaturized<a title="Select to navigate to references" href="#cit112"><sup><span class="sup_ref">112</span></sup></a> and has made great contributions to disease diagnosis, drug development, and disease treatment.<a title="Select to navigate to reference" href="#cit113"><sup><span class="sup_ref">113–117,121,122</span></sup></a> In materials synthesis, microfluidic technology has become an excellent material synthesis platform with the characteristics of high throughput, efficient mass transfer, excellent heat transfer rate, and precise liquid manipulation. With the addition of artificial intelligence, we have a deeper understanding of the reaction process, successfully optimized and predicted the synthesis results, and obtained products with better performance and higher yield.<a title="Select to navigate to reference" href="#cit114"><sup><span class="sup_ref">114,118–120</span></sup></a></span>
      <p class="otherpara">However, reviewing existing research, it is not difficult to find that there are still shortcomings. First of all, most of the artificial intelligence currently used is supervised learning, which requires manual use of a large amount of data to train the model, which will consume a lot of time and labor and it may be more convenient to use unsupervised learning. Secondly, the general applicability of existing research is poor. In the application of disease diagnosis, most of them are for the identification and sorting of cancer cells, while less attention is paid to other diseases.</p>
      <p class="otherpara">In the future, common diseases that are difficult to cure, such as AIDS, diabetes, and COVID-19, can be studied. In the field of materials synthesis, most of the current programs are only for the synthesis of a certain material, and a public program has yet to be developed. In short, as emerging technologies, artificial intelligence and microfluidics have achieved satisfactory results, and as we further study them, they will definitely collide with more intense novel ideas.</p>
    
    
      <h2 id="sect681"><span class="a_heading">Conflicts of interest</span></h2>
      <span>There are no conflicts to declare.</span>
    
  
    <span id="sect685"><h2 id="sect684"><span class="a_heading">Notes and references</span></h2></span><ol type="1">
      <li><span id="cit1">J. Mccarthy, M. L. Minsky and C. E. Shannon, <span class="italic">AI Mag.</span>, 2006, <span class="bold">27</span>, 12–14 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=AI%20Mag.%5BJour%5D%20AND%2027%5Bvolume%5D%20AND%2012%5Bpage%5D%20and%202006%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit2">G. E. Hinton, S. Osindero and Y. Teh, <span class="italic">Neural Comput.</span>, 2006, <span class="bold">18</span>, 1527–1554 <a target="_blank" class="DOILink" href="https://doi.org/10.1162/neco.2006.18.7.1527" title="DOI Link to resource 10.1162/neco.2006.18.7.1527">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16764513%5Buid%5D" title="PubMed Link to resource 16764513">PubMed</a>.</span></li>
      <li><span id="cit3">
          B. E. Boser, I. M. Guyon and V. N. Vapnik., Proceedings of the fifth annual workshop on Computational learning theory,  1992, pp. 144–152.</span></li>
      <li><span id="cit4">M. W. Fagerland, <span class="italic">Stata J.</span>, 2014, <span class="bold">14</span>, 947–964 <a target="_blank" class="DOILink" href="https://doi.org/10.1177/1536867X1401400414" title="DOI Link to resource 10.1177/1536867X1401400414">CrossRef</a>.</span></li>
      <li><span id="cit5">
          L. Breiman, <span class="italic">Arcing classifiers</span>,  1996 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Arcing%20classifiers%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201996%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit6">W. Y. Loh, <span class="italic">Wiley Interdiscip. Rev. Data Min. Knowl. Discov.</span>, 2011, <span class="bold">1</span>, 14–23 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/widm.8" title="DOI Link to resource 10.1002/widm.8">CrossRef</a>.</span></li>
      <li><span id="cit7">D. E. Rumelhart, G. E. Hinton and R. J. Williams, <span class="italic">Nature</span>, 1986, <span class="bold">323</span>, 533–536 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/323533a0" title="DOI Link to resource 10.1038/323533a0">CrossRef</a>.</span></li>
      <li><span id="cit8">
          D. S. Broomhead and D. Lowe, <span class="italic">Radial basis functions, multi-variable functional interpolation and adaptive networks</span>, Royal Signals and Radar Establishment Malvern, United Kingdom,  1988 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Radial%20basis%20functions,%20multi-variable%20functional%20interpolation%20and%20adaptive%20networks%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%201988%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit9">K. Fukushima, <span class="italic">Biol. Cybern.</span>, 1980, <span class="bold">36</span>, 193–202 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/BF00344251" title="DOI Link to resource 10.1007/BF00344251">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADyaL3c7nsFKntw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=7370364%5Buid%5D" title="PubMed Link to resource 7370364">PubMed</a>.</span></li>
      <li><span id="cit10">C. Cortes and V. Vapnik, <span class="italic">Mach. Learn.</span>, 1995, <span class="bold">20</span>, 273–297 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Mach.%20Learn.%5BJour%5D%20AND%2020%5Bvolume%5D%20AND%20273%5Bpage%5D%20and%201995%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit11">
          V. Vapnik, <span class="italic">The nature of statistical learning theory</span>, Springer science &amp; business media,  2013 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=The%20nature%20of%20statistical%20learning%20theory%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202013%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit12">L. Rampasek and A. Goldenberg, <span class="italic">Cell Syst.</span>, 2016, <span class="bold">2</span>, 12–14 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cels.2016.01.009" title="DOI Link to resource 10.1016/j.cels.2016.01.009">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhtFakt78%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27136685%5Buid%5D" title="PubMed Link to resource 27136685">PubMed</a>.</span></li>
      <li><span id="cit13">X. Chen, H. Chen, Z. Chen, J. Gong and C. Y. Chen, <span class="italic">J. Mater. Chem. B</span>, 2020, <span class="bold">8</span>, 2063–2081 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0TB00061B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0TB00061B">RSC</a>.</span></li>
      <li><span id="cit14">B. Huang, Z. Li and J. Li, <span class="italic">Nanoscale</span>, 2018, <span class="bold">10</span>, 21320–21326 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8NR06734A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8NR06734A">RSC</a>.</span></li>
      <li><span id="cit15">A. Thakkar, S. Johansson, K. Jorner, D. Buttar, J. Reymond and O. Engkvist, <span class="italic">React. Chem. Eng.</span>, 2021, <span class="bold">6</span>, 27–51 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0RE00340A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0RE00340A">RSC</a>.</span></li>
      <li><span id="cit16">J. C. Caicedo, S. Cooper, F. Heigwer, S. Warchal, P. Qiu, C. Molnar, A. S. Vasilevich, J. D. Barry, H. S. Bansal, O. Kraus, M. Wawer, L. Paavolainen, M. D. Herrmann, M. Rohban, J. Hung, H. Hennig, J. Concannon, I. Smith, P. A. Clemons, S. Singh, P. Rees, P. Horvath, R. G. Linington and A. E. Carpenter, <span class="italic">Nat. Methods</span>, 2017, <span class="bold">14</span>, 849–863 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nmeth.4397" title="DOI Link to resource 10.1038/nmeth.4397">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVektrbJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28858338%5Buid%5D" title="PubMed Link to resource 28858338">PubMed</a>.</span></li>
      <li><span id="cit17">M. Xiao, H. Yang, Q. Xiao, H. Zhang and X. Chen, <span class="italic">Nanoscale</span>, 2019, <span class="bold">11</span>, 19179–19189 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9NR00667B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9NR00667B">RSC</a>.</span></li>
      <li><span id="cit18">C. Zhai, T. Li, H. Shi and J. Yeo, <span class="italic">J. Mater. Chem. B</span>, 2020, <span class="bold">8</span>, 6562–6587 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0TB00896F&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0TB00896F">RSC</a>.</span></li>
      <li><span id="cit19">J. Bostrom, D. G. Brown, R. J. Young and G. M. Keseru, <span class="italic">Nat. Rev. Drug Discovery</span>, 2018, <span class="bold">17</span>, 709–727 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nrd.2018.116" title="DOI Link to resource 10.1038/nrd.2018.116">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30140018%5Buid%5D" title="PubMed Link to resource 30140018">PubMed</a>.</span></li>
      <li><span id="cit20">Y. D. Hezaveh, L. P. Levasseur and P. J. Marshall, <span class="italic">Nature</span>, 2017, <span class="bold">548</span>, 555–557 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature23463" title="DOI Link to resource 10.1038/nature23463">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXhsVektrnJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28858317%5Buid%5D" title="PubMed Link to resource 28858317">PubMed</a>.</span></li>
      <li><span id="cit21">C. Lei, H. Kobayashi, Y. Wu, M. Li, A. Isozaki, A. Yasumoto, H. Mikami, T. Ito, N. Nitta, T. Sugimura, M. Yamada, Y. Yatomi, D. Di Carlo, Y. Ozeki and K. Goda, <span class="italic">Nat. Protoc.</span>, 2018, <span class="bold">13</span>, 1603–1631 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41596-018-0008-7" title="DOI Link to resource 10.1038/s41596-018-0008-7">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWjsb%252FE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29976951%5Buid%5D" title="PubMed Link to resource 29976951">PubMed</a>.</span></li>
      <li><span id="cit22">G. Wetzstein, A. Ozcan, S. Gigan, S. Fan, D. Englund, M. Soljačić, C. Denz, D. A. B. Miller and D. Psaltis, <span class="italic">Nature</span>, 2020, <span class="bold">588</span>, 39–47 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41586-020-2973-6" title="DOI Link to resource 10.1038/s41586-020-2973-6">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXisVOks7rI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33268862%5Buid%5D" title="PubMed Link to resource 33268862">PubMed</a>.</span></li>
      <li><span id="cit23">M. Liu, P. Karuso, Y. Feng, E. Kellenberger, F. Liu, C. Wang and R. J. Quinn, <span class="italic">MedChemComm</span>, 2019, <span class="bold">10</span>, 1667–1677 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9MD00128J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9MD00128J">RSC</a>.</span></li>
      <li><span id="cit24">X. Zou, J. Pan, Z. Sun, B. Wang, Z. Jin, G. Xu and F. Yan, <span class="italic">Energy Environ. Sci.</span>, 2021, 3965–3975 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D1EE01170G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D1EE01170G">RSC</a>.</span></li>
      <li><span id="cit25">H. Yang, Z. Zhang, J. Zhang and X. C. Zeng, <span class="italic">Nanoscale</span>, 2018, <span class="bold">1</span>, 1992–1999 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Nanoscale%5BJour%5D%20AND%201%5Bvolume%5D%20AND%201992%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit26">M. Zhang, J. Li, L. Kang, N. Zhang, C. Huang, Y. He, M. Hu, X. Zhou and J. Zhang, <span class="italic">Nanoscale</span>, 2020, <span class="bold">12</span>, 3988–3996 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9NR09146G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9NR09146G">RSC</a>.</span></li>
      <li><span id="cit27">S. Wang, Z. Shen, Z. Shen, Y. Dong, Y. Li, Y. Cao, Y. Zhang, S. Guo, J. Shuai, Y. Yang, C. Lin, X. Chen, X. Zhang and Q. Huang, <span class="italic">Nano Today</span>, 2021, <span class="bold">38</span>, 101152 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.nantod.2021.101152" title="DOI Link to resource 10.1016/j.nantod.2021.101152">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvVCrs7vO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit28">Z. Tang, L. Zhao, G. Chen and C. Y. Chen, <span class="italic">RSC Adv.</span>, 2021, <span class="bold">11</span>, 6423–6446 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0RA10077C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0RA10077C">RSC</a>.</span></li>
      <li><span id="cit29">J. He, C. He, C. Zheng, Q. Wang and J. Ye, <span class="italic">Nanoscale</span>, 2019, <span class="bold">11</span>, 17444–17459 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9NR03450A&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9NR03450A">RSC</a>.</span></li>
      <li><span id="cit30">H. S. Stein, J. M. Gregoire and P. C. U. S. California Institute Of Technology Caltech, <span class="italic">Chem. Sci.</span>, 2019, <span class="bold">1</span>, 964–9649 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Chem.%20Sci.%5BJour%5D%20AND%201%5Bvolume%5D%20AND%20964%5Bpage%5D%20and%202019%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit31">P. L. Gentili, <span class="italic">RSC Adv.</span>, 2013, <span class="bold">3</span>, 25523 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3RA44657C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3RA44657C">RSC</a>.</span></li>
      <li><span id="cit32">G. K. Anumanchipalli, J. Chartier and E. F. Chang, <span class="italic">Nature</span>, 2019, <span class="bold">568</span>, 493–498 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41586-019-1119-1" title="DOI Link to resource 10.1038/s41586-019-1119-1">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXovFejsbk%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31019317%5Buid%5D" title="PubMed Link to resource 31019317">PubMed</a>.</span></li>
      <li><span id="cit33">P. W. Hatfield, J. A. Gaffney, G. J. Anderson, S. Ali, L. Antonelli, S. D. P. Başeğmez, J. Citrin, M. Fajardo, P. Knapp, B. Kettle, B. Kustowski, M. J. Macdonald, D. Mariscal, M. E. Martin, T. Nagayama, C. A. J. Palmer, J. L. Peterson, S. Rose, J. J. Ruby, C. Shneider, M. J. V. Streeter, W. Trickey and B. Williams, <span class="italic">Nature</span>, 2021, <span class="bold">593</span>, 351–361 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41586-021-03382-w" title="DOI Link to resource 10.1038/s41586-021-03382-w">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhtFanurzJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34012079%5Buid%5D" title="PubMed Link to resource 34012079">PubMed</a>.</span></li>
      <li><span id="cit34">O. Inderwildi, C. Zhang, X. Wang and M. Kraft, <span class="italic">Energy Environ. Sci.</span>, 2020, <span class="bold">13</span>, 744–771 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9EE01919G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9EE01919G">RSC</a>.</span></li>
      <li><span id="cit35">Y. Wang, L. Lu, G. Zheng and X. Zhang, <span class="italic">ACS Nano</span>, 2020, <span class="bold">14</span>, 9861–9872 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.0c02701" title="DOI Link to resource 10.1021/acsnano.0c02701">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmt7rE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32701267%5Buid%5D" title="PubMed Link to resource 32701267">PubMed</a>.</span></li>
      <li><span id="cit36">G. M. Whitesides, <span class="italic">Nature</span>, 2006, <span class="bold">442</span>, 368–373 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature05058" title="DOI Link to resource 10.1038/nature05058">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD28XnsVaju74%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16871203%5Buid%5D" title="PubMed Link to resource 16871203">PubMed</a>.</span></li>
      <li><span id="cit37">Z. Tang, N. Kong, X. Zhang, Y. Liu, P. Hu, S. Mou, P. Liljeström, J. Shi, W. Tan, J. S. Kim, Y. Cao, R. Langer, K. W. Leong, O. C. Farokhzad and W. Tao, <span class="italic">Nat. Rev. Mater.</span>, 2020, <span class="bold">5</span>, 847–860 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41578-020-00247-y" title="DOI Link to resource 10.1038/s41578-020-00247-y">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXit1Gqt7jO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33078077%5Buid%5D" title="PubMed Link to resource 33078077">PubMed</a>.</span></li>
      <li><span id="cit38">P. N. Nge, C. I. Rogers and A. T. Woolley, <span class="italic">Chem. Rev.</span>, 2013, <span class="bold">113</span>, 2550–2583 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/cr300337x" title="DOI Link to resource 10.1021/cr300337x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3sXisF2jtLw%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=23410114%5Buid%5D" title="PubMed Link to resource 23410114">PubMed</a>.</span></li>
      <li><span id="cit39">J. Atencia and D. J. Beebe, <span class="italic">Nature</span>, 2005, <span class="bold">437</span>, 648–655 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/nature04163" title="DOI Link to resource 10.1038/nature04163">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD2MXhtVCjsLzN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=16193039%5Buid%5D" title="PubMed Link to resource 16193039">PubMed</a>.</span></li>
      <li><span id="cit40">A. A. Darhuber and S. M. Troian, <span class="italic">Annu. Rev. Fluid Mech.</span>, 2005, <span class="bold">37</span>, 425–455 <a target="_blank" class="DOILink" href="https://doi.org/10.1146/annurev.fluid.36.050802.122052" title="DOI Link to resource 10.1146/annurev.fluid.36.050802.122052">CrossRef</a>.</span></li>
      <li><span id="cit41">M. Chowdhury, W. Zheng, S. Kumari, J. Heyman, X. Zhang, P. Dey, D. Weitz and R. Haag, <span class="italic">Nat. Commun.</span>, 2019, <span class="bold">10</span>, 4546 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-019-12462-5" title="DOI Link to resource 10.1038/s41467-019-12462-5">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31586046%5Buid%5D" title="PubMed Link to resource 31586046">PubMed</a>.</span></li>
      <li><span id="cit42">Z. Li, X. Zhang, J. Ouyang, D. Chu, F. Han, L. Shi, R. Liu, Z. Guo, G. X. Gu, W. Tao, L. Jin and J. Li, <span class="italic">Bioact. Mater.</span>, 2021, <span class="bold">6</span>, 4053–4064 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bioactmat.2021.04.014" title="DOI Link to resource 10.1016/j.bioactmat.2021.04.014">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhtFKjtL%252FF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33997492%5Buid%5D" title="PubMed Link to resource 33997492">PubMed</a>.</span></li>
      <li><span id="cit43">S. Han, Q. Zhang, X. Zhang, X. Liu, L. Lu, J. Wei, Y. Li, Y. Wang and G. Zheng, <span class="italic">Biosens. Bioelectron.</span>, 2019, <span class="bold">143</span>, 111957 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bios.2019.111597" title="DOI Link to resource 10.1016/j.bios.2019.111597">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31442755%5Buid%5D" title="PubMed Link to resource 31442755">PubMed</a>.</span></li>
      <li><span id="cit44">R. Yang, Z. Gong, X. Zhang and L. Que, <span class="italic">J. Nanosci. Nanotechnol.</span>, 2019, <span class="bold">19</span>, 7591–7595 <a target="_blank" class="DOILink" href="https://doi.org/10.1166/jnn.2019.16752" title="DOI Link to resource 10.1166/jnn.2019.16752">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXisVyrsrrE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31196265%5Buid%5D" title="PubMed Link to resource 31196265">PubMed</a>.</span></li>
      <li><span id="cit45">Z. Li, D. Chu, Y. Gao, L. Jin, X. Zhang, W. Cui and J. Li, <span class="italic">Mater. Today Adv.</span>, 2019, <span class="bold">3</span>, 100014 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mtadv.2019.100014" title="DOI Link to resource 10.1016/j.mtadv.2019.100014">CrossRef</a>.</span></li>
      <li><span id="cit46">Q. Zhang, X. Zhang, X. Zhang, L. Jiang, J. Yin, P. Zhang, S. Han, Y. Wang and G. Zheng, <span class="italic">Mar. Pollut. Bull.</span>, 2019, <span class="bold">144</span>, 20–27 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.marpolbul.2019.04.063" title="DOI Link to resource 10.1016/j.marpolbul.2019.04.063">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXpsVWkt70%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31179989%5Buid%5D" title="PubMed Link to resource 31179989">PubMed</a>.</span></li>
      <li><span id="cit47">Z. Gong, S. Karandikar, X. Zhang, V. Kotipalli, Y. Lvov and L. Que, <span class="italic">IEEE Sensors</span>, 2010, <span class="bold">10</span>, 29–32 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=IEEE%20Sensors%5BJour%5D%20AND%2010%5Bvolume%5D%20AND%2029%5Bpage%5D%20and%202010%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit48">G. Zheng, Q. Gao, Y. Jiang, L. Lu, J. Li, X. Zhang, H. Zhao, P. Fan, Y. Cui, F. Gu and Y. Wang, <span class="italic">Anal. Chem.</span>, 2021, <span class="bold">93</span>, 9728–9736 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.analchem.1c00917" title="DOI Link to resource 10.1021/acs.analchem.1c00917">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhsVOgurfO" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34228918%5Buid%5D" title="PubMed Link to resource 34228918">PubMed</a>.</span></li>
      <li><span id="cit49">E. Brouzes, M. Medkova, N. Savenelli, D. Marran, M. Twardowski, J. B. Hutchison, J. M. Rothberg, D. R. Link, N. Perrimon and M. L. Samuels, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2009, <span class="bold">106</span>, 14195–14200 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0903542106" title="DOI Link to resource 10.1073/pnas.0903542106">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXhtFaiurfE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19617544%5Buid%5D" title="PubMed Link to resource 19617544">PubMed</a>.</span></li>
      <li><span id="cit50">J. J. Agresti, E. Antipov, A. R. Abate, K. Ahn, A. C. Rowat, J. C. Baret, M. Marquez, A. M. Klibanov, A. D. Griffiths and D. A. Weitz, <span class="italic">Proc. Natl. Acad. Sci. U. S. A.</span>, 2010, <span class="bold">107</span>, 4004–4009 <a target="_blank" class="DOILink" href="https://doi.org/10.1073/pnas.0910781107" title="DOI Link to resource 10.1073/pnas.0910781107">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXjsFWitr0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20142500%5Buid%5D" title="PubMed Link to resource 20142500">PubMed</a>.</span></li>
      <li><span id="cit51">L. Liu, N. Xiang, Z. Ni, X. Huang, J. Zheng, Y. Wang and X. Zhang, <span class="italic">BioTechniques</span>, 2020, <span class="bold">68</span>, 114–116 <a target="_blank" class="DOILink" href="https://doi.org/10.2144/btn-2019-0134" title="DOI Link to resource 10.2144/btn-2019-0134">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXkvVWnsL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31973559%5Buid%5D" title="PubMed Link to resource 31973559">PubMed</a>.</span></li>
      <li><span id="cit52">A. Isozaki, J. Harmon, Y. Zhou, S. Li, Y. Nakagawa, M. Hayashi, H. Mikami, C. Lei and K. Goda, <span class="italic">Lab Chip</span>, 2020, <span class="bold">20</span>, 3074–3090 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0LC00521E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0LC00521E">RSC</a>.</span></li>
      <li><span id="cit53">N. Nitta, T. Sugimura, A. Isozaki, H. Mikami, K. Hiraki, S. Sakuma, T. Iino, F. Arai, T. Endo, Y. Fujiwaki, H. Fukuzawa, M. Hase, T. Hayakawa, K. Hiramatsu, Y. Hoshino, M. Inaba, T. Ito, H. Karakawa, Y. Kasai, K. Koizumi, S. Lee, C. Lei, M. Li, T. Maeno, S. Matsusaka, D. Murakami, A. Nakagawa, Y. Oguchi, M. Oikawa, T. Ota, K. Shiba, H. Shintaku, Y. Shirasaki, K. Suga, Y. Suzuki, N. Suzuki, Y. Tanaka, H. Tezuka, C. Toyokawa, Y. Yalikun, M. Yamada, M. Yamagishi, T. Yamano, A. Yasumoto, Y. Yatomi, M. Yazawa, D. Di Carlo, Y. Hosokawa, S. Uemura, Y. Ozeki and K. Goda, <span class="italic">Cell</span>, 2018, <span class="bold">175</span>, 266–276 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.cell.2018.08.028" title="DOI Link to resource 10.1016/j.cell.2018.08.028">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2ltr3I" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30166209%5Buid%5D" title="PubMed Link to resource 30166209">PubMed</a>.</span></li>
      <li><span id="cit54">A. F. Chrimes, K. Khoshmanesh, P. R. Stoddart, A. Mitchell and K. Kalantar-Zadeh, <span class="italic">Chem. Soc. Rev.</span>, 2013, <span class="bold">42</span>, 588–596 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C3CS35515B&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C3CS35515B">RSC</a>.</span></li>
      <li><span id="cit55">M. Uddin, Y. Wang and M. Woodbury-Smith, <span class="italic">NPJ Digit. Med.</span>, 2019, <span class="bold">2</span>, 112 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41746-019-0191-0" title="DOI Link to resource 10.1038/s41746-019-0191-0">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31799421%5Buid%5D" title="PubMed Link to resource 31799421">PubMed</a>.</span></li>
      <li><span id="cit56">S. Y. Wang, S. Pershing and A. Y. Lee, <span class="italic">Curr. Opin. Ophthalmol.</span>, 2020, <span class="bold">31</span>, 318–323 <a target="_blank" class="DOILink" href="https://doi.org/10.1097/ICU.0000000000000676" title="DOI Link to resource 10.1097/ICU.0000000000000676">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32657996%5Buid%5D" title="PubMed Link to resource 32657996">PubMed</a>.</span></li>
      <li><span id="cit57">N. L. Bragazzi, H. Dai, G. Damiani, M. Behzadifar, M. Martini and J. Wu, <span class="italic">Int. J. Environ. Res. Public Health</span>, 2020, <span class="bold">17</span>, 3176 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/ijerph17093176" title="DOI Link to resource 10.3390/ijerph17093176">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhvVKqtbfJ" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32370204%5Buid%5D" title="PubMed Link to resource 32370204">PubMed</a>.</span></li>
      <li><span id="cit58">A. F. de Almeida, R. Moreira and T. Rodrigues, <span class="italic">Nat. Rev. Chem.</span>, 2019, <span class="bold">3</span>, 589–604 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41570-019-0124-0" title="DOI Link to resource 10.1038/s41570-019-0124-0">CrossRef</a>.</span></li>
      <li><span id="cit59">S. M. Moghimi, A. C. Hunter and J. C. Murray, <span class="italic">FASEB J.</span>, 2004, <span class="bold">19</span>, 311–330 <a target="_blank" class="DOILink" href="https://doi.org/10.1096/fj.04-2747rev" title="DOI Link to resource 10.1096/fj.04-2747rev">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=15746175%5Buid%5D" title="PubMed Link to resource 15746175">PubMed</a>.</span></li>
      <li><span id="cit60">E. H. Chang, J. B. Harford, M. A. W. Eaton, P. M. Boisseau, A. Dube, R. Hayeshi, H. Swai and D. S. Lee, <span class="italic">Biochem. Biophys. Res. Commun.</span>, 2015, <span class="bold">468</span>, 511–517 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bbrc.2015.10.136" title="DOI Link to resource 10.1016/j.bbrc.2015.10.136">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgtL7K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26518648%5Buid%5D" title="PubMed Link to resource 26518648">PubMed</a>.</span></li>
      <li><span id="cit61">B. Pelaz, C. Alexiou, R. A. Alvarez-Puebla, F. Alves, A. M. Andrews, S. Ashraf, L. P. Balogh, L. Ballerini, A. Bestetti, C. Brendel, S. Bosi, M. Carril, W. C. W. Chan, C. Chen, X. Chen, X. Chen, Z. Cheng, D. Cui, J. Du, C. Dullin, A. Escudero, N. Feliu, M. Gao, M. George, Y. Gogotsi, A. Grünweller, Z. Gu, N. J. Halas, N. Hampp, R. K. Hartmann, M. C. Hersam, P. Hunziker, J. Jian, X. Jiang, P. Jungebluth, P. Kadhiresan, K. Kataoka, A. Khademhosseini, J. Kopeček, N. A. Kotov, H. F. Krug, D. S. Lee, C. Lehr, K. W. Leong, X. Liang, M. L. Lim, L. M. Liz-Marzán, X. Ma, P. Macchiarini, H. Meng, H. Möhwald, P. Mulvaney, A. E. Nel, S. Nie, P. Nordlander, T. Okano, J. Oliveira, T. H. Park, R. M. Penner, M. Prato, V. Puntes, V. M. Rotello, A. Samarakoon, R. E. Schaak, Y. Shen, S. Sjöqvist, A. G. Skirtach, M. G. Soliman, M. M. Stevens, H. Sung, B. Z. Tang, R. Tietze, B. N. Udugama, J. S. Vanepps, T. Weil, P. S. Weiss, I. Willner, Y. Wu, L. Yang, Z. Yue, Q. Zhang, Q. Zhang, X. Zhang, Y. Zhao, X. Zhou and W. J. Parak, <span class="italic">ACS Nano</span>, 2017, <span class="bold">11</span>, 2313–2381 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.6b06040" title="DOI Link to resource 10.1021/acsnano.6b06040">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2sXktV2isLg%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28290206%5Buid%5D" title="PubMed Link to resource 28290206">PubMed</a>.</span></li>
      <li><span id="cit62">X. Ji, L. Ge, C. Liu, Z. Tang, Y. Xiao, W. Chen, Z. Lei, W. Gao, S. Blake, D. De, B. Shi, X. Zeng, N. Kong, X. Zhang and W. Tao, <span class="italic">Nat. Commun.</span>, 2021, <span class="bold">12</span>, 1124 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-021-21436-5" title="DOI Link to resource 10.1038/s41467-021-21436-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXkvFejsL8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33602928%5Buid%5D" title="PubMed Link to resource 33602928">PubMed</a>.</span></li>
      <li><span id="cit63">D. Gao, X. Guo, X. Zhang, S. Chen, Y. Wang, T. Chen, G. Huang, Y. Gao, Z. Tian and Z. Yang, <span class="italic">Mater. Today Bio</span>, 2020, <span class="bold">5</span>, 100035 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.mtbio.2019.100035" title="DOI Link to resource 10.1016/j.mtbio.2019.100035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADC%252BB383ntlOntw%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32211603%5Buid%5D" title="PubMed Link to resource 32211603">PubMed</a>.</span></li>
      <li><span id="cit64">M. Buttarello and M. Plebani, <span class="italic">Am. J. Clin. Pathol.</span>, 2008, <span class="bold">130</span>, 104–116 <a target="_blank" class="DOILink" href="https://doi.org/10.1309/EK3C7CTDKNVPXVTN" title="DOI Link to resource 10.1309/EK3C7CTDKNVPXVTN">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=18550479%5Buid%5D" title="PubMed Link to resource 18550479">PubMed</a>.</span></li>
      <li><span id="cit65">L. A. Bertschi, <span class="italic">Am. J. Nurs.</span>, 2021, <span class="bold">121</span>, 38–45 <a target="_blank" class="DOILink" href="https://doi.org/10.1097/01.NAJ.0000731656.00453.12" title="DOI Link to resource 10.1097/01.NAJ.0000731656.00453.12">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33350695%5Buid%5D" title="PubMed Link to resource 33350695">PubMed</a>.</span></li>
      <li><span id="cit66">L. R. Dixon, <span class="italic">J. Perinat. Neonatal. Nurs.</span>, 1997, <span class="bold">11</span>, 1–18 <a target="_blank" class="DOILink" href="https://doi.org/10.1097/00005237-199712000-00003" title="DOI Link to resource 10.1097/00005237-199712000-00003">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ASTN%3A280%3ADyaK1c7htlSiug%253D%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=9451188%5Buid%5D" title="PubMed Link to resource 9451188">PubMed</a>.</span></li>
      <li><span id="cit67">C. P. Hornik, D. K. Benjamin, K. C. Becker, D. K. Benjamin, J. Li, R. H. Clark, M. Cohen-Wolkowiez and P. B. Smith, <span class="italic">Pediatr. Infect. Dis. J.</span>, 2012, <span class="bold">31</span>, 803–807 <a target="_blank" class="DOILink" href="https://doi.org/10.1097/INF.0b013e31825691e4" title="DOI Link to resource 10.1097/INF.0b013e31825691e4">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22531232%5Buid%5D" title="PubMed Link to resource 22531232">PubMed</a>.</span></li>
      <li><span id="cit68">X. Huang, Y. Jiang, X. Liu, H. Xu, Z. Han, H. Rong, H. Yang, M. Yan and H. Yu, <span class="italic">Sensors</span>, 2016, <span class="bold">16</span>, 1836 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/s16111836" title="DOI Link to resource 10.3390/s16111836">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27827837%5Buid%5D" title="PubMed Link to resource 27827837">PubMed</a>.</span></li>
      <li><span id="cit69">Y. Liao, N. Yu, D. Tian, S. Li and Z. Li, <span class="italic">Sensors</span>, 2019, <span class="bold">19</span>, 5103 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/s19235103" title="DOI Link to resource 10.3390/s19235103">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31766471%5Buid%5D" title="PubMed Link to resource 31766471">PubMed</a>.</span></li>
      <li><span id="cit70">M. Wang, L. S. Ong, J. Dauwels and H. H. Asada, <span class="italic">J. Med. Imaging</span>, 2018, <span class="bold">5</span>, 1 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=J.%20Med.%20Imaging%5BJour%5D%20AND%205%5Bvolume%5D%20AND%201%5Bpage%5D%20and%202018%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit71">J. Lamanna, E. Y. Scott, H. S. Edwards, M. D. Chamberlain, M. D. M. Dryden, J. Peng, B. Mair, A. Lee, C. Chan, A. A. Sklavounos, A. Heffernan, F. Abbas, C. Lam, M. E. Olson, J. Moffat and A. R. Wheeler, <span class="italic">Nat. Commun.</span>, 2020, <span class="bold">11</span>, 5632 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-020-19394-5" title="DOI Link to resource 10.1038/s41467-020-19394-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitlClsr3K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33177493%5Buid%5D" title="PubMed Link to resource 33177493">PubMed</a>.</span></li>
      <li><span id="cit72">F. Bray, J. Ferlay, I. Soerjomataram, R. L. Siegel, L. A. Torre and A. Jemal, <span class="italic">CA-Cancer J. Clin.</span>, 2020, <span class="bold">70</span>, 313 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=CA-Cancer%20J.%20Clin.%5BJour%5D%20AND%2070%5Bvolume%5D%20AND%20313%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit73">V. K. Jagannadh, G. Gopakumar, G. R. K. Sai Subrahmanyam and S. S. Gorthi, <span class="italic">Med. Biol. Eng. Comput.</span>, 2017, <span class="bold">55</span>, 711–718 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s11517-016-1549-y" title="DOI Link to resource 10.1007/s11517-016-1549-y">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27447709%5Buid%5D" title="PubMed Link to resource 27447709">PubMed</a>.</span></li>
      <li><span id="cit74">K. D. Nyberg, S. L. Bruce, A. V. Nguyen, C. K. Chan, N. K. Gill, T. Kim, E. K. Sloan and A. C. Rowat, <span class="italic">Integr. Biol.</span>, 2018, <span class="bold">10</span>, 218–231 <a target="_blank" class="DOILink" href="https://doi.org/10.1039/C7IB00222J" title="DOI Link to resource 10.1039/C7IB00222J">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1cXltlSrtr8%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=29589844%5Buid%5D" title="PubMed Link to resource 29589844">PubMed</a>.</span></li>
      <li><span id="cit75">M. S. Manak, J. S. Varsanik, B. J. Hogan, M. J. Whitfield, W. R. Su, N. Joshi, N. Steinke, A. Min, D. Berger, R. J. Saphirstein, G. Dixit, T. Meyyappan, H. Chu, K. B. Knopf, D. M. Albala, G. R. Sant and A. C. Chander, <span class="italic">Nat. Biomed. Eng.</span>, 2018, <span class="bold">2</span>, 761–772 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41551-018-0285-z" title="DOI Link to resource 10.1038/s41551-018-0285-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtFegu7nL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30854249%5Buid%5D" title="PubMed Link to resource 30854249">PubMed</a>.</span></li>
      <li><span id="cit76">C. R. Oliver, M. A. Altemus, T. M. Westerhof, H. Cheriyan, X. Cheng, M. Dziubinski, Z. Wu, J. Yates, A. Morikawa, J. Heth, M. G. Castro, B. M. Leung, S. Takayama and S. D. Merajver, <span class="italic">Lab Chip</span>, 2019, <span class="bold">19</span>, 1162–1173 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8LC01387J&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8LC01387J">RSC</a>.</span></li>
      <li><span id="cit77">
          J. Sun, L. Wang, Q. Liu and X. Su, <span class="italic">Optics in Health Care and Biomedical Optics IX</span>,  2020, 11190 <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/sites/entrez?orig_db=PubMed&amp;db=pubmed&amp;cmd=Search&amp;term=Optics%20in%20Health%20Care%20and%20Biomedical%20Optics%20IX%5BJour%5D%20AND%20%5Bvolume%5D%20AND%20%5Bpage%5D%20and%202020%5Bpdat%5D" title="Search PubMed for this citation">Search PubMed</a>.</span></li>
      <li><span id="cit78">F. Ellett, J. Jorgensen, A. L. Marand, Y. M. Liu, M. M. Martinez, V. Sein, K. L. Butler, J. Lee and D. Irimia, <span class="italic">Nat. Biomed. Eng.</span>, 2018, <span class="bold">2</span>, 207–214 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41551-018-0208-z" title="DOI Link to resource 10.1038/s41551-018-0208-z">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhtFensr%252FF" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=30283724%5Buid%5D" title="PubMed Link to resource 30283724">PubMed</a>.</span></li>
      <li><span id="cit79">X. Huang, J. Guo, X. Wang, M. Yan, Y. Kang and H. Yu, <span class="italic">PLoS One</span>, 2014, <span class="bold">9</span>, e104539 <a target="_blank" class="DOILink" href="https://doi.org/10.1371/journal.pone.0104539" title="DOI Link to resource 10.1371/journal.pone.0104539">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25111497%5Buid%5D" title="PubMed Link to resource 25111497">PubMed</a>.</span></li>
      <li><span id="cit80">S. Heynen-Genel, L. Pache, S. K. Chanda and J. Rosen, <span class="italic">Expert Opin. Drug Discovery</span>, 2012, <span class="bold">7</span>, 955–968 <a target="_blank" class="DOILink" href="https://doi.org/10.1517/17460441.2012.711311" title="DOI Link to resource 10.1517/17460441.2012.711311">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsbvN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=22860749%5Buid%5D" title="PubMed Link to resource 22860749">PubMed</a>.</span></li>
      <li><span id="cit81">M. M. Usaj, E. B. Styles, A. J. Verster, H. Friesen, C. Boone and B. J. Andrews, <span class="italic">Trends Cell Biol.</span>, 2016, <span class="bold">26</span>, 598–611 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.tcb.2016.03.008" title="DOI Link to resource 10.1016/j.tcb.2016.03.008">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27118708%5Buid%5D" title="PubMed Link to resource 27118708">PubMed</a>.</span></li>
      <li><span id="cit82">H. Kobayashi, C. Lei, Y. Wu, A. Mao, Y. Jiang, B. Guo, Y. Ozeki and K. Goda, <span class="italic">Sci. Rep.</span>, 2017, <span class="bold">7</span>, 955–968 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-017-01154-z" title="DOI Link to resource 10.1038/s41598-017-01154-z">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=28424509%5Buid%5D" title="PubMed Link to resource 28424509">PubMed</a>.</span></li>
      <li><span id="cit83">H. Kobayashi, C. Lei, Y. Wu, C. Huang, A. Yasumoto, M. Jona, W. Li, Y. Wu, Y. Yalikun, Y. Jiang, B. Guo, C. Sun, Y. Tanaka, M. Yamada, Y. Yatomi and K. Goda, <span class="italic">Lab Chip</span>, 2019, <span class="bold">19</span>, 2688–2698 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8LC01370E&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8LC01370E">RSC</a>.</span></li>
      <li><span id="cit84">Y. Wu, Y. Zhou, C. Huang, H. Kobayashi, S. Yan, Y. Ozeki, Y. Wu, C. Sun, A. Yasumoto, Y. Yatomi, C. Lei and K. Goda, <span class="italic">Opt. Express</span>, 2020, <span class="bold">28</span>, 519 <a target="_blank" class="DOILink" href="https://doi.org/10.1364/OE.380679" title="DOI Link to resource 10.1364/OE.380679">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXht12htLzP" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32118978%5Buid%5D" title="PubMed Link to resource 32118978">PubMed</a>.</span></li>
      <li><span id="cit85">D. Gao, T. Chen, Y. Han, S. Chen, Y. Wang, X. Guo, H. Wang, X. Chen, M. Guo, Y. S. Zhang, G. Hong, X. Zhang, Z. Tian and Z. Yang, <span class="italic">Nano-Micro Lett.</span>, 2021, <span class="bold">13</span>, 99 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s40820-021-00616-4" title="DOI Link to resource 10.1007/s40820-021-00616-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhsFGktbrL" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34138317%5Buid%5D" title="PubMed Link to resource 34138317">PubMed</a>.</span></li>
      <li><span id="cit86">Z. Yang, D. Gao, X. Guo, L. Jin, J. Zhang, Y. Wang, S. Chen, X. Zheng, L. Zeng, M. Guo, X. Zhang and Z. Tian, <span class="italic">ACS Nano</span>, 2020, <span class="bold">14</span>, 17442–17457 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.0c07721" title="DOI Link to resource 10.1021/acsnano.0c07721">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXit1Kgsr3E" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33166111%5Buid%5D" title="PubMed Link to resource 33166111">PubMed</a>.</span></li>
      <li><span id="cit87">Y. Wang, D. Gao, Y. Liu, X. Guo, S. Chen, L. Zeng, J. Ma, X. Zhang, Z. Tian and Z. Yang, <span class="italic">Bioact. Mater.</span>, 2020, <span class="bold">6</span>, 1513–1527 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.bioactmat.2020.11.016" title="DOI Link to resource 10.1016/j.bioactmat.2020.11.016">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33294730%5Buid%5D" title="PubMed Link to resource 33294730">PubMed</a>.</span></li>
      <li><span id="cit88">C. Liu, S. Sun, Q. Feng, G. Wu, Y. Wu, N. Kong, Z. Yu, J. Yao, X. Zhang, W. Chen, Z. Tang, Y. Xiao, X. Huang, A. Lv, C. Yao, H. Cheng, A. Wu, T. Xie and W. Tao, <span class="italic">Adv. Mater.</span>, 2021, <span class="bold">33</span>, 2102054 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.202102054" title="DOI Link to resource 10.1002/adma.202102054">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhs1Khtb7K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34309925%5Buid%5D" title="PubMed Link to resource 34309925">PubMed</a>.</span></li>
      <li><span id="cit89">J. M. Ayuso, M. Virumbrales-Muñoz, A. Lacueva, P. M. Lanuza, E. Checa-Chavarria, P. Botella, E. Fernández, M. Doblare, S. J. Allison, R. M. Phillips, J. Pardo, L. J. Fernandez and I. Ochoa, <span class="italic">Sci. Rep.</span>, 2016, <span class="bold">6</span>, 36086 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/srep36086" title="DOI Link to resource 10.1038/srep36086">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitL3I" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=27796335%5Buid%5D" title="PubMed Link to resource 27796335">PubMed</a>.</span></li>
      <li><span id="cit90">N. Moore, D. Doty, M. Zietstorff, I. Kariv, L. Y. Moy, A. Gimbel, J. R. Chevillet, N. Lowry, J. Santos, V. Mott, L. Kratchman, T. Lau, G. Addona, H. Chen and J. T. Borenstein, <span class="italic">Lab Chip</span>, 2018, <span class="bold">18</span>, 1844–1858 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C8LC00256H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C8LC00256H">RSC</a>.</span></li>
      <li><span id="cit91">M. S. Chowdhury, X. Zhang, L. Amini, P. Dey, A. K. Singh, A. Faghani, M. Schmueck-Henneresse and R. Haag, <span class="italic">Nano-Micro Lett.</span>, 2021, <span class="bold">13</span>, 147 <a target="_blank" class="DOILink" href="https://doi.org/10.1007/s40820-021-00663-x" title="DOI Link to resource 10.1007/s40820-021-00663-x">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXitFGmu7fE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34146147%5Buid%5D" title="PubMed Link to resource 34146147">PubMed</a>.</span></li>
      <li><span id="cit92">X. Chen, Y. Chen, L. Zou, X. Zhang, Y. Dong, J. Tang, D. J. Mcclements and W. Liu, <span class="italic">J. Agric. Food Chem.</span>, 2019, <span class="bold">67</span>, 6574–6584 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.jafc.9b02028" title="DOI Link to resource 10.1021/acs.jafc.9b02028">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXpvFamt7s%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31117503%5Buid%5D" title="PubMed Link to resource 31117503">PubMed</a>.</span></li>
      <li><span id="cit93">A. R. Kirmani, J. M. Luther, M. Abolhasani and A. Amassian, <span class="italic">ACS Energy Lett.</span>, 2020, <span class="bold">5</span>, 3069–3100 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsenergylett.0c01453" title="DOI Link to resource 10.1021/acsenergylett.0c01453">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhs1KltrzO" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit94">J. Puigmartí-Luis, M. Rubio-Martínez, U. Hartfelder, I. Imaz, D. Maspoch and P. S. Dittrich, <span class="italic">J. Am. Chem. Soc.</span>, 2011, <span class="bold">133</span>, 4216–4219 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/ja110834j" title="DOI Link to resource 10.1021/ja110834j">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=21384917%5Buid%5D" title="PubMed Link to resource 21384917">PubMed</a>.</span></li>
      <li><span id="cit95">D. Liu, H. Zhang, E. Mäkilä, J. Fan, B. Herranz-Blanco, C. Wang, R. Rosa, A. J. Ribeiro, J. Salonen, J. Hirvonen and H. A. Santos, <span class="italic">Biomaterials</span>, 2015, <span class="bold">39</span>, 249–259 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.biomaterials.2014.10.079" title="DOI Link to resource 10.1016/j.biomaterials.2014.10.079">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC2cXhvFWitrnE" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=25468375%5Buid%5D" title="PubMed Link to resource 25468375">PubMed</a>.</span></li>
      <li><span id="cit96">C. L. C. Smith, D. K. C. Wu, M. W. Lee, C. Monat, S. Tomljenovic-Hanic, C. Grillet, B. J. Eggleton, D. Freeman, Y. Ruan, S. Madden, B. Luther-Davies, H. Giessen and Y. Lee, <span class="italic">Appl. Phys. Lett.</span>, 2007, <span class="bold">91</span>, 121103 <a target="_blank" class="DOILink" href="https://doi.org/10.1063/1.2785988" title="DOI Link to resource 10.1063/1.2785988">CrossRef</a>.</span></li>
      <li><span id="cit97">J. Wu, S. Yadavali, D. Lee and D. A. Issadore, <span class="italic">Appl. Phys. Rev.</span>, 2021, <span class="bold">8</span>, 31304 <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhs1ejt7rF" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit98">L. Wang, L. R. Karadaghi, R. L. Brutchey and N. Malmstadt, <span class="italic">Chem. Commun.</span>, 2020, <span class="bold">56</span>, 3745–3748 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0CC00064G&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0CC00064G">RSC</a>.</span></li>
      <li><span id="cit99">I. Lignos, S. Stavrakis, G. Nedelcu, L. Protesescu, A. J. Demello and M. V. Kovalenko, <span class="italic">Nano Lett.</span>, 2016, <span class="bold">16</span>, 1869–1877 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acs.nanolett.5b04981" title="DOI Link to resource 10.1021/acs.nanolett.5b04981">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC28XhvV2mtro%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=26836149%5Buid%5D" title="PubMed Link to resource 26836149">PubMed</a>.</span></li>
      <li><span id="cit100">R. Huang, X. Chen, Y. Dong, X. Zhang, Y. Wei, Z. Yang, W. Li, Y. Guo, J. Liu, Z. Yang, H. Wang and L. Jin, <span class="italic">ACS Appl. Bio Mater.</span>, 2020, <span class="bold">3</span>, 2125–2131 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsabm.0c00007" title="DOI Link to resource 10.1021/acsabm.0c00007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXlsFaisLY%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit101">J. P. Mcmullen and K. F. Jensen, <span class="italic">Org. Process Res. Dev.</span>, 2010, <span class="bold">14</span>, 1169–1176 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/op100123e" title="DOI Link to resource 10.1021/op100123e">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCns7fM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit102">Y. Orimoto, K. Watanabe, K. Yamashita, M. Uehara, H. Nakamura, T. Furuya and H. Maeda, <span class="italic">J. Phys. Chem. C</span>, 2012, <span class="bold">116</span>, 17885–17896 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/jp3031122" title="DOI Link to resource 10.1021/jp3031122">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC38XhtFWiurfK" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit103">A. Sokhansanj, S. M. Abdoli and M. Zabihi, <span class="italic">Process Saf. Environ.</span>, 2020, <span class="bold">141</span>, 321–332 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.psep.2020.05.035" title="DOI Link to resource 10.1016/j.psep.2020.05.035">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtFOitbrM" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit104">D. Salley, G. Keenan, J. Grizou, A. Sharma, S. Martín and L. Cronin, <span class="italic">Nat. Commun.</span>, 2020, <span class="bold">11</span>, 2771 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-020-16501-4" title="DOI Link to resource 10.1038/s41467-020-16501-4">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXhtV2ktrfI" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32488034%5Buid%5D" title="PubMed Link to resource 32488034">PubMed</a>.</span></li>
      <li><span id="cit105">H. Tao, T. Wu, S. Kheiri, M. Aldeghi, A. Aspuru-Guzik and E. Kumacheva, <span class="italic">Adv. Funct. Mater.</span>, 2021, 2106725 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adfm.202106725" title="DOI Link to resource 10.1002/adfm.202106725">CrossRef</a>.</span></li>
      <li><span id="cit106">O. Voznyy, L. Levina, J. Z. Fan, M. Askerka, A. Jain, M. Choi, O. Ouellette, P. Todorović, L. K. Sagar and E. H. Sargent, <span class="italic">ACS Nano</span>, 2019, <span class="bold">13</span>, 11122–11128 <a target="_blank" class="DOILink" href="https://doi.org/10.1021/acsnano.9b03864" title="DOI Link to resource 10.1021/acsnano.9b03864">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BC1MXhvVWksrnN" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=31539477%5Buid%5D" title="PubMed Link to resource 31539477">PubMed</a>.</span></li>
      <li><span id="cit107">K. Abdel-Latif, R. W. Epps, F. Bateni, S. Han, K. G. Reyes and M. Abolhasani, <span class="italic">Adv. Intell. Syst.</span>, 2021, <span class="bold">3</span>, 2000245 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/aisy.202000245" title="DOI Link to resource 10.1002/aisy.202000245">CrossRef</a>.</span></li>
      <li><span id="cit108">H. S. M. Ali, N. Blagden, P. York, A. Amani and T. Brook, <span class="italic">Eur. J. Pharm. Sci.</span>, 2009, <span class="bold">37</span>, 514–522 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.ejps.2009.04.007" title="DOI Link to resource 10.1016/j.ejps.2009.04.007">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BD1MXms1Oktr0%253D" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19406230%5Buid%5D" title="PubMed Link to resource 19406230">PubMed</a>.</span></li>
      <li><span id="cit109">S. A. Damiati, D. Rossi, H. N. Joensson and S. Damiati, <span class="italic">Sci. Rep.</span>, 2020, <span class="bold">10</span>, 19517 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41598-020-76477-5" title="DOI Link to resource 10.1038/s41598-020-76477-5">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXitlClsrfK" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33177577%5Buid%5D" title="PubMed Link to resource 33177577">PubMed</a>.</span></li>
      <li><span id="cit110">M. Flore, R. Zekun, H. Tan, W. W. Kuan, Z. Fang, X. Jiaxun, I. P. S. Tian, J. Senthilnath, M. Zackaria, B. Daniil, H. Kedar, K. Saif, B. Tonio, L. Qianxiao and W. Xiaonan, <span class="italic">npj Comput. Mater.</span>, 2021, <span class="bold">7</span>, 1–10 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41524-020-00473-6" title="DOI Link to resource 10.1038/s41524-020-00473-6">CrossRef</a>.</span></li>
      <li><span id="cit111">J. A. Fine, A. A. Rajasekar, K. P. Jethava and G. Chopra, <span class="italic">Chem. Sci.</span>, 2020, <span class="bold">11</span>, 4618–4630 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=C9SC06240H&amp;newsite=1" title="Link to RSC resource DOI:10.1039/C9SC06240H">RSC</a>.</span></li>
      <li><span id="cit112">J. R. Mejía-Salazar, K. R. Cruz, E. M. M. Vásques and O. N. D. Oliveira Jr., <span class="italic">Sensors</span>, 2020, <span class="bold">20</span>, 1951 <a target="_blank" class="DOILink" href="https://doi.org/10.3390/s20071951" title="DOI Link to resource 10.3390/s20071951">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=32244343%5Buid%5D" title="PubMed Link to resource 32244343">PubMed</a>.</span></li>
      <li><span id="cit113">F. Grisoni, B. Huisman, A. L. Button, M. Moret, K. Atz, D. Merk and G. Schneider, <span class="italic">Sci. Adv.</span>, 2021, <span class="bold">7</span>, eabg3338 <a target="_blank" class="DOILink" href="https://doi.org/10.1126/sciadv.abg3338" title="DOI Link to resource 10.1126/sciadv.abg3338">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvVShtr3K" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34117066%5Buid%5D" title="PubMed Link to resource 34117066">PubMed</a>.</span></li>
      <li><span id="cit114">E. A. Galan, H. Zhao, X. Wang, Q. Dai, W. T. S. Huck and S. Ma, <span class="italic">Matter</span>, 2020, <span class="bold">3</span>, 1893–1922 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.matt.2020.08.034" title="DOI Link to resource 10.1016/j.matt.2020.08.034">CrossRef</a>.</span></li>
      <li><span id="cit115">K. Jiang, D. S. Jokhun and C. T. Lim, <span class="italic">J. Biomech.</span>, 2021, <span class="bold">117</span>, 110235 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.jbiomech.2021.110235" title="DOI Link to resource 10.1016/j.jbiomech.2021.110235">CrossRef</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33486262%5Buid%5D" title="PubMed Link to resource 33486262">PubMed</a>.</span></li>
      <li><span id="cit116">J. Liu, Z. Xu, Y. Shan and X. Huang, <span class="italic">Analyst</span>, 2021, <span class="bold">146</span>, 1529–1537 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0AN01945C&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0AN01945C">RSC</a>.</span></li>
      <li><span id="cit117">J. Yang, X. Zhang, C. Liu, Z. Wang, L. Deng, C. Feng, W. Tao, X. Xua and W. Cui, <span class="italic">Prog. Mater. Sci.</span>, 2021, <span class="bold">118</span>, 100768 <a target="_blank" class="DOILink" href="https://doi.org/10.1016/j.pmatsci.2020.100768" title="DOI Link to resource 10.1016/j.pmatsci.2020.100768">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXks1Clsrw%253D" title="Link to resource in CAS">CAS</a>.</span></li>
      <li><span id="cit118">W. Xiao, L. Xin, R. Cao, X. Wu, R. Tian, L. Che, L. Sun, P. Ferraro and F. Pan, <span class="italic">Lab Chip</span>, 2021, <span class="bold">21</span>, 1385–1394 <a target="_blank" class="RSCLink" href="http://xlink.rsc.org/?doi=D0LC01113D&amp;newsite=1" title="Link to RSC resource DOI:10.1039/D0LC01113D">RSC</a>.</span></li>
      <li><span id="cit119">A. A. Volk, R. W. Epps and M. Abolhasani, <span class="italic">Adv. Mater.</span>, 2021, <span class="bold">33</span>, 2004495 <a target="_blank" class="DOILink" href="https://doi.org/10.1002/adma.202004495" title="DOI Link to resource 10.1002/adma.202004495">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3cXisVyisbbM" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=33289177%5Buid%5D" title="PubMed Link to resource 33289177">PubMed</a>.</span></li>
      <li><span id="cit120">B. A. Rizkin, A. S. Shkolnik, N. J. Ferraro and R. L. Hartman, <span class="italic">Nat. Mach. Intell.</span>, 2020, <span class="bold">2</span>, 200–209 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s42256-020-0166-5" title="DOI Link to resource 10.1038/s42256-020-0166-5">CrossRef</a>.</span></li>
      <li><span id="cit121">Y. Yang, X. Wei, N. Zhang, J. Zheng, X. Chen, Q. Wen, X. Luo, C. Lee, X. Liu, X. Zhang, J. Chen, C. Tao, W. Zhang and X. Fan, <span class="italic">Nat. Commun.</span>, 2021, <span class="bold">12</span>, 4876 <a target="_blank" class="DOILink" href="https://doi.org/10.1038/s41467-021-25075-8" title="DOI Link to resource 10.1038/s41467-021-25075-8">CrossRef</a> <a target="_blank" class="COILink" href="/en/content/coiresolver?coi=1%3ACAS%3A528%3ADC%252BB3MXhvVSlt77P" title="Link to resource in CAS">CAS</a> <a target="_blank" class="PMedLink" href="http://www.ncbi.nlm.nih.gov/pubmed/?term=34385436%5Buid%5D" title="PubMed Link to resource 34385436">PubMed</a>.</span></li>
      <li><span id="cit122">Y. Li, B. Liu and X. Zhang, <span class="italic">Mater. Today</span>, 2021<small> DOI:<a class="DOILink" href="https://doi.org/10.1016/j.mattod.2021.10.008" TARGET="_BLANK" title="DOI Link to 10.1016/j.mattod.2021.10.008">10.1016/j.mattod.2021.10.008</a></small>.</span></li>
    </ol>
  <hr/><table><tr><td><h3>Footnote</h3></td></tr><tr><td><span class="sup_ref">† <span id="fn1">These authors contributed equally to this work.</span></span></td></tr></table><table><tr><td><hr/></td></tr><tr><td><b>This journal is © The Royal Society of Chemistry 2021</b></td></tr></table></div></div></div></body><script src="/content/scripts/CrossMarkIE.js"> </script><SaxonLicenceTest result="pass" message="Licenced Enterprise Edition [ EE 9.3.0.4 ]"/></html>